//FILE GENERATED BY build.js on Tue Sep 13 2022 16:43:47 GMT+0200 (Central European Summer Time), DO NOT MODIFY!
const proteins = {"10640885":{"id":10640885,"externalId":"192222.Cj0001","name":"dnaA","hasFamily":true,"annotation":"Plays an important role in the initiation and regulation of chromosomal replication. Binds to the origin of replication; it binds specifically double-stranded DNA at a 9 bp consensus (dnaA box): 5'- TTATC[CA]A[CA]A-3'. DnaA binds to ATP and to acidic phospholipids (By similarity)","uniprotId":"DNAA_CAMJE"},"10640886":{"id":10640886,"externalId":"192222.Cj0002","name":"dnaN","hasFamily":true,"annotation":"Confers DNA tethering and processivity to DNA polymerases and other proteins. Acts as a clamp, forming a ring around DNA (a reaction catalyzed by the clamp-loading complex) which diffuses in an ATP- independent manner freely and bidirectionally along dsDNA. Initially characterized for its ability to contact the catalytic subunit of DNA polymerase III (Pol III), a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria; Pol III exhibits 3'-5' exonuclease proofreading activity. The beta chain is required for initiation of replication as well as for proces [...] ","uniprotId":"Q0PCC3_CAMJE"},"10640887":{"id":10640887,"externalId":"192222.Cj0003","name":"gyrB","hasFamily":true,"annotation":"A type II topoisomerase that negatively supercoils closed circular double-stranded (ds) DNA in an ATP-dependent manner to modulate DNA topology and maintain chromosomes in an underwound state. Negative supercoiling favors strand separation, and DNA replication, transcription, recombination and repair, all of which involve strand separation. Also able to catalyze the interconversion of other topological isomers of dsDNA rings, including catenanes and knotted rings. Type II topoisomerases break and join 2 DNA strands simultaneously in an ATP-dependent manner","uniprotId":"GYRB_CAMJE"},"10640888":{"id":10640888,"externalId":"192222.Cj0004c","name":"Cj0004c","hasFamily":true,"annotation":"Monohaem cytochrome C; Original (2000) note: Cj0004c, possible periplasmic protein, len: 113 aa. No Hp match. Contains possible N-terminal signal sequence; Updated (2006) note: NCTC11168 Cj0004c encodes a monohaem cytochrome C (PMID:15632441). Together with Cj0005 (molybdopterin oxidoreductase), they are homologous to sulphite:cytochrome C oxidoreductase (SOR) of Starkeya novella (PMID:11557133). Cj0004 mutants showed wild-type rates of formate-dependent respiration but were unable to respire with sulphite or metabisulphite as electron donors. Periplasmic extracts of wild-type NCTC1116 [...] ","uniprotId":"Q0PCC1_CAMJE"},"10640889":{"id":10640889,"externalId":"192222.Cj0005c","name":"Cj0005c","hasFamily":true,"annotation":"Original (2000) note: Cj0005c, possible molybdenum containing oxidoreductase, len: 412 aa; similar to many eukaryotic oxidoreductases e.g. SUOX_HUMAN sulfite oxidase precursor (488 aa), fasta scores; opt: 543 z-score: 338.2 E(): 1.4e-11, 29.8% identity in 372 aa overlap and NIA_PETHY nitrate reductase from Petunia hybrida (909 aa),fasta scores; opt: 281 z-score: 312.5 E(): 3.8e-10, 28.9% identity in 395 aa overlap. No Hp match. Contains Pfam match to entry PF00174 oxidored_molyb, Oxidoreductase molybdopterin binding domain, score 45.20, E-value 1e-12; Updated (2006) note: Pfam domain P [...] ","uniprotId":"Q0PCC0_CAMJE"},"10640890":{"id":10640890,"externalId":"192222.Cj0006","name":"Cj0006","hasFamily":true,"annotation":"Putative Na+/H+ antiporter family protein; Original (2000) note: Cj0006, probable integral membrane protein, len: 435 aa; similar to e.g. Y325_HAEIN hypothetical protein HI0325 (450 aa), fasta scores; opt: 757 z-score: 1654.9 E(): 0, 44.0% identity in 450 aa overlap. 47.2% identity to HP0758; Updated (2006) note: Pfam domain PF03553 Na+/H+ antiporter family identified within CDS. Product modified to more specific family member due to motif match. Also,eleven probable transmembrane helices predicted by TMHMM2.0 giving further support to product function. No specific characterisation wit [...] ","uniprotId":"Q0PCB9_CAMJE"},"10640891":{"id":10640891,"externalId":"192222.Cj0007","name":"gltB","hasFamily":true,"annotation":"Original (2000) note: Cj0007, gltB, probable glutamate synthase (NADPH) large subunit, len: 1496 aa; similar to many e.g. GLTB_ECOLI glutamate synthase (NADPH) large chain (1517 aa), fasta scores; opt: 3846 z-score: 3828.3 E(): 0, 42.5% identity in 1488 aa overlap. No Hp match; Updated (2006) note: Pfam domains PF00310 Glutamine amidotransferases class-II, PF04898 Glutamate synthase central domain, PF01645 Conserved region in glutamate synthase and PF01493 GXGXG motif were all identified within CDS. Further support given to product function. Characterised within Bacillus subtilis with  [...] ","uniprotId":"Q0PCB8_CAMJE"},"10640892":{"id":10640892,"externalId":"192222.Cj0008","name":"Cj0008","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0008, unknown, len: 583 aa; similar to Y686_METJA hypothetical protein MJ0686 (580 aa), fasta scores; opt: 171 z-score: 228.3 E(): 1.9e-05,22.8% identity in 609 aa overlap. 37.5% identity in N-term to HP0938, 30.0% identity in C-term to HP0937; Updated (2006) note: Pfam PF03235 Protein of unknown function (DUF262), was present within CDS. Conserved added to product function as motif was identified. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PCB7_CAMJE"},"10640893":{"id":10640893,"externalId":"192222.Cj0009","name":"gltD","hasFamily":true,"annotation":"Original (2000) note: Cj0009, gltD, probable glutamate synthase (NADPH) small subunit, len: 481 aa; similar to many e.g. GLTD_ECOLI glutamate synthase (NADPH) small chain (EC 1.4.1.13) (471 aa), fasta scores; opt: 807 z-score: 909.0 E(): 0, 33.0% identity in 482 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF00070 Pyridine nucleotide-disulphide oxidoreductase, PF07992 Pyridine nucleotide-disulphide oxidoreductase, PF03486 HI0933-like protein, PF01266 FAD dependent oxidoreductase, PF00890 FAD binding domain and PF01593 Flavin containing amine oxidoreductase were identified  [...] ","uniprotId":"Q0PCB6_CAMJE"},"10640894":{"id":10640894,"externalId":"192222.Cj0010c","name":"rnhB","hasFamily":true,"annotation":"Endonuclease that specifically degrades the RNA of RNA-DNA hybrids","uniprotId":"RNH2_CAMJE"},"10640895":{"id":10640895,"externalId":"192222.Cj0011c","name":"Cj0011c","hasFamily":true,"annotation":"Updated (2006) note: Characterisation work carried out within Bacillus subtilis, however, sequence alignment was only partial. Putative kept within product function; Original (2000) note: Cj0011c, possible non-specific DNA binding protein, len: 79 aa; similar to the C-terminus of CME1_BACSU comE operon protein 1 (comeA) (205 aa), fasta scores; opt: 223 z-score: 359.5 E(): 9e-13, 56.9% identity in 65 aa overlap. No Hp match. Contains Pfam match to entry PF00633 HHH,Helix-hairpin-helix motif., score 31.80, E-value 1.6e-05; PMID:7968523, PMID:7768800","uniprotId":"Q0PCB4_CAMJE"},"10640896":{"id":10640896,"externalId":"192222.Cj0012c","name":"rrc","hasFamily":true,"annotation":"Non-haem iron protein; Original (2000) note: Cj0012c, non-haem iron protein, len: 215 aa; has three possible iron-binding domains; C-terminus is similar to rubrerythrins e.g. RUBY_DESVH rubrerythrin (191 aa), fasta scores; opt: 454 z-score: 746.8 E(): 0, 39.5% identity in 190 aa overlap,with the iron-binding residues of both the rubredoxin-like center non-sulfur oxo-bridged di-iron center conserved. N-terminal 33 aa are simialar to the iron-bindingdomain of desulfoferrodoxins e.g. DESR_DESVH desulfoferrodoxin (125 aa), fasta scores; opt: 161 z-score: 275.8 E(): 4.2e-08,68.8% identity i [...] ","uniprotId":"Q0PCB3_CAMJE"},"10640897":{"id":10640897,"externalId":"192222.Cj0013","name":"ilvD","hasFamily":true,"annotation":"Belongs to the IlvD/Edd family","uniprotId":"ILVD_CAMJE"},"10640898":{"id":10640898,"externalId":"192222.Cj0014c","name":"Cj0014c","hasFamily":true,"annotation":"Original (2000) note: Cj0014c, probable integral membrane protein, len: 174 aa; similar to YAGU_ECOLI hypothetical 23.0 kd protein in intF-eaeH intergenic region (204 aa), fasta scores; opt: 402 z-score: 821.6 E(): 0, 45.1% identity in 184 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF07274 Protein of unknown function (DUF1440) identified within CDS. Also,three probable transmembrane helices identified by TMHMM2.0. Further support given to product function. No specific characterisation has been carried out yet. Thus,putative kept within product function. Functional classi [...] ","uniprotId":"Q0PCB1_CAMJE"},"10640899":{"id":10640899,"externalId":"192222.Cj0015c","name":"Cj0015c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0015c, unknown, len: 408 aa; 35.4% identity to HP0711; Updated (2006) note: Literature search identified paper (PMID:15647906) giving potential clues to product functionality. Functional classification - Conserved hypothetical proteins; PMID:15647906","uniprotId":"Q0PCB0_CAMJE"},"10640900":{"id":10640900,"externalId":"192222.Cj0016","name":"queC","hasFamily":true,"annotation":"Catalyzes the ATP-dependent conversion of 7-carboxy-7- deazaguanine (CDG) to 7-cyano-7-deazaguanine (preQ(0)). ","uniprotId":"QUEC_CAMJE"},"10640901":{"id":10640901,"externalId":"192222.Cj0017c","name":"dsbI","hasFamily":true,"annotation":"Disulphide bond formation protein; Original (2000) note: Cj0017c, probable ATP/GTP binding protein, len: 508 aa; 37.4% identity to HP0595 hypothetical protein. Contains PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Pfam domain PF02600 Disulfide bond formation protein DsbB identified within CDS. Gram negative bacterial cells have disulfide bond formation occuring in the oxidative environment of the periplasm and are catalysed by Dsb (disulfide bond) proteins found in the periplasm and in the inner membrane. Supporting paper (PMID:15632440) demonstrates via deletion [...] ","uniprotId":"Q0PCA8_CAMJE"},"10640902":{"id":10640902,"externalId":"192222.Cj0018c","name":"dba","hasFamily":true,"annotation":"Disulphide bond formation protein; Original (2000) note: Cj0018c, small hydrophobic protein, len: 55 aa; similar to TR:E1314390 (EMBL:AL031184) small hydrophobic protein SC2A11.18 from Streptomyces coelicolor (55 aa), fasta scores; opt: 72 z-score: 141.4 E(): 1.3, 32.7% identity in 49 aa overlap. 49.1% identity to HP0594 hypothetical protein; Updated (2006) note: Gram negative bacterial cells have disulfide bond formation occuring in the oxidative environment of the periplasm and are catalysed by Dsb (disulfide bond) proteins found in the periplasm and in the inner membrance. Supportin [...] ","uniprotId":"Q0PCA7_CAMJE"},"10640903":{"id":10640903,"externalId":"192222.Cj0019c","name":"Cj0019c","hasFamily":true,"annotation":"Original (2000) note: Cj0019c, probable MCP-domain signal transduction protein, len: 592 aa; similar to other proteins containing the MCP domain e.g. HLYB_VIBCH hemolysin secretion protein precursor (548 aa), fasta scores; opt: 418 z-score: 492.5 E(): 3.5e-20, 22.7% identity in 565 aa overlap, and AER_ECOLI aerotaxis receptor (506 aa), fasta scores; opt: 299 z-score: 391.2 E(): 1.6e-14, 25.5% identity in 357 aa overlap. 27.5% identity to HP0103. Contains N-terminal signal sequence and transmembrane anchor around aa 250. Also contains Pfam match to entry PF00015 MCPsignal, Methyl-accept [...] ","uniprotId":"Q0PCA6_CAMJE"},"10640904":{"id":10640904,"externalId":"192222.Cj0020c","name":"Cj0020c","hasFamily":true,"annotation":"Cytochrome C551 peroxidase; Original (2000) note: Cj0020c, probable cytochrome C551 peroxidase, len: 304 aa; similar to CCPR_PSEAE cytochrome C551 peroxidase precursor (EC 1.11.1.5) (346 aa), fasta scores; opt: 775 z-score: 1167.7 E(): 0, 42.3% identity in 298 aa overlap. No Hp ortholog. Contains 2x PS00190 Cytochrome c family heme-binding site signature and probable N-terminal signal sequence; Updated (2006) note: Characterisation has been carried out within Pseudomonas aeruginosa with acceptable identity score. Appropriate motifs also identified. Putative not added to product functio [...] ","uniprotId":"Q0PCA5_CAMJE"},"10640905":{"id":10640905,"externalId":"192222.Cj0021c","name":"Cj0021c","hasFamily":true,"annotation":"Putative fumarylacetoacetate (FAA) hydrolase family protein; Original (2000) note: Cj0021c, unknown, len: 292 aa; similar to hypothetical proteins e.g. TR:O06724 (EMBL:Z99109) B. subtilis yisK (301 aa), fasta scores; opt: 665 z-score: 1018.0 E(): 0, 43.1% identity in 255 aa overlap, and to C-terminal half of HPCE_ECOLI 2-hydroxyhepta-2,4-diene-1,7-dioate isomeraase/5-carboxymethyl-2-oxo-hex-3-ene-1,7-dioate decarboxylase (405 aa), fasta scores; opt: 328 z-score: 381.2 E(): 5.6e-14, 31.2% identity in 199 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF01557 Fumarylacetoaceta [...] ","uniprotId":"Q0PCA4_CAMJE"},"10640906":{"id":10640906,"externalId":"192222.Cj0022c","name":"Cj0022c","hasFamily":true,"annotation":"Original (2000) note: Cj0022c, possible ribosomal pseudouridine synthase, len: 300 aa; similar to e.g. RLUD_ECOLI ribosomal large subunit pseudouridine synthase D (EC 4.2.1.70) (326 aa), fasta scores; opt: 344 z-score: 412.1 E(): 1.2e-15, 29.6% identity in 243 aa overlap. 41.1% identity to HP0347. Contains PS01129 Hypothetical yabO/yceC/sfhB family signature and Pfam match to entry PF00849 YABO, Hypothetical yabO/yceC/sfhB family, score 182.90, E-value 5e-51. Also similar to Cj1280c (31.0% identity) and Cj0708 (28.0% identity); Updated (2006) note: Characterisation has been carried out [...] ","uniprotId":"Q0PCA3_CAMJE"},"10640907":{"id":10640907,"externalId":"192222.Cj0023","name":"purB","hasFamily":true,"annotation":"Belongs to the lyase 1 family. Adenylosuccinate lyase subfamily","uniprotId":"Q0PCA2_CAMJE"},"10640908":{"id":10640908,"externalId":"192222.Cj0024","name":"nrdA","hasFamily":true,"annotation":"Provides the precursors necessary for DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding ribonucleotides","uniprotId":"Q0PCA1_CAMJE"},"10640909":{"id":10640909,"externalId":"192222.Cj0025c","name":"Cj0025c","hasFamily":true,"annotation":"Belongs to the dicarboxylate/amino acid:cation symporter (DAACS) (TC 2.A.23) family","uniprotId":"Q0PCA0_CAMJE"},"10640910":{"id":10640910,"externalId":"192222.Cj0026c","name":"thyX","hasFamily":true,"annotation":"Catalyzes the reductive methylation of 2'-deoxyuridine-5'- monophosphate (dUMP) to 2'-deoxythymidine-5'-monophosphate (dTMP) while utilizing 5,10-methylenetetrahydrofolate (mTHF) as the methyl donor, and NADPH and FADH(2) as the reductant","uniprotId":"THYX_CAMJE"},"10640911":{"id":10640911,"externalId":"192222.Cj0027","name":"pyrG","hasFamily":true,"annotation":"Catalyzes the ATP-dependent amination of UTP to CTP with either L-glutamine or ammonia as the source of nitrogen. Regulates intracellular CTP levels through interactions with the four ribonucleotide triphosphates","uniprotId":"PYRG_CAMJE"},"10640912":{"id":10640912,"externalId":"192222.Cj0028","name":"recJ","hasFamily":true,"annotation":"Original (2000) note: Cj0028, recJ, probable single-stranded-DNA-specific exonuclease, len: 523 aa; similar to e.g. RECJ_ECOLI single-stranded-DNA-specific exonuclease (EC 3.1.-.-) (577 aa), fasta scores; opt: 385 z-score: 861.6 E(): 0, 29.8% identity in 526 aa overlap. 39.0% identity to HP0348; Updated (2006) note: Pfam domain HMMPfam PF01368 DHH family and HMMPfam PF02272 DHHA1 domain were identified within CDS. Further support given to product function. Characterised within Escherichia coli, however,marginal identity score was obtained. Putative kept within product function. Functio [...] ","uniprotId":"Q0PC97_CAMJE"},"10640913":{"id":10640913,"externalId":"192222.Cj0029","name":"ansA","hasFamily":true,"annotation":"Belongs to the asparaginase 1 family. ","uniprotId":"Q0PC96_CAMJE"},"10640914":{"id":10640914,"externalId":"192222.Cj0030","name":"Cj0030","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0030, unknown, len: 465 aa. No Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PC95_CAMJE"},"10640915":{"id":10640915,"externalId":"192222.Cj0031","name":"Cj0031","hasFamily":true,"annotation":"Putative type IIS restriction/modification enzyme; Original (2000) note: Cj0031, probable DNA restriction/modification enzyme, N-terminal half, len: 867 aa; similar to N-terminus of T257_ECOLI type IIS restriction enzyme Eco57I (which contains an adenine-specific methyltransferase activity) (997 aa),fasta scores; opt: 126 z-score: 218.0 E(): 6.9e-05, 24.0% identity in 516 aa overlap. 40.9% identity to N-terminus of HP1517. Some similarity to N-terminus of Cj0690c (24.8% identity in 858 aa overlap). Contains PS00092 N-6 Adenine-specific DNA methylases signature. Note that there is a pol [...] ","uniprotId":"Q0PC94_CAMJE"},"10640916":{"id":10640916,"externalId":"192222.Cj0033","name":"Cj0033","hasFamily":true,"annotation":"Original (2000) note: Cj0033, probable integral membrane protein, len: 593 aa; contains 6 possible transmembrane domains in C-terminal half. No Hp match. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PC93_CAMJE"},"10640917":{"id":10640917,"externalId":"192222.Cj0034c","name":"Cj0034c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0034c, probable periplasmic protein, len: 233 aa; similar to e.g. TR:O50817 (EMBL:AE000788) Borrelia burgdorferi conserved hypothetical protein BBK13 (238 aa), fasta scores; opt: 306 z-score: 257.5 E(): 4.4e-07, 28.1% identity in 231 aa overlap. No Hp match. Contains possible N-terminal signal sequence. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PC92_CAMJE"},"10640918":{"id":10640918,"externalId":"192222.Cj0035c","name":"Cj0035c","hasFamily":true,"annotation":"Putative efflux protein; Original (2000) note: Cj0035c, possible efflux protein, len: 400 aa; similar to many members of the bcr/cmlA subfamily of the major facilitator family e.g. BCR_ECOLI bicyclomycin resistance protein (396 aa), fasta scores; opt: 717 z-score: 1046.3 E(): 0, 28.1% identity in 398 aa overlap. No Hp ortholog. Contains 12 possible transmembrane domains. Also similar to Cj1375 (25.5% identity in 373 aa overlap); Updated (2006) note: Pfam domain PF07690 Major Facilitator Superfamily was identified within CDS. Further support given to product function. Some characterisat [...] ","uniprotId":"Q0PC91_CAMJE"},"10640919":{"id":10640919,"externalId":"192222.Cj0036","name":"Cj0036","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0036, unknown, len: 449 aa; 37.8% identity to HP1143 hypothetical protein. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PC90_CAMJE"},"10640920":{"id":10640920,"externalId":"192222.Cj0037c","name":"Cj0037c","hasFamily":true,"annotation":"Putative cytochrome C; Original (2000) note: Cj0037c, possible cytochrome C, len: 347 aa; weak simlarity to e.g. TR:O69780 (EMBL:U76906) di-heme cytochrome c (fixP) from Rhizobium etli (287 aa), fasta scores; opt: 97 z-score: 205.7 E(): 0.00034, 27.2% identity in 151 aa overlap. No Hp match. Contains 2x PS00190 Cytochrome c family heme-binding site signature and Pfam match to entry PF00034 cytochrome_c,Cytochrome c, score 5.90, E-value 0.11; Updated (2006) note: No specific characterisation with acceptable identity scores has been carried out yet. Thus, putative kept within product fun [...] ","uniprotId":"Q0PC89_CAMJE"},"10640921":{"id":10640921,"externalId":"192222.Cj0038c","name":"Cj0038c","hasFamily":true,"annotation":"Putative poly(A) polymerase family protein; Original (2000) note: Cj0038c, possible membrane protein, len: 215 aa; no HP match. Contains two possible transmembrane domains at the N-terminus; Updated (2006) note: Pfam domain PF01743 Poly A polymerase family identified within CDS by carrying out own Pfam search. This family includes nucleic acid independent RNA polymerases, such as Poly(A) polymerase,which adds the poly(A) tail to mRNA (EC:2.7.7.19). This family also includes the tRNA nucleotidyltransferase that adds the CCA to the 3' of the tRNA (EC:2.7.7.25). Product modified to more s [...] ","uniprotId":"Q0PC88_CAMJE"},"10640922":{"id":10640922,"externalId":"192222.Cj0039c","name":"typA","hasFamily":true,"annotation":"Original (2000) note: Cj0039c, typA, GTP-binding protein typA homolog, len: 602 aa; similar to TYPA_ECOLI GTP-binding protein typA/bipA (591 aa), fasta scores; opt: 2145 z-score: 2659.1 E(): 0, 54.2% identity in 592 aa overlap, and to many EF-G proteins e.g. EFG_AGRTU elongation factor G (EF-G) (699 aa), fasta scores; opt: 344 z-score: 797.3 E(): 0, 26.2% identity in 715 aa overlap. 77.3% identity to HP0480. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00301 GTP-binding elongation factors signature and Pfam match to entry PF00009 GTP_EFTU, Elongation factor Tu family (cont [...] ","uniprotId":"Q0PC87_CAMJE"},"10640923":{"id":10640923,"externalId":"192222.Cj0040","name":"Cj0040","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0040, unknown, len: 107 aa; no Hp match; Updated (2006) note: Literature search identified paper giving clues to product function. Functional classification - Unknown; PMID:14985343","uniprotId":"Q0PC86_CAMJE"},"10640924":{"id":10640924,"externalId":"192222.Cj0041","name":"fliK","hasFamily":true,"annotation":"Putative flagellar hook-length control protein; Original (2000) note: Cj0041, unknown, len: 598 aa; 25.2% identity to HP0906, but shorter at N-term; Updated (2006) note: Literature search identified papers linking product function to be involved in motility and invasion. Unpublished observation by N.Kamal & C. Penn have predicted this CDS to be fliK. Literature search identified work carried out in Campylobacter jejuni,Helicobacter pylori and Escherichia coli. Product function modified to more specific family member. Putative was kept in product function as not fully characterised. Fun [...] ","uniprotId":"Q0PC85_CAMJE"},"10640925":{"id":10640925,"externalId":"192222.Cj0042","name":"flgD","hasFamily":true,"annotation":"Required for flagellar hook formation. May act as a scaffolding protein","uniprotId":"Q0PC84_CAMJE"},"10640926":{"id":10640926,"externalId":"192222.Cj0043","name":"flgE","hasFamily":true,"annotation":"Flagellar basal body protein; Original (2000) note: Cj0043, flgE, probable flagellar hook protein, len: 545 aa; similar to e.g. FLGE_BORBU flagellar hook protein flgE (442 aa), fasta scores; opt: 350 z-score: 665.8 E(): 7.9e-30, 27.5% identity in 553 aa overlap. 24.5% identity to HP0908. Contains PS00588 Flagella basal body rod proteins signature and Pfam match to entry PF00460 flg_bb_rod,Flagella basal body rod proteins, score 37.00, E-value 6.7e-08; Updated (2006) note: Characterised within Borrilia burgdorferi and also some work within Campylobacter jejuni. Identity scores were marg [...] ","uniprotId":"Q0PC83_CAMJE"},"10640927":{"id":10640927,"externalId":"192222.Cj0044c","name":"Cj0044c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0044c, unknown, len: 623 aa; no Hp ortholog, Also similar to Cj0800c (28.5% identity in 625 aa overlap). Preceeded by a C(9-11) polymorphic region; C11 would allow the translation of Cj0045c to extend by another 15aa, overlapping the start of this CDS; C9 and C10 (consensus) have no effect; Updated (2006) note: Literature search identified paper giving further clues to product function. Functional classification - Conserved hypothetical proteins; PMID:14985343","uniprotId":"Q0PC82_CAMJE"},"10640928":{"id":10640928,"externalId":"192222.Cj0045c","name":"Cj0045c","hasFamily":true,"annotation":"Putative iron-binding protein; Original (2000) note: Cj0045c, possible iron-binding protein, len: 240 aa; weak similarity to members of the eukaryotic iron-binding hemerythrin family,e.g. HEMT_LINUN hemerythrin alpha chain (117 aa), fasta scores; opt: 105 z-score: 137.1 E(): 2.5, 23.8% identity in 122 aa overlap. Note that all of the iron-binding residues are conserved. No Hp match, similar to Cj0072c (pseudogene), Cj0241c (30.9% identity in 123 aa overlap) and Cj1224 (27.7% identity in 202 aa overlap). Contains a C(9-11) polymorphic region at the N-terminus; C11 would allow the transl [...] ","uniprotId":"Q0PC81_CAMJE"},"10640929":{"id":10640929,"externalId":"192222.Cj0053c","name":"trmU","hasFamily":true,"annotation":"Catalyzes the 2-thiolation of uridine at the wobble position (U34) of tRNA, leading to the formation of s(2)U34","uniprotId":"MNMA_CAMJE"},"10640930":{"id":10640930,"externalId":"192222.Cj0054c","name":"Cj0054c","hasFamily":true,"annotation":"Belongs to the LOG family","uniprotId":"Q0PC79_CAMJE"},"10640931":{"id":10640931,"externalId":"192222.Cj0055c","name":"Cj0055c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0055c, unknown, len: 274 aa; no Hp match, similar in C-terminus to Cj0122 (46.6% identity in 148 aa overlap); Updated (2006) note: Contains Prosite match PS01164 COPPER_AMINE_OXID, Copper amine oxidase. Amine oxidases are enzymes that catalyze the oxidation of different biogenic amines e.g. neurotransmitters. Copper-containing amine oxidases are found in bacteria. This particular form binds one copper ion per subunit as well as a 2,4,5-trihydroxyphenylalanine quinone (or topaquinone) (TPQ) cofactor (PMID:8591028). Functional classificati [...] ","uniprotId":"Q0PC78_CAMJE"},"10640932":{"id":10640932,"externalId":"192222.Cj0056c","name":"Cj0056c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0056c, unknown, len: 138 aa. No Hp match; Updated (2006) note: Literature search identified paper giving clues to product function. Functional classification - Unknown; PMID:14702320","uniprotId":"Q0PC77_CAMJE"},"10640933":{"id":10640933,"externalId":"192222.Cj0057","name":"Cj0057","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0057, possible periplasmic protein, len: 284 aa; no Hp match. Contains possible N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PC76_CAMJE"},"10640934":{"id":10640934,"externalId":"192222.Cj0058","name":"Cj0058","hasFamily":true,"annotation":"Putative peptidase C39 family protein; Original (2000) note: Cj0058, possible periplasmic protein, len: 199 aa; no Hp match. Contains possible N-terminal signal sequence; Updated (2006) note: Pfam domain PF03412 Peptidase C39 family identified within CDS. Product modified to more specific family member based on motif match. No specific characterisation with acceptable identity scores carried out yet. Thus, putative kept within product function. Functional classification - Degradation of macromolecules - Proteins, peptides and glycopeptides","uniprotId":"Q0PC75_CAMJE"},"10640935":{"id":10640935,"externalId":"192222.Cj0059c","name":"fliY","hasFamily":true,"annotation":"Original (2000) note: Cj0059c, fliY, probable flagellar motor switch protein, len: 280 aa; similar to e.g. TR:P74928 (EMBL:U36839) Treponema pallidum flagellar motor switch protein fliY (348 aa), fasta scores; opt: 277 z-score: 496.5 E(): 2.1e-20, 26.2% identity in 309 aa overlap. Also similar in C-terminus to FLIN_BORBU flagellar motor switch protein fliN (113 aa), fasta scores; opt: 231 z-score: 362.5 E(): 6.2e-13, 44.2% identity in 77 aa overlap. 39.7% identity to HP1030. Contains Pfam match to entry PF01052 SPOA_protein, Surface presentation of antigens (SPOA) protein, score 52.80, [...] ","uniprotId":"Q0PC74_CAMJE"},"10640936":{"id":10640936,"externalId":"192222.Cj0060c","name":"fliM","hasFamily":true,"annotation":"Flagellar motor switch protein; Original (2000) note: Cj0060c, fliM, probable flagellar motor switch protein, len: 359 aa; simlar to e.g. FLIM_BACSU flagellar motor switch protein fliM (332 aa), fasta scores; opt: 728 z-score: 1046.5 E(): 0, 35.6% identity in 329 aa overlap. 61.1% identity to HP1031; Updated (2006) note: Pfam domain PF01052 Surface presentation of antigens (SPOA) protein and PF02154 Flagellar motor switch protein FliM were identified within CDS. Further support given to product function. Characterised within Bacillus subtilis with acceptable identity score. Thus, putat [...] ","uniprotId":"Q0PC73_CAMJE"},"10640937":{"id":10640937,"externalId":"192222.Cj0061c","name":"fliA","hasFamily":true,"annotation":"Belongs to the sigma-70 factor family. ","uniprotId":"Q0PC72_CAMJE"},"10640938":{"id":10640938,"externalId":"192222.Cj0062c","name":"Cj0062c","hasFamily":true,"annotation":"Original (2000) note: Cj0062c, probable integral membrane protein, len: 118 aa; no Hp match. Contains two possible transmembrane domains. Functional classification - Cell envelope - Membranes, lipoproteins and porins","uniprotId":"Q0PC71_CAMJE"},"10640939":{"id":10640939,"externalId":"192222.Cj0063c","name":"Cj0063c","hasFamily":true,"annotation":"Site-determining protein; Original (2000) note: Cj0063c, probable ATP-binding protein, len: 288 aa; similar to hypothetical proteins in other flagellar operons e.g. YLXH_BACSU hypothetical 33.2 kd protein in flhF-cheB intergenic region (298 aa), fasta scores; opt: 390 z-score: 493.5 E(): 3.1e-20, 29.0% identity in 293 aa overlap, and to cell division proteins e.g. MIND_BACSU septum site-determining protein minD (268 aa), fasta scores; opt: 323 z-score: 417.6 E(): 5.3e-16,32.9% identity in 249 aa overlap. 45.7% identity to HP1034. Contains PS00017 ATP/GTP-binding site motif A (P-loop) a [...] ","uniprotId":"Q0PC70_CAMJE"},"10640940":{"id":10640940,"externalId":"192222.Cj0064c","name":"flhF","hasFamily":true,"annotation":"Necessary for flagellar biosynthesis. May be involved in translocation of the flagellum (By similarity)","uniprotId":"FLHF_CAMJE"},"10640941":{"id":10640941,"externalId":"192222.Cj0065c","name":"folK","hasFamily":true,"annotation":"Belongs to the HPPK family","uniprotId":"HPPK_CAMJE"},"10640942":{"id":10640942,"externalId":"192222.Cj0066c","name":"aroQ","hasFamily":true,"annotation":"Catalyzes a trans-dehydration via an enolate intermediate. ","uniprotId":"AROQ_CAMJE"},"10640943":{"id":10640943,"externalId":"192222.Cj0067","name":"Cj0067","hasFamily":true,"annotation":"Putative amidohydrolase family protein; Original (2000) note: Cj0067, unknown, len: 409 aa; identical to, but shorter than, C. jejuni hypothetical protein TR:P71126 (EMBL:U38524) upstream of pspA (518 aa) (99.5% identity in 409 aa overlap), and to many hypothetical proteins e.g. YF41_METJA hypothetical protein MJ1541 (420 aa), fasta scores; opt: 226 z-score: 413.4 E(): 9.2e-16, 26.9% identity in 383 aa overlap. 42.0% identity to HP0267 chlorohydrolase; Updated (2006) note: Pfam domain PF01979 Amidohydrolase family identified within CDS. Product modified to more specific family member d [...] ","uniprotId":"Q0PC66_CAMJE"},"10640944":{"id":10640944,"externalId":"192222.Cj0068","name":"pspA","hasFamily":true,"annotation":"Protease; Original (2000) note: Cj0068, pspA, probable protease required for pilus-like appendage formation, len: 298 aa; almost identical to TR:P71127 (EMBL:U38524) pspA (298 aa) (99.0% identity in 298 aa overlap) and similar to the C-terminus of e.g. SPPA_ECOLI protease IV (618 aa),fasta scores; opt: 285 z-score: 266.8 E(): 1.3e-07, 24.7% identity in 251 aa overlap. 42.0% identity to HP1435; Updated (2006) note: Pfam domain PF01343 Peptidase family S49 identified within CDS. Product function based on direct submission with paper (PMID:8793884). Putative not added to product function  [...] ","uniprotId":"Q0PC65_CAMJE"},"10640945":{"id":10640945,"externalId":"192222.Cj0069","name":"Cj0069","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0069, unknown, len: 343 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Literature search identified paper giving further clues to product function. Functional classification - Unknown; PMID:14702320","uniprotId":"Q0PC64_CAMJE"},"10640946":{"id":10640946,"externalId":"192222.Cj0070c","name":"Cj0070c","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0070c, questionable CDS,len: 52 aa; no Hp match; Updated (2006) note: Similar to proteins in other bacteria. Functional classification - Unknown","uniprotId":"Q0PC63_CAMJE"},"10640947":{"id":10640947,"externalId":"192222.Cj0073c","name":"Cj0073c","hasFamily":true,"annotation":"Uncharacterized protein; HMMPfam hit to PF01814, Hemerythrin HHE cation binding domain, score 0.007","uniprotId":"Q0PC62_CAMJE"},"10640948":{"id":10640948,"externalId":"192222.Cj0074c","name":"Cj0074c","hasFamily":true,"annotation":"Putative iron-sulfur protein; Original (2000) note: Cj0074c, probable iron-sulfur protein, len: 479 aa; similar to hypothetical proteins e.g. YKGF_ECOLI (475 aa), fasta scores; opt: 1142 z-score: 1335.9 E(): 0, 38.3% identity in 462 aa overlap. 58.1% identity to HP0138. Contains 2x PS00198 4Fe-4S ferredoxins, iron-sulfur binding region signature and 2x Pfam match to entry PF00037 fer4, 4Fe-4S ferredoxins and related iron-sulfur cluster binding domains., scores 13.80, E-value 0.0095, and 12.30, E-value 0.026; Updated (2006) note: No specific characterisation with acceptable identity sco [...] ","uniprotId":"Q0PC61_CAMJE"},"10640949":{"id":10640949,"externalId":"192222.Cj0075c","name":"Cj0075c","hasFamily":true,"annotation":"Original (2000) note: Cj0075c, possible oxidoreductase iron-sulfur subunit, len: 246 aa; similar to hypothetical proteins and to the C-terminus of e.g. TR:O53142 (EMBL:AJ000942) Methanobacterium thermoautotrophicum strain Marburg, Thiol:fumarate reductase subunit B (488 aa), fasta scores; opt: 217 z-score: 476.0 E(): 3e-1, 24.3% identity in 235 aa overlap. 60.7% identity to HP0139; Updated (2006) note: Pfam domain x2 PF02754 Cysteine-rich domain identified within CDS. Further support given to product function. No specific characterisation with acceptable identity score. Thus,putative k [...] ","uniprotId":"Q0PC60_CAMJE"},"10640950":{"id":10640950,"externalId":"192222.Cj0076c","name":"lctP","hasFamily":true,"annotation":"Transports L-lactate across the membrane. Can also transport D-lactate and glycolate","uniprotId":"Q0PC59_CAMJE"},"10640951":{"id":10640951,"externalId":"192222.Cj0077c","name":"cdtC","hasFamily":true,"annotation":"Original (2000) note: Cj0077c, cdtC, cytolethal distending toxin, len: 189 aa, identical to TR:Q46102 (EMBL:U51121) cdtC (189 aa). No Hp match. Contains N-terminal signal sequence; Updated (2006) note: Pfam domain PF03499 Cytolethal distending toxin C identified within CDS. Product modified to more specific family member due to motif match. Characterised within Campylobacter jejuni, so putative not added to product function. Functional classification -Pathogenicity; PMID:10688204, PMID:8675309, PMID:11083762","uniprotId":"Q0PC58_CAMJE"},"10640952":{"id":10640952,"externalId":"192222.Cj0078c","name":"cdtB","hasFamily":true,"annotation":"Original (2000) note: Cj0078c, cdtB, cytolethal distending toxin, len: 265 aa; identical toTR:Q46101 (EMBL:U51121) cdtB (265 aa) and similar to e.g. TR:Q47089 (EMBL:U03293) Escherichia coli cdtB (273 aa), fasta scores; opt: 946 z-score: 1533.6 E(): 0, 55.2% identity in 268 aa overlap. No Hp match. Contains N-terminal signal sequence; Updated (2006) note: Prosite domain PRO1388 CDTOXINB, Cytolethal distending toxin B signature identified within CDS. Product modified to more specific family member due to motif match. Characterised within Campylobacter jejuni, so putative not added to pro [...] ","uniprotId":"Q0PC57_CAMJE"},"10640953":{"id":10640953,"externalId":"192222.Cj0079c","name":"cdtA","hasFamily":true,"annotation":"CDTs are cytotoxins which induce cell distension, growth arrest in G2/M phase, nucleus swelling, and chromatin fragmentation in HeLa cells","uniprotId":"CDTA_CAMJE"},"10640954":{"id":10640954,"externalId":"192222.Cj0080","name":"Cj0080","hasFamily":true,"annotation":"Putative membrane protein. Functional classification-Membranes, lipoproteins and porins; Original (2000) note: Cj0080, possible membrane protein, len: 267 aa; no Hp match; Contains one possible transmembrane domain; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0","uniprotId":"Q0PC55_CAMJE"},"10640955":{"id":10640955,"externalId":"192222.Cj0081","name":"cydA","hasFamily":true,"annotation":"Original (2000) note: Cj0081, cydA, probable cytochrome bd oxidase subunit I, len: 520 aa;similar to many e.g. APPC_ECOLI cytochrome bd-ii oxidase subunit I (EC 1.10.3.-) (514 aa), fasta scores; opt: 1412 z-score: 1888.2 E(): 0, 45.7% identity in 530 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF01654 Bacterial Cytochrome Ubiquinol Oxidase identified within CDS. Also,nine probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Characterised within Escherichia coli with acceptable identity score. Thus, putative not added to product  [...] ","uniprotId":"Q0PC54_CAMJE"},"10640956":{"id":10640956,"externalId":"192222.Cj0082","name":"cydB","hasFamily":true,"annotation":"Cytochrome bd oxidase subunit II; Original (2000) note: Cj0082, cydB, probable cytochrome bd oxidase subunit II, len: 520 aa;similar to many e.g. CYDB_ECOLI cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-) (379 aa), fasta scores; opt: 466 z-score: 449.6 E(): 8.7e-18, 27.7% identity in 386 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02322 Cytochrome oxidase subunit II identified within CDS. Also, nine probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Characterised within Escherichia coli with marginal identity score.  [...] ","uniprotId":"Q0PC53_CAMJE"},"10640957":{"id":10640957,"externalId":"192222.Cj0085c","name":"Cj0085c","hasFamily":true,"annotation":"Original (2000) note: Cj0085c, probable amino acid recemase, len: 231 aa; similar to members of the aspartate/glutamate racemases family e.g. YGEA_ECOLI (230 aa), fasta scores; opt: 667 z-score: 1155.6 E(): 0, 46.7% identity in 225 aa overlap, and RACD_STRTR aspartate racemase (243 aa), fasta scores; opt: 254 z-score: 230.9 E(): 1.3e-05, 25.4% identity in 232 aa overlap. No Hp match. Contains PS00924 Aspartate and glutamate racemases signature 2 andPfam match to entry PF01177 Asp_Glu_race,Aspartate / glutamate racemase, score 108.40, E-value 1.3e-28; Updated (2006) note: Pfam domain PF [...] ","uniprotId":"Q0PC52_CAMJE"},"10640958":{"id":10640958,"externalId":"192222.Cj0086c","name":"ung","hasFamily":true,"annotation":"Excises uracil residues from the DNA which can arise as a result of misincorporation of dUMP residues by DNA polymerase or due to deamination of cytosine","uniprotId":"UNG_CAMJE"},"10640959":{"id":10640959,"externalId":"192222.Cj0087","name":"aspA","hasFamily":true,"annotation":"Original (2000) note: Cj0087, aspA, aspartate ammonia-lyase, len: 468 aa; similar to many e.g.ASPA_ECOLI aspartate ammonia-lyase (EC 4.3.1.1) (ASPAR (478 aa),fasta scores; opt: 1853 z-score: 2526.1 E(): 0, 58.2% identity in 466 aa overlap, 67.5% identity to HP0649. Contains PS00163 Fumarate lyases signature and Pfam match to entry PF00206 lyase_1, Lyases, score 615.50, E-value 3.1e-181; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Thus,putative not added to product function. Functional classification - Central intermediary metabolism -Gener [...] ","uniprotId":"Q0PC50_CAMJE"},"10640960":{"id":10640960,"externalId":"192222.Cj0088","name":"dcuA","hasFamily":true,"annotation":"Responsible for the transport of C4-dicarboxylates from the periplasm across the inner membrane","uniprotId":"Q0PC49_CAMJE"},"10640961":{"id":10640961,"externalId":"192222.Cj0089","name":"Cj0089","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0089, probable lipoprotein,len: 453 aa; 27.4% identity to HP0018. Contains N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PC48_CAMJE"},"10640962":{"id":10640962,"externalId":"192222.Cj0090","name":"Cj0090","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0090, probable lipoprotein,len: 122 aa; no Hp match. Contains N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PC47_CAMJE"},"10640963":{"id":10640963,"externalId":"192222.Cj0091","name":"Cj0091","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0091, probable lipoprotein,len: 207 aa; 33.5% identity to HP1457. Contains N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PC46_CAMJE"},"10640964":{"id":10640964,"externalId":"192222.Cj0092","name":"Cj0092","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0092, probable periplasmic protein, len: 445 aa; no Hp match. Contains probable N-terminal signal sequence and PS00017 ATP/GTP-binding site motif A (P-loop). Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PC45_CAMJE"},"10640965":{"id":10640965,"externalId":"192222.Cj0093","name":"Cj0093","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0093, probable periplasmic protein, len: 400 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PC44_CAMJE"},"10640966":{"id":10640966,"externalId":"192222.Cj0094","name":"rplU","hasFamily":true,"annotation":"This protein binds to 23S rRNA in the presence of protein L20","uniprotId":"RL21_CAMJE"},"10640967":{"id":10640967,"externalId":"192222.Cj0095","name":"rpmA","hasFamily":true,"annotation":"Belongs to the bacterial ribosomal protein bL27 family. ","uniprotId":"RL27_CAMJE"},"10640968":{"id":10640968,"externalId":"192222.Cj0096","name":"obg","hasFamily":true,"annotation":"An essential GTPase which binds GTP, GDP and possibly (p)ppGpp with moderate affinity, with high nucleotide exchange rates and a fairly low GTP hydrolysis rate. Plays a role in control of the cell cycle, stress response, ribosome biogenesis and in those bacteria that undergo differentiation, in morphogenesis control. ","uniprotId":"OBG_CAMJE"},"10640969":{"id":10640969,"externalId":"192222.Cj0097","name":"proB","hasFamily":true,"annotation":"Catalyzes the transfer of a phosphate group to glutamate to form L-glutamate 5-phosphate","uniprotId":"PROB_CAMJE"},"10640970":{"id":10640970,"externalId":"192222.Cj0098","name":"fmt","hasFamily":true,"annotation":"Attaches a formyl group to the free amino group of methionyl- tRNA(fMet). The formyl group appears to play a dual role in the initiator identity of N-formylmethionyl-tRNA by promoting its recognition by IF2 and preventing the misappropriation of this tRNA by the elongation apparatus","uniprotId":"FMT_CAMJE"},"10640971":{"id":10640971,"externalId":"192222.Cj0099","name":"birA","hasFamily":true,"annotation":"Original (2000) note: Cj0099, birA, possible biotin--[acetyl-CoA-carboxylase] synthetase, len: 217 aa; weak similarity to e.g. SW:BIRA_BACSU P42975 birA bifunctional protein (biotin operon repressor) (biotin--[acetyl-CoA-carboxylase] synthetase) (EC 6.3.4.15)(325 aa), blastp scores; E= 0.00011, 21% identity in 163 aa overlap; similarity does not include DNA-binding domain. 39.0% identity to HP1140; Updated (2006) note: Pfam domain PF03099 Biotin/lipoate A/B protein ligase family identified within CDS. Further support given to product function. Specific characterisation with acceptable  [...] ","uniprotId":"Q0PC38_CAMJE"},"10640972":{"id":10640972,"externalId":"192222.Cj0100","name":"Cj0100","hasFamily":true,"annotation":"ParA family protein; Original (2000) note: Cj0100, putative parA family protein, len: 261 aa; similar to members of the parA family e.g. SOJ_BACSU soj protein (253 aa) fasta scores; opt: 762 z-score: 1251.6 E(): 0, 46.3% identity in 259 aa overlap. 61.1% identity to HP1139. Contains Pfam match to entry PF00991 ParA, ParA family ATPase, score 118.40,E-value 1.3e-31; Updated (2006) note: Pfam domain PF01656 CobQ/CobB/MinD/ParA nucleotide binding domain identified within CDS. This family consists of putative cobyrinic acid a,c-diamide synthases family protein. Some characterisation work w [...] ","uniprotId":"Q0PC37_CAMJE"},"10640973":{"id":10640973,"externalId":"192222.Cj0101","name":"parB","hasFamily":true,"annotation":"Involved in chromosome partition. Localize to both poles of the predivisional cell following completion of DNA replication. Binds to the DNA origin of replication (By similarity)","uniprotId":"PARB_CAMJE"},"10640974":{"id":10640974,"externalId":"192222.Cj0102","name":"atpF","hasFamily":true,"annotation":"Belongs to the ATPase B chain family. ","uniprotId":"Q0PC35_CAMJE"},"10640975":{"id":10640975,"externalId":"192222.Cj0103","name":"Cj0103","hasFamily":true,"annotation":"Component of the F(0) channel, it forms part of the peripheral stalk, linking F(1) to F(0)","uniprotId":"Q0PC34_CAMJE"},"10640976":{"id":10640976,"externalId":"192222.Cj0104","name":"atpH","hasFamily":true,"annotation":"F(1)F(0) ATP synthase produces ATP from ADP in the presence of a proton or sodium gradient. F-type ATPases consist of two structural domains, F(1) containing the extramembraneous catalytic core and F(0) containing the membrane proton channel, linked together by a central stalk and a peripheral stalk. During catalysis, ATP synthesis in the catalytic domain of F(1) is coupled via a rotary mechanism of the central stalk subunits to proton translocation","uniprotId":"ATPD_CAMJE"},"10640977":{"id":10640977,"externalId":"192222.Cj0105","name":"atpA","hasFamily":true,"annotation":"Produces ATP from ADP in the presence of a proton gradient across the membrane. The alpha chain is a regulatory subunit. ","uniprotId":"ATPA_CAMJE"},"10640978":{"id":10640978,"externalId":"192222.Cj0106","name":"atpG","hasFamily":true,"annotation":"Produces ATP from ADP in the presence of a proton gradient across the membrane. The gamma chain is believed to be important in regulating ATPase activity and the flow of protons through the CF(0) complex","uniprotId":"ATPG_CAMJE"},"10640979":{"id":10640979,"externalId":"192222.Cj0107","name":"atpD","hasFamily":true,"annotation":"Produces ATP from ADP in the presence of a proton gradient across the membrane. The catalytic sites are hosted primarily by the beta subunits","uniprotId":"ATPB_CAMJE"},"10640980":{"id":10640980,"externalId":"192222.Cj0108","name":"atpC","hasFamily":true,"annotation":"Produces ATP from ADP in the presence of a proton gradient across the membrane","uniprotId":"ATPE_CAMJE"},"10640981":{"id":10640981,"externalId":"192222.Cj0109","name":"exbB3","hasFamily":true,"annotation":"Putative MotA/TolQ/ExbB proton channel family protein; Original (2000) note: Cj0109, exbB3, probable exbB/tolQ family transport protein, len: 184 aa; similar to members of the exbB/tolQ family, e.g. EXBB_ECOLI biopolymer transport exbB protein (244 aa), fasta scores; opt: 208 z-score: 287.0 E(): 9.9e-09, 33.6% identity in 128 aa overlap, and TOLQ_ECOLI tolQ protein (230 aa),fasta scores; opt: 187 z-score: 280.1 E(): 2.4e-08, 48.4% identity in 62 aa overlap. 42.6% identity to HP1130. Also similar to Cj0179 (30.6% identity in 186 aa overlap) and Cj1628 (44.4% identity in 81 aa overlap);  [...] ","uniprotId":"Q0PC28_CAMJE"},"10640982":{"id":10640982,"externalId":"192222.Cj0110","name":"exbD3","hasFamily":true,"annotation":"Belongs to the ExbD/TolR family. ","uniprotId":"Q0PC27_CAMJE"},"10640983":{"id":10640983,"externalId":"192222.Cj0111","name":"Cj0111","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0111, possible periplasmic protein, len: 259 aa; no Hp match. Contains possible N-terminal signal sequence; Updated (2006) note: Literature search identified paper (PMID:15231804) suggesting a number of differentially expressed genes (e.g. Cj0111) in microarray experiments may be co-trasncribed. Cj0111 seems to be iron induced and based on RT-PCR seems to be co-transcribed with Cj1658. Functional classification - Miscellaneous periplasmic proteins; PMID:15231804","uniprotId":"Q0PC26_CAMJE"},"10640984":{"id":10640984,"externalId":"192222.Cj0112","name":"tolB","hasFamily":true,"annotation":"Part of the Tol-Pal system, which plays a role in outer membrane invagination during cell division and is important for maintaining outer membrane integrity","uniprotId":"TOLB_CAMJE"},"10640985":{"id":10640985,"externalId":"192222.Cj0113","name":"pal","hasFamily":true,"annotation":"Part of the Tol-Pal system, which plays a role in outer membrane invagination during cell division and is important for maintaining outer membrane integrity","uniprotId":"Q0PC24_CAMJE"},"10640986":{"id":10640986,"externalId":"192222.Cj0114","name":"Cj0114","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0114, possible periplasmic protein, len: 315 aa; similar to hypothetical protein YBGF_ECOLI hypothetical 28.2 kd protein in pal-lysT intergenic region (263 aa), fasta scores; opt: 144 z-score: 238.8 E(): 4.8e-06, 21.3% identity in 249 aa overlap. 28.6% identity to HP1124. Contains possible N-terminal signal sequence; Updated (2006) note: Prosite PS50005 TPR_REPEAT,Tetratricopeptide region identified within CDS. Supporting paper identifies protein as a glycoprotein. Functional classification - Miscellaneous periplasmic proteins; PMID [...] ","uniprotId":"Q0PC23_CAMJE"},"10640987":{"id":10640987,"externalId":"192222.Cj0115","name":"slyD","hasFamily":true,"annotation":"Original (2000) note: Cj0115, slyD, probable peptidyl-prolyl cis-trans isomerase, len: 189 aa; similar to many of the FKBP-type e.g. SLYD_ECOLI FKBP-type peptidyl-prolyl cis-trans isomerase (196 aa), fasta scores; opt: 478 z-score: 296.9 E(): 2.8e-09, 38.5% identity in 200 aa overlap. 39.2% identity to HP1123; Updated (2006) note: Prosite PS50059 FKBP_PPIASE,Peptidylprolyl isomerase, FKBP-type domain identified within CDS. Product modified to more specific family member due to motif match. Characterised within Escherichia coli with acceptable identity score. Putative not added to produ [...] ","uniprotId":"Q0PC22_CAMJE"},"10640988":{"id":10640988,"externalId":"192222.Cj0116","name":"fabD","hasFamily":true,"annotation":"Original (2000) note: Cj0116, fabD, probable malonyl CoA-acyl carrier protein transacylase, len: 306 aa; similar to many e.g. FABD_ECOLI malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39) (308 aa), fasta scores; opt: 555 z-score: 655.2 E(): 3.1e-29, 36.1% identity in 296 aa overlap. 49.5% identity to HP0090. Contains Pfam match to entry PF00698 Acyl_transf, Acyl transferase domain, score 58.10, E-value 1.9e-13; Updated (2006) note: Characterised within Escherichia coli with acceptable identity scores. Thus,putative not added to product function. Functional classification - Fat [...] ","uniprotId":"Q0PC21_CAMJE"},"10640989":{"id":10640989,"externalId":"192222.Cj0117","name":"pfs","hasFamily":true,"annotation":"Catalyzes the direct conversion of aminodeoxyfutalosine (AFL) into dehypoxanthine futalosine (DHFL) and adenine via the hydrolysis of the N-glycosidic bond; this reaction seems to represent an essential step in the menaquinone biosynthesis pathway in Campylobacter species. Also catalyzes the hydrolysis of 5'-methylthioadenosine (MTA) to adenine and 5'-methylthioribose. Can also probably use S- adenosylhomocysteine (SAH) as substrate, leading to adenine and S- ribosylhomocysteine. These other activities highlight the tremendous versatility of the enzyme, which also plays key roles in S- [...] ","uniprotId":"MQMTN_CAMJE"},"10640990":{"id":10640990,"externalId":"192222.Cj0118","name":"Cj0118","hasFamily":true,"annotation":"Belongs to the TtcA family. ","uniprotId":"Q0PC19_CAMJE"},"10640991":{"id":10640991,"externalId":"192222.Cj0119","name":"Cj0119","hasFamily":true,"annotation":"Putative hydrolase; Original (2000) note: Cj0119, unknown, len: 173 aa; similar to a hypothetical protein from B. subtilis TR:O32091 (EMBL:Z99120) YUEJ protein (183 aa), fasta scores; opt: 172 z-score: 282.5 E(): 1.8e-08, 29.5% identity in 166 aa overlap. No Hp match; Updated (2006) note: Motif match (IPR000868) identified link to hydrolase. Product modified to more specific family member based on motif match. No specific characterisation with acceptable identity score has been carried out yet. Putative kept within product function. Functional classification - Misc","uniprotId":"Q0PC18_CAMJE"},"10640992":{"id":10640992,"externalId":"192222.Cj0120","name":"Cj0120","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0120, unknown, len: 204 aa; 41.7% identity to HP0951. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PC17_CAMJE"},"10640993":{"id":10640993,"externalId":"192222.Cj0121","name":"ybeY","hasFamily":true,"annotation":"Single strand-specific metallo-endoribonuclease involved in late-stage 70S ribosome quality control and in maturation of the 3' terminus of the 16S rRNA","uniprotId":"YBEY_CAMJE"},"10640994":{"id":10640994,"externalId":"192222.Cj0122","name":"Cj0122","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0122, unknown, len: 229 aa; no Hp match. Similar in C-terminus to Cj0055c (46.6% identity in 148 aa overlap), and in N-terminus to Cj0416 (88.2% identity in 34 aa overlap); Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PC15_CAMJE"},"10640995":{"id":10640995,"externalId":"192222.Cj0123c","name":"Cj0123c","hasFamily":true,"annotation":"Catalyzes the synthesis of 5,6-dihydrouridine (D), a modified base found in the D-loop of most tRNAs, via the reduction of the C5-C6 double bond in target uridines","uniprotId":"Q0PC14_CAMJE"},"10640996":{"id":10640996,"externalId":"192222.Cj0124c","name":"Cj0124c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0124c, probable membrane protein, len: 328 aa; 21.1% identity to HP0948. Contains two possible transmembrane domains. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PC13_CAMJE"},"10640997":{"id":10640997,"externalId":"192222.Cj0125c","name":"Cj0125c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0125c, unknown, len: 120 aa; some similarity to e.g. DKSA_ECOLI dnaK suppressor protein (151 aa), fasta scores; opt: 186 z-score: 276.6 E(): 3.8e-08, 26.3% identity in 118 aa overlap. No Hp match. Contains PS01102 Prokaryotic dksA/traR C4-type zinc finger and Pfam match to entry PF01258 zf_dskA_traR,Prokaryotic dksA/traR C4-type zinc finger, score 39.20,E-value 9.7e-08; Updated (2006) note: No specific characterisation has been carried out yet. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PC12_CAMJE"},"10640998":{"id":10640998,"externalId":"192222.Cj0126c","name":"rlmH","hasFamily":true,"annotation":"Specifically methylates the pseudouridine at position 1915 (m3Psi1915) in 23S rRNA","uniprotId":"RLMH_CAMJE"},"10640999":{"id":10640999,"externalId":"192222.Cj0127c","name":"accD","hasFamily":true,"annotation":"Component of the acetyl coenzyme A carboxylase (ACC) complex. Biotin carboxylase (BC) catalyzes the carboxylation of biotin on its carrier protein (BCCP) and then the CO(2) group is transferred by the transcarboxylase to acetyl-CoA to form malonyl-CoA","uniprotId":"Q0PC10_CAMJE"},"10641000":{"id":10641000,"externalId":"192222.Cj0128c","name":"Cj0128c","hasFamily":true,"annotation":"Inositol-1-monophosphatase; Original (2000) note: Cj0128c, unknown, len: 240 aa; some similarity to e.g. SUHB_ECOLI extragenic suppressor protein suhB (267 aa), fasta scores; opt: 168 z-score: 229.0 E(): 1.7e-05, 29.9% identity in 134 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF00459 Inositol monophosphatase family identified within CDS. Product modified to more specific family member due to motif match. Characterised within Escherichia coli, however,identity scores were not acceptable. Functional classification - Misc; PMID:10747806, PMID:8002619","uniprotId":"Q0PC09_CAMJE"},"10641001":{"id":10641001,"externalId":"192222.Cj0129c","name":"bamA","hasFamily":true,"annotation":"Part of the outer membrane protein assembly complex, which is involved in assembly and insertion of beta-barrel proteins into the outer membrane","uniprotId":"Q0PC08_CAMJE"},"10641002":{"id":10641002,"externalId":"192222.Cj0130","name":"tyrA","hasFamily":true,"annotation":"Putative prephenate dehydrogenase; Original (2000) note: Cj0130, tyrA, probable prephenate dehydrogenase, len: 275 aa; similar to e.g. TYRA_BACSU prephenate dehydrogenase (EC 1.3.1.12) (372 aa), fasta scores; opt: 474 z-score: 529.7 E(): 3e-22,31.0% identity in 284 aa overlap. 45.3% identity to HP1380; Updated (2006) note: Characterised within Bacillus subtilis with marginal identity score. Alignment does not cover full sequence length. Putative kept within product function. Functional classification - Amino acid biosynthesis - Aromatic amino acid family; PMID:3106153","uniprotId":"Q0PC07_CAMJE"},"10641003":{"id":10641003,"externalId":"192222.Cj0131","name":"Cj0131","hasFamily":true,"annotation":"Putative peptidase M23 family protein; Original (2000) note: Cj0131, probable periplasmic protein, len: 457 aa; similar to hypothetical proteins e.g.TR:O25694 (EMBL:AE000754) Aquifex aeolicus AQ_1743 (422 aa), fasta scores; opt: 401 z-score: 635.2 E(): 4e-28, 30.0% identity in 420 aa overlap. 30.0% identity to HP1054. Contains probable N-terminal signal sequence. Also similar in C-terminus to Cj1235 (30.2% identity in 222 aa overlap); Updated (2006) note: Pfam domain PF01551 Peptidase family M23 identified within CDS. Product modified to more specific family member due to motif match.  [...] ","uniprotId":"Q0PC06_CAMJE"},"10641004":{"id":10641004,"externalId":"192222.Cj0132","name":"lpxC","hasFamily":true,"annotation":"Catalyzes the hydrolysis of UDP-3-O-myristoyl-N- acetylglucosamine to form UDP-3-O-myristoylglucosamine and acetate, the committed step in lipid A biosynthesis","uniprotId":"LPXC_CAMJE"},"10641005":{"id":10641005,"externalId":"192222.Cj0133","name":"Cj0133","hasFamily":true,"annotation":"Putative glycoprotease family protein; Original (2000) note: Cj0133, unknown, len: 133 aa; 33.6% identity to HP1051; Updated (2006) note: Pfam domain PF00814 Glycoprotease family identified within CDS. Further support given to product function. No specific characterisation with acceptable identity score carried out yet. Putative kept within product function. Functional classification - Degradation of macromolecules - Proteins,peptides and glycopeptides","uniprotId":"Q0PC04_CAMJE"},"10641006":{"id":10641006,"externalId":"192222.Cj0134","name":"thrB","hasFamily":true,"annotation":"Catalyzes the ATP-dependent phosphorylation of L-homoserine to L-homoserine phosphate","uniprotId":"KHSE_CAMJE"},"10641007":{"id":10641007,"externalId":"192222.Cj0135","name":"Cj0135","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0135, unknown, len: 85 aa; 44.6% identity to HP1049; Updated (2006) note: Pfam PF04296 Protein of unknown function (DUF448) was present within CDS. Conserved added to product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PC02_CAMJE"},"10641008":{"id":10641008,"externalId":"192222.Cj0136","name":"infB","hasFamily":true,"annotation":"One of the essential components for the initiation of protein synthesis. Protects formylmethionyl-tRNA from spontaneous hydrolysis and promotes its binding to the 30S ribosomal subunits. Also involved in the hydrolysis of GTP during the formation of the 70S ribosomal complex","uniprotId":"IF2_CAMJE"},"10641009":{"id":10641009,"externalId":"192222.Cj0137","name":"rbfA","hasFamily":true,"annotation":"One of several proteins that assist in the late maturation steps of the functional core of the 30S ribosomal subunit. Associates with free 30S ribosomal subunits (but not with 30S subunits that are part of 70S ribosomes or polysomes). Required for efficient processing of 16S rRNA. May interact with the 5'-terminal helix region of 16S rRNA","uniprotId":"RBFA_CAMJE"},"10641010":{"id":10641010,"externalId":"192222.Cj0138","name":"rimP","hasFamily":true,"annotation":"Required for maturation of 30S ribosomal subunits. ","uniprotId":"RIMP_CAMJE"},"10641011":{"id":10641011,"externalId":"192222.Cj0139","name":"Cj0139","hasFamily":true,"annotation":"Putative endonuclease; Original (2000) note: Cj0139, possible endonuclease, len: 783 aa; similar in part to MCRB_ECOLI 5-methylcytosine-specific restriction enzyme B (465 aa),fasta scores; opt: 305 z-score: 505.4 E(): 6.7e-21, 40.9% identity in 186 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF07728 ATPase family associated with various cellular activities was identified within CDS. Characterised within Eschereichia coli with acceptable identity scores. Sequence alignment was only partial. Thus, putative kept within product function. Functional classification - DNA replic [...] ","uniprotId":"Q0PBZ8_CAMJE"},"10641012":{"id":10641012,"externalId":"192222.Cj0140","name":"Cj0140","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0140, unknown, len: 443 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PBZ7_CAMJE"},"10641013":{"id":10641013,"externalId":"192222.Cj0141c","name":"Cj0141c","hasFamily":true,"annotation":"Original (2000) note: Cj0141c, ABC transporter integral membrane protein, len: 267 aa; similar to many e.g. MNTB_SYNY3 manganese transport system membrane protein (306 aa), fasta scores; opt: 425 z-score: 449.6 E(): 8.7e-18, 25.8% identity in 264 aa overlap. No Hp match. Contains Pfam match to entry PF00950 ABC-3, ABC 3 transport family, score 276.20, E-value 4.3e-79; Updated (2006) note: Seven probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. No specific characterisation with acceptable identity score identified. Thus, putative added to p [...] ","uniprotId":"Q0PBZ6_CAMJE"},"10641014":{"id":10641014,"externalId":"192222.Cj0142c","name":"Cj0142c","hasFamily":true,"annotation":"Original (2000) note: Cj0142c, ABC transporter ATP-binding protein, len: 284 aa; simlar to many e.g. MNTA_SYNY3 manganese transport system ATP-binding protein (260 aa), fasta scores; opt: 417 z-score: 613.7 E(): 6.2e-27, 32.9% identity in 252 aa overlap. No Hp ortholog but 30.4% identity to HP0250 oligopeptide ABC transporter. Contains PS00017 ATP/GTP-binding site motif A (P-loop),PS00211 ABC transporters family signature and Pfam match to entry PF00005 ABC_tran, ABC transporters, score 165.90,E-value 6.9e-46; Updated (2006) note: Some characterisation within Bacillus subtilis with acc [...] ","uniprotId":"Q0PBZ5_CAMJE"},"10641015":{"id":10641015,"externalId":"192222.Cj0143c","name":"Cj0143c","hasFamily":true,"annotation":"Original (2000) note: Cj0143c, probable periplasmic solute binding protein for ABC transport system, len: 296 aa; similar to e.g. ZNUA_ECOLI high-affinity zinc uptake system protein znuA (328 aa), fasta scores; opt: 386 z-score: 471.6 E(): 5.2e-19, 26.4% identity in 288 aa overlap. No Hp match. Contains probable N-terminal signal sequence and Pfam match to entry PF01297 Lipoprotein_4,score 133.20, E-value 4.6e-36; Updated (2006) note: Pfam domain PPF01297 Periplasmic solute binding protein identified within CDS. Further support given to product function. Some characterisation within Es [...] ","uniprotId":"Q0PBZ4_CAMJE"},"10641016":{"id":10641016,"externalId":"192222.Cj0144","name":"Cj0144","hasFamily":true,"annotation":"Original (2000) note: Cj0144, probable methyl-accepting chemotaxis signal transduction protein,len: 659 aa; similar to mnay e.g. MCPS_ENTAE methyl-accepting chemotaxis serine transducer (557 aa),fasta scores; opt: 297 z-score: 493.4 E(): 3.2e-20, 28.6% identity in 377 aa overlap. No Hp ortholog. C-terminus contains repeat1, and is identical to the other repeat 1-containing genes Cj0262c and Cj1564. Contains N-terminal signal sequence and transmembrane anchor around aa 300. Also contains Pfam match to entry PF00015 MCPsignal,Methyl-accepting chemotaxis protein (MCP) signaling domain, sc [...] ","uniprotId":"Q0PBZ3_CAMJE"},"10641017":{"id":10641017,"externalId":"192222.Cj0145","name":"Cj0145","hasFamily":true,"annotation":"Putative TAT (Twin-Arginine Translocation) pathway signal sequence domain protein; Original (2000) note: Cj0145, unknown, len: 593 aa; some similarity to a hypothetical protein from Streptomyces coelicolor TR:O69840 (EMBL:AL023517) SC1B5.12C (465 aa), fasta scores; opt: 124 z-score: 327.1 E(): 5.8e-11, 23.0% identity in 574 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF05787 Bacterial protein of unknown function (DUF83), identified within CDS. TIGRFAM TIGR01409 TAT_signal_seq, Tat (twin-arginine translocation) pathway signal sequence identified within CDS. Product modifie [...] ","uniprotId":"Q0PBZ2_CAMJE"},"10641018":{"id":10641018,"externalId":"192222.Cj0146c","name":"trxB","hasFamily":true,"annotation":"Thioredoxin reductase; Original (2000) note: Cj0146c, trxB, probable thioredoxin reductase, len: 312 aa; similar to many e.g. TRXB_LISMO thioredoxin reductase (EC 1.6.4.5) (319 aa),fasta scores; opt: 749 z-score: 926.7 E(): 0, 38.0% identity in 313 aa overlap. 64.2% identity to HP0825. Contains PS00573 Pyridine nucleotide-disulphide oxidoreductases class-II active site; Updated (2006) note: Pfam domain PF00070 Pyridine nucleotide-disulphide oxidoreductase identified within CDS. Further support given to product function. Characterised within Listeria monocytogenes with acceptable identi [...] ","uniprotId":"Q0PBZ1_CAMJE"},"10641019":{"id":10641019,"externalId":"192222.Cj0147c","name":"trxA","hasFamily":true,"annotation":"Belongs to the thioredoxin family. ","uniprotId":"Q0PBZ0_CAMJE"},"10641020":{"id":10641020,"externalId":"192222.Cj0148c","name":"Cj0148c","hasFamily":true,"annotation":"Belongs to the UPF0102 family","uniprotId":"Y148_CAMJE"},"10641021":{"id":10641021,"externalId":"192222.Cj0149c","name":"hom","hasFamily":true,"annotation":"Homoserine dehydrogenase; Original (2000) note: Cj0149c, hom, probable homoserine dehydrogenase, len: 415 aa; similar to many e.g. DHOM_BACSU homoserine dehydrogenase (EC 1.1.1.3) (433 aa), fasta scores; opt: 932 z-score: 1191.7 E(): 0, 36.4% identity in 426 aa overlap. 53.3% identity to HP0822. Contains PS01042 Homoserine dehydrogenase signature and Pfam match to entry PF00742 Homoserine_dh, Homoserine dehydrogenase, score 329.20, E-value 4.7e-95; Updated (2006) note: Characterised within Bacillus subtilis with acceptable identity score. Putative not added to product function. Functio [...] ","uniprotId":"Q0PBY8_CAMJE"},"10641022":{"id":10641022,"externalId":"192222.Cj0150c","name":"Cj0150c","hasFamily":true,"annotation":"Aminotransferase; Original (2000) note: Cj0150c, probable aminotransferase, len: 400 aa; similar to many hypothetical aminotransferases and to e.g. AAT2_BACSU probable aspartate aminotransferase (EC 2.6.1.1) (399 aa),fasta scores; opt: 722 z-score: 1151.8 E(): 0, 33.8% identity in 390 aa overlap, and AAT_BACST aspartate aminotransferase (393 aa), fasta scores; opt: 513 z-score: 536.0 E(): 1.3e-22, 28.9% identity in 377 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00155 aminotran_1, Aminotransferases class-I, score 129.10,E-value 8.1e-35; Updated (2006) note: Characterised [...] ","uniprotId":"Q0PBY7_CAMJE"},"10641023":{"id":10641023,"externalId":"192222.Cj0151c","name":"Cj0151c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0151c, possible periplasmic protein, len: 268 aa; 26.0% identity to HP0555. Contains possible N-terminal signal sequence. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PBY6_CAMJE"},"10641024":{"id":10641024,"externalId":"192222.Cj0152c","name":"Cj0152c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0152c, possible membrane protein, len: 312 aa; 25.0% identity to HP0554. Contains one probable transmembrane domain around aa 110. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBY5_CAMJE"},"10641025":{"id":10641025,"externalId":"192222.Cj0153c","name":"Cj0153c","hasFamily":true,"annotation":"Belongs to the class IV-like SAM-binding methyltransferase superfamily. RNA methyltransferase TrmH family. ","uniprotId":"Q0PBY4_CAMJE"},"10641026":{"id":10641026,"externalId":"192222.Cj0154c","name":"rsmI","hasFamily":true,"annotation":"Catalyzes the 2'-O-methylation of the ribose of cytidine 1402 (C1402) in 16S rRNA","uniprotId":"Q0PBY3_CAMJE"},"10641027":{"id":10641027,"externalId":"192222.Cj0155c","name":"rpmE","hasFamily":true,"annotation":"Binds the 23S rRNA","uniprotId":"RL31_CAMJE"},"10641028":{"id":10641028,"externalId":"192222.Cj0156c","name":"Cj0156c","hasFamily":true,"annotation":"Specifically methylates the N3 position of the uracil ring of uridine 1498 (m3U1498) in 16S rRNA. Acts on the fully assembled 30S ribosomal subunit","uniprotId":"Q0PBY1_CAMJE"},"10641029":{"id":10641029,"externalId":"192222.Cj0157c","name":"Cj0157c","hasFamily":true,"annotation":"Original (2000) note: Cj0157c, probable integral membrane protein, len: 135 aa; no Hp match. Contains four probable transmembrane domains. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBY0_CAMJE"},"10641030":{"id":10641030,"externalId":"192222.Cj0158c","name":"Cj0158c","hasFamily":true,"annotation":"Original (2000) note: Cj0158c, possible haem-binding lipoprotein, len: 141 aa; no Hp match. Contains N-terminal signal sequence with appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site, PS00190 Cytochrome c family heme-binding site signature, and Pfam match to entry PF00034 cytochrome_c, Cytochrome c, score -3.00, E-value 0.87; Updated (2006) note: No specific characterisation with acceptable identity scores identified yet. Putative kept within product function. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PBX9_CAMJE"},"10641031":{"id":10641031,"externalId":"192222.Cj0159c","name":"Cj0159c","hasFamily":true,"annotation":"Putative 6-pyruvoyl tetrahydropterin synthase; Original (2000) note: Cj0159c, unknown, len: 193 aa; similar in N-terminus to hypothetical proteins e.g.TR:O27296 (EMBL:AE000890) Methanobacterium thermoautotrophicum MTH1228 (161 aa), fasta scores; opt: 140 z-score: 234.0 E(): 8.9e-06, 37.9% identity in 66 aa overlap. 35.8% identity to HP0933; Updated (2006) note: Pfam domain PF01242 6-pyruvoyl tetrahydropterin synthase was identified within CDS. Product modified to more specific family member due to motif match with own search. No specific characterisation with acceptable identity scores [...] ","uniprotId":"Q0PBX8_CAMJE"},"10641032":{"id":10641032,"externalId":"192222.Cj0160c","name":"queE","hasFamily":true,"annotation":"Catalyzes the complex heterocyclic radical-mediated conversion of 6-carboxy-5,6,7,8-tetrahydropterin (CPH4) to 7-carboxy-7- deazaguanine (CDG), a step common to the biosynthetic pathways of all 7-deazapurine-containing compounds","uniprotId":"QUEE_CAMJE"},"10641033":{"id":10641033,"externalId":"192222.Cj0161c","name":"moaA","hasFamily":true,"annotation":"Catalyzes the cyclization of GTP to (8S)-3',8-cyclo-7,8- dihydroguanosine 5'-triphosphate","uniprotId":"MOAA_CAMJE"},"10641034":{"id":10641034,"externalId":"192222.Cj0162c","name":"Cj0162c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0162c, probable periplasmic protein, len: 171 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PBX5_CAMJE"},"10641035":{"id":10641035,"externalId":"192222.Cj0163c","name":"Cj0163c","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0163c, unknown, len: 165 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PBX4_CAMJE"},"10641036":{"id":10641036,"externalId":"192222.Cj0164c","name":"ubiA","hasFamily":true,"annotation":"Belongs to the UbiA prenyltransferase family. ","uniprotId":"Q0PBX3_CAMJE"},"10641037":{"id":10641037,"externalId":"192222.Cj0166","name":"miaA","hasFamily":true,"annotation":"Catalyzes the transfer of a dimethylallyl group onto the adenine at position 37 in tRNAs that read codons beginning with uridine, leading to the formation of N6-(dimethylallyl)adenosine (i(6)A)","uniprotId":"MIAA_CAMJE"},"10641038":{"id":10641038,"externalId":"192222.Cj0167c","name":"mntP","hasFamily":true,"annotation":"Probably functions as a manganese efflux pump. ","uniprotId":"MNTP_CAMJE"},"10641039":{"id":10641039,"externalId":"192222.Cj0168c","name":"Cj0168c","hasFamily":false,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0168c, probable periplasmic protein, len: 55 aa; No Hp ortholog. Some similarity in C-term to Cj0909 (35.0% identity in 40 aa overlap). Contains probable N-terminal signal sequence; Updated (2006) note: Literature search identified paper giving further support to product function. N-terminal matches to Yersinia pestis, acid shock protein 2 precursor. Functional classification - Miscellaneous periplasmic proteins; PMID:15632442","uniprotId":"Q0PBX0_CAMJE"},"10641040":{"id":10641040,"externalId":"192222.Cj0169","name":"sodB","hasFamily":true,"annotation":"Destroys superoxide anion radicals which are normally produced within the cells and which are toxic to biological systems","uniprotId":"SODF_CAMJE"},"10641041":{"id":10641041,"externalId":"192222.Cj0170","name":"Cj0170","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0171, unknown, len: 166 aa; no Hp match. Contains polyporphic sequence G(9,11) at N-terminus; G11 allows translation from the upstream CDS Cj0170, G9 (consensus) would only allow this CDS to start at aa 47. Similar to Cj1326 (51.9% identity in 135 aa overlap), which also contains a polymorphic sequence allowing possible translation from Cj1325; Original (2000) note: Cj0170, unknown, len: 90 aa; no Hp match. Contains polyporphic sequence G(9,11) at C-terminus; G9 (consensus) gives this translation stop,G11 allows translation into the down [...] ","uniprotId":"Q0PBW8_CAMJE"},"10641042":{"id":10641042,"externalId":"192222.Cj0172c","name":"Cj0172c","hasFamily":true,"annotation":"Putative saccharopine dehydrogenase; Original (2000) note: Cj0172c, unknown, len: 401 aa; similar to hypothetical proteins e.g. TR:Q55131 (EMBL:D64001) Synechocystis sp. SLR0049 (398 aa), fasta scores; opt: 1534 z-score: 2324.2 E(): 0, 55.7% identity in 397 aa overlap. 63.8% identity to HP1507 conserved hypothetical ATP-binding protein; Updated (2006) note: Pfam domain PF03435 Saccharopine dehydrogenase was identified within CDS. Product modified to more specific family member due to motif match. No specific characterisation has yet been carried out. Putative kept within product functi [...] ","uniprotId":"Q0PBW7_CAMJE"},"10641043":{"id":10641043,"externalId":"192222.Cj0173c","name":"cfbpC","hasFamily":true,"annotation":"Original (2000) note: Cj0173c, possible iron-uptake ABC transport system ATP-binding protein, len: 302 aa; similar to many e.g. HITC_HAEIN iron(iii)-transport ATP-binding protein hitC (356 aa), fasta scores; opt: 389 z-score: 783.2 E(): 0, 38.0% identity in 250 aa overlap. Contains PS00017 ATP/GTP-binding site motif A (P-loop),PS00211 ABC transporters family signature and Pfam match to entry PF00005 ABC_tran, ABC transporters, score 163.10,E-value 4.9e-45. No Hp ortholog; Updated (2006) note: Literature search identified many supporting papers within Campylobacter jejuni (PMID:12069882 [...] ","uniprotId":"Q0PBW6_CAMJE"},"10641044":{"id":10641044,"externalId":"192222.Cj0174c","name":"cfbpB","hasFamily":true,"annotation":"Original (2000) note: Cj0174c, possible iron-uptake ABC transport system permease protein, len: 538 aa; similar to e.g. SFUB_SERMA iron(iii)-transport system permease protein (527 aa), fasta scores; opt: 333 z-score: 335.6 E(): 1.9e-11, 21.5% identity in 522 aa overlap. No Hp match. Contains Pfam match to entry PF00528 BPD_transp,Binding-protein-dependent transport systems inner membrane component, score 27.90, E-value 0.00024; Updated (2006) note: Literature search identified many supporting papers within Campylobacter jejuni (PMID:12069882) and also paper carrying out the original wo [...] ","uniprotId":"Q0PBW5_CAMJE"},"10641045":{"id":10641045,"externalId":"192222.Cj0175c","name":"cfbpA","hasFamily":true,"annotation":"Original (2000) note: Cj0175c, possible iron-uptake ABC transport system periplasmic iron-binding protein,len: 334 aa; similar to e.g. HITA_HAEIN iron-utilization periplasmic protein precursor (332 aa), fasta scores; opt: 466 z-score: 500.8 E(): 1.2e-20, 29.8% identity in 332 aa overlap. No Hp match. Contains N-terminal signal sequence; Updated (2006) note: Pfam domain PF01547 Bacterial extracellular solute-binding protein identified within CDS. Further support given to product function. Literature search identified many supporting papers within Campylobacter jejuni (PMID:12069882) and [...] ","uniprotId":"Q0PBW4_CAMJE"},"10641046":{"id":10641046,"externalId":"192222.Cj0176c","name":"Cj0176c","hasFamily":false,"annotation":"Putative lipoprotein; Original (2000) note: Cj0176c, probable lipoprotein, len: 43 aa; no Hp match. Contains N-terminal signal sequence with appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site; Updated (2006) note: Literature search identified papers giving further clues to product function. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBW3_CAMJE"},"10641047":{"id":10641047,"externalId":"192222.Cj0177","name":"Cj0177","hasFamily":true,"annotation":"Putative iron transport protein; Original (2000) note: Cj0177, probable lipoprotein,len: 283 aa; simlar to TR:O68876 (EMBL:AF055999) hypothetical protein PhuW from Pseudomonas aeruginosa hemin uptake locus (295 aa), fasta scores; opt: 375 z-score: 307.6 E(): 7.1e-10, 31.3% identity in 246 aa overlap. No Hp match. Contains N-terminal signal sequence with appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site; Updated (2006) note: Pfam domain PF04187 Protein of unknown function, DUF399 identified within CDS. Literature search identified papers giving furt [...] ","uniprotId":"Q0PBW2_CAMJE"},"10641048":{"id":10641048,"externalId":"192222.Cj0178","name":"Cj0178","hasFamily":true,"annotation":"Original (2000) note: Cj0178, possible outer membrane siderophore receptor, len: 755 aa; similar to e.g. YC17_HAEIN probable tonB-dependent receptor HI1217 (913 aa), fasta scores; opt: 440 z-score: 1818.4 E(): 0,32.7% identity in 912 aa overlap, and TR:O68881 (EMBL:AF055999) outer membrane hemin receptor PhuR from Pseudomonas aeruginosa (764 aa), fasta scores; opt: 145 z-score: 207.4 E(): 0.00027 22.3% identity in 806 aa overlap. No Hp match. Contains N-terminal signal sequence; Updated (2006) note: Pfam domains PF07715 TonB-dependent Receptor Plug Domain and PF00593 TonB dependent rec [...] ","uniprotId":"Q0PBW1_CAMJE"},"10641049":{"id":10641049,"externalId":"192222.Cj0179","name":"exbB1","hasFamily":true,"annotation":"Biopolymer transport protein; Original (2000) note: Cj0179, exbB1, biopolymer transport protein, len: 253 aa; highly similar to e.g. EXBB_ECOLI biopolymer transport exbB protein (244 aa),fasta scores; opt: 537 z-score: 811.2 E(): 0, 46.3% identity in 216 aa overlap. No Hp ortholog; Updated (2006) note: Pfam domain PF01618 MotA/TolQ/ExbB proton channel family identified within CDS. Also, three probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Characterised within Escherichia coli with acceptable identity score. Thus, putative not added. Cj0 [...] ","uniprotId":"Q0PBW0_CAMJE"},"10641050":{"id":10641050,"externalId":"192222.Cj0180","name":"exbD1","hasFamily":true,"annotation":"Biopolymer transport protein; Original (2000) note: Cj0180, exbD1, biopolymer transport protein, len: 136 aa; highly similar to e.g. EXBD_ECOLI biopolymer transport exbD protein (141 aa),fasta scores; opt: 356 z-score: 629.2 E(): 8.7e-28, 42.9% identity in 119 aa overlap. No Hp ortholog; Updated (2006) note: Pfam domain PF02472 Biopolymer transport protein ExbD/TolR identified within CDS. Also,one probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Characterised within Escherichia coli with acceptable identity scores. Thus, putative not adde [...] ","uniprotId":"Q0PBV9_CAMJE"},"10641051":{"id":10641051,"externalId":"192222.Cj0181","name":"tonB1","hasFamily":true,"annotation":"TonB transport protein; Original (2000) note: Cj0181, tonB1, possible tonB transport protein, len 248 aa; similar to e.g. TONB_PSEPU tonB protein (243 aa), fasta scores; opt: 220 z-score: 317.2 E(): 2.1e-10, 29.5% identity in 149 aa overlap. No Hp match; Updated (2006) note: One probable transmembrane helices predicted by TMHMM2.0. Characterised within Pseudomonas putida with marginal identity score. Putative not added. Cj0179-Cj0181 (exbB1, exbD1, tonB1) function as an outer-membrane energy transducer system. Functional classification - Transport/binding proteins - Other; PMID:7642501 [...] ","uniprotId":"Q0PBV8_CAMJE"},"10641052":{"id":10641052,"externalId":"192222.Cj0182","name":"Cj0182","hasFamily":true,"annotation":"Original (2000) note: Cj0182, probable transmembrane transport protein, len: 401 aa; similar to e.g. SBMA_ECOLI sbmA protein (406 aa), fasta scores; opt: 450 z-score: 1235.4 E(): 0, 35.4% identity in 395 aa overlap. No Hp ortholog; Updated (2006) note: Prosite domain PS50929, ABC transporter integral membrane type-1 fused domain profile identified within CDS. Six probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. No specific characterisation with acceptable identity scores (with full sequence alignment), has been carried out yet. Functional [...] ","uniprotId":"Q0PBV7_CAMJE"},"10641053":{"id":10641053,"externalId":"192222.Cj0183","name":"Cj0183","hasFamily":true,"annotation":"Original (2000) note: Cj0183, probable integral membrane protein with haemolysin domain, len: 452 aa; similar to many members of the UPF0053 family e.g. YUGS_BACSU (434 aa), fasta scores; opt: 999 z-score: 1508.9 E(): 0, 38.1% identity in 430 aa overlap. Also similar in C-terminal half to haemolysins e.g. HLYC_TREHY Treponema hyodysenteriae hemolysin C (268 aa), fasta scores; opt: 447 z-score: 468.1 E(): 8.1e-19, 32.5% identity in 237 aa overlap. 54.3% identity to HP1490. Contains 2x Pfam match to entry PF00571 CBS, CBS domain,scores 26.10, E-value 0.00082 and 40.50, E-value 3.7e-08; U [...] ","uniprotId":"Q0PBV6_CAMJE"},"10641054":{"id":10641054,"externalId":"192222.Cj0184c","name":"Cj0184c","hasFamily":true,"annotation":"Original (2000) note: Cj0184c, possible serine/threonine protein phosphatase, len: 384 aa; similar in part to many eukarotic serine/threonine protein phosphatases e.g. PP12_SCHPO serine/threonine protein phosphatase PP1-2 (322 aa), fasta scores; opt: 140 z-score: 290.0 E(): 6.8e-09, 31.3% identity in 131 aa overlap. No Hp match. Contains Pfam match to entry PF00149 STphosphatase, Ser/Thr protein phosphatases, score 6.90,E-value 0.39; Updated (2006) note: Pfam domain PF00149 STphosphatase was changed in Pfam database to Calcineurin-like phosphoesterase. Characterisation work has been ca [...] ","uniprotId":"Q0PBV5_CAMJE"},"10641055":{"id":10641055,"externalId":"192222.Cj0185c","name":"Cj0185c","hasFamily":true,"annotation":"Putative phnA domain protein; Original (2000) note: Cj0185c, unknown, len: 69 aa; similar to part of phnA proteins (involved in alkylphosphonate uptake) from several organisms e.g. PHNA_ECOLI phnA protein (111 aa), fasta scores; opt: 238 z-score: 482.4 E(): 1.3e-19, 55.9% identity in 68 aa overlap. 45.6% identity in 68 aa overlap to HP0872 (109 aa); Updated (2006) note: Pfam domain PF03831 PhnA protein was identified within CDS. Characterisation work within Escherichia coli has been carried out with acceptable identity scores. However, sequence alignment was only partial. Product chang [...] ","uniprotId":"Q0PBV4_CAMJE"},"10641056":{"id":10641056,"externalId":"192222.Cj0186c","name":"Cj0186c","hasFamily":true,"annotation":"Original (2000) note: Cj0186c, probable integral membrane protein, len: 239 aa; similar to many hypothetical proteins e.g. YGDQ_HAEIN HI0056 (237 aa),fasta scores; opt: 964 z-score: 1727.7 E(): 0, 64.3% identity in 230 aa overlap. 50.0% identity to HP1343; Updated (2006) note: Pfam domain PF03741 Integral membrane protein TerC family was identified within CDS. Seven probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Product modified to more specific family member based on motif match. No specific characterisation with acceptable identity sc [...] ","uniprotId":"Q0PBV3_CAMJE"},"10641057":{"id":10641057,"externalId":"192222.Cj0187c","name":"purN","hasFamily":true,"annotation":"Catalyzes the transfer of a formyl group from 10- formyltetrahydrofolate to 5-phospho-ribosyl-glycinamide (GAR), producing 5-phospho-ribosyl-N-formylglycinamide (FGAR) and tetrahydrofolate","uniprotId":"Q0PBV2_CAMJE"},"10641058":{"id":10641058,"externalId":"192222.Cj0188c","name":"Cj0188c","hasFamily":true,"annotation":"Bifunctional enzyme that catalyzes the epimerization of the S- and R-forms of NAD(P)HX and the dehydration of the S-form of NAD(P)HX at the expense of ADP, which is converted to AMP. This allows the repair of both epimers of NAD(P)HX, a damaged form of NAD(P)H that is a result of enzymatic or heat-dependent hydration. ","uniprotId":"Q0PBV1_CAMJE"},"10641059":{"id":10641059,"externalId":"192222.Cj0189c","name":"Cj0189c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0189c, unknown, len: 232 aa; similar to hypothetical proteins e.g. TR:P73622 (EMBL:D90908) Synechocystis sp. (249 aa), fasta scores; opt: 351 z-score: 629.7 E(): 8.1e-28, 30.1% identity in 246 aa overlap. No Hp match; Updated (2006) note: Pfam PF01936 Protein of unknown function (DUF88) was present within CDS. Conserved added to product function. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PBV0_CAMJE"},"10641060":{"id":10641060,"externalId":"192222.Cj0190c","name":"Cj0190c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0190c, unknown, len: 501 aa; similar to prokaryotic hypothetical proteins e.g. YIFB_ECOLI (516 aa), fasta scores; opt: 670 z-score: 1088.3 E(): 0, 34.0% identity in 497 aa overlap, and more distantly to members of the eukaryotic MCM family of DNA replication regulatory factors e.g. MCM6_RAT DNA replication licensing factor MCM6 (509 aa), fasta scores; opt: 152 z-score: 247.8 E(): 1.5e-06, 23.5% identity in 345 aa overlap. 47.7% identity to HP0792 (anotated as sigma-54 interacting protein). Contains PS00017 ATP/GTP-binding site motif A (P [...] ","uniprotId":"Q0PBU9_CAMJE"},"10641061":{"id":10641061,"externalId":"192222.Cj0191c","name":"def","hasFamily":true,"annotation":"Removes the formyl group from the N-terminal Met of newly synthesized proteins. Requires at least a dipeptide for an efficient rate of reaction. N-terminal L-methionine is a prerequisite for activity but the enzyme has broad specificity at other positions. ","uniprotId":"DEF_CAMJE"},"10641062":{"id":10641062,"externalId":"192222.Cj0192c","name":"clpP","hasFamily":true,"annotation":"Cleaves peptides in various proteins in a process that requires ATP hydrolysis. Has a chymotrypsin-like activity. Plays a major role in the degradation of misfolded proteins. ","uniprotId":"CLPP_CAMJE"},"10641063":{"id":10641063,"externalId":"192222.Cj0193c","name":"tig","hasFamily":true,"annotation":"Involved in protein export. Acts as a chaperone by maintaining the newly synthesized protein in an open conformation. Functions as a peptidyl-prolyl cis-trans isomerase (By similarity). ","uniprotId":"TIG_CAMJE"},"10641064":{"id":10641064,"externalId":"192222.Cj0194","name":"folE","hasFamily":true,"annotation":"Original (2000) note: Cj0194, folE, GTP cyclohydrolase I, len: 190 aa; identical to, but slightly longer than, GCH1_CAMJE (179 aa), and highly similar to many e.g. GCH1_SYNY3 GTP cyclohydrolase I (EC 3.5.4.16) (234 aa), fasta scores; opt: 760 z-score: 1327.9 E(): 0,60.8% identity in 181 aa overlap. 49.7% identity to HP0928. Contains PS00859 GTP cyclohydrolase I signature 1 and Pfam match to entry PF01227 GTP_cyclohydro_I, GTP cyclohydrolase I, score 377.10, E-value 1.8e-109; Updated (2006) note: Characterised within Campylobacter jejuni and many others. Putative not added. Functional c [...] ","uniprotId":"GCH1_CAMJE"},"10641065":{"id":10641065,"externalId":"192222.Cj0195","name":"fliI","hasFamily":true,"annotation":"Original (2000) note: Cj0195, fliI,flagellum-specific ATP synthase, len: FLII_ECOLI flagellum-specific ATP synthase (EC 3.6.1.34) (457 aa),fasta scores; opt: 1115 z-score: 1219.0 E(): 0, 43.8% identity in 443 aa overlap. 63.9% identity to HP1420. Contains PS00017 ATP/GTP-binding site motif A (P-loop),PS00152 ATP synthase alpha and beta subunits signature and Pfam match to entry PF00006 ATP-synt_ab, ATP synthase alpha and beta subunits, score 349.10, E-value 4.7e-101; Updated (2006) note: Characterised within Helicobacter pylori with acceptable identity score. Putative not added to prod [...] ","uniprotId":"Q0PBU4_CAMJE"},"10641066":{"id":10641066,"externalId":"192222.Cj0196c","name":"purF","hasFamily":true,"annotation":"Catalyzes the formation of phosphoribosylamine from phosphoribosylpyrophosphate (PRPP) and glutamine","uniprotId":"Q0PBU3_CAMJE"},"10641067":{"id":10641067,"externalId":"192222.Cj0197c","name":"dapB","hasFamily":true,"annotation":"Catalyzes the conversion of 4-hydroxy-tetrahydrodipicolinate (HTPA) to tetrahydrodipicolinate","uniprotId":"DAPB_CAMJE"},"10641068":{"id":10641068,"externalId":"192222.Cj0198c","name":"Cj0198c","hasFamily":true,"annotation":"Helicase-like protein; Original (2000) note: Cj0198c, unknown, len: 393 aa; similar to hypothetical proteins e.g. YCAJ_ECOLI (447 aa), fasta scores; opt: 339 z-score: 796.4 E(): 0, 32.8% identity in 381 aa overlap, and less strongly to several helicases e.g. RUVB_HELPY holliday junction DNA helicase ruvB (336 aa), fasta scores; opt: 204 z-score: 280.5 E(): 2.3e-08, 28.8% identity in 278 aa overlap. 49.9% identity to HP1026. Contains PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Pfam domain PF00004 ATPase family associated with various cellular activities (AAA),was [...] ","uniprotId":"Q0PBU1_CAMJE"},"10641069":{"id":10641069,"externalId":"192222.Cj0199c","name":"Cj0199c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0199c, probable periplasmic protein, len: 373 aa; no Hp match. Contains N-terminal signal sequence. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PBU0_CAMJE"},"10641070":{"id":10641070,"externalId":"192222.Cj0200c","name":"Cj0200c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0200c, probable periplasmic protein, len: 115 aa; no Hp match. Contains N-terminal signal sequence; Updated (2006) note: Literature search papers giving further clues to product function. Paper (PMID:12186869) links protein to glycoprotein. Functional classification - Miscellaneous periplasmic proteins; PMID:15554967, PMID:12186869","uniprotId":"Q0PBT9_CAMJE"},"10641071":{"id":10641071,"externalId":"192222.Cj0201c","name":"Cj0201c","hasFamily":true,"annotation":"Original (2000) note: Cj0201c, probable integral membrane protein, len: 201 aa; no Hp match. Contains possible transmembrane domain at N-terminus and PS00216 Sugar transport proteins signature 1. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBT8_CAMJE"},"10641072":{"id":10641072,"externalId":"192222.Cj0202c","name":"Cj0202c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0202c, unknown, len: 113 aa. No Hp match. Functional classification - Unknown","uniprotId":"Q0PBT7_CAMJE"},"10641073":{"id":10641073,"externalId":"192222.Cj0203","name":"Cj0203","hasFamily":true,"annotation":"Putative citrate transporter; Original (2000) note: Cj0203, probable transmembrane transport protein, len: 448 aa; some similarity to e.g. CITM_BACSU Mg2+/citrate complex secondary transporter (433 aa), fasta scores; opt: 702 z-score: 964.3 E(): 0, 28.7% identity in 449 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF03600 Citrate transporter was identified within CDS. Twelve probable transmembrane helices predicted by TMHMM2.0. Product modified to more specific family member due to motif match. Some characterisation work has been carried out within Bacillus subtilis, howev [...] ","uniprotId":"Q0PBT6_CAMJE"},"10641074":{"id":10641074,"externalId":"192222.Cj0204","name":"Cj0204","hasFamily":true,"annotation":"Putative oligopeptide transporter, OPT family; Original (2000) note: Cj0204, probable integral membrane protein, len: 665 aa; 98.9% identity in 369 aa overlap to TR:Q46110 (EMBL:X76062) C. jejuni ORF CJO1.1 (381 aa). Similar to hypothetical proteins e.g. Y561_HAEIN HI0561/560 (633 aa), fasta scores; opt: 2119 z-score: 1501.8 E(): 0, 53.6% identity in 640 aa overlap, and in C-terminus to TR:Q51156 (EMBL:L09189) Neisseria meningitidis ORF associated with capsule gene complex (282 aa), fasta scores; opt: 880 z-score: 1097.6 E(): 0, 51.3% identity in 273 aa overlap. No Hp match; Updated (2 [...] ","uniprotId":"Q0PBT5_CAMJE"},"10641075":{"id":10641075,"externalId":"192222.Cj0205","name":"uppP","hasFamily":true,"annotation":"Catalyzes the dephosphorylation of undecaprenyl diphosphate (UPP). Confers resistance to bacitracin","uniprotId":"UPPP_CAMJE"},"10641076":{"id":10641076,"externalId":"192222.Cj0206","name":"thrS","hasFamily":true,"annotation":"Catalyzes the attachment of threonine to tRNA(Thr) in a two- step reaction: L-threonine is first activated by ATP to form Thr-AMP and then transferred to the acceptor end of tRNA(Thr). Also edits incorrectly charged L-seryl-tRNA(Thr)","uniprotId":"SYT_CAMJE"},"10641077":{"id":10641077,"externalId":"192222.Cj0207","name":"infC","hasFamily":true,"annotation":"IF-3 binds to the 30S ribosomal subunit and shifts the equilibrum between 70S ribosomes and their 50S and 30S subunits in favor of the free subunits, thus enhancing the availability of 30S subunits on which protein synthesis initiation begins. ","uniprotId":"IF3_CAMJE"},"10641078":{"id":10641078,"externalId":"192222.Cj0208","name":"Cj0208","hasFamily":true,"annotation":"Original (2000) note: Cj0208, probable DNA modification methylase (adenine-specific methyltransferase), len: 363 aa; similar to many e.g. MTN3_NEILA modification methylase NlaIII (EC 2.1.1.72) (334 aa), fasta scores; opt: 215 z-score: 401.1 E(): 4.4e-15, 31.7% identity in 366 aa overlap. 29.6% identity to HP1208 ulcer associated adenine specific DNA methyltransferase. Contains PS00092 N-6 Adenine-specific DNA methylases signature; Updated (2006) note: Pfam domain PF02086 D12 class N6 adenine-specific DNA methyltransferase was identified within CDS. Further support given to product func [...] ","uniprotId":"Q0PBT1_CAMJE"},"10641079":{"id":10641079,"externalId":"192222.Cj0224","name":"argC","hasFamily":true,"annotation":"Catalyzes the NADPH-dependent reduction of N-acetyl-5- glutamyl phosphate to yield N-acetyl-L-glutamate 5-semialdehyde. ","uniprotId":"ARGC_CAMJE"},"10641080":{"id":10641080,"externalId":"192222.Cj0225","name":"Cj0225","hasFamily":true,"annotation":"Putative acetyltransferase; Original (2000) note: Cj0225, probable acetyltransferase, len: 148 aa; some simlarity to STA_ECOLI streptothricin acetyltransferase (plasmid borne)(EC 2.3.-.-) (174 aa), fasta scores; opt: 118 z-score: 186.2 E(): 0.0041, 27.3% identity in 110 aa overlap. No Hp match. Contains Pfam match to entry PF00583 Acetyltransf, Acetyltransferase (GNAT) family, score 57.00, E-value 4.1e-13; Updated (2006) note: Characterisation carried out within Escherichia coil, however, identity score was marginal. Putative kept within product function. Functional classification - Mi [...] ","uniprotId":"Q0PBS9_CAMJE"},"10641081":{"id":10641081,"externalId":"192222.Cj0226","name":"argB","hasFamily":true,"annotation":"Catalyzes the ATP-dependent phosphorylation of N-acetyl-L- glutamate","uniprotId":"ARGB_CAMJE"},"10641082":{"id":10641082,"externalId":"192222.Cj0227","name":"argD","hasFamily":true,"annotation":"Belongs to the class-III pyridoxal-phosphate-dependent aminotransferase family. ArgD subfamily","uniprotId":"ARGD_CAMJE"},"10641083":{"id":10641083,"externalId":"192222.Cj0228c","name":"pcm","hasFamily":true,"annotation":"Original (2000) note: Cj0228c, pcm, probable protein-L-isoaspartate O-methyltransferase, len: 209 aa PIMT_ECOLI protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77) (207 aa), fasta scores; opt: 463 z-score: 633.3 E(): 5.1e-28, 35.3% identity in 204 aa overlap. 53.8% identity to HP0363. Contains Pfam match to entry PF01135 PCMT, Protein-L-isoaspartate(D-aspartate) O-methyltransferase (PCMT), score 181.40, E-value 1.4e-50; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Putative not added to product function. Functional classification -Prote [...] ","uniprotId":"Q0PBS6_CAMJE"},"10641084":{"id":10641084,"externalId":"192222.Cj0229","name":"Cj0229","hasFamily":true,"annotation":"Putative acetyltransferase; Original (2000) note: Cj0229, possible acetyltransferase, len: 182 aa; similar to e.g. CAIE_ECOLI carnitine operon protein caiE (203 aa), fasta scores; opt: 251 z-score: 517.5 E(): 1.4e-21, 33.1% identity in 169 aa overlap. No Hp ortholog. Contains 3x Pfam match to entry PF00132 hexapep, Bacterial transferase hexapeptide (four repeats), scores 16.60, E-value 0.0071; 9.10, E-value 0.97 and 24.70, E-value 3.7e-05; Updated (2006) note: Some characterisation work within Escherichia coli, however, identity score was marginal. Putative kept within product function [...] ","uniprotId":"Q0PBS5_CAMJE"},"10641085":{"id":10641085,"externalId":"192222.Cj0230c","name":"Cj0230c","hasFamily":true,"annotation":"Catalyzes the first step in the biosynthesis of NAD from nicotinic acid, the ATP-dependent synthesis of beta-nicotinate D- ribonucleotide from nicotinate and 5-phospho-D-ribose 1-phosphate. ","uniprotId":"Q0PBS4_CAMJE"},"10641086":{"id":10641086,"externalId":"192222.Cj0231c","name":"nrdB","hasFamily":true,"annotation":"Provides the precursors necessary for DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding ribonucleotides","uniprotId":"Q0PBS3_CAMJE"},"10641087":{"id":10641087,"externalId":"192222.Cj0232c","name":"Cj0232c","hasFamily":true,"annotation":"Original (2000) note: Cj0232c, probable integral membrane protein, len: 136 aa; 37.9% identity to HP1258 (called conserved hypothetical mitochondrial protein 4). Contains three possible transmembrane domains; Updated (2006) note: Pfam domain PF06271 RDD family was identified within CDS. This family of proteins contain three highly conserved amino acids: one arginine and two aspartates, hence the name of RDD family. This region contains two predicted transmembrane regions. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBS2_CAMJE"},"10641088":{"id":10641088,"externalId":"192222.Cj0233c","name":"pyrE","hasFamily":true,"annotation":"Catalyzes the transfer of a ribosyl phosphate group from 5- phosphoribose 1-diphosphate to orotate, leading to the formation of orotidine monophosphate (OMP)","uniprotId":"PYRE_CAMJE"},"10641089":{"id":10641089,"externalId":"192222.Cj0234c","name":"frr","hasFamily":true,"annotation":"Responsible for the release of ribosomes from messenger RNA at the termination of protein biosynthesis. May increase the efficiency of translation by recycling ribosomes from one round of translation to another","uniprotId":"RRF_CAMJE"},"10641090":{"id":10641090,"externalId":"192222.Cj0235c","name":"secG","hasFamily":true,"annotation":"Involved in protein export. Participates in an early event of protein translocation","uniprotId":"Q0PBR9_CAMJE"},"10641091":{"id":10641091,"externalId":"192222.Cj0236c","name":"Cj0236c","hasFamily":true,"annotation":"Uncharacterized protein Cj0236c; Original (2000) note: Cj0236c, probable integral membrane protein, len: 231 aa similar to members of the UPF0005 family e.g. YCCA_PSEAE (222 aa), fasta scores; opt: 377 z-score: 628.9 E(): 8.9e-28, 37.7% identity in 207 aa overlap. 49.1% identity to HP0920. Contains Pfam match to entry PF01027 UPF0005, Uncharacterized protein family, score 128.30, E-value 1.4e-34; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Y236_CAMJE"},"10641092":{"id":10641092,"externalId":"192222.Cj0237","name":"cynT","hasFamily":true,"annotation":"Reversible hydration of carbon dioxide. ","uniprotId":"Q0PBR7_CAMJE"},"10641093":{"id":10641093,"externalId":"192222.Cj0238","name":"Cj0238","hasFamily":true,"annotation":"Putative mechanosensitive ion channel family protein; Original (2000) note: Cj0238, probable integral membrane protein, len: 627 aa; similar in C-terminus to many hypothetical proteins, e.g. YB43_METJA MJ1143 (361 aa), fasta scores; opt: 413 z-score: 535.7 E(): 1.4e-22,23.9% identity in 372 aa overlap. 36.3% identity in 608 aa overlap to HP0415. Contains Pfam match to entry PF00924 UPF0003, Uncharacterized protein family UPF0003, score 145.90, E-value 6.9e-40; Updated (2006) note: Pfam domain PF00924 Mechanosensitive ion channel was identified within CDS. Product modified to more speci [...] ","uniprotId":"Q0PBR6_CAMJE"},"10641094":{"id":10641094,"externalId":"192222.Cj0239c","name":"Cj0239c","hasFamily":true,"annotation":"May be involved in the formation or repair of [Fe-S] clusters present in iron-sulfur proteins","uniprotId":"Q0PBR5_CAMJE"},"10641095":{"id":10641095,"externalId":"192222.Cj0240c","name":"iscS","hasFamily":true,"annotation":"Original (2000) note: Cj0240c, probable aminotransferase (nifS protein homolog), len: 393 aa; similar to many e.g. NIFS_KLEPN nifS protein (397 aa),fasta scores; opt: 1067 z-score: 1533.6 E(): 0, 47.0% identity in 383 aa overlap. 62.8% identity to HP0220. Contains PS00132 Zinc carboxypeptidases, zinc-binding region 1 signature, PS00595 Aminotransferases class-V pyridoxal-phosphate attachment site, and Pfam match to entry PF00266 aminotran_5, Aminotransferases class-V,score 358.80, E-value 5.9e-104; Updated (2006) note: Characterised within Escherichia coli with acceptable identity scor [...] ","uniprotId":"Q0PBR4_CAMJE"},"10641096":{"id":10641096,"externalId":"192222.Cj0241c","name":"Cj0241c","hasFamily":true,"annotation":"Oxygen-binding protein. May be involved in a storage mechanism or for delivery to oxygen-requiring enzymes. The oxygen- binding site contains two iron atoms","uniprotId":"HEMTB_CAMJE"},"10641097":{"id":10641097,"externalId":"192222.Cj0243c","name":"Cj0243c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0243c, unknown, len: 388 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PBR2_CAMJE"},"10641098":{"id":10641098,"externalId":"192222.Cj0244","name":"rpmI","hasFamily":true,"annotation":"Belongs to the bacterial ribosomal protein bL35 family. ","uniprotId":"RL35_CAMJE"},"10641099":{"id":10641099,"externalId":"192222.Cj0245","name":"rplT","hasFamily":true,"annotation":"Binds directly to 23S ribosomal RNA and is necessary for the in vitro assembly process of the 50S ribosomal subunit. It is not involved in the protein synthesizing functions of that subunit. ","uniprotId":"RL20_CAMJE"},"10641100":{"id":10641100,"externalId":"192222.Cj0246c","name":"Cj0246c","hasFamily":true,"annotation":"Original (2000) note: Cj0246c, probable MCP-domain signal transduction protein, len: 375 aa; similar in C-terminus to many MCP-domain containing proteins e.g. MCPA_BACSU methyl-accepting chemotaxis protein mcpA (661 aa), fasta scores; opt: 257 z-score: 408.0 E(): 1.8e-15,33.2% identity in 205 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00015 MCPsignal, Methyl-accepting chemotaxis protein (MCP) signaling domain, score 93.20,E-value 3.9e-25; Updated (2006) note: Characterisation within Bacillus subtilis, however, identity score was marginal. Putative kept within product fu [...] ","uniprotId":"Q0PBQ9_CAMJE"},"10641101":{"id":10641101,"externalId":"192222.Cj0247c","name":"Cj0247c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0247c, unknown, len: 177 aa; no Hp match; Updated (2006) note: Similar to proteins from other bacteria. Functional classification - Unknown","uniprotId":"Q0PBQ8_CAMJE"},"10641102":{"id":10641102,"externalId":"192222.Cj0248","name":"Cj0248","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0248, unknown, len: 285 aa; no Hp match; Updated (2006) note: Similar to proteins from other bacteria. Literature search identified papers giving structural characterisation and clues to product function. Currently unknown function but may be some kind of hydrolase or signal-transduction protein. Cj0248 has been implicated as a virulence factor and has been shown to play a role in motility. Functional classification -Unknown; PMID:16287129, PMID:15066034, PMID:11298288","uniprotId":"Q0PBQ7_CAMJE"},"10641103":{"id":10641103,"externalId":"192222.Cj0249","name":"Cj0249","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0249, unknown, len: 159 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PBQ6_CAMJE"},"10641104":{"id":10641104,"externalId":"192222.Cj0250c","name":"Cj0250c","hasFamily":true,"annotation":"Putative MFS (Major Facilitator Superfamily) transport protein; Original (2000) note: Cj0250c, probable transmembrane transport protein, len: 436 aa; similar to many members of the sugar transport protein family, e.g. PROP_ECOLI proline/betaine transporter (500 aa), fasta scores; opt: 602 z-score: 791.3 E(): 0, 31.7% identity in 394 aa overlapCIT1_SALTY citrate-proton symport (citrate transporter) (434 aa), fasta scores; opt: 580 z-score: 734.9 E(): 0, 28.0% identity in 396 aa overlap. 50.9% identity to HP0936. Contains Pfam match to entry PF00083 sugar_tr, Sugar (and other) transporte [...] ","uniprotId":"Q0PBQ5_CAMJE"},"10641105":{"id":10641105,"externalId":"192222.Cj0251c","name":"Cj0251c","hasFamily":true,"annotation":"Highly acidic protein; Original (2000) note: Cj0251c, highly acidic protein, len: 51 aa; 47.9% identity to HP0756. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PBQ4_CAMJE"},"10641106":{"id":10641106,"externalId":"192222.Cj0252","name":"moaC","hasFamily":true,"annotation":"Catalyzes the conversion of (8S)-3',8-cyclo-7,8- dihydroguanosine 5'-triphosphate to cyclic pyranopterin monophosphate (cPMP)","uniprotId":"MOAC_CAMJE"},"10641107":{"id":10641107,"externalId":"192222.Cj0253","name":"Cj0253","hasFamily":true,"annotation":"To H.pylori HP0495/JHP0447","uniprotId":"Y253_CAMJE"},"10641108":{"id":10641108,"externalId":"192222.Cj0254","name":"Cj0254","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0254, unknown, len: 386 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PBQ1_CAMJE"},"10641109":{"id":10641109,"externalId":"192222.Cj0255c","name":"exoA","hasFamily":true,"annotation":"Exodeoxyribonuclease; Original (2000) note: Cj0255c, probable exodeoxyribonuclease, len: 252 aa; similar to many e.g. EXOA_BACSU exodeoxyribonuclease (EC 3.1.11.2)(252 aa),fasta scores; opt: 676 z-score: 868.1 E(): 0, 40.5% identity in 259 aa overlap. 41.1% identity to HP1526. Contains Pfam match to entry PF01260 AP_endonulease1, AP endonucleases family 1, score 282.10, E-value 7.3e-81; Updated (2006) note: Pfam domain PF03372,Endonuclease/Exonuclease/phosphatase family identified within CDS. Further support given to product function. Characterised within Bacillus subtilis with accepta [...] ","uniprotId":"Q0PBQ0_CAMJE"},"10641110":{"id":10641110,"externalId":"192222.Cj0256","name":"Cj0256","hasFamily":true,"annotation":"Putative sulfatase family protein; Original (2000) note: Cj0256, probable integral membrane protein, len: 512 aa; similar to hypothetical proteins e.g. YJDB_ECOLI (547 aa), fasta scores; opt: 646 z-score: 1169.8 E(): 0, 38.9% identity in 530 aa overlap. 34.1% identity to HP0022; Updated (2006) note: Pfam domain PF00884 Sulfatase identified within CDS. Also, five probable transmembrane helices predicted by TMHMM2.0. Product modified to more specific family member based on motif match. No specific characterisation has been carried out yet, so putative kept within product function. Litera [...] ","uniprotId":"Q0PBP9_CAMJE"},"10641111":{"id":10641111,"externalId":"192222.Cj0257","name":"dgkA","hasFamily":true,"annotation":"Recycling of diacylglycerol produced during the turnover of membrane phospholipid","uniprotId":"Q0PBP8_CAMJE"},"10641112":{"id":10641112,"externalId":"192222.Cj0258","name":"Cj0258","hasFamily":true,"annotation":"Putative bacterial regulatory protein, ArsR family; Original (2000) note: Cj0258, possible DNA binding protein, len: 81 aa; no Hp match. Contains probable helix-turn-helix motif at aa 35-56 (Score 1128, +3.03 SD); Updated (2006) note: Pfam domain PF01022 Bacterial regulatory protein, arsR family was identified within CDS. This was found by carrying out own Pfam search. Product modified to more specific family member due to motif match. No specific characterisation has been carried out yet, so putative kept within product function. Literature search identified paper with some additional [...] ","uniprotId":"Q0PBP7_CAMJE"},"10641113":{"id":10641113,"externalId":"192222.Cj0259","name":"pyrC","hasFamily":true,"annotation":"Catalyzes the reversible cyclization of carbamoyl aspartate to dihydroorotate","uniprotId":"Q0PBP6_CAMJE"},"10641114":{"id":10641114,"externalId":"192222.Cj0260c","name":"Cj0260c","hasFamily":true,"annotation":"Small hydrophobic protein; Original (2000) note: Cj0260c, unknown, len: 71 aa; no Hp match, very hydrophobic; Updated (2006) note: Two probable transmembrane helices identified by TMHMM2.0. Functional classification - Unknown","uniprotId":"Q0PBP5_CAMJE"},"10641115":{"id":10641115,"externalId":"192222.Cj0261c","name":"Cj0261c","hasFamily":true,"annotation":"Putative SAM-dependent methyltransferase; Original (2000) note: Cj0261c, unknown, len: 239 aa; some similarity in N-terminus to YC73_HAEIN hypothetical protein HI1273 (268 aa), fasta scores; opt: 208 z-score: 260.9 E(): 2.8e-07, 29.0% identity in 176 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF08241 and PF08242 Methyltransferase domains were identified within CDS. ProfileScan also identified PS50193 SAM_BIND domain. Members from methyltransferase family are SAM dependent methyltransferases. This was found by carrying out own Pfam search. Product modified to more specifi [...] ","uniprotId":"Q0PBP4_CAMJE"},"10641116":{"id":10641116,"externalId":"192222.Cj0262c","name":"Cj0262c","hasFamily":true,"annotation":"Original (2000) note: Cj0262c, probable methyl-accepting chemotaxis signal transduction protein,len: 665 aa; similar to many e.g. TR:O32443 (EMBL:D86947) Pseudomonas aeruginosa chemotactic transducer (629 aa),fasta scores; opt: 283 z-score: 635.0 E(): 4.1e-28, 25.3% identity in 667 aa overlap, and MCP1_ECOLI methyl-accepting chemotaxis protein I (551 aa), fasta scores; opt: 280 z-score: 519.1 E(): 1.2e-21. 29.1% identity in 302 aa overlap. No Hp ortholog. C-terminus contains repeat1, and is identical to the other repeat 1-containing genes Cj0262c and Cj1564. Contains N-terminal signal  [...] ","uniprotId":"Q0PBP3_CAMJE"},"10641117":{"id":10641117,"externalId":"192222.Cj0263","name":"zupT","hasFamily":true,"annotation":"Mediates zinc uptake. May also transport other divalent cations (By similarity)","uniprotId":"ZUPT_CAMJE"},"10641118":{"id":10641118,"externalId":"192222.Cj0264c","name":"Cj0264c","hasFamily":true,"annotation":"Belongs to the prokaryotic molybdopterin-containing oxidoreductase family","uniprotId":"Q0PBP1_CAMJE"},"10641119":{"id":10641119,"externalId":"192222.Cj0265c","name":"Cj0265c","hasFamily":true,"annotation":"Original (2000) note: Cj0265c, probable cytochrome C-type haem-binding periplasmic protein, len: 191 aa; possibly part of electron-transport chain for oxidoreductase Cj0264c. Similar to C-terminus of many e.g. YECK_ECOLI probable cytochrome C-TYPE protein in bisZ-cutC intergenic region (366 aa), fasta scores; opt: 208 z-score: 216.4 E(): 8.5e-05, 27.0% identity in 163 aa overlap, and TORC_ECOLI cytochrome C-type protein torC (390 aa), fasta scores; opt: 264 z-score: 216.0 E(): 8.9e-05, 26.4% identity in 178 aa overlap. No Hp match. Contains N-terminal signal sequence and PS00190 Cytoch [...] ","uniprotId":"Q0PBP0_CAMJE"},"10641120":{"id":10641120,"externalId":"192222.Cj0266c","name":"Cj0266c","hasFamily":true,"annotation":"Original (2000) note: Cj0266c, probable integral membrane protein, len: 169 aa; no Hp match; Updated (2006) note: Two probable transmembrane helices identified by TMHMM2.0. Prosite PS00430 TONB_DEPENDENT_REC_1, TonB box, N-terminal also identified. No specific characterisation has been carried out yet. Putative kept within product function. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBN9_CAMJE"},"10641121":{"id":10641121,"externalId":"192222.Cj0267c","name":"Cj0267c","hasFamily":true,"annotation":"Original (2000) note: Cj0267c, probable integral membrane protein, len: 176 aa; no Hp match; Updated (2006) note: Two probable transmembrane helices identified by TMHMM2.0. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBN8_CAMJE"},"10641122":{"id":10641122,"externalId":"192222.Cj0268c","name":"Cj0268c","hasFamily":true,"annotation":"Putative transmembrane protein; Original (2000) note: Cj0268c, probable transmembrane protein, len: 362 aa; 55.9% identity to HP0248. Contains one transmembrane domain around aa 60; Updated (2006) note: Pfam domain PF01145 SPFH domain / Band was identified within CDS. FPrintScan PR00679 Prohibitin was also identified. The FPrintScan identified is a child of the Pfam protein. No specific characherisation with acceptable identity scores have been carried out yet. Putative kept within product function. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBN7_CAMJE"},"10641123":{"id":10641123,"externalId":"192222.Cj0269c","name":"ilvE","hasFamily":true,"annotation":"Acts on leucine, isoleucine and valine. ","uniprotId":"Q0PBN6_CAMJE"},"10641124":{"id":10641124,"externalId":"192222.Cj0270","name":"Cj0270","hasFamily":true,"annotation":"Belongs to the 4-oxalocrotonate tautomerase family. ","uniprotId":"Y270_CAMJE"},"10641125":{"id":10641125,"externalId":"192222.Cj0271","name":"Cj0271","hasFamily":true,"annotation":"Original (2000) note: Cj0271, bacterioferritin comigratory protein homolog, len: 151 aa; similar to many e.g. BCP_ECOLI bacterioferritin comigratory protein (156 aa), fasta scores; opt: 390 z-score: 675.8 E(): 2.2e-30,39.6% identity in 149 aa overlap. 57.0% identity to HP0136. Contains Pfam match to entry PF00578 AhpC-TSA,AhpC/TSA family, score 146.30, E-value 5.4e-40; Updated (2006) note: Some characterisation work within Escherichia coli shows Cj0271 to be putative peroxiredoxin (EC.1.11.1.15), bcp. Literature search also identified work within Helicobacter pylori. Product function k [...] ","uniprotId":"Q0PBN4_CAMJE"},"10641126":{"id":10641126,"externalId":"192222.Cj0272","name":"queH","hasFamily":true,"annotation":"Catalyzes the conversion of epoxyqueuosine (oQ) to queuosine (Q), which is a hypermodified base found in the wobble positions of tRNA(Asp), tRNA(Asn), tRNA(His) and tRNA(Tyr)","uniprotId":"Q0PBN3_CAMJE"},"10641127":{"id":10641127,"externalId":"192222.Cj0273","name":"fabZ","hasFamily":true,"annotation":"Involved in unsaturated fatty acids biosynthesis. Catalyzes the dehydration of short chain beta-hydroxyacyl-ACPs and long chain saturated and unsaturated beta-hydroxyacyl-ACPs","uniprotId":"FABZ_CAMJE"},"10641128":{"id":10641128,"externalId":"192222.Cj0274","name":"lpxA","hasFamily":true,"annotation":"Involved in the biosynthesis of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell","uniprotId":"LPXA_CAMJE"},"10641129":{"id":10641129,"externalId":"192222.Cj0275","name":"clpX","hasFamily":true,"annotation":"ATP-dependent specificity component of the Clp protease. It directs the protease to specific substrates. Can perform chaperone functions in the absence of ClpP","uniprotId":"CLPX_CAMJE"},"10641130":{"id":10641130,"externalId":"192222.Cj0276","name":"mreB","hasFamily":true,"annotation":"Original (2000) note: Cj0276, mreB, homolog of E. coli rod shape-determining protein, len: 346 aa; highly similar to many e.g. MREB_ECOLI rod shape-determining protein mreB (347 aa), fasta scores; opt: 1199 z-score: 1757.3 E(): 0, 55.8% identity in 339 aa overlap. 71.4% identity to HP1373. Contains PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Pfam domain PF06723 MreB/Mbl protein was identified within CDS. MreB is known to be a rod shape-determining protein in bacteria and makes up the bacterial cytoskeleton. Further support given to product function. Characterise [...] ","uniprotId":"Q0PBM9_CAMJE"},"10641131":{"id":10641131,"externalId":"192222.Cj0277","name":"mreC","hasFamily":true,"annotation":"Homolog of E. coli rod shape-determining protein; Original (2000) note: Cj0277, probable periplasmic protein, len: 249 aa; 30.2% identity to HP1372 annotated as rod shape-determining protein (mreC), no apparent match to any other mreC genes. Contains N-terminal signal sequence; Updated (2006) note: Pfam domain PF04085 rod shape-determining protein MreC was identified within CDS. MreC (murein formation C) is involved in the rod shape determination in E. coli, and more generally in cell shape determination of bacteria whether or not they are rod-shaped. Product changed due to motif match [...] ","uniprotId":"Q0PBM8_CAMJE"},"10641132":{"id":10641132,"externalId":"192222.Cj0279","name":"carB","hasFamily":true,"annotation":"Original (2000) note: Cj0279, carB, probable carbamoyl-phosphate synthase large chain, len: 1089 aa; highly similar to many e.g. CARB_ECOLI carbamoyl-phosphate synthase large chain (EC 6.3.5.5) (1072 aa), fasta scores; opt: 3268 z-score: 4517.4 E(): 0, 55.3% identity in 1098 aa overlap. 62.5% identity to HP0919. Contains 2x PS00866 Carbamoyl-phosphate synthase subdomain signature 1, 2x PS00867 Carbamoyl-phosphate synthase subdomain signature 2, and 2x Pfam match to entry PF00289 CPSase_L_chain,Carbamoyl-phosphate synthase (CPSase). Also contains apparent N-terminal signal sequence with [...] ","uniprotId":"CARB_CAMJE"},"10641133":{"id":10641133,"externalId":"192222.Cj0280","name":"Cj0280","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0280, unknown, len: 140 aa; 52.8% identity to HP0918. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PBM6_CAMJE"},"10641134":{"id":10641134,"externalId":"192222.Cj0281c","name":"tal","hasFamily":true,"annotation":"Transaldolase is important for the balance of metabolites in the pentose-phosphate pathway","uniprotId":"TAL_CAMJE"},"10641135":{"id":10641135,"externalId":"192222.Cj0282c","name":"serB","hasFamily":true,"annotation":"Putative phosphoserine phosphatase; Original (2000) note: Cj0282c, serB, possible phosphoserine phosphatase, len: 207 aa; similar to C-terminus of SERB_ECOLI phosphoserine phosphatase (EC 3.1.3.3) (322 aa), fasta scores; opt: 361 z-score: 572.4 E(): 1.3e-24, 34.3% identity in 204 aa overlap. 48.8% identity to HP0652; Updated (2006) note: Pfam domain PF00702 haloacid dehalogenase-like hydrolase were identified within CDS. TIGRFAM motif TIGR00338, serB phosphoserine phosphatase identified along with TIGR01488 HAD-SF-IB: HAD-superfamily hydrolase within CDS. Further support for product fu [...] ","uniprotId":"Q0PBM4_CAMJE"},"10641136":{"id":10641136,"externalId":"192222.Cj0283c","name":"cheW","hasFamily":true,"annotation":"Chemotaxis protein; Original (2000) note: Cj0283c, cheW, probable chemotaxis protein, len: 173 aa; similar to many e.g. CHEW_ECOLI purine-binding chemotaxis protein (167 aa),fasta scores; opt: 236 z-score: 304.4 E(): 1.1e-09, 32.4% identity in 139 aa overlap. 51.4% identity to HP0391; Updated (2006) note: Pfam domain PF01584 CheW-like domain identified within CDS. Further support given to product function. Characterised within Escherichia coli with marginal identity score. Putative not added to product function. Functional classification - Chemotaxis and mobility; PMID:2068106, PMID:3510184","uniprotId":"Q0PBM3_CAMJE"},"10641137":{"id":10641137,"externalId":"192222.Cj0284c","name":"cheA","hasFamily":true,"annotation":"Chemotaxis histidine kinase; Original (2000) note: Cj0284c, cheA, probable chemotaxis histidine kinase, len: 769 aa; similar to many e.g. CHEA_ECOLI chemotaxis protein cheA (EC 2.7.3.-) (654 aa), fasta scores; opt: 1022 z-score: 1416.0 E(): 0, 35.7% identity in 658 aa overlap. 64.6% identity to HP0392. Contains Pfam matches to entry PF00072 response_reg,Response regulator receiver domain, score 103.80, E-value 3.4e-27 and to entry PF00512 signal, Signal carboxyl-terminal domain, score -12.30, E-value 0.00013; Updated (2006) note: Characterised within Escherichia coli with marginal iden [...] ","uniprotId":"Q0PBM2_CAMJE"},"10641138":{"id":10641138,"externalId":"192222.Cj0285c","name":"cheV","hasFamily":true,"annotation":"Chemotaxis protein; Original (2000) note: Cj0285c, cheV, probable chemotaxis protein cheV, len: 318 aa; similar to e.g. CHEV_BACSU chemotaxis cheV protein (EC 2.7.3.-) (303 aa),fasta scores; opt: 414 z-score: 344.9 E(): 5.9e-12, 28.2% identity in 309 aa overlap. 50.8% identity to HP0393. Contains Pfam match to entry PF00072 response_reg,Response regulator receiver domain, score 50.70, E-value 3.1e-11; Updated (2006) note: Pfam domain PF01584 CheW-like domain identified within CDS. Further support given to product function. Characterised within Bacillus subtilis with marginal identity s [...] ","uniprotId":"Q0PBM1_CAMJE"},"10641139":{"id":10641139,"externalId":"192222.Cj0286c","name":"Cj0286c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0286c, unknown, len: 205 aa; 28.4% identity to HP0394 (252 aa). Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PBM0_CAMJE"},"10641140":{"id":10641140,"externalId":"192222.Cj0287c","name":"greA","hasFamily":true,"annotation":"Necessary for efficient RNA polymerase transcription elongation past template-encoded arresting sites. The arresting sites in DNA have the property of trapping a certain fraction of elongating RNA polymerases that pass through, resulting in locked ternary complexes. Cleavage of the nascent transcript by cleavage factors such as GreA or GreB allows the resumption of elongation from the new 3'terminus. GreA releases sequences of 2 to 3 nucleotides. ","uniprotId":"GREA_CAMJE"},"10641141":{"id":10641141,"externalId":"192222.Cj0288c","name":"lpxB","hasFamily":true,"annotation":"Condensation of UDP-2,3-diacylglucosamine and 2,3- diacylglucosamine-1-phosphate to form lipid A disaccharide, a precursor of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell","uniprotId":"LPXB_CAMJE"},"10641142":{"id":10641142,"externalId":"192222.Cj0289c","name":"peb3","hasFamily":true,"annotation":"Original (2000) note: Cj0289c, peb3, major antigenic peptide PEB3, len: 250 aa; 94.9% identical to TR:G234575, sequenced by direct peptide sequencing; see: Pei at al. J. Biol. Chem. 266:16363-16369 (1991). Also similar to TR:Q57410 (EMBL:U39068) Vibrio cholerae accessory colonization factor acfC (253 aa), fasta scores; opt: 855 z-score: 1301.2 E(): 0, 53.6% identity in 239 aa overlap and TR:P96759 (EMBL:U82533) anm (E. coli 045 attachment and effacement negative mutant) (251 aa), fasta scores; opt: 849 z-score: 824.0 E(): 0, 51.8% identity in 251 aa overlap. No Hp match. Also similar t [...] ","uniprotId":"Q0PBL7_CAMJE"},"10641143":{"id":10641143,"externalId":"192222.Cj0293","name":"surE","hasFamily":true,"annotation":"Nucleotidase that shows phosphatase activity on nucleoside 5'-monophosphates","uniprotId":"SURE_CAMJE"},"10641144":{"id":10641144,"externalId":"192222.Cj0294","name":"Cj0294","hasFamily":true,"annotation":"Putative MoeB/ThiF family protein; Original (2000) note: Cj0294, unknown, len: 219 aa; similar to members of the hesA/moeB/thiF family, e.g. YGDL_ECOLI (268 aa), fasta scores; opt: 219 z-score: 563.3 E(): 4e-24, 33.5% identity in 254 aa overlap. 48.3% identity to HP0755 (described as moeB). Contains Pfam match to entry PF00899 ThiF_family, ThiF family, score 33.10, E-value 2e-07. Also similar to Cj1046c (33.6% identity in 146 aa overlap); Updated (2006) note: No specific characterisation with acceptable identity scores carried out yet. Putative kept within product function. Functional  [...] ","uniprotId":"Q0PBL5_CAMJE"},"10641145":{"id":10641145,"externalId":"192222.Cj0295","name":"Cj0295","hasFamily":true,"annotation":"Putative acetyltransferase; Original (2000) note: Cj0295, possible acetyltransferase, len: 152 aa; similar to hypothetical proteins e.g. YAFP_ECOLI (150 aa), fasta scores; opt: 246 z-score: 340.4 E(): 1.1e-11, 27.8% identity in 144 aa overlap. No Hp match. Contains Pfam match to entry PF00583 Acetyltransf, Acetyltransferase (GNAT) family, score 20.20, E-value 0.011; Updated (2006) note: Some characterisation within Escherichia coli with marginal identity score. Putative kept within product function. Functional classification -Misc","uniprotId":"Q0PBL4_CAMJE"},"10641146":{"id":10641146,"externalId":"192222.Cj0296c","name":"panD","hasFamily":true,"annotation":"Catalyzes the pyruvoyl-dependent decarboxylation of aspartate to produce beta-alanine","uniprotId":"PAND_CAMJE"},"10641147":{"id":10641147,"externalId":"192222.Cj0297c","name":"panC","hasFamily":true,"annotation":"Catalyzes the condensation of pantoate with beta-alanine in an ATP-dependent reaction via a pantoyl-adenylate intermediate. ","uniprotId":"PANC_CAMJE"},"10641148":{"id":10641148,"externalId":"192222.Cj0298c","name":"panB","hasFamily":true,"annotation":"Catalyzes the reversible reaction in which hydroxymethyl group from 5,10-methylenetetrahydrofolate is transferred onto alpha- ketoisovalerate to form ketopantoate","uniprotId":"PANB_CAMJE"},"10641149":{"id":10641149,"externalId":"192222.Cj0299","name":"Cj0299","hasFamily":true,"annotation":"Original (2000) note: Cj0299, possible periplasmic beta-lactamase, len: 257 aa; similar to members of the class-D beta-lactamase family e.g. BLL1_PSEAE BETA-LACTAMASE LCR-1 PRECURSOR (EC 3.5.2.6) (Pseudomonas aeruginosa plasmid PMG76) (260 aa), fasta scores; opt: 423 z-score: 305.2 E(): 9.6e-10, 33.7% identity in 255 aa overlap. No Hp match. Contains N-terminal signal sequence; Updated (2006) note: Some characterisation work within Pseudomonas aeruginosa and Escherichia coli with marginal identity scores. Putative kept within product function. Functional classification - Antibiotic res [...] ","uniprotId":"Q0PBL0_CAMJE"},"10641150":{"id":10641150,"externalId":"192222.Cj0300c","name":"modC","hasFamily":true,"annotation":"Original (2000) note: Cj0300c, modC, probable molybdenum transport ATP-binding protein, len: 294 aa; similar to many ABC-transporter ATP binding proteins, and to MODC_ECOLI molybdenum transport ATP-binding protein (352 aa), fasta scores; opt: 487 z-score: 700.9 E(): 8.7e-32, 34.8% identity in 270 aa overlap. Identification as modC is based on presence of probable molybdenum-specific permease and periplasmic binding proteins upstream. 46.8% identity to HP0475. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature and Pfam match to entry PF0000 [...] ","uniprotId":"Q0PBK9_CAMJE"},"10641151":{"id":10641151,"externalId":"192222.Cj0301c","name":"modB","hasFamily":true,"annotation":"Part of the binding-protein-dependent transport system for molybdenum; probably responsible for the translocation of the substrate across the membrane","uniprotId":"Q0PBK8_CAMJE"},"10641152":{"id":10641152,"externalId":"192222.Cj0302c","name":"Cj0302c","hasFamily":true,"annotation":"Putative molybdenum-pterin binding protein; Original (2000) note: Cj0302c, unknonw, len: 133 aa; no Hp match; Updated (2006) note: Pfam domain PF03459 TOBE domain identified within CDS. TOBE (Transport-associated OB) has had work carried out within Esherichia coli where it has been shown to be involved in the recognition of small ligands such as molybdenum. TOBE has been found in ABC transporters immediately after the ATPase domain. A child process of this family is Molybdenum-pterin binding protein. Product modified to more specific family member based on motif match. Some characteris [...] ","uniprotId":"Q0PBK7_CAMJE"},"10641153":{"id":10641153,"externalId":"192222.Cj0303c","name":"modA","hasFamily":true,"annotation":"Original (2000) note: Cj0303c, modA, probable molybdate-binding lipoprotein, len: 249 aa; similar to e.g. MODA_RHOCA molybdate-binding periplasmic protein precursor (252 aa), fasta scores; opt: 354 z-score: 449.0 E(): 9.4e-18, 30.8% identity in 224 aa overlap, and MODA_ECOLI (257 aa), fasta scores; opt: 220 z-score: 245.4 E(): 2.1e-06, 25.0% identity in 232 aa overlap. 42.3% identity to HP0473. Contains probable N-terminal signal sequence with appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site; Updated (2006) note: Pfam domain PF01547 Bacterial extr [...] ","uniprotId":"Q0PBK6_CAMJE"},"10641154":{"id":10641154,"externalId":"192222.Cj0304c","name":"bioC","hasFamily":true,"annotation":"Converts the free carboxyl group of a malonyl-thioester to its methyl ester by transfer of a methyl group from S-adenosyl-L- methionine (SAM). It allows to synthesize pimeloyl-ACP via the fatty acid synthetic pathway","uniprotId":"Q0PBK5_CAMJE"},"10641155":{"id":10641155,"externalId":"192222.Cj0305c","name":"Cj0305c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0305c, unknown, len: 203 aa; similar to a hypothetical protein from H. infuenzae YF52_HAEIN HI1552 (215 aa), fasta scores; opt: 297 z-score: 372.7 E(): 1.7e-13, 29.8% identity in 215 aa overlap. No Hp match; Updated (2006) note: Pfam PF04301, Protein of unknown function (DUF452) was present within CDS. Conserved added to product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PBK4_CAMJE"},"10641156":{"id":10641156,"externalId":"192222.Cj0306c","name":"bioF","hasFamily":true,"annotation":"8-amino-7-oxononanoate synthase; Original (2000) note: Cj0306c, bioF, probable 8-amino-7-oxononanoate synthase, len: 380 aa; similar to many e.g. BIOF_ECOLI 8-amino-7-oxononanoate synthase (EC 2.3.1.47) (384 aa), fasta scores; opt: 525 z-score: 550.3 E(): 2.1e-23, 28.9% identity in 387 aa overlap. 29.0% identity to HP0598. Contains PS00599 Aminotransferases class-II pyridoxal-phosphate attachment site and Pfam match to entry PF00222 aminotran_2, Aminotransferases class-II, score 196.30, E-value 4.7e-55; Updated (2006) note: Characterised within Escherichia coli with marginal identity s [...] ","uniprotId":"Q0PBK3_CAMJE"},"10641157":{"id":10641157,"externalId":"192222.Cj0307","name":"bioA","hasFamily":true,"annotation":"Belongs to the class-III pyridoxal-phosphate-dependent aminotransferase family","uniprotId":"Q0PBK2_CAMJE"},"10641158":{"id":10641158,"externalId":"192222.Cj0308c","name":"bioD","hasFamily":true,"annotation":"Catalyzes a mechanistically unusual reaction, the ATP- dependent insertion of CO2 between the N7 and N8 nitrogen atoms of 7,8- diaminopelargonic acid (DAPA) to form an ureido ring. ","uniprotId":"BIOD_CAMJE"},"10641159":{"id":10641159,"externalId":"192222.Cj0309c","name":"Cj0309c","hasFamily":true,"annotation":"Putative efflux protein; Original (2000) note: Cj0309c, probable efflux protein, len: 104 aa; similar to many membrane protein of known and unknown function e.g. TR:O34690 (EMBL:AJ002571) Bacillus subtilis YKKD (105 aa), fasta scores; opt: 303 z-score: 547.1 E(): 3.2e-23, 44.2% identity in 104 aa overlap, and EBR_STAAU ethidium bromide resistance protein (multidrug resistance protein) (107 aa), fasta scores; opt: 187 z-score: 272.5 E(): 6.4e-08, 27.2% identity in 103 aa overlap. No Hp match. Also similar to upstream CDS Cj0310c (32.4% identity in 105 aa overlap), and to Cj1174 (28.7% i [...] ","uniprotId":"Q0PBK0_CAMJE"},"10641160":{"id":10641160,"externalId":"192222.Cj0310c","name":"Cj0310c","hasFamily":true,"annotation":"Putative efflux protein; Original (2000) note: Cj0310c, possible efflux protein, len: 104 aa; similar to many membrane proteins of known and unknown function e.g. YKKC_BACSU (112 aa), fasta scores; opt: 273 z-score: 458.2 E(): 2.9e-18, 40.8% identity in 103 aa overlap, and EBR_STAAU ethidium bromide resistance protein (107 aa), fasta scores; opt: 135 z-score: 198.9 E(): 0.00088, 27.4% identity in 84 aa overlap. No Hp match. Also similar to downstream CDS Cj0309c (32.4% identity in 105 aa overlap) and to Cj1173 (27.1% identity in 107 aa overlap). Contains Pfam match to entry PF00893 DUF [...] ","uniprotId":"Q0PBJ9_CAMJE"},"10641161":{"id":10641161,"externalId":"192222.Cj0311","name":"rplY","hasFamily":true,"annotation":"This is one of the proteins that binds to the 5S RNA in the ribosome where it forms part of the central protuberance. ","uniprotId":"RL25_CAMJE"},"10641162":{"id":10641162,"externalId":"192222.Cj0312","name":"pth","hasFamily":true,"annotation":"The natural substrate for this enzyme may be peptidyl-tRNAs which drop off the ribosome during protein synthesis. ","uniprotId":"PTH_CAMJE"},"10641163":{"id":10641163,"externalId":"192222.Cj0313","name":"Cj0313","hasFamily":true,"annotation":"Original (2000) note: Cj0313, probable integral membrane protein, len: 352 aa; 35.9% identity to HP1498; Updated (2006) note: Pfam domains PF03739 Predicted permease YjgP/YjgQ family were identified. Members of this family are predicted integral membrane proteins of unknown function. They are about 350 amino acids long and contain about 6 transmembrane regions. They are predicted to be permeases although there is no verification of this. Six probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Functional classification - Membranes, lipoprotei [...] ","uniprotId":"Q0PBJ6_CAMJE"},"10641164":{"id":10641164,"externalId":"192222.Cj0314","name":"lysA","hasFamily":true,"annotation":"Specifically catalyzes the decarboxylation of meso- diaminopimelate (meso-DAP) to L-lysine","uniprotId":"DCDA_CAMJE"},"10641165":{"id":10641165,"externalId":"192222.Cj0315","name":"Cj0315","hasFamily":true,"annotation":"Putative HAD-superfamily hydrolase, subfamily IIA; Original (2000) note: Cj0315, unknown, len: 255 aa; similar to the hypothetical proteins TR:O59622 (EMBL:AB009530) Pyrococcus horikoshii PHBR011 (263 aa),fasta scores; opt: 185 z-score: 375.5 E(): 1.2e-13, 28.1% identity in 267 aa overlapNAGD_ECOLI nagD protein (250 aa), fasta scores; opt: 259 z-score: 266.1 E(): 1.4e-07,26.0% identity in 250 aa overlap. No Hp match; Updated (2006) note: Pfam domains PF00702 haloacid dehalogenase-like hydrolase, PF02142 MGS-like domain and PF03558 TBSV core protein P21/P22 were identified with separate [...] ","uniprotId":"Q0PBJ4_CAMJE"},"10641166":{"id":10641166,"externalId":"192222.Cj0316","name":"pheA","hasFamily":true,"annotation":"Original (2000) note: Cj0316, pheA, probable chorismate mutase/prephenate dehydratase, len: 357 aa; similar to many e.g. PHEA_ECOLI chorismate mutase (EC 5.4.99.5)/prephenate dehydratase (EC 4.2.1.51) (386 aa),fasta scores; opt: 482 z-score: 733.8 E(): 0, 33.1% identity in 366 aa overlap. Similar in N-term to possible truncated pheA in Hp; HP0291 hypothetical protein (96 aa) 35.8% identity in 81 aa overlap. Contains PS00857 and PS00858 Prephenate dehydratase signatures 1 and 2, and Pfam match to entry PF00800 PDT, Prephenate dehydratase,score 268.20, E-value 1.1e-76; Updated (2006) not [...] ","uniprotId":"Q0PBJ3_CAMJE"},"10641167":{"id":10641167,"externalId":"192222.Cj0317","name":"hisC","hasFamily":true,"annotation":"Belongs to the class-II pyridoxal-phosphate-dependent aminotransferase family. Histidinol-phosphate aminotransferase subfamily","uniprotId":"HIS8_CAMJE"},"10641168":{"id":10641168,"externalId":"192222.Cj0318","name":"fliF","hasFamily":true,"annotation":"The M ring may be actively involved in energy transduction. ","uniprotId":"Q0PBJ1_CAMJE"},"10641169":{"id":10641169,"externalId":"192222.Cj0319","name":"fliG","hasFamily":true,"annotation":"Original (2000) note: Cj0319, fliG, probable flagellar motor switch protein, len: 342 aa; similar to many e.g. FLIG_ECOLI flagellar motor switch protein fliG (331 aa), fasta scores; opt: 609 z-score: 927.6 E(): 0,30.1% identity in 326 aa overlap. 61.5% identity to HP0352; Updated (2006) note: Pfam domain PF01706 FliG C-terminal domain was identified. Further supoport given to product function. Characterised within Escherichia coli with marginal identity score. Putative not added to product function. Functional classification - Surface structures; PMID:1551848, PMID:8308888, PMID:8550421","uniprotId":"Q0PBJ0_CAMJE"},"10641170":{"id":10641170,"externalId":"192222.Cj0320","name":"fliH","hasFamily":true,"annotation":"Original (2000) note: Cj0320, fliH, possible flagellar assembly protein, len: 364 aa; weakly similar to e.g. SW:FLIH_BACSU P23449 probable flagellar assembly protein fliH (208 aa), wublastp scores; E= 6.3e-07, 22% identity in 190 aa overlap. 34.6% identity to HP0353; Updated (2006) note: Characterised within Bacillus subtilis, however, identity score unnacceptable. Putative kept within product function. Functional classification -Surface structures; PMID:1828465","uniprotId":"Q0PBI9_CAMJE"},"10641171":{"id":10641171,"externalId":"192222.Cj0321","name":"dxs","hasFamily":true,"annotation":"Catalyzes the acyloin condensation reaction between C atoms 2 and 3 of pyruvate and glyceraldehyde 3-phosphate to yield 1-deoxy-D- xylulose-5-phosphate (DXP)","uniprotId":"DXS_CAMJE"},"10641172":{"id":10641172,"externalId":"192222.Cj0322","name":"perR","hasFamily":true,"annotation":"Belongs to the Fur family","uniprotId":"Q0PBI7_CAMJE"},"10641173":{"id":10641173,"externalId":"192222.Cj0323","name":"Cj0323","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0323, unknown, len: 398 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PBI6_CAMJE"},"10641174":{"id":10641174,"externalId":"192222.Cj0324","name":"ubiE","hasFamily":true,"annotation":"Methyltransferase required for the conversion of demethylmenaquinol (DMKH2) to menaquinol (MKH2) and the conversion of 2-polyprenyl-6-methoxy-1,4-benzoquinol (DDMQH2) to 2-polyprenyl-3- methyl-6-methoxy-1,4-benzoquinol (DMQH2)","uniprotId":"UBIE_CAMJE"},"10641175":{"id":10641175,"externalId":"192222.Cj0325","name":"xseA","hasFamily":true,"annotation":"Bidirectionally degrades single-stranded DNA into large acid- insoluble oligonucleotides, which are then degraded further into small acid-soluble oligonucleotides","uniprotId":"EX7L_CAMJE"},"10641176":{"id":10641176,"externalId":"192222.Cj0326","name":"serC","hasFamily":true,"annotation":"Catalyzes the reversible conversion of 3- phosphohydroxypyruvate to phosphoserine and of 3-hydroxy-2-oxo-4- phosphonooxybutanoate to phosphohydroxythreonine","uniprotId":"SERC_CAMJE"},"10641177":{"id":10641177,"externalId":"192222.Cj0327","name":"Cj0327","hasFamily":true,"annotation":"Putative endoribonuclease L-PSP family protein; Original (2000) note: Cj0327, unknown, len: 116 aa; similar to hypothetical proteins e.g. YABJ_BACSU (125 aa),fasta scores; opt: 211 z-score: 261.9 E(): 2.5e-07, 33.0% identity in 109 aa overlap, and to e.g. UK14_HUMAN 14.5 kd translational inhibitor protein (137 aa), fasta scores; opt: 173 z-score: 249.5 E(): 1.2e-06, 29.2% identity in 106 aa overlap. No Hp ortholog. Also similar to Cj1388 (33.3% identity in 96 aa overlap). Contains Pfam match to entry PF01042 DUF10, Domain of unknown function, score 69.30, E-value 8.4e-17; Updated (2006 [...] ","uniprotId":"Q0PBI2_CAMJE"},"10641178":{"id":10641178,"externalId":"192222.Cj0328c","name":"fabH","hasFamily":true,"annotation":"Catalyzes the condensation reaction of fatty acid synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP. Catalyzes the first condensation reaction which initiates fatty acid synthesis and may therefore play a role in governing the total rate of fatty acid production. Possesses both acetoacetyl-ACP synthase and acetyl transacylase activities. Its substrate specificity determines the biosynthesis of branched-chain and/or straight-chain of fatty acids","uniprotId":"FABH_CAMJE"},"10641179":{"id":10641179,"externalId":"192222.Cj0329c","name":"plsX","hasFamily":true,"annotation":"Catalyzes the reversible formation of acyl-phosphate (acyl- PO(4)) from acyl-[acyl-carrier-protein] (acyl-ACP). This enzyme utilizes acyl-ACP as fatty acyl donor, but not acyl-CoA. ","uniprotId":"PLSX_CAMJE"},"10641180":{"id":10641180,"externalId":"192222.Cj0330c","name":"rpmF","hasFamily":true,"annotation":"Belongs to the bacterial ribosomal protein bL32 family. ","uniprotId":"RL32_CAMJE"},"10641181":{"id":10641181,"externalId":"192222.Cj0331c","name":"Cj0331c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0331c, unknown, len: 117 aa; 34.7% identity to HP0199. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PBH8_CAMJE"},"10641182":{"id":10641182,"externalId":"192222.Cj0332c","name":"ndk","hasFamily":true,"annotation":"Major role in the synthesis of nucleoside triphosphates other than ATP. The ATP gamma phosphate is transferred to the NDP beta phosphate via a ping-pong mechanism, using a phosphorylated active-site intermediate","uniprotId":"NDK_CAMJE"},"10641183":{"id":10641183,"externalId":"192222.Cj0333c","name":"fdxA","hasFamily":true,"annotation":"Ferredoxin; Original (2000) note: Cj0333c, fdxA, probable ferredoxin, len: 94 aa; similar to many e.g. FER_CLOTM ferredoxin (55 aa), fasta scores; opt: 142 z-score: 311.2 E(): 4.5e-10, 45.8% identity in 59 aa overlap. No Hp ortholog. Also similar to Cj0354c (38.8% identity in 67 aa overlap). Contains PS00198 4Fe-4S ferredoxins, iron-sulfur binding region signature and Pfam match to entry PF00037 fer4, 4Fe-4S ferredoxins and related iron-sulfur cluster binding domains., score 51.80, E-value 8.8e-14; Updated (2006) note: Similar to many e.g. Clostridium sp. with acceptable identity score [...] ","uniprotId":"Q0PBH6_CAMJE"},"10641184":{"id":10641184,"externalId":"192222.Cj0334","name":"ahpC","hasFamily":true,"annotation":"Alkyl hydroperoxide reductase; Original (2000) note: Cj0334, ahpC, probable alkyl hydroperoxide reductase, len: 198 aa; highly simlar to e.g. TDX1_HUMAN thioredoxin peroxidase 1 (198 aa), fasta scores; opt: 615 z-score: 915.3 E(): 0, 46.1% identity in 193 aa overlap, and to e.g. AHPC_ECOLI P26427 alkyl hydroperoxide reductase C22 protein (EC 1.6.4.-) (186 aa),wublastp scores E= 8.9e-23, 34% identity in 149 aa overlap. 67.7% identity to HP1563 (Hp 26 kD antigen). Contains Pfam match to entry PF00578 AhpC-TSA, AhpC/TSA family, score 208.20, E-value 1.2e-58; Updated (2006) note: Character [...] ","uniprotId":"Q0PBH5_CAMJE"},"10641185":{"id":10641185,"externalId":"192222.Cj0335","name":"flhB","hasFamily":true,"annotation":"Required for formation of the rod structure in the basal body of the flagellar apparatus. Together with FliI and FliH, may constitute the export apparatus of flagellin","uniprotId":"Q0PBH4_CAMJE"},"10641186":{"id":10641186,"externalId":"192222.Cj0336c","name":"motB","hasFamily":true,"annotation":"Original (2000) note: Cj0336c, motB, possible flagellar motor protein, len: 247 aa; similar to MOTB_BACSU chemotaxis motB protein (motility protein B) (261 aa), fasta scores; opt: 195 z-score: 275.7 E(): 4.2e-08, 30.7% identity in 261 aa overlap. 43.3% identity to HP0816. Contains Pfam match to entry PF00691 OmpA, OmpA family, score 28.50, E-value 5.6e-07; Updated (2006) note: One probable transmembrace helices identified by TMHMM2.0 within CDS. Some characterisation within Bacillus subtilis with marginal identity score. Putative kept within product function. Functional classification  [...] ","uniprotId":"Q0PBH3_CAMJE"},"10641187":{"id":10641187,"externalId":"192222.Cj0337c","name":"motA","hasFamily":true,"annotation":"Original (2000) note: Cj0337c, motA, probable flagellar motor proton channel, len: 258 aa; similar to e.g. MOTA_BACSU chemotaxis motA protein (motility protein A) (270 aa), fasta scores; opt: 493 z-score: 784.5 E(): 0,34.0% identity in 259 aa overlap. 58.6% identity to HP0815. Contains PS01307 Flagellar motor protein motA family signature; Updated (2006) note: Three probable transmembrace helices identified by TMHMM2.0 within CDS. Pfam domain PF01618 MotA/TolQ/ExbB proton channel family protein identified within CDS. Further support given to product function. Some characterisation work [...] ","uniprotId":"Q0PBH2_CAMJE"},"10641188":{"id":10641188,"externalId":"192222.Cj0338c","name":"polA","hasFamily":true,"annotation":"In addition to polymerase activity, this DNA polymerase exhibits 5'-3' exonuclease activity","uniprotId":"Q0PBH1_CAMJE"},"10641189":{"id":10641189,"externalId":"192222.Cj0339","name":"Cj0339","hasFamily":true,"annotation":"Putative MFS (Major Facilitator Superfamily) transport protein; Original (2000) note: Cj0339, probable transmembrane transport protein, len: 453 aa; similar to e.g. YDFJ_ECOLI hypothetical metabolite transport protein (427 aa), fasta scores; opt: 1214 z-score: 1428.6 E(): 0,42.5% identity in 416 aa overlap, and SHIA_ECOLI shikimate transporter (438 aa), fasta scores; opt: 876 z-score: 1104.0 E(): 0, 32.3% identity in 421 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00083 sugar_tr,Sugar (and other) transporters, score 123.20, E-value 4.9e-33; Updated (2006) note: Ten proba [...] ","uniprotId":"Q0PBH0_CAMJE"},"10641190":{"id":10641190,"externalId":"192222.Cj0340","name":"Cj0340","hasFamily":true,"annotation":"Putative nucleoside hydrolase; Original (2000) note: Cj0340, possible nucleoside hydrolase, len: 335 aa; similar to hypothetical proteins e.g. YAAF_ECOLI (304 aa), fasta scores; opt: 252 z-score: 377.2 E(): 9.4e-14, 25.9% identity in 309 aa overlap, and to the eukaryotic IUNH_CRIFA inosine-uridine preferring nucleoside hydrolase (314 aa), fasta scores; opt: 261 z-score: 363.4 E(): 5.5e-13, 25.3% identity in 296 aa overlap. No Hp match. Contains Pfam match to entry PF01156 IU_nuc_hydro, Inosine-uridine preferring nucleoside hydrolase, score -11.20, E-value 4.8e-12; Updated (2006) note:  [...] ","uniprotId":"Q0PBG9_CAMJE"},"10641191":{"id":10641191,"externalId":"192222.Cj0341c","name":"Cj0341c","hasFamily":true,"annotation":"Original (2000) note: Cj0341c, possible integral membrane protein, len: 147 aa; similar to hypothetical membrane proteins e.g. Y489_HAEIN HI0489 (157 aa), fasta scores; opt: 252 z-score: 350.5 E(): 2.9e-12, 34.0% identity in 159 aa overlap. No Hp match; Updated (2006) note: Four probable transmembrace helices identified by TMHMM2.0 within CDS. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBG8_CAMJE"},"10641192":{"id":10641192,"externalId":"192222.Cj0342c","name":"uvrA","hasFamily":true,"annotation":"The UvrABC repair system catalyzes the recognition and processing of DNA lesions. UvrA is an ATPase and a DNA-binding protein. A damage recognition complex composed of 2 UvrA and 2 UvrB subunits scans DNA for abnormalities. When the presence of a lesion has been verified by UvrB, the UvrA molecules dissociate. ","uniprotId":"Q0PBG7_CAMJE"},"10641193":{"id":10641193,"externalId":"192222.Cj0343c","name":"Cj0343c","hasFamily":true,"annotation":"Original (2000) note: Cj0343c, probable integral membrane protein, len: 261 aa; similar to many hypothetical membrane proteins e.g. Y441_METJA MJ0441 (267 aa), fasta scores; opt: 152 z-score: 301.8 E(): 1.5e-09,30.7% identity in 257 aa overlap. 45.1% identity to HP0677 and 44.0% identity to HP0226; Updated (2006) note: Nine probable transmembrace helices identified by TMHMM2.0 within CDS. Pfam PF01925 Domain of unknown function (DUF81) protein identified within CDS. Further support given to product function. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBG6_CAMJE"},"10641194":{"id":10641194,"externalId":"192222.Cj0344","name":"Cj0344","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0344, unknown, len: 41 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PBG5_CAMJE"},"10641195":{"id":10641195,"externalId":"192222.Cj0345","name":"trpE","hasFamily":true,"annotation":"Original (2000) note: Cj0345, trpE, possible anthranilate synthase component I, len: 416 aa; similar in C-terminal half to many e.g. TRPE_CLOTM anthranilate synthase component I (EC 4.1.3.27) (494 aa), fasta scores; opt: 784 z-score: 1209.9 E(): 0, 47.0% identity in 281 aa overlap. 35.6% identity in 365 aa overlap to HP1282. Contains Pfam match to entry PF00425 chorismate_bind,chorismate binding enzyme, score 309.50, E-value 3.9e-89; Updated (2006) note: Characterised within Clostridium thermocellum with acceptable identity score,however, sequence alignment was only partial. Thus,putat [...] ","uniprotId":"Q0PBG4_CAMJE"},"10641196":{"id":10641196,"externalId":"192222.Cj0346","name":"trpD","hasFamily":true,"annotation":"Catalyzes the transfer of the phosphoribosyl group of 5- phosphorylribose-1-pyrophosphate (PRPP) to anthranilate to yield N-(5'- phosphoribosyl)-anthranilate (PRA)","uniprotId":"Q0PBG3_CAMJE"},"10641197":{"id":10641197,"externalId":"192222.Cj0347","name":"trpF","hasFamily":true,"annotation":"Belongs to the TrpF family","uniprotId":"TRPF_CAMJE"},"10641198":{"id":10641198,"externalId":"192222.Cj0348","name":"trpB","hasFamily":true,"annotation":"The beta subunit is responsible for the synthesis of L- tryptophan from indole and L-serine","uniprotId":"TRPB_CAMJE"},"10641199":{"id":10641199,"externalId":"192222.Cj0349","name":"trpA","hasFamily":true,"annotation":"The alpha subunit is responsible for the aldol cleavage of indoleglycerol phosphate to indole and glyceraldehyde 3-phosphate. ","uniprotId":"TRPA_CAMJE"},"10641200":{"id":10641200,"externalId":"192222.Cj0350","name":"Cj0350","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0350, unknown, len: 140 aa; no Hp match. . Functional classification - Unknown","uniprotId":"Q0PBF9_CAMJE"},"10641201":{"id":10641201,"externalId":"192222.Cj0351","name":"fliN","hasFamily":true,"annotation":"Original (2000) note: Cj0351, fliN, flagellar motor switch protein, len: 102 aa; identical to TR:O32370 (EMBL:AJ000400), and similar to many e.g. FLIN_ECOLI flagellar motor switch protein FLIN (137 aa), fasta scores; opt: 196 z-score: 359.1 E(): 9.5e-13, 36.0% identity in 89 aa overlap. 53.4% identity to HP0584; Updated (2006) note: Pfam PF01052 Surface presentation of antigens (SPOA) protein identified within CDS. Further support given to product function. Characterisation paper identified within Campylobacter jejuni and also Escherichia coli. Putative not added to product function. F [...] ","uniprotId":"Q0PBF8_CAMJE"},"10641202":{"id":10641202,"externalId":"192222.Cj0352","name":"Cj0352","hasFamily":true,"annotation":"Original (2000) note: Cj0352, probable transmembrane protein, len: 265 aa; 99.6% identity to TR:O32371 (EMBL:AJ000400). No Hp match. Contains probable N-terminal signal sequence and transmembrane domain around aa 160; Updated (2006) note: Two probable transmembrane helices identified by TMHMM2.0 within CDS. Literature search identified paper linking CDS with potential new functional assignment as FliO flagellar protein. Functional classification - Membranes, lipoproteins and porins; PMID:15817382, PMID:9894591","uniprotId":"Q0PBF7_CAMJE"},"10641203":{"id":10641203,"externalId":"192222.Cj0353c","name":"Cj0353c","hasFamily":true,"annotation":"Phosphatase; Original (2000) note: Cj0353c, probable phosphatase, len: 486 aa; similar to e.g. GPPA_ECOLI guanosine-5'-triphosphate,3'-diphosphate pyrophosphatase (EC 3.6.1.40) (494 aa), fasta scores; opt: 368 z-score: 272.7 E(): 6.2e-08, 25.8% identity in 480 aa overlap, and to PPX_ECOLI EXOPOLYPHOSPHATASE (EC 3.6.1.11) (512 aa),fasta scores; opt: 340 z-score: 350.9 E(): 2.7e-12, 23.8% identity in 441 aa overlap. 41.0% identity to HP0278; Updated (2006) note: Pfam domain PF02541 Ppx/GppA phosphatase family identified within CDS. Further support given to product function. Similar to ma [...] ","uniprotId":"Q0PBF6_CAMJE"},"10641204":{"id":10641204,"externalId":"192222.Cj0354c","name":"fdxB","hasFamily":true,"annotation":"Ferredoxin; Original (2000) note: Cj0354c, fdxB, probable ferredoxin, len: 81 aa; similar to many e.g. FER_CHRVI ferredoxin (82 aa), fasta scores; opt: 282 z-score: 445.5 E(): 1.5e-17, 45.3% identity in 75 aa overlap. 58.4% identity to HP0277. Also similar to Cj0333c (fdx); 38.8% identity in 67 aa overlap. Contains PS00198 4Fe-4S ferredoxins, iron-sulfur binding region signature and Pfam match to entry PF00037 fer4, 4Fe-4S ferredoxins and related iron-sulfur cluster binding domains., score 39.00,E-value 4.8e-10; Updated (2006) note: Characterised within Chromatium vinosum with acceptab [...] ","uniprotId":"Q0PBF5_CAMJE"},"10641205":{"id":10641205,"externalId":"192222.Cj0355c","name":"Cj0355c","hasFamily":true,"annotation":"Two-component regulator; Original (2000) note: Cj0355c, probable two-component regulator, len: 223 aa; similar to many e.g. CZCR_ALCEU transcriptional activator protein CZCR (225 aa), fasta scores; opt: 375 z-score: 570.8 E(): 1.5e-24,31.7% identity in 218 aa overlap. 59.9% identity to HP1043. Contains Pfam match to entry PF00072 response_reg,Response regulator receiver domain, score 70.70, E-value 3.1e-17 and Pfam match to entry PF00486 trans_reg_C,Transcriptional regulatory protein, C terminal, score 56.60, E-value 1.5e-15; Updated (2006) note: Characterisation work within many e.g.  [...] ","uniprotId":"Q0PBF4_CAMJE"},"10641206":{"id":10641206,"externalId":"192222.Cj0356c","name":"folB","hasFamily":true,"annotation":"Putative dihydroneopterin aldolase; Original (2000) note: Cj0356c, unknown, len: 105 aa; 31.0% identity to HP1510; Updated (2006) note: Pfam domain PF02152 FolB Dihydroneopterin aldolase was identified within CDS. This enzyme catalyses the conversion of 7,8-dihydroneopterin to 6-hydroxymethyl-7,8-dihydropterin in the biosynthetic pathway of tetrahydrofolate. This was found by carrying out own Pfam search. Product modified to new family member due to motif match. No specific characterisation has been carried out yet. Putative kept within product function. Functional classification - Misc","uniprotId":"Q0PBF3_CAMJE"},"10641207":{"id":10641207,"externalId":"192222.Cj0357c","name":"plsY","hasFamily":true,"annotation":"Catalyzes the transfer of an acyl group from acyl-phosphate (acyl-PO(4)) to glycerol-3-phosphate (G3P) to form lysophosphatidic acid (LPA). This enzyme utilizes acyl-phosphate as fatty acyl donor, but not acyl-CoA or acyl-ACP","uniprotId":"PLSY_CAMJE"},"10641208":{"id":10641208,"externalId":"192222.Cj0358","name":"Cj0358","hasFamily":true,"annotation":"Original (2000) note: Cj0358, probable cytochrome C551 peroxidase, len: 341 aa; similar to e.g. CCPR_PSEAE cytochrome C551 peroxidase precursor (EC 1.11.1.5) (346 aa), fasta scores; opt: 281 z-score: 1028.0 E(): 0, 48.9% identity in 313 aa overlap. 58.5% identity to HP1461. Also similar to Cj0020c (45.4% identity in 273 aa overlap). Contains 2x PS00190 Cytochrome c family heme-binding site signature and probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF03150 Di-haem cytochrome c peroxidase identified within CDS. Further support given to product function. Character [...] ","uniprotId":"Q0PBF1_CAMJE"},"10641209":{"id":10641209,"externalId":"192222.Cj0360","name":"glmM","hasFamily":true,"annotation":"Catalyzes the conversion of glucosamine-6-phosphate to glucosamine-1-phosphate","uniprotId":"GLMM_CAMJE"},"10641210":{"id":10641210,"externalId":"192222.Cj0361","name":"lspA","hasFamily":true,"annotation":"This protein specifically catalyzes the removal of signal peptides from prolipoproteins","uniprotId":"LSPA_CAMJE"},"10641211":{"id":10641211,"externalId":"192222.Cj0362","name":"Cj0362","hasFamily":true,"annotation":"Original (2000) note: Cj0362, probable integral membrane protein, len: 150 aa; similar to hypothetical proteins e.g. TR:Q53231 (EMBL:L76097) from Rhodobacter sphaeroides (154 aa), fasta scores; opt: 301 z-score: 419.6 E(): 4.1e-16, 34.2% identity in 149 aa overlap. 47.6% identity to HP1484; Updated (2006) note: Four probable transmembrace helices identified by TMHMM2.0 within CDS. Pfam domain PF03653 Uncharacterised protein family (UPF0093) identified within CDS. Further support given to product function. Functional classification - Membranes,lipoproteins and porins","uniprotId":"Q0PBE8_CAMJE"},"10641212":{"id":10641212,"externalId":"192222.Cj0363c","name":"hemN","hasFamily":true,"annotation":"Putative oxygen-independent coproporphyrinogen III oxidase; Original (2000) note: Cj0363c, probable oxidoreductase, len: 448 aa; some similarity to e.g. HEMN_BACSU probable oxygen-independent coproporphyrinogen III oxidase (366 aa), fasta scores; opt: 354 z-score: 228.9 E(): 1.7e-05, 26.6% identity in 368 aa overlap. No Hp ortholog; Updated (2006) note: Pfam domains PF06969 HemN C-terminal region and PF04055 Radical SAM superfamily were identified within CDS. Further support given to product function. Characterised within Bacillus subtilis and many others with marginal identity score.  [...] ","uniprotId":"Q0PBE7_CAMJE"},"10641213":{"id":10641213,"externalId":"192222.Cj0364","name":"Cj0364","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0364, unknown, len: 83 aa; no Hp match; Updated (2006) note: Literature search identified paper giving further clues to product function. Functional classification - Unknown","uniprotId":"Q0PBE6_CAMJE"},"10641214":{"id":10641214,"externalId":"192222.Cj0365c","name":"cmeC","hasFamily":true,"annotation":"Outer membrane channel protein CmeC (Multidrug efflux system CmeABC); Original (2000) note: Cj0365c, possible outer membrane channel protein, len: 492 aa; similar to members of the nodT/fusA family e.g. TR:O31101 (EMBL:AF029405) Pseudomonas putida outer membrane channel protein SRPC (470 aa), fasta scores; opt: 645 z-score: 834.4 E(): 0,29.2% identity in 465 aa overlap. and TR:Q51006 (EMBL:X95635) Neisseria gonorrhoeae mtrE gene (467 aa,fasta scores; opt: 663 z-score: 829.8 E(): 0, 31.1% identity in 476 aa overlap. No Hp match. Also some similarity to Cj0608 (28.6% identity in 455 aa o [...] ","uniprotId":"Q0PBE5_CAMJE"},"10641215":{"id":10641215,"externalId":"192222.Cj0366c","name":"cmeB","hasFamily":true,"annotation":"Original (2000) note: Cj0366c, probable transmembrane efflux protein, len: 1040 aa; similar to e.g. TR:P95422 (EMBL:X99514) Pseudomonas aeruginosa mexF (antibiotic efflux system protein) (1062 aa), fasta scores; opt: 1584 z-score: 3208.4 E(): 0, 41.2% identity in 1056 aa overlap, and ACRB_ECOLI acriflavin resistance protein B (1049 aa), fasta scores; opt: 2739 z-score: 3055.6 E(): 0, 41.7% identity in 1031 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00873 ACR_tran,AcrB/AcrD/AcrF family, score 1415.10, E-value 0; Updated (2006) note: Thirteen probable transmembrace helice [...] ","uniprotId":"Q0PBE4_CAMJE"},"10641216":{"id":10641216,"externalId":"192222.Cj0367c","name":"cmeA","hasFamily":true,"annotation":"Belongs to the membrane fusion protein (MFP) (TC 8.A.1) family","uniprotId":"Q0PBE3_CAMJE"},"10641217":{"id":10641217,"externalId":"192222.Cj0368c","name":"cmeR","hasFamily":true,"annotation":"Transcriptional regulator CmeR; Original (2000) note: Cj0368c, probable transcriptional regulatory protein, len: 210 aa; similar to members of the tetR family e.g. TR:G3327947 (EMBL:AF053772) Staphylococcus aureus transcriptional regulator qacR (regulator of multidrug efflux protein QacB) (188 aa), fasta scores; opt: 162 z-score: 285.0 E(): 1.3e-08, 29.3% identity in 198 aa overlap, and MTRR_NEIGO regulatory protein MTRR (210 aa), fasta scores; opt: 137 z-score: 210.6 E(): 0.00018, 31.3% identity in 83 aa overlap. No Hp match. Contains Pfam match to entry PF00440 tetR, Bacterial regula [...] ","uniprotId":"Q0PBE2_CAMJE"},"10641218":{"id":10641218,"externalId":"192222.Cj0369c","name":"Cj0369c","hasFamily":true,"annotation":"Original (2000) note: Cj0369c, ferredoxin domain-containing integral membrane protein, len: 458 aa; some similarity to e.g. RDXA_RHOSH RDXA protein (469 aa),fasta scores; opt: 474 z-score: 657.7 E(): 2.2e-29, 24.9% identity in 402 aa overlap, and FIXG_RHIME nitrogen fixation protein FIXG (524 aa), fasta scores; opt: 476 z-score: 533.8 E(): 1.8e-22, 28.7% identity in 296 aa overlap. 45.7% identity to HP1508. Contains PS00198 4Fe-4S ferredoxins, iron-sulfur binding region signature and Pfam match to entry PF00037 fer4, 4Fe-4S ferredoxins and related iron-sulfur cluster binding domains.,  [...] ","uniprotId":"Q0PBE1_CAMJE"},"10641219":{"id":10641219,"externalId":"192222.Cj0370","name":"rpsU","hasFamily":true,"annotation":"Belongs to the bacterial ribosomal protein bS21 family. ","uniprotId":"RS21_CAMJE"},"10641220":{"id":10641220,"externalId":"192222.Cj0371","name":"Cj0371","hasFamily":true,"annotation":"Original (2000) note: Cj0371, probable lipoprotein,len: 201 aa; similar to H. pylori gene fragment reported to be involved in flagellar motility TR:Q48273 (EMBL:Z68311) (122 aa), fasta scores; opt: 322 z-score: 449.5 E(): 8.8e-18, 46.1% identity in 115 aa overlap. 46.4% identity to HP0232. Contains N-terminal signal sequence and PS00013 Prokaryotic membrane lipoprotein lipid attachment site; Updated (2006) note: One probable transmembrace helices identified by TMHMM2.0 within CDS. Some similarity to Helicobacter pylori gene fragment. Putative kept within product function. Functional cl [...] ","uniprotId":"Y371_CAMJE"},"10641221":{"id":10641221,"externalId":"192222.Cj0372","name":"Cj0372","hasFamily":true,"annotation":"Putative glutathionylspermidine synthase; Original (2000) note: Cj0372, unknown, len: 389 aa; similar to hypothetical proteins e.g. YGIC_ECOLI (386 aa),fasta scores; opt: 552 z-score: 508.4 E(): 4.6e-21, 31.3% identity in 383 aa overlap. Also similar to N-terminus of GSP_ECOLI glutathionylspermidine synthase (619 aa), fasta scores; opt: 314 z-score: 248.3 E(): 1.4e-06, 24.4% identity in 381 aa overlap. 60.5% identity to HP0233; Updated (2006) note: Pfam domain PF03738 Glutathionylspermidine synthase identified within CDS. Characterised within Escherichia coli with marginal identity sco [...] ","uniprotId":"Q0PBD8_CAMJE"},"10641222":{"id":10641222,"externalId":"192222.Cj0373","name":"Cj0373","hasFamily":true,"annotation":"Belongs to the D-isomer specific 2-hydroxyacid dehydrogenase family","uniprotId":"Q0PBD7_CAMJE"},"10641223":{"id":10641223,"externalId":"192222.Cj0374","name":"Cj0374","hasFamily":true,"annotation":"Belongs to the UPF0234 family","uniprotId":"Y374_CAMJE"},"10641224":{"id":10641224,"externalId":"192222.Cj0375","name":"Cj0375","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0375, probable lipoprotein,len: 158 aa; no Hp match. Contains N-terminal signal sequence and PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PBD5_CAMJE"},"10641225":{"id":10641225,"externalId":"192222.Cj0376","name":"Cj0376","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0376, possible periplasmic protein, len: 274 aa; no Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Literature search identified paper linking protein to glycoprotein. Functional classification - Miscellaneous periplasmic proteins; PMID:12186869","uniprotId":"Q0PBD4_CAMJE"},"10641226":{"id":10641226,"externalId":"192222.Cj0377","name":"Cj0377","hasFamily":true,"annotation":"Putative AAA family ATPase; Original (2000) note: Cj0377, probable ATPase, len: 570 aa; weak similarity to many members of the AAA family e.g. TR:O28182 (EMBL:O28182) cell division control protein 48 (Archaeoglobus fulgidus AF2098) (811 aa), fasta scores; opt: 341 z-score: 312.6 E(): 3.7e-10, 26.4% identity in 435 aa overlap. No Hp match. Contains PS00017 ATP/GTP-binding site motif A (P-loop) and Pfam match to entry PF00004 AAA, ATPases associated with various cellular activities (AAA), score 106.20, E-value 6.3e-28; Updated (2006) note: Probable changed to putative. No specific charac [...] ","uniprotId":"Q0PBD3_CAMJE"},"10641227":{"id":10641227,"externalId":"192222.Cj0378c","name":"Cj0378c","hasFamily":true,"annotation":"Original (2000) note: Cj0378c, probable integral membrane protein, len: 179 aa; no Hp match; Updated (2006) note: Pfam domain PF01794 ferric reductase like transmembrane component identified within CDS. This was from a separate Pfam search. Pfam domain PF05252 Uncharacterised protein family (UPF0191) identified within CDS. Six probable transmembrace helices identified by TMHMM2.0 within CDS. Product functionality changed due to motif match. Literature search identified paper giving further support to product function. No specific characterisation has been carried out yet. Putative kept [...] ","uniprotId":"Q0PBD2_CAMJE"},"10641228":{"id":10641228,"externalId":"192222.Cj0379c","name":"msrP","hasFamily":true,"annotation":"Part of the MsrPQ system that repairs oxidized periplasmic proteins containing methionine sulfoxide residues (Met-O), using respiratory chain electrons. Thus protects these proteins from oxidative-stress damage caused by reactive species of oxygen and chlorine generated by the host defense mechanisms. MsrPQ is essential for the maintenance of envelope integrity under bleach stress, rescuing a wide series of structurally unrelated periplasmic proteins from methionine oxidation. The catalytic subunit MsrP is non-stereospecific, being able to reduce both (R-) and (S-) diastereoisomers of  [...] ","uniprotId":"MSRP_CAMJE"},"10641229":{"id":10641229,"externalId":"192222.Cj0380c","name":"Cj0380c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0380c, unknonw, len: 261 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PBD0_CAMJE"},"10641230":{"id":10641230,"externalId":"192222.Cj0381c","name":"pyrF","hasFamily":true,"annotation":"Catalyzes the decarboxylation of orotidine 5'-monophosphate (OMP) to uridine 5'-monophosphate (UMP)","uniprotId":"PYRF_CAMJE"},"10641231":{"id":10641231,"externalId":"192222.Cj0382c","name":"nusB","hasFamily":true,"annotation":"Involved in transcription antitermination. Required for transcription of ribosomal RNA (rRNA) genes. Binds specifically to the boxA antiterminator sequence of the ribosomal RNA (rrn) operons. ","uniprotId":"NUSB_CAMJE"},"10641232":{"id":10641232,"externalId":"192222.Cj0383c","name":"ribH","hasFamily":true,"annotation":"Catalyzes the formation of 6,7-dimethyl-8-ribityllumazine by condensation of 5-amino-6-(D-ribitylamino)uracil with 3,4-dihydroxy-2- butanone 4-phosphate. This is the penultimate step in the biosynthesis of riboflavin","uniprotId":"RISB_CAMJE"},"10641233":{"id":10641233,"externalId":"192222.Cj0384c","name":"kdsA","hasFamily":true,"annotation":"Belongs to the KdsA family","uniprotId":"KDSA_CAMJE"},"10641234":{"id":10641234,"externalId":"192222.Cj0385c","name":"Cj0385c","hasFamily":true,"annotation":"Original (2000) note: Cj0385c, probable integral membrane protein, len: 310 aa; similar to many hypothetical integral membrane proteins e.g. TR:O29973 (EMBL:AE001087) Archaeoglobus fulgidus AF0266 (276 aa),fasta scores; opt: 155 z-score: 221.8 E(): 4.5e-05, 26.9% identity in 279 aa overlap. 30.8% identity to HP1234. Contains Pfam match to entry PF00892 DUF6, Integral membrane protein, score 29.90, E-value 5.9e-05; Updated (2006) note: Pfam domain PF00892 Integral membrane protein DUF6 identified within CDS. Ten probable transmembrace helices identified by TMHMM2.0 within CDS. Further s [...] ","uniprotId":"Q0PBC5_CAMJE"},"10641235":{"id":10641235,"externalId":"192222.Cj0386","name":"der","hasFamily":true,"annotation":"GTPase that plays an essential role in the late steps of ribosome biogenesis","uniprotId":"DER_CAMJE"},"10641236":{"id":10641236,"externalId":"192222.Cj0387","name":"aroK","hasFamily":true,"annotation":"Catalyzes the specific phosphorylation of the 3-hydroxyl group of shikimic acid using ATP as a cosubstrate","uniprotId":"Q0PBC3_CAMJE"},"10641237":{"id":10641237,"externalId":"192222.Cj0388","name":"trpS","hasFamily":true,"annotation":"Catalyzes the attachment of tryptophan to tRNA(Trp). ","uniprotId":"SYW_CAMJE"},"10641238":{"id":10641238,"externalId":"192222.Cj0389","name":"serS","hasFamily":true,"annotation":"Catalyzes the attachment of serine to tRNA(Ser). Is also able to aminoacylate tRNA(Sec) with serine, to form the misacylated tRNA L- seryl-tRNA(Sec), which will be further converted into selenocysteinyl- tRNA(Sec)","uniprotId":"SYS_CAMJE"},"10641239":{"id":10641239,"externalId":"192222.Cj0390","name":"Cj0390","hasFamily":true,"annotation":"Original (2000) note: Cj0390, possible transmembrane protein, len: 820 aa; 28.7% identity in 703 aa overlap to HP1479. Contains possible transmembrane domain around aa 100 and Pfam match to entry PF00515 TPR,TPR Domain, score 26.90, E-value 0.00047; Updated (2006) note: Pfam domains x4 PF07719 Tetratricopeptide repeat were identified within CDS. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBC0_CAMJE"},"10641240":{"id":10641240,"externalId":"192222.Cj0391c","name":"Cj0391c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0391c, unknown, len: 211 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Literature search identified paper giving further clues to product function. Functional classification - Unknown","uniprotId":"Q0PBB9_CAMJE"},"10641241":{"id":10641241,"externalId":"192222.Cj0392c","name":"pyk","hasFamily":true,"annotation":"Belongs to the pyruvate kinase family. ","uniprotId":"Q0PBB8_CAMJE"},"10641242":{"id":10641242,"externalId":"192222.Cj0393c","name":"mqo","hasFamily":true,"annotation":"Catalyzes oxidation of malate to oxaloacetate in the citric acid cycle. Donates electrons to quinones of the electron transfer chain (By similarity)","uniprotId":"MQO_CAMJE"},"10641243":{"id":10641243,"externalId":"192222.Cj0394c","name":"coaX","hasFamily":true,"annotation":"Catalyzes the phosphorylation of pantothenate (Pan), the first step in CoA biosynthesis","uniprotId":"COAX_CAMJE"},"10641244":{"id":10641244,"externalId":"192222.Cj0395c","name":"Cj0395c","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0395c, unknown, len: 96 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PBB5_CAMJE"},"10641245":{"id":10641245,"externalId":"192222.Cj0396c","name":"Cj0396c","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0396c, probable lipoprotein, len: 332 aa; 28.9% identity to HP0863. Conatins N-terminal signal sequence and PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBB4_CAMJE"},"10641246":{"id":10641246,"externalId":"192222.Cj0397c","name":"Cj0397c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0397c, unknown, len: 192 aa; almost identical to C. jejuni hypothetical protein fragment TR:Q46096 (EMBL:U27272), 28.5% identity to HP0864; Updated (2006) note: One probable transmembrace helices identified by TMHMM2.0 within CDS. Literature serach identified paper giving further clues about product function. Functional classification - Conserved hypothetical proteins; PMID:10913692","uniprotId":"Q0PBB3_CAMJE"},"10641247":{"id":10641247,"externalId":"192222.Cj0398","name":"gatC","hasFamily":true,"annotation":"Allows the formation of correctly charged Asn-tRNA(Asn) or Gln-tRNA(Gln) through the transamidation of misacylated Asp-tRNA(Asn) or Glu-tRNA(Gln) in organisms which lack either or both of asparaginyl- tRNA or glutaminyl-tRNA synthetases. The reaction takes place in the presence of glutamine and ATP through an activated phospho-Asp- tRNA(Asn) or phospho-Glu-tRNA(Gln) (By similarity)","uniprotId":"GATC_CAMJE"},"10641248":{"id":10641248,"externalId":"192222.Cj0399","name":"Cj0399","hasFamily":true,"annotation":"Colicin V production protein homolog; Original (2000) note: Cj0399, probable integral membrane protein, len: 187 aa; identical to C. jejuni hypothetical protein fragment TR:O68605 (EMBL:AF052056) (123 aa). 26.9% identity in 171 aa overlap to HP0181 (224 aa); Updated (2006) note: Four probable transmembrace helices identified by TMHMM2.0 within CDS. Pfam domain PF02674 Colicin V production protein identified within CDS. Product modified to more speicific family member based on motif matches. No specific characterisation with acceptable identity score has been carried out yet. Putative k [...] ","uniprotId":"Q0PBB1_CAMJE"},"10641249":{"id":10641249,"externalId":"192222.Cj0400","name":"fur","hasFamily":true,"annotation":"Acts as a global negative controlling element, employing Fe(2+) as a cofactor to bind the operator of the repressed genes. ","uniprotId":"FUR_CAMJE"},"10641250":{"id":10641250,"externalId":"192222.Cj0401","name":"lysS","hasFamily":true,"annotation":"Belongs to the class-II aminoacyl-tRNA synthetase family. ","uniprotId":"SYK_CAMJE"},"10641251":{"id":10641251,"externalId":"192222.Cj0402","name":"glyA","hasFamily":true,"annotation":"Catalyzes the reversible interconversion of serine and glycine with tetrahydrofolate (THF) serving as the one-carbon carrier. This reaction serves as the major source of one-carbon groups required for the biosynthesis of purines, thymidylate, methionine, and other important biomolecules. Also exhibits THF-independent aldolase activity toward beta-hydroxyamino acids, producing glycine and aldehydes, via a retro-aldol mechanism (By similarity)","uniprotId":"GLYA_CAMJE"},"10641252":{"id":10641252,"externalId":"192222.Cj0403","name":"Cj0403","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0403, unknown, len: 181 aa; 57.8% identity to HP0184; Updated (2006) note: Literature search identified paper giving further clues to product function. Functional classification - Conserved hypothetical proteins; PMID:12595250","uniprotId":"Q0PBA7_CAMJE"},"10641253":{"id":10641253,"externalId":"192222.Cj0404","name":"Cj0404","hasFamily":true,"annotation":"Original (2000) note: Cj0404, probable transmembrane protein, len: 278 aa; no Hp match. Contains probable transmembrane domain around 40 aa; Updated (2006) note: Literature search identified paper giving further clues to product function. Functional classification - Membranes, lipoproteins and porins; PMID:12595250","uniprotId":"Q0PBA6_CAMJE"},"10641254":{"id":10641254,"externalId":"192222.Cj0405","name":"aroE","hasFamily":true,"annotation":"Involved in the biosynthesis of the chorismate, which leads to the biosynthesis of aromatic amino acids. Catalyzes the reversible NADPH linked reduction of 3-dehydroshikimate (DHSA) to yield shikimate (SA)","uniprotId":"Q0PBA5_CAMJE"},"10641255":{"id":10641255,"externalId":"192222.Cj0406c","name":"Cj0406c","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0406c, probable lipoprotein, len: 299 aa; no Hp match. Contains N-terminal signal sequence and PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PBA4_CAMJE"},"10641256":{"id":10641256,"externalId":"192222.Cj0407","name":"lgt","hasFamily":true,"annotation":"Catalyzes the transfer of the diacylglyceryl group from phosphatidylglycerol to the sulfhydryl group of the N-terminal cysteine of a prolipoprotein, the first step in the formation of mature lipoproteins","uniprotId":"LGT_CAMJE"},"10641257":{"id":10641257,"externalId":"192222.Cj0408","name":"frdC","hasFamily":true,"annotation":"Original (2000) note: Cj0408, frdC, probable fumarate reductase cytochrome B subunit, len: 260 aa; similar to FRDC_WOLSU fumarate reductase cytochrome B subunit (256 aa), fasta scores; opt: 823 z-score: 1258.2 E(): 0, 53.6% identity in 235 aa overlap. 50.0% identity to HP0193; Updated (2006) note: Pfam domain PF02967 Fumarate reductase respiratory complex was identified within CDS. Five probable transmembrane helices predicted by TMHMM2.0. Characterised within Wolinella succinogenes with acceptable identity score. Putative not added to product function. Functional classification - Ener [...] ","uniprotId":"Q0PBA2_CAMJE"},"10641258":{"id":10641258,"externalId":"192222.Cj0409","name":"frdA","hasFamily":true,"annotation":"Original (2000) note: Cj0409, frdA, probable fumarate reductase flavoprotein subunit, len: 663 aa; FRDA_WOLSU fumarate reductase flavoprotein subunit (EC 1.3.99.1) (656 aa), fasta scores; opt: 3050 z-score: 3174.9 E(): 0, 68.2% identity in 660 aa overlap. 62.2% identity to HP0192. Contains Pfam match to entry PF00890 FAD_binding_2, FAD binding domain, score 535.80, E-value 2.9e-157; Updated (2006) note: Pfam domain PF02910 Fumarate reductase/succinate dehydrogenase was identified within CDS. Characterised within Wolinella succinogenes with acceptable identity score. Putative not added  [...] ","uniprotId":"Q0PBA1_CAMJE"},"10641259":{"id":10641259,"externalId":"192222.Cj0410","name":"frdB","hasFamily":true,"annotation":"Belongs to the succinate dehydrogenase/fumarate reductase iron-sulfur protein family","uniprotId":"Q0PBA0_CAMJE"},"10641260":{"id":10641260,"externalId":"192222.Cj0411","name":"Cj0411","hasFamily":true,"annotation":"Original (2000) note: Cj0411, probable ATP/GTP binding protein, len: 728 aa; some similarity in C-term to YOR6_CALSR hypothetical protein (402 aa), fasta scores; opt: 220 z-score: 284.9 E(): 1.4e-08, 25.2% identity in 425 aa overlap, and to C-term of YPBR_BACSU hypothetical protein (1193 aa), fasta scores; opt: 173 z-score: 275.1 E(): 5e-08, 20.1% identity in 722 aa overlap. No Hp match. Contains PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Pfam domain PF00350 Dynamin family protein was identified within CDS. Dynamin is a member of the G-protein superfamily clan. [...] ","uniprotId":"Q0PB99_CAMJE"},"10641261":{"id":10641261,"externalId":"192222.Cj0412","name":"Cj0412","hasFamily":true,"annotation":"Original (2000) note: Cj0412, probable ATP/GTP binding protein, len: 609 aa; some similarity to YPBR_BACSU hypothetical protein (1193 aa), fasta scores; opt: 172 z-score: 237.5 E(): 6.2e-06, 23.7% identity in 439 aa overlap. No Hp match. Contains PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Pfam domain PF00350 Dynamin family protein was identified within CDS. Dynamin is a member of the G-protein superfamily clan. ATP/GTP binding proteins fall under this particular clan. Further support given to product function. Literature search identified paper giving further c [...] ","uniprotId":"Q0PB98_CAMJE"},"10641262":{"id":10641262,"externalId":"192222.Cj0413","name":"Cj0413","hasFamily":true,"annotation":"Hydrolyzes 6-aminopenicillinic acid and 7- aminocephalosporanic acid (ACA) derivatives. ","uniprotId":"Q0PB97_CAMJE"},"10641263":{"id":10641263,"externalId":"192222.Cj0414","name":"Cj0414","hasFamily":true,"annotation":"Original (2000) note: Cj0414, possible oxidoreductase subunit, len: 242 aa; some similarity to TR:O34213 (EMBL:U97665) Erwinia cypripedii gluconate dehydrogenase subunit III precursor (220 aa), fasta scores; opt: 343 z-score: 502.3 E(): 1.1e-20, 32.6% identity in 193 aa overlap. No Hp match; Updated (2006) note: No specific characterisation with acceptable identity score has been carried out yet. Putative kept within product function. Literature search identified paper giving further clues to product function. Functional classification - Misc; PMID:15231804","uniprotId":"Q0PB96_CAMJE"},"10641264":{"id":10641264,"externalId":"192222.Cj0415","name":"Cj0415","hasFamily":true,"annotation":"Original (2000) note: Cj0415, possible oxidoreductase subunit, len: 573 aa; similar to e.g. TR:O34214 (EMBL:U97665) Erwinia cypripedii gluconate dehydrogenase subunit precursor (615 aa), fasta scores; opt: 1025 z-score: 1674.4 E(): 0, 40.8% identity in 583 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF00732 GMC (glucose-methanol-choline) oxidoreductase was identified within CDS when carrying out own Pfam search. This family is a member of the FAD/NAD(P)-binding Rossmann fold Superfamily clan. These proteins bind FAD as a cofactor. Product modified to more specific family  [...] ","uniprotId":"Q0PB95_CAMJE"},"10641265":{"id":10641265,"externalId":"192222.Cj0416","name":"Cj0416","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0416, unknown, len: 32 aa; no Hp match, highly simlar to N-terminus of Cj0122 (88.2% identity in 34 aa overlap). Functional classification -Unknown","uniprotId":"Q0PB94_CAMJE"},"10641266":{"id":10641266,"externalId":"192222.Cj0417","name":"Cj0417","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0417, unknown, len: 48 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PB93_CAMJE"},"10641267":{"id":10641267,"externalId":"192222.Cj0418c","name":"Cj0418c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0418c, unknown, len: 234 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PB92_CAMJE"},"10641268":{"id":10641268,"externalId":"192222.Cj0419","name":"Cj0419","hasFamily":true,"annotation":"Putative histidine triad (HIT) family protein; Original (2000) note: Cj0419, unknown, len: 120 aa; no Hp match; Updated (2006) note: Pfam domain PF01230 HIT (histidine triad) was identified within CDS by carrying out own Pfam search. Product modified to more specific family member due to motif match. No specific characterisation has been carried out yet. Literature search identified paper giving further information on HIT domains. Functional classification - Misc; PMID:1472710, PMID:9164465","uniprotId":"Q0PB91_CAMJE"},"10641269":{"id":10641269,"externalId":"192222.Cj0420","name":"Cj0420","hasFamily":true,"annotation":"Belongs to the UPF0312 family. ","uniprotId":"Q0PB90_CAMJE"},"10641270":{"id":10641270,"externalId":"192222.Cj0421c","name":"Cj0421c","hasFamily":true,"annotation":"Original (2000) note: Cj0421c, probable integral membrane protein, len: 318 aa; no Hp match; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PB89_CAMJE"},"10641271":{"id":10641271,"externalId":"192222.Cj0422c","name":"Cj0422c","hasFamily":true,"annotation":"Putative H-T-H containing protein; Original (2000) note: Cj0422c, unknown, len: 69 aa; no Hp match. Contains probable helix-turn-helix motif at aa 5-26 (Score 1410, +3.99 SD). Functional classification - Broad regulatory functions","uniprotId":"Q0PB88_CAMJE"},"10641272":{"id":10641272,"externalId":"192222.Cj0423","name":"Cj0423","hasFamily":false,"annotation":"Original (2000) note: Cj0423, probable integral membrane protein, len: 75 aa; similar to immunity proteins of T4 and T2 phage e.g. IMM_BPT4 immunity protein (83 aa),fasta scores; opt: 174 z-score: 240.8 E(): 4e-06, 40.2% identity in 82 aa overlap. No Hp match; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PB87_CAMJE"},"10641273":{"id":10641273,"externalId":"192222.Cj0424","name":"Cj0424","hasFamily":true,"annotation":"Original (2000) note: Cj0424, probable acidic periplasmic protein, len: 210 aa; no Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF07661 MORN repeat variant was identified within CDS. The MORN (Membrane Occupation and Recognition Nexus) repeat is found in multiple copies in several proteins. The function of this motif is unknown. Literature search identified paper giving functional link to iron stimulation. Functional classification - Miscellaneous periplasmic proteins; PMID:15632442","uniprotId":"Q0PB86_CAMJE"},"10641274":{"id":10641274,"externalId":"192222.Cj0425","name":"Cj0425","hasFamily":false,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0425, probable periplasmic protein, len: 138 aa; no Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Literature search identified paper giving functional link to iron stimulation. Functional classification - Miscellaneous periplasmic proteins; PMID:15632442","uniprotId":"Q0PB85_CAMJE"},"10641275":{"id":10641275,"externalId":"192222.Cj0426","name":"Cj0426","hasFamily":true,"annotation":"Original (2000) note: Cj0426, probable ABC transporter ATP-binding protein, len: 525 aa; simlar to many e.g. YBIT_ECOLI hypothetical ABC transporter ATP-binding protein (530 aa), fasta scores; opt: 1461 z-score: 1750.6 E(): 0, 42.8% identity in 526 aa overlap. 60.5% identity to HP0853. Contains 2x PS00017 ATP/GTP-binding site motif A (P-loop) and Pfam match to entry PF00005 ABC_tran, ABC transporters, scores 115.00, E-value 1.4e-30, and 120.70, E-value 2.7e-32; Updated (2006) note: Some characterisation within Escherichia coli with acceptable identity score. Putative added to product f [...] ","uniprotId":"Q0PB84_CAMJE"},"10641276":{"id":10641276,"externalId":"192222.Cj0427","name":"Cj0427","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0427, unknown, len: 111 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PB83_CAMJE"},"10641277":{"id":10641277,"externalId":"192222.Cj0428","name":"Cj0428","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0428, unknown, len: 127 aa; no Hp match; Updated (2006) note: Literature search identified papers linking product function to be involved with motility. Functional classification - Unknown; PMID:15812042, PMID:14985343","uniprotId":"Q0PB82_CAMJE"},"10641278":{"id":10641278,"externalId":"192222.Cj0429c","name":"Cj0429c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0429c, unknown, len: 194 aa; similar to hypothetical proteins e.g. YIGZ_ECOLI (205 aa), fasta scores; opt: 241 z-score: 370.8 E(): 2.3e-13,40.2% identity in 112 aa overlap. 39.1% identity to HP1485 (misannotated as proline dipeptidase); Updated (2006) note: Pfam domain PF01205 Uncharacterized protein family (UPF0029) was identified within CDS. Conserved added to product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PB81_CAMJE"},"10641279":{"id":10641279,"externalId":"192222.Cj0430","name":"Cj0430","hasFamily":true,"annotation":"Original (2000) note: Cj0430, probable integral membrane protein, len: 408 aa; 33.8% identity to HP1235; Updated (2006) note: Nine probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Literature search identified paper linking product function to stress response. Functional classification - Membranes, lipoproteins and porins; PMID:16045618","uniprotId":"Q0PB80_CAMJE"},"10641280":{"id":10641280,"externalId":"192222.Cj0431","name":"Cj0431","hasFamily":true,"annotation":"Original (2000) note: Cj0431, possible periplasmic ATP/GTP-binding protein, len: 204 aa; no Hp match; contains possible N-terminal signal sequence and PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: One probable transmembrane helices predicted by TMHMM2.0. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PB79_CAMJE"},"10641281":{"id":10641281,"externalId":"192222.Cj0432c","name":"murD","hasFamily":true,"annotation":"Cell wall formation. Catalyzes the addition of glutamate to the nucleotide precursor UDP-N-acetylmuramoyl-L-alanine (UMA). ","uniprotId":"MURD_CAMJE"},"10641282":{"id":10641282,"externalId":"192222.Cj0433c","name":"mraY","hasFamily":true,"annotation":"First step of the lipid cycle reactions in the biosynthesis of the cell wall peptidoglycan","uniprotId":"MRAY_CAMJE"},"10641283":{"id":10641283,"externalId":"192222.Cj0434","name":"pgm","hasFamily":true,"annotation":"Catalyzes the interconversion of 2-phosphoglycerate and 3- phosphoglycerate","uniprotId":"GPMI_CAMJE"},"10641284":{"id":10641284,"externalId":"192222.Cj0435","name":"fabG","hasFamily":true,"annotation":"Catalyzes the NADPH-dependent reduction of beta-ketoacyl-ACP substrates to beta-hydroxyacyl-ACP products, the first reductive step in the elongation cycle of fatty acid biosynthesis. ","uniprotId":"Q0PB75_CAMJE"},"10641285":{"id":10641285,"externalId":"192222.Cj0436","name":"Cj0436","hasFamily":true,"annotation":"Putative pyridoxamine 5'-phosphate oxidase; Original (2000) note: Cj0436, unknown, len: 136 aa; no Hp match; Updated (2006) note: Pfam domain PF01243 Pyridoxamine 5'-phosphate oxidase was identified within CDS. Product function modified to more specific family member based on motif match. Pyridoxamine 5'-phosphate oxidase (1.4.3.5) is a FMN flavoprotein involved in the de novo synthesis of pyridoxine (vitamin B6) and pyridoxal phosphate. It oxidizes pyridoxamine-5-P (PMP) and pyridoxine-5-P (PNP) to pyridoxal-5-P. No specific characterisation with acceptable identity scores has been ca [...] ","uniprotId":"Q0PB74_CAMJE"},"10641286":{"id":10641286,"externalId":"192222.Cj0437","name":"sdhA","hasFamily":true,"annotation":"Original (2000) note: Cj0437, sdhA, probable succinate dehydrogenase flavoprotein subunit, len: 611 aa; similar to many e.g. DHSA_ECOLI succinate dehydrogenase flavoprotein subunit (EC 1.3.99.1) (588 aa), fasta scores; opt: 1036 z-score: 1136.5 E(): 0, 37.3% identity in 569 aa overlap and to e.g. FRDA_WOLSU fumarate reductase flavoprotein subunit (656 aa), fasta scores; opt: 862 z-score: 1211.2 E(): 0, 33.3% identity in 595 aa overlap. No Hp ortholog, but 32.3% identity to HP0192 fumarate reductase, flavoprotein subunit (frdA), and 33.8% identity to C. jejuni frdA (Cj0409). Contains Pf [...] ","uniprotId":"Q0PB73_CAMJE"},"10641287":{"id":10641287,"externalId":"192222.Cj0438","name":"sdhB","hasFamily":true,"annotation":"Original (2000) note: Cj0438, sdhB, possibleble succinate dehydrogenase iron-sulfur protein, len: 321 aa; similar to e.g. DHSB_MYCGR succinate dehydrogenase (ubiquinone) iron-sulfur protein (EC 1.3.5.1) (297 aa),fasta scores; opt: 352 z-score: 442.4 E(): 2.4e-17, 29.3% identity in 215 aa overlap. No Hp ortholog, but 28.1% identical to HP0191 fumarate reductase, iron-sulfur subunit (frdB). Contains PS00197 2Fe-2S ferredoxins,iron-sulfur binding region signature, PS00198 4Fe-4S ferredoxins, iron-sulfur binding region signature, and Pfam match to entry PF00037 fer4, 4Fe-4S ferredoxins and [...] ","uniprotId":"Q0PB72_CAMJE"},"10641288":{"id":10641288,"externalId":"192222.Cj0439","name":"sdhC","hasFamily":true,"annotation":"Original (2000) note: Cj0439, sdhC, probable succinate dehydrogenase subunit C, len: 285 aa; similar to e.g. TR:P77945 (EMBL:Y09041) Sulfolobus acidocaldarius succinate dehydrogenase subunit C (EC 1.3.99.1) (290 aa),fasta scores; opt: 612 z-score: 739.6 E(): 0, 34.2% identity in 275 aa overlap. No Hp match; Updated (2006) note: Pfam domains x2 PF02754 Cysteine-rich domain was identified within CDS. This domain is usually found in two copies per protein. It contains up to four conserved cysteines. No specific characterisation with acceptable identity score has been carried out yet. Puta [...] ","uniprotId":"Q0PB71_CAMJE"},"10641289":{"id":10641289,"externalId":"192222.Cj0440c","name":"Cj0440c","hasFamily":true,"annotation":"Catalyzes an amino-pyrimidine hydrolysis reaction at the C5' of the pyrimidine moiety of thiamine compounds, a reaction that is part of a thiamine salvage pathway","uniprotId":"Q0PB70_CAMJE"},"10641290":{"id":10641290,"externalId":"192222.Cj0441","name":"acpP","hasFamily":true,"annotation":"Carrier of the growing fatty acid chain in fatty acid biosynthesis","uniprotId":"ACP_CAMJE"},"10641291":{"id":10641291,"externalId":"192222.Cj0442","name":"fabF","hasFamily":true,"annotation":"Catalyzes the condensation reaction of fatty acid synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP. ","uniprotId":"Q0PB68_CAMJE"},"10641292":{"id":10641292,"externalId":"192222.Cj0443","name":"accA","hasFamily":true,"annotation":"Component of the acetyl coenzyme A carboxylase (ACC) complex. First, biotin carboxylase catalyzes the carboxylation of biotin on its carrier protein (BCCP) and then the CO(2) group is transferred by the carboxyltransferase to acetyl-CoA to form malonyl-CoA. ","uniprotId":"ACCA_CAMJE"},"10641293":{"id":10641293,"externalId":"192222.Cj0447","name":"Cj0447","hasFamily":true,"annotation":"Putative NUDIX hydrolase family protein; HMMPfam hit to PF00593, TonB dependent receptor,score 3.1e-33","uniprotId":"Q0PB66_CAMJE"},"10641294":{"id":10641294,"externalId":"192222.Cj0448c","name":"Cj0448c","hasFamily":true,"annotation":"Original (2000) note: Cj0448c, probable MCP-type signal transduction protein, len: 365 aa; similar to many proteins with MCP domains e.g. TR:Q56633 (EMBL:L25660) Vibrio cholerae acfB accessory colonization factor (626 aa), fasta scores; opt: 364 z-score: 455.7 E(): 4.4e-18,29.9% identity in 278 aa overlap, and PILJ_PSEAE PILJ protein (682 aa), fasta scores; opt: 314 z-score: 451.9 E(): 7.1e-18, 30.7% identity in 189 aa overlap. 31.9% identity to HP0599 (called hemolysin secretion protein precursor (hylB), but previously sequenced as TR:P94847 H.pylori 48 kDa antigen (433 aa), fasta sco [...] ","uniprotId":"Q0PB65_CAMJE"},"10641295":{"id":10641295,"externalId":"192222.Cj0449c","name":"Cj0449c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0449c, unknown, len: 71 aa; similar to E. coli hypothetical protein YDCH_ECOLI (55 aa), fasta scores; opt: 112 z-score: 209.2 E(): 0.00023,42.6% identity in 47 aa overlap. 45.1% identity to HP1242; Updated (2006) note: Pfam domain PF04325 Protein of unknown function (DUF465) was identified within CDS. Conserved added to product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PB64_CAMJE"},"10641296":{"id":10641296,"externalId":"192222.Cj0450c","name":"rpmB","hasFamily":true,"annotation":"Belongs to the bacterial ribosomal protein bL28 family. ","uniprotId":"RL28_CAMJE"},"10641297":{"id":10641297,"externalId":"192222.Cj0451","name":"rep","hasFamily":true,"annotation":"Belongs to the ribulose-phosphate 3-epimerase family. ","uniprotId":"Q0PB62_CAMJE"},"10641298":{"id":10641298,"externalId":"192222.Cj0452","name":"dnaQ","hasFamily":true,"annotation":"Original (2000) note: Cj0452, dnaQ, exonuclease,possibly dna polymerase III epsilon subunit, len: 253 aa; similar to part of e.g. DP3A_MYCPU DNA polymerase III,alpha chain (EC 2.7.7.7) (1435 aa), fasta scores; opt: 220 z-score: 322.2 E(): 1.2e-10, 27.7% identity in 267 aa overlap, and DING_BACSU probable ATP-dependent helicase DING homolog (931 aa), fasta scores; opt: 275 z-score: 312.6 E(): 4.1e-10, 35.3% identity in 170 aa overlap. Also some similarity to DP3E_ECOLI DNA polymerase III, epsilon chain (EC 2.7.7.7), wublastn scores; E= 3.2e-05, 25% identity in 174 aa overlap. 41.5% iden [...] ","uniprotId":"Q0PB61_CAMJE"},"10641299":{"id":10641299,"externalId":"192222.Cj0453","name":"thiC","hasFamily":true,"annotation":"Catalyzes the synthesis of the hydroxymethylpyrimidine phosphate (HMP-P) moiety of thiamine from aminoimidazole ribotide (AIR) in a radical S-adenosyl-L-methionine (SAM)-dependent reaction. ","uniprotId":"THIC_CAMJE"},"10641300":{"id":10641300,"externalId":"192222.Cj0454c","name":"Cj0454c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0454c, possible membrane protein, len: 189 aa; no Hp match; Updated (2006) note: One probable transmembrane helices predicted by TMHMM2.0 within CDS. Literature search identified paper linking product function to motility. Functional classification - Membranes,lipoproteins and porins; PMID:15066034","uniprotId":"Q0PB59_CAMJE"},"10641301":{"id":10641301,"externalId":"192222.Cj0455c","name":"Cj0455c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0455c, possible membrane protein, len: 114 aa; no Hp match; Updated (2006) note: One probable transmembrane helices predicted by TMHMM2.0 within CDS. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PB58_CAMJE"},"10641302":{"id":10641302,"externalId":"192222.Cj0456c","name":"Cj0456c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0456c, unknown, len: 319 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PB57_CAMJE"},"10641303":{"id":10641303,"externalId":"192222.Cj0457c","name":"Cj0457c","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0457c, probable lipoprotein, len: 209 aa; similar to hypothetical protein fromRICKETTSIA PROWAZEKII TR:E1342403 (EMBL:AJ235270) RP090 (218 aa), fasta scores; opt: 372 z-score: 531.3 E(): 2.7e-22, 33.0% identity in 212 aa overlap. 35.9% identity in 153 aa overlap to HP0270. Contains N-terminal signal sequence and PS00013 Prokaryotic membrane lipoprotein lipid attachment site; Updated (2006) note: Pfam domain PF04028 Domain of unknown function (DUF374) identified within CDS. One probable transmembrane helices predicted by TMHMM2.0 within CDS. [...] ","uniprotId":"Q0PB56_CAMJE"},"10641304":{"id":10641304,"externalId":"192222.Cj0458c","name":"miaB","hasFamily":true,"annotation":"Catalyzes the methylthiolation of N6-(dimethylallyl)adenosine (i(6)A), leading to the formation of 2-methylthio-N6- (dimethylallyl)adenosine (ms(2)i(6)A) at position 37 in tRNAs that read codons beginning with uridine","uniprotId":"MIAB_CAMJE"},"10641305":{"id":10641305,"externalId":"192222.Cj0459c","name":"Cj0459c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0459c, unknown, len: 88 aa; 32.5% identical to HP0268. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PB54_CAMJE"},"10641306":{"id":10641306,"externalId":"192222.Cj0460","name":"nusA","hasFamily":true,"annotation":"Participates in both transcription termination and antitermination","uniprotId":"Q0PB53_CAMJE"},"10641307":{"id":10641307,"externalId":"192222.Cj0461c","name":"Cj0461c","hasFamily":true,"annotation":"Putative MFS (Major Facilitator Superfamily) transport protein; Original (2000) note: Cj0461c, probable integral membrane protein, len: 395 aa; some similarity to a hypothetical protein from B. subtilis TR:O31400 (EMBL:AJ222587) YKUC (430 aa), fasta scores; opt: 172 z-score: 369.7 E(): 2.7e-13, 23.8% identity in 407 aa overlap. No Hp match; Updated (2006) note: Pfam domains PF07690 Major Facilitator Superfamily (MFS) and PF04332 Protein of unknown function (DUF475) were identified within CDS. Twelve probable transmembrane helices predicted by TMHMM2.0. Product modified to more specific [...] ","uniprotId":"Q0PB52_CAMJE"},"10641308":{"id":10641308,"externalId":"192222.Cj0462","name":"mqnC","hasFamily":true,"annotation":"Radical SAM enzyme that catalyzes the cyclization of dehypoxanthine futalosine (DHFL) into cyclic dehypoxanthine futalosine (CDHFL), a step in the biosynthesis of menaquinone (MK, vitamin K2). ","uniprotId":"Q0PB51_CAMJE"},"10641309":{"id":10641309,"externalId":"192222.Cj0463","name":"Cj0463","hasFamily":true,"annotation":"Original (2000) note: Cj0463, zinc protease-like protein, len: 406 aa; some similarity to members of the peptidase M16 (insulinase) family e.g. Y4WB_RHISN hypothetical zinc protease-like protein Y4WB (447 aa),fasta scores; opt: 375 z-score: 306.4 E(): 9.1e-10, 22.7% identity in 396 aa overlap. 39.3% identity to HP0657; Updated (2006) note: Pfam domain PF05193 Peptidase M16 inactive domain identified within CDS. No specific characterisation has been carried out yet. Product function left unchanged. Functional classification -Degradation of macromolecules - Proteins, peptides and glycopeptides","uniprotId":"Q0PB50_CAMJE"},"10641310":{"id":10641310,"externalId":"192222.Cj0464","name":"recG","hasFamily":true,"annotation":"ATP-dependent DNA helicase; Original (2000) note: Cj0464, recG, probable ATP-dependent DNA helicase, len: 607 aa; similarto many e.g. RECG_ECOLI ATP-dependent DNA helicase RECG (EC 3.6.1.-) (693 aa), fasta scores; opt: 491 z-score: 773.3 E(): 0,34.6% identity in 431 aa overlap. 42.9% identity to HP1523. Contains PS00017 ATP/GTP-binding site motif A (P-loop) and Pfam match to entry PF00271 helicase_C,Helicases conserved C-terminal domain, score 53.20,E-value 5.5e-12; Updated (2006) note: Pfam domain PF04851 Type III restriction enzyme, res subunit and PF00270 DEAD/DEAH box helicase iden [...] ","uniprotId":"Q0PB49_CAMJE"},"10641311":{"id":10641311,"externalId":"192222.Cj0465c","name":"ctb","hasFamily":true,"annotation":"Has been suggested to be involved in cytochrome c peroxidase or P450-like oxygen chemistry or cyanide detoxification. The high oxygen affinity of this protein suggests that it probably does not function as an oxygen transporter","uniprotId":"TRHBP_CAMJE"},"10641312":{"id":10641312,"externalId":"192222.Cj0466","name":"nssR","hasFamily":true,"annotation":"Transcriptional regulator; Original (2000) note: Cj0466, probable transcriptional regulator, len: 198 aa; similar to members of the crp/fnr family e.g. CRP_SALTY catabolite gene activator (210 aa), fasta scores; opt: 191 z-score: 236.7 E(): 6.9e-06, 24.1% identity in 195 aa overlap, and FNR_BACSU anaerobic regulatory protein (238 aa), fasta scores; opt: 182 z-score: 215.5 E(): 0.0001, 23.0% identity in 209 aa overlap. No Hp match. Contains helix-turn-helix motif at aa 150-171 (Score 1452, +4.13 SD); Updated (2006) note: Pfam domain PF00027 Cyclic nucleotide-binding domain identified wi [...] ","uniprotId":"Q0PB47_CAMJE"},"10641313":{"id":10641313,"externalId":"192222.Cj0467","name":"Cj0467","hasFamily":true,"annotation":"Original (2000) note: Cj0467, probable amino-acid ABC transporter integral membrane protein, len: 233 aa; similar to many e.g. GLNP_ECOLI glutamine transport system permease protein (219 aa), fasta scores; opt: 237 z-score: 468.8 E(): 8.1e-19, 32.6% identity in 233 aa overlap. 45.3% identity to HP1169. Contains PS00402 Binding-protein-dependent transport systems inner membrane comp sign. and Pfam match to entry PF00528 BPD_transp,Binding-protein-dependent transport systems inner membrane component, score 50.10, E-value 4.9e-11; Updated (2006) note: Literature search identified paper wi [...] ","uniprotId":"Q0PB46_CAMJE"},"10641314":{"id":10641314,"externalId":"192222.Cj0468","name":"Cj0468","hasFamily":true,"annotation":"Original (2000) note: Cj0468, probable amino-acid ABC transporter integral membrane protein, len: 218 aa; similar to many e.g. GLTJ_ECOLI glutamate/aspartate transport system permease (246 aa), fasta scores; opt: 194 z-score: 425.1 E(): 2.2e-16, 25.7% identity in 222 aa overlap. 49.3% identity to HP1170. Contains PS00402 Binding-protein-dependent transport systems inner membrane comp sign. and Pfam match to entry PF00528 BPD_transp,Binding-protein-dependent transport systems inner membrane component, score 52.80, E-value 7.6e-12; Updated (2006) note: Literature search identified paper  [...] ","uniprotId":"Q0PB45_CAMJE"},"10641315":{"id":10641315,"externalId":"192222.Cj0469","name":"Cj0469","hasFamily":true,"annotation":"Original (2000) note: Cj0469, probable amino-acid ABC transporter ATP-binding protein, len: 2t53 aa; highly similar to many e.g. GLNQ_BACST glutamine transport ATP-binding protein (242 aa), fasta scores; opt: 902 z-score: 1279.6 E(): 0, 55.4% identity in 242 aa overlap. 61.4% identity to HP1171. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran, ABC transporters, score 227.80, E-value 1.6e-64; Updated (2006) note: Literature search identified paper within Campylobacter jejuni linking Cj0467, Cj046 [...] ","uniprotId":"Q0PB44_CAMJE"},"10641316":{"id":10641316,"externalId":"192222.Cj0470","name":"tuf","hasFamily":true,"annotation":"This protein promotes the GTP-dependent binding of aminoacyl- tRNA to the A-site of ribosomes during protein biosynthesis. ","uniprotId":"EFTU_CAMJE"},"10641317":{"id":10641317,"externalId":"192222.Cj0471","name":"rpmG","hasFamily":true,"annotation":"Belongs to the bacterial ribosomal protein bL33 family. ","uniprotId":"RL33_CAMJE"},"10641318":{"id":10641318,"externalId":"192222.Cj0472","name":"secE","hasFamily":true,"annotation":"Essential subunit of the Sec protein translocation channel SecYEG. Clamps together the 2 halves of SecY. May contact the channel plug during translocation","uniprotId":"Q0PB41_CAMJE"},"10641319":{"id":10641319,"externalId":"192222.Cj0473","name":"nusG","hasFamily":true,"annotation":"Participates in transcription elongation, termination and antitermination","uniprotId":"NUSG_CAMJE"},"10641320":{"id":10641320,"externalId":"192222.Cj0474","name":"rplK","hasFamily":true,"annotation":"Forms part of the ribosomal stalk which helps the ribosome interact with GTP-bound translation factors","uniprotId":"RL11_CAMJE"},"10641321":{"id":10641321,"externalId":"192222.Cj0475","name":"rplA","hasFamily":true,"annotation":"Binds directly to 23S rRNA. The L1 stalk is quite mobile in the ribosome, and is involved in E site tRNA release. ","uniprotId":"RL1_CAMJE"},"10641322":{"id":10641322,"externalId":"192222.Cj0476","name":"rplJ","hasFamily":true,"annotation":"Forms part of the ribosomal stalk, playing a central role in the interaction of the ribosome with GTP-bound translation factors. ","uniprotId":"RL10_CAMJE"},"10641323":{"id":10641323,"externalId":"192222.Cj0477","name":"rplL","hasFamily":true,"annotation":"Forms part of the ribosomal stalk which helps the ribosome interact with GTP-bound translation factors. Is thus essential for accurate translation","uniprotId":"RL7_CAMJE"},"10641324":{"id":10641324,"externalId":"192222.Cj0478","name":"rpoB","hasFamily":true,"annotation":"DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates. ","uniprotId":"RPOB_CAMJE"},"10641325":{"id":10641325,"externalId":"192222.Cj0479","name":"rpoC","hasFamily":true,"annotation":"DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates. ","uniprotId":"RPOC_CAMJE"},"10641326":{"id":10641326,"externalId":"192222.Cj0480c","name":"Cj0480c","hasFamily":true,"annotation":"Original (2000) note: Cj0480c, probable transcriptional regulator, len: 253 aa; similar to many e.g. KDGR_ECOLI transcriptional regulator KDGR (263 aa),fasta scores; opt: 305 z-score: 319.1 E(): 1.8e-10, 24.1% identity in 212 aa overlap. No Hp match. Contains helix-turn-helix motif at aa 21-42 (Score 1424, +4.04 SD); Updated (2006) note: Pfam domain PF01614 Bacterial transcriptional regulator identified within CDS. This family of bacterial transcriptional regulators includes the glycerol operon regulatory protein and acetate operon repressor both of which are members of the iclR family [...] ","uniprotId":"Q0PB33_CAMJE"},"10641327":{"id":10641327,"externalId":"192222.Cj0481","name":"dapA","hasFamily":true,"annotation":"Catalyzes the reversible aldol cleavage of N-acetylneuraminic acid (sialic acid; Neu5Ac) to form pyruvate and N-acetylmannosamine (ManNAc) via a Schiff base intermediate. ","uniprotId":"Q0PB32_CAMJE"},"10641328":{"id":10641328,"externalId":"192222.Cj0482","name":"uxaA'","hasFamily":true,"annotation":"Original (2000) note: Cj0482, uxaA' possible altronate hydrolase N-terminus, len: 87 aa; similar to N-terminus of e.g. UXAA_ECOLI altronate hydrolase (495 aa), fasta scores; opt: 199 z-score: 322.2 E(): 1.2e-10. 46.2% identity in 78 aa overlap. Simlarity continues in downstream ORF Cj0485. No Hp match. Note that B. subtilis paralog of uxaA, YCBI_BACSU, also has a separate N-terminus (YCBH_BACSU); Updated (2006) note: Pfam domain PF04292 D-galactarate dehydratase / Altronate hydrolase identified within CDS. Further support given to product function. Characterisation work carried out wit [...] ","uniprotId":"Q0PB31_CAMJE"},"10641329":{"id":10641329,"externalId":"192222.Cj0483","name":"Cj0483","hasFamily":true,"annotation":"Original (2000) note: Cj0483, uxaA' possible altronate hydrolase C-terminus, len: 390 aa; similar to C-terminus of e.g. UXAA_ECOLI altronate hydrolase (495 aa), fasta scores; opt: 832 z-score: 1121.7 E(): 0, 36.1% identity in 391 aa overlap. Simlarity continues from upstream ORF Cj0482. No Hp match. Note that B. subtilis paralog of uxaA, YCBI_BACSU, also has a separate N-terminus (YCBH_BACSU); Updated (2006) note: Pfam domain PF04295 D-galactarate dehydratase / Altronate hydrolase identified within CDS. Further support given to product function. Characterisation work carried out within [...] ","uniprotId":"Q0PB30_CAMJE"},"10641330":{"id":10641330,"externalId":"192222.Cj0484","name":"Cj0484","hasFamily":true,"annotation":"Putative MFS (Major Facilitator Superfamily) transport protein; Original (2000) note: Cj0484, probable transmembrane transport protein, len: 410 aa; similar to members of the phalate transporter family e.g. TR:Q44470 (EMBL:U25634) Agrobacterium vitis putative tartrate transporter (433 aa), fasta scores; opt: 1004 z-score: 1322.2 E(): 0, 40.4% identity in 408 aa overlap, and PHT1_PSEPU phthalate transporter (451 aa), fasta scores; opt: 663 z-score: 835.4 E(): 0, 29.8% identity in 413 aa overlap. No Hp ortholog; Updated (2006) note: Pfam domain PF07690 Major Facilitator Superfamily (MFS) [...] ","uniprotId":"Q0PB29_CAMJE"},"10641331":{"id":10641331,"externalId":"192222.Cj0485","name":"Cj0485","hasFamily":true,"annotation":"Putative oxidoreductase; Original (2000) note: Cj0485, probable oxidoreductase, len: 262 aa; similar to many e.g. FABG_AQUAE 3-oxoacyl-[acyl-carrier protein] reductase (248 aa), fasta scores; opt: 384 z-score: 465.5 E(): 1.2e-18,34.3% identity in 254 aa overlap, and BA71_EUBSP 7-alpha-hydroxysteroid dehydrogenase (249 aa), fasta scores; opt: 349 z-score: 421.2 E(): 3.6e-16, 32.3% identity in 254 aa overlap. No Hp ortholog. Also similar to C. jejuni fabG, Cj0435 (31.7% identity in 252 aa overlap). Contains Pfam matches to entry PF00106 adh_short, Alcohol/other dehydrogenases, short chai [...] ","uniprotId":"Q0PB28_CAMJE"},"10641332":{"id":10641332,"externalId":"192222.Cj0486","name":"Cj0486","hasFamily":true,"annotation":"Putative sugar transporter; Original (2000) note: Cj0486, probable sugar transporter, len: 418 aa; simlar to members of the FHS family e.g. FUCP_ECOLI L-fucose permease (438 aa), fasta scores; opt: 848 z-score: 1022.1 E(): 0, 37.2% identity in 390 aa overlap, and GLUP_BRUAB glucose/galactose transporter (412 aa), fasta scores; opt: 456 z-score: 830.5 E(): 0, 33.4% identity in 410 aa overlap. 32.6% identity to HP1174; Updated (2006) note: Pfam domain PF07690 Major Facilitator Superfamily identified within CDS. Prosite PS50850 MFS, Major facilitator superfamily also identified along with [...] ","uniprotId":"Q0PB27_CAMJE"},"10641333":{"id":10641333,"externalId":"192222.Cj0487","name":"Cj0487","hasFamily":true,"annotation":"Putative amidohydrolase; Original (2000) note: Cj0487, unknown, len: 256 aa; some similarity to Y4MH_RHISN hypothetical protein Y4MH (297 aa), fasta scores; opt: 98 z-score: 200.8 E(): 0.00069, 22.7% identity in 295 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF04909 Amidohydrolase identified within CDS. Product modified to more specific family member due to motif match. No specific characterisation has been carried out yet. Thus,putative kept within product function. Functional classification - Misc","uniprotId":"Q0PB26_CAMJE"},"10641334":{"id":10641334,"externalId":"192222.Cj0488","name":"Cj0488","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0488, unknown, len: ; similar to a hypothetical protein from B. subtilis TR:O05263 (EMBL:Z93938) YULD(104 aa), fasta scores; opt: 183 z-score: 266.1 E(): 1.6e-07, 33.6% identity in 107 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF05336 Protein of unknown function (DUF718) identified within CDS. Conserved added to product function. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PB25_CAMJE"},"10641335":{"id":10641335,"externalId":"192222.Cj0489","name":"ald'","hasFamily":true,"annotation":"Original (2000) note: Cj0489, ald', aldehyde dehydrogenase N-terminus, len: 73 aa; highly similar to ALDA_ECOLI aldehyde dehydrogenase A (EC 1.2.1.22) (478 aa), fasta scores; opt: 229 z-score: 319.3 E(): 1.7e-10,51.5% identity in 68 aa overlap. Similarity continues into downstream ORF Cj0490. No error in the sequence can be found. No Hp match; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Gene names for Cj0489 and Cj0490 are identical with a ' denoting two separate terminals for the same CDS. May well be a pseudogene. Putative kept within pr [...] ","uniprotId":"Q0PB24_CAMJE"},"10641336":{"id":10641336,"externalId":"192222.Cj0490","name":"Cj0490","hasFamily":true,"annotation":"Belongs to the aldehyde dehydrogenase family. ","uniprotId":"Q0PB23_CAMJE"},"10641337":{"id":10641337,"externalId":"192222.Cj0491","name":"rpsL","hasFamily":true,"annotation":"With S4 and S5 plays an important role in translational accuracy","uniprotId":"RS12_CAMJE"},"10641338":{"id":10641338,"externalId":"192222.Cj0492","name":"rpsG","hasFamily":true,"annotation":"One of the primary rRNA binding proteins, it binds directly to 16S rRNA where it nucleates assembly of the head domain of the 30S subunit. Is located at the subunit interface close to the decoding center, probably blocks exit of the E-site tRNA","uniprotId":"RS7_CAMJE"},"10641339":{"id":10641339,"externalId":"192222.Cj0493","name":"fusA","hasFamily":true,"annotation":"Catalyzes the GTP-dependent ribosomal translocation step during translation elongation. During this step, the ribosome changes from the pre-translocational (PRE) to the post-translocational (POST) state as the newly formed A-site-bound peptidyl-tRNA and P-site-bound deacylated tRNA move to the P and E sites, respectively. Catalyzes the coordinated movement of the two tRNA molecules, the mRNA and conformational changes in the ribosome","uniprotId":"EFG_CAMJE"},"10641340":{"id":10641340,"externalId":"192222.Cj0494","name":"Cj0494","hasFamily":false,"annotation":"Putative exporting protein; Original (2000) note: Cj0494, small hydrophobic protein, len: 49 aa; no Hp match; Updated (2006) note: Product function modified to more specific family member based on SignalP 2.0 HMM results. Putative kept within product function. Functional classification - Transport/binding proteins - Other","uniprotId":"Q0PB19_CAMJE"},"10641341":{"id":10641341,"externalId":"192222.Cj0495","name":"Cj0495","hasFamily":true,"annotation":"Belongs to the methyltransferase superfamily. ","uniprotId":"Q0PB18_CAMJE"},"10641342":{"id":10641342,"externalId":"192222.Cj0496","name":"Cj0496","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0496, unknown, len: 124 aa; similar to hypothetical proteins e.g. TR:O59308 (EMBL:AB009522) Pyrococcus horikoshii PHAV002 (141 aa),fasta scores; opt: 154 z-score: 270.6 E(): 8.8e-08,30.1% identity in 113 aa overlap. 47.5% identity to HP0274; Updated (2006) note: Pfam domain PF03692 Uncharacterised protein family (UPF0153) identified within CDS. Conserved added to product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PB17_CAMJE"},"10641343":{"id":10641343,"externalId":"192222.Cj0497","name":"Cj0497","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0497, probable lipoprotein,len: 425 aa; some similarity to hypothetical prteins e.g. TR:O67021 (EMBL:AE000710) Aquifex aeolicus AQ_854 (545 aa), fasta scores; opt: 266 z-score: 326.7 E(): 6.7e-11,29.1% identity in 412 aa overlap. 26.7% identity to HP0275 (misannotated as ATP-dependent nuclease (addB)). Contains N-terminal signal sequence and PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PB16_CAMJE"},"10641344":{"id":10641344,"externalId":"192222.Cj0498","name":"trpC","hasFamily":true,"annotation":"Belongs to the TrpC family","uniprotId":"TRPC_CAMJE"},"10641345":{"id":10641345,"externalId":"192222.Cj0499","name":"Cj0499","hasFamily":true,"annotation":"Putative histidine triad (HIT) family protein; Original (2000) note: Cj0499, unknown, len: 161 aa; similar to many members of the HIT family e.g. HIT_BACSU HIT protein (145 aa), fasta scores; opt: 176 z-score: 274.7 E(): 5.2e-08, 26.1% identity in 111 aa overlap. 42.6% identity to HP0741. Contains Pfam match to entry PF01230 HIT, HIT family, score 19.90, E-value 3.6e-06; Updated (2006) note: Product modified to more specific family member due to motif match. No specific characterisation has been carried out yet. Putative kept within product function. Functional classification - Misc","uniprotId":"Q0PB14_CAMJE"},"10641346":{"id":10641346,"externalId":"192222.Cj0500","name":"Cj0500","hasFamily":true,"annotation":"Belongs to the SelU family. ","uniprotId":"Q0PB13_CAMJE"},"10641347":{"id":10641347,"externalId":"192222.Cj0503c","name":"hemH","hasFamily":true,"annotation":"Catalyzes the ferrous insertion into protoporphyrin IX. ","uniprotId":"HEMH_CAMJE"},"10641348":{"id":10641348,"externalId":"192222.Cj0504c","name":"Cj0504c","hasFamily":true,"annotation":"Putative oxidoreductase; Original (2000) note: Cj0504c, unknown, len: 288 aa; similar to hypothetical proteins e.g. YISS_BACSU 342 aa), fasta scores; opt: 224 z-score: 359.9 E(): 9.4e-13,28.1% identity in 192 aa overlap. 26.4% identity to HP0679; Updated (2006) note: Pfam domain PF01408 Oxidoreductase family, NAD-binding Rossmann fold identified within CDS. Product function modified due to motif match. No specific characterisation has been carried out yet. Putative kept within product function. Functional classification - Misc","uniprotId":"Q0PB11_CAMJE"},"10641349":{"id":10641349,"externalId":"192222.Cj0505c","name":"Cj0505c","hasFamily":true,"annotation":"Belongs to the DegT/DnrJ/EryC1 family. ","uniprotId":"Q0PB10_CAMJE"},"10641350":{"id":10641350,"externalId":"192222.Cj0506","name":"alaS","hasFamily":true,"annotation":"Catalyzes the attachment of alanine to tRNA(Ala) in a two- step reaction: alanine is first activated by ATP to form Ala-AMP and then transferred to the acceptor end of tRNA(Ala). Also edits incorrectly charged Ser-tRNA(Ala) and Gly-tRNA(Ala) via its editing domain","uniprotId":"SYA_CAMJE"},"10641351":{"id":10641351,"externalId":"192222.Cj0507","name":"maf","hasFamily":true,"annotation":"Nucleoside triphosphate pyrophosphatase. May have a dual role in cell division arrest and in preventing the incorporation of modified nucleotides into cellular nucleic acids","uniprotId":"NTPP_CAMJE"},"10641352":{"id":10641352,"externalId":"192222.Cj0508","name":"pbpA","hasFamily":true,"annotation":"Penicillin-binding protein; Original (2000) note: Cj0508, pbpA, probable penicillin-binding protein, len: 643 aa; similar to many e.g. PBPB_ECOLI penicillin-binding protein 1B (PBP-1B) (844 aa), fasta scores; opt: 774 z-score: 915.8 E(): 0,32.5% identity in 616 aa overlap. 48.0% identity to HP0597. Also similar to Cj0525c (22.9% identity in 407 aa overlap). Contains Pfam matches to entry PF00912 Transglycosyl, Transglycosylase, score 299.40, E-value 4.5e-86 and Pfam match to entry PF00905 Transpeptidase,Penicillin binding protein transpeptidase domain, score 103.80, E-value 3.3e-27; Up [...] ","uniprotId":"Q0PB07_CAMJE"},"10641353":{"id":10641353,"externalId":"192222.Cj0509c","name":"clpB","hasFamily":true,"annotation":"Part of a stress-induced multi-chaperone system, it is involved in the recovery of the cell from heat-induced damage, in cooperation with DnaK, DnaJ and GrpE. Acts before DnaK, in the processing of protein aggregates. Protein binding stimulates the ATPase activity; ATP hydrolysis unfolds the denatured protein aggregates, which probably helps expose new hydrophobic binding sites on the surface of ClpB-bound aggregates, contributing to the solubilization and refolding of denatured protein aggregates by DnaK (By similarity). ","uniprotId":"CLPB_CAMJE"},"10641354":{"id":10641354,"externalId":"192222.Cj0510c","name":"Cj0510c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0510c, unknown, len: 96 aa; similar to hypothetical proteins e.g. TR:O32222 (EMBL:Z99121) B. subtilis YVGZ protein (101 aa), fasta scores; opt: 212 z-score: 338.9 E(): 1.4e-11, 40.0% identity in 95 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02583 Uncharacterised BCR, COG1937 identified within CDS. Conserved added to product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PB05_CAMJE"},"10641355":{"id":10641355,"externalId":"192222.Cj0511","name":"Cj0511","hasFamily":true,"annotation":"Belongs to the peptidase S41A family. ","uniprotId":"Q0PB04_CAMJE"},"10641356":{"id":10641356,"externalId":"192222.Cj0512","name":"purC","hasFamily":true,"annotation":"Belongs to the SAICAR synthetase family. ","uniprotId":"PUR7_CAMJE"},"10641357":{"id":10641357,"externalId":"192222.Cj0513","name":"purS","hasFamily":true,"annotation":"Part of the phosphoribosylformylglycinamidine synthase complex involved in the purines biosynthetic pathway. Catalyzes the ATP-dependent conversion of formylglycinamide ribonucleotide (FGAR) and glutamine to yield formylglycinamidine ribonucleotide (FGAM) and glutamate. The FGAM synthase complex is composed of three subunits. PurQ produces an ammonia molecule by converting glutamine to glutamate. PurL transfers the ammonia molecule to FGAR to form FGAM in an ATP- dependent manner. PurS interacts with PurQ and PurL and is thought to assist in the transfer of the ammonia molecule from Pu [...] ","uniprotId":"Q0PB02_CAMJE"},"10641358":{"id":10641358,"externalId":"192222.Cj0514","name":"purQ","hasFamily":true,"annotation":"Part of the phosphoribosylformylglycinamidine synthase complex involved in the purines biosynthetic pathway. Catalyzes the ATP-dependent conversion of formylglycinamide ribonucleotide (FGAR) and glutamine to yield formylglycinamidine ribonucleotide (FGAM) and glutamate. The FGAM synthase complex is composed of three subunits. PurQ produces an ammonia molecule by converting glutamine to glutamate. PurL transfers the ammonia molecule to FGAR to form FGAM in an ATP- dependent manner. PurS interacts with PurQ and PurL and is thought to assist in the transfer of the ammonia molecule from Pu [...] ","uniprotId":"PURQ_CAMJE"},"10641359":{"id":10641359,"externalId":"192222.Cj0515","name":"Cj0515","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0515, possible periplasmic protein, len: 398 aa; 27.2% identity to HP1349; Updated (2006) note: One probable transmembrane helices predicted by TMHMM2.0 within CDS. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PB00_CAMJE"},"10641360":{"id":10641360,"externalId":"192222.Cj0516","name":"plsC","hasFamily":true,"annotation":"Belongs to the 1-acyl-sn-glycerol-3-phosphate acyltransferase family","uniprotId":"Q0PAZ9_CAMJE"},"10641361":{"id":10641361,"externalId":"192222.Cj0517","name":"crcB","hasFamily":true,"annotation":"Important for reducing fluoride concentration in the cell, thus reducing its toxicity","uniprotId":"CRCB_CAMJE"},"10641362":{"id":10641362,"externalId":"192222.Cj0518","name":"htpG","hasFamily":true,"annotation":"Molecular chaperone. Has ATPase activity","uniprotId":"HTPG_CAMJE"},"10641363":{"id":10641363,"externalId":"192222.Cj0519","name":"Cj0519","hasFamily":true,"annotation":"Putative rhodanese-like domain protein; Original (2000) note: Cj0519, unknown, len: 105 aa; similar to hypothetical proteins e.g. TR:E1342888 (EMBL:AJ235272) Rickettsia prowazekii RP600 (123 aa),fasta scores; opt: 151 z-score: 214.9 E(): 0.00011, 41.6% identity in 89 aa overlap. 34.3% identity to HP1223; Updated (2006) note: Prosite domain PS50206 RHODANESE_3, Rhodanese domain profile also identified within CDS. Rhodanese, a sulphurtransferase involved in cyanide detoxification shares evolutionary relationship with a large family of proteins. This domain is found as a single copy in ot [...] ","uniprotId":"Q0PAZ6_CAMJE"},"10641364":{"id":10641364,"externalId":"192222.Cj0520","name":"Cj0520","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0520, possible membrane protein, len: 144 aa; no Hp match. Contains one possible transmembrane domain at N-terminus. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PAZ5_CAMJE"},"10641365":{"id":10641365,"externalId":"192222.Cj0522","name":"Cj0522","hasFamily":true,"annotation":"Putative Na+/Pi cotransporter protein; Original (2000) note: Cj0522, possible membrane protein, len: 167 aa; no Hp match. May be fragment of pseudogene; Updated (2006) note: Pfam domain PF02690 Na+/Pi-cotransporter identified within CDS. Three probable transmembrane helices predicted by TMHMM2.0. Product modified to more specific family member due to motif match. No specific characterisation has been carried out yet. Putative kept within product function. Functional classification - Transport/binding proteins - Cations","uniprotId":"Q0PAZ4_CAMJE"},"10641366":{"id":10641366,"externalId":"192222.Cj0523","name":"Cj0523","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0523, possible membrane protein, len: 91 aa; no Hp match. May be fragment of pseudogene; Updated (2006) note: Three probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PAZ3_CAMJE"},"10641367":{"id":10641367,"externalId":"192222.Cj0524","name":"Cj0524","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0524, unknown, len: 233 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins and cotransporter family proteins from other bacteria. Functional classification - Unknown","uniprotId":"Q0PAZ2_CAMJE"},"10641368":{"id":10641368,"externalId":"192222.Cj0525c","name":"pbpB","hasFamily":true,"annotation":"Putative penicillin-binding protein; Original (2000) note: Cj0525c, pbpB, probable penicillin-binding protein, len: 602 aa; similar to many e.g. PBP2_NEIME penicillin-binding protein 2 (581 aa),fasta scores; opt: 184 z-score: 511.3 E(): 3.5e-21. 28.9% identity in 443 aa overlap. 35.8% identity to HP1556 (misannotated as cell division protein (ftsI)). Also similar to Cj0508 (22.6% identity in 407 aa overlap). Contains PS00017 ATP/GTP-binding site motif A (P-loop),Pfam match to entry PF00905 Transpeptidase, Penicillin binding protein transpeptidase domain, score 259.30,E-value 5.2e-74 an [...] ","uniprotId":"Q0PAZ1_CAMJE"},"10641369":{"id":10641369,"externalId":"192222.Cj0526c","name":"fliE","hasFamily":true,"annotation":"Original (2000) note: Cj0526c, fliE, probable flagellar hook-basal body complex protein, len: 98 aa; similar to e.g. FLIE_BACSU flagellar hook-basal body complex protein (106 aa), fasta scores; opt: 182 z-score: 262.0 E(): 2.7e-07 34.0% identity in 97 aa overlap. 60.9% identity to HP1557; Updated (2006) note: Pfam domain PF02049 Flagellar hook-basal body complex protein FliE identified within CDS. These genes form proteins that are thought be involved in bacterial flagellar assembly. FliE is said to form part of the hook-basal body complex of bacterial flagellar. The FlgB, FlgC and Flg [...] ","uniprotId":"FLIE_CAMJE"},"10641370":{"id":10641370,"externalId":"192222.Cj0527c","name":"flgC","hasFamily":true,"annotation":"Belongs to the flagella basal body rod proteins family. ","uniprotId":"Q0PAY9_CAMJE"},"10641371":{"id":10641371,"externalId":"192222.Cj0528c","name":"flgB","hasFamily":true,"annotation":"Structural component of flagellum, the bacterial motility apparatus. Part of the rod structure of flagellar basal body. ","uniprotId":"Q0PAY8_CAMJE"},"10641372":{"id":10641372,"externalId":"192222.Cj0529c","name":"mltG","hasFamily":true,"annotation":"Functions as a peptidoglycan terminase that cleaves nascent peptidoglycan strands endolytically to terminate their elongation. ","uniprotId":"Q0PAY7_CAMJE"},"10641373":{"id":10641373,"externalId":"192222.Cj0530","name":"Cj0530","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0530, probable periplasmic protein, len: 856 aa; 22.7% identity to HP0586. Contains N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PAY6_CAMJE"},"10641374":{"id":10641374,"externalId":"192222.Cj0531","name":"icd","hasFamily":true,"annotation":"Belongs to the monomeric-type IDH family. ","uniprotId":"Q0PAY5_CAMJE"},"10641375":{"id":10641375,"externalId":"192222.Cj0532","name":"mdh","hasFamily":true,"annotation":"Catalyzes the reversible oxidation of malate to oxaloacetate. ","uniprotId":"MDH_CAMJE"},"10641376":{"id":10641376,"externalId":"192222.Cj0533","name":"sucC","hasFamily":true,"annotation":"Succinyl-CoA synthetase functions in the citric acid cycle (TCA), coupling the hydrolysis of succinyl-CoA to the synthesis of either ATP or GTP and thus represents the only step of substrate-level phosphorylation in the TCA. The beta subunit provides nucleotide specificity of the enzyme and binds the substrate succinate, while the binding sites for coenzyme A and phosphate are found in the alpha subunit","uniprotId":"SUCC_CAMJE"},"10641377":{"id":10641377,"externalId":"192222.Cj0534","name":"sucD","hasFamily":true,"annotation":"Succinyl-CoA synthetase functions in the citric acid cycle (TCA), coupling the hydrolysis of succinyl-CoA to the synthesis of either ATP or GTP and thus represents the only step of substrate-level phosphorylation in the TCA. The alpha subunit of the enzyme binds the substrates coenzyme A and phosphate, while succinate binding and nucleotide specificity is provided by the beta subunit. ","uniprotId":"Q0PAY2_CAMJE"},"10641378":{"id":10641378,"externalId":"192222.Cj0535","name":"oorD","hasFamily":true,"annotation":"Original (2000) note: Cj0535, oorD, probable OORD subunit of 2-oxoglutarate:acceptor oxidoreductase, len: 103 aa; similar to TR:O68227 (EMBL:AF021094) H. pylori OORD subunit of 2-oxoglutarate:acceptor oxidoreductase (113 aa), fasta scores; opt: 436 z-score: 768.9 E(): 0,59.8% identity in 97 aa overlap. 59.8% identity to HP0588. Contains 2x PS00198 4Fe-4S ferredoxins, iron-sulfur binding region signature and Pfam match to entry PF00037 fer4, 4Fe-4S ferredoxins and related iron-sulfur cluster binding domains., score 17.40, E-value 0.00091; Updated (2006) note: Characterised within Helico [...] ","uniprotId":"Q0PAY1_CAMJE"},"10641379":{"id":10641379,"externalId":"192222.Cj0536","name":"oorA","hasFamily":true,"annotation":"Original (2000) note: Cj0536, oorA, probable OORA subunit of 2-oxoglutarate:acceptor oxidoreductase, len: 374 aa; similar to TR:O68228 (EMBL:AF021094) H. pylori OORA subunit of 2-oxoglutarate:acceptor oxidoreductase (371 aa), fasta scores; opt: 1447 z-score: 2256.7 E(): 0,62.7% identity in 351 aa overlap. 63.2% identity to HP0589; Updated (2006) note: Pfam domain PF01855 Pyruvate flavodoxin/ferredoxin oxidoreductase identified within CDS. Further support given to product function. Characterised within Helicobacter pylori with acceptable identity score. Putative not added to product fun [...] ","uniprotId":"Q0PAY0_CAMJE"},"10641380":{"id":10641380,"externalId":"192222.Cj0537","name":"oorB","hasFamily":true,"annotation":"Original (2000) note: Cj0537, oorB, probable OORB subunit of 2-oxoglutarate:acceptor oxidoreductase, len: 281 aa; similar to TR:O68229 (EMBL:AF021094) H. pylori OORB subunit of 2-oxoglutarate:acceptor oxidoreductase (274 aa), fasta scores; opt: 1356 z-score: 2118.6 E(): 0,69.7% identity in 274 aa overlap. 72.9% identity to HP0590; Updated (2006) note: Pfam domain PF02775 Thiamine pyrophosphate enzyme, C-terminal identified within CDS. Further support given to product function. Characterised within Helicobacter pylori with acceptable identity score. Putative not added to product functio [...] ","uniprotId":"Q0PAX9_CAMJE"},"10641381":{"id":10641381,"externalId":"192222.Cj0538","name":"oorC","hasFamily":true,"annotation":"Original (2000) note: Cj0538, oorC, probable OORC subunit of 2-oxoglutarate:acceptor oxidoreductase, len: 185 aa; similar to TR:O68230 (EMBL:AF021094) H. pylori OORC subunit of 2-oxoglutarate:acceptor oxidoreductase (184 aa), fasta scores; opt: 714 z-score: 1191.4 E(): 0,59.7% identity in 181 aa overlap. 60.8% identity to HP0591; Updated (2006) note: Pfam domain PF01558 Pyruvate ferredoxin/flavodoxin oxidoreductase identified within CDS. Further support given to product function. Characterised within Helicobacter pylori with acceptable identity score. Putative not added to product func [...] ","uniprotId":"Q0PAX8_CAMJE"},"10641382":{"id":10641382,"externalId":"192222.Cj0539","name":"Cj0539","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0539, unknown, len: 87 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PAX7_CAMJE"},"10641383":{"id":10641383,"externalId":"192222.Cj0540","name":"Cj0540","hasFamily":true,"annotation":"Putative exporting protein; Original (2000) note: Cj0540, unknown, len: 178 aa; 30.6% identity in 72 aa overlap (N-terminus) to HP0241; Updated (2006) note: Product function modified to more specific family member based on SignalP 2.0 HMM results. Putative kept within product function. Functional classification - Protein and peptide secretion","uniprotId":"Q0PAX6_CAMJE"},"10641384":{"id":10641384,"externalId":"192222.Cj0541","name":"Cj0541","hasFamily":true,"annotation":"Belongs to the FPP/GGPP synthase family. ","uniprotId":"Q0PAX5_CAMJE"},"10641385":{"id":10641385,"externalId":"192222.Cj0542","name":"hemA","hasFamily":true,"annotation":"Catalyzes the NADPH-dependent reduction of glutamyl-tRNA(Glu) to glutamate 1-semialdehyde (GSA)","uniprotId":"HEM1_CAMJE"},"10641386":{"id":10641386,"externalId":"192222.Cj0543","name":"proS","hasFamily":true,"annotation":"Catalyzes the attachment of proline to tRNA(Pro) in a two- step reaction: proline is first activated by ATP to form Pro-AMP and then transferred to the acceptor end of tRNA(Pro). As ProRS can inadvertently accommodate and process non-cognate amino acids such as alanine and cysteine, to avoid such errors it has two additional distinct editing activities against alanine. One activity is designated as 'pretransfer' editing and involves the tRNA(Pro)-independent hydrolysis of activated Ala-AMP. The other activity is designated 'posttransfer' editing and involves deacylation of mischarged A [...] ","uniprotId":"SYP_CAMJE"},"10641387":{"id":10641387,"externalId":"192222.Cj0544","name":"Cj0544","hasFamily":false,"annotation":"Original (2000) note: Cj0544, probable integral membrane protein, len: 129 aa; no Hp match; Updated (2006) note: Three probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PAX2_CAMJE"},"10641388":{"id":10641388,"externalId":"192222.Cj0545","name":"hemC","hasFamily":true,"annotation":"Tetrapolymerization of the monopyrrole PBG into the hydroxymethylbilane pre-uroporphyrinogen in several discrete steps. ","uniprotId":"HEM3_CAMJE"},"10641389":{"id":10641389,"externalId":"192222.Cj0546","name":"ubiD","hasFamily":true,"annotation":"Belongs to the UbiD family. ","uniprotId":"Q0PAX0_CAMJE"},"10641390":{"id":10641390,"externalId":"192222.Cj0547","name":"flaG","hasFamily":true,"annotation":"Flagellar protein; Original (2000) note: Cj0547, flaG, possible flagellar protein, len: 121 aa; some similarity to e.g. TR:O33420 (EMBL:L81176) Pseudomonas aeruginosa FLAG (128 aa), fasta scores; opt: 169 z-score: 197.2 E(): 0.0011,31.4% identity in 102 aa overlap. 43.5% identity to HP0751; Updated (2006) note: Pfam domain PF03646 FlaG protein was identified within CDS. Further support given to product function. The FlgB, FlgC and FlgG proteins are components of the proximal and distal rods; FliF forms the M-ring that anchors the rod assembly to the membrane. Characterised within Pseud [...] ","uniprotId":"Q0PAW9_CAMJE"},"10641391":{"id":10641391,"externalId":"192222.Cj0548","name":"fliD","hasFamily":true,"annotation":"Required for the morphogenesis and for the elongation of the flagellar filament by facilitating polymerization of the flagellin monomers at the tip of growing filament. Forms a capping structure, which prevents flagellin subunits (transported through the central channel of the flagellum) from leaking out without polymerization at the distal end (By similarity)","uniprotId":"FLID_CAMJE"},"10641392":{"id":10641392,"externalId":"192222.Cj0549","name":"fliS","hasFamily":true,"annotation":"Flagellar protein FliS; Original (2000) note: Cj0549, fliS, probable flagellar protein, len: 128 aa; similar to e.g. FLIS_SALTY flagellar protein FLIS (135 aa), fasta scores; opt: 190 z-score: 343.0 E(): 8.2e-12, 34.2% identity in 117 aa overlap. 60.5% identity to HP0753; Updated (2006) note: Pfam domain PF02561 Flagellar protein FliS was identified within CDS. Further support given to product function. Characterised within Bacillus subtilis and others with marginal identity scores. Putative not added to product function. Functional classification - Surface structures; PMID:8195064, PM [...] ","uniprotId":"Q0PAW7_CAMJE"},"10641393":{"id":10641393,"externalId":"192222.Cj0550","name":"Cj0550","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0550, unknown, len: 91 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PAW6_CAMJE"},"10641394":{"id":10641394,"externalId":"192222.Cj0551","name":"efp","hasFamily":true,"annotation":"Involved in peptide bond synthesis. Stimulates efficient translation and peptide-bond synthesis on native or reconstituted 70S ribosomes in vitro. Probably functions indirectly by altering the affinity of the ribosome for aminoacyl-tRNA, thus increasing their reactivity as acceptors for peptidyl transferase","uniprotId":"EFP_CAMJE"},"10641395":{"id":10641395,"externalId":"192222.Cj0552","name":"Cj0552","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0552, unknown, len: 230 aa; no Hp match. Hydrophobic; Updated (2006) note: Pfam domain PF06149 Protein of unknown function (DUF969) identified within CDS. This is a family of uncharacterised bacterial membrane proteins. Also identified were four probable transmembrane helices predicted by TMHMM2.0. Product function has been changed due to these results. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PAW4_CAMJE"},"10641396":{"id":10641396,"externalId":"192222.Cj0553","name":"Cj0553","hasFamily":true,"annotation":"Original (2000) note: Cj0553, probable integral membrane protein, len: 317 aa; no Hp match; Updated (2006) note: Pfam domain PF06166 Protein of unknown function (DUF979) identified within CDS. This family consists of several putative bacterial membrane proteins. The function of this family is unclear. Also identified were nine probable transmembrane helices predicted by TMHMM2.0. Further support for product function. Functional classification - Membranes,lipoproteins and porins","uniprotId":"Q0PAW3_CAMJE"},"10641397":{"id":10641397,"externalId":"192222.Cj0554","name":"Cj0554","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0554, unknown, len: 334 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PAW2_CAMJE"},"10641398":{"id":10641398,"externalId":"192222.Cj0555","name":"Cj0555","hasFamily":true,"annotation":"Putative dicarboxylate carrier protein MatC; Original (2000) note: Cj0555, probable integral membrane protein, len: 430 aa; similar in N-terminus to an ORF within the Streptococcus pneumoniae cps3E and rpt pseudogene region TR:P72556 (EMBL:U66846) orf5 (197 aa),fasta scores; opt: 493 z-score: 726.3 E(): 3.7e-33, 47.4% identity in 152 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF07158 Dicarboxylate carrier protein MatC N-terminus identified within CDS. This family represents the N-terminal region of the bacterial dicarboxylate carrier protein MatC. MatC protein is an inte [...] ","uniprotId":"Q0PAW1_CAMJE"},"10641399":{"id":10641399,"externalId":"192222.Cj0556","name":"Cj0556","hasFamily":true,"annotation":"Putative amidohydrolase family protein; Original (2000) note: Cj0556, unknown, len: 265 aa; no Hp match; Updated (2006) note: Pfam domain PF04909 Amidohydrolase identified within CDS. Product modified to more specific family member due to motif match. No specific characterisation has been carried out yet. Putative kept within product function. Functional classification - Misc","uniprotId":"Q0PAW0_CAMJE"},"10641400":{"id":10641400,"externalId":"192222.Cj0557c","name":"Cj0557c","hasFamily":true,"annotation":"Original (2000) note: Cj0557c, probable integral membrane protein, len: 361 aa; similar in C terminus to hypothetical proteins from B. subtilis and E. coli: TR:O32225 (EMBL:O32225) YVAC (631 aa), fasta scores; opt: 271 z-score: 562.0 E(): 5.2e-24, 29.9% identity in 187 aa overlap, and YHFK_ECOLI (696 aa), fasta scores; opt: 182 z-score: 212.9 E(): 0.00015, 27.8% identity in 133 aa overlap. No Hp match; Updated (2006) note: Nine probable transmembrane helices predicted by TMHMM2.0. Further support added to product function. Functional classification - Membranes,lipoproteins and porins","uniprotId":"Q0PAV9_CAMJE"},"10641401":{"id":10641401,"externalId":"192222.Cj0558c","name":"proA","hasFamily":true,"annotation":"Catalyzes the NADPH-dependent reduction of L-glutamate 5- phosphate into L-glutamate 5-semialdehyde and phosphate. The product spontaneously undergoes cyclization to form 1-pyrroline-5-carboxylate. ","uniprotId":"PROA_CAMJE"},"10641402":{"id":10641402,"externalId":"192222.Cj0559","name":"Cj0559","hasFamily":true,"annotation":"Original (2000) note: Cj0559, probable oxidoreductase, len: 311 aa; similar to members of the pyridine nucleotide-disulfide oxidoreductases class-II family e.g. R34K_CLOPA 34.2 KD protein in rubredoxin operon (308 aa), fasta scores; opt: 217 z-score: 335.8 E(): 2.1e-11, 23.2% identity in 314 aa overlap, and TRXB_BACSU thioredoxin reductase (315 aa), fasta scores; opt: 192 z-score: 279.9 E(): 2.7e-08, 25.9% identity in 313 aa overlap. 46.3% identity to HP1164. Contains possible N-terminal signal sequence and PS00013 Prokaryotic membrane lipoprotein lipid attachment site; Updated (2006)  [...] ","uniprotId":"Q0PAV7_CAMJE"},"10641403":{"id":10641403,"externalId":"192222.Cj0560","name":"Cj0560","hasFamily":true,"annotation":"Putative MATE family transport protein; Original (2000) note: Cj0560, probable integral membrane protein, len: 442 aa; similar to many hypothetical proteins e.g. Y709_METJA MJ0709 (450 aa),fasta scores; opt: 402 z-score: 324.8 E(): 8.5e-11, 21.0% identity in 429 aa overlap. 26.3% identity to HP1184; Updated (2006) note: Pfam domain PF01554 MatE identified within CDS. Prosite domain PS00589 PTS_HPR_SER,Phosphotransferase system, HPr serine phosphorylation site also identified. Twelve probable transmembrane helices predicted by TMHMM2.0. Characterised members of the Multi Antimicrobial E [...] ","uniprotId":"Q0PAV6_CAMJE"},"10641404":{"id":10641404,"externalId":"192222.Cj0561c","name":"Cj0561c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0561c, possible periplasmic protein, len: 309 aa; no Hp match; Updated (2006) note: possible changed to putative. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PAV5_CAMJE"},"10641405":{"id":10641405,"externalId":"192222.Cj0562","name":"dnaB","hasFamily":true,"annotation":"Participates in initiation and elongation during chromosome replication; it exhibits DNA-dependent ATPase activity and contains distinct active sites for ATP binding, DNA binding, and interaction with DnaC protein, primase, and other prepriming proteins. ","uniprotId":"Q0PAV4_CAMJE"},"10641406":{"id":10641406,"externalId":"192222.Cj0563","name":"Cj0563","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0563, unknown, len: 200 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PAV3_CAMJE"},"10641407":{"id":10641407,"externalId":"192222.Cj0564","name":"Cj0564","hasFamily":false,"annotation":"Original (2000) note: Cj0564, probable integral membrane protein, len: 68 aa; no Hp match; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PAV2_CAMJE"},"10641408":{"id":10641408,"externalId":"192222.Cj0566","name":"Cj0566","hasFamily":false,"annotation":"Uncharacterized protein; HMMPfam hit to PF03667, Uncharacterised protein family (UPF0174), score 3.1e-51","uniprotId":"Q0PAV1_CAMJE"},"10641409":{"id":10641409,"externalId":"192222.Cj0567","name":"Cj0567","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0567, unknown, len: 37 aa; no Hp match; Updated (2006) note: Paper identified linking product function to iron transport or metabolism. Functional classification - Unknown; PMID:15632442","uniprotId":"Q0PAV0_CAMJE"},"10641410":{"id":10641410,"externalId":"192222.Cj0568","name":"Cj0568","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0568, unknown, len: 93 aa; no Hp match; Updated (2006) note: One probable transmembrane helices predicted by TMHMM2.0. Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PAU9_CAMJE"},"10641411":{"id":10641411,"externalId":"192222.Cj0569","name":"Cj0569","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0569, unknown, len: 289 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PAU8_CAMJE"},"10641412":{"id":10641412,"externalId":"192222.Cj0570","name":"Cj0570","hasFamily":true,"annotation":"Original (2000) note: Cj0570, probable ATP/GTP binding protein, len: 355 aa; 85.1% identity to HP0729. Contains PS00017 ATP/GTP-binding site motif A (P-loop). Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PAU7_CAMJE"},"10641413":{"id":10641413,"externalId":"192222.Cj0571","name":"Cj0571","hasFamily":true,"annotation":"Original (2000) note: Cj0571, possible transcriptional regulator; len: 290 aa; similar in C-terminus to two hypothetical deoR family transcriptional regulators from E. coli; YFJR_ECOLI (233 aa), fasta scores; opt: 279 z-score: 374.9 E(): 1.4e-13, 27.7% identity in 202 aa overlap, and YAFY_ECOLI (285 aa), fasta scores; opt: 167 z-score: 203.3 E(): 0.0005, 40.4% identity in 57 aa overlap. No Hp match, Contains probable helix-turn-helix motif at aa 25-46 (Score 1541, +4.44 SD); Updated (2006) note: No specific characterisation with acceptable identity scores carried out yet. Putative kept [...] ","uniprotId":"Q0PAU6_CAMJE"},"10641414":{"id":10641414,"externalId":"192222.Cj0572","name":"ribB","hasFamily":true,"annotation":"Catalyzes the conversion of D-ribulose 5-phosphate to formate and 3,4-dihydroxy-2-butanone 4-phosphate","uniprotId":"RIBB_CAMJE"},"10641415":{"id":10641415,"externalId":"192222.Cj0573","name":"Cj0573","hasFamily":true,"annotation":"Putative GatB/Yqey family protein; Original (2000) note: Cj0573, unknown, len: 147 aa; similar to several hypothetical proteins e.g.YQEY_BACSU (148 aa), fasta scores; opt: 265 z-score: 348.7 E(): 4e-12, 32.9% identity in 140 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02637 GatB/Yqey identified within CDS. This domain is found in GatB and proteins related to bacterial Yqey. Typically about 140 aa residues long. This domain is found at the C terminus of GatB which transamidates Glu-tRNA to Gln-tRNA. The function of this domain is uncertain. It does however suggest that Y [...] ","uniprotId":"Q0PAU4_CAMJE"},"10641416":{"id":10641416,"externalId":"192222.Cj0574","name":"ilvI","hasFamily":true,"annotation":"Acetolactate synthase; Original (2000) note: Cj0574, ilvI, probable acetolactate synthase large subunit, len: 566 aa; highly similar to many e.g.ILVI_ECOLI acetolactate synthase isozyme III large subunit (EC 4.1.3.18) (574 aa), fasta scores; opt: 1713 z-score: 2101.4 E(): 0, 46.5% identity in 572 aa overlap (also 43.4% identity to ILVB_ECOLI, and 42.0% identity to ILVG_ECOLI). No Hp match. Contains Pfam match to entry PF00205 TPP_enzymes, Thiamine pyrophosphate enzymes, score 769.00, E-value 6.9e-238; Updated (2006) note: Characterised within Escherichia coli with acceptable identity s [...] ","uniprotId":"Q0PAU3_CAMJE"},"10641417":{"id":10641417,"externalId":"192222.Cj0575","name":"ilvH","hasFamily":true,"annotation":"Original (2000) note: Cj0575, ilvH, probable acetolactate synthase small subunit, len 145 aa; similar to many e.g. ILVH_ECOLI acetolactate synthase isozyme III small subunit (EC 4.1.3.18) (163 aa), fasta scores; opt: 341 z-score: 523.3 E(): 7.5e-22, 40.0% identity in 155 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF01842 ACT domain identified within CDS. Further support given to product function. Characterised within Escherichia coli with acceptable identity score. Putative not added to product function. EC number updated. Functional classification -Amino acid biosynthes [...] ","uniprotId":"Q0PAU2_CAMJE"},"10641418":{"id":10641418,"externalId":"192222.Cj0576","name":"lpxD","hasFamily":true,"annotation":"Catalyzes the N-acylation of UDP-3-O-acylglucosamine using 3- hydroxyacyl-ACP as the acyl donor. Is involved in the biosynthesis of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell. ","uniprotId":"LPXD_CAMJE"},"10641419":{"id":10641419,"externalId":"192222.Cj0577c","name":"queA","hasFamily":true,"annotation":"Transfers and isomerizes the ribose moiety from AdoMet to the 7-aminomethyl group of 7-deazaguanine (preQ1-tRNA) to give epoxyqueuosine (oQ-tRNA)","uniprotId":"QUEA_CAMJE"},"10641420":{"id":10641420,"externalId":"192222.Cj0578c","name":"tatC","hasFamily":true,"annotation":"Part of the twin-arginine translocation (Tat) system that transports large folded proteins containing a characteristic twin- arginine motif in their signal peptide across membranes. Together with TatB, TatC is part of a receptor directly interacting with Tat signal peptides","uniprotId":"TATC_CAMJE"},"10641421":{"id":10641421,"externalId":"192222.Cj0579c","name":"tatB","hasFamily":true,"annotation":"Part of the twin-arginine translocation (Tat) system that transports large folded proteins containing a characteristic twin- arginine motif in their signal peptide across membranes. Together with TatC, TatB is part of a receptor directly interacting with Tat signal peptides. TatB may form an oligomeric binding site that transiently accommodates folded Tat precursor proteins before their translocation. ","uniprotId":"TATB_CAMJE"},"10641422":{"id":10641422,"externalId":"192222.Cj0580c","name":"Cj0580c","hasFamily":true,"annotation":"Probably acts as a heme chaperone, transferring heme to an unknown acceptor. Binds one molecule of heme per monomer, possibly covalently. Binds 1 [4Fe-4S] cluster. The cluster is coordinated with 3 cysteines and an exchangeable S-adenosyl-L-methionine. ","uniprotId":"Q0PAT7_CAMJE"},"10641423":{"id":10641423,"externalId":"192222.Cj0581","name":"nudH","hasFamily":true,"annotation":"Accelerates the degradation of transcripts by removing pyrophosphate from the 5'-end of triphosphorylated RNA, leading to a more labile monophosphorylated state that can stimulate subsequent ribonuclease cleavage","uniprotId":"RPPH_CAMJE"},"10641424":{"id":10641424,"externalId":"192222.Cj0582","name":"lysC","hasFamily":true,"annotation":"Belongs to the aspartokinase family. ","uniprotId":"Q0PAT5_CAMJE"},"10641425":{"id":10641425,"externalId":"192222.Cj0583","name":"Cj0583","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0583, unknown, len: 178 aa; 30.8% identity to HP1230. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PAT4_CAMJE"},"10641426":{"id":10641426,"externalId":"192222.Cj0584","name":"Cj0584","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0584, unknown, len: 199 aa; 36.3% identity to HP1231 (annotated as DNA polymerase III delta prime subunit (holB)). No significant similarity to other holB proteins. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PAT3_CAMJE"},"10641427":{"id":10641427,"externalId":"192222.Cj0585","name":"folP","hasFamily":true,"annotation":"Dihydropteroate synthase; Original (2000) note: Cj0585, folP, probable dihydropteroate synthase, len: 380 aa; simliar in C-terminus to e.g. DHPS_ECOLI dihydropteroate synthase (EC 2.5.1.15) (282 aa), fasta scores; opt: 508 z-score: 504.3 E(): 8.5e-21, 35.8% identity in 274 aa overlap. 41.6% identity to HP1232. Contains Pfam match to entry PF00809 DHPS, Dihydropteroate synthase; Updated (2006) note: Pfam domain PF00809 DHPS,Dihydropteroate synthase has now been changed to Pterin binding enzyme. Characterised within Campylobacter jejuni and Escherichia coli with acceptable identity score [...] ","uniprotId":"Q0PAT2_CAMJE"},"10641428":{"id":10641428,"externalId":"192222.Cj0586","name":"ligA","hasFamily":true,"annotation":"DNA ligase that catalyzes the formation of phosphodiester linkages between 5'-phosphoryl and 3'-hydroxyl groups in double- stranded DNA using NAD as a coenzyme and as the energy source for the reaction. It is essential for DNA replication and repair of damaged DNA","uniprotId":"DNLJ_CAMJE"},"10641429":{"id":10641429,"externalId":"192222.Cj0587","name":"Cj0587","hasFamily":true,"annotation":"Original (2000) note: Cj0587, probable integral membrane protein, len: 334 aa; no Hp match; Updated (2006) note: Eight probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PAT0_CAMJE"},"10641430":{"id":10641430,"externalId":"192222.Cj0588","name":"tlyA","hasFamily":true,"annotation":"Putative haemolysin; Original (2000) note: Cj0588, tlyA, possible haemolysin, len: 253 aa; similar to e.g. HLYA_TREHY hemolysin A from Treponema hyodysenteriae (Serpulina hyodysenteriae) (240 aa), fasta scores; opt: 342 z-score: 596.3 E(): 6.4e-26, 33.7% identity in 261 aa overlap. 40.7% identity to HP1086; Updated (2006) note: Pfam domain PF01479 S4 and PF01728 FtsJ-like methyltransferase domain have been identified within CDS. Further support given to product function. Characterisation work carried out within Treponema hyodysenteriae with marginal identity score. Putative kept within [...] ","uniprotId":"Q0PAS9_CAMJE"},"10641431":{"id":10641431,"externalId":"192222.Cj0589","name":"ribF","hasFamily":true,"annotation":"Belongs to the ribF family. ","uniprotId":"Q0PAS8_CAMJE"},"10641432":{"id":10641432,"externalId":"192222.Cj0590","name":"cmoA","hasFamily":true,"annotation":"Catalyzes the conversion of S-adenosyl-L-methionine (SAM) to carboxy-S-adenosyl-L-methionine (Cx-SAM)","uniprotId":"CMOA_CAMJE"},"10641433":{"id":10641433,"externalId":"192222.Cj0591c","name":"Cj0591c","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0591c, probable lipoprotein, len: 73 aa; no Hp match. Contains N-terminal signal sequence and PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Some similarity to upstream gene Cj0592c (39.7% identity in 63 aa overlap). Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PAS6_CAMJE"},"10641434":{"id":10641434,"externalId":"192222.Cj0592c","name":"Cj0592c","hasFamily":false,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0592c, probable periplasmic protein, len: 144 aa; no Hp match. Contains N-terminal signal sequence. Some similarity to downstream gene Cj0591c (39.7% identity in 63 aa overlap). Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PAS5_CAMJE"},"10641435":{"id":10641435,"externalId":"192222.Cj0593c","name":"Cj0593c","hasFamily":true,"annotation":"Original (2000) note: Cj0593c, probable integral membrane protein, len: 210 aa; similar to hypothetical proteins e.g. YICG_ECOLI (205 aa), fasta scores; opt: 783 z-score: 1285.7 E(): 0, 57.4% identity in 204 aa overlap. No Hp match; Updated (2006) note: Pfam domains x2 PF03458 UPF0126 domain was identified within CDS. This domain is found duplicated in bacterial membrane proteins of unknown function and contains three transmembrane helices. Also seven probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Functional classification - Membranes,  [...] ","uniprotId":"Y593_CAMJE"},"10641436":{"id":10641436,"externalId":"192222.Cj0594c","name":"Cj0594c","hasFamily":true,"annotation":"Putative DNA/RNA non-specific endonuclease; Original (2000) note: Cj0594c, probable periplasmic protein, len: 216 aa; no Hp match. Contains N-terminal signal sequence; Updated (2006) note: Pfam domain PF01223 DNA/RNA non-specific endonuclease was identified within CDS. Product modified to more specific family member due to motif match. No specific characterisation with acceptable identity score has been carried out yet. Thus, putative kept within product function. Functional classification -DNA replication, restriction/modification, recombination and repair","uniprotId":"Q0PAS3_CAMJE"},"10641437":{"id":10641437,"externalId":"192222.Cj0595c","name":"nth","hasFamily":true,"annotation":"DNA repair enzyme that has both DNA N-glycosylase activity and AP-lyase activity. The DNA N-glycosylase activity releases various damaged pyrimidines from DNA by cleaving the N-glycosidic bond, leaving an AP (apurinic/apyrimidinic) site. The AP-lyase activity cleaves the phosphodiester bond 3' to the AP site by a beta-elimination, leaving a 3'-terminal unsaturated sugar and a product with a terminal 5'- phosphate","uniprotId":"Q0PAS2_CAMJE"},"10641438":{"id":10641438,"externalId":"192222.Cj0596","name":"peb4-cbf2","hasFamily":true,"annotation":"Putative peptidyl-prolyl cis-trans isomerase Cbf2; Original (2000) note: Cj0596, peb4-cbf2, major antigenic peptide PEB4-cell binding factor 2, len: 273 aa; identical to CBF2_CAMJE cell binding factor 2 precursor,and TR:G451285 peb4=major antigenic peptide (34 aa). Also similar to memebers of the PPIC-parvulin family of rotamases e.g. PRTM_LACPA protease maturation protein precursor (299 aa), fasta scores; opt: 238 z-score: 303.5 E(): 1.3e-09, 31.3% identity in 240 aa overlap, and CYPD_ECOLI peptidyl-prolyl cis-trans isomerase D (623 aa),fasta scores; opt: 244 z-score: 293.9 E(): 4.5e- [...] ","uniprotId":"CBF2_CAMJE"},"10641439":{"id":10641439,"externalId":"192222.Cj0597","name":"fba","hasFamily":true,"annotation":"Catalyzes the aldol condensation of dihydroxyacetone phosphate (DHAP or glycerone-phosphate) with glyceraldehyde 3-phosphate (G3P) to form fructose 1,6-bisphosphate (FBP) in gluconeogenesis and the reverse reaction in glycolysis","uniprotId":"ALF_CAMJE"},"10641440":{"id":10641440,"externalId":"192222.Cj0598","name":"Cj0598","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0598, probable membrane protein, len: 480 aa; no Hp match; Updated (2006) note: Four probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PAR9_CAMJE"},"10641441":{"id":10641441,"externalId":"192222.Cj0599","name":"Cj0599","hasFamily":true,"annotation":"Putative OmpA family membrane protein; Original (2000) note: Cj0599, probable periplasmic protein, len: 317 aa; no Hp match. Contains N-terminal signal sequence; Updated (2006) note: Pfam domain PF00691 OmpA family protein was identified within CDS. Also, one probable transmembrane helix predicted by TMHMM2.0. Further support given to product function. No specific characterisation with acceptable identity scores has been carried out yet. Putative kept within product function. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PAR8_CAMJE"},"10641442":{"id":10641442,"externalId":"192222.Cj0600","name":"Cj0600","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0600, unknown, len: 291 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PAR7_CAMJE"},"10641443":{"id":10641443,"externalId":"192222.Cj0601c","name":"Cj0601c","hasFamily":true,"annotation":"Belongs to the sodium:neurotransmitter symporter (SNF) (TC 2.A.22) family","uniprotId":"Q0PAR6_CAMJE"},"10641444":{"id":10641444,"externalId":"192222.Cj0602c","name":"Cj0602c","hasFamily":true,"annotation":"MOSC-domain containing protein; Original (2000) note: Cj0602c, unknown, len: 222 aa; similar to hypothetical proteins e.g. YIIM_ECOLI (234 aa), fasta scores; opt: 298 z-score: 406.4 E(): 2.4e-15,30.4% identity in 207 aa overlap. No Hp match; Updated (2006) note: Pfam domains PF03475 3-alpha domain and PF03473 MOSC domain were identified within CDS. Product modified to more specific family member due to motif match. Functional classification - Misc","uniprotId":"Q0PAR5_CAMJE"},"10641445":{"id":10641445,"externalId":"192222.Cj0603c","name":"dsbD","hasFamily":true,"annotation":"Required to facilitate the formation of correct disulfide bonds in some periplasmic proteins and for the assembly of the periplasmic c-type cytochromes. Acts by transferring electrons from cytoplasmic thioredoxin to the periplasm. This transfer involves a cascade of disulfide bond formation and reduction steps (By similarity)","uniprotId":"DSBD_CAMJE"},"10641446":{"id":10641446,"externalId":"192222.Cj0604","name":"Cj0604","hasFamily":true,"annotation":"Putative polyphosphate kinase; Original (2000) note: Cj0604, unknown, len: 293 aa; similar to hypothetical proteins e.g. TR:O05877 (EMBL:O05877) Mycobacterium tuberculosis Rv3232c (MTCY20B11.07c) (295 aa), fasta scores; opt: 858 z-score: 1317.7 E(): 0, 52.0% identity in 227 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF03976 Domain of unknown function (DUF344) was identified within CDS This Pfam has now been modified to Polyphosphate kinase 2. Product modified to more specific family member due to motif match. No specific characterisation with acceptable identity score ha [...] ","uniprotId":"Q0PAR3_CAMJE"},"10641447":{"id":10641447,"externalId":"192222.Cj0605","name":"Cj0605","hasFamily":true,"annotation":"Putative amidohydrolase; Original (2000) note: Cj0605, probable amidohydrolase, len: 396 aa; similar to members of the AMA/HIPO/HYUC family of hydrolases (peptidase family M40) e.g. AMAA_BACST N-acyl-L-amino acid amidohydrolase (370 aa), fasta scores; opt: 805 z-score: 1165.8 E(): 0, 41.4% identity in 307 aa overlap. Also similar to HIPO_CAMJE hippurate hydrolase (Cj0985c) (383 aa) (39.1% identity in 376 aa overlap). No Hp match. Contains PS00070 Aldehyde dehydrogenases cysteine active site; Updated (2006) note: Pfam domain PF01546 Peptidase family M20/M25/M40 was identified within CDS [...] ","uniprotId":"Q0PAR2_CAMJE"},"10641448":{"id":10641448,"externalId":"192222.Cj0606","name":"Cj0606","hasFamily":true,"annotation":"Belongs to the membrane fusion protein (MFP) (TC 8.A.1) family","uniprotId":"Q0PAR1_CAMJE"},"10641449":{"id":10641449,"externalId":"192222.Cj0607","name":"macB","hasFamily":true,"annotation":"Non-canonical ABC transporter that contains transmembrane domains (TMD), which form a pore in the inner membrane, and an ATP- binding domain (NBD), which is responsible for energy generation. Confers resistance against macrolides","uniprotId":"MACB_CAMJE"},"10641450":{"id":10641450,"externalId":"192222.Cj0608","name":"Cj0608","hasFamily":true,"annotation":"Original (2000) note: Cj0608, possible outer membrane protein, len: 456 aa; similar to hypothetical proteins e.g. YE62_HAEIN HI1462 (454 aa), fasta scores; opt: 711 z-score: 933.7 E(): 0, 29.8% identity in 450 aa overlap, and weakly to outer membrene proteins e.g. TR:Q51006 (EMBL:X95635) Neisseria gonorrhoeae MTRE gene (fragment) (467 aa), fasta scores; opt: 289 z-score: 325.5 E(): 8.3e-11, 22.8% identity in 465 aa overlap. No Hp match. Also similar to Cj0365c (28.6% identity in 455 aa overlap). Contains N-terminal signal sequence, PS00013 Prokaryotic membrane lipoprotein lipid attachm [...] ","uniprotId":"Q0PAQ9_CAMJE"},"10641451":{"id":10641451,"externalId":"192222.Cj0609c","name":"Cj0609c","hasFamily":true,"annotation":"Original (2000) note: Cj0609c, possible periplasmic protein, len: 392 aa; no Hp match. Contains possible N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PAQ8_CAMJE"},"10641452":{"id":10641452,"externalId":"192222.Cj0610c","name":"Cj0610c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0610c, probable periplasmic protein, len: 336 aa; no Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF04311 Protein of unknown function (DUF459) was identified within CDS. Product function kept the same. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PAQ7_CAMJE"},"10641453":{"id":10641453,"externalId":"192222.Cj0611c","name":"Cj0611c","hasFamily":true,"annotation":"Belongs to the membrane-bound acyltransferase family. ","uniprotId":"Q0PAQ6_CAMJE"},"10641454":{"id":10641454,"externalId":"192222.Cj0612c","name":"cft","hasFamily":true,"annotation":"Iron-storage protein","uniprotId":"FTN_CAMJE"},"10641455":{"id":10641455,"externalId":"192222.Cj0613","name":"pstS","hasFamily":true,"annotation":"Original (2000) note: Cj0613, pstS, possible periplasmic phosphate binding protein, len: 167 aa; similar to e.g. SPHX_SYNP7 SPHX protein precursor (337 aa), fasta scores; opt: 346 z-score: 602.6 E(): 2.9e-26,33.2% identity in 301 aa overlap. No Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Characterised within Escherichia coli, however, identity score was unnacceptable. Putative kept within product function. Functional classification - Transport/binding proteins -Anions; PMID:2651888, PMID:7741855","uniprotId":"Q0PAQ4_CAMJE"},"10641456":{"id":10641456,"externalId":"192222.Cj0614","name":"pstC","hasFamily":true,"annotation":"Part of the binding-protein-dependent transport system for phosphate; probably responsible for the translocation of the substrate across the membrane","uniprotId":"Q0PAQ3_CAMJE"},"10641457":{"id":10641457,"externalId":"192222.Cj0615","name":"pstA","hasFamily":true,"annotation":"Original (2000) note: Cj0615, pstA, probable phosphate transport system permease protein, len: 362 aa; similar to e.g. PSTA_ECOLI phosphate transport system permease protein (296 aa), fasta scores; opt: 346 z-score: 447.5 E(): 1.3e-17, 29.2% identity in 226 aa overlap. No Hp ortholog. Also similar to upstream gene Cj0614 (30.7% identity in 254 aa overlap). Contains Pfam match to entry PF00528 BPD_transp, Binding-protein-dependent transport systems inner membrane component; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Characterised within Escherichia co [...] ","uniprotId":"Q0PAQ2_CAMJE"},"10641458":{"id":10641458,"externalId":"192222.Cj0616","name":"pstB","hasFamily":true,"annotation":"Part of the ABC transporter complex PstSACB involved in phosphate import. Responsible for energy coupling to the transport system","uniprotId":"PSTB_CAMJE"},"10641459":{"id":10641459,"externalId":"192222.Cj0617","name":"Cj0617","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0617, unknown, len: 203 aa; no Hp match. A member of the 617 family of C.j. proteins containing homopolymeric tracts; similar to N-terminus of Cj1305c (38.9% identity in 180 aa overlap), Cj1306c (39.2% identity in 181 aa overlap), Cj1310c (37.4% identity in 182 aa overlap) and Cj1342c (49.7% identity in 191 aa overlap). The similarities continue in the downstream ORF Cj0618, suggesting that these two ORFs may encode a single protein, but are frameshifted w.r.t. each other. There is a poly-G tract at the site of the possible frameshift (5 [...] ","uniprotId":"Q0PAQ0_CAMJE"},"10641460":{"id":10641460,"externalId":"192222.Cj0618","name":"Cj0618","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0618, unknown, len: 215 aa; no Hp match. A member of the 617 family of C.j. proteins containing homopolymeric tracts; similar to C-terminus of Cj1305c (36.7% identity in 221 aa overlap), Cj1306c (39.1% identity in 220 aa overlap), Cj1310c (33.6% identity in 220 aa overlap) and Cj1342c (43.4% identity in 221 aa overlap). The similarities continue from the upstream ORF Cj0617, suggesting that these two ORFs may encode a single protein, but are frameshifted w.r.t. each other. There is a poly-G tract at thesite of the possible frameshift (57 [...] ","uniprotId":"Q0PAP9_CAMJE"},"10641461":{"id":10641461,"externalId":"192222.Cj0619","name":"Cj0619","hasFamily":true,"annotation":"Putative MATE family transport protein; Original (2000) note: Cj0619, probable integral membrane protein, len: 438 aa; similar to hypothetical proteins e.g. TR:O07940 B. subtilis YISQ (455 aa), fasta scores; opt: 880 z-score: 1364.9 E(): 0, 32.1% identity in 446 aa overlap, and YEEO_ECOLI (547 aa), fasta scores; opt: 164 z-score: 529.6 E(): 3.3e-22, 22.7% identity in 437 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF01554 MatE was identified within CDS. This family of protein function as drug/sodium antiporters. These proteins mediate resistance to a wide range of cationi [...] ","uniprotId":"Q0PAP8_CAMJE"},"10641462":{"id":10641462,"externalId":"192222.Cj0620","name":"Cj0620","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0620, unknown, len: 215 aa; some simlarity to TR:O58443 (EMBL:AB009490) Pyrococcus horikoshii PHCF016 (230 aa), fasta scores; opt: 229 z-score: 237.3 E(): 6.4e-06, 26.9% identity in 197 aa overlap. 33.7% identity to HP0806, 25.0% identity to HP1401; Updated (2006) note: Pfam domain PF01863 Protein of unknown function DUF45 identified within CDS. This motif has no known function. Members are found in some archaebacteria, as well as Helicobacter pylori (Epsilon Proteobacteria). The proteins are 190-240 amino acids long, with the C terminus [...] ","uniprotId":"Q0PAP7_CAMJE"},"10641463":{"id":10641463,"externalId":"192222.Cj0621","name":"Cj0621","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0621, unknown, len: 506 aa; similar to TR:E1186594 (EMBL:AJ003049) hypothetical protein HYDE from Wolinella succinogenes (546 aa), fasta scores; opt: 207 z-score: 285.3 E(): 1.3e-08, 25.8% identity in 551 aa overlap. 23.2% identity to HP0635; Updated (2006) note: Literature search identified paper linking product function to hydE-like protein. Functional classification - Conserved hypothetical proteins; PMID:10533287","uniprotId":"Q0PAP6_CAMJE"},"10641464":{"id":10641464,"externalId":"192222.Cj0622","name":"hypF","hasFamily":true,"annotation":"Involved in the maturation of [NiFe] hydrogenases. Along with HypE, it catalyzes the synthesis of the CN ligands of the active site iron of [NiFe]-hydrogenases. HypF functions as a carbamoyl transferase using carbamoylphosphate as a substrate and transferring the carboxamido moiety in an ATP-dependent reaction to the thiolate of the C-terminal cysteine of HypE yielding a protein-S-carboxamide. ","uniprotId":"Q0PAP5_CAMJE"},"10641465":{"id":10641465,"externalId":"192222.Cj0623","name":"hypB","hasFamily":true,"annotation":"Original (2000) note: Cj0623, hypB, probable hydrogenase isoenzymes formation protein, len: 247 aa; similar to many e.g. HYPB_ECOLI hydrogenase isoenzymes formation protein (290 aa), fasta scores; opt: 691 z-score: 1142.3 E(): 0, 46.1% identity in 230 aa overlap. 44.2% identity to HP0900. Contains PS00028 Zinc finger,C2H2 type, domain; Updated (2006) note: Pfam domain PF02492 CobW/HypB/UreG, nucleotide-binding domain was identified within CDS. Further support given to product function. Characterised within Escherichia coli with acceptable identity score. Putative not added to product f [...] ","uniprotId":"Q0PAP4_CAMJE"},"10641466":{"id":10641466,"externalId":"192222.Cj0624","name":"hypC","hasFamily":true,"annotation":"Original (2000) note: Cj0624, hypC, probable hydrogenase isoenzymes formation protein, len: 93 aa; similar to many e.g. HYPC_ECOLI hydrogenase isoenzymes formation protein (90 aa), fasta scores; opt: 145 z-score: 173.2 E(): 0.024, 35.9% identity in 78 aa overlap. 53.3% identity to HP0899; Updated (2006) note: Pfam domain PF01455 HupF/HypC family protein was identified within CDS. Further support given to product function. Characterised within Escherichia coli with acceptable identity score. Putative not added to product function. Functional classification -Energy metabolism - Respirati [...] ","uniprotId":"Q0PAP3_CAMJE"},"10641467":{"id":10641467,"externalId":"192222.Cj0625","name":"hypD","hasFamily":true,"annotation":"Belongs to the HypD family. ","uniprotId":"Q0PAP2_CAMJE"},"10641468":{"id":10641468,"externalId":"192222.Cj0626","name":"hypE","hasFamily":true,"annotation":"Original (2000) note: Cj0626, hypE, probable hydrogenase isoenzymes formation protein, len: 324 aa; similar to many e.g. HYPE_ECOLI hydrogenase isoenzymes formation protein (322 aa), fasta scores; opt: 705 z-score: 923.2 E(): 0, 40.4% identity in 327 aa overlap. 46.8% identity to HP0047. Contains Pfam match to entry PF00586 AIRS, AIR synthase related proteins; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Putative not added to product function. Functional classification -Energy metabolism - Respiration; PMID:1849603","uniprotId":"Q0PAP1_CAMJE"},"10641469":{"id":10641469,"externalId":"192222.Cj0627","name":"hypA","hasFamily":true,"annotation":"Involved in the maturation of [NiFe] hydrogenases. Required for nickel insertion into the metal center of the hydrogenase. ","uniprotId":"HYPA_CAMJE"},"10641470":{"id":10641470,"externalId":"192222.Cj0628","name":"Cj0628","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0628, probable lipoprotein,len: 171 aa; no Hp match. Contains N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. There is a variable poly-T tract at the C-terminus T(4-5). T5 causes the ORF to stop here, T4 allows translation into the downstream ORF Cj0629. Similar to Cj1677 (42.4% identity in 172 aa overlap). Cj1677 has a non-variable poly-T tract in an quivalent position, which could allow translation into Cj1678 (highly similar to Cj0629); Original (2000) note: Cj0629,  [...] ","uniprotId":"Q0PAN9_CAMJE"},"10641471":{"id":10641471,"externalId":"192222.Cj0630c","name":"holA","hasFamily":true,"annotation":"Putative DNA polymerase III, delta subunit; Original (2000) note: Cj0630c, unknown, len: 321 aa; 26.6% identity to HP1247; Updated (2006) note: Pfam domain PF06144 DNA polymerase III, delta subunit identified within CDS. Product modified to more specific family member due to motif match. No specific characterisation has been carried out. Putative kept within product function. Functional classification - DNA replication,restriction/modification, recombination and repair; PMID:11432857","uniprotId":"Q0PAN8_CAMJE"},"10641472":{"id":10641472,"externalId":"192222.Cj0631c","name":"Cj0631c","hasFamily":true,"annotation":"Putative ribonuclease; Original (2000) note: Cj0631c, possible ribonuclease, len: 644 aa; similar to members of the ribonuclease II (RNB) family, e.g. VACB_ECOLI VACB protein (813 aa), fasta scores; opt: 503 z-score: 829.1 E(): 0,31.0% identity in 580 aa overlap. 35.1% identity to HP1248. Contains PS01175 Ribonuclease II family signature,and Pfam match to entry PF00773 RNB, RNB-like proteins; Updated (2006) note: Some characterisation within Escherichia coli, however, identity scores were marginal. Putative kept within product function. Functional classification - Degradation of macrom [...] ","uniprotId":"Q0PAN7_CAMJE"},"10641473":{"id":10641473,"externalId":"192222.Cj0632","name":"ilvC","hasFamily":true,"annotation":"Involved in the biosynthesis of branched-chain amino acids (BCAA). Catalyzes an alkyl-migration followed by a ketol-acid reduction of (S)-2-acetolactate (S2AL) to yield (R)-2,3-dihydroxy-isovalerate. In the isomerase reaction, S2AL is rearranged via a Mg-dependent methyl migration to produce 3-hydroxy-3-methyl-2-ketobutyrate (HMKB). In the reductase reaction, this 2-ketoacid undergoes a metal-dependent reduction by NADPH to yield (R)-2,3-dihydroxy-isovalerate. ","uniprotId":"ILVC_CAMJE"},"10641474":{"id":10641474,"externalId":"192222.Cj0633","name":"Cj0633","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0633, probable periplasmic protein, len: 360 aa; similar in C-terminus to hypothetical proteins e.g. TR:O51711 (EMBL:AE001176) Borrelia burgdorferi BB0770 (293 aa), fasta scores; opt: 425 z-score: 553.2 E(): 1.6e-23, 36.3% identity in 237 aa overlap. No Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF04748 Divergent polysaccharide deacetylase identified within CDS. No specific characterisation has been carried out yet. Product function unchanged. Functional classification -Miscellaneous per [...] ","uniprotId":"Q0PAN5_CAMJE"},"10641475":{"id":10641475,"externalId":"192222.Cj0634","name":"dprA","hasFamily":true,"annotation":"DNA processing protein A; Original (2000) note: Cj0634, unknown, len: 257 aa; similar, in part, to members of the SMF family e.g. SMF_HAEIN SMF protein (DNA processing chain A) (373 aa),fasta scores; opt: 329 z-score: 487.6 E(): 7.2e-20, 31.5% identity in 213 aa overlap. 48.8% identity to HP0333; Updated (2006) note: Pfam domain PF02481 SMF family identified within CDS. Further support given to product function. Literature search identified Campylobacter jejuni characterising paper. Product modified to more specific family member. Functional classification - DNA replication, restrictio [...] ","uniprotId":"Q0PAN4_CAMJE"},"10641476":{"id":10641476,"externalId":"192222.Cj0635","name":"Cj0635","hasFamily":true,"annotation":"Could be a nuclease involved in processing of the 5'-end of pre-16S rRNA","uniprotId":"YQGF_CAMJE"},"10641477":{"id":10641477,"externalId":"192222.Cj0636","name":"Cj0636","hasFamily":true,"annotation":"Belongs to the class I-like SAM-binding methyltransferase superfamily. RsmB/NOP family","uniprotId":"Q0PAN2_CAMJE"},"10641478":{"id":10641478,"externalId":"192222.Cj0637c","name":"mrsA","hasFamily":true,"annotation":"Has an important function as a repair enzyme for proteins that have been inactivated by oxidation. Catalyzes the reversible oxidation-reduction of methionine sulfoxide in proteins to methionine. ","uniprotId":"MSRA_CAMJE"},"10641479":{"id":10641479,"externalId":"192222.Cj0638c","name":"ppa","hasFamily":true,"annotation":"Catalyzes the hydrolysis of inorganic pyrophosphate (PPi) forming two phosphate ions","uniprotId":"IPYR_CAMJE"},"10641480":{"id":10641480,"externalId":"192222.Cj0639c","name":"adk","hasFamily":true,"annotation":"Catalyzes the reversible transfer of the terminal phosphate group between ATP and AMP. Plays an important role in cellular energy homeostasis and in adenine nucleotide metabolism","uniprotId":"KAD_CAMJE"},"10641481":{"id":10641481,"externalId":"192222.Cj0640c","name":"aspS","hasFamily":true,"annotation":"Aspartyl-tRNA synthetase with relaxed tRNA specificity since it is able to aspartylate not only its cognate tRNA(Asp) but also tRNA(Asn). Reaction proceeds in two steps: L-aspartate is first activated by ATP to form Asp-AMP and then transferred to the acceptor end of tRNA(Asp/Asn)","uniprotId":"SYDND_CAMJE"},"10641482":{"id":10641482,"externalId":"192222.Cj0641","name":"nadK","hasFamily":true,"annotation":"Involved in the regulation of the intracellular balance of NAD and NADP, and is a key enzyme in the biosynthesis of NADP. Catalyzes specifically the phosphorylation on 2'-hydroxyl of the adenosine moiety of NAD to yield NADP","uniprotId":"NADK_CAMJE"},"10641483":{"id":10641483,"externalId":"192222.Cj0642","name":"recN","hasFamily":true,"annotation":"May be involved in recombinational repair of damaged DNA. ","uniprotId":"RECN_CAMJE"},"10641484":{"id":10641484,"externalId":"192222.Cj0643","name":"cbrR","hasFamily":true,"annotation":"Original (2000) note: Cj0643, probable two-component response regulator, len: 414 aa; similar to e.g. TR:P97018 (EMBL:U67070) Aeromonas jandaei response regulator protein RRPX (415 aa), fasta scores; opt: 597 z-score: 530.0 E(): 3.2e-22, 25.8% identity in 414 aa overlap, andTR:Q46020 (EMBL:L42554) Caulobacter crescentus response regulator PLED (454 aa), fasta scores; opt: 286 z-score: 309.9 E(): 5.8e-10, 22.0% identity in 451 aa overlap. No Hp ortholog. Contains 2x Pfam match to entry PF00072 response_reg, Response regulator receiver domain and Pfam match to entry PF00990 DUF9, Domain  [...] ","uniprotId":"Q0PAM5_CAMJE"},"10641485":{"id":10641485,"externalId":"192222.Cj0644","name":"Cj0644","hasFamily":true,"annotation":"Putative TatD-related deoxyribonuclease protein; Original (2000) note: Cj0644, unknown, len: 271 aa; similar to hypothetical proteins e.g. YABD_BACSU (255 aa),fasta scores; opt: 614 z-score: 1024.7 E(): 0, 38.6% identity in 259 aa overlap. 49.2% identity to HP1573. Contains PS01137 Uncharacterized protein family UPF0006 signature 1 and Pfam match to entry PF01026 UPF0006,Uncharacterized protein family; Updated (2006) note: Pfam domain PF01026 TatD related DNase and Prosite domain PS01137 TATD_1,TatD-related deoxyribonuclease were identified within CDS. Product modified to more specific [...] ","uniprotId":"Q0PAM4_CAMJE"},"10641486":{"id":10641486,"externalId":"192222.Cj0645","name":"Cj0645","hasFamily":true,"annotation":"Original (2000) note: Cj0645, possible secreted transglycosylase, len: 372 aa; similar to members of the SLT family e.g. MLTD_ECOLI membrane-bound lytic murein transglycosylase D precursor (EC 3.2.1.-) (452 aa), fasta scores; opt: 425 z-score: 431.3 E(): 1e-16, 28.4% identity in 348 aa overlap. 37.5% identity to HP1572. Contains N-terminal signal sequence; Updated (2006) note: Pfam domains PF01464 Transglycosylase SLT domain PF01476 LysM domain were identified within CDS. Further support given to product function. Characterised within Escherichia coli with marginal identity score and o [...] ","uniprotId":"Q0PAM3_CAMJE"},"10641487":{"id":10641487,"externalId":"192222.Cj0646","name":"rlpA","hasFamily":true,"annotation":"Lytic transglycosylase with a strong preference for naked glycan strands that lack stem peptides","uniprotId":"RLPA_CAMJE"},"10641488":{"id":10641488,"externalId":"192222.Cj0647","name":"Cj0647","hasFamily":true,"annotation":"Putative HAD-superfamily hydrolase; Original (2000) note: Cj0647, unknown, len: 162 aa; similar to hypothetical proteins e.g. YRBI_ECOLI (188 aa), fasta scores; opt: 322 z-score: 399.6 E(): 5.8e-15,40.0% identity in 155 aa overlap. 40.9% identity to HP1570; Updated (2006) note: Pfam domain PF08282 haloacid dehalogenase-like hydrolase was identified by running own search within CDS. Product modified to more specific family member due to motif match. No specific characterisation carried out yet. Putative kept within product function. Some similarity to recently characterised Escherichia  [...] ","uniprotId":"Q0PAM1_CAMJE"},"10641489":{"id":10641489,"externalId":"192222.Cj0648","name":"Cj0648","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0648, unknwn, len: 171 aa; no Hp match; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Also, one N-terminal signal peptide predicted by SignalP 2.0 HMM. Product function modified to more specific family member. Putative kept within product function. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PAM0_CAMJE"},"10641490":{"id":10641490,"externalId":"192222.Cj0649","name":"Cj0649","hasFamily":true,"annotation":"Putative OstA family protein; Original (2000) note: Cj0649, unknown, len: 153 aa; 31.6% identity to HP1568; Updated (2006) note: Pfam domain PF03968 OstA-like protein was identified within CDS. Product modified to more specific family member due to Pfam match. No specific characterisation has been carried out. Putative kept within product function. Functional classification -Conserved hypothetical proteins; PMID:7811102","uniprotId":"Q0PAL9_CAMJE"},"10641491":{"id":10641491,"externalId":"192222.Cj0650","name":"engB","hasFamily":true,"annotation":"Necessary for normal cell division and for the maintenance of normal septation","uniprotId":"ENGB_CAMJE"},"10641492":{"id":10641492,"externalId":"192222.Cj0651","name":"Cj0651","hasFamily":true,"annotation":"Original (2000) note: Cj0651, possible integral membrane protein, len: 164 aa; no Hp match; Updated (2006) note: Five probable transmembrane helices predicted by TMHMM2.0 identified within CDS. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PAL7_CAMJE"},"10641493":{"id":10641493,"externalId":"192222.Cj0652","name":"pbpC","hasFamily":true,"annotation":"Penicillin-binding protein; Original (2000) note: Cj0652, pbpC, probable penicillin-binding protein, len: 601 aa; similar to many e.g. PBP2_ECOLI penicillin-binding protein 2 (PBP-2) (633 aa), fasta scores; opt: 954 z-score: 1474.4 E(): 0, 32.7% identity in 609 aa overlap. 43.5% identity to HP1565. Contains Pfam match to entry PF00905 Transpeptidase,Penicillin binding protein transpeptidase domain; Updated (2006) note: Pfam domain PF03717 Penicillin-binding Protein dimerisation domain was identified within CDS. Further support given to product function. Characterised within Escherichia [...] ","uniprotId":"Q0PAL6_CAMJE"},"10641494":{"id":10641494,"externalId":"192222.Cj0653c","name":"Cj0653c","hasFamily":true,"annotation":"Putative aminopeptidase; Original (2000) note: Cj0653c, probable aminopeptidase, len: 596 aa; similar to e.g. TR:O54975 (EMBL:AF038591) rattus norvegicus cytoplasmic aminopeptidase P (623 aa), fasta scores; opt: 763 z-score: 1470.5 E(): 0, 36.8% identity in 612 aa overlap, and TR:O51094 (EMBL:AE001119) Borrelia burgdorferi putative peptidase BB0067 (592 aa), fasta scores; opt: 1234 z-score: 1316.7 E(): 0, 41.1% identity in 609 aa overlap. 25.8% identity to HP1037. Contains Pfam match to entry PF00557 pep_M24, metallopeptidase family M24; Updated (2006) note: No specific characterisatio [...] ","uniprotId":"Q0PAL5_CAMJE"},"10641495":{"id":10641495,"externalId":"192222.Cj0659c","name":"Cj0659c","hasFamily":true,"annotation":"Putative periplasmic protein; HMMPfam hit to PF00854, POT family, score 8e-11","uniprotId":"Q0PAL4_CAMJE"},"10641496":{"id":10641496,"externalId":"192222.Cj0660c","name":"Cj0660c","hasFamily":true,"annotation":"Putative transmembrane protein; Original (2000) note: Cj0660c, possible transmembrane protein, len: 368 aa; no Hp match. Contains a possible transmembrane domain around aa 220; Updated (2006) note: One probable transmembrane helices predicted by TMHMM2.0 identified within CDS. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PAL3_CAMJE"},"10641497":{"id":10641497,"externalId":"192222.Cj0661c","name":"era","hasFamily":true,"annotation":"An essential GTPase that binds both GDP and GTP, with rapid nucleotide exchange. Plays a role in 16S rRNA processing and 30S ribosomal subunit biogenesis and possibly also in cell cycle regulation and energy metabolism","uniprotId":"ERA_CAMJE"},"10641498":{"id":10641498,"externalId":"192222.Cj0662c","name":"hslU","hasFamily":true,"annotation":"ATPase subunit of a proteasome-like degradation complex; this subunit has chaperone activity. The binding of ATP and its subsequent hydrolysis by HslU are essential for unfolding of protein substrates subsequently hydrolyzed by HslV. HslU recognizes the N-terminal part of its protein substrates and unfolds these before they are guided to HslV for hydrolysis","uniprotId":"HSLU_CAMJE"},"10641499":{"id":10641499,"externalId":"192222.Cj0663c","name":"hslV","hasFamily":true,"annotation":"Protease subunit of a proteasome-like degradation complex believed to be a general protein degrading machinery. ","uniprotId":"HSLV_CAMJE"},"10641500":{"id":10641500,"externalId":"192222.Cj0664c","name":"rplI","hasFamily":true,"annotation":"Binds to the 23S rRNA","uniprotId":"RL9_CAMJE"},"10641501":{"id":10641501,"externalId":"192222.Cj0665c","name":"argG","hasFamily":true,"annotation":"Belongs to the argininosuccinate synthase family. Type 1 subfamily","uniprotId":"ASSY_CAMJE"},"10641502":{"id":10641502,"externalId":"192222.Cj0667","name":"Cj0667","hasFamily":true,"annotation":"Putative S4 domain protein; Original (2000) note: Cj0667, unknown, len: 81 aa; similar to many hypothetical proteins e.g. YABO_BACSU (86 aa), fasta scores; opt: 149 z-score: 259.2 E(): 3.8e-07,35.4% identity in 79 aa overlap. 43.6% identity to HP1423; Updated (2006) note: Pfam domain PF01479 S4 domain was identified within CDS. The S4 domain probably mediates binding to RNA. Product modified to more specific family member due to motif match. No specific characterisation has been carried out. Putative kept within product function. Functional classification - Misc","uniprotId":"Q0PAK7_CAMJE"},"10641503":{"id":10641503,"externalId":"192222.Cj0668","name":"Cj0668","hasFamily":true,"annotation":"Original (2000) note: Cj0668, probable ATP/GTP-binding protein, len: 135 aa; similar to hypothetical proteins e.g. YJEE_ECOLI (153 aa), fasta scores; opt: 164 z-score: 205.3 E(): 0.00039, 33.8% identity in 80 aa overlap. 44.2% identity to HP0716. Contains PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Pfam domain PF02367 Uncharacterised P-loop hydrolase UPF0079 was identified within CDS. No specific characterisation has been carried out yet. Putative kept within product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PAK6_CAMJE"},"10641504":{"id":10641504,"externalId":"192222.Cj0669","name":"Cj0669","hasFamily":true,"annotation":"Original (2000) note: Cj0669, probable ABC-transporter ATP-binding protein, len: 242 aa; highly similar to many e.g. YHBG_ECOLI probable ABC transporter ATP-binding protein (240 aa), fasta scores; opt: 840 z-score: 1144.2 E(): 0, 54.2% identity in 236 aa overlap. 65.8% identity to HP0715. Contains PS00017 ATP/GTP-binding site motif A (P-loop) and Pfam match to entry PF00005 ABC_tran, ABC transporters; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Appropriate motifs also present. Putative not added to product function. Functional classificati [...] ","uniprotId":"Q0PAK5_CAMJE"},"10641505":{"id":10641505,"externalId":"192222.Cj0670","name":"rpoN","hasFamily":true,"annotation":"Sigma factors are initiation factors that promote the attachment of RNA polymerase to specific initiation sites and are then released","uniprotId":"Q0PAK4_CAMJE"},"10641506":{"id":10641506,"externalId":"192222.Cj0671","name":"dcuB","hasFamily":true,"annotation":"Responsible for the transport of C4-dicarboxylates from the periplasm across the inner membrane","uniprotId":"Q0PAK3_CAMJE"},"10641507":{"id":10641507,"externalId":"192222.Cj0672","name":"Cj0672","hasFamily":false,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0672, possible periplasmic protein, len: 62 aa; no Hp match. Contains possible N-terminal signal sequence and second hydrophobic domain; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PAK2_CAMJE"},"10641508":{"id":10641508,"externalId":"192222.Cj0677","name":"kdpB","hasFamily":true,"annotation":"Part of the high-affinity ATP-driven potassium transport (or Kdp) system, which catalyzes the hydrolysis of ATP coupled with the electrogenic transport of potassium into the cytoplasm. This subunit is responsible for energy coupling to the transport system. ","uniprotId":"Q0PAK1_CAMJE"},"10641509":{"id":10641509,"externalId":"192222.Cj0679","name":"kdpD","hasFamily":true,"annotation":"Truncated KdpD protein; 1 probable transmembrane helix predicted for Cj0678 by TMHMM2.0 at aa 5-24","uniprotId":"Q0PAK0_CAMJE"},"10641510":{"id":10641510,"externalId":"192222.Cj0680c","name":"uvrB","hasFamily":true,"annotation":"The UvrABC repair system catalyzes the recognition and processing of DNA lesions. A damage recognition complex composed of 2 UvrA and 2 UvrB subunits scans DNA for abnormalities. Upon binding of the UvrA(2)B(2) complex to a putative damaged site, the DNA wraps around one UvrB monomer. DNA wrap is dependent on ATP binding by UvrB and probably causes local melting of the DNA helix, facilitating insertion of UvrB beta-hairpin between the DNA strands. Then UvrB probes one DNA strand for the presence of a lesion. If a lesion is found the UvrA subunits dissociate and the UvrB-DNA preincision [...] ","uniprotId":"UVRB_CAMJE"},"10641511":{"id":10641511,"externalId":"192222.Cj0681","name":"Cj0681","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0681, unknown, len: 76 aa; 36.2% identity to HP0385. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PAJ8_CAMJE"},"10641512":{"id":10641512,"externalId":"192222.Cj0682","name":"Cj0682","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0682, unknown, len: 80 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PAJ7_CAMJE"},"10641513":{"id":10641513,"externalId":"192222.Cj0683","name":"Cj0683","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0683, probable periplasmic protein, len: 145 aa; no Hp match; Updated (2006) note: One probable transmembrane helices predicted by TMHMM2.0. Further support for product function. Pfam domain PF07963 Prokaryotic N-terminal methylation motif was identified within CDS. The motif is involved in methylation and is said to be found at the N-terminus of pilins and other proteins involved in secretion. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PAJ6_CAMJE"},"10641514":{"id":10641514,"externalId":"192222.Cj0684","name":"priA","hasFamily":true,"annotation":"Involved in the restart of stalled replication forks. Recognizes and binds the arrested nascent DNA chain at stalled replication forks. It can open the DNA duplex, via its helicase activity, and promote assembly of the primosome and loading of the major replicative helicase DnaB onto DNA","uniprotId":"Q0PAJ5_CAMJE"},"10641515":{"id":10641515,"externalId":"192222.Cj0685c","name":"cipA","hasFamily":true,"annotation":"Invasion protein CipA; Original (2000) note: Cj0685c, possible sugar transferase, len: 450 aa; no Hp match. Contains variable poly-G tract near C-terminus; G9 (consensus) allows complete translation, G10 would cause truncation at aa 233. Some similarity to Cj1421c (28.0% identity in 404 aa overlap) and Cj1422c (25.3% identity in 442 aa overlap),both of which contain variable polyG tracts, though in different positions. Putative functional assignment is based on family clustering (BLASTP) with other C. jejuni predicted sugar transferases e,g, Cj1434c, and Cj1438c; Updated (2006) note: N [...] ","uniprotId":"Q0PAJ4_CAMJE"},"10641516":{"id":10641516,"externalId":"192222.Cj0686","name":"ispG","hasFamily":true,"annotation":"Converts 2C-methyl-D-erythritol 2,4-cyclodiphosphate (ME- 2,4cPP) into 1-hydroxy-2-methyl-2-(E)-butenyl 4-diphosphate. ","uniprotId":"ISPG_CAMJE"},"10641517":{"id":10641517,"externalId":"192222.Cj0687c","name":"flgH","hasFamily":true,"annotation":"Assembles around the rod to form the L-ring and probably protects the motor/basal body from shearing forces during rotation. ","uniprotId":"FLGH_CAMJE"},"10641518":{"id":10641518,"externalId":"192222.Cj0688","name":"pta","hasFamily":true,"annotation":"Original (2000) note: Cj0688, pta, probable phosphate acetyltransferase, len: 501 aa; highly similar in C-terminus to e.g. PTA_ECOLI phosphate acetyltransferase (EC 2.3.1.8) (713 aa), fasta scores; opt: 1012 z-score: 1161.8 E(): 0, 48.8% identity in 326 aa overlap. 24.8% identity in N-terminus to HP0905; Updated (2006) note: Pfam domain PF01515 Phosphate acetyl/butaryl transferase was identified within CDS. Further support given to product function. Characterised within Escherichia coli with acceptable identity score,however, sequence alignment was only partial. Putative kept within pr [...] ","uniprotId":"Q0PAJ1_CAMJE"},"10641519":{"id":10641519,"externalId":"192222.Cj0689","name":"ackA","hasFamily":true,"annotation":"Catalyzes the formation of acetyl phosphate from acetate and ATP. Can also catalyze the reverse reaction","uniprotId":"ACKA_CAMJE"},"10641520":{"id":10641520,"externalId":"192222.Cj0690c","name":"Cj0690c","hasFamily":true,"annotation":"Original (2000) note: Cj0690c, possible restriction/modification enzyme, len: 1250 aa; similar to Borrelia burgdorferi plasmid TR:O50698 (EMBL:AE000785) hypotheticalprotein BBE02 (1277 aa), fasta scores; opt: 1517 z-score: 1607.6 E(): 0, 35.0% identity in 1304 aa overlap. Very weak similarity to T257_ECOLI type IIS restriction enzyme ECO57I (997 aa), fasta scores; opt: 156 z-score: 163.4 E(): 0.083, 19.6% identity in 981 aa overlap. No Hp ortholog. Some simlarity to Cj0031 (25.0% identity in 859 aa overlap). Contains PS00092 N-6 Adenine-specific DNA methylases signature; Updated (2006) [...] ","uniprotId":"Q0PAI9_CAMJE"},"10641521":{"id":10641521,"externalId":"192222.Cj0691","name":"Cj0691","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0691, possible membrane protein, len: 163 aa; no Hp match. Contains two possible transmembrane domains. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PAI8_CAMJE"},"10641522":{"id":10641522,"externalId":"192222.Cj0692c","name":"Cj0692c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0692c, possible membrane protein, len: 312 aa; 34.7% identity in N-terminus to HP0708 (114 aa). Contains one possible transmembrane domain. Functional classification - Membranes,lipoproteins and porins","uniprotId":"Q0PAI7_CAMJE"},"10641523":{"id":10641523,"externalId":"192222.Cj0693c","name":"mraW","hasFamily":true,"annotation":"Specifically methylates the N4 position of cytidine in position 1402 (C1402) of 16S rRNA","uniprotId":"RSMH_CAMJE"},"10641524":{"id":10641524,"externalId":"192222.Cj0694","name":"Cj0694","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0694, probable periplasmic protein, len: 496 aa; C-terminus is almost identical to C. jejuni ORF TR:Q46115 (EMBL:X95910). 31.9% identity toHP0977. Contains probable N-terminal signal sequence; Updated (2006) note: Literature search identified paper linking product function to glycoprotein. Functional classification - Miscellaneous periplasmic proteins; PMID:12186869","uniprotId":"Q0PAI5_CAMJE"},"10641525":{"id":10641525,"externalId":"192222.Cj0695","name":"ftsA","hasFamily":true,"annotation":"Cell division protein that is involved in the assembly of the Z ring. May serve as a membrane anchor for the Z ring. ","uniprotId":"Q0PAI4_CAMJE"},"10641526":{"id":10641526,"externalId":"192222.Cj0696","name":"ftsZ","hasFamily":true,"annotation":"Essential cell division protein that forms a contractile ring structure (Z ring) at the future cell division site. The regulation of the ring assembly controls the timing and the location of cell division. One of the functions of the FtsZ ring is to recruit other cell division proteins to the septum to produce a new cell wall between the dividing cells. Binds GTP and shows GTPase activity. ","uniprotId":"Q0PAI3_CAMJE"},"10641527":{"id":10641527,"externalId":"192222.Cj0697","name":"flgG2","hasFamily":true,"annotation":"Flagellar basal body protein; Original (2000) note: Cj0697, flgG2, probable flagellar basal-body rod protein, len: 270 aa; similar to e.g. FLGG_ECOLI flagellar basal-body rod protein FLGG (260 aa), fasta scores; opt: 290 z-score: 318.7 E(): 1.9e-10,28.7% identity in 272 aa overlap. 42.6% identity to HP1092. Also similar to downstream gene Cj0698 (flgG,29.9% identity in 281 aa overlap). Contains PS00588 Flagella basal body rod proteins signature and Pfam match to entry PF00460 flg_bb_rod, Flagella basal body rod proteins; Updated (2006) note: Characterised within Escherichia coli with m [...] ","uniprotId":"Q0PAI2_CAMJE"},"10641528":{"id":10641528,"externalId":"192222.Cj0698","name":"flgG","hasFamily":true,"annotation":"Belongs to the flagella basal body rod proteins family. ","uniprotId":"Q0PAI1_CAMJE"},"10641529":{"id":10641529,"externalId":"192222.Cj0699c","name":"glnA","hasFamily":true,"annotation":"Glutamine synthetase; Original (2000) note: Cj0699c, glnA, probable glutamine synthetase, len: 476 aa; highly similar to many e.g. GLNA_ECOLI glutamine synthetase (EC 6.3.1.2) (468 aa), fasta scores; opt: 1529 z-score: 1812.8 E(): 0,51.5% identity in 462 aa overlap. 63.6% identity to HP0512. Contains PS00180 Glutamine synthetase signature 1,PS00181 Glutamine synthetase putative ATP-binding region signature, and Pfam match to entry PF00120 gln-synt; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Putative not added to product function. Function [...] ","uniprotId":"Q0PAI0_CAMJE"},"10641530":{"id":10641530,"externalId":"192222.Cj0700","name":"Cj0700","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0700, unknown, len: 231 aa; 49.4% identity to HP0170. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PAH9_CAMJE"},"10641531":{"id":10641531,"externalId":"192222.Cj0701","name":"Cj0701","hasFamily":true,"annotation":"Putative protease; Original (2000) note: Cj0701, probable protease,len: 417 aa; similar to many putative proteases belonging to the peptidase family U32, e.g. YEGQ_ECOLI putative protease in baer-ogrk intergenic Region (453 aa), fasta scores; opt: 567 z-score: 648.6 E(): 7.9e-29, 34.9% identity in 404 aa overlap. 55.5% identity to HP0169. Contains PS01276 Peptidase family U32 signature and Pfam match to entry PF01136 Peptidase_U32; Updated (2006) note: Similar to many with marginal identity scores. Putative kept within product function. Functional classification - Degradation of macrom [...] ","uniprotId":"Q0PAH8_CAMJE"},"10641532":{"id":10641532,"externalId":"192222.Cj0702","name":"purE","hasFamily":true,"annotation":"Catalyzes the conversion of N5-carboxyaminoimidazole ribonucleotide (N5-CAIR) to 4-carboxy-5-aminoimidazole ribonucleotide (CAIR)","uniprotId":"Q0PAH7_CAMJE"},"10641533":{"id":10641533,"externalId":"192222.Cj0703","name":"Cj0703","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0703, unknown, len: 178 aa; 51.0% identity to HP0406. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PAH6_CAMJE"},"10641534":{"id":10641534,"externalId":"192222.Cj0704","name":"glyQ","hasFamily":true,"annotation":"Glycine--tRNA ligase alpha subunit; Original (2000) note: Cj0704, glyQ, probable glycyl-tRNA synthetase alpha chain, len: 287 aa; similar to many e.g. SYGA_ECOLI glycyl-tRNA synthetase alpha chain (EC 6.1.1.14) (303 aa), fasta scores; opt: 1228 z-score: 1442.3 E(): 0, 66.1% identity in 280 aa overlap. 69.1% identity to HP0960. Contains PS00339 Aminoacyl-transfer RNA synthetases class-II signature 2; Updated (2006) note: Pfam domain PF02091 Glycyl-tRNA synthetase alpha subunit identified within CDS. Further support given to product function. Characterised within Escherichia coli with ac [...] ","uniprotId":"SYGA_CAMJE"},"10641535":{"id":10641535,"externalId":"192222.Cj0705","name":"Cj0705","hasFamily":true,"annotation":"GTP cyclohydrolase 1 type 2 homolog; Original (2000) note: Cj0705, unknown, len: 241 aa; similar to hypothetical proteins e.g. TR:O51424 (EMBL:AE001151) Borrelia burgdorferi BB0468 (248 aa),fasta scores; opt: 240 z-score: 292.4 E(): 5.5e-09, 27.0% identity in 244 aa overlap. 42.7% identity to HP0959; Updated (2006) note: Pfam domain PF01784 NIF3 (NGG1p interacting factor 3) identified within CDS. Conserved added to product function. Functional classification - Conserved hypothetical proteins","uniprotId":"GCH1L_CAMJE"},"10641536":{"id":10641536,"externalId":"192222.Cj0706","name":"Cj0706","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0706, unknown, len: 238 aa; similar to hypothetical proteins e.g. TR:O67273 (EMBL:AE000729) Aquifex aeolicus AQ_1223 (235 aa), fasta scores; opt: 298 z-score: 287.3 E(): 1.1e-08, 30.1% identity in 236 aa overlap. 37.4% identity to HP0958; Updated (2006) note: Pfam domain PF02591 Uncharacterized ACR, COG1579 identified within CDS. Conserved added to product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PAH3_CAMJE"},"10641537":{"id":10641537,"externalId":"192222.Cj0707","name":"kdtA","hasFamily":true,"annotation":"Involved in lipopolysaccharide (LPS) biosynthesis. Catalyzes the transfer of 3-deoxy-D-manno-octulosonate (Kdo) residue(s) from CMP- Kdo to lipid IV(A), the tetraacyldisaccharide-1,4'-bisphosphate precursor of lipid A","uniprotId":"Q0PAH2_CAMJE"},"10641538":{"id":10641538,"externalId":"192222.Cj0708","name":"Cj0708","hasFamily":true,"annotation":"Original (2000) note: Cj0708, possible ribosomal pseudouridine synthase, len: 250 aa; similar to many hypothetical proteins and to e.g. RLUC_ECOLI ribosomal large subunit pseudouridine synthase D (EC 4.2.1.70) (319 aa), fasta scores; opt: 202 z-score: 252.9 E(): 8.6e-07,29.7% identity in 175 aa overlap. 47.0% identity to HP0956. Contains PS01129 Hypothetical yabO/yceC/sfhB family signature and Pfam match to entry PF00849 YABO,Hypothetical yabO/yceC/sfhB family. Also similar to Cj1280c (26.9% identity in 223 aa overlap) and Cj0022c (27.6% identity in 275 aa overlap); Updated (2006) note [...] ","uniprotId":"Q0PAH1_CAMJE"},"10641539":{"id":10641539,"externalId":"192222.Cj0709","name":"ffh","hasFamily":true,"annotation":"Original (2000) note: Cj0709, ffh, probable signal recognition particle protein, len: 445 aa; highly similar to many e.g. SR54_ECOLI signal recognition particle protein (453 aa), fasta scores; opt: 1136 z-score: 1195.2 E(): 0, 41.5% identity in 441 aa overlap. 61.0% identity to HP1152. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00300 SRP54-type proteins GTP-binding domain signature, and Pfam match to entry PF00448 SRP54,SRP54-type protein; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Putative not added to product function. Fu [...] ","uniprotId":"Q0PAH0_CAMJE"},"10641540":{"id":10641540,"externalId":"192222.Cj0710","name":"rpsP","hasFamily":true,"annotation":"Belongs to the bacterial ribosomal protein bS16 family. ","uniprotId":"RS16_CAMJE"},"10641541":{"id":10641541,"externalId":"192222.Cj0711","name":"Cj0711","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0711, unknown, len: 80 aa; simlar to small hypothetical proteins downstream of rpsP in many organisms e.g. TR:O31738 (EMBL:Z99112) Bacillus subtilis YLQC (81 aa), fasta scores; opt: 115 z-score: 178.5 E(): 0.012, 32.3% identity in 62 aa overlap. 37.7% identity to HP1150. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PAG8_CAMJE"},"10641542":{"id":10641542,"externalId":"192222.Cj0712","name":"rimM","hasFamily":true,"annotation":"An accessory protein needed during the final step in the assembly of 30S ribosomal subunit, possibly for assembly of the head region. Probably interacts with S19. Essential for efficient processing of 16S rRNA. May be needed both before and after RbfA during the maturation of 16S rRNA. It has affinity for free ribosomal 30S subunits but not for 70S ribosomes","uniprotId":"RIMM_CAMJE"},"10641543":{"id":10641543,"externalId":"192222.Cj0713","name":"trmD","hasFamily":true,"annotation":"Specifically methylates guanosine-37 in various tRNAs. ","uniprotId":"TRMD_CAMJE"},"10641544":{"id":10641544,"externalId":"192222.Cj0714","name":"rplS","hasFamily":true,"annotation":"This protein is located at the 30S-50S ribosomal subunit interface and may play a role in the structure and function of the aminoacyl-tRNA binding site","uniprotId":"RL19_CAMJE"},"10641545":{"id":10641545,"externalId":"192222.Cj0715","name":"Cj0715","hasFamily":true,"annotation":"Belongs to the transthyretin family. 5-hydroxyisourate hydrolase subfamily","uniprotId":"Q0PAG4_CAMJE"},"10641546":{"id":10641546,"externalId":"192222.Cj0716","name":"Cj0716","hasFamily":true,"annotation":"Original (2000) note: Cj0716, probable phospho-2-dehydro-3-deoxyheptonate aldolase, len: 446 aa; similar to enzymes from plants e.g. AROF_ARATH phospho-2-dehydro-3-deoxyheptonate aldolase (EC 4.1.2.15) (525 aa), fasta scores; opt: 1680 z-score: 1935.9 E(): 0,54.2% identity in 445 aa overlap. 59.6% identity to HP0134; Updated (2006) note: Pfam domain PF01474 Class-II DAHP synthetase family identified within CDS. Product function kept the same (along with putative), as no specific characterisation within related bacteria has been carried out. Functional classification - Amino acid biosyn [...] ","uniprotId":"Q0PAG3_CAMJE"},"10641547":{"id":10641547,"externalId":"192222.Cj0717","name":"Cj0717","hasFamily":true,"annotation":"Belongs to the ArsC family. ","uniprotId":"Q0PAG2_CAMJE"},"10641548":{"id":10641548,"externalId":"192222.Cj0718","name":"dnaE","hasFamily":true,"annotation":"DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. This DNA polymerase also exhibits 3' to 5' exonuclease activity. The alpha chain is the DNA polymerase (By similarity)","uniprotId":"DPO3A_CAMJE"},"10641549":{"id":10641549,"externalId":"192222.Cj0719c","name":"Cj0719c","hasFamily":true,"annotation":"Pyridoxal 5'-phosphate (PLP)-binding protein, which is involved in PLP homeostasis","uniprotId":"Q0PAG0_CAMJE"},"10641550":{"id":10641550,"externalId":"192222.Cj0720c","name":"flaC","hasFamily":true,"annotation":"Plays a role in virulence","uniprotId":"FLAC_CAMJE"},"10641551":{"id":10641551,"externalId":"192222.Cj0721c","name":"Cj0721c","hasFamily":true,"annotation":"Putative integral membrane protein. Functional classification-Membranes, lipoproteins and porins; Original (2000) note: Cj0721c, probable integral membrane protein, len: 156 aa; 35.9% identity to HP0288; Updated (2006) note: Four probable transmembrane helices predicted by TMHMM2.0","uniprotId":"Q0PAF8_CAMJE"},"10641552":{"id":10641552,"externalId":"192222.Cj0722c","name":"Cj0722c","hasFamily":true,"annotation":"Putative DNA methylase; Original (2000) note: Cj0722c, probable DNA methylase, len: 271 aa; similar to members of the hemK family of methylases, e.g. HEMK_ECOLI HEMK protein (277 aa), fasta scores; opt: 262 z-score: 315.7 E(): 2.7e-10,26.1% identity in 276 aa overlap. 36.9% identity to HP0381. Contains PS00092 N-6 Adenine-specific DNA methylases signature; Updated (2006) note: Prosite domain PS50193 SAM_BIND, SAM (and some other nucleotide) binding motif identified within CDS. Further support given to product function. No specific characterisation with acceptable identity score carried [...] ","uniprotId":"Q0PAF7_CAMJE"},"10641553":{"id":10641553,"externalId":"192222.Cj0723c","name":"Cj0723c","hasFamily":true,"annotation":"Original (2000) note: Cj0723c, probable integral membrane zinc-metalloprotease, len: 395 aa; simimlar to yeast proteases e.g. ST24_YEAST CAAX prenyl protease 1 (453 aa), fasta scores; opt: 494 z-score: 565.5 E(): 3.4e-24, 31.6% identity in 430 aa overlap, and to prokaryotic htpX, e.g. HTPX_ECOLI heat shock protein HTPX (293 aa), fasta scores; opt: 183 z-score: 216.0 E(): 9.9e-05, 24.1% identity in 295 aa overlap. 38.6% identity to HP0382. Contains PS00142 Neutral zinc metallopeptidases, zinc-binding region signature; Updated (2006) note: Pfam domain PF01435 Peptidase family M48 and Pro [...] ","uniprotId":"Q0PAF6_CAMJE"},"10641554":{"id":10641554,"externalId":"192222.Cj0724","name":"Cj0724","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0724, unknown, len: 61 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PAF5_CAMJE"},"10641555":{"id":10641555,"externalId":"192222.Cj0725c","name":"Cj0725c","hasFamily":true,"annotation":"Molybdopterin biosynthesis protein; Original (2000) note: Cj0725c, mog, probable molybdopterin biosynthesis protein, len: 180 aa; similar to e.g. MOG_ECOLI molybdopterin biosynthesis MOG protein (195 aa), fasta scores; opt: 506 z-score: 615.6 E(): 5.5e-27, 50.8% identity in 187 aa overlap. 65.3% identity to HP0799; Updated (2006) note: Pfam domain PF00994 Probable molybdopterin binding domain found within CDS. Characterised within Escherichia coli with acceptable identity score. Putative not added to product function. Functional classification - Biosynthesis of cofactors,prosthetic gro [...] ","uniprotId":"Q0PAF4_CAMJE"},"10641556":{"id":10641556,"externalId":"192222.Cj0726c","name":"corA","hasFamily":true,"annotation":"Mediates influx of magnesium ions. Can also mediate cobalt and manganese uptake (By similarity). Alternates between open and closed states. Activated by low cytoplasmic Mg(2+) levels. Inactive when cytoplasmic Mg(2+) levels are high (By similarity). ","uniprotId":"CORA_CAMJE"},"10641557":{"id":10641557,"externalId":"192222.Cj0727","name":"Cj0727","hasFamily":true,"annotation":"Original (2000) note: Cj0727, probable periplasmic solute-binding protein, len: 348 aa; similar to e.g. SUBI_ECOLI sulfate-binding protein precursor (329 aa),fasta scores; opt: 220 z-score: 256.1 E(): 5.8e-07, 25.0% identity in 220 aa overlap. No Hp match. Contains N-terminal signal sequence; Updated (2006) note: Pfam domain PF01547 Bacterial extracellular solute-binding protein identified within CDS. Further support given to product function. Characterised within Escherichia coli, however, identity scores were unnacceptable. Putative kept within product function. Functional classifica [...] ","uniprotId":"Q0PAF2_CAMJE"},"10641558":{"id":10641558,"externalId":"192222.Cj0728","name":"Cj0728","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0728, probable periplasmic protein, len: 441 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PAF1_CAMJE"},"10641559":{"id":10641559,"externalId":"192222.Cj0729","name":"Cj0729","hasFamily":true,"annotation":"Putative type I phosphodiesterase/nucleotide pyrophosphatase; Original (2000) note: Cj0729, unknown, len: 269 aa; no Hp match; Updated (2006) note: Pfam domain PF01663 Type I phosphodiesterase / nucleotide pyrophosphatase identified within CDS. Product modified to more specific family member due to motif match. No specific characterisation has been carried out yet, so putative kept within product function. Functional classification - Purines,pyrimidines, nucleosides and nucleotides - Miscellaneous nucleoside/nucleotide reactions","uniprotId":"Q0PAF0_CAMJE"},"10641560":{"id":10641560,"externalId":"192222.Cj0730","name":"Cj0730","hasFamily":true,"annotation":"Original (2000) note: Cj0730, probable ABC transport system permease, len: 279 aa; similar to members of the binding protein-dependent permease family e.g. UGPA_ECOLI SN-glycerol-3-phosphate transport system permease (295 aa), fasta scores; opt: 273 z-score: 324.4 E(): 9.1e-11, 25.0% identity in 280 aa overlap. No Hp match. Contains Pfam match to entry PF00528 BPD_transp,Binding-protein-dependent transport systems inner membrane component; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. No specific characterisati [...] ","uniprotId":"Q0PAE9_CAMJE"},"10641561":{"id":10641561,"externalId":"192222.Cj0731","name":"Cj0731","hasFamily":true,"annotation":"Original (2000) note: Cj0731, probable ABC transport system permease, len: 259 aa; similar to members of the binding protein-dependent permease family e.g. POTC_ECOLI spermidine/putrescine transport system permease (264 aa), fasta scores; opt: 271 z-score: 337.3 E(): 1.7e-11, 24.1% identity in 241 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00528 BPD_transp,Binding-protein-dependent transport systems inner membrane component; Updated (2006) note: Seven probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. No specific characteris [...] ","uniprotId":"Q0PAE8_CAMJE"},"10641562":{"id":10641562,"externalId":"192222.Cj0732","name":"Cj0732","hasFamily":true,"annotation":"Belongs to the ABC transporter superfamily. ","uniprotId":"Q0PAE7_CAMJE"},"10641563":{"id":10641563,"externalId":"192222.Cj0733","name":"Cj0733","hasFamily":true,"annotation":"Putative HAD-superfamily hydrolase; Original (2000) note: Cj0733, unknown, len: 212 aa; some similarity to hypotheical proteins from Actinomycetes e.g. TR:O69629 (EMBL:AL022121) Rv3661 (MTV025.009) (287 aa), fasta scores; opt: 264 z-score: 321.7 E(): 1.3e-10,28.4% identity in 215 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF00702 haloacid dehalogenase-like hydrolase identified within CDS. No specific characterisation with acceptable identity score has been carried out yet. Putative kept within product function. Functional classification - Misc","uniprotId":"Q0PAE6_CAMJE"},"10641564":{"id":10641564,"externalId":"192222.Cj0734c","name":"hisJ","hasFamily":true,"annotation":"Bacterial periplasmic substrate-binding protein; Original (2000) note: Cj0734c, hisJ, probable histidine-binding protein precursor, len: 251 aa; almost identical to HISJ_CAMJE histidine-binding protein precursor (256 aa) (88.7% identity in 256 aa overlap) and similar to many priplasmic amino-acid binding proteins e.g. HISJ_ECOLI histidine-binding periplasmic protein precursor (260 aa), fasta scores; opt: 343 z-score: 390.5 E(): 1.9e-14. 33.5% identity in 263 aa overlap. No Hp ortholog. Contains N-terminal signal sequence, PS00013 Prokaryotic membrane lipoprotein lipid attachment site,P [...] "},"10641565":{"id":10641565,"externalId":"192222.Cj0735","name":"Cj0735","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0735, probable periplasmic protein, len: 239 aa; no Hp match. Similar to N-terminus of Cj0967 (E(): 6.7e-15, 34.5% identity in 116 aa overlap); similarity continues in downstream CDS Cj0736. Contains probable N-terminal signal sequence; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PAE4_CAMJE"},"10641566":{"id":10641566,"externalId":"192222.Cj0736","name":"Cj0736","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0736, unknown, len: 239 aa; no Hp match. Similar to C-terminus of Cj0967(E(): 0, 42.3% identity in 562 aa overlap); similarity continues from upstream CDS Cj0735; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PAE3_CAMJE"},"10641567":{"id":10641567,"externalId":"192222.Cj0737","name":"Cj0737","hasFamily":true,"annotation":"Putative haemagglutination activity domain containing protein; Original (2000) note: Cj0737, probable periplasmic protein, len: 358 aa; simlar to the N-terminus of HXA3_HAEIN heme:hemopexin-binding protein precursor (hxuA) (917 aa), fasta scores; opt: 236 z-score: 251.5 E(): 1e-06, 32.8% identity in 180 aa overlap, and N-terminus of TR:Q48028 H. influenzae adhesin (1477 aa), fasta scores; opt: 292 z-score: 307.3 E(): 8.1e-10, 28.9% identity in 381 aa overlap. No Hp match. Similar in part to Cj0968 and Cj0969, probably representing a pseudogene copy of this CDS; Updated (2006) note: Pfa [...] ","uniprotId":"Q0PAE2_CAMJE"},"10641568":{"id":10641568,"externalId":"192222.Cj0738","name":"Cj0738","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0738, unknown, len: 54 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PAE1_CAMJE"},"10641569":{"id":10641569,"externalId":"192222.Cj0739","name":"Cj0739","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0739, unknown, len: 57 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PAE0_CAMJE"},"10641570":{"id":10641570,"externalId":"192222.Cj0740","name":"Cj0740","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0740, unknown, len: 54 aa; no Hp match. Functional classification - . Functional classification - Unknown","uniprotId":"Q0PAD9_CAMJE"},"10641571":{"id":10641571,"externalId":"192222.Cj0741","name":"Cj0741","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0741, unknown, len: 308 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PAD8_CAMJE"},"10641572":{"id":10641572,"externalId":"192222.Cj0747","name":"tRNA-Ile","hasFamily":false,"annotation":"Uncharacterized protein; tRNA Ile anticodon GAT, Cove score 93.24","uniprotId":"Q0PAD7_CAMJE"},"10641573":{"id":10641573,"externalId":"192222.Cj0748","name":"Cj0748","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0748, unknown, len: 30 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PAD6_CAMJE"},"10641574":{"id":10641574,"externalId":"192222.Cj0753c","name":"tonB3","hasFamily":true,"annotation":"Interacts with outer membrane receptor proteins that carry out high-affinity binding and energy dependent uptake into the periplasmic space of specific substrates. It could act to transduce energy from the cytoplasmic membrane to specific energy-requiring processes in the outer membrane, resulting in the release into the periplasm of ligands bound by these outer membrane proteins. ","uniprotId":"Q0PAD5_CAMJE"},"10641575":{"id":10641575,"externalId":"192222.Cj0755","name":"cfrA","hasFamily":true,"annotation":"Ferric enterobactin uptake receptor; Original (2000) note: Cj0755, cfrA, probable iron uptake protein (ferric receptor), len: 696 aa; almost identical to TR:O07651 (EMBL:U80812) C. coli ferric receptor CFRA (696 aa); 98.7% identity in 696 aa overlap. Also simimlar to e.g. TR:O87518 (EMBL:AF081285) E. coli exogenous ferric siderophore receptor R4 (669 aa), fasta scores; opt: 1127 z-score: 1247.0 E(): 0, 34.5% identity in 693 aa overlap. 23.4% identity to HP0686; Updated (2006) note: Pfam domains PF07715 TonB-dependent Receptor Plug Domain and PF00593 TonB dependent receptor identified w [...] ","uniprotId":"Q0PAD4_CAMJE"},"10641576":{"id":10641576,"externalId":"192222.Cj0757","name":"hrcA","hasFamily":true,"annotation":"Negative regulator of class I heat shock genes (grpE-dnaK- dnaJ and groELS operons). Prevents heat-shock induction of these operons","uniprotId":"HRCA_CAMJE"},"10641577":{"id":10641577,"externalId":"192222.Cj0758","name":"grpE","hasFamily":true,"annotation":"Participates actively in the response to hyperosmotic and heat shock by preventing the aggregation of stress-denatured proteins, in association with DnaK and GrpE. It is the nucleotide exchange factor for DnaK and may function as a thermosensor. Unfolded proteins bind initially to DnaJ; upon interaction with the DnaJ-bound protein, DnaK hydrolyzes its bound ATP, resulting in the formation of a stable complex. GrpE releases ADP from DnaK; ATP binding to DnaK triggers the release of the substrate protein, thus completing the reaction cycle. Several rounds of ATP-dependent interactions be [...] ","uniprotId":"GRPE_CAMJE"},"10641578":{"id":10641578,"externalId":"192222.Cj0759","name":"dnaK","hasFamily":true,"annotation":"Acts as a chaperone","uniprotId":"DNAK_CAMJE"},"10641579":{"id":10641579,"externalId":"192222.Cj0760","name":"Cj0760","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0760, unknown, len: 340 aa; similar to hypothetical proteins e.g. YZ34_MYCTU Rv0906,(MTCY31.34) (372 aa), fasta scores; opt: 598 z-score: 724.2 E(): 4.8e-33, 29.9% identity in 335 aa overlap. Some similarity in C-terminus to ROMA_KLEPN outer membrane protein roma (fragment) (132 aa), fasta scores; opt: 335 z-score: 415.0 E(): 8.1e-16, 43.0% identity in 114 aa overlap. No Hp match; Updated (2006) note: Literature search identified paper giving clues to product function. Product function modified based on literature search. Putative kept w [...] ","uniprotId":"Q0PAD0_CAMJE"},"10641580":{"id":10641580,"externalId":"192222.Cj0761","name":"Cj0761","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0761, unknown, len: aa; similar to hypothetical proteins e.g. TR:O67079 (EMBL:AE000715) Aquifex aeolicus AQ_943 (107 aa), fasta scores; opt: 130 z-score: 186.5 E(): 0.0043, 28.6% identity in 84 aa overlap. 55.8% identity to HP0902; Updated (2006) note: Literature search identified paper giving clues to product function. Functional classification - Conserved hypothetical proteins; PMID:15758235","uniprotId":"Q0PAC9_CAMJE"},"10641581":{"id":10641581,"externalId":"192222.Cj0762c","name":"aspB","hasFamily":true,"annotation":"Aspartate aminotransferase; Original (2000) note: Cj0762c, aspB, probable aspartate aminotransferase, len: 389 aa; similar to e.g. AAT1_BACSU aspartate aminotransferase (EC 2.6.1.1) (393 aa), fasta scores; opt: 1216 z-score: 1352.8 E(): 0, 50.6% identity in 393 aa overlap. 57.5% identity to HP0672. Contains Pfam match to entry PF00155 aminotran_1,Aminotransferases class-I; Updated (2006) note: Characterised in Bacillus subtilis with acceptable identity score. Putative not added to product function. Functional classification -Amino acid biosynthesis - Aspartate family; PMID:10712618","uniprotId":"Q0PAC8_CAMJE"},"10641582":{"id":10641582,"externalId":"192222.Cj0763c","name":"cysE","hasFamily":true,"annotation":"Serine acetyltransferase; Original (2000) note: Cj0763c, cysE, probable serine acetyltransferase, len: 212 aa; highly similar to many e.g. CYSE_BACSU serine acetyltransferase (EC 2.3.1.30) (217 aa), fasta scores; opt: 598 z-score: 700.2 E(): 1e-31, 46.6% identity in 204 aa overlap. 50.9% identity to HP1210. Contains PS00101 Hexapeptide-repeat containing-transferases signature and 2x Pfam match to entry PF00132 hexapep, Bacterial transferase hexapeptide (four repeats); Updated (2006) note: Characterised in Bacillus subtilis with acceptable identity score. Putative not added to product f [...] ","uniprotId":"Q0PAC7_CAMJE"},"10641583":{"id":10641583,"externalId":"192222.Cj0764c","name":"speA","hasFamily":true,"annotation":"Catalyzes the biosynthesis of agmatine from arginine. ","uniprotId":"Q0PAC6_CAMJE"},"10641584":{"id":10641584,"externalId":"192222.Cj0765c","name":"hisS","hasFamily":true,"annotation":"Histidine--tRNA ligase; Original (2000) note: Cj0765c, hisS, probable histidyl-tRNA synthetase, len: 408 aa; similar to many e.g. SYH_ECOLI histidyl-tRNA synthetase (EC 6.1.1.21) (423 aa), fasta scores; opt: 927 z-score: 1052.2 E(): 0, 37.6% identity in 407 aa overlap. 29.8% identity to HP1190. Contains PS00339 Aminoacyl-transfer RNA synthetases class-II signature 2 and Pfam match to entry PF00587 tRNA-synt_2b, tRNA synthetases class II (Gly, His, Pro and Ser); Updated (2006) note: Characterised in Escherichia coli with acceptable identity score. Putative not added to product function. [...] ","uniprotId":"SYH_CAMJE"},"10641585":{"id":10641585,"externalId":"192222.Cj0766c","name":"tmk","hasFamily":true,"annotation":"Phosphorylation of dTMP to form dTDP in both de novo and salvage pathways of dTTP synthesis","uniprotId":"KTHY_CAMJE"},"10641586":{"id":10641586,"externalId":"192222.Cj0767c","name":"coaD","hasFamily":true,"annotation":"Reversibly transfers an adenylyl group from ATP to 4'- phosphopantetheine, yielding dephospho-CoA (dPCoA) and pyrophosphate. ","uniprotId":"COAD_CAMJE"},"10641587":{"id":10641587,"externalId":"192222.Cj0768c","name":"ubiX","hasFamily":true,"annotation":"Flavin prenyltransferase that catalyzes the synthesis of the prenylated FMN cofactor (prenyl-FMN) for 4-hydroxy-3-polyprenylbenzoic acid decarboxylase UbiD. The prenyltransferase is metal-independent and links a dimethylallyl moiety from dimethylallyl monophosphate (DMAP) to the flavin N5 and C6 atoms of FMN","uniprotId":"UBIX_CAMJE"},"10641588":{"id":10641588,"externalId":"192222.Cj0769c","name":"flgA","hasFamily":true,"annotation":"Involved in the assembly process of the P-ring formation. It may associate with FlgF on the rod constituting a structure essential for the P-ring assembly or may act as a modulator protein for the P- ring assembly","uniprotId":"Q0PAC1_CAMJE"},"10641589":{"id":10641589,"externalId":"192222.Cj0770c","name":"Cj0770c","hasFamily":true,"annotation":"Putative NLPA family lipoprotein; Original (2000) note: Cj0770c, probable periplasmic protein, len: 258 aa; similar to lipoproteins from other organisms e.g. PLPC_PASHA outer membrane lipoprotein 3 precursor (263 aa), fasta scores; opt: 569 z-score: 657.0 E(): 2.7e-29, 38.9% identity in 265 aa overlap, and TR:O69441 (EMBL:AJ005668) Legionella pneumophila 29 kda immunogenic protein (259 aa), fasta scores; opt: 584 z-score: 674.2 E(): 3e-30, 41.0% identity in 261 aa overlap. Also similar to Cj0772c (55.0% identity), Cj0771c (54.8% identity) and Cj1200 (44.9% identity). Single copy in Hp  [...] ","uniprotId":"Q0PAC0_CAMJE"},"10641590":{"id":10641590,"externalId":"192222.Cj0771c","name":"Cj0771c","hasFamily":true,"annotation":"Putative NLPA family lipoprotein; Original (2000) note: Cj0771c, probable periplasmic protein, len: 258 aa; similar to lipoproteins from other organisms e.g. NLPA_ECOLI lipoprotein-28 precursor (272 aa), fasta scores; opt: 623 z-score: 705.2 E(): 5.6e-32,46.4% identity in 237 aa overlap, and TR:O69441 (EMBL:AJ005668) Legionella pneumophila 29 kda immunogenic protein (259 aa), fasta scores; opt: 661 z-score: 747.9 E(): 0, 44.2% identity in 258 aa overlap. Also similar to Cj0772c (52.5% identity), Cj0770c (54.8% identity) and Cj1200 (45.6% identity). Single copy in Hp (HP1564); Updated ( [...] ","uniprotId":"Q0PAB9_CAMJE"},"10641591":{"id":10641591,"externalId":"192222.Cj0772c","name":"Cj0772c","hasFamily":true,"annotation":"Putative NLPA family lipoprotein; Original (2000) note: Cj0772c, probable periplasmic protein, len: 258 aa; similar to lipoproteins from other organisms e.g. PLPC_PASHA outer membrane lipoprotein 3 precursor (263 aa), fasta scores; opt: 642 z-score: 721.7 E(): 6.7e-33. 40.8% identity in 260 aa overlap, and TR:O69441 (EMBL:AJ005668) Legionella pneumophila 29 kda immunogenic protein (259 aa), fasta scores; opt: 638 z-score: 717.3 E(): 1.2e-32. 39.8% identity in 259 aa overlap. Also similar to Cj0770c (55.0% identity), Cj0771c (52.5% identity) and Cj1200 (57.5% identity). Single copy in H [...] ","uniprotId":"Q0PAB8_CAMJE"},"10641592":{"id":10641592,"externalId":"192222.Cj0773c","name":"Cj0773c","hasFamily":true,"annotation":"Original (2000) note: Cj0773c, probable binding-protein dependent transport system permease protein, len: 303 aa; similar to many e.g. PROW_BACSU glycine betaine/L-proline transport system permease protein (217 aa), fasta scores; opt: 212 z-score: 249.9 E(): 1.3e-06, 30.0% identity in 190 aa overlap. 43.9% identity in 205 aa overlap to HP1577. Contains Pfam match to entry PF00528 BPD_transp, Binding-protein-dependent transport systems inner membrane component; Updated (2006) note: Seven probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. No  [...] ","uniprotId":"Q0PAB7_CAMJE"},"10641593":{"id":10641593,"externalId":"192222.Cj0774c","name":"metN","hasFamily":true,"annotation":"Part of the ABC transporter complex MetNIQ involved in methionine import. Responsible for energy coupling to the transport system","uniprotId":"METN_CAMJE"},"10641594":{"id":10641594,"externalId":"192222.Cj0775c","name":"valS","hasFamily":true,"annotation":"Catalyzes the attachment of valine to tRNA(Val). As ValRS can inadvertently accommodate and process structurally similar amino acids such as threonine, to avoid such errors, it has a 'posttransfer' editing activity that hydrolyzes mischarged Thr-tRNA(Val) in a tRNA- dependent manner","uniprotId":"SYV_CAMJE"},"10641595":{"id":10641595,"externalId":"192222.Cj0776c","name":"Cj0776c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0776c, probable periplasmic protein, len: 340 aa; no Hp match. Contains N-terminal signal sequence. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PAB4_CAMJE"},"10641596":{"id":10641596,"externalId":"192222.Cj0777","name":"Cj0777","hasFamily":true,"annotation":"DNA helicase; Original (2000) note: Cj0777, probable ATP-dependent DNA helicase, len: 676 aa; similar to many members of the uvrD family of helicases, e.g. PCRA_STAAU ATP-dependent helicase PCRA (675 aa), fasta scores; opt: 561 z-score: 634.4 E(): 4.9e-28, 29.0% identity in 694 aa overlap, and REP_ECOLI ATP-dependent DNA helicase REP (673 aa), fasta scores; opt: 423 z-score: 478.5 E(): 2.4e-19,27.3% identity in 682 aa overlap. 47.5% identity to HP0911. Contains PS00017 ATP/GTP-binding site motif A (P-loop) and Pfam match to entry PF00580 UvrD-helicase,UvrD/REP helicase; Updated (2006)  [...] ","uniprotId":"Q0PAB3_CAMJE"},"10641597":{"id":10641597,"externalId":"192222.Cj0778","name":"peb2","hasFamily":true,"annotation":"Original (2000) note: Cj0778, peb2, major antigenic peptide PEB2, len: 245 aa; 94.9% identical to TR:G234576,sequenced by direct peptide sequencing; see: J. Biol. Chem. 266:16363-16369 (1991). Also similar to TR:Q57410 (EMBL:U39068, U15606) Vibrio cholerae accessory colonization factor ACFC (253 aa), fasta scores; opt: 430 z-score: 499.2 E(): 1.7e-20, 33.9% identity in 248 aa overlap, and TR:P96759 (EMBL:U82533)E. coli O45 ANM protein (251 aa), fasta scores; opt: 417 z-score: 484.4 E(): 1.1e-19, 33.2% identity in 247 aa overlap. Contains N-terminal signal sequence; Updated (2006) note: [...] ","uniprotId":"Q0PAB2_CAMJE"},"10641598":{"id":10641598,"externalId":"192222.Cj0779","name":"tpx","hasFamily":true,"annotation":"Thiol-specific peroxidase that catalyzes the reduction of hydrogen peroxide and organic hydroperoxides to water and alcohols, respectively. Plays a role in cell protection against oxidative stress by detoxifying peroxides","uniprotId":"TPX_CAMJE"},"10641599":{"id":10641599,"externalId":"192222.Cj0780","name":"napA","hasFamily":true,"annotation":"Catalytic subunit of the periplasmic nitrate reductase complex NapAB. Receives electrons from NapB and catalyzes the reduction of nitrate to nitrite","uniprotId":"NAPA_CAMJE"},"10641600":{"id":10641600,"externalId":"192222.Cj0781","name":"napG","hasFamily":true,"annotation":"Putative ferredoxin; Original (2000) note: Cj0781, napG, probable ferredoxin, len: 246 aa; similar to e.g. NAPG_ECOLI ferredoxin-type protein NAPG (231 aa), fasta scores; opt: 763 z-score: 884.9 E(): 0, 48.2% identity in 228 aa overlap, and MAUM_METEX methylamine utilization ferredoxin-type protein (220 aa), fasta scores; opt: 635 z-score: 738.5 E(): 0, 45.7% identity in 210 aa overlap. No Hp match. Contains PS00198 4Fe-4S ferredoxins,iron-sulfur binding region signature, and 3x Pfam match to entry PF00037 fer4, 4Fe-4S ferredoxins and related iron-sulfur cluster binding domains; Update [...] ","uniprotId":"Q0PAA9_CAMJE"},"10641601":{"id":10641601,"externalId":"192222.Cj0782","name":"napH","hasFamily":true,"annotation":"Putative ferredoxin; Original (2000) note: Cj0782, napH, probable ferredoxin, len: 260 aa; similar to e.g. NAPH_ECOLI ferredoxin-type protein NAPH (287 aa), fasta scores; opt: 588 z-score: 663.2 E(): 1.2e-29, 35.3% identity in 255 aa overlap, and MAUN_PARDE methylamine utilization ferredoxin-type protein (283 aa), fasta scores; opt: 596 z-score: 672.2 E(): 3.8e-30, 37.8% identity in 251 aa overlap. No Hp ortholog. Contains PS00198 4Fe-4S ferredoxins, iron-sulfur binding region signature, and Pfam match to entry PF00037 fer4, 4Fe-4S ferredoxins and related iron-sulfur cluster binding do [...] ","uniprotId":"Q0PAA8_CAMJE"},"10641602":{"id":10641602,"externalId":"192222.Cj0783","name":"napB","hasFamily":true,"annotation":"Electron transfer subunit of the periplasmic nitrate reductase complex NapAB","uniprotId":"Q0PAA7_CAMJE"},"10641603":{"id":10641603,"externalId":"192222.Cj0784","name":"napL","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0784, probable periplasmic protein, len: 304 aa; no Hp match. Contains N-terminal signal sequence; Updated (2006) note: Some characterisation work within Wolinella succinogenes with acceptable identity score. Functional classification - Miscellaneous periplasmic proteins; PMID:12823811","uniprotId":"Q0PAA6_CAMJE"},"10641604":{"id":10641604,"externalId":"192222.Cj0785","name":"napD","hasFamily":true,"annotation":"Chaperone for NapA, the catalytic subunit of the periplasmic nitrate reductase. It binds directly and specifically to the twin- arginine signal peptide of NapA, preventing premature interaction with the Tat translocase and premature export","uniprotId":"Q0PAA5_CAMJE"},"10641605":{"id":10641605,"externalId":"192222.Cj0786","name":"Cj0786","hasFamily":true,"annotation":"Original (2000) note: Cj0786, small hydrophobic protein, len: 57 aa; no Hp match; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Functional classification -Unknown","uniprotId":"Q0PAA4_CAMJE"},"10641606":{"id":10641606,"externalId":"192222.Cj0787","name":"Cj0787","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0787, unknown, len: 82 aa; 38.1% identity to HP0168; Updated (2006) note: Prosite domain PS50293 TPR_REGION, Tetratricopeptide region identified within CDS. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PAA3_CAMJE"},"10641607":{"id":10641607,"externalId":"192222.Cj0788","name":"Cj0788","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0788, unknown, len: 163 aa; 30.8% identity to HP0167. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PAA2_CAMJE"},"10641608":{"id":10641608,"externalId":"192222.Cj0789","name":"cca","hasFamily":true,"annotation":"Belongs to the tRNA nucleotidyltransferase/poly(A) polymerase family","uniprotId":"Q0PAA1_CAMJE"},"10641609":{"id":10641609,"externalId":"192222.Cj0790","name":"purU","hasFamily":true,"annotation":"Catalyzes the hydrolysis of 10-formyltetrahydrofolate (formyl-FH4) to formate and tetrahydrofolate (FH4)","uniprotId":"Q0PAA0_CAMJE"},"10641610":{"id":10641610,"externalId":"192222.Cj0791c","name":"Cj0791c","hasFamily":true,"annotation":"Putative aminotransferase; Original (2000) note: Cj0791c, probable aminotransferase, len: 424 aa; similar to many members of the nifS family e.g. TR:O32164 (EMBL:Z99120) B. subtilis YURW protein (406 aa), fasta scores; opt: 289 z-score: 340.6 E(): 1.1e-11, 23.9% identity in 431 aa overlap. 45.4% identity to HP0405. Contains PS00455 Putative AMP-binding domain signature and PS00595 Aminotransferases class-V pyridoxal-phosphate attachment site; Updated (2006) note: No specific characterisation with acceptable identity score carried out yet. Putative kept within product function. Function [...] ","uniprotId":"Q0PA99_CAMJE"},"10641611":{"id":10641611,"externalId":"192222.Cj0792","name":"Cj0792","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0792, unknown, len: 292 aa; some similarity to hypothetical proteins e.g. YZ04_METJA hypothetical protein MJECL04 (439 aa), fasta scores; opt: 175 z-score: 208.0 E(): 0.00027, 26.5% identity in 309 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF03008 Archaea bacterial proteins of unknown function identified within CDS. Conserved added to product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PA98_CAMJE"},"10641612":{"id":10641612,"externalId":"192222.Cj0793","name":"flgS","hasFamily":true,"annotation":"Original (2000) note: Cj0793, probable signal transduction histidine kinase, len: 339 aa; similar to e.g. the C-terminus of ATOS_ECOLI sensor protein ATOS (608 aa), fasta scores; opt: 357 z-score: 408.3 E(): 1.9e-15,33.3% identity in 222 aa overlap. 42.4% identity to HP0244. Contains Pfam match to entry PF00512 signal,Signal carboxyl-terminal domain; Updated (2006) note: Characterised within Campylobacter jejuni and Helicobacter pylori. Putative not added to product function. Literature search identified papers linking Cj0793 and Cj1024 to a two-component signal transduction system (Fl [...] ","uniprotId":"Q0PA97_CAMJE"},"10641613":{"id":10641613,"externalId":"192222.Cj0794","name":"Cj0794","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0794, unknown, len: 426 aa; no Hp match. Has consecutive regions of near-identity to Cj0814 (73.9% identity in 253 aa overlap), Cj0815 (61.6% identity in 86 aa overlap) and Cj0816 (63.6% identity in 44 aa overlap), which may represent a pseudogene copy of this CDS; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PA96_CAMJE"},"10641614":{"id":10641614,"externalId":"192222.Cj0795c","name":"murF","hasFamily":true,"annotation":"UDP-N-acetylmuramoyl-tripeptide D-alanyl-D-alanine ligase; Original (2000) note: Cj0795c, murF, possible UDP-N-acetylmuramoylalanyl-D-glutamyl-2,6-diaminopimelate -- ligase, len: 475 aa; some similarity to e.g. MURF_BACSU UDP-N-acetylmuramoylalanyl-D-glutamyl-2,6-diaminopimelate -- ligase (EC 6.3.2.15) (457 aa), fasta scores; opt: 235 z-score: 258.7 E(): 4.1e-07, 26.7% identity in 348 aa overlap. 41.5% identity to HP0740; Updated (2006) note: Pfam domains PF01225 Mur ligase family, catalytic domain and PF02875 Mur ligase family, glutamate ligase domain were identified within CDS. Also, [...] ","uniprotId":"Q0PA95_CAMJE"},"10641615":{"id":10641615,"externalId":"192222.Cj0796c","name":"Cj0796c","hasFamily":true,"annotation":"Putative hydrolase; Original (2000) note: Cj0796c, unknown, len: 239 aa; 46.3% identity to HP0739 (annotated as 2-hydroxy-6-oxohepta-2,4-dienoate hydrolase); Updated (2006) note: Pfam domain PF00561 alpha/beta hydrolase fold and Prosite domain PS50187 Esterase/lipase/thioesterase active site serine identified within CDS. The latter are found in alpha/beta fold hydrolase. Product function modified due to motif match. No specific characterisation with acceptable identity score carried out yet. Putative kept within product function. Functional classification - Misc","uniprotId":"Q0PA94_CAMJE"},"10641616":{"id":10641616,"externalId":"192222.Cj0797c","name":"Cj0797c","hasFamily":true,"annotation":"Antitoxin component of a type II toxin-antitoxin (TA) system. ","uniprotId":"Q0PA93_CAMJE"},"10641617":{"id":10641617,"externalId":"192222.Cj0798c","name":"ddlA","hasFamily":true,"annotation":"Cell wall formation","uniprotId":"DDL_CAMJE"},"10641618":{"id":10641618,"externalId":"192222.Cj0799c","name":"ruvA","hasFamily":true,"annotation":"The RuvA-RuvB complex in the presence of ATP renatures cruciform structure in supercoiled DNA with palindromic sequence, indicating that it may promote strand exchange reactions in homologous recombination. RuvAB is a helicase that mediates the Holliday junction migration by localized denaturation and reannealing. RuvA stimulates, in the presence of DNA, the weak ATPase activity of RuvB. ","uniprotId":"RUVA_CAMJE"},"10641619":{"id":10641619,"externalId":"192222.Cj0800c","name":"Cj0800c","hasFamily":true,"annotation":"Putative ATPase; Original (2000) note: Cj0800c, unknown, len: 619 aa; 24.7% identity to HP0884. Also similar to Cj0044c (28.6% identity in 622 aa overlap); Updated (2006) note: Prosite domain PS00389 ATPase,F1 complex, OSCP/delta subunit signature identified within CDS. Product modified to more specific family member due to motif match. Not fully characterised so putative kept within product function. Functional classification - Misc","uniprotId":"Q0PA90_CAMJE"},"10641620":{"id":10641620,"externalId":"192222.Cj0801","name":"murJ","hasFamily":true,"annotation":"Involved in peptidoglycan biosynthesis. Transports lipid- linked peptidoglycan precursors from the inner to the outer leaflet of the cytoplasmic membrane","uniprotId":"Q0PA89_CAMJE"},"10641621":{"id":10641621,"externalId":"192222.Cj0802","name":"cysS","hasFamily":true,"annotation":"Belongs to the class-I aminoacyl-tRNA synthetase family. ","uniprotId":"SYC_CAMJE"},"10641622":{"id":10641622,"externalId":"192222.Cj0803","name":"msbA","hasFamily":true,"annotation":"Original (2000) note: Cj0803, msbA, probable lipid export ABC transport protein, len: 580 aa; similar to lipid export protein MSBA_ECOLI probable transport ATP-binding protein MSBA (582 aa), fasta scores; opt: 1032 z-score: 1135.9 E(): 0, 29.6% identity in 575 aa overlap,and to other members of the mutidrug-resistance family of ABC transporters. 42.7% identity to HP1082. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, Pfam match to entry PF00664 ABC_membrane, ABC transporter transmembrane region, and Pfam match to entry PF00005 ABC_tra [...] ","uniprotId":"Q0PA87_CAMJE"},"10641623":{"id":10641623,"externalId":"192222.Cj0804","name":"pyrD","hasFamily":true,"annotation":"Catalyzes the conversion of dihydroorotate to orotate with quinone as electron acceptor","uniprotId":"Q0PA86_CAMJE"},"10641624":{"id":10641624,"externalId":"192222.Cj0805","name":"Cj0805","hasFamily":true,"annotation":"Belongs to the peptidase M16 family. ","uniprotId":"Q0PA85_CAMJE"},"10641625":{"id":10641625,"externalId":"192222.Cj0806","name":"Cj0806","hasFamily":true,"annotation":"Catalyzes the condensation of (S)-aspartate-beta-semialdehyde [(S)-ASA] and pyruvate to 4-hydroxy-tetrahydrodipicolinate (HTPA). ","uniprotId":"DAPA_CAMJE"},"10641626":{"id":10641626,"externalId":"192222.Cj0807","name":"Cj0807","hasFamily":true,"annotation":"Putative oxidoreductase; Updated (2006) note: Similarity to more than one genus. Putative kept within product function; Original (2000) note: Cj0807, probable oxidoreductase, len: 259 aa; similar to members of the short-chain dehydrogenase/reductase family e.g. SOU1_CANAL sorbitol utilization protein SOU1 (281 aa), fasta scores; opt: 422 z-score: 477.6 E(): 2.6e-19, 32.9% identity in 258 aa overlap, and HDHA_ECOLI 7-alpha-hydroxysteroid dehydrogenase (255 aa), fasta scores; opt: 377 z-score: 428.4 E(): 1.4e-16, 34.3% identity in 254 aa overlap. 65.3% identity to HP1014. Contains Pfam m [...] ","uniprotId":"Q0PA83_CAMJE"},"10641627":{"id":10641627,"externalId":"192222.Cj0808c","name":"Cj0808c","hasFamily":true,"annotation":"Original (2000) note: Cj0808c, small hydrophobic protein, len: 101 aa; no Hp match; Updated (2006) note: Three probable transmembrane helices predicted by TMHMM2.0. Functional classification -Unknown","uniprotId":"Q0PA82_CAMJE"},"10641628":{"id":10641628,"externalId":"192222.Cj0809c","name":"Cj0809c","hasFamily":true,"annotation":"Putative hydrolase; Original (2000) note: Cj0809c, possible hydrolase,len: 198 aa; similar to many hypothetical proteins belonging to the glyoxalase II family e.g. YQGX_BACSU (211 aa), fasta scores; opt: 289 z-score: 365.7 E(): 4.5e-13,32.8% identity in 204 aa overlap. 44.3% identity to HP0813. Contains Pfam match to entry PF00753 lactamase_B,Metallo-beta-lactamase superfamily; Updated (2006) note: No specific characterisation with acceptable identity scores carried out yet. Putative kept within product function. Functional classification -Misc","uniprotId":"Q0PA81_CAMJE"},"10641629":{"id":10641629,"externalId":"192222.Cj0810","name":"nadE","hasFamily":true,"annotation":"Catalyzes the ATP-dependent amidation of deamido-NAD to form NAD. Uses ammonia as a nitrogen source","uniprotId":"NADE_CAMJE"},"10641630":{"id":10641630,"externalId":"192222.Cj0811","name":"lpxK","hasFamily":true,"annotation":"Transfers the gamma-phosphate of ATP to the 4'-position of a tetraacyldisaccharide 1-phosphate intermediate (termed DS-1-P) to form tetraacyldisaccharide 1,4'-bis-phosphate (lipid IVA). ","uniprotId":"LPXK_CAMJE"},"10641631":{"id":10641631,"externalId":"192222.Cj0812","name":"thrC","hasFamily":true,"annotation":"Threonine synthase. Functional classification-Amino acid biosynthesis-Aspartate family; Updated (2006) note: Characterised within Pseudomonas aeruginosa with acceptable identity score. Appropriate motifs present. Putative not added to product function; Original (2000) note: Cj0812, thrC, probable threonine synthase, len: 470 aa; similar to e.g. THRC_YEAST threonine synthase (EC 4.2.99.2) (514 aa),fasta scores; opt: 595 z-score: 671.2 E(): 4.3e-30, 33.6% identity in 464 aa overlap, and THRC_PSEAE (470 aa), fasta scores; opt: 647 z-score: 730.2 E(): 0, 32.3% identity in 434 aa overlap. 4 [...] ","uniprotId":"Q0PA78_CAMJE"},"10641632":{"id":10641632,"externalId":"192222.Cj0813","name":"kdsB","hasFamily":true,"annotation":"Belongs to the KdsB family. ","uniprotId":"Q0PA77_CAMJE"},"10641633":{"id":10641633,"externalId":"192222.Cj0814","name":"Cj0814","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0814, unknown, len: 251 aa; no Hp match. Almost identical to the N-teminus of Cj0794; similarity continues in Cj0815 and Cj0816: may represent a pseudogene fragment; Updated (2006) note: Similar to hypothetical proteins from other bacteria","uniprotId":"Q0PA76_CAMJE"},"10641634":{"id":10641634,"externalId":"192222.Cj0815","name":"Cj0815","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0815, unknown, len: 114 aa; no Hp match. Almost identical to the centre of Cj0794; similarity continues from Cj0814 and into Cj0816: may represent a pseudogene fragment. Functional classification - Unknown","uniprotId":"Q0PA75_CAMJE"},"10641635":{"id":10641635,"externalId":"192222.Cj0816","name":"Cj0816","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0816, unknown, len: 56 aa; no Hp match. Almost identical to the C-terminus of Cj0794; similarity continues from Cj0814 and Cj0815: may represent a pseudogene fragment. Functional classification -Unknown","uniprotId":"Q0PA74_CAMJE"},"10641636":{"id":10641636,"externalId":"192222.Cj0817","name":"glnH","hasFamily":true,"annotation":"Original (2000) note: Cj0817, glnH, probable glutamine-binding periplasmic protein, len: 256 aa; similar to e.g. GLNH_ECOLI glutamine-binding periplasmic protein precursor (248 aa), fasta scores; opt: 382 z-score: 441.9 E(): 2.6e-17, 32.5% identity in 243 aa overlap. 29.0% identity to HP0940. Contains Pfam match to entry PF00497 SBP_bac_3, Bacterial extracellular solute-binding proteins, family 3; Updated (2006) note: Characterised with marginal identity scores in more than one bacteria. Prosite domain PS50121, Solute binding protein/glutamate receptor domain identified within CDS. Fur [...] ","uniprotId":"Q0PA73_CAMJE"},"10641637":{"id":10641637,"externalId":"192222.Cj0818","name":"Cj0818","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0818, probable lipoprotein,len: 75 aa; no Hp match. Contains N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PA72_CAMJE"},"10641638":{"id":10641638,"externalId":"192222.Cj0819","name":"Cj0819","hasFamily":true,"annotation":"Original (2000) note: Cj0819, small hydrophobic protein, len 74 aa; no Hp match; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Functional classification -Unknown","uniprotId":"Q0PA71_CAMJE"},"10641639":{"id":10641639,"externalId":"192222.Cj0820c","name":"fliP","hasFamily":true,"annotation":"Plays a role in the flagellum-specific transport system. ","uniprotId":"Q0PA70_CAMJE"},"10641640":{"id":10641640,"externalId":"192222.Cj0821","name":"glmU","hasFamily":true,"annotation":"Catalyzes the last two sequential reactions in the de novo biosynthetic pathway for UDP-N-acetylglucosamine (UDP-GlcNAc). The C- terminal domain catalyzes the transfer of acetyl group from acetyl coenzyme A to glucosamine-1-phosphate (GlcN-1-P) to produce N- acetylglucosamine-1-phosphate (GlcNAc-1-P), which is converted into UDP-GlcNAc by the transfer of uridine 5-monophosphate (from uridine 5- triphosphate), a reaction catalyzed by the N-terminal domain. ","uniprotId":"GLMU_CAMJE"},"10641641":{"id":10641641,"externalId":"192222.Cj0822","name":"dfp","hasFamily":true,"annotation":"Catalyzes two steps in the biosynthesis of coenzyme A. In the first step cysteine is conjugated to 4'-phosphopantothenate to form 4- phosphopantothenoylcysteine, in the latter compound is decarboxylated to form 4'-phosphopantotheine","uniprotId":"Q0PA68_CAMJE"},"10641642":{"id":10641642,"externalId":"192222.Cj0823","name":"Cj0823","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0823, unknown, len: 205 aa; 24.3% identity to HP0842; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Functional classification -Unknown","uniprotId":"Q0PA67_CAMJE"},"10641643":{"id":10641643,"externalId":"192222.Cj0824","name":"uppS","hasFamily":true,"annotation":"Catalyzes the condensation of isopentenyl diphosphate (IPP) with allylic pyrophosphates generating different type of terpenoids. ","uniprotId":"ISPT_CAMJE"},"10641644":{"id":10641644,"externalId":"192222.Cj0825","name":"Cj0825","hasFamily":true,"annotation":"Putative processing peptidase; Original (2000) note: Cj0825, possible processing peptidase, len: 258 aa; similar to e.g. LEP3_VIBCH type 4 prepilin-like protein specific leader peptidase (253 aa),fasta scores; opt: 261 z-score: 323.9 E(): 9.7e-11, 28.5% identity in 260 aa overlap. No Hp match. Contains several probable membrane spanning domains; Updated (2006) note: Pfam domains PF06750 Bacterial Peptidase A24 N-terminal domain and PF01478 Type IV leader peptidase family were identified within CDS. Further support given to product function. No specific characterisation with acceptable  [...] ","uniprotId":"Q0PA65_CAMJE"},"10641645":{"id":10641645,"externalId":"192222.Cj0826","name":"Cj0826","hasFamily":true,"annotation":"Original (2000) note: Cj0826, probable integral membrane protein, len: 341 aa; similar to hypothetical membrane proteins e.g. YJGP_HAEIN (372 aa), fasta scores; opt: 203 z-score: 236.9 E(): 6.8e-06, 21.7% identity in 314 aa overlap. 32.3% identity to HP0362; Updated (2006) note: Pfam domain PF03739 Predicted permease YjgP/YjgQ family identified within CDS. Members of this family are predicted integral membrane proteins of unknown function. Also, six probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PA64_CAMJE"},"10641646":{"id":10641646,"externalId":"192222.Cj0827","name":"truA","hasFamily":true,"annotation":"Formation of pseudouridine at positions 38, 39 and 40 in the anticodon stem and loop of transfer RNAs","uniprotId":"TRUA_CAMJE"},"10641647":{"id":10641647,"externalId":"192222.Cj0828c","name":"ilvA","hasFamily":true,"annotation":"Original (2000) note: Cj0828c, ilvA, probable threonine dehydratase biosynthetic, len: 403 aa; similar to e.g. THD1_BACSU threonine dehydratase biosynthetic (EC 4.2.1.16) (422 aa), fasta scores; opt: 808 z-score: 906.6 E(): 0, 35.6% identity in 407 aa overlap. No Hp match. Contains PS00165 Serine/threonine dehydratases pyridoxal-phosphate attachment site, and Pfam match to entry PF00291 S_T_dehydratase, Pyridoxal-phosphate dependant enzymes; Updated (2006) note: Characterised in Caulobacter crescentus and Bacillus subtilis with acceptable identity scores. Putative not added to product  [...] ","uniprotId":"Q0PA62_CAMJE"},"10641648":{"id":10641648,"externalId":"192222.Cj0829c","name":"Cj0829c","hasFamily":true,"annotation":"Putative CoA binding domain containing protein; Original (2000) note: Cj0829c, unknown, len: 137 aa; similar to hypothetical proteins e.g. TR:P75874 (EMBL:AE000198) E. coli b0965 (164 aa), fasta scores; opt: 240 z-score: 312.1 E(): 4.4e-10, 30.3% identity in 132 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02629 CoA binding domain identified within CDS. Product modified to new family member based on motif match. No specific characterisation has been carried out yet. Putative kept within product function. Functional classification - Misc","uniprotId":"Q0PA61_CAMJE"},"10641649":{"id":10641649,"externalId":"192222.Cj0830","name":"Cj0830","hasFamily":true,"annotation":"Original (2000) note: Cj0830, probable integral membrane protein, len: 138 aa; 42.7% identity to HP1502; Updated (2006) note: Pfam domain PF04323 Protein of unknown function (DUF474) identified within CDS. Also,four probable transmembrane helices predicted by TMHMM2.0. Literature search identified paper giving potential clues to product function. Functional classification -Membranes, lipoproteins and porins; PMID:16045618","uniprotId":"Q0PA60_CAMJE"},"10641650":{"id":10641650,"externalId":"192222.Cj0831c","name":"trmA","hasFamily":true,"annotation":"Dual-specificity methyltransferase that catalyzes the formation of 5-methyluridine at position 54 (m5U54) in all tRNAs, and that of position 341 (m5U341) in tmRNA (transfer-mRNA). ","uniprotId":"TRMA_CAMJE"},"10641651":{"id":10641651,"externalId":"192222.Cj0832c","name":"Cj0832c","hasFamily":true,"annotation":"Putative Na+/H+ antiporter family protein; Original (2000) note: Cj0832c, probable integral membrane protein, len: 577 aa; similar to hypothetical proteins e.g. TR:O59530 (EMBL:AP000007) Pyrococcus horikoshii PH1889 (533 aa), fasta scores; opt: 628 z-score: 744.4 E(): 0, 39.2% identity in 558 aa overlap. 26.9% identity to HP0946; Updated (2006) note: Pfam domain PF03553 Na+/H+ antiporter family identified within CDS. Also, thirteen probable transmembrane helices predicted by TMHMM2. Product modified to more specific family member based on motif match. No specific characterisation carri [...] ","uniprotId":"Q0PA58_CAMJE"},"10641652":{"id":10641652,"externalId":"192222.Cj0833c","name":"Cj0833c","hasFamily":true,"annotation":"Belongs to the short-chain dehydrogenases/reductases (SDR) family","uniprotId":"Q0PA57_CAMJE"},"10641653":{"id":10641653,"externalId":"192222.Cj0834c","name":"Cj0834c","hasFamily":true,"annotation":"Original (2000) note: Cj0834c, possible periplasmic protein, len: 412 aa; similar in C-terminus to ankyrin repeat-containing proteins e.g. ANKH_CHRVI ankyrin homolog precursor (323 aa), fasta scores; opt: 194 z-score: 229.9 E(): 1.7e-05, 28.9% identity in 194 aa overlap. No Hp match. Some similarity to ankyrin repeats in Cj1386 (29.3% identity in 99 aa overlap). Contains 2x Pfam match to entry PF00023 ank, Ank repeat, and possible N-terminal signal sequence; Updated (2006) note: No specific characterisation with acceptable identity scores carried out yet. Putative kept within product f [...] ","uniprotId":"Q0PA56_CAMJE"},"10641654":{"id":10641654,"externalId":"192222.Cj0835c","name":"acnB","hasFamily":true,"annotation":"Belongs to the aconitase/IPM isomerase family. ","uniprotId":"Q0PA55_CAMJE"},"10641655":{"id":10641655,"externalId":"192222.Cj0836","name":"ogt","hasFamily":true,"annotation":"Involved in the cellular defense against the biological effects of O6-methylguanine (O6-MeG) and O4-methylthymine (O4-MeT) in DNA. Repairs the methylated nucleobase in DNA by stoichiometrically transferring the methyl group to a cysteine residue in the enzyme. This is a suicide reaction: the enzyme is irreversibly inactivated. ","uniprotId":"Q0PA54_CAMJE"},"10641656":{"id":10641656,"externalId":"192222.Cj0837c","name":"Cj0837c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0837c, unknown, len: 313 aa; no Hp match. Contains PS00179 Aminoacyl-transfer RNA synthetases class-II signature 1, and PS00030 Eukaryotic putative RNA-binding region RNP-1 signature. Functional classification - Unknown","uniprotId":"Q0PA53_CAMJE"},"10641657":{"id":10641657,"externalId":"192222.Cj0838c","name":"metS","hasFamily":true,"annotation":"Is required not only for elongation of protein synthesis but also for the initiation of all mRNA translation through initiator tRNA(fMet) aminoacylation","uniprotId":"SYM_CAMJE"},"10641658":{"id":10641658,"externalId":"192222.Cj0839c","name":"Cj0839c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0839c, unknown, len: 65 aa; 43.1% identity to HP1384. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PA51_CAMJE"},"10641659":{"id":10641659,"externalId":"192222.Cj0840c","name":"fbp","hasFamily":true,"annotation":"Fructose-1,6-bisphosphatase class 1; Original (2000) note: Cj0840c, fbp, probable fructose-1,6-bisphosphatase, len: 280 aa; similar to many e.g. F16Q_SACHY fructose-1,6-bisphosphatase, cytosolic (EC 3.1.3.11) (331 aa), fasta scores; opt: 376 z-score: 455.9 E(): 4.3e-18, 37.5% identity in 272 aa overlap. 39.9% identity to HP1385. Contains 2x Pfam match to entry PF00316 FBPase, Fructose-1-6-bisphosphatase; Updated (2006) note: Characterisation work carried out in Escherichia coli and Bacillus subtilis with marginal identity scores. Appropriate motifs present. Putative removed form produc [...] ","uniprotId":"F16PA_CAMJE"},"10641660":{"id":10641660,"externalId":"192222.Cj0841c","name":"mobB","hasFamily":true,"annotation":"Putative molybdopterin-guanine dinucleotide biosynthesis protein; Original (2000) note: Cj0841c, probable ATP/GTP binding protein, len: 163 aa; some similarity to e.g. MOBB_ECOLI molybdopterin-guanine dinucleotide biosynthesis protein B (174 aa), fasta scores; opt: 148 z-score: 194.1 E(): 0.0016, 25.6% identity in 168 aa overlap. No Hp match. Contains S00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Pfam domain PF03205 Molybdopterin guanine dinucleotide synthesis protein B identified within CDS. Further support for product function. Characterisation papers form Escher [...] ","uniprotId":"Q0PA49_CAMJE"},"10641661":{"id":10641661,"externalId":"192222.Cj0842","name":"Cj0842","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0842, probable lipoprotein,len: 161 aa; no Hp match. Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PA48_CAMJE"},"10641662":{"id":10641662,"externalId":"192222.Cj0843c","name":"Cj0843c","hasFamily":true,"annotation":"Original (2000) note: Cj0843c, probable secreted transglycosylase, len: 541 aa; some simialarity in C-terminus to SLT_ECOLI soluble lytic murein transglycosylase precursor (645 aa), fasta scores; opt: 257 z-score: 285.7 E(): 1.3e-08, 29.3% identity in 208 aa overlap. 31.4% identity to HP0645. Contains PS00922 Prokaryotic transglycosylases signature and N-terminal signal sequence; Updated (2006) note: Pfam domain PF01464 Transglycosylase SLT domain identified within CDS. Further support given to product function. Putative kept within product function as identity scores were marginal and [...] ","uniprotId":"Q0PA47_CAMJE"},"10641663":{"id":10641663,"externalId":"192222.Cj0844c","name":"Cj0844c","hasFamily":true,"annotation":"Original (2000) note: Cj0844c, possible integral membrane protein, len: 93 aa; 38.4% identity to HP0644; Updated (2006) note: Pfam domain PF02325 YGGT family identified within CDS. This family consists of a repeat found in conserved hypothetical integral membrane proteins. The function of this region and the proteins which possess it is unknown. Also, two probable transmembrane helices predicted by TMHMM2.0. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PA46_CAMJE"},"10641664":{"id":10641664,"externalId":"192222.Cj0845c","name":"gltX","hasFamily":true,"annotation":"Catalyzes the attachment of glutamate to tRNA(Glu) in a two- step reaction: glutamate is first activated by ATP to form Glu-AMP and then transferred to the acceptor end of tRNA(Glu)","uniprotId":"SYE1_CAMJE"},"10641665":{"id":10641665,"externalId":"192222.Cj0846","name":"Cj0846","hasFamily":true,"annotation":"Uncharacterized metallophosphoesterase Cj0846; Original (2000) note: Cj0846, probable integral membrane protein, len: 374 aa; contains several probable membrane spanning regions in N-terminal half, C-terminal half is simalar to several hypothetical proteins e.g. YPBG_BACSU (259 aa), fasta scores; opt: 197 z-score: 233.7 E(): 1e-05, 28.6% identity in 238 aa overlap. 38.8% identity to HP1044; Updated (2006) note: Pfam domain PF00149 Calcineurin-like phosphoesterase and Prosite domain PS00142 Peptidase M, neutral zinc metallopeptidases,zinc-binding site identified within CDS. Also, four p [...] ","uniprotId":"Y846_CAMJE"},"10641666":{"id":10641666,"externalId":"192222.Cj0847","name":"psd","hasFamily":true,"annotation":"Catalyzes the formation of phosphatidylethanolamine (PtdEtn) from phosphatidylserine (PtdSer)","uniprotId":"PSD_CAMJE"},"10641667":{"id":10641667,"externalId":"192222.Cj0848c","name":"Cj0848c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0848c, unknown, len: 93 aa; similar to hypothetical proteins e.g. TR:O34867 (EMBL:AJ000975) B. subtilis YLQH protein (93 aa), fasta scores; opt: 284 z-score: 365.6 E(): 4.5e-13, 47.3% identity in 91 aa overlap. Also similar to C-terminus of flhB possible flagellar transport proteins e.g. FLHB_ECOLI flagellar biosynthetic protein FLHB (382 aa), fasta scores; opt: 182 z-score: 232.8 E(): 1.1e-05, 41.6% identity in 77 aa overlap. 39.2% identity to HP1575 (annotated as flhB). Also similar to C-terminus of Cj0335,flhB (40.7% identity in 81 aa [...] ","uniprotId":"Q0PA42_CAMJE"},"10641668":{"id":10641668,"externalId":"192222.Cj0849c","name":"Cj0849c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0849c, unknown, len: 719 aa; no Hp ortholog, some similarity to Cj0041 (23.1% identity in 566 aa overlap). Contain Asn-rich region near centre; Updated (2006) note: Prosite domain PS00063 Aldo/keto reductase family putative active site signature found within CDS. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0PA41_CAMJE"},"10641669":{"id":10641669,"externalId":"192222.Cj0850c","name":"Cj0850c","hasFamily":true,"annotation":"Putative MFS (Major Facilitator Superfamily) transport protein; Original (2000) note: Cj0850c, transmembrane transport protein, len: 395 aa; similar to hypothetical transport proteins e.g. YCAD_ECOLI (382 aa), fasta scores; opt: 431 z-score: 481.6 E(): 1.6e-19, 27.6% identity in 366 aa overlap. No Hp ortholog. Contains S00216 Sugar transport proteins signature 1, and Pfam match to entry PF00083 sugar_tr, Sugar (and other) transporters; Updated (2006) note: Pfam domain PF07690 Major Facilitator Superfamily identified within CDS. Also,twelve probable transmembrane helices predicted by TM [...] ","uniprotId":"Q0PA40_CAMJE"},"10641670":{"id":10641670,"externalId":"192222.Cj0851c","name":"Cj0851c","hasFamily":true,"annotation":"Original (2000) note: Cj0851c, probable integral membrane protein, len: 167 aa; some similarity to YCIB_BUCAP putative integral membrane protein from Buchnera aphidicola (177 aa), fasta scores; opt: 139 z-score: 180.5 E(): 0.0093, 30.0% identity in 160 aa overlap. No Hp match; Updated (2006) note: Four probable transmembrane helices predicted by TMHMM2.0. Paper identified giving further clues to product function. Functional classification - Membranes, lipoproteins and porins; PMID:16045618","uniprotId":"Q0PA39_CAMJE"},"10641671":{"id":10641671,"externalId":"192222.Cj0852c","name":"Cj0852c","hasFamily":true,"annotation":"Original (2000) note: Cj0852c, possible integral membrane protein, len: 109 aa; some similarity to hypothetical proteins e.g. TR:O27758 (EMBL:AE000929) Methanobacterium thermoautotrophicum MTH1725 (92 aa),fasta scores; opt: 123 z-score: 165.5 E(): 0.064, 32.7% identity in 98 aa overlap. 55.0% identity to HP0307; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0PA38_CAMJE"},"10641672":{"id":10641672,"externalId":"192222.Cj0853c","name":"hemL","hasFamily":true,"annotation":"Original (2000) note: Cj0853c, hemL, probable glutamate-1-semialdehyde 2,1-aminomutase, len: 424 aa; highly similar to many e.g. GSA_ECOLI glutamate-1-semialdehyde 2,1-aminomutase (EC 5.4.3.8) (426 aa), fasta scores; opt: 1377 z-score: 1553.3 E(): 0, 49.9% identity in 417 aa overlap. 59.8% identity to HP0306 Contains PS00600 Aminotransferases class-III pyridoxal-phosphate attachment site, and Pfam match to entry PF00202 aminotran_3, Aminotransferases class-III pyridoxal-phosphate; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Putative not ad [...] ","uniprotId":"GSA_CAMJE"},"10641673":{"id":10641673,"externalId":"192222.Cj0854c","name":"Cj0854c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0854c, probable periplasmic protein, len: 115 aa; 37.9% identity to HP0236. Contains N-terminal signal sequence and PS00190 Cytochrome c family heme-binding site signature. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0PA36_CAMJE"},"10641674":{"id":10641674,"externalId":"192222.Cj0855","name":"folD","hasFamily":true,"annotation":"Catalyzes the oxidation of 5,10-methylenetetrahydrofolate to 5,10-methenyltetrahydrofolate and then the hydrolysis of 5,10- methenyltetrahydrofolate to 10-formyltetrahydrofolate. ","uniprotId":"FOLD_CAMJE"},"10641675":{"id":10641675,"externalId":"192222.Cj0856","name":"lepP","hasFamily":true,"annotation":"Belongs to the peptidase S26 family. ","uniprotId":"Q0PA34_CAMJE"},"10641676":{"id":10641676,"externalId":"192222.Cj0857c","name":"moeA","hasFamily":true,"annotation":"Catalyzes the insertion of molybdate into adenylated molybdopterin with the concomitant release of AMP. ","uniprotId":"Q0PA33_CAMJE"},"10641677":{"id":10641677,"externalId":"192222.Cj0858c","name":"murA","hasFamily":true,"annotation":"Cell wall formation. Adds enolpyruvyl to UDP-N- acetylglucosamine","uniprotId":"MURA_CAMJE"},"10641678":{"id":10641678,"externalId":"192222.Cj0859c","name":"Cj0859c","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0859c, unknown, len: 142 aa; no Hp match. Gln-rich in C-term; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PA31_CAMJE"},"10641679":{"id":10641679,"externalId":"192222.Cj0860","name":"Cj0860","hasFamily":true,"annotation":"Putative integral membrane protein; Updated (2006) note: Ten probable transmembrane helices predicted by TMHMM2.0; Original (2000) note: Cj0860, probable integral membrane protein, len: 290 aa; similar to hypothetical proteins e.g. YICL_ECOLI (307 aa), fasta scores; opt: 516 z-score: 611.4 E(): 9.3e-27, 32.0% identity in 284 aa overlap. No Hp match. Contains 2x Pfam match to entry PF00892 DUF6, Integral membrane protein. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0PA30_CAMJE"},"10641680":{"id":10641680,"externalId":"192222.Cj0861c","name":"pabA","hasFamily":true,"annotation":"Original (2000) note: Cj0861c, pabA, probable para-aminobenzoate synthase glutamine amidotransferase component II, len: 188 aa; simlar to e.g. PABA_BACSU para-aminobenzoate synthase glutamine amidotransferase component II (EC 4.1.3.-) (194 aa), fasta scores; opt: 534 z-score: 632.3 E(): 6.4e-28, 44.6% identity in 186 aa overlap, and PABA_ECOLI (187 aa), fasta scores; opt: 475 z-score: 564.2 E(): 4e-24, 36.8% identity in 185 aa overlap. 38.6% identity to HP1281 annotated as anthranilate synthase component II (trpD). Contains Pfam match to entry PF00117 GATase, Glutamine amidotransferase [...] ","uniprotId":"Q0PA29_CAMJE"},"10641681":{"id":10641681,"externalId":"192222.Cj0862c","name":"pabB","hasFamily":true,"annotation":"Original (2000) note: Cj0862c, pabB, probable para-aminobenzoate synthase component I, len: 594 aa; simlar to e.g. PABB_ECOLI para-aminobenzoate synthase component I (EC 4.1.3.-) (453 aa), fasta scores; opt: 541 z-score: 604.3 E(): 2.3e-26, 35.7% identity in 252 aa overlap. 39.3% identity to HP0293. Contains Pfam match to entry PF00425 chorismate_bind, chorismate binding enzyme; Updated (2006) note: Characterised in Escherichia coli with acceptable identity score. Putative not added to product function. Functional classification - Biosynthesis of cofactors, prosthetic groups and carrie [...] ","uniprotId":"Q0PA28_CAMJE"},"10641682":{"id":10641682,"externalId":"192222.Cj0863c","name":"xerD","hasFamily":true,"annotation":"Site-specific tyrosine recombinase, which acts by catalyzing the cutting and rejoining of the recombining DNA molecules. Binds to the complete atypical dif motif (difH) site and to both halves separately","uniprotId":"XERH_CAMJE"},"10641683":{"id":10641683,"externalId":"192222.Cj0864","name":"Cj0864","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0864, probable periplasmic protein, len: 119 aa; some similarity in C-terminus to C-terminus of thiol:disulfide interchange proteins (not including active site) e.g. TR:P97037 (EMBL:U32616) Klebsiella pneumoniae disulfide isomerase (222 aa), fasta scores; opt: 184 z-score: 240.3 E(): 4.4e-06, 36.9% identity in 84 aa overlap. No Hp match. Also similar to C-term of Cj0872 dsbA (57.3% identity in 82 aa overlap); Updated (2006) note: Interpro result (ipr012335) shows a thioredoxin fold within the CDS. Further support given to product fu [...] ","uniprotId":"Q0PA26_CAMJE"},"10641684":{"id":10641684,"externalId":"192222.Cj0865","name":"dsbB","hasFamily":true,"annotation":"Required for disulfide bond formation in some proteins. Part of a redox system composed of DsbI and DsbL that mediates formation of an essential disulfide bond in AssT (By similarity)","uniprotId":"DSBI_CAMJE"},"10641685":{"id":10641685,"externalId":"192222.Cj0872","name":"dsbA","hasFamily":true,"annotation":"Uncharacterized protein; HMMPfam hit to PF05935, Arylsulfotransferase (ASST), score 8.2e-179","uniprotId":"Q0PA24_CAMJE"},"10641686":{"id":10641686,"externalId":"192222.Cj0873c","name":"Cj0873c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0873c, unknown, len: 70 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0PA23_CAMJE"},"10641687":{"id":10641687,"externalId":"192222.Cj0874c","name":"Cj0874c","hasFamily":true,"annotation":"Putative cytochrome C; Original (2000) note: Cj0874c, possible cytochrome C, len: 186 aa; very weak similarity to many cytochrome C proteins e.g. CYC_APIME cytochrome C (107 aa), fasta scores; opt: 106 z-score: 137.1 E(): 2.5, 36.5% identity in 52 aa overlap. No Hp match. Contains 2x PS00190 Cytochrome c family heme-binding site signature. Also some similarity to Cj0037c possible cytochrome C (36.0% identity in 178 aa overlap); Updated (2006) note: No specific characterisation in related bacteria with acceptable identity score. Appropriate motifs present. Putative added to product func [...] ","uniprotId":"Q0PA22_CAMJE"},"10641688":{"id":10641688,"externalId":"192222.Cj0876c","name":"Cj0876c","hasFamily":false,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0876c, probable periplasmic protein, len: 96 aa; no Hp match. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PA21_CAMJE"},"10641689":{"id":10641689,"externalId":"192222.Cj0877c","name":"Cj0877c","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0877c, questionable ORF,len: 30 aa; no Hp match. Functional classification -Unknown","uniprotId":"Q0PA20_CAMJE"},"10641690":{"id":10641690,"externalId":"192222.Cj0878","name":"Cj0878","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0878, unknown, len: 48 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PA19_CAMJE"},"10641691":{"id":10641691,"externalId":"192222.Cj0879c","name":"Cj0879c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0879c, probable periplasmic protein, len: 210 aa; no Hp match. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PA18_CAMJE"},"10641692":{"id":10641692,"externalId":"192222.Cj0880c","name":"Cj0880c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0880c, unknown, len: 240 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0PA17_CAMJE"},"10641693":{"id":10641693,"externalId":"192222.Cj0881c","name":"Cj0881c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0881c, unknown, len: 348 aa; 40.6% identity to HP1042. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0PA16_CAMJE"},"10641694":{"id":10641694,"externalId":"192222.Cj0882c","name":"flhA","hasFamily":true,"annotation":"Required for formation of the rod structure of the flagellar apparatus. Together with FliI and FliH, may constitute the export apparatus of flagellin","uniprotId":"Q0PA15_CAMJE"},"10641695":{"id":10641695,"externalId":"192222.Cj0883c","name":"Cj0883c","hasFamily":true,"annotation":"Putative transcriptional regulator; Original (2000) note: Cj0883c, unknown, len: 136 aa; identical to TR:Q46090 (EMBL:U06951) Campylobacter jejuni 81-176 ORF3 in region upstream of the flhA gene. (136 aa), and similar to hypothetical proteins e.g. Y379_HAEIN (150 aa), fasta scores; opt: 174 z-score: 226.5 E(): 2.6e-05, 28.3% identity in 138 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02082 Transcriptional regulator identified within CDS. Product modified to more specific family member based on motif match. No specific characterisation carried out yet, so putative kept i [...] ","uniprotId":"Q0PA14_CAMJE"},"10641696":{"id":10641696,"externalId":"192222.Cj0884","name":"rpsO","hasFamily":true,"annotation":"One of the primary rRNA binding proteins, it binds directly to 16S rRNA where it helps nucleate assembly of the platform of the 30S subunit by binding and bridging several RNA helices of the 16S rRNA. ","uniprotId":"RS15_CAMJE"},"10641697":{"id":10641697,"externalId":"192222.Cj0886c","name":"ftsK","hasFamily":true,"annotation":"Essential cell division protein that coordinates cell division and chromosome segregation. The N-terminus is involved in assembly of the cell-division machinery. The C-terminus functions as a DNA motor that moves dsDNA in an ATP-dependent manner towards the dif recombination site, which is located within the replication terminus region. Translocation stops specifically at Xer-dif sites, where FtsK interacts with the Xer recombinase, allowing activation of chromosome unlinking by recombination. FtsK orienting polar sequences (KOPS) guide the direction of DNA translocation. FtsK can remo [...] ","uniprotId":"FTSK_CAMJE"},"10641698":{"id":10641698,"externalId":"192222.Cj0887c","name":"flaD","hasFamily":true,"annotation":"Flagellin is the subunit protein which polymerizes to form the filaments of bacterial flagella","uniprotId":"Q0PA11_CAMJE"},"10641699":{"id":10641699,"externalId":"192222.Cj0888c","name":"Cj0888c","hasFamily":true,"annotation":"Original (2000) note: Cj0888c, ABC transport system ATP-binding protein, len: 643 aa; similar to UUP_ECOLI ABC transporter atp-binding protein UUP (635 aa), fasta scores; opt: 744 z-score: 763.9 E(): 0, 37.1% identity in 634 aa overlap. No Hp ortholog. Contains 2x PS00017 ATP/GTP-binding site motif A (P-loop), 2x PS00211 ABC transporters family signature, and 2x Pfam match to entry PF00005 ABC_tran; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Putative not added to product function. Functional classification -Transport/binding proteins - Ot [...] ","uniprotId":"Q0PA10_CAMJE"},"10641700":{"id":10641700,"externalId":"192222.Cj0889c","name":"Cj0889c","hasFamily":true,"annotation":"Original (2000) note: Cj0889c, probable sensory transduction histidine kinase, len: 429 aa; similar in C-terminus to many e.g. ENVZ_ECOLI osmolarity sensor protein ENVZ (450 aa), fasta scores; opt: 210 z-score: 229.3 E(): 1.8e-05. 26.5% identity in 204 aa overlap No Hp ortholog. Contains two probable transmembrane domains in N-terminus and Pfam match to entry PF00512 signal, Signal carboxyl-terminal domain; Updated (2006) note: Pfam domain PF02518 Histidine kinase-, DNA gyrase B-, and HSP90-like ATPase identified within CDS. Further support given to product function. Some characterisat [...] ","uniprotId":"Q0PA09_CAMJE"},"10641701":{"id":10641701,"externalId":"192222.Cj0890c","name":"Cj0890c","hasFamily":true,"annotation":"Original (2000) note: Cj0890c, probable sensory transduction transcriptional regulator, len: 220 aa; similar to many e.g. PHOB_SHIFL phosphate regulon transcriptional regulator (229 aa), fasta scores; opt: 396 z-score: 471.5 E(): 5.8e-19, 29.3% identity in 222 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00072 response_reg, Response regulator receiver domain and Pfam match to entry PF00486 trans_reg_C,Transcriptional regulatory protein, C terminal; Updated (2006) note: Characterisation within more than one genus, however, identity scores were marginal. Putative kept withi [...] ","uniprotId":"Q0PA08_CAMJE"},"10641702":{"id":10641702,"externalId":"192222.Cj0891c","name":"serA","hasFamily":true,"annotation":"Belongs to the D-isomer specific 2-hydroxyacid dehydrogenase family","uniprotId":"Q0PA07_CAMJE"},"10641703":{"id":10641703,"externalId":"192222.Cj0892c","name":"Cj0892c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0892c, probable periplasmic protein, len: 162 aa; no Hp match. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0PA06_CAMJE"},"10641704":{"id":10641704,"externalId":"192222.Cj0893c","name":"rpsA","hasFamily":true,"annotation":"30S ribosomal protein S1; Original (2000) note: Cj0893c, rpsA, probable 30S ribosomal protein S1, len: 556 aa; similar to many e.g. RS1_ECOLI 30S ribosomal protein S1 (557 aa), fasta scores; opt: 1053 z-score: 1148.3 E(): 0, 33.4% identity in 548 aa overlap. 47.0% identity to HP0399. Contains 6x Pfam match to entry PF00575 S1, S1 RNA binding domain; Updated (2006) note: Characterised in Escherichia coli with marginal identity score. Appropriate motifs present. Putative not added to product function. Functional classification - Ribosomal protein synthesis and modification; PMID:6267426","uniprotId":"Q0PA05_CAMJE"},"10641705":{"id":10641705,"externalId":"192222.Cj0894c","name":"ispH","hasFamily":true,"annotation":"Catalyzes the conversion of 1-hydroxy-2-methyl-2-(E)-butenyl 4-diphosphate (HMBPP) into a mixture of isopentenyl diphosphate (IPP) and dimethylallyl diphosphate (DMAPP). Acts in the terminal step of the DOXP/MEP pathway for isoprenoid precursor biosynthesis. ","uniprotId":"ISPH_CAMJE"},"10641706":{"id":10641706,"externalId":"192222.Cj0895c","name":"aroA","hasFamily":true,"annotation":"Catalyzes the transfer of the enolpyruvyl moiety of phosphoenolpyruvate (PEP) to the 5-hydroxyl of shikimate-3-phosphate (S3P) to produce enolpyruvyl shikimate-3-phosphate and inorganic phosphate","uniprotId":"AROA_CAMJE"},"10641707":{"id":10641707,"externalId":"192222.Cj0896c","name":"pheT","hasFamily":true,"annotation":"Belongs to the phenylalanyl-tRNA synthetase beta subunit family. Type 1 subfamily","uniprotId":"SYFB_CAMJE"},"10641708":{"id":10641708,"externalId":"192222.Cj0897c","name":"pheS","hasFamily":true,"annotation":"Belongs to the class-II aminoacyl-tRNA synthetase family. Phe-tRNA synthetase alpha subunit type 1 subfamily","uniprotId":"SYFA_CAMJE"},"10641709":{"id":10641709,"externalId":"192222.Cj0898","name":"Cj0898","hasFamily":true,"annotation":"Putative histidine triad (HIT) family protein; Original (2000) note: Cj0898, HIT-family protein,len: 121 aa; similar to many members of the HIT family e.g. IPK1_HUMAN HINT protein (protein kinase C inhibitor 1) (125 aa), fasta scores; opt: 286 z-score: 385.9 E(): 3.4e-14, 40.6% identity in 106 aa overlap, and YCFF_ECOLI HYPOTHETICAL 13.2 KD PROTEIN HIT-LIKE PROTE (119 aa),fasta scores; opt: 276 z-score: 373.2 E(): 1.7e-13, 38.2% identity in 110 aa overlap. 56.6% identity to HP0404. Contains Pfam match to entry PF01230 HIT, HIT family; Updated (2006) note: Specific characterisation has  [...] ","uniprotId":"Q0PA00_CAMJE"},"10641710":{"id":10641710,"externalId":"192222.Cj0899c","name":"thiJ","hasFamily":true,"annotation":"Original (2000) note: Cj0899c, thiJ, probable 4-methyl-5(beta-hydroxyethyl)-thiazole monophosphate synthesis protein, len: 189 aa; simlar to e.g. THIJ_ECOLI 4-methyl-5(b-hydroxyethyl)-thiazole monophosphate biosynthesis enzyme (198 aa), fasta scores; opt: 265 z-score: 330.3 E(): 4.2e-11, 32.6% identity in 181 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF01965 DJ-1/PfpI family identified within CDS. Further support given to product function. Characterised in Escherichia coli with marginal identity score. Putative not added to protein function. Functional classification -  [...] ","uniprotId":"Q0P9Z9_CAMJE"},"10641711":{"id":10641711,"externalId":"192222.Cj0900c","name":"Cj0900c","hasFamily":true,"annotation":"Original (2000) note: Cj0900c, small hydrophobic protein, len: 59 aa; 29.8% identity to HP0085. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P9Z8_CAMJE"},"10641712":{"id":10641712,"externalId":"192222.Cj0901","name":"Cj0901","hasFamily":true,"annotation":"Original (2000) note: Cj0901, probable amino acid ABC tansporter permease protein, len: 247 aa; similar to many e.g. YQIY_BACSU probable amino-acid ABC transporter permease protein (219 aa), fasta scores; opt: 552 z-score: 683.2 E(): 9.3e-31, 43.6% identity in 204 aa overlap. 28.6% identity to HP0939. Contains PS00402 Binding-protein-dependent transport systems inner membrane comp signature, and Pfam match to entry PF00528 BPD_transp, Binding-protein-dependent transport systems inner membrane component; Updated (2006) note: Four probable transmembrane helices predicted by TMHMM2.0. Fur [...] ","uniprotId":"Q0P9Z7_CAMJE"},"10641713":{"id":10641713,"externalId":"192222.Cj0902","name":"glnQ","hasFamily":true,"annotation":"Original (2000) note: Cj0902, glnQ, probable glutamine transport ATP-binding protein, len: 247 aa; highly similar to e.g. GLNQ_BACST glutamine transport ATP-binding protein (242 aa), fasta scores; opt: 940 z-score: 1074.4 E(): 0, 56.2% identity in 242 aa overlap. No Hp ortholog. Also similar to Cj0469 (52.9% identity in 238 aa overlap). Contains S00017 ATP/GTP-binding site motif A (P-loop), S00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran, ABC transporters; Updated (2006) note: Characterised in Bacillus stearothermophilus with acceptable identity score [...] ","uniprotId":"Q0P9Z6_CAMJE"},"10641714":{"id":10641714,"externalId":"192222.Cj0903c","name":"Cj0903c","hasFamily":true,"annotation":"Original (2000) note: Cj0903c, probable amino-acid transport protein, len: 488 aa; simlar to many hypothetical membaer of the sodium:alanine symporter family e.g. ALST_BACSU amino acid carrier protein ALST (465 aa), fasta scores; opt: 821 z-score: 946.6 E(): 0,41.9% identity in 473 aa overlap. 33.6% identity to HP0942. Contains PS00873 Sodium:alanine symporter family signature, and Pfam match to entry PF01235 Na_Ala_symp,Sodium:alanine symporter family; Updated (2006) note: Nine probable transmembrane helices predicted for TMHMM2.0. Further support given to product function. No specfic [...] ","uniprotId":"Q0P9Z5_CAMJE"},"10641715":{"id":10641715,"externalId":"192222.Cj0904c","name":"trmL","hasFamily":true,"annotation":"Methylates the ribose at the nucleotide 34 wobble position in the two leucyl isoacceptors tRNA(Leu)(CmAA) and tRNA(Leu)(cmnm5UmAA). Catalyzes the methyl transfer from S-adenosyl-L-methionine to the 2'-OH of the wobble nucleotide","uniprotId":"Q0P9Z4_CAMJE"},"10641716":{"id":10641716,"externalId":"192222.Cj0905c","name":"alr","hasFamily":true,"annotation":"Catalyzes the interconversion of L-alanine and D-alanine. May also act on other amino acids","uniprotId":"ALR_CAMJE"},"10641717":{"id":10641717,"externalId":"192222.Cj0906c","name":"Cj0906c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0906c, probable periplasmic protein, len: 325 aa; similar to hypothetical proteins e.g. TR:O66569 (EMBL:AE000677) Aquifex aeolicus AQ_183 (397 aa), fasta scores; opt: 380 z-score: 432.9 E(): 8.2e-17, 31.1% identity in 270 aa overlap. 36.6% identity to HP0518. Contains probable N-terminal signal sequence; Updated (2006) note: Paper identified linking protein to glycoprotein. Functional classification -Miscellaneous periplasmic proteins; PMID:12186869","uniprotId":"Q0P9Z2_CAMJE"},"10641718":{"id":10641718,"externalId":"192222.Cj0908","name":"Cj0908","hasFamily":false,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0908, probable periplasmic protein, len: 156 aa; no Hp match. Contains probable N-terminal signal sequence. Similar to downstream gene Cj0910 (41.3% identity in 143 aa overlap). Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0P9Z1_CAMJE"},"10641719":{"id":10641719,"externalId":"192222.Cj0909","name":"Cj0909","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0909, probable periplasmic protein, len: 139 aa; similar to TR:O67294 (EMBL:AE000731) Aquifex aeolicus AQ_1253 (151 aa), fasta scores; opt: 208 z-score: 273.4 E(): 6.3e-08, 34.3% identity in 134 aa overlap. No Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF04314 Protein of unknown function (DUF461) identified within CDS. This Pfam is linked to being a putative membrane or periplasmic protein. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0P9Z0_CAMJE"},"10641720":{"id":10641720,"externalId":"192222.Cj0910","name":"Cj0910","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0910, probable periplasmic protein, len: 150 aa; no Hp match. Contains probable N-terminal signal sequence. Similar to downstream gene Cj0908 (41.3% identity in 145 aa overlap). Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0P9Y9_CAMJE"},"10641721":{"id":10641721,"externalId":"192222.Cj0911","name":"Cj0911","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0911, probable periplasmic protein, len: 186 aa; similar to bacterial hypotheticals e.g. TR:O67113 (EMBL:AE000717) Aquifex aeolicus AQ_988 (197 aa), fasta scores; opt: 260 z-score: 314.1 E(): 3.4e-10, 28.1% identity in 167 aa overlap, and to N-terminus of eukaryotic proteins e.g. SCO1_YEAST SCO1 protein precursor (295 aa), opt: 213 z-score: 257.2 E(): 5e-07, 32.1% identity in 137 aa overlap. No Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF02630 SCO1/SenC identified within CDS. Characteri [...] ","uniprotId":"Q0P9Y8_CAMJE"},"10641722":{"id":10641722,"externalId":"192222.Cj0912c","name":"cysM","hasFamily":true,"annotation":"Cysteine synthase B; Original (2000) note: Cj0912c, cysM, cysteine synthase, len: 299 aa; 98.7% identical to CYSM_CAMJE cysteine synthase B (EC 4.2.99.8) (299 aa), and similar to many e.g. CYSK_BACSU CYSTEINE SYNTHASE (EC 4.2.99.8) (307 aa), fasta scores; opt: 907 z-score: 1020.2 E(): 0, 48.5% identity in 305 aa overlap. 38.9% identity to HP0107. Contains PS00165 Serine/threonine dehydratases pyridoxal-phosphate attachment site, PS00901 Cysteine synthase/cystathionine beta-synthase P-phosphate attachment site, and Pfam match to entry PF00291 S_T_dehydratase, Pyridoxal-phosphate dependa [...] ","uniprotId":"CYSM_CAMJE"},"10641723":{"id":10641723,"externalId":"192222.Cj0913c","name":"hupB","hasFamily":true,"annotation":"Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions","uniprotId":"DBH_CAMJE"},"10641724":{"id":10641724,"externalId":"192222.Cj0914c","name":"ciaB","hasFamily":true,"annotation":"CiaB protein; Original (2000) note: Cj0914c, ciaB, unknown function, len: 610 aa; no Hp match. Product is translocated into host cells. Mutants fail to translocate,and fail to translocate other proteins. Contains PS00142 Neutral zinc metallopeptidases, zinc-binding region signature; Updated (2006) note: Papers attached giving further information regarding this CDS. Functional classification - Pathogenicity; PMID:10361274, PMID:10540297, PMID:10659361,PMID:15722140","uniprotId":"Q0P9Y5_CAMJE"},"10641725":{"id":10641725,"externalId":"192222.Cj0915","name":"Cj0915","hasFamily":true,"annotation":"Putative hydrolase; Original (2000) note: Cj0915, possible hydrolase,len: 137 aa; similar to prokaryotic hypotheticals e.g. YCIA_ECOLI (132 aa), fasta scores; opt: 351 z-score: 440.7 E(): 3e-17, 41.5% identity in 123 aa overlap, and to eukaryotic acyl-CoA hydrolases e.g. THCC_HUMAN cytosolic acyl coenzyme A thioester hydrolase (372 aa), fasta scores; opt: 174 z-score: 220.2 E(): 5.7e-05, 31.6% identity in 114 aa overlap. 25.4% identity to HP0891 (VDLD_HELPY); Updated (2006) note: Pfam domain PF03061 Thioesterase superfamily identified within CDS. No specific characterisation with accep [...] ","uniprotId":"Q0P9Y4_CAMJE"},"10641726":{"id":10641726,"externalId":"192222.Cj0916c","name":"Cj0916c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0916c, unknown, len: 65 aa; similar to two hypothetical E. coli proteins; YBDD_ECOLI (65 aa), fasta scores; opt: 154 z-score: 227.0 E(): 2.5e-05, 41.3% identity in 46 aa overlap, and YJIX_ECOLI (67 aa), fasta scores; opt: 141 z-score: 209.5 E(): 0.00023, 35.8% identity in 67 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF04328 Protein of unknown function (DUF466) identified within CDS. Conserved added to product function. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P9Y3_CAMJE"},"10641727":{"id":10641727,"externalId":"192222.Cj0917c","name":"cstA","hasFamily":true,"annotation":"Involved in the uptake of dipeptides and tripeptides. May influence host-pathogen interactions. Involved in motility and agglutination, and has a role in stimulation of dendritic cells. ","uniprotId":"CSTA_CAMJE"},"10641728":{"id":10641728,"externalId":"192222.Cj0918c","name":"prsA","hasFamily":true,"annotation":"Involved in the biosynthesis of the central metabolite phospho-alpha-D-ribosyl-1-pyrophosphate (PRPP) via the transfer of pyrophosphoryl group from ATP to 1-hydroxyl of ribose-5-phosphate (Rib- 5-P)","uniprotId":"KPRS_CAMJE"},"10641729":{"id":10641729,"externalId":"192222.Cj0919c","name":"Cj0919c","hasFamily":true,"annotation":"Original (2000) note: Cj0919c, probable ABC-type amino-acid transporter permease protein, len: 219 aa; similar to many e.g. GLTK_ECOLI glutamate/aspartate transport system permease protein (224 aa), fasta scores; opt: 380 z-score: 457.9 E(): 3.3e-18, 32.6% identity in 221 aa overlap. No Hp ortholog. Contains PS00402 Binding-protein-dependent transport systems inner membrane comp sign., and Pfam match to entry PF00528 BPD_transp,Binding-protein-dependent transport systems inner membrane component; Updated (2006) note: Three probable transmembrane helices predicted by TMHMM2.0. No specif [...] ","uniprotId":"Q0P9Y0_CAMJE"},"10641730":{"id":10641730,"externalId":"192222.Cj0920c","name":"Cj0920c","hasFamily":true,"annotation":"Original (2000) note: Cj0920c, probable ABC-type amino-acid transporter permease protein, len: 250 aa; similar to many e.g. YCKA_BACSU probable amino-acid ABC transporter permease protein (226 aa), fasta scores; opt: 448 z-score: 560.9 E(): 6e-24, 32.0% identity in 219 aa overlap. No Hp ortholog. Contains PS00402 Binding-protein-dependent transport systems inner membrane comp sign., and Pfam match to entry PF00528 BPD_transp,Binding-protein-dependent transport systems inner membrane component; Updated (2006) note: Three probable transmembrane helices predicted by TMHMM2.0. No specific  [...] ","uniprotId":"Q0P9X9_CAMJE"},"10641731":{"id":10641731,"externalId":"192222.Cj0921c","name":"peb1A","hasFamily":true,"annotation":"Common antigen and a major cell adherence molecule. Most probably involved, with PEB1C, in a binding-protein-dependent transport system for an amino acid. May be involved in binding to intestinal cells","uniprotId":"PEB1A_CAMJE"},"10641732":{"id":10641732,"externalId":"192222.Cj0922c","name":"pebC","hasFamily":true,"annotation":"Most probably involved, with PEB1, in a binding-protein- dependent transport system for an amino acid. Probably responsible for energy coupling to the transport system (By similarity)","uniprotId":"PEB1C_CAMJE"},"10641733":{"id":10641733,"externalId":"192222.Cj0923c","name":"cheR","hasFamily":true,"annotation":"Methylation of the membrane-bound methyl-accepting chemotaxis proteins (MCP) to form gamma-glutamyl methyl ester residues in MCP. ","uniprotId":"CHER_CAMJE"},"10641734":{"id":10641734,"externalId":"192222.Cj0924c","name":"cheB","hasFamily":true,"annotation":"Protein-glutamate methylesterase; Original (2000) note: Cj0924c, cheB' probable MCP protein-glutamate methylesterase, len: 184 aa; similar to the methylesterase domain of e.g. CHEB_ECOLI protein-glutamate methylesterase (EC 3.1.1.61) (349 aa),fasta scores; opt: 339 z-score: 430.2 E(): 1.1e-16, 34.4% identity in 183 aa overlap. Note that the 'response reciever domain' normally found in cheB is not present. No Hp match; Updated (2006) note: Characterised within Escherichia coli, however, identity scores were marginal. Possible truncated protein as missing receiver domain of CheB protein. [...] ","uniprotId":"Q0P9X5_CAMJE"},"10641735":{"id":10641735,"externalId":"192222.Cj0925","name":"rpiB","hasFamily":true,"annotation":"Ribose 5-phosphate isomerase; Original (2000) note: Cj0925, rpiB, probable ribose 5-phosphate isomerase, len: 145 aa; highly similar to e.g. RPIB_ECOLI ribose 5-phosphate isomerase B (EC 5.3.1.6) (149 aa), fasta scores; opt: 399 z-score: 505.9 E(): 7e-21, 45.0% identity in 140 aa overlap. 45.0% identity to HP0574, misannotated as galactosidase acetyltransferase (lacA); Updated (2006) note: Prosite domain PS00237 G_PROTEIN_RECEP_F1_1, Rhodopsin-like GPCR superfamily and Pfam domain PF02502 Ribose/Galactose Isomerase were identified within CDS. Further support given to product function.  [...] ","uniprotId":"Q0P9X4_CAMJE"},"10641736":{"id":10641736,"externalId":"192222.Cj0926","name":"Cj0926","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0926, possible membrane protein, len: 110 aa; 50.0% identity to HP0573. Contains possible transmembrane domain at N-terminus; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P9X3_CAMJE"},"10641737":{"id":10641737,"externalId":"192222.Cj0927","name":"apt","hasFamily":true,"annotation":"Catalyzes a salvage reaction resulting in the formation of AMP, that is energically less costly than de novo synthesis. ","uniprotId":"APT_CAMJE"},"10641738":{"id":10641738,"externalId":"192222.Cj0928","name":"Cj0928","hasFamily":true,"annotation":"Original (2000) note: Cj0928, probable integral membrane protein, len: 198 aa; similar to members of the dedA family of hypothetical integral membrane proteins e.g. YOHD_ECOLI (192 aa), fasta scores; opt: 303 z-score: 394.8 E(): 1.1e-14, 29.1% identity in 172 aa overlap, and DEDA_ECOLI DEDA protein (219 aa), fasta scores; opt: 169 z-score: 224.4 E(): 3.3e-05, 25.3% identity in 162 aa overlap. 61.4% identity to HP0571. Contains Pfam match to entry PF00597 DedA, DedA family. Also similar to Cj1210 (27.1% identity in 177 aa overlap); Updated (2006) note: Five probable transmembrane helice [...] ","uniprotId":"Q0P9X1_CAMJE"},"10641739":{"id":10641739,"externalId":"192222.Cj0929","name":"pepA","hasFamily":true,"annotation":"Presumably involved in the processing and regular turnover of intracellular proteins. Catalyzes the removal of unsubstituted N- terminal amino acids from various peptides","uniprotId":"AMPA_CAMJE"},"10641740":{"id":10641740,"externalId":"192222.Cj0930","name":"ychF","hasFamily":true,"annotation":"ATPase that binds to both the 70S ribosome and the 50S ribosomal subunit in a nucleotide-independent manner. ","uniprotId":"Q0P9W9_CAMJE"},"10641741":{"id":10641741,"externalId":"192222.Cj0931c","name":"argH","hasFamily":true,"annotation":"Argininosuccinate lyase; Original (2000) note: Cj0931c, argH,argininosuccinate lyase, len: 460 aa; 97.4% identical to ARLY_CAMJE argininosuccinate lyase (EC 4.3.2.1) (459 aa),and highly similar to many e.g. ARLY_CANAL argininosuccinate lyase (EC 4.3.2.1) (468 aa), fasta scores; opt: 1337 z-score: 1518.4 E(): 0, 44.7% identity in 459 aa overlap. No Hp ortholog. Contains PS00163 Fumarate lyases signature, and Pfam match to entry PF00206 lyase_1, Lyases; Updated (2006) note: Characterised in Campylobacter jejuni, so putative not added to product function. Functional classification - Amino [...] ","uniprotId":"ARLY_CAMJE"},"10641742":{"id":10641742,"externalId":"192222.Cj0932c","name":"pckA","hasFamily":true,"annotation":"Involved in the gluconeogenesis. Catalyzes the conversion of oxaloacetate (OAA) to phosphoenolpyruvate (PEP) through direct phosphoryl transfer between the nucleoside triphosphate and OAA. ","uniprotId":"PCKA_CAMJE"},"10641743":{"id":10641743,"externalId":"192222.Cj0933c","name":"pycB","hasFamily":true,"annotation":"Original (2000) note: Cj0933c, pycB, possible possible pyruvate carboxylase B subunit, len: 599 aa; similar to TR:O27179 (EMBL:AE000942) Methanobacterium thermoautotrophicum characterised pyruvate carboxylase B (EC 6.4.1.1) (568 aa), fasta scores; opt: 425 z-score: 482.7 E(): 1.4e-19, 29.1% identity in 601 aa overlap. Also similar to other biotin-requiring decarboxlases e.g. DCOA_SALTY oxaloacetate decarboxylase alpha chain (590 aa), fasta scores; opt: 515 z-score: 584.5 E(): 2.9e-25,27.3% identity in 605 aa overlap, and C-terminus of PYC1_YEAST pyruvate carboxylase 1 (1178 aa), fasta  [...] ","uniprotId":"Q0P9W6_CAMJE"},"10641744":{"id":10641744,"externalId":"192222.Cj0934c","name":"Cj0934c","hasFamily":true,"annotation":"Putative sodium:amino-acid symporter family protein; Original (2000) note: Cj0934c, probable transmembrane transport protein, len: 446 aa; similar to members of the sodium:neurotransmitter symporter family e.g. YD19_METJA hypothetical sodium-dependent transporter MJ1319 (492 aa), fasta scores; opt: 628 z-score: 711.7 E(): 2.4e-32, 27.4% identity in 457 aa overlap. 42.6% identity to HP0498. Also similar to Cj0935c (51.0% identity in 441 aa overlap) and Cj0601c (28.2% identity in 454 aa overlap); Updated (2006) note: Pfam domain PF00209 Sodium:neurotransmitter symporter family identified [...] ","uniprotId":"Q0P9W5_CAMJE"},"10641745":{"id":10641745,"externalId":"192222.Cj0935c","name":"Cj0935c","hasFamily":true,"annotation":"Putative sodium:amino-acid symporter family protein; Original (2000) note: Cj0935c, probable transmembrane transport protein, len: 444 aa; similar to members of the sodium:neurotransmitter symporter family e.g. YG90_HAEIN hypothetical sodium-dependent transporter HI1690 (457 aa), fasta scores; opt: 694 z-score: 771.9 E(): 0, 33.0% identity in 449 aa overlap. 40.4% identity to HP0497. Pfam match to entry PF00209 SNF,Sodium:neurotransmitter symporter family. Also similar to Cj0934c (51.0% identity in 441 aa overlap) and Cj0601c (32.6% identity in 451 aa overlap); Updated (2006) note: Pfa [...] ","uniprotId":"Q0P9W4_CAMJE"},"10641746":{"id":10641746,"externalId":"192222.Cj0936","name":"atpE","hasFamily":true,"annotation":"F(1)F(0) ATP synthase produces ATP from ADP in the presence of a proton or sodium gradient. F-type ATPases consist of two structural domains, F(1) containing the extramembraneous catalytic core and F(0) containing the membrane proton channel, linked together by a central stalk and a peripheral stalk. During catalysis, ATP synthesis in the catalytic domain of F(1) is coupled via a rotary mechanism of the central stalk subunits to proton translocation","uniprotId":"ATPL_CAMJE"},"10641747":{"id":10641747,"externalId":"192222.Cj0937","name":"Cj0937","hasFamily":true,"annotation":"Original (2000) note: Cj0937, probable integral membrane protein, len: 303 aa; simimlar to hypothetical membrane proteins e.g. YA31_METJA MJ1031 (308 aa), fasta scores; opt: 379 z-score: 446.4 E(): 1.4e-17, 26.1% identity in 318 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF03547 Auxin Efflux Carrier identified within CDS. Also, ten probable transmembrane helices predicted by TMHMM2.0. PProteins in this group are mostly uncharacterised and of unknown function. Functional classification - Membranes,lipoproteins and porins","uniprotId":"Q0P9W2_CAMJE"},"10641748":{"id":10641748,"externalId":"192222.Cj0938c","name":"aas","hasFamily":true,"annotation":"Original (2000) note: Cj0938c, aas, probable 2-acylglycerophosphoethanolamine acyltransferase / acyl-acyl carrier protein synthetase, len: 1170 aa; contains 2 domains; C-terminal domain (420 aa to end) is simlar to the bifunctional enzyme AAS_ECOLI 2-acylglycerophosphoethanolamine acyltransferase / acyl-acyl carrier protein synthetase (719 aa), fasta scores; opt: 438 z-score: 471.6 E(): 5.7e-19, 31.0% identity in 762 aa overlap; N-terminal domain contains approx 12 transmembrane domains, and is similar to the hypothetical membrane protein immediately upstreram of the E. coli aas; YGED_ [...] ","uniprotId":"Q0P9W1_CAMJE"},"10641749":{"id":10641749,"externalId":"192222.Cj0939c","name":"Cj0939c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0939c, unknown, len: 126 aa; 29.5% identity to HP1065. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P9W0_CAMJE"},"10641750":{"id":10641750,"externalId":"192222.Cj0940c","name":"glnP","hasFamily":true,"annotation":"Original (2000) note: Cj0940c, glnP, probable glutamine transport system permease, len: 261 aa; similar to many e.g. GLNP_ECOLI glutamine transport system permease protein (219 aa), fasta scores; opt: 414 z-score: 509.0 E(): 4.7e-21, 37.5% identity in 216 aa overlap. No Hp otholog. Contains PS00402 Binding-protein-dependent transport systems inner membrane comp sign., and Pfam match to entry PF00528 BPD_transp,Binding-protein-dependent transport systems inner membrane component; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Not fully characterised, so p [...] ","uniprotId":"Q0P9V9_CAMJE"},"10641751":{"id":10641751,"externalId":"192222.Cj0941c","name":"Cj0941c","hasFamily":true,"annotation":"Putative permease; Original (2000) note: Cj0941c, probable integral membrane protein, len: 401 aa; similar to many hypothetical membrane proteins e.g. YCFU_ECOLI(399 aa),fasta scores; opt: 414 z-score: 478.9 E(): 2.2e-19, 22.1% identity in 408 aa overlap. 53.5% identity to HP0787; Updated (2006) note: Pfam domain PF02687 Predicted permease identified within CDS. Also, four probable transmembrane helices predicted by TMHMM2.0. Product modified to more specific family member based on motif match. Not specific characterisation has been carried out yet, so putative kept within product func [...] ","uniprotId":"Q0P9V8_CAMJE"},"10641752":{"id":10641752,"externalId":"192222.Cj0942c","name":"secA","hasFamily":true,"annotation":"Part of the Sec protein translocase complex. Interacts with the SecYEG preprotein conducting channel. Has a central role in coupling the hydrolysis of ATP to the transfer of proteins into and across the cell membrane, serving as an ATP-driven molecular motor driving the stepwise translocation of polypeptide chains across the membrane","uniprotId":"SECA_CAMJE"},"10641753":{"id":10641753,"externalId":"192222.Cj0943","name":"lolA","hasFamily":true,"annotation":"Participates in the translocation of lipoproteins from the inner membrane to the outer membrane. Only forms a complex with a lipoprotein if the residue after the N-terminal Cys is not an aspartate (The Asp acts as a targeting signal to indicate that the lipoprotein should stay in the inner membrane) (By similarity)","uniprotId":"LOLA_CAMJE"},"10641754":{"id":10641754,"externalId":"192222.Cj0944c","name":"Cj0944c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0944c, probable periplasmic protein, len: 244 aa; no Hp match; Updated (2006) note: Paper identified linking protein to glycoprotein. Functional classification -Miscellaneous periplasmic proteins; PMID:12186869","uniprotId":"Q0P9V5_CAMJE"},"10641755":{"id":10641755,"externalId":"192222.Cj0945c","name":"pif1","hasFamily":true,"annotation":"DNA-dependent ATPase and 5'-3' DNA helicase that efficiently unwinds G-quadruplex (G4) DNA structures. May be involved in resolving commom issues that arise during DNA replication, recombination, and repair","uniprotId":"PIF1_CAMJE"},"10641756":{"id":10641756,"externalId":"192222.Cj0946","name":"Cj0946","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0946, probable lipoprotein,len: 448 aa; 29.0% identity to HP0087. Contains probable N-terminal signal sequence with appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site, and Pfam match to entry PF00877 NLPC_P60, NLP/P60 family. Functional classification - Membranes,lipoproteins and porins","uniprotId":"Q0P9V3_CAMJE"},"10641757":{"id":10641757,"externalId":"192222.Cj0947c","name":"Cj0947c","hasFamily":true,"annotation":"Putative carbon-nitrogen hydrolase; Original (2000) note: Cj0947c, probable hydrolase,len: 290 aa; similar to prokaryotic hypothetical hydrolases, and to e.g. BUP_RAT beta-ureidopropionase (393 aa), fasta scores; opt: 319 z-score: 392.0 E(): 1.6e-14,26.0% identity in 304 aa overlap. 60.9% identity to HP0757; Updated (2006) note: Pfam domain PF00795 Carbon-nitrogen hydrolase identified within CDS. Product modified to more specific family member based on motif match. Specific characterisation has been carried out yet,so putative kept in product funciton. Functional classification - Misc","uniprotId":"Q0P9V2_CAMJE"},"10641758":{"id":10641758,"externalId":"192222.Cj0948c","name":"Cj0948c","hasFamily":true,"annotation":"Belongs to the cation diffusion facilitator (CDF) transporter (TC 2.A.4) family","uniprotId":"Q0P9V1_CAMJE"},"10641759":{"id":10641759,"externalId":"192222.Cj0949c","name":"Cj0949c","hasFamily":true,"annotation":"Belongs to the agmatine deiminase family. ","uniprotId":"Q0P9V0_CAMJE"},"10641760":{"id":10641760,"externalId":"192222.Cj0950c","name":"Cj0950c","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0950c, probable lipoprotein, len: 144 aa; some similarity to HSLJ_ECOLI heat shock protein HSLJ (140 aa), fasta scores; opt: 152 z-score: 202.3 E(): 0.00057, 26.2% identity in 145 aa overlap. 33.3% identity to HP1462 (secreted protein involved in flagellar motility). Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site; Updated (2006) note: Pfam domain PF03724 Domain of unknown function (306) identified within CDS. Functional classification - Membranes, lipo [...] ","uniprotId":"Q0P9U9_CAMJE"},"10641761":{"id":10641761,"externalId":"192222.Cj0951c","name":"Cj0951c","hasFamily":true,"annotation":"Original (2000) note: Cj0951c, probable MCP-domain signal transduction protein, len: 224 aa; similar to MCP domains in many proteins e.g. CPS_CLOTM putative sensory transducer protein (557 aa), fasta scores; opt: 270 z-score: 283.0 E(): 1.8e-08, 30.3% identity in 198 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00015 MCPsignal, Methyl-accepting chemotaxis protein (MCP) signaling domain. The ORF continues upstream of this start, and contains BLASTX hits to other MCP proteins,suggesting that this may be a pseudogene, and should be read through from the upstream CDS. The seq [...] ","uniprotId":"Q0P9U8_CAMJE"},"10641762":{"id":10641762,"externalId":"192222.Cj0952c","name":"Cj0952c","hasFamily":true,"annotation":"Putative HAMP containing membrane protein; Original (2000) note: Cj0952c, probable membrane protein, len: 239 aa; some simlarity to TR:O66691 (EMBL:AE000686) Aquifex aeolicus AQ_367 (850 aa), fasta scores; opt: 229 z-score: 256.9 E(): 5.2e-07, 26.4% identity in 208 aa overlap. No Hp match. Contains PS00190 Cytochrome c family heme-binding site signature. BLASTX hits to MCP proteins in the C-terminus of this CDS continue into the downstream ORF, suggesting that this may be a pseudogene, and should be read through into the downstream CDS. The sequence has been checked, and is believed to [...] ","uniprotId":"Q0P9U7_CAMJE"},"10641763":{"id":10641763,"externalId":"192222.Cj0953c","name":"purH","hasFamily":true,"annotation":"Bifunctional purine biosynthesis protein PurH; Original (2000) note: Cj0953c, purH,phosphoribosylaminoimidazolecarboxamide formyltransferase / IMP cyclohydrolase, len: 510 aa; highly similar to many e.g. PUR9_ECOLI phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3) / IMP cyclohydrolase (EC 3.5.4.10) (529 aa), fasta scores; opt: 812 z-score: 935.5 E(): 0, 44.6% identity in 529 aa overlap. No Hp match; Updated (2006) note: Pfam domains PF01808 AICARFT/IMPCHase bienzyme and PF02142 MGS-like domain were identified within CDS. Further support given to product function. C [...] ","uniprotId":"PUR9_CAMJE"},"10641764":{"id":10641764,"externalId":"192222.Cj0954c","name":"Cj0954c","hasFamily":true,"annotation":"Putative DnaJ-like protein; Original (2000) note: Cj0954c, possible dnaJ-like protein, len: 256 aa; weak similarity to e.g. DJLA_COXBU DNAJ-like protein DJLA (270 aa), fasta scores; opt: 167 z-score: 193.0 E(): 0.0019, 26.9% identity in 201 aa overlap. No Hp match. Contains Pfam match to entry PF00226 DnaJ, DnaJ domain; Updated (2006) note: No specific characterisation with acceptable identity scores carried out yet. Putative kept within product function. Functional classification -Chaperones, chaperonins, heat shock","uniprotId":"Q0P9U5_CAMJE"},"10641765":{"id":10641765,"externalId":"192222.Cj0955c","name":"purL","hasFamily":true,"annotation":"Part of the phosphoribosylformylglycinamidine synthase complex involved in the purines biosynthetic pathway. Catalyzes the ATP-dependent conversion of formylglycinamide ribonucleotide (FGAR) and glutamine to yield formylglycinamidine ribonucleotide (FGAM) and glutamate. The FGAM synthase complex is composed of three subunits. PurQ produces an ammonia molecule by converting glutamine to glutamate. PurL transfers the ammonia molecule to FGAR to form FGAM in an ATP- dependent manner. PurS interacts with PurQ and PurL and is thought to assist in the transfer of the ammonia molecule from Pu [...] ","uniprotId":"PURL_CAMJE"},"10641766":{"id":10641766,"externalId":"192222.Cj0956c","name":"trmE","hasFamily":true,"annotation":"Exhibits a very high intrinsic GTPase hydrolysis rate. Involved in the addition of a carboxymethylaminomethyl (cmnm) group at the wobble position (U34) of certain tRNAs, forming tRNA- cmnm(5)s(2)U34","uniprotId":"MNME_CAMJE"},"10641767":{"id":10641767,"externalId":"192222.Cj0957c","name":"Cj0957c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0957c, unknown, len: 272 aa; 42.5% identity to HP1451. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P9U2_CAMJE"},"10641768":{"id":10641768,"externalId":"192222.Cj0958c","name":"yidC","hasFamily":true,"annotation":"Required for the insertion and/or proper folding and/or complex formation of integral membrane proteins into the membrane. Involved in integration of membrane proteins that insert both dependently and independently of the Sec translocase complex, as well as at least some lipoproteins. Aids folding of multispanning membrane proteins","uniprotId":"YIDC_CAMJE"},"10641769":{"id":10641769,"externalId":"192222.Cj0959c","name":"Cj0959c","hasFamily":true,"annotation":"Could be involved in insertion of integral membrane proteins into the membrane","uniprotId":"YIDD_CAMJE"},"10641770":{"id":10641770,"externalId":"192222.Cj0960c","name":"rnpA","hasFamily":true,"annotation":"RNaseP catalyzes the removal of the 5'-leader sequence from pre-tRNA to produce the mature 5'-terminus. It can also cleave other RNA substrates such as 4.5S RNA. The protein component plays an auxiliary but essential role in vivo by binding to the 5'-leader sequence and broadening the substrate specificity of the ribozyme. ","uniprotId":"RNPA_CAMJE"},"10641771":{"id":10641771,"externalId":"192222.Cj0961c","name":"rpmH","hasFamily":true,"annotation":"Belongs to the bacterial ribosomal protein bL34 family. ","uniprotId":"RL34_CAMJE"},"10641772":{"id":10641772,"externalId":"192222.Cj0962","name":"Cj0962","hasFamily":true,"annotation":"Putative acetyltransferase; Original (2000) note: Cj0962, possible acetyltransferase, len: 159 aa; most similar to eukaryotic acetyltransferases, e.g. TR:P79081 (EMBL:U82218) Schizosaccharomyces pombe putative acetyltransferase ATS1 (168 aa), fasta scores; opt: 317 z-score: 406.2 E(): 2.5e-15, 34.6% identity in 162 aa overlap, and ATDA_MOUSE diamine acetyltransferase (171 aa), fasta scores; opt: 211 z-score: 274.9 E(): 5.2e-08, 27.4% identity in 168 aa overlap. No Hp match. Contains Pfam match to entry PF00583 Acetyltransf, Acetyltransferase (GNAT) family; Updated (2006) note: No speci [...] ","uniprotId":"Q0P9T7_CAMJE"},"10641773":{"id":10641773,"externalId":"192222.Cj0963","name":"Cj0963","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0963, unknown, len: 202 aa; similar to hypothetical proteins e.g. TR:O67598 (EMBL:AE000751) Aquifex aeolicus DPLF (195 aa), fasta scores; opt: 234 z-score: 301.4 E(): 1.7e-09, 27.2% identity in 184 aa overlap. 21.4% identity to HP0650. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P9T6_CAMJE"},"10641774":{"id":10641774,"externalId":"192222.Cj0964","name":"Cj0964","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0964, probable periplasmic protein, len: 396 aa; 26.6% identity to HP0781. Contains probable N-terminal signal sequence. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0P9T5_CAMJE"},"10641775":{"id":10641775,"externalId":"192222.Cj0965c","name":"Cj0965c","hasFamily":true,"annotation":"Putative acyl-CoA thioester hydrolase; Original (2000) note: Cj0965c, unknown, len: 124 aa; similar to hypothetical proteins e.g. YBGC_ECOLI (134 aa), fasta scores; opt: 235 z-score: 313.7 E(): 3.5e-10,38.1% identity in 113 aa overlap. 40.6% identity to HP0496; Updated (2006) note: Pfam domain PF03061 Thioesterase superfamily and Prosite domain PS01328 4HBCOA_THIOESTERASE, 4-hydroxybenzoyl-CoA thioesterase,active site identified within CDS. Product modified to more specific family member based on motif match. Some characterisation carried out in Escerichia coli with acceptable identity [...] ","uniprotId":"Q0P9T4_CAMJE"},"10641776":{"id":10641776,"externalId":"192222.Cj0967","name":"Cj0967","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0967, possible periplasmic protein, len: 762 aa; no Hp match. N- and C- termini are similar to two consecutive genes Cj0735 (37.8% identity in 214 aa overlap) and Cj0736 (42.3% identity in 562 aa overlap). Contains probable N-terminal signal sequence. Note that an ATA initiation codon has been used: this is consistent with the RBS, the similarity to Cj0735, and measures of coding likelyhood based on aa usage. Contains S00639 Eukaryotic thiol (cysteine) proteases histidine active site, PS00133 Zinc carboxypeptidases, zinc-binding reg [...] ","uniprotId":"Q0P9T3_CAMJE"},"10641777":{"id":10641777,"externalId":"192222.Cj0970","name":"Cj0970","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0968, probable periplasmic protein, len: 80 aa; no Hp match. Contains probable N-terminal signal sequence. Similar to N-terminus of Cj0737 (53.3% identity in 92 aa overlap). The similarity continues in Cj0968, suggesting that this may be the start of a pseudogene; Original (2000) note: Cj0969, probable pseudogene,len 132 bp; 65.4% identical to part of Cj0737, similarity continues from upstream ORF. No Hp match; Updated (2006) note: No specific characterisation with acceptable identity scores has been carried out yet. Thus, putative kept  [...] ","uniprotId":"Q0P9T2_CAMJE"},"10641778":{"id":10641778,"externalId":"192222.Cj0971","name":"Cj0971","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0971, unknown, len: 130 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0P9T1_CAMJE"},"10641779":{"id":10641779,"externalId":"192222.Cj0972","name":"Cj0972","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0972, unknown, len: 107 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0P9T0_CAMJE"},"10641780":{"id":10641780,"externalId":"192222.Cj0973","name":"Cj0973","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0973, unknown, len: 36 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0P9S9_CAMJE"},"10641781":{"id":10641781,"externalId":"192222.Cj0974","name":"Cj0974","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0974, questionable CDS, len 46 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0P9S8_CAMJE"},"10641782":{"id":10641782,"externalId":"192222.Cj0975","name":"Cj0975","hasFamily":true,"annotation":"Original (2000) note: Cj0975, probable outer-membrane protein, len: 584 aa; similar to OM proteins involved in specific protein secretion/activation e.g. HXB1_HAEIN heme-hemopexin utilization protein B precursor (565 aa), fasta scores; opt: 489 z-score: 539.9 E(): 9e-23, 23.9% identity in 570 aa overlap, and HLYB_PROMI hemolysin activator protein precursor (561 aa),fasta scores; opt: 220 z-score: 244.2 E(): 2.7e-06, 21.0% identity in 539 aa overlap. No Hp match. Apparently a complete copy of pseudogene Cj0742; Updated (2006) note: No specific characterisation with acceptable identity s [...] ","uniprotId":"Q0P9S7_CAMJE"},"10641783":{"id":10641783,"externalId":"192222.Cj0976","name":"cmoB","hasFamily":true,"annotation":"Catalyzes carboxymethyl transfer from carboxy-S-adenosyl-L- methionine (Cx-SAM) to 5-hydroxyuridine (ho5U) to form 5- carboxymethoxyuridine (cmo5U) at position 34 in tRNAs. ","uniprotId":"CMOB_CAMJE"},"10641784":{"id":10641784,"externalId":"192222.Cj0977","name":"Cj0977","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj0977, unknown, len: 192 aa; 29.2% identity to HP0420; Updated (2006) note: Literature search identified paper linking product function to be involved with invasion of intestinal epithelial cells. Functional classification - Conserved hypothetical proteins; PMID:16369037","uniprotId":"Q0P9S5_CAMJE"},"10641785":{"id":10641785,"externalId":"192222.Cj0978c","name":"Cj0978c","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj0978c, probable lipoprotein, len: 57 aa; no Hp match. Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P9S4_CAMJE"},"10641786":{"id":10641786,"externalId":"192222.Cj0979c","name":"Cj0979c","hasFamily":true,"annotation":"Putative secreted nuclease; Original (2000) note: Cj0979c, probable secreted nuclease, len: 175 aa; similar to many e.g. NUC_STAHY thermonuclease precursor (EC 3.1.31.1) (169 aa), fasta scores; opt: 206 z-score: 261.7 E(): 2.8e-07, 36.9% identity in 122 aa overlap. No Hp match. Contains possible N-terminal signal sequence and Pfam match to entry PF00565 SNase, Staphylococcal nuclease homologues; Updated (2006) note: Some characterisation work in Staphylococcus spp. Putative kept in product function. Functional classification - Degradation of macromolecules - DNA; PMID:67041918","uniprotId":"Q0P9S3_CAMJE"},"10641787":{"id":10641787,"externalId":"192222.Cj0980","name":"Cj0980","hasFamily":true,"annotation":"Putative peptidase; Original (2000) note: Cj0980, possible peptidase,len: 422 aa; similar to e.g. PEPD_ECOLI aminoacyl-histidine dipeptidase (484 aa), fasta scores; opt: 175 z-score: 206.9 E(): 0.00032, 31.1% identity in 476 aa overlap. No Hp match; Updated (2006) note: No specific characterisation with acceptable identity score carried out yet. Putative kept within product function. Functional classification -Degradation of macromolecules - Proteins, peptides and glycopeptides; PMID:2651887","uniprotId":"Q0P9S2_CAMJE"},"10641788":{"id":10641788,"externalId":"192222.Cj0981c","name":"cjaB","hasFamily":true,"annotation":"Putative MFS (Major Facilitator Superfamily) transport protein; Original (2000) note: Cj0981c, transmembrane transport protein, len: 415 aa; similar to many members of the sugar transporter family e.g. YHJE_ECOLI hypothetical metabolite transport protein I (440 aa), fasta scores; opt: 548 z-score: 610.3 E(): 1.1e-26, 30.4% identity in 418 aa overlap, and PROP_ECOLI proline/betaine transporter (500 aa), fasta scores; opt: 474 z-score: 528.2 E(): 4e-22, 28.0% identity in 421 aa overlap. No Hp ortholog. Contains PS00216 Sugar transport protein; Updated (2006) note: Pfam domains PF07690 Ma [...] ","uniprotId":"Q0P9S1_CAMJE"},"10641789":{"id":10641789,"externalId":"192222.Cj0982c","name":"cjaA","hasFamily":true,"annotation":"Original (2000) note: Cj0982c, probable amino-acid transporter periplasmic solute-binding protein, len: 279 aa; similar to e.g. GLNH_BACST glutamine-binding protein precursor (262 aa), fasta scores; opt: 410 z-score: 484.3 E(): 1.1e-19, 32.4% identity in 253 aa overlap. 98.2% identical to TR:O05414 (EMBL:U93169) C. jejuni OMPH1, and 89.2% identical to TR:P94643 (EMBL:U93169) C. jejuni CjaA. 56.1% identity to HP1172. Contains Pfam match to entry PF00497 SBP_bac_3, Bacterial extracellular solute-binding proteins, N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic [...] ","uniprotId":"Q0P9S0_CAMJE"},"10641790":{"id":10641790,"externalId":"192222.Cj0983","name":"Cj0983","hasFamily":true,"annotation":"Original (2000) note: Cj0983, probable lipoprotein,len: 372 aa; 97.0% identical to YHI4_CAMJE. No Hp match. Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Y983_CAMJE"},"10641791":{"id":10641791,"externalId":"192222.Cj0984","name":"Cj0984","hasFamily":true,"annotation":"Uncharacterized protein Cj0984; Original (2000) note: Cj0984, unknown, len: 246 aa; 97.2% identical to YHI3_CAMJE. No Hp match. Weak similarity to TR:O66885 (EMBL:AE000700) Aquifex aeolicus AQ_644 (227 aa), fasta scores; opt: 154 z-score: 190.3 E(): 0.0027, 23.6% identity in 212 aa overlap; Updated (2006) note: Pfam domain PF03883 Protein of unknown function (DUF328) identified within CDS. Conserved added to product function. Functional classification -Conserved hypothetical proteins","uniprotId":"Y984_CAMJE"},"10641792":{"id":10641792,"externalId":"192222.Cj0985c","name":"hipO","hasFamily":true,"annotation":"Cleaves hippuric acid into benzoic acid and glycine. ","uniprotId":"HIPO_CAMJE"},"10641793":{"id":10641793,"externalId":"192222.Cj0986c","name":"Cj0986c","hasFamily":false,"annotation":"Original (2000) note: Cj0986c, possible integral membrane protein, len: 62 aa; 98.4% identical to C-terminus of YHI2_CAMJE hypothetical protein (203 aa). No Hp match; Updated (2006) note: Two probable transmembrane helices predicted for Cj0986c by TMHMM2.0. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P9R6_CAMJE"},"10641794":{"id":10641794,"externalId":"192222.Cj0987c","name":"Cj0987c","hasFamily":true,"annotation":"Uncharacterized protein Cj0987c; Original (2000) note: Cj0987c, probable integral membrane protein, len: 321 aa; identical to N- and C-terminus of YHI2_CAMJE hypothetical protein (203 aa),which has a central deletion relative to this CDS. Some similarity to hypothetical membrane proteins e.g. YJHB_ECOLI (405 aa), fasta scores; opt: 239 z-score: 285.2 E(): 1.4e-08, 24.6% identity in 301 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF07690 Major Facilitator Superfamily identified within CDS. Also, nine probable transmembrane helices predicted by TMHMM2.0. Product modified to [...] ","uniprotId":"Y987_CAMJE"},"10641795":{"id":10641795,"externalId":"192222.Cj0988c","name":"Cj0988c","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj0988c, questionable CDS,len: 78 aa; 98.3% identical to TR:Q46117 (EMBL:Z36940) C. jejuni hypothetical protein. No Hp match; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Functional classification -Unknown","uniprotId":"Q0P9R4_CAMJE"},"10641796":{"id":10641796,"externalId":"192222.Cj0989","name":"Cj0989","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj0989, possible membrane protein, len: 68 aa; identical to TR:Q46118 (EMBL:Z36940) C. jejuni hypothetical protein. No Hp match; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P9R3_CAMJE"},"10641797":{"id":10641797,"externalId":"192222.Cj0990c","name":"Cj0990c","hasFamily":true,"annotation":"Uncharacterized protein Cj0990c; Original (2000) note: Cj0990c, unknown, len: 253 aa; identical in C-terminus to YHI1_CAMJE C. jejuni hypothetical protein fragment (94 aa). No Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Y990_CAMJE"},"10641798":{"id":10641798,"externalId":"192222.Cj0991c","name":"Cj0991c","hasFamily":true,"annotation":"Original (2000) note: Cj0991c, probable oxidoreductase ferredoxin-type electron transport protein,len: 421; weak similarity to e.g. GLPC_ECOLI anaerobic glycerol-3-phosphate dehydrogenase subunit C (396 aa),fasta scores; opt: 280 z-score: 323.3 E(): 1e-10, 23.3% identity in 425 aa overlap, and GLCF_ECOLI glycolate oxidase iron-sulfur subunit (407 aa), fasta scores; opt: 199 z-score: 231.5 E(): 1.4e-05, 22.3% identity in 413 aa overlap. 44.7% identity to HP0666 (annotated as glpC, but there are no homologs of glpA or glpB in Hp or Cj). Contains PS00198 4Fe-4S ferredoxins, iron-sulfur bi [...] ","uniprotId":"Q0P9R1_CAMJE"},"10641799":{"id":10641799,"externalId":"192222.Cj0992c","name":"Cj0992c","hasFamily":true,"annotation":"Belongs to the anaerobic coproporphyrinogen-III oxidase family","uniprotId":"Q0P9R0_CAMJE"},"10641800":{"id":10641800,"externalId":"192222.Cj0993c","name":"Cj0993c","hasFamily":true,"annotation":"Belongs to the UPF0763 family","uniprotId":"Y993_CAMJE"},"10641801":{"id":10641801,"externalId":"192222.Cj0994c","name":"argF","hasFamily":true,"annotation":"Reversibly catalyzes the transfer of the carbamoyl group from carbamoyl phosphate (CP) to the N(epsilon) atom of ornithine (ORN) to produce L-citrulline, which is a substrate for argininosuccinate synthetase (ArgG) involved in the final step in arginine biosynthesis. ","uniprotId":"OTCA_CAMJE"},"10641802":{"id":10641802,"externalId":"192222.Cj0995c","name":"hemB","hasFamily":true,"annotation":"Belongs to the ALAD family. ","uniprotId":"Q0P9Q7_CAMJE"},"10641803":{"id":10641803,"externalId":"192222.Cj0996","name":"ribA","hasFamily":true,"annotation":"GTP cyclohydrolase II; Original (2000) note: Cj0996, ribA, probable GTP cyclohydrolase II, len: 186 aa; similar to many e.g. GCH2_ECOLI GTP cyclohydrolase II (EC 3.5.4.25) (196 aa),fasta scores; opt: 569 z-score: 678.3 E(): 1.8e-30, 47.2% identity in 193 aa overlap. 53.2% identity to HP0802. Contains Pfam match to entry PF00925 GTP_cyclohydro2, GTP cyclohydrolase II; Updated (2006) note: Characterised in Escherichia coli with acceptable identity score, so putative not added to product function. Functional classification -Biosynthesis of cofactors, prosthetic groups and carriers - Ribof [...] ","uniprotId":"Q0P9Q6_CAMJE"},"10641804":{"id":10641804,"externalId":"192222.Cj0997","name":"rsmG","hasFamily":true,"annotation":"Specifically methylates the N7 position of guanine in position 527 of 16S rRNA","uniprotId":"RSMG_CAMJE"},"10641805":{"id":10641805,"externalId":"192222.Cj0998c","name":"Cj0998c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj0998c, probable periplasmic protein, len: 190 aa; 33.9% identity to HP0305. Contains probable N-terminal signal sequence; Updated (2006) note: Paper identified linking protein to glycoprotein. Functional classification -Miscellaneous periplasmic proteins; PMID:12186869","uniprotId":"Q0P9Q4_CAMJE"},"10641806":{"id":10641806,"externalId":"192222.Cj0999c","name":"Cj0999c","hasFamily":true,"annotation":"Original (2000) note: Cj0999c, probable integral membrane protein, len: 365 aa; similar to hypothetical membrane proteins e.g. YEIH_ECOLI (349 aa), fasta scores; opt: 631 z-score: 729.2 E(): 0, 35.7% identity in 347 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF03601 Conserved hypothetical protein identified within CDS. Also, eleven probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Y999_CAMJE"},"10641807":{"id":10641807,"externalId":"192222.Cj1000","name":"Cj1000","hasFamily":true,"annotation":"Belongs to the LysR transcriptional regulatory family. ","uniprotId":"Q0P9Q2_CAMJE"},"10641808":{"id":10641808,"externalId":"192222.Cj1001","name":"rpoD","hasFamily":true,"annotation":"Sigma factors are initiation factors that promote the attachment of RNA polymerase to specific initiation sites and are then released. This sigma factor is the primary sigma factor during exponential growth","uniprotId":"Q0P9Q1_CAMJE"},"10641809":{"id":10641809,"externalId":"192222.Cj1002c","name":"Cj1002c","hasFamily":true,"annotation":"Putative phosphoglycerate/bisphosphoglycerate mutase; Original (2000) note: Cj1002c, unknown, len: 170 aa; similar to hypothetical proteins e.g. TR:O67234 (EMBL:AE000727) Aquifex aeolicus AQ_1173 (149 aa), fasta scores; opt: 185 z-score: 235.0 E(): 8.6e-06, 34.2% identity in 117 aa overlap. No Hp match; Updated (2006) note: Running own Pfam search allowed identification of Pfam domain PF00300 Phosphoglycerate mutase family. This is an enzyme that catalyses reactions involving the transfer of phospho groups between the 3 carbon atoms of phosphoglycerate. Not specifically characterised,  [...] ","uniprotId":"Q0P9Q0_CAMJE"},"10641810":{"id":10641810,"externalId":"192222.Cj1003c","name":"Cj1003c","hasFamily":true,"annotation":"Original (2000) note: Cj1003c, probable integral membrane protein, len: 172 aa; similar to hypothetical proteins e.g. TR:O59166 (EMBL:AP000006) Pyrococcus horikoshii PH1497 (197 aa), fasta scores; opt: 233 z-score: 292.8 E(): 5.2e-09, 30.8% identity in 172 aa overlap. No Hp ortholog; Updated (2006) note: Pfam domain PF01694 Rhomboid family identified within CDS. Also, five probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. No specific charactersation carried out yet, so putative kept in product function. Functional classification - Membrane [...] ","uniprotId":"Q0P9P9_CAMJE"},"10641811":{"id":10641811,"externalId":"192222.Cj1004","name":"Cj1004","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1004, probable periplasmic protein, len: 138 aa; very weak similarity to TR:P75137 (EMBL:AE000019) Mycoplasma pneumoniae putative lipoprotein E09_orf129 (129 aa), fasta scores; opt: 110 z-score: 147.3 E(): 0.66, 34.9% identity in 129 aa overlap. No Hp match. Also similar to Cj1626c (38.6% identity in 140 aa overlap); Updated (2006) note: Literature search identified paper giving further clues on product funciton. Functional classification - Miscellaneous periplasmic proteins; PMID:15554967","uniprotId":"Q0P9P8_CAMJE"},"10641812":{"id":10641812,"externalId":"192222.Cj1005c","name":"Cj1005c","hasFamily":true,"annotation":"Belongs to the AAA ATPase family. ","uniprotId":"Q0P9P7_CAMJE"},"10641813":{"id":10641813,"externalId":"192222.Cj1006c","name":"Cj1006c","hasFamily":true,"annotation":"Putative MiaB-like tRNA modifying enzyme; Original (2000) note: Cj1006c, unknown, len: 416 aa; similar to hypothetical proteins belonging to the UPF0004 family e.g. Y16R_MYCIO (438 aa), fasta scores; opt: 580 z-score: 664.7 E(): 1e-29, 30.5% identity in 413 aa overlap. 50.1% identity to HP0285. Also similar to Cj0458c (28.6% identity in 391 aa overlap) and Cj1454c (28.4% identity in 419 aa overlap). Contains PS01278 Uncharacterized protein family UPF0004 signature and Pfam match to entry PF00919 UPF0004; Updated (2006) note: Pfam domain PF04055 Radical SAM superfamily identified within [...] ","uniprotId":"Q0P9P6_CAMJE"},"10641814":{"id":10641814,"externalId":"192222.Cj1007c","name":"Cj1007c","hasFamily":true,"annotation":"Putative mechanosensitive ion channel family protein; Original (2000) note: Cj1007c, probable membrane protein, len: 523 aa; similar in C-terminus to hypothetical proteins e.g. Y700_METJA hypothetical protein MJ0700 (324 aa), fasta scores; opt: 281 z-score: 300.2 E(): 2e-09, 29.0% identity in 276 aa overlap. 37.4% identity to HP0284; Updated (2006) note: Pfam domain PF00924 Mechanosensitive ion channel identified within CDS. Also,three probable transmembrane helices predicted by TMHMM2.0. Product modified to more specific family member based on motif match. No specific characterisation [...] ","uniprotId":"Q0P9P5_CAMJE"},"10641815":{"id":10641815,"externalId":"192222.Cj1008c","name":"aroB","hasFamily":true,"annotation":"Catalyzes the conversion of 3-deoxy-D-arabino-heptulosonate 7-phosphate (DAHP) to dehydroquinate (DHQ)","uniprotId":"AROB_CAMJE"},"10641816":{"id":10641816,"externalId":"192222.Cj1009c","name":"Cj1009c","hasFamily":true,"annotation":"Conserved hypothetical proetin Cj1009c; Original (2000) note: Cj1009c, unknown, len: 466 aa; 29.1% identity to HP0282; Updated (2006) note: Pfam domain PF02080 TrkA-C domain identified within CDS. The TrkA-C domain is typically found next to the -N domain. Currently unknown function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P9P3_CAMJE"},"10641817":{"id":10641817,"externalId":"192222.Cj1010","name":"tgt","hasFamily":true,"annotation":"Catalyzes the base-exchange of a guanine (G) residue with the queuine precursor 7-aminomethyl-7-deazaguanine (PreQ1) at position 34 (anticodon wobble position) in tRNAs with GU(N) anticodons (tRNA-Asp, -Asn, -His and -Tyr). Catalysis occurs through a double-displacement mechanism. The nucleophile active site attacks the C1' of nucleotide 34 to detach the guanine base from the RNA, forming a covalent enzyme-RNA intermediate. The proton acceptor active site deprotonates the incoming PreQ1, allowing a nucleophilic attack on the C1' of the ribose to form the product. After dissociation, tw [...] ","uniprotId":"TGT_CAMJE"},"10641818":{"id":10641818,"externalId":"192222.Cj1011","name":"Cj1011","hasFamily":true,"annotation":"Putative CorA-like Mg2+ transporter protein; Original (2000) note: Cj1011, probable membrane protein, len: 255 aa; no Hp match; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Running own Pfam search identified PF01544 CorA-like Mg2+ transporter protein. Product modified to more specific family member based on motif match. No specific characterisation has been carried out yet. Putative kept in product function. Functional classification - Transport/binding proteins - Cations","uniprotId":"Q0P9P1_CAMJE"},"10641819":{"id":10641819,"externalId":"192222.Cj1012c","name":"Cj1012c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj1012c, possible membrane protein, len: 149 aa; no Hp match; Updated (2006) note: One probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P9P0_CAMJE"},"10641820":{"id":10641820,"externalId":"192222.Cj1013c","name":"Cj1013c","hasFamily":true,"annotation":"Putative cytochrome C biogenesis protein; Original (2000) note: Cj1013c, probable membrane protein, len: 1081 aa; contains three domains; aa 1-90 contains three membrane spanning domains; aa 90-780 is non-membrane, and aa 780-1081 contains ten possible membrane spanning domains. Some similarity in C-teminal membrane domain to eukaryotic proteins e.g. CCSA_CHLRE cytochrome C biogenesis protein CCSA (353 aa), fasta scores; opt: 459 z-score: 515.4 E(): 2.1e-21, 36.0% identity in 267 aa overlap, and to prokaryotic equivalents e.g. CCMF_ECOLI cytochrome C-type biogenesis protein CCMF (647 a [...] ","uniprotId":"Q0P9N9_CAMJE"},"10641821":{"id":10641821,"externalId":"192222.Cj1014c","name":"livF","hasFamily":true,"annotation":"Original (2000) note: Cj1014c, livF, probable branched-chain amino-acid ABC transport system ATP-binding protein, len: 231 aa; similar to e.g. LIVF_ECOLI high-affinity branched-chain amino acid transport ATP-binding protein (237 aa), fasta scores; opt: 686 z-score: 797.6 E(): 0, 46.4% identity in 233 aa overlap. No Hp ortholog. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran; Updated (2006) note: Characterised within Pseudomonas aeruginosa and Escherichia coli with acceptable identity scores. Pu [...] ","uniprotId":"Q0P9N8_CAMJE"},"10641822":{"id":10641822,"externalId":"192222.Cj1015c","name":"livG","hasFamily":true,"annotation":"Original (2000) note: Cj1015c, livG, probable branched-chain amino-acid ABC transport system ATP-binding protein, len: 256 aa; similar to e.g. LIVG_ECOLI high-affinity branched-chain amino acid transport ATP-binding protein (255 aa), fasta scores; opt: 680 z-score: 800.2 E(): 0, 41.2% identity in 250 aa overlap. No Hp ortholog. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran; Updated (2006) note: Characterised within Pseudomonas aeruginosa and Escherichia coli with acceptable identity scores. Pu [...] ","uniprotId":"Q0P9N7_CAMJE"},"10641823":{"id":10641823,"externalId":"192222.Cj1016c","name":"livM","hasFamily":true,"annotation":"Belongs to the binding-protein-dependent transport system permease family","uniprotId":"Q0P9N6_CAMJE"},"10641824":{"id":10641824,"externalId":"192222.Cj1017c","name":"livH","hasFamily":true,"annotation":"Belongs to the binding-protein-dependent transport system permease family","uniprotId":"Q0P9N5_CAMJE"},"10641825":{"id":10641825,"externalId":"192222.Cj1018c","name":"livK","hasFamily":true,"annotation":"Original (2000) note: Cj1018c, livK, probable branched-chain amino-acid ABC transport system periplasmic binding protein, len: 369 aa; similar to e.g. LIVK_ECOLI LEU/ILE/VAL-binding protein precursor (367 aa), fasta scores; opt: 386 z-score: 436.5 E(): 5.1e-17, 28.3% identity in 272 aa overlap. No Hp match. Highly similar to Cj1019c (62.0% identity in 371 aa overlap); Updated (2006) note: Pfam domain PF01094 Receptor family ligand binding region identified within CDS. Further support given to product function. Paper identified linking protein to glycoprotein (PMID:12186869). Characteri [...] ","uniprotId":"Q0P9N4_CAMJE"},"10641826":{"id":10641826,"externalId":"192222.Cj1019c","name":"livJ","hasFamily":true,"annotation":"Original (2000) note: Cj1019c, livJ, probable branched-chain amino-acid ABC transport system periplasmic binding protein, len: 369 aa; similar to e.g. LIVJ_ECOLI LEU/ILE/VAL-binding protein precursor (367 aa), fasta scores; opt: 419 z-score: 465.8 E(): 1.2e-18, 27.8% identity in 352 aa overlap. No Hp match. Highly similar to Cj1018c (62.0% identity in 371 aa overlap); Updated (2006) note: Pfam domain PF01094 Receptor family ligand binding region identified within CDS. Further support given to producnt fucntion. Characterised in Escherichia coli with marginal identity score. Appropriate [...] ","uniprotId":"Q0P9N3_CAMJE"},"10641827":{"id":10641827,"externalId":"192222.Cj1020c","name":"Cj1020c","hasFamily":true,"annotation":"Putative cytochrome C; Original (2000) note: Cj1020c, probable cytochrome C, len: 144 aa; similar to e.g.C553_DESDN cytochrome C-553 (80 aa), fasta scores; opt: 142 z-score: 184.4 E(): 0.0056, 30.8% identity in 78 aa overlap. Also similar to Cj1153 (27.6% identity in 76 aa overlap). No Hp ortholog. Contains N-terminal signal sequence, PS00190 Cytochrome c family heme-binding site signature, and Pfam match to entry PF00034 cytochrome_c; Updated (2006) note: Putative kept in product function as no specific characterisation has been carried out. Functional classification - Energy metaboli [...] ","uniprotId":"Q0P9N2_CAMJE"},"10641828":{"id":10641828,"externalId":"192222.Cj1021c","name":"Cj1021c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1021c, probable periplasmic protein, len: 63 aa; similar to the N-terminus of C. jejuni major outer membrane protein TR:G2529699 (EMBL:U96452) (MOMP_CAMJE) momp, porA, Cj1259 (424 aa),fasta scores; opt: 110 z-score: 165.3 E(): 0.066, 36.4% identity in 55 aa overlap. No Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF05538 Campylobacter major outer membrane protein identified when running own Pfam search. Some characterisation work within Cj1259, however, alignment was only partial. Putative [...] ","uniprotId":"Q0P9N1_CAMJE"},"10641829":{"id":10641829,"externalId":"192222.Cj1022c","name":"Cj1022c","hasFamily":true,"annotation":"Original (2000) note: Cj1022c, probable integral membrane protein, len: 168 aa; similar to e.g. TR:O66639 (EMBL:AE000682) Aquifex aeolicus AQ_286 (175 aa), fasta scores; opt: 354 z-score: 451.1 E(): 7.9e-18, 34.3% identity in 166 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF07053 Protein of unknown function (DUF1334) identified within CDS. Also,four probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P9N0_CAMJE"},"10641830":{"id":10641830,"externalId":"192222.Cj1023c","name":"asd","hasFamily":true,"annotation":"Catalyzes the NADPH-dependent formation of L-aspartate- semialdehyde (L-ASA) by the reductive dephosphorylation of L-aspartyl- 4-phosphate","uniprotId":"DHAS_CAMJE"},"10641831":{"id":10641831,"externalId":"192222.Cj1024c","name":"flgR","hasFamily":true,"annotation":"Sigma-54 associated transcriptional activator; Original (2000) note: Cj1024c, probable signal-transduction regulatory protein, len: 433 aa; similar to e.g. NTRC_PROVU nitrogen regulation protein NR(I) (473 aa), fasta scores; opt: 957 z-score: 1002.5 E(): 0, 38.8% identity in 397 aa overlap, and ATOC_ECOLI acetoacetate metabolism regulatory protein (461 aa), fasta scores; opt: 929 z-score: 973.5 E(): 0, 38.2% identity in 453 aa overlap. 57.1% identity to HP0703. Contains Pfam match to entry PF00072 response_reg, Response regulator receiver domain, PS00675 and PS00676 Sigma-54 interactio [...] ","uniprotId":"Q0P9M8_CAMJE"},"10641832":{"id":10641832,"externalId":"192222.Cj1025c","name":"Cj1025c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1025c, unknown, len: 149 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0P9M7_CAMJE"},"10641833":{"id":10641833,"externalId":"192222.Cj1026c","name":"Cj1026c","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj1026c, probable lipoprotein, len: 171 aa; 51.0% identity in N-terminus to HP0837 (102 aa). Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site; Updated (2006) note: Pfam domain PF04164 Protein of unknown function, DUF400 identified within CDS. Literature search identified paper linking product to motility. Functional classification - Membranes, lipoproteins and porins; PMID:14985343","uniprotId":"Q0P9M6_CAMJE"},"10641834":{"id":10641834,"externalId":"192222.Cj1027c","name":"gyrA","hasFamily":true,"annotation":"A type II topoisomerase that negatively supercoils closed circular double-stranded (ds) DNA in an ATP-dependent manner to modulate DNA topology and maintain chromosomes in an underwound state. Negative supercoiling favors strand separation, and DNA replication, transcription, recombination and repair, all of which involve strand separation. Also able to catalyze the interconversion of other topological isomers of dsDNA rings, including catenanes and knotted rings. Type II topoisomerases break and join 2 DNA strands simultaneously in an ATP-dependent manner","uniprotId":"GYRA_CAMJE"},"10641835":{"id":10641835,"externalId":"192222.Cj1028c","name":"Cj1028c","hasFamily":true,"annotation":"Original (2000) note: Cj1028c, possible purine/pyrimidine phosphoribosyltransferase, len: 191 aa; weak similarity to e.g. orotate phosphoribosyltransferase domain of PYR5_HUMAN uridine 5'-monophosphate synthase (480 aa), fasta scores; opt: 114 z-score: 145.4 E(): 0.85,25.5% identity in 149 aa overlap. 37.3% identity to HP1473. Contains PS00103 Purine/pyrimidine phosphoribosyl transferases signature; Updated (2006) note: Pfam domain PF00156 Phosphoribosyl transferase domain identified within CDS. Further support given to product function. Product kept the same as no specific characteris [...] ","uniprotId":"Q0P9M4_CAMJE"},"10641836":{"id":10641836,"externalId":"192222.Cj1029c","name":"mapA","hasFamily":true,"annotation":"Original (2000) note: Cj1029c, mapA, probable lipoprotein, len: 214 aa; identical to TR:Q46122 (EMBL:X80135) C. jeuni species specific mapA gene (214 aa). No Hp match. Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site; Updated (2006) note: Studied in Campylobacter jejuni. No specific characterisation has been carried out yet. Functional classification - Membranes, lipoproteins and porins; PMID:7790451","uniprotId":"MAPA_CAMJE"},"10641837":{"id":10641837,"externalId":"192222.Cj1030c","name":"lepA","hasFamily":true,"annotation":"Required for accurate and efficient protein synthesis under certain stress conditions. May act as a fidelity factor of the translation reaction, by catalyzing a one-codon backward translocation of tRNAs on improperly translocated ribosomes. Back-translocation proceeds from a post-translocation (POST) complex to a pre- translocation (PRE) complex, thus giving elongation factor G a second chance to translocate the tRNAs correctly. Binds to ribosomes in a GTP- dependent manner","uniprotId":"LEPA_CAMJE"},"10641838":{"id":10641838,"externalId":"192222.Cj1031","name":"cmeD","hasFamily":true,"annotation":"Original (2000) note: Cj1031, possible outer membrane component of efflux system, len: 424 aa; similar to members of the PRTF family of secretion proteins e.g. TR:O68960 (EMBL:AF059041) putative outer membrane pore protein hefA (=HP0605) (477 aa), fasta scores; opt: 238 z-score: 261.5 E(): 2.9e-07, 25.2% identity in 409 aa overlap, and TOLC_SALEN outer membrane protein TOLC precursor (491 aa), fasta scores; opt: 174 z-score: 192.9 E(): 0.0019, 21.5% identity in 437 aa overlap. 24.8% identity to HP0605. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domains x2 P [...] ","uniprotId":"Q0P9M1_CAMJE"},"10641839":{"id":10641839,"externalId":"192222.Cj1032","name":"cmeE","hasFamily":true,"annotation":"Belongs to the membrane fusion protein (MFP) (TC 8.A.1) family","uniprotId":"Q0P9M0_CAMJE"},"10641840":{"id":10641840,"externalId":"192222.Cj1033","name":"cmeF","hasFamily":true,"annotation":"Belongs to the resistance-nodulation-cell division (RND) (TC 2.A.6) family","uniprotId":"Q0P9L9_CAMJE"},"10641841":{"id":10641841,"externalId":"192222.Cj1034c","name":"Cj1034c","hasFamily":true,"annotation":"Putative DnaJ-like protein; Original (2000) note: Cj1034c, possible dnaJ-like protein, len: 268 aa; no Hp match. Some similarity to dnaJ-like protein Cj0954c (23.2% identity in 254 aa overlap). Contains weak Pfam match to entry PF00226 DnaJ,DnaJ domain; Updated (2006) note: Prosite domain PS50076 DNAJ_2,Heat shock protein DnaJ, N-terminal identified within CDS. No specific characterisation has been carried out yet, so putative kept in product function. Functional classification - Chaperones, chaperonins, heat shock","uniprotId":"Q0P9L8_CAMJE"},"10641842":{"id":10641842,"externalId":"192222.Cj1035c","name":"ate","hasFamily":true,"annotation":"Functions in the N-end rule pathway of protein degradation where it conjugates Leu from its aminoacyl-tRNA to the N-termini of proteins containing an N-terminal aspartate or glutamate. ","uniprotId":"BPT_CAMJE"},"10641843":{"id":10641843,"externalId":"192222.Cj1036c","name":"Cj1036c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1036c, possible transcriptional regulator, len: 83 aa; no Hp match. Contains probable helix-turn-helix motif at aa 19-40 (Score 1070, +2.83 SD); Updated (2006) note: Conserved added to product function. Functional classification - Broad regulatory functions","uniprotId":"Q0P9L6_CAMJE"},"10641844":{"id":10641844,"externalId":"192222.Cj1037c","name":"pycA","hasFamily":true,"annotation":"This protein is a component of the acetyl coenzyme A carboxylase complex; first, biotin carboxylase catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA. ","uniprotId":"Q0P9L5_CAMJE"},"10641845":{"id":10641845,"externalId":"192222.Cj1038","name":"Cj1038","hasFamily":true,"annotation":"Belongs to the SEDS family. ","uniprotId":"Q0P9L4_CAMJE"},"10641846":{"id":10641846,"externalId":"192222.Cj1039","name":"murG","hasFamily":true,"annotation":"Cell wall formation. Catalyzes the transfer of a GlcNAc subunit on undecaprenyl-pyrophosphoryl-MurNAc-pentapeptide (lipid intermediate I) to form undecaprenyl-pyrophosphoryl-MurNAc- (pentapeptide)GlcNAc (lipid intermediate II)","uniprotId":"MURG_CAMJE"},"10641847":{"id":10641847,"externalId":"192222.Cj1040c","name":"Cj1040c","hasFamily":true,"annotation":"Putative MFS (Major Facilitator Superfamily) transport protein; Original (2000) note: Cj1040c, probable transmembrane transport protein, len: 387 aa; simlar to hypothetical membrane proteins e.g. YEAN_ECOLI (393 aa),fasta scores; opt: 641 z-score: 726.5 E(): 0, 31.2% identity in 381 aa overlap, and to CYNX_ECOLI cyanate transport protein CYNX (384 aa), fasta scores; opt: 421 z-score: 480.1 E(): 1.9e-19, 23.9% identity in 368 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF07690 Major Facilitator Superfamily (MFS) protein identified within CDS. Also, Twelve probable transmem [...] ","uniprotId":"Q0P9L2_CAMJE"},"10641848":{"id":10641848,"externalId":"192222.Cj1041c","name":"Cj1041c","hasFamily":true,"annotation":"Original (2000) note: Cj1041c, probable periplasmic ATP/GTP-binding protein, len: 283 aa; no Hp match. Contains probable N-terminal signal sequence and PS00017 ATP/GTP-binding site motif A (P-loop). Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0P9L1_CAMJE"},"10641849":{"id":10641849,"externalId":"192222.Cj1042c","name":"Cj1042c","hasFamily":true,"annotation":"Original (2000) note: Cj1042c, probable transcriptional regulatory protein, len: 296 aa; similar to many members of the ARAC/XYLS family e.g. VIRF_YEREN VIRULENCE regulon transcriptional activator (271 aa),fasta scores; opt: 220 z-score: 270.6 E(): 9e-08, 23.9% identity in 247 aa overlap, and LCRF_YERPE thermoregulatory protein LCRF (271 aa), fasta scores; opt: 219 z-score: 269.4 E(): 1e-07, 23.5% identity in 247 aa overlap. No Hp match. Contains PS00041 Bacterial regulatory proteins, araC family signature, and Pfam match to entry PF00165 HTH_2, Bacterial regulatory helix-turn-helix pr [...] ","uniprotId":"Q0P9L0_CAMJE"},"10641850":{"id":10641850,"externalId":"192222.Cj1043c","name":"Cj1043c","hasFamily":true,"annotation":"Putative thiamine-phosphate pyrophosphorylase; Original (2000) note: Cj1043c, possible transferase, len: 201 aa; similar to e.g. TENI_BACSU regulatory protein TENI (205 aa), fasta scores; opt: 217 z-score: 277.4 E(): 3.8e-08, 26.3% identity in 171 aa overlap, and THIE_ECOLI thiamin-phosphate pyrophosphorylase (211 aa), fasta scores; opt: 146 z-score: 190.3 E(): 0.0026, 27.9% identity in 154 aa overlap. No Hp ortholog. Also similar to Cj1081c (30.0% identity in 190 aa overlap); Updated (2006) note: Pfam domain PF02581 Thiamine monophosphate synthase/TENI identified by running own Pfam s [...] ","uniprotId":"Q0P9K9_CAMJE"},"10641851":{"id":10641851,"externalId":"192222.Cj1044c","name":"thiH","hasFamily":true,"annotation":"Original (2000) note: Cj1044c, thiH, probable thiamine biosynthesis protein, len: 381 aa; similar to THIH_ECOLI THIH protein (377 aa), fasta scores; opt: 785 z-score: 909.4 E(): 0, 37.4% identity in 350 aa overlap. No Hp match; Updated (2006) note: Pfam domains PF06968 Biotin and Thiamin Synthesis associated domain and PF04055 Radical SAM superfamily were identified within CDS. Further support given to product function. Characterised in Escherichia coli with acceptable identity score, so putative not added to product function. Based on recent characterisation papers within Escherichia  [...] ","uniprotId":"Q0P9K8_CAMJE"},"10641852":{"id":10641852,"externalId":"192222.Cj1045c","name":"thiG","hasFamily":true,"annotation":"Catalyzes the rearrangement of 1-deoxy-D-xylulose 5-phosphate (DXP) to produce the thiazole phosphate moiety of thiamine. Sulfur is provided by the thiocarboxylate moiety of the carrier protein ThiS. In vitro, sulfur can be provided by H(2)S","uniprotId":"THIG_CAMJE"},"10641853":{"id":10641853,"externalId":"192222.Cj1046c","name":"moeB","hasFamily":true,"annotation":"Original (2000) note: Cj1046c, moeB, probable molybdopterin biosynthesis protein, len: 267 aa; similar to e.g. MOEB_ECOLI molybdopterin biosynthesis MOEB protein (249 aa), fasta scores; opt: 256 z-score: 309.6 E(): 6e-1,28.6% identity in 192 aa overlap. 34.2% identity to HP0814. Contains Pfam match to entry PF00899 ThiF_family; Updated (2006) note: Characterisation has been carried out in Escerichia coli, however, identity score was marginal, so putative kept in product function. Recent literature search has shown strong similarity with ThiF protein. Functional classification - Biosynt [...] ","uniprotId":"Q0P9K6_CAMJE"},"10641854":{"id":10641854,"externalId":"192222.Cj1047c","name":"thiS","hasFamily":true,"annotation":"Putative thiamine biosynthesis protein; Original (2000) note: Cj1047c, unknown, len: 63 aa; similar to hypothetical proteins e.g. YC40_ODOSI (73 aa),fasta scores; opt: 131 z-score: 200.4 E(): 0.00072, 37.1% identity in 62 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02597 ThiS family identified within CDS. Product modified to a more specific family member based on motif match. Characterisation has been carried out within Escherichia coli and Bacillus subtilis with marginal simlilarity score. Putativae kept in product function. Functional classification - Biosynthesis of  [...] ","uniprotId":"Q0P9K5_CAMJE"},"10641855":{"id":10641855,"externalId":"192222.Cj1048c","name":"dapE","hasFamily":true,"annotation":"Catalyzes the hydrolysis of N-succinyl-L,L-diaminopimelic acid (SDAP), forming succinate and LL-2,6-diaminoheptanedioate (DAP), an intermediate involved in the bacterial biosynthesis of lysine and meso-diaminopimelic acid, an essential component of bacterial cell walls","uniprotId":"DAPE_CAMJE"},"10641856":{"id":10641856,"externalId":"192222.Cj1049c","name":"Cj1049c","hasFamily":true,"annotation":"Putative LysE family transporter protein; Original (2000) note: Cj1049c, probable integral membrane protein, len: 199 aa; similar to hypothetical membrane proteins belonging to the UPF0056 (MARC) family e.g. YVBG_BACSU (211 aa), fasta scores; opt: 173 z-score: 220.6 E(): 5.4e-05, 32.8% identity in 189 aa overlap. No Hp match; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Also, Pfam domain PF01810 LysE type translocator identified within CDS. Lysine exporter protein is involved in the efflux of excess L-lysine as a control for intracellular levels of L-l [...] ","uniprotId":"Q0P9K3_CAMJE"},"10641857":{"id":10641857,"externalId":"192222.Cj1050c","name":"npdA","hasFamily":true,"annotation":"NAD-dependent lysine deacetylase and desuccinylase that specifically removes acetyl and succinyl groups on target proteins. Modulates the activities of several proteins which are inactive in their acylated form","uniprotId":"NPD_CAMJE"},"10641858":{"id":10641858,"externalId":"192222.Cj1051c","name":"cjeI","hasFamily":true,"annotation":"Restriction modification enzyme; Original (2000) note: Cj1051c, probable restriction modification enzyme, len: 1339 aa; similar in N-terminal 70% to TR:O85299 (EMBL:AF060542) Haemophilus influenzae biogroup aegyptius HaeIV restriction/modification system (953 aa), fasta scores; opt: 1818 z-score: 1949.8 E(): 0 42.7% identity in 976 aa overlap, and in extreme C-terminus to e.g. T1SA_ECOLI type I restriction enzyme ECOAI specificity protein (589 aa), fasta scores; opt: 393 z-score: 422.5 E(): 3.1e-16, 40.4% identity in 178 aa overlap. No Hp ortholog. Contains PS00092 N-6 Adenine-specific [...] ","uniprotId":"Q0P9K1_CAMJE"},"10641859":{"id":10641859,"externalId":"192222.Cj1052c","name":"mutS","hasFamily":true,"annotation":"Endonuclease that is involved in the suppression of homologous recombination and may therefore have a key role in the control of bacterial genetic diversity","uniprotId":"Q0P9K0_CAMJE"},"10641860":{"id":10641860,"externalId":"192222.Cj1053c","name":"Cj1053c","hasFamily":true,"annotation":"Original (2000) note: Cj1053c, possible integral membrane protein, len: 116 aa; 25.4% identity to HP0622; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P9J9_CAMJE"},"10641861":{"id":10641861,"externalId":"192222.Cj1054c","name":"murC","hasFamily":true,"annotation":"Cell wall formation","uniprotId":"MURC_CAMJE"},"10641862":{"id":10641862,"externalId":"192222.Cj1055c","name":"Cj1055c","hasFamily":true,"annotation":"Putative sulfatase family protein; Original (2000) note: Cj1055c, probable integral membrane protein, len: 657 aa; weakly similar to hypothetical proteins, e.g. YQGS_BACSU (638 aa), fasta scores; opt: 197 z-score: 215.2 E(): 0.00011, 22.1% identity in 602 aa overlap. 24.3% identity to HP0578. Contains PS00044 Bacterial regulatory proteins, lysR family signature, though this may be spurious; Updated (2006) note: Pfam domain PF00884 Sulfatase identified within CDS. Sulfatases are enzymes that hydrolyze various sulphate esters. Five probable transmembrane helices predicted by TMHMM2.0. Pr [...] ","uniprotId":"Q0P9J7_CAMJE"},"10641863":{"id":10641863,"externalId":"192222.Cj1056c","name":"Cj1056c","hasFamily":true,"annotation":"Putative carbon-nitrogen hydrolase family protein; Original (2000) note: Cj1056c, unknown, len: 258 aa; similar to hypothetical proteins e.g. TR:O59829 (EMBL:AL023590) Schizosaccharomyces pombe putative nitrilase SPCC965.09 (272 aa), fasta scores; opt: 175 z-score: 224.3 E(): 3.4e-05, 25.1% identity in 187 aa overlap. 38.8% identity to HP1481; Updated (2006) note: Prosite domain PS50263 CN_HYDROLASE, Carbon-nitrogen hydrolase domain,Nitrilase/cyanide hydratase and apolipoprotein N-acyltransferase identified within CDS. Product modified to more specific family member due to motif match. [...] ","uniprotId":"Q0P9J6_CAMJE"},"10641864":{"id":10641864,"externalId":"192222.Cj1057c","name":"Cj1057c","hasFamily":true,"annotation":"Bidirectionally degrades single-stranded DNA into large acid- insoluble oligonucleotides, which are then degraded further into small acid-soluble oligonucleotides","uniprotId":"Q0P9J5_CAMJE"},"10641865":{"id":10641865,"externalId":"192222.Cj1058c","name":"guaB","hasFamily":true,"annotation":"Catalyzes the conversion of inosine 5'-phosphate (IMP) to xanthosine 5'-phosphate (XMP), the first committed and rate-limiting step in the de novo synthesis of guanine nucleotides, and therefore plays an important role in the regulation of cell growth. ","uniprotId":"Q0P9J4_CAMJE"},"10641866":{"id":10641866,"externalId":"192222.Cj1059c","name":"gatA","hasFamily":true,"annotation":"Allows the formation of correctly charged Gln-tRNA(Gln) through the transamidation of misacylated Glu-tRNA(Gln) in organisms which lack glutaminyl-tRNA synthetase. The reaction takes place in the presence of glutamine and ATP through an activated gamma-phospho-Glu- tRNA(Gln) (By similarity)","uniprotId":"GATA_CAMJE"},"10641867":{"id":10641867,"externalId":"192222.Cj1060c","name":"Cj1060c","hasFamily":false,"annotation":"Putative membrane protein; Original (2000) note: Cj1060c, small hydrophobic protein, len: 36 aa. No Hp match; Updated (2006) note: One probable transmembrane helix predicted for TMHMM2.0. Also, one signal peptide predicted by SignalP 2.0 HMM. Product function modified to more specific family member. Putative kept within product function. Functional classification - Membranes,lipoproteins and porins","uniprotId":"Q0P9J2_CAMJE"},"10641868":{"id":10641868,"externalId":"192222.Cj1061c","name":"ileS","hasFamily":true,"annotation":"Catalyzes the attachment of isoleucine to tRNA(Ile). As IleRS can inadvertently accommodate and process structurally similar amino acids such as valine, to avoid such errors it has two additional distinct tRNA(Ile)-dependent editing activities. One activity is designated as 'pretransfer' editing and involves the hydrolysis of activated Val-AMP. The other activity is designated 'posttransfer' editing and involves deacylation of mischarged Val-tRNA(Ile). ","uniprotId":"SYI_CAMJE"},"10641869":{"id":10641869,"externalId":"192222.Cj1062","name":"Cj1062","hasFamily":true,"annotation":"Belongs to the CinA family. ","uniprotId":"Q0P9J0_CAMJE"},"10641870":{"id":10641870,"externalId":"192222.Cj1063","name":"Cj1063","hasFamily":true,"annotation":"Putative acetyltransferase; Original (2000) note: Cj1063, possible acetyltransferase, len: 141 aa; very weak similarity to PHNO_ECOLI PHNO protein (involved in alkylphosphonate uptake) (144 aa), fasta scores; opt: 135 z-score: 184.7 E(): 0.0054, 26.8% identity in 138 aa overlap. No Hp match. Contains Pfam match to entry PF00583 Acetyltransf,Acetyltransferase (GNAT) family; Updated (2006) note: Specific characterisation with acceptable identity score has not been carried out yet, so putative was kept within product function. Functional classification - Misc","uniprotId":"Q0P9I9_CAMJE"},"10641871":{"id":10641871,"externalId":"192222.Cj1066","name":"rdxA","hasFamily":true,"annotation":"Nitroreductase; HMMPfam hit to PF00881, Nitroreductase family,score 1.3e-32","uniprotId":"Q0P9I8_CAMJE"},"10641872":{"id":10641872,"externalId":"192222.Cj1067","name":"pgsA","hasFamily":true,"annotation":"Belongs to the CDP-alcohol phosphatidyltransferase class-I family","uniprotId":"Q0P9I7_CAMJE"},"10641873":{"id":10641873,"externalId":"192222.Cj1068","name":"Cj1068","hasFamily":true,"annotation":"Belongs to the peptidase M50B family","uniprotId":"Y1068_CAMJE"},"10641874":{"id":10641874,"externalId":"192222.Cj1069","name":"Cj1069","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1069, unknown, len: 288 aa; similar to hypothetical proteins e.g. YB19_HAEIN HI1119 (292 aa), fasta scores; opt: 428 z-score: 534.0 E(): 1.9e-22, 34.1% identity in 290 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF04393 Protein of unknown function (DUF535) identified within CDS. Conserved added to product function. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P9I5_CAMJE"},"10641875":{"id":10641875,"externalId":"192222.Cj1070","name":"rpsF","hasFamily":true,"annotation":"Binds together with S18 to 16S ribosomal RNA","uniprotId":"RS6_CAMJE"},"10641876":{"id":10641876,"externalId":"192222.Cj1071","name":"ssb","hasFamily":true,"annotation":"Single-stranded DNA-binding protein; Original (2000) note: Cj1071, ssb, single-strand DNA binding protein, len: 183 aa; 99.5% identical to TR:O69302 (EMBL:Y17166) C. jejuni ssb (183 aa), and similar to many e.g. SSB_ECOLI (177 aa), fasta scores; opt: 322 z-score: 365.8 E(): 4.5e-13, 34.9% identity in 186 aa overlap. 40.7% identity to HP1245. Contains PS00735 Single-strand binding protein family signature 1, and Pfam match to entry PF00436 SSB, Single-strand binding protein family; Updated (2006) note: Characterised within Campylobacter jejuni and similar to others with marginal identit [...] ","uniprotId":"SSB_CAMJE"},"10641877":{"id":10641877,"externalId":"192222.Cj1072","name":"rpsR","hasFamily":true,"annotation":"Binds as a heterodimer with protein S6 to the central domain of the 16S rRNA, where it helps stabilize the platform of the 30S subunit","uniprotId":"RS18_CAMJE"},"10641878":{"id":10641878,"externalId":"192222.Cj1073c","name":"lon","hasFamily":true,"annotation":"ATP-dependent serine protease that mediates the selective degradation of mutant and abnormal proteins as well as certain short- lived regulatory proteins. Required for cellular homeostasis and for survival from DNA damage and developmental changes induced by stress. Degrades polypeptides processively to yield small peptide fragments that are 5 to 10 amino acids long. Binds to DNA in a double-stranded, site-specific manner","uniprotId":"LON_CAMJE"},"10641879":{"id":10641879,"externalId":"192222.Cj1074c","name":"bamD","hasFamily":true,"annotation":"Part of the outer membrane protein assembly complex, which is involved in assembly and insertion of beta-barrel proteins into the outer membrane","uniprotId":"Q0P9I0_CAMJE"},"10641880":{"id":10641880,"externalId":"192222.Cj1075","name":"fliW","hasFamily":true,"annotation":"Acts as an anti-CsrA protein, binds CsrA and prevents it from repressing translation of its target genes, one of which is flagellin. Binds to flagellin and participates in the assembly of the flagellum (Probable). Antagonizes CsrA-mediated translational repression of flaA in a promoter-independent manner, leading to expression of FlaA and probably other flagellar genes. Binds to FlaA, which releases CsrA to repress translation of flaA mRNA. Also has a negative effect on flaA transcription, and influences the localization of flaA mRNA to the poles of short, probably elongating, cells. ","uniprotId":"FLIW_CAMJE"},"10641881":{"id":10641881,"externalId":"192222.Cj1076","name":"proC","hasFamily":true,"annotation":"Catalyzes the reduction of 1-pyrroline-5-carboxylate (PCA) to L-proline","uniprotId":"Q0P9H8_CAMJE"},"10641882":{"id":10641882,"externalId":"192222.Cj1077","name":"ctsT","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1077, probable periplasmic protein, len: 100 aa; no Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Some characterisation work carried out within Campylobacter jejuni. Product unchanged, but gene name added. Functional classification - Miscellaneous periplasmic proteins; PMID:12949093","uniprotId":"Q0P9H7_CAMJE"},"10641883":{"id":10641883,"externalId":"192222.Cj1078","name":"Cj1078","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1078, probable periplasmic protein, len: 220 aa; no Hp match. Contains probable N-terminal signal sequence and PS00572 Glycosyl hydrolases family 1 active site; Updated (2006) note: Some research carried out within Campylobacter jejuni. Product unchanged. Functional classification - Miscellaneous periplasmic proteins; PMID:12949093","uniprotId":"Q0P9H6_CAMJE"},"10641884":{"id":10641884,"externalId":"192222.Cj1079","name":"Cj1079","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1079, probable periplasmic protein, len: 150 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P9H5_CAMJE"},"10641885":{"id":10641885,"externalId":"192222.Cj1080c","name":"hemD","hasFamily":true,"annotation":"Catalyzes cyclization of the linear tetrapyrrole, hydroxymethylbilane, to the macrocyclic uroporphyrinogen III. ","uniprotId":"Q0P9H4_CAMJE"},"10641886":{"id":10641886,"externalId":"192222.Cj1081c","name":"thiE","hasFamily":true,"annotation":"Condenses 4-methyl-5-(beta-hydroxyethyl)thiazole monophosphate (THZ-P) and 2-methyl-4-amino-5-hydroxymethyl pyrimidine pyrophosphate (HMP-PP) to form thiamine monophosphate (TMP). ","uniprotId":"THIE_CAMJE"},"10641887":{"id":10641887,"externalId":"192222.Cj1082c","name":"thiD","hasFamily":true,"annotation":"Phosphomethylpyrimidine kinase; Original (2000) note: Cj1082c, thiD, probable phosphomethylpyrimidine kinase, len: 270 aa; similar to many e.g. THID_ECOLI phosphomethylpyrimidine kinase (EC 2.7.4.7) (266 aa), fasta scores; opt: 582 z-score: 703.6 E(): 6.8e-32, 41.0% identity in 261 aa overlap. 43.4% identity to HP0844; Updated (2006) note: Characterised in Escherichia coli with acceptable identity score, so putative not added to product function. Functional classification -Biosynthesis of cofactors, prosthetic groups and carriers - Thiamine; PMID:10075431","uniprotId":"Q0P9H2_CAMJE"},"10641888":{"id":10641888,"externalId":"192222.Cj1083c","name":"Cj1083c","hasFamily":true,"annotation":"Putative nuclease; Original (2000) note: Cj1083c, possible nuclease,len: 228 aa; some similarity with predicted proteins showing similarity to endonucleases e.g. END3_RICPR endonuclease III (EC 4.2.99.18) (212 aa), fasta scores; opt: 147 z-score: 181.9 E(): 0.0078, 25.7% identity in 152 aa overlap. 43.0% identity to HP0602 (annotated as endonuclease III). No detectable similarity to C. jejuni endonuclease III (Cj0595c, equivalent to HP0585); Updated (2006) note: Pfam domain PF00730 HhH-GPD superfamily base excision DNA repair identified within CDS. Further support given to product func [...] ","uniprotId":"Q0P9H1_CAMJE"},"10641889":{"id":10641889,"externalId":"192222.Cj1084c","name":"Cj1084c","hasFamily":true,"annotation":"Original (2000) note: Cj1084c, possible ATP/GTP-binding protein, len: 253 aa; similar to part of hypothetical proteins from Synechocystis TR:Q55502 (EMBL:D64006) (436 aa), fasta scores; opt: 586 z-score: 675.2 E(): 2.6e-30, 42.4% identity in 236 aa overlap, and Aquifex aeolicus TR:O66668 (EMBL:AE000684) AQ_336 (196 aa), fasta scores; opt: 383 z-score: 449.2 E(): 1e-17,40.4% identity in 136 aa overlap. No Hp match. Contains PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Pfam domain PF05673 Protein of unknown function (DUF815) identified within CDS. No specific chara [...] ","uniprotId":"Q0P9H0_CAMJE"},"10641890":{"id":10641890,"externalId":"192222.Cj1085c","name":"mfd","hasFamily":true,"annotation":"Couples transcription and DNA repair by recognizing RNA polymerase (RNAP) stalled at DNA lesions. Mediates ATP-dependent release of RNAP and its truncated transcript from the DNA, and recruitment of nucleotide excision repair machinery to the damaged site","uniprotId":"Q0P9G9_CAMJE"},"10641891":{"id":10641891,"externalId":"192222.Cj1086c","name":"Cj1086c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1086c, unknown, len: 131 aa; some similarity to hypothetical proteins e.g. TR:O51261 (EMBL:AE001134) Borrelia burgdorferi BB0245 (184 aa), fasta scores; opt: 197 z-score: 255.1 E(): 6.6e-07,30.0% identity in 110 aa overlap. 39.8% identity to HP1542; Updated (2006) note: Pfam domain PF04519 Protein of unknown function, DUF583 are identified within CDS. Prosite domain PS00430 TONB_DEPENDENT_REC_1,TonB-dependent receptor proteins signature 1 was also identified. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P9G8_CAMJE"},"10641892":{"id":10641892,"externalId":"192222.Cj1087c","name":"Cj1087c","hasFamily":true,"annotation":"Putative peptidase; Original (2000) note: Cj1087c, possible periplasmic protein, len: 300 aa; similar to TR:Q57503 (EMBL:U39068) Vibrio cholerae ToxR activated gene (302 aa), fasta scores; opt: 575 z-score: 642.1 E(): 1.8e-28, 34.4% identity in 305 aa overlap. Also contains a segment of similarity to e.g. LSTP_STASI lysostaphin precursor (493 aa), fasta scores; opt: 262 z-score: 294.5 E(): 4.2e-09,28.8% identity in 208 aa overlap. 36.9% identity to HP1543. Also similar to Cj1215 (40.6% identity in 133 aa overlap), and Cj1235 (40.7% identity in 123 aa overlap). Contains possible N-termi [...] ","uniprotId":"Q0P9G7_CAMJE"},"10641893":{"id":10641893,"externalId":"192222.Cj1088c","name":"folC","hasFamily":true,"annotation":"Original (2000) note: Cj1088c, folC, probable folylpolyglutamate synthase/dihydrofolate synthase, len: 394 aa; similar to many e.g. FOLC_ECOLI folylpolyglutamate synthase (EC 6.3.2.17) / dihydrofolate synthase (EC 6.3.2.12) (422 aa), fasta sores; opt: 315 z-score: 360.4 E(): 8.9e-13, 30.7% identity in 313 aa overlap. 42.7% identity to HP1545. Contains Pfam match to entry PF01225 FPGS, Folylpolyglutamate synthase; Updated (2006) note: Characterised in Escherichia coli with marginal identity score. Appropriate motifs present. Putative not added to product function. Functional classificat [...] ","uniprotId":"Q0P9G6_CAMJE"},"10641894":{"id":10641894,"externalId":"192222.Cj1089c","name":"Cj1089c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1089c, unknown, len: 65 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0P9G5_CAMJE"},"10641895":{"id":10641895,"externalId":"192222.Cj1090c","name":"Cj1090c","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj1090c, probable lipoprotein, len: 170 aa; 26.5% identity to HP1546. Contains N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes,lipoproteins and porins","uniprotId":"Q0P9G4_CAMJE"},"10641896":{"id":10641896,"externalId":"192222.Cj1091c","name":"leuS","hasFamily":true,"annotation":"Belongs to the class-I aminoacyl-tRNA synthetase family. ","uniprotId":"SYL_CAMJE"},"10641897":{"id":10641897,"externalId":"192222.Cj1092c","name":"secF","hasFamily":true,"annotation":"Part of the Sec protein translocase complex. Interacts with the SecYEG preprotein conducting channel. SecDF uses the proton motive force (PMF) to complete protein translocation after the ATP-dependent function of SecA","uniprotId":"Q0P9G2_CAMJE"},"10641898":{"id":10641898,"externalId":"192222.Cj1093c","name":"secD","hasFamily":true,"annotation":"Part of the Sec protein translocase complex. Interacts with the SecYEG preprotein conducting channel. SecDF uses the proton motive force (PMF) to complete protein translocation after the ATP-dependent function of SecA","uniprotId":"Q0P9G1_CAMJE"},"10641899":{"id":10641899,"externalId":"192222.Cj1094c","name":"yajC","hasFamily":true,"annotation":"The SecYEG-SecDF-YajC-YidC holo-translocon (HTL) protein secretase/insertase is a supercomplex required for protein secretion, insertion of proteins into membranes, and assembly of membrane protein complexes. While the SecYEG complex is essential for assembly of a number of proteins and complexes, the SecDF-YajC-YidC subcomplex facilitates these functions","uniprotId":"Q0P9G0_CAMJE"},"10641900":{"id":10641900,"externalId":"192222.Cj1095","name":"lnt","hasFamily":true,"annotation":"Catalyzes the phospholipid dependent N-acylation of the N- terminal cysteine of apolipoprotein, the last step in lipoprotein maturation","uniprotId":"LNT_CAMJE"},"10641901":{"id":10641901,"externalId":"192222.Cj1096c","name":"metK","hasFamily":true,"annotation":"Belongs to the AdoMet synthase family. ","uniprotId":"Q0P9F8_CAMJE"},"10641902":{"id":10641902,"externalId":"192222.Cj1097","name":"sstT","hasFamily":true,"annotation":"Involved in the import of serine and threonine into the cell, with the concomitant import of sodium (symport system). ","uniprotId":"SSTT_CAMJE"},"10641903":{"id":10641903,"externalId":"192222.Cj1098","name":"pyrB","hasFamily":true,"annotation":"Belongs to the aspartate/ornithine carbamoyltransferase superfamily. ATCase family","uniprotId":"PYRB_CAMJE"},"10641904":{"id":10641904,"externalId":"192222.Cj1099","name":"Cj1099","hasFamily":true,"annotation":"Peptidase; Original (2000) note: Cj1099, probable peptidase,len: 573 aa; similar to many members of the peptidase family M3 (zinc metalloprotease) e.g. PEPF_LACLA oligoendopeptidase F (601 aa), fasta scores; opt: 603 z-score: 673.0 E(): 3.4e-30, 25.9% identity in 595 aa overlap. 53.5% identity to HP0470. Contains PS00142 Neutral zinc metallopeptidases, zinc-binding region signature; Updated (2006) note: Pfam domain PF01432 Peptidase family M3 identified within CDS. Further support given to product function. Specific characterisation has not been carried out yet. Putative not added to p [...] ","uniprotId":"Q0P9F5_CAMJE"},"10641905":{"id":10641905,"externalId":"192222.Cj1100","name":"Cj1100","hasFamily":true,"annotation":"Hypothteical protein Cj1100; Original (2000) note: Cj1100, unknown, len: 145 aa; 34.7% identity to HP0469. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P9F4_CAMJE"},"10641906":{"id":10641906,"externalId":"192222.Cj1101","name":"Cj1101","hasFamily":true,"annotation":"DNA helicase; Original (2000) note: Cj1101, probable ATP-dependent DNA helicase, len: 691 aa; similar to many e.g REP_ECOLI ATP-dependent DNA helicase REP (EC 3.6.1.-) (673 aa), fasta scores; opt: 977 z-score: 1067.3 E(): 0,36.9% identity in 640 aa overlap, and UVRD_ECOLI DNA helicase II (EC 3.6.1.-) (720 aa), fasta scores; opt: 1129 z-score: 1232.6 E(): 0, 35.2% identity in 714 aa overlap. 43.7% identity to HP1478. Contains PS00017 ATP/GTP-binding site motif A (P-loop), and Pfam match to entry PF00580 UvrD-helicase, UvrD/REP helicase; Updated (2006) note: Similar to more than one heli [...] ","uniprotId":"Q0P9F3_CAMJE"},"10641907":{"id":10641907,"externalId":"192222.Cj1102","name":"truB","hasFamily":true,"annotation":"Responsible for synthesis of pseudouridine from uracil-55 in the psi GC loop of transfer RNAs","uniprotId":"TRUB_CAMJE"},"10641908":{"id":10641908,"externalId":"192222.Cj1103","name":"csrA","hasFamily":true,"annotation":"A translational regulator that binds mRNA to regulate translation initiation and/or mRNA stability. Usually binds in the 5'- UTR at or near the Shine-Dalgarno sequence preventing ribosome-binding, thus repressing translation. Binds mRNA; 77% of enriched bound RNA is for flagellin A (flaA) while another 13% encodes other flagellar or motility-related genes. Binds mRNA in 5'-UTR or intergenic regions, binds consensus 5'-AAGGA-3' in the loop of a predicted stem-loop structure. Binds at least 2 sites in the 5'-UTR of flaA mRNA and represses its translation; mutation of the binding sites ab [...] ","uniprotId":"CSRA_CAMJE"},"10641909":{"id":10641909,"externalId":"192222.Cj1104","name":"ispE","hasFamily":true,"annotation":"Catalyzes the phosphorylation of the position 2 hydroxy group of 4-diphosphocytidyl-2C-methyl-D-erythritol","uniprotId":"ISPE_CAMJE"},"10641910":{"id":10641910,"externalId":"192222.Cj1105","name":"smpB","hasFamily":true,"annotation":"Required for rescue of stalled ribosomes mediated by trans- translation. Binds to transfer-messenger RNA (tmRNA), required for stable association of tmRNA with ribosomes. tmRNA and SmpB together mimic tRNA shape, replacing the anticodon stem-loop with SmpB. tmRNA is encoded by the ssrA gene; the 2 termini fold to resemble tRNA(Ala) and it encodes a 'tag peptide', a short internal open reading frame. During trans-translation Ala-aminoacylated tmRNA acts like a tRNA, entering the A-site of stalled ribosomes, displacing the stalled mRNA. The ribosome then switches to translate the ORF on  [...] ","uniprotId":"SSRP_CAMJE"},"10641911":{"id":10641911,"externalId":"192222.Cj1106","name":"Cj1106","hasFamily":true,"annotation":"Original (2000) note: Cj1106, possible periplasmic thioredoxin, len: 200 aa; similar to active site region of thioredoxins e.g. THIO_BACSU thioredoxin (103 aa), fasta scores; opt: 112 z-score: 150.6 E(): 0.43, 37.7% identity in 69 aa overlap. Contains probable N-terminal signal sequence. No Hp match. Also similar to Cj1207c (29.4% identity in 201 aa overlap); Updated (2006) note: No specific characterisation with acceptable identity score has been carried out yet,so putative kept in product function. Functional classification - Energy metabolism - Electron transport","uniprotId":"Q0P9E8_CAMJE"},"10641912":{"id":10641912,"externalId":"192222.Cj1107","name":"clpS","hasFamily":true,"annotation":"Involved in the modulation of the specificity of the ClpAP- mediated ATP-dependent protein degradation","uniprotId":"CLPS_CAMJE"},"10641913":{"id":10641913,"externalId":"192222.Cj1108","name":"clpA","hasFamily":true,"annotation":"Belongs to the ClpA/ClpB family. ","uniprotId":"Q0P9E6_CAMJE"},"10641914":{"id":10641914,"externalId":"192222.Cj1109","name":"aat","hasFamily":true,"annotation":"Functions in the N-end rule pathway of protein degradation where it conjugates Leu, Phe and, less efficiently, Met from aminoacyl- tRNAs to the N-termini of proteins containing an N-terminal arginine or lysine","uniprotId":"LFTR_CAMJE"},"10641915":{"id":10641915,"externalId":"192222.Cj1110c","name":"Cj1110c","hasFamily":true,"annotation":"Original (2000) note: Cj1110c, probable MCP-type signal transduction protein, len: 429 aa; similar to TR:O68016 (EMBL:AF010180) Agrobacterium tumefaciens plasmid pTiC58 MCPA (579 aa), fasta scores; opt: 749 z-score: 785.2 E(): 0, 34.5% identity in 496 aa overlap,and to many other proteins with MCP domains e.g. PILJ_PSEAE PILJ protein (682 aa), fasta scores; opt: 379 z-score: 399.1 E(): 6.3e-15, 26.9% identity in 443 aa overlap, and HLYB_VIBCH hemolysin secretion protein precursor (548 aa), fasta scores; opt: 357 z-score: 377.6 E(): 9.8e-14, 37.0% identity in 173 aa overlap. No Hp ortho [...] ","uniprotId":"Q0P9E4_CAMJE"},"10641916":{"id":10641916,"externalId":"192222.Cj1111c","name":"Cj1111c","hasFamily":true,"annotation":"UPF0056 inner membrane protein; Original (2000) note: Cj1111c, probable integral membrane protein, len: 208 aa; similar to many hypothetical membrane proteoins, e.g. YVBG_BACSU (211 aa),fasta scores; opt: 324 z-score: 391.8 E(): 1.6e-14, 28.8% identity in 198 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF01914 MarC family integral membrane protein identified within CDS. Also, six probable transmembrane helices predicted by TMHMM2.0. Product modified to more specific family member based on Pfam match. Specific characterisation has not yet been carried out, so putative kept [...] ","uniprotId":"Q0P9E3_CAMJE"},"10641917":{"id":10641917,"externalId":"192222.Cj1112c","name":"Cj1112c","hasFamily":true,"annotation":"Belongs to the MsrB Met sulfoxide reductase family. ","uniprotId":"Q0P9E2_CAMJE"},"10641918":{"id":10641918,"externalId":"192222.Cj1113","name":"Cj1113","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1113, unknown, len: 265 aa; similar to a hypothetical protein from Haemophilus Y077_HAEIN HI0077 (288 aa), fasta scores; opt: 441 z-score: 521.9 E(): 9.1e-22, 34.5% identity in 275 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF04305 Protein of unknown function (DUF455) identified within CDS. Conserved added to product function. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P9E1_CAMJE"},"10641919":{"id":10641919,"externalId":"192222.Cj1114c","name":"pssA","hasFamily":true,"annotation":"Belongs to the CDP-alcohol phosphatidyltransferase class-I family","uniprotId":"Q0P9E0_CAMJE"},"10641920":{"id":10641920,"externalId":"192222.Cj1115c","name":"Cj1115c","hasFamily":true,"annotation":"Putative phosphatidylserine decarboxylase-related protein; Original (2000) note: Cj1115c, probable membrane protein, len: 205 aa; simimlar to hypothetical proteins e.g. TR:Q50967 (EMBL:U34760) Neisseria gonorrhoeae ORF259 (259 aa), fasta scores; opt: 223 z-score: 281.2 E(): 2.3e-08, 24.6% identity in 191 aa overlap. No Hp match; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Characterised within Escherichia coli and Bacillus subtilis with identity scores marginal identity scores. Putative kept within product function. Functional classification - Misc; PMID [...] ","uniprotId":"Q0P9D9_CAMJE"},"10641921":{"id":10641921,"externalId":"192222.Cj1116c","name":"ftsH","hasFamily":true,"annotation":"Acts as a processive, ATP-dependent zinc metallopeptidase for both cytoplasmic and membrane proteins. Plays a role in the quality control of integral membrane proteins","uniprotId":"Q0P9D8_CAMJE"},"10641922":{"id":10641922,"externalId":"192222.Cj1117c","name":"prmA","hasFamily":true,"annotation":"Methylates ribosomal protein L11","uniprotId":"PRMA_CAMJE"},"10641923":{"id":10641923,"externalId":"192222.Cj1118c","name":"cheY","hasFamily":true,"annotation":"Involved in the transmission of sensory signals from the chemoreceptors to the flagellar motors. CheY seems to regulate the clockwise (CW) rotation (By similarity)","uniprotId":"CHEY_CAMJE"},"10641924":{"id":10641924,"externalId":"192222.Cj1119c","name":"pglG","hasFamily":true,"annotation":"Putative integral membrane protein; Original (2000) note: Cj1119c, wlaM, pglG, probable integral membrane protein, len: 297 aa; 99.3% identical to TR:O86160 (EMBL:Y11648) C. jejuni 81116 wlaM (297 aa). Also contains short section of similarity to TR:Q56633 (EMBL:L25660) Vibrio cholerae accessory colonization factor (ACFB) (626 aa), fasta scores; opt: 165 z-score: 202.6 E(): 0.00055, 34.0% identity in 103 aa overlap. 37.6% identity to HP0158; Updated (2006) note: Four probable transmembrane helices predicted by TMHMM2.0. Characterised in Campylobacter jejuni. The putative integral membr [...] ","uniprotId":"Q0P9D5_CAMJE"},"10641925":{"id":10641925,"externalId":"192222.Cj1120c","name":"pglF","hasFamily":true,"annotation":"C6 dehydratase involved in the bacillosamine biosynthesis pathway by generating UDP-2-acetamido-2,6-dideoxy-alpha-D-xylo-4- hexulose from UDP-N-acetyl-alpha-D-glucosamine. Involved in the N- linked protein glycosylation pathway","uniprotId":"PGLF_CAMJE"},"10641926":{"id":10641926,"externalId":"192222.Cj1121c","name":"pglE","hasFamily":true,"annotation":"Aminotransferase involved in the bacillosamine biosynthesis pathway by producing UDP-4-amino-4,6-dideoxy-alpha-D-GlcNAc (UDP-2- acetamido-4-amino-2,4,6-trideoxy-alpha-D-glucopyranose), a precursor used in the production of the glycan component 2,4-diacetamido-2,4,6- trideoxy-alpha-D-glucopyranose. Required for host colonization and virulence. Involved in the N-linked protein glycosylation pathway. ","uniprotId":"PGLE_CAMJE"},"10641927":{"id":10641927,"externalId":"192222.Cj1122c","name":"Cj1122c","hasFamily":false,"annotation":"Original (2000) note: Cj1122c, wlaJ, possible integral membrane protein, len: 217 aa; 99.5% identical to TR:G4100601 (EMBL:AF001497) C. jejuni 11168 orfE (217 aa). Not present in C. jejuni 81116. No Hp match; Updated (2006) note: Three probable transmembrane helices predicted by TMHMM2.0","uniprotId":"Q0P9D2_CAMJE"},"10641928":{"id":10641928,"externalId":"192222.Cj1123c","name":"pglD","hasFamily":true,"annotation":"Acetyltransferase that modifies the UDP-4-amino-sugar to form UDP-N,N'-diacetylbacillosamine in the N-linked protein glycosylation pathway","uniprotId":"PGLD_CAMJE"},"10641929":{"id":10641929,"externalId":"192222.Cj1124c","name":"pglC","hasFamily":true,"annotation":"Glycosyl-1-phosphate transferase that mediates the first step in the biosynthesis of the undecaprenyl-linked heptasaccharide donor in the N-linked protein glycosylation pathway. Catalyzes the linking of uridine 5'-diphosphobacillosamine (UDP-Bac) to undecaprenyl phosphate to create the first membrane-associated intermediate undecaprenylpyrophosphate-linked Bac (Und-PP-Bac). ","uniprotId":"PGLC_CAMJE"},"10641930":{"id":10641930,"externalId":"192222.Cj1125c","name":"pglA","hasFamily":true,"annotation":"Adds the first GalNAc residue on to the isoprenoid-linked bacillosamine (2,4-diacetamido-2,4,6-trideoxyglucose) carrier in the N- linked protein glycosylation pathway. Acts first on the undecaprenylpyrophosphate-linked bacillosamine (Und-PP-Bac) substrate to yield the disaccharide","uniprotId":"PGLA_CAMJE"},"10641931":{"id":10641931,"externalId":"192222.Cj1126c","name":"pglB","hasFamily":true,"annotation":"Oligosaccharyl transferase (OST) that catalyzes the initial transfer of a defined glycan (GalNAc(2)GlcGalNAc(3)Bac(NAc)(2) in eubacteria, where Bac(NAc)(2) is di-N-acetyl bacillosamine) from the lipid carrier undecaprenol-pyrophosphate to an asparagine residue within an Asp/Glu-Asn-X-Ser/Thr consensus motif in nascent polypeptide chains, the first step in protein N-glycosylation. ","uniprotId":"PGLB_CAMJE"},"10641932":{"id":10641932,"externalId":"192222.Cj1127c","name":"pglJ","hasFamily":true,"annotation":"Adds a GalNAc residue on to the Und-PP-Bac2,4diNAc-GalNAc disaccharide in the N-linked protein glycosylation pathway. Transfers the third sugar in the heptasaccharide biosynthesis. ","uniprotId":"PGLJ_CAMJE"},"10641933":{"id":10641933,"externalId":"192222.Cj1128c","name":"pglI","hasFamily":true,"annotation":"Glucosyltransferase that adds he final branching glucose to complete the final heptasaccharide structure in the N-linked protein glycosylation pathway","uniprotId":"PGLI_CAMJE"},"10641934":{"id":10641934,"externalId":"192222.Cj1129c","name":"pglH","hasFamily":true,"annotation":"Processive glycosyltransferase that is part of the biosynthetic pathway of the lipid-linked oligosaccharide (LLO) that serves as the glycan donor in bacterial protein N-glycosylation. Catalyzes the transfer of exactly three alpha-(1->4)-N- acetylgalactosamine (GalNAc) units to the growing LLO precursor, GalNAc-alpha-(1->4)-GalNAc-alpha-(1->3)-diNAcBac-PP-undecaprenyl . Cannot accept UDP- GlcNAc as substrate ","uniprotId":"PGLH_CAMJE"},"10641935":{"id":10641935,"externalId":"192222.Cj1130c","name":"pglK","hasFamily":true,"annotation":"Mediates the ATP-dependent translocation of the undecaprenylpyrophosphate-linked heptasaccharide intermediate across the cell membrane; this is an essential step during the N-linked protein glycosylation pathway. Transport across the membrane is effected via ATP-driven conformation changes. Most likely, only the polar and charged part of the glycolipid enter the substrate-binding cavity, and the lipid tail remains exposed to the membrane lipids during the transmembrane flipping process. ","uniprotId":"PGLK_CAMJE"},"10641936":{"id":10641936,"externalId":"192222.Cj1131c","name":"gne","hasFamily":true,"annotation":"Belongs to the NAD(P)-dependent epimerase/dehydratase family","uniprotId":"Q0P9C3_CAMJE"},"10641937":{"id":10641937,"externalId":"192222.Cj1132c","name":"Cj1132c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1132c, unknown, len: 264 aa; some similarity to part of DNA polymerases e.g. DPOL_PYRFU DNA polymerase (775 aa), fasta scores; opt: 168 z-score: 197.4 E(): 0.0011, 29.6% identity in 247 aa overlap. Similar in N- and C-termini to sequential Hp CDSs; HP0688 (46.7% identity in 169 aa overlap), and HP0689 (47.4% identity in 95 aa overlap). Contains Pfam match to entry PF00136 DNA_pol_B, DNA polymerase family B; Updated (2006) note: Literature search identified paper giving further clues about product function. Conserved added to product func [...] ","uniprotId":"Q0P9C2_CAMJE"},"10641938":{"id":10641938,"externalId":"192222.Cj1133","name":"waaC","hasFamily":true,"annotation":"Original (2000) note: Cj1133, waaC, probable lipopolysaccharide heptosyltransferase, len: 342 aa; 98.2% identical to TR:O87585 (EMBL:AF086705) C. jejuni heptosyltransferase WaaC (342 aa), and similar to e.g. RFAC_ECOLI lipopolysaccharide heptosyltransferase-1 (319 aa), fasta scores; opt: 212 z-score: 252.9 E(): 8.6e-07,26.2% identity in 302 aa overlap. 38.5% identity to HP0279. Contains Pfam match to entry PF01075 Heptosyltranf, Heptosyltransferase; Updated (2006) note: Characterised within Campylobacter jejuni. Product function further updated. Putative not added to product function.  [...] ","uniprotId":"Q0P9C1_CAMJE"},"10641939":{"id":10641939,"externalId":"192222.Cj1134","name":"htrB","hasFamily":true,"annotation":"Original (2000) note: Cj1134, htrB, probable lipid A biosynthesis lauroyl acyltransferase, len: 295 aa; similar to e.g. HTRB_ECOLI lipid A biosynthesis lauroyl acyltransferase (EC 2.3.1.-) (306 aa), fasta scores; opt: 274 z-score: 340.3 E(): 1.2e-11, 23.4% identity in 299 aa overlap. 27.9% identity to HP0280; Updated (2006) note: Pfam domain PF03279 Bacterial lipid A biosynthesis acyltransferase domain was identified within CDS. Further support given to product function. No specific characterisation with acceptable identity score has been carried out yet, so putative kept within produc [...] ","uniprotId":"Q0P9C0_CAMJE"},"10641940":{"id":10641940,"externalId":"192222.Cj1135","name":"Cj1135","hasFamily":true,"annotation":"Putative two-domain glucosyltransferase; Original (2000) note: Cj1135, probable two-domain glycosyltransferase, len: 515 aa; contains similarities in N- and C-termini to glycosyltransferases e.g. TR:P95373 (EMBL:U58765) Neisseria meningitidis beta 1,4 glucosyltransferase lgtF (252 aa), fatsa scores; opt: 380 z-score: 453.4 E(): 5.9e-18, 30.9% identity in 262 aa overlap, and TR:O85457 Pasteurella multocida glycosyl transferase hyaD (864 aa), fasta scores; opt: 259 z-score: 302.8 E(): 1.4e-09, 28.0% identity in 239 aa overlap. No Hp match. Contains 2x Pfam match to entry PF00535 Glycos_t [...] ","uniprotId":"Q0P9B9_CAMJE"},"10641941":{"id":10641941,"externalId":"192222.Cj1136","name":"Cj1136","hasFamily":true,"annotation":"Putative glycosyltransferase; Original (2000) note: Cj1136, probable galactosyltransferase, len: 390 aa; similar to e.g. TR:O07340 (EMBL:X85787) Streptococcus pneumoniae SS-1,4-galactosyltransferase CPS14J (318 aa), fasta scores; opt: 309 z-score: 361.4 E(): 7.8e-13, 29.0% identity in 290 aa overlap. No Hp match. Contains Pfam match to entry PF00535 Glycos_transf_2, Glycosyl transferases; Updated (2006) note: Pfam domain PF00535 Glycosyl transferase identified within CDS. Product function updated based on motif match. No specific characterisation has been carried out yet, so putative k [...] ","uniprotId":"Q0P9B8_CAMJE"},"10641942":{"id":10641942,"externalId":"192222.Cj1137c","name":"Cj1137c","hasFamily":true,"annotation":"Putative glycosyltransferase; Original (2000) note: Cj1137c, unknown, len: 331 aa; similar to hypothetcal proteins in polysaccharide biosynthesis operons e.g. TR:O86892 (EMBL:AJ006986) Streptococcus pneumoniae CAPP33FI protein (322 aa), fasta scores; opt: 295 z-score: 346.5 E(): 5.3e-12, 26.5% identity in 294 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF05704 Capsular polysaccharide synthesis protein identified within CDS. Product function modified to more specific family member. No specific characterisation has been carried out yet. Putative kept within product function [...] ","uniprotId":"Q0P9B7_CAMJE"},"10641943":{"id":10641943,"externalId":"192222.Cj1138","name":"Cj1138","hasFamily":true,"annotation":"Putative glycosyltransferase; Original (2000) note: Cj1138, probable galactosyltransferase, len: 389 aa; similar to e.g. TR:O07340 (EMBL:X85787) Streptococcus pneumoniae SS-1,4-galactosyltransferase CPS14J (318 aa), fasta scores; opt: 304 z-score: 361.2 E(): 8e-13, 25.8% identity in 383 aa overlap. No Hp match. Contains Pfam match to entry PF00535 Glycos_transf_2, Glycosyl transferases; Updated (2006) note: No specific characterisation with acceptable identity score has been carried out yet. Putative kept within product function. Product function modified to more specific family member [...] ","uniprotId":"Q0P9B6_CAMJE"},"10641944":{"id":10641944,"externalId":"192222.Cj1139c","name":"wlaN","hasFamily":true,"annotation":"Beta-1,3 galactosyltransferase; Original (2000) note: Cj1139c, probable galactosyltransferase, len: 303 aa; similar to e.g. TR:O07340 (EMBL:X85787) Streptococcus pneumoniae SS-1,4-galactosyltransferase CPS14J (318 aa), fasta scores; opt: 309 z-score: 362.2 E(): 7.1e-13, 26.1% identity in 249 aa overlap. No Hp match. Contains Pfam match to entry PF00535 Glycos_transf_2, Glycosyl transferases. Contains C(8-9) polymorphic region at aa 111. C(8), the consensus, allows translation of the full length protein. C(9) would cause a premature truncation after a further 20 aa; Updated (2006) note: [...] ","uniprotId":"Q0P9B5_CAMJE"},"10641945":{"id":10641945,"externalId":"192222.Cj1140","name":"cstIII","hasFamily":true,"annotation":"Alpha-2,3 sialyltransferase; Original (2000) note: Cj1140, unknown, len: 294 aa; similar to H. influenzae hypothetical protein in lipopolysaccharide biosynthesis gene cluster Y352_HAEIN HI0352 (ORF1) (231 aa), fasta scores; opt: 558 z-score: 672.8 E(): 3.5e-30, 41.5% identity in 229 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF06002 Alpha-2,3-sialyltransferase (CST-I) identified within CDS. Characterisation work carried out within Campylobacter jejuni, so product function modified to new family member. Putative not added to product function. This protein transfers sialic [...] ","uniprotId":"Q0P9B4_CAMJE"},"10641946":{"id":10641946,"externalId":"192222.Cj1141","name":"neuB1","hasFamily":true,"annotation":"Sialic acid synthase (N-acetylneuraminic acid synthetase); Original (2000) note: Cj1141, neuB1, probable N-acetylneuraminic acid synthetase, len: 343 aa; highly similar to e.g. TR:Q57265 (EMBL:U40740) N-acetylneuraminic acid synthetase (EC 4.1.3.-) (349 aa), fasta scores; opt: 1327 z-score: 1554.1 E(): 0, 59.0% identity in 344 aa overlap. No Hp ortholog. Also similar to Cj1327 (37.1% identity in 329 aa overlap) and Cj1317 (34.4% identity in 340 aa overlap); Updated (2006) note: Pfam domains PF03102 NeuB family and PF01354 Antifreeze-like domain were identified within CDS. Characterisat [...] ","uniprotId":"Q0P9B3_CAMJE"},"10641947":{"id":10641947,"externalId":"192222.Cj1142","name":"neuC1","hasFamily":true,"annotation":"Putative UDP-N-acetylglucosamine 2-epimerase; Original (2000) note: Cj1142, neuC1, probable N-acetylglucosamine-6-phosphate 2-epimerase/N-acetylglucosamine-6-phosphatase, len: 371 aa; similar to e.g. TR:Q57141 (EMBL:X78068) Neisseria meningitidis putative N-acetylglucosamine-6-phosphate 2-epimerase (EC 5.1.3.-) / N-acetylglucosamine-6-phosphatase (EC 3.1.3.-) (377 aa),fatsta scores; opt: 1029 z-score: 1162.4 E(): 0, 44.0% identity in 377 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02350 UDP-N-acetylglucosamine 2-epimerase identified within CDS. Further support given to  [...] ","uniprotId":"Q0P9B2_CAMJE"},"10641948":{"id":10641948,"externalId":"192222.Cj1143","name":"neuA1,","hasFamily":true,"annotation":"Two-domain bifunctional protein (Beta-1,4-N-acetylgalactosaminyltransferase/CMP-Neu5Ac synthase); Original (2000) note: Cj1143, neuA1, probable acylneuraminate cytidylyltransferase (CMP-N-acetylneuraminic acid synthetase), len: 536 aa; N-terminus is similar to HP0217 (49.1% identity in 279 aa overlap), C-traminus is similar to NEUA_NEIME acylneuraminate cytidylyltransferase (EC 2.7.7.43) (228 aa), fasta scores; opt: 471 z-score: 550.3 E(): 2.3e-23,39.4% identity in 221 aa overlap, and to Cj1311 (31.6% identity in 215 aa overlap) and Cj1331 (34.7% identity in 219 aa overlap); Updated (2 [...] ","uniprotId":"Q0P9B1_CAMJE"},"10641949":{"id":10641949,"externalId":"192222.Cj1144c","name":"Cj1144c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1144c, unknown, len: 197 aa; no Hp match. Contains C(8-9) polymorphism at aa 15; C(8), the consnsus gives this CDS; C(9) would cause truncation after 4 aa, although a Met at aa 23 would allow translation to re-initiate. C(7) or C(10), although not seen in shotgun, would allow translation from upstream CDS Cj1145c; Original (2000) note: Cj1145c, unknown, len: 105 aa; no Hp match. Contains C(8-9) polymorphism at aa 99; C(8), the consnsus gives this CDS; C(9) would cause truncation after4 aa; C(7) or C(10), although not seen in shotgun, wou [...] ","uniprotId":"Q0P9B0_CAMJE"},"10641950":{"id":10641950,"externalId":"192222.Cj1146c","name":"waaV","hasFamily":true,"annotation":"Putative glucosyltransferase; Original (2000) note: Cj1146c, waaV, possible glucosyltransferase, len: 274 aa; similar to e.g. TR:O87182 (EMBL:AB017355) Streptococcus agalactiae N-acetylglucosaminyltransferase CPSH (333 aa), fasta scores; opt: 339 z-score: 402.4 E(): 4.1e-15, 31.4% identity in 274 aa overlap. No Hp match. Contains Pfam match to entry PF00535 Glycos_transf_2, Glycosyl transferases; Updated (2006) note: Some characterisation within Escherichia coli, however, identity score was marginal. Putative kept within product function. Identifed as part of Campylobacter jejuni lipoo [...] ","uniprotId":"Q0P9A9_CAMJE"},"10641951":{"id":10641951,"externalId":"192222.Cj1148","name":"waaF","hasFamily":true,"annotation":"Heptosyltransferase II; Original (2000) note: Cj1148, waaF, probable ADP-heptose--LPS heptosyltransferase, len: 319 aa; similar to e.g. TR:Q51063 (EMBL:Z37141) Neisseria gonorrhoeae ADP-heptose:LPS heptosyltransferase II (336 aa), fasta scores; opt: 384 z-score: 447.0 E(): 1.3e-17, 29.7% identity in 333 aa overlap, and RFAF_ECOLI ADP-heptose--LPS heptosyltransferase II (348 aa), fasta scores; opt: 292 z-score: 341.5 E(): 1e-11, 24.1% identity in 348 aa overlap. 37.6% identity to HP1191. Also similar to Cj1133 (24.8% identity in 351 aa overlap). Contains Pfam match to entry PF01075 Hept [...] ","uniprotId":"Q0P9A8_CAMJE"},"10641952":{"id":10641952,"externalId":"192222.Cj1149c","name":"gmhA","hasFamily":true,"annotation":"Catalyzes the isomerization of sedoheptulose 7-phosphate in D-glycero-D-manno-heptose 7-phosphate","uniprotId":"GMHA1_CAMJE"},"10641953":{"id":10641953,"externalId":"192222.Cj1150c","name":"hldE","hasFamily":true,"annotation":"Catalyzes the phosphorylation of D-glycero-D-manno-heptose 7- phosphate at the C-1 position to selectively form D-glycero-beta-D- manno-heptose-1,7-bisphosphate","uniprotId":"HLDE_CAMJE"},"10641954":{"id":10641954,"externalId":"192222.Cj1151c","name":"hldD","hasFamily":true,"annotation":"ADP-glyceromanno-heptose 6-epimerase; Original (2000) note: Cj1151c, waaD, probable ADP-L-glycero-D-manno-heptose-6-epimerase, len: 317 aa; similar to e.g. RFAD_ECOLI ADP-L-glycero-D-manno-heptose-6-epimerase (EC 5.1.3.20) (310 aa), fasta scores; opt: 533 z-score: 600.5 E(): 3.8e-26. 32.9% identity in 319 aa overlap. 48.3% identity to HP0859; Updated (2006) note: Pfam domain PF01370 NAD dependent epimerase/dehydratase family identified within CDS. Further support given to product function. Characterised within Escherichia coli with marginal identity score. Product function has been upd [...] ","uniprotId":"Q0P9A5_CAMJE"},"10641955":{"id":10641955,"externalId":"192222.Cj1152c","name":"gmhB","hasFamily":true,"annotation":"Converts the D-glycero-alpha-D-manno-heptose 1,7-bisphosphate intermediate into D-glycero-alpha-D-manno-heptose 1-phosphate by removing the phosphate group at the C-7 position","uniprotId":"GMHBA_CAMJE"},"10641956":{"id":10641956,"externalId":"192222.Cj1153","name":"Cj1153","hasFamily":true,"annotation":"Original (2000) note: Cj1153, probable periplasmic cytochrome C, len: 100 aa; similar to many e.g. 553_DESVH cytochrome C-553 precursor (103 aa), fasta scores; opt: 187 z-score: 243.8 E(): 2.8e-06, 34.6% identity in 107 aa overlap. 31.0% identity to HP1227. Contains PS00190 Cytochrome c family heme-binding site signature, and Pfam match to entry PF00034 cytochrome_c; Updated (2006) note: Characterisation work has been carried out in Helicobacter pylori, however, identity searches were marginal. Putative kept in product function. Functional classification - Energy metabolism - Electron  [...] ","uniprotId":"Q0P9A3_CAMJE"},"10641957":{"id":10641957,"externalId":"192222.Cj1154c","name":"Cj1154c","hasFamily":true,"annotation":"Putative cytochrome oxidase maturation protein cbb3-type; Original (2000) note: Cj1154c, small hydrophobic protein, len: 68 aa; 38.7% identity to HP1163; Updated (2006) note: Pfam domain PF03597 Cytochrome oxidase maturation protein cbb3-type identified within CDS. Also, one probable transmembrane helix predicted by TMHMM2.0. Product modified to more specific family member based on motif match. Specific characterisation has not been carried out yet, so putative kept within product function. Functional classification - Protein translation and modification","uniprotId":"Q0P9A2_CAMJE"},"10641958":{"id":10641958,"externalId":"192222.Cj1155c","name":"Cj1155c","hasFamily":true,"annotation":"Original (2000) note: Cj1155c, probable cation-transporting ATPase, len: 785 aa; similar to members of the E1-E2 ATPase family e.g. ATCU_ECOLI probable copper-transporting ATPase (834 aa), fasta scores; opt: 708 z-score: 767.6 E(): 0, 28.9% identity in 830 aa overlap. 36.5% identity to HP1503. Contains 2x Pfam match to entry PF00122 E1-E2_ATPase; Updated (2006) note: Pfam domains PF00702 haloacid dehalogenase-like hydrolase and PF00403 Heavy-metal-associated domain were identified within CDS. Eight probable transmembrane helices predicted by TMHMM2.0. Some characterisation within Esche [...] ","uniprotId":"Q0P9A1_CAMJE"},"10641959":{"id":10641959,"externalId":"192222.Cj1156","name":"rho","hasFamily":true,"annotation":"Facilitates transcription termination by a mechanism that involves Rho binding to the nascent RNA, activation of Rho's RNA- dependent ATPase activity, and release of the mRNA from the DNA template","uniprotId":"Q0P9A0_CAMJE"},"10641960":{"id":10641960,"externalId":"192222.Cj1157","name":"dnaX","hasFamily":true,"annotation":"DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. This DNA polymerase also exhibits 3' to 5' exonuclease activity. ","uniprotId":"Q0P999_CAMJE"},"10641961":{"id":10641961,"externalId":"192222.Cj1158c","name":"Cj1158c","hasFamily":true,"annotation":"Original (2000) note: Cj1158c, small hydrophobic protein, len: 75 aa; no Hp match; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0P998_CAMJE"},"10641962":{"id":10641962,"externalId":"192222.Cj1159c","name":"Cj1159c","hasFamily":false,"annotation":"Original (2000) note: Cj1159c, small hydrophobic protein, len: 40 aa; no Hp match; Updated (2006) note: One probable transmembrane helices predicted by TMHMM2.0. Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0P997_CAMJE"},"10641963":{"id":10641963,"externalId":"192222.Cj1160c","name":"Cj1160c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj1160c, small hydrophobic protein, len: 59 aa; no Hp match; Updated (2006) note: One probable transmembrane helix predicted for TMHMM2.0. Also, one signal peptide predicted by SignalP 2.0 HMM. Product function modified to more specific family member. Putative kept within product function. Functional classification - Membranes,lipoproteins and porins","uniprotId":"Q0P996_CAMJE"},"10641964":{"id":10641964,"externalId":"192222.Cj1161c","name":"Cj1161c","hasFamily":true,"annotation":"Original (2000) note: Cj1161c, probable cation-transporting ATPase, len: 699 aa; similar to members of the E1-E2 ATPase family e.g. COPA_HELFE copper-transporting ATPase (732 aa), fasta scores; opt: 1106 z-score: 1124.4 E(): 0, 32.8% identity in 729 aa overlap. 33.2% identity to HP1072. Contains Pfam match to entry PF00403 HMA, Heavy-metal-associated domain containing proteins, 2x Pfam match to entry PF00122 E1-E2_ATPase, and PS00154 E1-E2 ATPases phosphorylation site; Updated (2006) note: Characterisation in more than one bacteria with marginal identity scores. Putative kept within pr [...] ","uniprotId":"Q0P995_CAMJE"},"10641965":{"id":10641965,"externalId":"192222.Cj1162c","name":"Cj1162c","hasFamily":true,"annotation":"Putative heavy-metal-associated domain protein; Original (2000) note: Cj1162c, unknown, len: 64 aa; no Hp match; Updated (2006) note: Prosite domain PS50846 HMA_2,Heavy metal transport/detoxification protein within CDS. Product modified to more specific family member based on motif match. No specific characterisation carried out yet,so putative kept within product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P994_CAMJE"},"10641966":{"id":10641966,"externalId":"192222.Cj1163c","name":"Cj1163c","hasFamily":true,"annotation":"Original (2000) note: Cj1163c, possible cation transport protein, len: 316 aa; similar to e.g. CZCD_ALCEU cobalt-zinc-cadmium resistance protein CZCD (316 aa),fasta scores; opt: 609 z-score: 699.8 E(): 1.1e-31, 36.2% identity in 287 aa overlap. No Hp match. Contains His-rich regin at N-terminus. Also similar to Cj0948c (25.7% identity in 280 aa overlap); Updated (2006) note: Pfam domain PF01545 Cation efflux family domain identified within CDS. Also, six probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Characterisation has been carried ou [...] ","uniprotId":"Q0P993_CAMJE"},"10641967":{"id":10641967,"externalId":"192222.Cj1164c","name":"Cj1164c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1164c, unknown, len: 87 aa; no Hp match. Contains two Cys pairs; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0P992_CAMJE"},"10641968":{"id":10641968,"externalId":"192222.Cj1165c","name":"Cj1165c","hasFamily":true,"annotation":"Original (2000) note: Cj1165c, probable integral membrane protein, len: 164 aa; some similarity to hypothetical proteins e.g. YJJB_ECOLI (128 aa), fasta scores; opt: 171 z-score: 229.8 E(): 1.7e-05, 29.2% identity in 106 aa overlap. No Hp match. Contains one Cys pair; Updated (2006) note: Five probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P991_CAMJE"},"10641969":{"id":10641969,"externalId":"192222.Cj1166c","name":"Cj1166c","hasFamily":true,"annotation":"Original (2000) note: Cj1166c, probable integral membrane protein, len: 258 aa; some similarity to hypothetical proteins e.g. YJJP_ECOLI (277 aa), fasta scores; opt: 292 z-score: 367.7 E(): 3.5e-13, 25.8% identity in 236 aa overlap. No Hp match. Contains one Cys pair; Updated (2006) note: Pfam domain PF06738 Protein of unknown function (DUF1212) identified within CDS. Five probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P990_CAMJE"},"10641970":{"id":10641970,"externalId":"192222.Cj1167","name":"ldh","hasFamily":true,"annotation":"Belongs to the LDH/MDH superfamily. ","uniprotId":"Q0P989_CAMJE"},"10641971":{"id":10641971,"externalId":"192222.Cj1168c","name":"Cj1168c","hasFamily":true,"annotation":"Putative integral membrane protein (DedA homolog); Updated (2006) note: Four probable transmembrane helices predicted by TMHMM2.0; Original (2000) note: Cj1168c, probable integral membrane protein (dedA homolog), len: 200 aa; similar to many members of the dedA family e.g. APL_LACLA alkaline phosphatase like protein (242 aa), fasta scores; opt: 424 z-score: 531.1 E(): 2.8e-22, 37.5% identity in 208 aa overlap, and DEDA_ECOLI DEDA protein (219 aa), fasta scores; opt: 240 z-score: 306.1 E(): 9.5e-10, 24.7% identity in 190 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00597 D [...] ","uniprotId":"Q0P988_CAMJE"},"10641972":{"id":10641972,"externalId":"192222.Cj1169c","name":"Cj1169c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1169c, probable periplasmic protein, len: 75 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P987_CAMJE"},"10641973":{"id":10641973,"externalId":"192222.Cj1170c","name":"omp50","hasFamily":true,"annotation":"Original (2000) note: Cj1170c, omp50, outer membrane protein, len: 473 aa; no Hp match. Contains N-terminal signal sequence and coiled-coil region between aa 60-90. Identification as an outer membrane protein by M-J. Bolla (Jean-Michel.Bolla@medicine.univ-mrs.fr) personal communication; Updated (2006) note: Further characterisation with attached papers give more information on protein naming. Functional classification - Membranes, lipoproteins and porins; PMID:11104668, PMID:15131219","uniprotId":"Q0P986_CAMJE"},"10641974":{"id":10641974,"externalId":"192222.Cj1171c","name":"ppi","hasFamily":true,"annotation":"PPIases accelerate the folding of proteins. It catalyzes the cis-trans isomerization of proline imidic peptide bonds in oligopeptides","uniprotId":"Q0P985_CAMJE"},"10641975":{"id":10641975,"externalId":"192222.Cj1172c","name":"Cj1172c","hasFamily":true,"annotation":"Probable transcriptional regulatory protein Cj1172c; Original (2000) note: Cj1172c, unknown, len: 235 aa; similar to hypothetical proteins e.g. YEBC_ECOLI (246 aa), fasta scores; opt: 508 z-score: 579.3 E(): 5.7e-25,37.1% identity in 229 aa overlap. 58.1% identity to HP0162; Updated (2006) note: Pfam domain PF01709 Domain of unknown function DUF28 identified within CDS. Conserved added to product function. Functional classification -Conserved hypothetical proteins","uniprotId":"Y1172_CAMJE"},"10641976":{"id":10641976,"externalId":"192222.Cj1173","name":"Cj1173","hasFamily":true,"annotation":"Putative efflux protein (Multidrug resistance protein); Original (2000) note: Cj1173, possible efflux protein, len: 113 aa; similar to many e.g. EBR_STAAU ethidium bromide resistance protein (107 aa), fasta scores; opt: 232 z-score: 309.2 E(): 6.3e-10, 38.0% identity in 100 aa overlap. No Hp match. Contains Pfam match to entry PF00893 DUF7, Integral membrane protein. Also similar to Cj0310c (27.1% identity in 107 aa overlap); Updated (2006) note: Pfam domain PF00893 Small Multidrug Resistance protein identified within CDS. Also,four probable transmembrane helices predicted by TMHMM2.0. [...] ","uniprotId":"Q0P983_CAMJE"},"10641977":{"id":10641977,"externalId":"192222.Cj1174","name":"Cj1174","hasFamily":true,"annotation":"Putative efflux protein (Multidrug resistance protein); Original (2000) note: Cj1174, possible efflux protein, len: 102 aa; similar to many e.g. EBR_STAAU ethidium bromide resistance protein (107 aa), fasta scores; opt: 228 z-score: 302.9 E(): 1.4e-09, 33.0% identity in 100 aa overlap. No Hp match. Contains Pfam match to entry PF00893 DUF7, Integral membrane protein. Also similar to Cj0309c (28.7% identity in 94 aa overlap); Updated (2006) note: Pfam domain PF00893 Small Multidrug Resistance protein identified within CDS. Also,four probable transmembrane helices predicted by TMHMM2.0.  [...] ","uniprotId":"Q0P982_CAMJE"},"10641978":{"id":10641978,"externalId":"192222.Cj1175c","name":"argS","hasFamily":true,"annotation":"Arginine--tRNA ligase; Original (2000) note: Cj1175c, argS, arginyl-tRNA synthetase, len: 530 aa; similar to many e.g. SYR_BACSU arginyl-tRNA synthetase (EC 6.1.1.19) (556 aa), fasta scores; opt: 1206 z-score: 1360.5 E(): 0, 38.2% identity in 529 aa overlap. 51.1% identity to HP0319. Contains PS00178 Aminoacyl-transfer RNA synthetases class-I signature, and Pfam match to entry PF00750 tRNA-synt_1d,tRNA synthetases class I (R); Updated (2006) note: Characterised in Bacillus subtilis with acceptable identity score. Putative not added to product function. Functional classification -Aminoa [...] ","uniprotId":"SYR_CAMJE"},"10641979":{"id":10641979,"externalId":"192222.Cj1176c","name":"tatA","hasFamily":true,"annotation":"Part of the twin-arginine translocation (Tat) system that transports large folded proteins containing a characteristic twin- arginine motif in their signal peptide across membranes. TatA could form the protein-conducting channel of the Tat system. ","uniprotId":"TATA_CAMJE"},"10641980":{"id":10641980,"externalId":"192222.Cj1177c","name":"gmk","hasFamily":true,"annotation":"Essential for recycling GMP and indirectly, cGMP. ","uniprotId":"KGUA_CAMJE"},"10641981":{"id":10641981,"externalId":"192222.Cj1178c","name":"Cj1178c","hasFamily":true,"annotation":"Highly acidic protein; Original (2000) note: Cj1178c, highly acidic protein, len: 542 aa; no Hp match. Very Asp- Glu- rich; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0P978_CAMJE"},"10641982":{"id":10641982,"externalId":"192222.Cj1179c","name":"fliR","hasFamily":true,"annotation":"Role in flagellar biosynthesis. ","uniprotId":"Q0P977_CAMJE"},"10641983":{"id":10641983,"externalId":"192222.Cj1180c","name":"Cj1180c","hasFamily":true,"annotation":"Original (2000) note: Cj1180c, probable ABC transporter ATP binding protein, len: 211 aa; similar to many e.g. YBBA_ECOLI hypothetical ABC transporter ATP-binding protein (228 aa), fasta scores; opt: 457 z-score: 554.6 E(): 1.4e-23, 39.7% identity in 199 aa overlap. 49.5% identity to HP0179. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran, ABC transporters; Updated (2006) note: Similar to many identified as lipoprotein-releasing system ATP-binding protein. Putative kept within product function.  [...] ","uniprotId":"Q0P976_CAMJE"},"10641984":{"id":10641984,"externalId":"192222.Cj1181c","name":"tsf","hasFamily":true,"annotation":"Associates with the EF-Tu.GDP complex and induces the exchange of GDP to GTP. It remains bound to the aminoacyl-tRNA.EF- Tu.GTP complex up to the GTP hydrolysis stage on the ribosome. ","uniprotId":"EFTS_CAMJE"},"10641985":{"id":10641985,"externalId":"192222.Cj1182c","name":"rpsB","hasFamily":true,"annotation":"Belongs to the universal ribosomal protein uS2 family. ","uniprotId":"RS2_CAMJE"},"10641986":{"id":10641986,"externalId":"192222.Cj1183c","name":"cfa","hasFamily":true,"annotation":"Original (2000) note: Cj1183c, cfa, probable cyclopropane-fatty-acyl-phospholipid synthase, len: 387 aa; CFA_ECOLI cyclopropane-fatty-acyl-phospholipid synthase (EC 2.1.1.79) (381 aa), fasta scores; opt: 721 z-score: 802.3 E(): 0, 38.7% identity in 362 aa overlap. 62.1% identity to HP0416; Updated (2006) note: Pfam domain PF02353 Cyclopropane-fatty-acyl-phosphol lipid synthase identified within CDS. Further support given to product function. Characterised in Escherichia coli with acceptable identity score. Appropiate motifs present. Putative removed from product function. Functional cl [...] ","uniprotId":"Q0P973_CAMJE"},"10641987":{"id":10641987,"externalId":"192222.Cj1184c","name":"petC","hasFamily":true,"annotation":"Original (2000) note: Cj1184c, petC, possible ubiquinol-cytochrome C reductase cytochrome C subunit,len: 374 aa; some similarity to e.g. CYC_TETPY cytochrome C (109 aa), fasta scores; opt: 124 z-score: 145.5 E(): 0.83, 25.6% identity in 121 aa overlap. 36.4% identity to HP1538. Contains Pfam match to entry PF00034 cytochrome_c,Cytochrome c, probable N-terminal signal sequence, and possible membrane anchor at C-terminus; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Specific characterisation with acceptable identity score not carried out yet, so putative [...] ","uniprotId":"Q0P972_CAMJE"},"10641988":{"id":10641988,"externalId":"192222.Cj1185c","name":"petB","hasFamily":true,"annotation":"Component of the ubiquinol-cytochrome c reductase complex (complex III or cytochrome b-c1 complex), which is a respiratory chain that generates an electrochemical potential coupled to ATP synthesis. ","uniprotId":"Q0P971_CAMJE"},"10641989":{"id":10641989,"externalId":"192222.Cj1186c","name":"petA","hasFamily":true,"annotation":"Component of the ubiquinol-cytochrome c reductase complex (complex III or cytochrome b-c1 complex), which is a respiratory chain that generates an electrochemical potential coupled to ATP synthesis. ","uniprotId":"Q0P970_CAMJE"},"10641990":{"id":10641990,"externalId":"192222.Cj1187c","name":"arsB","hasFamily":true,"annotation":"Original (2000) note: Cj1187c, arsB, possible arsenical pump membrane protein, len: 428 aa; simlar to many e.g. ARSB_ECOLI arsenical pump membrane protein (429 aa), fasta scores; opt: 805 z-score: 920.7 E(): 0, 33.5% identity in 433 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02040 Arsenical pump membrane protein identified within CDS. Also, eleven probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Characterised in Escherichia coli with marginal identity score. Appropriate motifs present. Putative removed from product functi [...] ","uniprotId":"Q0P969_CAMJE"},"10641991":{"id":10641991,"externalId":"192222.Cj1188c","name":"gidA","hasFamily":true,"annotation":"NAD-binding protein involved in the addition of a carboxymethylaminomethyl (cmnm) group at the wobble position (U34) of certain tRNAs, forming tRNA-cmnm(5)s(2)U34","uniprotId":"MNMG_CAMJE"},"10641992":{"id":10641992,"externalId":"192222.Cj1189c","name":"cetB","hasFamily":true,"annotation":"Bipartate energy taxis response protein cetB; Original (2000) note: Cj1189c, possible signal-transduction sensor protein, len: 165 aa; similar to N-terminus of genes involved in oxygen-sensing e.g. AER_ECOLI aerotaxis receptor (506 aa), fasta scores; opt: 342 z-score: 407.5 E(): 2.1e-15, 36.4% identity in 140 aa overlap, and NIFL_AZOVI nitrogen fixation regulatory protein (519 aa), fasta scores; opt: 175 z-score: 212.5 E(): 0.00015, 33.6% identity in 110 aa overlap. No Hp match. Also similar to Cj1191c (63.4% identity in 161 aa overlap); Updated (2006) note: Pfam domain PF00989 PAS fol [...] ","uniprotId":"Q0P967_CAMJE"},"10641993":{"id":10641993,"externalId":"192222.Cj1190c","name":"cetA","hasFamily":true,"annotation":"Bipartate energy taxis response protein cetA; Original (2000) note: Cj1190c, probable MCP-domain signal transduction protein, len: 459 aa; similar to many e.g. TLPC_BACSU methyl-accepting chemotaxis protein TLPC (573 aa), fasta scores; opt: 371 z-score: 397.9 E(): 7.3e-15, 26.2% identity in 404 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00015 MCPsignal,Methyl-accepting chemotaxis protein (MCP) signaling domain, and probable membrane spanning domain at N-terminus; Updated (2006) note: Characterisation work carried out in Campylobacter jejuni. Product function updated. Li [...] ","uniprotId":"Q0P966_CAMJE"},"10641994":{"id":10641994,"externalId":"192222.Cj1191c","name":"Cj1191c","hasFamily":true,"annotation":"Putative PAS domain containing signal-transduction sensor protein; Original (2000) note: Cj1191c, possible signal-transduction sensor protein, len: 164 aa; similar to N-terminus of genes involved in oxygen-sensing e.g. AER_ECOLI aerotaxis receptor (506 aa), fasta scores; opt: 297 z-score: 373.5 E(): 1.7e-13, 34.3% identity in 134 aa overlap, and NIFL_AZOVI nitrogen fixation regulatory protein (519 aa), fasta scores; opt: 178 z-score: 226.6 E(): 2.5e-05, 33.0% identity in 103 aa overlap. No Hp match. Also similar to Cj1189c (63.4% identity in 161 aa overlap); Updated (2006) note: Pfam d [...] ","uniprotId":"Q0P965_CAMJE"},"10641995":{"id":10641995,"externalId":"192222.Cj1192","name":"dctA","hasFamily":true,"annotation":"Belongs to the dicarboxylate/amino acid:cation symporter (DAACS) (TC 2.A.23) family","uniprotId":"Q0P964_CAMJE"},"10641996":{"id":10641996,"externalId":"192222.Cj1193c","name":"Cj1193c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1193c, probable periplasmic protein, len: 268 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P963_CAMJE"},"10641997":{"id":10641997,"externalId":"192222.Cj1194","name":"Cj1194","hasFamily":true,"annotation":"Phosphate transporter; Original (2000) note: Cj1194, possible phosphate permease, len: 508 aa; similar to many predicted phosphate permeases e.g. TR:Q50684 Mycobacterium tuberculosis putative phosphate permease Rv2281 MTCY339.29C (552 aa),fasta scores; opt: 900 z-score: 971.5 E(): 0, 46.6% identity in 511 aa overlap. 32.7% identity to HP1491; Updated (2006) note: Pfam domain PF01384 Phosphate transporter family identified within CDS. Also, thirteen probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. No specific characterisation has been carr [...] ","uniprotId":"Q0P962_CAMJE"},"10641998":{"id":10641998,"externalId":"192222.Cj1195c","name":"pyrC2","hasFamily":true,"annotation":"Putative dihydroorotase; Original (2000) note: Cj1195c, pyrC2, probable dihydroorotase, len: 392 aa; similar to e.g. PYRC_BACSU dihydroorotase (EC 3.5.2.3) (428 aa), fasta scores; opt: 256 z-score: 307.9 E(): 7.5e-10, 26.1% identity in 402 aa overlap. 30.1% identity to HP0266. No apparent similarity to pyrC, Cj0259; Updated (2006) note: Pfam domain PF01979 Amidohydrolase family identified within CDS. Further support given to product function. Characterised in Bacillus subtilis with marginal identity score. Putative kept in product function. Functional classification -Pyrimidine ribonuc [...] ","uniprotId":"Q0P961_CAMJE"},"10641999":{"id":10641999,"externalId":"192222.Cj1196c","name":"gpsA","hasFamily":true,"annotation":"Belongs to the NAD-dependent glycerol-3-phosphate dehydrogenase family","uniprotId":"GPDA_CAMJE"},"10642000":{"id":10642000,"externalId":"192222.Cj1197c","name":"gatB","hasFamily":true,"annotation":"Allows the formation of correctly charged Asn-tRNA(Asn) or Gln-tRNA(Gln) through the transamidation of misacylated Asp-tRNA(Asn) or Glu-tRNA(Gln) in organisms which lack either or both of asparaginyl- tRNA or glutaminyl-tRNA synthetases. The reaction takes place in the presence of glutamine and ATP through an activated phospho-Asp- tRNA(Asn) or phospho-Glu-tRNA(Gln)","uniprotId":"GATB_CAMJE"},"10642001":{"id":10642001,"externalId":"192222.Cj1198","name":"luxS","hasFamily":true,"annotation":"Involved in the synthesis of autoinducer 2 (AI-2) which is secreted by bacteria and is used to communicate both the cell density and the metabolic potential of the environment. The regulation of gene expression in response to changes in cell density is called quorum sensing. Catalyzes the transformation of S-ribosylhomocysteine (RHC) to homocysteine (HC) and 4,5-dihydroxy-2,3-pentadione (DPD). ","uniprotId":"LUXS_CAMJE"},"10642002":{"id":10642002,"externalId":"192222.Cj1199","name":"Cj1199","hasFamily":true,"annotation":"Belongs to the iron/ascorbate-dependent oxidoreductase family","uniprotId":"Q0P957_CAMJE"},"10642003":{"id":10642003,"externalId":"192222.Cj1200","name":"Cj1200","hasFamily":true,"annotation":"Putative NLPA family lipoprotein; Original (2000) note: Cj1200, probable periplasmic protein, len: 262 aa; similar to members of the nlpA family of outem membrane lipoproteins, e.g. NLPA_ECOLI lipoprotein-28 precursor (272 aa), fasta scores; opt: 560 z-score: 650.6 E(): 6.1e-29, 39.7% identity in 239 aa overlap, and PLPC_PASHA outer membrane lipoprotein 3 precursor (263 aa), fasta scores; opt: 564 z-score: 655.4 E(): 3.3e-29, 38.6% identity in 259 aa overlap. No Hp ortholog. Contains probable N-terminal signal sequence. Also similar to Cj0772c (57.5% identity in 254 aa overlap), Cj0771 [...] ","uniprotId":"Q0P956_CAMJE"},"10642004":{"id":10642004,"externalId":"192222.Cj1201","name":"metE","hasFamily":true,"annotation":"Catalyzes the transfer of a methyl group from 5- methyltetrahydrofolate to homocysteine resulting in methionine formation","uniprotId":"METE_CAMJE"},"10642005":{"id":10642005,"externalId":"192222.Cj1202","name":"metF","hasFamily":true,"annotation":"Belongs to the methylenetetrahydrofolate reductase family. ","uniprotId":"Q0P954_CAMJE"},"10642006":{"id":10642006,"externalId":"192222.Cj1203c","name":"Cj1203c","hasFamily":false,"annotation":"Original (2000) note: Cj1203c, probable integral membrane protein, len: 63 aa; no Hp match; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Functional classification -Central intermediary metabolism - General","uniprotId":"Q0P953_CAMJE"},"10642007":{"id":10642007,"externalId":"192222.Cj1204c","name":"atpB","hasFamily":true,"annotation":"Key component of the proton channel; it plays a direct role in the translocation of protons across the membrane. ","uniprotId":"ATP6_CAMJE"},"10642008":{"id":10642008,"externalId":"192222.Cj1205c","name":"radA","hasFamily":true,"annotation":"DNA-dependent ATPase involved in processing of recombination intermediates, plays a role in repairing DNA breaks. Stimulates the branch migration of RecA-mediated strand transfer reactions, allowing the 3' invading strand to extend heteroduplex DNA faster. Binds ssDNA in the presence of ADP but not other nucleotides, has ATPase activity that is stimulated by ssDNA and various branched DNA structures, but inhibited by SSB. Does not have RecA's homology-searching function. ","uniprotId":"RADA_CAMJE"},"10642009":{"id":10642009,"externalId":"192222.Cj1206c","name":"ftsY","hasFamily":true,"annotation":"Involved in targeting and insertion of nascent membrane proteins into the cytoplasmic membrane. Acts as a receptor for the complex formed by the signal recognition particle (SRP) and the ribosome-nascent chain (RNC). Interaction with SRP-RNC leads to the transfer of the RNC complex to the Sec translocase for insertion into the membrane, the hydrolysis of GTP by both Ffh and FtsY, and the dissociation of the SRP-FtsY complex into the individual components. ","uniprotId":"Q0P950_CAMJE"},"10642010":{"id":10642010,"externalId":"192222.Cj1207c","name":"Cj1207c","hasFamily":true,"annotation":"Original (2000) note: Cj1207c, possible lipoprotein thiredoxin, len: 185 aa; some similarity to TR:G4100188 (EMBL:U95250) Treponema pallidum predicted thioredoxin (185 aa), fasta scores; opt: 196 z-score: 242.3 E(): 3.4e-06, 31.5% identity in 111 aa overlap, and HELX_RHOCA thiol:disulfide interchange protein(176 aa), fatsa scores; opt: 153 z-score: 192.6 E(): 0.002, 23.2% identity in 142 aa overlap. 23.7% identity to HP0762. Also similar to Cj1106 (29.4% identity in 201 aa overlap). Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane l [...] ","uniprotId":"Q0P949_CAMJE"},"10642011":{"id":10642011,"externalId":"192222.Cj1208","name":"Cj1208","hasFamily":true,"annotation":"Belongs to the 5-formyltetrahydrofolate cyclo-ligase family","uniprotId":"Q0P948_CAMJE"},"10642012":{"id":10642012,"externalId":"192222.Cj1209","name":"rny","hasFamily":true,"annotation":"Endoribonuclease that initiates mRNA decay. ","uniprotId":"RNY_CAMJE"},"10642013":{"id":10642013,"externalId":"192222.Cj1210","name":"Cj1210","hasFamily":true,"annotation":"Original (2000) note: Cj1210, probable integral membrane protein, len: 185 aa; similar to members of the dedA family e.g. DEDA_ECOLI DEDA protein (219 aa), fasta scores; opt: 193 z-score: 248.3 E(): 1.6e-06, 26.7% identity in 172 aa overlap. 35.6% identity to HP1162. Also similar to Cj0928 (27.7% identity in 177 aa overlap), and Cj1168c (23.0% identity in 200 aa overlap). Contains Pfam match to entry PF00597 DedA, DedA family; Updated (2006) note: Four probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P946_CAMJE"},"10642014":{"id":10642014,"externalId":"192222.Cj1211","name":"Cj1211","hasFamily":true,"annotation":"Putative competence family protein; Original (2000) note: Cj1211, probable integral membrane protein, len: 419 aa; similar to hypothetical membrane proteins e.g. TR:O51537 (EMBL:AE001160) BB0591 (416 aa), fasta scores; opt: 222 z-score: 251.2 E(): 1.1e-06, 26.4% identity in 405 aa overlap. 33.9% identity to HP1361 (called competence locus E (comE3)); Updated (2006) note: Pfam motif PF03772 Competence protein was identified within CDS. Nine probable transmembrane helices predicted by TMHMM2.0. Product function modified to more specific family member based on motif match. No specific cha [...] ","uniprotId":"Q0P945_CAMJE"},"10642015":{"id":10642015,"externalId":"192222.Cj1212c","name":"rbn","hasFamily":true,"annotation":"Putative ribonuclease BN; Original (2000) note: Cj1212c, rbn, possible ribonuclease BN, len: RBN_ECOLI ribonuclease BN (EC 3.1.-.-) (290 aa), fasta scores; opt: 365 z-score: 430.3 E(): 1.1e-16, 28.5% identity in 242 aa overlap. 29.2% identity to HP1407; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Characterised in Escherichia coli with marginal identity score. Putative kept within product function. Functional classification -Aminoacyl tRNA synthetases and their modification; PMID:8955422","uniprotId":"Q0P944_CAMJE"},"10642016":{"id":10642016,"externalId":"192222.Cj1213c","name":"glcD","hasFamily":true,"annotation":"Original (2000) note: Cj1213c, glcD, probable glycolate oxidase subunit D, len: 460 aa; similar to e.g. GLCD_ECOLI glycolate oxidase subunit GLCD (499 aa), fasta scores; opt: 952 z-score: 1099.5 E(): 0, 37.0% identity in 419 aa overlap. 64.8% identity to HP0509; Updated (2006) note: Pfam domains PF02913 FAD linked oxidases, C-terminal domain and PF01565 FAD binding domain were identified within CDS. Further support given to product function. Characterised in Escherichia coli with marginal identity score. Putative kept in product function. Functional classification - Degradation - Carbo [...] ","uniprotId":"Q0P943_CAMJE"},"10642017":{"id":10642017,"externalId":"192222.Cj1214c","name":"Cj1214c","hasFamily":true,"annotation":"Putative exporting protein; Original (2000) note: Cj1214c, unknown, len: 241 aa; similar to N-terminal half of HP0508 (452 aa) (24.9% identity in 245 aa overlap); Updated (2006) note: One signal peptide predicted by SignalP 2.0 HMM. Product function modified to more specific family member. Putative kept within product function. Paper identified linking protein to glycoprotein (PMID:12186869). Functional classification -Transport/binding proteins - Other; PMID:12186869","uniprotId":"Q0P942_CAMJE"},"10642018":{"id":10642018,"externalId":"192222.Cj1215","name":"Cj1215","hasFamily":true,"annotation":"Putative peptidase M23 family protein; Original (2000) note: Cj1215, probable periplasmic protein, len: 386 aa; similar to hypothetical proteins e.g. YEBA_ECOLI (419 aa), fasta scores; opt: 491 z-score: 556.7 E(): 1e-23, 30.2% identity in 338 aa overlap. 44.0% identity to HP0506. Also similar, in part, to Cj1087c (40.6% identity in 133 aa overlap), and Cj1235 (29.6% identity in 206 aa overlap); Updated (2006) note: Pfam domain PF01551 Peptidase family M23 identified within CDS. Further support given to product function. Product modified to more specific family member based on motif res [...] ","uniprotId":"Q0P941_CAMJE"},"10642019":{"id":10642019,"externalId":"192222.Cj1216c","name":"Cj1216c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1216c, unknown, len: 67 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0P940_CAMJE"},"10642020":{"id":10642020,"externalId":"192222.Cj1217c","name":"Cj1217c","hasFamily":true,"annotation":"Multicopper oxidase with polyphenol oxidase activity. ","uniprotId":"POLOX_CAMJE"},"10642021":{"id":10642021,"externalId":"192222.Cj1218c","name":"Cj1218c","hasFamily":true,"annotation":"Original (2000) note: Cj1218c, ribA, probable riboflavin synthase alpha chain, len: 203 aa; similar to e.g. RISA_BACSU riboflavin synthase alpha chain (EC 2.5.1.9) (215 aa), fasta scores; opt: 369 z-score: 435.1 E(): 6.2e-17, 32.7% identity in 208 aa overlap. 47.6% identity to HP1574. Contains Pfam match to entry PF00677 Lum_binding, Lumazine binding domain; Updated (2006) note: No specific characterisation with acceptable identity score carried out yet. Putative kept within product function. Functional classification -Biosynthesis of cofactors, prosthetic groups and carriers - Ribofla [...] ","uniprotId":"Q0P938_CAMJE"},"10642022":{"id":10642022,"externalId":"192222.Cj1219c","name":"Cj1219c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1219c, probable periplasmic protein, len: 846 aa; 27.3% identity to HP0358. Weak similarity to FLID_ECOLI flagellar hook-associated protein 2 (467 aa), fasta scores; opt: 168 z-score: 180.0 E(): 0.0099, 24.2% identity in 446 aa overlap, and to Cj0530 (22.1% identity in 891 aa overlap). Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P937_CAMJE"},"10642023":{"id":10642023,"externalId":"192222.Cj1220","name":"groES","hasFamily":true,"annotation":"Binds to Cpn60 in the presence of Mg-ATP and suppresses the ATPase activity of the latter","uniprotId":"CH10_CAMJE"},"10642024":{"id":10642024,"externalId":"192222.Cj1221","name":"groEL","hasFamily":true,"annotation":"Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions. ","uniprotId":"CH60_CAMJE"},"10642025":{"id":10642025,"externalId":"192222.Cj1222c","name":"dccS","hasFamily":true,"annotation":"Original (2000) note: Cj1222c, probable two-component sensor (histidine kinase), len: 396 aa; similar to many sensor proteins e.g. RSTB_ECOLI sensor protein RSTB (433 aa), fasta scores; opt: 226 z-score: 267.7 E(): 1.3e-07, 24.5% identity in 237 aa overlap. 28.5% identity to HP1364. Contains Pfam match to entry PF00512 signal, Signal carboxyl-terminal domain, and two possible membrane spanning domains; Updated (2006) note: Pfam domains PF02518 Histidine kinase-, DNA gyrase B-, and HSP90-like ATPase motif and PF00512 His Kinase A (phosphoacceptor) domain were identified within CDS. Pros [...] ","uniprotId":"Q0P934_CAMJE"},"10642026":{"id":10642026,"externalId":"192222.Cj1223c","name":"dccR","hasFamily":true,"annotation":"Two-component regulator; Original (2000) note: Cj1223c, probable two-component regulator, len: 221 aa; similar to many e.g. CIAR_STRPN transcriptional regulatory protein CIAR (224 aa), fasta scores; opt: 426 z-score: 522.6 E(): 8.2e-22,37.0% identity in 219 aa overlap. 39.4% identity to HP1365. Contains Pfam match to entry PF00072 response_reg,Response regulator receiver domain, and Pfam match to entry PF00486 trans_reg_C, Transcriptional regulatory protein, C terminal; Updated (2006) note: Characterised in Campylobacter jejuni. Literature search identified characterisation paper where [...] ","uniprotId":"Q0P933_CAMJE"},"10642027":{"id":10642027,"externalId":"192222.Cj1224","name":"Cj1224","hasFamily":true,"annotation":"Putative iron-binding protein; Original (2000) note: Cj1224, possible iron-binding protein, len: 199 aa; some simlarity to TR:Q58157 (EMBL:U67520) Methanococcus jannaschii MJ0747 (169 aa),fasta scores; opt: 230 z-score: 274.3 E(): 5.6e-08, 33.9% identity in 127 aa overlap. Weak similarity to members of the eukaryotic oxygen-binding hemerythrin family, e.g. HEMM_THEZO myohemerythrin (118 aa), fasta scores; opt: 159 z-score: 196.6 E(): 0.0012, 30.5% identity in 118 aa overlap. Note that all of the iron-binding residues are conserved. No Hp match. Also similar to Cj0045c (29.2% identity i [...] ","uniprotId":"Q0P932_CAMJE"},"10642028":{"id":10642028,"externalId":"192222.Cj1225","name":"Cj1225","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1225, unknown, len: 82 aa; weak similarity to CDS from Borrelia burgdorferi repeated DNA element TR:Q45025 (EMBL:X87127) (210 aa), fasta scores; opt: 117 z-score: 159.0 E(): 0.15, 33.3% identity in 78 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF01402 Ribbon-helix-helix protein, copG family identified within CDS. No specific characterisation has yet been carried out yet. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P931_CAMJE"},"10642029":{"id":10642029,"externalId":"192222.Cj1226c","name":"Cj1226c","hasFamily":true,"annotation":"Histidine kinase; Original (2000) note: Cj1226c, probable two-component sensor (histidine kinase), len: 415 aa; similar to many sensor proteins e.g. CPXA_ECOLI sensor protein CPXA (457 aa), fasta scores; opt: 205 z-score: 241.7 E(): 3.7e-06, 25.2% identity in 270 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00512 signal,Signal carboxyl-terminal domain, and two possible membrane spanning domains. Also similar to Cj1262 (31.0% identity in 410 aa overlap); Updated (2006) note: Pfam domains PF02518 Histidine kinase-, DNA gyrase B-, and HSP90-like ATPase motifs,PF00512 His Kin [...] ","uniprotId":"Q0P930_CAMJE"},"10642030":{"id":10642030,"externalId":"192222.Cj1227c","name":"Cj1227c","hasFamily":true,"annotation":"Original (2000) note: Cj1227c, probable two-component regulator, len: 224 aa; identical to TR:Q46095 (EMBL:U27271) C. jejuni CDS (called ompR) (198 aa), and similar to many e.g. YYCF_BACSU hypothetical 27.2 kd sensory transduction protein (235 aa), fasta scores; opt: 490 z-score: 561.1 E(): 5.9e-24, 36.4% identity in 225 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00072 response_reg, Response regulator receiver domain, and Pfam match to entry PF00486 trans_reg_C,Transcriptional regulatory protein, C terminal. Also similar to Cj1261 (50.7% identity in 223 aa overlap); Upd [...] ","uniprotId":"Q0P929_CAMJE"},"10642031":{"id":10642031,"externalId":"192222.Cj1228c","name":"htrA","hasFamily":true,"annotation":"Belongs to the peptidase S1C family. ","uniprotId":"Q0P928_CAMJE"},"10642032":{"id":10642032,"externalId":"192222.Cj1229","name":"cbpA","hasFamily":true,"annotation":"Original (2000) note: Cj1229, cbpA, probable curved-DNA binding protein, len: 297 aa; similar to e.g. CBPA_ECOLI curved DNA-binding protein (306 aa), fatsa scores; opt: 514 z-score: 550.9 E(): 2.2e-23, 32.8% identity in 287 aa overlap. 52.0% identity to HP1024. Contains S00636 Nt-dnaJ domain signature, and Pfam match to entry PF00226 DnaJ, DnaJ domain; Updated (2006) note: Pfam domain PF01556 DnaJ C terminal region identified within CDS. Further support given to product function. Characterised in Escherichia coli with marginal identity score. Putative kept within product function. Func [...] ","uniprotId":"Q0P927_CAMJE"},"10642033":{"id":10642033,"externalId":"192222.Cj1230","name":"hspR","hasFamily":true,"annotation":"Original (2000) note: Cj1230, hspR, possible heat shock transcriptional regulator, len: 124 aa; similar to SPR_STRCO putative heat shock protein HSPR (151 aa), fatsa scores; opt: 309 z-score: 385.2 E(): 3.7e-14, 42.4% identity in 118 aa overlap, and to other members of the MerR family of transcriptional regulators. 63.3% identity to HP1025. Contains PS00552 Bacterial regulatory proteins,merR family signature, Pfam match to entry PF00376 merR,and probable helix-turn-helix motif at aa 11-32 (Score 1333, +3.73 SD); Updated (2006) note: Characterised in Campylobacter jejuni (PMID:15758235) [...] ","uniprotId":"Q0P926_CAMJE"},"10642034":{"id":10642034,"externalId":"192222.Cj1231","name":"kefB","hasFamily":true,"annotation":"Belongs to the monovalent cation:proton antiporter 2 (CPA2) transporter (TC 2.A.37) family","uniprotId":"Q0P925_CAMJE"},"10642035":{"id":10642035,"externalId":"192222.Cj1232","name":"Cj1232","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1232, unknown, len: 110 aa; no Hp match. Lys-rich. Functional classification -Unknown","uniprotId":"Q0P924_CAMJE"},"10642036":{"id":10642036,"externalId":"192222.Cj1233","name":"Cj1233","hasFamily":true,"annotation":"Putative HAD-superfamily hydrolase; Original (2000) note: Cj1233, probable hydrolase,len: 206 aa; similar to e.g. GPH_ECOLI phosphoglycolate phosphatase (252 aa), fasta scores; opt: 177 z-score: 216.3 E(): 9.5e-05, 27.6% identity in 199 aa overlap. No Hp match. Contains Pfam match to entry PF00702 Hydrolase,haloacid dehalogenase-like hydrolase. Also similar to Cj1477c (27.3% identity in 205 aa overlap); Updated (2006) note: Characterisation work within Escherichia coli, however, identity scores were marginal. Product modified to more specific family member based on motif matches. Putat [...] ","uniprotId":"Q0P923_CAMJE"},"10642037":{"id":10642037,"externalId":"192222.Cj1234","name":"glyS","hasFamily":true,"annotation":"Glycine--tRNA ligase beta subunit; Original (2000) note: Cj1234, glyS, probable glycyl-tRNA synthetase beta chain, len: 664 aa; similar to e.g. SYGB_BACSU glycyl-tRNA synthetase beta chain (EC 6.1.1.14) (679 aa), fasta scores; opt: 1252 z-score: 1414.7 E(): 0, 35.9% identity in 682 aa overlap. 42.9% identity to HP0972; Updated (2006) note: Pfam domain PF02092 Glycyl-tRNA synthetase beta subunit identified within CDS. Further support given to product function. Characterised in Escherichia coli with acceptable identity score, so putative not added to product function. Functional classifi [...] ","uniprotId":"Q0P922_CAMJE"},"10642038":{"id":10642038,"externalId":"192222.Cj1235","name":"Cj1235","hasFamily":true,"annotation":"Putative peptidase M23 family protein; Original (2000) note: Cj1235, probable periplasmic protein, len: 273 aa; similar to many e.g. TR:Q57503 (EMBL:U07173) Vibrio cholerae ToxR-activated gene, tagE (302 aa), fasta scores; opt: 319 z-score: 373.7 E(): 1.6e-13, 37.0% identity in 192 aa overlap, and YEBA_ECOLI (419 aa), fasta scores; opt: 299 z-score: 348.5 E(): 4.1e-12, 44.3% identity in 115 aa overlap. No Hp ortholog. Also similar to Cj0131 (30.2% identity in 222 aa overlap),Cj1087c (40.7% identity in 123 aa overlap), and Cj1215 (29.6% identity in 206 aa overlap); Updated (2006) note:  [...] ","uniprotId":"Q0P921_CAMJE"},"10642039":{"id":10642039,"externalId":"192222.Cj1236","name":"Cj1236","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1236, unknown, len: 316 aa; similar to hypothetical proteins e.g. TR:O66873 (EMBL:AE000698) Aquifex aeolicus AQ_622 (320 aa), fasta scores; opt: 263 z-score: 318.9 E(): 1.8e-10, 23.8% identity in 323 aa overlap. 32.4% identity to HP0812; Updated (2006) note: Pfam domain PF02636 Uncharacterized ACR, COG1565 identified within CDS. Conserved added to product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P920_CAMJE"},"10642040":{"id":10642040,"externalId":"192222.Cj1237c","name":"Cj1237c","hasFamily":true,"annotation":"Putative phosphatase; Original (2000) note: Cj1237c, possible phosphatase, len: 324 aa; weak similarity to e.g. GPPA_ECOLI guanosine-5'-triphosphate,3'-diphosphate pyrophosphatase (494 aa), fasta scores; opt: 174 z-score: 201.9 E(): 0.0006, 25.7% identity in 323 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02541 Ppx/GppA phosphatase family was identified within CDS. Further support given to product function. No specific characterisation with acceptable identity score carried out yet, so putative kept within product function (previously was posssible). Functional classifi [...] ","uniprotId":"Q0P919_CAMJE"},"10642041":{"id":10642041,"externalId":"192222.Cj1238","name":"pdxJ","hasFamily":true,"annotation":"Catalyzes the complicated ring closure reaction between the two acyclic compounds 1-deoxy-D-xylulose-5-phosphate (DXP) and 3-amino- 2-oxopropyl phosphate (1-amino-acetone-3-phosphate or AAP) to form pyridoxine 5'-phosphate (PNP) and inorganic phosphate. ","uniprotId":"PDXJ_CAMJE"},"10642042":{"id":10642042,"externalId":"192222.Cj1239","name":"pdxA","hasFamily":true,"annotation":"Catalyzes the NAD(P)-dependent oxidation of 4-(phosphooxy)-L- threonine (HTP) into 2-amino-3-oxo-4-(phosphooxy)butyric acid which spontaneously decarboxylates to form 3-amino-2-oxopropyl phosphate (AHAP)","uniprotId":"PDXA_CAMJE"},"10642043":{"id":10642043,"externalId":"192222.Cj1240c","name":"Cj1240c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1240c, probable periplasmic protein, len: 200 aa; no Hp match. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0P916_CAMJE"},"10642044":{"id":10642044,"externalId":"192222.Cj1241","name":"Cj1241","hasFamily":true,"annotation":"Putative MFS (Major Facilitator Superfamily) transporter protein; Original (2000) note: Cj1241, probable transmembrane transport protein, len: 399 aa; similar to e.g. ARAJ_ECOLI protein ARAJ precursor (394 aa), fasta scores; opt: 579 z-score: 680.4 E(): 1.3e-30, 28.8% identity in 385 aa overlap, and CMLR_STRLI chloramphenicol resistance protein (392 aa), fasta scores; opt: 444 z-score: 523.2 E(): 7.6e-22, 25.5% identity in 380 aa overlap. 27.4% identity to HP1185; Updated (2006) note: Pfam domain PF07690 Major Facilitator Superfamily identified within CDS. Also,twelve probable transmem [...] ","uniprotId":"Q0P915_CAMJE"},"10642045":{"id":10642045,"externalId":"192222.Cj1242","name":"Cj1242","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1242, unknown, len: 106 aa; no Hp match; Updated (2006) note: Literature search identified papers giving potential clues to product function. Functional classification - Unknown; PMID:14985343, PMID:15812042","uniprotId":"Q0P914_CAMJE"},"10642046":{"id":10642046,"externalId":"192222.Cj1243","name":"hemE","hasFamily":true,"annotation":"Catalyzes the decarboxylation of four acetate groups of uroporphyrinogen-III to yield coproporphyrinogen-III. ","uniprotId":"DCUP_CAMJE"},"10642047":{"id":10642047,"externalId":"192222.Cj1244","name":"Cj1244","hasFamily":true,"annotation":"Putative radical SAM domain protein; Original (2000) note: Cj1244, unknown, len: 300 aa; similar to hypothetical proteins e.g. YD12_METJA Methanococcus jannaschii MJ1312 (321 aa), fasta scores; opt: 436 z-score: 499.9 E(): 1.5e-20, 30.7% identity in 313 aa overlap. 40.1% identity to HP0117; Updated (2006) note: Pfam domain PF04055 Radical SAM superfamily identified within CDS. Product modified to more specific family member based on motif match. No specific characterisation carried out yet, so putative kept within product function. Functional classification -Misc","uniprotId":"Q0P912_CAMJE"},"10642048":{"id":10642048,"externalId":"192222.Cj1245c","name":"Cj1245c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj1245c, possible membrane protein, len: 398 aa; no Hp match; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P911_CAMJE"},"10642049":{"id":10642049,"externalId":"192222.Cj1246c","name":"uvrC","hasFamily":true,"annotation":"The UvrABC repair system catalyzes the recognition and processing of DNA lesions. UvrC both incises the 5' and 3' sides of the lesion. The N-terminal half is responsible for the 3' incision and the C-terminal half is responsible for the 5' incision","uniprotId":"UVRC_CAMJE"},"10642050":{"id":10642050,"externalId":"192222.Cj1247c","name":"Cj1247c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1247c, unknown, len: 164 aa; 27.1% identity to HP0820. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P909_CAMJE"},"10642051":{"id":10642051,"externalId":"192222.Cj1248","name":"guaA","hasFamily":true,"annotation":"Catalyzes the synthesis of GMP from XMP","uniprotId":"GUAA_CAMJE"},"10642052":{"id":10642052,"externalId":"192222.Cj1249","name":"Cj1249","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1249, unknown, len: 486 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0P907_CAMJE"},"10642053":{"id":10642053,"externalId":"192222.Cj1250","name":"purD","hasFamily":true,"annotation":"Phosphoribosylamine--glycine ligase; Original (2000) note: Cj1250, purD, probable phosphoribosylamine--glycine ligase, len: 416 aa; similar to e.g. PUR2_ECOLI phosphoribosylamine--glycine ligase (EC 6.3.4.13) (429 aa), fasta scores; opt: 981 z-score: 1186.4 E(): 0, 39.3% identity in 428 aa overlap. 38.2% identity to HP1218. Contains Pfam match to entry PF01071 GARS,Phosphoribosylglycinamide synthetase (GARS); Updated (2006) note: Pfam domain PF02844, PF01071,PF02842 and PF02843 Phosphoribosylglycinamide synthetase,N domain, B domain, C domain and ATP-grasp (A) domains were identified w [...] ","uniprotId":"PUR2_CAMJE"},"10642054":{"id":10642054,"externalId":"192222.Cj1251","name":"Cj1251","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1251, unknown, len: 150 aa; similar to hypothetical proteins e.g. TR:Q56614 (EMBL:U39068) Vibrio cholerae orfY (145 aa), fasta scores; opt: 220 z-score: 291.8 E(): 5.9e-09, 28.6% identity in 140 aa overlap. 27.9% identity to HP1217; Updated (2006) note: Pfam domain PF06271 RDD family identified within CDS. Also, two probable transmembrane helices predicted by TMHMM2.0. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P905_CAMJE"},"10642055":{"id":10642055,"externalId":"192222.Cj1252","name":"lptD","hasFamily":true,"annotation":"Involved in the assembly of lipopolysaccharide (LPS) at the surface of the outer membrane","uniprotId":"Q0P904_CAMJE"},"10642056":{"id":10642056,"externalId":"192222.Cj1253","name":"pnp","hasFamily":true,"annotation":"Involved in mRNA degradation. Catalyzes the phosphorolysis of single-stranded polyribonucleotides processively in the 3'- to 5'- direction","uniprotId":"PNP_CAMJE"},"10642057":{"id":10642057,"externalId":"192222.Cj1254","name":"Cj1254","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1254, unknown, len: 160 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins and also to G/U mismatch-specific DNA glycosyl from other bacteria. Functional classification - Unknown","uniprotId":"Q0P902_CAMJE"},"10642058":{"id":10642058,"externalId":"192222.Cj1255","name":"Cj1255","hasFamily":true,"annotation":"Belongs to the 4-oxalocrotonate tautomerase family. ","uniprotId":"Q0P901_CAMJE"},"10642059":{"id":10642059,"externalId":"192222.Cj1256c","name":"Cj1256c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj1256c, possible membrane protein, len: 220 aa; no Hp match. Contains possible membrane spanning domain at N-terminus. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P900_CAMJE"},"10642060":{"id":10642060,"externalId":"192222.Cj1257c","name":"Cj1257c","hasFamily":true,"annotation":"Putative efflux pump; Original (2000) note: Cj1257c, possible efflux pump, len: 394 aa; similar to TR:CAA07482 (EMBL:AJ007367) Streptococcus pneumoniae multi-drug resistance efflux pump pmrA (399 aa), fasta scores; opt: 854 z-score: 1007.9 E(): 0, 37.6% identity in 380 aa overlap. Also similar hypothetical transporters e.g. YCEE_ECOLI (408 aa), fasta scores; opt: 830 z-score: 965.7 E(): 0, 35.8% identity in 383 aa overlap, and to other members of the major facilitator family, e.g. BMR1_BACSU multidrug resistance protein 1 (389 aa), fasta scores; opt: 345 z-score: 405.2 E(): 2.8e-15, 24 [...] ","uniprotId":"Q0P8Z9_CAMJE"},"10642061":{"id":10642061,"externalId":"192222.Cj1258","name":"Cj1258","hasFamily":true,"annotation":"Belongs to the low molecular weight phosphotyrosine protein phosphatase family","uniprotId":"Q0P8Z8_CAMJE"},"10642062":{"id":10642062,"externalId":"192222.Cj1259","name":"porA","hasFamily":true,"annotation":"Assembles to form a functional porin. May be one of the structures responsible for adhesion to intestinal cells","uniprotId":"PORA_CAMJE"},"10642063":{"id":10642063,"externalId":"192222.Cj1260c","name":"dnaJ","hasFamily":true,"annotation":"Participates actively in the response to hyperosmotic and heat shock by preventing the aggregation of stress-denatured proteins and by disaggregating proteins, also in an autonomous, DnaK-independent fashion. Unfolded proteins bind initially to DnaJ; upon interaction with the DnaJ-bound protein, DnaK hydrolyzes its bound ATP, resulting in the formation of a stable complex. GrpE releases ADP from DnaK; ATP binding to DnaK triggers the release of the substrate protein, thus completing the reaction cycle. Several rounds of ATP-dependent interactions between DnaJ, DnaK and GrpE are require [...] ","uniprotId":"DNAJ_CAMJE"},"10642064":{"id":10642064,"externalId":"192222.Cj1261","name":"racR","hasFamily":true,"annotation":"Two-component regulator; Original (2000) note: Cj1261, racR, two-component regulator, len: 223 aa; identical to TR:O68795 (EMBL:AF053960) C. jejuni response regulator protein (223 aa), and similar to e.g. BAER_ECOLI transcriptional regulatory protein BAER (240 aa), fasta scores; opt: 485 z-score: 583.5 E(): 3.3e-25, 37.4% identity in 219 aa overlap. 52.0% identity to HP0166. Also simlar to Cj1227c (50.7% identity in 223 aa overlap). Contains Pfam match to entry PF00072 response_reg, Response regulator receiver domain, and Pfam match to entry PF00486 trans_reg_C,Transcriptional regulato [...] ","uniprotId":"Q0P8Z5_CAMJE"},"10642065":{"id":10642065,"externalId":"192222.Cj1262","name":"racS","hasFamily":true,"annotation":"Original (2000) note: Cj1262, racS, probable two-component sensor (histidine kinase), len: 411 aa; 99.5% identical to TR:O68796 (EMBL:AF053961) C. jejuni putative histidine protein kinase (fragment) (189 aa), and similar to e.g. ENVZ_ECOLI osmolarity sensor protein ENVZ (450 aa), fast scores; opt: 188 z-score: 217.1 E(): 8.5e-05, 24.4% identity in 242 aa overlap. N-term has 28.2% identity to HP0165, C-term has 37.9% identity to HP0164. Also similar to Cj1226c (31.3% identity in 412 aa overlap). Contains Pfam match to entry PF00512 signal,Signal carboxyl-terminal domain, and Pfam match  [...] ","uniprotId":"Q0P8Z4_CAMJE"},"10642066":{"id":10642066,"externalId":"192222.Cj1263","name":"recR","hasFamily":true,"annotation":"May play a role in DNA repair. It seems to be involved in an RecBC-independent recombinational process of DNA repair. It may act with RecF and RecO","uniprotId":"RECR_CAMJE"},"10642067":{"id":10642067,"externalId":"192222.Cj1264c","name":"hydD","hasFamily":true,"annotation":"Original (2000) note: Cj1264c, hydD, probable hydrogenase maturation protease, len: 178 aa; simlar to e.g. HYDD_WOLSU HYDD protein (159 aa), fasta scores; opt: 443 z-score: 561.5 E(): 5.6e-24, 42.5% identity in 160 aa overlap, and HYBD_ECOLI hydrogenase 2 maturation protease (164 aa), fasta scores; opt: 212 z-score: 275.6 E(): 4.7e-08, 28.1% identity in 128 aa overlap. 42.7% identity to HP0634; Updated (2006) note: Pfam domain PF01750 Hydrogenase maturation protease identified within CDS. Further support given to product function. Some characterisation work within Wolinella succinogene [...] ","uniprotId":"Q0P8Z2_CAMJE"},"10642068":{"id":10642068,"externalId":"192222.Cj1265c","name":"hydC","hasFamily":true,"annotation":"Original (2000) note: Cj1265c, hydC, probable Ni/Fe-hydrogenase B-type cytochrome subunit, len: 230 aa; similar to e.g. CYBH_WOLSU quinone-reactive Ni/Fe-hydrogenase B-type cytochrome subunit (230 aa),fasta scores; opt: 771 z-score: 968.2 E(): 0, 49.8% identity in 225 aa overlap, and CYBH_ECOLI probable Ni/Fe-hydrogenase B-type cytochrome subunit (235 aa),fasta scores; opt: 344 z-score: 437.4 E(): 4.6e-17, 25.5% identity in 231 aa overlap. 41.1% identity to HP0633. Contains PS00883 Nickel-dependent hydrogenases b-type cytochrome subunit signature 2, and fam match to entry PF01292 Ni_hy [...] ","uniprotId":"Q0P8Z1_CAMJE"},"10642069":{"id":10642069,"externalId":"192222.Cj1266c","name":"hydB","hasFamily":true,"annotation":"Belongs to the [NiFe]/[NiFeSe] hydrogenase large subunit family","uniprotId":"Q0P8Z0_CAMJE"},"10642070":{"id":10642070,"externalId":"192222.Cj1267c","name":"hydA","hasFamily":true,"annotation":"Ni/Fe-hydrogenase small chain; Original (2000) note: Cj1267c, hydA, probable Ni/Fe-hydrogenase small subunit, len: 379 aa; similar to e.g. MBHS_WOLSU quinone-reactive Ni/Fe-hydrogenase small chain precursor (354 aa), fasta scores; opt: 1968 z-score: 2288.8 E(): 0, 78.1% identity in 351 aa overlap, and MBHS_ECOLI hydrogenase-1 small chain precursor (372 aa),fasta scores; opt: 1058 z-score: 1232.4 E(): 0, 44.0% identity in 361 aa overlap. 68.2% identity to HP0631. Also similar to Cj1399c (50.8% identity in 329 aa overlap); Updated (2006) note: Pfam domain PF01058 NADH ubiquinone oxidored [...] ","uniprotId":"Q0P8Y9_CAMJE"},"10642071":{"id":10642071,"externalId":"192222.Cj1268c","name":"mnmC","hasFamily":true,"annotation":"Catalyzes the last two steps in the biosynthesis of 5- methylaminomethyl-2-thiouridine (mnm(5)s(2)U) at the wobble position (U34) in tRNA. Catalyzes the FAD-dependent demodification of cmnm(5)s(2)U34 to nm(5)s(2)U34, followed by the transfer of a methyl group from S-adenosyl-L-methionine to nm(5)s(2)U34, to form mnm(5)s(2)U34","uniprotId":"MNMC_CAMJE"},"10642072":{"id":10642072,"externalId":"192222.Cj1269c","name":"amiA","hasFamily":true,"annotation":"N-acetylmuramoyl-L-alanine amidase; Original (2000) note: Cj1269c, amiA, probable N-acetylmuramoyl-L-alanine amidase, len: 659 aa; similar in N-terminus to many e.g. AMIA_ECOLI probable N-acetylmuramoyl-L-alanine amidase AMIA precursor (EC 3.5.1.28) (289 aa), fasta scores; opt: 393 z-score: 409.5 E(): 1.6e-15, 32.2% identity in 264 aa overlap, and CWLB_BACSU N-acetylmuramoyl-L-alanine amidase CWLB precursor (496 aa), fasta scores; opt: 250 z-score: 260.4 E(): 3.3e-07, 24.9% identity in 365 aa overlap. Similar in N-term to HP0772 (51.8% identity in 253 aa overlap). C-term approx 400 aa  [...] ","uniprotId":"Q0P8Y7_CAMJE"},"10642073":{"id":10642073,"externalId":"192222.Cj1270c","name":"Cj1270c","hasFamily":true,"annotation":"Putative 2-nitropropane dioxygenase,oxidoreductase protein; Original (2000) note: Cj1270c, unknown, len: 363 aa; similar to hypothetical proteins e.g. TR:O66549 (EMBL:AE000676) Aquifex aeolicus AQ_159 (182 aa), fasta scores; opt: 550 z-score: 647.2 E(): 9.5e-29, 52.2% identity in 161 aa overlap. 61.1% identity to HP0773; Updated (2006) note: Pfam domain PF03060 2-nitropropane dioxygenase identified within CDS. Product function modified to more specific family member based on motif match. No specific characterisation has been carried out yet, so putative kept within product function. Fu [...] ","uniprotId":"Q0P8Y6_CAMJE"},"10642074":{"id":10642074,"externalId":"192222.Cj1271c","name":"tyrS","hasFamily":true,"annotation":"Catalyzes the attachment of tyrosine to tRNA(Tyr) in a two- step reaction: tyrosine is first activated by ATP to form Tyr-AMP and then transferred to the acceptor end of tRNA(Tyr)","uniprotId":"SYY_CAMJE"},"10642075":{"id":10642075,"externalId":"192222.Cj1272c","name":"spoT","hasFamily":true,"annotation":"In eubacteria ppGpp (guanosine 3'-diphosphate 5-' diphosphate) is a mediator of the stringent response that coordinates a variety of cellular activities in response to changes in nutritional abundance","uniprotId":"Q0P8Y4_CAMJE"},"10642076":{"id":10642076,"externalId":"192222.Cj1273c","name":"rpoZ","hasFamily":true,"annotation":"Promotes RNA polymerase assembly. Latches the N- and C- terminal regions of the beta' subunit thereby facilitating its interaction with the beta and alpha subunits (By similarity). ","uniprotId":"RPOZ_CAMJE"},"10642077":{"id":10642077,"externalId":"192222.Cj1274c","name":"pyrH","hasFamily":true,"annotation":"Catalyzes the reversible phosphorylation of UMP to UDP. ","uniprotId":"PYRH_CAMJE"},"10642078":{"id":10642078,"externalId":"192222.Cj1275c","name":"Cj1275c","hasFamily":true,"annotation":"Putative peptidase M23 family protein; Original (2000) note: Cj1275c, probable periplasmic protein, len: 397 aa; weak similarity to hypothetical proteins e.g. YIBP_ECOLI (419 aa), fasta scores; opt: 203 z-score: 198.7 E(): 0.0009, 21.5% identity in 396 aa overlap. 36.4% identity to HP0750. Contains probable N-terminal signal sequence and coiled-coil regions between approx. aa 30-100 and 165-230; Updated (2006) note: Pfam domain PF01551 Peptidase family M23 identified within CDS. Also, one probable transmembrane helix predicted by TMHMM2.0. Product modified to more specific family membe [...] ","uniprotId":"Q0P8Y1_CAMJE"},"10642079":{"id":10642079,"externalId":"192222.Cj1276c","name":"Cj1276c","hasFamily":true,"annotation":"Original (2000) note: Cj1276c, probable integral membrane protein, len: 268 aa; 32.1% identity to HP0749; Updated (2006) note: Four probable transmembrane helices predicted by TMHMM2.0. No specific characterisation with acceptable identity score carried out yet. Putative kept within product function. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P8Y0_CAMJE"},"10642080":{"id":10642080,"externalId":"192222.Cj1277c","name":"Cj1277c","hasFamily":true,"annotation":"Original (2000) note: Cj1277c, probable ABC transporter ATP-binding protein, len: 221 aa; similar to many e.g. YCFV_ECOLI hypothetical ABC transporter ATP-binding protein (233 aa), fast scores; opt: 464 z-score: 566.5 E(): 2.9e-24, 38.2% identity in 220 aa overlap, and FTSE_ECOLI cell division ATP-binding protein FTSE (222 aa), fasta scores; opt: 455 z-score: 556.0 E(): 1.1e-23, 38.5% identity in 213 aa overlap. 54.8% identity to HP0748. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran, ABC trans [...] ","uniprotId":"Q0P8X9_CAMJE"},"10642081":{"id":10642081,"externalId":"192222.Cj1278c","name":"trmB","hasFamily":true,"annotation":"Catalyzes the formation of N(7)-methylguanine at position 46 (m7G46) in tRNA","uniprotId":"TRMB_CAMJE"},"10642082":{"id":10642082,"externalId":"192222.Cj1279c","name":"Cj1279c","hasFamily":true,"annotation":"Original (2000) note: Cj1279c, probable fibronectin domain-containing lipoprotein, len: 411 aa; 31.1% identity to HP0746. Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Also contains 4x Pfam match to entry PF00041 fn3, Fibronectin type III domain; Updated (2006) note: Specific characterisation with acceptable identity score has not been carried out yet, so putative kept within product function. Functional classification - Pathogenicity","uniprotId":"Q0P8X7_CAMJE"},"10642083":{"id":10642083,"externalId":"192222.Cj1280c","name":"Cj1280c","hasFamily":true,"annotation":"Responsible for synthesis of pseudouridine from uracil. ","uniprotId":"Q0P8X6_CAMJE"},"10642084":{"id":10642084,"externalId":"192222.Cj1282","name":"mrdB","hasFamily":true,"annotation":"Belongs to the SEDS family. ","uniprotId":"Q0P8X5_CAMJE"},"10642085":{"id":10642085,"externalId":"192222.Cj1283","name":"ktrB","hasFamily":true,"annotation":"Putative K+ uptake protein; Original (2000) note: Cj1283, ktrB, probable K+ uptake protein, len: 447 aa; similar to e.g. TR:O87953 (EMBL:D89592) Vibrio alginolyticus K+-uptake system KTRB protein (455 aa), fasta scores; opt: 844 z-score: 981.2 E(): 0, 33.8% identity in 438 aa overlap. Also similar to NTPJ_ENTHR V-type sodium ATP synthase subunit J (451 aa),fasta scores; opt: 853 z-score: 991.7 E(): 0, 33.3% identity in 447 aa overlap, and TRKH_ECOLI TRK system potassium uptake protein TRKH (483 aa), fasta scores; opt: 210 z-score: 247.5 E(): 1.7e-06, 26.8% identity in 426 aa overlap. N [...] ","uniprotId":"Q0P8X4_CAMJE"},"10642086":{"id":10642086,"externalId":"192222.Cj1284","name":"ktrA","hasFamily":true,"annotation":"Putative K+ uptake protein; Original (2000) note: Cj1284, ktrA, probable K+ uptake protein, len: 216 aa; similar to e.g. TR:O87952 (EMBL:D89592) Vibrio alginolyticus K+-uptake system KTRA protein (220 aa), fasta scores; opt: 316 z-score: 371.1 E(): 2.3e-13, 29.2% identity in 212 aa overlap. No Hp match; Updated (2006) note: Pfam domains PF02254 TrkA-N domain and PF02080 TrkA-C domain identified within CDS. Further support given to product function. No specific characterisation with acceptable identity score yet, so putative kept within product function. Functional classification - Tran [...] ","uniprotId":"Q0P8X3_CAMJE"},"10642087":{"id":10642087,"externalId":"192222.Cj1285c","name":"mqnA","hasFamily":true,"annotation":"Catalyzes the dehydration of chorismate into 3-[(1- carboxyvinyl)oxy]benzoate, a step in the biosynthesis of menaquinone (MK, vitamin K2)","uniprotId":"Q0P8X2_CAMJE"},"10642088":{"id":10642088,"externalId":"192222.Cj1286c","name":"upp","hasFamily":true,"annotation":"Catalyzes the conversion of uracil and 5-phospho-alpha-D- ribose 1-diphosphate (PRPP) to UMP and diphosphate","uniprotId":"UPP_CAMJE"},"10642089":{"id":10642089,"externalId":"192222.Cj1287c","name":"Cj1287c","hasFamily":true,"annotation":"Malate oxidoreductase; Original (2000) note: Cj1287c, probable malate oxidoreductase, len: 411 aa; highly similar to many e.g. MAOX_BACST malate oxidoreductase (NAD) (EC 1.1.1.38) (478 aa), fasta scores; opt: 1193 z-score: 1304.5 E(): 0, 49.9% identity in 411 aa overlap. No Hp match. Contains Pfam match to entry PF00390 malic, Malic enzymes; Updated (2006) note: Characterised in more than one genus with acceptable identity scores, so putative not added to product function. Functional classification -Central intermediary metabolism - Gluconeogenesis; PMID:1371255, PMID:7798145, PMID:953 [...] ","uniprotId":"Q0P8X0_CAMJE"},"10642090":{"id":10642090,"externalId":"192222.Cj1288c","name":"gltX2","hasFamily":true,"annotation":"Catalyzes the attachment of glutamate to tRNA(Glu) in a two- step reaction: glutamate is first activated by ATP to form Glu-AMP and then transferred to the acceptor end of tRNA(Glu)","uniprotId":"SYE2_CAMJE"},"10642091":{"id":10642091,"externalId":"192222.Cj1289","name":"Cj1289","hasFamily":true,"annotation":"Original (2000) note: Cj1289, possible periplasmic protein, len: 271 aa; 25.8% identity to C-terminus of HP0659. Contains possible N-terminal signal sequence. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0P8W8_CAMJE"},"10642092":{"id":10642092,"externalId":"192222.Cj1290c","name":"accC","hasFamily":true,"annotation":"This protein is a component of the acetyl coenzyme A carboxylase complex; first, biotin carboxylase catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA. ","uniprotId":"Q0P8W7_CAMJE"},"10642093":{"id":10642093,"externalId":"192222.Cj1291c","name":"accB","hasFamily":true,"annotation":"This protein is a component of the acetyl coenzyme A carboxylase complex; first, biotin carboxylase catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA. ","uniprotId":"Q0P8W6_CAMJE"},"10642094":{"id":10642094,"externalId":"192222.Cj1292","name":"dcd","hasFamily":true,"annotation":"Catalyzes the deamination of dCTP to dUTP. ","uniprotId":"DCD_CAMJE"},"10642095":{"id":10642095,"externalId":"192222.Cj1293","name":"pseB","hasFamily":true,"annotation":"Catalyzes the first step in the biosynthesis of pseudaminic acid, a sialic-acid-like sugar that is used to modify flagellin. Has both C6 dehydratase and C5 epimerase activities that result in the production of both UDP-2-acetamido-2,6-dideoxy-beta-L-arabino-4- hexulose and UDP-2-acetamido-2,6-dideoxy-alpha-D-xylo-4-hexulose. ","uniprotId":"PSEB_CAMJE"},"10642096":{"id":10642096,"externalId":"192222.Cj1294","name":"pseC","hasFamily":true,"annotation":"Catalyzes the second step in the biosynthesis of pseudaminic acid, a sialic-acid-like sugar that is used to modify flagellin. Uses UDP-2-acetamido-2,6-dideoxy-beta-L-arabino-4-hexulose as substrate producing UDP-4-amino-4,6-dideoxy-beta-L-AltNAc. ","uniprotId":"PSEC_CAMJE"},"10642097":{"id":10642097,"externalId":"192222.Cj1295","name":"Cj1295","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1295, unknown, len: 435 aa; no Hp match; Updated (2006) note: PIR (Protein Information Resource) database has a match with IPR012353 Uncharacterised conserved protein, PIRSF01524 polysaccharide biosynthesis aminopeptidase-like protein. Members of this group from Streptomyces sp. and Brucella melitensis are encoded by polysaccharide biosynthesis gene clusters, and are believed to be involved in polysaccharide biosynthesis (PMID:12644223). Members of this group exhibit distant sequence similarity to aminopeptidases. Identifed as part of Ca [...] ","uniprotId":"Q0P8W2_CAMJE"},"10642098":{"id":10642098,"externalId":"192222.Cj1296","name":"Cj1296","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1296, unknown, len: 119 aa; weak similarity to aminoglycoside N3'-acetyltransferases e.g. AAC3_PSEAE aminoglycoside N3'-acetyltransferase III (271 aa), fasta scores; opt: 102 z-score: 144.2 E(): 1,31.7% identity in 60 aa overlap. No Hp match. Similar to N-terminus of Cj1298 (65.7% identity in 105 aa overlap). Similarity continues in downstream ORF Cj1297. There is a G(9) tract at aa 104 that, if variable, would allow translation to continue into Cj1297; Updated (2006) note: Literature search identified paper linking product function of C [...] ","uniprotId":"Q0P8W1_CAMJE"},"10642099":{"id":10642099,"externalId":"192222.Cj1297","name":"Cj1297","hasFamily":true,"annotation":"Hypotehtical protein Cj1297; Original (2000) note: Cj1297, unknown, len: 145 aa; no Hp match. Similar to C-terminus of Cj1298 (44.7% identity in 141 aa overlap). Similarity continues from upstream gene Cj1296. There is a G(9) tract upstream of this start that, if variable, would allow translation from Cj1296; Updated (2006) note: Literature search identified paper linking product function of Cj1296/Cj1297/Cj1298 to antimicrobial efflux pumps. Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Functional classification - Antibiotic resistance; PMID [...] ","uniprotId":"Q0P8W0_CAMJE"},"10642100":{"id":10642100,"externalId":"192222.Cj1298","name":"Cj1298","hasFamily":true,"annotation":"Putative N-acetyltransferase; Original (2000) note: Cj1298, unknown, len: 263 aa; weak similarity to aminoglycoside N3'-acetyltransferases e.g. AAC4_SALSP aminoglycoside N3'-acetyltransferase IV (261 aa), fasta scores; opt: 139 z-score: 177.8 E(): 0.014, 25.7% identity in 175 aa overlap. Similar in N- and C-terminus to upstream ORFs Cj1296 (65.7% identity in 105 aa overlap), and Cj1297 (44.7% identity in 141 aa overlap). No Hp match. There is a G(9) tract that, if variable, would allow fusion of Cj1296 and Cj1297. There is no equivalent repetitive tract in this CDS; Updated (2006) note [...] ","uniprotId":"Q0P8V9_CAMJE"},"10642101":{"id":10642101,"externalId":"192222.Cj1299","name":"acpP2","hasFamily":true,"annotation":"Original (2000) note: Cj1299, acpP2, probable acyl carrier protein, len: 76 aa; similar to many e.g. ACPH_MYCGE acyl carrier protein homolog (84 aa), fasta scores; opt: 111 z-score: 172.6 E(): 0.026, 35.8% identity in 81 aa overlap. No Hp match. Also similar to acpP Cj0441 (34.1% identity in 41 aa overlap); Updated (2006) note: No specific characterisation with acceptable identity score carried out yet. Putative kept within product function. Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Functional classification - Fatty acid biosynthesis; PMI [...] ","uniprotId":"Q0P8V8_CAMJE"},"10642102":{"id":10642102,"externalId":"192222.Cj1300","name":"Cj1300","hasFamily":true,"annotation":"Putative SAM domain containing methyltransferase; Original (2000) note: Cj1300, unknown, len: 297 aa; no Hp match; Updated (2006) note: Prosite domains PS50193 SAM_BIND, SAM (and some other nucleotide) binding motif and PS50124 MET_TRANS, Generic methyltransferase were both identified within CDS. Product modified to more specific family member based on motif results. Specific characterisation has not been carried out yet. Putative kept within product function. Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Functional classification - Misc; PMI [...] ","uniprotId":"Q0P8V7_CAMJE"},"10642103":{"id":10642103,"externalId":"192222.Cj1301","name":"Cj1301","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1301, unknown, len: 138 aa; similar to a hypothetical protein from Pyrococcus horikoshii TR:O58010 (EMBL:AP000001) PH0272 (136 aa),fasta scores; opt: 137 z-score: 188.8 E(): 0.0032, 32.8% identity in 137 aa overlap. No Hp match; Updated (2006) note: Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Functional classification - Conserved hypothetical proteins; PMID:16573682, PMID:12781527","uniprotId":"Q0P8V6_CAMJE"},"10642104":{"id":10642104,"externalId":"192222.Cj1302","name":"Cj1302","hasFamily":true,"annotation":"Putative HAD-superfamily phosphatase, subfamily IIIC; Original (2000) note: Cj1302, unknown, len: 520 aa; no Hp match; Updated (2006) note: TIGRFAM domain TIGR01681 HAD-SF-IIIC, HAD-superfamily phosphatase subfamily IIIC identified within CDS. Product modified to more specific family member based on motif match. No specific characterisation with acceptable identity score carried out yet, so putative kept within product function. Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Functional classification - Misc; PMID:16573682, PMID:12781527","uniprotId":"Q0P8V5_CAMJE"},"10642105":{"id":10642105,"externalId":"192222.Cj1303","name":"fabH2","hasFamily":true,"annotation":"May catalyze the condensation reaction of fatty acid synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP","uniprotId":"FABHL_CAMJE"},"10642106":{"id":10642106,"externalId":"192222.Cj1304","name":"acpP3","hasFamily":true,"annotation":"Original (2000) note: Cj1304, acpP3, probable acyl carrier protein, len: 73 aa; similar to e.g. CP_MYXXA acyl carrier protein (78 aa), fasta scores; opt: 101 z-score: 157.0 E(): 0.19, 28.4% identity in 74 aa overlap. No Hp match. Also similar to acpP2 Cj1299 (25.4% identity in 63 aa overlap); Updated (2006) note: No specific charactersiation with acceptable identity score has been carried out yet,so putative kept within product function. Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Functional classification - Fatty acid biosynthesis; PMID:16 [...] ","uniprotId":"Q0P8V3_CAMJE"},"10642107":{"id":10642107,"externalId":"192222.Cj1305c","name":"Cj1305c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1305c, unknown, len: 405 aa; no Hp match. A member of the 617 family of C.j. proteins containing homopolymeric tracts; similar to Cj0617/Cj0618 (38.9% identity in 180 aa overlap/37.1% identity in 221 aa overlap), Cj1306c (78.1% identity in 407 aa overlap), Cj1310c (63.7% identity in 402 aa overlap), Cj1342c (38.0% identity in 426 aa overlap). Contains a variable polyG-tract at aa 194; G(9) (consensus) gives this ORF, G(10) would cause truncation after 22 aa; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Ident [...] ","uniprotId":"Q0P8V2_CAMJE"},"10642108":{"id":10642108,"externalId":"192222.Cj1306c","name":"Cj1306c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1306c, unknown, len: 405 aa; no Hp match. A member of the 617 family of C.j. proteins containing homopolymeric tracts; similar to Cj1310c (59.8% identity in 410 aa overlap), Cj1305c (78.1% identity in 407 aa overlap), Cj1342c (38.5% identity in 426 aa overlap) and Cj0617/Cj0618 (39.2% identity in 181 aa overlap/39.5% identity in 220 aa overlap). Contains a variable polyG-tract at aa 194; G(9) (consensus) gives this ORF, G(8) would cause truncation after 5 aa; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Iden [...] ","uniprotId":"Q0P8V1_CAMJE"},"10642109":{"id":10642109,"externalId":"192222.Cj1307","name":"Cj1307","hasFamily":true,"annotation":"Original (2000) note: Cj1307, possible amino acid activating enzyme, len: 502 aa; similar to e.g. DLTA_BACSU D-alanine-activating enzyme (EC 6.3.2.-) (503 aa), fasta scores; opt: 726 z-score: 837.7 E(): 0, 30.3% identity in 495 aa overlap, and to parts of ENTF_ECOLI enterobactin synthetase component F (1293 aa), fasta scores; opt: 549 z-score: 627.8 E(): 1.1e-27, 27.4% identity in 460 aa overlap, and GRSB_BACBR gramicidin S synthetase II (4451 aa), fasta scores; opt: 571 z-score: 644.7 E(): 1.3e-28,29.9% identity in 508 aa overlap. No Hp match. Contains PS00455 Putative AMP-binding dom [...] ","uniprotId":"Q0P8V0_CAMJE"},"10642110":{"id":10642110,"externalId":"192222.Cj1308","name":"Cj1308","hasFamily":true,"annotation":"Original (2000) note: Cj1308, acpP4, possible acyl carrier protein, len: 75 aa; some similartiy to e.g. ACP_CYAPA acyl carrier protein (103 aa), fasta scores; opt: 84 z-score: 126.5 E(): 9.5, 31.3% identity in 64 aa overlap. No Hp match; Updated (2006) note: No specific characterisation with acceptable identity score carried out yet. Putative kept within product function. Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Functional classification - Fatty acid biosynthesis; PMID:7797557, PMID:16573682, PMID:12781527","uniprotId":"Q0P8U9_CAMJE"},"10642111":{"id":10642111,"externalId":"192222.Cj1309c","name":"Cj1309c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1309c, unknown, len 356 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Functional classification - Unknown; PMID:16573682, PMID:12781527","uniprotId":"Q0P8U8_CAMJE"},"10642112":{"id":10642112,"externalId":"192222.Cj1310c","name":"Cj1310c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1310c, unknown, len: 404 aa; no Hp match. A member of the 617 family of C.j. proteins containing homopolymeric tracts; similar to Cj1305c (63.7% identity in 402 aa overlap), Cj1306c (59.8% identity in 410 aa overlap), Cj1342c (37.1% identity in 426 aa overlap) and Cj0617/Cj0618 (37.4% identity in 182 aa overlap/34.1% identity in 220 aa overlap). Contains a variable polyG-tract at aa 194; G(9) (consensus) gives this ORF, G(10) would cause truncation after 22 aa; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Id [...] ","uniprotId":"Q0P8U7_CAMJE"},"10642113":{"id":10642113,"externalId":"192222.Cj1311","name":"pseF","hasFamily":true,"annotation":"Catalyzes the final step in the biosynthesis of pseudaminic acid, a sialic-acid-like sugar that is used to modify flagellin. Mediates the activation of pseudaminic acid with CMP by forming CMP- pseudaminic acid (By similarity)","uniprotId":"PSEF_CAMJE"},"10642114":{"id":10642114,"externalId":"192222.Cj1312","name":"pseG","hasFamily":true,"annotation":"Nucleotide sugar hydrolase that catalyzes the fourth step in the biosynthesis of pseudaminic acid, a sialic-acid-like sugar that is used to modify flagellin. Mediates the removal of UDP from C-1 of UDP- 2,4-diacetamido-2,4,6-trideoxy-beta-L-altropyranose forming 2,4- diacetamido-2,4,6-trideoxy-beta-L-altropyranose. ","uniprotId":"PSEG_CAMJE"},"10642115":{"id":10642115,"externalId":"192222.Cj1313","name":"pseH","hasFamily":true,"annotation":"Catalyzes the third step in the biosynthesis of pseudaminic acid, a sialic-acid-like sugar that is used to modify flagellin. Mediates N-4 acetylation of UDP-4-amino-4,6-dideoxy-beta-L-AltNAc to form UDP-2,4-diacetamido-2,4,6-trideoxy-beta-L-altropyranose (By similarity)","uniprotId":"PSEH_CAMJE"},"10642116":{"id":10642116,"externalId":"192222.Cj1314c","name":"hisF2","hasFamily":true,"annotation":"IGPS catalyzes the conversion of PRFAR and glutamine to IGP, AICAR and glutamate. The HisF subunit catalyzes the cyclization activity that produces IGP and AICAR from PRFAR using the ammonia provided by the HisH subunit (By similarity)","uniprotId":"HIS62_CAMJE"},"10642117":{"id":10642117,"externalId":"192222.Cj1315c","name":"hisH1","hasFamily":true,"annotation":"IGPS catalyzes the conversion of PRFAR and glutamine to IGP, AICAR and glutamate. The HisH subunit provides the glutamine amidotransferase activity that produces the ammonia necessary to HisF for the synthesis of IGP and AICAR (By similarity)","uniprotId":"HIS51_CAMJE"},"10642118":{"id":10642118,"externalId":"192222.Cj1316c","name":"pseA","hasFamily":true,"annotation":"Pseudaminic acid biosynthesis PseA protein; Original (2000) note: Cj1316c, unknown, len: 378 aa; similar to e.g. TR:P72140 (EMBL:U50396) Pseudomonas aeruginosa WBPG (lipopolysaccharide gene cluster) (376 aa), fasta scores; opt: 257 z-score: 302.0 E(): 1.6e-09,23.5% identity in 366 aa overlap. No Hp match. Also similar to Cj1324 (33.2% identity in 373 aa overlap); Updated (2006) note: Prosite domain PS00761 SPASE_I_3, Peptidase S26A, signal peptidase I identified within CDS. Product function modified to more specific family member based on motif match. PseA has been proposed to play a r [...] ","uniprotId":"Q0P8U1_CAMJE"},"10642119":{"id":10642119,"externalId":"192222.Cj1317","name":"pseI","hasFamily":true,"annotation":"Catalyzes the fifth step in the biosynthesis of pseudaminic acid, a sialic-acid-like sugar that is used to modify flagellin. Catalyzes the condensation of phosphoenolpyruvate with 2,4-diacetamido- 2,4,6-trideoxy-beta-l-altropyranose, forming pseudaminic acid. ","uniprotId":"PSEI_CAMJE"},"10642120":{"id":10642120,"externalId":"192222.Cj1318","name":"maf1","hasFamily":true,"annotation":"Original (2000) note: Cj1318, unknown, len: 649 aa; no Hp ortholog, but 33.9% identity to HP0114. A member of the 1318 family of proteins. Similar to Cj1335/Cj1336 (100.0% identity in 59 aa overlap/100.0% identity in 592 aa overlap), Cj1333 (72.5% identity in 658 aa overlap),Cj1334 (41.5% identity in 648 aa overlap), Cj1337 (48.1% identity in 643 aa overlap), Cj1341c (30.2% identity in 640 aa overlap), Cj1340c (31.4% identity in 641 aa overlap). Contains G(10-11) polymorphic tract at aa 57. G(11) (consensus) gives this ORF, G(10) causes truncation after 1 aa; Updated (2006) note: Pfam  [...] ","uniprotId":"Q0P8S3_CAMJE"},"10642121":{"id":10642121,"externalId":"192222.Cj1319","name":"Cj1319","hasFamily":true,"annotation":"Original (2000) note: Cj1319, probable nucleotide sugar dehydratase, len: 323 aa; similar to e.g. TR:Q54116 (EMBL:L37354) thymidine diphosphoglucose 4,6-dehydratase (329 aa), fasta scores; opt: 526 z-score: 600.3 E(): 3.9e-26, 29.5% identity in 315 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00106 adh_short,Alcohol/other dehydrogenases, short chain type; Updated (2006) note: Pfam domain now altered to PF01370 NAD dependent epimerase/dehydratase family. Further support given to product function. No specific characterisation with acceptable identity score carried out yet.  [...] ","uniprotId":"Q0P8T8_CAMJE"},"10642122":{"id":10642122,"externalId":"192222.Cj1320","name":"Cj1320","hasFamily":true,"annotation":"Belongs to the DegT/DnrJ/EryC1 family. ","uniprotId":"Q0P8T7_CAMJE"},"10642123":{"id":10642123,"externalId":"192222.Cj1321","name":"Cj1321","hasFamily":true,"annotation":"Putative transferase; Original (2000) note: Cj1321, probable transferase,len: 180 aa; similar to TR:Q46674 (EMBL:U05248) E. coli polysialic acid gene cluster region 2 neuD (207 aa), fasta scores; opt: 267 z-score: 337.2 E(): 1.8e-11, 29.6% identity in 162 aa overlap, some simialrity to e.g. LPXA_BRUAB acyl-[acyl-carrier-protein]--UDP-N-acetylglucosamine O-acyltransferase (283 aa), fasta scores; opt: 134 z-score: 173.4 E(): 0.025, 31.6% identity in 95 aa overlap. No Hp match. Also similar to Cj1123c wlaI (30.2% identity in 179 aa overlap). Contains 2x Pfam match to entry PF00132 hexapep [...] ","uniprotId":"Q0P8T6_CAMJE"},"10642124":{"id":10642124,"externalId":"192222.Cj1322","name":"Cj1322","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj1322, unknown, len: 106 aa; no Hp match; Updated (2006) note: Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Literature search identified paper linking Cj1321-Cj1326 as being a glycosylation islet (PMID:16230626). Functional classification - Unknown; PMID:16230626, PMID:12586395, PMID:16573682,PMID:12781527","uniprotId":"Q0P8T5_CAMJE"},"10642125":{"id":10642125,"externalId":"192222.Cj1323","name":"Cj1323","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1323, unknown, len: 128 aa; no Hp match; Updated (2006) note: Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Literature search identified paper linking Cj1321-Cj1326 as being a glycosylation islet (PMID:16230626). Functional classification - Unknown; PMID:16230626, PMID:16573682, PMID:12781527","uniprotId":"Q0P8T4_CAMJE"},"10642126":{"id":10642126,"externalId":"192222.Cj1324","name":"Cj1324","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1324, unknown, len: 373 aa; some similarity to TR:P72140 (EMBL:U50396) Pseudomonas aeruginosa WBPG (lipopolysaccharide gene cluster) (376 aa), fasta scores; opt: 166 z-score: 199.7 E(): 0.0008,20.1% identity in 378 aa overlap. No Hp match. Also similar to Cj1316c; Updated (2006) note: Identifed as part of Campylobacter jejuni O-linked glycosylation locus (Cj1293 - Cj1342). Literature search identified paper linking Cj1321-Cj1326 as being a glycosylation islet (PMID:16230626). Also shows some similarity to Cj1316 pseudaminic acid biosynth [...] ","uniprotId":"Q0P8T3_CAMJE"},"10642127":{"id":10642127,"externalId":"192222.Cj1325","name":"Cj1325","hasFamily":true,"annotation":"Putative methyltransferase; Original (2000) note: Cj1326, unknown, len: 140 aa; no Hp match. Contains polyporphic sequence G(9-10) at N-terminus; G(9) allows translation from the upstream CDS Cj1325, G(10) (consensus) would only allow this CDS to start at aa 22. Similar to Cj0171 (51.9% identity in 135 aa overlap), which also contains a polymorphic sequence allowing possible translation from Cj0170; Original (2000) note: Cj1325, unknown, len: 89 aa; no Hp match. Contains polyporphic sequence G(9-10) at C-terminus; G(10) (consensus) gives this translation stop,G(9) allows translation in [...] ","uniprotId":"Q0P8T2_CAMJE"},"10642128":{"id":10642128,"externalId":"192222.Cj1327","name":"neuB2","hasFamily":true,"annotation":"Involved in biosynthesis of legionaminic acid (5,7-diamino- 3,5,7,9-tetradeoxy-D-glycero-D-galacto-non-2-ulosonic acid)(Leg), a sialic acid-like derivative that is incorporated into flagellin via O- linkage to Ser/Thr. Catalyzes the condensation of 2,4-diacetamido- 2,4,6-trideoxymannose with phosphoenolpyruvate (PEP) to give N,N'- diacetyllegionaminic acid","uniprotId":"NEUBH_CAMJE"},"10642129":{"id":10642129,"externalId":"192222.Cj1328","name":"neuC2","hasFamily":true,"annotation":"Involved in biosynthesis of legionaminic acid (5,7-diamino- 3,5,7,9-tetradeoxy-D-glycero-D-galacto-non-2-ulosonic acid)(Leg), a sialic acid-like derivative that is incorporated into flagellin via O- linkage to Ser/Thr. Catalyzes the conversion of GDP-N,N'- diacetylbacillosamine (Bac2Ac4Ac) into 2,4-diacetamido-2,4,6- trideoxymannose and GDP. It can also use UDP-N,N'-diacetylbacillosamine however it generates small quantities of 2,4-diacetamido-2,4,6- trideoxymannose","uniprotId":"NEUCH_CAMJE"},"10642130":{"id":10642130,"externalId":"192222.Cj1329","name":"Cj1329","hasFamily":true,"annotation":"Original (2000) note: Cj1329, probable sugar-phosphate nucleotide transferase, len: 341 aa; similar in C-term (120 aa - end) to many predicted transferases, and to MPG1_YEAST mannose-1-phosphate guanyltransferase (361 aa), fasta scores; opt: 304 z-score: 343.2 E(): 8.1e-12, 28.9% identity in 225 aa overlap. N-term has some similarity to CBS domains in nucleotide dehydrogenases e.g. IMDH_BACSU inosine-5'-monophosphate dehydrogenase (513 aa), blastp scores; E=0.00034, 30% identity in 116 overlap. N-term contains 2x Pfam match to entry PF00571 CBS, CBS domain,C-term contains Pfam match to [...] ","uniprotId":"Q0P8S9_CAMJE"},"10642131":{"id":10642131,"externalId":"192222.Cj1330","name":"Cj1330","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1330, unknown, len: 301 aa; some similarity to predicted oxidoreductases e.g. TR:O26961 (EMBL:AE000863) Methanobacterium thermoautotrophicum MTH875 (predicted 3-chlorobenzoate-3,4-dioxygenase dyhydrogenase related protein) (318 aa), fasta scores; opt: 199 z-score: 244.0 E(): 2.7e-06, 24.7% identity in 267 aa overlap, and TR:Q53739 (EMBL:X92429) Streptomyces alboniger predicted oxido-reductase (338 aa), fasta scores; opt: 135 z-score: 167.8 E(): 0.048, 27.0% identity in 196 aa overlap. No Hp match; Updated (2006) note: Identifed as part o [...] ","uniprotId":"Q0P8S8_CAMJE"},"10642132":{"id":10642132,"externalId":"192222.Cj1331","name":"ptmB","hasFamily":true,"annotation":"Involved in biosynthesis of legionaminic acid (5,7-diamino- 3,5,7,9-tetradeoxy-D-glycero-D-galacto-non-2-ulosonic acid)(Leg), a sialic acid-like derivative that is incorporated into flagellin via O- linkage to Ser/Thr. Catalyzes the conversion of N,N'- diacetyllegionaminic acid (Leg5Ac7Ac) and CTP into CMP-N,N'- diacetyllegionaminic acid (CMP-Leg5Ac7Ac). ","uniprotId":"NEUAH_CAMJE"},"10642133":{"id":10642133,"externalId":"192222.Cj1332","name":"ptmA","hasFamily":true,"annotation":"Putative oxidoreductase (Flagellin modification); Original (2000) note: Cj1332, ptmA, probable oxidoreductase, len: 256 aa; 79.3% identical to PTMA_CAMCO posttranslational flagellin modification protein A (256 aa), and similar to e.g. DHG2_BACME glucose 1-dehydrogenase II (261 aa), fasta scores; opt: 236 z-score: 281.8 E(): 2.1e-08, 24.1% identity in 261 aa overlap. No Hp ortholog. Contains fam match to entry PF00106 adh_short, Alcohol/other dehydrogenases, short chain type, and Pfam match to entry PF00678 adh_short_C2,Short chain dehydrogenase/reductase C-terminus; Updated (2006) note [...] ","uniprotId":"Q0P8S6_CAMJE"},"10642134":{"id":10642134,"externalId":"192222.Cj1333","name":"pseD","hasFamily":true,"annotation":"PseD protein; Original (2000) note: Cj1333, unknown, len: 653 aa; no Hp ortholog, but 34.6% identity to HP0114. A member of the 1318 family of proteins. Similar to Cj1318 (72.5% identity in 658 aa overlap), Cj1337 (48.6% identity in 636 aa overlap), Cj1334 (41.8% identity in 644 aa overlap),Cj1340c (30.4% identity in 615 aa overlap), Cj1341c (31.2% identity in 616 aa overlap) and Cj1335/Cj1336 (94.2% identity in 52 aa overlap/71.7% identity in 597 aa overlap); Updated (2006) note: Pfam domain PF01973 Protein of unknown function DUF115 identified within CDS. Characterised work has been  [...] ","uniprotId":"Q0P8S5_CAMJE"},"10642135":{"id":10642135,"externalId":"192222.Cj1334","name":"maf3","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1334, unknown, len: 619 aa; no Hp ortholog, but 34.8% identity to HP0114. A member of the 1318 family of proteins. Similar to Cj1333 (41.9% identity in 645 aa overlap), Cj1318 (41.0% identity in 647 aa overlap), Cj1336 (41.9% identity in 590 aa overlap),Cj1337 (43.8% identity in 628 aa overlap), Cj1341c (34.3% identity in 621 aa overlap), and Cj1340c (34.0% identity in 618 aa overlap); Updated (2006) note: Pfam domain PF01973 Protein of unknown function DUF115 identified within CDS. Some characterisation work has been carried out within  [...] ","uniprotId":"Q0P8S4_CAMJE"},"10642136":{"id":10642136,"externalId":"192222.Cj1335","name":"maf4","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1335, unknown, len: 60 aa; no Hp match. A member of the 1318 family of proteins. Identical to the N-terminus of Cj1318 (100.0% identity in 59 aa overlap) and similar to Cj1333 (94.2% identity in 52 aa overlap). Contains a G(9-10) polymorphic region at aa 57. G(9) (consensus) gives this ORF, G(10) would extend the ORF by 18 aa. G(8) or G(11) (not seen in shotgun) would allow translation into the downstream ORF Cj1336,giving a protein similar to other members of the 1318 family; Original (2000) note: Cj1336, unknown, len: 593 aa; no Hp ort [...] "},"10642137":{"id":10642137,"externalId":"192222.Cj1337","name":"pseE","hasFamily":true,"annotation":"PseE protein; Original (2000) note: Cj1337, unknown, len: 628 aa; 35.0% identity to HP0114. A member of the 1318 family of proteins. Simlar to Cj1333 (48.6% identity in 636 aa overlap), Cj1318 (48.1% identity in 643 aa overlap),Cj1336 (48.6% identity in 586 aa overlap), Cj1334 (43.6% identity in 628 aa overlap), Cj1340c (32.2% identity in 624 aa overlap), Cj1341c (31.9% identity in 627 aa overlap); Updated (2006) note: Pfam domain PF01973 Protein of unknown function DUF115 identified within CDS. Some characterisation work carried out within Campylobacter jejuni. Product function modifi [...] ","uniprotId":"Q0P8S2_CAMJE"},"10642138":{"id":10642138,"externalId":"192222.Cj1338c","name":"flaB","hasFamily":true,"annotation":"Flagellin is the subunit protein which polymerizes to form the filaments of bacterial flagella","uniprotId":"FLB1_CAMJE"},"10642139":{"id":10642139,"externalId":"192222.Cj1339c","name":"flaA","hasFamily":true,"annotation":"Flagellin is the subunit protein which polymerizes to form the filaments of bacterial flagella. FlaA binds to flagellar assembly factor FliW protein, preventing FliW from binding to CsrA, so that CsrA can then bind flaA mRNA and represses its translation. ","uniprotId":"FLA1_CAMJE"},"10642140":{"id":10642140,"externalId":"192222.Cj1340c","name":"Cj1340c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1340c, unknown, len: 605 aa; no Hp ortholog, but 28.5% identity to HP0114. A member of the 1318 family of proteins. Similar to Cj1341c (83.4% identity in 598 aa overlap), Cj1337 (32.5% identity in 625 aa overlap), Cj1333 (30.4% identity in 615 aa overlap),Cj1336 (32.0% identity in 579 aa overlap), Cj1318 (31.4% identity in 641 aa overlap), and Cj1334 (34.1% identity in 618 aa overlap); Updated (2006) note: Pfam domain PF01973 Protein of unknown function DUF115 identified within CDS. Conserved added to product function. Literature search  [...] ","uniprotId":"Q0P8R9_CAMJE"},"10642141":{"id":10642141,"externalId":"192222.Cj1341c","name":"maf6","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1341c, unknown, len: 607 aa; no Hp ortholog, but 28.4% identity to HP0114. A member of the 1318 family of proteins. Similar to Cj1340c (83.4% identity in 598 aa overlap), Cj1334 (34.6% identity in 622 aa overlap), Cj1336 (30.8% identity in 578 aa overlap),Cj1333 (31.2% identity in 616 aa overlap), Cj1318 (30.2% identity in 640 aa overlap), and Cj1337 (32.0% identity in 628 aa overlap); Updated (2006) note: Pfam domain PF01973 Protein of unknown function DUF115 identified within CDS. Some characterisation work carried out within Campyloba [...] ","uniprotId":"Q0P8R8_CAMJE"},"10642142":{"id":10642142,"externalId":"192222.Cj1342c","name":"maf7","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1342c, unknown, len: 413 aa; no Hp match. A member of the 617 family of C.j. proteins containing homopolymeric tracts; similar to Cj1310c (37.3% identity in 424 aa overlap), Cj1305c (38.5% identity in 423 aa overlap), Cj0617/Cj0618 (49.7% identity in 191 aa overlap/43.4% identity in 221 aa overlap),Cj1306c (38.3% identity in 426 aa overlap). Contains C(9-10) polymorphic region at aa 188. C(9) consensus gives this ORF, C(10) gives truncation after a further 19 aa; Updated (2006) note: Some characterisation work carried out within Campylob [...] ","uniprotId":"Q0P8R7_CAMJE"},"10642143":{"id":10642143,"externalId":"192222.Cj1343c","name":"Cj1343c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1343c, probable periplasmic protein, len: 171 aa; no Hp match. Some similarity to N-terminus of secretion proteins e.g. GSPG_ECOLI putative general secretion pathway protein (145 aa), fasta scores; opt: 153 z-score: 192.8 E(): 0.0019, 35.6% identity in 59 aa overlap. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF07963 Prokaryotic N-terminal methylation motif identified within CDS. Further support given to product function. Literature search identified paper giving further clues to product function. [...] ","uniprotId":"Q0P8R6_CAMJE"},"10642144":{"id":10642144,"externalId":"192222.Cj1344c","name":"tsaD","hasFamily":true,"annotation":"Required for the formation of a threonylcarbamoyl group on adenosine at position 37 (t(6)A37) in tRNAs that read codons beginning with adenine. Is involved in the transfer of the threonylcarbamoyl moiety of threonylcarbamoyl-AMP (TC-AMP) to the N6 group of A37, together with TsaE and TsaB. TsaD likely plays a direct catalytic role in this reaction","uniprotId":"TSAD_CAMJE"},"10642145":{"id":10642145,"externalId":"192222.Cj1345c","name":"Cj1345c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1345c, probable periplasmic protein, len: 464 aa; 41.2% identity to HP1075. Contains probable N-terminal signal sequence; Updated (2006) note: Paper identified linking protein to glycoprotein. Functional classification -Miscellaneous periplasmic proteins; PMID:12186869","uniprotId":"Q0P8R4_CAMJE"},"10642146":{"id":10642146,"externalId":"192222.Cj1346c","name":"dxr","hasFamily":true,"annotation":"Catalyzes the NADP-dependent rearrangement and reduction of 1-deoxy-D-xylulose-5-phosphate (DXP) to 2-C-methyl-D-erythritol 4- phosphate (MEP)","uniprotId":"DXR_CAMJE"},"10642147":{"id":10642147,"externalId":"192222.Cj1347c","name":"cdsA","hasFamily":true,"annotation":"Belongs to the CDS family","uniprotId":"Q0P8R2_CAMJE"},"10642148":{"id":10642148,"externalId":"192222.Cj1348c","name":"Cj1348c","hasFamily":true,"annotation":"Putative coiled-coil protein; Original (2000) note: Cj1348c, probable coiled-coil protein, len: 112 aa; Lys-, Glu-rich, highly hydrophilic,contains probable coiled-coil from approx aa 50-80. No Hp match. Functional classification - Unknown","uniprotId":"Q0P8R1_CAMJE"},"10642149":{"id":10642149,"externalId":"192222.Cj1349c","name":"Cj1349c","hasFamily":true,"annotation":"Original (2000) note: Cj1349c, possible fibronectin/fibrinogen-binding protein, len: aa; similar to TR:Q54858 (EMBL:L28919) Streptococcus pyogenes fibronectin/fibrinogen-binding protein FBP54 (474 aa),fasta scores; opt: 213 z-score: 230.0 E(): 1.7e-05, 23.2% identity in 466 aa overlap. 30.6% identity to HP1392; Updated (2006) note: Pfam domain PF05670 Domain of unknown function (DUF814) identified within CDS. No specific characterisation with acceptable identity score carried out yet, so putative kept within product function. Functional classification - Pathogenicity","uniprotId":"Q0P8R0_CAMJE"},"10642150":{"id":10642150,"externalId":"192222.Cj1350","name":"mobA","hasFamily":true,"annotation":"Transfers a GMP moiety from GTP to Mo-molybdopterin (Mo-MPT) cofactor (Moco or molybdenum cofactor) to form Mo-molybdopterin guanine dinucleotide (Mo-MGD) cofactor","uniprotId":"MOBA_CAMJE"},"10642151":{"id":10642151,"externalId":"192222.Cj1351","name":"pldA","hasFamily":true,"annotation":"Hydrolysis of phosphatidylcholine with phospholipase A2 (EC 3.1.1.4) and phospholipase A1 (EC 3.1.1.32) activities. ","uniprotId":"Q0P8Q8_CAMJE"},"10642152":{"id":10642152,"externalId":"192222.Cj1352","name":"ceuB","hasFamily":true,"annotation":"Belongs to the binding-protein-dependent transport system permease family. FecCD subfamily","uniprotId":"Q0P8Q7_CAMJE"},"10642153":{"id":10642153,"externalId":"192222.Cj1353","name":"ceuC","hasFamily":true,"annotation":"Belongs to the binding-protein-dependent transport system permease family. FecCD subfamily","uniprotId":"Q0P8Q6_CAMJE"},"10642154":{"id":10642154,"externalId":"192222.Cj1354","name":"ceuD","hasFamily":true,"annotation":"Original (2000) note: Cj1354, ceuD, probable enterochelin uptake ATP-binding protein, len: 251 aa; highly similar to TR:Q46005 (EMBL:X88849) Campylobacter coli ceuD (251 aa), fasta scores; opt: 1258 z-score: 1347.9 E(): 0, 82.5% identity in 246 aa overlap, and similar to e.g. FHUC_BACSU ferrichrome transport ATP-binding protein (269 aa), fasta scores; opt: 613 z-score: 663.4 E(): 1.2e-29, 38.0% identity in 245 aa overlap. No Hp ortholog. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran, ABC trans [...] ","uniprotId":"Q0P8Q5_CAMJE"},"10642155":{"id":10642155,"externalId":"192222.Cj1355","name":"ceuE","hasFamily":true,"annotation":"Original (2000) note: Cj1355, ceuE, probable enterochelin uptake periplasmic binding protein, len: 330 aa; highly similar to TR:Q46006 (EMBL:X88849) Campylobacter coli ceuE (328 aa), fasta scores; opt: 1903 z-score: 2111.3 E(): 0, 90.6% identity in 330 aa overlap,and similar to e.g. FATB_VIBAN ferric anguibactin-binding protein precursor (322 aa), fasta scores; opt: 540 z-score: 605.4 E(): 2e-26, 30.1% identity in 279 aa overlap. No Hp match. Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site; Updated [...] ","uniprotId":"Q0P8Q4_CAMJE"},"10642156":{"id":10642156,"externalId":"192222.Cj1356c","name":"Cj1356c","hasFamily":true,"annotation":"Original (2000) note: Cj1356c, probable integral membrane protein, len: 184 aa; highly similar to TR:Q46007 (EMBL:X88849) Campylobacter coli ORF (210 aa), fasta scores; opt: 545 z-score: 652.8 E(): 4.6e-29, 67.1% identity in 210 aa overlap. 31.8% identity to HP0234; Updated (2006) note: Three probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P8Q3_CAMJE"},"10642157":{"id":10642157,"externalId":"192222.Cj1357c","name":"nrfA","hasFamily":true,"annotation":"Belongs to the cytochrome c-552 family. ","uniprotId":"Q0P8Q2_CAMJE"},"10642158":{"id":10642158,"externalId":"192222.Cj1358c","name":"nrfH","hasFamily":true,"annotation":"Cytochrome c-type protein; Original (2000) note: Cj1358c, probable periplasmic cytochrome C, len: 171 aa; similar to e.g. NAPC_ECOLI cytochrome C-type protein NAPC (200 aa), fasta scores opt: 201 z-score: 242.9 E(): 3.1e-06, 34.5% identity in 165 aa overlap. No Hp match. Contains probable N-terminal signal sequence and 3x PS00190 Cytochrome c family heme-binding site signature; Updated (2006) note: Pfam domain PF03264 NapC/NirT cytochrome c family, N-terminal was identified within CDS. Further support given to product function. Characterisation work carried out within Wolinella succino [...] ","uniprotId":"Q0P8Q1_CAMJE"},"10642159":{"id":10642159,"externalId":"192222.Cj1359","name":"ppk","hasFamily":true,"annotation":"Catalyzes the reversible transfer of the terminal phosphate of ATP to form a long-chain polyphosphate (polyP)","uniprotId":"PPK1_CAMJE"},"10642160":{"id":10642160,"externalId":"192222.Cj1361c","name":"Cj1361c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1360c, probable proteolysis tag for 10Sa_RNA ((A)ANNVKFAPAYAKAA), len: 14 aa; similar to TR:P96347 (EMBL:U68082) H. pylori coded portion of proteolysis tag (fragment) (14 aa), fasta scores; opt: 66 z-score: 192.9 E(): 0.0019, 76.9% identity in 13 aa overlap. Corresponds to c(838688..838726) in Hp seq. Functional classification - Degradation of macromolecules - Proteins, peptides and glycopeptides","uniprotId":"Q0P8P8_CAMJE"},"10642161":{"id":10642161,"externalId":"192222.Cj1362","name":"ruvB","hasFamily":true,"annotation":"The RuvA-RuvB complex in the presence of ATP renatures cruciform structure in supercoiled DNA with palindromic sequence, indicating that it may promote strand exchange reactions in homologous recombination. RuvAB is a helicase that mediates the Holliday junction migration by localized denaturation and reannealing. ","uniprotId":"RUVB_CAMJE"},"10642162":{"id":10642162,"externalId":"192222.Cj1363","name":"amaA","hasFamily":true,"annotation":"Original (2000) note: Cj1363, amaA, acid membrane antigen A, len: 347 aa; similar to many hypothetical membrane proteins e.g. TR:O66948 (EMBL:AE000705) Aquifex aeolicus AQ_740 (353 aa), fasta scores; opt: 268 z-score: 311.7 E(): 4.6e-10, 24.1% identity in 352 aa overlap. 32.2% identity to HP0567. Identical in part to TR:O69293 (EMBL:Y16882) C. jejuni predicted outer membrane protein from rearranged lambda clone (364 aa). (Phillippa Connerton(p.connerton@ic.ac.uk) pers. comm.); Updated (2006) note: Seven probable transmembrane helices predicted by TMHMM2.0. Pfam domain PF01594 Domain of [...] ","uniprotId":"Q0P8P6_CAMJE"},"10642163":{"id":10642163,"externalId":"192222.Cj1364c","name":"fumC","hasFamily":true,"annotation":"Involved in the TCA cycle. Catalyzes the stereospecific interconversion of fumarate to L-malate","uniprotId":"FUMC_CAMJE"},"10642164":{"id":10642164,"externalId":"192222.Cj1365c","name":"Cj1365c","hasFamily":true,"annotation":"Original (2000) note: Cj1365c, probable secreted serine protease, len: 1041 aa; similar to e.g. TR:Q54484 (EMBL:D78380) Serratia marcescens serine protease homologue SSP-H2 (1034 aa), fasta scores; opt: 245 z-score: 257.1 E(): 5.1e-07, 23.5% identity in 591 aa overlap, and PRTS_SERMA extracellular serine protease precursor (EC 3.4.21.-) (1045 aa), fasta scores; opt: 205 z-score: 214.6 E(): 0.00012, 23.7% identity in 615 aa overlap. No Hp match. Contains PS00137 Serine proteases,subtilase family, histidine active site, PS00138 Serine proteases, subtilase family, serine active site, Pfam [...] ","uniprotId":"Q0P8P4_CAMJE"},"10642165":{"id":10642165,"externalId":"192222.Cj1366c","name":"glmS","hasFamily":true,"annotation":"Catalyzes the first step in hexosamine metabolism, converting fructose-6P into glucosamine-6P using glutamine as a nitrogen source. ","uniprotId":"GLMS_CAMJE"},"10642166":{"id":10642166,"externalId":"192222.Cj1367c","name":"Cj1367c","hasFamily":true,"annotation":"Original (2000) note: Cj1367c, possible nucleotidyltransferase, len: 782 aa; weak similarity to TR:Q53245 (EMBL:U47030) Rhizobium tropici putative uridylyltransferase/uridylyl-removing enzyme (971 aa),fasta scores; opt: 202 z-score: 226.1 E(): 2.7e-05, 19.7% identity in 857 aa overlap, and GLND_HAEIN [protein-PII] uridylyltransferase (863 aa), fasta scores; opt: 119 z-score: 133.2 E(): 4, 19.5% identity in 822 aa overlap,(blastp score; E=4.3e-07). No Hp match. Contains C(1-2) polymorphic region at aa 148; C(2) (consensus) gives this ORF, C(1) gives truncation after +8 aa; Updated (2006 [...] ","uniprotId":"Q0P8P2_CAMJE"},"10642167":{"id":10642167,"externalId":"192222.Cj1368","name":"mqnE","hasFamily":true,"annotation":"Radical SAM enzyme that catalyzes the addition of the adenosyl radical to the double bond of 3-[(1-carboxyvinyl)oxy]benzoate, leading to aminodeoxyfutalosine (AFL), a key intermediate in the formation of menaquinone (MK, vitamin K2) from chorismate. ","uniprotId":"Q0P8P1_CAMJE"},"10642168":{"id":10642168,"externalId":"192222.Cj1369","name":"Cj1369","hasFamily":true,"annotation":"Putative permease; Original (2000) note: Cj1369, probable transmembrane transport protein, len: 439 aa; similar to many hypothetical membrane proteins e.g. YJCD_ECOLI (449 aa), fasta scores; opt: 897 z-score: 1039.1 E(): 0, 34.7% identity in 429 aa overlap. 50.7% identity to HP1175. Contains Pfam match to entry PF00860 xan_ur_permease,Xanthine/uracil permeases family; Updated (2006) note: Pfam domain PF00860 Permease family protein identified within CDS. Also, thirteen probable transmembrane helices predicted by TMHMM2.0. Product modified to more specific family member based on motif s [...] ","uniprotId":"Q0P8P0_CAMJE"},"10642169":{"id":10642169,"externalId":"192222.Cj1370","name":"Cj1370","hasFamily":true,"annotation":"Original (2000) note: Cj1370, possible nucleotide phosphoribosyltransferase, len: 147 aa; similar to e.g. TR:O50284 (EMBL:AF035967) hypoxanthine ribosyl transferase (176 aa), fasta scores; opt: 158 z-score: 205.5 E(): 0.00038, 26.2% identity in 145 aa overlap, and XGPT_ECOLI xanthine-guanine phosphoribosyltransferase (152 aa), fasta scores; opt: 128 z-score: 170.3 E(): 0.035, 24.6% identity in 142 aa overlap. 39.5% identity to HP0735. Contains PS00103 Purine/pyrimidine phosphoribosyl transferases signature; Updated (2006) note: Pfam domain PF00156 Phosphoribosyl transferase domain iden [...] ","uniprotId":"Q0P8N9_CAMJE"},"10642170":{"id":10642170,"externalId":"192222.Cj1371","name":"Cj1371","hasFamily":true,"annotation":"Original (2000) note: Cj1371, probable periplasmic protein (vacJ homolog), len: 232 aa; similar to e.g. VACJ_SHIFL VACJ lipoprotein precursor (required for intercellular spreading of Shigella flexneri) (251 aa),fasta scores; opt: 310 z-score: 377.2 E(): 1e-13, 32.5% identity in 197 aa overlap. No Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF04333 VacJ like lipoprotein identitied within CDS. Further support given to product function. Putative kept within product function. Functional classification - Pathogenicity; PMID:8145644","uniprotId":"Q0P8N8_CAMJE"},"10642171":{"id":10642171,"externalId":"192222.Cj1372","name":"Cj1372","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1372, probable periplasmic protein, len: 189 aa; similar to e.g. YRBC_ECOLI (211 aa),fasta scores; opt: 189 z-score: 222.3 E(): 4.4e-05, 22.5% identity in 200 aa overlap. No Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF05494 Toluene tolerance, Ttg2 identified within CDS. This family is a transporter which shows similarity to ABC transporters. No specific characterisation with acceptable identity score carried out yet, so putative kept within product function. Functional classification -  [...] ","uniprotId":"Q0P8N7_CAMJE"},"10642172":{"id":10642172,"externalId":"192222.Cj1373","name":"Cj1373","hasFamily":true,"annotation":"Original (2000) note: Cj1373, probable integral membrane protein, len: 823 aa; similar to hypothetical membrane proteins e.g. TR:O51268 (EMBL:AE001135) Borrelia burgdorferi conserved hypothetical integral membrane protein (767 aa), fasta scores; opt: 331 z-score: 348.2 E(): 4.2e-12, 24.8% identity in 840 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02460 Patched family protein identified within CDS. Prosite domain PS50156 SSD x2, Sterol-sensing 5TM box was also identified within CDS. Also, thirteen probable transmembrane helices predicted by TMHMM2.0. No specific charact [...] ","uniprotId":"Q0P8N6_CAMJE"},"10642173":{"id":10642173,"externalId":"192222.Cj1374c","name":"Cj1374c","hasFamily":true,"annotation":"Pyrophosphatase that catalyzes the hydrolysis of nucleoside triphosphates to their monophosphate derivatives, with a high preference for the non-canonical purine nucleotides XTP (xanthosine triphosphate), dITP (deoxyinosine triphosphate) and ITP. Seems to function as a house-cleaning enzyme that removes non-canonical purine nucleotides from the nucleotide pool, thus preventing their incorporation into DNA/RNA and avoiding chromosomal lesions. ","uniprotId":"IXTPA_CAMJE"},"10642174":{"id":10642174,"externalId":"192222.Cj1375","name":"Cj1375","hasFamily":true,"annotation":"Putative multidrug efflux transporter; Original (2000) note: Cj1375, probable efflux protein, len: 431 aa; similar to members of the major facilitator family e.g. BMR1_BACSU multidrug resistance protein 1 (multidrug-efflux transporter 1) (389 aa), fasta scores; opt: 364 z-score: 415.1 E(): 8e-16, 27.1% identity in 384 aa overlap, and NORA_STAAU quinolone resistance NORA protein (388 aa), fasta scores; opt: 353 z-score: 402.8 E(): 3.9e-15, 26.7% identity in 375 aa overlap. 41.9% identity to HP1181. Also similar to Cj0035c (24.7% identity in 372 aa overlap). Contains Pfam match to entry  [...] ","uniprotId":"Q0P8N4_CAMJE"},"10642175":{"id":10642175,"externalId":"192222.Cj1376","name":"Cj1376","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1376, probable periplasmic protein, len: 269 aa; similar to TR:O32102 (EMBL:Z99120) B. subtilis YUII protein (314 aa), fasta scores; opt: 209 z-score: 257.4 E(): 4.9e-07, 30.6% identity in 242 aa overlap, and TR:O54506 (EMBL:U97227) Salmonella enterica IROE (305 aa), fasta scores; opt: 195 z-score: 240.9 E(): 4.1e-06, 26.4% identity in 277 aa overlap. No Hp match. Contains probable N-terminal signal sequence. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0P8N3_CAMJE"},"10642176":{"id":10642176,"externalId":"192222.Cj1377c","name":"Cj1377c","hasFamily":true,"annotation":"Putative ferredoxin; Original (2000) note: Cj1377c, probable ferredoxin,len: 553 aa; limited similarity to e.g. HYCF_ECOLI formate hydrogenlyase subunit 6 (180 aa), fasta scores; opt: 212 z-score: 241.0 E(): 4e-06, 30.6% identity in 134 aa overlap. No Hp match. Contains PS00198 4Fe-4S ferredoxins,iron-sulfur binding region signature, and 2x Pfam match to entry PF00037 fer4, 4Fe-4S ferredoxins and related iron-sulfur cluster binding domains; Updated (2006) note: No specific characterisation with acceptable identity score carried out yet, so putative kept within product function. Functio [...] ","uniprotId":"Q0P8N2_CAMJE"},"10642177":{"id":10642177,"externalId":"192222.Cj1378","name":"selA","hasFamily":true,"annotation":"Converts seryl-tRNA(Sec) to selenocysteinyl-tRNA(Sec) required for selenoprotein biosynthesis","uniprotId":"SELA_CAMJE"},"10642178":{"id":10642178,"externalId":"192222.Cj1379","name":"selB","hasFamily":true,"annotation":"Original (2000) note: Cj1379, selB, probable selenocysteine-specific elongation factor, len: 601 aa; similar to e.g. SELB_ECOLI selenocysteine-specific elongation factor (614 aa), fasta scores; opt: 479 z-score: 491.6 E(): 4.4e-20, 26.2% identity in 362 aa overlap. No Hp ortholog. Contains PS00017 ATP/GTP-binding site motif A (P-loop), and Pfam match to entry PF00009 GTP_EFTU, Elongation factor Tu family; Updated (2006) note: Putative kept within product function as no characterisation with acceptable identity score has been carried out yet. Functional classification - Protein translat [...] ","uniprotId":"Q0P8N0_CAMJE"},"10642179":{"id":10642179,"externalId":"192222.Cj1380","name":"Cj1380","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1380, probable periplasmic protein, len: 236 aa; some similarity to e.g. DSBG_ECOLI thiol:disulfide interchange protein DSBG precursor (248 aa), fasta scores; opt: 138 z-score: 165.8 E(): 0.062,26.8% identity in 164 aa overlap (the active site is conserved). 34.1% identity to HP0231. Contains probable N-terminal signal sequence; Updated (2006) note: Paper identified linking protein to glycoprotein. Functional classification -Miscellaneous periplasmic proteins; PMID:12186869","uniprotId":"Q0P8M9_CAMJE"},"10642180":{"id":10642180,"externalId":"192222.Cj1381","name":"Cj1381","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj1381, probable lipoprotein,len: 176 aa; no Hp match. Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P8M8_CAMJE"},"10642181":{"id":10642181,"externalId":"192222.Cj1382c","name":"fldA","hasFamily":true,"annotation":"Low-potential electron donor to a number of redox enzymes. ","uniprotId":"Q0P8M7_CAMJE"},"10642182":{"id":10642182,"externalId":"192222.Cj1383c","name":"Cj1383c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1383c, unknown, len: 231 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Literature search identified paper giving further clues to product function. Functional classification - Unknown; PMID:15231804","uniprotId":"Q0P8M6_CAMJE"},"10642183":{"id":10642183,"externalId":"192222.Cj1384c","name":"Cj1384c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1384c, unknown, len: 104 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Literature search identified paper giving further clues to product function. Functional classification - Unknown; PMID:15231804","uniprotId":"Q0P8M5_CAMJE"},"10642184":{"id":10642184,"externalId":"192222.Cj1385","name":"katA","hasFamily":true,"annotation":"Decomposes hydrogen peroxide into water and oxygen; serves to protect cells from the toxic effects of hydrogen peroxide","uniprotId":"CATA_CAMJE"},"10642185":{"id":10642185,"externalId":"192222.Cj1386","name":"Cj1386","hasFamily":true,"annotation":"Original (2000) note: Cj1386, ankyrin-repeat containing protein, len: 412 aa; similar to proteins downstream of catalase in Pseudomonads e.g. TR:P95546 (EMBL:U16026) Pseudomonas syringae ANKF precursor (183 aa), fasta scores; opt: 217 z-score: 286.8 E(): 1.2e-08,31.3% identity in 134 aa overlap, and to eukaryotic ankyrin proteins e.g. ANK1_MOUSE ankyrin (1862 aa), fasta scores; opt: 181 z-score: 226.3 E(): 2.8e-05, 29.1% identity in 117 aa overlap. No Hp match. Also similar to Cj0834c (29.3% identity in 99 aa overlap). Contains 2x Pfam match to entry PF00023 ank, Ank repeat; Updated (2 [...] ","uniprotId":"Q0P8M3_CAMJE"},"10642186":{"id":10642186,"externalId":"192222.Cj1387c","name":"Cj1387c","hasFamily":true,"annotation":"Helix-turn-helix containing protein; Original (2000) note: Cj1387c, unknown, len: 218 aa; similar to hypothetical proteins e.g. YHEO_ECOLI (240 aa), fasta scores; opt: 164 z-score: 202.0 E(): 0.00063,29.4% identity in 204 aa overlap. No Hp match. Contains probable helix-turn-helix motif at aa 194-215 (Score 1100,+2.93 SD). Functional classification - Broad regulatory functions","uniprotId":"Q0P8M2_CAMJE"},"10642187":{"id":10642187,"externalId":"192222.Cj1388","name":"Cj1388","hasFamily":true,"annotation":"Putative endoribonuclease L-PSP; Original (2000) note: Cj1388, unknown, len: 120 aa; similar to prokaryotic hypothetical proteins e.g. YA28_PYRHO Pyrococcus horikoshii PHAL028 (137 aa), fasta scores; opt: 399 z-score: 503.4 E(): 1e-20, 53.8% identity in 117 aa overlap, and to e.g. UK14_HUMAN 14.5 KD translational inhibitor protein (137 aa), fasta scores; opt: 384 z-score: 485.1 E(): 1.1e-19, 48.3% identity in 116 aa overlap. 53.0% identity to HP0944. Also similar to Cj0327 (34.4% identity in 96 aa overlap). Contains Pfam match to entry PF01042 DUF10, Domain of unknown function; Updated [...] ","uniprotId":"Q0P8M1_CAMJE"},"10642188":{"id":10642188,"externalId":"192222.Cj1392","name":"metC'","hasFamily":false,"annotation":"Putative cystathionine beta-lyase, N-terminus; 12 probable transmembrane helices predicted for Cj1389 by TMHMM2.0 at aa 4-19, 26-48, 68-90, 110-132,137-159, 191-213, 254-276, 297-317, 337-359, 372-394,427-449 and 456-478","uniprotId":"Q0P8M0_CAMJE"},"10642189":{"id":10642189,"externalId":"192222.Cj1393","name":"Cj1393","hasFamily":true,"annotation":"Putative cystathionine beta-lyase; Original (2000) note: Cj1393, metC', probable cystathionine beta-lyase, len: 348 aa; similar to many e.g. METC_ECOLI cystathionine beta-lyase (EC 4.4.1.8) (395 aa), fasta scores; opt: 802 z-score: 979.2 E(): 0, 35.6% identity in 354 aa overlap. 36.2% identity to HP0106. Also similar to Cj1727c, 30.0% identity in 363 aa overlap. Contains PS00868 Cys/Met metabolism enzymes pyridoxal-phosphate attachment site, and Pfam match to entry PF01053 Cys_Met_Meta_PP, Gamma-family of PLP-dependent enzymes. Note that the N-terminal 50 aa of the protein appear to be [...] ","uniprotId":"Q0P8L9_CAMJE"},"10642190":{"id":10642190,"externalId":"192222.Cj1394","name":"Cj1394","hasFamily":true,"annotation":"Belongs to the lyase 1 family. Adenylosuccinate lyase subfamily","uniprotId":"Q0P8L8_CAMJE"},"10642191":{"id":10642191,"externalId":"192222.Cj1397","name":"feoA","hasFamily":true,"annotation":"Might be involved in Fe(2+) ion uptake (By similarity). ","uniprotId":"FEOA_CAMJE"},"10642192":{"id":10642192,"externalId":"192222.Cj1398","name":"feoB","hasFamily":true,"annotation":"Probable transporter of a GTP-driven Fe(2+) uptake system across the cell inner membrane into the cytoplasm, shown for strain NCTC 11168 ","uniprotId":"FEOB_CAMJE"},"10642193":{"id":10642193,"externalId":"192222.Cj1399c","name":"hydA2","hasFamily":true,"annotation":"Original (2000) note: Cj1399c, hydA2, probable Ni/Fe-hydrogenase small subunit, len: 497 aa; similar to e.g. MBHS_WOLSU quinone-reactive Ni/Fe-hydrogenase small chain (354 aa), fasta scores; opt: 1122 z-score: 1317.8 E(): 0, 52.6% identity in 285 aa overlap. No Hp ortholog. Also similar to Cj1267c (51.5% identity in 330 aa overlap). Contains C-terminal extension not present in other hydrogenase small subunits; Updated (2006) note: Pfam domain PF01058 NADH ubiquinone oxidoreductase, 20 Kd subunit identified within CDS. Further support given to product function. Characterised within Woli [...] ","uniprotId":"Q0P8L5_CAMJE"},"10642194":{"id":10642194,"externalId":"192222.Cj1400c","name":"fabI","hasFamily":true,"annotation":"Original (2000) note: Cj1400c, fabI, probable enoyl-[acyl-carrier-protein] reductase [NADH], len: 274 aa; highly similar to many e.g. FABI_ECOLI enoyl-[acyl-carrier-protein] reductase [NADH] (EC 1.3.1.9)(261 aa), fasta scores; opt: 789 z-score: 906.5 E(): 0, 47.1% identity in 257 aa overlap. 67.9% identity to HP0195. Also similar to Cj0807 (29.2% identity in 257 aa overlap), Cj0485 (28.3% identity in 254 aa overlap),Cj1332 (25.4% identity in 244 aa overlap), and Cj0435 (25.0% identity in 256 aa overlap). Contains Pfam match to entry PF00106 adh_short, Alcohol/other dehydrogenases,short [...] ","uniprotId":"Q0P8L4_CAMJE"},"10642195":{"id":10642195,"externalId":"192222.Cj1401c","name":"tpiA","hasFamily":true,"annotation":"Involved in the gluconeogenesis. Catalyzes stereospecifically the conversion of dihydroxyacetone phosphate (DHAP) to D- glyceraldehyde-3-phosphate (G3P)","uniprotId":"TPIS_CAMJE"},"10642196":{"id":10642196,"externalId":"192222.Cj1402c","name":"pgk","hasFamily":true,"annotation":"Belongs to the phosphoglycerate kinase family. ","uniprotId":"PGK_CAMJE"},"10642197":{"id":10642197,"externalId":"192222.Cj1403c","name":"gapA","hasFamily":true,"annotation":"Belongs to the glyceraldehyde-3-phosphate dehydrogenase family","uniprotId":"Q0P8L1_CAMJE"},"10642198":{"id":10642198,"externalId":"192222.Cj1404","name":"nadD","hasFamily":true,"annotation":"Catalyzes the reversible adenylation of nicotinate mononucleotide (NaMN) to nicotinic acid adenine dinucleotide (NaAD). ","uniprotId":"NADD_CAMJE"},"10642199":{"id":10642199,"externalId":"192222.Cj1405","name":"rsfS","hasFamily":true,"annotation":"Functions as a ribosomal silencing factor. Interacts with ribosomal protein L14 (rplN), blocking formation of intersubunit bridge B8. Prevents association of the 30S and 50S ribosomal subunits and the formation of functional ribosomes, thus repressing translation. ","uniprotId":"Q0P8K9_CAMJE"},"10642200":{"id":10642200,"externalId":"192222.Cj1406c","name":"Cj1406c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1406c, probable periplasmic protein, len: 116 aa; similar to YQJC_ECOLI (predicted periplasmic protein) (127 aa), fasta scores; opt: 160 z-score: 185.2 E(): 0.0054 32.4% identity in 111 aa overlap. No Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF06476 Protein of unknown function (DUF1090) identified within CDS. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0P8K8_CAMJE"},"10642201":{"id":10642201,"externalId":"192222.Cj1407c","name":"Cj1407c","hasFamily":true,"annotation":"Putative phospho-sugar mutase; Original (2000) note: Cj1407c, probable phospho-sugar mutase, len: 462 aa; simial to e.g. ALGC_PSEAE phosphomannomutase (462 aa), fasta scores; opt: 1037 z-score: 1216.2 E(): 0, 40.7% identity in 457 aa overlap, and PGMU_NEIGO phosphoglucomutase (460 aa), fasta scores; opt: 884 z-score: 1037.2 E(): 0, 35.4% identity in 435 aa overlap. 44.7% identity to HP1275. Contains Pfam match to entry PF00408 PGM_PMM, Phosphoglucomutase and phosphomannomutase, and PS00017 ATP/GTP-binding site motif A (P-loop). Also similar to Cj0360 (28.8% identity in 386 aa overlap); [...] ","uniprotId":"Q0P8K7_CAMJE"},"10642202":{"id":10642202,"externalId":"192222.Cj1408","name":"fliL","hasFamily":true,"annotation":"Controls the rotational direction of flagella during chemotaxis","uniprotId":"Q0P8K6_CAMJE"},"10642203":{"id":10642203,"externalId":"192222.Cj1409","name":"acpS","hasFamily":true,"annotation":"Transfers the 4'-phosphopantetheine moiety from coenzyme A to a Ser of acyl-carrier-protein","uniprotId":"ACPS_CAMJE"},"10642204":{"id":10642204,"externalId":"192222.Cj1410c","name":"Cj1410c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj1410c, possible membrane protein, len: 139 aa; no Hp match. Contains hydrophobic domain and PS00041 Bacterial regulatory proteins, araC family signature. Functional classification - Membranes,lipoproteins and porins","uniprotId":"Q0P8K4_CAMJE"},"10642205":{"id":10642205,"externalId":"192222.Cj1411c","name":"Cj1411c","hasFamily":true,"annotation":"Belongs to the cytochrome P450 family. ","uniprotId":"Q0P8K3_CAMJE"},"10642206":{"id":10642206,"externalId":"192222.Cj1412c","name":"Cj1412c","hasFamily":true,"annotation":"Original (2000) note: Cj1412c, probable integral membrane protein, len: 373 aa; no Hp match; Updated (2006) note: Three probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P8K2_CAMJE"},"10642207":{"id":10642207,"externalId":"192222.Cj1413c","name":"kpsS","hasFamily":true,"annotation":"Original (2000) note: Cj1413c, possible polysaccharide modification protein, len: 394 aa; similar to e.g. LIPB_NEIME capsule polysaccharide modification protein (419 aa), fasta scores; opt: 744 z-score: 897.9 E(): 0, 38.5% identity in 395 aa overlap, and KSS5_ECOLI capsule polysaccharide export protein KPSS (389 aa), fasta scores; opt: 919 z-score: 1108.7 E(): 0, 40.2% identity in 383 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF05159 Capsule polysaccharide biosynthesis protein identified within CDS. Further support given to product function. Possible removed from produc [...] ","uniprotId":"Q0P8K1_CAMJE"},"10642208":{"id":10642208,"externalId":"192222.Cj1414c","name":"kpsC","hasFamily":true,"annotation":"Original (2000) note: Cj1414c, possible polysaccharide modification protein, len: 689 aa; similar to e.g. LIPA_NEIME capsule polysaccharide modification protein (400 aa), fasta scores; opt: 927 z-score: 1062.5 E(): 0, 39.6% identity in 394 aa overlap, and KSC5_ECOLI capsule polysaccharide export protein KPSC (675 aa), fasta scores; opt: 724 z-score: 827.0 E(): 0, 38.5% identity in 655 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF05159 Capsule polysaccharide biosynthesis protein identified within CDS. Further support given to product function. Possible removed from produc [...] ","uniprotId":"Q0P8K0_CAMJE"},"10642209":{"id":10642209,"externalId":"192222.Cj1415c","name":"cysC","hasFamily":true,"annotation":"Involved in the biosynthesis of the O-methyl phosphoramidate (MeOPN) group found on the capsular polysaccharide (CPS) of C.jejuni . Catalyzes the ATP-dependent phosphorylation of cytidine diphosphoramidate (CDP-NH(2)) to form cytidine 3'-phosphate 5'-diphosphoramidate . Can also use other substrates such as the corresponding adenine and uridine diphosphoramidate derivatives or cytidine diphosphoramidate analogs, with lower efficiency ","uniprotId":"CDK_CAMJE"},"10642210":{"id":10642210,"externalId":"192222.Cj1416c","name":"Cj1416c","hasFamily":true,"annotation":"Involved in the biosynthesis of the O-methyl phosphoramidate (MeOPN) group found on the capsular polysaccharide (CPS) of C.jejuni . Catalyzes the formation of CDP-L-glutamine from CTP and L-glutamine phosphate . ","uniprotId":"CTDTR_CAMJE"},"10642211":{"id":10642211,"externalId":"192222.Cj1417c","name":"Cj1417c","hasFamily":true,"annotation":"Involved in the biosynthesis of the O-methyl phosphoramidate (MeOPN) group found on the capsular polysaccharide (CPS) of C.jejuni . Catalyzes the hydrolysis of CDP-L-glutamine to L- glutamate and cytidine diphosphoramidate . ","uniprotId":"GCDPH_CAMJE"},"10642212":{"id":10642212,"externalId":"192222.Cj1418c","name":"Cj1418c","hasFamily":true,"annotation":"Involved in the biosynthesis of the O-methyl phosphoramidate (MeOPN) group found on the capsular polysaccharide (CPS) of C.jejuni . Catalyzes the ATP-dependent phosphorylation of the amide nitrogen of L-glutamine to form L-glutamine phosphate, the first committed step in MeOPN biosynthesis . Does not phosphorylate L-glutamate or L-asparagine . ","uniprotId":"GLKIN_CAMJE"},"10642213":{"id":10642213,"externalId":"192222.Cj1419c","name":"Cj1419c","hasFamily":true,"annotation":"Putative methyltransferase; Original (2000) note: Cj1419c, possible methyltransferase, len: 253 aa; similar to hypothetical proteins e.g. TR:O53392 (EMBL:AL021841) Mycobacterium tuberculosis Rv3342 (MTV016.42) (243 aa), fasta scores; opt: 210 z-score: 250.1 E(): 1.3e-06, 32.0% identity in 122 aa overlap. Also contains weak similarity in part to several methyltransferases e.g. UBIE_ECOLI ubiquinone/menaquinone biosynthesis methyltransferase (251 aa), fasta scores; opt: 141 z-score: 171.3 E(): 0.032,24.2% identity in 132 aa overlap. No Hp match. Also similar to Cj1420c (33.7% identity in [...] ","uniprotId":"Q0P8J5_CAMJE"},"10642214":{"id":10642214,"externalId":"192222.Cj1420c","name":"Cj1420c","hasFamily":true,"annotation":"Putative methyltransferase; Original (2000) note: Cj1420c, unknown, len: 258 aa; similar to hypothetical proteins e.g. TR:O53392 (EMBL:AL021841) Mycobacterium tuberculosis Rv3342 (MTV016.42) (243 aa), fasta scores; opt: 219 z-score: 263.3 E(): 2.4e-07, 30.7% identity in 137 aa overlap. Also similar to Cj1419c (34.1% identity in 255 aa overlap). Contains C(9-10) polymorphic region at aa 132. C9 (consensus) gives this ORF, C10 would give truncation after a further 19 aa; Updated (2006) note: Prostie domain PS00213 LIPOCALIN, Lipocalin signature identified within CDS. Conserved added to p [...] ","uniprotId":"Q0P8J4_CAMJE"},"10642215":{"id":10642215,"externalId":"192222.Cj1421c","name":"Cj1421c","hasFamily":true,"annotation":"Putative sugar transferase; Original (2000) note: Cj1421c, possible sugar transferase, len: 612 aa; no Hp match. Similar to polymorphic tract-containing genes Cj1422c (overall 58.2% identity in 636 aa overlap, but N- and C-terminal 250 and 40 aa are identical), Cj0685c (28.0% identity in 404 aa overlap). N-terminus is simlar to parts of Cj1431c (38.6% identity in 114 aa overlap), and putative sugar transferases Cj1434c (39.6% identity in 111 aa overlap),and Cj1438c (34.6% identity in 127 aa overlap). Contains C(8-10) polymorphic tract at aa 30. C9 (consensus) gives this ORF, C8 gives t [...] ","uniprotId":"Q0P8J3_CAMJE"},"10642216":{"id":10642216,"externalId":"192222.Cj1422c","name":"Cj1422c","hasFamily":true,"annotation":"Putative sugar transferase; Original (2000) note: Cj1422c, possible sugar transferase, len: 625 aa; no Hp match. Similar to polymorphic tract-containing genes Cj1421c (overall 58.2% identity in 636 aa overlap, but N- and C-terminal 250 and 40 aa are identical), Cj0685c (25.6% identity in 442 aa overlap). N-terminus is simlar to parts of Cj1431c (38.6% identity in 114 aa overlap), and putative sugar transferases Cj1434c (39.6% identity in 111 aa overlap),and Cj1438c (23.5% identity in 452 aa overlap). Contains C(9-10) polymorphic tract at aa 30. C9 (consensus) gives this ORF, C10 gives  [...] ","uniprotId":"Q0P8J2_CAMJE"},"10642217":{"id":10642217,"externalId":"192222.Cj1423c","name":"hddC","hasFamily":true,"annotation":"Putative D-glycero-D-manno-heptose 1-phosphate guanosyltransferase; Original (2000) note: Cj1423c, possible sugar-phosphate nucleotidyltransferase, len: 221 aa; similar to many predicted nucleotidyltransferases, e.g. TR:O58775 (EMBL:AP000004) Pyrococcus horikoshii hypothetical sugar-phosphate nucleotydyl transferase PH1022 (416 aa), fasta scores; opt: 413 z-score: 496.0 E(): 2.6e-20, 34.3% identity in 230 aa overlap, and MPG1_YEAST mannose-1-phosphate guanyltransferase (361 aa),fasta scores; opt: 258 z-score: 313.4 E(): 3.9e-10, 28.9% identity in 232 aa overlap. Contains Pfam match to  [...] ","uniprotId":"Q0P8J1_CAMJE"},"10642218":{"id":10642218,"externalId":"192222.Cj1424c","name":"gmhA2","hasFamily":true,"annotation":"Catalyzes the isomerization of sedoheptulose 7-phosphate in D-glycero-D-manno-heptose 7-phosphate","uniprotId":"GMHA2_CAMJE"},"10642219":{"id":10642219,"externalId":"192222.Cj1425c","name":"hddA","hasFamily":true,"annotation":"Putative D-glycero-D-manno-heptose 7-phosphate kinase; Original (2000) note: Cj1425c, possible sugar kinase, len: 339 aa; similar to e.g. GAL1_LACHE galactokinase (388 aa), fasta scores; opt: 186 z-score: 227.6 E(): 2.4e-05, 24.3% identity in 371 aa overlap. Contains PS00012 Phosphopantetheine attachment site. No Hp match; Updated (2006) note: Pfam domain PF00288 GHMP kinases putative ATP-binding protein identified within CDS. Further support given to product function. Literature search identified work carried out within Campylobacter jejuni. Not yet fully characterised, so putative ke [...] ","uniprotId":"Q0P8I9_CAMJE"},"10642220":{"id":10642220,"externalId":"192222.Cj1426c","name":"Cj1426c","hasFamily":true,"annotation":"Putative methyltransferase family protein; Original (2000) note: Cj1426c, unknown, len: 283 aa; no Hp match. Contains C(10-11) polymorphic region at aa 99. C10 (consensus) gives this ORF, C11 gives truncation after +8 aa; Updated (2006) note: TIGRFAM motif TIGR01444 Methyltransferase FkbM, identified within CDS. Product modified to more specific family member based on motif match. No specific characterisation has been carried out yet, so putative kept within product function. Identifed as part of Campylobacter jejuni capsule locus (Cj1413c -Cj1448c). Functional classification - Misc; P [...] ","uniprotId":"Q0P8I8_CAMJE"},"10642221":{"id":10642221,"externalId":"192222.Cj1427c","name":"Cj1427c","hasFamily":true,"annotation":"Original (2000) note: Cj1427c, probable sugar-nucleotide epimerase/dehydratease, len: 313 aa; similar to many predicted epimerases/dehydrateases e.g. TR:Q25130 (EMBL:D25537) HALOCYNTHIA RORETZI (SEA SQUIRT) gene with similarity to UDPglucose4-epimerases and 3beta-hydroxysteroid dehydrogenase/isomerases HREPIB (241 aa), fasta scores; opt: 585 z-score: 691.2 E(): 3.5e-31,44.2% identity in 208 aa overlap, and TR:O73960 (EMBL:AP000002) Pyrococcus horikoshii hypothetical UDP-glucose 4-epimerase (318 aa), fasta scores; opt: 304 z-score: 361.7 E(): 7.9e-13, 26.4% identity in 326 aa overlap. N [...] ","uniprotId":"Q0P8I7_CAMJE"},"10642222":{"id":10642222,"externalId":"192222.Cj1428c","name":"fcl","hasFamily":true,"annotation":"Catalyzes the two-step NADP-dependent conversion of GDP-4- dehydro-6-deoxy-D-mannose to GDP-fucose, involving an epimerase and a reductase reaction","uniprotId":"Q0P8I6_CAMJE"},"10642223":{"id":10642223,"externalId":"192222.Cj1429c","name":"Cj1429c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1429c, unknown, len: 308 aa; no Hp match. Contains C(9-10) polymorphic region at aa 98. C10 (consensus) gives this ORF, C9 would give truncation after +1 aa; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Identifed as part of Campylobacter jejuni capsule locus (Cj1413c - Cj1448c). Functional classification - Unknown; PMID:15612919, PMID:10672176","uniprotId":"Q0P8I5_CAMJE"},"10642224":{"id":10642224,"externalId":"192222.Cj1430c","name":"rfbC","hasFamily":true,"annotation":"Putative dTDP-4-dehydrorhamnose 3,5-epimerase; Original (2000) note: Cj1430c, probable nucleotide-sugar epimerase/dehydratase, len: 181 aa; similar to e.g. RFBC_ECOLI DTDP-4-dehydrorhamnose 3,5-epimerase (185 aa), fasta scores; opt: 303 z-score: 386.2 E(): 3.4e-14, 33.9% identity in 165 aa overlap, and TR:Q46770 (EMBL:U23775) E. coli rmlC DTDP-6-deoxy-l-lyxo-4-hexulose reductase (183 aa), fasta scores; opt: 306 z-score: 390.0 E(): 2.1e-14, 33.5% identity in 164 aa overlap. No Hp match. Contains Pfam match to entry PF00908 dTDP_sugar_isom,dTDP-4-dehydrorhamnose 3,5-epimerase; Updated (2 [...] ","uniprotId":"Q0P8I4_CAMJE"},"10642225":{"id":10642225,"externalId":"192222.Cj1431c","name":"Cj1431c","hasFamily":true,"annotation":"Capsular polysaccharide heptosyltransferase; Original (2000) note: Cj1431c, unknown, len: 582 aa; no Hp match. Similar in part to N-termini of Cj1421c (38.6% identity in 114 aa overlap), and Cj1422c (38.6% identity in 114 aa overlap). Contains PS00455 Putative AMP-binding domain signature; Updated (2006) note: Characterised within Campylobacter jejuni. Product modified to more specific family member based on literature search results. Identifed as part of Campylobacter jejuni capsule locus (Cj1413c - Cj1448c). Functional classification - Surface polysaccharides, lipopolysaccharides and [...] ","uniprotId":"Q0P8I3_CAMJE"},"10642226":{"id":10642226,"externalId":"192222.Cj1432c","name":"Cj1432c","hasFamily":true,"annotation":"Putative sugar transferase; Original (2000) note: Cj1432c, possible sugar transferase, len: 1031 aa; weakly similar to e.g. TR:Q48232 (EMBL:X78559) Haemophilus influenzae orf3 in DNA for serotypE B capsulation locus (1215 aa), fasta scores; opt: 361 z-score: 398.4 E(): 7.2e-15, 22.8% identity in 1029 aa overlap, and TR:O84910 (EMBL:AF010183) glycosyltransferase WBPZ (381 aa), fasta scores; opt: 161 z-score: 182.7 E(): 0.0074, 23.1% identity in 251 aa overlap. Contains Pfam match to entry PF00534 Glycos_transf_1, Glycosyl transferases group 1. Also similar to Cj1434c (23.1% identity in  [...] ","uniprotId":"Q0P8I2_CAMJE"},"10642227":{"id":10642227,"externalId":"192222.Cj1433c","name":"Cj1433c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1433c, unknown, len: 368 aa; no Hp match. Contains 9 repeats of the aa sequence KIDLNNT near the N-terminus; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Identifed as part of Campylobacter jejuni capsule locus (Cj1413c - Cj1448c). Functional classification - Unknown; PMID:15612919, PMID:10672176","uniprotId":"Q0P8I1_CAMJE"},"10642228":{"id":10642228,"externalId":"192222.Cj1434c","name":"Cj1434c","hasFamily":true,"annotation":"Putative sugar transferase; Original (2000) note: Cj1434c, probable sugar transferase, len: 445 aa; similar in N-terminus to e.g. TR:O07340 (EMBL:O07340) Streptococcus pneumoniae SS-1,4-galactosyltransferase (318 aa), fasta scores; opt: 344 z-score: 405.7 E(): 2.8e-15, 27.3% identity in 326 aa overlap. No Hp ortholog. Highly similar to Cj1438c (72.4% identity in 456 aa overlap). Contains Pfam match to entry PF00535 Glycos_transf_2, Glycosyl transferases; Updated (2006) note: No specific characterisation has been carried out yet, so putative kept within product function. Literature sear [...] ","uniprotId":"Q0P8I0_CAMJE"},"10642229":{"id":10642229,"externalId":"192222.Cj1435c","name":"Cj1435c","hasFamily":true,"annotation":"Putative phosphatase; Original (2000) note: Cj1435c, unknown, len: 211 aa; similar to TR:Q58989 (EMBL:U67599) Methanococcus jannaschii hypothetical protein MJ1594 (211 aa), fasta scores; opt: 177 z-score: 228.6 E(): 2.1e-05, 23.5% identity in 196 aa overlap. No Hp match; Updated (2006) note: TIGRFAM motifs identified HAD-superfamily proteins within CDS. Literature search identified paper giving further information to product function. Product function modified to more specific family member due to motif identification and literature search. Specific characterisation still required, so  [...] ","uniprotId":"Q0P8H9_CAMJE"},"10642230":{"id":10642230,"externalId":"192222.Cj1436c","name":"Cj1436c","hasFamily":true,"annotation":"Aminotransferase; Original (2000) note: Cj1436c, probable aminotransferase, len: 390 aa; weakly similar to e.g. HIS8_BACSU histidinol-phosphate aminotransferase (360 aa),fasta scores; opt: 304 z-score: 351.9 E(): 2.8e-12, 25.5% identity in 318 aa overlap. No Hp match. Also similar to Cj0317 (hisC) (25.7% identity in 304 aa overlap), and Cj1437c (23.1% identity in 347 aa overlap). Contains PS00105 Aminotransferases class-I pyridoxal-phosphate attachment site; Updated (2006) note: Pfam domain PF00155 Aminotransferase class I and II identified within CDS. Further support given to product  [...] ","uniprotId":"Q0P8H8_CAMJE"},"10642231":{"id":10642231,"externalId":"192222.Cj1437c","name":"Cj1437c","hasFamily":true,"annotation":"Aminotransferase; Original (2000) note: Cj1437c, probable aminotransferase, len: 367 aa; similar to e.g. HIS8_BACSU histidinol-phosphate aminotransferase (360 aa), fasta scores; opt: 408 z-score: 478.1 E(): 2.6e-19, 28.6% identity in 360 aa overlap. No Hp match. Also similar to Cj0317 (hisC) (29.9% identity in 344 aa overlap), and Cj1436c (23.3% identity in 347 aa overlap). Contains Pfam match to entry PF00222 aminotran_2, Aminotransferases class-II, and PS00599 Aminotransferases class-II pyridoxal-phosphate attachment site; Updated (2006) note: Pfam domain PF00155 Aminotransferase cla [...] ","uniprotId":"Q0P8H7_CAMJE"},"10642232":{"id":10642232,"externalId":"192222.Cj1438c","name":"Cj1438c","hasFamily":true,"annotation":"Putative sugar transferase; Original (2000) note: Cj1438c, probable sugar transferase, len: 776 aa; similar in N-terminus to e.g. TR:O07340 (EMBL:O07340) Streptococcus pneumoniae SS-1,4-galactosyltransferase (318 aa), fasta scores; opt: 325 z-score: 375.2 E(): 1.4e-13, 26.9% identity in 316 aa overlap. Similar in C-terminus to TR:O31348 (EMBL:Y11138) Bacillus cereus hypothetical protein (ORF2) (300 aa),fasta scores; opt: 355 z-score: 409.9 E(): 1.7e-15, 31.2% identity in 340 aa overlap. 27.7% identity in 260 aa overlap to HP0102. Also highly similar to Cj1434c (72.9% identity in 457 aa [...] ","uniprotId":"Q0P8H6_CAMJE"},"10642233":{"id":10642233,"externalId":"192222.Cj1439c","name":"glf","hasFamily":true,"annotation":"UDP-galactopyranose mutase; Original (2000) note: Cj1439c, glf, probable UDP-galactopyranose mutase, len: aa; highly simiar to e.g. TR:O86897 (EMBL:AJ006986) Streptococcus pneumoniae polysaccharide capsule bisynthesis cluster UDP-galactopyranose mutase (EC 5.4.99.9) (369 aa), fasta scores; opt: 1546 z-score: 1775.3 E(): 0, 59.9% identity in 364 aa overlap, and GLF_ECOLI UDP-galactopyranose mutase (EC 5.4.99.9) (367 aa), fasta scores; opt: 1545 z-score: 1774.2 E(): 0, 60.8% identity in 362 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF03275 UDP-galactopyranose mutase ident [...] ","uniprotId":"Q0P8H5_CAMJE"},"10642234":{"id":10642234,"externalId":"192222.Cj1440c","name":"Cj1440c","hasFamily":true,"annotation":"Putative sugar transferase; Original (2000) note: Cj1440c, probable sugar transferase, len: 407 aa; similar in N-terminus to e.g. TR:O07340 (EMBL:O07340) Streptococcus pneumoniae SS-1,4-galactosyltransferase (318 aa), fasta scores; opt: 280 z-score: 326.2 E(): 7.5e-11, 26.7% identity in 374 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00535 Glycos_transf_2, Glycosyl transferases; Updated (2006) note: Literature search identified papers giving further information towards product function. Specific characterisation still required, so putative kept within product function. I [...] ","uniprotId":"Q0P8H4_CAMJE"},"10642235":{"id":10642235,"externalId":"192222.Cj1441c","name":"kfiD","hasFamily":true,"annotation":"UDP-glucose 6-dehydrogenase; Original (2000) note: Cj1441c, kfiD, probable UDP-glucose 6-dehydrogenase, len: 393 aa; similar to many e.g. UDG5_ECOLI UDP-glucose 6-dehydrogenase (EC 1.1.1.22) (392 aa), fasta scores; opt: 1386 z-score: 1511.4 E(): 0,52.3% identity in 392 aa overlap. No Hp match. Contains Pfam match to entry PF00984 UDPG_MGDP_dh,UDP-glucose/GDP-mannose dehydrogenase family; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Putative removed from product function. Identifed as part of Campylobacter jejuni capsule locus (Cj1413c - Cj1 [...] ","uniprotId":"Q0P8H3_CAMJE"},"10642236":{"id":10642236,"externalId":"192222.Cj1442c","name":"Cj1442c","hasFamily":true,"annotation":"Putative sugar transferase; Original (2000) note: Cj1442c, unknown, len: 544 aa; no Hp match; Updated (2006) note: Literature search identified papers giving further information about product function. Specific characterisation still required, so putative kept within product function. Identifed as part of Campylobacter jejuni capsule locus (Cj1413c - Cj1448c). Functional classification - Surface polysaccharides,lipopolysaccharides and antigens; PMID:15612919, PMID:10672176, PMID:11422559","uniprotId":"Q0P8H2_CAMJE"},"10642237":{"id":10642237,"externalId":"192222.Cj1443c","name":"kpsF","hasFamily":true,"annotation":"Belongs to the SIS family. GutQ/KpsF subfamily. ","uniprotId":"Q0P8H1_CAMJE"},"10642238":{"id":10642238,"externalId":"192222.Cj1444c","name":"kpsD","hasFamily":true,"annotation":"Original (2000) note: Cj1444c, kpsD, probable capsule polysaccharide export system periplasmic protein,len: aa; similar to e.g. KSD1_ECOLI polysialic acid transport protein KPSD PREC (558 aa), fasta scores; opt: 1187 z-score: 1306.9 E(): 0, 36.1% identity in 560 aa overlap. Contains probable N-terminal signal sequence. No Hp match; Updated (2006) note: Pfam domain PF02563 Polysaccharide biosynthesis/export protein identified within CDS. Paper identified linking protein to glycoprotein (PMID:Cj12186869). Characterised within Escherichia coli with acceptable identity score. Putative remo [...] ","uniprotId":"Q0P8H0_CAMJE"},"10642239":{"id":10642239,"externalId":"192222.Cj1445c","name":"kpsE","hasFamily":true,"annotation":"Original (2000) note: Cj1445c, kpsE, probable capsule polysaccharide export system inner membrane protein, len: 372 aa; similar to e.g. KSE1_ECOLI capsule polysaccharide export inner-membrane protein (382 aa),fasta scores; opt: 752 z-score: 755.4 E(): 0, 34.2% identity in 351 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02706 Chain length determinant protein identified within CDS. Further support given to product function. Two probable transmembrane helices predicted by TMHMM2.0. Characterised within Escherichia coli with acceptable identity score. Putative removed from  [...] ","uniprotId":"Q0P8G9_CAMJE"},"10642240":{"id":10642240,"externalId":"192222.Cj1447c","name":"kpsT","hasFamily":true,"annotation":"Original (2000) note: Cj1447c, kpsT, probable capsule polysaccharide export ATP-binding protein, len: 220 aa; identical to TR:O69286 (EMBL:AJ000856) C. jejuni ABC transporter protein (220 aa), and highly similar to e.g. KST1_ECOLI polysialic acid transport ATP-binding protein (219 aa), fasta scores; opt: 810 z-score: 945.3 E(): 0, 58.7% identity in 218 aa overlap. No Hp ortholog. Contains PS00017 ATP/GTP-binding site motif A (P-loop),PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran, ABC transporters; Updated (2006) note: Characterised within Campyloba [...] ","uniprotId":"Q0P8G8_CAMJE"},"10642241":{"id":10642241,"externalId":"192222.Cj1448c","name":"kpsM","hasFamily":true,"annotation":"Transport permease protein; Original (2000) note: Cj1448c, kpsM, probable capsule polysaccharide export system inner membrane protein, len: 260 aa; identical to TR:O69285 (EMBL:AJ000856) C. jejuni ABC transporter protein (260 aa), and highly similar to e.g. KPM1_ECOLI polysialic acid transport protein KPSM (258 aa), fasta scores; opt: 488 z-score: 558.3 E(): 8.9e-24, 34.3% identity in 254 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF01061 ABC-2 type transporter identified within CDS. Further support given to product function. Also, six probable transmembrane helices pred [...] ","uniprotId":"Q0P8G7_CAMJE"},"10642242":{"id":10642242,"externalId":"192222.Cj1449c","name":"Cj1449c","hasFamily":true,"annotation":"Belongs to the UPF0306 family","uniprotId":"Y1449_CAMJE"},"10642243":{"id":10642243,"externalId":"192222.Cj1450","name":"Cj1450","hasFamily":true,"annotation":"Original (2000) note: Cj1450, probable ATP/GTP-binding protein, len: 187 aa; no Hp match. Contains PS00017 ATP/GTP-binding site motif A (P-loop),and probable coiled-coil domain at C-terminus (aa 150-175); Updated (2006) note: Literature search identified paper giving further clues to product function. Functional classification - Unknown; PMID:16740937","uniprotId":"Q0P8G5_CAMJE"},"10642244":{"id":10642244,"externalId":"192222.Cj1451","name":"dut","hasFamily":true,"annotation":"dUTPase; Original (2000) note: Cj1451, dut, possible dUTPase, len: 229 aa; similar to two enzymes of eukaryotic origin e.g. TR:O15826 (EMBL:L76742) Leishmania major characterised dUTPase (EC 3.6.1.23) (dUTP pyrophosphatase) (268 aa), fasta scores; opt: 311 z-score: 382.3 E(): 5.7e-14, 33.6% identity in 226 aa overlap, and TR:O15923 (EMBL:U93211) Trypanosoma cruzi deoxyuridine triphosphatase (283 aa), fasta scores; opt: 198 z-score: 246.6 E(): 2.1e-06, 32.9% identity in 228 aa overlap. No Hp match. C.j. does not have a homolog of the E. coli dut gene. H.p. has an E. coli-like dut gene;  [...] ","uniprotId":"Q0P8G4_CAMJE"},"10642245":{"id":10642245,"externalId":"192222.Cj1452","name":"Cj1452","hasFamily":true,"annotation":"Original (2000) note: Cj1452, possible integral membrane protein, len: 237 aa; 29.7% identity to HP0174. Contains four large hydrophobic domains; Updated (2006) note: Pfam domain PF04401 Protein of unknown function (DUF540) identified within CDS. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P8G3_CAMJE"},"10642246":{"id":10642246,"externalId":"192222.Cj1453c","name":"tilS","hasFamily":true,"annotation":"Ligates lysine onto the cytidine present at position 34 of the AUA codon-specific tRNA(Ile) that contains the anticodon CAU, in an ATP-dependent manner. Cytidine is converted to lysidine, thus changing the amino acid specificity of the tRNA from methionine to isoleucine. ","uniprotId":"TILS_CAMJE"},"10642247":{"id":10642247,"externalId":"192222.Cj1454c","name":"rimO","hasFamily":true,"annotation":"Catalyzes the methylthiolation of an aspartic acid residue of ribosomal protein S12","uniprotId":"RIMO_CAMJE"},"10642248":{"id":10642248,"externalId":"192222.Cj1455","name":"prfB","hasFamily":true,"annotation":"Peptide chain release factor 2 directs the termination of translation in response to the peptide chain termination codons UGA and UAA","uniprotId":"RF2_CAMJE"},"10642249":{"id":10642249,"externalId":"192222.Cj1456c","name":"Cj1456c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1456c, probable periplasmic protein, len: 104 aa; no Hp match. Contains possible N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P8F9_CAMJE"},"10642250":{"id":10642250,"externalId":"192222.Cj1457c","name":"truD","hasFamily":true,"annotation":"Responsible for synthesis of pseudouridine from uracil-13 in transfer RNAs","uniprotId":"TRUD_CAMJE"},"10642251":{"id":10642251,"externalId":"192222.Cj1458c","name":"thiL","hasFamily":true,"annotation":"Catalyzes the ATP-dependent phosphorylation of thiamine- monophosphate (TMP) to form thiamine-pyrophosphate (TPP), the active form of vitamin B1","uniprotId":"Q0P8F7_CAMJE"},"10642252":{"id":10642252,"externalId":"192222.Cj1459","name":"Cj1459","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1459, unknown, len: 357 aa; 24.8% identity to HP0468. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P8F6_CAMJE"},"10642253":{"id":10642253,"externalId":"192222.Cj1460","name":"Cj1460","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1460, unknown, len: 127 aa; 41.0% identity to HP0811. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P8F5_CAMJE"},"10642254":{"id":10642254,"externalId":"192222.Cj1461","name":"Cj1461","hasFamily":true,"annotation":"Putative DNA methylase; Original (2000) note: Cj1461, possible DNA methylase, len: 227 aa; similar to hypothetical proteins e.g. YHHF_ECOLI (198 aa), fasta scores; opt: 186 z-score: 224.4 E(): 3.5e-05, 27.1% identity in 181 aa overlap. 37.2% identity to HP0810. Contains PS00092 N-6 Adenine-specific DNA methylases signature; Updated (2006) note: Pfam domain PF03602 Conserved hypothetical protein identified within CDS. Specific characterisation with acceptable identity score has not been carried out yet, so putative kept within product function. Functional classification - DNA replicatio [...] ","uniprotId":"Q0P8F4_CAMJE"},"10642255":{"id":10642255,"externalId":"192222.Cj1462","name":"flgI","hasFamily":true,"annotation":"Assembles around the rod to form the L-ring and probably protects the motor/basal body from shearing forces during rotation. ","uniprotId":"FLGI_CAMJE"},"10642256":{"id":10642256,"externalId":"192222.Cj1463","name":"flgJ","hasFamily":true,"annotation":"To H.pylori HP0245/JHP0230","uniprotId":"Y1463_CAMJE"},"10642257":{"id":10642257,"externalId":"192222.Cj1464","name":"flgM","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1464, unknown, len: 65 aa; no Hp match; Updated (2006) note: Literature search identified papers giving further clues to product function. Has been linked to flagellar anti FliA (sigma 28) factor FlgM protein. Functional classification - Unknown; PMID:14617189, PMID:14702320, PMID:14985343","uniprotId":"Q0P8F1_CAMJE"},"10642258":{"id":10642258,"externalId":"192222.Cj1465","name":"Cj1465","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1465, unknown, len: 144 aa; 31.9% identity to HP1120; Updated (2006) note: Literature search identified paper giving clues to product function. Product function linked to possible flagellar gene. Similar story for uncharacterised upstream genes Cj1463 and Cj1464. Functional classification - Conserved hypothetical proteins; PMID:14702320","uniprotId":"Q0P8F0_CAMJE"},"10642259":{"id":10642259,"externalId":"192222.Cj1466","name":"flgK","hasFamily":true,"annotation":"Belongs to the flagella basal body rod proteins family. ","uniprotId":"Q0P8E9_CAMJE"},"10642260":{"id":10642260,"externalId":"192222.Cj1467","name":"Cj1467","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1467, unknown, len: 254 aa; some similarity to TR:O51480 (EMBL:AE001154) Borrelia burgdorferi BB0530 (217 aa), fasta scores; opt: 331 z-score: 404.6 E(): 3.2e-15, 32.8% identity in 189 aa overlap. No Hp match. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P8E8_CAMJE"},"10642261":{"id":10642261,"externalId":"192222.Cj1468","name":"Cj1468","hasFamily":true,"annotation":"Original (2000) note: Cj1468, probable integral membrane protein, len: 254 aa; similar to hypothetical membrane proteins e.g. YFCA_ECOLI (269 aa), fasta scores; opt: 612 z-score: 731.5 E(): 0, 43.4% identity in 242 aa overlap. No Hp ortholog; Updated (2006) note: Pfam domain PF01925 Domain of unknown function DUF81 identified within CDS. Also, seven probable transmembrane helices predicted by TMHMM2.0. Furhter support given to product function. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P8E7_CAMJE"},"10642262":{"id":10642262,"externalId":"192222.Cj1471c","name":"ctsE","hasFamily":true,"annotation":"HMMPfam hit to PF00482, Bacterial type II secretion system protein F, score 5.9e-21","uniprotId":"Q0P8E6_CAMJE"},"10642263":{"id":10642263,"externalId":"192222.Cj1472c","name":"Cj1472c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj1472c, probable membrane protein, len: 195 aa; no Hp match; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Literature search identified paper giving further clues to product function. Functional classification - Membranes, lipoproteins and porins; PMID:12949093","uniprotId":"Q0P8E5_CAMJE"},"10642264":{"id":10642264,"externalId":"192222.Cj1473c","name":"ctsP","hasFamily":true,"annotation":"Original (2000) note: Cj1473c, probable ATP/GTP-binding protein, len: 202 aa; no Hp match. Contains PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Literature search identified paper giving further clues to product function. Functional classification - Unknown; PMID:12949093","uniprotId":"Q0P8E4_CAMJE"},"10642265":{"id":10642265,"externalId":"192222.Cj1474c","name":"ctsD","hasFamily":true,"annotation":"Belongs to the bacterial secretin family. ","uniprotId":"Q0P8E3_CAMJE"},"10642266":{"id":10642266,"externalId":"192222.Cj1475c","name":"ctsR","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1475c, unknown, len: 105 aa; no Hp match; Updated (2006) note: Literature search identified paper giving further clues to product function. Functional classification - Unknown; PMID:12949093","uniprotId":"Q0P8E2_CAMJE"},"10642267":{"id":10642267,"externalId":"192222.Cj1476c","name":"Cj1476c","hasFamily":true,"annotation":"annotation not available","uniprotId":"Q0P8E1_CAMJE"},"10642268":{"id":10642268,"externalId":"192222.Cj1477c","name":"Cj1477c","hasFamily":true,"annotation":"Putative hydrolase; Original (2000) note: Cj1477c, probable hydrolase,len: 206 aa; similar to e.g. GPH_ECOLI phosphoglycolate phosphatase (252 aa), fasta scores; opt: 209 z-score: 257.5 E(): 5.1e-07, 27.5% identity in 222 aa overlap. Contains Pfam match to entry PF00702 Hydrolase, haloacid dehalogenase-like hydrolase. Also similar to Cj1233 (27.3% identity in 205 aa overlap). No Hp match; Updated (2006) note: Similar to different types of hydrolases. Putative kept within product function. Functional classification - Misc; PMID:12576589","uniprotId":"Q0P8E0_CAMJE"},"10642269":{"id":10642269,"externalId":"192222.Cj1478c","name":"cadF","hasFamily":true,"annotation":"Original (2000) note: Cj1478c, cadF, outer membrane fibronectin-binding protein, len: 319 aa; 82.8% identical to TR:O06895 (EMBL:U87559) C. jejuni CADF precursor (fibronectin-binding protein) (326 aa), and similar to many oprF proteins e.g. PORF_PSEFL outer membrane porin F precursor (root adhesin) (326 aa), fasta scores; opt: 401 z-score: 458.6 E(): 3.2e-18, 29.1% identity in 316 aa overlap. No Hp match. Contains PS01068 OmpA-like domain,and Pfam match to entry PF00691 OmpA, OmpA family; Updated (2006) note: Characterised within Campylobacter jejuni, so putative not added to product f [...] ","uniprotId":"Q0P8D9_CAMJE"},"10642270":{"id":10642270,"externalId":"192222.Cj1479c","name":"rpsI","hasFamily":true,"annotation":"Belongs to the universal ribosomal protein uS9 family. ","uniprotId":"RS9_CAMJE"},"10642271":{"id":10642271,"externalId":"192222.Cj1480c","name":"rplM","hasFamily":true,"annotation":"This protein is one of the early assembly proteins of the 50S ribosomal subunit, although it is not seen to bind rRNA by itself. It is important during the early stages of 50S assembly. ","uniprotId":"Q0P8D7_CAMJE"},"10642272":{"id":10642272,"externalId":"192222.Cj1481c","name":"Cj1481c","hasFamily":true,"annotation":"Putative helicase; Original (2000) note: Cj1481c, possible helicase,len: 921 aa; weak similarity to several predicted helicases e.g. UVRD_MYCGE putative DNA helicase II homolog (703 aa), fasta scores; opt: 220 z-score: 237.1 E(): 7e-06, 24.5% identity in 715 aa overlap, and to e.g. ADDA_BACSU ATP-dependent nuclease subunit A (contains helicase activity) (1232 aa), fasta scores; opt: 204 z-score: 216.2 E(): 0.0001, 23.0% identity in 868 aa overlap. 30.5% identity to HP1553. Also similar to predicted helicases Cj1101 (23.9% identity in 532 aa overlap), and Cj0777 (24.1% identity in 673 a [...] ","uniprotId":"Q0P8D6_CAMJE"},"10642273":{"id":10642273,"externalId":"192222.Cj1482c","name":"Cj1482c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1482c, unknown, len: 788 aa; 29.5% identity to HP1089. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P8D5_CAMJE"},"10642274":{"id":10642274,"externalId":"192222.Cj1483c","name":"Cj1483c","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj1483c, probable lipoprotein, len: 173 aa; 30.8% identity (in N-term) to HP0150. Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P8D4_CAMJE"},"10642275":{"id":10642275,"externalId":"192222.Cj1484c","name":"Cj1484c","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj1484c, probable membrane protein, len: 200 aa; no Hp match. Contains probable N-terminal signal sequence and probable C-terminal transmembrane domain. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P8D3_CAMJE"},"10642276":{"id":10642276,"externalId":"192222.Cj1485c","name":"Cj1485c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1485c, probable periplasmic protein, len: 33 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P8D2_CAMJE"},"10642277":{"id":10642277,"externalId":"192222.Cj1486c","name":"Cj1486c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1486c, probable periplasmic protein, len: 73 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P8D1_CAMJE"},"10642278":{"id":10642278,"externalId":"192222.Cj1487c","name":"ccoP","hasFamily":true,"annotation":"C-type cytochrome. Part of the cbb3-type cytochrome c oxidase complex","uniprotId":"Q0P8D0_CAMJE"},"10642279":{"id":10642279,"externalId":"192222.Cj1488c","name":"ccoQ","hasFamily":true,"annotation":"Original (2000) note: Cj1488c, ccoQ, cb-type cytochrome C oxidase subunit IV, len: 87 aa; similar to TR:O87195 (EMBL:AB018105) H. pylori characterised cb-type cytochrome C oxidase subunit IV (EC 1.9.3.1) (72 aa),fasta scores; opt: 143 z-score: 226.7 E(): 2.6e-05, 35.1% identity in 57 aa overlap (=HP0146); Updated (2006) note: Characterised in Helicobacter pylori with acceptable identity score. Putative not added to product function. Functional classification - Energy metabolism - Electron transport; PMID:9880817","uniprotId":"Q0P8C9_CAMJE"},"10642280":{"id":10642280,"externalId":"192222.Cj1489c","name":"ccoO","hasFamily":true,"annotation":"Original (2000) note: Cj1489c, ccoO, cb-type cytochrome C oxidase subunit II, len: 221 aa; similar to TR:O87194 (EMBL:AB018105) H. pylori characterised cb-type cytochrome C oxidase subunit II (EC 1.9.3.1) (232 aa),fasta scores; opt: 1010 z-score: 1158.4 E(): 0, 65.9% identity in 220 aa overlap (=HP0145, 66.8% identity). Contains PS00190 Cytochrome c family heme-binding site signature; Updated (2006) note: PF02433 Cytochrome C oxidase,mono-heme subunit/FixO subunit identified within CDS. Further support given to product function. Characterised in Helicobacter pylori with acceptable iden [...] ","uniprotId":"Q0P8C8_CAMJE"},"10642281":{"id":10642281,"externalId":"192222.Cj1490c","name":"ccoN","hasFamily":true,"annotation":"Belongs to the heme-copper respiratory oxidase family. ","uniprotId":"Q0P8C7_CAMJE"},"10642282":{"id":10642282,"externalId":"192222.Cj1491c","name":"Cj1491c","hasFamily":true,"annotation":"Original (2000) note: Cj1491c, probable two-component regulator, len: 226 aa; similar to many e.g. VANR_ENTFA regulatory protein VANRB (220 aa), fasta scores; opt: 322 z-score: 369.3 E(): 3e-13, 29.9% identity in 214 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00072 response_reg, Response regulator receiver domain; Updated (2006) note: Pfam domain PF00486 Transcriptional regulatory protein, C terminal identified within CDS. Further support given to product function. No specific characterisation with acceptable identity score has been carried out yet, so putative kept wit [...] ","uniprotId":"Q0P8C6_CAMJE"},"10642283":{"id":10642283,"externalId":"192222.Cj1492c","name":"Cj1492c","hasFamily":true,"annotation":"Putative two-component sensor; Original (2000) note: Cj1492c, probable two-component sensor, len: 403 aa; similar to many e.g. KINC_BACSU sporulation kinase C (428 aa), fasta scores; opt: 281 z-score: 301.1 E(): 1.9e-09, 29.3% identity in 242 aa overlap. No Hp ortholog. Contains Pfam match to entry PF00512 signal, Signal carboxyl-terminal domain; Updated (2006) note: No specific characterisation with acceptable identity score has been carried out yet,so putative kept within product function. Functional classification - Signal transduction","uniprotId":"Q0P8C5_CAMJE"},"10642284":{"id":10642284,"externalId":"192222.Cj1493c","name":"Cj1493c","hasFamily":true,"annotation":"Original (2000) note: Cj1493c, probable integral membrane protein, len: 219 aa; 33.9% identity to HP0861; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P8C4_CAMJE"},"10642285":{"id":10642285,"externalId":"192222.Cj1494c","name":"carA","hasFamily":true,"annotation":"Belongs to the CarA family","uniprotId":"CARA_CAMJE"},"10642286":{"id":10642286,"externalId":"192222.Cj1495c","name":"Cj1495c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1495c, unknown, len: 183 aa; similar to hypothetical proteins e.g. TR:O67633 (EMBL:AE000754) Aquifex aeolicus AQ_1752 (183 aa), fasta scores; opt: 268 z-score: 310.8 E(): 5.4e-10, 24.6% identity in 183 aa overlap. 45.4% identity to HP1236; Updated (2006) note: Pfam domain PF04368 Protein of unknown function (DUF507) identified within CDS. Conserved added to product function. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P8C2_CAMJE"},"10642287":{"id":10642287,"externalId":"192222.Cj1496c","name":"Cj1496c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1496c, probable periplasmic protein, len: 172 aa; similar to TR:O30719 (EMBL:AF016039) H. pylori PDP protein (HP0257) (223 aa), fasta scores; opt: 232 z-score: 243.0 E(): 3.3e-06, 33.3% identity in 177 aa overlap. Contains probable N-terminal signal sequence; Updated (2006) note: Paper identified linking protein to glycoprotein. Functional classification -Miscellaneous periplasmic proteins; PMID:12186869","uniprotId":"Q0P8C1_CAMJE"},"10642288":{"id":10642288,"externalId":"192222.Cj1497c","name":"Cj1497c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1497c, unknown, len: 142 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0P8C0_CAMJE"},"10642289":{"id":10642289,"externalId":"192222.Cj1498c","name":"purA","hasFamily":true,"annotation":"Plays an important role in the de novo pathway of purine nucleotide biosynthesis. Catalyzes the first committed step in the biosynthesis of AMP from IMP","uniprotId":"PURA_CAMJE"},"10642290":{"id":10642290,"externalId":"192222.Cj1500","name":"Cj1500","hasFamily":true,"annotation":"Original (2000) note: Cj1500, probable integral membrane protein, len: 402 aa; similar to hypothetical membrane protens e.g. YEDE_ECOLI (401 aa), fasta scores; opt: 722 z-score: 819.1 E(): 0, 44.3% identity in 395 aa overlap. No Hp match; Updated (2006) note: Pfam domain x2 PF04143 YeeE/YedE family (DUF395) identified within CDS. Also, ten probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P8B8_CAMJE"},"10642291":{"id":10642291,"externalId":"192222.Cj1501","name":"Cj1501","hasFamily":true,"annotation":"Belongs to the sulfur carrier protein TusA family. ","uniprotId":"Q0P8B7_CAMJE"},"10642292":{"id":10642292,"externalId":"192222.Cj1502c","name":"putP","hasFamily":true,"annotation":"Catalyzes the sodium-dependent uptake of extracellular L- proline","uniprotId":"Q0P8B6_CAMJE"},"10642293":{"id":10642293,"externalId":"192222.Cj1503c","name":"putA","hasFamily":true,"annotation":"Oxidizes proline to glutamate for use as a carbon and nitrogen source","uniprotId":"Q0P8B5_CAMJE"},"10642294":{"id":10642294,"externalId":"192222.Cj1504c","name":"selD","hasFamily":true,"annotation":"Synthesizes selenophosphate from selenide and ATP. ","uniprotId":"SELD_CAMJE"},"10642295":{"id":10642295,"externalId":"192222.Cj1505c","name":"Cj1505c","hasFamily":true,"annotation":"Belongs to the sulfur carrier protein TusA family. ","uniprotId":"Q0P8B3_CAMJE"},"10642296":{"id":10642296,"externalId":"192222.Cj1506c","name":"Cj1506c","hasFamily":true,"annotation":"Original (2000) note: Cj1506c, probable MCP-type signal transduction protein, len: 700 aa; similar to many e.g. TR:CAB38318 (EMBL:AJ132321) Halobacterium salinarium arginine chemotaxis CAR protein (452 aa), fasta scores; opt: 382 z-score: 382.7 E(): 5.4e-14, 29.9% identity in 281 aa overlap. Contains two possible transmembrane domains and Pfam match to entry PF00015 MCPsignal,Methyl-accepting chemotaxis protein (MCP) signaling domain. 35.3% identity to HP0099; Updated (2006) note: No specific characterisation with acceptable identity score carried out yet. Putative kept in product func [...] ","uniprotId":"Q0P8B2_CAMJE"},"10642297":{"id":10642297,"externalId":"192222.Cj1507c","name":"Cj1507c","hasFamily":true,"annotation":"Putative regulatory protein; Original (2000) note: Cj1507c, unknown, len: 245 aa; no Hp match; Updated (2006) note: Pfam domain PF00126 Bacterial regulatory helix-turn-helix protein, lysR family identified within CDS. Product modified to more specific family based on motif match. No specific characterisation has been carried out yet, so putative kept within product function. Literature search identified paper giving further clues to product function. Functional classification - Broad regulatory functions; PMID:14641908","uniprotId":"Q0P8B1_CAMJE"},"10642298":{"id":10642298,"externalId":"192222.Cj1508c","name":"fdhD","hasFamily":true,"annotation":"Required for formate dehydrogenase (FDH) activity. Acts as a sulfur carrier protein that transfers sulfur from IscS to the molybdenum cofactor prior to its insertion into FDH. ","uniprotId":"Q0P8B0_CAMJE"},"10642299":{"id":10642299,"externalId":"192222.Cj1509c","name":"fdhC","hasFamily":true,"annotation":"Original (2000) note: Cj1509c, fdhC, probable formate dehydrogenase, cytochrome B subunit, len: 310 aa; similar to e.g. TR:P28180 (EMBL:X54057) Wolinella succinogenes formate dehydrogenase C precursor (306 aa),fasta scores; opt: 634 z-score: 759.3 E(): 0, 42.8% identity in 318 aa overlap, and FDNI_ECOLI formate dehydrogenase, nitrate-inducible, cytochrome B556(fdn) subunit (217 aa), fasta scores; opt: 189 z-score: 234.3 E(): 9.9e-06, 25.5% identity in 212 aa overlap. No Hp ortholog; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Some characterisation in  [...] ","uniprotId":"Q0P8A9_CAMJE"},"10642300":{"id":10642300,"externalId":"192222.Cj1510c","name":"fdhB","hasFamily":true,"annotation":"Original (2000) note: Cj1510c, fdhB, probable formate dehydrogenase iron-sulfur subunit, len: 213 aa; similar to e.g. FDHB_WOLSU formate dehydrogenase iron-sulfur subunit (EC 1.2.1.2) (200 aa), fasta scores; opt: 1025 z-score: 1153.1 E(): 0, 69.5% identity in 197 aa overlap. No Hp ortholog. Contains PS00198 4Fe-4S ferredoxins, iron-sulfur binding region signature, and Pfam match to entry PF00037 fer4, 4Fe-4S ferredoxins and related iron-sulfur cluster binding domains; Updated (2006) note: Some characterisation in Wolinella succinogenes with acceptable identity score. Putative kept with [...] ","uniprotId":"Q0P8A8_CAMJE"},"10642301":{"id":10642301,"externalId":"192222.Cj1511c","name":"fdhA","hasFamily":true,"annotation":"Belongs to the prokaryotic molybdopterin-containing oxidoreductase family","uniprotId":"Q0P8A7_CAMJE"},"10642302":{"id":10642302,"externalId":"192222.Cj1513c","name":"Cj1513c","hasFamily":true,"annotation":"Original (2000) note: Cj1513c, possible periplasmic protein, len: 63 aa; no Hp match. Contains possible N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P8A6_CAMJE"},"10642303":{"id":10642303,"externalId":"192222.Cj1514c","name":"Cj1514c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1514c, unknown, len: 237 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0P8A5_CAMJE"},"10642304":{"id":10642304,"externalId":"192222.Cj1515c","name":"Cj1515c","hasFamily":true,"annotation":"Catalyzes the decarboxylation of carboxynorspermidine and carboxyspermidine","uniprotId":"Q0P8A4_CAMJE"},"10642305":{"id":10642305,"externalId":"192222.Cj1516","name":"Cj1516","hasFamily":true,"annotation":"Original (2000) note: Cj1516, probable periplasmic oxidoreductase, len: 1539 aa; similar to members of the multicoper oxidase family e.g. YACK_ECOLI probable blue-copper protein yack precursor (516 aa), fasta scores; opt: 376 z-score: 429.0 E(): 1.4e-16, 28.7% identity in 460 aa overlap, and BLRO_MYRVE bilirubin oxidase precursor (572 aa), fasta scores; opt: 309 z-score: 352.2 E(): 2.7e-12, 24.8% identity in 499 aa overlap. No Hp match. Contains possible N-terminal signal sequence, Pfam match to entry PF00394 Cu-oxidase, Multicopper oxidase, and PS00080 Multicopper oxidases signature 2 [...] ","uniprotId":"Q0P8A3_CAMJE"},"10642306":{"id":10642306,"externalId":"192222.Cj1517","name":"moaD","hasFamily":true,"annotation":"Original (2000) note: Cj1517, moaD, possible molybdopterin converting factor, subunit 1, len: 73 aa; similar to many predicted moaD genes, e.g. TR:O31706 (EMBL:AF012285) Bacillus subtilis molybdopterin converting factor (subunit 1) (77 aa), fasta scores; opt: 111 z-score: 167.7 E(): 0.051, 30.5% identity in 59 aa overlap, and weakly similar to MOAD_ECOLI molybdopterin [mpt] converting factor, subunit 1 (81 aa), fasta scores; opt: 81 z-score: 127.5 E(): 8.8, 38.5% identity in 26 aa overlap. 61.6% identity to HP0801; Updated (2006) note: Pfam domain PF02597 ThiS family identified within  [...] ","uniprotId":"Q0P8A2_CAMJE"},"10642307":{"id":10642307,"externalId":"192222.Cj1518","name":"moaE","hasFamily":true,"annotation":"Original (2000) note: Cj1518, moaE, possible molybdopterin converting factor, subunit 2, len: 148 aa; similar to many predicted moaE genes, e.g. TR:O31705 (EMBL:AF012285) Bacillus subtilis molybdopterin converting factor (subunit 2) (157 aa), fasta scores; opt: 134 z-score: 182.4 E(): 0.0078, 25.2% identity in 131 aa overlap, and to MOAE_ECOLI molybdopterin [mpt] converting factor, subunit 2 (149 aa), fasta scores opt: 133 z-score: 181.4 E(): 0.0087, 26.0% identity in 123 aa overlap. 50.4% identity to HP0800; Updated (2006) note: Pfam domain PF02391 MoaE protein identified within CDS.  [...] ","uniprotId":"Q0P8A1_CAMJE"},"10642308":{"id":10642308,"externalId":"192222.Cj1519","name":"moeA2","hasFamily":true,"annotation":"Catalyzes the insertion of molybdate into adenylated molybdopterin with the concomitant release of AMP. ","uniprotId":"Q0P8A0_CAMJE"},"10642309":{"id":10642309,"externalId":"192222.Cj1521c","name":"cas2","hasFamily":true,"annotation":"CRISPR (clustered regularly interspaced short palindromic repeat), is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain sequences complementary to antecedent mobile elements and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA). Functions as a ssRNA-specific endoribonuclease. Involved in the integration of spacer DNA into the CRISPR cassette","uniprotId":"CAS2_CAMJE"},"10642310":{"id":10642310,"externalId":"192222.Cj1522c","name":"cas1","hasFamily":true,"annotation":"CRISPR (clustered regularly interspaced short palindromic repeat), is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain spacers, sequences complementary to antecedent mobile elements, and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA). Acts as a dsDNA endonuclease. Involved in the integration of spacer DNA into the CRISPR cassette","uniprotId":"Q0P898_CAMJE"},"10642311":{"id":10642311,"externalId":"192222.Cj1523c","name":"cas9","hasFamily":true,"annotation":"CRISPR (clustered regularly interspaced short palindromic repeat) is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain spacers, sequences complementary to antecedent mobile elements, and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA). In type II CRISPR systems correct processing of pre-crRNA requires a trans- encoded small RNA (tracrRNA), endogenous ribonuclease 3 (rnc) and this protein. The tracrRNA serves as a guide  [...] ","uniprotId":"CAS9_CAMJE"},"10642312":{"id":10642312,"externalId":"192222.Cj1529c","name":"purM","hasFamily":true,"annotation":"Phosphoribosylformylglycinamidine cyclo-ligase; 10 probable transmembrane helices predicted for Cj1528 by TMHMM2.0 at aa 4-21, 26-48, 68-90, 119-153,200-222, 248-270, 274-291, 304-326, 352-374 and 444-461","uniprotId":"Q0P896_CAMJE"},"10642313":{"id":10642313,"externalId":"192222.Cj1530","name":"coaE","hasFamily":true,"annotation":"Catalyzes the phosphorylation of the 3'-hydroxyl group of dephosphocoenzyme A to form coenzyme A","uniprotId":"COAE_CAMJE"},"10642314":{"id":10642314,"externalId":"192222.Cj1531","name":"dapF","hasFamily":true,"annotation":"Catalyzes the stereoinversion of LL-2,6-diaminoheptanedioate (L,L-DAP) to meso-diaminoheptanedioate (meso-DAP), a precursor of L- lysine and an essential component of the bacterial peptidoglycan. ","uniprotId":"DAPF_CAMJE"},"10642315":{"id":10642315,"externalId":"192222.Cj1532","name":"Cj1532","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1532, possible periplasmic protein, len: 243 aa; some similarity to N-terminus of BAX_ECOLI BAX protein (274 aa), fasta scores; opt: 240 z-score: 286.1 E(): 1.3e-08, 38.3% identity in 120 aa overlap. No Hp match. Contains possible N-terminal signal sequence; Updated (2006) note: Pfam domain PF01832 Mannosyl-glycoprotein endo-beta-N-acetylglucosaminidase (Glucosaminidase) identified within CDS. No specific characterisation carried out yet, so putative kept within product function. Functional classification -Miscellaneous periplasmic  [...] ","uniprotId":"Q0P893_CAMJE"},"10642316":{"id":10642316,"externalId":"192222.Cj1533c","name":"Cj1533c","hasFamily":true,"annotation":"Putative helix-turn-helix containing protein; Original (2000) note: Cj1533c, unknown, len: 345 aa; 27.9% identity to HP1321. Contains possible helix-turn-helix motif at aa 191-212 (Score 1065, +2.81 SD). Functional classification - Broad regulatory functions","uniprotId":"Q0P892_CAMJE"},"10642317":{"id":10642317,"externalId":"192222.Cj1534c","name":"dps","hasFamily":true,"annotation":"Protects DNA from oxidative damage by sequestering intracellular Fe(2+) ion and storing it in the form of Fe(3+) oxyhydroxide mineral. One hydrogen peroxide oxidizes two Fe(2+) ions, which prevents hydroxyl radical production by the Fenton reaction (Probable). Does not bind DNA","uniprotId":"DPS_CAMJE"},"10642318":{"id":10642318,"externalId":"192222.Cj1535c","name":"pgi","hasFamily":true,"annotation":"Original (2000) note: Cj1535c, pgi, possible glucose-6-phosphate isomerase, len: 406 aa; similar to e.g. G6PI_BACSU glucose-6-phosphate isomerase (EC 5.3.1.9) (450 aa), fats scores; opt: 469 z-score: 569.6 E(): 2.1e-24, 29.3% identity in 382 aa overlap. 26.1% identity to HP1166. Contains PS00174 Phosphoglucose isomerase signature 2, and Pfam match to entry PF00342 PGI,Phosphoglucose isomerase; Updated (2006) note: Some characterisation has been carried out with different genus, however, not matching with acceptable identity score, so putative kept within product function. Functional cl [...] ","uniprotId":"G6PI_CAMJE"},"10642319":{"id":10642319,"externalId":"192222.Cj1536c","name":"galU","hasFamily":true,"annotation":"Original (2000) note: Cj1536c, galU, probable UTP--glucose-1-phosphate uridylyltransferase, len: 274 aa; highly similar to many e.g. GALU_PSEAE UTP--glucose-1-phosphate uridylyltransferase (EC 2.7.7.9) (279 aa), fasta scores; opt: 1245 z-score: 1547.3 E(): 0,65.9% identity in 270 aa overlap. 61.3% identity to HP0646. Contains Pfam match to entry PF00483 NTP_transferase, Nucleotidyl transferase; Updated (2006) note: Characterised in Pseudomonas aeruginosa with acceptable identity score. Thus, putative was not added to product function. Functional classification - Degradation - Carbon co [...] ","uniprotId":"Q0P889_CAMJE"},"10642320":{"id":10642320,"externalId":"192222.Cj1537c","name":"acs","hasFamily":true,"annotation":"Catalyzes the conversion of acetate into acetyl-CoA (AcCoA), an essential intermediate at the junction of anabolic and catabolic pathways. AcsA undergoes a two-step reaction. In the first half reaction, AcsA combines acetate with ATP to form acetyl-adenylate (AcAMP) intermediate. In the second half reaction, it can then transfer the acetyl group from AcAMP to the sulfhydryl group of CoA, forming the product AcCoA","uniprotId":"ACSA_CAMJE"},"10642321":{"id":10642321,"externalId":"192222.Cj1538c","name":"tupC","hasFamily":true,"annotation":"Part of an ABC transporter complex involved in ultra-high affinity tungstate uptake. Probably responsible for energy coupling to the transport system","uniprotId":"TUPC_CAMJE"},"10642322":{"id":10642322,"externalId":"192222.Cj1539c","name":"tupB","hasFamily":true,"annotation":"Part of an ABC transporter complex involved in ultra-high affinity tungstate uptake. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"TUPB_CAMJE"},"10642323":{"id":10642323,"externalId":"192222.Cj1540","name":"tupA","hasFamily":true,"annotation":"Part of an ABC transporter complex involved in ultra-high affinity tungstate uptake. Specifically binds tungstate. ","uniprotId":"TUPA_CAMJE"},"10642324":{"id":10642324,"externalId":"192222.Cj1541","name":"pxpA","hasFamily":true,"annotation":"Catalyzes the cleavage of 5-oxoproline to form L-glutamate coupled to the hydrolysis of ATP to ADP and inorganic phosphate. ","uniprotId":"PXPA_CAMJE"},"10642325":{"id":10642325,"externalId":"192222.Cj1542","name":"Cj1542","hasFamily":true,"annotation":"Putative allophanate hydrolase subunit 1; Original (2000) note: Cj1542, unknown, len: 246 aa; similar to hypothetical proteins e.g. YCSJ_BACSU (578 aa),fasta scores; opt: 569 z-score: 690.1 E(): 4.1e-31, 41.8% identity in 237 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02682 Allophanate hydrolase subunit 1 identified within CDS. Product modified to more specific family member based on motif match. No specific characterisation has been carried out yet, so putative kept within product function. Also,similarity to Bacillus subtilis kinase a inhibitor (sporulation inhibitor [...] ","uniprotId":"Q0P883_CAMJE"},"10642326":{"id":10642326,"externalId":"192222.Cj1543","name":"Cj1543","hasFamily":true,"annotation":"Putative allophanate hydrolase subunit 2; Original (2000) note: Cj1543, unknown, len: 321 aa; similar to hypothetical proteins e.g. YBGK_ECOLI (310 aa),fasta scores; opt: 553 z-score: 652.4 E(): 5.1e-29, 32.7% identity in 312 aa overlap. Also similar to part of DUR1_YEAST urea amidolyase [includes: urea carboxylase; allophanate hydrolase] (1835 aa), fasta scores; opt: 307 z-score: 353.8 E(): 2.2e-12, 28.5% identity in 333 aa overlap. No Hp match; Updated (2006) note: Pfam domain PF02626 Allophanate hydrolase subunit 2 identified within CDS. Product modified to more specific family memb [...] ","uniprotId":"Q0P882_CAMJE"},"10642327":{"id":10642327,"externalId":"192222.Cj1544c","name":"Cj1544c","hasFamily":true,"annotation":"Original (2000) note: Cj1544c, probable integral membrane protein, len: 298 aa; simlar to hypothetical membrane proteins e.g. YYAM_BACSU (305 aa), fasta scores; opt: 244 z-score: 300.4 E(): 2.1e-09, 22.3% identity in 300 aa overlap. No Hp match. Contains Pfam match to entry PF00892 DUF6, Integral membrane protein; Updated (2006) note: Ten probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P881_CAMJE"},"10642328":{"id":10642328,"externalId":"192222.Cj1545c","name":"Cj1545c","hasFamily":true,"annotation":"MdaB protein homolog; Original (2000) note: Cj1545c, unknown, len: 192 aa; similar to e.g. MDAB_ECOLI modulator of drug activity B (193 aa), fasta scores; opt: 777 z-score: 934.0 E(): 0,57.8% identity in 192 aa overlap. Also weak simlarity to many boxidoreductases e.g. YCAK_ECOLI putative NAD(P)H oxidoreductase YCAK (196 aa), fasta scores; opt: 124 z-score: 159.3 E(): 0.15, 24.7% identity in 174 aa overlap. 65.1% identity to HP0630; Updated (2006) note: Pfam domain PF02525 Flavodoxin-like fold identified within CDS. Further support given to product function. Homolog designation kept wi [...] ","uniprotId":"Q0P880_CAMJE"},"10642329":{"id":10642329,"externalId":"192222.Cj1546","name":"Cj1546","hasFamily":true,"annotation":"Putative transcriptional regulator; Original (2000) note: Cj1546, unknown, len: 115 aa; similar to hypothetical proteins e.g. YYBR_BACSU (125 aa),fasta scores; opt: 278 z-score: 389.5 E(): 2.3e-14, 42.5% identity in 106 aa overlap. No Hp match. Also similar to Cj1556 (43.6% identity in 101 aa overlap); Updated (2006) note: Pfam domain PF01638 Transcriptional regulator identified within CDS. Product modified to more specific family member based on motif match. Putative kept within product function. Functional classification - Broad regulatory functions","uniprotId":"Q0P879_CAMJE"},"10642330":{"id":10642330,"externalId":"192222.Cj1547","name":"Cj1547","hasFamily":true,"annotation":"Involved in the storage or transport of lipids necessary for membrane maintenance under stressful conditions. Displays a binding preference for lysophospholipids","uniprotId":"Q0P878_CAMJE"},"10642331":{"id":10642331,"externalId":"192222.Cj1548c","name":"Cj1548c","hasFamily":true,"annotation":"Original (2000) note: Cj1548c, probable NADP-dependent alcohol dehydrogenase, len: 358 aa; similar to e.g. ADH_MYCTU NADP-dependent alcohol dehydrogenase (EC 1.1.1.2) (346 aa), fasta scores; opt: 997 z-score: 1095.7 E(): 0, 43.9% identity in 344 aa overlap. Also similar to plant lignin production alcohol dehydrogenases e.g. CAD3_ARATH cinnamyl-alcohol dehydrogenase ELI3-2 (359 aa),fasta scores; opt: 949 z-score: 1043.1 E(): 0, 40.9% identity in 352 aa overlap. 65.8% identity to HP1104. Contains PS00059 Zinc-containing alcohol dehydrogenases signature, and Pfam match to entry PF00107 ad [...] ","uniprotId":"Q0P877_CAMJE"},"10642332":{"id":10642332,"externalId":"192222.Cj1549c","name":"hsdR","hasFamily":true,"annotation":"Subunit R is required for both nuclease and ATPase activities, but not for modification","uniprotId":"Q0P876_CAMJE"},"10642333":{"id":10642333,"externalId":"192222.Cj1550c","name":"rloH","hasFamily":true,"annotation":"Original (2000) note: Cj1550c, probable ATP/GTP-binding protein, len: 588 aa; some similarity to hypothetical proteins e.g. TR:O50349 Lactobacillus helveticus plasmid plh1 hypothetical protein (495 aa),fasta scores; opt: 209 z-score: 231.6 E(): 1.4e-05, 22.6% identity in 412 aa overlap. No Hp match. Contains PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Prosite domain PS50893 ABC_TRANSPORTER_2, ATP-binding cassette, ABC transporter-type domain profile identified within CDS. Literature search identified work carried out within Helicobacter pylori and Campylobacter  [...] ","uniprotId":"Q0P875_CAMJE"},"10642334":{"id":10642334,"externalId":"192222.Cj1551c","name":"hsdS","hasFamily":true,"annotation":"Original (2000) note: Cj1551c, probable type I restriction enzyme S protein, len: 588 aa; simlar to many e.g. T1S_SALTY type I restriction enzymE StySJI specificity protein (S protein) (469 aa), fasta scores; opt: 173 z-score: 199.9 E(): 0.00082, 25.6% identity in 301 aa overlap. 24.1% identity to HP0462; Updated (2006) note: Literature search identified work carried out within Helicobacter pylori and Campylobacter jejuni. This CDS is within a putative type I locus (genes Cj1549-Cj1553). Some characterisation work within Escherichia coli, however, putative kept within product function  [...] ","uniprotId":"Q0P874_CAMJE"},"10642335":{"id":10642335,"externalId":"192222.Cj1552c","name":"mloB","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1552c, unknown, len: 488 aa; no Hp match; Updated (2006) note: Pfam domain PF04326 Divergent AAA domain identified within CDS. Conserved added to product function. Literature search identified work carried out within Helicobacter pylori and Campylobacter jejuni. This CDS is within a putative type I locus (genes Cj1549-Cj1553). Functional classification - Unknown; PMID:15699185","uniprotId":"Q0P873_CAMJE"},"10642336":{"id":10642336,"externalId":"192222.Cj1553c","name":"hsdM","hasFamily":true,"annotation":"Original (2000) note: Cj1553c, probable type I restriction enzyme M protein, len: 500 aa; simlar to many e.g. T1M1_ECOLI type I restriction enzyme EcoR124II M protein (EC 2.1.1.72) (520 aa), fasta scores; opt: 447 z-score: 527.4 E(): 4.7e-22, 25.9% identity in 521 aa overlap. 30.2% identity to HP0850. Contains PS00092 N-6 Adenine-specific DNA methylases signature; Updated (2006) note: Pfam domains PF02384 N-6 DNA Methylase and PF02506 Type I restriction modification system were identified within CDS. Further support given to product function. Literature search identified work carried o [...] ","uniprotId":"Q0P872_CAMJE"},"10642337":{"id":10642337,"externalId":"192222.Cj1555c","name":"Cj1555c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1555c, unknown, len: 211 aa; similar to TR:P71037 Bacillus subtilis YWNB (213 aa),fasta scores; opt: 519 z-score: 635.5 E(): 4.5e-28, 42.9% identity in 212 aa overlap. No Hp match. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P871_CAMJE"},"10642338":{"id":10642338,"externalId":"192222.Cj1556","name":"Cj1556","hasFamily":true,"annotation":"Putative transcriptional regulator; Original (2000) note: Cj1556, unknown, len: 110 aa; similar to hypothetical proteins e.g. YYBR_BACSU (125 aa),fasta scores; opt: 307 z-score: 402.3 E(): 4.3e-15, 43.7% identity in 103 aa overlap. No Hp match. Also similar to Cj1546 (43.6% identity in 101 aa overlap); Updated (2006) note: Pfam domain PF01638 Transcriptional regulator identified within CDS. Product modified to more specific family member based on motif match. No specific characterisation has been carried out yet, so putative kept within product function. Functional classification - Bro [...] ","uniprotId":"Q0P870_CAMJE"},"10642339":{"id":10642339,"externalId":"192222.Cj1558","name":"Cj1558","hasFamily":false,"annotation":"Putative membrane protein; Original (2000) note: Cj1558, probable membrane protein, len: 57 aa; no Hp match. May constitute the 5' end of Cj1560, which may be a pseudogene; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P869_CAMJE"},"10642340":{"id":10642340,"externalId":"192222.Cj1560","name":"Cj1560","hasFamily":true,"annotation":"Putative permease; Original (2000) note: Cj1560, probable membrane protein, len: 274 aa; simlar to hypothetical proteins e.g. Y584_METJA MJ0584 (365 aa), fasta scores; opt: 213 z-score: 246.7 E(): 2.1e-06, 23.5% identity in 294 aa overlap. No Hp match. Note that the similarities continue upstream of the first possible start site (at aa 62), and that this gene may therfore be a pseudogene, with Cj1558 representing the 5' end; Updated (2006) note: Pfam domain PF03773 Predicted permease identified within CDS. Product modified to more specific family member based on motif match. No specifi [...] ","uniprotId":"Q0P868_CAMJE"},"10642341":{"id":10642341,"externalId":"192222.Cj1561","name":"Cj1561","hasFamily":true,"annotation":"Original (2000) note: Cj1561, probable transcriptional regulator, len: 59 aa; similar to N-terminus of e.g. ARR1_ECOLI arsenical resistance operon repressor (117 aa), fasta scores; opt: 142 z-score: 236.5 E(): 7.4e-06, 38.2% identity in 55 aa overlap, and CADC_LISMO cadmium efflux system accessory protein (119 aa), fasta scores; opt: 134 z-score: 224.1 E(): 3.7e-05,42.0% identity in 50 aa overlap. No Hp match. Contains helix-turn-helix motif at aa 30-51 (Score 1478, +4.22 SD); Updated (2006) note: Similar to more than one characterised bacteria. Putative designation kept within product [...] ","uniprotId":"Q0P867_CAMJE"},"10642342":{"id":10642342,"externalId":"192222.Cj1562","name":"Cj1562","hasFamily":false,"annotation":"Uncharacterized protein; Original (2000) note: Cj1562, unknown, len: 50 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0P866_CAMJE"},"10642343":{"id":10642343,"externalId":"192222.Cj1563c","name":"Cj1563c","hasFamily":true,"annotation":"Original (2000) note: Cj1563c, probable transcriptional regulator, len: 143 aa; similar to members of the MerR family e.g. Y186_HAEIN hypothetical transcriptional regulator HI0186 (135 aa), fasta scores; opt: 241 z-score: 290.6 E(): 7.3e-09, 31.9% identity in 119 aa overlap, and BLTR_BACSU multidrug-efflux transporter 2 regulator (273 aa), fasta scores; opt: 149 z-score: 182.1 E(): 0.008, 24.4% identity in 127 aa overlap. No Hp match. Contains helix-turn-helix motif at aa 4-25 (Score 1172, +3.18 SD); Updated (2006) note: Pfam domain PF00376 MerR family regulatory protein identified wit [...] ","uniprotId":"Q0P865_CAMJE"},"10642344":{"id":10642344,"externalId":"192222.Cj1564","name":"Cj1564","hasFamily":true,"annotation":"Original (2000) note: Cj1564, probable methyl-accepting chemotaxis signal transduction protein,len: 662 aa; similar to many e.g. TLPC_BACSU methyl-accepting chemotaxis protein TLPC (573 aa), fasta scores; opt: 351 z-score: 363.6 E(): 6.2e-13, 26.2% identity in 553 aa overlap. No Hp ortholog. C-terminus contains repeat1, and is identical to the other repeat 1-containing genes Cj0262c and Cj0144. Contains N-terminal signal sequence and transmembrane anchor around aa 300. Also contains Pfam match to entry PF00015 MCPsignal,Methyl-accepting chemotaxis protein (MCP) signaling domain; Update [...] ","uniprotId":"Q0P864_CAMJE"},"10642345":{"id":10642345,"externalId":"192222.Cj1565c","name":"pflA","hasFamily":true,"annotation":"Paralysed flagellum protein; Original (2000) note: Cj1565c, pflA, required for flagellar function, len: 788 aa; identical to TR:Q46092 (EMBL:U09019) C. jejuni pflA (788 aa). 22.7% identity to HP1274. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF07719 Tetratricopeptide repeat identified within CDS. Similar to previously characterised PflA protein in Campylobacter jejuni. Paper identified linking protein to glycoprotein. Functional classification - Surface structures; PMID:7715450, PMID:12186869","uniprotId":"Q0P863_CAMJE"},"10642346":{"id":10642346,"externalId":"192222.Cj1566c","name":"nuoN","hasFamily":true,"annotation":"NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"Q0P862_CAMJE"},"10642347":{"id":10642347,"externalId":"192222.Cj1567c","name":"nuoM","hasFamily":true,"annotation":"Original (2000) note: Cj1567c, nuoM, probable NADH dehydrogenase I chain M, len: 495 aa; similar to many e.g. NU4M_CHOCR NADH-ubiquinone oxidoreductase chain 4 (EC 1.6.5.3) (492 aa), fasta scores; opt: 1055 z-score: 1213.7 E(): 0, 36.4% identity in 495 aa overlap, and NUOM_ECOLI NADH dehydrogenase I chain M(EC 1.6.5.3) (509 aa), blastp scores; E= 1.5e-58, 32% identity in 419 aa overlap. 40.1% identity to HP1272. Contains Pfam match to entry PF00361 oxidored_q1, NADH-Ubiquinone/plastoquinone (complex I),various chains; Updated (2006) note: Characterised operon within Escherichia coli. A [...] ","uniprotId":"Q0P861_CAMJE"},"10642348":{"id":10642348,"externalId":"192222.Cj1568c","name":"nuoL","hasFamily":true,"annotation":"NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be menaquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient (By similarity)","uniprotId":"NUOL_CAMJE"},"10642349":{"id":10642349,"externalId":"192222.Cj1569c","name":"nuoK","hasFamily":true,"annotation":"NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"Q0P859_CAMJE"},"10642350":{"id":10642350,"externalId":"192222.Cj1570c","name":"nuoJ","hasFamily":true,"annotation":"NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient. ","uniprotId":"Q0P858_CAMJE"},"10642351":{"id":10642351,"externalId":"192222.Cj1571c","name":"nuoI","hasFamily":true,"annotation":"NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"NUOI_CAMJE"},"10642352":{"id":10642352,"externalId":"192222.Cj1572c","name":"nuoH","hasFamily":true,"annotation":"NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient. This subunit may bind ubiquinone. ","uniprotId":"NUOH_CAMJE"},"10642353":{"id":10642353,"externalId":"192222.Cj1573c","name":"nuoG","hasFamily":true,"annotation":"Original (2000) note: Cj1573c, nuoG, probable NADH dehydrogenase I chain G, len: 820 aa; similar to many e.g. NUAM_RECAM NADH-ubiquinone oxidoreductase 75 kD subunit (691 aa), fasta scores; opt: 320 z-score: 333.6 E(): 2.9e-11, 25.3% identity in 450 aa overlap, and NUOG_ECOLI NADH dehydrogenase I chain G (907 aa), blastp scores; E= 1.8e-10, 26% identity in 161 aa overlap, 30% identity in 43 aa overlap and 36% identity in 22 overlap. 36.3% identity to HP1266. Contains PS00198 4Fe-4S ferredoxins,iron-sulfur binding region signature, and 2x Pfam match to entry PF00037 fer4, 4Fe-4S ferredo [...] ","uniprotId":"Q0P855_CAMJE"},"10642354":{"id":10642354,"externalId":"192222.Cj1574c","name":"Cj1574c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1574c, unknown, len: 230 aa; 26.5% identity to HP1265. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P854_CAMJE"},"10642355":{"id":10642355,"externalId":"192222.Cj1575c","name":"Cj1575c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1575c, unknown, len: 75 aa; 52.0% identity to HP1264. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P853_CAMJE"},"10642356":{"id":10642356,"externalId":"192222.Cj1576c","name":"nuoD","hasFamily":true,"annotation":"NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"NUOD_CAMJE"},"10642357":{"id":10642357,"externalId":"192222.Cj1577c","name":"nuoC","hasFamily":true,"annotation":"NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. ","uniprotId":"Q0P851_CAMJE"},"10642358":{"id":10642358,"externalId":"192222.Cj1578c","name":"nuoB","hasFamily":true,"annotation":"NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"NUOB_CAMJE"},"10642359":{"id":10642359,"externalId":"192222.Cj1579c","name":"nuoA","hasFamily":true,"annotation":"NDH-1 shuttles electrons from NADH, via FMN and iron-sulfur (Fe-S) centers, to quinones in the respiratory chain. ","uniprotId":"Q0P849_CAMJE"},"10642360":{"id":10642360,"externalId":"192222.Cj1580c","name":"Cj1580c","hasFamily":true,"annotation":"Original (2000) note: Cj1580c, probable peptide ABC-transport system ATP-binding protein, len: 221 aa; similar to e.g. DPPF_ECOLI dipeptide transport ATP-binding protein (334 aa), fasta scores; opt: 436 z-score: 498.1 E(): 2e-20, 34.8% identity in 210 aa overlap, and APPF_BACSU oligopeptide transport ATP-binding protein (329 aa), fasta scores; opt: 419 z-score: 479.2 E(): 2.3e-19,30.8% identity in 211 aa overlap. 30.6% identity to HP0302. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran, ABC tran [...] ","uniprotId":"Q0P848_CAMJE"},"10642361":{"id":10642361,"externalId":"192222.Cj1581c","name":"Cj1581c","hasFamily":true,"annotation":"Original (2000) note: Cj1581c, probable peptide ABC-transport system ATP-binding protein, len: 236 aa; similar to e.g. OPPD_ECOLI oligopeptide transport ATP-binding protein (337 aa), fasta scores opt: 385 z-score: 442.8 E(): 2.4e-17, 31.5% identity in 232 aa overlap. 29.8% identity to HP0301. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran, ABC transporters; Updated (2006) note: Similar to different transport systems with acceptable identity scores. Putative kept within product function. Functio [...] ","uniprotId":"Q0P847_CAMJE"},"10642362":{"id":10642362,"externalId":"192222.Cj1582c","name":"Cj1582c","hasFamily":true,"annotation":"Original (2000) note: Cj1582c, probable peptide ABC-transport system permease protein, len: 264 aa; similar to e.g. PPC_BACSU oligopeptide transport permease protein (303 aa), fasta scores; opt: 397 z-score: 467.3 E(): 1e-18, 29.1% identity in 261 aa overlap. 26.3% identity to HP0300. Contains Pfam match to entry PF00528 BPD_transp, Binding-protein-dependent transport systems inner membrane component; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Similar to different transport systems with acceptable identity scores. Putative kept within product functio [...] ","uniprotId":"Q0P846_CAMJE"},"10642363":{"id":10642363,"externalId":"192222.Cj1583c","name":"Cj1583c","hasFamily":true,"annotation":"Original (2000) note: Cj1583c, probable peptide ABC-transport system permease protein, len: 314 aa; similar to e.g. OPPB_LACLC oligopeptide transport system permease protein (319 aa), fasta scores; opt: 476 z-score: 540.7 E(): 8.5e-23, 28.4% identity in 317 aa overlap. 24.1% identity to HP0299; Updated (2006) note: Six probable transmembrane helices predicted by TMHMM2.0. Pfam domain PF00528 Binding-protein-dependent transport system inner membrane component identified within CDS. Similar to different transport systems with acceptable identity scores. Putative kept within product funct [...] ","uniprotId":"Q0P845_CAMJE"},"10642364":{"id":10642364,"externalId":"192222.Cj1584c","name":"Cj1584c","hasFamily":true,"annotation":"Original (2000) note: Cj1584c, probable peptide ABC-transport system periplasmic peptide-binding protein,len: 511 aa; similar to e.g. APPA_BACSU oligopeptide-binding protein APPA precursor (543 aa),fasta scores; opt: 700 z-score: 791.8 E(): 0, 30.5% identity in 509 aa overlap. 26.7% identity to HP0298. Contains probable N-terminal signal sequence and Pfam match to entry PF00496 SBP_bac_5, Bacterial extracellular solute-binding proteins, family 5; Updated (2006) note: Prosite domain PS00092 N6_MTASE, N-6 Adenine-specific DNA methylases signature identified within CDS. Similar to differe [...] ","uniprotId":"Q0P844_CAMJE"},"10642365":{"id":10642365,"externalId":"192222.Cj1585c","name":"Cj1585c","hasFamily":true,"annotation":"Putative oxidoreductase; Original (2000) note: Cj1585c, probable oxidoreductase, len: 923 aa; similar to e.g. YDIJ_ECOLI (1018 aa), fasta scores; opt: 299 z-score: 326.0 E(): 7.7e-11, 24.0% identity in 668 aa overlap, and GLCD_ECOLI glycolate oxidase subunit GLCD (499 aa), fasta scores; opt: 258 z-score: 285.3 E(): 1.4e-08, 23.4% identity in 483 aa overlap. 49.2% identity to HP1222 (misannotated as D-lactate dehydrogenase). Contains PS00551 Prokaryotic molybdopterin oxidoreductases signature 1, ands 2x PS00198 4Fe-4S ferredoxins, iron-sulfur binding region signature; Updated (2006) not [...] ","uniprotId":"Q0P843_CAMJE"},"10642366":{"id":10642366,"externalId":"192222.Cj1586","name":"cgb","hasFamily":true,"annotation":"Belongs to the globin family. ","uniprotId":"Q0P842_CAMJE"},"10642367":{"id":10642367,"externalId":"192222.Cj1587c","name":"Cj1587c","hasFamily":true,"annotation":"Putative ABC transporter; Original (2000) note: Cj1587c, probable ABC transporter, len: 543 aa; similar to e.g. YOJI_ECOLI hypothetical ABC transporter ATP-binding protein (547 aa),fasta scores; opt: 1391 z-score: 1550.2 E(): 0, 39.0% identity in 546 aa overlap. No Hp ortholog. N-terminus contains several probable transmembrane domains,C-terminus contains PS00017 ATP/GTP-binding site motif A (P-loop), and Pfam match to entry PF00005 ABC_tran, ABC transporters; Updated (2006) note: Pfam domain PF00664 ABC transporter transmembrane region identified within CDS. Further support given to p [...] ","uniprotId":"Q0P841_CAMJE"},"10642368":{"id":10642368,"externalId":"192222.Cj1588c","name":"Cj1588c","hasFamily":true,"annotation":"Putative MFS (Major Facilitator Superfamily) transport protein; Original (2000) note: Cj1588c, probable transmembrane transport protein, len: 431 aa; similar to e.g. YHJE_ECOLI hypothetical metabolite transport protein (440 aa), fasta scores; opt: 682 z-score: 771.4 E(): 0,28.9% identity in 422 aa overlap, and PROP_ECOLI proline/betaine transporter (proline porter II) (500 aa),fasta scores; opt: 655 z-score: 740.4 E(): 0, 30.1% identity in 415 aa overlap. No Hp ortholog. Contains PS00217 Sugar transport proteins signature 2, and Pfam match to entry PF00083 sugar_tr, Sugar (and other) t [...] ","uniprotId":"Q0P840_CAMJE"},"10642369":{"id":10642369,"externalId":"192222.Cj1589","name":"Cj1589","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1589, unknown, len: 265 aa; some similarity to TR:O34910 (EMBL:AF027868) Bacillus subtilis YOBT (233 aa), fasta scores; opt: 178 z-score: 211.5 E(): 0.00019, 27.6% identity in 203 aa overlap. No Hp match. Contains Pfam match to entry PF00753 lactamase_B, Metallo-beta-lactamase superfamily. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P839_CAMJE"},"10642370":{"id":10642370,"externalId":"192222.Cj1590","name":"infA","hasFamily":true,"annotation":"One of the essential components for the initiation of protein synthesis. Stabilizes the binding of IF-2 and IF-3 on the 30S subunit to which N-formylmethionyl-tRNA(fMet) subsequently binds. Helps modulate mRNA selection, yielding the 30S pre-initiation complex (PIC). Upon addition of the 50S ribosomal subunit IF-1, IF-2 and IF-3 are released leaving the mature 70S translation initiation complex. ","uniprotId":"IF1_CAMJE"},"10642371":{"id":10642371,"externalId":"192222.Cj1591","name":"rpmJ","hasFamily":true,"annotation":"Belongs to the bacterial ribosomal protein bL36 family. ","uniprotId":"RL36_CAMJE"},"10642372":{"id":10642372,"externalId":"192222.Cj1592","name":"rpsM","hasFamily":true,"annotation":"Located at the top of the head of the 30S subunit, it contacts several helices of the 16S rRNA. In the 70S ribosome it contacts the 23S rRNA (bridge B1a) and protein L5 of the 50S subunit (bridge B1b), connecting the 2 subunits; these bridges are implicated in subunit movement. Contacts the tRNAs in the A and P-sites. ","uniprotId":"RS13_CAMJE"},"10642373":{"id":10642373,"externalId":"192222.Cj1593","name":"rpsK","hasFamily":true,"annotation":"Located on the platform of the 30S subunit, it bridges several disparate RNA helices of the 16S rRNA. Forms part of the Shine- Dalgarno cleft in the 70S ribosome","uniprotId":"RS11_CAMJE"},"10642374":{"id":10642374,"externalId":"192222.Cj1594","name":"rpsD","hasFamily":true,"annotation":"One of the primary rRNA binding proteins, it binds directly to 16S rRNA where it nucleates assembly of the body of the 30S subunit. ","uniprotId":"RS4_CAMJE"},"10642375":{"id":10642375,"externalId":"192222.Cj1595","name":"rpoA","hasFamily":true,"annotation":"DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates. ","uniprotId":"RPOA_CAMJE"},"10642376":{"id":10642376,"externalId":"192222.Cj1596","name":"rplQ","hasFamily":true,"annotation":"50S ribosomal protein L17; Original (2000) note: Cj1596, rplQ, probable 50S ribosomal protein L17, len: 117 aa; similar to many e.g. RL17_ECOLI 50S ribosomal protein L17 (127 aa), fasta scores; opt: 346 z-score: 460.2 E(): 2.6e-18, 49.1% identity in 116 aa overlap. 73.3% identity to HP1292. Contains PS01167 Ribosomal protein L17 signature, and Pfam match to entry PF01196 Ribosomal_L17, Ribosomal protein L17; Updated (2006) note: Characterised within Escherichia coli with acceptable identity score. Putative not added to product function. Functional classification -Ribosomal protein synt [...] ","uniprotId":"RL17_CAMJE"},"10642377":{"id":10642377,"externalId":"192222.Cj1597","name":"hisG","hasFamily":true,"annotation":"Catalyzes the condensation of ATP and 5-phosphoribose 1- diphosphate to form N'-(5'-phosphoribosyl)-ATP (PR-ATP). Has a crucial role in the pathway because the rate of histidine biosynthesis seems to be controlled primarily by regulation of HisG enzymatic activity. ","uniprotId":"HIS1_CAMJE"},"10642378":{"id":10642378,"externalId":"192222.Cj1598","name":"hisD","hasFamily":true,"annotation":"Catalyzes the sequential NAD-dependent oxidations of L- histidinol to L-histidinaldehyde and then to L-histidine. ","uniprotId":"HISX_CAMJE"},"10642379":{"id":10642379,"externalId":"192222.Cj1599","name":"hisB","hasFamily":true,"annotation":"In the C-terminal section; belongs to the imidazoleglycerol-phosphate dehydratase family","uniprotId":"HIS7_CAMJE"},"10642380":{"id":10642380,"externalId":"192222.Cj1600","name":"hisH2","hasFamily":true,"annotation":"IGPS catalyzes the conversion of PRFAR and glutamine to IGP, AICAR and glutamate. The HisH subunit provides the glutamine amidotransferase activity that produces the ammonia necessary to HisF for the synthesis of IGP and AICAR (By similarity)","uniprotId":"HIS52_CAMJE"},"10642381":{"id":10642381,"externalId":"192222.Cj1601","name":"hisA","hasFamily":true,"annotation":"1-(5-phosphoribosyl)-5-[(5-phosphoribosylamino)methylideneamino] imidazole-4-carboxamide isomerase; Original (2000) note: Cj1601, hisA, probable phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase, len: 243 aa; similar to many e.g. HIS4_ECOLI phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16) (245 aa),fasta scores; opt: 778 z-score: 919.6 E(): 0, 50.6% identity in 243 aa overlap. No Hp match. Contains fam match to entry PF00977 His_biosynth, Histidine biosynthesis protein; Updated (2006) note: Characterised within Escherichia coli  [...] ","uniprotId":"HIS4_CAMJE"},"10642382":{"id":10642382,"externalId":"192222.Cj1602","name":"Cj1602","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1602, unknown, len: 318 aa; 35.8% identity to HP0852; Updated (2006) note: Pfam domain PF04373 Protein of unknown function (DUF511) identified within CDS. Conserved added to product function. Functional classification -Conserved hypothetical proteins; PMID:11956101, PMID:15901688","uniprotId":"Q0P826_CAMJE"},"10642383":{"id":10642383,"externalId":"192222.Cj1603","name":"hisF1","hasFamily":true,"annotation":"IGPS catalyzes the conversion of PRFAR and glutamine to IGP, AICAR and glutamate. The HisF subunit catalyzes the cyclization activity that produces IGP and AICAR from PRFAR using the ammonia provided by the HisH subunit (By similarity)","uniprotId":"HIS61_CAMJE"},"10642384":{"id":10642384,"externalId":"192222.Cj1604","name":"hisI","hasFamily":true,"annotation":"In the C-terminal section; belongs to the PRA-PH family. ","uniprotId":"HIS2_CAMJE"},"10642385":{"id":10642385,"externalId":"192222.Cj1605c","name":"dapD","hasFamily":true,"annotation":"Catalyzes the conversion of the cyclic tetrahydrodipicolinate (THDP) into the acyclic N-succinyl-L-2-amino-6-oxopimelate using succinyl-CoA","uniprotId":"DAPD_CAMJE"},"10642386":{"id":10642386,"externalId":"192222.Cj1606c","name":"mrp","hasFamily":true,"annotation":"Binds and transfers iron-sulfur (Fe-S) clusters to target apoproteins. Can hydrolyze ATP","uniprotId":"Q0P822_CAMJE"},"10642387":{"id":10642387,"externalId":"192222.Cj1607","name":"ispDF","hasFamily":true,"annotation":"Bifunctional enzyme that catalyzes the formation of 4- diphosphocytidyl-2-C-methyl-D-erythritol from CTP and 2-C-methyl-D- erythritol 4-phosphate (MEP) (IspD), and catalyzes the conversion of 4- diphosphocytidyl-2-C-methyl-D-erythritol 2-phosphate (CDP-ME2P) to 2-C- methyl-D-erythritol 2,4-cyclodiphosphate (ME-CPP) with a corresponding release of cytidine 5-monophosphate (CMP) (IspF)","uniprotId":"ISPDF_CAMJE"},"10642388":{"id":10642388,"externalId":"192222.Cj1608","name":"Cj1608","hasFamily":true,"annotation":"Original (2000) note: Cj1608, possible two-component regulator, len: 295 aa, some similarity to regulatory components of tw-component systems e.g. TR:O87527 (EMBL:AF082668) Streptococcus pyogenes capsule synthesis regulator CSRR (228 aa), fasta scores; opt: 170 z-score: 203.9 E(): 0.00049, 35.0% identity in 123 aa overlap, and BASR_ECOLI transcriptional regulatory protein (222 aa), blastp scores; E = 1.6e-06 29% identity in 113 aa overlap. 33.8% identity to HP1021. Contains helix-turn-helix motif at aa 271-292 (Score 997, +2.58 SD). Also some similarity to Cj1024c (22.0% identity in 34 [...] ","uniprotId":"Q0P820_CAMJE"},"10642389":{"id":10642389,"externalId":"192222.Cj1609","name":"Cj1609","hasFamily":true,"annotation":"Original (2000) note: Cj1609, possible sulfate adenylyltransferase, len: 386 aa; similar to e.g. TR:O34764 (EMBL:) BAcillus subtilis putative sulfate adenylyltransferase (382 aa), fasta scores; opt: 360 z-score: 414.1 E(): 9.6e-16, 23.4% identity in 351 aa overlap, and TR:O66036 (EMBL:U84759) Chromatium vinosum sulfate adenylyltransferase (EC 2.7.7.4) (397 aa), fasta scores; opt: 273 z-score: 315.2 E(): 3.1e-10, 21.8% identity in 357 aa overlap. No Hp match; Updated (2006) note: No specific characterisation with acceptable identity score carried out yet. Putative kept within product fu [...] ","uniprotId":"Q0P819_CAMJE"},"10642390":{"id":10642390,"externalId":"192222.Cj1610","name":"pgpA","hasFamily":true,"annotation":"Lipid phosphatase which dephosphorylates phosphatidylglycerophosphate (PGP) to phosphatidylglycerol (PG). ","uniprotId":"Q0P818_CAMJE"},"10642391":{"id":10642391,"externalId":"192222.Cj1611","name":"rpsT","hasFamily":true,"annotation":"Binds directly to 16S ribosomal RNA","uniprotId":"RS20_CAMJE"},"10642392":{"id":10642392,"externalId":"192222.Cj1612","name":"prfA","hasFamily":true,"annotation":"Peptide chain release factor 1 directs the termination of translation in response to the peptide chain termination codons UAG and UAA","uniprotId":"RF1_CAMJE"},"10642393":{"id":10642393,"externalId":"192222.Cj1613c","name":"Cj1613c","hasFamily":true,"annotation":"Putative pyridoxamine 5'-phosphate oxidase; Original (2000) note: Cj1613c, unknown, len: 251 aa; similar to hypothetical proteins e.g. Y854_HAEIN hypothetical protein HI0854 (253 aa), fasta scores; opt: 827 z-score: 991.6 E(): 0, 48.4% identity in 248 aa overlap. 56.1% identity to HP0318; Updated (2006) note: Pfam domain PF01243 Pyridoxamine 5'-phosphate oxidase identified within CDS. Product modified to more specific family member based on motif match. No specific characterisation with acceptable identity score has been carried out yet. Putative kept within product function. Literatur [...] ","uniprotId":"Q0P815_CAMJE"},"10642394":{"id":10642394,"externalId":"192222.Cj1614","name":"chuA","hasFamily":true,"annotation":"Original (2000) note: Cj1614, chuA, haemin uptake system outer membrane receptor, len: 709 aa; mutants are unable to grow on haemin - see van Vliet et al, J. Bact 180, 5921-5298 (1998). Similar to TR:O85161 (EMBL:AF047484) Vibrio vulnificus heme receptor (712 aa),fasta scores; opt: 270 z-score: 302.6 E(): 1.6e-09, 22.9% identity in 750 aa overlap, and TR:Q56644 (EMBL:L27149) Vibrio cholerae heme receptor (693 aa), fast scores; opt: 242 z-score: 271.3 E(): 8.6e-08, 23.0% identity in 744 aa overlap. 21.6% identity to HP0807. Also similar to Cj0755 (23.6% identity in 770 aa overlap), and  [...] ","uniprotId":"Q0P814_CAMJE"},"10642395":{"id":10642395,"externalId":"192222.Cj1615","name":"chuB","hasFamily":true,"annotation":"Belongs to the binding-protein-dependent transport system permease family. FecCD subfamily","uniprotId":"Q0P813_CAMJE"},"10642396":{"id":10642396,"externalId":"192222.Cj1616","name":"chuC","hasFamily":true,"annotation":"Original (2000) note: Cj1616, chuC, probable haemin uptake system ATP-binding protein, len: 258 aa; similar to e.g. FEPC_ECOLI ferric enterobactin transport ATP-binding protein (271 aa), fasta scores; opt: 477 z-score: 544.5 E(): 5.2e-23, 32.6% identity in 242 aa overlap. 32.9% identity to HP0888. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran, ABC transporters. Predicted function is based on apparent co-transcription with Cj1614 chuA; Updated (2006) note: Yet to be specifically characterised w [...] ","uniprotId":"Q0P812_CAMJE"},"10642397":{"id":10642397,"externalId":"192222.Cj1617","name":"chuD","hasFamily":true,"annotation":"Original (2000) note: Cj1617, chuD, probable haemin uptake system periplasmic haemin-binding protein, len: 268 aa; similar to e.g. HMUT_YERPE hemin-binding periplasmic protein HMUT precursor (278 aa), fasta scores; opt: 189 z-score: 223.7 E(): 3.9e-05, 23.9% identity in 230 aa overlap. No Hp match. Contains probable N-terminal signal sequence. Predicted function is based on apparent co-transcription with Cj1614 chuA; Updated (2006) note: Pfam domain PF01497 Periplasmic binding protein identified within CDS. Further support given to product function. Yet to be specifically characterised [...] ","uniprotId":"Q0P811_CAMJE"},"10642398":{"id":10642398,"externalId":"192222.Cj1618c","name":"Cj1618c","hasFamily":true,"annotation":"Putative radical SAM domain protein; Original (2000) note: Cj1618c, unknown, len: 305 aa; weak similarity to hypothetical proteins e.g. TR:Q59026 (EMBL:U67603) Methanococcus jannaschii hypothetical protein MJ1632 (255 aa), fasta scores; opt: 138 z-score: 169.9 E(): 0.038, 25.8% identity in 256 aa overlap (contains predicted iron-sulfur cluster which is conserved in Cj1618c). 34.5% identity to HP0568; Updated (2006) note: Pfam domain PF04055 Radical SAM superfamily identified within CDS. Product modified to more specific family member based on motif match. No specific characterisation h [...] ","uniprotId":"Q0P810_CAMJE"},"10642399":{"id":10642399,"externalId":"192222.Cj1619","name":"kgtP","hasFamily":true,"annotation":"Alpha-ketoglutarate permease; Original (2000) note: Cj1619, kgtP, probable alpha-ketoglutarate permease, len: 419 aa; similar to KGTP_ECOLI alpha-ketoglutarate permease (432 aa), fasta scores; opt: 1348 z-score: 1575.2 E(): 0, 49.0% identity in 414 aa overlap. 49.9% identity to HP1091. Contains PS00217 Sugar transport proteins signature 2, and Pfam match to entry PF00083 sugar_tr, Sugar (and other) transporters; Updated (2006) note: PMID:2053984; Updated (2006) note: Twelve probable transmembrane helices predicted by TMHMM2.0. Characterised within Escherichia coli with acceptable ident [...] ","uniprotId":"Q0P809_CAMJE"},"10642400":{"id":10642400,"externalId":"192222.Cj1620c","name":"mutY","hasFamily":true,"annotation":"A/G-specific adenine glycosylase; Original (2000) note: Cj1620c, mutY, probable A/G-specific adenine glycosylase, len: 339 aa; similar to e.g. MUTY_ECOLI A/G-specific adenine glycosylase (EC 3.2.2.-) (350 aa), blastp scores; E= 1.0e-41, 39% identity in 271 aa overlap. 47.6% identity to HP0142. Contains Pfam match to entry PF00730 Endonuclease_3, Endonuclease III; Updated (2006) note: Pfam domain PF00730 HhH-GPD superfamily base excision DNA repair and PF00633 Helix-hairpin-helix motif were both identified within CDS. Further support given to product function. Characterisation work has  [...] ","uniprotId":"Q0P808_CAMJE"},"10642401":{"id":10642401,"externalId":"192222.Cj1621","name":"Cj1621","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1621, possible periplasmic protein, len: 250 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P807_CAMJE"},"10642402":{"id":10642402,"externalId":"192222.Cj1622","name":"ribD","hasFamily":true,"annotation":"Original (2000) note: Cj1622, ribD, probable riboflavin-specific deaminase, len: 336 aa; similar to e.g. RIBD_ACTPL riboflavin-specific deaminase (EC 3.5.4.-) (410 aa), fasta scores; opt: 320 z-score: 381.1 E(): 6.6e-14, 35.7% identity in 235 aa overlap, and RIBD_ECOLI riboflavin-specific deaminase (EC 3.5.4.-). (367 aa), opt: 249 z-score: 298.4 E(): 2.7e-09, 28.4% identity in 282 aa overlap. 37.9% identity to ; HP1505. Contains Pfam match to entry PF00383 dCMP_cyt_deam, Cytidine and deoxycytidylate deaminases zinc-binding region; Updated (2006) note: Characterisation has been carried  [...] ","uniprotId":"Q0P806_CAMJE"},"10642403":{"id":10642403,"externalId":"192222.Cj1623","name":"Cj1623","hasFamily":true,"annotation":"Putative membrane protein; Original (2000) note: Cj1623, probable membrane protein, len: 173 aa; no Hp match; Updated (2006) note: Two probable transmembrane helices predicted by TMHMM2.0. Functional classification -Membranes, lipoproteins and porins","uniprotId":"Q0P805_CAMJE"},"10642404":{"id":10642404,"externalId":"192222.Cj1624c","name":"sdaA","hasFamily":true,"annotation":"Belongs to the iron-sulfur dependent L-serine dehydratase family","uniprotId":"Q0P804_CAMJE"},"10642405":{"id":10642405,"externalId":"192222.Cj1625c","name":"sdaC","hasFamily":true,"annotation":"Amino acid transporter; Original (2000) note: Cj1625c, sdaC, probable serine transporter, len: 416 aa; similar to e.g. SDAC_ECOLI serine transporter (429 aa), fasta scores; opt: 1392 z-score: 1574.7 E(): 0, 50.1% identity in 417 aa overlap, and TDCC_ECOLI threonine/serine transporter (443 aa), fasta scores; opt: 848 z-score: 961.6 E(): 0, 43.8% identity in 432 aa overlap. 42.0% identity to HP0133; Updated (2006) note: Prosite domains PS50286 AROMATIC_AA_PERM_2, Amino acid/polyamine transporter II and PS50285 AMINO_ACID_PERMEASE_2, Amino acid/polyamine transporter I were identified with [...] ","uniprotId":"Q0P803_CAMJE"},"10642406":{"id":10642406,"externalId":"192222.Cj1626c","name":"Cj1626c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1626c, probable periplasmic protein, len: 138 aa; no Hp match. Also similar to Cj1004 (38.6% identity in 140 aa overlap), and in N-terminus to Cj1722c (53.1% identity in 32 aa overlap); Updated (2006) note: Literature search identified paper giving further clues to product function. Functional classification - Miscellaneous periplasmic proteins; PMID:15554967","uniprotId":"Q0P802_CAMJE"},"10642407":{"id":10642407,"externalId":"192222.Cj1627c","name":"Cj1627c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1627c, unknown, len: 247 aa; no Hp match; Updated (2006) note: Similar to hypothetical proteins from other bacteria. Functional classification -Unknown","uniprotId":"Q0P801_CAMJE"},"10642408":{"id":10642408,"externalId":"192222.Cj1628","name":"exbB2","hasFamily":true,"annotation":"Original (2000) note: Cj1628, exbB2, probable exbB/tolQ family transport protein, len: 141 aa; similar to many e.g. EXBB_HAEDU biopolymer transport EXBB protein (150 aa), fasta scores; opt: 426 z-score: 531.7 E(): 2.7e-22, 51.2% identity in 129 aa overlap. 54.4% identity to HP1339. Also similar to Cj0179 exbB (31.3% identity in 150 aa overlap), and Cj0109 exbB2 (44.4% identity in 81 aa overlap); Updated (2006) note: Pfam domain PF01618 MotA/TolQ/ExbB proton channel family identified within CDS. Further support given to product function. Three probable transmembrane helices predicted by [...] ","uniprotId":"Q0P800_CAMJE"},"10642409":{"id":10642409,"externalId":"192222.Cj1629","name":"exbD2","hasFamily":true,"annotation":"Original (2000) note: Cj1629, exbD2, probable exbD/tolR family transport protein, len: 129 aa; similar to many e.g. EXBD_HAEDU biopolymer transport EXBD protein (129 aa), fasta scores; opt: 337 z-score: 414.6 E(): 9e-16, 44.8% identity in 125 aa overlap. 49.6% identity to HP1340. Also similar to Cj0180 exbD (31.1% identity in 122 aa overlap) and Cj0110 exbD2 (26.4% identity in 121 aa overlap); Updated (2006) note: Pfam domain PF02472 Biopolymer transport protein ExbD/TolR identified within CDS. Further support given to product function. One probable transmembrane helices predicted by T [...] ","uniprotId":"Q0P7Z9_CAMJE"},"10642410":{"id":10642410,"externalId":"192222.Cj1630","name":"tonB2","hasFamily":true,"annotation":"Interacts with outer membrane receptor proteins that carry out high-affinity binding and energy dependent uptake into the periplasmic space of specific substrates. It could act to transduce energy from the cytoplasmic membrane to specific energy-requiring processes in the outer membrane, resulting in the release into the periplasm of ligands bound by these outer membrane proteins. ","uniprotId":"Q0P7Z8_CAMJE"},"10642411":{"id":10642411,"externalId":"192222.Cj1631c","name":"Cj1631c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1631c, unknown, len: 289 aa; no Hp match. Contains PS00018 EF-hand calcium-binding domain. Functional classification - Unknown","uniprotId":"Q0P7Z7_CAMJE"},"10642412":{"id":10642412,"externalId":"192222.Cj1632c","name":"Cj1632c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1632c, possible periplasmic protein, len: 53 aa; no Hp match. Contains possible N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P7Z6_CAMJE"},"10642413":{"id":10642413,"externalId":"192222.Cj1633","name":"Cj1633","hasFamily":true,"annotation":"Putative ATP-binding protein; Original (2000) note: Cj1633, unknown, len: 327 aa; similar to hypothetical proteins e.g. TR:O67046 (EMBL:AE000713) Aquifex aeolicus AQ_898 (251 aa), fasta scores; opt: 544 z-score: 648.2 E(): 8.8e-29, 50.2% identity in 221 aa overlap. 44.7% identity to HP0013. Contains HP0013; Updated (2006) note: Pfam domain PF01902 ATP-binding region identified within CDS. Product function modified to more specific family member based on motif match. Putative kept within product function. Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P7Z5_CAMJE"},"10642414":{"id":10642414,"externalId":"192222.Cj1634c","name":"aroC","hasFamily":true,"annotation":"Catalyzes the anti-1,4-elimination of the C-3 phosphate and the C-6 proR hydrogen from 5-enolpyruvylshikimate-3-phosphate (EPSP) to yield chorismate, which is the branch point compound that serves as the starting substrate for the three terminal pathways of aromatic amino acid biosynthesis. This reaction introduces a second double bond into the aromatic ring system","uniprotId":"AROC_CAMJE"},"10642415":{"id":10642415,"externalId":"192222.Cj1635c","name":"rnc","hasFamily":true,"annotation":"Digests double-stranded RNA. Involved in the processing of primary rRNA transcript to yield the immediate precursors to the large and small rRNAs (23S and 16S). Also processes some mRNAs, and tRNAs when they are encoded in the rRNA operon (By similarity). ","uniprotId":"RNC_CAMJE"},"10642416":{"id":10642416,"externalId":"192222.Cj1636c","name":"rnhA","hasFamily":true,"annotation":"Endonuclease that specifically degrades the RNA of RNA-DNA hybrids","uniprotId":"RNH_CAMJE"},"10642417":{"id":10642417,"externalId":"192222.Cj1637c","name":"Cj1637c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1637c, probable periplasmic protein, len: 328 aa; 26.1% identity to HP0660. Contains possible N-terminal signal sequence; Updated (2006) note: Pfam domain PF07719 Tetratricopeptide repeat identified within CDS. Functional classification - Miscellaneous periplasmic proteins","uniprotId":"Q0P7Z1_CAMJE"},"10642418":{"id":10642418,"externalId":"192222.Cj1638","name":"dnaG","hasFamily":true,"annotation":"RNA polymerase that catalyzes the synthesis of short RNA molecules used as primers for DNA polymerase during DNA replication. ","uniprotId":"DNAG_CAMJE"},"10642419":{"id":10642419,"externalId":"192222.Cj1639","name":"Cj1639","hasFamily":true,"annotation":"NifU protein homolog; Original (2000) note: Cj1639, unknown, len: 90 aa; similar to members of the NIFU family of hypothetical poteins e.g. NIU1_RHOCA NIFU protein 1 (135 aa), fasta scores; opt: 158 z-score: 219.3 E(): 6.8e-05 36.5% identity in 63 aa overlap. Identical to TR:O69292 (EMBL:Y16882) C. jejuni NIFU-like protein. 52.2% identity to HP1492. Contains Pfam match to entry PF01106 NifU-like,NifU-like domain. Also similar to N-terminus of Cj0239c (38.2% identity in 76 aa overlap). Functional classification - Conserved hypothetical proteins","uniprotId":"Q0P7Y9_CAMJE"},"10642420":{"id":10642420,"externalId":"192222.Cj1640","name":"Cj1640","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1640, unknown, len: 183 aa; 27.7% identity to HP1493. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P7Y8_CAMJE"},"10642421":{"id":10642421,"externalId":"192222.Cj1641","name":"murE","hasFamily":true,"annotation":"Catalyzes the addition of meso-diaminopimelic acid to the nucleotide precursor UDP-N-acetylmuramoyl-L-alanyl-D-glutamate (UMAG) in the biosynthesis of bacterial cell-wall peptidoglycan. ","uniprotId":"MURE_CAMJE"},"10642422":{"id":10642422,"externalId":"192222.Cj1642","name":"Cj1642","hasFamily":true,"annotation":"Binds to DNA and alters its conformation. May be involved in regulation of gene expression, nucleoid organization and DNA protection","uniprotId":"Y1642_CAMJE"},"10642423":{"id":10642423,"externalId":"192222.Cj1643","name":"Cj1643","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1643, probable periplasmic protein, len: 364 aa; similar to hypothetical proteins e.g. TR:O34248 (EMBL:AJ003049) Wolinella succinogenes ORF341 protein (341 aa), fasta scores; opt: 238 z-score: 283.8 E(): 1.7e-08, 23.6% identity in 314 aa overlap. 25.5% identity to HP0036. Contains probable N-terminal signal sequence, and Pfam match to entry PF00595 PDZ, PDZ domain (Also known as DHR or GLGF); Updated (2006) note: Paper identified linking protein to glycoprotein. Functional classification -Miscellaneous periplasmic proteins; PMID:12186869","uniprotId":"Q0P7Y5_CAMJE"},"10642424":{"id":10642424,"externalId":"192222.Cj1644","name":"ispA","hasFamily":true,"annotation":"Belongs to the FPP/GGPP synthase family. ","uniprotId":"Q0P7Y4_CAMJE"},"10642425":{"id":10642425,"externalId":"192222.Cj1645","name":"tkt","hasFamily":true,"annotation":"Catalyzes the transfer of a two-carbon ketol group from a ketose donor to an aldose acceptor, via a covalent intermediate with the cofactor thiamine pyrophosphate","uniprotId":"Q0P7Y3_CAMJE"},"10642426":{"id":10642426,"externalId":"192222.Cj1646","name":"iamB","hasFamily":true,"annotation":"Original (2000) note: Cj1646, iamB, possible ABC transport system permease protein, len: 369 aa; 91.6% identity to TR:O68254 (EMBL:AF023133) C. jejuni iamB gene fragment (133 aa). Similar to hypothetical proteins e.g. YRBE_ECOLI (260 aa), fast scores; opt: 363 z-score: 423.9 E(): 2.7e-16, 28.1% identity in 256 aa overlap, and to TR:AAD17958 (EMBL:AF106002) Pseudomonas putida toluene tolerance protein TTG2B (265 aa), fast scores; opt: 381 z-score: 444.3 E(): 2e-17, 30.0% identity in 203 aa overlap. 41.1% identity to HP1466; Updated (2006) note: Pfam domain PF02405 Domain of unknown func [...] ","uniprotId":"Q0P7Y2_CAMJE"},"10642427":{"id":10642427,"externalId":"192222.Cj1647","name":"iamA","hasFamily":true,"annotation":"Original (2000) note: Cj1647, iamA, probable ABC transport system ATP-binding protein, len: 240 aa; 89.5% identity to TR:O68255(EMBL:AF023133) C. jejuni iamA ABC-transporter. Similar to hypothetical proteins e.g. YRBF_ECOLI HYPOTHETICAL ABC transporter ATP-binding protein (269 aa), fasta scores; opt: 453 z-score: 533.0 E(): 2.3e-22, 31.8% identity in 239 aa overlap, and to TR:AAD17957 (EMBL:AF106002) Pseudomonas putida toluene tolerance protein TTG2A (269 aa), fasta scores; opt: 491 z-score: 576.9 E(): 8.2e-25, 34.5% identity in 238 aa overlap. 40.5% identity to HP1465. Contains PS0001 [...] ","uniprotId":"Q0P7Y1_CAMJE"},"10642428":{"id":10642428,"externalId":"192222.Cj1648","name":"Cj1648","hasFamily":true,"annotation":"Original (2000) note: Cj1648, possible ABC transport system periplasmic substrate-binding protein,len: 296 aa; similar to hypothetical proteins e.g. TR:O67491 (EMBL:AE000744) Aquifex aeolicus AQ_1533 (300 aa), fasta scores; opt: 362 z-score: 406.8 E(): 2.4e-15,29.7% identity in 303 aa overlap, and, weakly, to TR:AAD17959 (EMBL:AF106002) Pseudomonas putida toluene tolerance protein TTG2C (161 aa), fasta scores; opt: 161 z-score: 190.8 E(): 0.0026, 28.7% identity in 122 aa overlap. 24.0% identity to HP1464. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF [...] ","uniprotId":"Q0P7Y0_CAMJE"},"10642429":{"id":10642429,"externalId":"192222.Cj1649","name":"Cj1649","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj1649, probable lipoprotein,len: 199 aa; some similarity to YFXK_BRAJA hypothetical protein in (fragment) (180 aa), fast scores; opt: 184 z-score: 235.4 E(): 8.6e-06, 24.5% identity in 143 aa overlap. Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. No Hp match. Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P7X9_CAMJE"},"10642430":{"id":10642430,"externalId":"192222.Cj1650","name":"Cj1650","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1650, unknown, len: 166 aa; 30.7% identity to HP1076. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P7X8_CAMJE"},"10642431":{"id":10642431,"externalId":"192222.Cj1651c","name":"map","hasFamily":true,"annotation":"Removes the N-terminal methionine from nascent proteins. The N-terminal methionine is often cleaved when the second residue in the primary sequence is small and uncharged (Met-Ala-, Cys, Gly, Pro, Ser, Thr, or Val). Requires deformylation of the N(alpha)-formylated initiator methionine before it can be hydrolyzed","uniprotId":"Q0P7X7_CAMJE"},"10642432":{"id":10642432,"externalId":"192222.Cj1652c","name":"murI","hasFamily":true,"annotation":"Provides the (R)-glutamate required for cell wall biosynthesis","uniprotId":"MURI_CAMJE"},"10642433":{"id":10642433,"externalId":"192222.Cj1653c","name":"Cj1653c","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj1653c, probable lipoprotein, len: 152 aa; similar to hypothetical proteins e.g. NLPC_ECOLI probable lipoprotein NLPC precursor (154 aa), fasta scores; opt: 381 z-score: 485.2 E(): 1e-19,40.3% identity in 154 aa overlap, and SPR_ECOLI lipoprotein SPR precursor (188 aa), fasta scores; opt: 355 z-score: 451.8 E(): 7.6e-18, 44.7% identity in 123 aa overlap. No Hp match. Contains probable N-terminal signal sequence with appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site, and Pfam match to entry PF00877 NLPC_P [...] ","uniprotId":"Q0P7X5_CAMJE"},"10642434":{"id":10642434,"externalId":"192222.Cj1654c","name":"nhaA2","hasFamily":true,"annotation":"Na(+)/H(+) antiporter that extrudes sodium in exchange for external protons","uniprotId":"NHAA1_CAMJE"},"10642435":{"id":10642435,"externalId":"192222.Cj1655c","name":"nhaA1","hasFamily":true,"annotation":"Na(+)/H(+) antiporter that extrudes sodium in exchange for external protons","uniprotId":"NHAA2_CAMJE"},"10642436":{"id":10642436,"externalId":"192222.Cj1656c","name":"Cj1656c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1656c, unknown, len: 180 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0P7X2_CAMJE"},"10642437":{"id":10642437,"externalId":"192222.Cj1658","name":"Cj1658","hasFamily":true,"annotation":"Putative iron permease; Original (2000) note: Cj1658, probable integral membrane protein, len: 696 aa; similar to TR:CAA21360 (EMBL:AL031866) Yersinia pestis ORF37 upstream of p19 homolog in 102 kbases unstable region (639 aa), fasta scores; opt: 1055 z-score: 1112.1 E(): 0, 31.6% identity in 706 aa overlap. Contains possible N-terminal signal sequence, followed by non-membrane domain with three coiled-coil regions to aa 440. C-terminal domain contains six possible membrane spanning sequences, and is similar to hypothetical membrane proteins e.g. YWBL_BACSU (481 aa), fasta scores; opt: [...] ","uniprotId":"Q0P7X1_CAMJE"},"10642438":{"id":10642438,"externalId":"192222.Cj1659","name":"p19","hasFamily":true,"annotation":"Original (2000) note: Cj1659, p19, periplasmic protein, len: 179 aa; identical to TR:O07089 (EMBL:Y13641) C. jejuni 19 kDa periplamic protein (fragment) (152 aa),and similar to TR:CAA21359 (EMBL:AL031866) Yersinia pestis ORF36 in 102 kbases unstable region (175 aa), fasta scores; opt: 677 z-score: 805.8 E(): 0, 57.9% identity in 178 aa overlap, and TA34_TREPA 34 kD membrane antigen precursor (pathogen-specific membrane antigen) (204 aa),fasta scores; opt: 353 z-score: 425.6 E(): 2.2e-16, 35.8% identity in 159 aa overlap. No Hp match. Contains probable N-terminal signal sequence; Update [...] ","uniprotId":"Q0P7X0_CAMJE"},"10642439":{"id":10642439,"externalId":"192222.Cj1660","name":"Cj1660","hasFamily":true,"annotation":"Original (2000) note: Cj1660, probable integral membrane protein, len: 467 aa; 97.3% identity to TR:O07090 (EMBL:Y13641) C. jejuni hypothetical protein (fragment) (73 aa), and similar to TR:CAA21358 (EMBL:AL031866) Yersinia pestis ORF35 in 102 kbases unstable region (469 aa), fasta scores; opt: 687 z-score: 761.3 E(): 0, 27.5% identity in 465 aa overlap. No Hp match; Updated (2006) note: Eight probable transmembrane helices predicted by TMHMM2.0. Further support given to product function. Pfam domain PF04945 YHS domain identified within CDS. No specific characterisation with acceptable [...] ","uniprotId":"Q0P7W9_CAMJE"},"10642440":{"id":10642440,"externalId":"192222.Cj1661","name":"Cj1661","hasFamily":true,"annotation":"Original (2000) note: Cj1661, possible ABC transport system permease protein, len: 430 aa; similar to hypothetical membrane proteins e.g. TR:CAA21357 (EMBL:AL031866) Yersinia pestis ORF34 in 102 kbases unstable region (430 aa), fasta scores; opt: 929 z-score: 1043.7 E(): 0, 36.1% identity in 435 aa overlap, and to the membrane domain of hypothetical ACB-transporters e.g. YBJZ_ECOLI hypothetical ABC transporter ATP-binding protein (648 aa), fasta scores; opt: 189 z-score: 214.0 E(): 0.00014, 22.0% identity in 241 aa overlap. No Hp match. Also similar to Cj1662 (28.1% identity in 437 aa  [...] ","uniprotId":"Q0P7W8_CAMJE"},"10642441":{"id":10642441,"externalId":"192222.Cj1662","name":"Cj1662","hasFamily":true,"annotation":"Original (2000) note: Cj1662, probable integral membrane protein, len: 372 aa; similar to hypothetical membrane proteins e.g. TR:CAA21356 (EMBL:AL031866) Yersinia pestis ORF33 in 102 kbases unstable region (387 aa), fasta scores; opt: 637 z-score: 710.8 E(): 2.9e-32,32.5% identity in 375 aa overlap. No Hp match. Also similar to Cj1661 (28.1% identity in 437 aa overlap). Possibly a permease protein for an ABC transport system; ABC-binding protein is downstream; Updated (2006) note: Four probable transmembrane helices predicted by TMHMM2.0. Pfam domain PF02687 Predicted permease identifi [...] ","uniprotId":"Q0P7W7_CAMJE"},"10642442":{"id":10642442,"externalId":"192222.Cj1663","name":"Cj1663","hasFamily":true,"annotation":"Original (2000) note: Cj1663, probable ABC transport system ATP-binding protein, len: 217 aa; similar to many e.g. TR:CAA21355(EMBL:AL031866) Yersinia pestis ORF32 in 102 kbases unstable region (237 aa), fasta scores; opt: 859 z-score: 985.9 E(): 0, 60.8% identity in 209 aa overlap, and GLNQ_ECOLI glutamine transport ATP-binding protein (240 aa), fasta scores; opt: 555 z-score: 641.1 E(): 2.2e-28, 43.5% identity in 214 aa overlap. No Hp ortholog. Contains PS00017 ATP/GTP-binding site motif A (P-loop), PS00211 ABC transporters family signature, and Pfam match to entry PF00005 ABC_tran,  [...] ","uniprotId":"Q0P7W6_CAMJE"},"10642443":{"id":10642443,"externalId":"192222.Cj1664","name":"Cj1664","hasFamily":true,"annotation":"Original (2000) note: Cj1664, possible periplasmic thiredoxin, len: 162 aa; similar to e.g. TR:CAA21354 (EMBL:AL031866) Yersinia pestis ORF31 in 102 kbases unstable region (165 aa), fast scores; opt: 225 z-score: 278.4 E(): 3.5e-08, 33.3% identity in 141 aa overlap, and weakly to RESA_BACSU RESA protein (contains thiredoxin domain) (181 aa), fasta scores; opt: 124 z-score: 159.5 E(): 0.15, 25.0% identity in 112 aa overlap. No Hp match. Contains PS00194 Thioredoxin family active site and probable N-terminal signal sequence; Updated (2006) note: Specific characterisation with acceptable  [...] ","uniprotId":"Q0P7W5_CAMJE"},"10642444":{"id":10642444,"externalId":"192222.Cj1665","name":"Cj1665","hasFamily":true,"annotation":"Original (2000) note: Cj1665, possible lipoprotein thiredoxin, len: 167 aa; similar to TR:O33920 (EMBL:U75949) Salmonella typhimurium suppressor for copper-sensitivity D (168 aa), fasta scores; opt: 121 z-score: 163.1 E(): 0.093, 27.5% identity in 153 aa overlap, and weakly to many thiredoxins, e.g. THIO_BUCAP thioredoxin (108 aa), fasta scores; opt: 108 z-score: 149.9 E(): 0.5, 28.6% identity in 112 aa overlap. Contains probable N-terminal signal sequence and appropriately positioned PS00013 Prokaryotic membrane lipoprotein lipid attachment site. No Hp match; Updated (2006) note: Spec [...] ","uniprotId":"Q0P7W4_CAMJE"},"10642445":{"id":10642445,"externalId":"192222.Cj1666c","name":"Cj1666c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1666c, probable periplasmic protein, len: 145 aa; similar to TR:AAD11752 (EMBL:AF067954) Salmonella typhimurium ORF105 in silver resistance cluster (105 aa), fasta scores; opt: 244 z-score: 293.2 E(): 5.2e-09, 39.2% identity in 102 aa overlap, and TR:O67205 (EMBL:AE000724) Aquifex aeolicus AQ_1129 (122 aa), fats scores; opt: 231 z-score: 277.7 E(): 3.8e-08, 33.3% identity in 117 aa overlap. No Hp match. Contains probable N-terminal signal sequence; Updated (2006) note: Pfam domain PF04214 Protein of unknown function, DUF identified  [...] ","uniprotId":"Q0P7W3_CAMJE"},"10642446":{"id":10642446,"externalId":"192222.Cj1667c","name":"Cj1667c","hasFamily":false,"annotation":"RepA protein homolog; Original (2000) note: Cj1667c, repA protein homolog, len: 50 aa; highly similar to TR:Q46016 (EMBL:X82080) Campylobacter coli plasmid PCCT2 repA protein (26 aa), fast scores; opt: 93 z-score: 179.3 E(): 0.011, 57.7% identity in 26 aa overlap. No Hp match. This gene sequence is < 19% G+C and has an atypical positional base preference. Functional classification - Plasmid related functions","uniprotId":"Q0P7W2_CAMJE"},"10642447":{"id":10642447,"externalId":"192222.Cj1668c","name":"Cj1668c","hasFamily":false,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1668c, probable periplasmic protein, len: 138 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P7W1_CAMJE"},"10642448":{"id":10642448,"externalId":"192222.Cj1669c","name":"Cj1669c","hasFamily":true,"annotation":"Original (2000) note: Cj1669c, probable ATP-dependent DNA ligase, len: 282 aa; similar to DNLI_HAEIN DNA ligase (EC 6.5.1.1) (268 aa), fasta scores; opt: 484 z-score: 564.1 E(): 4.3e-24, 43.4% identity in 279 aa overlap, and weakly to several bacteriphage DNA ligases e.g. DNLI_ASFM2 DNA ligase (EC 6.5.1.1) (419 aa),fasta scores; opt: 186 z-score: 219.5 E(): 6.6e-05, 29.0% identity in 162 aa overlap. No Hp match. No similarity to Cj. jejuni ligA Cj0586. Contains PS00333 ATP-dependent DNA ligase signature 2, and Pfam match to entry PF01068 DNA_ligase; Updated (2006) note: Characterisatio [...] ","uniprotId":"Q0P7W0_CAMJE"},"10642449":{"id":10642449,"externalId":"192222.Cj1670c","name":"cgpA","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1670c, probable periplasmic protein, len: 220 aa; 27.3% identity to HP0156. Contains probable N-terminal signal sequence; Updated (2006) note: Characterisation paper within Campylobacter jejuni identified new gene name. Papers identified linking protein to glycoprotein (PMID:12186869). Functional classification - Miscellaneous periplasmic proteins; PMID:12186869, PMID:11985725","uniprotId":"Q0P7V9_CAMJE"},"10642450":{"id":10642450,"externalId":"192222.Cj1671c","name":"Cj1671c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1671c, unknown, len: 67 aa; 38.7% identity to HP0155; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P7V8_CAMJE"},"10642451":{"id":10642451,"externalId":"192222.Cj1672c","name":"eno","hasFamily":true,"annotation":"Catalyzes the reversible conversion of 2-phosphoglycerate into phosphoenolpyruvate. It is essential for the degradation of carbohydrates via glycolysis","uniprotId":"ENO_CAMJE"},"10642452":{"id":10642452,"externalId":"192222.Cj1673c","name":"recA","hasFamily":true,"annotation":"Can catalyze the hydrolysis of ATP in the presence of single- stranded DNA, the ATP-dependent uptake of single-stranded DNA by duplex DNA, and the ATP-dependent hybridization of homologous single-stranded DNAs. It interacts with LexA causing its activation and leading to its autocatalytic cleavage","uniprotId":"RECA_CAMJE"},"10642453":{"id":10642453,"externalId":"192222.Cj1674","name":"mqnD","hasFamily":true,"annotation":"Catalyzes the conversion of cyclic dehypoxanthine futalosine (cyclic DHFL) into 1,4-dihydroxy-6-naphthoate, a step in the biosynthesis of menaquinone (MK, vitamin K2)","uniprotId":"Q0P7V5_CAMJE"},"10642454":{"id":10642454,"externalId":"192222.Cj1675","name":"fliQ","hasFamily":true,"annotation":"Role in flagellar biosynthesis. ","uniprotId":"Q0P7V4_CAMJE"},"10642455":{"id":10642455,"externalId":"192222.Cj1676","name":"murB","hasFamily":true,"annotation":"Cell wall formation","uniprotId":"MURB_CAMJE"},"10642456":{"id":10642456,"externalId":"192222.Cj1677","name":"Cj1677","hasFamily":true,"annotation":"Putative lipoprotein; Original (2000) note: Cj1678, possible lipoprotein,len: 928 aa; no Hp match. Contains a poly-T tract at aa 1 which, if variable, would allow translation from Cj1677 (which contains a signal sequence and lipid attachment site). Otherwise translation could start at aa 46. Highly similar to Cj0629 (90.7% identity in 901 aa overlap). Cj0629 has a non-variable poly-T tract in an quivalent position, which could allow translation from Cj0628 (similar to Cj1677); Original (2000) note: Cj1677, probable lipoprotein,len: 216 aa; no Hp match. Contains N-terminal signal sequen [...] ","uniprotId":"Q0P7V2_CAMJE"},"10642457":{"id":10642457,"externalId":"192222.Cj1679","name":"Cj1679","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1679, unknown, len: 584 aa; no Hp match; Updated (2006) note: Pfam domain x2 PF07719 Tetratricopeptide repeat identified within CDS. Functional classification - Unknown","uniprotId":"Q0P7V1_CAMJE"},"10642458":{"id":10642458,"externalId":"192222.Cj1680c","name":"Cj1680c","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1680c, probable periplasmic protein, len: 255 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P7V0_CAMJE"},"10642459":{"id":10642459,"externalId":"192222.Cj1681c","name":"cysQ","hasFamily":true,"annotation":"Inositol-1-monophosphatase; Original (2000) note: Cj1681c, cysQ, probable cysQ protein homolog, len: 254 aa; similar to e.g. CYSQ_ECOLI CYSQ protein (246 aa), fasta scores; opt: 421 z-score: 500.9 E(): 1.4e-20, 32.4% identity in 247 aa overlap. No Hp match. Contains Pfam match to entry PF00459 inositol_P,Inositol monophosphatase family, and PS00629 Inositol monophosphatase family signature 1. Functional classification - Central intermediary metabolism - Sulphur metabolism; PMID:1729235, PMID:7493934","uniprotId":"Q0P7U9_CAMJE"},"10642460":{"id":10642460,"externalId":"192222.Cj1682c","name":"gltA","hasFamily":true,"annotation":"Belongs to the citrate synthase family. ","uniprotId":"Q0P7U8_CAMJE"},"10642461":{"id":10642461,"externalId":"192222.Cj1684c","name":"Cj1684c","hasFamily":true,"annotation":"Original (2000) note: Cj1684c, probable transmembrane transport protein, len: 391 aa; simlar to many predicted Na+/H+ antiporters, and to KEFB_ECOLI Glutathione-regulated potassium-efflux system protein (601 aa), fasta scores; opt: 197 z-score: 237.9 E(): 6.2e-06,21.9% identity in 324 aa overlap, and NAPA_ENTHR Na(+)/H(+) antiporter (383 aa), blastp scores; E= 3.1e-10,22% identity in 372 aa overlap. 35.9% identity to HP1183 (called NA+/H+ antiporter (napA) on basis of match to NAPA_ENTHR); Updated (2006) note: Eleven probable transmembrane helices predicted by TMHMM2.0. Further support [...] ","uniprotId":"Q0P7U7_CAMJE"},"10642462":{"id":10642462,"externalId":"192222.Cj1685c","name":"bioB","hasFamily":true,"annotation":"Catalyzes the conversion of dethiobiotin (DTB) to biotin by the insertion of a sulfur atom into dethiobiotin via a radical-based mechanism","uniprotId":"BIOB_CAMJE"},"10642463":{"id":10642463,"externalId":"192222.Cj1686c","name":"topA","hasFamily":true,"annotation":"Releases the supercoiling and torsional tension of DNA, which is introduced during the DNA replication and transcription, by transiently cleaving and rejoining one strand of the DNA duplex. Introduces a single-strand break via transesterification at a target site in duplex DNA. The scissile phosphodiester is attacked by the catalytic tyrosine of the enzyme, resulting in the formation of a DNA- (5'-phosphotyrosyl)-enzyme intermediate and the expulsion of a 3'-OH DNA strand. The free DNA strand then undergoes passage around the unbroken strand, thus removing DNA supercoils. Finally, in t [...] ","uniprotId":"TOP1_CAMJE"},"10642464":{"id":10642464,"externalId":"192222.Cj1687","name":"Cj1687","hasFamily":true,"annotation":"Putative efflux protein; Original (2000) note: Cj1687, possible efflux protein, len: 423 aa; similar to members of the major facilitator family e.g. NORA_STAAU quinolone resistance NORA protein (388 aa), fasta scores; opt: 333 z-score: 386.0 E(): 3.5e-14, 23.1% identity in 350 aa overlap, and BMR1_BACSU multidrug resistance protein 1 (389 aa), fasta scores; opt: 282 z-score: 327.8 E(): 6.1e-11, 24.6% identity in 305 aa overlap. No Hp match. Contains Pfam match to entry PF00083 sugar_tr, Sugar (and other) transporters; Updated (2006) note: Pfam domain PF07690 Major Facilitator Superfami [...] ","uniprotId":"Q0P7U4_CAMJE"},"10642465":{"id":10642465,"externalId":"192222.Cj1688c","name":"secY","hasFamily":true,"annotation":"The central subunit of the protein translocation channel SecYEG. Consists of two halves formed by TMs 1-5 and 6-10. These two domains form a lateral gate at the front which open onto the bilayer between TMs 2 and 7, and are clamped together by SecE at the back. The channel is closed by both a pore ring composed of hydrophobic SecY resides and a short helix (helix 2A) on the extracellular side of the membrane which forms a plug. The plug probably moves laterally to allow the channel to open. The ring and the pore may move independently. ","uniprotId":"Q0P7U3_CAMJE"},"10642466":{"id":10642466,"externalId":"192222.Cj1689c","name":"rp10","hasFamily":true,"annotation":"Binds to the 23S rRNA","uniprotId":"Q0P7U2_CAMJE"},"10642467":{"id":10642467,"externalId":"192222.Cj1690c","name":"rpsE","hasFamily":true,"annotation":"With S4 and S12 plays an important role in translational accuracy","uniprotId":"RS5_CAMJE"},"10642468":{"id":10642468,"externalId":"192222.Cj1691c","name":"rplR","hasFamily":true,"annotation":"This is one of the proteins that binds and probably mediates the attachment of the 5S RNA into the large ribosomal subunit, where it forms part of the central protuberance","uniprotId":"RL18_CAMJE"},"10642469":{"id":10642469,"externalId":"192222.Cj1692c","name":"rplF","hasFamily":true,"annotation":"This protein binds to the 23S rRNA, and is important in its secondary structure. It is located near the subunit interface in the base of the L7/L12 stalk, and near the tRNA binding site of the peptidyltransferase center","uniprotId":"RL6_CAMJE"},"10642470":{"id":10642470,"externalId":"192222.Cj1693c","name":"rpsH","hasFamily":true,"annotation":"One of the primary rRNA binding proteins, it binds directly to 16S rRNA central domain where it helps coordinate assembly of the platform of the 30S subunit","uniprotId":"RS8_CAMJE"},"10642471":{"id":10642471,"externalId":"192222.Cj1694c","name":"rpsN","hasFamily":true,"annotation":"Binds 16S rRNA, required for the assembly of 30S particles and may also be responsible for determining the conformation of the 16S rRNA at the A site","uniprotId":"RS14Z_CAMJE"},"10642472":{"id":10642472,"externalId":"192222.Cj1695c","name":"rplE","hasFamily":true,"annotation":"This is 1 of the proteins that binds and probably mediates the attachment of the 5S RNA into the large ribosomal subunit, where it forms part of the central protuberance. In the 70S ribosome it contacts protein S13 of the 30S subunit (bridge B1b), connecting the 2 subunits; this bridge is implicated in subunit movement. Contacts the P site tRNA; the 5S rRNA and some of its associated proteins might help stabilize positioning of ribosome-bound tRNAs","uniprotId":"RL5_CAMJE"},"10642473":{"id":10642473,"externalId":"192222.Cj1696c","name":"rplX","hasFamily":true,"annotation":"One of two assembly initiator proteins, it binds directly to the 5'-end of the 23S rRNA, where it nucleates assembly of the 50S subunit","uniprotId":"RL24_CAMJE"},"10642474":{"id":10642474,"externalId":"192222.Cj1697c","name":"rplN","hasFamily":true,"annotation":"Binds to 23S rRNA. Forms part of two intersubunit bridges in the 70S ribosome","uniprotId":"RL14_CAMJE"},"10642475":{"id":10642475,"externalId":"192222.Cj1698c","name":"rpsQ","hasFamily":true,"annotation":"One of the primary rRNA binding proteins, it binds specifically to the 5'-end of 16S ribosomal RNA","uniprotId":"RS17_CAMJE"},"10642476":{"id":10642476,"externalId":"192222.Cj1699c","name":"rpmC","hasFamily":true,"annotation":"Belongs to the universal ribosomal protein uL29 family. ","uniprotId":"RL29_CAMJE"},"10642477":{"id":10642477,"externalId":"192222.Cj1700c","name":"rplP","hasFamily":true,"annotation":"Binds 23S rRNA and is also seen to make contacts with the A and possibly P site tRNAs","uniprotId":"RL16_CAMJE"},"10642478":{"id":10642478,"externalId":"192222.Cj1701c","name":"rpsC","hasFamily":true,"annotation":"Binds the lower part of the 30S subunit head. Binds mRNA in the 70S ribosome, positioning it for translation","uniprotId":"RS3_CAMJE"},"10642479":{"id":10642479,"externalId":"192222.Cj1702c","name":"rplV","hasFamily":true,"annotation":"This protein binds specifically to 23S rRNA; its binding is stimulated by other ribosomal proteins, e.g. L4, L17, and L20. It is important during the early stages of 50S assembly. It makes multiple contacts with different domains of the 23S rRNA in the assembled 50S subunit and ribosome (By similarity)","uniprotId":"RL22_CAMJE"},"10642480":{"id":10642480,"externalId":"192222.Cj1703c","name":"rpsS","hasFamily":true,"annotation":"Protein S19 forms a complex with S13 that binds strongly to the 16S ribosomal RNA","uniprotId":"RS19_CAMJE"},"10642481":{"id":10642481,"externalId":"192222.Cj1704c","name":"rplB","hasFamily":true,"annotation":"One of the primary rRNA binding proteins. Required for association of the 30S and 50S subunits to form the 70S ribosome, for tRNA binding and peptide bond formation. It has been suggested to have peptidyltransferase activity; this is somewhat controversial. Makes several contacts with the 16S rRNA in the 70S ribosome. ","uniprotId":"RL2_CAMJE"},"10642482":{"id":10642482,"externalId":"192222.Cj1705c","name":"rplW","hasFamily":true,"annotation":"One of the early assembly proteins it binds 23S rRNA. One of the proteins that surrounds the polypeptide exit tunnel on the outside of the ribosome. Forms the main docking site for trigger factor binding to the ribosome","uniprotId":"RL23_CAMJE"},"10642483":{"id":10642483,"externalId":"192222.Cj1706c","name":"rplD","hasFamily":true,"annotation":"One of the primary rRNA binding proteins, this protein initially binds near the 5'-end of the 23S rRNA. It is important during the early stages of 50S assembly. It makes multiple contacts with different domains of the 23S rRNA in the assembled 50S subunit and ribosome (By similarity)","uniprotId":"RL4_CAMJE"},"10642484":{"id":10642484,"externalId":"192222.Cj1707c","name":"rplC","hasFamily":true,"annotation":"One of the primary rRNA binding proteins, it binds directly near the 3'-end of the 23S rRNA, where it nucleates assembly of the 50S subunit","uniprotId":"RL3_CAMJE"},"10642485":{"id":10642485,"externalId":"192222.Cj1708c","name":"rpsJ","hasFamily":true,"annotation":"Involved in the binding of tRNA to the ribosomes. ","uniprotId":"RS10_CAMJE"},"10642486":{"id":10642486,"externalId":"192222.Cj1709c","name":"Cj1709c","hasFamily":true,"annotation":"Belongs to the pseudouridine synthase RsuA family. ","uniprotId":"Q0P7S2_CAMJE"},"10642487":{"id":10642487,"externalId":"192222.Cj1710c","name":"rnj","hasFamily":true,"annotation":"An RNase that has 5'-3' exonuclease and possibly endonuclease activity. Involved in maturation of rRNA and in some organisms also mRNA maturation and/or decay","uniprotId":"Q0P7S1_CAMJE"},"10642488":{"id":10642488,"externalId":"192222.Cj1711c","name":"ksgA","hasFamily":true,"annotation":"Specifically dimethylates two adjacent adenosines (A1518 and A1519) in the loop of a conserved hairpin near the 3'-end of 16S rRNA in the 30S particle. May play a critical role in biogenesis of 30S subunits","uniprotId":"RSMA_CAMJE"},"10642489":{"id":10642489,"externalId":"192222.Cj1712","name":"Cj1712","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1712, unknown, len: 178 aa; 40.9% identity to HP1530 (called purine nucleoside phosphorylase (punB). Functional classification -Conserved hypothetical proteins","uniprotId":"Q0P7R9_CAMJE"},"10642490":{"id":10642490,"externalId":"192222.Cj1713","name":"rlmN","hasFamily":true,"annotation":"Specifically methylates position 2 of adenine 2503 in 23S rRNA and position 2 of adenine 37 in tRNAs. m2A2503 modification seems to play a crucial role in the proofreading step occurring at the peptidyl transferase center and thus would serve to optimize ribosomal fidelity","uniprotId":"RLMN_CAMJE"},"10642491":{"id":10642491,"externalId":"192222.Cj1714","name":"Cj1714","hasFamily":false,"annotation":"Original (2000) note: Cj1714, small hydrophobic protein, len: 32 aa; no Hp match; Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Functional classification -Unknown","uniprotId":"Q0P7R7_CAMJE"},"10642492":{"id":10642492,"externalId":"192222.Cj1715","name":"Cj1715","hasFamily":true,"annotation":"Putative acetyltransferase; Original (2000) note: Cj1715, possible acetyltransferase, len: 176 aa; similar to several plasmid-or integron-encoded genes e.g. STA_ECOLI streptothricin acetyltransferase (174 aa), fasta scores; opt: 149 z-score: 192.6 E(): 0.0021, 21.7% identity in 161 aa overlap, and TR:P75025 (EMBL:P75025) plasmid PVS1 hypothetical protein (166 aa), fasta scores; opt: 201 z-score: 255.1 E(): 6.9e-07, 26.1% identity in 161 aa overlap. Also simlilar to TR:O31513 (EMBL:Z99107) Bacillus subtilis chromosomal YESJ protein (180 aa), fasta scores; opt: 157 z-score: 202.0 E(): 0. [...] ","uniprotId":"Q0P7R6_CAMJE"},"10642493":{"id":10642493,"externalId":"192222.Cj1716c","name":"leuD","hasFamily":true,"annotation":"Catalyzes the isomerization between 2-isopropylmalate and 3- isopropylmalate, via the formation of 2-isopropylmaleate. ","uniprotId":"LEUD_CAMJE"},"10642494":{"id":10642494,"externalId":"192222.Cj1717c","name":"leuC","hasFamily":true,"annotation":"Catalyzes the isomerization between 2-isopropylmalate and 3- isopropylmalate, via the formation of 2-isopropylmaleate. ","uniprotId":"LEUC_CAMJE"},"10642495":{"id":10642495,"externalId":"192222.Cj1718c","name":"leuB","hasFamily":true,"annotation":"Catalyzes the oxidation of 3-carboxy-2-hydroxy-4- methylpentanoate (3-isopropylmalate) to 3-carboxy-4-methyl-2- oxopentanoate. The product decarboxylates to 4-methyl-2 oxopentanoate","uniprotId":"LEU3_CAMJE"},"10642496":{"id":10642496,"externalId":"192222.Cj1719c","name":"leuA","hasFamily":true,"annotation":"Catalyzes the condensation of the acetyl group of acetyl-CoA with 3-methyl-2-oxobutanoate (2-oxoisovalerate) to form 3-carboxy-3- hydroxy-4-methylpentanoate (2-isopropylmalate)","uniprotId":"LEU1_CAMJE"},"10642497":{"id":10642497,"externalId":"192222.Cj1720","name":"Cj1720","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1720, unknown, len: 209 aa; no Hp match. Functional classification - Unknown","uniprotId":"Q0P7R1_CAMJE"},"10642498":{"id":10642498,"externalId":"192222.Cj1721c","name":"Cj1721c","hasFamily":true,"annotation":"Original (2000) note: Cj1721c, possible outer membrane protein, len: 214 aa; very weak similarity to several e.g. TR:P95343 (EMBL:U52069) Neisseria gonorrhoeae outer membrane protein precursor (174 aa), blastp scores; E= 0.0043, 29% identity in 176 aa overlap. Contains probable N-terminal signal sequence. 22.3% identity to HP0706 (outer membrane protein (omp15)). Functional classification - Membranes, lipoproteins and porins","uniprotId":"Q0P7R0_CAMJE"},"10642499":{"id":10642499,"externalId":"192222.Cj1722c","name":"Cj1722c","hasFamily":true,"annotation":"Uncharacterized protein; Original (2000) note: Cj1722c, unknown, len: 33 aa; no Hp match. Similar to the N-termini of predicted periplasmic proteins Cj1626c (53.1% identity in 32 aa overlap), and Cj1004 (46.2% identity in 26 aa overlap). Functional classification - Unknown","uniprotId":"Q0P7Q9_CAMJE"},"10642500":{"id":10642500,"externalId":"192222.Cj1723c","name":"Cj1723c","hasFamily":false,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1723c, probable periplasmic protein, len: 74 aa; no Hp match. Contains probable N-terminal signal sequence. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P7Q8_CAMJE"},"10642501":{"id":10642501,"externalId":"192222.Cj1724c","name":"queF","hasFamily":true,"annotation":"Catalyzes the NADPH-dependent reduction of 7-cyano-7- deazaguanine (preQ0) to 7-aminomethyl-7-deazaguanine (preQ1). ","uniprotId":"QUEF_CAMJE"},"10642502":{"id":10642502,"externalId":"192222.Cj1725","name":"Cj1725","hasFamily":true,"annotation":"Putative periplasmic protein; Original (2000) note: Cj1725, probable periplasmic protein, len: 199 aa; similar to TR:O66724 (EMBL:AE000688) Aquifex aeolicus AQ_407 (203 aa), fasta scores; opt: 411 z-score: 483.0 E(): 1.4e-19, 34.0% identity in 203 aa overlap. No Hp match. Functional classification -Miscellaneous periplasmic proteins","uniprotId":"Q0P7Q6_CAMJE"},"10642503":{"id":10642503,"externalId":"192222.Cj1726c","name":"metA","hasFamily":true,"annotation":"Transfers an acetyl group from acetyl-CoA to L-homoserine, forming acetyl-L-homoserine","uniprotId":"METAA_CAMJE"},"10642504":{"id":10642504,"externalId":"192222.Cj1727c","name":"metB","hasFamily":true,"annotation":"Original (2000) note: Cj1727c, metY, possible O-acetylhomoserine (thiol)-lyase, len: 423 aa; TR:P94890 (EMBL:Y10744) Leptospira meyeri O-acetylhomoserine sulfhydrylase (metY) (442 aa), fasta scores; opt: 1198 z-score: 1372.4 E(): 0, 43.0% identity in 433 aa overlap,and CYSD_EMENI O-acetylhomoserine (thiol)-lyase (EC 4.2.99.10) (437 aa), fasta scores; opt: 1001 z-score: 1147.5 E(): 0, 39.4% identity in 419 aa overlap. No Hp ortholog. Also similar to Cj1393 metC (30.0% identity in 363 aa overlap). Contains Pfam match to entry PF01053 Cys_Met_Meta_PP, Gamma-family of PLP-dependent enzymes [...] ","uniprotId":"Q0P7Q4_CAMJE"},"10642505":{"id":10642505,"externalId":"192222.Cj1728c","name":"Cj1728c","hasFamily":false,"annotation":"Original (2000) note: Cj1728c, small hydrophobic protein, len: 51 aa; no Hp match. Some similarity to Cj1203c (33.3% identity in 42 aa overlap); Updated (2006) note: One probable transmembrane helix predicted by TMHMM2.0. Functional classification -Unknown","uniprotId":"Q0P7Q3_CAMJE"},"10642506":{"id":10642506,"externalId":"192222.Cj1729c","name":"flgE2","hasFamily":true,"annotation":"Flagellar hook protein FlgE; Original (2000) note: Cj1729c, flgE2, probable flagellar hook subunit protein, len: 865 aa; highly similar to e.g. TR:O86144 (EMBL:AJ002074) C. jejuni flagellar hook subunit protein (862 aa), fasta scores; opt: 4848 z-score: 4883.1 E(): 0, 86.4% identity in 869 aa overlap. 35.5% identity to HP0870. Also similar to Cj0043 flgE (25.4% identity in 488 aa overlap). Contains Pfam match to entry PF00460 flg_bb_rod, Flagella basal body rod proteins, and PS00017 ATP/GTP-binding site motif A (P-loop); Updated (2006) note: Characterised in Campylobacter jejuni. Appro [...] ","uniprotId":"Q0P7Q2_CAMJE"},"10642507":{"id":10642507,"externalId":"192222.Cj1731c","name":"ruvC","hasFamily":true,"annotation":"Nuclease that resolves Holliday junction intermediates in genetic recombination. Cleaves the cruciform structure in supercoiled DNA by nicking to strands with the same polarity at sites symmetrically opposed at the junction in the homologous arms and leaves a 5'-terminal phosphate and a 3'-terminal hydroxyl group","uniprotId":"RUVC_CAMJE"}};
module.exports = proteins;