//FILE GENERATED BY build.js on Thu Sep 01 2016 17:21:45 GMT+0200 (CEST), DO NOT MODIFY!
const proteins = {"6880885":{"id":6880885,"externalId":"511145.b0001","name":"thrL","hasFamily":false,"annotation":"thr operon leader peptide; This protein is involved in control of the biosynthesis of threonine","uniprotId":"LPT_ECOLI"},"6880886":{"id":6880886,"externalId":"511145.b0002","name":"thrA","hasFamily":true,"annotation":"fused aspartokinase I and homoserine dehydrogenase I","uniprotId":"AK1H_ECOLI"},"6880887":{"id":6880887,"externalId":"511145.b0003","name":"thrB","hasFamily":true,"annotation":"homoserine kinase; Catalyzes the ATP-dependent phosphorylation of L- homoserine to L-homoserine phosphate. Is also able to phosphorylate the hydroxy group on gamma-carbon of L-homoserine analogs when the functional group at the alpha-position is a carboxyl, an ester, or even a hydroxymethyl group. Neither L- threonine nor L-serine are substrates of the enzyme","uniprotId":"KHSE_ECOLI"},"6880888":{"id":6880888,"externalId":"511145.b0004","name":"thrC","hasFamily":true,"annotation":"threonine synthase; Catalyzes the gamma-elimination of phosphate from L- phosphohomoserine and the beta-addition of water to produce L- threonine. To a lesser extent, is able to slowly catalyze the deamination of L-threonine into alpha-ketobutyrate and that of L- serine and 3-chloroalanine into pyruvate. Is also able to rapidly convert vinylglycine to threonine, which proves that the pyridoxal p-quinonoid of vinylglycine is an intermediate in the TS reaction","uniprotId":"THRC_ECOLI"},"6880889":{"id":6880889,"externalId":"511145.b0005","name":"yaaX","hasFamily":false,"annotation":"predicted protein","uniprotId":"YAAX_ECOLI"},"6880890":{"id":6880890,"externalId":"511145.b0006","name":"yaaA","hasFamily":true,"annotation":"Peroxide resistance protein, lowers intracellular iron","uniprotId":"YAAA_ECOLI"},"6880891":{"id":6880891,"externalId":"511145.b0007","name":"yaaJ","hasFamily":true,"annotation":"putative transporter","uniprotId":"YAAJ_ECOLI"},"6880892":{"id":6880892,"externalId":"511145.b0008","name":"talB","hasFamily":true,"annotation":"transaldolase B; Transaldolase is important for the balance of metabolites in the pentose-phosphate pathway (By similarity)","uniprotId":"TALB_ECOLI"},"6880893":{"id":6880893,"externalId":"511145.b0009","name":"mog","hasFamily":true,"annotation":"molybdochelatase incorporating molybdenum into molybdopterin; Catalyzes the adenylation of molybdopterin as part of the biosynthesis of the molybdenum-cofactor","uniprotId":"MOG_ECOLI"},"6880894":{"id":6880894,"externalId":"511145.b0010","name":"yaaH","hasFamily":true,"annotation":"inner membrane protein, Grp1_Fun34_YaaH family","uniprotId":"YAAH_ECOLI"},"6880895":{"id":6880895,"externalId":"511145.b0011","name":"yaaW","hasFamily":true,"annotation":"conserved protein, UPF0174 family","uniprotId":"YAAW_ECOLI"},"6880896":{"id":6880896,"externalId":"511145.b0013","name":"yaaI","hasFamily":true,"annotation":"conserved protein, UPF0412 family","uniprotId":"YAAI_ECOLI"},"6880897":{"id":6880897,"externalId":"511145.b0014","name":"dnaK","hasFamily":true,"annotation":"chaperone Hsp70, co-chaperone with DnaJ; Plays an essential role in the initiation of phage lambda DNA replication, where it acts in an ATP-dependent fashion with the DnaJ protein to release lambda O and P proteins from the preprimosomal complex. DnaK is also involved in chromosomal DNA replication, possibly through an analogous interaction with the DnaA protein. Also participates actively in the response to hyperosmotic shock","uniprotId":"DNAK_ECOLI"},"6880898":{"id":6880898,"externalId":"511145.b0015","name":"dnaJ","hasFamily":true,"annotation":"chaperone Hsp40, co-chaperone with DnaK; Participates actively in the response to hyperosmotic and heat shock by preventing the aggregation of stress-denatured proteins and by disaggregating proteins, also in an autonomous, DnaK-independent fashion. Unfolded proteins bind initially to DnaJ; upon interaction with the DnaJ-bound protein, DnaK hydrolyzes its bound ATP, resulting in the formation of a stable complex. GrpE releases ADP from DnaK; ATP binding to DnaK triggers the release of the substrate protein, thus completing the reaction cycle. Several rounds of ATP-dependent interaction [...] ","uniprotId":"DNAJ_ECOLI"},"6880899":{"id":6880899,"externalId":"511145.b0016","name":"b0016","hasFamily":false,"annotation":"IS186 transposase"},"6880900":{"id":6880900,"externalId":"511145.b0018","name":"gef","hasFamily":false,"annotation":"toxic membrane protein, small; When overexpressed, kills the cells from the inside by interfering with a vital function in the cell membrane","uniprotId":"HOKC_ECOLI"},"6880901":{"id":6880901,"externalId":"511145.b0019","name":"nhaA","hasFamily":true,"annotation":"sodium-proton antiporter; Na(+)/H(+) antiporter that extrudes sodium in exchange for external protons. Catalyzes the exchange of 2 H(+) per Na(+). Can mediate sodium uptake when a transmembrane pH gradient is applied. Active at alkaline pH. Activity is strongly down- regulated below pH 6.5","uniprotId":"NHAA_ECOLI"},"6880902":{"id":6880902,"externalId":"511145.b0020","name":"nhaR","hasFamily":true,"annotation":"DNA-binding transcriptional activator; Plays a role in the positive regulation of NhaA","uniprotId":"NHAR_ECOLI"},"6880903":{"id":6880903,"externalId":"511145.b0021","name":"b0021","hasFamily":true,"annotation":"IS1 transposase B"},"6880904":{"id":6880904,"externalId":"511145.b0022","name":"b0022","hasFamily":false,"annotation":"IS1 repressor TnpA"},"6880905":{"id":6880905,"externalId":"511145.b0023","name":"rpsT","hasFamily":true,"annotation":"30S ribosomal subunit protein S20; Binds directly to 16S ribosomal RNA (By similarity)","uniprotId":"RS20_ECOLI"},"6880906":{"id":6880906,"externalId":"511145.b0024","name":"yaaY","hasFamily":true,"annotation":"predicted protein","uniprotId":"YAAY_ECOLI"},"6880907":{"id":6880907,"externalId":"511145.b0025","name":"ribF","hasFamily":true,"annotation":"bifunctional riboflavin kinase/FAD synthetase","uniprotId":"RIBF_ECOLI"},"6880908":{"id":6880908,"externalId":"511145.b0026","name":"ileS","hasFamily":true,"annotation":"isoleucyl-tRNA synthetase; Catalyzes the attachment of isoleucine to tRNA(Ile). As IleRS can inadvertently accommodate and process structurally similar amino acids such as valine, to avoid such errors it has two additional distinct tRNA(Ile)-dependent editing activities. One activity is designated as 'pretransfer' editing and involves the hydrolysis of activated Val-AMP. The other activity is designated 'posttransfer' editing and involves deacylation of mischarged Val-tRNA(Ile) (By similarity)","uniprotId":"SYI_ECOLI"},"6880909":{"id":6880909,"externalId":"511145.b0027","name":"lspA","hasFamily":true,"annotation":"prolipoprotein signal peptidase (signal peptidase II); This protein specifically catalyzes the removal of signal peptides from prolipoproteins (By similarity)","uniprotId":"LSPA_ECOLI"},"6880910":{"id":6880910,"externalId":"511145.b0028","name":"fkpB","hasFamily":true,"annotation":"FKBP-type peptidyl-prolyl cis-trans isomerase (rotamase); PPIases accelerate the folding of proteins. Substrate specificity investigated with 'Suc-Ala-Xaa-Pro-Phe-4-nitroanilide' where Xaa is the amino acid tested, was found to be Phe > Leu >> Ile > Lys = Ala > Trp > His >> Gln","uniprotId":"FKBX_ECOLI"},"6880911":{"id":6880911,"externalId":"511145.b0029","name":"ispH","hasFamily":true,"annotation":"4-hydroxy-3-methylbut-2-enyl diphosphate reductase, 4Fe-4S protein; Converts 1-hydroxy-2-methyl-2-(E)-butenyl 4-diphosphate into isopentenyl diphosphate (IPP) and dimethylallyl diphosphate (DMAPP). Is also involved in penicillin tolerance and control of the stringent response. Seems to directly or indirectly interact with RelA to maintain it in an inactive form during normal growth","uniprotId":"ISPH_ECOLI"},"6880912":{"id":6880912,"externalId":"511145.b0030","name":"rihC","hasFamily":true,"annotation":"ribonucleoside hydrolase 3; Hydrolyzes both purine and pyrimidine ribonucleosides with a broad-substrate specificity with decreasing activity in the order uridine, xanthosine, inosine, adenosine, cytidine, guanosine","uniprotId":"RIHC_ECOLI"},"6880913":{"id":6880913,"externalId":"511145.b0031","name":"dapB","hasFamily":true,"annotation":"dihydrodipicolinate reductase; Catalyzes the conversion of 4-hydroxy- tetrahydrodipicolinate (HTPA) to tetrahydrodipicolinate. Can use both NADH and NADPH as a reductant, with NADH being twice as effective as NADPH","uniprotId":"DAPB_ECOLI"},"6880914":{"id":6880914,"externalId":"511145.b0032","name":"carA","hasFamily":true,"annotation":"carbamoyl phosphate synthetase small subunit, glutamine amidotransferase","uniprotId":"CARA_ECOLI"},"6880915":{"id":6880915,"externalId":"511145.b0033","name":"carB","hasFamily":true,"annotation":"carbamoyl-phosphate synthase large subunit","uniprotId":"CARB_ECOLI"},"6880916":{"id":6880916,"externalId":"511145.b0034","name":"caiF","hasFamily":true,"annotation":"DNA-binding transcriptional activator; Potential transcriptional activator of carnitine metabolism","uniprotId":"CAIF_ECOLI"},"6880917":{"id":6880917,"externalId":"511145.b0035","name":"caiE","hasFamily":true,"annotation":"stimulator of CaiD and CaiB enzyme activities; Overproduction of CaiE stimulates the activity of CaiB and CaiD (By similarity)","uniprotId":"CAIE_ECOLI"},"6880918":{"id":6880918,"externalId":"511145.b0036","name":"caiD","hasFamily":true,"annotation":"carnitinyl-CoA dehydratase; Catalyzes the reversible dehydration of L-carnitinyl-CoA to crotonobetainyl-CoA (By similarity)","uniprotId":"CAID_ECOLI"},"6880919":{"id":6880919,"externalId":"511145.b0037","name":"caiC","hasFamily":true,"annotation":"probable crotonobetaine/carnitine-CoA ligase; Could catalyze the transfer of CoA to carnitine, generating the initial carnitinyl-CoA needed for the CaiB reaction cycle","uniprotId":"CAIC_ECOLI"},"6880920":{"id":6880920,"externalId":"511145.b0038","name":"caiB","hasFamily":true,"annotation":"crotonobetainyl CoA:carnitine CoA transferase; Catalyzes the reversible transfer of the CoA moiety from gamma-butyrobetainyl-CoA to L-carnitine to generate L-carnitinyl- CoA and gamma-butyrobetaine. Is also able to catalyze the reversible transfer of the CoA moiety from gamma-butyrobetainyl- CoA or L-carnitinyl-CoA to crotonobetaine to generate crotonobetainyl-CoA (By similarity)","uniprotId":"CAIB_ECOLI"},"6880921":{"id":6880921,"externalId":"511145.b0039","name":"caiA","hasFamily":true,"annotation":"crotonobetaine reductase subunit II, FAD-binding; Catalyzes the reduction of crotonobetainyl-CoA to gamma- butyrobetainyl-CoA (By similarity)","uniprotId":"CAIA_ECOLI"},"6880922":{"id":6880922,"externalId":"511145.b0040","name":"caiT","hasFamily":true,"annotation":"putative transporter; Catalyzes the exchange of L-carnitine for gamma- butyrobetaine and related betaines (By similarity)","uniprotId":"CAIT_ECOLI"},"6880923":{"id":6880923,"externalId":"511145.b0041","name":"fixA","hasFamily":true,"annotation":"putative electron transfer flavoprotein subunit, required for anaerobic carnitine reduction; Required for anaerobic carnitine reduction. May bring reductant to CaiA (By similarity)","uniprotId":"FIXA_ECOLI"},"6880924":{"id":6880924,"externalId":"511145.b0042","name":"fixB","hasFamily":true,"annotation":"putative electron transfer flavoprotein, NAD/FAD-binding domain and ETFP adenine nucleotide-binding domain-like protein; Required for anaerobic carnitine reduction. May bring reductant to CaiA (By similarity)","uniprotId":"FIXB_ECOLI"},"6880925":{"id":6880925,"externalId":"511145.b0043","name":"fixC","hasFamily":true,"annotation":"putative oxidoreductase with FAD/NAD(P)-binding domain; Could be part of an electron transfer system required for anaerobic carnitine reduction","uniprotId":"FIXC_ECOLI"},"6880926":{"id":6880926,"externalId":"511145.b0044","name":"fixX","hasFamily":true,"annotation":"putative 4Fe-4S ferredoxin-type protein; Could be part of an electron transfer system required for anaerobic carnitine reduction. Could be a 3Fe-4S cluster- containing protein","uniprotId":"FIXX_ECOLI"},"6880927":{"id":6880927,"externalId":"511145.b0045","name":"yaaU","hasFamily":true,"annotation":"putative transporter","uniprotId":"YAAU_ECOLI"},"6880928":{"id":6880928,"externalId":"511145.b0046","name":"kefF","hasFamily":true,"annotation":"potassium-efflux system ancillary protein for KefC, glutathione-regulated; quinone oxidoreductase, FMN-dependent; Required for full activity of KefC (By similarity)","uniprotId":"KEFF_ECOLI"},"6880929":{"id":6880929,"externalId":"511145.b0047","name":"kefC","hasFamily":true,"annotation":"potassium:proton antiporter; Transport system that facilitates potassium-efflux, possibly by potassium-proton antiport (By similarity)","uniprotId":"KEFC_ECOLI"},"6880930":{"id":6880930,"externalId":"511145.b0048","name":"folA","hasFamily":true,"annotation":"dihydrofolate reductase; Key enzyme in folate metabolism. Catalyzes an essential reaction for de novo glycine and purine synthesis, and for DNA precursor synthesis (By similarity)","uniprotId":"DYR_ECOLI"},"6880931":{"id":6880931,"externalId":"511145.b0049","name":"apaH","hasFamily":true,"annotation":"diadenosine tetraphosphatase; Hydrolyzes diadenosine 5',5'''-P1,P4-tetraphosphate to yield ADP (By similarity)","uniprotId":"APAH_ECOLI"},"6880932":{"id":6880932,"externalId":"511145.b0050","name":"apaG","hasFamily":true,"annotation":"protein associated with Co2+ and Mg2+ efflux","uniprotId":"APAG_ECOLI"},"6880933":{"id":6880933,"externalId":"511145.b0051","name":"ksgA","hasFamily":true,"annotation":"16S rRNA dimethyladenosine transferase, SAM-dependent; Specifically dimethylates two adjacent adenosines (A1518 and A1519) in the loop of a conserved hairpin near the 3'-end of 16S rRNA in the 30S particle. May play a critical role in biogenesis of 30S subunits. Has also a DNA glycosylase/AP lyase activity that removes C mispaired with oxidized T from DNA, and may play a role in protection of DNA against oxidative stress","uniprotId":"RSMA_ECOLI"},"6880934":{"id":6880934,"externalId":"511145.b0052","name":"pdxA","hasFamily":true,"annotation":"4-hydroxy-L-threonine phosphate dehydrogenase, NAD-dependent; Catalyzes the NAD(P)-dependent oxidation of 4- (phosphohydroxy)-L-threonine (HTP) into 2-amino-3-oxo-4- (phosphohydroxy)butyric acid which spontaneously decarboxylates to form 3-amino-2-oxopropyl phosphate (AHAP) (By similarity)","uniprotId":"PDXA_ECOLI"},"6880935":{"id":6880935,"externalId":"511145.b0053","name":"surA","hasFamily":true,"annotation":"peptidyl-prolyl cis-trans isomerase (PPIase); Chaperone involved in the correct folding and assembly of outer membrane proteins, such as OmpA, OmpF and LamB. Recognizes specific patterns of aromatic residues and the orientation of their side chains, which are found more frequently in integral outer membrane proteins. May act in both early periplasmic and late outer membrane-associated steps of protein maturation. Essential for the survival of E.coli in stationary phase. Required for pilus biogenesis","uniprotId":"SURA_ECOLI"},"6880936":{"id":6880936,"externalId":"511145.b0054","name":"imp","hasFamily":true,"annotation":"LPS assembly OM complex LptDE, beta-barrel component; Together with LptE, is involved in the assembly of LPS in the outer leaflet of the outer membrane. Determines N-hexane tolerance and is involved in outer membrane permeability. Essential for envelope biogenesis (By similarity)","uniprotId":"LPTD_ECOLI"},"6880937":{"id":6880937,"externalId":"511145.b0055","name":"djlA","hasFamily":true,"annotation":"DnaJ-like protein, membrane anchored; Regulatory DnaK co-chaperone. Direct interaction between DnaK and DjlA is needed for the induction of the wcaABCDE operon, involved in the synthesis of a colanic acid polysaccharide capsule, possibly through activation of the RcsB/RcsC phosphotransfer signaling pathway. The colanic acid capsule may help the bacterium survive conditions outside the host (By similarity)","uniprotId":"DJLA_ECOLI"},"6880938":{"id":6880938,"externalId":"511145.b0058","name":"rluA","hasFamily":true,"annotation":"pseudouridine synthase for 23S rRNA (position 746) and tRNAphe(position 32); Responsible for synthesis of pseudouridine from uracil- 746 in 23S ribosomal RNA and from uracil-32 in the anticodon stem and loop of transfer RNAs","uniprotId":"RLUA_ECOLI"},"6880939":{"id":6880939,"externalId":"511145.b0059","name":"hepA","hasFamily":true,"annotation":"RNA polymerase-associated helicase protein (ATPase and RNA polymerase recycling factor); Transcription regulator that activates transcription by stimulating RNA polymerase (RNAP) recycling in case of stress conditions such as supercoiled DNA or high salt concentrations. Probably acts by releasing the RNAP, when it is trapped or immobilized on tightly supercoiled DNA. Does not activate transcription on linear DNA. Probably not involved in DNA repair (By similarity)","uniprotId":"RAPA_ECOLI"},"6880940":{"id":6880940,"externalId":"511145.b0060","name":"polB","hasFamily":true,"annotation":"DNA polymerase II; Thought to be involved in DNA repair and/or mutagenesis","uniprotId":"DPO2_ECOLI"},"6880941":{"id":6880941,"externalId":"511145.b0061","name":"araD","hasFamily":true,"annotation":"L-ribulose-5-phosphate 4-epimerase","uniprotId":"ARAD_ECOLI"},"6880942":{"id":6880942,"externalId":"511145.b0062","name":"araA","hasFamily":true,"annotation":"L-arabinose isomerase; Catalyzes the conversion of L-arabinose to L-ribulose (By similarity)","uniprotId":"ARAA_ECOLI"},"6880943":{"id":6880943,"externalId":"511145.b0063","name":"araB","hasFamily":true,"annotation":"L-ribulokinase","uniprotId":"ARAB_ECOLI"},"6880944":{"id":6880944,"externalId":"511145.b0064","name":"araC","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; This protein controls the expression of at least six genes that are involved in the transport and catabolism of L- arabinose. It regulates initiation of transcription of the araBAD operon and it also controls its own synthesis. The L-arabinose operon displays both positive and negative regulation through AraC","uniprotId":"ARAC_ECOLI"},"6880945":{"id":6880945,"externalId":"511145.b0065","name":"yabI","hasFamily":true,"annotation":"inner membrane protein, SNARE_assoc family","uniprotId":"YABI_ECOLI"},"6880946":{"id":6880946,"externalId":"511145.b0066","name":"thiQ","hasFamily":true,"annotation":"thiamin transporter subunit; Part of the ABC transporter complex ThiBPQ involved in thiamine import. Responsible for energy coupling to the transport system (Probable)","uniprotId":"THIQ_ECOLI"},"6880947":{"id":6880947,"externalId":"511145.b0067","name":"thiP","hasFamily":true,"annotation":"fused thiamin transporter subunits of ABC superfamily: membrane components; Part of the ABC transporter complex ThiBPQ involved in thiamine import. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"THIP_ECOLI"},"6880948":{"id":6880948,"externalId":"511145.b0068","name":"tbpA","hasFamily":true,"annotation":"thiamin transporter subunit; Part of the ABC transporter complex ThiBPQ involved in thiamine import","uniprotId":"THIB_ECOLI"},"6880949":{"id":6880949,"externalId":"511145.b0069","name":"sgrR","hasFamily":true,"annotation":"transcriptional DNA-binding transcriptional activator of sgrS sRNA; Activates the small RNA gene sgrS under glucose- phosphate stress conditions as well as yfdZ. Represses its own transcription under both stress and non-stress conditions; this repression likely provides one measure of control over sgrR at the level of synthesis. Might act as a sensor of the intracellular accumulation of phosphoglucose by binding these molecules in its C-terminal solute-binding domain","uniprotId":"SGRR_ECOLI"},"6880950":{"id":6880950,"externalId":"511145.b0070","name":"setA","hasFamily":false,"annotation":"broad specificity sugar efflux system; Involved in the efflux of sugars. The physiological role may be the detoxification of non-metabolizable sugar analogs. Can transport IPTG, lactose and glucose. Has broad substrate specificity, with preferences for glucosides or galactosides with alkyl or aryl substituents","uniprotId":"SETA_ECOLI"},"6880951":{"id":6880951,"externalId":"511145.b0071","name":"leuD","hasFamily":true,"annotation":"3-isopropylmalate dehydratase small subunit; Catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate, via the formation of 2-isopropylmaleate (By similarity)","uniprotId":"LEUD_ECOLI"},"6880952":{"id":6880952,"externalId":"511145.b0072","name":"leuC","hasFamily":true,"annotation":"3-isopropylmalate dehydratase large subunit; Catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate, via the formation of 2-isopropylmaleate (By similarity)","uniprotId":"LEUC_ECOLI"},"6880953":{"id":6880953,"externalId":"511145.b0073","name":"leuB","hasFamily":true,"annotation":"3-isopropylmalate dehydrogenase, NAD(+)-dependent; Catalyzes the oxidation of 3-carboxy-2-hydroxy-4- methylpentanoate (3-isopropylmalate) to 3-carboxy-4-methyl-2- oxopentanoate. The product decarboxylates to 4-methyl-2 oxopentanoate (By similarity)","uniprotId":"LEU3_ECOLI"},"6880954":{"id":6880954,"externalId":"511145.b0074","name":"leuA","hasFamily":true,"annotation":"2-isopropylmalate synthase; Catalyzes the condensation of the acetyl group of acetyl-CoA with 3-methyl-2-oxobutanoate (2-oxoisovalerate) to form 3-carboxy-3-hydroxy-4-methylpentanoate (2-isopropylmalate) (By similarity)","uniprotId":"LEU1_ECOLI"},"6880955":{"id":6880955,"externalId":"511145.b0075","name":"leuL","hasFamily":false,"annotation":"leu operon leader peptide; Involved in control of the biosynthesis of leucine","uniprotId":"LPL_ECOLI"},"6880956":{"id":6880956,"externalId":"511145.b0076","name":"leuO","hasFamily":true,"annotation":"DNA-binding transcriptional activator; A global transcription factor. Activates transcription of the 9 following operons; yjjQ-bglJ, yjjP, acrEF, ybdO, yjcRQP, casABCDE12, rhsD-ybbC, fepE and gltF, in most cases it probably interferes with silencing by H-NS and activates transcription. Represses transcription of the 3 following operons; uxaCA, sdaCB and yjiY. H-NS repression of the bgl operon, leading to the ability to metabolize some beta-glucosides. It also directly activates the bgl operon. Activation is H-NS and BglJ-RcsB independent","uniprotId":"LEUO_ECOLI"},"6880957":{"id":6880957,"externalId":"511145.b0077","name":"ilvI","hasFamily":true,"annotation":"acetolactate synthase III, large subunit","uniprotId":"ILVI_ECOLI"},"6880958":{"id":6880958,"externalId":"511145.b0078","name":"ilvH","hasFamily":true,"annotation":"acetolactate synthase III, thiamin-dependent, small subunit","uniprotId":"ILVH_ECOLI"},"6880959":{"id":6880959,"externalId":"511145.b0080","name":"cra","hasFamily":true,"annotation":"DNA-binding transcriptional repressor-activator for carbon metabolism; Global transcriptional regulator, which plays an important role in the regulation of carbon metabolism. Activates transcription of genes encoding biosynthetic and oxidative enzymes (involved in Krebs cycle, glyoxylate shunt and gluconeogenesis, such as ppsA and fbp). Represses genes involved in sugar catabolism, such as fruB, pfkA, pykF and adhE. Binds asymmetrically to the two half-sites of its operator","uniprotId":"CRA_ECOLI"},"6880960":{"id":6880960,"externalId":"511145.b0081","name":"mraZ","hasFamily":true,"annotation":"RsmH methytransferase inhibitor","uniprotId":"MRAZ_ECOLI"},"6880961":{"id":6880961,"externalId":"511145.b0082","name":"mraW","hasFamily":true,"annotation":"16S rRNA m(4)C1402 methyltranserfase, SAM-dependent; Specifically methylates the N4 position of cytidine in position 1402 (C1402) of 16S rRNA. In vitro, active on the assembled 30S subunit, but not naked 16S rRNA or 70S ribosomes","uniprotId":"RSMH_ECOLI"},"6880962":{"id":6880962,"externalId":"511145.b0083","name":"ftsL","hasFamily":true,"annotation":"membrane bound cell division protein at septum containing leucine zipper motif; Essential cell division protein. May link together the upstream cell division proteins, which are predominantly cytoplasmic, with the downstream cell division proteins, which are predominantly periplasmic. Can also mediate Zn(2+)-sensitivity probably by increasing the permeability of the inner membrane","uniprotId":"FTSL_ECOLI"},"6880963":{"id":6880963,"externalId":"511145.b0084","name":"ftsI","hasFamily":true,"annotation":"transpeptidase involved in septal peptidoglycan synthesis (penicillin-binding protein 3); Cell wall formation. Essential for the formation of a septum of the murein sacculus. Synthesis of cross-linked peptidoglycan from the lipid intermediates","uniprotId":"FTSI_ECOLI"},"6880964":{"id":6880964,"externalId":"511145.b0085","name":"murE","hasFamily":true,"annotation":"UDP-N-acetylmuramoyl-L-alanyl-D-glutamate:meso- diaminopimelate ligase; Catalyzes the addition of meso-diaminopimelic acid to the nucleotide precursor UDP-N-acetylmuramoyl-L-alanyl-D-glutamate (UMAG) in the biosynthesis of bacterial cell-wall peptidoglycan. Is also able to use many meso-diaminopimelate analogs as substrates, although much less efficiently, but not L-lysine","uniprotId":"MURE_ECOLI"},"6880965":{"id":6880965,"externalId":"511145.b0086","name":"murF","hasFamily":true,"annotation":"UDP-N-acetylmuramoyl-tripeptide:D-alanyl-D- alanine ligase; Involved in cell wall formation. Catalyzes the final step in the synthesis of UDP-N-acetylmuramoyl-pentapeptide, the precursor of murein (By similarity)","uniprotId":"MURF_ECOLI"},"6880966":{"id":6880966,"externalId":"511145.b0087","name":"mraY","hasFamily":true,"annotation":"phospho-N-acetylmuramoyl-pentapeptide transferase; First step of the lipid cycle reactions in the biosynthesis of the cell wall peptidoglycan (By similarity)","uniprotId":"MRAY_ECOLI"},"6880967":{"id":6880967,"externalId":"511145.b0088","name":"murD","hasFamily":true,"annotation":"UDP-N-acetylmuramoyl-L-alanine:D-glutamate ligase; Cell wall formation. Catalyzes the addition of glutamate to the nucleotide precursor UDP-N-acetylmuramoyl-L-alanine (UMA) (By similarity)","uniprotId":"MURD_ECOLI"},"6880968":{"id":6880968,"externalId":"511145.b0089","name":"ftsW","hasFamily":true,"annotation":"Lipid II flippase; integral membrane protein involved in stabilizing FstZ ring during cell division; Essential cell division protein. Transports lipid-linked peptidoglycan precursors from the inner to the outer leaflet of the cytoplasmic membrane. Required for localization of FtsI. May play a role in the stabilization of the FtsZ ring during cell division","uniprotId":"FTSW_ECOLI"},"6880969":{"id":6880969,"externalId":"511145.b0090","name":"murG","hasFamily":true,"annotation":"N-acetylglucosaminyl transferase; Cell wall formation. Catalyzes the transfer of a GlcNAc subunit on undecaprenyl-pyrophosphoryl-MurNAc-pentapeptide (lipid intermediate I) to form undecaprenyl-pyrophosphoryl-MurNAc- (pentapeptide)GlcNAc (lipid intermediate II) (By similarity)","uniprotId":"MURG_ECOLI"},"6880970":{"id":6880970,"externalId":"511145.b0091","name":"murC","hasFamily":true,"annotation":"UDP-N-acetylmuramate:L-alanine ligase; Cell wall formation (By similarity)","uniprotId":"MURC_ECOLI"},"6880971":{"id":6880971,"externalId":"511145.b0092","name":"ddlB","hasFamily":true,"annotation":"D-alanine:D-alanine ligase; Cell wall formation (By similarity)","uniprotId":"DDLB_ECOLI"},"6880972":{"id":6880972,"externalId":"511145.b0093","name":"ftsQ","hasFamily":true,"annotation":"Divisome assembly protein, membrane anchored protein involved in growth of wall at septum; Essential cell division protein. May link together the upstream cell division proteins, which are predominantly cytoplasmic, with the downstream cell division proteins, which are predominantly periplasmic. May control correct divisome assembly (By similarity)","uniprotId":"FTSQ_ECOLI"},"6880973":{"id":6880973,"externalId":"511145.b0094","name":"ftsA","hasFamily":true,"annotation":"ATP-binding cell division protein involved in recruitment of FtsK to Z ring; This protein may be involved in anomalous filament growth (By similarity). May be a component of the septum","uniprotId":"FTSA_ECOLI"},"6880974":{"id":6880974,"externalId":"511145.b0095","name":"ftsZ","hasFamily":true,"annotation":"GTP-binding tubulin-like cell division protein; Essential cell division protein that forms a contractile ring structure (Z ring) at the future cell division site. The regulation of the ring assembly controls the timing and the location of cell division. One of the functions of the FtsZ ring is to recruit other cell division proteins to the septum to produce a new cell wall between the dividing cells. Binds GTP and shows GTPase activity. Polymerization and bundle formation is enhanced by CbeA","uniprotId":"FTSZ_ECOLI"},"6880975":{"id":6880975,"externalId":"511145.b0096","name":"lpxC","hasFamily":true,"annotation":"UDP-3-O-acyl N-acetylglucosamine deacetylase; The key enzyme in the biosynthesis of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell. Degraded by FtsH; when the activity of FtsH is reduced too much lipid A and not enough phospholipids are made (both pathways use the same precursor), which is lethal","uniprotId":"LPXC_ECOLI"},"6880976":{"id":6880976,"externalId":"511145.b0097","name":"secM","hasFamily":true,"annotation":"regulator of secA translation; Regulates secA expression by translational coupling of the secM secA operon. Ribosomes translating the C-terminal region of secM can disrupt an RNA repressor helix that normally blocks secA translation initiation, derepressing the expression of secA. Translational pausing of secM at Pro-166 under secretion-limiting conditions increases the duration of the disruption and thus increases secA expression. This is controlled by interaction of the secM signal peptide with secA and the translocon, possibly by secA pulling the paused secM out of the ribosome. The [...] ","uniprotId":"SECM_ECOLI"},"6880977":{"id":6880977,"externalId":"511145.b0098","name":"secA","hasFamily":true,"annotation":"preprotein translocase subunit, ATPase; Part of the Sec protein translocase complex. Interacts with the SecYEG preprotein conducting channel. Has a central role in coupling the hydrolysis of ATP to the transfer of proteins into and across the cell membrane, serving both as a receptor for the preprotein-SecB complex and as an ATP-driven molecular motor driving the stepwise translocation of polypeptide chains across the membrane (By similarity)","uniprotId":"SECA_ECOLI"},"6880978":{"id":6880978,"externalId":"511145.b0099","name":"mutT","hasFamily":true,"annotation":"nucleoside triphosphate pyrophosphohydrolase, marked preference for dGTP; Involved in the GO system responsible for removing an oxidatively damaged form of guanine (7,8-dihydro-8-oxoguanine) from DNA and the nucleotide pool. 8-oxo-dGTP is inserted opposite dA and dC residues of template DNA with almost equal efficiency thus leading to A.T to G.C transversions. MutT specifically degrades 8-oxo-dGTP to the monophosphate","uniprotId":"MUTT_ECOLI"},"6880979":{"id":6880979,"externalId":"511145.b0101","name":"yacG","hasFamily":true,"annotation":"DNA gyrase inhibitor; Inhibits all the catalytic activities of DNA gyrase by preventing its interaction with DNA. Acts by binding directly to the C-terminal domain of GyrB, which probably disrupts DNA binding by the gyrase (By similarity)","uniprotId":"YACG_ECOLI"},"6880980":{"id":6880980,"externalId":"511145.b0102","name":"yacF","hasFamily":true,"annotation":"FtsZ stabilizer; Cell division factor that enhances FtsZ-ring assembly. Directly interacts with FtsZ and promotes bundling of FtsZ protofilaments, with a reduction in FtsZ GTPase activity (By similarity)","uniprotId":"ZAPD_ECOLI"},"6880981":{"id":6880981,"externalId":"511145.b0103","name":"coaE","hasFamily":true,"annotation":"dephospho-CoA kinase; Catalyzes the phosphorylation of the 3'-hydroxyl group of dephosphocoenzyme A to form coenzyme A (By similarity)","uniprotId":"COAE_ECOLI"},"6880982":{"id":6880982,"externalId":"511145.b0104","name":"guaC","hasFamily":true,"annotation":"GMP reductase; Catalyzes the irreversible NADPH-dependent deamination of GMP to IMP. It functions in the conversion of nucleobase, nucleoside and nucleotide derivatives of G to A nucleotides, and in maintaining the intracellular balance of A and G nucleotides (By similarity)","uniprotId":"GUAC_ECOLI"},"6880983":{"id":6880983,"externalId":"511145.b0106","name":"hofC","hasFamily":true,"annotation":"assembly protein in type IV pilin biogenesis, transmembrane protein","uniprotId":"HOFC_ECOLI"},"6880984":{"id":6880984,"externalId":"511145.b0107","name":"hofB","hasFamily":true,"annotation":"conserved protein with nucleoside triphosphate hydrolase domain","uniprotId":"HOFB_ECOLI"},"6880985":{"id":6880985,"externalId":"511145.b0108","name":"ppdD","hasFamily":true,"annotation":"putative major pilin subunit; Not yet known","uniprotId":"PPDD_ECOLI"},"6880986":{"id":6880986,"externalId":"511145.b0109","name":"nadC","hasFamily":true,"annotation":"quinolinate phosphoribosyltransferase; Involved in the catabolism of quinolinic acid (QA) (By similarity)","uniprotId":"NADC_ECOLI"},"6880987":{"id":6880987,"externalId":"511145.b0110","name":"ampD","hasFamily":true,"annotation":"1,6-anhydro-N-acetylmuramyl-L-alanine amidase, Zn-dependent; murein amidase; Involved in both cell wall peptidoglycans recycling and beta-lactamase induction. Specifically cleaves the amide bond between the lactyl group of N-acetylmuramic acid and the alpha- amino group of the L-alanine in degradation products containing an anhydro N-acetylmuramyl moiety (By similarity)","uniprotId":"AMPD_ECOLI"},"6880988":{"id":6880988,"externalId":"511145.b0111","name":"ampE","hasFamily":true,"annotation":"putative inner membrane protein; Putative signaling protein in beta-lactamase regulation. AmpE seems not to act as a direct sensor for beta-lactams","uniprotId":"AMPE_ECOLI"},"6880989":{"id":6880989,"externalId":"511145.b0112","name":"aroP","hasFamily":true,"annotation":"aromatic amino acid transporter; Permease that is involved in the transport across the cytoplasmic membrane of the aromatic amino acids (phenylalanine, tyrosine, and tryptophan)","uniprotId":"AROP_ECOLI"},"6880990":{"id":6880990,"externalId":"511145.b0113","name":"pdhR","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; Transcriptional repressor for the pyruvate dehydrogenase complex genes aceEF and lpd","uniprotId":"PDHR_ECOLI"},"6880991":{"id":6880991,"externalId":"511145.b0114","name":"aceE","hasFamily":true,"annotation":"pyruvate dehydrogenase, decarboxylase component E1, thiamin-binding; Component of the pyruvate dehydrogenase (PDH) complex, that catalyzes the overall conversion of pyruvate to acetyl-CoA and CO(2) (By similarity)","uniprotId":"ODP1_ECOLI"},"6880992":{"id":6880992,"externalId":"511145.b0115","name":"aceF","hasFamily":true,"annotation":"pyruvate dehydrogenase, dihydrolipoyltransacetylase component E2; The pyruvate dehydrogenase complex catalyzes the overall conversion of pyruvate to acetyl-CoA and CO(2). It contains multiple copies of three enzymatic components: pyruvate dehydrogenase (E1), dihydrolipoamide acetyltransferase (E2) and lipoamide dehydrogenase (E3)","uniprotId":"ODP2_ECOLI"},"6880993":{"id":6880993,"externalId":"511145.b0116","name":"lpd","hasFamily":true,"annotation":"lipoamide dehydrogenase, E3 component is part of three enzyme complexes; Lipoamide dehydrogenase is a component of the glycine cleavage system as well as of the alpha-ketoacid dehydrogenase complexes","uniprotId":"DLDH_ECOLI"},"6880994":{"id":6880994,"externalId":"511145.b0117","name":"yacH","hasFamily":true,"annotation":"predicted protein","uniprotId":"YACH_ECOLI"},"6880995":{"id":6880995,"externalId":"511145.b0118","name":"acnB","hasFamily":true,"annotation":"bifunctional aconitate hydratase 2/2-methylisocitrate dehydratase; Catalyzes the isomerization of citrate to isocitrate via cis-aconitate as well as the dehydration of 2-methylisocitrate to cis-2-methylaconitate","uniprotId":"ACON2_ECOLI"},"6880996":{"id":6880996,"externalId":"511145.b0119","name":"yacL","hasFamily":true,"annotation":"conserved protein","uniprotId":"YACL_ECOLI"},"6880997":{"id":6880997,"externalId":"511145.b0120","name":"speD","hasFamily":true,"annotation":"S-adenosylmethionine decarboxylase; Catalyzes the decarboxylation of S-adenosylmethionine to S-adenosylmethioninamine (dcAdoMet), the propylamine donor required for the synthesis of the polyamines spermine and spermidine from the diamine putrescine (By similarity)","uniprotId":"SPED_ECOLI"},"6880998":{"id":6880998,"externalId":"511145.b0121","name":"speE","hasFamily":true,"annotation":"spermidine synthase (putrescine aminopropyltransferase); Catalyzes the production of spermidine from putrescine and decarboxylated S-adenosylmethionine (dcSAM), which serves as an aminopropyl donor (By similarity)","uniprotId":"SPEE_ECOLI"},"6880999":{"id":6880999,"externalId":"511145.b0122","name":"yacC","hasFamily":true,"annotation":"conserved protein, PulS_OutS family","uniprotId":"YACC_ECOLI"},"6881000":{"id":6881000,"externalId":"511145.b0123","name":"cueO","hasFamily":true,"annotation":"multicopper oxidase (laccase); Probably involved in periplasmic detoxification of copper by oxidizing Cu(+) to Cu(2+) and thus preventing its uptake into the cytoplasm. Possesses phenoloxidase and ferroxidase activities and might be involved in the production of polyphenolic compounds and the prevention of oxidative damage in the periplasm","uniprotId":"CUEO_ECOLI"},"6881001":{"id":6881001,"externalId":"511145.b0124","name":"gcd","hasFamily":true,"annotation":"glucose dehydrogenase; GDH is probably involved in energy conservation rather than in sugar metabolism","uniprotId":"DHG_ECOLI"},"6881002":{"id":6881002,"externalId":"511145.b0125","name":"hpt","hasFamily":true,"annotation":"hypoxanthine phosphoribosyltransferase; Acts preferentially on hypoxanthine; has very low activity towards guanine. Inactive towards xanthine","uniprotId":"HPRT_ECOLI"},"6881003":{"id":6881003,"externalId":"511145.b0126","name":"can","hasFamily":true,"annotation":"carbonic anhydrase","uniprotId":"CAN_ECOLI"},"6881004":{"id":6881004,"externalId":"511145.b0127","name":"yadG","hasFamily":true,"annotation":"putative transporter subunit: ATP-binding component of ABC superfamily","uniprotId":"YADG_ECOLI"},"6881005":{"id":6881005,"externalId":"511145.b0128","name":"yadH","hasFamily":true,"annotation":"putative transporter subunit: membrane component of ABC superfamily","uniprotId":"YADH_ECOLI"},"6881006":{"id":6881006,"externalId":"511145.b0129","name":"yadI","hasFamily":true,"annotation":"putative PTS Enzyme IIA; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (By similarity)","uniprotId":"YADI_ECOLI"},"6881007":{"id":6881007,"externalId":"511145.b0130","name":"yadE","hasFamily":true,"annotation":"putative polysaccharide deacetylase lipoprotein","uniprotId":"YADE_ECOLI"},"6881008":{"id":6881008,"externalId":"511145.b0131","name":"panD","hasFamily":true,"annotation":"aspartate 1-decarboxylase; Catalyzes the pyruvoyl-dependent decarboxylation of aspartate to produce beta-alanine (By similarity)","uniprotId":"PAND_ECOLI"},"6881009":{"id":6881009,"externalId":"511145.b0132","name":"yadD","hasFamily":false,"annotation":"putative transposase","uniprotId":"YADD_ECOLI"},"6881010":{"id":6881010,"externalId":"511145.b0133","name":"panC","hasFamily":true,"annotation":"pantothenate synthetase; Catalyzes the condensation of pantoate with beta-alanine in an ATP-dependent reaction via a pantoyl-adenylate intermediate (By similarity)","uniprotId":"PANC_ECOLI"},"6881011":{"id":6881011,"externalId":"511145.b0134","name":"panB","hasFamily":true,"annotation":"3-methyl-2-oxobutanoate hydroxymethyltransferase; Catalyzes the reversible reaction in which hydroxymethyl group from 5,10-methylenetetrahydrofolate is tranferred onto alpha-ketoisovalerate to form ketopantoate (By similarity)","uniprotId":"PANB_ECOLI"},"6881012":{"id":6881012,"externalId":"511145.b0135","name":"yadC","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YADC_ECOLI"},"6881013":{"id":6881013,"externalId":"511145.b0136","name":"yadK","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YADK_ECOLI"},"6881014":{"id":6881014,"externalId":"511145.b0137","name":"yadL","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YADL_ECOLI"},"6881015":{"id":6881015,"externalId":"511145.b0138","name":"yadM","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YADM_ECOLI"},"6881016":{"id":6881016,"externalId":"511145.b0139","name":"htrE","hasFamily":true,"annotation":"putative outer membrane usher protein; Probable porin-like protein necessary for the assembly of a pilin-type protein","uniprotId":"HTRE_ECOLI"},"6881017":{"id":6881017,"externalId":"511145.b0140","name":"ecpD","hasFamily":true,"annotation":"putative periplasmic pilin chaperone; Probably mediates assembly of a pilin","uniprotId":"ECPD_ECOLI"},"6881018":{"id":6881018,"externalId":"511145.b0141","name":"yadN","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YADN_ECOLI"},"6881019":{"id":6881019,"externalId":"511145.b0142","name":"folK","hasFamily":true,"annotation":"2-amino-4-hydroxy-6-hydroxymethyldihyropteridine pyrophosphokinase","uniprotId":"HPPK_ECOLI"},"6881020":{"id":6881020,"externalId":"511145.b0143","name":"pcnB","hasFamily":true,"annotation":"poly(A) polymerase I; Adds poly(A) tail to the 3' end of many RNAs, which usually targets these RNAs for decay. Plays a significant role in the global control of gene expression, through influencing the rate of transcript degradation, and in the general RNA quality control. Rho-independent transcription terminators may serve as polyadenylation signals. Seems to be involved in plasmid copy number control","uniprotId":"PCNB_ECOLI"},"6881021":{"id":6881021,"externalId":"511145.b0144","name":"gluQ","hasFamily":true,"annotation":"glutamyl-Q tRNA(Asp) synthetase; Catalyzes the tRNA-independent activation of glutamate in presence of ATP and the subsequent transfer of glutamate onto tRNA(Asp). Glutamate is transferred on the 2-amino-5-(4,5- dihydroxy-2-cyclopenten-1-yl) moiety of the queuosine in position 34 of the tRNA(Asp), the wobble position of the QUC anticodon. Does not transfer glutamate to either tRNA(Glu) or tRNA(Gln). The incapacity of the glutamylated tRNA(Asp) to bind elongation factor Tu suggests that it is not involved in ribosomal protein biosynthesis","uniprotId":"GLUQ_ECOLI"},"6881022":{"id":6881022,"externalId":"511145.b0145","name":"dksA","hasFamily":true,"annotation":"DNA-binding transcriptional regulator of rRNA transcription, DnaK suppressor protein; Transcription factor that acts by binding directly to the RNA polymerase (RNAP). Required for negative regulation of rRNA expression and positive regulation of several amino acid biosynthesis promoters. Also required for regulation of fis expression. Binding to RNAP disrupts interaction of RNAP with DNA, inhibits formation of initiation complexes, and amplifies effects of ppGpp and the initiating NTP on rRNA transcription. Inhibits transcript elongation, exonucleolytic RNA cleavage and pyrophosphoroly [...] ","uniprotId":"DKSA_ECOLI"},"6881023":{"id":6881023,"externalId":"511145.b0146","name":"sfsA","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator; Binds to DNA non-specifically. Could be a regulatory factor involved in maltose metabolism (By similarity)","uniprotId":"SFSA_ECOLI"},"6881024":{"id":6881024,"externalId":"511145.b0147","name":"ligT","hasFamily":true,"annotation":"2'-5' RNA ligase; Specifically ligates tRNA half-molecules containing nucleoside base modifications and shows a preference among different tRNA species. Acts in the absence of ATP to form a 2'-5' phosphodiester linkage","uniprotId":"LIGT_ECOLI"},"6881025":{"id":6881025,"externalId":"511145.b0148","name":"hrpB","hasFamily":true,"annotation":"putative ATP-dependent helicase","uniprotId":"HRPB_ECOLI"},"6881026":{"id":6881026,"externalId":"511145.b0149","name":"mrcB","hasFamily":true,"annotation":"fused glycosyl transferase and transpeptidase; Cell wall formation. Synthesis of cross-linked peptidoglycan from the lipid intermediates. The enzyme has a penicillin-insensitive transglycosylase N-terminal domain (formation of linear glycan strands) and a penicillin-sensitive transpeptidase C-terminal domain (cross-linking of the peptide subunits)","uniprotId":"PBPB_ECOLI"},"6881027":{"id":6881027,"externalId":"511145.b0150","name":"fhuA","hasFamily":true,"annotation":"ferrichrome outer membrane transporter; This receptor binds the ferrichrome-iron ligand. It interacts with the TonB protein, which is responsible for energy coupling of the ferrichrome-promoted iron transport system. Acts as a receptor for bacteriophage T5 as well as T1, phi80 and colicin M. Binding of T5 triggers the opening of a high conductance ion channel. Can also transport the antibiotic albomycin","uniprotId":"FHUA_ECOLI"},"6881028":{"id":6881028,"externalId":"511145.b0151","name":"fhuC","hasFamily":true,"annotation":"iron-hydroxamate transporter subunit; Part of the ABC transporter complex FhuCDB involved in iron(3+)-hydroxamate import. Responsible for energy coupling to the transport system","uniprotId":"FHUC_ECOLI"},"6881029":{"id":6881029,"externalId":"511145.b0152","name":"fhuD","hasFamily":true,"annotation":"iron-hydroxamate transporter subunit; Part of the ABC transporter complex FhuCDB involved in iron(3+)-hydroxamate import. Binds the iron(3+)-hydroxamate complex and transfers it to the membrane-bound permease. Required for the transport of all iron(3+)-hydroxamate siderophores such as ferrichrome, gallichrome, desferrioxamine, coprogen, aerobactin, shizokinen, rhodotorulic acid and the antibiotic albomycin","uniprotId":"FHUD_ECOLI"},"6881030":{"id":6881030,"externalId":"511145.b0153","name":"fhuB","hasFamily":true,"annotation":"fused iron-hydroxamate transporter subunits of ABC superfamily: membrane components; Part of the ABC transporter complex FhuCDB involved in iron(3+)-hydroxamate import. Responsible for the translocation of the substrate across the membrane","uniprotId":"FHUB_ECOLI"},"6881031":{"id":6881031,"externalId":"511145.b0154","name":"hemL","hasFamily":true,"annotation":"glutamate-1-semialdehyde aminotransferase (aminomutase)","uniprotId":"GSA_ECOLI"},"6881032":{"id":6881032,"externalId":"511145.b0155","name":"clcA","hasFamily":true,"annotation":"H(+)/Cl(-) exchange transporter; Proton-coupled chloride transporter. Functions as antiport system and exchanges two chloride ions for 1 proton. Probably acts as an electrical shunt for an outwardly-directed proton pump that is linked to amino acid decarboxylation, as part of the extreme acid resistance (XAR) response (By similarity)","uniprotId":"CLCA_ECOLI"},"6881033":{"id":6881033,"externalId":"511145.b0156","name":"erpA","hasFamily":true,"annotation":"iron-sulfur cluster insertion protein; Probably involved in the insertion of Fe-S clusters into apoproteins in vivo including IspG and/or IspH. Essential for growth under aerobic conditions and for anaerobic respiration but not for fermentation. In vitro it binds Fe-S clusters and transfers them to apo-IspG, which is involved in quinone biosynthesis among many other cell components. Experiments indicate that it is probably also involved in the insertion of other Fe-S clusters than IspG/IspH","uniprotId":"ERPA_ECOLI"},"6881034":{"id":6881034,"externalId":"511145.b0157","name":"yadS","hasFamily":true,"annotation":"inner membrane protein, UPF0126 family","uniprotId":"YADS_ECOLI"},"6881035":{"id":6881035,"externalId":"511145.b0158","name":"btuF","hasFamily":true,"annotation":"vitamin B12 transporter subunit: periplasmic-binding component of ABC superfamily; Part of the ABC transporter complex BtuCDF involved in vitamin B12 import. Binds vitamin B12 and delivers it to the periplasmic surface of BtuC (By similarity)","uniprotId":"BTUF_ECOLI"},"6881036":{"id":6881036,"externalId":"511145.b0159","name":"mtn","hasFamily":true,"annotation":"5'-methylthioadenosine/S-adenosylhomocysteine nucleosidase; Catalyzes the irreversible cleavage of the glycosidic bond in both 5'-methylthioadenosine (MTA) and S- adenosylhomocysteine (SAH/AdoHcy) to adenine and the corresponding thioribose, 5'-methylthioribose and S-ribosylhomocysteine, respectively. Can also use 5'-isobutylthioadenosine, 5'-n- butylthioadenosine, S-adenosyl-D-homocysteine, decarboxylated adenosylhomocysteine, deaminated adenosylhomocysteine and S-2-aza- adenosylhomocysteine as substrates","uniprotId":"MTNN_ECOLI"},"6881037":{"id":6881037,"externalId":"511145.b0160","name":"dgt","hasFamily":true,"annotation":"deoxyguanosine triphosphate triphosphohydrolase; dGTPase preferentially hydrolyzes dGTP over the other canonical NTPs (By similarity)","uniprotId":"DGTP_ECOLI"},"6881038":{"id":6881038,"externalId":"511145.b0161","name":"degP","hasFamily":true,"annotation":"serine endoprotease (protease Do), membrane-associated; DegP acts as a chaperone at low temperatures but switches to a peptidase (heat shock protein) at higher temperatures. It degrades transiently denatured and unfolded proteins which accumulate in the periplasm following heat shock or other stress conditions. DegP is efficient with Val-Xaa and Ile- Xaa peptide bonds, suggesting a preference for beta-branched side chain amino acids. Only unfolded proteins devoid of disulfide bonds appear capable of being cleaved, thereby preventing non- specific proteolysis of folded proteins. Its pro [...] ","uniprotId":"DEGP_ECOLI"},"6881039":{"id":6881039,"externalId":"511145.b0162","name":"cdaR","hasFamily":true,"annotation":"DNA-binding transcriptional regulator for gar and gud operons; carbohydrate diacid regulator; Seems to regulate the expression of the operons for the enzymes involved in D-galactarate, D-glucarate and D-glycerate utilization","uniprotId":"CDAR_ECOLI"},"6881040":{"id":6881040,"externalId":"511145.b0163","name":"yaeH","hasFamily":true,"annotation":"conserved protein, UPF0325 family","uniprotId":"YAEH_ECOLI"},"6881041":{"id":6881041,"externalId":"511145.b0164","name":"yaeI","hasFamily":true,"annotation":"phosphodiesterase with model substrate bis-pNPP; Shows phosphodiesterase activity, hydrolyzing phosphodiester bond in the artificial chromogenic substrate bis-p- nitrophenyl phosphate (bis-pNPP)","uniprotId":"YAEI_ECOLI"},"6881042":{"id":6881042,"externalId":"511145.b0166","name":"dapD","hasFamily":true,"annotation":"2,3,4,5-tetrahydropyridine-2-carboxylate N-succinyltransferase","uniprotId":"DAPD_ECOLI"},"6881043":{"id":6881043,"externalId":"511145.b0167","name":"glnD","hasFamily":true,"annotation":"uridylyltransferase; Modifies, by uridylylation or deuridylylation the PII (GlnB) regulatory protein (By similarity)","uniprotId":"GLND_ECOLI"},"6881044":{"id":6881044,"externalId":"511145.b0168","name":"map","hasFamily":true,"annotation":"methionine aminopeptidase; Removes the N-terminal methionine from nascent proteins (By similarity)","uniprotId":"AMPM_ECOLI"},"6881045":{"id":6881045,"externalId":"511145.b0169","name":"rpsB","hasFamily":true,"annotation":"30S ribosomal subunit protein S2","uniprotId":"RS2_ECOLI"},"6881046":{"id":6881046,"externalId":"511145.b0170","name":"tsf","hasFamily":true,"annotation":"protein chain elongation factor EF-Ts; Associates with the EF-Tu.GDP complex and induces the exchange of GDP to GTP. It remains bound to the aminoacyl-tRNA.EF- Tu.GTP complex up to the GTP hydrolysis stage on the ribosome (By similarity)","uniprotId":"EFTS_ECOLI"},"6881047":{"id":6881047,"externalId":"511145.b0171","name":"pyrH","hasFamily":true,"annotation":"uridylate kinase; Catalyzes the reversible phosphorylation of UMP to UDP, with ATP as the most efficient phosphate donor","uniprotId":"PYRH_ECOLI"},"6881048":{"id":6881048,"externalId":"511145.b0172","name":"frr","hasFamily":true,"annotation":"ribosome recycling factor; Responsible for the release of ribosomes from messenger RNA at the termination of protein biosynthesis. May increase the efficiency of translation by recycling ribosomes from one round of translation to another (By similarity)","uniprotId":"RRF_ECOLI"},"6881049":{"id":6881049,"externalId":"511145.b0173","name":"dxr","hasFamily":true,"annotation":"1-deoxy-D-xylulose 5-phosphate reductoisomerase; Catalyzes the NADP-dependent rearrangement and reduction of 1-deoxy-D-xylulose-5-phosphate (DXP) to 2-C-methyl-D-erythritol 4-phosphate (MEP) (By similarity)","uniprotId":"DXR_ECOLI"},"6881050":{"id":6881050,"externalId":"511145.b0174","name":"ispU","hasFamily":true,"annotation":"undecaprenyl pyrophosphate synthase; Catalyzes the sequential condensation of isopentenyl diphosphate (IPP) with (2E,6E)-farnesyl diphosphate (E,E-FPP) to yield (2Z,6Z,10Z,14Z,18Z,22Z,26Z,30Z,34E,38E)-undecaprenyl diphosphate (di-trans,octa-cis-UPP). UPP is the precursor of glycosyl carrier lipid in the biosynthesis of bacterial cell wall polysaccharide components such as peptidoglycan and lipopolysaccharide (By similarity)","uniprotId":"UPPS_ECOLI"},"6881051":{"id":6881051,"externalId":"511145.b0175","name":"cdsA","hasFamily":true,"annotation":"CDP-diglyceride synthase","uniprotId":"CDSA_ECOLI"},"6881052":{"id":6881052,"externalId":"511145.b0176","name":"rseP","hasFamily":true,"annotation":"inner membrane zinc RIP metalloprotease; RpoE activator, by degrading RseA; Cleaves the peptide bond between 'Ala-108' and 'Cys-109' in the transmembrane region of RseA. Acts on DegS-cleaved RseA to release the cytoplasmic domain of RseA. This provides the cell with sigma E (RpoE) activity through the proteolysis of RseA. In vitro, can also cleave sequences in transmembrane regions of other proteins as well as the signal sequence of beta-lactamase. Purified RseP was also shown to degrade RpoH and RpoE in vitro","uniprotId":"RSEP_ECOLI"},"6881053":{"id":6881053,"externalId":"511145.b0177","name":"bamA","hasFamily":true,"annotation":"outer membrane protein assembly factor, forms pores; required for OM biogenesis; Part of the outer membrane protein assembly complex, which is involved in assembly and insertion of beta-barrel proteins into the outer membrane. Constitutes, with BamD, the core component of the assembly machinery (By similarity)","uniprotId":"BAMA_ECOLI"},"6881054":{"id":6881054,"externalId":"511145.b0178","name":"skp","hasFamily":true,"annotation":"periplasmic chaperone; Molecular chaperone that interacts specifically with outer membrane proteins, thus maintaining the solubility of early folding intermediates during passage through the periplasm. Required for the efficient release of OmpA from the inner membrane, the maintenance of its solubility in the periplasm, and, in association with lipopolysaccharide (LPS), for the efficient folding and insertion of OmpA into the outer membrane","uniprotId":"SKP_ECOLI"},"6881055":{"id":6881055,"externalId":"511145.b0179","name":"lpxD","hasFamily":true,"annotation":"UDP-3-O-(3-hydroxymyristoyl)-glucosamine N-acyltransferase; Catalyzes the N-acylation of UDP-3-O- (hydroxymyristoyl)glucosamine using 3-hydroxymyristoyl-ACP as the acyl donor. Is involved in the biosynthesis of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell. Prefers R-3-hydroxymyristoyl-ACP over R-3-hydroxypalmitoyl-ACP as the acyl donor in vitro, which is consistent with the structure of E.coli lipid A that contains over 95% (R)-3-hydroxymyristate at the 2 and 2' positions","uniprotId":"LPXD_ECOLI"},"6881056":{"id":6881056,"externalId":"511145.b0180","name":"fabZ","hasFamily":true,"annotation":"(3R)-hydroxymyristol acyl carrier protein dehydratase; Involved in unsaturated fatty acids biosynthesis. Catalyzes the dehydration of short chain beta-hydroxyacyl-ACPs and long chain saturated and unsaturated beta-hydroxyacyl-ACPs (By similarity)","uniprotId":"FABZ_ECOLI"},"6881057":{"id":6881057,"externalId":"511145.b0181","name":"lpxA","hasFamily":true,"annotation":"UDP-N-acetylglucosamine acetyltransferase; Involved in the biosynthesis of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell (By similarity)","uniprotId":"LPXA_ECOLI"},"6881058":{"id":6881058,"externalId":"511145.b0182","name":"lpxB","hasFamily":true,"annotation":"tetraacyldisaccharide-1-P synthase; Condensation of UDP-2,3-diacylglucosamine and 2,3- diacylglucosamine-1-phosphate to form lipid A disaccharide, a precursor of lipid A, a phosphorylated glycolipid that anchors the lipopolysaccharide to the outer membrane of the cell (By similarity)","uniprotId":"LPXB_ECOLI"},"6881059":{"id":6881059,"externalId":"511145.b0183","name":"rnhB","hasFamily":true,"annotation":"ribonuclease HII, degrades RNA of DNA-RNA hybrids; Endonuclease that specifically degrades the RNA of RNA- DNA hybrids (By similarity)","uniprotId":"RNH2_ECOLI"},"6881060":{"id":6881060,"externalId":"511145.b0184","name":"dnaE","hasFamily":true,"annotation":"DNA polymerase III alpha subunit; DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. This DNA polymerase also exhibits 3' to 5' exonuclease activity. The alpha chain is the DNA polymerase","uniprotId":"DPO3A_ECOLI"},"6881061":{"id":6881061,"externalId":"511145.b0185","name":"accA","hasFamily":true,"annotation":"acetyl-CoA carboxylase, carboxytransferase, alpha subunit; Component of the acetyl coenzyme A carboxylase (ACC) complex. First, biotin carboxylase catalyzes the carboxylation of biotin on its carrier protein (BCCP) and then the CO(2) group is transferred by the carboxyltransferase to acetyl-CoA to form malonyl-CoA (By similarity)","uniprotId":"ACCA_ECOLI"},"6881062":{"id":6881062,"externalId":"511145.b0186","name":"ldcC","hasFamily":true,"annotation":"lysine decarboxylase 2, constitutive; LDC is constitutively but weakly expressed under various conditions","uniprotId":"DCLZ_ECOLI"},"6881063":{"id":6881063,"externalId":"511145.b0187","name":"yaeR","hasFamily":true,"annotation":"putative lyase","uniprotId":"YAER_ECOLI"},"6881064":{"id":6881064,"externalId":"511145.b0188","name":"mesJ","hasFamily":true,"annotation":"tRNA(Ile)-lysidine synthetase; Ligates lysine onto the cytidine present at position 34 of the AUA codon-specific tRNA(Ile) that contains the anticodon CAU, in an ATP-dependent manner. Cytidine is converted to lysidine, thus changing the amino acid specificity of the tRNA from methionine to isoleucine. This enzyme is essential for viability","uniprotId":"TILS_ECOLI"},"6881065":{"id":6881065,"externalId":"511145.b0189","name":"rof","hasFamily":true,"annotation":"modulator of Rho-dependent transcription termination; Suppresses temperature-sensitive mutations in essential genes by modulating rho-dependent transcription termination","uniprotId":"ROF_ECOLI"},"6881066":{"id":6881066,"externalId":"511145.b0190","name":"yaeQ","hasFamily":true,"annotation":"conserved protein","uniprotId":"YAEQ_ECOLI"},"6881067":{"id":6881067,"externalId":"511145.b0191","name":"yaeJ","hasFamily":true,"annotation":"petidyl-tRNA hydrolase, ribosome-attached; stalled ribosome rescue factor; Rescues stalled ribosomes. Can hydrolyze peptidyl-tRNA on ribosomes stalled by both non-stop mRNAs and mRNAs that contain rare codon clusters. May function as a complementary rescue system when the stalled ribosome can not be rescued by the SsrA(tmRNA)- SmpB quality control system or the alternative ribosome-rescue factor A","uniprotId":"ARFB_ECOLI"},"6881068":{"id":6881068,"externalId":"511145.b0192","name":"cutF","hasFamily":true,"annotation":"lipoprotein involved with copper homeostasis and adhesion; Involved in copper homeostasis. Could be involved in both copper efflux and the delivery of copper to copper-dependent enzymes. When overproduced induces degP through the activation of the two-component system CpxA/CpxR","uniprotId":"NLPE_ECOLI"},"6881069":{"id":6881069,"externalId":"511145.b0193","name":"yaeF","hasFamily":true,"annotation":"putative lipoprotein","uniprotId":"YAEF_ECOLI"},"6881070":{"id":6881070,"externalId":"511145.b0194","name":"proS","hasFamily":true,"annotation":"prolyl-tRNA synthetase; Catalyzes the attachment of proline to tRNA(Pro) in a two-step reaction: proline is first activated by ATP to form Pro- AMP and then transferred to the acceptor end of tRNA(Pro). As ProRS can inadvertently accommodate and process non-cognate amino acids such as alanine and cysteine, to avoid such errors it has two additional distinct editing activities against alanine. One activity is designated as 'pretransfer' editing and involves the tRNA(Pro)-independent hydrolysis of activated Ala-AMP. The other activity is designated 'posttransfer' editing and involves dea [...] ","uniprotId":"SYP_ECOLI"},"6881071":{"id":6881071,"externalId":"511145.b0195","name":"tsaA","hasFamily":true,"annotation":"conserved protein, UPF0066 family; Is likely the methyltransferase responsible for the addition of the methyl group in the formation of N6-methyl-N6- threonylcarbamoyladenosine at position 37 (m6t6A37) in some tRNAs species that read codons starting with adenosine. The methyl group of m6t6A37 appears to slightly improve the efficiency of the tRNA","uniprotId":"TSAA_ECOLI"},"6881072":{"id":6881072,"externalId":"511145.b0196","name":"rcsF","hasFamily":true,"annotation":"putative outer membrane protein, signal; Essential component of the Rcs signaling system to promote transcription of the genes for colanic acid capsule synthesis. Together with RcsC, they probably promote the phosphorylation of RcsB","uniprotId":"RCSF_ECOLI"},"6881073":{"id":6881073,"externalId":"511145.b0197","name":"metQ","hasFamily":true,"annotation":"DL-methionine transporter subunit; This protein is a component of a D-methionine permease, a binding protein-dependent, ATP-driven transport system","uniprotId":"METQ_ECOLI"},"6881074":{"id":6881074,"externalId":"511145.b0198","name":"metI","hasFamily":true,"annotation":"DL-methionine transporter subunit; Part of the binding-protein-dependent transport system for D-methionine and the toxic methionine analog alpha-methyl- methionine. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"METI_ECOLI"},"6881075":{"id":6881075,"externalId":"511145.b0199","name":"abc","hasFamily":true,"annotation":"DL-methionine transporter subunit; Part of the ABC transporter complex MetNIQ involved in methionine import. Responsible for energy coupling to the transport system (Probable). It has also been shown to be involved in formyl-L-methionine transport","uniprotId":"METN_ECOLI"},"6881076":{"id":6881076,"externalId":"511145.b0200","name":"gmhB","hasFamily":true,"annotation":"D,D-heptose 1,7-bisphosphate phosphatase; Converts the D-glycero-beta-D-manno-heptose 1,7- bisphosphate intermediate into D-glycero-beta-D-manno-heptose 1- phosphate (HBP) by removing the phosphate group at the C-7 position. Also catalyzes the dephosphorylation of fructose-1,6- bisphosphate (Fru1,6bisP)","uniprotId":"GMHB_ECOLI"},"6881077":{"id":6881077,"externalId":"511145.b0207","name":"dkgB","hasFamily":true,"annotation":"2,5-diketo-D-gluconate reductase B; Catalyzes the reduction of 2,5-diketo-D-gluconic acid (25DKG) to 2-keto-L-gulonic acid (2KLG)","uniprotId":"DKGB_ECOLI"},"6881078":{"id":6881078,"externalId":"511145.b0208","name":"yafC","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YAFC_ECOLI"},"6881079":{"id":6881079,"externalId":"511145.b0209","name":"yafD","hasFamily":true,"annotation":"conserved protein","uniprotId":"YAFD_ECOLI"},"6881080":{"id":6881080,"externalId":"511145.b0210","name":"yafE","hasFamily":true,"annotation":"putative S-adenosyl-L-methionine-dependent methyltransferase","uniprotId":"YAFE_ECOLI"},"6881081":{"id":6881081,"externalId":"511145.b0211","name":"mltD","hasFamily":true,"annotation":"predicted membrane-bound lytic murein transglycosylase D; Murein-degrading enzyme. May play a role in recycling of muropeptides during cell elongation and/or cell division (By similarity)","uniprotId":"MLTD_ECOLI"},"6881082":{"id":6881082,"externalId":"511145.b0212","name":"gloB","hasFamily":true,"annotation":"hydroxyacylglutathione hydrolase; Thiolesterase that catalyzes the hydrolysis of S-D- lactoyl-glutathione to form glutathione and D-lactic acid (By similarity)","uniprotId":"GLO2_ECOLI"},"6881083":{"id":6881083,"externalId":"511145.b0213","name":"yafS","hasFamily":true,"annotation":"putative S-adenosyl-L-methionine-dependent methyltransferase","uniprotId":"YAFS_ECOLI"},"6881084":{"id":6881084,"externalId":"511145.b0214","name":"rnhA","hasFamily":true,"annotation":"ribonuclease HI, degrades RNA of DNA-RNA hybrids; Endonuclease that specifically degrades the RNA of RNA- DNA hybrids. RNase H participates in DNA replication; it helps to specify the origin of genomic replication by suppressing initiation at origins other than the oriC locus; along with the 5'-3' exonuclease of pol1, it removes RNA primers from the Okazaki fragments of lagging strand synthesis; and it defines the origin of replication for ColE1-type plasmids by specific cleavage of an RNA preprimer","uniprotId":"RNH_ECOLI"},"6881085":{"id":6881085,"externalId":"511145.b0215","name":"dnaQ","hasFamily":true,"annotation":"DNA polymerase III epsilon subunit; DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. The epsilon subunit contain the editing function and is a proofreading 3'-5' exonuclease","uniprotId":"DPO3E_ECOLI"},"6881086":{"id":6881086,"externalId":"511145.b0217","name":"yafT","hasFamily":false,"annotation":"lipoprotein","uniprotId":"YAFT_ECOLI"},"6881087":{"id":6881087,"externalId":"511145.b0219","name":"aec5","hasFamily":true,"annotation":"putative C-N hydrolase family amidase, NAD(P)-binding","uniprotId":"YAFV_ECOLI"},"6881088":{"id":6881088,"externalId":"511145.b0220","name":"ivy","hasFamily":true,"annotation":"inhibitor of c-type lysozyme, periplasmic; Strong inhibitor of lysozyme C","uniprotId":"IVY_ECOLI"},"6881089":{"id":6881089,"externalId":"511145.b0221","name":"fadE","hasFamily":true,"annotation":"acyl coenzyme A dehydrogenase; Catalyzes the dehydrogenation of acyl-CoA","uniprotId":"FADE_ECOLI"},"6881090":{"id":6881090,"externalId":"511145.b0222","name":"gmhA","hasFamily":true,"annotation":"D-sedoheptulose 7-phosphate isomerase; Catalyzes the isomerization of sedoheptulose 7-phosphate in D-glycero-D-manno-heptose 7-phosphate (By similarity)","uniprotId":"GMHA_ECOLI"},"6881091":{"id":6881091,"externalId":"511145.b0223","name":"yafJ","hasFamily":true,"annotation":"putative amidotransfease","uniprotId":"YAFJ_ECOLI"},"6881092":{"id":6881092,"externalId":"511145.b0224","name":"yafK","hasFamily":true,"annotation":"conserved protein","uniprotId":"YAFK_ECOLI"},"6881093":{"id":6881093,"externalId":"511145.b0225","name":"yafQ","hasFamily":true,"annotation":"translation inhibitor toxin of toxin-antitoxin pair YafQ/DinJ; Toxic component of a toxin-antitoxin (TA) module. A sequence-specific mRNA endoribonuclease that inhibits translation elongation and induces bacterial stasis. Cleavage occurs between the second and third residue of the Lys codon followed by a G or A (5'AAA(G/A)3'), is reading-frame dependent and occurs within the 5' end of most mRNAs. Ribosome-binding confers the sequence specificity and reading frame-dependence. When overexpressed in liquid media YafQ partially inhibits protein synthesis, with a reduction in growth rate an [...] ","uniprotId":"YAFQ_ECOLI"},"6881094":{"id":6881094,"externalId":"511145.b0226","name":"dinJ","hasFamily":true,"annotation":"antitoxin of YafQ-DinJ toxin-antitoxin system; Antitoxin component of a toxin-antitoxin (TA) module. A labile antitoxin that counteracts the effect of the YafQ toxin. YafQ and DinJ together bind their own promoter, and by analogy to other TA modules probably repress its expression","uniprotId":"DINJ_ECOLI"},"6881095":{"id":6881095,"externalId":"511145.b0227","name":"yafL","hasFamily":true,"annotation":"putative lipoprotein and C40 family peptidase","uniprotId":"YAFL_ECOLI"},"6881096":{"id":6881096,"externalId":"511145.b0228","name":"yafM","hasFamily":true,"annotation":"conserved protein","uniprotId":"YAFM_ECOLI"},"6881097":{"id":6881097,"externalId":"511145.b0231","name":"dinB","hasFamily":true,"annotation":"DNA polymerase IV; Poorly processive, error-prone DNA polymerase involved in untargeted mutagenesis. Copies undamaged DNA at stalled replication forks, which arise in vivo from mismatched or misaligned primer ends. These misaligned primers can be extended by PolIV. Exhibits no 3'-5' exonuclease (proofreading) activity. May be involved in translesional synthesis, in conjunction with the beta clamp from PolIII (By similarity)"},"6881098":{"id":6881098,"externalId":"511145.b0232","name":"yafN","hasFamily":false,"annotation":"antitoxin of the YafO-YafN toxin-antitoxin system; Antitoxin component of a toxin-antitoxin (TA) module. Functions as an mRNA interferase antitoxin; overexpression prevents YafO-mediated cessation of cell growth and inhibition of cell proliferation","uniprotId":"YAFN_ECOLI"},"6881099":{"id":6881099,"externalId":"511145.b0233","name":"yafO","hasFamily":false,"annotation":"mRNA interferase toxin of the YafO-YafN toxin-antitoxin system; Toxic component of a toxin-antitoxin (TA) module. A translation-dependent mRNA interferase. Overexpression causes cessation of cell growth and inhibits cell proliferation via inhibition of translation; this blockage is overcome by subsequent expression of antitoxin YafN. Overexpression causes cleavage of a number of mRNAs in a ribosome-dependent fashion. YafO binding to the 50S ribosomal subunit in the translation complex induces mRNA cleavage 3' to the region protected by the ribosome; YafO alone is not able to digest mRNA","uniprotId":"YAFO_ECOLI"},"6881100":{"id":6881100,"externalId":"511145.b0234","name":"yafP","hasFamily":true,"annotation":"putative acyltransferase with acyl-CoA N-acyltransferase domain","uniprotId":"YAFP_ECOLI"},"6881101":{"id":6881101,"externalId":"511145.b0237","name":"pepD","hasFamily":true,"annotation":"aminoacyl-histidine dipeptidase (peptidase D); Dipeptidase with broad substrate specificity. Requires dipeptide substrates with an unblocked N-terminus and the amino group in the alpha or beta position. Non-protein amino acids and proline are not accepted in the C-terminal position, whereas some dipeptide amides and formyl amino acids are hydrolyzed. Also shows cysteinylglycinase activity, which is sufficient for E.coli to utilize cysteinylglycine as a cysteine source","uniprotId":"PEPD_ECOLI"},"6881102":{"id":6881102,"externalId":"511145.b0238","name":"gpt","hasFamily":true,"annotation":"guanine-hypoxanthine phosphoribosyltransferase; Acts on guanine, xanthine and to a lesser extent hypoxanthine (By similarity)","uniprotId":"XGPT_ECOLI"},"6881103":{"id":6881103,"externalId":"511145.b0239","name":"frsA","hasFamily":true,"annotation":"fermentation-respiration switch protein; PTS Enzyme IIA(Glc)-binding protein; Displays esterase activity toward pNP-butyrate (By similarity)","uniprotId":"FRSA_ECOLI"},"6881104":{"id":6881104,"externalId":"511145.b0240","name":"crl","hasFamily":true,"annotation":"sigma factor-binding protein, stimulates RNA polymerase holoenzyme formation; Binds to the sigma-S subunit of RNA polymerase, activating expression of sigma-S-regulated genes, such as the csgBAC operon encoding the subunits of curli proteins. Stimulates RNA polymerase holoenzyme formation and may bind to several other sigma factors, such as sigma-70 and sigma-32","uniprotId":"CRL_ECOLI"},"6881105":{"id":6881105,"externalId":"511145.b0241","name":"phoE","hasFamily":true,"annotation":"outer membrane phosphoporin protein E; Uptake of inorganic phosphate, phosphorylated compounds, and some other negatively charged solutes","uniprotId":"PHOE_ECOLI"},"6881106":{"id":6881106,"externalId":"511145.b0242","name":"proB","hasFamily":true,"annotation":"gamma-glutamate kinase; Catalyzes the transfer of a phosphate group to glutamate to form glutamate 5-phosphate which rapidly cyclizes to 5- oxoproline (By similarity)","uniprotId":"PROB_ECOLI"},"6881107":{"id":6881107,"externalId":"511145.b0243","name":"proA","hasFamily":true,"annotation":"gamma-glutamylphosphate reductase; Catalyzes the NADPH dependent reduction of L-gamma- glutamyl 5-phosphate into L-glutamate 5-semialdehyde and phosphate. The product spontaneously undergoes cyclization to form 1-pyrroline-5-carboxylate (By similarity)","uniprotId":"PROA_ECOLI"},"6881108":{"id":6881108,"externalId":"511145.b0245","name":"ykfI","hasFamily":false,"annotation":"CP4-6 prophage; Toxic component of a toxin-antitoxin (TA) module. Overexpression results in inhibition of growth in liquid cultures and a decrease in colony formation; these effects are overcome by concomitant expression of antitoxin YafW, which leads to toxin loss by an unknown mechanism","uniprotId":"YKFI_ECOLI"},"6881109":{"id":6881109,"externalId":"511145.b0246","name":"yafW","hasFamily":true,"annotation":"CP4-6 prophage; Antitoxin component of a toxin-antitoxin (TA) module. Labile antitoxin that counteracts the effect of the YkfI toxin. It does not seem to bind to the cognate toxin but instead induces toxin loss by an unknown mechanism","uniprotId":"YAFW_ECOLI"},"6881110":{"id":6881110,"externalId":"511145.b0247","name":"ykfG","hasFamily":false,"annotation":"CP4-6 prophage","uniprotId":"YKFG_ECOLI"},"6881111":{"id":6881111,"externalId":"511145.b0248","name":"yafX","hasFamily":false,"annotation":"CP4-6 prophage","uniprotId":"YAFX_ECOLI"},"6881112":{"id":6881112,"externalId":"511145.b0249","name":"ykfF","hasFamily":false,"annotation":"CP4-6 prophage","uniprotId":"YKFF_ECOLI"},"6881113":{"id":6881113,"externalId":"511145.b0250","name":"ykfB","hasFamily":false,"annotation":"CP4-6 prophage","uniprotId":"YKFB_ECOLI"},"6881114":{"id":6881114,"externalId":"511145.b0251","name":"yafY","hasFamily":false,"annotation":"lipoprotein, inner membrane; overproduction stimulates degP expression","uniprotId":"YAFY_ECOLI"},"6881115":{"id":6881115,"externalId":"511145.b0252","name":"yafZ","hasFamily":false,"annotation":"CP4-6 prophage","uniprotId":"YAFZ_ECOLI"},"6881116":{"id":6881116,"externalId":"511145.b0253","name":"ykfA","hasFamily":true,"annotation":"CP4-6 prophage; predicted GTP-binding protein","uniprotId":"YKFA_ECOLI"},"6881117":{"id":6881117,"externalId":"511145.b0254","name":"perR","hasFamily":false,"annotation":"CP4-6 prophage; Apparent regulatory gene involved in peroxide resistance in stationary phase","uniprotId":"PERR_ECOLI"},"6881118":{"id":6881118,"externalId":"511145.b0256","name":"is30","hasFamily":true,"annotation":"IS30 transposase; Required for the transposition of the insertion element (Potential)","uniprotId":"INSI1_ECOLI"},"6881119":{"id":6881119,"externalId":"511145.b0259","name":"b0259","hasFamily":true,"annotation":"IS5 transposase and trans-activator"},"6881120":{"id":6881120,"externalId":"511145.b0260","name":"MmuP","hasFamily":true,"annotation":"CP4-6 prophage; predicted S-methylmethionine transporter; Transporter for the intake of S-methylmethionine","uniprotId":"MMUP_ECOLI"},"6881121":{"id":6881121,"externalId":"511145.b0261","name":"MmuM","hasFamily":true,"annotation":"CP4-6 prophage; S-methylmethionine:homocysteine methyltransferase; Catalyzes methyl transfer from S-methylmethionine or S- adenosylmethionine (less efficient) to homocysteine, selenohomocysteine and less efficiently selenocysteine","uniprotId":"MMUM_ECOLI"},"6881122":{"id":6881122,"externalId":"511145.b0262","name":"fbpC","hasFamily":true,"annotation":"CP4-6 prophage; predicted ferric transporter subunit; Part of the ABC transporter complex FbpABC involved in Fe(3+) ions import. Responsible for energy coupling to the transport system (By similarity)","uniprotId":"FBPC_ECOLI"},"6881123":{"id":6881123,"externalId":"511145.b0264","name":"b0264","hasFamily":true,"annotation":"IS1 transposase B"},"6881124":{"id":6881124,"externalId":"511145.b0265","name":"b0265","hasFamily":false,"annotation":"IS1 repressor TnpA"},"6881125":{"id":6881125,"externalId":"511145.b0267","name":"yagA","hasFamily":true,"annotation":"CP4-6 prophage","uniprotId":"YAGA_ECOLI"},"6881126":{"id":6881126,"externalId":"511145.b0268","name":"yagE","hasFamily":true,"annotation":"2-keto-3-deoxy gluconate (KDG) aldolase; CP4-6 prophage; Catalyzes the formation of 2-keto-3-deoxy-galactonate (KDGal) from pyruvate and glyceraldehyde. Overexpression leads to increased growth (over 2 hours) in the presence of the antibiotics norfloxacin, ampicillin and streptomycin","uniprotId":"YAGE_ECOLI"},"6881127":{"id":6881127,"externalId":"511145.b0269","name":"yagF","hasFamily":true,"annotation":"CP4-6 prophage","uniprotId":"YAGF_ECOLI"},"6881128":{"id":6881128,"externalId":"511145.b0270","name":"yagG","hasFamily":true,"annotation":"CP4-6 prophage","uniprotId":"YAGG_ECOLI"},"6881129":{"id":6881129,"externalId":"511145.b0271","name":"yagH","hasFamily":true,"annotation":"CP4-6 prophage; predicted xylosidase/arabinosidase","uniprotId":"YAGH_ECOLI"},"6881130":{"id":6881130,"externalId":"511145.b0272","name":"yagI","hasFamily":true,"annotation":"CP4-6 prophage","uniprotId":"YAGI_ECOLI"},"6881131":{"id":6881131,"externalId":"511145.b0273","name":"argF","hasFamily":true,"annotation":"ornithine carbamoyltransferase 2, chain F; CP4-6 prophage; Reversibly catalyzes the transfer of the carbamoyl group from carbamoyl phosphate (CP) to the N(epsilon) atom of ornithine (ORN) to produce L-citrulline, which is a substrate for argininosuccinate synthetase, the enzyme involved in the final step in arginine biosynthesis","uniprotId":"OTC2_ECOLI"},"6881132":{"id":6881132,"externalId":"511145.b0274","name":"aec38","hasFamily":true,"annotation":"IS1 transposase B; Absolutely required for transposition of IS1","uniprotId":"INSB2_ECOLI"},"6881133":{"id":6881133,"externalId":"511145.b0275","name":"b0275","hasFamily":false,"annotation":"IS1 repressor TnpA"},"6881134":{"id":6881134,"externalId":"511145.b0277","name":"yagK","hasFamily":true,"annotation":"CP4-6 prophage","uniprotId":"YAGK_ECOLI"},"6881135":{"id":6881135,"externalId":"511145.b0278","name":"yagL","hasFamily":false,"annotation":"CP4-6 prophage","uniprotId":"YAGL_ECOLI"},"6881136":{"id":6881136,"externalId":"511145.b0279","name":"yagM","hasFamily":false,"annotation":"CP4-6 prophage","uniprotId":"YAGM_ECOLI"},"6881137":{"id":6881137,"externalId":"511145.b0280","name":"yagN","hasFamily":false,"annotation":"CP4-6 prophage","uniprotId":"YAGN_ECOLI"},"6881138":{"id":6881138,"externalId":"511145.b0281","name":"intF","hasFamily":true,"annotation":"CP4-6 prophage; Integrase is necessary for integration of the phage into the host genome by site-specific recombination. In conjunction with excisionase, integrase is also necessary for excision of the prophage from the host genome","uniprotId":"INTF_ECOLI"},"6881139":{"id":6881139,"externalId":"511145.b0283","name":"yagQ","hasFamily":false,"annotation":"moco insertion factor for PaoABC aldehyde oxidoreductase","uniprotId":"YAGQ_ECOLI"},"6881140":{"id":6881140,"externalId":"511145.b0284","name":"yagR","hasFamily":true,"annotation":"PaoABC aldehyde oxidoreductase, Moco-containing subunit","uniprotId":"YAGR_ECOLI"},"6881141":{"id":6881141,"externalId":"511145.b0285","name":"yagS","hasFamily":true,"annotation":"PaoABC aldehyde oxidoreductase, FAD-containing subunit","uniprotId":"YAGS_ECOLI"},"6881142":{"id":6881142,"externalId":"511145.b0286","name":"yagT","hasFamily":true,"annotation":"PaoABC aldehyde oxidoreductase, 2Fe-2S subunit; Iron-sulfur subunit of the xanthine dehydrogenase complex (By similarity)","uniprotId":"YAGT_ECOLI"},"6881143":{"id":6881143,"externalId":"511145.b0287","name":"yagU","hasFamily":true,"annotation":"inner membrane protein, DUF1440 family","uniprotId":"YAGU_ECOLI"},"6881144":{"id":6881144,"externalId":"511145.b0288","name":"ykgJ","hasFamily":true,"annotation":"predicted ferredoxin","uniprotId":"YKGJ_ECOLI"},"6881145":{"id":6881145,"externalId":"511145.b0289","name":"matF","hasFamily":false,"annotation":"conserved protein","uniprotId":"YAGV_ECOLI"},"6881146":{"id":6881146,"externalId":"511145.b0290","name":"matE","hasFamily":false,"annotation":"putative receptor","uniprotId":"YAGW_ECOLI"},"6881147":{"id":6881147,"externalId":"511145.b0291","name":"matD","hasFamily":true,"annotation":"putative aromatic compound dioxygenase","uniprotId":"YAGX_ECOLI"},"6881148":{"id":6881148,"externalId":"511145.b0292","name":"matC","hasFamily":false,"annotation":"predicted protein","uniprotId":"MATC_ECOLI"},"6881149":{"id":6881149,"externalId":"511145.b0293","name":"ecpA","hasFamily":false,"annotation":"cryptic Mat fimbrillin gene","uniprotId":"MATB_ECOLI"},"6881150":{"id":6881150,"externalId":"511145.b0294","name":"matA","hasFamily":false,"annotation":"putative regulator; Represses expression of the flagellar master operon flhDC, and consequently prevents flagellum biosynthesis and motility. Acts by binding to the regulatory region of the flhDC operon (Probable)","uniprotId":"MATA_ECOLI"},"6881151":{"id":6881151,"externalId":"511145.b0295","name":"ykgL","hasFamily":false,"annotation":"predicted protein","uniprotId":"YKGL_ECOLI"},"6881152":{"id":6881152,"externalId":"511145.b0296","name":"ykgM","hasFamily":true,"annotation":"rpmE (L31) paralog","uniprotId":"RL31B_ECOLI"},"6881153":{"id":6881153,"externalId":"511145.b0298","name":"b0298","hasFamily":true,"annotation":"IS3 transposase A"},"6881154":{"id":6881154,"externalId":"511145.b0299","name":"b0299","hasFamily":true,"annotation":"IS3 transposase B"},"6881155":{"id":6881155,"externalId":"511145.b0301","name":"ykgB","hasFamily":true,"annotation":"inner membrane protein, DUF417 family","uniprotId":"YKGB_ECOLI"},"6881156":{"id":6881156,"externalId":"511145.b0303","name":"ykgI","hasFamily":true,"annotation":"predicted protein","uniprotId":"YKGI_ECOLI"},"6881157":{"id":6881157,"externalId":"511145.b0304","name":"ykgC","hasFamily":true,"annotation":"putative pyridine nucleotide-disulfide oxidoreductase","uniprotId":"YKGC_ECOLI"},"6881158":{"id":6881158,"externalId":"511145.b0305","name":"ykgD","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YKGD_ECOLI"},"6881159":{"id":6881159,"externalId":"511145.b0306","name":"ykgE","hasFamily":true,"annotation":"putative oxidoreductase","uniprotId":"YKGE_ECOLI"},"6881160":{"id":6881160,"externalId":"511145.b0307","name":"ykgF","hasFamily":true,"annotation":"putative electron transport protein with ferridoxin-like domai","uniprotId":"YKGF_ECOLI"},"6881161":{"id":6881161,"externalId":"511145.b0308","name":"ykgG","hasFamily":true,"annotation":"putative transporter","uniprotId":"YKGG_ECOLI"},"6881162":{"id":6881162,"externalId":"511145.b0310","name":"ykgH","hasFamily":false,"annotation":"putative inner membrane protein","uniprotId":"YKGH_ECOLI"},"6881163":{"id":6881163,"externalId":"511145.b0311","name":"betA","hasFamily":true,"annotation":"choline dehydrogenase, a flavoprotein; Can catalyze the oxidation of choline to betaine aldehyde and betaine aldehyde to glycine betaine at the same rate","uniprotId":"BETA_ECOLI"},"6881164":{"id":6881164,"externalId":"511145.b0312","name":"betB","hasFamily":true,"annotation":"betaine aldehyde dehydrogenase, NAD-dependent","uniprotId":"BETB_ECOLI"},"6881165":{"id":6881165,"externalId":"511145.b0313","name":"betI","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor involved in choline regulation of the bet genes (By similarity)","uniprotId":"BETI_ECOLI"},"6881166":{"id":6881166,"externalId":"511145.b0314","name":"betT","hasFamily":true,"annotation":"choline transporter of high affinity; High-affinity uptake of choline driven by a proton- motive force","uniprotId":"BETT_ECOLI"},"6881167":{"id":6881167,"externalId":"511145.b0315","name":"yahA","hasFamily":false,"annotation":"c-di-GMP-specific phosphodiesterase; Hydrolyzes c-di-GMP (cyclic bis(3'-5') dimeric GMP) to 5'-pGpG, known as PDE-A activity. PDE-B activity, that is hydrolysis of 5'-pGpG to GMP, proceeds only very slowly","uniprotId":"YAHA_ECOLI"},"6881168":{"id":6881168,"externalId":"511145.b0316","name":"yahB","hasFamily":false,"annotation":"putative DNA-bindng transcriptional regulator","uniprotId":"YAHB_ECOLI"},"6881169":{"id":6881169,"externalId":"511145.b0317","name":"yahC","hasFamily":false,"annotation":"putative inner membrane protein","uniprotId":"YAHC_ECOLI"},"6881170":{"id":6881170,"externalId":"511145.b0318","name":"yahD","hasFamily":true,"annotation":"ankyrin repeat protein","uniprotId":"YAHD_ECOLI"},"6881171":{"id":6881171,"externalId":"511145.b0319","name":"yahE","hasFamily":false,"annotation":"predicted protein","uniprotId":"YAHE_ECOLI"},"6881172":{"id":6881172,"externalId":"511145.b0320","name":"yahF","hasFamily":false,"annotation":"putative acyl-CoA synthetase with NAD(P)-binding domain and succinyl-CoA synthetase domain","uniprotId":"YAHF_ECOLI"},"6881173":{"id":6881173,"externalId":"511145.b0321","name":"yahG","hasFamily":true,"annotation":"conserved protein","uniprotId":"YAHG_ECOLI"},"6881174":{"id":6881174,"externalId":"511145.b0323","name":"yahI","hasFamily":false,"annotation":"carbamate kinase-like protein","uniprotId":"ARCM_ECOLI"},"6881175":{"id":6881175,"externalId":"511145.b0324","name":"yahJ","hasFamily":true,"annotation":"putative deaminase with metallo-dependent hydrolase domain","uniprotId":"YAHJ_ECOLI"},"6881176":{"id":6881176,"externalId":"511145.b0325","name":"yahK","hasFamily":true,"annotation":"putative oxidoreductase, Zn-dependent and NAD(P)-binding","uniprotId":"YAHK_ECOLI"},"6881177":{"id":6881177,"externalId":"511145.b0326","name":"yahL","hasFamily":false,"annotation":"predicted protein","uniprotId":"YAHL_ECOLI"},"6881178":{"id":6881178,"externalId":"511145.b0327","name":"yahM","hasFamily":false,"annotation":"predicted protein","uniprotId":"YAHM_ECOLI"},"6881179":{"id":6881179,"externalId":"511145.b0328","name":"yahN","hasFamily":true,"annotation":"amino acid exporter for proline, lysine, glutamate, homoserine","uniprotId":"YAHN_ECOLI"},"6881180":{"id":6881180,"externalId":"511145.b0329","name":"yahO","hasFamily":true,"annotation":"periplasmic protein, function unknown, YhcN family","uniprotId":"YAHO_ECOLI"},"6881181":{"id":6881181,"externalId":"511145.b0330","name":"prpR","hasFamily":true,"annotation":"propionate catabolism operon regulatory protein; Involved in the transcriptional regulation of the propionate catabolism operon","uniprotId":"PRPR_ECOLI"},"6881182":{"id":6881182,"externalId":"511145.b0331","name":"ECs0385","hasFamily":true,"annotation":"2-methylisocitrate lyase; Catalyzes the formation of pyruvate and succinate from 2-methylisocitrate (By similarity)","uniprotId":"PRPB_ECOLI"},"6881183":{"id":6881183,"externalId":"511145.b0333","name":"prpC","hasFamily":true,"annotation":"2-methylcitrate synthase; Catalyzes the synthesis of 2-methylcitrate from propionyl-CoA and oxaloacetate. Also catalyzes the condensation of oxaloacetate with acetyl-CoA but with a lower specificity","uniprotId":"PRPC_ECOLI"},"6881184":{"id":6881184,"externalId":"511145.b0334","name":"prpD","hasFamily":true,"annotation":"2-methylcitrate dehydratase; Catalyzes the dehydration of 2-methylcitrate to 2- methyl-cis-aconitate. Also seems to be responsible for the residual aconitase activity of the acnAB-null strain","uniprotId":"PRPD_ECOLI"},"6881185":{"id":6881185,"externalId":"511145.b0335","name":"prpE","hasFamily":true,"annotation":"propionate--CoA ligase; Catalyzes the synthesis of propionyl-CoA from propionate and CoA. Also converts acetate to acetyl-CoA but with a lower specific activity (By similarity)","uniprotId":"PRPE_ECOLI"},"6881186":{"id":6881186,"externalId":"511145.b0336","name":"codB","hasFamily":true,"annotation":"cytosine transporter; Required for cytosine transport into the cell","uniprotId":"CODB_ECOLI"},"6881187":{"id":6881187,"externalId":"511145.b0337","name":"codA","hasFamily":true,"annotation":"cytosine/isoguanine deaminase","uniprotId":"CODA_ECOLI"},"6881188":{"id":6881188,"externalId":"511145.b0338","name":"cynR","hasFamily":true,"annotation":"transcriptional activator of cyn operon; autorepressor; Positively regulates the cynTSX operon, and negatively regulates its own transcription. Binds specifically to the cynR- cynTSX intergenic region","uniprotId":"CYNR_ECOLI"},"6881189":{"id":6881189,"externalId":"511145.b0339","name":"cynT","hasFamily":true,"annotation":"carbonic anhydrase; Reversible hydration of carbon dioxide. Carbon dioxide formed in the bicarbonate-dependent decomposition of cyanate by cyanase (CynS) diffuses out of the cell faster than it would be hydrated to bicarbonate, so the apparent function of this enzyme is to catalyze the hydration of carbon dioxide and thus prevent depletion of cellular bicarbonate","uniprotId":"CYNT_ECOLI"},"6881190":{"id":6881190,"externalId":"511145.b0340","name":"cynS","hasFamily":true,"annotation":"cyanate aminohydrolase; Catalyzes the reaction of cyanate with bicarbonate to produce ammonia and carbon dioxide (By similarity)","uniprotId":"CYNS_ECOLI"},"6881191":{"id":6881191,"externalId":"511145.b0341","name":"cynX","hasFamily":true,"annotation":"putative cyanate transporter; This protein is part of an active transport system that transports exogenous cyanate into E.coli cells","uniprotId":"CYNX_ECOLI"},"6881192":{"id":6881192,"externalId":"511145.b0342","name":"lacA","hasFamily":true,"annotation":"thiogalactoside acetyltransferase; May assist cellular detoxification by acetylating non- metabolizable pyranosides, thereby preventing their reentry into the cell","uniprotId":"B8LFD8_ECOLI"},"6881193":{"id":6881193,"externalId":"511145.b0343","name":"lacY","hasFamily":false,"annotation":"lactose permease; Responsible for transport of beta-galactosides into the cell, with the concomitant import of a proton (symport system)","uniprotId":"B8LFD7_ECOLI"},"6881194":{"id":6881194,"externalId":"511145.b0344","name":"lacZ","hasFamily":true,"annotation":"beta-D-galactosidase","uniprotId":"B8LFD6_ECOLI"},"6881195":{"id":6881195,"externalId":"511145.b0345","name":"lacI","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor of the lactose operon. Binds allolactose as an inducer","uniprotId":"B8LFD5_ECOLI"},"6881196":{"id":6881196,"externalId":"511145.b0346","name":"mhpR","hasFamily":false,"annotation":"DNA-binding transcriptional activator, 3HPP-binding; Activator of the mhpRABCDEF operon coding for components of the 3-hydroxyphenylpropionate degradation pathway","uniprotId":"MHPR_ECOLI"},"6881197":{"id":6881197,"externalId":"511145.b0347","name":"mhpA","hasFamily":true,"annotation":"3-(3-hydroxyphenyl)propionate hydroxylase; Catalyzes the insertion of one atom of molecular oxygen into position 2 of the phenyl ring of 3-(3- hydroxyphenyl)propionate (3-HPP) and hydroxycinnamic acid (3HCI) (By similarity)","uniprotId":"MHPA_ECOLI"},"6881198":{"id":6881198,"externalId":"511145.b0348","name":"mhpB","hasFamily":false,"annotation":"2,3-dihydroxyphenylpropionate 1,2-dioxygenase; Catalyzes the non-heme iron(II)-dependent oxidative cleavage of 2,3-dihydroxyphenylpropionic acid and 2,3- dihydroxicinnamic acid into 2-hydroxy-6-ketononadienedioate and 2- hydroxy-6-ketononatrienedioate, respectively (By similarity)","uniprotId":"MHPB_ECOLI"},"6881199":{"id":6881199,"externalId":"511145.b0349","name":"mhpC","hasFamily":true,"annotation":"2-hydroxy-6-ketonona-2,4-dienedioic acid hydrolase; Catalyzes the cleavage of the C5-C6 bond of 2-hydroxy-6- oxononadienedioate and 2-hydroxy-6-oxononatrienedioate, a dienol ring fission product of the bacterial meta-cleavage pathway for degradation of phenylpropionic acid. MhpC shows some selectivity for the carboxylate of the side chain","uniprotId":"MHPC_ECOLI"},"6881200":{"id":6881200,"externalId":"511145.b0350","name":"mhpD","hasFamily":false,"annotation":"2-keto-4-pentenoate hydratase; Catalyzes the conversion of 2-hydroxypentadienoic acid (enolic form of 2-oxopent-4-enoate) to 4-hydroxy-2-ketopentanoic acid (By similarity)","uniprotId":"MHPD_ECOLI"},"6881201":{"id":6881201,"externalId":"511145.b0351","name":"mhpF","hasFamily":true,"annotation":"acetaldehyde-CoA dehydrogenase II, NAD-binding; Catalyzes the conversion of acetaldehyde to acetyl-CoA, using NAD(+) and coenzyme A. Is the final enzyme in the meta- cleavage pathway for the degradation of 3-phenylpropanoate. Functions as a chaperone protein for folding of MhpE","uniprotId":"ACDH_ECOLI"},"6881202":{"id":6881202,"externalId":"511145.b0352","name":"mhpE","hasFamily":true,"annotation":"4-hyroxy-2-oxovalerate/4-hydroxy-2-oxopentanoic acid aldolase, class I; Catalyzes the retro-aldol cleavage of 4-hydroxy-2- oxopentanoate to pyruvate and acetaldehyde. Is involved in the meta-cleavage pathway for the degradation of aromatic compounds (By similarity)","uniprotId":"HOA_ECOLI"},"6881203":{"id":6881203,"externalId":"511145.b0353","name":"mhpT","hasFamily":true,"annotation":"putative 3-hydroxyphenylpropionic transporter; Could be a transporter for 3-phenylpropionate (hydrocinnamic acid)","uniprotId":"MHPT_ECOLI"},"6881204":{"id":6881204,"externalId":"511145.b0354","name":"yaiL","hasFamily":true,"annotation":"predicted protein","uniprotId":"YAIL_ECOLI"},"6881205":{"id":6881205,"externalId":"511145.b0355","name":"frmB","hasFamily":true,"annotation":"S-formylglutathione hydrolase; Serine hydrolase involved in the detoxification of formaldehyde. Hydrolyzes S-formylglutathione to glutathione and formate. Shows also esterase activity against two pNP-esters (pNP- acetate and pNP-propionate), alpha-naphthyl acetate and lactoylglutathione","uniprotId":"SFGH1_ECOLI"},"6881206":{"id":6881206,"externalId":"511145.b0356","name":"adhC","hasFamily":true,"annotation":"alcohol dehydrogenase class III/glutathione-dependent formaldehyde dehydrogenase; Has high formaldehyde dehydrogenase activity in the presence of glutathione and catalyzes the oxidation of normal alcohols in a reaction that is not GSH-dependent. In addition, hemithiolacetals other than those formed from GSH, including omega-thiol fatty acids, also are substrates","uniprotId":"FRMA_ECOLI"},"6881207":{"id":6881207,"externalId":"511145.b0357","name":"frmR","hasFamily":true,"annotation":"regulator protein that represses frmRAB operon; Repressor of the frmRAB operon","uniprotId":"FRMR_ECOLI"},"6881208":{"id":6881208,"externalId":"511145.b0358","name":"yaiO","hasFamily":false,"annotation":"outer membrane protein","uniprotId":"YAIO_ECOLI"},"6881209":{"id":6881209,"externalId":"511145.b0360","name":"b0360","hasFamily":true,"annotation":"IS2 repressor TnpA"},"6881210":{"id":6881210,"externalId":"511145.b0361","name":"b0361","hasFamily":true,"annotation":"IS2 transposase TnpB"},"6881211":{"id":6881211,"externalId":"511145.b0363","name":"yaiP","hasFamily":false,"annotation":"putative glucosyltransferase","uniprotId":"YAIP_ECOLI"},"6881212":{"id":6881212,"externalId":"511145.b0364","name":"yaiS","hasFamily":false,"annotation":"conserved protein","uniprotId":"YAIS_ECOLI"},"6881213":{"id":6881213,"externalId":"511145.b0365","name":"tauA","hasFamily":true,"annotation":"taurine transporter subunit; Part of a binding-protein-dependent transport system for taurine","uniprotId":"TAUA_ECOLI"},"6881214":{"id":6881214,"externalId":"511145.b0366","name":"tauB","hasFamily":true,"annotation":"taurine transporter subunit; Part of the ABC transporter complex TauABC involved in taurine import. Responsible for energy coupling to the transport system","uniprotId":"TAUB_ECOLI"},"6881215":{"id":6881215,"externalId":"511145.b0367","name":"tauC","hasFamily":true,"annotation":"taurine transporter subunit; Part of a binding-protein-dependent transport system for taurine. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"TAUC_ECOLI"},"6881216":{"id":6881216,"externalId":"511145.b0368","name":"tauD","hasFamily":true,"annotation":"taurine dioxygenase, 2-oxoglutarate-dependent; Catalyzes the conversion of taurine and alpha ketoglutarate to sulfite, aminoacetaldehyde and succinate. Required for the utilization of taurine (2-aminoethanesulfonic acid) as an alternative sulfur source. Pentane-sulfonic acid, 3- (N-morpholino)propanesulfonic acid and 1,3-dioxo-2- isoindolineethanesulfonic acid are also substrates for this enzyme","uniprotId":"TAUD_ECOLI"},"6881217":{"id":6881217,"externalId":"511145.b0369","name":"hemB","hasFamily":true,"annotation":"5-aminolevulinate dehydratase (porphobilinogen synthase); Catalyzes an early step in the biosynthesis of tetrapyrroles. Binds two molecules of 5-aminolevulinate per subunit, each at a distinct site, and catalyzes their condensation to form porphobilinogen","uniprotId":"HEM2_ECOLI"},"6881218":{"id":6881218,"externalId":"511145.b0372","name":"b0372","hasFamily":true,"annotation":"IS3 transposase B"},"6881219":{"id":6881219,"externalId":"511145.b0373","name":"b0373","hasFamily":true,"annotation":"IS3 transposase A"},"6881220":{"id":6881220,"externalId":"511145.b0375","name":"yaiV","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YAIV_ECOLI"},"6881221":{"id":6881221,"externalId":"511145.b0376","name":"ampH","hasFamily":true,"annotation":"penicillin-binding protein; Hydrolyzes the cross-linked dimers tetrapentapeptide (D45) and tetratetrapeptide (D44). Removes the terminal D-alanine from muropeptides and disaccharide pentapeptide M5 with a C- terminal D-Ala-D-Ala dipeptide. Associated with recycling and remodeling of peptidoglycan (PG). Also displays a low beta- lactamase activity","uniprotId":"AMPH_ECOLI"},"6881222":{"id":6881222,"externalId":"511145.b0377","name":"sbmA","hasFamily":true,"annotation":"microcin B17 transporter; Uptake of antimicrobial peptides. Required for the transport of microcin B17 (MccB17), microcin 25 (Mcc25) and proline-rich antimicrobial peptides into the cell","uniprotId":"SBMA_ECOLI"},"6881223":{"id":6881223,"externalId":"511145.b0378","name":"yaiW","hasFamily":true,"annotation":"probable lipoprotein required for swarming phenotype","uniprotId":"YAIW_ECOLI"},"6881224":{"id":6881224,"externalId":"511145.b0379","name":"yaiY","hasFamily":true,"annotation":"inner membrane protein, DUF2755 family","uniprotId":"YAIY_ECOLI"},"6881225":{"id":6881225,"externalId":"511145.b0380","name":"yaiZ","hasFamily":true,"annotation":"putative inner membrane protein, DUF2754 family","uniprotId":"YAIZ_ECOLI"},"6881226":{"id":6881226,"externalId":"511145.b0381","name":"ddlA","hasFamily":true,"annotation":"D-alanine-D-alanine ligase A; Cell wall formation (By similarity)","uniprotId":"DDLA_ECOLI"},"6881227":{"id":6881227,"externalId":"511145.b0382","name":"iraP","hasFamily":true,"annotation":"anti-RssB factor, RpoS stabilzer during Pi starvation; Inhibits RpoS proteolysis by regulating RssB activity, thereby increasing the stability of the sigma stress factor RpoS especially during phosphate starvation, but also in stationary phase and during nitrogen starvation. Its effect on RpoS stability is due to its interaction with RssB, which probably blocks the interaction of RssB with RpoS, and the consequent delivery of the RssB-RpoS complex to the ClpXP protein degradation pathway (By similarity)","uniprotId":"IRAP_ECOLI"},"6881228":{"id":6881228,"externalId":"511145.b0383","name":"phoA","hasFamily":true,"annotation":"bacterial alkaline phosphatase","uniprotId":"B8YM76_ECOLI"},"6881229":{"id":6881229,"externalId":"511145.b0384","name":"psiF","hasFamily":true,"annotation":"conserved protein, PsiF family, pho regulon","uniprotId":"PSIF_ECOLI"},"6881230":{"id":6881230,"externalId":"511145.b0385","name":"adrA","hasFamily":true,"annotation":"diguanylate cyclase, cellulose regualtor; A probable diguanylate cyclase. The last member of a cascade of expressed proteins, its expression requires YdaM. AdrA production induces biosynthesis of cellulose in some E.coli isolates, but not in K12 strains. Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"ADRA_ECOLI"},"6881231":{"id":6881231,"externalId":"511145.b0386","name":"proC","hasFamily":true,"annotation":"pyrroline-5-carboxylate reductase, NAD(P)-binding","uniprotId":"P5CR_ECOLI"},"6881232":{"id":6881232,"externalId":"511145.b0387","name":"yaiI","hasFamily":true,"annotation":"conserved protein, UPF0178 family, downregulated by beryllium","uniprotId":"YAII_ECOLI"},"6881233":{"id":6881233,"externalId":"511145.b0388","name":"aroL","hasFamily":true,"annotation":"shikimate kinase II; Catalyzes the specific phosphorylation of the 3-hydroxyl group of shikimic acid using ATP as a cosubstrate (By similarity)","uniprotId":"AROL_ECOLI"},"6881234":{"id":6881234,"externalId":"511145.b0389","name":"yaiA","hasFamily":true,"annotation":"predicted protein","uniprotId":"YAIA_ECOLI"},"6881235":{"id":6881235,"externalId":"511145.b0390","name":"aroM","hasFamily":true,"annotation":"conserved protein, AroM family; This protein of unknown function is encoded by a gene that cotranscribes with the aroL gene, which codes for shikimate kinase II","uniprotId":"AROM_ECOLI"},"6881236":{"id":6881236,"externalId":"511145.b0391","name":"yaiE","hasFamily":true,"annotation":"conserved protein, UPF0345 family","uniprotId":"YAIE_ECOLI"},"6881237":{"id":6881237,"externalId":"511145.b0393","name":"rdgC","hasFamily":true,"annotation":"nucleoid-associated ssDNA and dsDNA binding protein; competitive inhibitor of RecA function; May be involved in recombination (By similarity)","uniprotId":"RDGC_ECOLI"},"6881238":{"id":6881238,"externalId":"511145.b0394","name":"mak","hasFamily":true,"annotation":"manno(fructo)kinase; Catalyzes the phosphorylation of fructose to fructose-6- P. Has also low level glucokinase activity in vitro. Is not able to phosphorylate D-ribose, D-mannitol, D-sorbitol, inositol, and L-threonine","uniprotId":"MAK_ECOLI"},"6881239":{"id":6881239,"externalId":"511145.b0396","name":"araJ","hasFamily":true,"annotation":"arabinose-inducible predicted transporter, MFS family; May be involved in either the transport or processing of arabinose polymers","uniprotId":"ARAJ_ECOLI"},"6881240":{"id":6881240,"externalId":"511145.b0397","name":"sbcC","hasFamily":true,"annotation":"exonuclease, dsDNA, ATP-dependent; SbcCD cleaves DNA hairpin structures. These structures can inhibit DNA replication and are intermediates in certain DNA recombination reactions. The complex acts as a 3'->5' double strand exonuclease that can open hairpins. It also has a 5' single-strand endonuclease activity","uniprotId":"SBCC_ECOLI"},"6881241":{"id":6881241,"externalId":"511145.b0398","name":"sbcD","hasFamily":true,"annotation":"exonuclease, dsDNA, ATP-dependent; SbcCD cleaves DNA hairpin structures. These structures can inhibit DNA replication and are intermediates in certain DNA recombination reactions. The complex acts as a 3'->5' double strand exonuclease that can open hairpins. It also has a 5' single-strand endonuclease activity","uniprotId":"SBCD_ECOLI"},"6881242":{"id":6881242,"externalId":"511145.b0399","name":"phoB","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with PhoR (or CreC); This protein is a positive regulator for the phosphate regulon. Transcription of this operon is positively regulated by PhoB and PhoR when phosphate is limited","uniprotId":"PHOB_ECOLI"},"6881243":{"id":6881243,"externalId":"511145.b0400","name":"phoR","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with PhoB; Member of the two-component regulatory system PhoR/PhoB involved in the phosphate regulon genes expression. PhoR may function as a membrane-associated protein kinase that phosphorylates PhoB in response to environmental signals","uniprotId":"PHOR_ECOLI"},"6881244":{"id":6881244,"externalId":"511145.b0401","name":"brnQ","hasFamily":true,"annotation":"Branched-chain amino acid transport system 2 carrier protein; LIV-II transport system for Ile, Leu, and Val; Component of the LIV-II transport system for branched- chain amino acids. This LIV-II transport system may be H(+)- coupled","uniprotId":"BRNQ_ECOLI"},"6881245":{"id":6881245,"externalId":"511145.b0402","name":"proY","hasFamily":true,"annotation":"proline-specific permease; Permease that is involved in the transport across the cytoplasmic membrane of proline","uniprotId":"PROY_ECOLI"},"6881246":{"id":6881246,"externalId":"511145.b0403","name":"malZ","hasFamily":true,"annotation":"maltodextrin glucosidase; May play a role in regulating the intracellular level of maltotriose. Cleaves glucose from the reducing end of maltotriose and longer maltodextrins with a chain length of up to 7 glucose units","uniprotId":"MALZ_ECOLI"},"6881247":{"id":6881247,"externalId":"511145.b0404","name":"acpH","hasFamily":true,"annotation":"conserved protein; Converts holo-ACP to apo-ACP by hydrolytic cleavage of the phosphopantetheine prosthetic group from ACP (By similarity)","uniprotId":"ACPH_ECOLI"},"6881248":{"id":6881248,"externalId":"511145.b0405","name":"queA","hasFamily":true,"annotation":"S-adenosylmethionine:tRNA ribosyltransferase-isomerase; Transfers and isomerizes the ribose moiety from AdoMet to the 7-aminomethyl group of 7-deazaguanine (preQ1-tRNA) to give epoxyqueuosine (oQ-tRNA) (By similarity)","uniprotId":"QUEA_ECOLI"},"6881249":{"id":6881249,"externalId":"511145.b0406","name":"tgt","hasFamily":true,"annotation":"tRNA-guanine transglycosylase; Exchanges the guanine residue with 7-aminomethyl-7- deazaguanine in tRNAs with GU(N) anticodons (tRNA-Asp, -Asn, -His and -Tyr). After this exchange, a cyclopentendiol moiety is attached to the 7-aminomethyl group of 7-deazaguanine, resulting in the hypermodified nucleoside queuosine (Q) (7-(((4,5-cis- dihydroxy-2-cyclopenten-1-yl)amino)methyl)-7-deazaguanosine) (By similarity)","uniprotId":"TGT_ECOLI"},"6881250":{"id":6881250,"externalId":"511145.b0407","name":"yajC","hasFamily":true,"annotation":"SecYEG protein translocase auxillary subunit","uniprotId":"YAJC_ECOLI"},"6881251":{"id":6881251,"externalId":"511145.b0408","name":"secD","hasFamily":true,"annotation":"SecYEG protein translocase auxillary subunit; Part of the Sec protein translocase complex. Interacts with the SecYEG preprotein conducting channel. SecDF uses the proton motive force (PMF) to complete protein translocation after the ATP-dependent function of SecA. The large periplasmic domain is thought to have a base and head domain joined by a hinge; movement of the hinge may be coupled to both proton transport and protein export, with the head domain capturing substrate, and a conformational change preventing backward movement and driving forward movement. Expression of V.alginolyti [...] ","uniprotId":"SECD_ECOLI"},"6881252":{"id":6881252,"externalId":"511145.b0409","name":"secF","hasFamily":true,"annotation":"SecYEG protein translocase auxillary subunit; Part of the Sec protein translocase complex. Interacts with the SecYEG preprotein conducting channel. SecDF uses the proton motive force (PMF) to complete protein translocation after the ATP-dependent function of SecA. The large periplasmic domain is thought to have a base and head domain joined by a hinge; movement of the hinge may be coupled to both proton transport and protein export, with the head domain capturing substrate, and a conformational change preventing backward movement and driving forward movement. Expression of V.alginolyti [...] ","uniprotId":"SECF_ECOLI"},"6881253":{"id":6881253,"externalId":"511145.b0410","name":"yajD","hasFamily":true,"annotation":"conserved protein, HNH family","uniprotId":"YAJD_ECOLI"},"6881254":{"id":6881254,"externalId":"511145.b0411","name":"tsx","hasFamily":true,"annotation":"nucleoside channel, receptor of phage T6 and colicin K; Constitutes the receptor for colicin K and phage T6, and functions as substrate-specific channel for nucleosides and deoxynucleosides","uniprotId":"TSX_ECOLI"},"6881255":{"id":6881255,"externalId":"511145.b0412","name":"yajI","hasFamily":true,"annotation":"putative lipoprotein","uniprotId":"YAJI_ECOLI"},"6881256":{"id":6881256,"externalId":"511145.b0413","name":"nrdR","hasFamily":true,"annotation":"conserved protein; Represses transcription of the class Ib RNR genes nrdHIEF but has much smaller effect on transcription of the class Ia RNR genes nrdAB and class III RNR genes nrdDG. By binding to nrdR boxes in the promoter regions to alter promoter activity, nrdR differentially regulates nrdAB, nrdHIEF and nrdD transcription in aerobic growth","uniprotId":"NRDR_ECOLI"},"6881257":{"id":6881257,"externalId":"511145.b0414","name":"ribD","hasFamily":true,"annotation":"fused diaminohydroxyphosphoribosylaminopyrimidine deaminase and 5-amino-6-(5-phosphoribosylamino) uracil reductase; Converts 2,5-diamino-6-(ribosylamino)-4(3h)-pyrimidinone 5'-phosphate into 5-amino-6-(ribosylamino)-2,4(1h,3h)- pyrimidinedione 5'-phosphate (By similarity)","uniprotId":"RIBD_ECOLI"},"6881258":{"id":6881258,"externalId":"511145.b0415","name":"ribE","hasFamily":true,"annotation":"riboflavin synthase beta chain; Catalyzes the formation of 6,7-dimethyl-8- ribityllumazine by condensation of 5-amino-6-(D- ribitylamino)uracil with 3,4-dihydroxy-2-butanone 4-phosphate. This is the penultimate step in the biosynthesis of riboflavin (By similarity)","uniprotId":"RISB_ECOLI"},"6881259":{"id":6881259,"externalId":"511145.b0416","name":"nusB","hasFamily":true,"annotation":"transcription antitermination protein; One of the proteins essential for the formation of the RNA polymerase antitermination complex in the presence of lambda phage N protein. However, it is involved in the transcription termination process at certain sites during normal bacterial growth. Binds to the BoxA RNA motif","uniprotId":"NUSB_ECOLI"},"6881260":{"id":6881260,"externalId":"511145.b0417","name":"thiL","hasFamily":true,"annotation":"thiamin-monophosphate kinase; Catalyzes the ATP-dependent phosphorylation of thiamine- monophosphate (TMP) to form thiamine-pyrophosphate (TPP), the active form of vitamin B1. Can not use thiamine as substrate. Is highly specific for ATP as phosphate donor","uniprotId":"THIL_ECOLI"},"6881261":{"id":6881261,"externalId":"511145.b0418","name":"pgpA","hasFamily":true,"annotation":"phosphatidylglycerophosphatase A; One of the three phospholipid phosphatases, specifically hydrolyzes phosphatidylglycerophosphate","uniprotId":"PGPA_ECOLI"},"6881262":{"id":6881262,"externalId":"511145.b0419","name":"yajO","hasFamily":true,"annotation":"2-carboxybenzaldehyde reductase","uniprotId":"YAJO_ECOLI"},"6881263":{"id":6881263,"externalId":"511145.b0420","name":"dxs","hasFamily":true,"annotation":"1-deoxyxylulose-5-phosphate synthase, thiamine-requiring, FAD-requiring; Catalyzes the acyloin condensation reaction between C atoms 2 and 3 of pyruvate and glyceraldehyde 3-phosphate to yield 1-deoxy-D-xylulose-5-phosphate (DXP) (By similarity)","uniprotId":"DXS_ECOLI"},"6881264":{"id":6881264,"externalId":"511145.b0421","name":"ispA","hasFamily":true,"annotation":"geranyltranstransferase","uniprotId":"ISPA_ECOLI"},"6881265":{"id":6881265,"externalId":"511145.b0422","name":"xseB","hasFamily":true,"annotation":"exonuclease VII small subunit; Bidirectionally degrades single-stranded DNA into large acid-insoluble oligonucleotides, which are then degraded further into small acid-soluble oligonucleotides. It can also degrade 3' or 5' ss regions extending from the termini of duplex DNA molecules and displaced ss regions","uniprotId":"EX7S_ECOLI"},"6881266":{"id":6881266,"externalId":"511145.b0423","name":"thiI","hasFamily":true,"annotation":"tRNA s(4)U8 sulfurtransferase; Catalyzes the ATP-dependent transfer of a sulfur to tRNA to produce 4-thiouridine in position 8 of tRNAs, which functions as a near-UV photosensor. Also catalyzes the transfer of sulfur to the sulfur carrier protein ThiS, forming ThiS-thiocarboxylate. This is a step in the synthesis of thiazole, in the thiamine biosynthesis pathway. The sulfur is donated as persulfide by IscS (By similarity)","uniprotId":"THII_ECOLI"},"6881267":{"id":6881267,"externalId":"511145.b0424","name":"thiJ","hasFamily":true,"annotation":"Oxidative-stress-resistance chaperone; Involved in biogenesis of ribosomal proteins, probably as a ribosomal protein-folding chaperone. Interacts with ribosomal subunits, ribosomes and polysomes. Confers resistance to oxidative stress","uniprotId":"YAJL_ECOLI"},"6881268":{"id":6881268,"externalId":"511145.b0425","name":"apbA","hasFamily":true,"annotation":"2-dehydropantoate reductase, NADPH-specific; Catalyzes the NADPH-dependent reduction of ketopantoate into pantoic acid (By similarity)","uniprotId":"PANE_ECOLI"},"6881269":{"id":6881269,"externalId":"511145.b0426","name":"yajQ","hasFamily":true,"annotation":"Phi6 host factor; binds ATP and GTP; Binds nucleotides, may bind tRNA","uniprotId":"YAJQ_ECOLI"},"6881270":{"id":6881270,"externalId":"511145.b0427","name":"yajR","hasFamily":true,"annotation":"putative transporter","uniprotId":"YAJR_ECOLI"},"6881271":{"id":6881271,"externalId":"511145.b0428","name":"cyoE","hasFamily":true,"annotation":"protoheme IX farnesyltransferase; Converts heme B (protoheme IX) to heme O by substitution of the vinyl group on carbon 2 of heme B porphyrin ring with a hydroxyethyl farnesyl side group (By similarity)","uniprotId":"CYOE_ECOLI"},"6881272":{"id":6881272,"externalId":"511145.b0429","name":"cyoD","hasFamily":true,"annotation":"cytochrome o ubiquinol oxidase subunit IV; Cytochrome o terminal oxidase complex is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at high aeration","uniprotId":"CYOD_ECOLI"},"6881273":{"id":6881273,"externalId":"511145.b0430","name":"cyoC","hasFamily":true,"annotation":"cytochrome o ubiquinol oxidase subunit III; Cytochrome o terminal oxidase complex is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at high aeration","uniprotId":"CYOC_ECOLI"},"6881274":{"id":6881274,"externalId":"511145.b0431","name":"cyoB","hasFamily":true,"annotation":"cytochrome o ubiquinol oxidase subunit I; Cytochrome o terminal oxidase complex is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at high aeration. This ubiquinol oxidase shows proton pump activity across the membrane in addition to the electron transfer","uniprotId":"CYOB_ECOLI"},"6881275":{"id":6881275,"externalId":"511145.b0432","name":"cyoA","hasFamily":true,"annotation":"cytochrome o ubiquinol oxidase subunit II; Cytochrome o terminal oxidase complex is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at high aeration","uniprotId":"CYOA_ECOLI"},"6881276":{"id":6881276,"externalId":"511145.b0433","name":"ampG","hasFamily":true,"annotation":"muropeptide transporter; Probably acts as a permease in the beta-lactamase induction system and in peptidoglycan recycling","uniprotId":"AMPG_ECOLI"},"6881277":{"id":6881277,"externalId":"511145.b0434","name":"yajG","hasFamily":true,"annotation":"putative lipoprotein","uniprotId":"YAJG_ECOLI"},"6881278":{"id":6881278,"externalId":"511145.b0435","name":"bolA","hasFamily":true,"annotation":"stationary-phase morphogene, transcriptional repressor for mreB; also regulator for dacA, dacC, and ampC; Plays an important role in general stress response (heat shock, acidic stress, oxidative stress, carbon-starvation stress and osmotic shock). Involved in morphogenetic pathway. Has a role in the induction of the expression of dacA (PBP5), dacC (PBP6) and ampC","uniprotId":"BOLA_ECOLI"},"6881279":{"id":6881279,"externalId":"511145.b0436","name":"tig","hasFamily":true,"annotation":"peptidyl-prolyl cis/trans isomerase (trigger factor); Involved in protein export. Acts as a chaperone by maintaining the newly synthesized secretory and non-secretory proteins in an open conformation. Binds to nascent polypeptide chains via ribosomal protein L23 (PubMed:12226666). Functions as a peptidyl-prolyl cis-trans isomerase in vitro, this activity is dispensible in vivo for chaperone activity","uniprotId":"TIG_ECOLI"},"6881280":{"id":6881280,"externalId":"511145.b0437","name":"clpP","hasFamily":true,"annotation":"proteolytic subunit of ClpA-ClpP and ClpX-ClpP ATP-dependent serine proteases; Cleaves peptides in various proteins in a process that requires ATP hydrolysis. Has a chymotrypsin-like activity. Plays a major role in the degradation of misfolded proteins. May play the role of a master protease which is attracted to different substrates by different specificity factors such as ClpA or ClpX","uniprotId":"CLPP_ECOLI"},"6881281":{"id":6881281,"externalId":"511145.b0438","name":"clpX","hasFamily":true,"annotation":"ATPase and specificity subunit of ClpX-ClpP ATP-dependent serine protease; ATP-dependent specificity component of the Clp protease. It directs the protease to specific substrates. It may bind to the lambda O substrate protein and present it to the ClpP protease in a form that can be recognized and readily hydrolyzed by ClpP. Can perform chaperone functions in the absence of ClpP","uniprotId":"CLPX_ECOLI"},"6881282":{"id":6881282,"externalId":"511145.b0439","name":"lon","hasFamily":true,"annotation":"DNA-binding ATP-dependent protease La; ATP-dependent serine protease that mediates the selective degradation of mutant and abnormal proteins as well as certain short-lived regulatory proteins, including some antitoxins. Required for cellular homeostasis and for survival from DNA damage and developmental changes induced by stress. Degrades polypeptides processively to yield small peptide fragments that are 5 to 10 amino acids long. Binds to DNA in a double-stranded, site-specific manner. Endogenous substrates include the regulatory proteins RcsA and SulA, the transcriptional activator S [...] ","uniprotId":"LON_ECOLI"},"6881283":{"id":6881283,"externalId":"511145.b0440","name":"hupB","hasFamily":true,"annotation":"HU, DNA-binding transcriptional regulator, beta subunit; Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions","uniprotId":"DBHB_ECOLI"},"6881284":{"id":6881284,"externalId":"511145.b0441","name":"ppiD","hasFamily":true,"annotation":"periplasmic folding chaperone, has an inactive PPIase domain; PPIases accelerate the folding of proteins. Seems to be involved in the folding of outer membrane proteins. Its preference at the P1 position of the peptide substrate is Glu > Leu > Ala > His > Val > Phe > Ile > Gly > Lys > Thr","uniprotId":"PPID_ECOLI"},"6881285":{"id":6881285,"externalId":"511145.b0442","name":"ybaV","hasFamily":true,"annotation":"conserved protein, ComEA homolog","uniprotId":"YBAV_ECOLI"},"6881286":{"id":6881286,"externalId":"511145.b0443","name":"fadM","hasFamily":true,"annotation":"long-chain acyl-CoA thioesterase III; Long-chain acyl-CoA thioesterase with a preference for 3,5-tetradecadienoyl-CoA. Could be involved in beta-oxidation of fatty acids","uniprotId":"FADM_ECOLI"},"6881287":{"id":6881287,"externalId":"511145.b0444","name":"queC","hasFamily":true,"annotation":"7-cyano-7-deazaguanine (preQ0) synthase; queuosine biosynthesis; Catalyzes the ATP-dependent conversion of 7-carboxy-7- deazaguanine (CDG) to 7-cyano-7-deazaguanine (preQ(0)) (By similarity)","uniprotId":"QUEC_ECOLI"},"6881288":{"id":6881288,"externalId":"511145.b0445","name":"ybaE","hasFamily":true,"annotation":"putative transporter subunit: periplasmic-binding component of ABC superfamily","uniprotId":"YBAE_ECOLI"},"6881289":{"id":6881289,"externalId":"511145.b0446","name":"cof","hasFamily":true,"annotation":"thiamin pyrimidine pyrophosphate hydrolase; HMP-PP phosphatase; Catalyzes the hydrolysis of 4-amino-2-methyl-5- hydroxymethylpyrimidine pyrophosphate (HMP-PP) to 4-amino-2- methyl-5-hydroxymethylpyrimidine phosphate (HMP-P). Can also hydrolyze other substrates such as MeO-HMP-PP and 4-amino-2- trifluoromethyl 5-hydroxymethylpyrimidine pyrophosphate (CF3-HMP- PP) to give MeO-HMP-P and 4-amino-2-trifluoromethyl-5- hydroxymethylpyrimidine phosphate. This hydrolysis generates resistance to the antibiotics (bacimethrin, CF3-HMP) by reducing the formation of their toxic forms, 2'-methoxythia [...] ","uniprotId":"COF_ECOLI"},"6881290":{"id":6881290,"externalId":"511145.b0447","name":"ybaO","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YBAO_ECOLI"},"6881291":{"id":6881291,"externalId":"511145.b0448","name":"mdlA","hasFamily":true,"annotation":"fused predicted multidrug transporter subunits of ABC superfamily: ATP-binding components","uniprotId":"MDLA_ECOLI"},"6881292":{"id":6881292,"externalId":"511145.b0449","name":"mdlB","hasFamily":true,"annotation":"fused predicted multidrug transporter subunits of ABC superfamily: ATP-binding components","uniprotId":"MDLB_ECOLI"},"6881293":{"id":6881293,"externalId":"511145.b0450","name":"glnK","hasFamily":true,"annotation":"nitrogen assimilation regulatory protein for GlnL, GlnE, and AmtB; P-II indirectly controls the transcription of the glutamine synthetase gene (glnA). P-II prevents NR-II-catalyzed conversion of NR-I to NR-I-phosphate, the transcriptional activator of GlnA. When P-II is uridylylated to P-II-UMP, these events are reversed. When the ratio of Gln to 2-ketoglutarate decreases, P-II is uridylylated to P-II-UMP, which causes the deadenylation of glutamine synthetase by GlnE, so activating the enzyme (By similarity)","uniprotId":"GLNK_ECOLI"},"6881294":{"id":6881294,"externalId":"511145.b0451","name":"amtB","hasFamily":true,"annotation":"ammonium transporter; Involved in the uptake of ammonia","uniprotId":"AMTB_ECOLI"},"6881295":{"id":6881295,"externalId":"511145.b0452","name":"tesB","hasFamily":true,"annotation":"acyl-CoA thioesterase II; Can hydrolyze a broad range of acyl-CoA thioesters. Its physiological function is not known","uniprotId":"TESB_ECOLI"},"6881296":{"id":6881296,"externalId":"511145.b0453","name":"ybaY","hasFamily":true,"annotation":"outer membrane lipoprotein","uniprotId":"YBAY_ECOLI"},"6881297":{"id":6881297,"externalId":"511145.b0454","name":"ybaZ","hasFamily":true,"annotation":"excision repair protein, alkyltransferase-like protein ATL","uniprotId":"YBAZ_ECOLI"},"6881298":{"id":6881298,"externalId":"511145.b0456","name":"ybaA","hasFamily":false,"annotation":"conserved protein, DUF1428 family","uniprotId":"YBAA_ECOLI"},"6881299":{"id":6881299,"externalId":"511145.b0457","name":"ylaB","hasFamily":true,"annotation":"predicted cyclic-di-GMP phosphodiesterase","uniprotId":"YLAB_ECOLI"},"6881300":{"id":6881300,"externalId":"511145.b0458","name":"ylaC","hasFamily":true,"annotation":"inner membrane protein, DUF1449 family","uniprotId":"YLAC_ECOLI"},"6881301":{"id":6881301,"externalId":"511145.b0459","name":"maa","hasFamily":true,"annotation":"maltose O-acetyltransferase; Acetylates maltose and other sugars","uniprotId":"MAA_ECOLI"},"6881302":{"id":6881302,"externalId":"511145.b0460","name":"hha","hasFamily":true,"annotation":"modulator of gene expression, with H-NS; Down-regulates hemolysin production and can also stimulate transposition events in vivo. Binds DNA and influences DNA topology in response to environmental stimuli. Involved in persister cell formation, acting downstream of mRNA interferase (toxin) MqsR. Decreases biofilm formation by repressing the transcription of fimbrial genes fimA and ihfA, and by repressing the transcription of tRNAs corresponding to rare codons, which are abundant in type 1 fimbrial genes","uniprotId":"HHA_ECOLI"},"6881303":{"id":6881303,"externalId":"511145.b0461","name":"tomB","hasFamily":true,"annotation":"Hha toxicity attenuator; Attenuates Hha toxicity and regulates biofilm formation. Binds to various coding and intergenic regions of genomic DNA","uniprotId":"TOMB_ECOLI"},"6881304":{"id":6881304,"externalId":"511145.b0462","name":"acrB","hasFamily":true,"annotation":"multidrug efflux system protein; AcrAB is a drug efflux protein with a broad substrate specificity","uniprotId":"ACRB_ECOLI"},"6881305":{"id":6881305,"externalId":"511145.b0463","name":"acrA","hasFamily":true,"annotation":"multidrug efflux system; AcrAB is a drug efflux protein with a broad substrate specificity","uniprotId":"ACRA_ECOLI"},"6881306":{"id":6881306,"externalId":"511145.b0464","name":"acrR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Potential regulator protein for the acrAB genes","uniprotId":"ACRR_ECOLI"},"6881307":{"id":6881307,"externalId":"511145.b0465","name":"aefA","hasFamily":true,"annotation":"fused conserved protein; Mechanosensitive channel that opens in response to stretch forces in the membrane lipid bilayer. May participate in the regulation of osmotic pressure changes within the cell, although it does not appear to have a major role in osmolarity regulation. Forms an ion channel of 1.0 nanosiemens conductance. The channel can remain active for between 30 seconds and over 3 minutes; it does not desensitize upon extended pressure. Its activity is masked in wild-type cells by the MscS channel","uniprotId":"KEFA_ECOLI"},"6881308":{"id":6881308,"externalId":"511145.b0466","name":"ybaM","hasFamily":true,"annotation":"predicted protein","uniprotId":"YBAM_ECOLI"},"6881309":{"id":6881309,"externalId":"511145.b0467","name":"priC","hasFamily":true,"annotation":"primosomal replication protein N''; PriA recognizes a specific hairpin sequence on bacteriophage phi X174 ssDNA. This structure is then recognized and bound by proteins PriB and PriC. Formation of the primosome proceeds with the subsequent actions of DnaB, DnaC, DnaT and primase","uniprotId":"PRIC_ECOLI"},"6881310":{"id":6881310,"externalId":"511145.b0468","name":"ybaN","hasFamily":true,"annotation":"inner membrane protein, DUF454 family","uniprotId":"YBAN_ECOLI"},"6881311":{"id":6881311,"externalId":"511145.b0469","name":"apt","hasFamily":true,"annotation":"adenine phosphoribosyltransferase; Catalyzes a salvage reaction resulting in the formation of AMP, that is energically less costly than de novo synthesis (By similarity)","uniprotId":"APT_ECOLI"},"6881312":{"id":6881312,"externalId":"511145.b0470","name":"dnaX","hasFamily":true,"annotation":"DNA polymerase III/DNA elongation factor III, tau and gamma subunits; DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. This DNA polymerase also exhibits 3' to 5' exonuclease activity","uniprotId":"DPO3X_ECOLI"},"6881313":{"id":6881313,"externalId":"511145.b0471","name":"ybaB","hasFamily":false,"annotation":"conserved protein; Binds to DNA and alters its conformation. May be involved in regulation of gene expression, nucleoid organization and DNA protection (By similarity)","uniprotId":"YBAB_ECOLI"},"6881314":{"id":6881314,"externalId":"511145.b0472","name":"recR","hasFamily":true,"annotation":"gap repair protein; May play a role in DNA repair. It seems to be involved in an RecBC-independent recombinational process of DNA repair. It may act with RecF and RecO (By similarity)","uniprotId":"RECR_ECOLI"},"6881315":{"id":6881315,"externalId":"511145.b0473","name":"htpG","hasFamily":true,"annotation":"protein refolding molecular co-chaperone Hsp90, Hsp70-dependent; heat-shock protein; Molecular chaperone. Has ATPase activity (By similarity)","uniprotId":"HTPG_ECOLI"},"6881316":{"id":6881316,"externalId":"511145.b0474","name":"adk","hasFamily":true,"annotation":"adenylate kinase; Catalyzes the reversible transfer of the terminal phosphate group between ATP and AMP. Plays an important role in cellular energy homeostasis and in adenine nucleotide metabolism (By similarity)","uniprotId":"KAD_ECOLI"},"6881317":{"id":6881317,"externalId":"511145.b0475","name":"hemH","hasFamily":true,"annotation":"ferrochelatase; Catalyzes the ferrous insertion into protoporphyrin IX (By similarity)","uniprotId":"HEMH_ECOLI"},"6881318":{"id":6881318,"externalId":"511145.b0476","name":"aes","hasFamily":true,"annotation":"acetyl esterase; Displays esterase activity towards short chain fatty esters (acyl chain length of up to 8 carbons). Able to hydrolyze triacetylglycerol (triacetin) and tributyrylglycerol (tributyrin), but not trioleylglycerol (triolein) or cholesterol oleate. Negatively regulates MalT activity by antagonizing maltotriose binding. Inhibits MelA galactosidase activity (By similarity)","uniprotId":"AES_ECOLI"},"6881319":{"id":6881319,"externalId":"511145.b0477","name":"gsk","hasFamily":true,"annotation":"inosine/guanosine kinase","uniprotId":"INGK_ECOLI"},"6881320":{"id":6881320,"externalId":"511145.b0478","name":"ybaL","hasFamily":true,"annotation":"putative transporter with NAD(P)-binding Rossmann-fold domain","uniprotId":"YBAL_ECOLI"},"6881321":{"id":6881321,"externalId":"511145.b0479","name":"fsr","hasFamily":true,"annotation":"putative fosmidomycin efflux system; Confers the resistance against fosmidomycin","uniprotId":"FSR_ECOLI"},"6881322":{"id":6881322,"externalId":"511145.b0480","name":"ushA","hasFamily":true,"annotation":"bifunctional UDP-sugar hydrolase/5'-nucleotidase; Degradation of external UDP-glucose to uridine monophosphate and glucose-1-phosphate, which can then be used by the cell","uniprotId":"USHA_ECOLI"},"6881323":{"id":6881323,"externalId":"511145.b0481","name":"ybaK","hasFamily":true,"annotation":"conserved protein; Functions in trans to edit the amino acid from incorrectly charged Cys-tRNA(Pro) via a Cys-tRNA(Pro) deacylase activity. May compensate for the lack of Cys-tRNA(Pro) editing by ProRS. Is also able to deacylate Cys-tRNA(Cys), and displays weak deacylase activity in vitro against Gly-tRNA(Gly), as well as, at higher concentrations, some other correctly charged tRNAs. Unlike some of its orthologs it is not able to remove the amino acid moiety from incorrectly charged Ala-tRNA(Pro)","uniprotId":"YBAK_ECOLI"},"6881324":{"id":6881324,"externalId":"511145.b0482","name":"ybaP","hasFamily":true,"annotation":"GumN family protein","uniprotId":"YBAP_ECOLI"},"6881325":{"id":6881325,"externalId":"511145.b0483","name":"ybaQ","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YBAQ_ECOLI"},"6881326":{"id":6881326,"externalId":"511145.b0484","name":"copA","hasFamily":true,"annotation":"copper transporter; Involved in copper export. May also be involved in silver export","uniprotId":"COPA_ECOLI"},"6881327":{"id":6881327,"externalId":"511145.b0485","name":"glsA","hasFamily":true,"annotation":"glutaminase","uniprotId":"GLSA1_ECOLI"},"6881328":{"id":6881328,"externalId":"511145.b0486","name":"ybaT","hasFamily":true,"annotation":"predicted transporter; Probable amino-acid or metabolite transport protein","uniprotId":"YBAT_ECOLI"},"6881329":{"id":6881329,"externalId":"511145.b0487","name":"cueR","hasFamily":true,"annotation":"DNA-binding transcriptional activator of copper-responsive regulon genes; Regulates the transcription of the copA and cueO genes. It detects cytoplasmic copper stress and activates transcription in response to increasing copper concentrations","uniprotId":"CUER_ECOLI"},"6881330":{"id":6881330,"externalId":"511145.b0488","name":"ybbJ","hasFamily":true,"annotation":"inner membrane protein that stimulates the ftsH htpX mutant suppressor activity of QmcA","uniprotId":"YBBJ_ECOLI"},"6881331":{"id":6881331,"externalId":"511145.b0489","name":"qmcA","hasFamily":true,"annotation":"multicopy suppressor of ftsH htpX double mutant; Identified as a multi-copy suppressor of an FtsH/HtpX protease double disruption mutant. May play a role in the quality control of integral membrane proteins","uniprotId":"QMCA_ECOLI"},"6881332":{"id":6881332,"externalId":"511145.b0490","name":"ybbL","hasFamily":true,"annotation":"putative transporter subunit: ATP-binding component of ABC superfamily","uniprotId":"YBBL_ECOLI"},"6881333":{"id":6881333,"externalId":"511145.b0491","name":"ybbM","hasFamily":true,"annotation":"inner membrane protein, UPF0014 family","uniprotId":"YBBM_ECOLI"},"6881334":{"id":6881334,"externalId":"511145.b0492","name":"ybbN","hasFamily":true,"annotation":"DnaK co-chaperone, thioredoxin-like protein","uniprotId":"YBBN_ECOLI"},"6881335":{"id":6881335,"externalId":"511145.b0493","name":"ybbO","hasFamily":true,"annotation":"putative oxidoreductase with NAD(P)-binding Rossmann-fold domain","uniprotId":"YBBO_ECOLI"},"6881336":{"id":6881336,"externalId":"511145.b0494","name":"tesA","hasFamily":true,"annotation":"multifunctional acyl-CoA thioesterase I and protease I and lysophospholipase L1; Hydrolyzes only long chain acyl thioesters (C12-C18). Specificity similar to chymotrypsin","uniprotId":"TESA_ECOLI"},"6881337":{"id":6881337,"externalId":"511145.b0495","name":"ybbA","hasFamily":true,"annotation":"putative transporter subunit: ATP-binding component of ABC superfamily","uniprotId":"YBBA_ECOLI"},"6881338":{"id":6881338,"externalId":"511145.b0496","name":"ybbP","hasFamily":true,"annotation":"putative ABC transporter permease","uniprotId":"YBBP_ECOLI"},"6881339":{"id":6881339,"externalId":"511145.b0497","name":"rhsD","hasFamily":true,"annotation":"rhsD element protein; Rhs elements have a nonessential function. They may play an important role in the natural ecology of the cell","uniprotId":"RHSD_ECOLI"},"6881340":{"id":6881340,"externalId":"511145.b0498","name":"ybbC","hasFamily":false,"annotation":"predicted protein","uniprotId":"YBBC_ECOLI"},"6881341":{"id":6881341,"externalId":"511145.b0503","name":"selU","hasFamily":true,"annotation":"tRNA 2-selenouridine synthase, selenophosphate-dependent; Catalyzes the transfer of selenium from selenophosphate for conversion of 2-thiouridine to 2-selenouridine at the wobble position in tRNA (By similarity)","uniprotId":"SELU_ECOLI"},"6881342":{"id":6881342,"externalId":"511145.b0504","name":"allS","hasFamily":true,"annotation":"DNA-binding transcriptional activator of the allD operon; Positive regulator essential for the expression of allD operon. Binds to the allD promoter","uniprotId":"ALLS_ECOLI"},"6881343":{"id":6881343,"externalId":"511145.b0505","name":"allA","hasFamily":true,"annotation":"ureidoglycolate hydrolase; Involved in the anaerobic utilization of allantoin. Reinforces the induction of genes involved in the degradation of allantoin and glyoxylate by producing glyoxylate (By similarity)","uniprotId":"ALLA_ECOLI"},"6881344":{"id":6881344,"externalId":"511145.b0506","name":"allR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor for all (allantoin) and gcl (glyoxylate) operons; glyoxylate-induced; Negative regulator of allantoin and glyoxylate utilization operons. Binds to the gcl promoter and to the allS- allA intergenic region. Binding to DNA is abolished by glyoxylate","uniprotId":"ALLR_ECOLI"},"6881345":{"id":6881345,"externalId":"511145.b0507","name":"gcl","hasFamily":true,"annotation":"glyoxylate carboligase; Catalyzes the condensation of two molecules of glyoxylate to give 2-hydroxy-3-oxopropanoate (also termed tartronate semialdehyde)","uniprotId":"GCL_ECOLI"},"6881346":{"id":6881346,"externalId":"511145.b0508","name":"gip","hasFamily":true,"annotation":"hydroxypyruvate isomerase; Catalyzes the reversible isomerization between hydroxypyruvate and 2-hydroxy-3-oxopropanoate (also termed tartronate semialdehyde). Does not catalyze the isomerization of D-fructose to D-glucose or that of D-xylulose to D-xylose. Also does not catalyze racemization of serine, alanine, glycerate or lactate","uniprotId":"HYI_ECOLI"},"6881347":{"id":6881347,"externalId":"511145.b0509","name":"glxR","hasFamily":true,"annotation":"tartronate semialdehyde reductase, NADH-dependent","uniprotId":"GLXR_ECOLI"},"6881348":{"id":6881348,"externalId":"511145.b0511","name":"ybbW","hasFamily":true,"annotation":"putative allantoin transporter; Transport of allantoin (By similarity)","uniprotId":"ALLP_ECOLI"},"6881349":{"id":6881349,"externalId":"511145.b0512","name":"allB","hasFamily":true,"annotation":"allantoinase; Catalyzes the conversion of allantoin (5- ureidohydantoin) to allantoic acid by hydrolytic cleavage of the five-member hydantoin ring (By similarity)","uniprotId":"ALLB_ECOLI"},"6881350":{"id":6881350,"externalId":"511145.b0513","name":"ybbY","hasFamily":true,"annotation":"putative uracil/xanthine transporter","uniprotId":"YBBY_ECOLI"},"6881351":{"id":6881351,"externalId":"511145.b0514","name":"glxK","hasFamily":true,"annotation":"glycerate kinase II","uniprotId":"GLXK2_ECOLI"},"6881352":{"id":6881352,"externalId":"511145.b0515","name":"ylbA","hasFamily":true,"annotation":"conserved protein","uniprotId":"YLBA_ECOLI"},"6881353":{"id":6881353,"externalId":"511145.b0516","name":"allC","hasFamily":true,"annotation":"allantoate amidohydrolase; Involved in the anaerobic utilization of allantoin. Converts allantoate to (S)-ureidoglycolate and ammonia","uniprotId":"ALLC_ECOLI"},"6881354":{"id":6881354,"externalId":"511145.b0517","name":"allD","hasFamily":true,"annotation":"ureidoglycolate dehydrogenase; Involved in the anaerobic utilization of allantoin","uniprotId":"ALLD_ECOLI"},"6881355":{"id":6881355,"externalId":"511145.b0518","name":"fdrA","hasFamily":true,"annotation":"multicopy suppressor of dominant negative ftsH mutations; predicted acyl-CoA synthetase with NAD(P)-binding Rossmann-fold domain; Not known; multicopy suppressor of dominant negative ftsH mutations","uniprotId":"FDRA_ECOLI"},"6881356":{"id":6881356,"externalId":"511145.b0520","name":"ylbF","hasFamily":true,"annotation":"conserved protein","uniprotId":"YLBF_ECOLI"},"6881357":{"id":6881357,"externalId":"511145.b0521","name":"arcC","hasFamily":true,"annotation":"putative carbamate kinase","uniprotId":"ARCC_ECOLI"},"6881358":{"id":6881358,"externalId":"511145.b0522","name":"purK","hasFamily":true,"annotation":"N5-carboxyaminoimidazole ribonucleotide synthase; Catalyzes the ATP-dependent conversion of 5- aminoimidazole ribonucleotide (AIR) and HCO(3)(-) to N5- carboxyaminoimidazole ribonucleotide (N5-CAIR)","uniprotId":"PURK_ECOLI"},"6881359":{"id":6881359,"externalId":"511145.b0523","name":"purE","hasFamily":true,"annotation":"N5-carboxyaminoimidazole ribonucleotide mutase; Catalyzes the conversion of N5-carboxyaminoimidazole ribonucleotide (N5-CAIR) to 4-carboxy-5-aminoimidazole ribonucleotide (CAIR) (By similarity)","uniprotId":"PURE_ECOLI"},"6881360":{"id":6881360,"externalId":"511145.b0524","name":"lpxH","hasFamily":true,"annotation":"UDP-2,3-diacylglucosamine pyrophosphatase; Catalyzes the hydrolysis of the pyrophosphate bond of UDP-2,3-diacylglucosamine to yield 2,3-diacylglucosamine 1- phosphate (lipid X) and UMP (By similarity)","uniprotId":"LPXH_ECOLI"},"6881361":{"id":6881361,"externalId":"511145.b0525","name":"ppiB","hasFamily":true,"annotation":"peptidyl-prolyl cis-trans isomerase B (rotamase B); PPIases accelerate the folding of proteins. It catalyzes the cis-trans isomerization of proline imidic peptide bonds in oligopeptides","uniprotId":"PPIB_ECOLI"},"6881362":{"id":6881362,"externalId":"511145.b0526","name":"cysS","hasFamily":true,"annotation":"cysteinyl-tRNA synthetase","uniprotId":"SYC_ECOLI"},"6881363":{"id":6881363,"externalId":"511145.b0527","name":"ybcI","hasFamily":true,"annotation":"inner membrane protein, DUF457 family","uniprotId":"YBCI_ECOLI"},"6881364":{"id":6881364,"externalId":"511145.b0528","name":"ybcJ","hasFamily":true,"annotation":"Ribosome-associated protein; predicted RNA-binding protein; Its structure and the presence of conserved basic residues indicates that it probably binds RNA","uniprotId":"YBCJ_ECOLI"},"6881365":{"id":6881365,"externalId":"511145.b0529","name":"folD","hasFamily":true,"annotation":"bifunctional 5,10-methylene-tetrahydrofolate dehydrogenase/ 5,10-methylene-tetrahydrofolate cyclohydrolase; Catalyzes the oxidation of 5,10- methylenetetrahydrofolate to 5,10-methenyltetrahydrofolate and then the hydrolysis of 5,10-methenyltetrahydrofolate to 10- formyltetrahydrofolate. This enzyme is specific for NADP","uniprotId":"FOLD_ECOLI"},"6881366":{"id":6881366,"externalId":"511145.b0530","name":"sfmA","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein; Fimbriae (also called pili), polar filaments radiating from the surface of the bacterium to a length of 0.5-1.5 micrometers and numbering 100-300 per cell, enable bacteria to colonize the epithelium of specific host organs (By similarity)","uniprotId":"SFMA_ECOLI"},"6881367":{"id":6881367,"externalId":"511145.b0531","name":"sfmC","hasFamily":true,"annotation":"pilin chaperone, periplasmic; Required for the biogenesis of SfmA fimbriae. Binds and interact with SfmH (By similarity)","uniprotId":"SFMC_ECOLI"},"6881368":{"id":6881368,"externalId":"511145.b0532","name":"sfmD","hasFamily":true,"annotation":"putative outer membrane export usher protein; Involved in the export and assembly of SfmA fimbrial subunits across the outer membrane (By similarity)","uniprotId":"SFMD_ECOLI"},"6881369":{"id":6881369,"externalId":"511145.b0533","name":"sfmH","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein; Involved in regulation of length and mediation of adhesion of SfmA fimbriae (but not necessary for the production of fimbriae). Seems to be mannose binding adhesin (By similarity)","uniprotId":"SFMH_ECOLI"},"6881370":{"id":6881370,"externalId":"511145.b0534","name":"sfmF","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"SFMF_ECOLI"},"6881371":{"id":6881371,"externalId":"511145.b0535","name":"fimZ","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"FIMZ_ECOLI"},"6881372":{"id":6881372,"externalId":"511145.b0537","name":"intD","hasFamily":true,"annotation":"DLP12 prophage; Integrase from the cryptic lambdoic prophage DLP12. Integrase is necessary for integration of the phage into the host genome by site-specific recombination. In conjunction with excisionase, integrase is also necessary for excision of the prophage from the host genome","uniprotId":"INTD_ECOLI"},"6881373":{"id":6881373,"externalId":"511145.b0540","name":"b0540","hasFamily":true,"annotation":"IS3 transposase A"},"6881374":{"id":6881374,"externalId":"511145.b0541","name":"b0541","hasFamily":true,"annotation":"IS3 transposase B"},"6881375":{"id":6881375,"externalId":"511145.b0543","name":"emrE","hasFamily":true,"annotation":"DLP12 prophage; multidrug resistance protein; Multidrug transporter that expels positively charged hydrophobic drugs across the inner membrane of E.coli., thereby conferring resistance to a wide range of toxic compounds. The drug efflux is coupled to an influx of protons. Is involved in the resistance of E.coli cells to methyl viologen, ethidium bromide and acriflavine. Is also able to transport tetraphenylphosphonium (TPP(+)) and benzalkonium","uniprotId":"EMRE_ECOLI"},"6881376":{"id":6881376,"externalId":"511145.b0544","name":"ybcK","hasFamily":false,"annotation":"DLP12 prophage","uniprotId":"YBCK_ECOLI"},"6881377":{"id":6881377,"externalId":"511145.b0545","name":"ybcL","hasFamily":true,"annotation":"DLP12 prophage; secreted protein, UPF0098 family","uniprotId":"YBCL_ECOLI"},"6881378":{"id":6881378,"externalId":"511145.b0546","name":"ybcM","hasFamily":true,"annotation":"DLP12 prophage","uniprotId":"YBCM_ECOLI"},"6881379":{"id":6881379,"externalId":"511145.b0547","name":"ybcN","hasFamily":false,"annotation":"DLP12 prophage","uniprotId":"YBCN_ECOLI"},"6881380":{"id":6881380,"externalId":"511145.b0548","name":"ninE","hasFamily":false,"annotation":"DLP12 prophage","uniprotId":"NINE_ECOLI"},"6881381":{"id":6881381,"externalId":"511145.b0549","name":"ybcO","hasFamily":false,"annotation":"DLP12 prophage","uniprotId":"YBCO_ECOLI"},"6881382":{"id":6881382,"externalId":"511145.b0550","name":"rusA","hasFamily":false,"annotation":"DLP12 prophage; endonuclease RUS; Endonuclease that resolves Holliday junction intermediates made during homologous genetic recombination and DNA repair. Exhibits sequence and structure-selective cleavage of four-way DNA junctions, where it introduces symmetrical nicks in two strands of the same polarity at the 5' side of dinucleotides. Corrects the defects in genetic recombination and DNA repair associated with inactivation of ruvAB or ruvC (By similarity)","uniprotId":"RUSA_ECOLI"},"6881383":{"id":6881383,"externalId":"511145.b0551","name":"quuD","hasFamily":true,"annotation":"DLP12 prophage; Positively regulate expression of some phage genes. Bacterial host RNA polymerase modified by antitermination proteins transcribes through termination sites that otherwise prevent expression of the regulated genes (By similarity)","uniprotId":"REQ1_ECOLI"},"6881384":{"id":6881384,"externalId":"511145.b0552","name":"b0552","hasFamily":true,"annotation":"IS5 transposase and trans-activator"},"6881385":{"id":6881385,"externalId":"511145.b0554","name":"essD","hasFamily":false,"annotation":"DLP12 prophage","uniprotId":"ESSD_ECOLI"},"6881386":{"id":6881386,"externalId":"511145.b0555","name":"rrrD","hasFamily":true,"annotation":"DLP12 prophage; predicted lysozyme; Essential for lysis of bacterial cell wall, by showing cell wall hydrolyzing activity. Exhibits lytic activity against E.coli and S.typhi cell wall substrate","uniprotId":"LYSD_ECOLI"},"6881387":{"id":6881387,"externalId":"511145.b0556","name":"rzpD","hasFamily":true,"annotation":"DLP12 prophage; predicted murein endopeptidase; Necessary for host cell lysis. It is believed to code for an endopeptidase that cleaves the amino-carboxyl cross-link between the diaminopimelic acid and D-alanine residues in the murein component of the bacterial cell wall (By similarity)","uniprotId":"RZPD_ECOLI"},"6881388":{"id":6881388,"externalId":"511145.b0557","name":"iss","hasFamily":false,"annotation":"DLP12 prophage","uniprotId":"BORD_ECOLI"},"6881389":{"id":6881389,"externalId":"511145.b0558","name":"ybcV","hasFamily":true,"annotation":"DLP12 prophage","uniprotId":"YBCV_ECOLI"},"6881390":{"id":6881390,"externalId":"511145.b0559","name":"ybcW","hasFamily":false,"annotation":"DLP12 prophage","uniprotId":"YBCW_ECOLI"},"6881391":{"id":6881391,"externalId":"511145.b0560","name":"nohA","hasFamily":true,"annotation":"DLP12 prophage","uniprotId":"NOHA_ECOLI"},"6881392":{"id":6881392,"externalId":"511145.b0564","name":"appY","hasFamily":false,"annotation":"DNA-binding global transcriptional activator; DLP12 prophage; Induces the synthesis of acid phosphatase (AppA) and several other polypeptides (such as AppBC) during the deceleration phase of growth. It also acts as a transcriptional repressor for one group of proteins that are synthesized preferentially in exponential growth and for one group synthesized only in the stationary phase. Also involved in the stabilization of the sigma stress factor RpoS during stress conditions","uniprotId":"APPY_ECOLI"},"6881393":{"id":6881393,"externalId":"511145.b0565","name":"ompT","hasFamily":true,"annotation":"DLP12 prophage; outer membrane protease VII (outer membrane protein 3b); Protease that can cleave T7 RNA polymerase, ferric enterobactin receptor protein (FEP), antimicrobial peptide protamine and other proteins. This protease has a specificity for paired basic residues","uniprotId":"OMPT_ECOLI"},"6881394":{"id":6881394,"externalId":"511145.b0566","name":"envY","hasFamily":false,"annotation":"DNA-binding transcriptional activator of porin biosynthesis; Influences the temperature-dependent expression of several E.coli envelope proteins, most notably the porins OmpF and OmpC and the lambda receptor, LamB","uniprotId":"ENVY_ECOLI"},"6881395":{"id":6881395,"externalId":"511145.b0567","name":"ybcH","hasFamily":false,"annotation":"predicted protein","uniprotId":"YBCH_ECOLI"},"6881396":{"id":6881396,"externalId":"511145.b0568","name":"nfrA","hasFamily":false,"annotation":"bacteriophage N4 receptor, outer membrane subunit; Required for bacteriophage N4 adsorption. Serves as the phage receptor","uniprotId":"NFRA_ECOLI"},"6881397":{"id":6881397,"externalId":"511145.b0569","name":"nfrB","hasFamily":false,"annotation":"bacteriophage N4 receptor, inner membrane subunit; Required for bacteriophage N4 adsorption. May be a component of the phage receptor","uniprotId":"NFRB_ECOLI"},"6881398":{"id":6881398,"externalId":"511145.b0570","name":"cusS","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with CusR, senses copper ions; Member of the two-component regulatory system CusS/CusR. Copper ion sensor. Could also be a silver ion sensor. Activates CusR by phosphorylation","uniprotId":"CUSS_ECOLI"},"6881399":{"id":6881399,"externalId":"511145.b0571","name":"cusR","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with CusS; Member of the two-component regulatory system CusS/CusR. Activates the expression of cusCFBA and plasmid pRJ1004 gene pcoE in response to increasing levels of copper ions, and probably silver ions. Can also increase the basal-level expression of copper resistance gene operon pcoABCD","uniprotId":"CUSR_ECOLI"},"6881400":{"id":6881400,"externalId":"511145.b0572","name":"cusC","hasFamily":true,"annotation":"copper/silver efflux system, outer membrane component; Forms pores that allow passive diffusion of cations across the outer membrane. Part of a cation efflux system that mediates resistance to copper and silver. In pathogenic strains it allows the bacteria to invade brain microvascular endothelial cells (BMEC) thus allowing it to cross the blood-brain barrier and cause neonatal meningitis","uniprotId":"CUSC_ECOLI"},"6881401":{"id":6881401,"externalId":"511145.b0573","name":"cusF","hasFamily":false,"annotation":"periplasmic copper- and silver-binding protein; Part of a cation efflux system that mediates resistance to copper and silver. Binds one copper per polypeptide","uniprotId":"CUSF_ECOLI"},"6881402":{"id":6881402,"externalId":"511145.b0574","name":"cusB","hasFamily":true,"annotation":"copper/silver efflux system, membrane fusion protein; Part of a cation efflux system that mediates resistance to copper and silver","uniprotId":"CUSB_ECOLI"},"6881403":{"id":6881403,"externalId":"511145.b0575","name":"cusA","hasFamily":true,"annotation":"copper/silver efflux system, membrane component; Part of a cation efflux system that mediates resistance to copper and silver","uniprotId":"CUSA_ECOLI"},"6881404":{"id":6881404,"externalId":"511145.b0576","name":"pheP","hasFamily":true,"annotation":"phenylalanine transporter; Permease that is involved in the transport across the cytoplasmic membrane of phenylalanine","uniprotId":"PHEP_ECOLI"},"6881405":{"id":6881405,"externalId":"511145.b0577","name":"mscM","hasFamily":true,"annotation":"mechanosensitive channel protein, miniconductance; Mechanosensitive channel of miniconductance that confers protection against mild hypoosmotic shock. Overexpression confers protection against severe shocks","uniprotId":"MSCM_ECOLI"},"6881406":{"id":6881406,"externalId":"511145.b0578","name":"NTR","hasFamily":true,"annotation":"dihydropteridine reductase, NAD(P)H-dependent, oxygen-insensitive; Reduction of a variety of nitroaromatic compounds using NADH (and to lesser extent NADPH) as source of reducing equivalents; two electrons are transferred. Capable of reducing nitrofurazone, quinones and the anti-tumor agent CB1954 (5- (aziridin-1-yl)-2,4-dinitrobenzamide). The reduction of CB1954 results in the generation of cytotoxic species","uniprotId":"F2X7N5_ECOLI"},"6881407":{"id":6881407,"externalId":"511145.b0579","name":"ybdF","hasFamily":true,"annotation":"conserved protein","uniprotId":"YBDF_ECOLI"},"6881408":{"id":6881408,"externalId":"511145.b0580","name":"ybdJ","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YBDJ_ECOLI"},"6881409":{"id":6881409,"externalId":"511145.b0581","name":"ybdK","hasFamily":true,"annotation":"weak gamma-glutamyl:cysteine ligase; ATP-dependent carboxylate-amine ligase which exhibits weak glutamate--cysteine ligase activity. However, because of the low catalytic rate, the question remains whether L-cysteine is the actual biological substrate","uniprotId":"CAAL_ECOLI"},"6881410":{"id":6881410,"externalId":"511145.b0582","name":"b0582","hasFamily":false,"annotation":"IS186 transposase"},"6881411":{"id":6881411,"externalId":"511145.b0583","name":"entD","hasFamily":true,"annotation":"phosphopantetheinyltransferase component of enterobactin synthase multienzyme complex; Catalyzes the transfer of the 4'-phosphopantetheine moiety from coenzyme A to apo-domains of both EntB (an ArCP domain) and EntF (a PCP domain). Plays an essential role in the assembly of the enterobactin","uniprotId":"ENTD_ECOLI"},"6881412":{"id":6881412,"externalId":"511145.b0584","name":"fepA","hasFamily":true,"annotation":"iron-enterobactin outer membrane transporter; This protein is involved in the initial step of iron uptake by binding ferrienterobactin (Fe-ENT), an iron chelatin siderophore that allows E.coli to extract iron from the environment. FepA also acts as a receptor for colicins B and D","uniprotId":"FEPA_ECOLI"},"6881413":{"id":6881413,"externalId":"511145.b0585","name":"fes","hasFamily":true,"annotation":"enterobactin/ferric enterobactin esterase; Upon internalization, ferric enterobactin is processed via an exquisitely specific pathway that is dependent on FES activity, making iron available for metabolic use","uniprotId":"FES_ECOLI"},"6881414":{"id":6881414,"externalId":"511145.b0586","name":"entF","hasFamily":true,"annotation":"enterobactin synthase multienzyme complex component, ATP-dependent; Activates the carboxylate group of L-serine via ATP- dependent PPi exchange reactions to the aminoacyladenylate, preparing that molecule for the final stages of enterobactin synthesis. Holo-EntF acts as the catalyst for the formation of the three amide and three ester bonds present in the cyclic (2,3- dihydroxybenzoyl)serine trimer enterobactin, using seryladenylate and acyl-holo-EntB (acylated with 2,3-dihydroxybenzoate by EntE)","uniprotId":"ENTF_ECOLI"},"6881415":{"id":6881415,"externalId":"511145.b0587","name":"fepE","hasFamily":true,"annotation":"regulator of length of O-antigen component of lipopolysaccharide chains; Part of the ferric enterobactin transport system","uniprotId":"FEPE_ECOLI"},"6881416":{"id":6881416,"externalId":"511145.b0588","name":"fepC","hasFamily":true,"annotation":"iron-enterobactin transporter subunit; Part of the binding-protein-dependent transport system for ferric enterobactin. Probably responsible for energy coupling to the transport system","uniprotId":"FEPC_ECOLI"},"6881417":{"id":6881417,"externalId":"511145.b0589","name":"fepG","hasFamily":true,"annotation":"iron-enterobactin transporter subunit; Part of the binding-protein-dependent transport system for ferric enterobactin. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"FEPG_ECOLI"},"6881418":{"id":6881418,"externalId":"511145.b0590","name":"fepD","hasFamily":true,"annotation":"iron-enterobactin transporter subunit; Part of the binding-protein-dependent transport system for ferric enterobactin. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"FEPD_ECOLI"},"6881419":{"id":6881419,"externalId":"511145.b0591","name":"entS","hasFamily":true,"annotation":"enterobactin exporter, iron-regulated; Exports the siderophore enterobactin out of the cell (By similarity)","uniprotId":"ENTS_ECOLI"},"6881420":{"id":6881420,"externalId":"511145.b0592","name":"fepB","hasFamily":true,"annotation":"iron-enterobactin transporter subunit; Binds ferrienterobactin; part of the binding-protein- dependent transport system for uptake of ferrienterobactin","uniprotId":"FEPB_ECOLI"},"6881421":{"id":6881421,"externalId":"511145.b0593","name":"entC","hasFamily":true,"annotation":"isochorismate synthase 1","uniprotId":"ENTC_ECOLI"},"6881422":{"id":6881422,"externalId":"511145.b0594","name":"entE","hasFamily":true,"annotation":"2,3-dihydroxybenzoate-AMP ligase component of enterobactin synthase multienzyme complex; Activates the carboxylate group of 2,3-dihydroxy- benzoate (2,3-DHB), via ATP-dependent PPi exchange reactions, to the acyladenylate. Then, catalyzes the acylation of holo-EntB with 2,3-DHB adenylate, preparing that molecule for amide bond formation with L-serine","uniprotId":"ENTE_ECOLI"},"6881423":{"id":6881423,"externalId":"511145.b0595","name":"entB","hasFamily":true,"annotation":"isochorismatase; Required for production of 2,3-DHB. Also serves as an aryl carrier protein and plays a role in enterobactin assembly","uniprotId":"ENTB_ECOLI"},"6881424":{"id":6881424,"externalId":"511145.b0596","name":"entA","hasFamily":true,"annotation":"2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase","uniprotId":"ENTA_ECOLI"},"6881425":{"id":6881425,"externalId":"511145.b0597","name":"entH","hasFamily":true,"annotation":"thioesterase required for efficient enterobactin production; Required for optimal enterobactin synthesis. Acts as a proofreading enzyme that prevents EntB misacylation by hydrolyzing the thioester bound existing between EntB and wrongly charged molecules. Displays esterase activity toward a wide range of substrates, including acyl-CoAs and aryl-CoAs","uniprotId":"ENTH_ECOLI"},"6881426":{"id":6881426,"externalId":"511145.b0598","name":"cstA","hasFamily":true,"annotation":"carbon starvation protein; Peptide utilization during carbon starvation","uniprotId":"CSTA_ECOLI"},"6881427":{"id":6881427,"externalId":"511145.b0599","name":"ybdH","hasFamily":true,"annotation":"putative oxidoreductase","uniprotId":"YBDH_ECOLI"},"6881428":{"id":6881428,"externalId":"511145.b0600","name":"ybdL","hasFamily":true,"annotation":"methionine aminotransferase, PLP-dependent; Shows aminotransferase activity with methionine and histidine as substrates, and to a lesser extent also with phenylalanine","uniprotId":"YBDL_ECOLI"},"6881429":{"id":6881429,"externalId":"511145.b0601","name":"ybdM","hasFamily":true,"annotation":"conserved protein","uniprotId":"YBDM_ECOLI"},"6881430":{"id":6881430,"externalId":"511145.b0602","name":"ybdN","hasFamily":true,"annotation":"conserved protein","uniprotId":"YBDN_ECOLI"},"6881431":{"id":6881431,"externalId":"511145.b0603","name":"ybdO","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YBDO_ECOLI"},"6881432":{"id":6881432,"externalId":"511145.b0604","name":"dsbG","hasFamily":true,"annotation":"thiol:disulfide interchange protein, periplasmic; Involved in disulfide bond formation. DsbG and DsbC are part of a periplasmic reducing system that controls the level of cysteine sulfenylation, and provides reducing equivalents to rescue oxidatively damaged secreted proteins such as ErfK, YbiS and YnhG. Probably also functions as a disulfide isomerase with a narrower substrate specificity than DsbC. DsbG is maintained in a reduced state by DsbD. Displays chaperone activity in both redox states in vitro","uniprotId":"DSBG_ECOLI"},"6881433":{"id":6881433,"externalId":"511145.b0605","name":"ahpC","hasFamily":true,"annotation":"alkyl hydroperoxide reductase, C22 subunit; Directly reduces organic hydroperoxides in its reduced dithiol form","uniprotId":"AHPC_ECOLI"},"6881434":{"id":6881434,"externalId":"511145.b0606","name":"ahpF","hasFamily":true,"annotation":"alkyl hydroperoxide reductase, F52a subunit, FAD/NAD(P)-binding; Serves to protect the cell against DNA damage by alkyl hydroperoxides. It can use either NADH or NADPH as electron donor for direct reduction of redox dyes or of alkyl hydroperoxides when combined with the AhpC protein","uniprotId":"AHPF_ECOLI"},"6881435":{"id":6881435,"externalId":"511145.b0607","name":"uspG","hasFamily":true,"annotation":"universal stress protein UP12","uniprotId":"USPG_ECOLI"},"6881436":{"id":6881436,"externalId":"511145.b0608","name":"ybdR","hasFamily":true,"annotation":"putative oxidoreductase, Zn-dependent and NAD(P)-binding","uniprotId":"YBDR_ECOLI"},"6881437":{"id":6881437,"externalId":"511145.b0610","name":"rnk","hasFamily":true,"annotation":"regulator of nucleoside diphosphate kinase; May act as an anti-Gre factor. Regulates the level of the nucleoside diphosphate kinase Ndk","uniprotId":"RNK_ECOLI"},"6881438":{"id":6881438,"externalId":"511145.b0611","name":"rna","hasFamily":true,"annotation":"ribonuclease I; One of the few RNases that cleaves the phosphodiester bond between any two nucleotide. Shows a preference for cytidylic or guanylic acid","uniprotId":"RNI_ECOLI"},"6881439":{"id":6881439,"externalId":"511145.b0612","name":"citT","hasFamily":true,"annotation":"citrate:succinate antiporter; Responsible for the uptake of citrate in exchange to the efflux of succinate. Has a relatively broad specificity for C(4)- dicarboxylates and tricarboxylates","uniprotId":"CITT_ECOLI"},"6881440":{"id":6881440,"externalId":"511145.b0613","name":"citG","hasFamily":true,"annotation":"2-(5''-triphosphoribosyl)-3'-dephosphocoenzyme-A synthase; Catalyzes the formation of 2-(5''-triphosphoribosyl)-3'- dephosphocoenzyme-A, the precursor of the prosthetic group of the holo-acyl carrier protein (gamma chain) of citrate lyase, from ATP and dephospho-CoA (By similarity)","uniprotId":"CITG_ECOLI"},"6881441":{"id":6881441,"externalId":"511145.b0614","name":"citX","hasFamily":true,"annotation":"apo-citrate lyase phosphoribosyl-dephospho-CoA transferase; Transfers 2-(5''-triphosphoribosyl)-3'- dephosphocoenzyme-A on a serine residue to the apo-acyl carrier protein (gamma chain) of the citrate lyase to yield holo-acyl carrier protein (By similarity)","uniprotId":"CITX_ECOLI"},"6881442":{"id":6881442,"externalId":"511145.b0615","name":"citF","hasFamily":true,"annotation":"citrate lyase, citrate-ACP transferase (alpha) subunit; Represents a citrate:acetyl-ACP transferase (By similarity)","uniprotId":"CILA_ECOLI"},"6881443":{"id":6881443,"externalId":"511145.b0616","name":"citE","hasFamily":true,"annotation":"citrate lyase, citryl-ACP lyase (beta) subunit; Represents a citryl-ACP lyase (By similarity)","uniprotId":"CITE_ECOLI"},"6881444":{"id":6881444,"externalId":"511145.b0617","name":"citD","hasFamily":true,"annotation":"citrate lyase, acyl carrier (gamma) subunit; Covalent carrier of the coenzyme of citrate lyase (By similarity)","uniprotId":"CITD_ECOLI"},"6881445":{"id":6881445,"externalId":"511145.b0618","name":"citC","hasFamily":true,"annotation":"[citrate [pro-3S]-lyase] ligase; Acetylation of prosthetic group (2-(5''-phosphoribosyl)- 3'-dephosphocoenzyme-A) of the gamma subunit of citrate lyase","uniprotId":"CITC_ECOLI"},"6881446":{"id":6881446,"externalId":"511145.b0619","name":"citA","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with citB; Member of the two-component regulatory system DpiA/DpiB, which is essential for expression of citrate-specific fermentation genes and genes involved in plasmid inheritance. Could be involved in response to both the presence of citrate and external redox conditions. Functions as a sensor kinase that phosphorylates DpiA in the presence of citrate","uniprotId":"DPIB_ECOLI"},"6881447":{"id":6881447,"externalId":"511145.b0620","name":"dpiA","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with citA; Member of the two-component regulatory system DpiA/DpiB, which is essential for expression of citrate-specific fermentation genes and genes involved in plasmid inheritance. Could be involved in response to both the presence of citrate and external redox conditions. Regulates the transcription of citCDEFXGT, dpiAB, mdh and exuT. Binds specifically to the dpiB-citC intergenic region","uniprotId":"DPIA_ECOLI"},"6881448":{"id":6881448,"externalId":"511145.b0621","name":"dcuC","hasFamily":true,"annotation":"anaerobic C4-dicarboxylate transport; Responsible for the transport of C4-dicarboxylates during anaerobic growth","uniprotId":"DCUC_ECOLI"},"6881449":{"id":6881449,"externalId":"511145.b0622","name":"crcA","hasFamily":true,"annotation":"phospholipid:lipid A palmitoyltransferase; PagP is required both for biosynthesis of hepta-acylated lipid A species containing palmitate and for resistance to cationic antimicrobial peptides (CAMPs). It catalyzes the transfer of a palmitate chain (16:0) from the sn-1 position of a glycerophospholipid to the free hydroxyl group of the (R)-3- hydroxymyristate chain at position 2 of lipid A (endotoxin). Modifications of lipid A with a palmitate chain allow to evade host immune defenses by resisting antimicrobial peptides and attenuating the inflammatory response to infection triggered by  [...] ","uniprotId":"PAGP_ECOLI"},"6881450":{"id":6881450,"externalId":"511145.b0623","name":"cspE","hasFamily":true,"annotation":"DNA-binding transcriptional repressor","uniprotId":"CSPE_ECOLI"},"6881451":{"id":6881451,"externalId":"511145.b0624","name":"crcB","hasFamily":true,"annotation":"inner membrane protein associated with chromosome condensation; Not known. Overproduction leads to camphor resistance and chromosome condensation; deletion increases sensibility to nucleoid decondensation. Probably interacts with the heat-shock protein CspE, possibly providing a way to anchor the chromosome to the membrane","uniprotId":"CRCB_ECOLI"},"6881452":{"id":6881452,"externalId":"511145.b0627","name":"tatE","hasFamily":true,"annotation":"TatABCE protein translocation system subunit; Part of the twin-arginine translocation (Tat) system that transports large folded proteins containing a characteristic twin-arginine motif in their signal peptide across membranes. TatE shares overlapping functions with TatA (By similarity)","uniprotId":"TATE_ECOLI"},"6881453":{"id":6881453,"externalId":"511145.b0628","name":"lipA","hasFamily":true,"annotation":"lipoate synthase; Catalyzes the radical-mediated insertion of two sulfur atoms into the C-6 and C-8 positions of the octanoyl moiety bound to the lipoyl domains of lipoate-dependent enzymes, thereby converting the octanoylated domains into lipoylated derivatives. Free octanoate is not a substrate for LipA","uniprotId":"LIPA_ECOLI"},"6881454":{"id":6881454,"externalId":"511145.b0629","name":"ybeF","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YBEF_ECOLI"},"6881455":{"id":6881455,"externalId":"511145.b0630","name":"lipB","hasFamily":true,"annotation":"octanoyltransferase; octanoyl-[ACP]:protein N-octanoyltransferase; Catalyzes the transfer of endogenously produced octanoic acid from octanoyl-acyl-carrier-protein onto the lipoyl domains of lipoate-dependent enzymes. Lipoyl-ACP can also act as a substrate although octanoyl-ACP is likely to be the physiological substrate (By similarity)","uniprotId":"LIPB_ECOLI"},"6881456":{"id":6881456,"externalId":"511145.b0631","name":"ybeD","hasFamily":true,"annotation":"conserved protein, UPF0250 family","uniprotId":"YBED_ECOLI"},"6881457":{"id":6881457,"externalId":"511145.b0632","name":"dacA","hasFamily":true,"annotation":"D-alanyl-D-alanine carboxypeptidase (penicillin-binding protein 5); Removes C-terminal D-alanyl residues from sugar-peptide cell wall precursors","uniprotId":"DACA_ECOLI"},"6881458":{"id":6881458,"externalId":"511145.b0633","name":"rlpA","hasFamily":true,"annotation":"septal ring protein, suppressor of prc, minor lipoprotein","uniprotId":"RLPA_ECOLI"},"6881459":{"id":6881459,"externalId":"511145.b0634","name":"mrdB","hasFamily":true,"annotation":"cell wall shape-determining protein; Required for the maintenance of the rod cell shape. Required for the expression of the enzymatic activity of the penicillin-binding protein 2 (PBP2)","uniprotId":"RODA_ECOLI"},"6881460":{"id":6881460,"externalId":"511145.b0635","name":"mrdA","hasFamily":true,"annotation":"transpeptidase involved in peptidoglycan synthesis (penicillin-binding protein 2); Cell wall formation; PBP-2 is responsible for the determination of the rod shape of the cell. Its synthesize cross- linked peptidoglycan from the lipid intermediates","uniprotId":"PBP2_ECOLI"},"6881461":{"id":6881461,"externalId":"511145.b0636","name":"rlmH","hasFamily":true,"annotation":"23S rRNA m(3)Psi1915 pseudouridine methyltransferase, SAM-dependent; Specifically methylates the pseudouridine at position 1915 (m3Psi1915) in 23S rRNA. Specific for fully assembled 70S ribosomes","uniprotId":"RLMH_ECOLI"},"6881462":{"id":6881462,"externalId":"511145.b0637","name":"rsfS","hasFamily":true,"annotation":"ribosomal silencing factor; Functions as a ribosomal silencing factor. Addition to isolated ribosomal subunits partially inhibits their association, preventing translation. Interacts with ribosomal protein L14 (rplN), blocking formation of intersubunit bridge B8, preventing association of the 30S and 50S ribosomal subunits and the formation of functional ribosomes, thus repressing translation","uniprotId":"IOJAP_ECOLI"},"6881463":{"id":6881463,"externalId":"511145.b0638","name":"cobC","hasFamily":true,"annotation":"putative alpha-ribazole-5'-P phosphatase; Converts N1-(5-phospho-alpha-D-ribosyl)-5,6- dimethylbenzimidazole into N1-alpha-D-ribosyl-5,6- dimethylbenzimidazole. Also catalyzes the conversion of adenosylcobalamin 5'-phosphate to adenosylcobalamin (vitamin B12); involved in the assembly of the nucleotide loop of cobalamin","uniprotId":"COBC_ECOLI"},"6881464":{"id":6881464,"externalId":"511145.b0639","name":"nadD","hasFamily":true,"annotation":"nicotinic acid mononucleotide adenylyltransferase, NAD(P)-dependent; Catalyzes the reversible adenylation of nicotinate mononucleotide (NaMN) to nicotinic acid adenine dinucleotide (NaAD) (By similarity)","uniprotId":"NADD_ECOLI"},"6881465":{"id":6881465,"externalId":"511145.b0640","name":"holA","hasFamily":true,"annotation":"DNA polymerase III, delta subunit; DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. This DNA polymerase also exhibits 3' to 5' exonuclease activity. The delta subunit seems to interact with the gamma subunit to transfer the beta subunit on the DNA","uniprotId":"HOLA_ECOLI"},"6881466":{"id":6881466,"externalId":"511145.b0641","name":"lptE","hasFamily":true,"annotation":"LPS assembly OM complex LptDE, lipoprotein component; Together with LptD, is involved in the assembly of LPS in the outer leaflet of the outer membrane (By similarity)","uniprotId":"LPTE_ECOLI"},"6881467":{"id":6881467,"externalId":"511145.b0642","name":"leuS","hasFamily":true,"annotation":"leucyl-tRNA synthetase","uniprotId":"SYL_ECOLI"},"6881468":{"id":6881468,"externalId":"511145.b0643","name":"ybeL","hasFamily":true,"annotation":"conserved protein, DUF1451 family","uniprotId":"YBEL_ECOLI"},"6881469":{"id":6881469,"externalId":"511145.b0644","name":"ybeQ","hasFamily":true,"annotation":"conserved protein","uniprotId":"YBEQ_ECOLI"},"6881470":{"id":6881470,"externalId":"511145.b0645","name":"ybeR","hasFamily":true,"annotation":"predicted protein","uniprotId":"YBER_ECOLI"},"6881471":{"id":6881471,"externalId":"511145.b0646","name":"djlB","hasFamily":false,"annotation":"putative co-chaperone","uniprotId":"DJLB_ECOLI"},"6881472":{"id":6881472,"externalId":"511145.b0647","name":"ybeT","hasFamily":true,"annotation":"conserved protein, Sel1 family","uniprotId":"YBET_ECOLI"},"6881473":{"id":6881473,"externalId":"511145.b0648","name":"ybeU","hasFamily":true,"annotation":"conserved protein, DUF1266 family","uniprotId":"YBEU_ECOLI"},"6881474":{"id":6881474,"externalId":"511145.b0649","name":"djlC","hasFamily":true,"annotation":"Hsc56 co-chaperone of HscC","uniprotId":"DJLC_ECOLI"},"6881475":{"id":6881475,"externalId":"511145.b0650","name":"hscC","hasFamily":true,"annotation":"Hsp70 family chaperone Hsc62, binds to RpoD and inhibits transcription; Probable chaperone. Has ATPase activity. Not stimulated by DnaJ","uniprotId":"HSCC_ECOLI"},"6881476":{"id":6881476,"externalId":"511145.b0651","name":"rihA","hasFamily":true,"annotation":"ribonucleoside hydrolase 1; Hydrolyzes with equal efficiency cytidine or uridine to ribose and cytosine or uracil, respectively (By similarity)","uniprotId":"RIHA_ECOLI"},"6881477":{"id":6881477,"externalId":"511145.b0652","name":"gltL","hasFamily":true,"annotation":"glutamate, aspartate ABC transporter ATP-binding subunit; Part of the binding-protein-dependent transport system for glutamate and aspartate. Probably responsible for energy coupling to the transport system","uniprotId":"GLTL_ECOLI"},"6881478":{"id":6881478,"externalId":"511145.b0653","name":"gltK","hasFamily":true,"annotation":"glutamate, aspartate ABC transporter permease subunit; Part of the binding-protein-dependent transport system for aspartate/glutamate; probably responsible for the translocation of the substrate across the membrane","uniprotId":"GLTK_ECOLI"},"6881479":{"id":6881479,"externalId":"511145.b0654","name":"gltJ","hasFamily":true,"annotation":"glutamate, aspartate ABC transporter permease subunit; Part of the binding-protein-dependent transport system for aspartate/glutamate; probably responsible for the translocation of the substrate across the membrane","uniprotId":"GLTJ_ECOLI"},"6881480":{"id":6881480,"externalId":"511145.b0655","name":"gltI","hasFamily":true,"annotation":"glutamate, aspartate binding protein, periplasmic; part of GltJKLI ABC transporter; Part of the binding-protein-dependent transport system for glutamate and aspartate. Binds to both aspartate and glutamate","uniprotId":"GLTI_ECOLI"},"6881481":{"id":6881481,"externalId":"511145.b0656","name":"b0656","hasFamily":true,"annotation":"IS5 transposase and trans-activator"},"6881482":{"id":6881482,"externalId":"511145.b0657","name":"lnt","hasFamily":true,"annotation":"apolipoprotein N-acyltransferase; Transfers the fatty acyl group on membrane lipoproteins (By similarity)","uniprotId":"LNT_ECOLI"},"6881483":{"id":6881483,"externalId":"511145.b0658","name":"corC","hasFamily":true,"annotation":"predicteed ion transport; Plays a role in the transport of magnesium and cobalt ions (By similarity)","uniprotId":"CORC_ECOLI"},"6881484":{"id":6881484,"externalId":"511145.b0659","name":"ybeY","hasFamily":true,"annotation":"metal-binding heat shock protein required for rRNA maturation; Single strand-specific metallo-endoribonuclease involved in late-stage 70S ribosome quality control and in maturation of the 3' terminus of the 16S rRNA. Acts together with the RNase R to eliminate defective 70S ribosomes, but not properly matured 70S ribosomes or individual subunits, by a process mediated specifically by the 30S ribosomal subunit. Involved in the processing of 16S, 23S and 5S rRNAs, with a particularly strong effect on maturation at both the 5'-and 3'-ends of 16S rRNA as well as maturation of the 5'-end of [...] ","uniprotId":"YBEY_ECOLI"},"6881485":{"id":6881485,"externalId":"511145.b0660","name":"ybeZ","hasFamily":true,"annotation":"putative protein with nucleoside triphosphate hydrolase domain","uniprotId":"PHOL_ECOLI"},"6881486":{"id":6881486,"externalId":"511145.b0661","name":"miaB","hasFamily":true,"annotation":"tRNA-i(6)A37 methylthiotransferase; Catalyzes the methylthiolation of N6- (dimethylallyl)adenosine (i(6)A), leading to the formation of 2- methylthio-N6-(dimethylallyl)adenosine (ms(2)i(6)A) at position 37 in tRNAs that read codons beginning with uridine (By similarity)","uniprotId":"MIAB_ECOLI"},"6881487":{"id":6881487,"externalId":"511145.b0662","name":"ubiF","hasFamily":true,"annotation":"2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol oxygenase; Oxygenase that introduces the hydroxyl group at carbon five of 2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol resulting in the formation of 2-octaprenyl-3-methyl-5-hydroxy-6-methoxy-1,4- benzoquinol","uniprotId":"UBIF_ECOLI"},"6881488":{"id":6881488,"externalId":"511145.b0674","name":"asnB","hasFamily":true,"annotation":"asparagine synthetase B; Catalyzes the ATP-dependent conversion of aspartate into asparagine, using glutamine as a source of nitrogen. Can also use ammonia as the nitrogen source in vitro, albeit with lower efficiency. As nucleotide substrates, ATP and dATP are utilized at a similar rate in both the glutamine- and ammonia-dependent reactions, whereas GTP utilization is only 15% that of ATP, and CTP, UTP, ITP and XTP are very poor or not substrates. Also exhibits glutaminase activity","uniprotId":"ASNB_ECOLI"},"6881489":{"id":6881489,"externalId":"511145.b0675","name":"nagD","hasFamily":true,"annotation":"UMP phosphatase; Catalyzes the dephosphorylation of an unusually broad range of substrate including deoxyribo- and ribonucleoside tri-, di-, and monophosphates, as well as polyphosphate and glucose-1-P (Glu1P)","uniprotId":"NAGD_ECOLI"},"6881490":{"id":6881490,"externalId":"511145.b0676","name":"nagC","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator, repressor of N-acetylglucosamine; Acts as a repressor of the nagEBACD operon and acts both as an activator and a repressor for the transcription of the glmSU operon","uniprotId":"NAGC_ECOLI"},"6881491":{"id":6881491,"externalId":"511145.b0677","name":"nagA","hasFamily":true,"annotation":"N-acetylglucosamine-6-phosphate deacetylase; Catalyzes the deacetylation of N-acetyl-D-glucosamine-6- phosphate. In vitro, can also hydrolyze substrate analogs such as N-thioacetyl-D-glucosamine-6-phosphate, N-trifluoroacetyl-D- glucosamine-6-phosphate, N-acetyl-D-glucosamine-6-sulfate, N- acetyl-D-galactosamine-6-phosphate, and N-formyl-D-glucosamine-6- phosphate","uniprotId":"NAGA_ECOLI"},"6881492":{"id":6881492,"externalId":"511145.b0678","name":"nagB","hasFamily":true,"annotation":"glucosamine-6-phosphate deaminase; Catalyzes the reversible isomerization-deamination of glucosamine 6-phosphate (GlcN6P) to form fructose 6-phosphate (Fru6P) and ammonium ion (By similarity)","uniprotId":"NAGB_ECOLI"},"6881493":{"id":6881493,"externalId":"511145.b0679","name":"nagE","hasFamily":true,"annotation":"fused N-acetyl glucosamine specific PTS enzyme: IIC, IIB, and IIA components; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in N-acetylglucosamine transport","uniprotId":"PTW3C_ECOLI"},"6881494":{"id":6881494,"externalId":"511145.b0680","name":"glnS","hasFamily":true,"annotation":"glutamyl-tRNA synthetase","uniprotId":"SYQ_ECOLI"},"6881495":{"id":6881495,"externalId":"511145.b0681","name":"chiP","hasFamily":true,"annotation":"chitoporin, uptake of chitosugars; Involved in the uptake of chitosugars (Probable)","uniprotId":"CHIP_ECOLI"},"6881496":{"id":6881496,"externalId":"511145.b0682","name":"chiQ","hasFamily":true,"annotation":"novel lipoprotein induced by chitosugars","uniprotId":"CHIQ_ECOLI"},"6881497":{"id":6881497,"externalId":"511145.b0683","name":"fur","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator of siderophore biosynthesis and transport; Acts as a global negative controlling element, employing Fe(2+) as a cofactor to bind the operator of the repressed genes. Regulates the expression of several outer-membrane proteins including the iron transport operon","uniprotId":"FUR_ECOLI"},"6881498":{"id":6881498,"externalId":"511145.b0684","name":"fldA","hasFamily":true,"annotation":"flavodoxin 1; Low-potential electron donor to a number of redox enzymes (Potential). Involved in the reactivation of inactive cob(II)alamin in methionine synthase","uniprotId":"FLAV_ECOLI"},"6881499":{"id":6881499,"externalId":"511145.b0685","name":"ybfE","hasFamily":true,"annotation":"LexA-regulated conserved protein","uniprotId":"YBFE_ECOLI"},"6881500":{"id":6881500,"externalId":"511145.b0686","name":"ybfF","hasFamily":true,"annotation":"acyl-CoA esterase; Displays esterase activity toward palmitoyl-CoA, malonyl-CoA and pNP-butyrate","uniprotId":"YBFF_ECOLI"},"6881501":{"id":6881501,"externalId":"511145.b0687","name":"seqA","hasFamily":true,"annotation":"regulatory protein for replication initiation; Negative regulator of replication initiation, which contributes to regulation of DNA replication and ensures that replication initiation occurs exactly once per chromosome per cell cycle. Binds to pairs of hemimethylated GATC sequences in the oriC region, thus preventing assembly of replication proteins and re- initiation at newly replicated origins. Repression is relieved when the region becomes fully methylated. Can also bind to hemimethylated GATC sequences outside of oriC region. Binds, with less affinity, to fully methylated GATC site [...] ","uniprotId":"SEQA_ECOLI"},"6881502":{"id":6881502,"externalId":"511145.b0688","name":"pgm","hasFamily":true,"annotation":"phosphoglucomutase; This enzyme participates in both the breakdown and synthesis of glucose","uniprotId":"PGM_ECOLI"},"6881503":{"id":6881503,"externalId":"511145.b0689","name":"ybfP","hasFamily":false,"annotation":"lipoprotein","uniprotId":"YBFP_ECOLI"},"6881504":{"id":6881504,"externalId":"511145.b0692","name":"potE","hasFamily":true,"annotation":"putrescine/proton symporter: putrescine/ornithine antiporter; Probable putrescine-ornithine antiporter","uniprotId":"POTE_ECOLI"},"6881505":{"id":6881505,"externalId":"511145.b0693","name":"speF","hasFamily":true,"annotation":"ornithine decarboxylase isozyme, inducible","uniprotId":"DCOS_ECOLI"},"6881506":{"id":6881506,"externalId":"511145.b0694","name":"kdpE","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with KdpD; Member of the two-component regulatory system KdpD/KdpE involved in the regulation of the kdp operon","uniprotId":"KDPE_ECOLI"},"6881507":{"id":6881507,"externalId":"511145.b0695","name":"kdpD","hasFamily":true,"annotation":"fused sensory histidine kinase in two-component regulatory system with KdpE: signal sensing protein; Member of the two-component regulatory system KdpD/KdpE involved in the regulation of the kdp operon. KdpD may function as a membrane-associated protein kinase that phosphorylates KdpE in response to environmental signals","uniprotId":"KDPD_ECOLI"},"6881508":{"id":6881508,"externalId":"511145.b0696","name":"kdpC","hasFamily":true,"annotation":"potassium translocating ATPase, subunit C; One of the components of the high-affinity ATP-driven potassium transport (or KDP) system, which catalyzes the hydrolysis of ATP coupled with the exchange of hydrogen and potassium ions. The C subunit may be involved in assembly of the KDP complex (By similarity)","uniprotId":"ATKC_ECOLI"},"6881509":{"id":6881509,"externalId":"511145.b0697","name":"kdpB","hasFamily":true,"annotation":"potassium translocating ATPase, subunit B; One of the components of the high-affinity ATP-driven potassium transport (or KDP) system, which catalyzes the hydrolysis of ATP coupled with the exchange of hydrogen and potassium ions (By similarity)","uniprotId":"ATKB_ECOLI"},"6881510":{"id":6881510,"externalId":"511145.b0698","name":"kdpA","hasFamily":true,"annotation":"potassium translocating ATPase, subunit A; One of the components of the high-affinity ATP-driven potassium transport (or KDP) system, which catalyzes the hydrolysis of ATP coupled with the exchange of hydrogen and potassium ions (By similarity)","uniprotId":"ATKA_ECOLI"},"6881511":{"id":6881511,"externalId":"511145.b0699","name":"ybfA","hasFamily":true,"annotation":"predicted protein","uniprotId":"YBFA_ECOLI"},"6881512":{"id":6881512,"externalId":"511145.b0700","name":"BN17_20751","hasFamily":true,"annotation":"rhsC element core protein RshC; Rhs elements have a nonessential function. They may play an important role in the natural ecology of the cell","uniprotId":"RHSC_ECOLI"},"6881513":{"id":6881513,"externalId":"511145.b0702","name":"ybfB","hasFamily":false,"annotation":"putative membrane protein","uniprotId":"YBFB_ECOLI"},"6881514":{"id":6881514,"externalId":"511145.b0704","name":"ybfC","hasFamily":false,"annotation":"predicted protein","uniprotId":"YBFC_ECOLI"},"6881515":{"id":6881515,"externalId":"511145.b0706","name":"ybfD","hasFamily":true,"annotation":"conserved protein","uniprotId":"YBFD_ECOLI"},"6881516":{"id":6881516,"externalId":"511145.b0707","name":"ybgA","hasFamily":true,"annotation":"conserved protein, DUF1722 family","uniprotId":"YBGA_ECOLI"},"6881517":{"id":6881517,"externalId":"511145.b0708","name":"phr","hasFamily":true,"annotation":"deoxyribodipyrimidine photolyase, FAD-binding; Involved in repair of UV radiation-induced DNA damage. Catalyzes the light-dependent monomerization (300-600 nm) of cyclobutyl pyrimidine dimers (in cis-syn configuration), which are formed between adjacent bases on the same DNA strand upon exposure to ultraviolet radiation","uniprotId":"PHR_ECOLI"},"6881518":{"id":6881518,"externalId":"511145.b0709","name":"dtpD","hasFamily":true,"annotation":"dipeptide and tripeptide permease D; Probable proton-dependent permease that transports dipeptides (By similarity)","uniprotId":"DTPD_ECOLI"},"6881519":{"id":6881519,"externalId":"511145.b0710","name":"ybgI","hasFamily":true,"annotation":"conserved metal-binding protein, NIF3 family","uniprotId":"YBGI_ECOLI"},"6881520":{"id":6881520,"externalId":"511145.b0711","name":"ybgJ","hasFamily":true,"annotation":"predicted enzyme subunit","uniprotId":"YBGJ_ECOLI"},"6881521":{"id":6881521,"externalId":"511145.b0712","name":"ybgK","hasFamily":true,"annotation":"predicted enzyme subunit","uniprotId":"YBGK_ECOLI"},"6881522":{"id":6881522,"externalId":"511145.b0713","name":"ybgL","hasFamily":true,"annotation":"conserved protein, UPF0271 family","uniprotId":"YBGL_ECOLI"},"6881523":{"id":6881523,"externalId":"511145.b0714","name":"nei","hasFamily":true,"annotation":"endonuclease VIII/ 5-formyluracil/5-hydroxymethyluracil DNA glycosylase; Involved in base excision repair of DNA damaged by oxidation or by mutagenic agents. Acts as DNA glycosylase that recognizes and removes damaged bases. Has a preference for oxidized pyrimidines, such as thymine glycol, 5,6-dihydrouracil and 5,6-dihydrothymine. Acts on DNA bubble and 3'-fork structures, suggesting a role in replication-associated DNA repair. Has AP (apurinic/apyrimidinic) lyase activity and introduces nicks in the DNA strand. Cleaves the DNA backbone by beta-delta elimination to generate a single-s [...] ","uniprotId":"END8_ECOLI"},"6881524":{"id":6881524,"externalId":"511145.b0715","name":"abrB","hasFamily":true,"annotation":"regulator of aidB expression; inner membrane protein; Seems to be involved in the regulation of AidB","uniprotId":"ABRB_ECOLI"},"6881525":{"id":6881525,"externalId":"511145.b0716","name":"ybgO","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein; May be involved in a fimbrial system chaperoned by YbgP and exported by YbgQ","uniprotId":"YBGO_ECOLI"},"6881526":{"id":6881526,"externalId":"511145.b0717","name":"ybgP","hasFamily":true,"annotation":"putative periplasmic pilus chaperone; Could be required for the biogenesis of the putative YbgD fimbria","uniprotId":"YBGP_ECOLI"},"6881527":{"id":6881527,"externalId":"511145.b0718","name":"ybgQ","hasFamily":true,"annotation":"putative outer membrane protein; Could be involved in the export and assembly of the putative YbgD fimbrial subunit across the outer membrane","uniprotId":"YBGQ_ECOLI"},"6881528":{"id":6881528,"externalId":"511145.b0719","name":"ybgD","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YBGD_ECOLI"},"6881529":{"id":6881529,"externalId":"511145.b0720","name":"gltA","hasFamily":true,"annotation":"citrate synthase","uniprotId":"CISY_ECOLI"},"6881530":{"id":6881530,"externalId":"511145.b0721","name":"sdhC","hasFamily":true,"annotation":"succinate dehydrogenase, membrane subunit, binds cytochrome b556; Membrane-anchoring subunit of succinate dehydrogenase (SDH)","uniprotId":"DHSC_ECOLI"},"6881531":{"id":6881531,"externalId":"511145.b0722","name":"sdhD","hasFamily":true,"annotation":"succinate dehydrogenase, membrane subunit, binds cytochrome b556; Membrane-anchoring subunit of succinate dehydrogenase (SDH)","uniprotId":"DHSD_ECOLI"},"6881532":{"id":6881532,"externalId":"511145.b0723","name":"sdhA","hasFamily":true,"annotation":"succinate dehydrogenase, flavoprotein subunit; Two distinct, membrane-bound, FAD-containing enzymes are responsible for the catalysis of fumarate and succinate interconversion; the fumarate reductase is used in anaerobic growth, and the succinate dehydrogenase is used in aerobic growth","uniprotId":"DHSA_ECOLI"},"6881533":{"id":6881533,"externalId":"511145.b0724","name":"sdhB","hasFamily":true,"annotation":"succinate dehydrogenase, FeS subunit; Two distinct, membrane-bound, FAD-containing enzymes are responsible for the catalysis of fumarate and succinate interconversion; the fumarate reductase is used in anaerobic growth, and the succinate dehydrogenase is used in aerobic growth","uniprotId":"DHSB_ECOLI"},"6881534":{"id":6881534,"externalId":"511145.b0726","name":"sucA","hasFamily":true,"annotation":"2-oxoglutarate decarboxylase, thiamin-requiring; The 2-oxoglutarate dehydrogenase complex catalyzes the overall conversion of 2-oxoglutarate to succinyl-CoA and CO(2). It contains multiple copies of three enzymatic components: 2- oxoglutarate dehydrogenase (E1), dihydrolipoamide succinyltransferase (E2) and lipoamide dehydrogenase (E3)","uniprotId":"ODO1_ECOLI"},"6881535":{"id":6881535,"externalId":"511145.b0727","name":"sucB","hasFamily":true,"annotation":"dihydrolipoyltranssuccinase; The 2-oxoglutarate dehydrogenase complex catalyzes the overall conversion of 2-oxoglutarate to succinyl-CoA and CO(2). It contains multiple copies of three enzymatic components: 2- oxoglutarate dehydrogenase (E1), dihydrolipoamide succinyltransferase (E2) and lipoamide dehydrogenase (E3)","uniprotId":"ODO2_ECOLI"},"6881536":{"id":6881536,"externalId":"511145.b0728","name":"sucC","hasFamily":true,"annotation":"succinyl-CoA synthetase, beta subunit","uniprotId":"SUCC_ECOLI"},"6881537":{"id":6881537,"externalId":"511145.b0729","name":"sucD","hasFamily":true,"annotation":"succinyl-CoA synthetase, NAD(P)-binding, alpha subunit","uniprotId":"SUCD_ECOLI"},"6881538":{"id":6881538,"externalId":"511145.b0730","name":"farR","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator, fatty-acyl-binding; Represses mngA and mngB. Regulates its own expression","uniprotId":"MNGR_ECOLI"},"6881539":{"id":6881539,"externalId":"511145.b0731","name":"hrsA","hasFamily":false,"annotation":"fused 2-O-a-mannosyl-D-glycerate specific PTS enzymes: IIA component/IIB component/IIC component; Involved in thermoinduction of ompC","uniprotId":"HRSA_ECOLI"},"6881540":{"id":6881540,"externalId":"511145.b0732","name":"mngB","hasFamily":true,"annotation":"alpha-mannosidase; May hydrolyze mannosyl-D-glycerate to mannose-6- phosphate and glycerate","uniprotId":"MNGB_ECOLI"},"6881541":{"id":6881541,"externalId":"511145.b0733","name":"cydA","hasFamily":true,"annotation":"cytochrome d terminal oxidase, subunit I; Cytochrome d terminal oxidase complex is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at low aeration","uniprotId":"CYDA_ECOLI"},"6881542":{"id":6881542,"externalId":"511145.b0734","name":"cydB","hasFamily":true,"annotation":"cytochrome d terminal oxidase, subunit II; Cytochrome d terminal oxidase complex is the component of the aerobic respiratory chain of E.coli that predominates when cells are grown at low aeration","uniprotId":"CYDB_ECOLI"},"6881543":{"id":6881543,"externalId":"511145.b0735","name":"ybgE","hasFamily":true,"annotation":"conserved protein","uniprotId":"YBGE_ECOLI"},"6881544":{"id":6881544,"externalId":"511145.b0736","name":"ybgC","hasFamily":true,"annotation":"acyl-CoA thioesterase, involved in phospholipid metabolism; Thioesterase that appears to be involved in phospholipid metabolism. Some specific acyl-ACPs could be physiological substrates. Displays acyl-CoA thioesterase activity on malonyl-CoA in vitro, catalyzing the hydrolysis of the thioester bond","uniprotId":"YBGC_ECOLI"},"6881545":{"id":6881545,"externalId":"511145.b0737","name":"tolQ","hasFamily":true,"annotation":"membrane spanning protein in TolA-TolQ-TolR complex; Involved in the TonB-independent uptake of group A colicins (colicins A, E1, E2, E3, and K)","uniprotId":"TOLQ_ECOLI"},"6881546":{"id":6881546,"externalId":"511145.b0738","name":"tolR","hasFamily":true,"annotation":"membrane spanning protein in TolA-TolQ-TolR complex; Involved in the TonB-independent uptake of group A colicins (colicins A, E1, E2, E3, and K)","uniprotId":"TOLR_ECOLI"},"6881547":{"id":6881547,"externalId":"511145.b0739","name":"tolA","hasFamily":true,"annotation":"membrane anchored protein in TolA-TolQ-TolR complex; Involved in the TonB-independent uptake of group A colicins (colicins A, E1, E2, E3, and K). Necessary for the colicins to reach their respective targets after initial binding to the bacteria. Also involved in the translocation of bacteriophage DNA","uniprotId":"TOLA_ECOLI"},"6881548":{"id":6881548,"externalId":"511145.b0740","name":"tolB","hasFamily":true,"annotation":"periplasmic protein; Involved in the TonB-independent uptake of group A colicins (colicins A, E1, E2, E3 and K). Necessary for the colicins to reach their respective targets after initial binding to the bacteria","uniprotId":"TOLB_ECOLI"},"6881549":{"id":6881549,"externalId":"511145.b0741","name":"pal","hasFamily":true,"annotation":"peptidoglycan-associated outer membrane lipoprotein; Thought to play a role in bacterial envelope integrity. Very strongly associated with the peptidoglycan","uniprotId":"PAL_ECOLI"},"6881550":{"id":6881550,"externalId":"511145.b0742","name":"ybgF","hasFamily":true,"annotation":"periplasmic TolA-binding protein","uniprotId":"YBGF_ECOLI"},"6881551":{"id":6881551,"externalId":"511145.b0750","name":"nadA","hasFamily":true,"annotation":"quinolinate synthase, subunit A; Catalyzes the condensation of iminoaspartate with dihydroxyacetone phosphate to form quinolinate (By similarity)","uniprotId":"NADA_ECOLI"},"6881552":{"id":6881552,"externalId":"511145.b0751","name":"pnuC","hasFamily":true,"annotation":"nicotinamide riboside transporter; Required for nicotinamide riboside transport across the inner membrane","uniprotId":"PNUC_ECOLI"},"6881553":{"id":6881553,"externalId":"511145.b0752","name":"ybgR","hasFamily":true,"annotation":"zinc efflux system; Involved in zinc efflux across the cytoplasmic membrane, thus reducing zinc accumulation in the cytoplasm and rendering bacteria more resistant to zinc. It may contribute to zinc homeostasis at low concentrations of zinc, whereas ZntA is required for growth at more toxic concentrations","uniprotId":"ZITB_ECOLI"},"6881554":{"id":6881554,"externalId":"511145.b0753","name":"ybgS","hasFamily":true,"annotation":"conserved protein","uniprotId":"YBGS_ECOLI"},"6881555":{"id":6881555,"externalId":"511145.b0754","name":"aroG","hasFamily":true,"annotation":"3-deoxy-D-arabino-heptulosonate-7-phosphate synthase, phenylalanine repressible; Stereospecific condensation of phosphoenolpyruvate (PEP) and D-erythrose-4-phosphate (E4P) giving rise to 3-deoxy-D- arabino-heptulosonate-7-phosphate (DAHP) (By similarity)","uniprotId":"AROG_ECOLI"},"6881556":{"id":6881556,"externalId":"511145.b0755","name":"gpmA","hasFamily":true,"annotation":"phosphoglyceromutase 1; Catalyzes the interconversion of 2-phosphoglycerate and 3-phosphoglycerate (By similarity)","uniprotId":"GPMA_ECOLI"},"6881557":{"id":6881557,"externalId":"511145.b0756","name":"galM","hasFamily":true,"annotation":"galactose-1-epimerase (mutarotase); Converts alpha-aldose to the beta-anomer. It is active on D-glucose, L-arabinose, D-xylose, D-galactose, maltose and lactose (By similarity)","uniprotId":"GALM_ECOLI"},"6881558":{"id":6881558,"externalId":"511145.b0757","name":"galK","hasFamily":true,"annotation":"galactokinase; Catalyzes the transfer of the gamma-phosphate of ATP to D-galactose to form alpha-D-galactose-1-phosphate (Gal-1-P). To a lesser extent, is also able to phosphorylate 2-deoxy-D-galactose and D-galactosamine. Is not able to use D-galacturonic acid, D- talose, L-altrose, and L-glucose as substrates","uniprotId":"GAL1_ECOLI"},"6881559":{"id":6881559,"externalId":"511145.b0758","name":"galT","hasFamily":true,"annotation":"galactose-1-phosphate uridylyltransferase","uniprotId":"GAL7_ECOLI"},"6881560":{"id":6881560,"externalId":"511145.b0759","name":"galE","hasFamily":true,"annotation":"UDP-galactose-4-epimerase","uniprotId":"GALE_ECOLI"},"6881561":{"id":6881561,"externalId":"511145.b0760","name":"modF","hasFamily":true,"annotation":"fused molybdate transporter subunits of ABC superfamily: ATP-binding components; Involved in the transport of molybdenum into the cell. Involved in photorepair. Could act on UV-induced DNA damage other than pyrimidine dimers","uniprotId":"MODF_ECOLI"},"6881562":{"id":6881562,"externalId":"511145.b0761","name":"modE","hasFamily":true,"annotation":"DNA-binding transcriptional repressor for the molybdenum transport operon modABC; The ModE-Mo complex acts as a repressor of the modABC operon, involved in the transport of molybdate. Upon binding molybdate, the conformation of the protein changes, promoting dimerization of ModE-Mo. The protein dimer is then competent to bind a DNA region, upstream of the modABC operon, which contains an 8-base inverted repeat 5'-TAACGTTA-3' flanked by two CAT boxes. Acts also as an enhancer of the expression of genes coding for molybdoenzymes, both directly and indirectly. ModE also interacts with tungstate","uniprotId":"MODE_ECOLI"},"6881563":{"id":6881563,"externalId":"511145.b0762","name":"ybhT","hasFamily":true,"annotation":"predicted protein","uniprotId":"YBHT_ECOLI"},"6881564":{"id":6881564,"externalId":"511145.b0763","name":"modA","hasFamily":true,"annotation":"molybdate transporter subunit; Involved in the transport of molybdenum into the cell. Binds molybdate with high specificity and affinity","uniprotId":"MODA_ECOLI"},"6881565":{"id":6881565,"externalId":"511145.b0764","name":"modB","hasFamily":true,"annotation":"molybdate transporter subunit; Part of the binding-protein-dependent transport system for molybdenum; probably responsible for the translocation of the substrate across the membrane","uniprotId":"MODB_ECOLI"},"6881566":{"id":6881566,"externalId":"511145.b0765","name":"modC","hasFamily":true,"annotation":"molybdate transporter subunit; Part of the ABC transporter complex ModABC involved in molybdenum import. Responsible for energy coupling to the transport system (By similarity)","uniprotId":"MODC_ECOLI"},"6881567":{"id":6881567,"externalId":"511145.b0766","name":"ybhA","hasFamily":true,"annotation":"pyridoxal phosphate (PLP) phosphatase; Catalyzes the dephosphorylation of pyridoxal-phosphate (PLP). Can also hydrolyze erythrose-4-phosphate (Ery4P) and fructose-1,6-bis-phosphate (Fru1,6bisP)","uniprotId":"YBHA_ECOLI"},"6881568":{"id":6881568,"externalId":"511145.b0767","name":"pgl","hasFamily":true,"annotation":"6-phosphogluconolactonase; Catalyzes the hydrolysis of 6-phosphogluconolactone to 6-phosphogluconate (By similarity)","uniprotId":"6PGL_ECOLI"},"6881569":{"id":6881569,"externalId":"511145.b0768","name":"ybhD","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YBHD_ECOLI"},"6881570":{"id":6881570,"externalId":"511145.b0769","name":"ybhH","hasFamily":true,"annotation":"conserved protein","uniprotId":"YBHH_ECOLI"},"6881571":{"id":6881571,"externalId":"511145.b0770","name":"ybhI","hasFamily":true,"annotation":"putative transporter","uniprotId":"YBHI_ECOLI"},"6881572":{"id":6881572,"externalId":"511145.b0771","name":"ybhJ","hasFamily":true,"annotation":"putative hydratase","uniprotId":"YBHJ_ECOLI"},"6881573":{"id":6881573,"externalId":"511145.b0772","name":"ybhC","hasFamily":true,"annotation":"acyl-CoA thioesterase, lipoprotein; Putative thioesterase. Does not bind pectin, and has no pectinesterase activity","uniprotId":"YBHC_ECOLI"},"6881574":{"id":6881574,"externalId":"511145.b0773","name":"ybhB","hasFamily":true,"annotation":"kinase inhibitor homolog, UPF0098 family","uniprotId":"YBHB_ECOLI"},"6881575":{"id":6881575,"externalId":"511145.b0774","name":"bioA","hasFamily":true,"annotation":"7,8-diaminopelargonic acid synthase, PLP-dependent; Catalyzes the transfer of the alpha-amino group from S- adenosyl-L-methionine (SAM) to 7-keto-8-aminopelargonic acid (KAPA) to form 7,8-diaminopelargonic acid (DAPA). It is the only animotransferase known to utilize SAM as an amino donor (By similarity)","uniprotId":"BIOA_ECOLI"},"6881576":{"id":6881576,"externalId":"511145.b0775","name":"bioB","hasFamily":true,"annotation":"biotin synthase; Catalyzes the conversion of dethiobiotin (DTB) to biotin by the insertion of a sulfur atom into dethiobiotin via a radical- based mechanism (By similarity)","uniprotId":"BIOB_ECOLI"},"6881577":{"id":6881577,"externalId":"511145.b0776","name":"bioF","hasFamily":true,"annotation":"8-amino-7-oxononanoate synthase; Catalyzes the decarboxylative condensation of pimeloyl- [acyl-carrier protein] and L-alanine to produce 8-amino-7- oxononanoate (AON), [acyl-carrier protein], and carbon dioxide. Can also use pimeloyl-CoA instead of pimeloyl-ACP as substrate, but it is believed that pimeloyl-ACP rather than pimeloyl-CoA is the physiological substrate of BioF","uniprotId":"BIOF_ECOLI"},"6881578":{"id":6881578,"externalId":"511145.b0777","name":"bioC","hasFamily":true,"annotation":"malonyl-CoA methyltransferase, SAM-dependent; Converts the free carboxyl group of a malonyl-thioester to its methyl ester by transfer of a methyl group from S-adenosyl- L-methionine (SAM). It allows to synthesize pimeloyl-ACP via the fatty acid synthetic pathway. E.coli employs a methylation and demethylation strategy to allow elongation of a temporarily disguised malonate moiety to a pimelate moiety by the fatty acid synthetic enzymes","uniprotId":"BIOC_ECOLI"},"6881579":{"id":6881579,"externalId":"511145.b0778","name":"bioD","hasFamily":true,"annotation":"dethiobiotin synthetase; Catalyzes a mechanistically unusual reaction, the ATP- dependent insertion of CO2 between the N7 and N8 nitrogen atoms of 7,8-diaminopelargonic acid (DAPA) to form an ureido ring. Only CTP can partially replace ATP while diaminobiotin is only 37% as effective as 7,8-diaminopelargonic acid","uniprotId":"BIOD1_ECOLI"},"6881580":{"id":6881580,"externalId":"511145.b0779","name":"uvrB","hasFamily":true,"annotation":"excinulease of nucleotide excision repair, DNA damage recognition component; The UvrABC repair system catalyzes the recognition and processing of DNA lesions. A damage recognition complex composed of 2 UvrA and 2 UvrB subunits scans DNA for abnormalities. Upon binding of the UvrA(2)B(2) complex to a putative damaged site, the DNA wraps around one UvrB monomer. DNA wrap is dependent on ATP binding by UvrB and probably causes local melting of the DNA helix, facilitating insertion of UvrB beta-hairpin between the DNA strands. Then UvrB probes one DNA strand for the presence of a lesion. I [...] ","uniprotId":"UVRB_ECOLI"},"6881581":{"id":6881581,"externalId":"511145.b0780","name":"ybhK","hasFamily":true,"annotation":"putative transferase with NAD(P)-binding Rossmann-fold domain","uniprotId":"YBHK_ECOLI"},"6881582":{"id":6881582,"externalId":"511145.b0781","name":"moaA","hasFamily":true,"annotation":"molybdopterin biosynthesis protein A; Catalyzes, together with MoaC, the conversion of 5'-GTP to cyclic pyranopterin monophosphate (cPMP or molybdopterin precursor Z) (By similarity)","uniprotId":"MOAA_ECOLI"},"6881583":{"id":6881583,"externalId":"511145.b0782","name":"moaB","hasFamily":true,"annotation":"molybdopterin biosynthesis protein B; May be involved in the biosynthesis of molybdopterin. Can bind GTP and has low GTPase activity. Can bind MPT, but has no MPT adenylyl transferase activity","uniprotId":"MOAB_ECOLI"},"6881584":{"id":6881584,"externalId":"511145.b0783","name":"moaC","hasFamily":true,"annotation":"molybdopterin biosynthesis, protein C; Together with MoaA, is involved in the conversion of 5'- GTP to cyclic pyranopterin monophosphate (cPMP or molybdopterin precursor Z) (By similarity)","uniprotId":"MOAC_ECOLI"},"6881585":{"id":6881585,"externalId":"511145.b0784","name":"moaD","hasFamily":true,"annotation":"molybdopterin synthase, small subunit; Involved in sulfur transfer in the conversion of molybdopterin precursor Z to molybdopterin","uniprotId":"MOAD_ECOLI"},"6881586":{"id":6881586,"externalId":"511145.b0785","name":"moaE","hasFamily":true,"annotation":"molybdopterin synthase, large subunit; Converts molybdopterin precursor Z to molybdopterin. This requires the incorporation of two sulfur atoms into precursor Z to generate a dithiolene group. The sulfur is provided by MoaD","uniprotId":"MOAE_ECOLI"},"6881587":{"id":6881587,"externalId":"511145.b0786","name":"ybhL","hasFamily":true,"annotation":"inner membrane protein, UPF0005 family","uniprotId":"YBHL_ECOLI"},"6881588":{"id":6881588,"externalId":"511145.b0787","name":"ybhM","hasFamily":true,"annotation":"inner membrane protein, UPF0005 family","uniprotId":"YBHM_ECOLI"},"6881589":{"id":6881589,"externalId":"511145.b0788","name":"ybhN","hasFamily":true,"annotation":"conserved inner membrane protein","uniprotId":"YBHN_ECOLI"},"6881590":{"id":6881590,"externalId":"511145.b0789","name":"ybhO","hasFamily":true,"annotation":"cardiolipin synthase 2; Catalyzes, in vitro, the phosphatidyl group transfer from one phosphatidylglycerol molecule to another to form cardiolipin (CL) (diphosphatidylglycerol) and glycerol. Can also catalyze phosphatidyl group transfer to water to form phosphatidate. Catalyzes little, if any, cardiolipin synthesis in vivo, even when the expression level is very high","uniprotId":"YBHO_ECOLI"},"6881591":{"id":6881591,"externalId":"511145.b0790","name":"ybhP","hasFamily":true,"annotation":"conserved protein, endo/exonuclease/phosphatase family PFAM PF03372","uniprotId":"YBHP_ECOLI"},"6881592":{"id":6881592,"externalId":"511145.b0791","name":"ybhQ","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YBHQ_ECOLI"},"6881593":{"id":6881593,"externalId":"511145.b0792","name":"ybhR","hasFamily":true,"annotation":"putative transporter subunit: membrane component of ABC superfamily","uniprotId":"YBHR_ECOLI"},"6881594":{"id":6881594,"externalId":"511145.b0793","name":"ybhS","hasFamily":true,"annotation":"putative transporter subunit: membrane component of ABC superfamily","uniprotId":"YBHS_ECOLI"},"6881595":{"id":6881595,"externalId":"511145.b0794","name":"ybhF","hasFamily":true,"annotation":"fused predicted transporter subunits of ABC superfamily: ATP-binding components","uniprotId":"YBHF_ECOLI"},"6881596":{"id":6881596,"externalId":"511145.b0795","name":"ybhG","hasFamily":true,"annotation":"putative membrane fusion protein (MFP) component of efflux pump, membrane anchor","uniprotId":"YBHG_ECOLI"},"6881597":{"id":6881597,"externalId":"511145.b0796","name":"ybiH","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YBIH_ECOLI"},"6881598":{"id":6881598,"externalId":"511145.b0797","name":"rhlE","hasFamily":true,"annotation":"ATP-dependent RNA helicase; Acts as an ATP-dependent RNA helicase, able to unwind ds-RNA","uniprotId":"RHLE_ECOLI"},"6881599":{"id":6881599,"externalId":"511145.b0798","name":"ybiA","hasFamily":true,"annotation":"conserved protein, DUF1768 family; Required for swarming motility","uniprotId":"YBIA_ECOLI"},"6881600":{"id":6881600,"externalId":"511145.b0799","name":"dinG","hasFamily":true,"annotation":"ATP-dependent DNA helicase; Probable helicase involved in DNA repair and perhaps also replication","uniprotId":"DING_ECOLI"},"6881601":{"id":6881601,"externalId":"511145.b0800","name":"ybiB","hasFamily":true,"annotation":"putative family 3 glycosyltransferase","uniprotId":"YBIB_ECOLI"},"6881602":{"id":6881602,"externalId":"511145.b0801","name":"ybiC","hasFamily":true,"annotation":"putative dehydrogenase","uniprotId":"YBIC_ECOLI"},"6881603":{"id":6881603,"externalId":"511145.b0802","name":"ybiJ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YBIJ_ECOLI"},"6881604":{"id":6881604,"externalId":"511145.b0803","name":"ybiI","hasFamily":true,"annotation":"DksA-type zinc finger protein","uniprotId":"YBII_ECOLI"},"6881605":{"id":6881605,"externalId":"511145.b0804","name":"ybiX","hasFamily":true,"annotation":"conserved protein, Fe(II)-dependent oxygenase superfamily","uniprotId":"YBIX_ECOLI"},"6881606":{"id":6881606,"externalId":"511145.b0805","name":"fiu","hasFamily":true,"annotation":"catecholate siderophore receptor Fiu; Involved in the active transport across the outer membrane of iron complexed with catecholate siderophores such as dihydroxybenzoylserine and dihydroxybenzoate. It derives its energy for transport by interacting with the trans-periplasmic membrane protein TonB. Can also transport catechol-substitued cephalosporins. Receptor for microcins M, H47 and E492","uniprotId":"FIU_ECOLI"},"6881607":{"id":6881607,"externalId":"511145.b0806","name":"mcbA","hasFamily":false,"annotation":"colanic acid mucoidy stimulation protein; Affects biofilm formation and mucoidy","uniprotId":"MCBA_ECOLI"},"6881608":{"id":6881608,"externalId":"511145.b0807","name":"rlmF","hasFamily":true,"annotation":"23S rRNA mA1618 methyltransferase, SAM-dependent; Specifically methylates the adenine in position 1618 of 23S rRNA (By similarity)","uniprotId":"RLMF_ECOLI"},"6881609":{"id":6881609,"externalId":"511145.b0808","name":"ybiO","hasFamily":true,"annotation":"predicted mechanosensitive channel","uniprotId":"YBIO_ECOLI"},"6881610":{"id":6881610,"externalId":"511145.b0809","name":"glnQ","hasFamily":true,"annotation":"glutamine transporter subunit; Part of the binding-protein-dependent transport system for glutamine. Probably responsible for energy coupling to the transport system","uniprotId":"GLNQ_ECOLI"},"6881611":{"id":6881611,"externalId":"511145.b0810","name":"glnP","hasFamily":true,"annotation":"glutamine transporter subunit; Part of the binding-protein-dependent transport system for glutamine; probably responsible for the translocation of the substrate across the membrane","uniprotId":"GLNP_ECOLI"},"6881612":{"id":6881612,"externalId":"511145.b0811","name":"glnH","hasFamily":true,"annotation":"glutamine transporter subunit; Involved in a glutamine-transport system GlnHPQ","uniprotId":"GLNH_ECOLI"},"6881613":{"id":6881613,"externalId":"511145.b0812","name":"dps","hasFamily":true,"annotation":"Fe-binding and storage protein; During stationary phase, binds the chromosome non- specifically, forming a highly ordered and stable dps-DNA co- crystal within which chromosomal DNA is condensed and protected from diverse damages. It protects DNA from oxidative damage by sequestering intracellular Fe(2+) ion and storing it in the form of Fe(3+) oxyhydroxide mineral, which can be released after reduction. One hydrogen peroxide oxidizes two Fe(2+) ions, which prevents hydroxyl radical production by the Fenton reaction. Dps also protects the cell from UV and gamma irradiation, iron and co [...] ","uniprotId":"DPS_ECOLI"},"6881614":{"id":6881614,"externalId":"511145.b0813","name":"rhtA","hasFamily":true,"annotation":"threonine and homoserine efflux system","uniprotId":"RHTA_ECOLI"},"6881615":{"id":6881615,"externalId":"511145.b0814","name":"ompX","hasFamily":true,"annotation":"outer membrane protein X","uniprotId":"OMPX_ECOLI"},"6881616":{"id":6881616,"externalId":"511145.b0815","name":"ybiP","hasFamily":true,"annotation":"putative hydrolase, inner membrane","uniprotId":"YBIP_ECOLI"},"6881617":{"id":6881617,"externalId":"511145.b0816","name":"yliL","hasFamily":false,"annotation":"predicted protein"},"6881618":{"id":6881618,"externalId":"511145.b0817","name":"mntR","hasFamily":true,"annotation":"DNA-binding transcriptional regulator of mntH; In the presence of manganese, represses expression of mntH and mntS. Up-regulates expression of mntP","uniprotId":"MNTR_ECOLI"},"6881619":{"id":6881619,"externalId":"511145.b0818","name":"ybiR","hasFamily":true,"annotation":"putative transporter","uniprotId":"YBIR_ECOLI"},"6881620":{"id":6881620,"externalId":"511145.b0819","name":"ybiS","hasFamily":true,"annotation":"L,D-transpeptidase linking Lpp to murein; Responsible, at least in part, for anchoring of the major outer membrane lipoprotein (Lpp, also known as the Braun lipoprotein) to the peptidoglycan via a meso-diaminopimelyl-L-Lys- bond on the terminal residue of Lpp. Can be oxidized in vivo, its reduction depends preferentially on DsbG, although DsbC is able to partially replace DsbG","uniprotId":"YBIS_ECOLI"},"6881621":{"id":6881621,"externalId":"511145.b0820","name":"ybiT","hasFamily":true,"annotation":"fused predicted transporter subunits of ABC superfamily: ATP-binding components","uniprotId":"YBIT_ECOLI"},"6881622":{"id":6881622,"externalId":"511145.b0821","name":"ybiU","hasFamily":true,"annotation":"predicted protein","uniprotId":"YBIU_ECOLI"},"6881623":{"id":6881623,"externalId":"511145.b0822","name":"ybiV","hasFamily":true,"annotation":"sugar phosphatase; preference for fructose-1-P, ribose-5-P and glucose-6-P; Catalyzes the hydrolysis of sugar phosphate to sugar and inorganic phosphate. Has a wide substrate specificity catalyzing the hydrolysis of fructose-1-phosphate (Fru1P) most efficiently, but it remains uncertain if this is the real substrate in vivo. Appears to have a low level of phosphotransferase activity using monophosphates as the phosphate donor","uniprotId":"SUPH_ECOLI"},"6881624":{"id":6881624,"externalId":"511145.b0823","name":"ybiW","hasFamily":true,"annotation":"putative pyruvate formate lyase","uniprotId":"PFLF_ECOLI"},"6881625":{"id":6881625,"externalId":"511145.b0824","name":"ybiY","hasFamily":true,"annotation":"putative pyruvate formate lyase activating enzyme; Activation of pyruvate formate-lyase 2 under anaerobic conditions by generation of an organic free radical, using S- adenosylmethionine and reduced flavodoxin as cosubstrates to produce 5'-deoxy-adenosine (By similarity)","uniprotId":"PFLE_ECOLI"},"6881626":{"id":6881626,"externalId":"511145.b0825","name":"fsaA","hasFamily":true,"annotation":"fructose-6-phosphate aldolase 1","uniprotId":"FSAA_ECOLI"},"6881627":{"id":6881627,"externalId":"511145.b0826","name":"moeB","hasFamily":true,"annotation":"molybdopterin synthase sulfurylase; Catalyzes the adenylation by ATP of the carboxyl group of the C-terminal glycine of sulfur carrier protein MoaD","uniprotId":"MOEB_ECOLI"},"6881628":{"id":6881628,"externalId":"511145.b0827","name":"moeA","hasFamily":true,"annotation":"molybdopterin biosynthesis protein; Catalyzes the insertion of molybdate into adenylated molybdopterin with the concomitant release of AMP","uniprotId":"MOEA_ECOLI"},"6881629":{"id":6881629,"externalId":"511145.b0828","name":"iaaA","hasFamily":true,"annotation":"Isoaspartyl peptidase; Degrades proteins damaged by L-isoaspartyl residue formation (also known as beta-Asp residues). Degrades L- isoaspartyl-containing di- and maybe also tripeptides. Also has L- asparaginase activity, although this may not be its principal function","uniprotId":"IAAA_ECOLI"},"6881630":{"id":6881630,"externalId":"511145.b0829","name":"gsiA","hasFamily":true,"annotation":"glutathione transporter ATP-binding protein, ABC superfamily; Part of the ABC transporter complex GsiABCD involved in glutathione import. Responsible for energy coupling to the transport system","uniprotId":"GSIA_ECOLI"},"6881631":{"id":6881631,"externalId":"511145.b0830","name":"gsiB","hasFamily":true,"annotation":"glutathione periplasmic binding protein, ABC superfamily transporter; Part of the ABC transporter complex GsiABCD involved in glutathione import","uniprotId":"D9IX93_ECOLI"},"6881632":{"id":6881632,"externalId":"511145.b0831","name":"gsiC","hasFamily":true,"annotation":"glutathione transporter, permease component, ABC superfamily; Part of the ABC transporter complex GsiABCD involved in glutathione import. Probably responsible for the translocation of the substrate across the membrane (Probable)","uniprotId":"GSIC_ECOLI"},"6881633":{"id":6881633,"externalId":"511145.b0832","name":"gsiD","hasFamily":true,"annotation":"glutathione transporter, permease component, ABC superfamily; Part of the ABC transporter complex GsiABCD involved in glutathione import. Probably responsible for the translocation of the substrate across the membrane (Probable)","uniprotId":"GSID_ECOLI"},"6881634":{"id":6881634,"externalId":"511145.b0833","name":"yliE","hasFamily":false,"annotation":"putative membrane-anchored cyclic-di-GMP phosphodiesterase; Overexpression reduces biofilm formation. Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"YLIE_ECOLI"},"6881635":{"id":6881635,"externalId":"511145.b0834","name":"yliF","hasFamily":false,"annotation":"predicted diguanylate cyclase","uniprotId":"YLIF_ECOLI"},"6881636":{"id":6881636,"externalId":"511145.b0835","name":"rimO","hasFamily":true,"annotation":"ribosomal protein S12 methylthiotransferase; radical SAM superfamily; Catalyzes the methylthiolation of an aspartic acid residue of ribosomal protein S12 (By similarity)","uniprotId":"RIMO_ECOLI"},"6881637":{"id":6881637,"externalId":"511145.b0836","name":"bssR","hasFamily":true,"annotation":"repressor of biofilm formation by indole transport regulation; Represses biofilm formation in M9C glu and LB glu media but not in M9C and LB media. Seems to act as a global regulator of several genes involved in catabolite repression and stress response and regulation of the uptake and export of signaling pathways. Could be involved the regulation of indole as well as uptake and export of AI-2 through a cAMP-dependent pathway","uniprotId":"BSSR_ECOLI"},"6881638":{"id":6881638,"externalId":"511145.b0837","name":"yliI","hasFamily":true,"annotation":"soluble aldose sugar dehydrogenase; Aldose sugar dehydrogenase with broad substrate specificity. The physiological substrate is unknown. Can oxidize glucose to gluconolactone. Can also utilize D-arabinose, L- arabinose and 2-deoxy-glucose. Has higher activity towards oligomeric sugars, such as maltose, maltotriose or cellobiose. It may function to input sugar-derived electrons into the respiratory network","uniprotId":"YLII_ECOLI"},"6881639":{"id":6881639,"externalId":"511145.b0838","name":"gstB","hasFamily":true,"annotation":"glutathione S-transferase; Conjugation of reduced glutathione to a wide number of exogenous and endogenous hydrophobic electrophiles. Catalyzes the glutathione-dependent dehalogenation of bromoacetate","uniprotId":"GSTB_ECOLI"},"6881640":{"id":6881640,"externalId":"511145.b0839","name":"dacC","hasFamily":true,"annotation":"D-alanyl-D-alanine carboxypeptidase (penicillin-binding protein 6a); Removes C-terminal D-alanyl residues from sugar-peptide cell wall precursors","uniprotId":"DACC_ECOLI"},"6881641":{"id":6881641,"externalId":"511145.b0840","name":"deoR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; This protein is one of the repressors that regulate the expression of deoCABD genes, which encode nucleotide and deoxy ribonucleotide catabolizing enzymes. It also negatively regulates the expression of nupG (a transport protein) and tsx (a pore- forming protein). The inducer is deoxyribose-5-phosphate","uniprotId":"DEOR_ECOLI"},"6881642":{"id":6881642,"externalId":"511145.b0841","name":"ybjG","hasFamily":true,"annotation":"undecaprenyl pyrophosphate phosphatase; Overexpression leads to increased undecaprenyl diphosphatase activity and to increased resistance to bacitracin. May have a preferred substrate other than undecaprenyl diphosphate in vivo","uniprotId":"YBJG_ECOLI"},"6881643":{"id":6881643,"externalId":"511145.b0842","name":"cmr","hasFamily":true,"annotation":"multidrug efflux system protein; Efflux pump driven by the proton motive force. Confers resistance to a broad spectrum of chemically unrelated drugs. Confers resistance to a diverse group of cationic or zwitterionic lipophilic compounds such as ethidium bromide, tetraphenylphosphonium, rhodamine, daunomycin, benzalkonium, rifampicin, tetracycline, puromycin, and to chemically unrelated, clinically important antibiotics such as chloramphenicol, erythromycin, and certain aminoglycosides and fluoroquinolones. Overexpression results in isopropyl-beta-D-thiogalactopyranoside (IPTG) exclusio [...] ","uniprotId":"MDFA_ECOLI"},"6881644":{"id":6881644,"externalId":"511145.b0843","name":"ybjH","hasFamily":false,"annotation":"predicted protein","uniprotId":"YBJH_ECOLI"},"6881645":{"id":6881645,"externalId":"511145.b0844","name":"ybjI","hasFamily":true,"annotation":"FMN and erythrose-4-P phosphatase; Catalyzes the dephosphorylation of flavin mononucleotide (FMN): Can also hydrolyze erythrose 4-phosphate (Ery4P)","uniprotId":"YBJI_ECOLI"},"6881646":{"id":6881646,"externalId":"511145.b0845","name":"ybjJ","hasFamily":true,"annotation":"putative transporter","uniprotId":"YBJJ_ECOLI"},"6881647":{"id":6881647,"externalId":"511145.b0846","name":"ybjK","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YBJK_ECOLI"},"6881648":{"id":6881648,"externalId":"511145.b0847","name":"ybjL","hasFamily":true,"annotation":"putative transporter","uniprotId":"YBJL_ECOLI"},"6881649":{"id":6881649,"externalId":"511145.b0848","name":"ybjM","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YBJM_ECOLI"},"6881650":{"id":6881650,"externalId":"511145.b0849","name":"grxA","hasFamily":true,"annotation":"glutaredoxin 1, redox coenzyme for ribonucleotide reductase (RNR1a); The disulfide bond functions as an electron carrier in the glutathione-dependent synthesis of deoxyribonucleotides by the enzyme ribonucleotide reductase. In addition, it is also involved in reducing some disulfide bonds in a coupled system with glutathione reductase","uniprotId":"GLRX1_ECOLI"},"6881651":{"id":6881651,"externalId":"511145.b0850","name":"ybjC","hasFamily":true,"annotation":"conserved protein, DUF1418 family","uniprotId":"YBJC_ECOLI"},"6881652":{"id":6881652,"externalId":"511145.b0851","name":"mdaA","hasFamily":true,"annotation":"nitroreductase A, NADPH-dependent, FMN-dependent; Reduction of nitroaromatic compounds using NADH. Reduces nitrofurazone by a ping-pong bi-bi mechanism possibly to generate a two-electron transfer product. Major component of the oxygen- insensitive nitroreductase activity in E.coli","uniprotId":"NFSA_ECOLI"},"6881653":{"id":6881653,"externalId":"511145.b0852","name":"rimK","hasFamily":true,"annotation":"ribosomal protein S6 modification protein; Responsible for the addition of glutamate residues to the C-terminus of ribosomal protein S6 (By similarity)","uniprotId":"RIMK_ECOLI"},"6881654":{"id":6881654,"externalId":"511145.b0853","name":"ybjN","hasFamily":true,"annotation":"predicted oxidoreductase","uniprotId":"YBJN_ECOLI"},"6881655":{"id":6881655,"externalId":"511145.b0854","name":"potF","hasFamily":true,"annotation":"putrescine transporter subunit: periplasmic-binding component of ABC superfamily; Required for the activity of the bacterial periplasmic transport system of putrescine. Polyamine binding protein","uniprotId":"POTF_ECOLI"},"6881656":{"id":6881656,"externalId":"511145.b0855","name":"potG","hasFamily":true,"annotation":"putrescine transporter subunit: ATP-binding component of ABC superfamily; Part of the ABC transporter complex PotABCD involved in spermidine/putrescine import. Responsible for energy coupling to the transport system (By similarity)","uniprotId":"POTG_ECOLI"},"6881657":{"id":6881657,"externalId":"511145.b0856","name":"potH","hasFamily":true,"annotation":"putrescine transporter subunit: membrane component of ABC superfamily; Required for the activity of the bacterial periplasmic transport system of putrescine","uniprotId":"POTH_ECOLI"},"6881658":{"id":6881658,"externalId":"511145.b0857","name":"potI","hasFamily":true,"annotation":"putrescine transporter subunit: membrane component of ABC superfamily; Required for the activity of the bacterial periplasmic transport system of putrescine","uniprotId":"POTI_ECOLI"},"6881659":{"id":6881659,"externalId":"511145.b0858","name":"ybjO","hasFamily":true,"annotation":"inner membrane protein, DUF2593 family","uniprotId":"YBJO_ECOLI"},"6881660":{"id":6881660,"externalId":"511145.b0859","name":"rlmC","hasFamily":true,"annotation":"23S rRNA m(5)U747 methyltransferase; Catalyzes the formation of 5-methyl-uridine at position 747 (m5U747) in 23S rRNA (By similarity)","uniprotId":"RLMC_ECOLI"},"6881661":{"id":6881661,"externalId":"511145.b0860","name":"artJ","hasFamily":true,"annotation":"arginine binding protein, periplasmic; Part of the ABC transporter complex ArtPIQMJ involved in arginine transport. Binds L-arginine with high affinity","uniprotId":"ARTJ_ECOLI"},"6881662":{"id":6881662,"externalId":"511145.b0861","name":"artM","hasFamily":true,"annotation":"arginine transporter subunit; Part of the ABC transporter complex ArtPIQMJ involved in arginine transport. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"ARTM_ECOLI"},"6881663":{"id":6881663,"externalId":"511145.b0862","name":"artQ","hasFamily":true,"annotation":"arginine transporter subunit; Part of the ABC transporter complex ArtPIQMJ involved in arginine transport. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"ARTQ_ECOLI"},"6881664":{"id":6881664,"externalId":"511145.b0863","name":"artI","hasFamily":true,"annotation":"arginine transporter subunit; Part of the ABC transporter complex ArtPIQMJ involved in arginine transport","uniprotId":"ARTI_ECOLI"},"6881665":{"id":6881665,"externalId":"511145.b0864","name":"artP","hasFamily":true,"annotation":"arginine transporter subunit; Part of the ABC transporter complex ArtPIQMJ involved in arginine transport. Probably responsible for energy coupling to the transport system","uniprotId":"ARTP_ECOLI"},"6881666":{"id":6881666,"externalId":"511145.b0865","name":"ybjP","hasFamily":true,"annotation":"lipoprotein","uniprotId":"YBJP_ECOLI"},"6881667":{"id":6881667,"externalId":"511145.b0866","name":"ybjQ","hasFamily":true,"annotation":"conserved protein, UPF0145 family","uniprotId":"YBJQ_ECOLI"},"6881668":{"id":6881668,"externalId":"511145.b0867","name":"amiD","hasFamily":true,"annotation":"1,6-anhydro-N-acetylmuramyl-L-alanine amidase, Zn-dependent; OM lipoprotein","uniprotId":"AMID_ECOLI"},"6881669":{"id":6881669,"externalId":"511145.b0868","name":"ybjS","hasFamily":true,"annotation":"putative NAD(P)H-binding oxidoreductase with NAD(P)-binding Rossmann-fold domain","uniprotId":"YBJS_ECOLI"},"6881670":{"id":6881670,"externalId":"511145.b0869","name":"ybjT","hasFamily":true,"annotation":"conserved protein with NAD(P)-binding Rossmann-fold domain","uniprotId":"YBJT_ECOLI"},"6881671":{"id":6881671,"externalId":"511145.b0870","name":"ltaE","hasFamily":true,"annotation":"L-allo-threonine aldolase, PLP-dependent; Catalyzes the cleavage of L-allo-threonine and L- threonine to glycine and acetaldehyde. L-threo-phenylserine and L- erythro-phenylserine are also good substrates","uniprotId":"LTAE_ECOLI"},"6881672":{"id":6881672,"externalId":"511145.b0871","name":"poxB","hasFamily":true,"annotation":"pyruvate dehydrogenase (pyruvate oxidase), thiamin-dependent, FAD-binding"},"6881673":{"id":6881673,"externalId":"511145.b0872","name":"hcr","hasFamily":true,"annotation":"HCP oxidoreductase, NADH-dependent; NADH oxidoreductase acting in concert with HCP","uniprotId":"HCR_ECOLI"},"6881674":{"id":6881674,"externalId":"511145.b0873","name":"hcp","hasFamily":true,"annotation":"hybrid-cluster [4Fe-2S-2O] protein in anaerobic terminal reductases; Catalyzes the reduction of hydroxylamine to form NH(3) and H(2)O. Is also able to reduce hydroxylamine analogs such as methylhydroxylamine and hydroxyquinone. Might have a role as a scavenger of potentially toxic by-products of nitrate metabolism","uniprotId":"HCP_ECOLI"},"6881675":{"id":6881675,"externalId":"511145.b0874","name":"ybjE","hasFamily":true,"annotation":"putative transporter","uniprotId":"YBJE_ECOLI"},"6881676":{"id":6881676,"externalId":"511145.b0875","name":"aqpZ","hasFamily":true,"annotation":"aquaporin Z; Channel that permits osmotically driven movement of water in both directions. It is involved in the osmoregulation and in the maintenance of cell turgor during volume expansion in rapidly growing cells. It mediates rapid entry or exit of water in response to abrupt changes in osmolarity (By similarity)","uniprotId":"AQPZ_ECOLI"},"6881677":{"id":6881677,"externalId":"511145.b0876","name":"ybjD","hasFamily":true,"annotation":"conserved protein with nucleoside triphosphate hydrolase domain","uniprotId":"YBJD_ECOLI"},"6881678":{"id":6881678,"externalId":"511145.b0877","name":"ybjX","hasFamily":true,"annotation":"conserved protein","uniprotId":"YBJX_ECOLI"},"6881679":{"id":6881679,"externalId":"511145.b0878","name":"macA","hasFamily":true,"annotation":"macrolide transporter subunit, membrane fusion protein (MFP) component; Efflux transporter for macrolide antibiotics","uniprotId":"MACA_ECOLI"},"6881680":{"id":6881680,"externalId":"511145.b0879","name":"macB","hasFamily":true,"annotation":"fused macrolide transporter subunits of ABC superfamily: ATP-binding component/membrane component; Part of the ABC transporter complex MacAB involved in macrolide export. Transmembrane domains (TMD) form a pore in the inner membrane and the ATP-binding domain (NBD) is responsible for energy generation. Seems to confer resistance against macrolides composed of 14- and 15-membered lactones but no or weak resistance against 16-membered ones (Probable)","uniprotId":"MACB_ECOLI"},"6881681":{"id":6881681,"externalId":"511145.b0880","name":"cspD","hasFamily":true,"annotation":"inhibitor of DNA replication, cold shock protein homolog; Inhibits DNA replication at both initiation and elongation steps, most probably by binding to the opened, single- stranded regions at replication forks. Plays a regulatory role in chromosomal replication in nutrient-depleted cells","uniprotId":"CSPD_ECOLI"},"6881682":{"id":6881682,"externalId":"511145.b0881","name":"clpS","hasFamily":true,"annotation":"regulatory protein for ClpA substrate specificity; Involved in the modulation of the specificity of the ClpAP-mediated ATP-dependent protein degradation (By similarity)","uniprotId":"CLPS_ECOLI"},"6881683":{"id":6881683,"externalId":"511145.b0882","name":"clpA","hasFamily":true,"annotation":"ATPase and specificity subunit of ClpA-ClpP ATP-dependent serine protease, chaperone activity; ATP-dependent specificity component of the ClpAP protease. It directs the protease to specific substrates. It has unfoldase activity. The primary function of the ClpA-ClpP complex appears to be the degradation of unfolded or abnormal proteins","uniprotId":"CLPA_ECOLI"},"6881684":{"id":6881684,"externalId":"511145.b0884","name":"infA","hasFamily":false,"annotation":"translation initiation factor IF-1; No specific function has so far been attributed to this initiation factor; however, it seems to stimulate more or less all the activities of the other two initiation factors, IF-2 and IF-3 (By similarity)","uniprotId":"IF1_ECOLI"},"6881685":{"id":6881685,"externalId":"511145.b0885","name":"aat","hasFamily":true,"annotation":"leucyl/phenylalanyl-tRNA-protein transferase; Functions in the N-end rule pathway of protein degradation where it conjugates Leu, Phe and, less efficiently, Met from aminoacyl-tRNAs to the N-termini of proteins containing an N-terminal arginine or lysine (By similarity)","uniprotId":"LFTR_ECOLI"},"6881686":{"id":6881686,"externalId":"511145.b0886","name":"cydC","hasFamily":true,"annotation":"fused glutathione, cysteine exporter subunits of ABC superfamily: membrane component/ATP-binding component; Somehow involved in the cytochrome D branch of aerobic respiration. Seems to be a component of a transport system","uniprotId":"CYDC_ECOLI"},"6881687":{"id":6881687,"externalId":"511145.b0887","name":"cydD","hasFamily":true,"annotation":"fused glutathione, cysteine exporter subunits of ABC superfamily: membrane component/ATP-binding component; Somehow involved in the cytochrome D branch of aerobic respiration. Seems to be a component of a transport system","uniprotId":"CYDD_ECOLI"},"6881688":{"id":6881688,"externalId":"511145.b0888","name":"trxB","hasFamily":true,"annotation":"thioredoxin reductase, FAD/NAD(P)-binding","uniprotId":"TRXB_ECOLI"},"6881689":{"id":6881689,"externalId":"511145.b0889","name":"lrp","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator, leucine-binding; Mediates a global response to leucine. Exogenous leucine affects the expression of a number of different operons; lrp mediates this effect for at least some of these operons. For example it is regulator of the branched-chain amino acid transport genes","uniprotId":"LRP_ECOLI"},"6881690":{"id":6881690,"externalId":"511145.b0890","name":"ftsK","hasFamily":true,"annotation":"DNA translocase at septal ring sorting daughter chromsomes; Essential cell division protein that coordinates cell division and chromosome segregation. The N-terminus is involved in assembly of the cell-division machinery. The C-terminus functions as a DNA motor that moves dsDNA in an ATP-dependent manner towards the dif recombination site, which is located within the replication terminus region. Translocation stops specifically at Xer-dif sites, where FtsK interacts with the Xer recombinase, allowing activation of chromosome unlinking by recombination. FtsK orienting polar sequences (K [...] ","uniprotId":"FTSK_ECOLI"},"6881691":{"id":6881691,"externalId":"511145.b0891","name":"lolA","hasFamily":true,"annotation":"lipoprotein chaperone; Participates in the translocation of lipoproteins from the inner membrane to the outer membrane. Only forms a complex with a lipoprotein if the residue after the N-terminal Cys is not an aspartate (The Asp acts as a targeting signal to indicate that the lipoprotein should stay in the inner membrane); the inner membrane retention signal functions at the release step","uniprotId":"LOLA_ECOLI"},"6881692":{"id":6881692,"externalId":"511145.b0892","name":"rarA","hasFamily":true,"annotation":"DNA-dependent ATPase involved in processing recombination intermediates at replication forks","uniprotId":"RARA_ECOLI"},"6881693":{"id":6881693,"externalId":"511145.b0893","name":"serS","hasFamily":true,"annotation":"seryl-tRNA synthetase, also charges selenocysteinyl-tRNA with serine; Catalyzes the attachment of serine to tRNA(Ser). Is also able to aminoacylate tRNA(Sec) with serine, to form the misacylated tRNA L-seryl-tRNA(Sec), which will be further converted into selenocysteinyl-tRNA(Sec) (By similarity)","uniprotId":"SYS_ECOLI"},"6881694":{"id":6881694,"externalId":"511145.b0894","name":"dmsA","hasFamily":true,"annotation":"dimethyl sulfoxide reductase, anaerobic, subunit A; Catalyzes the reduction of dimethyl sulfoxide (DMSO) to dimethyl sulfide (DMS). DMSO reductase serves as the terminal reductase under anaerobic conditions, with DMSO being the terminal electron acceptor. Terminal reductase during anaerobic growth on various sulfoxides and N-oxide compounds. Allows E.coli to grow anaerobically on DMSO as respiratory oxidant","uniprotId":"DMSA_ECOLI"},"6881695":{"id":6881695,"externalId":"511145.b0895","name":"dmsB","hasFamily":true,"annotation":"dimethyl sulfoxide reductase, anaerobic, subunit B; Electron transfer subunit of the terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds","uniprotId":"DMSB_ECOLI"},"6881696":{"id":6881696,"externalId":"511145.b0896","name":"dmsC","hasFamily":true,"annotation":"dimethyl sulfoxide reductase, anaerobic, subunit C; Terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds. DmsC anchors the DmsAB dimer to the membrane and stabilizes it","uniprotId":"DMSC_ECOLI"},"6881697":{"id":6881697,"externalId":"511145.b0897","name":"ycaC","hasFamily":true,"annotation":"putative hydrolase, isochorismatase family","uniprotId":"YCAC_ECOLI"},"6881698":{"id":6881698,"externalId":"511145.b0898","name":"ycaD","hasFamily":true,"annotation":"putative MFS-type transporter","uniprotId":"YCAD_ECOLI"},"6881699":{"id":6881699,"externalId":"511145.b0899","name":"ycaM","hasFamily":true,"annotation":"putative transporter","uniprotId":"YCAM_ECOLI"},"6881700":{"id":6881700,"externalId":"511145.b0900","name":"ycaN","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YCAN_ECOLI"},"6881701":{"id":6881701,"externalId":"511145.b0901","name":"ycaK","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCAK_ECOLI"},"6881702":{"id":6881702,"externalId":"511145.b0902","name":"pflA","hasFamily":true,"annotation":"pyruvate formate lyase activating enzyme 1; Activation of pyruvate formate-lyase 1 under anaerobic conditions by generation of an organic free radical, using S- adenosylmethionine and reduced flavodoxin as cosubstrates to produce 5'-deoxy-adenosine","uniprotId":"PFLA_ECOLI"},"6881703":{"id":6881703,"externalId":"511145.b0903","name":"pflB","hasFamily":true,"annotation":"pyruvate formate lyase I","uniprotId":"PFLB_ECOLI"},"6881704":{"id":6881704,"externalId":"511145.b0904","name":"focA","hasFamily":true,"annotation":"formate channel; Involved in the bidirectional transport of formate","uniprotId":"FOCA_ECOLI"},"6881705":{"id":6881705,"externalId":"511145.b0905","name":"ycaO","hasFamily":true,"annotation":"ribosomal protein S12 methylthiotransferase accessory factor; Involved in beta-methylthiolation of ribosomal protein S12","uniprotId":"YCAO_ECOLI"},"6881706":{"id":6881706,"externalId":"511145.b0906","name":"ycaP","hasFamily":true,"annotation":"putative inner membrane protein, UPF0702 family","uniprotId":"YCAP_ECOLI"},"6881707":{"id":6881707,"externalId":"511145.b0907","name":"serC","hasFamily":true,"annotation":"3-phosphoserine/phosphohydroxythreonine aminotransferase; Catalyzes the reversible conversion of 3- phosphohydroxypyruvate to phosphoserine and of 3-hydroxy-2-oxo-4- phosphonooxybutanoate to phosphohydroxythreonine. Is involved in both pyridoxine and serine biosynthesis","uniprotId":"SERC_ECOLI"},"6881708":{"id":6881708,"externalId":"511145.b0908","name":"aroA","hasFamily":true,"annotation":"5-enolpyruvylshikimate-3-phosphate synthetase","uniprotId":"AROA_ECOLI"},"6881709":{"id":6881709,"externalId":"511145.b0909","name":"ycaL","hasFamily":true,"annotation":"putative peptidase with chaperone function","uniprotId":"YCAL_ECOLI"},"6881710":{"id":6881710,"externalId":"511145.b0910","name":"cmk","hasFamily":true,"annotation":"cytidylate kinase; ATP, dATP, and GTP are equally effective as phosphate donors. CMP and dCMP are the best phosphate acceptors","uniprotId":"KCY_ECOLI"},"6881711":{"id":6881711,"externalId":"511145.b0911","name":"rpsA","hasFamily":true,"annotation":"30S ribosomal subunit protein S1; Binds mRNA; thus facilitating recognition of the initiation point. It is needed to translate mRNA with a short Shine-Dalgarno (SD) purine-rich sequence (By similarity)","uniprotId":"RS1_ECOLI"},"6881712":{"id":6881712,"externalId":"511145.b0912","name":"ihfB","hasFamily":true,"annotation":"integration host factor (IHF), DNA-binding protein, beta subunit; This protein is one of the two subunits of integration host factor, a specific DNA-binding protein that functions in genetic recombination as well as in transcriptional and translational control (By similarity)","uniprotId":"IHFB_ECOLI"},"6881713":{"id":6881713,"externalId":"511145.b0913","name":"ycaI","hasFamily":true,"annotation":"inner membrane protein, ComEC family of competence proteins","uniprotId":"YCAI_ECOLI"},"6881714":{"id":6881714,"externalId":"511145.b0914","name":"msbA","hasFamily":true,"annotation":"fused lipid transporter subunits of ABC superfamily: membrane component/ATP-binding component; Involved in lipid A export and possibly also in glycerophospholipid export and for biogenesis of the outer membrane. Transmembrane domains (TMD) form a pore in the inner membrane and the ATP-binding domain (NBD) is responsible for energy generation (By similarity)","uniprotId":"MSBA_ECOLI"},"6881715":{"id":6881715,"externalId":"511145.b0915","name":"lpxK","hasFamily":true,"annotation":"lipid A 4'kinase; Transfers the gamma-phosphate of ATP to the 4'-position of a tetraacyldisaccharide 1-phosphate intermediate (termed DS-1- P) to form tetraacyldisaccharide 1,4'-bis-phosphate (lipid IVA) (By similarity)","uniprotId":"LPXK_ECOLI"},"6881716":{"id":6881716,"externalId":"511145.b0916","name":"ycaQ","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCAQ_ECOLI"},"6881717":{"id":6881717,"externalId":"511145.b0917","name":"ycaR","hasFamily":true,"annotation":"conserved protein, UPF0434 family","uniprotId":"YCAR_ECOLI"},"6881718":{"id":6881718,"externalId":"511145.b0918","name":"kdsB","hasFamily":true,"annotation":"3-deoxy-manno-octulosonate cytidylyltransferase; Activates KDO (a required 8-carbon sugar) for incorporation into bacterial lipopolysaccharide in Gram-negative bacteria (By similarity)","uniprotId":"KDSB_ECOLI"},"6881719":{"id":6881719,"externalId":"511145.b0919","name":"ycbJ","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCBJ_ECOLI"},"6881720":{"id":6881720,"externalId":"511145.b0920","name":"ycbC","hasFamily":true,"annotation":"conserved protein, DUF218 superfamily","uniprotId":"YCBC_ECOLI"},"6881721":{"id":6881721,"externalId":"511145.b0921","name":"smtA","hasFamily":true,"annotation":"putative S-adenosyl-L-methionine-dependent methyltransferase","uniprotId":"SMTA_ECOLI"},"6881722":{"id":6881722,"externalId":"511145.b0922","name":"mukF","hasFamily":true,"annotation":"chromosome condensin MukBEF, kleisin-like subunit, binds calcium; Involved in chromosome condensation, segregation and cell cycle progression. May participate in facilitating chromosome segregation by condensation DNA from both sides of a centrally located replisome during cell division. Not required for mini-F plasmid partitioning. Probably acts via its interaction with MukB and MukE. Overexpression results in anucleate cells. It has a calcium binding activity (By similarity)","uniprotId":"MUKF_ECOLI"},"6881723":{"id":6881723,"externalId":"511145.b0923","name":"mukE","hasFamily":true,"annotation":"chromosome condensin MukBEF, MukE subunit; Involved in chromosome condensation, segregation and cell cycle progression. May participate in facilitating chromosome segregation by condensation DNA from both sides of a centrally located replisome during cell division. Probably acts via its interaction with MukB and MukF (By similarity)","uniprotId":"MUKE_ECOLI"},"6881724":{"id":6881724,"externalId":"511145.b0924","name":"mukB","hasFamily":true,"annotation":"chromosome condensin MukBEF, ATPase and DNA-binding subunit; Plays a central role in chromosome condensation, segregation and cell cycle progression. Functions as a homodimer, which is essential for chromosome partition. Involved in negative DNA supercoiling in vivo, and by this means organizes and compacts chromosomes. May achieve or facilitate chromosome segregation by condensation of DNA from both sides of a centrally located replisome during cell division. Stimulates both DNA relaxation and to a lesser extent decatenation activity of topoisomerase IV","uniprotId":"MUKB_ECOLI"},"6881725":{"id":6881725,"externalId":"511145.b0925","name":"ycbB","hasFamily":true,"annotation":"murein L,D-transpeptidase; Responsible, at least in part, for generating a meso- diaminopimelyl-3-a meso-diaminopimelyl-3 cross-link","uniprotId":"YCBB_ECOLI"},"6881726":{"id":6881726,"externalId":"511145.b0926","name":"ycbK","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCBK_ECOLI"},"6881727":{"id":6881727,"externalId":"511145.b0927","name":"ycbL","hasFamily":true,"annotation":"putative metal-binding enzyme","uniprotId":"YCBL_ECOLI"},"6881728":{"id":6881728,"externalId":"511145.b0928","name":"aspC","hasFamily":true,"annotation":"aspartate aminotransferase, PLP-dependent","uniprotId":"AAT_ECOLI"},"6881729":{"id":6881729,"externalId":"511145.b0929","name":"ompF","hasFamily":false,"annotation":"outer membrane porin 1a (Ia;b;F); Forms pores that allow passive diffusion of small molecules across the outer membrane. It is also a receptor for the bacteriophage T2","uniprotId":"OMPF_ECOLI"},"6881730":{"id":6881730,"externalId":"511145.b0930","name":"asnS","hasFamily":true,"annotation":"asparaginyl tRNA synthetase","uniprotId":"SYN_ECOLI"},"6881731":{"id":6881731,"externalId":"511145.b0931","name":"pncB","hasFamily":true,"annotation":"nicotinate phosphoribosyltransferase","uniprotId":"PNCB_ECOLI"},"6881732":{"id":6881732,"externalId":"511145.b0932","name":"pepN","hasFamily":true,"annotation":"aminopeptidase N; Aminopeptidase N is involved in the degradation of intracellular peptides generated by protein breakdown during normal growth as well as in response to nutrient starvation","uniprotId":"AMPN_ECOLI"},"6881733":{"id":6881733,"externalId":"511145.b0933","name":"ssuB","hasFamily":true,"annotation":"alkanesulfonate transporter subunit; Part of the ABC transporter complex SsuABC involved in aliphatic sulfonates import. Responsible for energy coupling to the transport system (Probable)","uniprotId":"SSUB_ECOLI"},"6881734":{"id":6881734,"externalId":"511145.b0934","name":"ssuC","hasFamily":true,"annotation":"putative alkanesulfonate transporter subunit; Part of a binding-protein-dependent transport system for aliphatic sulfonates. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"SSUC_ECOLI"},"6881735":{"id":6881735,"externalId":"511145.b0935","name":"ssuD","hasFamily":true,"annotation":"alkanesulfonate monooxygenase, FMNH(2)-dependent; Involved in desulfonation of aliphatic sulfonates. Catalyzes the conversion of pentanesulfonic acid to sulfite and pentaldehyde and is able to desulfonate a wide range of sulfonated substrates including C-2 to C-10 unsubstituted linear alkanesulfonates, substituted ethanesulfonic acids and sulfonated buffers","uniprotId":"SSUD_ECOLI"},"6881736":{"id":6881736,"externalId":"511145.b0936","name":"ssuA","hasFamily":true,"annotation":"putative aliphatic sulfonate binding protein; Part of a binding-protein-dependent transport system for aliphatic sulfonates. Putative binding protein","uniprotId":"SSUA_ECOLI"},"6881737":{"id":6881737,"externalId":"511145.b0937","name":"ssuE","hasFamily":true,"annotation":"NAD(P)H-dependent FMN reductase; Catalyzes an NADPH-dependent reduction of FMN, but is also able to reduce FAD or riboflavin","uniprotId":"SSUE_ECOLI"},"6881738":{"id":6881738,"externalId":"511145.b0938","name":"elfA","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein; Part of the elfADCG-ycbUVF fimbrial operon, which promotes adhesion of bacteria to different abiotic surfaces. ElfA is the major fimbrial subunit produced by this operon","uniprotId":"ELFA_ECOLI"},"6881739":{"id":6881739,"externalId":"511145.b0939","name":"elfD","hasFamily":false,"annotation":"putative periplasmic pilin chaperone; Part of the elfADCG-ycbUVF fimbrial operon, which promotes adhesion of bacteria to different abiotic surfaces. Could be required for the biogenesis of the ElfA fimbriae","uniprotId":"ELFD_ECOLI"},"6881740":{"id":6881740,"externalId":"511145.b0940","name":"elfC","hasFamily":true,"annotation":"putative outer membrane usher protein; Part of the elfADCG-ycbUVF fimbrial operon, which promotes adhesion of bacteria to different abiotic surfaces. Could be involved in the export and assembly of the ElfA fimbrial subunits across the outer membrane","uniprotId":"ELFC_ECOLI"},"6881741":{"id":6881741,"externalId":"511145.b0941","name":"elfG","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein; Part of the elfADCG-ycbUVF fimbrial operon, which promotes adhesion of bacteria to different abiotic surfaces","uniprotId":"ELFG_ECOLI"},"6881742":{"id":6881742,"externalId":"511145.b0942","name":"ycbU","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein; Part of the elfADCG-ycbUVF fimbrial operon, which promotes adhesion of bacteria to different abiotic surfaces","uniprotId":"YCBU_ECOLI"},"6881743":{"id":6881743,"externalId":"511145.b0943","name":"ycbV","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein; Part of the elfADCG-ycbUVF fimbrial operon, which promotes adhesion of bacteria to different abiotic surfaces","uniprotId":"YCBV_ECOLI"},"6881744":{"id":6881744,"externalId":"511145.b0944","name":"ycbF","hasFamily":true,"annotation":"putative periplasmic pilini chaperone; Part of the elfADCG-ycbUVF fimbrial operon, which promotes adhesion of bacteria to different abiotic surfaces. Could be required for the biogenesis of fimbriae","uniprotId":"YCBF_ECOLI"},"6881745":{"id":6881745,"externalId":"511145.b0945","name":"pyrD","hasFamily":true,"annotation":"dihydro-orotate oxidase, FMN-linked; Catalyzes the conversion of dihydroorotate to orotate with quinone as electron acceptor (By similarity)","uniprotId":"PYRD_ECOLI"},"6881746":{"id":6881746,"externalId":"511145.b0946","name":"ycbW","hasFamily":true,"annotation":"FtsZ stabilizer; Contributes to the efficiency of the cell division process by stabilizing the polymeric form of the cell division protein FtsZ. Acts by promoting interactions between FtsZ protofilaments and suppressing the GTPase activity of FtsZ (By similarity)","uniprotId":"ZAPC_ECOLI"},"6881747":{"id":6881747,"externalId":"511145.b0947","name":"ycbX","hasFamily":true,"annotation":"putative 2Fe-2S cluster-containing protein; 6-N-hydroxylaminopurine resistance protein","uniprotId":"YCBX_ECOLI"},"6881748":{"id":6881748,"externalId":"511145.b0948","name":"rlmL","hasFamily":true,"annotation":"23S rRNA mG2445 methyltransferase, SAM-dependent; Specifically methylates the guanine in position 2445 (m2G2445) and the guanine in position 2069 (m7G2069) of 23S rRNA. Methylation occurs before assembly of 23S rRNA into 50S subunits","uniprotId":"RLMKL_ECOLI"},"6881749":{"id":6881749,"externalId":"511145.b0949","name":"uup","hasFamily":true,"annotation":"fused predicted transporter subunits of ABC superfamily: ATP-binding components","uniprotId":"UUP_ECOLI"},"6881750":{"id":6881750,"externalId":"511145.b0950","name":"pqiA","hasFamily":true,"annotation":"paraquat-inducible membrane protein A","uniprotId":"PQIA_ECOLI"},"6881751":{"id":6881751,"externalId":"511145.b0951","name":"pqiB","hasFamily":true,"annotation":"paraquat-inducible protein B","uniprotId":"PQIB_ECOLI"},"6881752":{"id":6881752,"externalId":"511145.b0952","name":"ymbA","hasFamily":true,"annotation":"conserved protein","uniprotId":"YMBA_ECOLI"},"6881753":{"id":6881753,"externalId":"511145.b0953","name":"rmf","hasFamily":true,"annotation":"ribosome modulation factor; During stationary phase, converts 70S ribosomes to an inactive dimeric form (100S ribosomes). May form immature 90S particles, which are converted to mature 100S ribosomes by the hibernation promoting factor Hpf. Inactivates ribosomes by covering the peptidyl transferase (PTase) center of the 23S rRNA and the entrance of peptide exit tunnel. Also involved in protection against heat stress, but this role is not dependent on the maintenance of ribosome dimers","uniprotId":"RMF_ECOLI"},"6881754":{"id":6881754,"externalId":"511145.b0954","name":"fabA","hasFamily":true,"annotation":"beta-hydroxydecanoyl thioester dehydrase; Necessary for the introduction of cis unsaturation into fatty acids. Catalyzes the dehydration of (3R)-3-hydroxydecanoyl- ACP to E-(2)-decenoyl-ACP and then its isomerization to Z-(3)- decenoyl-ACP. Can catalyze the dehydratase reaction for beta- hydroxyacyl-ACPs with saturated chain lengths up to 16:0, being most active on intermediate chain length. Is inactive in the dehydration of long chain unsaturated beta-hydroxyacyl-ACP","uniprotId":"FABA_ECOLI"},"6881755":{"id":6881755,"externalId":"511145.b0955","name":"ycbZ","hasFamily":true,"annotation":"putative peptidase","uniprotId":"LONH_ECOLI"},"6881756":{"id":6881756,"externalId":"511145.b0956","name":"matP","hasFamily":true,"annotation":"Ter macrodomain organizer matS-binding protein; Required for spatial organization of the terminus region of the chromosome (Ter macrodomain) during the cell cycle. Prevents early segregation of duplicated Ter macrodomains during cell division. Binds specifically to matS, which is a 13 bp signature motif repeated within the Ter macrodomain (By similarity)","uniprotId":"MATP_ECOLI"},"6881757":{"id":6881757,"externalId":"511145.b0957","name":"ompA","hasFamily":true,"annotation":"outer membrane protein A (3a;II*;G;d); Required for the action of colicins K and L and for the stabilization of mating aggregates in conjugation. Serves as a receptor for a number of T-even like phages. Also acts as a porin with low permeability that allows slow penetration of small solutes","uniprotId":"OMPA_ECOLI"},"6881758":{"id":6881758,"externalId":"511145.b0958","name":"sulA","hasFamily":true,"annotation":"SOS cell division inhibitor; Component of the SOS system and an inhibitor of cell division. Accumulation of SulA causes rapid cessation of cell division and the appearance of long, non-septate filaments. In the presence of GTP, binds a polymerization-competent form of FtsZ in a 1:1 ratio, thus inhibiting FtsZ polymerization and therefore preventing it from participating in the assembly of the Z ring. This mechanism prevents the premature segregation of damaged DNA to daughter cells during cell division. The effect of overexpression of SulA is neutralized by CbeA","uniprotId":"SULA_ECOLI"},"6881759":{"id":6881759,"externalId":"511145.b0959","name":"sxy","hasFamily":true,"annotation":"required for expression of CRP-S-dependent promoters","uniprotId":"SXY_ECOLI"},"6881760":{"id":6881760,"externalId":"511145.b0960","name":"yccS","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YCCS_ECOLI"},"6881761":{"id":6881761,"externalId":"511145.b0961","name":"yccF","hasFamily":true,"annotation":"inner membrane protein, DUF307 family","uniprotId":"YCCF_ECOLI"},"6881762":{"id":6881762,"externalId":"511145.b0962","name":"helD","hasFamily":true,"annotation":"DNA helicase IV; Helicase IV catalyzes the unwinding of duplex DNA in the 3' to 5' direction with respect to the bound single strand in a reaction that is dependent upon the hydrolysis of ATP","uniprotId":"HELD_ECOLI"},"6881763":{"id":6881763,"externalId":"511145.b0963","name":"mgsA","hasFamily":true,"annotation":"methylglyoxal synthase","uniprotId":"MGSA_ECOLI"},"6881764":{"id":6881764,"externalId":"511145.b0964","name":"yccT","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCCT_ECOLI"},"6881765":{"id":6881765,"externalId":"511145.b0965","name":"yccU","hasFamily":true,"annotation":"putative CoA-binding protein","uniprotId":"YCCU_ECOLI"},"6881766":{"id":6881766,"externalId":"511145.b0966","name":"hspQ","hasFamily":true,"annotation":"heat shock protein involved in degradation of mutant DnaA; hemimethylated oriC DNA-binding protein; Involved in the degradation of certain denaturated proteins, including DnaA, during heat shock stress (By similarity)","uniprotId":"HSPQ_ECOLI"},"6881767":{"id":6881767,"externalId":"511145.b0967","name":"rlmI","hasFamily":true,"annotation":"23S rRNA m(5)C1962 methyltransferase, SAM-dependent; Specifically methylates the cytosine at position 1962 (m5C1962) of 23S rRNA. Methylation occurs before assembly of 23S rRNA into 50S subunits","uniprotId":"RLMI_ECOLI"},"6881768":{"id":6881768,"externalId":"511145.b0968","name":"yccX","hasFamily":true,"annotation":"weak acylphosphatase","uniprotId":"ACYP_ECOLI"},"6881769":{"id":6881769,"externalId":"511145.b0969","name":"tusE","hasFamily":true,"annotation":"sulfurtransferase required for 2-thiolation step of mnm(5)-s(2)U34-tRNA synthesis; Part of a sulfur-relay system required for 2-thiolation of 5-methylaminomethyl-2-thiouridine (mnm(5)s(2)U) at tRNA wobble positions. Could accept sulfur from TusD","uniprotId":"TUSE_ECOLI"},"6881770":{"id":6881770,"externalId":"511145.b0970","name":"yccA","hasFamily":true,"annotation":"HflBKC-binding inner membrane protein, UPF0005 family; Negatively modulates the activity of the FtsH protease for membrane substrates. Overexpression or stabilizing YccA counteracts the FtsH-mediated degradation of SecY when the SecYEG preprotein translocator is jammed","uniprotId":"YCCA_ECOLI"},"6881771":{"id":6881771,"externalId":"511145.b0972","name":"hyaA","hasFamily":true,"annotation":"hydrogenase 1, small subunit; This is one of three E.coli hydrogenases synthesized in response to different physiological conditions. HYD1 is believed to have a role in hydrogen cycling during fermentative growth","uniprotId":"MBHS_ECOLI"},"6881772":{"id":6881772,"externalId":"511145.b0973","name":"hyaB","hasFamily":true,"annotation":"hydrogenase 1, large subunit; This is one of three E.coli hydrogenases synthesized in response to different physiological conditions. HYD1 is believed to have a role in hydrogen cycling during fermentative growth","uniprotId":"MBHL_ECOLI"},"6881773":{"id":6881773,"externalId":"511145.b0974","name":"hyaC","hasFamily":true,"annotation":"hydrogenase 1, b-type cytochrome subunit; Probable b-type cytochrome","uniprotId":"CYBH_ECOLI"},"6881774":{"id":6881774,"externalId":"511145.b0975","name":"hyaD","hasFamily":true,"annotation":"hydrogenase 1 maturation protease; Protease involved in the C-terminal processing of HyaB, the large subunit of hydrogenase 1 (Potential)","uniprotId":"HYAD_ECOLI"},"6881775":{"id":6881775,"externalId":"511145.b0976","name":"hyaE","hasFamily":true,"annotation":"probable HyaA chaperone; Not known. Could form, along with HyaD, a complex involved in the processing of the hydrogenase 1 structural operon","uniprotId":"HYAE_ECOLI"},"6881776":{"id":6881776,"externalId":"511145.b0977","name":"hyaF","hasFamily":true,"annotation":"protein involved in nickel incorporation into hydrogenase-1 proteins; Not known. Could enhance the incorporation of nickel to the hydrogenase","uniprotId":"HYAF_ECOLI"},"6881777":{"id":6881777,"externalId":"511145.b0978","name":"appC","hasFamily":true,"annotation":"cytochrome bd-II oxidase, subunit I; Cytochrome oxidase subunit","uniprotId":"APPC_ECOLI"},"6881778":{"id":6881778,"externalId":"511145.b0979","name":"appB","hasFamily":true,"annotation":"cytochrome bd-II oxidase, subunit II; Cytochrome oxidase subunit","uniprotId":"APPB_ECOLI"},"6881779":{"id":6881779,"externalId":"511145.b0980","name":"appA","hasFamily":true,"annotation":"phosphoanhydride phosphorylase","uniprotId":"PPA_ECOLI"},"6881780":{"id":6881780,"externalId":"511145.b0981","name":"etk","hasFamily":true,"annotation":"cryptic autophosphorylating protein tyrosine kinase Etk","uniprotId":"ETK_ECOLI"},"6881781":{"id":6881781,"externalId":"511145.b0982","name":"etp","hasFamily":true,"annotation":"phosphotyrosine-protein phosphatase; Dephosphorylates etk","uniprotId":"ETP_ECOLI"},"6881782":{"id":6881782,"externalId":"511145.b0983","name":"gfcE","hasFamily":true,"annotation":"lipoprotein required for formation of the O-antigen capsule, predicted exopolysaccharide export protein; May be involved in polysaccharide transport","uniprotId":"GFCE_ECOLI"},"6881783":{"id":6881783,"externalId":"511145.b0984","name":"gfcD","hasFamily":false,"annotation":"conserved protein","uniprotId":"GFCD_ECOLI"},"6881784":{"id":6881784,"externalId":"511145.b0985","name":"gfcC","hasFamily":true,"annotation":"conserved protein","uniprotId":"GFCC_ECOLI"},"6881785":{"id":6881785,"externalId":"511145.b0986","name":"gfcB","hasFamily":true,"annotation":"putative outer membrane lipoprotein","uniprotId":"GFCB_ECOLI"},"6881786":{"id":6881786,"externalId":"511145.b0987","name":"gfcA","hasFamily":false,"annotation":"predicted protein","uniprotId":"GFCA_ECOLI"},"6881787":{"id":6881787,"externalId":"511145.b0988","name":"insB4","hasFamily":true,"annotation":"IS1 transposase B; Absolutely required for transposition of IS1","uniprotId":"INSB4_ECOLI"},"6881788":{"id":6881788,"externalId":"511145.b0989","name":"cspH","hasFamily":true,"annotation":"stress protein, member of the CspA-family","uniprotId":"CSPH_ECOLI"},"6881789":{"id":6881789,"externalId":"511145.b0990","name":"cspG","hasFamily":true,"annotation":"cold shock protein homolog, cold-inducible","uniprotId":"CSPG_ECOLI"},"6881790":{"id":6881790,"externalId":"511145.b0991","name":"sfa","hasFamily":false,"annotation":"cold shock gene","uniprotId":"YMCE_ECOLI"},"6881791":{"id":6881791,"externalId":"511145.b0992","name":"yccM","hasFamily":true,"annotation":"putative 4Fe-4S membrane protein","uniprotId":"YCCM_ECOLI"},"6881792":{"id":6881792,"externalId":"511145.b0993","name":"torS","hasFamily":true,"annotation":"hybrid sensory histidine kinase in two-component regulatory system with TorR; Member of the two-component regulatory system TorS/TorR involved in the anaerobic utilization of trimethylamine-N-oxide (TMAO). Detects the presence of TMAO in the medium and, in response, activates TorR via a four-step phosphorelay. When TMAO is removed, TorS can dephosphorylate TorR, probably by a reverse phosphorelay involving His-860 and Asp-733","uniprotId":"TORS_ECOLI"},"6881793":{"id":6881793,"externalId":"511145.b0994","name":"torT","hasFamily":true,"annotation":"periplasmic sensory protein associated with the TorRS two-component regulatory system; Upon binding a putative inducer it probably interacts with TorS and allows it to play a role in the induction of the torCAD operon for trimethylamine N-oxide reductase","uniprotId":"TORT_ECOLI"},"6881794":{"id":6881794,"externalId":"511145.b0995","name":"torR","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with TorS; Member of the two-component regulatory system TorS/TorR involved in the anaerobic utilization of trimethylamine-N-oxide (TMAO). Phosphorylated TorR activates the transcription of the torCAD operon by binding to four decameric boxes located in the torCAD promoter. Box1, 2 and 4 contain the DNA sequence 5'- CTGTTCATAT-3' and box3 contains the DNA sequence 5'-CCGTTCATCC-3'. Phosphorylated as well as unphosphorylated TorR negatively regulates its own expression by binding to box1 and 2","uniprotId":"TORR_ECOLI"},"6881795":{"id":6881795,"externalId":"511145.b0996","name":"torC","hasFamily":true,"annotation":"trimethylamine N-oxide (TMAO) reductase I, cytochrome c-type subunit; Part of the anaerobic respiratory chain of trimethylamine-N-oxide reductase TorA. Acts by transferring electrons from the membranous menaquinones to TorA. This transfer probably involves an electron transfer pathway from menaquinones to the N-terminal domain of TorC, then from the N-terminus to the C-terminus, and finally to TorA. TorC apocytochrome negatively autoregulates the torCAD operon probably by inhibiting the TorS kinase activity","uniprotId":"TORC_ECOLI"},"6881796":{"id":6881796,"externalId":"511145.b0997","name":"torA","hasFamily":true,"annotation":"trimethylamine N-oxide (TMAO) reductase I, catalytic subunit; Reduces trimethylamine-N-oxide (TMAO) into trimethylamine; an anaerobic reaction coupled to energy-yielding reactions","uniprotId":"TORA_ECOLI"},"6881797":{"id":6881797,"externalId":"511145.b0998","name":"torD","hasFamily":true,"annotation":"TorA-maturation chaperone; Involved in the biogenesis of TorA. Acts on TorA before the insertion of the molybdenum cofactor and, as a result, probably favors a conformation of the apoenzyme that is competent for acquiring the cofactor (By similarity)","uniprotId":"TORD_ECOLI"},"6881798":{"id":6881798,"externalId":"511145.b0999","name":"cbpM","hasFamily":true,"annotation":"modulator of CbpA co-chaperone; Interacts with CbpA and inhibits both the DnaJ-like co- chaperone activity and the DNA binding activity of CbpA. Together with CbpA, modulates the activity of the DnaK chaperone system. Does not inhibit the co-chaperone activity of DnaJ (By similarity)","uniprotId":"CBPM_ECOLI"},"6881799":{"id":6881799,"externalId":"511145.b1000","name":"cbpA","hasFamily":true,"annotation":"curved DNA-binding protein, DnaJ homologue that functions as a co-chaperone of DnaK; DNA-binding protein that preferentially recognizes a curved DNA sequence. It is probably a functional analog of DnaJ; displays overlapping activities with DnaJ, but functions under different conditions, probably acting as a molecular chaperone in an adaptive response to environmental stresses other than heat shock. Lacks autonomous chaperone activity; binds native substrates and targets them for recognition by DnaK. Its activity is inhibited by the binding of CbpM (By similarity)","uniprotId":"CBPA_ECOLI"},"6881800":{"id":6881800,"externalId":"511145.b1001","name":"yccE","hasFamily":false,"annotation":"predicted protein","uniprotId":"YCCE_ECOLI"},"6881801":{"id":6881801,"externalId":"511145.b1002","name":"agp","hasFamily":true,"annotation":"glucose-1-phosphatase/inositol phosphatase; Absolutely required for the growth of E.coli in a high- phosphate medium containing G-1-P as the sole carbon source","uniprotId":"AGP_ECOLI"},"6881802":{"id":6881802,"externalId":"511145.b1003","name":"yccJ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YCCJ_ECOLI"},"6881803":{"id":6881803,"externalId":"511145.b1004","name":"wrbA","hasFamily":true,"annotation":"NAD(P)H:quinone oxidoreductase; It seems to function in response to environmental stress when various electron transfer chains are affected or when the environment is highly oxidizing. It reduces quinones to the hydroquinone state to prevent interaction of the semiquinone with O2 and production of superoxide. It prefers NADH over NADPH","uniprotId":"K0JFS6_ECOLI"},"6881804":{"id":6881804,"externalId":"511145.b1006","name":"rutG","hasFamily":true,"annotation":"pyrimidine permease; May function as a proton-driven pyrimidine uptake system","uniprotId":"RUTG_ECOLI"},"6881805":{"id":6881805,"externalId":"511145.b1007","name":"rutF","hasFamily":true,"annotation":"flavin:NADH reductase; Catalyzes the reduction of FMN to FMNH2 which is used to reduce pyrimidine by RutA via the Rut pathway. In vitro, the flavin reductase Fre can substitute for the function of RutF, however, RutF is required for uracil utilization in vivo","uniprotId":"RUTF_ECOLI"},"6881806":{"id":6881806,"externalId":"511145.b1008","name":"rutE","hasFamily":false,"annotation":"malonic semialdehyde reductase; May reduce toxic product malonic semialdehyde to 3- hydroxypropionic acid, which is excreted. RutE is apparently supplemented by YdfG. Required in vivo, but not in vitro in pyrimidine nitrogen degradation","uniprotId":"RUTE_ECOLI"},"6881807":{"id":6881807,"externalId":"511145.b1009","name":"rutD","hasFamily":true,"annotation":"putative aminoacrylate hydrolase; May increase the rate of spontaneous hydrolysis of aminoacrylate to malonic semialdehyde. Required to remove a toxic intermediate produce in vivo, but not in vitro in the pyrimidine nitrogen degradation","uniprotId":"RUTD_ECOLI"},"6881808":{"id":6881808,"externalId":"511145.b1010","name":"rutC","hasFamily":false,"annotation":"putative aminoacrylate peracid reductase; May reduce aminoacrylate peracid to aminoacrylate. Required to remove a toxic intermediate produce in vivo, but not in vitro in the pyrimidine nitrogen degradation","uniprotId":"RUTC_ECOLI"},"6881809":{"id":6881809,"externalId":"511145.b1011","name":"rutB","hasFamily":true,"annotation":"ureidoacrylate amidohydrolase; In vivo, quickly hydrolyzes the ureidoacrylate peracid to avoid toxicity, but can also hydrolyzes ureidoacrylate that is formed spontaneously from ureidoacrylate peracid. One of the products of hydrolysis, carbamate, hydrolyzes spontaneously, thereby releasing one of the pyrimidine rings nitrogen atoms as ammonia and one of its carbons as CO2 (By similarity)","uniprotId":"RUTB_ECOLI"},"6881810":{"id":6881810,"externalId":"511145.b1012","name":"rutA","hasFamily":true,"annotation":"pyrimidine oxygenase, FMN-dependent; Catalyzes the pyrimidine ring opening between N-3 and C- 4 by an unusual flavin hydroperoxide-catalyzed mechanism to yield ureidoacrylate peracid. It cleaves pyrmidine rings directly by adding oxygen atoms, making a toxic ureidoacrylate peracid product which can be spontaneously reduced to ureidoacrylate. Requires the flavin reductase RutF to regenerate FMN in vivo. RutF can be substituted by Fre in vitro","uniprotId":"RUTA_ECOLI"},"6881811":{"id":6881811,"externalId":"511145.b1013","name":"rutR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor for rut operon; Master transcription regulator which represses the degradation of pyrimidines (rutABCDEFG) and purines (gcl operon) for maintenance of metabolic balance between pyrimidines and purines. It also regulates the synthesis of pyrimidine nucleotides and arginine from glutamine (carAB) and the supply of glutamate (gadABWX)","uniprotId":"RUTR_ECOLI"},"6881812":{"id":6881812,"externalId":"511145.b1014","name":"putA","hasFamily":true,"annotation":"fused DNA-binding transcriptional regulator/proline dehydrogenase/pyrroline-5-carboxylate dehydrogenase; Oxidizes proline to glutamate for use as a carbon and nitrogen source and also function as a transcriptional repressor of the put operon","uniprotId":"PUTA_ECOLI"},"6881813":{"id":6881813,"externalId":"511145.b1015","name":"putP","hasFamily":true,"annotation":"proline:sodium symporter; Catalyzes the sodium-dependent uptake of extracellular L-proline. This protein is also capable of using lithium as the transport cation. Also catalyzes the uptake of propionate","uniprotId":"PUTP_ECOLI"},"6881814":{"id":6881814,"externalId":"511145.b1018","name":"efeO","hasFamily":true,"annotation":"inactive ferrous ion transporter EfeUOB; Involved in Fe(2+) uptake. Could be an iron-binding and/or electron-transfer component","uniprotId":"EFEO_ECOLI"},"6881815":{"id":6881815,"externalId":"511145.b1019","name":"efeB","hasFamily":true,"annotation":"deferrrochelatase, periplasmic; Involved in the recovery of exogenous heme iron. Extracts iron from heme while preserving the tetrapyrrol ring intact. Also displays peroxidase activity on guaiacol in vitro","uniprotId":"EFEB_ECOLI"},"6881816":{"id":6881816,"externalId":"511145.b1020","name":"phoH","hasFamily":true,"annotation":"conserved protein with nucleoside triphosphate hydrolase domain","uniprotId":"PHOH_ECOLI"},"6881817":{"id":6881817,"externalId":"511145.b1021","name":"pgaD","hasFamily":false,"annotation":"required for biofilm adhesin polysaccharide PGA synthesis; Required for the synthesis of poly-beta-1,6-N-acetyl-D- glucosamine (PGA), a biofilm adhesin polysaccharide. May assist the glycosyltransferase PgaC in the polymerization of PGA","uniprotId":"PGAD_ECOLI"},"6881818":{"id":6881818,"externalId":"511145.b1022","name":"pgaC","hasFamily":true,"annotation":"predicted glycosyl transferase; Probable N-acetylglucosaminyltransferase that catalyzes the polymerization of single monomer units of UDP-N- acetylglucosamine to produce the linear homopolymer poly-beta-1,6- N-acetyl-D-glucosamine (PGA), a biofilm adhesin polysaccharide","uniprotId":"PGAC_ECOLI"},"6881819":{"id":6881819,"externalId":"511145.b1023","name":"pgaB","hasFamily":true,"annotation":"biofilm adhesin polysaccharide PGA export lipoprotein with a polysaccharide deacetylase activity needed for export; Catalyzes the N-deacetylation of poly-beta-1,6-N-acetyl- D-glucosamine (PGA), a biofilm adhesin polysaccharide. N- deacetylation promotes PGA export through the PgaA porin","uniprotId":"PGAB_ECOLI"},"6881820":{"id":6881820,"externalId":"511145.b1024","name":"pgaA","hasFamily":true,"annotation":"biofilm adhesin polysaccharide PGA secretin; OM porin; Exports the biofilm adhesin polysaccharide poly-beta- 1,6-N-acetyl-D-glucosamine (PGA) across the outer membrane. The PGA transported seems to be partially N-deacetylated since N- deacetylation of PGA by PgaB is needed for PGA export through the PgaA porin","uniprotId":"PGAA_ECOLI"},"6881821":{"id":6881821,"externalId":"511145.b1025","name":"ycdT","hasFamily":true,"annotation":"diguanylate cyclase; A probable diguanylate cyclase, overexpression leads to a strong repression of swimming; swimming returns to normal when residues 359-360 are both mutated to Ala. Overexpression also leads to a 20-fold increase in c-di-GMP levels in vivo. Cyclic-di- GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"YCDT_ECOLI"},"6881822":{"id":6881822,"externalId":"511145.b1026","name":"b1026","hasFamily":true,"annotation":"IS3 transposase B"},"6881823":{"id":6881823,"externalId":"511145.b1027","name":"b1027","hasFamily":true,"annotation":"IS3 transposase A"},"6881824":{"id":6881824,"externalId":"511145.b1029","name":"ycdU","hasFamily":false,"annotation":"putative inner membrane protein","uniprotId":"YCDU_ECOLI"},"6881825":{"id":6881825,"externalId":"511145.b1033","name":"ghrA","hasFamily":true,"annotation":"glyoxylate/hydroxypyruvate reductase A; Catalyzes the NADPH-dependent reduction of glyoxylate and hydroxypyruvate into glycolate and glycerate, respectively. Inactive towards 2-oxo-D-gluconate, 2-oxoglutarate, oxaloacetate and pyruvate. Only D- and L-glycerate are involved in the oxidative activity with NADP. Activity with NAD is very low","uniprotId":"GHRA_ECOLI"},"6881826":{"id":6881826,"externalId":"511145.b1034","name":"ycdX","hasFamily":true,"annotation":"predicted zinc-binding hydrolase; Hydrolyzes p-nitrophenyl phosphate (pNPP) in vitro. Involved in the swarming motility process","uniprotId":"YCDX_ECOLI"},"6881827":{"id":6881827,"externalId":"511145.b1035","name":"ycdY","hasFamily":true,"annotation":"conserved protein, REMP family; Acts as a chaperone that increases YcdX activity, maybe by facilitating the correct insertion of the zinc ions into the catalytic site of YcdX. Involved in the swarming motility process","uniprotId":"YCDY_ECOLI"},"6881828":{"id":6881828,"externalId":"511145.b1036","name":"ycdZ","hasFamily":true,"annotation":"inner membrane protein, DUF1097 family","uniprotId":"YCDZ_ECOLI"},"6881829":{"id":6881829,"externalId":"511145.b1037","name":"csgG","hasFamily":true,"annotation":"curli production assembly/transport outer membrane lipoprotein; May be involved in the biogenesis of curli organelles","uniprotId":"CSGG_ECOLI"},"6881830":{"id":6881830,"externalId":"511145.b1038","name":"csgF","hasFamily":true,"annotation":"outer membrane protein required for curli nucleation by CsgB; May be involved in the biogenesis of curli organelles","uniprotId":"CSGF_ECOLI"},"6881831":{"id":6881831,"externalId":"511145.b1039","name":"csgE","hasFamily":true,"annotation":"curlin secretion specificity factor; May be involved in the biogenesis of curli organelles","uniprotId":"CSGE_ECOLI"},"6881832":{"id":6881832,"externalId":"511145.b1040","name":"csgD","hasFamily":true,"annotation":"DNA-binding transcriptional activator for csgBA; The master regulator for adhesive curli fimbriae expression; necessary for transcription of the csgBAC/ymdA operon. Plays a positive role in biofilm formation. May have the capability to respond to starvation and/or high cell density by activating csgBA transcription. Low-level constitutive expression confers an adherent curli fimbriae-expressing phenotype, up- regulates 10 genes and down-regulates 14 others","uniprotId":"CSGD_ECOLI"},"6881833":{"id":6881833,"externalId":"511145.b1041","name":"csgB","hasFamily":true,"annotation":"curlin nucleator protein, minor subunit in curli complex; Curlin is the structural subunit of the curli fimbriae. Curli are coiled surface structures that assemble preferentially at growth temperatures below 37 degrees Celsius. Curli can bind to fibronectin. The minor subunit is the nucleation component of curlin monomers","uniprotId":"CSGB_ECOLI"},"6881834":{"id":6881834,"externalId":"511145.b1042","name":"csgA","hasFamily":true,"annotation":"curlin subunit, amyloid curli fibers, cryptic; Curlin is the structural subunit of the curli fimbriae. Curli are coiled surface structures that assemble preferentially at growth temperatures below 37 degrees Celsius. Curli can bind to fibronectin","uniprotId":"CSGA_ECOLI"},"6881835":{"id":6881835,"externalId":"511145.b1043","name":"csgC","hasFamily":true,"annotation":"curli assembly protein; Plays a role in the extracellular assembly of CsgA into thin aggregative fimbriae (Tafi) fibers. Assembly may also require CsgE. Tafi are thought to be assembled via an extracellular nucleation-precipitation (ENP) pathway, and possibly also via an intracellular non-CsgC-dependent pathway (By similarity)","uniprotId":"CSGC_ECOLI"},"6881836":{"id":6881836,"externalId":"511145.b1044","name":"ymdA","hasFamily":true,"annotation":"predicted protein","uniprotId":"YMDA_ECOLI"},"6881837":{"id":6881837,"externalId":"511145.b1045","name":"ymdB","hasFamily":true,"annotation":"O-acetyl-ADP-ribose deacetylase; RNase III inhibitor during cold shock; Deacetylates O-acetyl-ADP ribose. Down-regulates ribonuclease 3 (RNase III) activity. Acts by interacting directly with the region of the ribonuclease that is required for dimerization/activation (By similarity)","uniprotId":"YMDB_ECOLI"},"6881838":{"id":6881838,"externalId":"511145.b1046","name":"ymdC","hasFamily":true,"annotation":"predicted hydrolase","uniprotId":"YMDC_ECOLI"},"6881839":{"id":6881839,"externalId":"511145.b1047","name":"mdoC","hasFamily":true,"annotation":"membrane protein required for succinylation of osmoregulated periplasmic glucans (OPGs); Necessary for the succinyl substitution of periplasmic glucans. Could catalyze the transfer of succinyl residues from the cytoplasmic side of the membrane to the nascent glucan backbones on the periplasmic side of the membrane (By similarity)","uniprotId":"OPGC_ECOLI"},"6881840":{"id":6881840,"externalId":"511145.b1048","name":"mdoG","hasFamily":true,"annotation":"osmoregulated periplasmic glucan (OPG) biosynthesis periplasmic protein; Involved in the biosynthesis of osmoregulated periplasmic glucans (OPGs) (By similarity)","uniprotId":"OPGG_ECOLI"},"6881841":{"id":6881841,"externalId":"511145.b1049","name":"mdoH","hasFamily":true,"annotation":"membrane glycosyltransferase; Involved in the biosynthesis of osmoregulated periplasmic glucans (OPGs) (By similarity)","uniprotId":"OPGH_ECOLI"},"6881842":{"id":6881842,"externalId":"511145.b1050","name":"yceK","hasFamily":true,"annotation":"lipoprotein, DUF1375 family","uniprotId":"YCEK_ECOLI"},"6881843":{"id":6881843,"externalId":"511145.b1051","name":"msyB","hasFamily":true,"annotation":"multicopy suppressor of secY and secA; Could participate in the normal pathway of protein export","uniprotId":"MSYB_ECOLI"},"6881844":{"id":6881844,"externalId":"511145.b1053","name":"mdtG","hasFamily":true,"annotation":"putative drug efflux system; Confers resistance to fosfomycin and deoxycholate (By similarity)","uniprotId":"MDTG_ECOLI"},"6881845":{"id":6881845,"externalId":"511145.b1054","name":"htrB","hasFamily":true,"annotation":"lauryl-acyl carrier protein (ACP)-dependent acyltransferase; Acylates the intermediate (KDO)2-lipid IVA to form (KDO)2-(lauroyl)-lipid IVA. Has 10 fold selectivity for lauroyl- ACP over myristoyl-ACP","uniprotId":"HTRB_ECOLI"},"6881846":{"id":6881846,"externalId":"511145.b1055","name":"yceA","hasFamily":true,"annotation":"putative rhodanese-related sulfurtransferase","uniprotId":"YCEA_ECOLI"},"6881847":{"id":6881847,"externalId":"511145.b1056","name":"yceI","hasFamily":true,"annotation":"secreted protein","uniprotId":"YCEI_ECOLI"},"6881848":{"id":6881848,"externalId":"511145.b1057","name":"yceJ","hasFamily":true,"annotation":"putative cytochrome b561","uniprotId":"C56I_ECOLI"},"6881849":{"id":6881849,"externalId":"511145.b1058","name":"yceO","hasFamily":false,"annotation":"predicted protein","uniprotId":"YCEO_ECOLI"},"6881850":{"id":6881850,"externalId":"511145.b1059","name":"solA","hasFamily":true,"annotation":"N-methyltryptophan oxidase, FAD-binding; Catalyzes the oxidative demethylation of N-methyl-L- tryptophan. Can also use other N-methyl amino acids, including sarcosine, which, however, is a poor substrate","uniprotId":"MTOX_ECOLI"},"6881851":{"id":6881851,"externalId":"511145.b1060","name":"bssS","hasFamily":true,"annotation":"biofilm regulator; Represses biofilm formation in M9C glu and LB glu media but not in M9C and LB media. Seems to act as a global regulator of several genes involved in catabolite repression and stress response and regulation of the uptake and export of signaling pathways. Could be involved the regulation of indole as well as uptake and export of AI-2 through a cAMP-dependent pathway","uniprotId":"BSSS_ECOLI"},"6881852":{"id":6881852,"externalId":"511145.b1061","name":"dinI","hasFamily":true,"annotation":"DNA damage-inducible protein I; Involved in SOS regulation. Inhibits RecA by preventing RecA to bind ssDNA. Can displace ssDNA from RecA","uniprotId":"DINI_ECOLI"},"6881853":{"id":6881853,"externalId":"511145.b1062","name":"pyrC","hasFamily":true,"annotation":"dihydro-orotase","uniprotId":"PYRC_ECOLI"},"6881854":{"id":6881854,"externalId":"511145.b1063","name":"yceB","hasFamily":true,"annotation":"lipoprotein, DUF1439 family","uniprotId":"YCEB_ECOLI"},"6881855":{"id":6881855,"externalId":"511145.b1064","name":"grxB","hasFamily":true,"annotation":"glutaredoxin 2 (Grx2); Involved in reducing some disulfide bonds in a coupled system with glutathione reductase. Does not act as hydrogen donor for ribonucleotide reductase","uniprotId":"GLRX2_ECOLI"},"6881856":{"id":6881856,"externalId":"511145.b1065","name":"mdtH","hasFamily":true,"annotation":"multidrug resistance efflux transporter conferring overexpression resistance to norfloxacin and enoxacin; Confers resistance to norfloxacin and enoxacin (By similarity)","uniprotId":"MDTH_ECOLI"},"6881857":{"id":6881857,"externalId":"511145.b1066","name":"rimJ","hasFamily":true,"annotation":"ribosomal-protein-S5-alanine N-acetyltransferase; This enzyme acetylates the N-terminal alanine of ribosomal protein S5. Plays also a role in the temperature regulation of pap pilin transcription","uniprotId":"RIMJ_ECOLI"},"6881858":{"id":6881858,"externalId":"511145.b1067","name":"yceH","hasFamily":true,"annotation":"conserved protein, UPF0502 family","uniprotId":"YCEH_ECOLI"},"6881859":{"id":6881859,"externalId":"511145.b1068","name":"mviM","hasFamily":true,"annotation":"putative oxidoreductase with NAD(P)-binding Rossmann-fold domain","uniprotId":"YCEM_ECOLI"},"6881860":{"id":6881860,"externalId":"511145.b1069","name":"murJ","hasFamily":true,"annotation":"probable peptidoglycan lipid II flippase required for murein synthesis; Involved in peptidoglycan biosynthesis","uniprotId":"MURJ_ECOLI"},"6881861":{"id":6881861,"externalId":"511145.b1070","name":"flgN","hasFamily":true,"annotation":"export chaperone for FlgK and FlgL; Required for the efficient initiation of filament assembly (By similarity)","uniprotId":"FLGN_ECOLI"},"6881862":{"id":6881862,"externalId":"511145.b1071","name":"flgM","hasFamily":true,"annotation":"anti-sigma factor for FliA (sigma 28); Responsible for the coupling of flagellin expression to flagellar assembly by preventing expression of the flagellin genes when a component of the middle class of proteins is defective. It negatively regulates flagellar genes by inhibiting the activity of FliA by directly binding to FliA","uniprotId":"FLGM_ECOLI"},"6881863":{"id":6881863,"externalId":"511145.b1072","name":"flgA","hasFamily":true,"annotation":"assembly protein for flagellar basal-body periplasmic P ring; Involved in the assembly process of the P-ring formation. It may associate with FlgF on the rod constituting a structure essential for the P-ring assembly or may act as a modulator protein for the P-ring assembly","uniprotId":"FLGA_ECOLI"},"6881864":{"id":6881864,"externalId":"511145.b1073","name":"flgB","hasFamily":true,"annotation":"flagellar component of cell-proximal portion of basal-body rod; Structural component of flagellum, the bacterial motility apparatus. Part of the rod structure of flagellar basal body (By similarity)","uniprotId":"FLGB_ECOLI"},"6881865":{"id":6881865,"externalId":"511145.b1074","name":"flgC","hasFamily":true,"annotation":"flagellar component of cell-proximal portion of basal-body rod","uniprotId":"FLGC_ECOLI"},"6881866":{"id":6881866,"externalId":"511145.b1075","name":"flgD","hasFamily":true,"annotation":"flagellar hook assembly protein; Required for flagellar hook formation. May act as a scaffolding protein","uniprotId":"FLGD_ECOLI"},"6881867":{"id":6881867,"externalId":"511145.b1076","name":"flgE","hasFamily":true,"annotation":"flagellar hook protein","uniprotId":"FLGE_ECOLI"},"6881868":{"id":6881868,"externalId":"511145.b1077","name":"flgF","hasFamily":true,"annotation":"flagellar component of cell-proximal portion of basal-body rod","uniprotId":"FLGF_ECOLI"},"6881869":{"id":6881869,"externalId":"511145.b1078","name":"flgG","hasFamily":true,"annotation":"flagellar component of cell-distal portion of basal-body rod","uniprotId":"FLGG_ECOLI"},"6881870":{"id":6881870,"externalId":"511145.b1079","name":"flgH","hasFamily":true,"annotation":"flagellar protein of basal-body outer-membrane L ring; Assembles around the rod to form the L-ring and probably protects the motor/basal body from shearing forces during rotation (By similarity)","uniprotId":"FLGH_ECOLI"},"6881871":{"id":6881871,"externalId":"511145.b1080","name":"flgI","hasFamily":true,"annotation":"putative flagellar basal body protein; Assembles around the rod to form the L-ring and probably protects the motor/basal body from shearing forces during rotation (By similarity)","uniprotId":"FLGI_ECOLI"},"6881872":{"id":6881872,"externalId":"511145.b1081","name":"flgJ","hasFamily":true,"annotation":"muramidase; Flagellum-specific muramidase which hydrolyzes the peptidoglycan layer to assemble the rod structure in the periplasmic space (By similarity)","uniprotId":"FLGJ_ECOLI"},"6881873":{"id":6881873,"externalId":"511145.b1082","name":"flgK","hasFamily":true,"annotation":"flagellar hook-filament junction protein 1","uniprotId":"FLGK_ECOLI"},"6881874":{"id":6881874,"externalId":"511145.b1083","name":"flgL","hasFamily":true,"annotation":"flagellar hook-filament junction protein","uniprotId":"FLGL_ECOLI"},"6881875":{"id":6881875,"externalId":"511145.b1084","name":"rne","hasFamily":true,"annotation":"fused ribonucleaseE: endoribonuclease/RNA-binding protein/RNA degradosome binding protein; Matures 5S rRNA from its precursors from all the rRNA genes. It also cleaves RNA I, a molecule that controls the replication of colE1 plasmid DNA. It is the major endoribonuclease participating in mRNA turnover in E.coli. It initiates the decay of RNAs by cutting them internally near their 5'-end. It is able to remove poly(A) tails by an endonucleolytic process","uniprotId":"RNE_ECOLI"},"6881876":{"id":6881876,"externalId":"511145.b1085","name":"yceQ","hasFamily":false,"annotation":"predicted protein","uniprotId":"YCEQ_ECOLI"},"6881877":{"id":6881877,"externalId":"511145.b1086","name":"rluC","hasFamily":true,"annotation":"23S rRNA pseudouridine(955,2504,2580) synthase (EC:4.2.1.70); Responsible for synthesis of pseudouridine from uracil at positions 955, 2504 and 2580 in 23S ribosomal RNA","uniprotId":"RLUC_ECOLI"},"6881878":{"id":6881878,"externalId":"511145.b1087","name":"b1087","hasFamily":true,"annotation":"Maf-like protein","uniprotId":"YCEF_ECOLI"},"6881879":{"id":6881879,"externalId":"511145.b1088","name":"yceD","hasFamily":true,"annotation":"conserved protein, DUF177 family","uniprotId":"YCED_ECOLI"},"6881880":{"id":6881880,"externalId":"511145.b1089","name":"rpmF","hasFamily":false,"annotation":"50S ribosomal subunit protein L32","uniprotId":"RL32_ECOLI"},"6881881":{"id":6881881,"externalId":"511145.b1090","name":"plsX","hasFamily":true,"annotation":"probable phosphate acyltransferase; Catalyzes the reversible formation of acyl-phosphate (acyl-PO(4)) from acyl-[acyl-carrier-protein] (acyl-ACP). This enzyme utilizes acyl-ACP as fatty acyl donor, but not acyl-CoA (By similarity)","uniprotId":"PLSX_ECOLI"},"6881882":{"id":6881882,"externalId":"511145.b1091","name":"fabH","hasFamily":true,"annotation":"3-oxoacyl-[acyl-carrier-protein] synthase III; Catalyzes the condensation reaction of fatty acid synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP. Catalyzes the first condensation reaction which initiates fatty acid synthesis and may therefore play a role in governing the total rate of fatty acid production. Possesses both acetoacetyl-ACP synthase and acetyl transacylase activities. Has some substrate specificity for acetyl-CoA. Its substrate specificity determines the biosynthesis of straight-chain of fatty acids instead of branched-chain","uniprotId":"FABH_ECOLI"},"6881883":{"id":6881883,"externalId":"511145.b1092","name":"fabD","hasFamily":true,"annotation":"malonyl-CoA-[acyl-carrier-protein] transacylase","uniprotId":"FABD_ECOLI"},"6881884":{"id":6881884,"externalId":"511145.b1093","name":"fabG","hasFamily":true,"annotation":"3-oxoacyl-[acyl-carrier-protein] reductase; Catalyzes the NADPH-dependent reduction of beta- ketoacyl-ACP substrates to beta-hydroxyacyl-ACP products, the first reductive step in the elongation cycle of fatty acid biosynthesis","uniprotId":"FABG_ECOLI"},"6881885":{"id":6881885,"externalId":"511145.b1094","name":"acpP","hasFamily":false,"annotation":"acyl carrier protein (ACP); Carrier of the growing fatty acid chain in fatty acid biosynthesis (By similarity)","uniprotId":"ACP_ECOLI"},"6881886":{"id":6881886,"externalId":"511145.b1095","name":"fabF","hasFamily":true,"annotation":"3-oxoacyl-[acyl-carrier-protein] synthase II; Catalyzes the condensation reaction of fatty acid synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP. Has a preference for short chain acid substrates and may function to supply the octanoic substrates for lipoic acid biosynthesis","uniprotId":"FABF_ECOLI"},"6881887":{"id":6881887,"externalId":"511145.b1096","name":"pabC","hasFamily":true,"annotation":"4-amino-4-deoxychorismate lyase component of para-aminobenzoate synthase multienzyme complex; Converts 4-amino-4-deoxychorismate into 4-aminobenzoate (PABA) and pyruvate","uniprotId":"PABC_ECOLI"},"6881888":{"id":6881888,"externalId":"511145.b1097","name":"yceG","hasFamily":true,"annotation":"predicted aminodeoxychorismate lyase","uniprotId":"YCEG_ECOLI"},"6881889":{"id":6881889,"externalId":"511145.b1098","name":"tmk","hasFamily":true,"annotation":"thymidylate kinase; Catalyzes the reversible phosphorylation of deoxythymidine monophosphate (dTMP) to deoxythymidine diphosphate (dTDP), using ATP as its preferred phosphoryl donor. Situated at the junction of both de novo and salvage pathways of deoxythymidine triphosphate (dTTP) synthesis, is essential for DNA synthesis and cellular growth","uniprotId":"KTHY_ECOLI"},"6881890":{"id":6881890,"externalId":"511145.b1099","name":"holB","hasFamily":true,"annotation":"DNA polymerase III, delta prime subunit; DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. This DNA polymerase also exhibits 3' to 5' exonuclease activity","uniprotId":"HOLB_ECOLI"},"6881891":{"id":6881891,"externalId":"511145.b1100","name":"ycfH","hasFamily":true,"annotation":"putative DNAse; Putative deoxyribonuclease","uniprotId":"YCFH_ECOLI"},"6881892":{"id":6881892,"externalId":"511145.b1101","name":"ptsG","hasFamily":true,"annotation":"fused glucose-specific PTS enzymes: IIB component/IIC component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in glucose transport. This enzyme is also a chemoreceptor monitoring the environment for changes in sugar concentration","uniprotId":"PTGCB_ECOLI"},"6881893":{"id":6881893,"externalId":"511145.b1102","name":"fhuE","hasFamily":true,"annotation":"ferric-rhodotorulic acid outer membrane transporter; Required for the uptake of Fe(3+) via coprogen, ferrioxamine B, and rhodotorulic acid","uniprotId":"FHUE_ECOLI"},"6881894":{"id":6881894,"externalId":"511145.b1103","name":"hinT","hasFamily":true,"annotation":"purine nucleoside phosphoramidase, dadA activator protein; Hydrolyzes purine nucleotide phosphoramidates, including adenosine 5'monophosphoramidate (AMP-NH2), adenosine 5'monophosphomorpholidate (AMP-morpholidate), guanosine 5'monophosphomorpholidate (GMP-morpholidate) and tryptamine 5'guanosine monophosphate (TpGd). Hydrolyzes lysyl-AMP (AMP-N- epsilon-(N-alpha-acetyl lysine methyl ester)) generated by lysine tRNA ligase and lysyl-GMP (GMP-N-epsilon-(N-alpha-acetyl lysine methyl ester)). Required for growth at high salt concentrations. Required for normal DadA activity and the ability [...] ","uniprotId":"HINT_ECOLI"},"6881895":{"id":6881895,"externalId":"511145.b1104","name":"ycfL","hasFamily":true,"annotation":"predicted protein","uniprotId":"YCFL_ECOLI"},"6881896":{"id":6881896,"externalId":"511145.b1105","name":"lpoB","hasFamily":true,"annotation":"outer membrane lipoprotein; Regulator of peptidoglycan synthesis that is essential for the function of penicillin-binding protein 1B (PBP1b). Stimulates transpeptidase and transglycosylase activities of PBP1b in vitro. May also contribute to outer membrane constriction during cell division, in complex with PBP1b","uniprotId":"LPOB_ECOLI"},"6881897":{"id":6881897,"externalId":"511145.b1106","name":"thiK","hasFamily":true,"annotation":"thiamin kinase; Catalyzes the phosphorylation of thiamine to thiamine phosphate (By similarity)","uniprotId":"THIK_ECOLI"},"6881898":{"id":6881898,"externalId":"511145.b1107","name":"nagZ","hasFamily":true,"annotation":"beta N-acetyl-glucosaminidase; Cleaves GlcNAc linked beta-1,4 to MurNAc tripeptides (By similarity)","uniprotId":"NAGZ_ECOLI"},"6881899":{"id":6881899,"externalId":"511145.b1108","name":"ycfP","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCFP_ECOLI"},"6881900":{"id":6881900,"externalId":"511145.b1109","name":"ndh","hasFamily":true,"annotation":"respiratory NADH dehydrogenase 2/cupric reductase; Transfer of electrons from NADH to the respiratory chain. The immediate electron acceptor for the enzyme is believed to be ubiquinone. Does not couple the redox reaction to proton translocation","uniprotId":"DHNA_ECOLI"},"6881901":{"id":6881901,"externalId":"511145.b1110","name":"ycfJ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YCFJ_ECOLI"},"6881902":{"id":6881902,"externalId":"511145.b1111","name":"comR","hasFamily":true,"annotation":"repressor for bhsA(ycfR); Represses expression of BhsA/ComC by binding to its promoter region in the absence of copper","uniprotId":"COMR_ECOLI"},"6881903":{"id":6881903,"externalId":"511145.b1112","name":"bhsA","hasFamily":true,"annotation":"biofilm, cell surface and signaling protein; Reduces the permeability of the outer membrane to copper. Seems to be involved in the regulation of biofilm formation. May decrease biofilm formation by repressing cell-cell interaction and cell surface interaction","uniprotId":"BHSA_ECOLI"},"6881904":{"id":6881904,"externalId":"511145.b1113","name":"ycfS","hasFamily":true,"annotation":"L,D-transpeptidase linking Lpp to murein; Responsible, at least in part, for anchoring of the major outer membrane lipoprotein (Lpp, also known as the Braun lipoprotein) to the peptidoglycan via a meso-diaminopimelyl-L-Lys- bond on the terminal residue of Lpp","uniprotId":"YCFS_ECOLI"},"6881905":{"id":6881905,"externalId":"511145.b1114","name":"mfd","hasFamily":true,"annotation":"transcription-repair coupling factor; Necessary for strand-specific repair. A lesion in the template strand blocks the RNA polymerase complex (RNAP). The RNAP-DNA-RNA complex is specifically recognized by TRCF which releases RNAP and the truncated transcript; the TCRF may replace RNAP at the lesion site and then recruit the UvrA/B/C repair system","uniprotId":"MFD_ECOLI"},"6881906":{"id":6881906,"externalId":"511145.b1115","name":"ycfT","hasFamily":false,"annotation":"inner membrane protein","uniprotId":"YCFT_ECOLI"},"6881907":{"id":6881907,"externalId":"511145.b1116","name":"lolC","hasFamily":true,"annotation":"lipoprotein-releasing system transmembrane protein; Part of an ATP-dependent transport system LolCDE responsible for the release of lipoproteins targeted to the outer membrane from the inner membrane. Such a release is dependent of the sorting-signal (absence of an Asp at position 2 of the mature lipoprotein) and of LolA","uniprotId":"LOLC_ECOLI"},"6881908":{"id":6881908,"externalId":"511145.b1117","name":"lolD","hasFamily":true,"annotation":"outer membrane-specific lipoprotein transporter subunit; Part of the ABC transporter complex LolCDE involved in the translocation of mature outer membrane-directed lipoproteins, from the inner membrane to the periplasmic chaperone, LolA. Responsible for the formation of the LolA-lipoprotein complex in an ATP-dependent manner. Such a release is dependent of the sorting-signal (absence of an Asp at position 2 of the mature lipoprotein) and of LolA","uniprotId":"LOLD_ECOLI"},"6881909":{"id":6881909,"externalId":"511145.b1118","name":"lolE","hasFamily":true,"annotation":"lipoprotein-releasing system transmembrane protein; Part of an ATP-dependent transport system LolCDE responsible for the release of lipoproteins targeted to the outer membrane from the inner membrane. Such a release is dependent of the sorting-signal (absence of an Asp at position 2 of the mature lipoprotein) and of LolA","uniprotId":"LOLE_ECOLI"},"6881910":{"id":6881910,"externalId":"511145.b1119","name":"nagK","hasFamily":true,"annotation":"N-acetyl-D-glucosamine kinase; Catalyzes the phosphorylation of N-acetyl-D-glucosamine (GlcNAc) derived from cell-wall degradation, yielding GlcNAc-6-P. Has also low level glucokinase activity in vitro","uniprotId":"NAGK_ECOLI"},"6881911":{"id":6881911,"externalId":"511145.b1120","name":"cobB","hasFamily":true,"annotation":"deacetylase of acs and cheY, chemotaxis regulator; NAD-dependent protein deacetylase which modulates the activities of several enzymes which are inactive in their acetylated form. May also have NAD-dependent lysine demalonylase and desuccinylase activity (By similarity). Activates the enzyme acetyl-CoA synthetase by deacetylating 'Lys-609' in the inactive, acetylated form of the enzyme. May also modulate the activity of other propionyl-adenosine monophosphate (AMP)-forming enzymes (Probable)","uniprotId":"NPD_ECOLI"},"6881912":{"id":6881912,"externalId":"511145.b1121","name":"ycfZ","hasFamily":false,"annotation":"inner membrane protein","uniprotId":"YCFZ_ECOLI"},"6881913":{"id":6881913,"externalId":"511145.b1122","name":"ymfA","hasFamily":false,"annotation":"conserved inner membrane protein","uniprotId":"YMFA_ECOLI"},"6881914":{"id":6881914,"externalId":"511145.b1123","name":"potD","hasFamily":true,"annotation":"polyamine transporter subunit; Required for the activity of the bacterial periplasmic transport system of putrescine and spermidine. Polyamine binding protein","uniprotId":"POTD_ECOLI"},"6881915":{"id":6881915,"externalId":"511145.b1124","name":"potC","hasFamily":true,"annotation":"polyamine transporter subunit; Required for the activity of the bacterial periplasmic transport system of putrescine and spermidine","uniprotId":"POTC_ECOLI"},"6881916":{"id":6881916,"externalId":"511145.b1125","name":"potB","hasFamily":true,"annotation":"polyamine transporter subunit; Required for the activity of the bacterial periplasmic transport system of putrescine and spermidine","uniprotId":"POTB_ECOLI"},"6881917":{"id":6881917,"externalId":"511145.b1126","name":"potA","hasFamily":true,"annotation":"polyamine transporter subunit; Part of the ABC transporter complex PotABCD involved in spermidine/putrescine import. Responsible for energy coupling to the transport system (By similarity)","uniprotId":"POTA_ECOLI"},"6881918":{"id":6881918,"externalId":"511145.b1127","name":"pepT","hasFamily":true,"annotation":"peptidase T; Cleaves the N-terminal amino acid of tripeptides (By similarity)","uniprotId":"PEPT_ECOLI"},"6881919":{"id":6881919,"externalId":"511145.b1128","name":"ycfD","hasFamily":true,"annotation":"cupin superfamily protein; Growth-regulating oxygenase that catalyzes the hydroxylation of 50S ribosomal protein L16 on 'Arg-81'","uniprotId":"YCFD_ECOLI"},"6881920":{"id":6881920,"externalId":"511145.b1129","name":"phoQ","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with PhoP; Member of the two-component regulatory system PhoQ/PhoP involved in adaptation to low Mg(2+) environments and the control of acid resistance genes. In presence of low periplasmic Mg(2+) concentrations, PhoQ functions as a membrane-associated protein kinase that undergoes autophosphorylation and subsequently transfers the phosphate to PhoP, which results in the expression of PhoP-activated genes (PAG) and repression of PhoP-repressed genes (PRG). In presence of high periplasmic Mg(2+) concentrations, acts as a protei [...] ","uniprotId":"PHOQ_ECOLI"},"6881921":{"id":6881921,"externalId":"511145.b1130","name":"phoP","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with PhoQ; Member of the two-component regulatory system PhoQ/PhoP involved in adaptation to low Mg(2+) environments and the control of acid resistance genes. Mediates magnesium influx to the cytosol by activation of MgtA. Promotes expression of the two-component regulatory system rstA/rstB and transcription of the hemL, mgrB, nagA, slyB, vboR and yrbL genes","uniprotId":"PHOP_ECOLI"},"6881922":{"id":6881922,"externalId":"511145.b1131","name":"purB","hasFamily":true,"annotation":"adenylosuccinate lyase","uniprotId":"PUR8_ECOLI"},"6881923":{"id":6881923,"externalId":"511145.b1132","name":"hflD","hasFamily":true,"annotation":"putative lysogenization regulator; Negative regulator of phage lambda lysogenization. Contributes to the degradation of the phage regulatory protein CII. Acts probably by holding CII on the membrane surface, away from the target promoters, but close to the FtsH protease (By similarity)","uniprotId":"HFLD_ECOLI"},"6881924":{"id":6881924,"externalId":"511145.b1133","name":"mnmA","hasFamily":true,"annotation":"tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase; Catalyzes the 2-thiolation of uridine at the wobble position (U34) of tRNA(Lys), tRNA(Glu) and tRNA(Gln), leading to the formation of s(2)U34, the first step of tRNA-mnm(5)s(2)U34 synthesis. Sulfur is provided by IscS, via a sulfur-relay system. Binds ATP and its substrate tRNAs (By similarity)","uniprotId":"MNMA_ECOLI"},"6881925":{"id":6881925,"externalId":"511145.b1134","name":"nudJ","hasFamily":true,"annotation":"bifunctional thiamin pyrimidine pyrophosphate hydrolase/thiamin pyrophosphate hydrolase; Catalyzes the hydrolysis of 4-amino-2-methyl-5- hydroxymethylpyrimidine pyrophosphate (HMP-PP) to 4-amino-2- methyl-5-hydroxymethylpyrimidine phosphate (HMP-P), and hydrolysis of thiamine pyrophosphate (TPP) to thiamine monophosphate (TMP). Can hydrolyze other substrates such as MeO-HMP-PP, CF(3)-HMP-PP and MeO-TPP. Is also a non-specific nucleoside tri- and diphosphatase that releases inorganic orthophosphate","uniprotId":"NUDJ_ECOLI"},"6881926":{"id":6881926,"externalId":"511145.b1135","name":"rluE","hasFamily":true,"annotation":"23S rRNA U2457 pseudouridine synthase; Responsible for synthesis of pseudouridine from uracil- 2457 in 23S ribosomal RNA","uniprotId":"RLUE_ECOLI"},"6881927":{"id":6881927,"externalId":"511145.b1136","name":"icd","hasFamily":true,"annotation":"e14 prophage; isocitrate dehydrogenase, specific for NADP+","uniprotId":"IDH_ECOLI"},"6881928":{"id":6881928,"externalId":"511145.b1137","name":"ymfD","hasFamily":false,"annotation":"e14 prophage","uniprotId":"YMFD_ECOLI"},"6881929":{"id":6881929,"externalId":"511145.b1138","name":"ymfE","hasFamily":false,"annotation":"e14 prophage","uniprotId":"YMFE_ECOLI"},"6881930":{"id":6881930,"externalId":"511145.b1139","name":"lit","hasFamily":false,"annotation":"e14 prophage; Interacts with a short DNA sequence about one-quarter of the way into the major capsid protein gene 23 of T4, and the inhibition occurs when this late gene of the virus is expressed","uniprotId":"LIT_ECOLI"},"6881931":{"id":6881931,"externalId":"511145.b1140","name":"intE","hasFamily":false,"annotation":"e14 prophage; Integrase from the cryptic lambdoic prophage e14. Integrase is necessary for integration of the phage into the host genome by site-specific recombination. In conjunction with excisionase, integrase is also necessary for excision of the prophage from the host genome","uniprotId":"INTE_ECOLI"},"6881932":{"id":6881932,"externalId":"511145.b1141","name":"xisE","hasFamily":false,"annotation":"e14 prophage","uniprotId":"VXIS_ECOLI"},"6881933":{"id":6881933,"externalId":"511145.b1143","name":"ymfI","hasFamily":false,"annotation":"e14 prophage","uniprotId":"YMFI_ECOLI"},"6881934":{"id":6881934,"externalId":"511145.b1144","name":"ymfJ","hasFamily":false,"annotation":"e14 prophage","uniprotId":"YMFJ_ECOLI"},"6881935":{"id":6881935,"externalId":"511145.b1145","name":"cohE","hasFamily":true,"annotation":"e14 prophage","uniprotId":"COHE_ECOLI"},"6881936":{"id":6881936,"externalId":"511145.b1146","name":"croE","hasFamily":false,"annotation":"e14 prophage","uniprotId":"CROE_ECOLI"},"6881937":{"id":6881937,"externalId":"511145.b1147","name":"ymfL","hasFamily":false,"annotation":"e14 prophage","uniprotId":"YMFL_ECOLI"},"6881938":{"id":6881938,"externalId":"511145.b1148","name":"ymfM","hasFamily":false,"annotation":"e14 prophage","uniprotId":"YMFM_ECOLI"},"6881939":{"id":6881939,"externalId":"511145.b1150","name":"ymfR","hasFamily":false,"annotation":"e14 prophage","uniprotId":"YMFR_ECOLI"},"6881940":{"id":6881940,"externalId":"511145.b1153","name":"ymfQ","hasFamily":true,"annotation":"prohage e14 tail protein homolog","uniprotId":"YMFQ_ECOLI"},"6881941":{"id":6881941,"externalId":"511145.b1154","name":"stfP","hasFamily":true,"annotation":"e14 prophage","uniprotId":"STFP_ECOLI"},"6881942":{"id":6881942,"externalId":"511145.b1155","name":"ymfS","hasFamily":false,"annotation":"e14 prophage","uniprotId":"YMFS_ECOLI"},"6881943":{"id":6881943,"externalId":"511145.b1156","name":"tfaE","hasFamily":false,"annotation":"e14 prophage","uniprotId":"TFAE_ECOLI"},"6881944":{"id":6881944,"externalId":"511145.b1158","name":"pin","hasFamily":true,"annotation":"e14 prophage; This protein catalyzes the inversion of an 1800-bp E.coli DNA fragment, the P region, which can exist in either orientation. The function of the inversion is not yet clear","uniprotId":"PINE_ECOLI"},"6881945":{"id":6881945,"externalId":"511145.b1159","name":"mcrA","hasFamily":false,"annotation":"e14 prophage; 5-methylcytosine-specific restriction endonuclease B; Restriction of 5-methyl and 5-hydroxymethylcytosines at the specific DNA sequence C(me)CGG","uniprotId":"MCRA_ECOLI"},"6881946":{"id":6881946,"externalId":"511145.b1160","name":"iraM","hasFamily":true,"annotation":"RpoS stabilzer during Mg starvation, anti-RssB factor; Inhibits RpoS proteolysis by regulating RssB activity, thereby increasing the stability of the sigma stress factor RpoS during magnesium starvation. May also be involved in the early steps of isoprenoid biosynthesis, possibly through its role as RssB regulator","uniprotId":"IRAM_ECOLI"},"6881947":{"id":6881947,"externalId":"511145.b1161","name":"ycgX","hasFamily":true,"annotation":"predicted protein","uniprotId":"YCGX_ECOLI"},"6881948":{"id":6881948,"externalId":"511145.b1162","name":"ycgE","hasFamily":true,"annotation":"repressor of blue light-responsive genes; Controls the expression of 6 small proteins that may play a role in biofilm maturation. Binds to and represses the operator of the ycgZ-ymgA-ariR-ymgC operon and also regulates yliL and ynaK. Binding is antagonized by YcgF upon blue light (470 nm) irradiation. Blue light may increase the affinity of YcgF for YcgE, allowing it to be released from its operator","uniprotId":"YCGE_ECOLI"},"6881949":{"id":6881949,"externalId":"511145.b1163","name":"ycgF","hasFamily":true,"annotation":"anti-repressor for YcgE, blue light-responsive; FAD-binding; Binds to and releases the YcgE repressor from its bound DNA target in a blue light-dependent (470 nm) fashion. A shift to low temperature also triggers a YcgF-mediated relief of repression by YcgE, suggesting YcgF may serve as a thermometer. Blue light may act to increase the affinity of YcgF for YcgE, allowing it to be released from its operator. The protein has a reversible photocycle, and undergoes structural changes, probably in the EAL domain, in response to light","uniprotId":"YCGF_ECOLI"},"6881950":{"id":6881950,"externalId":"511145.b1164","name":"ycgZ","hasFamily":false,"annotation":"RcsB connector protein for regulation of biofilm and acid-resistance; Probably a connector protein for RcsB/C regulation of biofilm formation, providing additional signal input into the two- component signaling pathway. Partially antagonizes the activities of YmgA and AriR, proteins that, via the Rcs phosphorelay, promote the synthesis of colanic acid, an exopolysaccharide and matrix component","uniprotId":"YCGZ_ECOLI"},"6881951":{"id":6881951,"externalId":"511145.b1165","name":"ymgA","hasFamily":false,"annotation":"RcsB connector protein for regulation of biofilm; Probably a connector protein for RcsB/C regulation of biofilm formation, providing additional signal input into the two- component signaling pathway. May serve to stimulate biofilm maturation, probably via the Rcs phosphorelay. Mild overexpression at 16 degrees Celsius increases the production of colanic acid, an exopolysaccharide and matrix component, and reduces adhesive curli fimbriae expression. Both of these effects require RcsB","uniprotId":"YMGA_ECOLI"},"6881952":{"id":6881952,"externalId":"511145.b1166","name":"ariR","hasFamily":false,"annotation":"RcsB connector protein for regulation of biofilm and acid-resistance; Probably a connector protein for RcsB/C regulation of biofilm and acid-resistance, providing additional signal input into the two-component signaling pathway. May serve to stimulate biofilm maturation, via the Rcs phosphorelay. Regulates expression of genes involved in acid-resistance and biofilm formation, including the RcsB/C two-component system. May be a non-specific DNA-binding protein that binds genes and/or intergenic regions via a geometric recognition. Also confers resistance to H(2)O(2). Overexpression at 2 [...] ","uniprotId":"ARIR_ECOLI"},"6881953":{"id":6881953,"externalId":"511145.b1167","name":"ymgC","hasFamily":false,"annotation":"predicted protein","uniprotId":"YMGC_ECOLI"},"6881954":{"id":6881954,"externalId":"511145.b1168","name":"ycgG","hasFamily":true,"annotation":"predicted cyclic-di-GMP phosphodiesterase","uniprotId":"YCGG_ECOLI"},"6881955":{"id":6881955,"externalId":"511145.b1171","name":"ymgD","hasFamily":false,"annotation":"predicted protein","uniprotId":"YMGD_ECOLI"},"6881956":{"id":6881956,"externalId":"511145.b1172","name":"ymgG","hasFamily":false,"annotation":"conserved protein, UPF0757 family","uniprotId":"YMGG_ECOLI"},"6881957":{"id":6881957,"externalId":"511145.b1174","name":"minE","hasFamily":true,"annotation":"cell division topological specificity factor; Prevents the cell division inhibition by proteins MinC and MinD at internal division sites while permitting inhibition at polar sites. This ensures cell division at the proper site by restricting the formation of a division septum at the midpoint of the long axis of the cell (By similarity)","uniprotId":"MINE_ECOLI"},"6881958":{"id":6881958,"externalId":"511145.b1175","name":"minD","hasFamily":true,"annotation":"membrane ATPase of the MinC-MinD-MinE system; ATPase required for the correct placement of the division site. Cell division inhibitors MinC and MinD act in concert to form an inhibitor capable of blocking formation of the polar Z ring septums. Rapidly oscillates between the poles of the cell to destabilize FtsZ filaments that have formed before they mature into polar Z rings","uniprotId":"MIND_ECOLI"},"6881959":{"id":6881959,"externalId":"511145.b1176","name":"minC","hasFamily":true,"annotation":"cell division inhibitor; Cell division inhibitor that blocks the formation of polar Z ring septums. Rapidly oscillates between the poles of the cell to destabilize FtsZ filaments that have formed before they mature into polar Z rings. Prevents FtsZ polymerization (By similarity)","uniprotId":"MINC_ECOLI"},"6881960":{"id":6881960,"externalId":"511145.b1177","name":"ycgJ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YCGJ_ECOLI"},"6881961":{"id":6881961,"externalId":"511145.b1178","name":"pliG","hasFamily":true,"annotation":"predicted protein; Inhibits activity of g-type lysozyme, which confers increased lysozyme tolerance to the bacterium","uniprotId":"PLIG_ECOLI"},"6881962":{"id":6881962,"externalId":"511145.b1179","name":"ycgL","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCGL_ECOLI"},"6881963":{"id":6881963,"externalId":"511145.b1180","name":"ycgM","hasFamily":true,"annotation":"putative isomerase/hydrolase","uniprotId":"YCGM_ECOLI"},"6881964":{"id":6881964,"externalId":"511145.b1181","name":"ycgN","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCGN_ECOLI"},"6881965":{"id":6881965,"externalId":"511145.b1182","name":"clyA","hasFamily":false,"annotation":"hemolysin E; Toxin, which has some hemolytic activity towards mammalian cells. Acts by forming a pore-like structure upon contact with mammalian cells","uniprotId":"HLYE_ECOLI"},"6881966":{"id":6881966,"externalId":"511145.b1183","name":"umuD","hasFamily":true,"annotation":"DNA polymerase V, subunit D; Involved in UV protection and mutation. Essential for induced (or SOS) mutagenesis. May modify the DNA replication machinery to allow bypass synthesis across a damaged template","uniprotId":"UMUD_ECOLI"},"6881967":{"id":6881967,"externalId":"511145.b1184","name":"umuC","hasFamily":true,"annotation":"DNA polymerase V, subunit C; Involved in UV protection and mutation. Essential for induced (or SOS) mutagenesis. May modify the DNA replication machinery to allow bypass synthesis across a damaged template","uniprotId":"UMUC_ECOLI"},"6881968":{"id":6881968,"externalId":"511145.b1185","name":"dsbB","hasFamily":true,"annotation":"oxidoreductase that catalyzes reoxidation of DsbA protein disulfide isomerase I; Required for disulfide bond formation in some periplasmic proteins such as PhoA or OmpA. Acts by oxidizing the DsbA protein","uniprotId":"DSBB_ECOLI"},"6881969":{"id":6881969,"externalId":"511145.b1186","name":"nhaB","hasFamily":true,"annotation":"sodium:proton antiporter; Na(+)/H(+) antiporter that extrudes sodium in exchange for external protons. Catalyzes the exchange of 3 H(+) per 2 Na(+). Has a high affinity for sodium, but can also transport lithium. Activity is weakly pH-dependent. Essential for regulation of intracellular pH under alkaline conditions","uniprotId":"NHAB_ECOLI"},"6881970":{"id":6881970,"externalId":"511145.b1187","name":"fadR","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator of fatty acid metabolism; Multifunctional regulator of fatty acid metabolism. Represses transcription of at least eight genes required for fatty acid transport and beta-oxidation including fadA, fadB, fadD, fadL and fadE. Activates transcription of at least three genes required for unsaturated fatty acid biosynthesis: fabA, fabB and iclR, the gene encoding the transcriptional regulator of the aceBAK operon encoding the glyoxylate shunt enzymes. Binding of FadR is specifically inhibited by long chain fatty acyl-CoA compounds","uniprotId":"FADR_ECOLI"},"6881971":{"id":6881971,"externalId":"511145.b1188","name":"ycgB","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCGB_ECOLI"},"6881972":{"id":6881972,"externalId":"511145.b1189","name":"dadA","hasFamily":true,"annotation":"D-amino acid dehydrogenase; Oxidative deamination of D-amino acids (By similarity)","uniprotId":"DADA_ECOLI"},"6881973":{"id":6881973,"externalId":"511145.b1190","name":"dadX","hasFamily":true,"annotation":"alanine racemase 2, PLP-binding; Catalyzes the interconversion of L-alanine and D- alanine. May also act on other amino acids (By similarity)","uniprotId":"ALR2_ECOLI"},"6881974":{"id":6881974,"externalId":"511145.b1191","name":"cvrA","hasFamily":true,"annotation":"putative cation/proton antiporter; Participates in control of cell volume in low-osmolarity conditions (By similarity)","uniprotId":"CVRA_ECOLI"},"6881975":{"id":6881975,"externalId":"511145.b1192","name":"ldcA","hasFamily":true,"annotation":"L,D-carboxypeptidase A; Releases the terminal D-alanine residue from the cytoplasmic tetrapeptide recycling product L-Ala-gamma-D-Glu-meso- Dap-D-Ala. To a lesser extent, can also cleave D-Ala from murein derivatives containing the tetrapeptide, i.e. MurNAc-tetrapeptide, UDP-MurNAc-tetrapeptide, GlcNAc-MurNAc-tetrapeptide, and GlcNAc- anhMurNAc-tetrapeptide. Does not act on murein sacculi or cross- linked muropeptides. The tripeptides produced by the LcdA reaction can then be reused as peptidoglycan building blocks; LcdA is thereby involved in murein recycling. Is also essential for vi [...] ","uniprotId":"LDCA_ECOLI"},"6881976":{"id":6881976,"externalId":"511145.b1193","name":"emtA","hasFamily":true,"annotation":"lytic murein endotransglycosylase E; Murein-degrading enzyme. May play a role in recycling of muropeptides during cell elongation and/or cell division. Preferentially cleaves at a distance of more than two disaccharide units from the ends of the glycan chain. Prefers cross-linked murein in vivo","uniprotId":"EMTA_ECOLI"},"6881977":{"id":6881977,"externalId":"511145.b1194","name":"ycgR","hasFamily":true,"annotation":"flagellar velocity braking protein, c-di-GMP-regulated; Acts as a flagellar brake, regulating swimming and swarming in a bis-(3'-5') cyclic diguanylic acid (c-di-GMP)- dependent manner. When bound to c-di-GMP it binds to elements of the flagellar motor (MotA (PubMed:20303158) and/or FliG and FliM (PubMed:20346719), binding to FliM also occurs in the absence of c-di-GMP), causing the motor to slow down. Thus, increasing levels of c-di-GMP lead to decreased motility. Probably binds 1 c-di-GMP dimer per subunit","uniprotId":"YCGR_ECOLI"},"6881978":{"id":6881978,"externalId":"511145.b1195","name":"ymgE","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YMGE_ECOLI"},"6881979":{"id":6881979,"externalId":"511145.b1196","name":"ycgY","hasFamily":false,"annotation":"predicted protein","uniprotId":"YCGY_ECOLI"},"6881980":{"id":6881980,"externalId":"511145.b1197","name":"treA","hasFamily":true,"annotation":"periplasmic trehalase; Provides the cells with the ability to utilize trehalose at high osmolarity by splitting it into glucose molecules that can subsequently be taken up by the phosphotransferase-mediated uptake system (By similarity)","uniprotId":"TREA_ECOLI"},"6881981":{"id":6881981,"externalId":"511145.b1198","name":"dhaM","hasFamily":false,"annotation":"fused predicted dihydroxyacetone-specific PTS enzymes: HPr component/EI component; Phosphotransferase subunit of the dihydroxyacetone kinase, which is responsible for phosphorylating dihydroxyacetone. DhaM serves as the phosphoryl donor. It is phosphorylated by HPr, then it donates its phosphoryl group to DhaL-ADP, which eventually transmits it to dihydroxyacetone","uniprotId":"DHAM_ECOLI"},"6881982":{"id":6881982,"externalId":"511145.b1199","name":"dhaL","hasFamily":true,"annotation":"dihydroxyacetone kinase, C-terminal domain; ADP-binding subunit of the dihydroxyacetone kinase, which is responsible for phosphorylating dihydroxyacetone. DhaL- ADP receives a phosphoryl group from DhaM and transmits it to dihydroxyacetone. DhaL-ADP acts also as a coactivator by binding to the sensor domain of DhaR. DhaL-ATP is inactive","uniprotId":"DHAL_ECOLI"},"6881983":{"id":6881983,"externalId":"511145.b1200","name":"dhaK","hasFamily":true,"annotation":"dihydroxyacetone kinase, PTS-dependent, dihydroxyacetone-binding subunit; Dihydroxyacetone binding subunit of the dihydroxyacetone kinase, which is responsible for phosphorylating dihydroxyacetone. Binds covalently dihydroxyacetone in hemiaminal linkage. Acts also as a corepressor of DhaR by binding to its sensor domain, in the absence of dihydroxyacetone","uniprotId":"DHAK_ECOLI"},"6881984":{"id":6881984,"externalId":"511145.b1201","name":"dhaR","hasFamily":false,"annotation":"DNA-binding transcription activator of the dhaKLM operon; Positively regulates the dhaKLM operon from a sigma-70 promoter. Represses its own synthesis","uniprotId":"DHAR_ECOLI"},"6881985":{"id":6881985,"externalId":"511145.b1202","name":"ycgV","hasFamily":false,"annotation":"putative adhesin","uniprotId":"YCGV_ECOLI"},"6881986":{"id":6881986,"externalId":"511145.b1203","name":"ychF","hasFamily":true,"annotation":"ATPase, K+-dependent, ribosome-associated; ATPase that binds to both the 70S ribosome and the 50S ribosomal subunit in a nucleotide-independent manner. Does not hydrolyze GTP","uniprotId":"YCHF_ECOLI"},"6881987":{"id":6881987,"externalId":"511145.b1204","name":"pth","hasFamily":true,"annotation":"peptidyl-tRNA hydrolase; The natural substrate for this enzyme may be peptidyl- tRNAs which drop off the ribosome during protein synthesis. Involved in lambda inhibition of host protein synthesis. PTH activity may, directly or indirectly, be the target for lambda bar RNA leading to rap cell death","uniprotId":"PTH_ECOLI"},"6881988":{"id":6881988,"externalId":"511145.b1205","name":"ychH","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YCHH_ECOLI"},"6881989":{"id":6881989,"externalId":"511145.b1206","name":"dauA","hasFamily":true,"annotation":"predicted transporter; Responsible for the aerobic transport of succinate from the periplasm to the cytoplasm at acidic pH. Can transport other C4-dicarboxylic acids such as aspartate and fumarate. May also play a role in the regulation of C4-dicarboxylic acid metabolism at pH 7, via regulation of expression and/or activity of DctA. May act as a co-sensor of DcuS","uniprotId":"DAUA_ECOLI"},"6881990":{"id":6881990,"externalId":"511145.b1207","name":"prs","hasFamily":true,"annotation":"phosphoribosylpyrophosphate synthase","uniprotId":"KPRS_ECOLI"},"6881991":{"id":6881991,"externalId":"511145.b1208","name":"ispE","hasFamily":true,"annotation":"4-diphosphocytidyl-2-C-methylerythritol kinase; Catalyzes the phosphorylation of the position 2 hydroxy group of 4-diphosphocytidyl-2C-methyl-D-erythritol. Phosphorylates isopentenyl phosphate at low rates. Also acts on isopentenol, and, much less efficiently, dimethylallyl alcohol. Dimethylallyl monophosphate does not serve as a substrate","uniprotId":"ISPE_ECOLI"},"6881992":{"id":6881992,"externalId":"511145.b1209","name":"hemM","hasFamily":true,"annotation":"lipoprotein localization factor; Plays a critical role in the incorporation of lipoproteins in the outer membrane after they are released by the LolA protein. Essential for E.coli viability","uniprotId":"LOLB_ECOLI"},"6881993":{"id":6881993,"externalId":"511145.b1210","name":"hemA","hasFamily":true,"annotation":"glutamyl tRNA reductase; Catalyzes the NADPH-dependent reduction of glutamyl- tRNA(Glu) to glutamate 1-semialdehyde (GSA). In the absence of NADPH, exhibits substrate esterase activity, leading to the release of glutamate from tRNA","uniprotId":"HEM1_ECOLI"},"6881994":{"id":6881994,"externalId":"511145.b1211","name":"prfA","hasFamily":true,"annotation":"peptide chain release factor RF-1; Peptide chain release factor 1 directs the termination of translation in response to the peptide chain termination codons UAG and UAA (By similarity)","uniprotId":"RF1_ECOLI"},"6881995":{"id":6881995,"externalId":"511145.b1212","name":"hemK","hasFamily":true,"annotation":"N5-glutamine methyltransferase, modifies release factors RF-1 and RF-2; Methylates the class 1 translation termination release factors RF1/PrfA and RF2/PrfB on the glutamine residue of the universally conserved GGQ motif, i.e. on 'Gln-235' in RF1 and on 'Gln-252' in RF2","uniprotId":"PRMC_ECOLI"},"6881996":{"id":6881996,"externalId":"511145.b1213","name":"ychQ","hasFamily":true,"annotation":"putative inner membrane protein, SIRB family","uniprotId":"SIRB2_ECOLI"},"6881997":{"id":6881997,"externalId":"511145.b1214","name":"ychA","hasFamily":true,"annotation":"conserved protein","uniprotId":"SIRB1_ECOLI"},"6881998":{"id":6881998,"externalId":"511145.b1215","name":"kdsA","hasFamily":true,"annotation":"3-deoxy-D-manno-octulosonate 8-phosphate synthase; Synthesis of KDO 8-P which is required for lipid A maturation and cellular growth","uniprotId":"KDSA_ECOLI"},"6881999":{"id":6881999,"externalId":"511145.b1216","name":"chaA","hasFamily":true,"annotation":"calcium/sodium:proton antiporter; According to PubMed:8496184 the natural activity of this protein is as a calcium/proton antiporter, but it can act as a weak sodium/proton antiporter. The presence of a preferred substrate and normal pH cytoplasmic values might prevent significant sodium/proton antiporter activity","uniprotId":"CHAA_ECOLI"},"6882000":{"id":6882000,"externalId":"511145.b1217","name":"chaB","hasFamily":true,"annotation":"cation transport regulator","uniprotId":"CHAB_ECOLI"},"6882001":{"id":6882001,"externalId":"511145.b1218","name":"chaC","hasFamily":true,"annotation":"cation transport regulator","uniprotId":"CHAC_ECOLI"},"6882002":{"id":6882002,"externalId":"511145.b1219","name":"ychN","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCHN_ECOLI"},"6882003":{"id":6882003,"externalId":"511145.b1220","name":"ychO","hasFamily":true,"annotation":"putative invasin","uniprotId":"YCHO_ECOLI"},"6882004":{"id":6882004,"externalId":"511145.b1221","name":"narL","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with NarX (or NarQ); This protein activates the expression of the nitrate reductase (narGHJI) and formate dehydrogenase-N (fdnGHI) operons and represses the transcription of the fumarate reductase (frdABCD) operon in response to a nitrate/nitrite induction signal transmitted by either the NarX or NarQ proteins","uniprotId":"NARL_ECOLI"},"6882005":{"id":6882005,"externalId":"511145.b1222","name":"narX","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with NarL; Acts as a sensor for nitrate/nitrite and transduces signal of nitrate availability to the NarL protein and of both nitrate/nitrite to the NarP protein. NarX probably activates NarL and NarP by phosphorylation in the presence of nitrate. NarX also plays a negative role in controlling NarL activity, probably through dephosphorylation in the absence of nitrate","uniprotId":"NARX_ECOLI"},"6882006":{"id":6882006,"externalId":"511145.b1223","name":"narK","hasFamily":true,"annotation":"nitrate/nitrite transporter; Involved in excretion of nitrite produced by the dissimilatory reduction of nitrate","uniprotId":"NARK_ECOLI"},"6882007":{"id":6882007,"externalId":"511145.b1224","name":"narG","hasFamily":true,"annotation":"nitrate reductase 1, alpha subunit; The nitrate reductase enzyme complex allows E.coli to use nitrate as an electron acceptor during anaerobic growth. The alpha chain is the actual site of nitrate reduction","uniprotId":"NARG_ECOLI"},"6882008":{"id":6882008,"externalId":"511145.b1225","name":"narH","hasFamily":true,"annotation":"nitrate reductase 1, beta (Fe-S) subunit; The nitrate reductase enzyme complex allows E.coli to use nitrate as an electron acceptor during anaerobic growth. The beta chain is an electron transfer unit containing four cysteine clusters involved in the formation of iron-sulfur centers. Electrons are transferred from the gamma chain to the molybdenum cofactor of the alpha subunit","uniprotId":"NARH_ECOLI"},"6882009":{"id":6882009,"externalId":"511145.b1226","name":"narJ","hasFamily":true,"annotation":"molybdenum-cofactor-assembly chaperone subunit (delta subunit) of nitrate reductase 1; Chaperone required for proper molybdenum cofactor insertion and final assembly of the membrane-bound respiratory nitrate reductase 1. Required for the insertion of the molybdenum into the apo-NarG subunit, maybe by keeping NarG in an appropriate competent-open conformation for the molybdenum cofactor insertion to occur. NarJ maintains the apoNarGH complex in a soluble state. Upon insertion of the molybdenum cofactor, NarJ seems to dissociate from the activated soluble NarGH complex, before its associ [...] ","uniprotId":"NARJ_ECOLI"},"6882010":{"id":6882010,"externalId":"511145.b1227","name":"narI","hasFamily":true,"annotation":"nitrate reductase 1, gamma (cytochrome b(NR)) subunit; The nitrate reductase enzyme complex allows E.coli to use nitrate as an electron acceptor during anaerobic growth. The gamma chain is a membrane-embedded heme-iron unit resembling cytochrome b, which transfers electrons from quinones to the beta subunit","uniprotId":"NARI_ECOLI"},"6882011":{"id":6882011,"externalId":"511145.b1229","name":"tpr","hasFamily":false,"annotation":"protamine-like protein","uniprotId":"PRTL_ECOLI"},"6882012":{"id":6882012,"externalId":"511145.b1232","name":"purU","hasFamily":true,"annotation":"formyltetrahydrofolate hydrolase; Produces formate from formyl-tetrahydrofolate. Provides the major source of formate for the PurT-dependent synthesis of 5'-phosphoribosyl-N-formylglycinamide (FGAR) during aerobic growth. Has a role in regulating the one-carbon pool","uniprotId":"PURU_ECOLI"},"6882013":{"id":6882013,"externalId":"511145.b1233","name":"ychJ","hasFamily":true,"annotation":"conserved protein, UPF0225 family","uniprotId":"YCHJ_ECOLI"},"6882014":{"id":6882014,"externalId":"511145.b1234","name":"rssA","hasFamily":true,"annotation":"putative phospholipase, patatin-like family","uniprotId":"RSSA_ECOLI"},"6882015":{"id":6882015,"externalId":"511145.b1235","name":"hnr","hasFamily":true,"annotation":"response regulator binding RpoS to initiate proteolysis by ClpXP; required for the PcnB-degradosome interaction during stationary phase","uniprotId":"HNR_ECOLI"},"6882016":{"id":6882016,"externalId":"511145.b1236","name":"galU","hasFamily":true,"annotation":"glucose-1-phosphate uridylyltransferase; May play a role in stationary phase survival","uniprotId":"GALU_ECOLI"},"6882017":{"id":6882017,"externalId":"511145.b1237","name":"hns","hasFamily":true,"annotation":"global DNA-binding transcriptional dual regulator H-NS; A DNA-binding protein implicated in transcriptional repression (silencing) as well as in bacterial chromosome organization. H-NS binds tightly to AT-rich dsDNA, increases its thermal stability and inhibits transcription. Also binds to ssDNA and RNA but with a much lower affinity. H-NS has possible histone- like function. May be a global transcriptional regulator through its ability to bind to curved DNA sequences, which are found in regions upstream of a certain subset of promoters. Plays a role in the thermal control of pili and  [...] ","uniprotId":"HNS_ECOLI"},"6882018":{"id":6882018,"externalId":"511145.b1238","name":"tdk","hasFamily":true,"annotation":"thymidine kinase/deoxyuridine kinase; Phosphorylates both thymidine and deoxyuridine","uniprotId":"KITH_ECOLI"},"6882019":{"id":6882019,"externalId":"511145.b1241","name":"adhE","hasFamily":true,"annotation":"fused acetaldehyde-CoA dehydrogenase/iron-dependent alcohol dehydrogenase/pyruvate-formate lyase deactivase; This enzyme has three activities: ADH, ACDH, and PFL- deactivase. In aerobic conditions it acts as a hydrogen peroxide scavenger. The PFL deactivase activity catalyzes the quenching of the pyruvate-formate-lyase catalyst in an iron, NAD, and CoA dependent reaction","uniprotId":"ADHE_ECOLI"},"6882020":{"id":6882020,"externalId":"511145.b1242","name":"ychE","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YCHE_ECOLI"},"6882021":{"id":6882021,"externalId":"511145.b1243","name":"oppA","hasFamily":true,"annotation":"oligopeptide transporter subunit; This protein is a component of the oligopeptide permease, a binding protein-dependent transport system, it binds peptides up to five amino acids long with high affinity","uniprotId":"OPPA_ECOLI"},"6882022":{"id":6882022,"externalId":"511145.b1244","name":"oppB","hasFamily":true,"annotation":"oligopeptide transporter subunit; Part of the binding-protein-dependent transport system for oligopeptides; probably responsible for the translocation of the substrate across the membrane","uniprotId":"OPPB_ECOLI"},"6882023":{"id":6882023,"externalId":"511145.b1245","name":"oppC","hasFamily":true,"annotation":"oligopeptide transporter subunit; Part of the binding-protein-dependent transport system for oligopeptides; probably responsible for the translocation of the substrate across the membrane","uniprotId":"OPPC_ECOLI"},"6882024":{"id":6882024,"externalId":"511145.b1246","name":"oppD","hasFamily":true,"annotation":"oligopeptide transporter subunit; Part of the binding-protein-dependent transport system for oligopeptides. Probably responsible for energy coupling to the transport system","uniprotId":"OPPD_ECOLI"},"6882025":{"id":6882025,"externalId":"511145.b1247","name":"oppF","hasFamily":true,"annotation":"oligopeptide transporter subunit; Part of the binding-protein-dependent transport system for oligopeptides. Probably responsible for energy coupling to the transport system","uniprotId":"OPPF_ECOLI"},"6882026":{"id":6882026,"externalId":"511145.b1248","name":"yciU","hasFamily":true,"annotation":"conserved protein, UPF0263 family","uniprotId":"YCIU_ECOLI"},"6882027":{"id":6882027,"externalId":"511145.b1249","name":"cls","hasFamily":true,"annotation":"cardiolipin synthase 1; Catalyzes the reversible phosphatidyl group transfer from one phosphatidylglycerol molecule to another to form cardiolipin (CL) (diphosphatidylglycerol) and glycerol. Affects resistance to the gyrase inhibitor novobiocin","uniprotId":"CLS_ECOLI"},"6882028":{"id":6882028,"externalId":"511145.b1250","name":"kch","hasFamily":true,"annotation":"voltage-gated potassium channel; K(+)-specific ion channel. May play a role in the defense against osmotic shock","uniprotId":"KCH_ECOLI"},"6882029":{"id":6882029,"externalId":"511145.b1251","name":"yciI","hasFamily":true,"annotation":"putative enzyme","uniprotId":"YCII_ECOLI"},"6882030":{"id":6882030,"externalId":"511145.b1252","name":"tonB","hasFamily":true,"annotation":"membrane spanning protein in TonB-ExbB-ExbD transport complex"},"6882031":{"id":6882031,"externalId":"511145.b1253","name":"yciA","hasFamily":true,"annotation":"acyl-CoA esterase; Catalyzes the hydrolysis of the thioester bond in palmitoyl-CoA and malonyl-CoA","uniprotId":"YCIA_ECOLI"},"6882032":{"id":6882032,"externalId":"511145.b1254","name":"yciB","hasFamily":true,"annotation":"putative inner membrane protein; Involved in cell division; probably involved in intracellular septation (By similarity)","uniprotId":"YCIB_ECOLI"},"6882033":{"id":6882033,"externalId":"511145.b1255","name":"yciC","hasFamily":true,"annotation":"inner membrane protein, UPF0259 family"},"6882034":{"id":6882034,"externalId":"511145.b1256","name":"ompW","hasFamily":true,"annotation":"outer membrane protein W; Acts as a receptor for colicin S4","uniprotId":"OMPW_ECOLI"},"6882035":{"id":6882035,"externalId":"511145.b1257","name":"yciE","hasFamily":true,"annotation":"putative rubrerythrin/ferritin-like metal-binding protein","uniprotId":"YCIE_ECOLI"},"6882036":{"id":6882036,"externalId":"511145.b1258","name":"yciF","hasFamily":true,"annotation":"putative rubrerythrin/ferritin-like metal-binding protein","uniprotId":"YCIF_ECOLI"},"6882037":{"id":6882037,"externalId":"511145.b1259","name":"yciG","hasFamily":false,"annotation":"predicted protein","uniprotId":"YCIG_ECOLI"},"6882038":{"id":6882038,"externalId":"511145.b1260","name":"trpA","hasFamily":true,"annotation":"tryptophan synthase, alpha subunit; The alpha subunit is responsible for the aldol cleavage of indoleglycerol phosphate to indole and glyceraldehyde 3- phosphate (By similarity)","uniprotId":"TRPA_ECOLI"},"6882039":{"id":6882039,"externalId":"511145.b1261","name":"trpB","hasFamily":true,"annotation":"tryptophan synthase, beta subunit; The beta subunit is responsible for the synthesis of L- tryptophan from indole and L-serine (By similarity)","uniprotId":"TRPB_ECOLI"},"6882040":{"id":6882040,"externalId":"511145.b1262","name":"trpC","hasFamily":true,"annotation":"fused indole-3-glycerolphosphate synthetase/N-(5-phosphoribosyl)anthranilate isomerase; Bifunctional enzyme that catalyzes two sequential steps of tryptophan biosynthetic pathway. The first reaction is catalyzed by the isomerase, coded by the TrpF domain; the second reaction is catalyzed by the synthase, coded by the TrpC domain","uniprotId":"TRPC_ECOLI"},"6882041":{"id":6882041,"externalId":"511145.b1263","name":"trpD","hasFamily":true,"annotation":"fused glutamine amidotransferase (component II) of anthranilate synthase/anthranilate phosphoribosyl transferase","uniprotId":"TRPG_ECOLI"},"6882042":{"id":6882042,"externalId":"511145.b1264","name":"trpE","hasFamily":true,"annotation":"component I of anthranilate synthase","uniprotId":"TRPE_ECOLI"},"6882043":{"id":6882043,"externalId":"511145.b1265","name":"trpL","hasFamily":false,"annotation":"trp operon leader peptide"},"6882044":{"id":6882044,"externalId":"511145.b1266","name":"trpH","hasFamily":true,"annotation":"conserved protein","uniprotId":"TRPH_ECOLI"},"6882045":{"id":6882045,"externalId":"511145.b1267","name":"yciO","hasFamily":true,"annotation":"putative RNA binding protein","uniprotId":"YCIO_ECOLI"},"6882046":{"id":6882046,"externalId":"511145.b1268","name":"yciQ","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YCIQ_ECOLI"},"6882047":{"id":6882047,"externalId":"511145.b1269","name":"rluB","hasFamily":true,"annotation":"23S rRNA pseudouridylate synthase; Responsible for synthesis of pseudouridine from uracil- 2605 in 23S ribosomal RNA","uniprotId":"RLUB_ECOLI"},"6882048":{"id":6882048,"externalId":"511145.b1270","name":"btuR","hasFamily":true,"annotation":"cob(I)alamin adenolsyltransferase/cobinamide ATP-dependent adenolsyltransferase; Required for both de novo synthesis of the corrin ring for the assimilation of exogenous corrinoids. Participates in the adenosylation of a variety of incomplete and complete corrinoids (By similarity)","uniprotId":"BTUR_ECOLI"},"6882049":{"id":6882049,"externalId":"511145.b1271","name":"yciK","hasFamily":true,"annotation":"putative oxoacyl-(acyl carrier protein) reductase, EmrKY-TolC system","uniprotId":"YCIK_ECOLI"},"6882050":{"id":6882050,"externalId":"511145.b1272","name":"sohB","hasFamily":true,"annotation":"putative inner membrane peptidase; multicopy suppressor of htrA(degP); Multicopy suppressor of the HtrA (DegP) null phenotype. It is possibly a protease, not essential for bacterial viability","uniprotId":"SOHB_ECOLI"},"6882051":{"id":6882051,"externalId":"511145.b1273","name":"yciN","hasFamily":true,"annotation":"DUF2498 protein YciN","uniprotId":"YCIN_ECOLI"},"6882052":{"id":6882052,"externalId":"511145.b1274","name":"topA","hasFamily":true,"annotation":"DNA topoisomerase I, omega subunit; Releases the supercoiling and torsional tension of DNA, which is introduced during the DNA replication and transcription, by transiently cleaving and rejoining one strand of the DNA duplex. Introduces a single-strand break via transesterification at a target site in duplex DNA. The scissile phosphodiester is attacked by the catalytic tyrosine of the enzyme, resulting in the formation of a DNA-(5'-phosphotyrosyl)-enzyme intermediate and the expulsion of a 3'-OH DNA strand. The free DNA strand then undergoes passage around the unbroken strand, thus rem [...] ","uniprotId":"TOP1_ECOLI"},"6882053":{"id":6882053,"externalId":"511145.b1275","name":"cysB","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator, O-acetyl-L-serine-binding; This protein is a positive regulator of gene expression for the cysteine regulon, a system of 10 or more loci involved in the biosynthesis of L-cysteine from inorganic sulfate. The inducer for CysB is N-acetylserine. CysB inhibits its own transcription","uniprotId":"CYSB_ECOLI"},"6882054":{"id":6882054,"externalId":"511145.b1276","name":"acnA","hasFamily":true,"annotation":"aconitate hydratase 1; Catalyzes the isomerization of citrate to isocitrate via cis-aconitate. May have an iron-responsive regulatory function","uniprotId":"ACON1_ECOLI"},"6882055":{"id":6882055,"externalId":"511145.b1277","name":"ribA","hasFamily":true,"annotation":"GTP cyclohydrolase II; Catalyzes the conversion of GTP to 2,5-diamino-6- ribosylamino-4(3H)-pyrimidinone 5'-phosphate (DARP), formate and pyrophosphate (By similarity)","uniprotId":"RIBA_ECOLI"},"6882056":{"id":6882056,"externalId":"511145.b1278","name":"pgpB","hasFamily":true,"annotation":"phosphatidylglycerophosphatase B; Catalyzes the dephosphorylation of diacylglycerol diphosphate (DGPP) to phosphatidate (PA) and the subsequent dephosphorylation of PA to diacylglycerol (DAG). Also has undecaprenyl pyrophosphate phosphatase activity, required for the biosynthesis of the lipid carrier undecaprenyl phosphate. Can also use lysophosphatidic acid (LPA) and phosphatidylglycerophosphate as substrates. The pattern of activities varies according to subcellular location, PGP phosphatase activity is higher in the cytoplasmic membrane, whereas PA and LPA phosphatase activities are [...] ","uniprotId":"PGPB_ECOLI"},"6882057":{"id":6882057,"externalId":"511145.b1279","name":"yciS","hasFamily":true,"annotation":"inner membrane DUF1049 protein YciS","uniprotId":"YCIS_ECOLI"},"6882058":{"id":6882058,"externalId":"511145.b1280","name":"yciM","hasFamily":true,"annotation":"TPR-repeats-containing protein","uniprotId":"YCIM_ECOLI"},"6882059":{"id":6882059,"externalId":"511145.b1281","name":"pyrF","hasFamily":true,"annotation":"orotidine-5'-phosphate decarboxylase; Catalyzes the decarboxylation of orotidine 5'- monophosphate (OMP) to uridine 5'-monophosphate (UMP) (By similarity)","uniprotId":"PYRF_ECOLI"},"6882060":{"id":6882060,"externalId":"511145.b1282","name":"yciH","hasFamily":true,"annotation":"initiation factor function partial mimic, SUI1 family","uniprotId":"YCIH_ECOLI"},"6882061":{"id":6882061,"externalId":"511145.b1283","name":"osmB","hasFamily":true,"annotation":"lipoprotein; Provides resistance to osmotic stress. May be important for stationary-phase survival","uniprotId":"OSMB_ECOLI"},"6882062":{"id":6882062,"externalId":"511145.b1284","name":"yciT","hasFamily":true,"annotation":"global regulator of transcription","uniprotId":"YCIT_ECOLI"},"6882063":{"id":6882063,"externalId":"511145.b1285","name":"gmr","hasFamily":true,"annotation":"cyclic-di-GMP phosphodiesterase; csgD regulator; Part of a 2 protein system that seems to be dedicated to transcription of csgD. This protein decreases csgD transcription and thus decreases expression of adhesive curli fimbriae genes csgEFG, csgBAC/ymaD and adrA (yaic). Activity of this protein is antagonized by the diguanylate cyclase YdaM. In vitro has c-di-GMP phosphodiesterase activity. Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"GMR_ECOLI"},"6882064":{"id":6882064,"externalId":"511145.b1286","name":"rnb","hasFamily":true,"annotation":"ribonuclease II; Involved in mRNA degradation. Hydrolyzes single-stranded polyribonucleotides processively in the 3' to 5' direction (By similarity)","uniprotId":"RNB_ECOLI"},"6882065":{"id":6882065,"externalId":"511145.b1287","name":"yciW","hasFamily":true,"annotation":"putative oxidoreductase","uniprotId":"YCIW_ECOLI"},"6882066":{"id":6882066,"externalId":"511145.b1288","name":"fabI","hasFamily":true,"annotation":"enoyl-[acyl-carrier-protein] reductase, NADH-dependent; Catalyzes the reduction of a carbon-carbon double bond in an enoyl moiety that is covalently linked to an acyl carrier protein (ACP). Involved in the elongation cycle of fatty acid which are used in the lipid metabolism and in the biotin biosynthesis","uniprotId":"FABI_ECOLI"},"6882067":{"id":6882067,"externalId":"511145.b1289","name":"ycjD","hasFamily":true,"annotation":"conserved protein","uniprotId":"YCJD_ECOLI"},"6882068":{"id":6882068,"externalId":"511145.b1290","name":"sapF","hasFamily":true,"annotation":"antimicrobial peptide transport ABC system ATP-binding protein; Involved in a peptide intake transport system that plays a role in the resistance to antimicrobial peptides","uniprotId":"SAPF_ECOLI"},"6882069":{"id":6882069,"externalId":"511145.b1291","name":"sapD","hasFamily":true,"annotation":"antimicrobial peptide transport ABC system ATP-binding protein; Involved in a peptide intake transport system that plays a role in the resistance to antimicrobial peptides","uniprotId":"SAPD_ECOLI"},"6882070":{"id":6882070,"externalId":"511145.b1292","name":"sapC","hasFamily":true,"annotation":"antimicrobial peptide transport ABC transporter permease; Involved in a peptide intake transport system that plays a role in the resistance to antimicrobial peptides","uniprotId":"SAPC_ECOLI"},"6882071":{"id":6882071,"externalId":"511145.b1293","name":"sapB","hasFamily":true,"annotation":"antimicrobial peptide transport ABC transporter permease; Involved in a peptide intake transport system that plays a role in the resistance to antimicrobial peptides","uniprotId":"SAPB_ECOLI"},"6882072":{"id":6882072,"externalId":"511145.b1294","name":"sapA","hasFamily":true,"annotation":"antimicrobial peptide transport ABC transporter periplasmic binding protein; Involved in a peptide intake transport system that plays a role in the resistance to antimicrobial peptides","uniprotId":"SAPA_ECOLI"},"6882073":{"id":6882073,"externalId":"511145.b1295","name":"ymjA","hasFamily":false,"annotation":"predicted protein","uniprotId":"YMJA_ECOLI"},"6882074":{"id":6882074,"externalId":"511145.b1296","name":"puuP","hasFamily":true,"annotation":"putrescine importer; Involved in the uptake of putrescine","uniprotId":"PUUP_ECOLI"},"6882075":{"id":6882075,"externalId":"511145.b1297","name":"puuA","hasFamily":true,"annotation":"gamma-Glu-putrescine synthase; Involved in the breakdown of putrescine via the biosynthesis of gamma-L-glutamylputrescine. It is able to use several diamines, spermidine and spermine. Absolutely essential to utilize putrescine as both nitrogen and carbon sources and to decrease the toxicity of putrescine, which can lead to inhibition of cell growth and protein synthesis","uniprotId":"PUUA_ECOLI"},"6882076":{"id":6882076,"externalId":"511145.b1298","name":"puuD","hasFamily":true,"annotation":"gamma-Glu-GABA hydrolase; Involved in the breakdown of putrescine via hydrolysis of the gamma-glutamyl linkage of gamma-glutamyl-gamma- aminobutyrate","uniprotId":"PUUD_ECOLI"},"6882077":{"id":6882077,"externalId":"511145.b1299","name":"puuR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor for the puu divergon; Represses puuA, puuD and puuP","uniprotId":"PUUR_ECOLI"},"6882078":{"id":6882078,"externalId":"511145.b1300","name":"aldH","hasFamily":true,"annotation":"gamma-Glu-gamma-aminobutyraldehyde dehydrogenase, NAD(P)H-dependent; Catalyzes the oxidation of 3-hydroxypropionaldehyde (3- HPA) to 3-hydroxypropionic acid (3-HP). It acts preferentially with NAD but can also use NADP. 3-HPA appears to be the most suitable substrate for PuuC followed by isovaleraldehyde, propionaldehyde, butyraldehyde, and valeraldehyde. It might play a role in propionate and/or acetic acid metabolisms. Also involved in the breakdown of putrescine through the oxidation of gamma-Glu- gamma-aminobutyraldehyde to gamma-Glu-gamma-aminobutyrate (gamma- Glu-GABA)","uniprotId":"PUUC_ECOLI"},"6882079":{"id":6882079,"externalId":"511145.b1301","name":"ordL","hasFamily":true,"annotation":"gamma-Glu-putrescine oxidase, FAD/NAD(P)-binding; Involved in the breakdown of putrescine via the oxidation of L-glutamylputrescine","uniprotId":"PUUB_ECOLI"},"6882080":{"id":6882080,"externalId":"511145.b1302","name":"goaG","hasFamily":true,"annotation":"GABA aminotransferase, PLP-dependent; Catalyzes the transfer of the amino group from gamma- aminobutyrate (GABA) to alpha-ketoglutarate (KG) to yield succinic semialdehyde (SSA). PuuE is important for utilization of putrescine as the sole nitrogen or carbon source","uniprotId":"PUUE_ECOLI"},"6882081":{"id":6882081,"externalId":"511145.b1303","name":"pspF","hasFamily":true,"annotation":"DNA-binding transcriptional activator; Transcriptional activator for the phage shock protein (psp) operon (pspABCDE) and pspG gene","uniprotId":"PSPF_ECOLI"},"6882082":{"id":6882082,"externalId":"511145.b1304","name":"pspA","hasFamily":true,"annotation":"regulatory protein for phage-shock-protein operon; The phage shock protein (psp) operon (pspABCDE) may play a significant role in the competition for survival under nutrient- or energy-limited conditions. PspA negatively regulates expression of the pspABCDE promoter and of pspG through negative regulation of the psp-specific transcriptional activator PspF. Is also required for membrane integrity, efficient translocation and maintenance of the proton motive force","uniprotId":"PSPA_ECOLI"},"6882083":{"id":6882083,"externalId":"511145.b1305","name":"pspB","hasFamily":true,"annotation":"DNA-binding transcriptional regulator of psp operon; The phage shock protein (psp) operon (pspABCDE) may play a significant role in the competition for survival under nutrient- or energy-limited conditions. PspB is involved in transcription regulation","uniprotId":"PSPB_ECOLI"},"6882084":{"id":6882084,"externalId":"511145.b1306","name":"pspC","hasFamily":true,"annotation":"DNA-binding transcriptional activator; The phage shock protein (psp) operon (pspABCDE) may play a significant role in the competition for survival under nutrient- or energy-limited conditions. PspC is involved in transcription regulation","uniprotId":"PSPC_ECOLI"},"6882085":{"id":6882085,"externalId":"511145.b1307","name":"pspD","hasFamily":true,"annotation":"peripheral inner membrane phage-shock protein; The phage shock protein (psp) operon (pspABCDE) may play a significant role in the competition for survival under nutrient- or energy-limited conditions","uniprotId":"PSPD_ECOLI"},"6882086":{"id":6882086,"externalId":"511145.b1308","name":"pspE","hasFamily":true,"annotation":"thiosulfate:cyanide sulfurtransferase (rhodanese); The phage shock protein (psp) operon (pspABCDE) may play a significant role in the competition for survival under nutrient- or energy-limited conditions. PspE catalyzes the sulfur-transfer reaction from thiosulfate to cyanide, to form sulfite and thiocyanate. Also able to use dithiol (dithiothreitol) as an alternate sulfur acceptor. Also possesses a very low mercaptopyruvate sulfurtransferase activity","uniprotId":"PSPE_ECOLI"},"6882087":{"id":6882087,"externalId":"511145.b1309","name":"ycjM","hasFamily":true,"annotation":"putative glucosyltransferase","uniprotId":"SUCP_ECOLI"},"6882088":{"id":6882088,"externalId":"511145.b1310","name":"ycjN","hasFamily":true,"annotation":"putative sugar transporter subunit: periplasmic-binding component of ABC superfamily; Probably part of the binding-protein-dependent transport system YcjNOP","uniprotId":"YCJN_ECOLI"},"6882089":{"id":6882089,"externalId":"511145.b1311","name":"ycjO","hasFamily":true,"annotation":"putative sugar transporter subunit: membrane component of ABC superfamily; Probably part of the binding-protein-dependent transport system YcjNOP. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"YCJO_ECOLI"},"6882090":{"id":6882090,"externalId":"511145.b1312","name":"ycjP","hasFamily":true,"annotation":"putative sugar transporter subunit: membrane component of ABC superfamily; Probably part of the binding-protein-dependent transport system YcjNOP. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"YCJP_ECOLI"},"6882091":{"id":6882091,"externalId":"511145.b1313","name":"ycjQ","hasFamily":true,"annotation":"putative oxidoreductase, Zn-dependent and NAD(P)-binding","uniprotId":"YCJQ_ECOLI"},"6882092":{"id":6882092,"externalId":"511145.b1314","name":"ycjR","hasFamily":true,"annotation":"putative enzyme","uniprotId":"YCJR_ECOLI"},"6882093":{"id":6882093,"externalId":"511145.b1315","name":"ycjS","hasFamily":true,"annotation":"putative oxidoreductase, NADH-binding","uniprotId":"YCJS_ECOLI"},"6882094":{"id":6882094,"externalId":"511145.b1316","name":"ycjT","hasFamily":true,"annotation":"putative hydrolase","uniprotId":"YCJT_ECOLI"},"6882095":{"id":6882095,"externalId":"511145.b1317","name":"ycjU","hasFamily":true,"annotation":"beta-phosphoglucomutase; Catalyzes the interconversion of D-glucose 1-phosphate (G1P) and D-glucose 6-phosphate (G6P), forming beta-D-glucose 1,6- (bis)phosphate (beta-G16P) as an intermediate. The beta- phosphoglucomutase (Beta-PGM) acts on the beta-C(1) anomer of G1P. Glucose or lactose are used in preference to maltose, which is only utilized after glucose or lactose has been exhausted. It plays a key role in the regulation of the flow of carbohydrate intermediates in glycolysis and the formation of the sugar nucleotide UDP-glucose","uniprotId":"PGMB_ECOLI"},"6882096":{"id":6882096,"externalId":"511145.b1319","name":"ompG","hasFamily":false,"annotation":"outer membrane porin G; Forms channels functionally larger than those of classical porins","uniprotId":"OMPG_ECOLI"},"6882097":{"id":6882097,"externalId":"511145.b1320","name":"ycjW","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YCJW_ECOLI"},"6882098":{"id":6882098,"externalId":"511145.b1321","name":"ycjX","hasFamily":true,"annotation":"conserved protein with nucleoside triphosphate hydrolase domain","uniprotId":"YCJX_ECOLI"},"6882099":{"id":6882099,"externalId":"511145.b1322","name":"ycjF","hasFamily":true,"annotation":"inner membrane protein, UPF0283 family","uniprotId":"YCJF_ECOLI"},"6882100":{"id":6882100,"externalId":"511145.b1323","name":"tyrR","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator, tyrosine-binding; Involved in transcriptional regulation of aromatic amino acid biosynthesis and transport. Modulates the expression of at least 8 unlinked operons. Seven of these operons are regulated in response to changes in the concentration of the three aromatic amino acids (phenylalanine, tyrosine and tryptophan). These amino acids are suggested to act as co-effectors which bind to the TyrR protein to form an active regulatory protein. In most cases TyrR causes negative regulation, but positive effects on the tyrP gene have been observe [...] ","uniprotId":"TYRR_ECOLI"},"6882101":{"id":6882101,"externalId":"511145.b1324","name":"tpx","hasFamily":true,"annotation":"lipid hydroperoxide peroxidase; Has antioxidant activity. Could remove peroxides or H(2)O(2) within the catalase- and peroxidase-deficient periplasmic space","uniprotId":"TPX_ECOLI"},"6882102":{"id":6882102,"externalId":"511145.b1325","name":"ycjG","hasFamily":true,"annotation":"L-Ala-D/L-Glu epimerase; Catalyzes the epimerization of L-Ala-D-Glu to L-Ala-L- Glu and has a role in the recycling of the murein peptide, of which L-Ala-D-Glu is a component. Is also able to catalyze the reverse reaction and the epimerization of all the L-Ala-X dipeptides, except L-Ala-L-Arg, L-Ala-L-Lys and L-Ala-L-Pro. Is also active with L-Gly-L-Glu, L-Phe-L-Glu, and L-Ser-L-Glu, but not with L-Glu-L-Glu, L-Lys-L-Glu, L-Pro-L-Glu, L-Lys-L-Ala, or D- Ala-D-Ala","uniprotId":"AEEP_ECOLI"},"6882103":{"id":6882103,"externalId":"511145.b1326","name":"mpaA","hasFamily":true,"annotation":"murein peptide amidase A","uniprotId":"MPAA_ECOLI"},"6882104":{"id":6882104,"externalId":"511145.b1327","name":"ycjY","hasFamily":true,"annotation":"putative hydrolase","uniprotId":"YCJY_ECOLI"},"6882105":{"id":6882105,"externalId":"511145.b1328","name":"ycjZ","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YCJZ_ECOLI"},"6882106":{"id":6882106,"externalId":"511145.b1329","name":"mppA","hasFamily":true,"annotation":"murein tripeptide (L-ala-gamma-D-glutamyl-meso-DAP) transporter subunit; Essential for the uptake of the murein peptide L-alanyl- gamma-D-glutamyl-meso-diaminopimelate. Also transports some alpha- linked peptides such as Pro-Phe-Lys with low affinity. The transport is effected by the oligopeptide permease system","uniprotId":"MPPA_ECOLI"},"6882107":{"id":6882107,"externalId":"511145.b1330","name":"ynaI","hasFamily":true,"annotation":"MscS family inner membrane protein","uniprotId":"YNAI_ECOLI"},"6882108":{"id":6882108,"externalId":"511145.b1331","name":"b1331","hasFamily":true,"annotation":"IS5 transposase and trans-activator"},"6882109":{"id":6882109,"externalId":"511145.b1332","name":"ynaJ","hasFamily":true,"annotation":"putative inner membrane protein, DUF2534 family","uniprotId":"YNAJ_ECOLI"},"6882110":{"id":6882110,"externalId":"511145.b1333","name":"uspE","hasFamily":true,"annotation":"stress-induced protein; Required for resistance to DNA-damaging agents","uniprotId":"USPE_ECOLI"},"6882111":{"id":6882111,"externalId":"511145.b1334","name":"fnr","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator, global regulator of anaerobic growth; Global transcription factor that controls the expression of over 100 target genes in response to anoxia. It facilitates the adaptation to anaerobic growth conditions by regulating the expression of gene products that are involved in anaerobic energy metabolism. When the terminal electron acceptor, O(2), is no longer available, it represses the synthesis of enzymes involved in aerobic respiration and increases the synthesis of enzymes required for anaerobic respiration","uniprotId":"FNR_ECOLI"},"6882112":{"id":6882112,"externalId":"511145.b1335","name":"ogt","hasFamily":true,"annotation":"O-6-alkylguanine-DNA:cysteine-protein methyltransferase; Involved in the cellular defense against the biological effects of O6-methylguanine (O6-MeG) in DNA. Repairs alkylated guanine in DNA by stoichiometrically transferring the alkyl group at the O-6 position to a cysteine residue in the enzyme. This is a suicide reaction: the enzyme is irreversibly inactivated. Can also repair O-4-methylthymine","uniprotId":"OGT_ECOLI"},"6882113":{"id":6882113,"externalId":"511145.b1336","name":"abgT","hasFamily":true,"annotation":"p-aminobenzoyl-glutamate transporter; membrane protein; Essential for aminobenzoyl-glutamate utilization. It catalyzes the concentration-dependent uptake of p-aminobenzoyl- glutamate (PABA-GLU) into the cell and allows accumulation of PABA-GLU to a concentration enabling AbgAB to catalyze cleavage into p-aminobenzoate and glutamate. It seems also to increase the sensitivity to low levels of aminobenzoyl-glutamate. May actually serve physiologically as a transporter for some other molecule, perhaps a dipeptide, and that it transports p-aminobenzoyl- glutamate as a secondary activity. Th [...] ","uniprotId":"ABGT_ECOLI"},"6882114":{"id":6882114,"externalId":"511145.b1337","name":"abgB","hasFamily":true,"annotation":"p-aminobenzoyl-glutamate hydrolase, B subunit; Component of the p-aminobenzoyl-glutamate hydrolase multicomponent enzyme system which catalyzes the cleavage of p- aminobenzoyl-glutamate (PABA-GLU) to form p-aminobenzoate (PABA) and glutamate. AbgAB does not degrade dipeptides and the physiological role of abgABT should be clarified","uniprotId":"ABGB_ECOLI"},"6882115":{"id":6882115,"externalId":"511145.b1338","name":"abgA","hasFamily":true,"annotation":"p-aminobenzoyl-glutamate hydrolase, A subunit; Component of the p-aminobenzoyl-glutamate hydrolase multicomponent enzyme system which catalyzes the cleavage of p- aminobenzoyl-glutamate (PABA-GLU) to form p-aminobenzoate (PABA) and glutamate. AbgAB does not degrade dipeptides and the physiological role of abgABT should be clarified","uniprotId":"ABGA_ECOLI"},"6882116":{"id":6882116,"externalId":"511145.b1339","name":"abgR","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator of abgABT operon; Could be the regulator of the abg operon","uniprotId":"ABGR_ECOLI"},"6882117":{"id":6882117,"externalId":"511145.b1340","name":"smrA","hasFamily":true,"annotation":"DNA endonuclease; Has DNA endonuclease activity. Binds DNA","uniprotId":"SMRA_ECOLI"},"6882118":{"id":6882118,"externalId":"511145.b1341","name":"ydaM","hasFamily":true,"annotation":"diguanylate cyclase, csgD regulator; Part of a 2 protein system that seems to be dedicated to regulate transcription of csgD. The CsdG protein in turn regulates expression of adhesive curli fimbriae genes csgEFG, csgBAC/ymaD and adrA (yaic). Activity of this protein is antagonized by the phosphodiesterase Gmr (YciR). In vitro has weak diguanylate cyclase activity. Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"YDAM_ECOLI"},"6882119":{"id":6882119,"externalId":"511145.b1342","name":"ydaN","hasFamily":true,"annotation":"putative Zn(II) transporter; Mediates efflux of zinc ions (By similarity)","uniprotId":"ZNTB_ECOLI"},"6882120":{"id":6882120,"externalId":"511145.b1343","name":"dbpA","hasFamily":true,"annotation":"ATP-dependent RNA helicase, specific for 23S rRNA; Has a helix-destabilizing activity, which is not coupled to the ATPase activity. Can unwind the 23S rRNA as well as 16S rRNA. Exhibits an RNA-dependent ATPase activity, specifically stimulated by bacterial 23S rRNA. Could play a major role in ribosome assembly, specifically in the assembly process of the active center of 50S ribosomal subunits","uniprotId":"DBPA_ECOLI"},"6882121":{"id":6882121,"externalId":"511145.b1344","name":"ttcA","hasFamily":true,"annotation":"tRNA 2-thiocytidine biosynthesis protein; Required for the thiolation of cytidine in position 32 of tRNA, to form 2-thiocytidine (s(2)C32) (By similarity)","uniprotId":"TTCA_ECOLI"},"6882122":{"id":6882122,"externalId":"511145.b1345","name":"intR","hasFamily":true,"annotation":"Rac prophage; integrase; Integrase is necessary for integration of the phage into the host genome by site-specific recombination. In conjunction with excisionase, integrase is also necessary for excision of the prophage from the host genome (By similarity)","uniprotId":"INTR_ECOLI"},"6882123":{"id":6882123,"externalId":"511145.b1346","name":"ydaQ","hasFamily":false,"annotation":"Rac prophage","uniprotId":"YDAQ_ECOLI"},"6882124":{"id":6882124,"externalId":"511145.b1347","name":"rcbA","hasFamily":false,"annotation":"Rac prophage; predicted protein; Helps to maintain the integrity of the chromosome by lowering the steady-state level of double strand breaks","uniprotId":"RCBA_ECOLI"},"6882125":{"id":6882125,"externalId":"511145.b1348","name":"ralR","hasFamily":false,"annotation":"Rac prophage; Has a RAL-like activity","uniprotId":"RALR_ECOLI"},"6882126":{"id":6882126,"externalId":"511145.b1349","name":"recT","hasFamily":true,"annotation":"Rac prophage; recombination and repair protein; Binds to single-stranded DNA and also promotes the renaturation of complementary single-stranded DNA. Function in recombination. Has a function similar to that of lambda RedB","uniprotId":"RECT_ECOLI"},"6882127":{"id":6882127,"externalId":"511145.b1350","name":"recE","hasFamily":true,"annotation":"Rac prophage; exonuclease VIII, 5' -> 3' specific dsDNA exonuclease; Is involved in the RecE pathway of recombination. Has a strong preference for linear duplex substrate DNA and appears to be unable to initiate degradation from single-stranded breaks in DNA","uniprotId":"RECE_ECOLI"},"6882128":{"id":6882128,"externalId":"511145.b1351","name":"racC","hasFamily":false,"annotation":"Rac prophage","uniprotId":"RACC_ECOLI"},"6882129":{"id":6882129,"externalId":"511145.b1352","name":"kilR","hasFamily":false,"annotation":"Rac prophage; Causes inhibition of cell division. At high levels of expression, can also abolish the rod shape of the cells. Division inhibition by KilR can be relieved by overexpression of the cell division protein FtsZ","uniprotId":"KILR_ECOLI"},"6882130":{"id":6882130,"externalId":"511145.b1353","name":"sieB","hasFamily":false,"annotation":"Rac prophage","uniprotId":"SIEB_ECOLI"},"6882131":{"id":6882131,"externalId":"511145.b1355","name":"ydaG","hasFamily":false,"annotation":"Rac prophage","uniprotId":"YDAG_ECOLI"},"6882132":{"id":6882132,"externalId":"511145.b1356","name":"racR","hasFamily":false,"annotation":"Rac prophage; Repressor protein for rac prophage","uniprotId":"RACR_ECOLI"},"6882133":{"id":6882133,"externalId":"511145.b1357","name":"ydaS","hasFamily":false,"annotation":"Rac prophage","uniprotId":"YDAS_ECOLI"},"6882134":{"id":6882134,"externalId":"511145.b1358","name":"ydaT","hasFamily":false,"annotation":"Rac prophage","uniprotId":"YDAT_ECOLI"},"6882135":{"id":6882135,"externalId":"511145.b1359","name":"ydaU","hasFamily":true,"annotation":"Rac prophage","uniprotId":"YDAU_ECOLI"},"6882136":{"id":6882136,"externalId":"511145.b1360","name":"ydaV","hasFamily":true,"annotation":"Rac prophage; predicted DNA replication protein","uniprotId":"YDAV_ECOLI"},"6882137":{"id":6882137,"externalId":"511145.b1363","name":"trkG","hasFamily":true,"annotation":"Rac prophage; Low-affinity potassium transport system. Interacts with Trk system potassium uptake protein TrkA and requires TrkE for transport activity. TrkE increases the affinity of the system","uniprotId":"TRKG_ECOLI"},"6882138":{"id":6882138,"externalId":"511145.b1365","name":"ynaK","hasFamily":false,"annotation":"Rac prophage","uniprotId":"YNAK_ECOLI"},"6882139":{"id":6882139,"externalId":"511145.b1370","name":"insH5","hasFamily":true,"annotation":"IS5 transposase and trans-activator; Involved in the transposition of the insertion sequence IS5","uniprotId":"INSH5_ECOLI"},"6882140":{"id":6882140,"externalId":"511145.b1372","name":"stfR","hasFamily":true,"annotation":"Rac prophage","uniprotId":"STFR_ECOLI"},"6882141":{"id":6882141,"externalId":"511145.b1373","name":"tfaR","hasFamily":true,"annotation":"Rac prophage","uniprotId":"TFAR_ECOLI"},"6882142":{"id":6882142,"externalId":"511145.b1374","name":"pinR","hasFamily":true,"annotation":"Rac prophage; predicted site-specific recombinase","uniprotId":"PINR_ECOLI"},"6882143":{"id":6882143,"externalId":"511145.b1375","name":"ynaE","hasFamily":false,"annotation":"cold shock protein, function unknown, Rac prophage","uniprotId":"YNAE_ECOLI"},"6882144":{"id":6882144,"externalId":"511145.b1376","name":"uspF","hasFamily":true,"annotation":"stress-induced protein, ATP-binding protein","uniprotId":"USPF_ECOLI"},"6882145":{"id":6882145,"externalId":"511145.b1377","name":"ompN","hasFamily":true,"annotation":"outer membrane pore protein N, non-specific; Forms pores that allow passive diffusion of small molecules across the outer membrane (By similarity). Non-specific porin","uniprotId":"OMPN_ECOLI"},"6882146":{"id":6882146,"externalId":"511145.b1378","name":"ydbK","hasFamily":true,"annotation":"fused predicted pyruvate-flavodoxin oxidoreductase: conserved protein/conserved protein/FeS binding protein; Oxidoreductase required for the transfer of electrons from pyruvate to flavodoxin (By similarity)","uniprotId":"NIFJ_ECOLI"},"6882147":{"id":6882147,"externalId":"511145.b1379","name":"hslJ","hasFamily":true,"annotation":"heat-inducible lipoprotein involved in novobiocin resistance","uniprotId":"HSLJ_ECOLI"},"6882148":{"id":6882148,"externalId":"511145.b1380","name":"ldhA","hasFamily":true,"annotation":"fermentative D-lactate dehydrogenase, NAD-dependent; Fermentative lactate dehydrogenase","uniprotId":"LDHD_ECOLI"},"6882149":{"id":6882149,"externalId":"511145.b1381","name":"ydbH","hasFamily":true,"annotation":"predicted protein","uniprotId":"YDBH_ECOLI"},"6882150":{"id":6882150,"externalId":"511145.b1382","name":"ynbE","hasFamily":true,"annotation":"lipoprotein","uniprotId":"YNBE_ECOLI"},"6882151":{"id":6882151,"externalId":"511145.b1383","name":"ydbL","hasFamily":true,"annotation":"conserved protein","uniprotId":"YDBL_ECOLI"},"6882152":{"id":6882152,"externalId":"511145.b1384","name":"feaR","hasFamily":true,"annotation":"DNA-binding transcriptional activator for tynA and feaB; Positive regulator of tynA/maoA and feaB/padA, the genes for 2-phenylethylamine catabolism","uniprotId":"FEAR_ECOLI"},"6882153":{"id":6882153,"externalId":"511145.b1385","name":"feaB","hasFamily":true,"annotation":"phenylacetaldehyde dehydrogenase; Acts almost equally well on phenylacetaldehyde, 4- hydroxyphenylacetaldehyde and 3,4-dihydroxyphenylacetaldehyde","uniprotId":"FEAB_ECOLI"},"6882154":{"id":6882154,"externalId":"511145.b1386","name":"tynA","hasFamily":true,"annotation":"tyramine oxidase, copper-requiring; The enzyme prefers aromatic over aliphatic amines","uniprotId":"AMO_ECOLI"},"6882155":{"id":6882155,"externalId":"511145.b1387","name":"maoC","hasFamily":true,"annotation":"fused oxepin-CoA hydrolase/3-oxo-5,6-dehydrosuberyl-CoA semialdehyde dehydrogenase; Catalyzes the hydrolytic ring cleavage of 2-oxepin- 2(3H)-ylideneacetyl-CoA (oxepin-CoA) via the open-chain aldehyde intermediate to yield 3-oxo-5,6-dehydrosuberyl-CoA. The enzyme consists of a C-terminal (R)-specific enoyl-CoA hydratase domain (formerly MaoC) that cleaves the ring and produces the highly reactive 3-oxo-5,6-dehydrosuberyl-CoA semialdehyde and an N- terminal NADP-dependent aldehyde dehydrogenase domain that oxidizes the aldehyde to 3-oxo-5,6-dehydrosuberyl-CoA. Can also use crotonyl-CoA  [...] ","uniprotId":"PAAZ_ECOLI"},"6882156":{"id":6882156,"externalId":"511145.b1388","name":"paaA","hasFamily":true,"annotation":"ring 1,2-phenylacetyl-CoA epoxidase subunit; Component of 1,2-phenylacetyl-CoA epoxidase multicomponent enzyme system which catalyzes the reduction of phenylacetyl-CoA (PA-CoA) to form 1,2-epoxyphenylacetyl-CoA. The subunit A is the catalytic subunit involved in the incorporation of one atom of molecular oxygen into phenylacetyl-CoA","uniprotId":"PAAA_ECOLI"},"6882157":{"id":6882157,"externalId":"511145.b1389","name":"paaB","hasFamily":true,"annotation":"ring 1,2-phenylacetyl-CoA epoxidase possible subunit, not required for in vitro activity; Component of 1,2-phenylacetyl-CoA epoxidase multicomponent enzyme system which catalyzes the reduction of phenylacetyl-CoA (PA-CoA) to form 1,2-epoxyphenylacetyl-CoA. The subunit B may play a regulatory role or be directly involved in electron transport","uniprotId":"PAAB_ECOLI"},"6882158":{"id":6882158,"externalId":"511145.b1390","name":"paaC","hasFamily":true,"annotation":"ring 1,2-phenylacetyl-CoA epoxidase subunit; Component of 1,2-phenylacetyl-CoA epoxidase multicomponent enzyme system which catalyzes the reduction of phenylacetyl-CoA (PA-CoA) to form 1,2-epoxyphenylacetyl-CoA. The subunit C may be essential for structural integrity of the alpha subunit","uniprotId":"PAAC_ECOLI"},"6882159":{"id":6882159,"externalId":"511145.b1391","name":"paaD","hasFamily":true,"annotation":"ring 1,2-phenylacetyl-CoA epoxidase subunit; Possible component of 1,2-phenylacetyl-CoA epoxidase multicomponent enzyme system which catalyzes the reduction of phenylacetyl-CoA (PA-CoA) to form 1,2-epoxyphenylacetyl-CoA. The subunit D may have a function related to the maturation of the monooxygenase complex, rather than direct involvement in catalysis. PaaD could assist either in maturation of PaaE or PaaA","uniprotId":"PAAD_ECOLI"},"6882160":{"id":6882160,"externalId":"511145.b1392","name":"paaE","hasFamily":true,"annotation":"ring 1,2-phenylacetyl-CoA epoxidase, NAD(P)H oxidoreductase component; Component of 1,2-phenylacetyl-CoA epoxidase multicomponent enzyme system which catalyzes the reduction of phenylacetyl-CoA (PA-CoA) to form 1,2-epoxyphenylacetyl-CoA. The subunit E is a reductase with a preference for NADPH and FAD, capable of reducing cytochrome c","uniprotId":"PAAE_ECOLI"},"6882161":{"id":6882161,"externalId":"511145.b1393","name":"paaF","hasFamily":true,"annotation":"2,3-dehydroadipyl-CoA hydratase; Catalyzes the reversible conversion of enzymatically produced 2,3-dehydroadipyl-CoA into 3-hydroxyadipyl-CoA","uniprotId":"PAAF_ECOLI"},"6882162":{"id":6882162,"externalId":"511145.b1394","name":"paaG","hasFamily":true,"annotation":"1,2-epoxyphenylacetyl-CoA isomerase, oxepin-CoA-forming; Catalyzes the reversible conversion of the epoxide to 2- oxepin-2(3H)-ylideneacetyl-CoA (oxepin-CoA)","uniprotId":"PAAG_ECOLI"},"6882163":{"id":6882163,"externalId":"511145.b1395","name":"paaH","hasFamily":true,"annotation":"3-hydroxyadipyl-CoA dehydrogenase, NAD+-dependent; Catalyzes the oxidation of 3-hydroxyadipyl-CoA to yield 3-oxoadipyl-CoA","uniprotId":"PAAH_ECOLI"},"6882164":{"id":6882164,"externalId":"511145.b1396","name":"paaI","hasFamily":true,"annotation":"thioesterase, most active with ring-hydroxylated phenylacetyl-coenzyme A thioesters; Thioesterase with a preference for ring-hydroxylated phenylacetyl-CoA esters. Hydrolyzes 3,4-dihydroxyphenylacetyl-CoA, 3-hydroxyphenylacetyl-CoA and 4-hydroxyphenylacetyl-CoA. Inactive towards 4-hydroxybenzoyl-CoA and 4-hydroxyphenacyl-CoA","uniprotId":"PAAI_ECOLI"},"6882165":{"id":6882165,"externalId":"511145.b1397","name":"paaJ","hasFamily":true,"annotation":"3-oxoadipyl-CoA/3-oxo-5,6-dehydrosuberyl-CoA thiolase; Catalyzes the thiolytic cleavage of the beta-keto C8 intermediate 3-oxo-5,6-dehydrosuberyl-CoA with CoA to yield the C6 intermediate 2,3-dehydroadipyl-CoA and acetyl-CoA. Besides it catalyzes also the last step of the pathway, in which 3-oxoadipyl- CoA similarly is cleaved to acetyl-CoA and succinyl-CoA","uniprotId":"PAAJ_ECOLI"},"6882166":{"id":6882166,"externalId":"511145.b1398","name":"paaK","hasFamily":true,"annotation":"phenylacetyl-CoA ligase; Catalyzes the activation of phenylacetic acid (PA) to phenylacetyl-CoA (PA-CoA)","uniprotId":"PAAK_ECOLI"},"6882167":{"id":6882167,"externalId":"511145.b1399","name":"paaX","hasFamily":false,"annotation":"DNA-binding transcriptional repressor of phenylacetic acid degradation paa operon, phenylacetyl-CoA inducer; Negative regulator of the paaZ and paaABCDEFGHIJK catabolic operons. Binds the consensus sequence 5'- WWTRTGATTCGYGWT-3'. Binding of PaaX is specifically inhibited by phenylacetyl-coenzyme A (PA-CoA)","uniprotId":"PAAX_ECOLI"},"6882168":{"id":6882168,"externalId":"511145.b1400","name":"paaY","hasFamily":true,"annotation":"putative hexapeptide repeat acetyltransferase","uniprotId":"PAAY_ECOLI"},"6882169":{"id":6882169,"externalId":"511145.b1402","name":"b1402","hasFamily":true,"annotation":"IS2 transposase TnpB"},"6882170":{"id":6882170,"externalId":"511145.b1403","name":"b1403","hasFamily":true,"annotation":"IS2 repressor TnpA"},"6882171":{"id":6882171,"externalId":"511145.b1404","name":"b1404","hasFamily":true,"annotation":"IS30 transposase"},"6882172":{"id":6882172,"externalId":"511145.b1406","name":"ydbC","hasFamily":true,"annotation":"putative oxidoreductase, NAD(P)-binding","uniprotId":"YDBC_ECOLI"},"6882173":{"id":6882173,"externalId":"511145.b1407","name":"ydbD","hasFamily":false,"annotation":"putative PF10971 family periplasmic methylglyoxal resistance protein","uniprotId":"YDBD_ECOLI"},"6882174":{"id":6882174,"externalId":"511145.b1408","name":"ynbA","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YNBA_ECOLI"},"6882175":{"id":6882175,"externalId":"511145.b1409","name":"ynbB","hasFamily":true,"annotation":"putative CDP-diglyceride synthase","uniprotId":"YNBB_ECOLI"},"6882176":{"id":6882176,"externalId":"511145.b1410","name":"ynbC","hasFamily":true,"annotation":"putative hydrolase","uniprotId":"YNBC_ECOLI"},"6882177":{"id":6882177,"externalId":"511145.b1411","name":"ynbD","hasFamily":true,"annotation":"putative phosphatase inner membrane protein","uniprotId":"YNBD_ECOLI"},"6882178":{"id":6882178,"externalId":"511145.b1412","name":"azoR","hasFamily":true,"annotation":"NADH-azoreductase, FMN-dependent; Catalyzes the reductive cleavage of azo bond in aromatic azo compounds to the corresponding amines. Requires NADH, but not NADPH, as an electron donor for its activity. The enzyme can reduce ethyl red and methyl red, but is not able to convert sulfonated azo dyes","uniprotId":"AZOR_ECOLI"},"6882179":{"id":6882179,"externalId":"511145.b1413","name":"hrpA","hasFamily":true,"annotation":"putative ATP-dependent helicase; Not yet known","uniprotId":"HRPA_ECOLI"},"6882180":{"id":6882180,"externalId":"511145.b1414","name":"ydcF","hasFamily":true,"annotation":"conserved SAM-binding protein, DUF218 family; Binds S-adenosyl-L-methionine (AdoMet)","uniprotId":"YDCF_ECOLI"},"6882181":{"id":6882181,"externalId":"511145.b1415","name":"aldA","hasFamily":true,"annotation":"aldehyde dehydrogenase A, NAD-linked; Acts on lactaldehyde as well as other aldehydes","uniprotId":"ALDA_ECOLI"},"6882182":{"id":6882182,"externalId":"511145.b1418","name":"cybB","hasFamily":true,"annotation":"cytochrome b561; B-type di-heme cytochrome with a major alpha-absorption peak at 561 nm and a minor peak at 555 nm","uniprotId":"C561_ECOLI"},"6882183":{"id":6882183,"externalId":"511145.b1419","name":"ydcA","hasFamily":false,"annotation":"predicted protein","uniprotId":"YDCA_ECOLI"},"6882184":{"id":6882184,"externalId":"511145.b1420","name":"mokB","hasFamily":false,"annotation":"regulatory peptide; Overlapping regulatory peptide whose translation enables hokB expression (Probable)","uniprotId":"MOKB_ECOLI"},"6882185":{"id":6882185,"externalId":"511145.b1421","name":"trg","hasFamily":true,"annotation":"methyl-accepting chemotaxis protein III, ribose and galactose sensor receptor; Mediates taxis to the sugars ribose and galactose via an interaction with the periplasmic ribose- or galactose-binding proteins","uniprotId":"MCP3_ECOLI"},"6882186":{"id":6882186,"externalId":"511145.b1422","name":"ydcI","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YDCI_ECOLI"},"6882187":{"id":6882187,"externalId":"511145.b1423","name":"ydcJ","hasFamily":true,"annotation":"putative metalloenzyme","uniprotId":"YDCJ_ECOLI"},"6882188":{"id":6882188,"externalId":"511145.b1424","name":"mdoD","hasFamily":true,"annotation":"osmoregulated periplasmic glucan (OPG) biosynthesis periplasmic protein; Probably involved in the control of the structural glucose backbone of osmoregulated periplasmic glucans (OPGs) (By similarity)","uniprotId":"OPGD_ECOLI"},"6882189":{"id":6882189,"externalId":"511145.b1426","name":"ydcH","hasFamily":true,"annotation":"predicted protein","uniprotId":"YDCH_ECOLI"},"6882190":{"id":6882190,"externalId":"511145.b1427","name":"rimL","hasFamily":true,"annotation":"ribosomal-protein-L7/L12-serine acetyltransferase; This enzyme acetylates the N-terminal serine of ribosomal protein L7/L12","uniprotId":"RIML_ECOLI"},"6882191":{"id":6882191,"externalId":"511145.b1428","name":"ydcK","hasFamily":true,"annotation":"putative enzyme","uniprotId":"YDCK_ECOLI"},"6882192":{"id":6882192,"externalId":"511145.b1429","name":"tehA","hasFamily":true,"annotation":"potassium-tellurite ethidium and proflavin transporter; Responsible for potassium tellurite resistance when present in high copy number. Ion channel involved in potassium tellurite resistance (By similarity). Otherwise, phenotypically silent","uniprotId":"TEHA_ECOLI"},"6882193":{"id":6882193,"externalId":"511145.b1430","name":"tehB","hasFamily":true,"annotation":"tellurite, selenium methyltransferase, SAM-dependent; tellurite, selenium resistance protein; S-adenosyl-L-methionine dependent methyltransferase that catalyzes the methylation of tellurite and is responsible for tellurite resistance when present in high copy number. Can also methylate selenite and selenium dioxide. Is thus able to detoxify different chalcogens. Can not methylate arsenic compounds","uniprotId":"TEHB_ECOLI"},"6882194":{"id":6882194,"externalId":"511145.b1431","name":"ydcL","hasFamily":true,"annotation":"lipoprotein","uniprotId":"YDCL_ECOLI"},"6882195":{"id":6882195,"externalId":"511145.b1432","name":"insQ","hasFamily":true,"annotation":"IS609 transposase B; Required for the transposition of the insertion element (Potential)","uniprotId":"INSQ_ECOLI"},"6882196":{"id":6882196,"externalId":"511145.b1433","name":"ydcO","hasFamily":true,"annotation":"inner membrane protein, predicted transporter, function unknown","uniprotId":"YDCO_ECOLI"},"6882197":{"id":6882197,"externalId":"511145.b1434","name":"ydcN","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YDCN_ECOLI"},"6882198":{"id":6882198,"externalId":"511145.b1435","name":"ydcP","hasFamily":true,"annotation":"putative peptidase","uniprotId":"YDCP_ECOLI"},"6882199":{"id":6882199,"externalId":"511145.b1436","name":"yncJ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YNCJ_ECOLI"},"6882200":{"id":6882200,"externalId":"511145.b1438","name":"hicB","hasFamily":true,"annotation":"antitoxin for the HicAB toxin-antitoxin system; Antitoxin component of a toxin-antitoxin (TA) module. Functions as an mRNA interferase antitoxin; overexpression prevents HicA-mediated cessation of cell growth and inhibition of cell proliferation","uniprotId":"HICB_ECOLI"},"6882201":{"id":6882201,"externalId":"511145.b1439","name":"ydcR","hasFamily":true,"annotation":"fused predicted DNA-binding transcriptional regulator/predicted amino transferase","uniprotId":"YDCR_ECOLI"},"6882202":{"id":6882202,"externalId":"511145.b1440","name":"ydcS","hasFamily":false,"annotation":"polyhydroxybutyrate (PHB) synthase, ABC transporter periplasmic binding protein homolog; Probably part of the binding-protein-dependent transport system YdcSTUV","uniprotId":"YDCS_ECOLI"},"6882203":{"id":6882203,"externalId":"511145.b1441","name":"ydcT","hasFamily":true,"annotation":"putative spermidine/putrescine transporter subunit; Probably part of a binding-protein-dependent transport system YdcSTUV. Probably responsible for energy coupling to the transport system","uniprotId":"YDCT_ECOLI"},"6882204":{"id":6882204,"externalId":"511145.b1442","name":"ydcU","hasFamily":false,"annotation":"putative spermidine/putrescine transporter subunit; Probably part of the binding-protein-dependent transport system YdcSTUV; probably responsible for the translocation of the substrate across the membrane","uniprotId":"YDCU_ECOLI"},"6882205":{"id":6882205,"externalId":"511145.b1443","name":"ydcV","hasFamily":false,"annotation":"putative spermidine/putrescine transporter subunit; Probably part of the binding-protein-dependent transport system YdcSTUV; probably responsible for the translocation of the substrate across the membrane","uniprotId":"YDCV_ECOLI"},"6882206":{"id":6882206,"externalId":"511145.b1444","name":"prr","hasFamily":true,"annotation":"gamma-aminobutyraldehyde dehydrogenase; Catalyzes the oxidation of 1-pyrroline, which is spontaneously formed from 4-aminobutanal, leading to 4- aminobutanoate (GABA). Can also oxidize n-alkyl medium-chain aldehydes, but with a lower catalytic efficiency","uniprotId":"ABDH_ECOLI"},"6882207":{"id":6882207,"externalId":"511145.b1445","name":"ydcX","hasFamily":false,"annotation":"conserved protein, DUF2566 family","uniprotId":"YDCX_ECOLI"},"6882208":{"id":6882208,"externalId":"511145.b1446","name":"ydcY","hasFamily":true,"annotation":"predicted protein","uniprotId":"YDCY_ECOLI"},"6882209":{"id":6882209,"externalId":"511145.b1447","name":"ydcZ","hasFamily":true,"annotation":"inner membrane protein, DUF606 family","uniprotId":"YDCZ_ECOLI"},"6882210":{"id":6882210,"externalId":"511145.b1448","name":"yncA","hasFamily":true,"annotation":"methionine N-acyltransferase; L-amino acid N-acyltransferase; N-acylates some free L-amino acids and their derivatives using a CoA thioester as cofactor. Using acetyl-CoA as a cofactor, substrate specificity is methionine sulfone > methionine sulfoximine > methionine sulfoxide > methionine. Asparagine, lysine, glutamine, aspartate and glutamate are very poor substrates. Using methionine as a substrate, cofactor preference is propionyl-CoA > acetyl-CoA >> butyryl-CoA","uniprotId":"YNCA_ECOLI"},"6882211":{"id":6882211,"externalId":"511145.b1449","name":"curA","hasFamily":true,"annotation":"curcumin/dihydrocurcumin reductase, NADPH-dependent; Catalyzes the metal-independent reduction of curcumin to dihydrocurcumin (DHC) as an intermediate product, followed by further reduction to tetrahydrocurcumin (THC) as an end product. It also acts on 3-octene-2-one, 3-hepten-2-one, resveratrol, and trans-2-octenal","uniprotId":"CURA_ECOLI"},"6882212":{"id":6882212,"externalId":"511145.b1450","name":"mcbR","hasFamily":true,"annotation":"colanic acid and biofilm gene transcriptional regulator, MqsR-controlled; Important for biofilm formation. Represses expression of McbA by binding to its promoter region, which prevents colanic acid overproduction and mucoidy","uniprotId":"MCBR_ECOLI"},"6882213":{"id":6882213,"externalId":"511145.b1451","name":"yncD","hasFamily":true,"annotation":"putative iron outer membrane transporter; Probable receptor, TonB-dependent","uniprotId":"YNCD_ECOLI"},"6882214":{"id":6882214,"externalId":"511145.b1452","name":"yncE","hasFamily":true,"annotation":"ATP-binding protein, periplasmic, function unknown","uniprotId":"YNCE_ECOLI"},"6882215":{"id":6882215,"externalId":"511145.b1453","name":"ansP","hasFamily":true,"annotation":"L-asparagine transporter","uniprotId":"ANSP_ECOLI"},"6882216":{"id":6882216,"externalId":"511145.b1454","name":"yncG","hasFamily":false,"annotation":"glutathione S-transferase homolog","uniprotId":"YNCG_ECOLI"},"6882217":{"id":6882217,"externalId":"511145.b1455","name":"yncH","hasFamily":false,"annotation":"conserved protein","uniprotId":"YNCH_ECOLI"},"6882218":{"id":6882218,"externalId":"511145.b1457","name":"ydcD","hasFamily":false,"annotation":"predicted protein","uniprotId":"YDCD_ECOLI"},"6882219":{"id":6882219,"externalId":"511145.b1460","name":"ydcC","hasFamily":true,"annotation":"conserved protein","uniprotId":"YDCC_ECOLI"},"6882220":{"id":6882220,"externalId":"511145.b1461","name":"pptA","hasFamily":false,"annotation":"4-oxalocrotonate tautomerase; Can use enol isomers of phenylpyruvate, 2-hydroxy-2,4- pentadienoate and (p-hydroxyphenyl)pyruvate as substrates","uniprotId":"PPTA_ECOLI"},"6882221":{"id":6882221,"externalId":"511145.b1462","name":"yddH","hasFamily":true,"annotation":"conserved protein","uniprotId":"YDDH_ECOLI"},"6882222":{"id":6882222,"externalId":"511145.b1463","name":"nhoA","hasFamily":true,"annotation":"N-hydroxyarylamine O-acetyltransferase","uniprotId":"NHOA_ECOLI"},"6882223":{"id":6882223,"externalId":"511145.b1464","name":"yddE","hasFamily":true,"annotation":"conserved predicted enzyme, PhzC-PhzF family","uniprotId":"YDDE_ECOLI"},"6882224":{"id":6882224,"externalId":"511145.b1465","name":"narV","hasFamily":true,"annotation":"nitrate reductase 2 (NRZ), gamma subunit; This is a second nitrate reductase enzyme which can substitute for the NRA enzyme and allows E.coli to use nitrate as an electron acceptor during anaerobic growth. The gamma chain is a membrane-embedded heme-iron unit resembling cytochrome b, which transfers electrons from quinones to the beta subunit","uniprotId":"NARV_ECOLI"},"6882225":{"id":6882225,"externalId":"511145.b1466","name":"narW","hasFamily":true,"annotation":"nitrate reductase 2 (NRZ), delta subunit (assembly subunit); Chaperone required for proper molybdenum cofactor insertion and final assembly of the membrane-bound respiratory nitrate reductase 2 (By similarity)","uniprotId":"NARW_ECOLI"},"6882226":{"id":6882226,"externalId":"511145.b1467","name":"narY","hasFamily":true,"annotation":"nitrate reductase 2 (NRZ), beta subunit; This is a second nitrate reductase enzyme which can substitute for the NRA enzyme and allows E.coli to use nitrate as an electron acceptor during anaerobic growth. The beta chain is an electron transfer unit containing four cysteine clusters involved in the formation of iron-sulfur centers. Electrons are transferred from the gamma chain to the molybdenum cofactor of the alpha subunit","uniprotId":"NARY_ECOLI"},"6882227":{"id":6882227,"externalId":"511145.b1468","name":"narZ","hasFamily":true,"annotation":"nitrate reductase 2 (NRZ), alpha subunit; This is a second nitrate reductase enzyme which can substitute for the NRA enzyme and allows E.coli to use nitrate as an electron acceptor during anaerobic growth","uniprotId":"NARZ_ECOLI"},"6882228":{"id":6882228,"externalId":"511145.b1469","name":"narU","hasFamily":true,"annotation":"nitrate/nitrite transporter; Involved in excretion of nitrite produced by the dissimilatory reduction of nitrate","uniprotId":"NARU_ECOLI"},"6882229":{"id":6882229,"externalId":"511145.b1470","name":"yddJ","hasFamily":false,"annotation":"predicted protein","uniprotId":"YDDJ_ECOLI"},"6882230":{"id":6882230,"externalId":"511145.b1473","name":"yddG","hasFamily":true,"annotation":"aromatic amino acid exporter; Probable efflux pump. Overexpression confers resistance to phenylalanine and increases export of phenylalanine, tyrosine and tryptophan","uniprotId":"YDDG_ECOLI"},"6882231":{"id":6882231,"externalId":"511145.b1474","name":"fdnG","hasFamily":true,"annotation":"formate dehydrogenase-N, alpha subunit, nitrate-inducible; Formate dehydrogenase allows E.coli to use formate as major electron donor during anaerobic respiration, when nitrate is used as electron acceptor. The alpha subunit forms the active site","uniprotId":"FDNG_ECOLI"},"6882232":{"id":6882232,"externalId":"511145.b1475","name":"fdnH","hasFamily":true,"annotation":"formate dehydrogenase-N, Fe-S (beta) subunit, nitrate-inducible; Formate dehydrogenase allows E.coli to use formate as major electron donor during anaerobic respiration, when nitrate is used as electron acceptor. The beta chain is an electron transfer unit containing 4 cysteine clusters involved in the formation of iron-sulfur centers. Electrons are transferred from the gamma chain to the molybdenum cofactor of the alpha subunit","uniprotId":"FDNH_ECOLI"},"6882233":{"id":6882233,"externalId":"511145.b1476","name":"fdnI","hasFamily":true,"annotation":"formate dehydrogenase-N, cytochrome B556 (gamma) subunit, nitrate-inducible; Formate dehydrogenase allows E.coli to use formate as major electron donor during anaerobic respiration, when nitrate is used as electron acceptor. Subunit gamma is the cytochrome b556(FDN) component of the formate dehydrogenase","uniprotId":"FDNI_ECOLI"},"6882234":{"id":6882234,"externalId":"511145.b1477","name":"yddM","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YDDM_ECOLI"},"6882235":{"id":6882235,"externalId":"511145.b1478","name":"adhP","hasFamily":true,"annotation":"ethanol-active dehydrogenase/acetaldehyde-active reductase; Preferred specificity is towards 1-propanol","uniprotId":"ADHP_ECOLI"},"6882236":{"id":6882236,"externalId":"511145.b1479","name":"maeA","hasFamily":true,"annotation":"malate dehydrogenase, (decarboxylating, NAD-requiring) (malic enzyme)","uniprotId":"MAO1_ECOLI"},"6882237":{"id":6882237,"externalId":"511145.b1480","name":"sra","hasFamily":true,"annotation":"Stationary-phase-induced ribosome-associated protein; Although this protein associates with the 30S subunit of the ribosome it is not considered to be a bona fide ribosomal protein. It is not essential for bacterial growth","uniprotId":"SRA_ECOLI"},"6882238":{"id":6882238,"externalId":"511145.b1481","name":"bdm","hasFamily":true,"annotation":"biofilm-dependent modulation protein","uniprotId":"BDM_ECOLI"},"6882239":{"id":6882239,"externalId":"511145.b1482","name":"osmC","hasFamily":true,"annotation":"lipoyl-dependent Cys-based peroxidase, hydroperoxide resistance; salt-shock inducible membrane protein; Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide","uniprotId":"OSMC_ECOLI"},"6882240":{"id":6882240,"externalId":"511145.b1483","name":"ddpF","hasFamily":false,"annotation":"D,D-dipeptide permease system, ATP-binding component; Part of the ABC transporter complex DdpABCDF, which is probably involved in D,D-dipeptide transport. Probably responsible for energy coupling to the transport system","uniprotId":"DDPF_ECOLI"},"6882241":{"id":6882241,"externalId":"511145.b1484","name":"ddpD","hasFamily":false,"annotation":"D,D-dipeptide permease system, ATP-binding component; Part of the ABC transporter complex DdpABCDF, which is probably involved in D,D-dipeptide transport. Probably responsible for energy coupling to the transport system","uniprotId":"DDPD_ECOLI"},"6882242":{"id":6882242,"externalId":"511145.b1485","name":"ddpC","hasFamily":true,"annotation":"D-ala-D-ala transporter subunit; Part of the ABC transporter complex DdpABCDF, which is probably involved in D,D-dipeptide transport. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"DDPC_ECOLI"},"6882243":{"id":6882243,"externalId":"511145.b1486","name":"ddpB","hasFamily":false,"annotation":"D-ala-D-ala transporter subunit; Part of the ABC transporter complex DdpABCDF, which is probably involved in D,D-dipeptide transport. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"DDPB_ECOLI"},"6882244":{"id":6882244,"externalId":"511145.b1487","name":"ddpA","hasFamily":true,"annotation":"D-ala-D-a la transporter subunit; Part of the ABC transporter complex DdpABCDF, which is probably involved in D,D-dipeptide transport","uniprotId":"DDPA_ECOLI"},"6882245":{"id":6882245,"externalId":"511145.b1488","name":"ddpX","hasFamily":true,"annotation":"D-ala-D-ala dipeptidase, Zn-dependent; Hydrolyzes D-Ala-D-Ala. May have a role in cell-wall turnover","uniprotId":"DDPX_ECOLI"},"6882246":{"id":6882246,"externalId":"511145.b1489","name":"dos","hasFamily":true,"annotation":"oxygen sensor, c-di-GMP phosphodiesterase, heme-regulated; cold- and stationary phase-induced bioflim regulator; Heme-based oxygen sensor protein displaying phosphodiesterase (PDE) activity toward c-di-GMP in response to oxygen availability. Involved in the modulation of intracellular c-di-GMP levels, in association with DosC which catalyzes the biosynthesis of c-di-GMP (diguanylate cyclase activity). Cyclic- di-GMP is a second messenger which controls cell surface- associated traits in bacteria. Has very poor PDE activity on cAMP (PubMed:15995192) but is not active with cGMP, bis(p-ni [...] ","uniprotId":"DOSP_ECOLI"},"6882247":{"id":6882247,"externalId":"511145.b1490","name":"dosC","hasFamily":true,"annotation":"diguanylate cyclase, cold- and stationary phase-induced oxygen-dependent biofilm regulator; Globin-coupled heme-based oxygen sensor protein displaying diguanylate cyclase (DGC) activity in response to oxygen availability. Thus, catalyzes the synthesis of cyclic diguanylate (c-di-GMP) via the condensation of 2 GTP molecules. Is involved in the modulation of intracellular c-di-GMP levels, in association with DosP which catalyzes the degradation of c-di-GMP (PDE activity). Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria. DosC regulates biofilm [...] ","uniprotId":"DOSC_ECOLI"},"6882248":{"id":6882248,"externalId":"511145.b1491","name":"yddW","hasFamily":true,"annotation":"liprotein, glycosyl hydrolase homolog","uniprotId":"YDDW_ECOLI"},"6882249":{"id":6882249,"externalId":"511145.b1492","name":"gadC","hasFamily":true,"annotation":"glutamate:gamma-aminobutyric acid antiporter; Involved in glutamate-dependent acid resistance. Imports glutamate inside the cell while simultaneously exporting to the periplasm the GABA produced by GadA and GadB. The gad system helps to maintain a near-neutral intracellular pH when cells are exposed to extremely acidic conditions. The ability to survive transit through the acidic conditions of the stomach is essential for successful colonization of the mammalian host by commensal and pathogenic bacteria","uniprotId":"GADC_ECOLI"},"6882250":{"id":6882250,"externalId":"511145.b1493","name":"gadB","hasFamily":true,"annotation":"glutamate decarboxylase B, PLP-dependent; Converts glutamate to gamma-aminobutyrate (GABA), consuming one intracellular proton in the reaction. The gad system helps to maintain a near-neutral intracellular pH when cells are exposed to extremely acidic conditions. The ability to survive transit through the acidic conditions of the stomach is essential for successful colonization of the mammalian host by commensal and pathogenic bacteria","uniprotId":"DCEB_ECOLI"},"6882251":{"id":6882251,"externalId":"511145.b1494","name":"pqqL","hasFamily":true,"annotation":"putative peptidase","uniprotId":"PQQL_ECOLI"},"6882252":{"id":6882252,"externalId":"511145.b1495","name":"yddB","hasFamily":true,"annotation":"putative porin protein","uniprotId":"YDDB_ECOLI"},"6882253":{"id":6882253,"externalId":"511145.b1496","name":"yddA","hasFamily":true,"annotation":"fused predicted multidrug transporter subunits of ABC superfamily: membrane component/ATP-binding component","uniprotId":"YDDA_ECOLI"},"6882254":{"id":6882254,"externalId":"511145.b1497","name":"ydeM","hasFamily":true,"annotation":"conserved protein","uniprotId":"YDEM_ECOLI"},"6882255":{"id":6882255,"externalId":"511145.b1498","name":"ydeN","hasFamily":true,"annotation":"conserved protein","uniprotId":"YDEN_ECOLI"},"6882256":{"id":6882256,"externalId":"511145.b1499","name":"ydeO","hasFamily":false,"annotation":"transcriptional activator for mdtEF; Induces the expression of gadE and mdtEF. Could also regulate the expression of other genes involved in acid resistance","uniprotId":"YDEO_ECOLI"},"6882257":{"id":6882257,"externalId":"511145.b1500","name":"safA","hasFamily":false,"annotation":"Two component system connector membrane protein, EvgSA to PhoQP; Connects the signal transduction between the two- component systems EvgS/EvgA and PhoQ/PhoP, by directly interacting with PhoQ and thus activating the PhoQ/PhoP system, in response to acid stress conditions","uniprotId":"SAFA_ECOLI"},"6882258":{"id":6882258,"externalId":"511145.b1501","name":"ydeP","hasFamily":true,"annotation":"putative oxidoreductase; Probably involved in acid resistance","uniprotId":"YDEP_ECOLI"},"6882259":{"id":6882259,"externalId":"511145.b1502","name":"ydeQ","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YDEQ_ECOLI"},"6882260":{"id":6882260,"externalId":"511145.b1503","name":"ydeR","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YDER_ECOLI"},"6882261":{"id":6882261,"externalId":"511145.b1504","name":"ydeS","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YDES_ECOLI"},"6882262":{"id":6882262,"externalId":"511145.b1507","name":"hipA","hasFamily":true,"annotation":"EF-Tu kinase; serine protein kinase required for persister formation; Toxic component of a toxin-antitoxin (TA) module. Autophosphorylates (Ser-150) and phosphorylates EF-Tu in vitro (on 'Thr-383'), may act on other proteins as well. The hipA7 mutation leads to increased generation of persister cells, cells that survive antibiotic treatment probably by entering into a dormant state. Wild-type cells produce persisters at a frequency of 10-6 to 10-5 whereas mutant hipA7 cells produce persisters at a frequency of 10-2. Generation of persister cells requires (p)ppGpp as cells lacking relA  [...] ","uniprotId":"HIPA_ECOLI"},"6882263":{"id":6882263,"externalId":"511145.b1508","name":"hipB","hasFamily":false,"annotation":"antitoxin of HipAB toxin-antitoxin system; Antitoxin component of a toxin-antitoxin (TA) module. Neutralizes the toxic effect of cognate toxin HipA. Binds to operator sites with the consensus sequence 5-'TATCCN(8)GGATA-3' to repress the hipBA operon promoter","uniprotId":"HIPB_ECOLI"},"6882264":{"id":6882264,"externalId":"511145.b1511","name":"lsrK","hasFamily":true,"annotation":"autoinducer-2 (AI-2) kinase; Catalyzes the phosphorylation of autoinducer 2 (AI-2) to phospho-AI-2, which subsequently inactivates the transcriptional regulator LsrR and leads to the transcription of the lsr operon. Phosphorylates the ring-open form of 4,5-dihydroxy-2,3- pentanedione (DPD), which is the precursor to all AI-2 signaling molecules, at the C5 position (By similarity). Required for the regulation of the lsr operon and many other genes","uniprotId":"LSRK_ECOLI"},"6882265":{"id":6882265,"externalId":"511145.b1512","name":"lsrR","hasFamily":true,"annotation":"lsr operon transcriptional repressor; Regulates transcription of many different genes. In the absence of autoinducer 2 (AI-2), represses transcription of the lsrACDBFG operon and its own transcription. In the presence of AI- 2, LsrR is inactivated by binding phospho-AI-2, leading to the transcription of the lsr genes","uniprotId":"LSRR_ECOLI"},"6882266":{"id":6882266,"externalId":"511145.b1513","name":"lsrA","hasFamily":true,"annotation":"Autoinducer 2 import ATP-binding protein; Part of the ABC transporter complex LsrABCD involved in autoinducer 2 (AI-2) import. Responsible for energy coupling to the transport system (Probable). This protein is essential for aerobic growth","uniprotId":"LSRA_ECOLI"},"6882267":{"id":6882267,"externalId":"511145.b1514","name":"lsrC","hasFamily":true,"annotation":"Autoinducer 2 import system permease protein; Part of the ABC transporter complex LsrABCD involved in autoinducer 2 (AI-2) import. Probably responsible for the translocation of the substrate across the membrane (Probable)","uniprotId":"LSRC_ECOLI"},"6882268":{"id":6882268,"externalId":"511145.b1515","name":"lsrD","hasFamily":true,"annotation":"autoinducer 2 import system permease protein; Part of the ABC transporter complex LsrABCD involved in autoinducer 2 (AI-2) import. Probably responsible for the translocation of the substrate across the membrane (Probable)","uniprotId":"LSRD_ECOLI"},"6882269":{"id":6882269,"externalId":"511145.b1516","name":"lsrB","hasFamily":true,"annotation":"autoinducer 2-binding protein; Part of the ABC transporter complex LsrABCD involved in autoinducer 2 (AI-2) import. Binds AI-2 and delivers it to the LsrC and LsrD permeases (Probable)","uniprotId":"LSRB_ECOLI"},"6882270":{"id":6882270,"externalId":"511145.b1517","name":"lsrF","hasFamily":true,"annotation":"putative autoinducer-2 (AI-2) aldolase; Could be involved in the degradation of phospho-AI-2, thereby terminating induction of the lsr operon and closing the AI-2 signaling cycle (By similarity)","uniprotId":"LSRF_ECOLI"},"6882271":{"id":6882271,"externalId":"511145.b1518","name":"lsrG","hasFamily":true,"annotation":"autoinducer-2 (AI-2) degrading protein LsrG; Involved in the degradation of phospho-AI-2, thereby terminating induction of the lsr operon and closing the AI-2 signaling cycle. Catalyzes the cleavage of phosphorylated 4,5- dihydroxy-2,3-pentanedione (P-DPD) to 2-phosphoglycolic acid (PG) and another three-carbon compound","uniprotId":"LSRG_ECOLI"},"6882272":{"id":6882272,"externalId":"511145.b1519","name":"tam","hasFamily":true,"annotation":"trans-aconitate methyltransferase; Catalyzes the S-adenosylmethionine monomethyl esterification of trans-aconitate at high affinity and of cis- aconitate, isocitrate, and citrate at lower velocities and affinities","uniprotId":"TAM_ECOLI"},"6882273":{"id":6882273,"externalId":"511145.b1520","name":"yneE","hasFamily":true,"annotation":"putative inner membrane protein, bestrophin family","uniprotId":"YNEE_ECOLI"},"6882274":{"id":6882274,"externalId":"511145.b1521","name":"uxaB","hasFamily":true,"annotation":"altronate oxidoreductase, NAD-dependent","uniprotId":"UXAB_ECOLI"},"6882275":{"id":6882275,"externalId":"511145.b1522","name":"yneF","hasFamily":false,"annotation":"predicted diguanylate cyclase","uniprotId":"YNEF_ECOLI"},"6882276":{"id":6882276,"externalId":"511145.b1523","name":"yneG","hasFamily":true,"annotation":"conserved protein","uniprotId":"YNEG_ECOLI"},"6882277":{"id":6882277,"externalId":"511145.b1524","name":"yneH","hasFamily":true,"annotation":"glutaminase","uniprotId":"GLSA2_ECOLI"},"6882278":{"id":6882278,"externalId":"511145.b1525","name":"sad","hasFamily":true,"annotation":"succinate semialdehyde dehydrogenase, NAD(P)+-dependent; Catalyzes the NAD(+)-dependent oxidation of succinate semialdehyde to succinate. It acts preferentially with NAD as cosubstrate but can also use NADP. Prevents the toxic accumulation of succinate semialdehyde (SSA) and plays an important role when arginine and putrescine are used as the sole nitrogen or carbon sources","uniprotId":"SAD_ECOLI"},"6882279":{"id":6882279,"externalId":"511145.b1526","name":"yneJ","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YNEJ_ECOLI"},"6882280":{"id":6882280,"externalId":"511145.b1527","name":"yneK","hasFamily":false,"annotation":"predicted protein","uniprotId":"YNEK_ECOLI"},"6882281":{"id":6882281,"externalId":"511145.b1528","name":"sotB","hasFamily":true,"annotation":"predicted arabinose transporter; Involved in the efflux of sugars. The physiological role may be the reduction of the intracellular concentration of toxic sugars or sugar metabolites. Transports L-arabinose and to a lesser extent IPTG. Seems to contribute to the control of the arabinose regulon","uniprotId":"SOTB_ECOLI"},"6882282":{"id":6882282,"externalId":"511145.b1529","name":"marC","hasFamily":true,"annotation":"inner membrane protein, UPF0056 family","uniprotId":"MARC_ECOLI"},"6882283":{"id":6882283,"externalId":"511145.b1530","name":"marR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor of multiple antibiotic resistance; Repressor of the marRAB operon which is involved in the activation of both antibiotic resistance and oxidative stress genes. Binds to the marO operator/promoter site","uniprotId":"MARR_ECOLI"},"6882284":{"id":6882284,"externalId":"511145.b1531","name":"marA","hasFamily":true,"annotation":"DNA-binding transcriptional dual activator of multiple antibiotic resistance; May be a transcriptional activator of genes involved in the multiple antibiotic resistance (Mar) phenotype. It can also activate genes such as sodA, zwf and micF","uniprotId":"MARA_ECOLI"},"6882285":{"id":6882285,"externalId":"511145.b1532","name":"marB","hasFamily":false,"annotation":"predicted protein","uniprotId":"MARB_ECOLI"},"6882286":{"id":6882286,"externalId":"511145.b1533","name":"eamA","hasFamily":true,"annotation":"cysteine and O-acetyl-L-serine efflux system; May be an export pump for cysteine and other metabolites of the cysteine pathway (such as N-acetyl-L-serine (NAS) and O- acetyl-L-serine (OAS)), and for other amino acids and their metabolites","uniprotId":"EAMA_ECOLI"},"6882287":{"id":6882287,"externalId":"511145.b1534","name":"ydeE","hasFamily":true,"annotation":"putative transporter","uniprotId":"YDEE_ECOLI"},"6882288":{"id":6882288,"externalId":"511145.b1535","name":"ydeH","hasFamily":true,"annotation":"diguanylate cyclase, required for pgaD induction; A diguanylate cyclase, overexpression leads to a strong repression of swimming; swimming returnes to normal when residues 206-207 are both mutated to Ala. Overexpression also leads to a reduction in flagellar abundance and a 20-fold increase in c-di- GMP levels in vivo. Required for aminoglycoside-mediated induction of biofilm formation, it also plays a lesser role in biofilm production in response to other classes of translation inhibitors. The c-di-GMP produced by this enzyme up-regulates poly-GlcNAc production as well as the biofilm  [...] ","uniprotId":"YDEH_ECOLI"},"6882289":{"id":6882289,"externalId":"511145.b1536","name":"ydeI","hasFamily":true,"annotation":"conserved protein","uniprotId":"YDEI_ECOLI"},"6882290":{"id":6882290,"externalId":"511145.b1537","name":"ydeJ","hasFamily":true,"annotation":"conserved protein; Does not have nicotinamide-nucleotide (NMN) amidohydrolase activity","uniprotId":"YDEJ_ECOLI"},"6882291":{"id":6882291,"externalId":"511145.b1538","name":"dcp","hasFamily":true,"annotation":"dipeptidyl carboxypeptidase II; Removes dipeptides from the C-termini of N-blocked tripeptides, tetrapeptides and larger peptides","uniprotId":"DCP_ECOLI"},"6882292":{"id":6882292,"externalId":"511145.b1539","name":"ydfG","hasFamily":true,"annotation":"malonic semialdehyde reductase, NADPH-dependent; L-allo-threonine dehydrogenase, NAD(P)-dependent; Has a broad substrate specificity and is able to catalyze the reduction of the malonic semialdehyde to 3- hydroxypropionic acid. YdfG is apparently supplementing RutE, the presumed malonic semialdehyde reductase involved in pyrimidine degradation since both are able to detoxify malonic semialdehyde. Also catalyzes the NADP-dependent oxidation of L-allo-threonine to L-2-amino-3-keto-butyrate, which is spontaneously decarboxylated into aminoacetone. Also acts on D-threonine, L-serine, D-ser [...] ","uniprotId":"YDFG_ECOLI"},"6882293":{"id":6882293,"externalId":"511145.b1540","name":"ydfH","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YDFH_ECOLI"},"6882294":{"id":6882294,"externalId":"511145.b1541","name":"ydfZ","hasFamily":true,"annotation":"selenoprotein, function unknown","uniprotId":"YDFZ_ECOLI"},"6882295":{"id":6882295,"externalId":"511145.b1542","name":"ydfI","hasFamily":true,"annotation":"putative D-mannonate oxidoreductase, NAD-dependent","uniprotId":"YDFI_ECOLI"},"6882296":{"id":6882296,"externalId":"511145.b1544","name":"ydfK","hasFamily":false,"annotation":"cold shock protein, function unknown, Qin prophage","uniprotId":"YDFK_ECOLI"},"6882297":{"id":6882297,"externalId":"511145.b1545","name":"pinQ","hasFamily":true,"annotation":"Qin prophage","uniprotId":"PINQ_ECOLI"},"6882298":{"id":6882298,"externalId":"511145.b1546","name":"tfaD","hasFamily":true,"annotation":"Qin prophage","uniprotId":"TFAD_ECOLI"},"6882299":{"id":6882299,"externalId":"511145.b1547","name":"stfQ","hasFamily":true,"annotation":"Qin prophage","uniprotId":"STFQ_ECOLI"},"6882300":{"id":6882300,"externalId":"511145.b1549","name":"ydfO","hasFamily":true,"annotation":"Qin prophage","uniprotId":"YDFO_ECOLI"},"6882301":{"id":6882301,"externalId":"511145.b1550","name":"gnsB","hasFamily":true,"annotation":"Qin prophage; Overexpression increases levels of unsaturated fatty acids and suppresses both the temperature-sensitive fabA6 mutation and cold-sensitive secG null mutation","uniprotId":"GNSB_ECOLI"},"6882302":{"id":6882302,"externalId":"511145.b1551","name":"ynfN","hasFamily":false,"annotation":"Qin prophage","uniprotId":"YNFN_ECOLI"},"6882303":{"id":6882303,"externalId":"511145.b1552","name":"cspI","hasFamily":true,"annotation":"Qin prophage; cold shock protein","uniprotId":"CSPI_ECOLI"},"6882304":{"id":6882304,"externalId":"511145.b1553","name":"ydfP","hasFamily":true,"annotation":"Rz-like protein, Qin prophage","uniprotId":"RZPQ_ECOLI"},"6882305":{"id":6882305,"externalId":"511145.b1554","name":"arrQ","hasFamily":false,"annotation":"Qin prophage; predicted lysozyme; Essential for lysis of bacterial cell wall, by showing cell wall hydrolyzing activity (By similarity)","uniprotId":"LYSQ_ECOLI"},"6882306":{"id":6882306,"externalId":"511145.b1555","name":"ydfR","hasFamily":false,"annotation":"Qin prophage","uniprotId":"YDFR_ECOLI"},"6882307":{"id":6882307,"externalId":"511145.b1556","name":"essQ","hasFamily":false,"annotation":"Qin prophage","uniprotId":"ESSQ_ECOLI"},"6882308":{"id":6882308,"externalId":"511145.b1557","name":"cspB","hasFamily":true,"annotation":"Qin prophage; cold shock protein","uniprotId":"CSPB_ECOLI"},"6882309":{"id":6882309,"externalId":"511145.b1558","name":"cspF","hasFamily":false,"annotation":"Qin prophage; cold shock protein","uniprotId":"CSPF_ECOLI"},"6882310":{"id":6882310,"externalId":"511145.b1559","name":"quuQ","hasFamily":false,"annotation":"Qin prophage; Positively regulate expression of some phage genes. Bacterial host RNA polymerase modified by antitermination proteins transcribes through termination sites that otherwise prevent expression of the regulated genes (By similarity)","uniprotId":"REQ2_ECOLI"},"6882311":{"id":6882311,"externalId":"511145.b1560","name":"ydfU","hasFamily":true,"annotation":"Qin prophage","uniprotId":"YDFU_ECOLI"},"6882312":{"id":6882312,"externalId":"511145.b1561","name":"rem","hasFamily":false,"annotation":"Qin prophage","uniprotId":"REM_ECOLI"},"6882313":{"id":6882313,"externalId":"511145.b1562","name":"hokD","hasFamily":false,"annotation":"Qin prophage; When overexpressed kills the cells from the inside by interfering with a vital function in the cell membrane","uniprotId":"HOKD_ECOLI"},"6882314":{"id":6882314,"externalId":"511145.b1563","name":"relE","hasFamily":true,"annotation":"Qin prophage; toxin of the RelE-RelB toxin-antitoxin system; Toxic component of a toxin-antitoxin (TA) module. A sequence-specific, ribosome-dependent mRNA endoribonuclease that inhibits translation during amino acid starvation (the stringent response). Acts by cleaving mRNA with high codon specificity in the ribosomal A site between positions 2 and 3. The stop codon UAG is cleaved at a fast rate while UAA and UGA are cleaved with intermediate and slow rates. mRNA cleavage can also occur in the ribosomal E site after peptide release from peptidyl-tRNA in the P site as well as on free 3 [...] ","uniprotId":"RELE_ECOLI"},"6882315":{"id":6882315,"externalId":"511145.b1564","name":"relB","hasFamily":true,"annotation":"Qin prophage; Antitoxin component of a toxin-antitoxin (TA) module. Counteracts the effect of RelE via direct protein-protein interaction, enabling the reversion of translation inhibition. Also acts as an autorepressor of relBE transcription. DNA-binding and repression is stronger when complexed with corepressor RelE. Increased transcription rate of relBE and activation of relE is consistent with a lower level of RelB in starved cells due to degradation of RelB by protease Lon","uniprotId":"RELB_ECOLI"},"6882316":{"id":6882316,"externalId":"511145.b1565","name":"ydfV","hasFamily":false,"annotation":"Qin prophage","uniprotId":"YDFV_ECOLI"},"6882317":{"id":6882317,"externalId":"511145.b1566","name":"flxA","hasFamily":false,"annotation":"Qin prophage","uniprotId":"FLXA_ECOLI"},"6882318":{"id":6882318,"externalId":"511145.b1569","name":"dicC","hasFamily":false,"annotation":"Qin prophage; This protein is a repressor of division inhibition gene dicB","uniprotId":"DICC_ECOLI"},"6882319":{"id":6882319,"externalId":"511145.b1570","name":"dicA","hasFamily":false,"annotation":"Qin prophage; This protein is a repressor of division inhibition gene dicB","uniprotId":"DICA_ECOLI"},"6882320":{"id":6882320,"externalId":"511145.b1571","name":"ydfA","hasFamily":false,"annotation":"Qin prophage","uniprotId":"YDFA_ECOLI"},"6882321":{"id":6882321,"externalId":"511145.b1572","name":"ydfB","hasFamily":false,"annotation":"Qin prophage","uniprotId":"YDFB_ECOLI"},"6882322":{"id":6882322,"externalId":"511145.b1573","name":"ydfC","hasFamily":false,"annotation":"conserved protein, Qin prophage","uniprotId":"YDFC_ECOLI"},"6882323":{"id":6882323,"externalId":"511145.b1575","name":"dicB","hasFamily":false,"annotation":"Qin prophage; Involved in cell division inhibition; this function can be repressed by DicA and DicC proteins as well as CbeA","uniprotId":"DICB_ECOLI"},"6882324":{"id":6882324,"externalId":"511145.b1576","name":"ydfD","hasFamily":false,"annotation":"Qin prophage","uniprotId":"YDFD_ECOLI"},"6882325":{"id":6882325,"externalId":"511145.b1580","name":"rspB","hasFamily":true,"annotation":"putative oxidoreductase, Zn-dependent and NAD(P)-binding; Not known; probable catabolic enzyme","uniprotId":"RSPB_ECOLI"},"6882326":{"id":6882326,"externalId":"511145.b1581","name":"rspA","hasFamily":true,"annotation":"bifunctional D-altronate/D-mannonate dehydratase; Probably involved in the degradation of homoserine lactone (HSL) or of a metabolite of HSL that signals starvation","uniprotId":"RSPA_ECOLI"},"6882327":{"id":6882327,"externalId":"511145.b1582","name":"ynfA","hasFamily":true,"annotation":"inner membrane protein, UPF0060 family","uniprotId":"YNFA_ECOLI"},"6882328":{"id":6882328,"externalId":"511145.b1583","name":"ynfB","hasFamily":true,"annotation":"predicted protein","uniprotId":"YNFB_ECOLI"},"6882329":{"id":6882329,"externalId":"511145.b1584","name":"speG","hasFamily":true,"annotation":"spermidine N1-acetyltransferase; Regulates polyamine concentration by their degradation. Acts on spermidine, forming N(1)- and N(8)-acetylspermidine","uniprotId":"ATDA_ECOLI"},"6882330":{"id":6882330,"externalId":"511145.b1585","name":"ynfC","hasFamily":true,"annotation":"lipoprotein, UPF0257 family","uniprotId":"YNFC_ECOLI"},"6882331":{"id":6882331,"externalId":"511145.b1586","name":"ynfD","hasFamily":true,"annotation":"predicted protein","uniprotId":"YNFD_ECOLI"},"6882332":{"id":6882332,"externalId":"511145.b1587","name":"ynfE","hasFamily":true,"annotation":"probable selenate reductase, periplasmic; Terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds (By similarity)","uniprotId":"YNFE_ECOLI"},"6882333":{"id":6882333,"externalId":"511145.b1588","name":"ynfF","hasFamily":true,"annotation":"S- and N-oxide reductase, A subunit, periplasmic; Terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds (By similarity)","uniprotId":"YNFF_ECOLI"},"6882334":{"id":6882334,"externalId":"511145.b1589","name":"ynfG","hasFamily":true,"annotation":"oxidoreductase, Fe-S subunit; Electron transfer subunit of the terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds","uniprotId":"YNFG_ECOLI"},"6882335":{"id":6882335,"externalId":"511145.b1590","name":"ynfH","hasFamily":true,"annotation":"oxidoreductase, membrane subunit; Terminal reductase during anaerobic growth on various sulfoxide and N-oxide compounds. The C subunit anchors the other two subunits to the membrane and stabilize the catalytic subunits (By similarity)","uniprotId":"YNFH_ECOLI"},"6882336":{"id":6882336,"externalId":"511145.b1591","name":"dmsD","hasFamily":true,"annotation":"twin-argninine leader-binding protein for DmsA and TorA; Required for biogenesis/assembly of DMSO reductase, but not for the interaction of the DmsA signal peptide with the Tat system. May be part of a chaperone cascade complex that facilitates a folding-maturation pathway for the substrate protein","uniprotId":"DMSD_ECOLI"},"6882337":{"id":6882337,"externalId":"511145.b1592","name":"clcB","hasFamily":true,"annotation":"H(+)/Cl(-) exchange transporter; Probably acts as an electrical shunt for an outwardly- directed proton pump that is linked to amino acid decarboxylation, as part of the extreme acid resistance (XAR) response (By similarity)","uniprotId":"CLCB_ECOLI"},"6882338":{"id":6882338,"externalId":"511145.b1593","name":"ynfK","hasFamily":true,"annotation":"putative dethiobiotin synthetase; Catalyzes a mechanistically unusual reaction, the ATP- dependent insertion of CO2 between the N7 and N8 nitrogen atoms of 7,8-diaminopelargonic acid (DAPA) to form an ureido ring (By similarity)","uniprotId":"BIOD2_ECOLI"},"6882339":{"id":6882339,"externalId":"511145.b1594","name":"mlc","hasFamily":true,"annotation":"global DNA-binding transcriptional repressor; autorepressor; Transcriptional repressor that regulates the expression of proteins that are part of the phosphotransferase system for sugar uptake. Regulates the expression of malT","uniprotId":"MLC_ECOLI"},"6882340":{"id":6882340,"externalId":"511145.b1595","name":"ynfL","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YNFL_ECOLI"},"6882341":{"id":6882341,"externalId":"511145.b1596","name":"ynfM","hasFamily":true,"annotation":"predicted transporter","uniprotId":"YNFM_ECOLI"},"6882342":{"id":6882342,"externalId":"511145.b1597","name":"asr","hasFamily":false,"annotation":"acid shock-inducible periplasmic protein; Required for growth and/or survival at acidic conditions (pH 4.5). Needed for the adaptation process at pH 4.5 that enables cells to survive at extremely low pH (pH 2.0)","uniprotId":"ASR_ECOLI"},"6882343":{"id":6882343,"externalId":"511145.b1598","name":"ydgD","hasFamily":true,"annotation":"putative peptidase","uniprotId":"YDGD_ECOLI"},"6882344":{"id":6882344,"externalId":"511145.b1599","name":"mdtI","hasFamily":true,"annotation":"multidrug efflux system transporter; Catalyzes the excretion of spermidine. Can also confer resistance to deoxycholate and SDS","uniprotId":"MDTI_ECOLI"},"6882345":{"id":6882345,"externalId":"511145.b1600","name":"mdtJ","hasFamily":true,"annotation":"multidrug efflux system transporter; Catalyzes the excretion of spermidine. Can also confer resistance to deoxycholate and SDS","uniprotId":"MDTJ_ECOLI"},"6882346":{"id":6882346,"externalId":"511145.b1601","name":"tqsA","hasFamily":true,"annotation":"pheromone AI-2 transporter; Controls the transport of the quorum-sensing signal AI-2 either by enhancing its secretion or inhibiting its uptake and consequently represses biofilm formation and motility and affects the global gene expression in biofilms","uniprotId":"TQSA_ECOLI"},"6882347":{"id":6882347,"externalId":"511145.b1602","name":"pntB","hasFamily":true,"annotation":"pyridine nucleotide transhydrogenase, beta subunit; The transhydrogenation between NADH and NADP is coupled to respiration and ATP hydrolysis and functions as a proton pump across the membrane (By similarity)","uniprotId":"PNTB_ECOLI"},"6882348":{"id":6882348,"externalId":"511145.b1603","name":"pntA","hasFamily":true,"annotation":"pyridine nucleotide transhydrogenase, alpha subunit; The transhydrogenation between NADH and NADP is coupled to respiration and ATP hydrolysis and functions as a proton pump across the membrane (By similarity)","uniprotId":"PNTA_ECOLI"},"6882349":{"id":6882349,"externalId":"511145.b1604","name":"ydgH","hasFamily":true,"annotation":"predicted protein","uniprotId":"YDGH_ECOLI"},"6882350":{"id":6882350,"externalId":"511145.b1605","name":"ydgI","hasFamily":true,"annotation":"putative arginine/ornithine antiporter transporter; Catalyzes an electroneutral exchange between arginine and ornithine to allow high-efficiency energy conversion in the arginine deiminase pathway (By similarity)","uniprotId":"ARCD_ECOLI"},"6882351":{"id":6882351,"externalId":"511145.b1606","name":"folM","hasFamily":true,"annotation":"dihydromonapterin reductase, NADPH-dependent; dihydrofolate reductase isozyme; Catalyzes the reduction of dihydrofolate to tetrahydrofolate","uniprotId":"FOLM_ECOLI"},"6882352":{"id":6882352,"externalId":"511145.b1607","name":"ydgC","hasFamily":true,"annotation":"inner membrane protein, GlpM family","uniprotId":"YDGC_ECOLI"},"6882353":{"id":6882353,"externalId":"511145.b1608","name":"rstA","hasFamily":true,"annotation":"multicopy supressor of yjeE, yeaZ or ygjD deletion lethality, predicted response regulator of two-component regulatory system with sensor protein RstB; Member of the two-component regulatory system RstB/RstA","uniprotId":"RSTA_ECOLI"},"6882354":{"id":6882354,"externalId":"511145.b1609","name":"rstB","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with RstA; Member of the two-component regulatory system RstB/RstA. RstB functions as a membrane-associated protein kinase that phosphorylates RstA (Probable)","uniprotId":"RSTB_ECOLI"},"6882355":{"id":6882355,"externalId":"511145.b1610","name":"tus","hasFamily":true,"annotation":"inhibitor of replication at Ter, DNA-binding protein; Trans-acting protein required for termination of DNA replication. Binds to DNA replication terminator sequences (terA to terF) to prevent the passage of replication forks. The termination efficiency will be affected by the affinity of this protein for the terminator sequence (By similarity)","uniprotId":"TUS_ECOLI"},"6882356":{"id":6882356,"externalId":"511145.b1611","name":"fumC","hasFamily":true,"annotation":"fumarate hydratase (fumarase C),aerobic Class II; Catalyzes the reversible addition of water to fumarate to give L-malate","uniprotId":"FUMC_ECOLI"},"6882357":{"id":6882357,"externalId":"511145.b1612","name":"fumA","hasFamily":true,"annotation":"fumarate hydratase (fumarase A), aerobic Class I; It functions as an aerobic enzyme in the citric acid cycle. It accounts for about 80% of the fumarase activity when the bacteria grows aerobically","uniprotId":"FUMA_ECOLI"},"6882358":{"id":6882358,"externalId":"511145.b1613","name":"manA","hasFamily":true,"annotation":"mannose-6-phosphate isomerase; Involved in the conversion of glucose to GDP-L-fucose, which can be converted to L-fucose, a capsular polysaccharide","uniprotId":"MANA_ECOLI"},"6882359":{"id":6882359,"externalId":"511145.b1614","name":"ydgA","hasFamily":true,"annotation":"conserved protein, DUF945 family","uniprotId":"YDGA_ECOLI"},"6882360":{"id":6882360,"externalId":"511145.b1615","name":"uidC","hasFamily":false,"annotation":"putative outer membrane porin protein; Enhances the activity of the UidB (GusB) glucuronide transporter, on its own however it has no transport activity. Glucuronide transport does not occur in strain K12 due to a variant at position 100 of the UidB (GusB, AC P30868) protein","uniprotId":"UIDC_ECOLI"},"6882361":{"id":6882361,"externalId":"511145.b1616","name":"uidB","hasFamily":true,"annotation":"glucuronide transporter; Responsible for the transport of glucuronide into the cell energized by the proton motive force (probably by symport). Import is enhanced by UidC (GusC, AC Q47706)","uniprotId":"UIDB2_ECOLI"},"6882362":{"id":6882362,"externalId":"511145.b1617","name":"uidA","hasFamily":true,"annotation":"beta-D-glucuronidase","uniprotId":"BGLR_ECOLI"},"6882363":{"id":6882363,"externalId":"511145.b1618","name":"uidR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor for the uidRABC (gusRABC) operon","uniprotId":"UIDR_ECOLI"},"6882364":{"id":6882364,"externalId":"511145.b1619","name":"hdhA","hasFamily":true,"annotation":"7-alpha-hydroxysteroid dehydrogenase, NAD-dependent; 7-alpha-dehydroxylation of cholic acid, yielding deoxycholic acid and lithocholic acid, respectively. Highest affinity with taurochenodeoxycholic acid","uniprotId":"HDHA_ECOLI"},"6882365":{"id":6882365,"externalId":"511145.b1620","name":"malI","hasFamily":false,"annotation":"transcriptional repressor of Mal regulon; Repressor for the malX and malY genes. Also regulates its own expression. Binds maltose as an inducer","uniprotId":"MALI_ECOLI"},"6882366":{"id":6882366,"externalId":"511145.b1621","name":"malX","hasFamily":true,"annotation":"fused maltose and glucose-specific PTS enzymes: IIB component, IIC component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in maltose and glucose transport","uniprotId":"PTOCB_ECOLI"},"6882367":{"id":6882367,"externalId":"511145.b1622","name":"malY","hasFamily":true,"annotation":"bifunctional beta-cystathionase, PLP-dependent/ regulator of maltose regulon; Acts as a beta-cystathionase and as a repressor of the maltose regulon","uniprotId":"MALY_ECOLI"},"6882368":{"id":6882368,"externalId":"511145.b1623","name":"add","hasFamily":true,"annotation":"adenosine deaminase","uniprotId":"ADD_ECOLI"},"6882369":{"id":6882369,"externalId":"511145.b1624","name":"ydgJ","hasFamily":true,"annotation":"putative oxidoreductase","uniprotId":"YDGJ_ECOLI"},"6882370":{"id":6882370,"externalId":"511145.b1625","name":"cnu","hasFamily":true,"annotation":"oriC-binding complex H-NS/Cnu; binds 26 bp cnb site; The complex formed with H-NS binds to the specific 26-bp cnb site in the origin of replication oriC. Can complement, at least partially, the absence of the Hha protein in hha mutants","uniprotId":"CNU_ECOLI"},"6882371":{"id":6882371,"externalId":"511145.b1626","name":"ydgK","hasFamily":true,"annotation":"inner membrane protein, DUF2569 family","uniprotId":"YDGK_ECOLI"},"6882372":{"id":6882372,"externalId":"511145.b1627","name":"rnfA","hasFamily":true,"annotation":"electron transport complex protein required for the reduction of SoxR; May be part of a membrane complex involved in electron transport (By similarity)","uniprotId":"RNFA_ECOLI"},"6882373":{"id":6882373,"externalId":"511145.b1628","name":"rnfB","hasFamily":true,"annotation":"electron transport complex protein, iron-sulfur protein, required for the reduction of SoxR; May be part of a membrane complex involved in electron transport (By similarity)","uniprotId":"RNFB_ECOLI"},"6882374":{"id":6882374,"externalId":"511145.b1629","name":"rnfC","hasFamily":true,"annotation":"electron transport complex protein required for the reduction of SoxR; predicted membrane-associated NADH oxidoreductase; May be part of a membrane complex involved in electron transport (By similarity)","uniprotId":"RNFC_ECOLI"},"6882375":{"id":6882375,"externalId":"511145.b1630","name":"rnfD","hasFamily":true,"annotation":"electron transport complex protein required for the reduction of SoxR; predicted membrane protein; May be part of a membrane complex involved in electron transport (By similarity)","uniprotId":"RNFD_ECOLI"},"6882376":{"id":6882376,"externalId":"511145.b1631","name":"rnfG","hasFamily":true,"annotation":"electron transport complex protein required for the reduction of SoxR; May be part of a membrane complex involved in electron transport (By similarity)","uniprotId":"RNFG_ECOLI"},"6882377":{"id":6882377,"externalId":"511145.b1632","name":"rnfE","hasFamily":true,"annotation":"electron transport complex inner membrane NADH-quinone reductase required for the reduction of SoxR; May be part of a membrane complex involved in electron transport (By similarity)","uniprotId":"RNFE_ECOLI"},"6882378":{"id":6882378,"externalId":"511145.b1633","name":"nth","hasFamily":true,"annotation":"DNA glycosylase and apyrimidinic (AP) lyase (endonuclease III); DNA repair enzyme that has both DNA N-glycosylase activity and AP-lyase activity. The DNA N-glycosylase activity releases various damaged pyrimidines from DNA by cleaving the N- glycosidic bond, leaving an AP (apurinic/apyrimidinic) site. The AP-lyase activity cleaves the phosphodiester bond 3' to the AP site by a beta-elimination, leaving a 3'-terminal unsaturated sugar and a product with a terminal 5'-phosphate","uniprotId":"END3_ECOLI"},"6882379":{"id":6882379,"externalId":"511145.b1634","name":"dtpA","hasFamily":true,"annotation":"dipeptide and tripeptide permease A; Proton-dependent permease that transports di- and tripeptides as well as structurally related peptidomimetics such as aminocephalosporins into the cell. Has a clear preference for dipeptides and tripeptides composed of L-amino acids, and discriminates dipeptides on the basis of the position of charges within the substrate","uniprotId":"DTPA_ECOLI"},"6882380":{"id":6882380,"externalId":"511145.b1635","name":"gst","hasFamily":true,"annotation":"glutathionine S-transferase; Catalyzes the conjugation of reduced glutathione (GSH) to a wide number of exogenous and endogenous hydrophobic electrophiles. Shows activity toward 1-chloro-2,4-dinitrobenzene (CDNB) and ethacrynic acid. Also possesses thiol:disulfide oxidoreductase activity, using GSH to reduce bis-(2-hydroxyethyl) disulfide (HEDS). Has a low level of glutathione-dependent peroxidase activity toward cumene hydroperoxide. Is important for defense against oxidative stress, probably via its peroxidase activity","uniprotId":"GSTA_ECOLI"},"6882381":{"id":6882381,"externalId":"511145.b1636","name":"pdxY","hasFamily":true,"annotation":"pyridoxamine kinase; Phosphorylates B6 vitamers; functions in a salvage pathway. Uses pyridoxamine, but has negligible activity toward pyridoxal and pyridoxine as substrates","uniprotId":"PDXY_ECOLI"},"6882382":{"id":6882382,"externalId":"511145.b1637","name":"tyrS","hasFamily":true,"annotation":"tyrosyl-tRNA synthetase; Catalyzes the attachment of tyrosine to tRNA(Tyr) in a two-step reaction: tyrosine is first activated by ATP to form Tyr- AMP and then transferred to the acceptor end of tRNA(Tyr) (By similarity)","uniprotId":"SYY_ECOLI"},"6882383":{"id":6882383,"externalId":"511145.b1638","name":"pdxH","hasFamily":true,"annotation":"pyridoxine 5'-phosphate oxidase; Catalyzes the oxidation of either pyridoxine 5'- phosphate (PNP) or pyridoxamine 5'-phosphate (PMP) into pyridoxal 5'-phosphate (PLP) (By similarity)","uniprotId":"PDXH_ECOLI"},"6882384":{"id":6882384,"externalId":"511145.b1639","name":"mliC","hasFamily":true,"annotation":"inhibitor of c-type lysozyme, membrane-bound; Specifically inhibits C-type lysozymes","uniprotId":"MLIC_ECOLI"},"6882385":{"id":6882385,"externalId":"511145.b1640","name":"anmK","hasFamily":true,"annotation":"anhydro-N-acetylmuramic acid kinase; Catalyzes the specific phosphorylation of 1,6-anhydro-N- acetylmuramic acid (anhMurNAc) with the simultaneous cleavage of the 1,6-anhydro ring, generating MurNAc-6-P. Is required for the utilization of anhMurNAc either imported from the medium or derived from its own cell wall murein, and thus plays a role in cell wall recycling (By similarity)","uniprotId":"ANMK_ECOLI"},"6882386":{"id":6882386,"externalId":"511145.b1641","name":"slyB","hasFamily":true,"annotation":"outer membrane lipoprotein","uniprotId":"SLYB_ECOLI"},"6882387":{"id":6882387,"externalId":"511145.b1642","name":"slyA","hasFamily":true,"annotation":"DNA-binding transcriptional activator; Transcription regulator that can specifically activate or repress expression of target genes. Activates expression of genes such as molecular chaperones (groL, groS, dnaK, grpE, and cbpA), proteins involved in acid resistance (hdeA, hdeB, and gadA), the starvation lipoprotein slp, virulence protein hlyE/clyA. Represses expression of genes involved in the histidine biosynthetic pathway such as hisA, hisB, hisD, hisF and hisG. Required for the activation of virulence genes","uniprotId":"SLYA_ECOLI"},"6882388":{"id":6882388,"externalId":"511145.b1643","name":"ydhI","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YDHI_ECOLI"},"6882389":{"id":6882389,"externalId":"511145.b1644","name":"ydhJ","hasFamily":true,"annotation":"putative membrane fusion protein (MFP) of YdhJK efflux pump","uniprotId":"YDHJ_ECOLI"},"6882390":{"id":6882390,"externalId":"511145.b1645","name":"ydhK","hasFamily":true,"annotation":"putative efflux protein (PET) component of YdhJK efflux pump","uniprotId":"YDHK_ECOLI"},"6882391":{"id":6882391,"externalId":"511145.b1646","name":"sodC","hasFamily":true,"annotation":"superoxide dismutase, Cu, Zn; Destroys radicals which are normally produced within the cells and which are toxic to biological systems (By similarity)","uniprotId":"SODC_ECOLI"},"6882392":{"id":6882392,"externalId":"511145.b1647","name":"ydhF","hasFamily":true,"annotation":"putative oxidoreductase; May function as oxidoreductase (Potential)","uniprotId":"YDHF_ECOLI"},"6882393":{"id":6882393,"externalId":"511145.b1648","name":"ydhL","hasFamily":true,"annotation":"conserved protein","uniprotId":"YDHL_ECOLI"},"6882394":{"id":6882394,"externalId":"511145.b1649","name":"nemR","hasFamily":true,"annotation":"transcriptional repressor for the nem operon, NEM-inactivated; Represses the transcription of the nemRA operon by binding to the nemR box","uniprotId":"NEMR_ECOLI"},"6882395":{"id":6882395,"externalId":"511145.b1650","name":"nemA","hasFamily":true,"annotation":"N-ethylmaleimide reductase, FMN-linked; Catalyzes the reduction of N-ethylmaleimide (NEM) to N- ethylsuccinimide","uniprotId":"NEMA_ECOLI"},"6882396":{"id":6882396,"externalId":"511145.b1651","name":"gloA","hasFamily":true,"annotation":"glyoxalase I, Ni-dependent; Catalyzes the conversion of hemimercaptal, formed from methylglyoxal and glutathione, to S-lactoylglutathione","uniprotId":"LGUL_ECOLI"},"6882397":{"id":6882397,"externalId":"511145.b1652","name":"rnt","hasFamily":true,"annotation":"ribonuclease T (RNase T); Responsible for the end-turnover of tRNA: specifically removes the terminal AMP residue from uncharged tRNA (tRNA-C-C-A). Also appears to be involved in tRNA biosynthesis, especially in strains lacking other exoribonucleases","uniprotId":"RNT_ECOLI"},"6882398":{"id":6882398,"externalId":"511145.b1653","name":"lhr","hasFamily":true,"annotation":"putative ATP-dependent helicase","uniprotId":"LHR_ECOLI"},"6882399":{"id":6882399,"externalId":"511145.b1654","name":"grxD","hasFamily":true,"annotation":"glutaredoxin-4; Monothiol glutaredoxin involved in the biogenesis of iron-sulfur clusters (Probable)","uniprotId":"GLRX4_ECOLI"},"6882400":{"id":6882400,"externalId":"511145.b1655","name":"ydhO","hasFamily":true,"annotation":"putative peptidase, C40 clan","uniprotId":"YDHO_ECOLI"},"6882401":{"id":6882401,"externalId":"511145.b1656","name":"sodB","hasFamily":true,"annotation":"superoxide dismutase, Fe; Destroys superoxide anion radicals which are normally produced within the cells and which are toxic to biological systems","uniprotId":"SODF_ECOLI"},"6882402":{"id":6882402,"externalId":"511145.b1657","name":"ydhP","hasFamily":true,"annotation":"putative transporter","uniprotId":"YDHP_ECOLI"},"6882403":{"id":6882403,"externalId":"511145.b1658","name":"purR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor, hypoxanthine-binding; Is the main repressor of the genes involved in the de novo synthesis of purine nucleotides, regulating purB, purC, purEK, purF, purHD, purL, purMN and guaBA expression. In addition, it participates in the regulation or coregulation of genes involved in de novo pyrimidine nucleotide biosynthesis, salvage and uptake (pyrC, pyrD, carAB and codBA), and of several genes encoding enzymes necessary for nucleotide and polyamine biosynthesis (prsA, glyA, gcvTHP, speA, glnB). Binds to a 16-bp palindromic sequence located within the pro [...] ","uniprotId":"PURR_ECOLI"},"6882404":{"id":6882404,"externalId":"511145.b1659","name":"ydhB","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YDHB_ECOLI"},"6882405":{"id":6882405,"externalId":"511145.b1660","name":"ydhC","hasFamily":true,"annotation":"predicted transporter","uniprotId":"YDHC_ECOLI"},"6882406":{"id":6882406,"externalId":"511145.b1661","name":"cfa","hasFamily":true,"annotation":"cyclopropane fatty acyl phospholipid synthase (unsaturated-phospholipid methyltransferase); Transfers a methylene group from S-adenosyl-L-methionine to the cis double bond of an unsaturated fatty acid chain resulting in the replacement of the double bond with a methylene bridge (By similarity)","uniprotId":"CFA_ECOLI"},"6882407":{"id":6882407,"externalId":"511145.b1662","name":"ribC","hasFamily":true,"annotation":"riboflavin synthase, alpha subunit; Catalyzes the dismutation of two molecules of 6,7- dimethyl-8-ribityllumazine, resulting in the formation of riboflavin and 5-amino-6-(D-ribitylamino)uracil","uniprotId":"RISA_ECOLI"},"6882408":{"id":6882408,"externalId":"511145.b1663","name":"mdtK","hasFamily":true,"annotation":"multidrug efflux system transporter; Multidrug efflux pump that functions probably as a Na(+)/drug antiporter. Confers resistance to many drugs such as fluoroquinolones (norfloxacin, ciprofloxacin, enoxacin), tetraphenylphosphonium ion (TPP), deoxycholate, doxorubicin, trimethoprim, chloramphenicol, fosfomycin, acriflavine, ethidium bromide and benzalkonium","uniprotId":"MDTK_ECOLI"},"6882409":{"id":6882409,"externalId":"511145.b1664","name":"ydhQ","hasFamily":false,"annotation":"conserved protein","uniprotId":"YDHQ_ECOLI"},"6882410":{"id":6882410,"externalId":"511145.b1667","name":"ydhR","hasFamily":true,"annotation":"putative monooxygenase; May function as monoxygenase and play a role in the metabolism of aromatic compounds (Potential)","uniprotId":"YDHR_ECOLI"},"6882411":{"id":6882411,"externalId":"511145.b1668","name":"ydhS","hasFamily":false,"annotation":"conserved protein with FAD/NAD(P)-binding domain","uniprotId":"YDHS_ECOLI"},"6882412":{"id":6882412,"externalId":"511145.b1669","name":"ydhT","hasFamily":false,"annotation":"conserved protein","uniprotId":"YDHT_ECOLI"},"6882413":{"id":6882413,"externalId":"511145.b1670","name":"ydhU","hasFamily":true,"annotation":"putative cytochrome","uniprotId":"PHSC_ECOLI"},"6882414":{"id":6882414,"externalId":"511145.b1671","name":"ydhX","hasFamily":false,"annotation":"putative 4Fe-4S ferridoxin-type protein","uniprotId":"YDHX_ECOLI"},"6882415":{"id":6882415,"externalId":"511145.b1672","name":"ydhW","hasFamily":false,"annotation":"predicted protein","uniprotId":"YDHW_ECOLI"},"6882416":{"id":6882416,"externalId":"511145.b1673","name":"ydhV","hasFamily":true,"annotation":"putative oxidoreductase","uniprotId":"YDHV_ECOLI"},"6882417":{"id":6882417,"externalId":"511145.b1674","name":"ydhY","hasFamily":false,"annotation":"putative 4Fe-4S ferridoxin-type protein","uniprotId":"YDHY_ECOLI"},"6882418":{"id":6882418,"externalId":"511145.b1675","name":"ydhZ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YDHZ_ECOLI"},"6882419":{"id":6882419,"externalId":"511145.b1676","name":"pykF","hasFamily":true,"annotation":"pyruvate kinase I","uniprotId":"KPYK1_ECOLI"},"6882420":{"id":6882420,"externalId":"511145.b1677","name":"lpp","hasFamily":true,"annotation":"murein lipoprotein; Interacts with the peptidoglycan both covalently and noncovalently. This interaction contributes to the maintenance of the structural and functional integrity of the cell envelope","uniprotId":"LPP_ECOLI"},"6882421":{"id":6882421,"externalId":"511145.b1678","name":"ynhG","hasFamily":true,"annotation":"murein L,D-transpeptidase; Responsible, at least in part, for generating a meso- diaminopimelyl-3-a meso-diaminopimelyl-3 cross-link","uniprotId":"YNHG_ECOLI"},"6882422":{"id":6882422,"externalId":"511145.b1679","name":"sufE","hasFamily":true,"annotation":"sulfur acceptor protein; Participates in cysteine desulfuration mediated by SufS. Cysteine desulfuration mobilizes sulfur from L-cysteine to yield L-alanine and constitutes an essential step in sulfur metabolism for biosynthesis of a variety of sulfur-containing biomolecules. Functions as a sulfur acceptor for SufS, by mediating the direct transfer of the sulfur atom from the S-sulfanylcysteine of SufS, an intermediate product of cysteine desulfuration process. Together with the SufBCD complex, it thereby enhances up to 50- fold, the cysteine desulfurase activity of SufS. Component of  [...] ","uniprotId":"SUFE_ECOLI"},"6882423":{"id":6882423,"externalId":"511145.b1680","name":"sufS","hasFamily":true,"annotation":"cysteine desulfurase, stimulated by SufE; selenocysteine lyase, PLP-dependent; Cysteine desulfurases mobilize the sulfur from L- cysteine to yield L-alanine, an essential step in sulfur metabolism for biosynthesis of a variety of sulfur-containing biomolecules. Component of the suf operon, which is activated and required under specific conditions such as oxidative stress and iron limitation. Acts as a potent selenocysteine lyase in vitro, that mobilizes selenium from L-selenocysteine. Selenocysteine lyase activity is however unsure in vivo (By similarity)","uniprotId":"SUFS_ECOLI"},"6882424":{"id":6882424,"externalId":"511145.b1681","name":"sufD","hasFamily":true,"annotation":"FeS cluster assembly protein of SufBCD complex; The SufBCD complex acts synergistically with SufE to stimulate the cysteine desulfurase activity of SufS. The SufBCD complex contributes to the assembly or repair of oxygen-labile iron-sulfur clusters under oxidative stress. May facilitate iron uptake from extracellular iron chelators under iron limitation. Required for the stability of the FhuF protein","uniprotId":"SUFD_ECOLI"},"6882425":{"id":6882425,"externalId":"511145.b1682","name":"sufC","hasFamily":true,"annotation":"SufBCD Fe-S cluster assembly scaffold protein, ATP-binding protein; Has low ATPase activity. The SufBCD complex acts synergistically with SufE to stimulate the cysteine desulfurase activity of SufS. The SufBCD complex contributes to the assembly or repair of oxygen-labile iron-sulfur clusters under oxidative stress. May facilitate iron uptake from extracellular iron chelators under iron limitation","uniprotId":"SUFC_ECOLI"},"6882426":{"id":6882426,"externalId":"511145.b1683","name":"sufB","hasFamily":true,"annotation":"component of SufBCD complex; The SufBCD complex acts synergistically with SufE to stimulate the cysteine desulfurase activity of SufS. The SufBCD complex contributes to the assembly or repair of oxygen-labile iron-sulfur clusters under oxidative stress. May facilitate iron uptake from extracellular iron chelators under iron limitation","uniprotId":"SUFB_ECOLI"},"6882427":{"id":6882427,"externalId":"511145.b1684","name":"sufA","hasFamily":true,"annotation":"Fe-S cluster assembly protein","uniprotId":"SUFA_ECOLI"},"6882428":{"id":6882428,"externalId":"511145.b1685","name":"ydiH","hasFamily":true,"annotation":"predicted protein","uniprotId":"YDIH_ECOLI"},"6882429":{"id":6882429,"externalId":"511145.b1686","name":"ydiI","hasFamily":true,"annotation":"acyl-CoA esterase; Displays esterase activity toward palmitoyl-CoA and pNP- butyrate","uniprotId":"YDII_ECOLI"},"6882430":{"id":6882430,"externalId":"511145.b1687","name":"ydiJ","hasFamily":true,"annotation":"putative FAD-linked oxidoreductase","uniprotId":"YDIJ_ECOLI"},"6882431":{"id":6882431,"externalId":"511145.b1688","name":"ydiK","hasFamily":true,"annotation":"inner membrane protein, UPF0118 family","uniprotId":"YDIK_ECOLI"},"6882432":{"id":6882432,"externalId":"511145.b1689","name":"ydiL","hasFamily":true,"annotation":"conserved protein","uniprotId":"YDIL_ECOLI"},"6882433":{"id":6882433,"externalId":"511145.b1690","name":"ydiM","hasFamily":true,"annotation":"inner membrane protein, predicted transporter","uniprotId":"YDIM_ECOLI"},"6882434":{"id":6882434,"externalId":"511145.b1691","name":"ydiN","hasFamily":true,"annotation":"Inner membrane protein, predicted MFS superfamily transporter","uniprotId":"YDIN_ECOLI"},"6882435":{"id":6882435,"externalId":"511145.b1692","name":"ydiB","hasFamily":true,"annotation":"quinate/shikimate 5-dehydrogenase, NAD(P)-binding; The physiological substrate is not known","uniprotId":"YDIB_ECOLI"},"6882436":{"id":6882436,"externalId":"511145.b1693","name":"aroD","hasFamily":true,"annotation":"3-dehydroquinate dehydratase","uniprotId":"AROD_ECOLI"},"6882437":{"id":6882437,"externalId":"511145.b1694","name":"ydiF","hasFamily":true,"annotation":"fused predicted acetyl-CoA:acetoacetyl-CoA transferase: alpha subunit/beta subunit; CoA transferase having broad substrate specificity for short-chain acyl-CoA thioesters with the activity decreasing when the length of the carboxylic acid chain exceeds four carbons. May play a role in short-chain fatty acid metabolism in E.coli (By similarity)","uniprotId":"YDIF_ECOLI"},"6882438":{"id":6882438,"externalId":"511145.b1695","name":"ydiO","hasFamily":true,"annotation":"putative acyl-CoA dehydrogenase","uniprotId":"YDIO_ECOLI"},"6882439":{"id":6882439,"externalId":"511145.b1696","name":"ydiP","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YDIP_ECOLI"},"6882440":{"id":6882440,"externalId":"511145.b1697","name":"ydiQ","hasFamily":true,"annotation":"putative electron transfer flavoprotein subunit; May play a role in a redox process","uniprotId":"YDIQ_ECOLI"},"6882441":{"id":6882441,"externalId":"511145.b1698","name":"ydiR","hasFamily":true,"annotation":"putative electron transfer flavoprotein, FAD-binding subunit; May play a role in a redox process","uniprotId":"YDIR_ECOLI"},"6882442":{"id":6882442,"externalId":"511145.b1699","name":"ydiS","hasFamily":true,"annotation":"putative oxidoreductase with FAD/NAD(P)-binding domain; Probably accepts electrons from YdiQ/YdiR and reduces a quinone","uniprotId":"YDIS_ECOLI"},"6882443":{"id":6882443,"externalId":"511145.b1700","name":"ydiT","hasFamily":true,"annotation":"putative 3Fe-4S ferredoxin-type protein; Could be a 3Fe-4S cluster-containing protein. Probably participates in a redox process with YdiQ, YdiR and YdiS","uniprotId":"YDIT_ECOLI"},"6882444":{"id":6882444,"externalId":"511145.b1701","name":"fadK","hasFamily":true,"annotation":"short chain acyl-CoA synthetase, anaerobic; Catalyzes the esterification, concomitant with transport, of exogenous fatty acids into metabolically active CoA thioesters for subsequent degradation or incorporation into phospholipids. It has a low activity on medium or long chain fatty acids and is maximally active on C6 and C8 substrates","uniprotId":"FADK_ECOLI"},"6882445":{"id":6882445,"externalId":"511145.b1702","name":"pps","hasFamily":true,"annotation":"phosphoenolpyruvate synthase; Catalyzes the phosphorylation of pyruvate to phosphoenolpyruvate (By similarity)","uniprotId":"PPSA_ECOLI"},"6882446":{"id":6882446,"externalId":"511145.b1703","name":"ppsR","hasFamily":true,"annotation":"bifunctional regulatory protein: PEP synthase kinase and PEP synthase pyrophosphorylase; Bifunctional serine/threonine kinase and phosphorylase involved in the regulation of the phosphoenolpyruvate synthase (PEPS) by catalyzing its phosphorylation/dephosphorylation (By similarity)","uniprotId":"PSRP_ECOLI"},"6882447":{"id":6882447,"externalId":"511145.b1704","name":"aroH","hasFamily":true,"annotation":"3-deoxy-D-arabino-heptulosonate-7-phosphate synthase, tryptophan repressible; Stereospecific condensation of phosphoenolpyruvate (PEP) and D-erythrose-4-phosphate (E4P) giving rise to 3-deoxy-D- arabino-heptulosonate-7-phosphate (DAHP) (By similarity)","uniprotId":"AROH_ECOLI"},"6882448":{"id":6882448,"externalId":"511145.b1705","name":"ydiE","hasFamily":true,"annotation":"hemin uptake protein HemP homolog","uniprotId":"YDIE_ECOLI"},"6882449":{"id":6882449,"externalId":"511145.b1706","name":"ydiU","hasFamily":true,"annotation":"conserved protein, UPF0061 family","uniprotId":"YDIU_ECOLI"},"6882450":{"id":6882450,"externalId":"511145.b1707","name":"ydiV","hasFamily":true,"annotation":"anti-FlhD4C2 factor, inactive EAL family phosphodiesterase; Upon overexpression acts as a novel anti-FlhC(2)FlhD(4) factor, decreasing its DNA-binding activity, able to negatively regulate expression of flagellar class II operons including FliC","uniprotId":"YDIV_ECOLI"},"6882451":{"id":6882451,"externalId":"511145.b1708","name":"nlpC","hasFamily":true,"annotation":"putative peptidase, C40 clan, lipoprotein","uniprotId":"NLPC_ECOLI"},"6882452":{"id":6882452,"externalId":"511145.b1709","name":"btuD","hasFamily":true,"annotation":"vitamin B12 transporter subunit : ATP-binding component of ABC superfamily; Part of the ABC transporter complex BtuCDF involved in vitamin B12 import. Responsible for energy coupling to the transport system (By similarity)","uniprotId":"BTUD_ECOLI"},"6882453":{"id":6882453,"externalId":"511145.b1710","name":"btuE","hasFamily":true,"annotation":"glutathione peroxidase; Not essential for B12 transport; however, it is an auxiliary component of the transport system","uniprotId":"BTUE_ECOLI"},"6882454":{"id":6882454,"externalId":"511145.b1711","name":"btuC","hasFamily":true,"annotation":"vitamin B12 transporter subunit: membrane component of ABC superfamily; Part of the ABC transporter complex BtuCDF involved in vitamin B12 import. Involved in the translocation of the substrate across the membrane (By similarity)","uniprotId":"BTUC_ECOLI"},"6882455":{"id":6882455,"externalId":"511145.b1712","name":"ihfA","hasFamily":true,"annotation":"integration host factor (IHF), DNA-binding protein, alpha subunit; This protein is one of the two subunits of integration host factor, a specific DNA-binding protein that functions in genetic recombination as well as in transcriptional and translational control (By similarity)","uniprotId":"IHFA_ECOLI"},"6882456":{"id":6882456,"externalId":"511145.b1713","name":"pheT","hasFamily":true,"annotation":"phenylalanine tRNA synthetase, beta subunit","uniprotId":"SYFB_ECOLI"},"6882457":{"id":6882457,"externalId":"511145.b1714","name":"pheS","hasFamily":true,"annotation":"phenylalanine tRNA synthetase, alpha subunit","uniprotId":"SYFA_ECOLI"},"6882458":{"id":6882458,"externalId":"511145.b1715","name":"pheM","hasFamily":false,"annotation":"phenylalanyl-tRNA synthetase operon leader peptide"},"6882459":{"id":6882459,"externalId":"511145.b1716","name":"rplT","hasFamily":true,"annotation":"50S ribosomal subunit protein L20; Binds directly to 23S ribosomal RNA and is necessary for the in vitro assembly process of the 50S ribosomal subunit. It is not involved in the protein synthesizing functions of that subunit (By similarity)","uniprotId":"RL20_ECOLI"},"6882460":{"id":6882460,"externalId":"511145.b1717","name":"rpmI","hasFamily":false,"annotation":"50S ribosomal subunit protein L35","uniprotId":"RL35_ECOLI"},"6882461":{"id":6882461,"externalId":"511145.b1718","name":"infC","hasFamily":true,"annotation":"protein chain initiation factor IF-3; IF-3 binds to the 30S ribosomal subunit and shifts the equilibrum between 70S ribosomes and their 50S and 30S subunits in favor of the free subunits, thus enhancing the availability of 30S subunits on which protein synthesis initiation begins (By similarity)","uniprotId":"IF3_ECOLI"},"6882462":{"id":6882462,"externalId":"511145.b1719","name":"thrS","hasFamily":true,"annotation":"threonyl-tRNA synthetase; ThrS is also a translational repressor protein, it controls the translation of its own gene by binding to its mRNA","uniprotId":"SYT_ECOLI"},"6882463":{"id":6882463,"externalId":"511145.b1722","name":"ydiY","hasFamily":true,"annotation":"putative outer membrane protein, acid-inducible","uniprotId":"YDIY_ECOLI"},"6882464":{"id":6882464,"externalId":"511145.b1723","name":"pfkB","hasFamily":true,"annotation":"6-phosphofructokinase II","uniprotId":"K6PF2_ECOLI"},"6882465":{"id":6882465,"externalId":"511145.b1724","name":"ydiZ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YDIZ_ECOLI"},"6882466":{"id":6882466,"externalId":"511145.b1725","name":"yniA","hasFamily":true,"annotation":"putative phosphotransferase/kinase","uniprotId":"YNIA_ECOLI"},"6882467":{"id":6882467,"externalId":"511145.b1726","name":"yniB","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YNIB_ECOLI"},"6882468":{"id":6882468,"externalId":"511145.b1727","name":"yniC","hasFamily":true,"annotation":"2-deoxyglucose-6-P phosphatase; Catalyzes strongly the dephosphorylation of 2- deoxyglucose 6-phosphate (2dGlu6P) and slightly the dephosphorylation of mannose 6-phosphate (Man6P), 2-deoxyribose-5- phosphate (2dRib5P), ribose-5-phosphate (Rib5P) and glucose-6- phosphate (Glu6P)","uniprotId":"YNIC_ECOLI"},"6882469":{"id":6882469,"externalId":"511145.b1728","name":"ydjM","hasFamily":true,"annotation":"inner membrane protein regulated by LexA","uniprotId":"YDJM_ECOLI"},"6882470":{"id":6882470,"externalId":"511145.b1729","name":"ydjN","hasFamily":true,"annotation":"putative transporter","uniprotId":"YDJN_ECOLI"},"6882471":{"id":6882471,"externalId":"511145.b1730","name":"ydjO","hasFamily":false,"annotation":"predicted protein","uniprotId":"YDJO_ECOLI"},"6882472":{"id":6882472,"externalId":"511145.b1731","name":"cedA","hasFamily":true,"annotation":"cell division modulator; Activates the cell division inhibited by chromosomal DNA over-replication (By similarity)","uniprotId":"CEDA_ECOLI"},"6882473":{"id":6882473,"externalId":"511145.b1732","name":"katE","hasFamily":true,"annotation":"catalase HPII, heme d-containing; Decomposes hydrogen peroxide into water and oxygen; serves to protect cells from the toxic effects of hydrogen peroxide","uniprotId":"CATE_ECOLI"},"6882474":{"id":6882474,"externalId":"511145.b1733","name":"chbG","hasFamily":true,"annotation":"UPF0249 family protein","uniprotId":"CHBG_ECOLI"},"6882475":{"id":6882475,"externalId":"511145.b1734","name":"celF","hasFamily":true,"annotation":"phospho-chitobiase; general 6-phospho-beta-glucosidase activity; Hydrolyzes a wide variety of P-beta-glucosides including cellobiose-6P, salicin-6P, arbutin-6P, gentiobiose-6P, methyl- beta-glucoside-6P and p-nitrophenyl-beta-D-glucopyranoside-6P. Is also able to hydrolyze phospho-N,N'-diacetylchitobiose","uniprotId":"CHBF_ECOLI"},"6882476":{"id":6882476,"externalId":"511145.b1735","name":"chbR","hasFamily":true,"annotation":"rRepressor, chb operon for N,N'-diacetylchitobiose utilization; Dual-function repressor/activator of the chbBCARFG operon. In the absence of the inducing sugar chitobiose, together with NagC, represses the chbBCARFG operon for the uptake and metabolism of chitobiose. In association with Crp, and probably in the presence of chitobiose 6-phosphate, induces the transcription of the chbBCARFG operon","uniprotId":"A4UR75_ECOLI"},"6882477":{"id":6882477,"externalId":"511145.b1736","name":"celC","hasFamily":true,"annotation":"N,N'-diacetylchitobiose-specific enzyme IIA component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in N,N'-diacetylchitobiose transport","uniprotId":"PTQA_ECOLI"},"6882478":{"id":6882478,"externalId":"511145.b1737","name":"celB","hasFamily":true,"annotation":"N,N'-diacetylchitobiose-specific enzyme IIC component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in N,N'-diacetylchitobiose transport","uniprotId":"PTQC_ECOLI"},"6882479":{"id":6882479,"externalId":"511145.b1738","name":"celA","hasFamily":true,"annotation":"N,N'-diacetylchitobiose-specific enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in N,N'-diacetylchitobiose transport","uniprotId":"PTQB_ECOLI"},"6882480":{"id":6882480,"externalId":"511145.b1739","name":"osmE","hasFamily":true,"annotation":"DNA-binding transcriptional activator","uniprotId":"OSME_ECOLI"},"6882481":{"id":6882481,"externalId":"511145.b1740","name":"nadE","hasFamily":true,"annotation":"NAD synthetase, NH3/glutamine-dependent; Catalyzes a key step in NAD biosynthesis, transforming deamido-NAD into NAD by a two-step reaction","uniprotId":"NADE_ECOLI"},"6882482":{"id":6882482,"externalId":"511145.b1741","name":"cho","hasFamily":true,"annotation":"endonuclease of nucleotide excision repair; Incises the DNA at the 3' side of a lesion during nucleotide excision repair. Incises the DNA farther away from the lesion than UvrC. Not able to incise the 5' site of a lesion. In vitro, the incision activity of Cho is UvrA and UvrB dependent. When a lesion remains because UvrC is not able to induce the 3' incision, Cho incises the DNA. Then UvrC makes the 5' incision. The combined action of Cho and UvrC broadens the substrate range of nucleotide excision repair","uniprotId":"CHO_ECOLI"},"6882483":{"id":6882483,"externalId":"511145.b1742","name":"ves","hasFamily":true,"annotation":"conserved protein","uniprotId":"VES_ECOLI"},"6882484":{"id":6882484,"externalId":"511145.b1743","name":"spy","hasFamily":true,"annotation":"periplasmic ATP-independent protein refolding chaperone, stress-induced; Could be involved in zinc resistance or another aspect of zinc metabolism","uniprotId":"SPY_ECOLI"},"6882485":{"id":6882485,"externalId":"511145.b1744","name":"astE","hasFamily":true,"annotation":"succinylglutamate desuccinylase; Transforms N(2)-succinylglutamate into succinate and glutamate (By similarity)","uniprotId":"ASTE_ECOLI"},"6882486":{"id":6882486,"externalId":"511145.b1745","name":"astB","hasFamily":true,"annotation":"succinylarginine dihydrolase; Catalyzes the hydrolysis of N(2)-succinylarginine into N(2)-succinylornithine, ammonia and CO(2) (By similarity)","uniprotId":"ASTB_ECOLI"},"6882487":{"id":6882487,"externalId":"511145.b1746","name":"astD","hasFamily":true,"annotation":"succinylglutamic semialdehyde dehydrogenase; Catalyzes the NAD-dependent reduction of succinylglutamate semialdehyde into succinylglutamate. Also shows activity with decanal or succinic semialdehyde as the electron donor and NAD as the electron acceptor. No activity is detected with NADP as the electron acceptor. Therefore, is an aldehyde dehydrogenase with broad substrate specificity","uniprotId":"ASTD_ECOLI"},"6882488":{"id":6882488,"externalId":"511145.b1747","name":"astA","hasFamily":true,"annotation":"arginine succinyltransferase; Catalyzes the transfer of succinyl-CoA to arginine to produce N(2)-succinylarginine (By similarity)","uniprotId":"ASTA_ECOLI"},"6882489":{"id":6882489,"externalId":"511145.b1748","name":"astC","hasFamily":true,"annotation":"succinylornithine transaminase, PLP-dependent; Catalyzes the transamination of N(2)-succinylornithine and alpha-ketoglutarate into N(2)-succinylglutamate semialdehyde and glutamate. Can also act as an acetylornithine aminotransferase (By similarity)","uniprotId":"ASTC_ECOLI"},"6882490":{"id":6882490,"externalId":"511145.b1749","name":"xthA","hasFamily":true,"annotation":"exonuclease III; Major apurinic-apyrimidinic endonuclease of E.coli. It removes the damaged DNA at cytosines and guanines by cleaving on the 3'-side of the AP site by a beta-elimination reaction. It exhibits 3'-5'-exonuclease, 3'-phosphomonoesterase, 3'-repair diesterase and ribonuclease H activities","uniprotId":"EX3_ECOLI"},"6882491":{"id":6882491,"externalId":"511145.b1750","name":"ydjX","hasFamily":true,"annotation":"inner membrane protein, TVP38/TMEM64 family","uniprotId":"YDJX_ECOLI"},"6882492":{"id":6882492,"externalId":"511145.b1751","name":"ydjY","hasFamily":false,"annotation":"predicted protein","uniprotId":"YDJY_ECOLI"},"6882493":{"id":6882493,"externalId":"511145.b1752","name":"ydjZ","hasFamily":true,"annotation":"Inner membrane protein, TVP38/TMEM64 family","uniprotId":"YDJZ_ECOLI"},"6882494":{"id":6882494,"externalId":"511145.b1753","name":"ynjA","hasFamily":true,"annotation":"conserved protein","uniprotId":"YNJA_ECOLI"},"6882495":{"id":6882495,"externalId":"511145.b1754","name":"ynjB","hasFamily":true,"annotation":"conserved protein","uniprotId":"YNJB_ECOLI"},"6882496":{"id":6882496,"externalId":"511145.b1755","name":"ynjC","hasFamily":false,"annotation":"putative inner membrane ABC transporter permease, function unknown; Probably part of the binding-protein-dependent transport system YnjCD. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"YNJC_ECOLI"},"6882497":{"id":6882497,"externalId":"511145.b1756","name":"ynjD","hasFamily":false,"annotation":"putative transporter subunit: ATP-binding component of ABC superfamily; Probably part of a binding-protein-dependent transport system YnjCD. Probably responsible for energy coupling to the transport system","uniprotId":"YNJD_ECOLI"},"6882498":{"id":6882498,"externalId":"511145.b1757","name":"ynjE","hasFamily":false,"annotation":"predicted thiosulfate sulfur transferase","uniprotId":"YNJE_ECOLI"},"6882499":{"id":6882499,"externalId":"511145.b1758","name":"ynjF","hasFamily":false,"annotation":"inner membrane protein, phosphatidylglycerophosphate synthase homolog","uniprotId":"YNJF_ECOLI"},"6882500":{"id":6882500,"externalId":"511145.b1759","name":"nudG","hasFamily":true,"annotation":"CTP pyrophosphohydrolas; also hydrolyzes 2-hydroxy-dATP, 8-hydroxy-dGTP, CTP, dCTP and 5-methyl-dCTP; Specific for pyrimidine substrates. Acts on 5-methyl- dCTP, CTP and dCTP in decreasing order","uniprotId":"NUDG_ECOLI"},"6882501":{"id":6882501,"externalId":"511145.b1760","name":"ynjH","hasFamily":true,"annotation":"conserved protein, DUF1496 family","uniprotId":"YNJH_ECOLI"},"6882502":{"id":6882502,"externalId":"511145.b1761","name":"gdhA","hasFamily":true,"annotation":"glutamate dehydrogenase, NADP-specific; Catalyzes the reversible oxidative deamination of glutamate to alpha-ketoglutarate and ammonia","uniprotId":"DHE4_ECOLI"},"6882503":{"id":6882503,"externalId":"511145.b1762","name":"ynjI","hasFamily":false,"annotation":"inner membrane protein","uniprotId":"YNJI_ECOLI"},"6882504":{"id":6882504,"externalId":"511145.b1763","name":"topB","hasFamily":true,"annotation":"DNA topoisomerase III; Releases the supercoiling and torsional tension of DNA, which is introduced during the DNA replication and transcription, by transiently cleaving and rejoining one strand of the DNA duplex. Introduces a single-strand break via transesterification at a target site in duplex DNA. The scissile phosphodiester is attacked by the catalytic tyrosine of the enzyme, resulting in the formation of a DNA-(5'-phosphotyrosyl)-enzyme intermediate and the expulsion of a 3'-OH DNA strand. The free DNA strand then undergoes passage around the unbroken strand, thus removing DNA sup [...] ","uniprotId":"TOP3_ECOLI"},"6882505":{"id":6882505,"externalId":"511145.b1764","name":"selD","hasFamily":true,"annotation":"selenophosphate synthase; Synthesizes selenophosphate from selenide and ATP (By similarity)","uniprotId":"SELD_ECOLI"},"6882506":{"id":6882506,"externalId":"511145.b1765","name":"ydjA","hasFamily":true,"annotation":"putative oxidoreductase","uniprotId":"YDJA_ECOLI"},"6882507":{"id":6882507,"externalId":"511145.b1766","name":"sppA","hasFamily":true,"annotation":"protease IV (signal peptide peptidase); Digestion of the cleaved signal peptides. This activity is necessary to maintain proper secretion of mature proteins across the membrane","uniprotId":"SPPA_ECOLI"},"6882508":{"id":6882508,"externalId":"511145.b1767","name":"ansA","hasFamily":true,"annotation":"cytoplasmic L-asparaginase I","uniprotId":"ASPG1_ECOLI"},"6882509":{"id":6882509,"externalId":"511145.b1768","name":"pncA","hasFamily":true,"annotation":"nicotinamidase/pyrazinamidase","uniprotId":"PNCA_ECOLI"},"6882510":{"id":6882510,"externalId":"511145.b1769","name":"ydjE","hasFamily":true,"annotation":"putative transporter","uniprotId":"YDJE_ECOLI"},"6882511":{"id":6882511,"externalId":"511145.b1770","name":"ydjF","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YDJF_ECOLI"},"6882512":{"id":6882512,"externalId":"511145.b1771","name":"ydjG","hasFamily":true,"annotation":"alpha-Keto reductase, NADH-dependent","uniprotId":"YDJG_ECOLI"},"6882513":{"id":6882513,"externalId":"511145.b1772","name":"ydjH","hasFamily":true,"annotation":"putative kinase","uniprotId":"YDJH_ECOLI"},"6882514":{"id":6882514,"externalId":"511145.b1773","name":"ydjI","hasFamily":true,"annotation":"putative aldolase","uniprotId":"YDJI_ECOLI"},"6882515":{"id":6882515,"externalId":"511145.b1774","name":"ydjJ","hasFamily":true,"annotation":"putative oxidoreductase, Zn-dependent and NAD(P)-binding","uniprotId":"YDJJ_ECOLI"},"6882516":{"id":6882516,"externalId":"511145.b1775","name":"ydjK","hasFamily":false,"annotation":"putative transporter","uniprotId":"YDJK_ECOLI"},"6882517":{"id":6882517,"externalId":"511145.b1776","name":"ydjL","hasFamily":true,"annotation":"putative oxidoreductase, Zn-dependent and NAD(P)-binding","uniprotId":"YDJL_ECOLI"},"6882518":{"id":6882518,"externalId":"511145.b1777","name":"yeaC","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEAC_ECOLI"},"6882519":{"id":6882519,"externalId":"511145.b1778","name":"msrB","hasFamily":true,"annotation":"methionine sulfoxide reductase B","uniprotId":"MSRB_ECOLI"},"6882520":{"id":6882520,"externalId":"511145.b1779","name":"gapA","hasFamily":true,"annotation":"glyceraldehyde-3-phosphate dehydrogenase A","uniprotId":"G3P1_ECOLI"},"6882521":{"id":6882521,"externalId":"511145.b1780","name":"yeaD","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEAD_ECOLI"},"6882522":{"id":6882522,"externalId":"511145.b1781","name":"yeaE","hasFamily":true,"annotation":"aldo-keto reductase, methylglyoxal to acetol, NADPH-dependent","uniprotId":"YEAE_ECOLI"},"6882523":{"id":6882523,"externalId":"511145.b1782","name":"mipA","hasFamily":true,"annotation":"scaffolding protein for murein synthesizing machinery; May serve as a scaffold protein required for the formation of a complex with MrcB/PonB and MltA, this complex could play a role in enlargement and septation of the murein sacculus","uniprotId":"MIPA_ECOLI"},"6882524":{"id":6882524,"externalId":"511145.b1783","name":"yeaG","hasFamily":true,"annotation":"protein kinase, function unknown; autokinase","uniprotId":"YEAG_ECOLI"},"6882525":{"id":6882525,"externalId":"511145.b1784","name":"yeaH","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEAH_ECOLI"},"6882526":{"id":6882526,"externalId":"511145.b1785","name":"yeaI","hasFamily":true,"annotation":"predicted diguanylate cyclase","uniprotId":"YEAI_ECOLI"},"6882527":{"id":6882527,"externalId":"511145.b1786","name":"yeaJ","hasFamily":true,"annotation":"putative diguanylate cyclase; Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"YEAJ_ECOLI"},"6882528":{"id":6882528,"externalId":"511145.b1787","name":"yeaK","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEAK_ECOLI"},"6882529":{"id":6882529,"externalId":"511145.b1788","name":"yoaI","hasFamily":false,"annotation":"predicted protein","uniprotId":"YOAI_ECOLI"},"6882530":{"id":6882530,"externalId":"511145.b1789","name":"yeaL","hasFamily":true,"annotation":"putative inner membrane protein, UPF0756 family","uniprotId":"YEAL_ECOLI"},"6882531":{"id":6882531,"externalId":"511145.b1790","name":"yeaM","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YEAM_ECOLI"},"6882532":{"id":6882532,"externalId":"511145.b1791","name":"yeaN","hasFamily":true,"annotation":"putative transporter","uniprotId":"YEAN_ECOLI"},"6882533":{"id":6882533,"externalId":"511145.b1792","name":"yeaO","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEAO_ECOLI"},"6882534":{"id":6882534,"externalId":"511145.b1793","name":"yoaF","hasFamily":true,"annotation":"conserved outer membrane lipoprotein","uniprotId":"YOAF_ECOLI"},"6882535":{"id":6882535,"externalId":"511145.b1794","name":"yeaP","hasFamily":false,"annotation":"diguanylate cyclase; Catalyzes the condensation of two GTP molecules to the cyclic dinucleotide di-GMP (c-di-GMP), which acts as a secondary messenger (Probable). Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"YEAP_ECOLI"},"6882536":{"id":6882536,"externalId":"511145.b1795","name":"yeaQ","hasFamily":true,"annotation":"conserved protein, UPF0410 family","uniprotId":"YEAQ_ECOLI"},"6882537":{"id":6882537,"externalId":"511145.b1796","name":"yoaG","hasFamily":true,"annotation":"predicted protein","uniprotId":"YOAG_ECOLI"},"6882538":{"id":6882538,"externalId":"511145.b1797","name":"yeaR","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEAR_ECOLI"},"6882539":{"id":6882539,"externalId":"511145.b1798","name":"leuE","hasFamily":true,"annotation":"neutral amino-acid efflux system; Exporter of leucine. Can also transport its natural analog L-alpha-amino-n-butyric acid and some other structurally unrelated amino acids","uniprotId":"LEUE_ECOLI"},"6882540":{"id":6882540,"externalId":"511145.b1799","name":"dmlR","hasFamily":true,"annotation":"DNA-binding transcriptional activator for dmlA; Transcriptional regulator required for the aerobic growth on D-malate as the sole carbon source. Induces the expression of dmlA in response to D-malate or L- or meso-tartrate. Negatively regulates its own expression","uniprotId":"DMLR_ECOLI"},"6882541":{"id":6882541,"externalId":"511145.b1800","name":"dmlA","hasFamily":true,"annotation":"D-malate oxidase, NAD-dependent; probable tartrate dehydrogenase; Catalyzes the NAD(+)-dependent oxidative decarboxylation of D-malate into pyruvate. Is essential for aerobic growth on D- malate as the sole carbon source. But is not required for anaerobic D-malate utilization, although DmlA is expressed and active in those conditions. Appears to be not able to use L- tartrate as a substrate for dehydrogenation instead of D-malate","uniprotId":"DMLA_ECOLI"},"6882542":{"id":6882542,"externalId":"511145.b1801","name":"yeaV","hasFamily":true,"annotation":"putative transporter; Probable transporter whose substrate is unknown. Is not involved in aerobic D-malate transport","uniprotId":"YEAV_ECOLI"},"6882543":{"id":6882543,"externalId":"511145.b1802","name":"yeaW","hasFamily":true,"annotation":"putative 2Fe-2S cluster-containing protein","uniprotId":"YEAW_ECOLI"},"6882544":{"id":6882544,"externalId":"511145.b1803","name":"yeaX","hasFamily":true,"annotation":"putative oxidoreductase","uniprotId":"YEAX_ECOLI"},"6882545":{"id":6882545,"externalId":"511145.b1804","name":"rnd","hasFamily":true,"annotation":"ribonuclease D; Exonuclease involved in the 3' processing of various precursor tRNAs. Initiates hydrolysis at the 3'-terminus of an RNA molecule and releases 5'-mononucleotides (By similarity)","uniprotId":"RND_ECOLI"},"6882546":{"id":6882546,"externalId":"511145.b1805","name":"fadD","hasFamily":true,"annotation":"acyl-CoA synthetase (long-chain-fatty-acid--CoA ligase); Catalyzes the esterification, concomitant with transport, of exogenous long-chain fatty acids into metabolically active CoA thioesters for subsequent degradation or incorporation into phospholipids","uniprotId":"LCFA_ECOLI"},"6882547":{"id":6882547,"externalId":"511145.b1806","name":"yeaY","hasFamily":true,"annotation":"lipoprotein, RpoE-regulated, fucntion unknown","uniprotId":"YEAY_ECOLI"},"6882548":{"id":6882548,"externalId":"511145.b1807","name":"rpf","hasFamily":true,"annotation":"protease specific for Gcp(YgjD), essential for nucleoid maintanence","uniprotId":"YEAZ_ECOLI"},"6882549":{"id":6882549,"externalId":"511145.b1808","name":"yoaA","hasFamily":true,"annotation":"conserved protein with nucleoside triphosphate hydrolase domain","uniprotId":"YOAA_ECOLI"},"6882550":{"id":6882550,"externalId":"511145.b1809","name":"yoaB","hasFamily":true,"annotation":"conserved protein, UPF0076 famiily","uniprotId":"YOAB_ECOLI"},"6882551":{"id":6882551,"externalId":"511145.b1810","name":"yoaC","hasFamily":true,"annotation":"conserved protein, DUF1889 family","uniprotId":"YOAC_ECOLI"},"6882552":{"id":6882552,"externalId":"511145.b1811","name":"yoaH","hasFamily":true,"annotation":"conserved protein","uniprotId":"YOAH_ECOLI"},"6882553":{"id":6882553,"externalId":"511145.b1812","name":"pabB","hasFamily":true,"annotation":"aminodeoxychorismate synthase, subunit I; Catalyzes the biosynthesis of 4-amino-4-deoxychorismate (ADC) from chorismate and glutamine","uniprotId":"PABB_ECOLI"},"6882554":{"id":6882554,"externalId":"511145.b1813","name":"nudL","hasFamily":true,"annotation":"putative NUDIX hydrolase; Probably mediates the hydrolysis of some nucleoside diphosphate derivatives (By similarity)","uniprotId":"NUDL_ECOLI"},"6882555":{"id":6882555,"externalId":"511145.b1814","name":"sdaA","hasFamily":true,"annotation":"L-serine deaminase I; Deaminates also threonine, particularly when it is present in high concentration","uniprotId":"SDHL_ECOLI"},"6882556":{"id":6882556,"externalId":"511145.b1815","name":"adrB","hasFamily":true,"annotation":"predicted phosphodiesterase; May serve as a negative regulator of cellulose synthesis (as has been suggested for S.typhimurium); overexpression inhibits cell aggregation in strains able to produce adhesive curli fimbriae. Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"ADRB_ECOLI"},"6882557":{"id":6882557,"externalId":"511145.b1816","name":"yoaE","hasFamily":true,"annotation":"fused predicted membrane protein/conserved protein","uniprotId":"YOAE_ECOLI"},"6882558":{"id":6882558,"externalId":"511145.b1817","name":"manX","hasFamily":true,"annotation":"fused mannose-specific PTS enzymes: IIA component/IIB component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in mannose transport","uniprotId":"PTNAB_ECOLI"},"6882559":{"id":6882559,"externalId":"511145.b1818","name":"manY","hasFamily":true,"annotation":"mannose-specific enzyme IIC component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in mannose transport","uniprotId":"PTNC_ECOLI"},"6882560":{"id":6882560,"externalId":"511145.b1819","name":"manZ","hasFamily":true,"annotation":"mannose-specific enzyme IID component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in mannose transport","uniprotId":"PTND_ECOLI"},"6882561":{"id":6882561,"externalId":"511145.b1820","name":"yobD","hasFamily":true,"annotation":"inner membrane protein, UPF0266 family","uniprotId":"YOBD_ECOLI"},"6882562":{"id":6882562,"externalId":"511145.b1821","name":"mntP","hasFamily":true,"annotation":"conserved inner membrane protein; Probably functions as a manganese efflux pump (By similarity)","uniprotId":"MNTP_ECOLI"},"6882563":{"id":6882563,"externalId":"511145.b1822","name":"rlmA","hasFamily":true,"annotation":"23S rRNA m(1)G745 methyltransferase; Specifically methylates the guanosine in position 745 of 23S rRNA","uniprotId":"RLMA_ECOLI"},"6882564":{"id":6882564,"externalId":"511145.b1823","name":"cspC","hasFamily":true,"annotation":"stress protein, member of the CspA-family","uniprotId":"CSPC_ECOLI"},"6882565":{"id":6882565,"externalId":"511145.b1824","name":"yobF","hasFamily":true,"annotation":"predicted protein","uniprotId":"YOBF_ECOLI"},"6882566":{"id":6882566,"externalId":"511145.b1825","name":"yebO","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YEBO_ECOLI"},"6882567":{"id":6882567,"externalId":"511145.b1826","name":"mgrB","hasFamily":true,"annotation":"regulatory peptide for PhoPQ, feedback inhibition","uniprotId":"MGRB_ECOLI"},"6882568":{"id":6882568,"externalId":"511145.b1827","name":"kdgR","hasFamily":true,"annotation":"DNA-binding transcriptional regulator f kdgK, kdgT, eda; Transcriptional regulator of the kdgK gene for the kdg kinase and the kdgT gene for the kdg permease","uniprotId":"KDGR_ECOLI"},"6882569":{"id":6882569,"externalId":"511145.b1828","name":"yebQ","hasFamily":true,"annotation":"putative transporter","uniprotId":"YEBQ_ECOLI"},"6882570":{"id":6882570,"externalId":"511145.b1829","name":"htpX","hasFamily":true,"annotation":"putative endopeptidase; Membrane-localized protease able to endoproteolytically degrade overproduced SecY but not YccA, another membrane protein. It seems to cleave SecY at specific cytoplasmic sites. Does not require ATP. Its natural substrate has not been identified. Probably plays a role in the quality control of integral membrane proteins","uniprotId":"HTPX_ECOLI"},"6882571":{"id":6882571,"externalId":"511145.b1830","name":"prc","hasFamily":true,"annotation":"carboxy-terminal protease for penicillin-binding protein 3; Involved in the cleavage of a C-terminal peptide of 11 residues from the precursor form of penicillin-binding protein 3 (PBP3). May be involved in protection of the bacterium from thermal and osmotic stresses","uniprotId":"PRC_ECOLI"},"6882572":{"id":6882572,"externalId":"511145.b1831","name":"proQ","hasFamily":true,"annotation":"RNA chaperone, probable regulator of ProP translation; Not exactly known; may be a structural element that influences the osmotic activation of ProP at a post-translational level (By similarity)","uniprotId":"PROQ_ECOLI"},"6882573":{"id":6882573,"externalId":"511145.b1832","name":"msrC","hasFamily":true,"annotation":"free methionine-(R)-sulfoxide reductase; Catalyzes the reversible oxidation-reduction of the R- enantiomer of free methionine sulfoxide to methionine. Specific for free L-methionine-(R)-S-oxide","uniprotId":"MSRC_ECOLI"},"6882574":{"id":6882574,"externalId":"511145.b1833","name":"yebS","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YEBS_ECOLI"},"6882575":{"id":6882575,"externalId":"511145.b1834","name":"yebT","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEBT_ECOLI"},"6882576":{"id":6882576,"externalId":"511145.b1835","name":"rsmF","hasFamily":true,"annotation":"16S rRNA m(5)C1407 methyltransferase, SAM-dependent; Specifically methylates the cytosine at position 1407 (m5C1407) of 16S rRNA (By similarity)","uniprotId":"RSMF_ECOLI"},"6882577":{"id":6882577,"externalId":"511145.b1836","name":"yebV","hasFamily":true,"annotation":"predicted protein","uniprotId":"YEBV_ECOLI"},"6882578":{"id":6882578,"externalId":"511145.b1837","name":"yebW","hasFamily":true,"annotation":"predicted protein","uniprotId":"YEBW_ECOLI"},"6882579":{"id":6882579,"externalId":"511145.b1838","name":"pphA","hasFamily":true,"annotation":"serine/threonine-specific protein phosphatase 1; Plays a key role in signaling protein misfolding via the CpxR/CPXA transducing system. It also modulates the phosphorylated status of many phosphoproteins in E.coli, some of which acting as major chaperones. Has been shown, in vitro, to act on Ser, Thr and Tyr-phosphorylated substrates","uniprotId":"PRP1_ECOLI"},"6882580":{"id":6882580,"externalId":"511145.b1839","name":"yebY","hasFamily":true,"annotation":"predicted protein","uniprotId":"YEBY_ECOLI"},"6882581":{"id":6882581,"externalId":"511145.b1840","name":"yebZ","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YEBZ_ECOLI"},"6882582":{"id":6882582,"externalId":"511145.b1841","name":"yobA","hasFamily":true,"annotation":"conserved protein","uniprotId":"YOBA_ECOLI"},"6882583":{"id":6882583,"externalId":"511145.b1842","name":"holE","hasFamily":true,"annotation":"DNA polymerase III, theta subunit; DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. This DNA polymerase also exhibits 3' to 5' exonuclease activity","uniprotId":"HOLE_ECOLI"},"6882584":{"id":6882584,"externalId":"511145.b1843","name":"yobB","hasFamily":true,"annotation":"conserved protein","uniprotId":"YOBB_ECOLI"},"6882585":{"id":6882585,"externalId":"511145.b1844","name":"exoX","hasFamily":true,"annotation":"DNA exonuclease X; Capable of degrading both single-strand and double- strand DNA with 3' to 5' polarity. Has higher affinity for ssDNA ends than for dsDNA","uniprotId":"EXOX_ECOLI"},"6882586":{"id":6882586,"externalId":"511145.b1845","name":"ptrB","hasFamily":true,"annotation":"protease II; Cleaves peptide bonds on the C-terminal side of lysyl and argininyl residues","uniprotId":"PTRB_ECOLI"},"6882587":{"id":6882587,"externalId":"511145.b1846","name":"yebE","hasFamily":true,"annotation":"inner membrane protein, DUF533 family","uniprotId":"YEBE_ECOLI"},"6882588":{"id":6882588,"externalId":"511145.b1847","name":"yebF","hasFamily":true,"annotation":"secreted protein","uniprotId":"YEBF_ECOLI"},"6882589":{"id":6882589,"externalId":"511145.b1848","name":"yebG","hasFamily":true,"annotation":"conserved protein regulated by LexA","uniprotId":"YEBG_ECOLI"},"6882590":{"id":6882590,"externalId":"511145.b1849","name":"purT","hasFamily":true,"annotation":"phosphoribosylglycinamide formyltransferase 2; Catalyzes two reactions: the first one is the production of beta-formyl glycinamide ribonucleotide (GAR) from formate, ATP and beta GAR; the second, a side reaction, is the production of acetyl phosphate and ADP from acetate and ATP (By similarity)","uniprotId":"PURT_ECOLI"},"6882591":{"id":6882591,"externalId":"511145.b1850","name":"eda","hasFamily":true,"annotation":"multifunctional 2-keto-3-deoxygluconate 6-phosphate aldolase and 2-keto-4-hydroxyglutarate aldolase and oxaloacetate decarboxylase","uniprotId":"ALKH_ECOLI"},"6882592":{"id":6882592,"externalId":"511145.b1851","name":"edd","hasFamily":true,"annotation":"6-phosphogluconate dehydratase","uniprotId":"EDD_ECOLI"},"6882593":{"id":6882593,"externalId":"511145.b1852","name":"gdh","hasFamily":true,"annotation":"glucose-6-phosphate 1-dehydrogenase","uniprotId":"G6PD_ECOLI"},"6882594":{"id":6882594,"externalId":"511145.b1853","name":"hexR","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator; Represses the expression of the hex regulon (zwf, eda, glp and gap) (By similarity)","uniprotId":"HEXR_ECOLI"},"6882595":{"id":6882595,"externalId":"511145.b1854","name":"pykA","hasFamily":true,"annotation":"pyruvate kinase II","uniprotId":"KPYK2_ECOLI"},"6882596":{"id":6882596,"externalId":"511145.b1855","name":"lpxM","hasFamily":true,"annotation":"myristoyl-acyl carrier protein (ACP)-dependent acyltransferase; Transfers myristate or laurate, activated on ACP, to the lipid IVA moiety of (KDO)2-(lauroyl)-lipid IVA. Decanoyl, palmitoyl, palmitoleoyl, and (R)-3-hydroxymyristoyl-ACP are poor acyl donors. Functions optimally after laurate incorporation by HtrB has taken place. Acylates (KDO)2-(lauroyl)-lipid IVA about 100 times faster than (KDO)2-lipid IVA. Displays a preference for myristoyl-ACP over lauroyl-ACP","uniprotId":"MSBB_ECOLI"},"6882597":{"id":6882597,"externalId":"511145.b1856","name":"yebA","hasFamily":true,"annotation":"putative peptidase; Could be involved in cell wall degradation or formation","uniprotId":"YEBA_ECOLI"},"6882598":{"id":6882598,"externalId":"511145.b1857","name":"yebL","hasFamily":true,"annotation":"zinc transporter subunit: periplasmic-binding component of ABC superfamily; Involved in the high-affinity zinc uptake transport system","uniprotId":"ZNUA_ECOLI"},"6882599":{"id":6882599,"externalId":"511145.b1858","name":"znuC","hasFamily":true,"annotation":"zinc transporter subunit: ATP-binding component of ABC superfamily; Part of the ABC transporter complex ZnuABC involved in zinc import. Responsible for energy coupling to the transport system","uniprotId":"ZNUC_ECOLI"},"6882600":{"id":6882600,"externalId":"511145.b1859","name":"znuB","hasFamily":true,"annotation":"zinc transporter subunit: membrane component of ABC superfamily; Involved in the high-affinity zinc uptake transport system","uniprotId":"ZNUB_ECOLI"},"6882601":{"id":6882601,"externalId":"511145.b1860","name":"ruvB","hasFamily":true,"annotation":"ATP-dependent DNA helicase, component of RuvABC resolvasome; The RuvA-RuvB complex in the presence of ATP renatures cruciform structure in supercoiled DNA with palindromic sequence, indicating that it may promote strand exchange reactions in homologous recombination. RuvAB is a helicase that mediates the Holliday junction migration by localized denaturation and reannealing (By similarity)","uniprotId":"RUVB_ECOLI"},"6882602":{"id":6882602,"externalId":"511145.b1861","name":"ruvA","hasFamily":true,"annotation":"component of RuvABC resolvasome, regulatory subunit; The RuvA-RuvB complex in the presence of ATP renatures cruciform structure in supercoiled DNA with palindromic sequence, indicating that it may promote strand exchange reactions in homologous recombination. RuvAB is a helicase that mediates the Holliday junction migration by localized denaturation and reannealing. RuvA stimulates, in the presence of DNA, the weak ATPase activity of RuvB. Binds both single- and double-stranded DNA (dsDNA). Binds preferentially to supercoiled rather than to relaxed dsDNA","uniprotId":"RUVA_ECOLI"},"6882603":{"id":6882603,"externalId":"511145.b1862","name":"yebB","hasFamily":true,"annotation":"conserved protein, DUF830 family","uniprotId":"YEBB_ECOLI"},"6882604":{"id":6882604,"externalId":"511145.b1863","name":"ruvC","hasFamily":true,"annotation":"component of RuvABC resolvasome, endonuclease; Nuclease that resolves Holliday junction intermediates in genetic recombination. Cleaves the cruciform structure in supercoiled DNA by nicking to strands with the same polarity at sites symmetrically opposed at the junction in the homologous arms and leaves a 5'-terminal phosphate and a 3'-terminal hydroxyl group (By similarity)","uniprotId":"RUVC_ECOLI"},"6882605":{"id":6882605,"externalId":"511145.b1864","name":"yebC","hasFamily":true,"annotation":"conserved protein, UPF0082 family","uniprotId":"YEBC_ECOLI"},"6882606":{"id":6882606,"externalId":"511145.b1865","name":"ntpA","hasFamily":true,"annotation":"dihydroneopterin triphosphate pyrophosphatase; Catalyzes the hydrolysis of dihydroneopterin triphosphate to dihydroneopterin monophosphate and pyrophosphate. Required for efficient folate biosynthesis. Can also hydrolyze nucleoside triphosphates with a preference for dATP","uniprotId":"NUDB_ECOLI"},"6882607":{"id":6882607,"externalId":"511145.b1866","name":"aspS","hasFamily":true,"annotation":"aspartyl-tRNA synthetase","uniprotId":"SYD_ECOLI"},"6882608":{"id":6882608,"externalId":"511145.b1867","name":"yecD","hasFamily":true,"annotation":"isochorismatase family protein","uniprotId":"YECD_ECOLI"},"6882609":{"id":6882609,"externalId":"511145.b1868","name":"yecE","hasFamily":true,"annotation":"conserved protein, UPF0759 family","uniprotId":"YECE_ECOLI"},"6882610":{"id":6882610,"externalId":"511145.b1869","name":"yecN","hasFamily":true,"annotation":"inner membrane protein, MAPEG family","uniprotId":"YECN_ECOLI"},"6882611":{"id":6882611,"externalId":"511145.b1870","name":"cmoA","hasFamily":true,"annotation":"tRNA cmo(5)U34 methyltransferase, SAM-dependent; Catalyzes the conversion of 5-methoxyuridine (mo5U) to uridine-5-oxyacetic acid (cmo5U) at position 34 in tRNA. May also participate in the methylation of uridine-5-oxyacetic acid (cmo5U) to uridine-5-oxyacetic acid methyl ester (mcmo5U) (By similarity)","uniprotId":"CMOA_ECOLI"},"6882612":{"id":6882612,"externalId":"511145.b1871","name":"cmoB","hasFamily":true,"annotation":"tRNA mo(5)U34 methyltransferase, SAM-dependent; Catalyzes the conversion of 5-hydroxyuridine (ho5U) to 5-methoxyuridine (mo5U) at position 34 in tRNA (By similarity)","uniprotId":"CMOB_ECOLI"},"6882613":{"id":6882613,"externalId":"511145.b1872","name":"bisZ","hasFamily":true,"annotation":"trimethylamine N-oxide reductase system III, catalytic subunit; Reduces trimethylamine-N-oxide (TMAO) into trimethylamine; an anaerobic reaction coupled to energy-yielding reactions. Can also reduce other N- and S-oxide compounds such as 4-methylmorpholine-N-oxide and biotin sulfoxide (BSO), but with a lower catalytic efficiency","uniprotId":"TORZ_ECOLI"},"6882614":{"id":6882614,"externalId":"511145.b1873","name":"torY","hasFamily":false,"annotation":"TMAO reductase III (TorYZ), cytochrome c-type subunit; Part of the anaerobic respiratory chain of trimethylamine-N-oxide reductase TorZ. Required for electron transfer to the TorZ terminal enzyme","uniprotId":"TORY_ECOLI"},"6882615":{"id":6882615,"externalId":"511145.b1874","name":"cutC","hasFamily":true,"annotation":"copper homeostasis protein; Involved in copper homeostasis (By similarity)","uniprotId":"CUTC_ECOLI"},"6882616":{"id":6882616,"externalId":"511145.b1875","name":"yecM","hasFamily":true,"annotation":"putative metal-binding enzyme","uniprotId":"YECM_ECOLI"},"6882617":{"id":6882617,"externalId":"511145.b1876","name":"argS","hasFamily":true,"annotation":"arginyl-tRNA synthetase","uniprotId":"SYR_ECOLI"},"6882618":{"id":6882618,"externalId":"511145.b1877","name":"yecT","hasFamily":false,"annotation":"predicted protein","uniprotId":"YECT_ECOLI"},"6882619":{"id":6882619,"externalId":"511145.b1878","name":"flhE","hasFamily":true,"annotation":"conserved protein; Not essential for flagellar formation and function","uniprotId":"FLHE_ECOLI"},"6882620":{"id":6882620,"externalId":"511145.b1879","name":"flhA","hasFamily":true,"annotation":"putative flagellar export pore protein; Required for formation of the rod structure of the flagellar apparatus. Together with FliI and FliH, may constitute the export apparatus of flagellin","uniprotId":"FLHA_ECOLI"},"6882621":{"id":6882621,"externalId":"511145.b1880","name":"flhB","hasFamily":true,"annotation":"flagellin export apparatus, substrate specificity protein; Required for formation of the rod structure in the basal body of the flagellar apparatus. Together with FliI and FliH, may constitute the export apparatus of flagellin","uniprotId":"FLHB_ECOLI"},"6882622":{"id":6882622,"externalId":"511145.b1881","name":"cheZ","hasFamily":true,"annotation":"chemotaxis regulator, protein phosphatase for CheY; Plays an important role in bacterial chemotaxis signal transduction pathway by accelerating the dephosphorylation of phosphorylated CheY (CheY-P) (By similarity)","uniprotId":"CHEZ_ECOLI"},"6882623":{"id":6882623,"externalId":"511145.b1882","name":"cheY","hasFamily":true,"annotation":"chemotaxis regulator transmitting signal to flagellar motor component; Involved in the transmission of sensory signals from the chemoreceptors to the flagellar motors. In its active (phosphorylated or acetylated) form, CheY exhibits enhanced binding to a switch component, FliM, at the flagellar motor which induces a change from counterclockwise to clockwise flagellar rotation. Overexpression of CheY in association with MotA and MotB improves motility of a ycgR disruption, suggesting there is an interaction (direct or indirect) between the c-di-GMP-binding flagellar brake protein and th [...] ","uniprotId":"CHEY_ECOLI"},"6882624":{"id":6882624,"externalId":"511145.b1883","name":"cheB","hasFamily":true,"annotation":"fused chemotaxis regulator: protein-glutamate methylesterase in two-component regulatory system with CheA; Involved in the modulation of the chemotaxis system; catalyzes the demethylation of specific methylglutamate residues introduced into the chemoreceptors (methyl-accepting chemotaxis proteins) by CheR (By similarity)","uniprotId":"CHEB_ECOLI"},"6882625":{"id":6882625,"externalId":"511145.b1884","name":"cheR","hasFamily":true,"annotation":"chemotaxis regulator, protein-glutamate methyltransferase; Methylation of the membrane-bound methyl-accepting chemotaxis proteins (MCP) to form gamma-glutamyl methyl ester residues in MCP (By similarity)","uniprotId":"CHER_ECOLI"},"6882626":{"id":6882626,"externalId":"511145.b1885","name":"tap","hasFamily":true,"annotation":"methyl-accepting protein IV; Mediates taxis toward dipeptides via an interaction with the periplasmic dipeptide-binding protein","uniprotId":"MCP4_ECOLI"},"6882627":{"id":6882627,"externalId":"511145.b1886","name":"ECs2596","hasFamily":true,"annotation":"methyl-accepting chemotaxis protein II; Receptor for the attractant L-aspartate and related amino and dicarboxylic acids. Tar also mediates taxis to the attractant maltose via an interaction with the periplasmic maltose binding protein. Tar mediates taxis away from the repellents cobalt and nickel","uniprotId":"MCP2_ECOLI"},"6882628":{"id":6882628,"externalId":"511145.b1887","name":"cheW","hasFamily":true,"annotation":"purine-binding chemotaxis protein; Involved in the transmission of sensory signals from the chemoreceptors to the flagellar motors. It physically bridges CheA to the MCPs (methyl-accepting chemotaxis proteins) to allow regulated phosphotransfer to CheY and CheB","uniprotId":"CHEW_ECOLI"},"6882629":{"id":6882629,"externalId":"511145.b1888","name":"cheA","hasFamily":true,"annotation":"fused chemotactic sensory histidine kinase in two-component regulatory system with CheB and CheY: sensory histidine kinase/signal sensing protein; Involved in the transmission of sensory signals from the chemoreceptors to the flagellar motors. CheA is autophosphorylated; it can transfer its phosphate group to either CheB or CheY","uniprotId":"CHEA_ECOLI"},"6882630":{"id":6882630,"externalId":"511145.b1889","name":"motB","hasFamily":true,"annotation":"protein that enables flagellar motor rotation; MotA and MotB comprise the stator element of the flagellar motor complex. Required for the rotation of the flagellar motor. Probably a linker that fastens the torque- generating machinery to the cell wall. Overexpression of this protein with MotA improves motility in a yhjH disruption, (a c-di- GMP phosphodiesterase) suggesting there is an interaction (direct or indirect) between the c-di-GMP-binding flagellar brake protein YcgR and the flagellar stator","uniprotId":"MOTB_ECOLI"},"6882631":{"id":6882631,"externalId":"511145.b1890","name":"motA","hasFamily":true,"annotation":"proton conductor component of flagella motor; MotA and MotB comprise the stator element of the flagellar motor complex. Required for rotation of the flagellar motor. Probable transmembrane proton channel. Overexpression of MotA, with or without MotB, restores motility in a yhjH disruption, (a c-di-GMP phosphodiesterase) suggesting there is an interaction (direct or indirect) between the c-di-GMP-binding flagellar brake protein YcgR and the flagellar stator","uniprotId":"MOTA_ECOLI"},"6882632":{"id":6882632,"externalId":"511145.b1891","name":"flhC","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator with FlhD; Functions in complex with FlhD as a master transcriptional regulator that regulates transcription of several flagellar and non-flagellar operons by binding to their promoter region. Activates expression of class 2 flagellar genes, including fliA, which is a flagellum-specific sigma factor that turns on the class 3 genes. Also regulates genes whose products function in a variety of physiological pathways","uniprotId":"FLHC_ECOLI"},"6882633":{"id":6882633,"externalId":"511145.b1892","name":"flhD","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator with FlhC; Functions in complex with FlhC as a master transcriptional regulator that regulates transcription of several flagellar and non-flagellar operons by binding to their promoter region. Activates expression of class 2 flagellar genes, including fliA, which is a flagellum-specific sigma factor that turns on the class 3 genes. Also regulates genes whose products function in a variety of physiological pathways (By similarity)","uniprotId":"FLHD_ECOLI"},"6882634":{"id":6882634,"externalId":"511145.b1893","name":"b1893","hasFamily":true,"annotation":"IS1 transposase B"},"6882635":{"id":6882635,"externalId":"511145.b1894","name":"b1894","hasFamily":false,"annotation":"IS1 repressor TnpA"},"6882636":{"id":6882636,"externalId":"511145.b1895","name":"uspC","hasFamily":true,"annotation":"universal stress protein; Required for resistance to DNA-damaging agents","uniprotId":"USPC_ECOLI"},"6882637":{"id":6882637,"externalId":"511145.b1896","name":"otsA","hasFamily":true,"annotation":"trehalose-6-phosphate synthase; Catalyzes the transfer of glucose from UDP-glucose to glucose-6-phosphate to form alpha,alpha-1,1 trehalose-6-phosphate. Acts with retention of the anomeric configuration of the UDP-sugar donor. Essential for viability of the cells at low temperatures and at elevated osmotic strength","uniprotId":"OTSA_ECOLI"},"6882638":{"id":6882638,"externalId":"511145.b1897","name":"tpp","hasFamily":true,"annotation":"trehalose-6-phosphate phosphatase, biosynthetic; Removes the phosphate from trehalose 6-phosphate (Tre6P) to produce free trehalose. Also catalyzes the dephosphorylation of glucose-6-phosphate (Glu6P) and 2-deoxyglucose-6-phosphate (2dGlu6P)","uniprotId":"OTSB_ECOLI"},"6882639":{"id":6882639,"externalId":"511145.b1900","name":"araG","hasFamily":true,"annotation":"fused L-arabinose transporter subunits of ABC superfamily: ATP-binding components; Part of the ABC transporter complex AraFGH involved in arabinose import. Responsible for energy coupling to the transport system (Probable)","uniprotId":"ARAG_ECOLI"},"6882640":{"id":6882640,"externalId":"511145.b1901","name":"araF","hasFamily":true,"annotation":"L-arabinose transporter subunit; Involved in the high-affinity L-arabinose membrane transport system. Binds with high affinity to arabinose, but can also bind D-galactose (approximately 2-fold reduction) and D- fucose (approximately 40-fold reduction)","uniprotId":"ARAF_ECOLI"},"6882641":{"id":6882641,"externalId":"511145.b1902","name":"ftnB","hasFamily":true,"annotation":"ferritin B, probable ferrous iron reservoir","uniprotId":"FTNB_ECOLI"},"6882642":{"id":6882642,"externalId":"511145.b1904","name":"yecR","hasFamily":true,"annotation":"lipoprotein, function unknown","uniprotId":"YECR_ECOLI"},"6882643":{"id":6882643,"externalId":"511145.b1905","name":"ftnA","hasFamily":true,"annotation":"ferritin iron storage protein (cytoplasmic); Iron-storage protein","uniprotId":"FTNA_ECOLI"},"6882644":{"id":6882644,"externalId":"511145.b1906","name":"yecH","hasFamily":true,"annotation":"predicted protein","uniprotId":"YECH_ECOLI"},"6882645":{"id":6882645,"externalId":"511145.b1907","name":"tyrP","hasFamily":true,"annotation":"tyrosine transporter; Involved in transporting tyrosine across the cytoplasmic membrane","uniprotId":"TYRP_ECOLI"},"6882646":{"id":6882646,"externalId":"511145.b1908","name":"yecA","hasFamily":true,"annotation":"conserved protein, UPF0149 family","uniprotId":"YECA_ECOLI"},"6882647":{"id":6882647,"externalId":"511145.b1912","name":"pgsA","hasFamily":true,"annotation":"phosphatidylglycerophosphate synthetase; This protein catalyzes the committed step to the synthesis of the acidic phospholipids (By similarity)","uniprotId":"PGSA_ECOLI"},"6882648":{"id":6882648,"externalId":"511145.b1913","name":"uvrC","hasFamily":true,"annotation":"excinuclease UvrABC, endonuclease subunit; The UvrABC repair system catalyzes the recognition and processing of DNA lesions. UvrC both incises the 5' and 3' sides of the lesion. The N-terminal half is responsible for the 3' incision and the C-terminal half is responsible for the 5' incision (By similarity)","uniprotId":"UVRC_ECOLI"},"6882649":{"id":6882649,"externalId":"511145.b1914","name":"uvrY","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with BarA; Member of the two-component regulatory system UvrY/BarA involved in the regulation of carbon metabolism via the CsrA/CsrB regulatory system. UvrY activates the transcription of the untranslated csrB RNA and of barA, in an autoregulatory loop. Mediates the effects of CsrA on csrB RNA by BarA-dependent and BarA-independent mechanisms","uniprotId":"UVRY_ECOLI"},"6882650":{"id":6882650,"externalId":"511145.b1915","name":"yecF","hasFamily":true,"annotation":"conserved protein, DUF2594 family","uniprotId":"YECF_ECOLI"},"6882651":{"id":6882651,"externalId":"511145.b1916","name":"sdiA","hasFamily":true,"annotation":"quorum-sensing transcriptional activator; Activates cell division by specifically increasing transcription from one of the two promoters that lie immediately upstream of the ftsQAZ gene cluster. Activates ydiV expression in response to extracellular autoinducer AI-1 (Vibrio fischeri autoinducer oxoC6)","uniprotId":"SDIA_ECOLI"},"6882652":{"id":6882652,"externalId":"511145.b1917","name":"yecC","hasFamily":true,"annotation":"putative transporter subunit: ATP-binding component of ABC superfamily; Probably part of a binding-protein-dependent transport system YecCS for an amino acid. Probably responsible for energy coupling to the transport system","uniprotId":"YECC_ECOLI"},"6882653":{"id":6882653,"externalId":"511145.b1918","name":"yecS","hasFamily":true,"annotation":"inner membrane protein, predicted transporter subunit: membrane component of ABC superfamily; Probably part of the binding-protein-dependent transport system YecCS for an amino acid; probably responsible for the translocation of the substrate across the membrane","uniprotId":"YECS_ECOLI"},"6882654":{"id":6882654,"externalId":"511145.b1919","name":"dcyD","hasFamily":true,"annotation":"D-cysteine desulfhydrase, PLP-dependent; Catalyzes the alpha,beta-elimination reaction of D- cysteine and of several D-cysteine derivatives. It could be a defense mechanism against D-cysteine. Can also catalyze the degradation of 3-chloro-D-alanine","uniprotId":"DCYD_ECOLI"},"6882655":{"id":6882655,"externalId":"511145.b1920","name":"fliY","hasFamily":true,"annotation":"cystine transporter subunit; Part of a binding-protein-dependent transport system for cystine","uniprotId":"FLIY_ECOLI"},"6882656":{"id":6882656,"externalId":"511145.b1921","name":"fliZ","hasFamily":true,"annotation":"RpoS antagonist; putative regulator of FliA activity; During the post-exponential growth phase transiently interferes with RpoS (sigma S) activity without affecting expression of RpoS itself. It is probably not an anti-sigma factor as its overexpression is detrimental in rapidly growing cells where there is almost no sigma S factor. There is a strong overlap between Crl-activated genes and FliZ-down-regulated genes. FliZ acts as a timing device for expression of the genes for the adhesive curli fimbriae by indirectly decreasing expression of the curli regulator CsgD","uniprotId":"FLIZ_ECOLI"},"6882657":{"id":6882657,"externalId":"511145.b1922","name":"fliA","hasFamily":true,"annotation":"RNA polymerase, sigma 28 (sigma F) factor; Sigma factors are initiation factors that promote the attachment of RNA polymerase to specific initiation sites and are then released. This alternative sigma factor is specific for class 3 flagellar operons","uniprotId":"FLIA_ECOLI"},"6882658":{"id":6882658,"externalId":"511145.b1923","name":"fliC","hasFamily":true,"annotation":"flagellar filament structural protein (flagellin); Flagellin is the subunit protein which polymerizes to form the filaments of bacterial flagella","uniprotId":"FLIC_ECOLI"},"6882659":{"id":6882659,"externalId":"511145.b1924","name":"fliD","hasFamily":true,"annotation":"flagellar filament capping protein; Required for the morphogenesis and for the elongation of the flagellar filament by facilitating polymerization of the flagellin monomers at the tip of growing filament. Forms a capping structure, which prevents flagellin subunits (transported through the central channel of the flagellum) from leaking out without polymerization at the distal end","uniprotId":"FLID_ECOLI"},"6882660":{"id":6882660,"externalId":"511145.b1925","name":"fliS","hasFamily":true,"annotation":"flagellar protein potentiates polymerization","uniprotId":"FLIS_ECOLI"},"6882661":{"id":6882661,"externalId":"511145.b1926","name":"fliT","hasFamily":true,"annotation":"putative chaperone; Dual-function protein that regulates the transcription of class 2 flagellar operons and that also acts as an export chaperone for the filament-capping protein FliD. As a transcriptional regulator, acts as an anti-FlhDC factor; it directly binds FlhC, thus inhibiting the binding of the FlhC/FlhD complex to class 2 promoters, resulting in decreased expression of class 2 flagellar operons. As a chaperone, effects FliD transition to the membrane by preventing its premature polymerization, and by directing it to the export apparatus (By similarity)","uniprotId":"FLIT_ECOLI"},"6882662":{"id":6882662,"externalId":"511145.b1927","name":"amyA","hasFamily":true,"annotation":"cytoplasmic alpha-amylase","uniprotId":"AMY2_ECOLI"},"6882663":{"id":6882663,"externalId":"511145.b1928","name":"yedD","hasFamily":true,"annotation":"lipoprotein","uniprotId":"YEDD_ECOLI"},"6882664":{"id":6882664,"externalId":"511145.b1929","name":"yedE","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YEDE_ECOLI"},"6882665":{"id":6882665,"externalId":"511145.b1930","name":"yedF","hasFamily":false,"annotation":"conserved protein, UPF0033 family","uniprotId":"YEDF_ECOLI"},"6882666":{"id":6882666,"externalId":"511145.b1931","name":"yedK","hasFamily":true,"annotation":"predicted protein","uniprotId":"YEDK_ECOLI"},"6882667":{"id":6882667,"externalId":"511145.b1932","name":"yedL","hasFamily":true,"annotation":"putative acyltransferase","uniprotId":"YEDL_ECOLI"},"6882668":{"id":6882668,"externalId":"511145.b1937","name":"fliE","hasFamily":true,"annotation":"flagellar basal-body component","uniprotId":"FLIE_ECOLI"},"6882669":{"id":6882669,"externalId":"511145.b1938","name":"fliF","hasFamily":true,"annotation":"flagellar basal-body MS-ring and collar protein; The M ring may be actively involved in energy transduction (By similarity)","uniprotId":"FLIF_ECOLI"},"6882670":{"id":6882670,"externalId":"511145.b1939","name":"fliG","hasFamily":true,"annotation":"flagellar motor switching and energizing component; FliG is one of three proteins (FliG, FliN, FliM) that forms the rotor-mounted switch complex (C ring), located at the base of the basal body. This complex interacts with the CheY and CheZ chemotaxis proteins, in addition to contacting components of the motor that determine the direction of flagellar rotation (By similarity)","uniprotId":"FLIG_ECOLI"},"6882671":{"id":6882671,"externalId":"511145.b1940","name":"fliH","hasFamily":true,"annotation":"negative regulator of FliI ATPase activity; Needed for flagellar regrowth and assembly","uniprotId":"FLIH_ECOLI"},"6882672":{"id":6882672,"externalId":"511145.b1941","name":"fliI","hasFamily":true,"annotation":"flagellum-specific ATP synthase; Probable catalytic subunit of a protein translocase for flagellum-specific export, or a proton translocase involved in local circuits at the flagellum. May be involved in a specialized protein export pathway that proceeds without signal peptide cleavage","uniprotId":"FLII_ECOLI"},"6882673":{"id":6882673,"externalId":"511145.b1942","name":"fliJ","hasFamily":true,"annotation":"flagellar protein; The FliJ protein is a flagellar protein that affects chemotactic events. Mutations in FliJ results in failure to respond to chemotactic stimuli","uniprotId":"FLIJ_ECOLI"},"6882674":{"id":6882674,"externalId":"511145.b1943","name":"fliK","hasFamily":true,"annotation":"flagellar hook-length control protein; Controls the length of the flagellar hook","uniprotId":"FLIK_ECOLI"},"6882675":{"id":6882675,"externalId":"511145.b1944","name":"fliL","hasFamily":true,"annotation":"flagellar biosynthesis protein; Controls the rotational direction of flagella during chemotaxis","uniprotId":"FLIL_ECOLI"},"6882676":{"id":6882676,"externalId":"511145.b1945","name":"fliM","hasFamily":true,"annotation":"flagellar motor switching and energizing component; FliM is one of three proteins (FliG, FliN, FliM) that forms the rotor-mounted switch complex (C ring), located at the base of the basal body. This complex interacts with the CheY and CheZ chemotaxis proteins, in addition to contacting components of the motor that determine the direction of flagellar rotation (By similarity)","uniprotId":"FLIM_ECOLI"},"6882677":{"id":6882677,"externalId":"511145.b1946","name":"fliN","hasFamily":true,"annotation":"flagellar motor switching and energizing component; FliN is one of three proteins (FliG, FliN, FliM) that form a switch complex that is proposed to be located at the base of the basal body. This complex interacts with the CheY and CheZ chemotaxis proteins, in addition to contacting components of the motor that determine the direction of flagellar rotation","uniprotId":"FLIN_ECOLI"},"6882678":{"id":6882678,"externalId":"511145.b1947","name":"fliO","hasFamily":true,"annotation":"flagellar biosynthesis protein","uniprotId":"FLIO_ECOLI"},"6882679":{"id":6882679,"externalId":"511145.b1948","name":"fliP","hasFamily":true,"annotation":"flagellar biosynthesis protein; Plays a role in the flagellum-specific transport system","uniprotId":"FLIP_ECOLI"},"6882680":{"id":6882680,"externalId":"511145.b1949","name":"fliQ","hasFamily":true,"annotation":"flagellar biosynthesis protein; Required for the assembly of the rivet at the earliest stage of flagellar biosynthesis","uniprotId":"FLIQ_ECOLI"},"6882681":{"id":6882681,"externalId":"511145.b1950","name":"fliR","hasFamily":true,"annotation":"flagellar export pore protein; Role in flagellar biosynthesis (By similarity)","uniprotId":"FLIR_ECOLI"},"6882682":{"id":6882682,"externalId":"511145.b1951","name":"rcsA","hasFamily":true,"annotation":"DNA-binding transcriptional activator, co-regulator with RcsB; Positive regulator of capsular polysaccharide synthesis. RcsA and RcsB form a complex to promote transcription of the genes for capsule synthesis","uniprotId":"RCSA_ECOLI"},"6882683":{"id":6882683,"externalId":"511145.b1952","name":"dsrB","hasFamily":true,"annotation":"predicted protein","uniprotId":"DSRB_ECOLI"},"6882684":{"id":6882684,"externalId":"511145.b1953","name":"yodD","hasFamily":true,"annotation":"predicted protein","uniprotId":"YODD_ECOLI"},"6882685":{"id":6882685,"externalId":"511145.b1955","name":"yedP","hasFamily":true,"annotation":"putative mannosyl-3-phosphoglycerate phosphatase","uniprotId":"MPGP_ECOLI"},"6882686":{"id":6882686,"externalId":"511145.b1956","name":"yedQ","hasFamily":true,"annotation":"predicted diguanylate cyclase; Involved in the regulation of cellulose production. Cyclic-di-GMP is a second messenger which controls cell surface- associated traits in bacteria","uniprotId":"YEDQ_ECOLI"},"6882687":{"id":6882687,"externalId":"511145.b1957","name":"yodC","hasFamily":true,"annotation":"predicted protein","uniprotId":"YODC_ECOLI"},"6882688":{"id":6882688,"externalId":"511145.b1958","name":"yedI","hasFamily":true,"annotation":"conserved inner membrane protein","uniprotId":"YEDI_ECOLI"},"6882689":{"id":6882689,"externalId":"511145.b1959","name":"yedA","hasFamily":true,"annotation":"amino acid exporter for phenylalanine, threonine","uniprotId":"YEDA_ECOLI"},"6882690":{"id":6882690,"externalId":"511145.b1960","name":"vsr","hasFamily":true,"annotation":"DNA mismatch endonuclease of very short patch repair; Deamination of 5-methylcytosine in DNA results in T/G mismatches. If unrepaired, these mismatches can lead to C-to-C transition mutations. The very short patch (VSP) repair process in E.coli counteracts the mutagenic process by repairing the mismatches in favor of the G-containing strand. This enzyme is an endonuclease that nicks double-stranded DNA within the sequence CT(AT)GN or NT(AT)GG next to the thymidine residue that is mismatched to 2'-deoxyguanosine. The incision is mismatch- dependent and strand-specific","uniprotId":"VSR_ECOLI"},"6882691":{"id":6882691,"externalId":"511145.b1961","name":"dcm","hasFamily":true,"annotation":"DNA cytosine methyltransferase; This methylase recognizes the double-stranded sequence CCWGG, causes specific methylation on C-2 on both strands","uniprotId":"DCM_ECOLI"},"6882692":{"id":6882692,"externalId":"511145.b1962","name":"yedJ","hasFamily":true,"annotation":"putative hydrolase, HD superfamily","uniprotId":"YEDJ_ECOLI"},"6882693":{"id":6882693,"externalId":"511145.b1963","name":"yedR","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YEDR_ECOLI"},"6882694":{"id":6882694,"externalId":"511145.b1967","name":"hchA","hasFamily":true,"annotation":"Glyoxalase III and Hsp31 molecular chaperone; Functions as a holding molecular chaperone (holdase) which stabilizes unfolding intermediates and rapidly releases them in an active form once stress has abated. Plays an important role in protecting cells from severe heat shock and starvation, as well as in acid resistance of stationary-phase cells. It uses temperature-induced exposure of structured hydrophobic domains to capture and stabilizes early unfolding and denatured protein intermediates under severe thermal stress. Catalyzes the conversion of methylglyoxal (MG) to D-lactate in a s [...] ","uniprotId":"HCHA_ECOLI"},"6882695":{"id":6882695,"externalId":"511145.b1968","name":"yedV","hasFamily":true,"annotation":"putative sensory kinase in two-component regulatory system with YedW; Probable member of a two-component regulatory system YedW/YedV. Activates YedW by phosphorylation","uniprotId":"YEDV_ECOLI"},"6882696":{"id":6882696,"externalId":"511145.b1969","name":"yedW","hasFamily":true,"annotation":"putative DNA-binding response regulator in two-component system with YedV; Probable member of a two-component regulatory system YedW/YedV","uniprotId":"YEDW_ECOLI"},"6882697":{"id":6882697,"externalId":"511145.b1970","name":"hiuH","hasFamily":true,"annotation":"Hydroxyisourate hydrolase; Catalyzes the hydrolysis of 5-hydroxyisourate (HIU) to 2-oxo-4-hydroxy-4-carboxy-5-ureidoimidazoline (OHCU)","uniprotId":"HIUH_ECOLI"},"6882698":{"id":6882698,"externalId":"511145.b1971","name":"yedY","hasFamily":true,"annotation":"membrane-anchored, periplasmic TMAO, DMSO reductase; The exact function is not known. Can catalyze the reduction of a variety of substrates like dimethyl sulfoxide, trimethylamine N-oxide, phenylmethyl sulfoxide and L-methionine sulfoxide. Cannot reduce cyclic N-oxides. Shows no activity as sulfite oxidase (By similarity)","uniprotId":"YEDY_ECOLI"},"6882699":{"id":6882699,"externalId":"511145.b1972","name":"yedZ","hasFamily":true,"annotation":"inner membrane heme subunit for periplasmic YedYZ reductase; Is probably involved in the electron transfer between its heme and the molybdenum cofactor of the reductase catalytic subunit YedY (By similarity)","uniprotId":"YEDZ_ECOLI"},"6882700":{"id":6882700,"externalId":"511145.b1973","name":"yodA","hasFamily":true,"annotation":"zinc and cadmium binding protein, periplasmic; May function as a periplasmic zinc chaperone or mediate direct transport of zinc from the periplasm to the cytoplasm under zinc-limited conditions. Binds zinc with high affinity, and can also bind cadmium, mercury or nickel. Preferentially binds Zn(2+) over Cd(2+). Contains one high affinity metal binding site, and can bind additional metal ions at other sites","uniprotId":"ZINT_ECOLI"},"6882701":{"id":6882701,"externalId":"511145.b1974","name":"yodB","hasFamily":true,"annotation":"cytochrome b561 homolog","uniprotId":"C56H_ECOLI"},"6882702":{"id":6882702,"externalId":"511145.b1976","name":"mtfA","hasFamily":true,"annotation":"anti-repressor for DgsA(Mlc); Involved in the regulation of ptsG expression by binding and inactivating Mlc (By similarity)","uniprotId":"MTFA_ECOLI"},"6882703":{"id":6882703,"externalId":"511145.b1978","name":"yeeJ","hasFamily":true,"annotation":"probable adhesin","uniprotId":"YEEJ_ECOLI"},"6882704":{"id":6882704,"externalId":"511145.b1981","name":"shiA","hasFamily":true,"annotation":"shikimate transporter","uniprotId":"SHIA_ECOLI"},"6882705":{"id":6882705,"externalId":"511145.b1982","name":"amn","hasFamily":true,"annotation":"AMP nucleosidase; Involved in regulation of AMP concentrations","uniprotId":"AMN_ECOLI"},"6882706":{"id":6882706,"externalId":"511145.b1983","name":"yeeN","hasFamily":true,"annotation":"conserved protein, UPF0082 family","uniprotId":"YEEN_ECOLI"},"6882707":{"id":6882707,"externalId":"511145.b1985","name":"yeeO","hasFamily":true,"annotation":"putative multdrug exporter, MATE family","uniprotId":"YEEO_ECOLI"},"6882708":{"id":6882708,"externalId":"511145.b1987","name":"cbl","hasFamily":false,"annotation":"DNA-binding transcriptional activator for the ssuEADCB and tauABCD operons; May be an accessory regulatory protein within the cys regulon","uniprotId":"CBL_ECOLI"},"6882709":{"id":6882709,"externalId":"511145.b1988","name":"nac","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator of nitrogen assimilation; Transcriptional activator for the hut, put and ure operons and repressor for the gdh and gltB operons in response to nitrogen limitation. Negative regulator of its own expression (By similarity)","uniprotId":"NAC_ECOLI"},"6882710":{"id":6882710,"externalId":"511145.b1990","name":"erfK","hasFamily":true,"annotation":"L,D-transpeptidase linking Lpp to murein; Responsible, at least in part, for anchoring of the major outer membrane lipoprotein (Lpp, also known as the Braun lipoprotein) to the peptidoglycan via a meso-diaminopimelyl-L-Lys- bond on the terminal residue of Lpp","uniprotId":"ERFK_ECOLI"},"6882711":{"id":6882711,"externalId":"511145.b1991","name":"cobT","hasFamily":true,"annotation":"nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase; Catalyzes the synthesis of alpha-ribazole-5'-phosphate from nicotinate mononucleotide (NAMN) and 5,6- dimethylbenzimidazole (DMB) (By similarity)","uniprotId":"COBT_ECOLI"},"6882712":{"id":6882712,"externalId":"511145.b1992","name":"cobS","hasFamily":true,"annotation":"cobalamin synthase; Joins Ado-cobinamide-GDP and alpha-ribazole to generate adenosylcobalamin (Ado-cobalamin) (By similarity)","uniprotId":"COBS_ECOLI"},"6882713":{"id":6882713,"externalId":"511145.b1993","name":"cobU","hasFamily":true,"annotation":"bifunctional cobinamide kinase/cobinamide phosphate guanylyltransferase; ATP-dependent phosphorylation of adenosylcobinamide and adds GMP to adenosylcobinamide phosphate","uniprotId":"COBU_ECOLI"},"6882714":{"id":6882714,"externalId":"511145.b1994","name":"insH6","hasFamily":true,"annotation":"IS5 transposase and trans-activator; Involved in the transposition of the insertion sequence IS5","uniprotId":"INSH6_ECOLI"},"6882715":{"id":6882715,"externalId":"511145.b1996","name":"b1996","hasFamily":true,"annotation":"IS2 transposase TnpB"},"6882716":{"id":6882716,"externalId":"511145.b1997","name":"b1997","hasFamily":true,"annotation":"IS2 repressor TnpA"},"6882717":{"id":6882717,"externalId":"511145.b2000","name":"flu","hasFamily":false,"annotation":"CP4-44 prophage; antigen 43 (Ag43) phase-variable biofilm formation autotransporter; Controls colony form variation and autoaggregation. May function as an adhesin","uniprotId":"AG43_ECOLI"},"6882718":{"id":6882718,"externalId":"511145.b2001","name":"yeeR","hasFamily":false,"annotation":"CP4-44 prophage","uniprotId":"YEER_ECOLI"},"6882719":{"id":6882719,"externalId":"511145.b2002","name":"yeeS","hasFamily":false,"annotation":"CP4-44 prophage","uniprotId":"YEES_ECOLI"},"6882720":{"id":6882720,"externalId":"511145.b2003","name":"yeeT","hasFamily":false,"annotation":"CP4-44 prophage","uniprotId":"YEET_ECOLI"},"6882721":{"id":6882721,"externalId":"511145.b2004","name":"cbeA","hasFamily":false,"annotation":"CP4-44 prophage; cytoskeleton bundling-enhancing factor A; Antitoxin component of a toxin-antitoxin (TA) module. Labile antitoxin that counteracts the effect of the CbtA (YeeV) toxin. It does not bind to the cognate toxin but instead binds to MreB and FtsZ (the toxin targets), enhancing their polymerization by forming higher-order bundles; it is probably retained in the MreB and FtsZ filament bundles. The mechanism has been proposed to require intergenic DNA, in cis, between the cbeA (yeeU) and cbta (yeeV) genes (PubMed:14594833); the intergenic region was not found to be necessary in  [...] ","uniprotId":"CBEA_ECOLI"},"6882722":{"id":6882722,"externalId":"511145.b2005","name":"cbtA","hasFamily":false,"annotation":"CP4-44 prophage; Toxic component of a toxin-antitoxin (TA) module. Interacts with cytoskeletal proteins FtsZ and MreB; inhibits FtsZ GTP-dependent polymerization and GTPase activity as well as MreB ATP-dependent polymerization. Binds to both the N- and C-terminus of FtsZ, likely blocking its polymerization. Overexpression results in inhibition of growth in liquid cultures and decrease in colony formation; these effects are overcome by concomitant expression of antitoxin CbeA (YeeU). In other experiments after 6 hours cells are lemon-shaped and by 24 hours those that have not lysed are  [...] ","uniprotId":"CBTA_ECOLI"},"6882723":{"id":6882723,"externalId":"511145.b2007","name":"yeeX","hasFamily":true,"annotation":"conserved protein, UPF0265 family","uniprotId":"YEEX_ECOLI"},"6882724":{"id":6882724,"externalId":"511145.b2008","name":"yeeA","hasFamily":true,"annotation":"inner membrane protein, FUSC family","uniprotId":"YEEA_ECOLI"},"6882725":{"id":6882725,"externalId":"511145.b2009","name":"sbmC","hasFamily":true,"annotation":"DNA gyrase inhibitor; Inhibits the supercoiling activity of DNA gyrase. Acts by inhibiting DNA gyrase at an early step, prior to (or at the step of) binding of DNA by the gyrase. It protects cells against toxins that target DNA gyrase, by inhibiting activity of these toxins and reducing the formation of lethal double-strand breaks in the cell. Protects cells against the natural plasmid-encoded toxins microcin B17 (MccB17) and CcdB, and synthetic quinolones. Can also protect cells against alkylating agents that act independently of DNA gyrase, suggesting a more general role in protectin [...] ","uniprotId":"SBMC_ECOLI"},"6882726":{"id":6882726,"externalId":"511145.b2010","name":"dacD","hasFamily":true,"annotation":"D-alanyl-D-alanine carboxypeptidase (penicillin-binding protein 6b); Removes C-terminal D-alanyl residues from sugar-peptide cell wall precursors","uniprotId":"DACD_ECOLI"},"6882727":{"id":6882727,"externalId":"511145.b2011","name":"sbcB","hasFamily":true,"annotation":"exonuclease I; Also functions as a DNA deoxyribophosphodiesterase that releases deoxyribose-phosphate moieties following the cleavage DNA at an apurinic/apyrimidinic (AP) site by either an AP endonuclease AP lyase","uniprotId":"C0Z3X9_ECOLI"},"6882728":{"id":6882728,"externalId":"511145.b2012","name":"yeeD","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEED_ECOLI"},"6882729":{"id":6882729,"externalId":"511145.b2013","name":"yeeE","hasFamily":true,"annotation":"inner membrane protein, UPF0394 family","uniprotId":"YEEE_ECOLI"},"6882730":{"id":6882730,"externalId":"511145.b2014","name":"plaP","hasFamily":true,"annotation":"putrescine importer, low affinity; Putrescine importer. Required for induction of type 1 pili-driven surface motility","uniprotId":"PLAP_ECOLI"},"6882731":{"id":6882731,"externalId":"511145.b2015","name":"yeeY","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YEEY_ECOLI"},"6882732":{"id":6882732,"externalId":"511145.b2016","name":"yeeZ","hasFamily":true,"annotation":"putative epimerase, with NAD(P)-binding Rossmann-fold domain","uniprotId":"YEEZ_ECOLI"},"6882733":{"id":6882733,"externalId":"511145.b2017","name":"yefM","hasFamily":true,"annotation":"antitoxin of the YoeB-YefM toxin-antitoxin system; Antitoxin component of a toxin-antitoxin (TA) module. Antitoxin that counteracts the effect of the YoeB toxin. YefM binds to the promoter region of the yefM-yeoB operon to repress transcription, YeoB acts as a corepressor","uniprotId":"YEFM_ECOLI"},"6882734":{"id":6882734,"externalId":"511145.b2018","name":"hisL","hasFamily":false,"annotation":"his operon leader peptide"},"6882735":{"id":6882735,"externalId":"511145.b2019","name":"hisG","hasFamily":true,"annotation":"ATP phosphoribosyltransferase; Catalyzes the condensation of ATP and 5-phosphoribose 1- diphosphate to form N'-(5'-phosphoribosyl)-ATP (PR-ATP). Has a crucial role in the pathway because the rate of histidine biosynthesis seems to be controlled primarily by regulation of HisG enzymatic activity (By similarity)","uniprotId":"HIS1_ECOLI"},"6882736":{"id":6882736,"externalId":"511145.b2020","name":"hisD","hasFamily":true,"annotation":"bifunctional histidinal dehydrogenase/ histidinol dehydrogenase; Catalyzes the sequential NAD-dependent oxidations of L- histidinol to L-histidinaldehyde and then to L-histidine (By similarity)","uniprotId":"HISX_ECOLI"},"6882737":{"id":6882737,"externalId":"511145.b2021","name":"hisC","hasFamily":true,"annotation":"histidinol-phosphate aminotransferase","uniprotId":"HIS8_ECOLI"},"6882738":{"id":6882738,"externalId":"511145.b2022","name":"hisB","hasFamily":true,"annotation":"fused histidinol-phosphatase/imidazoleglycerol-phosphate dehydratase","uniprotId":"HIS7_ECOLI"},"6882739":{"id":6882739,"externalId":"511145.b2023","name":"hisH","hasFamily":true,"annotation":"imidazole glycerol phosphate synthase, glutamine amidotransferase subunit with HisF; IGPS catalyzes the conversion of PRFAR and glutamine to IGP, AICAR and glutamate. The HisH subunit provides the glutamine amidotransferase activity that produces the ammonia necessary to HisF for the synthesis of IGP and AICAR (By similarity)","uniprotId":"HIS5_ECOLI"},"6882740":{"id":6882740,"externalId":"511145.b2024","name":"hisA","hasFamily":true,"annotation":"N-(5'-phospho-L-ribosyl-formimino)-5-amino-1- (5'-phosphoribosyl)-4-imidazolecarboxamide isomerase","uniprotId":"HIS4_ECOLI"},"6882741":{"id":6882741,"externalId":"511145.b2025","name":"hisF","hasFamily":true,"annotation":"imidazole glycerol phosphate synthase, catalytic subunit with HisH; IGPS catalyzes the conversion of PRFAR and glutamine to IGP, AICAR and glutamate. The HisF subunit catalyzes the cyclization activity that produces IGP and AICAR from PRFAR using the ammonia provided by the HisH subunit (By similarity)","uniprotId":"HIS6_ECOLI"},"6882742":{"id":6882742,"externalId":"511145.b2026","name":"hisI","hasFamily":true,"annotation":"fused phosphoribosyl-AMP cyclohydrolase/phosphoribosyl-ATP pyrophosphatase","uniprotId":"HIS2_ECOLI"},"6882743":{"id":6882743,"externalId":"511145.b2027","name":"wzz","hasFamily":true,"annotation":"regulator of length of O-antigen component of lipopolysaccharide chains; Confers a modal distribution of chain length on the O- antigen component of lipopolysaccharide (LPS). Gives rise to a reduced number of short chain molecules and increases in numbers of longer molecules, with a modal value of 13 (in strain O111/M92) and of 17 (in strain K12)","uniprotId":"WZZB_ECOLI"},"6882744":{"id":6882744,"externalId":"511145.b2028","name":"ugd","hasFamily":true,"annotation":"UDP-glucose 6-dehydrogenase","uniprotId":"UDG_ECOLI"},"6882745":{"id":6882745,"externalId":"511145.b2029","name":"gnd","hasFamily":true,"annotation":"6-phosphogluconate dehydrogenase, decarboxylating; Catalyzes the oxidative decarboxylation of 6- phosphogluconate to ribulose 5-phosphate and CO(2), with concomitant reduction of NADP to NADPH (By similarity)"},"6882746":{"id":6882746,"externalId":"511145.b2030","name":"b2030","hasFamily":true,"annotation":"IS5 transposase and trans-activator"},"6882747":{"id":6882747,"externalId":"511145.b2032","name":"wbbK","hasFamily":true,"annotation":"lipopolysaccharide biosynthesis protein; May be a glycosyltransferase involved in the transfer of UDP-GalF and UDP-glucose","uniprotId":"WBBK_ECOLI"},"6882748":{"id":6882748,"externalId":"511145.b2033","name":"wbbJ","hasFamily":true,"annotation":"putative lipopolysaccharide biosynthesis O-acetyl transferase; Putative O-acetyltransferase that transfers an O-acetyl group to the O antigen","uniprotId":"WBBJ_ECOLI"},"6882749":{"id":6882749,"externalId":"511145.b2034","name":"wbbI","hasFamily":false,"annotation":"d-Galf:alpha-d-Glc beta-1,6-galactofuranosyltransferase; Involved in the transfer of galactofuranose (Galf) onto an alpha-D-gluco-configured acceptor substrate to form a beta-1,6- linkage. It uses n-octyl alpha-D-glucopyranoside as an acceptor substrate for the addition of galactofuranose from the donor substrate UDP-galactofuranose. It is not able to use beta-D- glucopyranoside isomers","uniprotId":"WBBI_ECOLI"},"6882750":{"id":6882750,"externalId":"511145.b2035","name":"rfc","hasFamily":false,"annotation":"O-antigen polymerase; May link the O-antigen tetrasaccharide units into long chains, giving rise to typical smooth LPS","uniprotId":"RFC_ECOLI"},"6882751":{"id":6882751,"externalId":"511145.b2036","name":"glf","hasFamily":true,"annotation":"UDP-galactopyranose mutase, FAD/NAD(P)-binding; Catalyzes the interconversion through a 2-keto intermediate of uridine diphosphogalactopyranose (UDP-GalP) into uridine diphosphogalactofuranose (UDP-GalF) (By similarity)","uniprotId":"GLF_ECOLI"},"6882752":{"id":6882752,"externalId":"511145.b2037","name":"rfbX","hasFamily":true,"annotation":"putative polisoprenol-linked O-antigen transporter; May be involved in the translocation process of the nascent O-polysaccharide molecules and/or its ligation to lipid A core units","uniprotId":"RFBX_ECOLI"},"6882753":{"id":6882753,"externalId":"511145.b2038","name":"rfbC","hasFamily":true,"annotation":"dTDP-4-deoxyrhamnose-3,5-epimerase; Catalyzes the epimerization of the C3' and C5'positions of dTDP-6-deoxy-D-xylo-4-hexulose, forming dTDP-6-deoxy-L-lyxo-4- hexulose (By similarity)","uniprotId":"RMLC_ECOLI"},"6882754":{"id":6882754,"externalId":"511145.b2039","name":"rmlA","hasFamily":true,"annotation":"glucose-1-phosphate thymidylyltransferase; Catalyzes the formation of dTDP-glucose, from dTTP and glucose 1-phosphate, as well as its pyrophosphorolysis (By similarity)","uniprotId":"RMLA1_ECOLI"},"6882755":{"id":6882755,"externalId":"511145.b2040","name":"rfbD","hasFamily":true,"annotation":"dTDP-4-dehydrorhamnose reductase subunit, NAD(P)-binding, of dTDP-L-rhamnose synthase; Catalyzes the reduction of dTDP-6-deoxy-L-lyxo-4- hexulose to yield dTDP-L-rhamnose. RmlD uses NADH and NADPH nearly equally well (By similarity)","uniprotId":"RMLD_ECOLI"},"6882756":{"id":6882756,"externalId":"511145.b2041","name":"rfbB","hasFamily":true,"annotation":"dTDP-glucose 4,6 dehydratase, NAD(P)-binding; Catalyzes the dehydration of dTDP-D-glucose to form dTDP-6-deoxy-D-xylo-4-hexulose via a three-step process involving oxidation, dehydration and reduction (By similarity)","uniprotId":"RMLB1_ECOLI"},"6882757":{"id":6882757,"externalId":"511145.b2042","name":"galF","hasFamily":true,"annotation":"putative regulatory subunit for GalU","uniprotId":"GALF_ECOLI"},"6882758":{"id":6882758,"externalId":"511145.b2043","name":"wcaM","hasFamily":true,"annotation":"colanic acid biosynthesis protein","uniprotId":"WCAM_ECOLI"},"6882759":{"id":6882759,"externalId":"511145.b2044","name":"wcaL","hasFamily":true,"annotation":"putative glycosyl transferase","uniprotId":"WCAL_ECOLI"},"6882760":{"id":6882760,"externalId":"511145.b2045","name":"wcaK","hasFamily":true,"annotation":"Colanic acid biosynthesis protein","uniprotId":"WCAK_ECOLI"},"6882761":{"id":6882761,"externalId":"511145.b2046","name":"wzxC","hasFamily":true,"annotation":"putative colanic acid exporter","uniprotId":"WZXC_ECOLI"},"6882762":{"id":6882762,"externalId":"511145.b2047","name":"wcaJ","hasFamily":true,"annotation":"predicted UDP-glucose lipid carrier transferase; Is the initiating enzyme for colanic acid (CA) synthesis. Catalyzes the transfer of the glucose-1-phosphate moiety from UDP-Glc onto the carrier lipid undecaprenyl phosphate (C55-P), forming a phosphoanhydride bond yielding to glucosyl- pyrophosphoryl-undecaprenol (Glc-PP-C55). Also possesses a weak galactose-1-P transferase activity","uniprotId":"WCAJ_ECOLI"},"6882763":{"id":6882763,"externalId":"511145.b2048","name":"cpsG","hasFamily":true,"annotation":"phosphomannomutase; Involved in GDP-mannose biosynthesis which serves as the activated sugar nucleotide precursor for mannose residues in cell surface polysaccharides. This enzyme participates in synthesis of the LPS O7 antigen","uniprotId":"MANB_ECOLI"},"6882764":{"id":6882764,"externalId":"511145.b2049","name":"cpsB","hasFamily":true,"annotation":"mannose-1-phosphate guanyltransferase; Involved in the biosynthesis of the capsular polysaccharide colanic acid","uniprotId":"MANC_ECOLI"},"6882765":{"id":6882765,"externalId":"511145.b2050","name":"wcaI","hasFamily":true,"annotation":"putative glycosyl transferase","uniprotId":"WCAI_ECOLI"},"6882766":{"id":6882766,"externalId":"511145.b2051","name":"gmm","hasFamily":true,"annotation":"GDP-mannose mannosyl hydrolase; Hydrolyzes both GDP-mannose and GDP-glucose. Could participate in the regulation of cell wall biosynthesis by influencing the concentration of GDP-mannose or GDP-glucose in the cell. Might also be involved in the biosynthesis of the slime polysaccharide colanic acid","uniprotId":"GMM_ECOLI"},"6882767":{"id":6882767,"externalId":"511145.b2052","name":"fcl","hasFamily":true,"annotation":"bifunctional GDP-fucose synthetase: GDP-4-dehydro-6-deoxy-D-mannose epimerase/ GDP-4-dehydro-6-L-deoxygalactose reductase; Catalyzes the two-step NADP-dependent conversion of GDP- 4-dehydro-6-deoxy-D-mannose to GDP-fucose, involving an epimerase and a reductase reaction","uniprotId":"FCL_ECOLI"},"6882768":{"id":6882768,"externalId":"511145.b2053","name":"gmd","hasFamily":true,"annotation":"GDP-D-mannose dehydratase, NAD(P)-binding; Catalyzes the conversion of GDP-D-mannose to GDP-4- dehydro-6-deoxy-D-mannose","uniprotId":"GM4D_ECOLI"},"6882769":{"id":6882769,"externalId":"511145.b2054","name":"wcaF","hasFamily":true,"annotation":"putative acyl transferase","uniprotId":"WCAF_ECOLI"},"6882770":{"id":6882770,"externalId":"511145.b2055","name":"wcaE","hasFamily":true,"annotation":"putative glycosyl transferase","uniprotId":"WCAE_ECOLI"},"6882771":{"id":6882771,"externalId":"511145.b2056","name":"wcaD","hasFamily":true,"annotation":"putative colanic acid polymerase","uniprotId":"WCAD_ECOLI"},"6882772":{"id":6882772,"externalId":"511145.b2057","name":"wcaC","hasFamily":true,"annotation":"putative glycosyl transferase","uniprotId":"WCAC_ECOLI"},"6882773":{"id":6882773,"externalId":"511145.b2058","name":"wcaB","hasFamily":true,"annotation":"putative acyl transferase","uniprotId":"WCAB_ECOLI"},"6882774":{"id":6882774,"externalId":"511145.b2059","name":"wcaA","hasFamily":true,"annotation":"putative glycosyl transferase","uniprotId":"WCAA_ECOLI"},"6882775":{"id":6882775,"externalId":"511145.b2060","name":"wzc","hasFamily":true,"annotation":"protein-tyrosine kinase; Required for the extracellular polysaccharide colanic acid synthesis. The autophosphorylated form is inactive. Probably involved in the export of colanic acid from the cell to medium. Phosphorylates udg","uniprotId":"WZC_ECOLI"},"6882776":{"id":6882776,"externalId":"511145.b2061","name":"wzb","hasFamily":true,"annotation":"protein-tyrosine phosphatase; Dephosphorylates wzc. Required for the extracellular polysaccharide colanic acid synthesis. Probably involved in the export of colanic acid from the cell to medium","uniprotId":"WZB_ECOLI"},"6882777":{"id":6882777,"externalId":"511145.b2062","name":"wza","hasFamily":true,"annotation":"lipoprotein required for capsular polysaccharide translocation through the outer membrane; Probably involved in the export of the extracellular polysaccharide colanic acid from the cell to medium","uniprotId":"WZA_ECOLI"},"6882778":{"id":6882778,"externalId":"511145.b2063","name":"yegH","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YEGH_ECOLI"},"6882779":{"id":6882779,"externalId":"511145.b2064","name":"asmA","hasFamily":true,"annotation":"putative assembly protein; Involved in the inhibition of assembly of mutant ompF proteins. In general, could be involved in the assembly of outer membrane proteins","uniprotId":"ASMA_ECOLI"},"6882780":{"id":6882780,"externalId":"511145.b2065","name":"dcd","hasFamily":true,"annotation":"2'-deoxycytidine 5'-triphosphate deaminase","uniprotId":"DCD_ECOLI"},"6882781":{"id":6882781,"externalId":"511145.b2066","name":"udk","hasFamily":true,"annotation":"uridine/cytidine kinase","uniprotId":"URK_ECOLI"},"6882782":{"id":6882782,"externalId":"511145.b2067","name":"yegE","hasFamily":true,"annotation":"putative diguanylate cyclase, GGDEF domain signaling protein; Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"YEGE_ECOLI"},"6882783":{"id":6882783,"externalId":"511145.b2068","name":"alkA","hasFamily":true,"annotation":"3-methyl-adenine DNA glycosylase II; Hydrolysis of the deoxyribose N-glycosidic bond to excise 3-methyladenine, 3-methylguanine, 7-methylguanine, O2- methylthymine, and O2-methylcytosine from the damaged DNA polymer formed by alkylation lesions","uniprotId":"3MG2_ECOLI"},"6882784":{"id":6882784,"externalId":"511145.b2069","name":"yegD","hasFamily":true,"annotation":"putative chaperone","uniprotId":"YEGD_ECOLI"},"6882785":{"id":6882785,"externalId":"511145.b2070","name":"yegI","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEGI_ECOLI"},"6882786":{"id":6882786,"externalId":"511145.b2071","name":"yegJ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YEGJ_ECOLI"},"6882787":{"id":6882787,"externalId":"511145.b2072","name":"yegK","hasFamily":true,"annotation":"predicted protein","uniprotId":"YEGK_ECOLI"},"6882788":{"id":6882788,"externalId":"511145.b2073","name":"yegL","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEGL_ECOLI"},"6882789":{"id":6882789,"externalId":"511145.b2074","name":"mdtA","hasFamily":true,"annotation":"multidrug efflux system, subunit A; The MdtABC tripartite complex confers resistance against novobiocin and deoxycholate (By similarity)","uniprotId":"MDTA_ECOLI"},"6882790":{"id":6882790,"externalId":"511145.b2075","name":"mdtB","hasFamily":true,"annotation":"multidrug efflux system, subunit B; The MdtABC tripartite complex confers resistance against novobiocin and deoxycholate (By similarity)","uniprotId":"MDTB_ECOLI"},"6882791":{"id":6882791,"externalId":"511145.b2076","name":"mdtC","hasFamily":true,"annotation":"multidrug efflux system, subunit C; The MdtABC tripartite complex confers resistance against novobiocin and deoxycholate (By similarity)","uniprotId":"MDTC_ECOLI"},"6882792":{"id":6882792,"externalId":"511145.b2077","name":"mdtD","hasFamily":true,"annotation":"putative arabinose efflux transporter","uniprotId":"MDTD_ECOLI"},"6882793":{"id":6882793,"externalId":"511145.b2078","name":"baeS","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with BaeR; Member of the two-component regulatory system BaeS/BaeR. Activates BaeR by phosphorylation which then activates the mdtABCD and probably the casABCDE-ygbT-ygbF operons","uniprotId":"BAES_ECOLI"},"6882794":{"id":6882794,"externalId":"511145.b2079","name":"baeR","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with BaeS; Member of the two-component regulatory system BaeS/BaeR. Activates the mdtABCD and probably the casABCDE-ygbT-ygbF operons","uniprotId":"BAER_ECOLI"},"6882795":{"id":6882795,"externalId":"511145.b2080","name":"yegP","hasFamily":true,"annotation":"conserved protein, UPF0339 family","uniprotId":"YEGP_ECOLI"},"6882796":{"id":6882796,"externalId":"511145.b2081","name":"yegQ","hasFamily":true,"annotation":"putative peptidase","uniprotId":"YEGQ_ECOLI"},"6882797":{"id":6882797,"externalId":"511145.b2082","name":"ogrK","hasFamily":true,"annotation":"positive regulator of P2 growth (insertion of P2 ogr gene into the chromosome); Cryptic version of the phage P2 OGR protein which acts as an activator of P2 late transcription","uniprotId":"OGRK_ECOLI"},"6882798":{"id":6882798,"externalId":"511145.b2085","name":"yegR","hasFamily":false,"annotation":"predicted protein","uniprotId":"YEGR_ECOLI"},"6882799":{"id":6882799,"externalId":"511145.b2086","name":"yegS","hasFamily":true,"annotation":"phosphatidylglycerol kinase, metal-dependent; In vitro phosphorylates phosphatidylglycerol but not diacylglycerol; the in vivo substrate is unknown","uniprotId":"YEGS_ECOLI"},"6882800":{"id":6882800,"externalId":"511145.b2088","name":"insE","hasFamily":true,"annotation":"IS3 transposase A; Involved in the transposition of the insertion sequence IS3","uniprotId":"INSE1_ECOLI"},"6882801":{"id":6882801,"externalId":"511145.b2089","name":"orfB","hasFamily":true,"annotation":"IS3 transposase B; Involved in the transposition of the insertion sequence IS3","uniprotId":"INSF1_ECOLI"},"6882802":{"id":6882802,"externalId":"511145.b2091","name":"gatD","hasFamily":true,"annotation":"galactitol-1-phosphate dehydrogenase, Zn-dependent and NAD(P)-binding; Converts galactitol 1-phosphate to tagatose 6-phosphate","uniprotId":"GATD_ECOLI"},"6882803":{"id":6882803,"externalId":"511145.b2092","name":"gatC","hasFamily":true,"annotation":"galactitol-specific enzyme IIC component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in galactitol transport","uniprotId":"PTKC_ECOLI"},"6882804":{"id":6882804,"externalId":"511145.b2093","name":"gatB","hasFamily":true,"annotation":"galactitol-specific enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in galactitol transport","uniprotId":"PTKB_ECOLI"},"6882805":{"id":6882805,"externalId":"511145.b2094","name":"gatA","hasFamily":true,"annotation":"galactitol-specific enzyme IIA component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in galactitol transport","uniprotId":"PTKA_ECOLI"},"6882806":{"id":6882806,"externalId":"511145.b2095","name":"gatZ","hasFamily":true,"annotation":"D-tagatose 1,6-bisphosphate aldolase 2, subunit; Component of the tagatose-1,6-bisphosphate aldolase GatYZ that is required for full activity and stability of the Y subunit. Could have a chaperone-like function for the proper and stable folding of GatY. When expressed alone, GatZ does not show any aldolase activity. Is involved in the catabolism of galactitol (By similarity)","uniprotId":"GATZ_ECOLI"},"6882807":{"id":6882807,"externalId":"511145.b2096","name":"gatY","hasFamily":true,"annotation":"D-tagatose 1,6-bisphosphate aldolase 2, catalytic subunit; Catalytic subunit of the tagatose-1,6-bisphosphate aldolase GatYZ, which catalyzes the reversible aldol condensation of dihydroxyacetone phosphate (DHAP or glycerone-phosphate) with glyceraldehyde 3-phosphate (G3P) to produce tagatose 1,6- bisphosphate (TBP). Requires GatZ subunit for full activity and stability. Is involved in the catabolism of galactitol (By similarity)","uniprotId":"GATY_ECOLI"},"6882808":{"id":6882808,"externalId":"511145.b2097","name":"fbaB","hasFamily":true,"annotation":"fructose-bisphosphate aldolase class I","uniprotId":"ALF1_ECOLI"},"6882809":{"id":6882809,"externalId":"511145.b2098","name":"yegT","hasFamily":true,"annotation":"nucleoside transporter, low affinity; Could be involved in nucleoside transport (By similarity)","uniprotId":"YEGT_ECOLI"},"6882810":{"id":6882810,"externalId":"511145.b2099","name":"yegU","hasFamily":true,"annotation":"putative hydrolase","uniprotId":"YEGU_ECOLI"},"6882811":{"id":6882811,"externalId":"511145.b2100","name":"yegV","hasFamily":true,"annotation":"putative kinase","uniprotId":"YEGV_ECOLI"},"6882812":{"id":6882812,"externalId":"511145.b2101","name":"yegW","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YEGW_ECOLI"},"6882813":{"id":6882813,"externalId":"511145.b2102","name":"yegX","hasFamily":true,"annotation":"putative hydrolase","uniprotId":"YEGX_ECOLI"},"6882814":{"id":6882814,"externalId":"511145.b2103","name":"thiD","hasFamily":true,"annotation":"bifunctional hydroxy-methylpyrimidine kinase/ hydroxy-phosphomethylpyrimidine kinase; Catalyzes the phosphorylation of hydroxymethylpyrimidine phosphate (HMP-P) to HMP-PP, and of HMP to HMP-P. Shows no activity with pyridoxal, pyridoxamine or pyridoxine","uniprotId":"THID_ECOLI"},"6882815":{"id":6882815,"externalId":"511145.b2104","name":"thiM","hasFamily":true,"annotation":"hydoxyethylthiazole kinase","uniprotId":"THIM_ECOLI"},"6882816":{"id":6882816,"externalId":"511145.b2105","name":"rcnR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor of rcnA; Repressor of rcnA expression. Acts by binding specifically to the rcnA promoter in the absence of nickel and cobalt. In the presence of one of these metals, it has a weaker affinity for rcnA promoter","uniprotId":"RCNR_ECOLI"},"6882817":{"id":6882817,"externalId":"511145.b2106","name":"rcnA","hasFamily":true,"annotation":"membrane protein conferring nickel and cobalt resistance; Efflux system for nickel and cobalt","uniprotId":"RCNA_ECOLI"},"6882818":{"id":6882818,"externalId":"511145.b2107","name":"rcnB","hasFamily":true,"annotation":"periplasmic modulator of Ni and Co efflux; Influences nickel and cobalt homeostasis. May act by modulating RcnA-mediated export of these ions to avoid excess efflux. Not involved in nickel import and does not bind nickel or cobalt ions directly","uniprotId":"RCNB_ECOLI"},"6882819":{"id":6882819,"externalId":"511145.b2108","name":"yehA","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YEHA_ECOLI"},"6882820":{"id":6882820,"externalId":"511145.b2109","name":"yehB","hasFamily":true,"annotation":"putative outer membrane protein; Involved in the export and assembly of a fimbrial subunit across the outer membrane (By similarity)","uniprotId":"YEHB_ECOLI"},"6882821":{"id":6882821,"externalId":"511145.b2110","name":"yehC","hasFamily":true,"annotation":"putative periplasmic pilin chaperone; Could be required for the biogenesis of a putative fimbria (By similarity)","uniprotId":"YEHC_ECOLI"},"6882822":{"id":6882822,"externalId":"511145.b2111","name":"yehD","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YEHD_ECOLI"},"6882823":{"id":6882823,"externalId":"511145.b2112","name":"yehE","hasFamily":true,"annotation":"predicted protein","uniprotId":"YEHE_ECOLI"},"6882824":{"id":6882824,"externalId":"511145.b2113","name":"mrp","hasFamily":true,"annotation":"antiporter inner membrane protein","uniprotId":"MRP_ECOLI"},"6882825":{"id":6882825,"externalId":"511145.b2114","name":"metG","hasFamily":true,"annotation":"methionyl-tRNA synthetase; Is required not only for elongation of protein synthesis but also for the initiation of all mRNA translation through initiator tRNA(fMet) aminoacylation (By similarity)","uniprotId":"SYM_ECOLI"},"6882826":{"id":6882826,"externalId":"511145.b2118","name":"yehI","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEHI_ECOLI"},"6882827":{"id":6882827,"externalId":"511145.b2119","name":"yehL","hasFamily":true,"annotation":"putative transporter subunit: ATP-binding component of ABC superfamily","uniprotId":"YEHL_ECOLI"},"6882828":{"id":6882828,"externalId":"511145.b2120","name":"yehM","hasFamily":true,"annotation":"predicted protein","uniprotId":"YEHM_ECOLI"},"6882829":{"id":6882829,"externalId":"511145.b2121","name":"yehP","hasFamily":true,"annotation":"conserved protein","uniprotId":"YEHP_ECOLI"},"6882830":{"id":6882830,"externalId":"511145.b2123","name":"yehR","hasFamily":true,"annotation":"lipoprotein, DUF1307 family","uniprotId":"YEHR_ECOLI"},"6882831":{"id":6882831,"externalId":"511145.b2124","name":"yehS","hasFamily":true,"annotation":"conserved protein, DUF1456 family","uniprotId":"YEHS_ECOLI"},"6882832":{"id":6882832,"externalId":"511145.b2125","name":"yehT","hasFamily":true,"annotation":"putative response regulator in two-component system withYehU; Member of the two-component regulatory system YehU/YehT, which may be involved in the stationary-phase control network. Regulates expression of yjiY by binding to its promoter region","uniprotId":"YEHT_ECOLI"},"6882833":{"id":6882833,"externalId":"511145.b2126","name":"yehU","hasFamily":true,"annotation":"putative sensory kinase in two-component system with YehT, inner membrane protein; Member of the two-component regulatory system YehU/YehT, which may be involved in the stationary-phase control network. Activates YehT by phosphorylation","uniprotId":"YEHU_ECOLI"},"6882834":{"id":6882834,"externalId":"511145.b2127","name":"mlrA","hasFamily":true,"annotation":"DNA-binding transcriptional regulator; Transcriptional activator of the csg genes required for production of the curli (AgF)","uniprotId":"MLRA_ECOLI"},"6882835":{"id":6882835,"externalId":"511145.b2128","name":"yehW","hasFamily":true,"annotation":"putative transporter subunit: membrane component of ABC superfamily; May be part of an ABC transporter complex involved in uptake of osmoprotectant molecules. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"YEHW_ECOLI"},"6882836":{"id":6882836,"externalId":"511145.b2129","name":"yehX","hasFamily":true,"annotation":"putative transporter subunit: ATP-binding component of ABC superfamily; May be part of an ABC transporter complex involved in uptake of osmoprotectant molecules. Probably responsible for energy coupling to the transport system","uniprotId":"YEHX_ECOLI"},"6882837":{"id":6882837,"externalId":"511145.b2130","name":"yehY","hasFamily":true,"annotation":"putative transporter subunit: membrane component of ABC superfamily; May be part of an ABC transporter complex involved in uptake of osmoprotectant molecules. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"YEHY_ECOLI"},"6882838":{"id":6882838,"externalId":"511145.b2131","name":"osmF","hasFamily":true,"annotation":"putative transporter subunit: periplasmic-binding component of ABC superfamily; May be part of an ABC transporter complex involved in uptake of osmoprotectant molecules","uniprotId":"OSMF_ECOLI"},"6882839":{"id":6882839,"externalId":"511145.b2132","name":"bglX","hasFamily":true,"annotation":"beta-D-glucoside glucohydrolase, periplasmic","uniprotId":"BGLX_ECOLI"},"6882840":{"id":6882840,"externalId":"511145.b2133","name":"dld","hasFamily":true,"annotation":"D-lactate dehydrogenase, FAD-binding, NADH independent; First component of the membrane-bound D-lactate oxidase, which is believed to play an important role in the energization of the active transport of a variety of sugars and amino acids","uniprotId":"DLD_ECOLI"},"6882841":{"id":6882841,"externalId":"511145.b2134","name":"pbpG","hasFamily":true,"annotation":"D-alanyl-D-alanine endopeptidase; Cell wall formation. May play a specialized role in remodeling the cell wall. Specifically hydrolyzes the DD- diaminopimelate-alanine bonds in high-molecular-mass murein sacculi","uniprotId":"PBP7_ECOLI"},"6882842":{"id":6882842,"externalId":"511145.b2135","name":"yohC","hasFamily":true,"annotation":"inner membrane protein, Yip1 family","uniprotId":"YOHC_ECOLI"},"6882843":{"id":6882843,"externalId":"511145.b2136","name":"yohD","hasFamily":true,"annotation":"inner membrane protein, DedA family","uniprotId":"YOHD_ECOLI"},"6882844":{"id":6882844,"externalId":"511145.b2137","name":"yohF","hasFamily":true,"annotation":"putative oxidoreductase with NAD(P)-binding Rossmann-fold domain","uniprotId":"YOHF_ECOLI"},"6882845":{"id":6882845,"externalId":"511145.b2140","name":"dusC","hasFamily":true,"annotation":"tRNA-dihydrouridine synthase C; Catalyzes the synthesis of dihydrouridine, a modified base found in the D-loop of most tRNAs (By similarity)","uniprotId":"DUSC_ECOLI"},"6882846":{"id":6882846,"externalId":"511145.b2141","name":"yohJ","hasFamily":true,"annotation":"inner membrane protein, UPF0299 family","uniprotId":"YOHJ_ECOLI"},"6882847":{"id":6882847,"externalId":"511145.b2142","name":"yohK","hasFamily":true,"annotation":"inner membrane protein, LrgB family","uniprotId":"YOHK_ECOLI"},"6882848":{"id":6882848,"externalId":"511145.b2143","name":"cdd","hasFamily":true,"annotation":"cytidine/deoxycytidine deaminase; This enzyme scavenge exogenous and endogenous cytidine and 2'-deoxycytidine for UMP synthesis (By similarity)","uniprotId":"CDD_ECOLI"},"6882849":{"id":6882849,"externalId":"511145.b2144","name":"sanA","hasFamily":true,"annotation":"vancomycin high temperature exclusion protein; mutants have a defective envelope more permeable to vancomycin at 42 degrees; Participates in the barrier function of the cell envelope","uniprotId":"SANA_ECOLI"},"6882850":{"id":6882850,"externalId":"511145.b2145","name":"yeiS","hasFamily":true,"annotation":"conserved protein, DUF2542 family","uniprotId":"YEIS_ECOLI"},"6882851":{"id":6882851,"externalId":"511145.b2146","name":"preT","hasFamily":true,"annotation":"Dihydropyrimidine dehydrogenase, NADH-dependent, subunit A; Involved in pyrimidine base degradation. Catalyzes physiologically the reduction of uracil to 5,6-dihydrouracil (DHU) by using NADH as a specific cosubstrate. It also catalyzes the reverse reaction and the reduction of thymine to 5,6- dihydrothymine (DHT)","uniprotId":"PRET_ECOLI"},"6882852":{"id":6882852,"externalId":"511145.b2147","name":"preA","hasFamily":true,"annotation":"Dihydropyrimidine dehydrogenase, NADH-dependent, subunit B; Involved in pyrimidine base degradation. Catalyzes physiologically the reduction of uracil to 5,6-dihydrouracil (DHU) by using NADH as a specific cosubstrate. It also catalyzes the reverse reaction and the reduction of thymine to 5,6- dihydrothymine (DHT)","uniprotId":"PREA_ECOLI"},"6882853":{"id":6882853,"externalId":"511145.b2148","name":"mglC","hasFamily":true,"annotation":"methyl-galactoside transporter subunit; Part of the binding-protein-dependent transport system for galactoside. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"MGLC_ECOLI"},"6882854":{"id":6882854,"externalId":"511145.b2149","name":"mglA","hasFamily":true,"annotation":"fused methyl-galactoside transporter subunits of ABC superfamily: ATP-binding components; Part of the ABC transporter complex MglABC involved in galactose/methyl galactoside import. Responsible for energy coupling to the transport system (Probable)","uniprotId":"MGLA_ECOLI"},"6882855":{"id":6882855,"externalId":"511145.b2150","name":"mglB","hasFamily":true,"annotation":"methyl-galactoside transporter subunit; This protein is involved in the active transport of galactose and glucose. It plays a role in the chemotaxis towards the two sugars by interacting with the trg chemoreceptor","uniprotId":"DGAL_ECOLI"},"6882856":{"id":6882856,"externalId":"511145.b2151","name":"galS","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor of the mgl operon. Binds galactose and D- fucose as inducers. GalS binds to an operator DNA sequence within its own coding sequence (corresponding to residues 15 to 20)","uniprotId":"GALS_ECOLI"},"6882857":{"id":6882857,"externalId":"511145.b2152","name":"yeiB","hasFamily":true,"annotation":"putative inner membrane protein; Involved in transport (Potential)","uniprotId":"YEIB_ECOLI"},"6882858":{"id":6882858,"externalId":"511145.b2153","name":"folE","hasFamily":true,"annotation":"GTP cyclohydrolase I","uniprotId":"GCH1_ECOLI"},"6882859":{"id":6882859,"externalId":"511145.b2154","name":"yeiG","hasFamily":true,"annotation":"S-formylglutathione hydrolase; Serine hydrolase involved in the detoxification of formaldehyde. Hydrolyzes S-formylglutathione to glutathione and formate. Shows also esterase activity against alpha-naphthyl acetate, lactoylglutathione, palmitoyl-CoA and several pNP-esters of short chain fatty acids","uniprotId":"SFGH2_ECOLI"},"6882860":{"id":6882860,"externalId":"511145.b2155","name":"cirA","hasFamily":true,"annotation":"catecholate siderophore receptor CirA; Not yet known. Postulated to participate in iron transport. Outer membrane receptor for colicins IA and IB","uniprotId":"CIRA_ECOLI"},"6882861":{"id":6882861,"externalId":"511145.b2156","name":"lysP","hasFamily":true,"annotation":"lysine transporter; Permease that is involved in the transport across the cytoplasmic membrane of lysine","uniprotId":"LYSP_ECOLI"},"6882862":{"id":6882862,"externalId":"511145.b2157","name":"yeiE","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YEIE_ECOLI"},"6882863":{"id":6882863,"externalId":"511145.b2158","name":"yeiH","hasFamily":true,"annotation":"inner membrane protein, UPF0324 family","uniprotId":"YEIH_ECOLI"},"6882864":{"id":6882864,"externalId":"511145.b2159","name":"nfo","hasFamily":true,"annotation":"endonuclease IV with intrinsic 3'-5' exonuclease activity; Endonuclease IV plays a role in DNA repair. It cleaves phosphodiester bonds at apurinic or apyrimidinic sites (AP sites) to produce new 5'-ends that are base-free deoxyribose 5-phosphate residues. It preferentially attacks modified AP sites created by bleomycin and neocarzinostatin (By similarity)","uniprotId":"END4_ECOLI"},"6882865":{"id":6882865,"externalId":"511145.b2160","name":"yeiI","hasFamily":true,"annotation":"putative kinase","uniprotId":"YEII_ECOLI"},"6882866":{"id":6882866,"externalId":"511145.b2161","name":"nupX","hasFamily":true,"annotation":"nucleoside permease; Nucleoside transporter","uniprotId":"NUPX_ECOLI"},"6882867":{"id":6882867,"externalId":"511145.b2162","name":"rihB","hasFamily":true,"annotation":"ribonucleoside hydrolase 2; Hydrolyzes cytidine or uridine to ribose and cytosine or uracil, respectively. Has a clear preference for cytidine over uridine. Strictly specific for ribonucleosides. Has a low but significant activity for the purine nucleoside xanthosine","uniprotId":"RIHB_ECOLI"},"6882868":{"id":6882868,"externalId":"511145.b2163","name":"yeiL","hasFamily":true,"annotation":"DNA-binding transcriptional activator of stationary phase nitrogen survival; Transcription regulator involved in mid-term, stationary-phase viability under nitrogen starvation. Might control expression of the salvage pathways or in some other way repress the recycling of nucleobases to nucleic acids and enhance their use as general nitrogen sources during nitrogen-limited growth","uniprotId":"YEIL_ECOLI"},"6882869":{"id":6882869,"externalId":"511145.b2164","name":"psuT","hasFamily":true,"annotation":"putative nucleoside transporter; Could be involved in pseudouridine transport","uniprotId":"PSUT_ECOLI"},"6882870":{"id":6882870,"externalId":"511145.b2165","name":"psuG","hasFamily":true,"annotation":"pseudouridine 5'-phosphate glycosidase; Catalyzes the hydrolysis of pseudouridine 5'-phosphate (PsiMP) to ribose 5-phosphate and uracil. In vitro, can also catalyze the reverse reaction","uniprotId":"PSUG_ECOLI"},"6882871":{"id":6882871,"externalId":"511145.b2166","name":"psuK","hasFamily":true,"annotation":"pseudouridine kinase; Catalyzes the phosphorylation of pseudouridine to pseudouridine 5'-phosphate (PsiMP)","uniprotId":"PSUK_ECOLI"},"6882872":{"id":6882872,"externalId":"511145.b2167","name":"fruA","hasFamily":true,"annotation":"fused fructose-specific PTS enzymes: IIBcomponent/IIC components; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in fructose transport","uniprotId":"PTFBC_ECOLI"},"6882873":{"id":6882873,"externalId":"511145.b2168","name":"fruK","hasFamily":true,"annotation":"fructose-1-phosphate kinase","uniprotId":"K1PF_ECOLI"},"6882874":{"id":6882874,"externalId":"511145.b2169","name":"fruB","hasFamily":true,"annotation":"fused fructose-specific PTS enzymes: IIA component/HPr component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in fructose transport","uniprotId":"PTFAH_ECOLI"},"6882875":{"id":6882875,"externalId":"511145.b2170","name":"setB","hasFamily":true,"annotation":"lactose/glucose efflux system; Involved in the efflux of sugars. The physiological role may be the detoxification of non-metabolizable sugar analogs. Can transport lactose and glucose","uniprotId":"SETB_ECOLI"},"6882876":{"id":6882876,"externalId":"511145.b2171","name":"yeiP","hasFamily":true,"annotation":"elongation factor P-like protein","uniprotId":"EFPL_ECOLI"},"6882877":{"id":6882877,"externalId":"511145.b2172","name":"yeiQ","hasFamily":true,"annotation":"putative D-mannonate oxidoreductase, NAD-dependent","uniprotId":"YEIQ_ECOLI"},"6882878":{"id":6882878,"externalId":"511145.b2173","name":"yeiR","hasFamily":true,"annotation":"predicted enzyme","uniprotId":"YEIR_ECOLI"},"6882879":{"id":6882879,"externalId":"511145.b2174","name":"yeiU","hasFamily":true,"annotation":"lipid A 1-diphosphate synthase","uniprotId":"YEIU_ECOLI"},"6882880":{"id":6882880,"externalId":"511145.b2175","name":"spr","hasFamily":true,"annotation":"mutational suppressor of prc thermosensitivity, outer membrane lipoprotein","uniprotId":"SPR_ECOLI"},"6882881":{"id":6882881,"externalId":"511145.b2176","name":"rtn","hasFamily":true,"annotation":"resistance protein for phages lambda and N4, when overexpressed; Involved in resistance to phages N4 and lambda","uniprotId":"RTN_ECOLI"},"6882882":{"id":6882882,"externalId":"511145.b2177","name":"yejA","hasFamily":true,"annotation":"microcin C transporter YejABEF, periplasmic binding protein; ABC family","uniprotId":"YEJA_ECOLI"},"6882883":{"id":6882883,"externalId":"511145.b2178","name":"yejB","hasFamily":true,"annotation":"microcin C transporter YejABEF, permease subunit; ABC family; Probably part of a binding-protein-dependent transport system. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"YEJB_ECOLI"},"6882884":{"id":6882884,"externalId":"511145.b2179","name":"yejE","hasFamily":true,"annotation":"microcin C transporter YejABEF, permease subunit; ABC family; Probably part of a binding-protein-dependent transport system. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"YEJE_ECOLI"},"6882885":{"id":6882885,"externalId":"511145.b2180","name":"yejF","hasFamily":true,"annotation":"microcin C transporter, ATP-binding subunit; ABC family","uniprotId":"YEJF_ECOLI"},"6882886":{"id":6882886,"externalId":"511145.b2181","name":"yejG","hasFamily":true,"annotation":"predicted protein","uniprotId":"YEJG_ECOLI"},"6882887":{"id":6882887,"externalId":"511145.b2182","name":"bcr","hasFamily":true,"annotation":"bicyclomycin/multidrug efflux system; Involved in sulfonamide (sulfathiazole) and bicyclomycin resistance. Probable membrane translocase","uniprotId":"BCR_ECOLI"},"6882888":{"id":6882888,"externalId":"511145.b2183","name":"rsuA","hasFamily":true,"annotation":"16S rRNA U516 pseudouridine synthase; Responsible for synthesis of pseudouridine from uracil- 516 in 16S ribosomal RNA","uniprotId":"RSUA_ECOLI"},"6882889":{"id":6882889,"externalId":"511145.b2184","name":"yejH","hasFamily":true,"annotation":"putative ATP-dependent DNA or RNA helicase","uniprotId":"YEJH_ECOLI"},"6882890":{"id":6882890,"externalId":"511145.b2185","name":"rplY","hasFamily":true,"annotation":"50S ribosomal subunit protein L25; This is one of the proteins that binds to the 5S RNA in the ribosome where it forms part of the central protuberance. Binds to the 5S rRNA independently of L5 and L18. Not required for binding of the 5S rRNA/L5/L18 subcomplex to 23S rRNA","uniprotId":"RL25_ECOLI"},"6882891":{"id":6882891,"externalId":"511145.b2186","name":"yejK","hasFamily":true,"annotation":"nucleotide associated protein","uniprotId":"NDPA_ECOLI"},"6882892":{"id":6882892,"externalId":"511145.b2187","name":"yejL","hasFamily":true,"annotation":"conserved protein, UPF0352 family","uniprotId":"YEJL_ECOLI"},"6882893":{"id":6882893,"externalId":"511145.b2188","name":"yejM","hasFamily":true,"annotation":"putative hydrolase, inner membrane","uniprotId":"YEJM_ECOLI"},"6882894":{"id":6882894,"externalId":"511145.b2192","name":"b2192","hasFamily":true,"annotation":"IS5 transposase and trans-activator"},"6882895":{"id":6882895,"externalId":"511145.b2193","name":"narP","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with NarQ or NarX; This protein activates the expression of the nitrate reductase (narGHJI) and formate dehydrogenase-N (fdnGHI) operons and represses the transcription of the fumarate reductase (frdABCD) operon in response to a nitrate/nitrite induction signal transmitted by either the NarX or NarQ proteins","uniprotId":"NARP_ECOLI"},"6882896":{"id":6882896,"externalId":"511145.b2194","name":"ccmH","hasFamily":true,"annotation":"heme lyase, CcmH subunit; May be required for the biogenesis of c-type cytochromes. Possible subunit of a heme lyase","uniprotId":"CCMH_ECOLI"},"6882897":{"id":6882897,"externalId":"511145.b2195","name":"ccmG","hasFamily":true,"annotation":"periplasmic thioredoxin of cytochrome c-type biogenesis; Involved in disulfide bond formation. Catalyzes a late, reductive step in the assembly of periplasmic c-type cytochromes, probably the reduction of disulfide bonds of the apocytochrome c to allow covalent linkage with the heme. Possible subunit of a heme lyase. DsbE is maintained in a reduced state by DsbD","uniprotId":"DSBE_ECOLI"},"6882898":{"id":6882898,"externalId":"511145.b2196","name":"ccmF","hasFamily":true,"annotation":"heme lyase, CcmF subunit; Required for the biogenesis of c-type cytochromes. Possible subunit of a heme lyase","uniprotId":"CCMF_ECOLI"},"6882899":{"id":6882899,"externalId":"511145.b2197","name":"ccmE","hasFamily":true,"annotation":"periplasmic heme chaperone; Heme chaperone required for the biogenesis of c-type cytochromes. Transiently binds heme delivered by CcmC and transfers the heme to apo-cytochromes in a process facilitated by CcmF and CcmH (By similarity)","uniprotId":"CCME_ECOLI"},"6882900":{"id":6882900,"externalId":"511145.b2198","name":"ccmD","hasFamily":true,"annotation":"cytochrome c biogenesis protein; Required for the export of heme to the periplasm for the biogenesis of c-type cytochromes (Potential)","uniprotId":"CCMD_ECOLI"},"6882901":{"id":6882901,"externalId":"511145.b2199","name":"ccmC","hasFamily":true,"annotation":"heme exporter subunit; Required for the export of heme to the periplasm for the biogenesis of c-type cytochromes","uniprotId":"CCMC_ECOLI"},"6882902":{"id":6882902,"externalId":"511145.b2200","name":"ccmB","hasFamily":true,"annotation":"heme exporter subunit; Required for the export of heme to the periplasm for the biogenesis of c-type cytochromes (By similarity)","uniprotId":"CCMB_ECOLI"},"6882903":{"id":6882903,"externalId":"511145.b2201","name":"ccmA","hasFamily":true,"annotation":"heme exporter subunit; Part of the ABC transporter complex CcmAB involved in the biogenesis of c-type cytochromes; once thought to export heme, this seems not to be the case, but its exact role is uncertain. Responsible for energy coupling to the transport system (By similarity)","uniprotId":"CCMA_ECOLI"},"6882904":{"id":6882904,"externalId":"511145.b2202","name":"napC","hasFamily":true,"annotation":"Quinol dehydrogenase, electron source for NapAB; Mediates electron flow from quinones to the NapAB complex","uniprotId":"NAPC_ECOLI"},"6882905":{"id":6882905,"externalId":"511145.b2203","name":"napB","hasFamily":true,"annotation":"nitrate reductase, small, cytochrome C550 subunit, periplasmic; Electron transfer subunit of the periplasmic nitrate reductase complex NapAB. Receives electrons from the membrane- anchored tetraheme c-type NapC protein and transfers these to NapA subunit, thus allowing electron flow between membrane and periplasm. Essential for periplasmic nitrate reduction with nitrate as the terminal electron acceptor","uniprotId":"NAPB_ECOLI"},"6882906":{"id":6882906,"externalId":"511145.b2204","name":"napH","hasFamily":true,"annotation":"ferredoxin-type protein essential for electron transfer from ubiquinol to periplasmic nitrate reductase (NapAB); Involved in electron transfer","uniprotId":"NAPH_ECOLI"},"6882907":{"id":6882907,"externalId":"511145.b2205","name":"napG","hasFamily":true,"annotation":"ferredoxin-type protein essential for electron transfer from ubiquinol to periplasmic nitrate reductase (NapAB); Involved in electron transfer (Probable)","uniprotId":"NAPG_ECOLI"},"6882908":{"id":6882908,"externalId":"511145.b2206","name":"napA","hasFamily":true,"annotation":"nitrate reductase, periplasmic, large subunit; Catalytic subunit of the periplasmic nitrate reductase (NAP). Only expressed at high levels during aerobic growth. NapAB complex receives electrons from the membrane-anchored tetraheme protein NapC, thus allowing electron flow between membrane and periplasm. Essential function for nitrate assimilation and may have a role in anaerobic metabolism (By similarity)","uniprotId":"NAPA_ECOLI"},"6882909":{"id":6882909,"externalId":"511145.b2207","name":"napD","hasFamily":true,"annotation":"assembly protein for periplasmic nitrate reductase; Plays a role in the correct assembly of subunits of the periplasmic NapAB enzyme","uniprotId":"NAPD_ECOLI"},"6882910":{"id":6882910,"externalId":"511145.b2208","name":"napF","hasFamily":true,"annotation":"ferredoxin-type protein, role in electron transfer to periplasmic nitrate reductase NapA; Involved in electron transfer","uniprotId":"NAPF_ECOLI"},"6882911":{"id":6882911,"externalId":"511145.b2209","name":"eco","hasFamily":true,"annotation":"ecotin, a serine protease inhibitor; General inhibitor of pancreatic serine proteases: inhibits chymotrypsin, trypsin, elastases, factor X, kallikrein as well as a variety of other proteases. The strength of inhibition does not appear to be correlated with a particular protease specificity","uniprotId":"ECOT_ECOLI"},"6882912":{"id":6882912,"externalId":"511145.b2210","name":"mqo","hasFamily":true,"annotation":"malate dehydrogenase, FAD/NAD(P)-binding domain","uniprotId":"MQO_ECOLI"},"6882913":{"id":6882913,"externalId":"511145.b2211","name":"yojI","hasFamily":true,"annotation":"Microcin J25 efflux pump, TolC-dependent; fused ABC transporter permease and ATP-binding components; Mediates resistance to the antibacterial peptide microcin J25. Functions as an efflux pump for microcin J25. Requires TolC for export of microcin J25","uniprotId":"YOJI_ECOLI"},"6882914":{"id":6882914,"externalId":"511145.b2212","name":"alkB","hasFamily":true,"annotation":"oxidative demethylase of N1-methyladenine or N3-methylcytosine DNA lesions; Dioxygenase that repairs alkylated DNA and RNA containing 3-methylcytosine or 1-methyladenine by oxidative demethylation. Has highest activity towards 3-methylcytosine. Has lower activity towards alkylated DNA containing ethenoadenine, and no detectable activity towards 1-methylguanine or 3-methylthymine. Accepts double-stranded and single-stranded substrates. Requires molecular oxygen, alpha-ketoglutarate and iron. Provides extensive resistance to alkylating agents such as MMS and DMS (SN2 agents), but not to  [...] ","uniprotId":"ALKB_ECOLI"},"6882915":{"id":6882915,"externalId":"511145.b2213","name":"ada","hasFamily":true,"annotation":"fused DNA-binding transcriptional dual regulator/O6-methylguanine-DNA methyltransferase; Is involved in the adaptive response to alkylation damage in DNA caused by alkylating agents. Repairs O6- methylguanine and 04-methylthymine residues in alkylated DNA by a direct and irreversible transfer of the methyl group from the base to one of its own cysteine residues (Cys-321). Also specifically repairs the Sp diastereomer of DNA methylphosphotriester lesions by the same mechanism, although the methyl transfer occurs onto a different cysteine residue (Cys-38). Can not demethylate the other d [...] ","uniprotId":"ADA_ECOLI"},"6882916":{"id":6882916,"externalId":"511145.b2214","name":"apbE","hasFamily":true,"annotation":"putative thiamine biosynthesis lipoprotein; Involved in the conversion of aminoimidazole ribotide (AIR), a purine intermediate, to the 4-amino-5-hydroxymethyl-2- methyl pyrimidine (HMP) moiety of thiamine (By similarity)","uniprotId":"APBE_ECOLI"},"6882917":{"id":6882917,"externalId":"511145.b2215","name":"ompC","hasFamily":true,"annotation":"outer membrane porin protein C; Forms pores that allow passive diffusion of small molecules across the outer membrane","uniprotId":"OMPC_ECOLI"},"6882918":{"id":6882918,"externalId":"511145.b2216","name":"rcsD","hasFamily":true,"annotation":"phosphotransfer intermediate protein in two-component regulatory system with RcsBC; May serve as a phosphotransfer intermediate between RcsC and RcsB. It may acquire a phosphoryl group at His-842 from RcsC 'Asp-866' and transmit it to 'Asp-56' of RcsB","uniprotId":"RCSD_ECOLI"},"6882919":{"id":6882919,"externalId":"511145.b2217","name":"rcsB","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with RcsC and YojN; Member of the two-component regulatory system RcsB/RcsC, which regulates the expression of genes involved in colanic acid capsule synthesis. The two-component system interacts with a phosphotransfer intermediate, RcsD, which receives a phosphoryl group from RcsC and transfers it to RscB. RcsB acts as the effector. Additional control is provided by the dependence on the alternate sigma factor, RpoN, for the synthesis of RcsB. Activity is probably up-regulated by YmgA/AriR, and possibly down-regulated b [...] ","uniprotId":"RCSB_ECOLI"},"6882920":{"id":6882920,"externalId":"511145.b2218","name":"rcsC","hasFamily":true,"annotation":"hybrid sensory kinase in two-component regulatory system with RcsB and YojN; Member of the two-component regulatory system RcsC/RcsB involved in the regulation of the expression of genes involved in colanic acid capsule synthesis. Together with RcsF, RcsC probably functions as a membrane-associated protein kinase that phosphorylates RcsB in response to environmental signals. The two- component system interacts with a phosphotransfer intermediate, RcsD, which receives a phosphoryl group from RcsC and transfers it to RscB. Activity is probably up-regulated by YmgA/AriR, and possibly down [...] ","uniprotId":"RCSC_ECOLI"},"6882921":{"id":6882921,"externalId":"511145.b2219","name":"atoS","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with AtoC; Member of the two-component regulatory system AtoS/AtoC; may activate AtoC by phosphorylation","uniprotId":"ATOS_ECOLI"},"6882922":{"id":6882922,"externalId":"511145.b2220","name":"atoC","hasFamily":true,"annotation":"fused response regulator of ato operon, in two-component system with AtoS: response regulator/sigma54 interaction protein; Member of the two-component regulatory system AtoS/AtoC involved in the transcriptional regulation of the ato genes for acetoacetate metabolism. Also an inhibitor of polyamine biosynthesis","uniprotId":"ATOC_ECOLI"},"6882923":{"id":6882923,"externalId":"511145.b2221","name":"atoD","hasFamily":true,"annotation":"acetyl-CoA:acetoacetyl-CoA transferase, alpha subunit","uniprotId":"ATOD_ECOLI"},"6882924":{"id":6882924,"externalId":"511145.b2222","name":"atoA","hasFamily":true,"annotation":"acetyl-CoA:acetoacetyl-CoA transferase, beta subunit","uniprotId":"ATOA_ECOLI"},"6882925":{"id":6882925,"externalId":"511145.b2223","name":"atoE","hasFamily":true,"annotation":"short chain fatty acid transporter; Responsible for the intake of short-chain fatty acids","uniprotId":"ATOE_ECOLI"},"6882926":{"id":6882926,"externalId":"511145.b2224","name":"atoB","hasFamily":true,"annotation":"acetyl-CoA acetyltransferase","uniprotId":"ATOB_ECOLI"},"6882927":{"id":6882927,"externalId":"511145.b2225","name":"yfaP","hasFamily":true,"annotation":"conserved protein","uniprotId":"YFAP_ECOLI"},"6882928":{"id":6882928,"externalId":"511145.b2226","name":"yfaQ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YFAQ_ECOLI"},"6882929":{"id":6882929,"externalId":"511145.b2229","name":"yfaT","hasFamily":true,"annotation":"predicted protein","uniprotId":"YFAT_ECOLI"},"6882930":{"id":6882930,"externalId":"511145.b2230","name":"yfaA","hasFamily":true,"annotation":"conserved protein, DUF2138 family","uniprotId":"YFAA_ECOLI"},"6882931":{"id":6882931,"externalId":"511145.b2231","name":"gyrA","hasFamily":true,"annotation":"DNA gyrase (type II topoisomerase), subunit A; DNA gyrase negatively supercoils closed circular double- stranded DNA in an ATP-dependent manner and also catalyzes the interconversion of other topological isomers of double-stranded DNA rings, including catenanes and knotted rings (By similarity)","uniprotId":"GYRA_ECOLI"},"6882932":{"id":6882932,"externalId":"511145.b2232","name":"ubiG","hasFamily":true,"annotation":"bifunctional 3-demethylubiquinone-9 3-methyltransferase/ 2-octaprenyl-6-hydroxy phenol methylase; Non-specific O-methyltransferase that catalyzes the 2 O- methylation steps in the ubiquinone biosynthetic pathway (By similarity)","uniprotId":"UBIG_ECOLI"},"6882933":{"id":6882933,"externalId":"511145.b2233","name":"yfaL","hasFamily":false,"annotation":"adhesin","uniprotId":"YFAL_ECOLI"},"6882934":{"id":6882934,"externalId":"511145.b2234","name":"nrdA","hasFamily":true,"annotation":"ribonucleoside-diphosphate reductase 1, alpha subunit; Provides the precursors necessary for DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding ribonucleotides. R1 contains the binding sites for both substrates and allosteric effectors and carries out the actual reduction of the ribonucleotide. It also provides redox- active cysteines","uniprotId":"RIR1_ECOLI"},"6882935":{"id":6882935,"externalId":"511145.b2235","name":"nrdB","hasFamily":true,"annotation":"ribonucleoside-diphosphate reductase 1, beta subunit, ferritin-like protein; Provides the precursors necessary for DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding ribonucleotides. R2 contains the tyrosyl radical required for catalysis","uniprotId":"RIR2_ECOLI"},"6882936":{"id":6882936,"externalId":"511145.b2236","name":"yfaE","hasFamily":true,"annotation":"ferredoxin involved with ribonucleotide reductase diferric-tyrosyl radical (Y*) cofactor maintenance","uniprotId":"YFAE_ECOLI"},"6882937":{"id":6882937,"externalId":"511145.b2237","name":"inaA","hasFamily":true,"annotation":"conserved protein, acid-induced; May be an environmental sensor responsive to several stimuli, including internal pH, proton motive force, temperature, and possibly other unknown factors","uniprotId":"INAA_ECOLI"},"6882938":{"id":6882938,"externalId":"511145.b2239","name":"glpQ","hasFamily":true,"annotation":"periplasmic glycerophosphodiester phosphodiesterase; Glycerophosphoryl diester phosphodiesterase hydrolyzes deacylated phospholipids to G3P and the corresponding alcohols","uniprotId":"GLPQ_ECOLI"},"6882939":{"id":6882939,"externalId":"511145.b2240","name":"glpT","hasFamily":true,"annotation":"sn-glycerol-3-phosphate transporter; Responsible for glycerol-3-phosphate uptake","uniprotId":"GLPT_ECOLI"},"6882940":{"id":6882940,"externalId":"511145.b2241","name":"glpA","hasFamily":true,"annotation":"sn-glycerol-3-phosphate dehydrogenase (anaerobic), large subunit, FAD/NAD(P)-binding; Conversion of glycerol 3-phosphate to dihydroxyacetone. Uses fumarate or nitrate as electron acceptor","uniprotId":"GLPA_ECOLI"},"6882941":{"id":6882941,"externalId":"511145.b2242","name":"glpB","hasFamily":true,"annotation":"sn-glycerol-3-phosphate dehydrogenase (anaerobic), membrane anchor subunit; Conversion of glycerol 3-phosphate to dihydroxyacetone. Uses fumarate or nitrate as electron acceptor (By similarity)","uniprotId":"GLPB_ECOLI"},"6882942":{"id":6882942,"externalId":"511145.b2243","name":"glpC","hasFamily":true,"annotation":"anaerobic sn-glycerol-3-phosphate dehydrogenase, C subunit, 4Fe-4S iron-sulfur cluster; Electron transfer protein; may also function as the membrane anchor for the GlpAB dimer","uniprotId":"GLPC_ECOLI"},"6882943":{"id":6882943,"externalId":"511145.b2244","name":"yfaD","hasFamily":true,"annotation":"conserved protein","uniprotId":"YFAD_ECOLI"},"6882944":{"id":6882944,"externalId":"511145.b2245","name":"rhmA","hasFamily":true,"annotation":"2-keto-3-deoxy-L-rhamnonate aldolase; Catalyzes the reversible retro-aldol cleavage of 2-keto- 3-deoxy-L-rhamnonate (KDR) to pyruvate and lactaldehyde. 2-keto-3- deoxy-L-mannonate, 2-keto-3-deoxy-L-lyxonate and 4-hydroxy-2- ketoheptane-1,7-dioate (HKHD) are also reasonably good substrates, although 2-keto-3-deoxy-L-rhamnonate is likely to be the physiological substrate","uniprotId":"RHMA_ECOLI"},"6882945":{"id":6882945,"externalId":"511145.b2246","name":"rhmT","hasFamily":true,"annotation":"putative L-rhamnonate transporter","uniprotId":"RHMT_ECOLI"},"6882946":{"id":6882946,"externalId":"511145.b2247","name":"rhmD","hasFamily":true,"annotation":"L-rhamnonate dehydratase; Catalyzes the dehydration of L-rhamnonate to 2-keto-3- deoxy-L-rhamnonate (KDR). Can also dehydrate L-lyxonate, L- mannonate and D-gulonate, although less efficiently, but not 2- keto-4-hydroxyheptane-1,7-dioate","uniprotId":"RHMD_ECOLI"},"6882947":{"id":6882947,"externalId":"511145.b2248","name":"rhmR","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator for the rhm operon","uniprotId":"RHMR_ECOLI"},"6882948":{"id":6882948,"externalId":"511145.b2249","name":"yfaY","hasFamily":true,"annotation":"conserved protein; Does not have nicotinamide-nucleotide (NMN) amidohydrolase activity","uniprotId":"CINAL_ECOLI"},"6882949":{"id":6882949,"externalId":"511145.b2250","name":"yfaZ","hasFamily":true,"annotation":"outer membrane protein, possible porin","uniprotId":"YFAZ_ECOLI"},"6882950":{"id":6882950,"externalId":"511145.b2251","name":"nudI","hasFamily":true,"annotation":"nucleoside triphosphatase; Catalyzes the hydrolysis of nucleoside triphosphates, with a preference for pyrimidine deoxynucleoside triphosphates (dUTP, dTTP and dCTP) (By similarity)","uniprotId":"NUDI_ECOLI"},"6882951":{"id":6882951,"externalId":"511145.b2252","name":"ais","hasFamily":true,"annotation":"putative LPS core heptose(II)-phosphate phosphatase; Catalyzes the dephosphorylation of heptose(II) of the outer membrane lipopolysaccharide core (By similarity)","uniprotId":"AIS_ECOLI"},"6882952":{"id":6882952,"externalId":"511145.b2253","name":"arnB","hasFamily":true,"annotation":"uridine 5'-(beta-1-threo-pentapyranosyl-4-ulose diphosphate) aminotransferase, PLP-dependent; Catalyzes the conversion of UDP-4-keto-arabinose (UDP- Ara4O) to UDP-4-amino-4-deoxy-L-arabinose (UDP-L-Ara4N). The modified arabinose is attached to lipid A and is required for resistance to polymyxin and cationic antimicrobial peptides (By similarity)","uniprotId":"ARNB_ECOLI"},"6882953":{"id":6882953,"externalId":"511145.b2254","name":"arnC","hasFamily":true,"annotation":"undecaprenyl phosphate-L-Ara4FN transferase; Catalyzes the transfer of 4-deoxy-4-formamido-L- arabinose from UDP to undecaprenyl phosphate. The modified arabinose is attached to lipid A and is required for resistance to polymyxin and cationic antimicrobial peptides (By similarity)","uniprotId":"ARNC_ECOLI"},"6882954":{"id":6882954,"externalId":"511145.b2255","name":"arnA","hasFamily":true,"annotation":"fused UDP-L-Ara4N formyltransferase/UDP-GlcA C-4'-decarboxylase; Bifunctional enzyme that catalyzes the oxidative decarboxylation of UDP-glucuronic acid (UDP-GlcUA) to UDP-4-keto- arabinose (UDP-Ara4O) and the addition of a formyl group to UDP-4- amino-4-deoxy-L-arabinose (UDP-L-Ara4N) to form UDP-L-4-formamido- arabinose (UDP-L-Ara4FN). The modified arabinose is attached to lipid A and is required for resistance to polymyxin and cationic antimicrobial peptides (By similarity)","uniprotId":"ARNA_ECOLI"},"6882955":{"id":6882955,"externalId":"511145.b2256","name":"arnD","hasFamily":true,"annotation":"Undecaprenyl phosphate-alpha-L-ara4FN deformylase; Catalyzes the deformylation of 4-deoxy-4-formamido-L- arabinose-phosphoundecaprenol to 4-amino-4-deoxy-L-arabinose- phosphoundecaprenol. The modified arabinose is attached to lipid A and is required for resistance to polymyxin and cationic antimicrobial peptides (By similarity)","uniprotId":"ARND_ECOLI"},"6882956":{"id":6882956,"externalId":"511145.b2257","name":"arnT","hasFamily":true,"annotation":"4-amino-4-deoxy-L-arabinose transferase; Catalyzes the transfer of the L-Ara4N moiety of the glycolipid undecaprenyl phosphate-alpha-L-Ara4N to lipid A. The modified arabinose is attached to lipid A and is required for resistance to polymyxin and cationic antimicrobial peptides (By similarity)","uniprotId":"ARNT_ECOLI"},"6882957":{"id":6882957,"externalId":"511145.b2258","name":"arnF","hasFamily":true,"annotation":"undecaprenyl phosphate-alpha-L-ara4N exporter; flippase ArnEF subunit; Translocates 4-amino-4-deoxy-L-arabinose- phosphoundecaprenol (alpha-L-Ara4N-phosphoundecaprenol) from the cytoplasmic to the periplasmic side of the inner membrane (By similarity)","uniprotId":"ARNF_ECOLI"},"6882958":{"id":6882958,"externalId":"511145.b2259","name":"pmrD","hasFamily":true,"annotation":"inactive two-component system connector protein"},"6882959":{"id":6882959,"externalId":"511145.b2260","name":"menE","hasFamily":true,"annotation":"o-succinylbenzoate-CoA ligase; Converts 2-succinylbenzoate (OSB) to 2-succinylbenzoyl- CoA (OSB-CoA)","uniprotId":"MENE_ECOLI"},"6882960":{"id":6882960,"externalId":"511145.b2261","name":"menC","hasFamily":true,"annotation":"o-succinylbenzoyl-CoA synthase; Converts SHCHC to OSB (By similarity)","uniprotId":"MENC_ECOLI"},"6882961":{"id":6882961,"externalId":"511145.b2262","name":"menB","hasFamily":true,"annotation":"dihydroxynaphthoic acid synthetase; Converts o-succinylbenzoyl-CoA (OSB-CoA) to 1,4- dihydroxy-2-naphthoyl-CoA (DHNA-CoA)","uniprotId":"MENB_ECOLI"},"6882962":{"id":6882962,"externalId":"511145.b2263","name":"menH","hasFamily":true,"annotation":"2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate synthase; Catalyzes a proton abstraction reaction that results in 2,5-elimination of pyruvate from 2-succinyl-5-enolpyruvyl-6- hydroxy-3-cyclohexene-1-carboxylate (SEPHCHC) and the formation of 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate (SHCHC). Is also able to catalyze the hydrolysis of the thioester bond in palmitoyl-CoA in vitro (By similarity)","uniprotId":"MENH_ECOLI"},"6882963":{"id":6882963,"externalId":"511145.b2264","name":"menD","hasFamily":true,"annotation":"bifunctional 2-oxoglutarate decarboxylase/ SHCHC synthase; Catalyzes the thiamine diphosphate-dependent decarboxylation of 2-oxoglutarate and the subsequent addition of the resulting succinic semialdehyde-thiamine pyrophosphate anion to isochorismate to yield 2-succinyl-5-enolpyruvyl-6-hydroxy-3- cyclohexene-1-carboxylate (SEPHCHC) (By similarity)","uniprotId":"MEND_ECOLI"},"6882964":{"id":6882964,"externalId":"511145.b2265","name":"menF","hasFamily":true,"annotation":"isochorismate synthase 2","uniprotId":"MENF_ECOLI"},"6882965":{"id":6882965,"externalId":"511145.b2266","name":"elaB","hasFamily":true,"annotation":"conserved protein","uniprotId":"ELAB_ECOLI"},"6882966":{"id":6882966,"externalId":"511145.b2267","name":"elaA","hasFamily":true,"annotation":"putative acyltransferase with acyl-CoA N-acyltransferase domain","uniprotId":"ELAA_ECOLI"},"6882967":{"id":6882967,"externalId":"511145.b2268","name":"rbn","hasFamily":true,"annotation":"RNase BN, tRNA processing enzyme; Zinc phosphodiesterase, which has both exoribonuclease and endoribonuclease activities, depending on the nature of the substrate and of the added divalent cation, and on its 3'-terminal structure. Can process the 3' termini of both CCA-less and CCA- containing tRNA precursors. CCA-less tRNAs are cleaved endonucleolytically after the discriminator base, whereas residues following the CCA sequence can be removed exonucleolytically or endonucleolytically in CCA-containing molecules. Does not remove the CCA sequence. May also be involved in the degradation [...] ","uniprotId":"RBN_ECOLI"},"6882968":{"id":6882968,"externalId":"511145.b2269","name":"elaD","hasFamily":true,"annotation":"protease, capable of cleaving an AMC-ubiquitin model substrate; Protease that can act as an efficient and specific deubiquitinating enzyme in vitro. Does not possess desumoylating and deneddylating activities. The physiological substrate is unknown","uniprotId":"ELAD_ECOLI"},"6882969":{"id":6882969,"externalId":"511145.b2270","name":"yfbK","hasFamily":true,"annotation":"conserved protein","uniprotId":"YFBK_ECOLI"},"6882970":{"id":6882970,"externalId":"511145.b2271","name":"yfbL","hasFamily":true,"annotation":"putative peptidase","uniprotId":"YFBL_ECOLI"},"6882971":{"id":6882971,"externalId":"511145.b2272","name":"yfbM","hasFamily":false,"annotation":"conserved protein, DUF1877 family","uniprotId":"YFBM_ECOLI"},"6882972":{"id":6882972,"externalId":"511145.b2273","name":"yfbN","hasFamily":false,"annotation":"conserved protein","uniprotId":"YFBN_ECOLI"},"6882973":{"id":6882973,"externalId":"511145.b2274","name":"yfbO","hasFamily":false,"annotation":"conserved protein","uniprotId":"YFBO_ECOLI"},"6882974":{"id":6882974,"externalId":"511145.b2275","name":"yfbP","hasFamily":false,"annotation":"TPR-like repeats-containing protein","uniprotId":"YFBP_ECOLI"},"6882975":{"id":6882975,"externalId":"511145.b2276","name":"nuoN","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, membrane subunit N; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient (By similarity)","uniprotId":"NUON_ECOLI"},"6882976":{"id":6882976,"externalId":"511145.b2277","name":"nuoM","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, membrane subunit M; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"NUOM_ECOLI"},"6882977":{"id":6882977,"externalId":"511145.b2278","name":"nuoL","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, membrane subunit L; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"NUOL_ECOLI"},"6882978":{"id":6882978,"externalId":"511145.b2279","name":"nuoK","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, membrane subunit K; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient (By similarity)","uniprotId":"NUOK_ECOLI"},"6882979":{"id":6882979,"externalId":"511145.b2280","name":"nuoJ","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, membrane subunit J; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"NUOJ_ECOLI"},"6882980":{"id":6882980,"externalId":"511145.b2281","name":"nuoI","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, chain I; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient (By similarity)","uniprotId":"NUOI_ECOLI"},"6882981":{"id":6882981,"externalId":"511145.b2282","name":"nuoH","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, membrane subunit H; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient. This subunit may bind ubiquinone (By similarity)","uniprotId":"NUOH_ECOLI"},"6882982":{"id":6882982,"externalId":"511145.b2283","name":"nuoG","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, chain G; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"NUOG_ECOLI"},"6882983":{"id":6882983,"externalId":"511145.b2284","name":"nuoF","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, chain F; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"NUOF_ECOLI"},"6882984":{"id":6882984,"externalId":"511145.b2285","name":"nuoE","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, chain E; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"NUOE_ECOLI"},"6882985":{"id":6882985,"externalId":"511145.b2286","name":"nuoC","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, fused CD subunit; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient (By similarity)","uniprotId":"NUOCD_ECOLI"},"6882986":{"id":6882986,"externalId":"511145.b2287","name":"nuoB","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, chain B; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient (By similarity)","uniprotId":"NUOB_ECOLI"},"6882987":{"id":6882987,"externalId":"511145.b2288","name":"nuoA","hasFamily":true,"annotation":"NADH:ubiquinone oxidoreductase, membrane subunit A; NDH-1 shuttles electrons from NADH, via FMN and iron- sulfur (Fe-S) centers, to quinones in the respiratory chain. The immediate electron acceptor for the enzyme in this species is believed to be ubiquinone. Couples the redox reaction to proton translocation (for every two electrons transferred, four hydrogen ions are translocated across the cytoplasmic membrane), and thus conserves the redox energy in a proton gradient","uniprotId":"NUOA_ECOLI"},"6882988":{"id":6882988,"externalId":"511145.b2289","name":"lrhA","hasFamily":true,"annotation":"DNA-binding transcriptional repressor of flagellar, motility and chemotaxis genes; Not known, does not seem to act on the proton translocating NADH dehydrogenase genes (nuoA-N) which are part of the lrhA operon","uniprotId":"LRHA_ECOLI"},"6882989":{"id":6882989,"externalId":"511145.b2290","name":"alaA","hasFamily":true,"annotation":"valine-pyruvate aminotransferase 2; Involved in the biosynthesis of alanine","uniprotId":"ALAA_ECOLI"},"6882990":{"id":6882990,"externalId":"511145.b2291","name":"yfbR","hasFamily":true,"annotation":"5'-nucleotidase; Nucleotidase that shows strict specificity toward deoxyribonucleoside 5'-monophosphates and does not dephosphorylate 5'-ribonucleotides or ribonucleoside 3'-monophosphates. Might be involved in the regulation of all dNTP pools in E.coli","uniprotId":"YFBR_ECOLI"},"6882991":{"id":6882991,"externalId":"511145.b2292","name":"yfbS","hasFamily":true,"annotation":"putative transporter","uniprotId":"YFBS_ECOLI"},"6882992":{"id":6882992,"externalId":"511145.b2293","name":"yfbT","hasFamily":true,"annotation":"sugar phosphatas; Catalyzes the dephosphorylation of different sugar phosphates including ribose-5-phosphate (Ribu5P), fructose-1- phosphate (Fru1P), fructose-6-phosphate (Fru6P), glucose-6- phosphate (Glu6P). Selectively hydrolyzes beta-D-glucose-1- phosphate (bGlu1P) and has no activity with the alpha form","uniprotId":"YFBT_ECOLI"},"6882993":{"id":6882993,"externalId":"511145.b2294","name":"yfbU","hasFamily":true,"annotation":"conserved protein, UPF0304 family","uniprotId":"YFBU_ECOLI"},"6882994":{"id":6882994,"externalId":"511145.b2295","name":"yfbV","hasFamily":true,"annotation":"inner membrane protein, UPF0208 family","uniprotId":"YFBV_ECOLI"},"6882995":{"id":6882995,"externalId":"511145.b2296","name":"ackA","hasFamily":true,"annotation":"acetate kinase A and propionate kinase 2; Catalyzes the formation of acetyl phosphate from acetate and ATP. Can also catalyze the reverse reaction. During anaerobic growth of the organism, this enzyme is also involved in the synthesis of most of the ATP formed catabolically","uniprotId":"ACKA_ECOLI"},"6882996":{"id":6882996,"externalId":"511145.b2297","name":"pta","hasFamily":true,"annotation":"phosphate acetyltransferase; Involved in acetate metabolism. Catalyzes the reversible interconversion of acetyl-CoA and acetyl phosphate. The direction of the overall reaction changes depending on growth conditions. On minimal medium acetyl-CoA is generated. In rich medium acetyl-CoA is converted to acetate and allowing the cell to dump the excess of acetylation potential in exchange for energy in the form of ATP","uniprotId":"PTA_ECOLI"},"6882997":{"id":6882997,"externalId":"511145.b2298","name":"yfcC","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YFCC_ECOLI"},"6882998":{"id":6882998,"externalId":"511145.b2299","name":"yfcD","hasFamily":true,"annotation":"putative NUDIX hydrolase","uniprotId":"YFCD_ECOLI"},"6882999":{"id":6882999,"externalId":"511145.b2300","name":"yfcE","hasFamily":true,"annotation":"phosphodiesterase activity on bis-pNPP; Shows phosphodiesterase activity, hydrolyzing phosphodiesters bonds in the artificial chromogenic substrates bis-p-nitrophenyl phosphate (bis-pNPP), and less efficiently thymidine 5'-monophosphate p-nitrophenyl ester (pNP-TMP) and p- nitrophenylphosphorylcholine (pNPPC). The physiological substrate is unknown","uniprotId":"YFCE_ECOLI"},"6883000":{"id":6883000,"externalId":"511145.b2301","name":"yfcF","hasFamily":true,"annotation":"glutathione S-transferase; Exhibits glutathione (GSH) S-transferase activity toward 1-chloro-2,4-dinitrobenzene (CDNB); however this activity is as low as 1% of that of GstA. Also displays a GSH-dependent peroxidase activity toward cumene hydroperoxide. Is involved in defense against oxidative stress, probably via its peroxidase activity","uniprotId":"YFCF_ECOLI"},"6883001":{"id":6883001,"externalId":"511145.b2302","name":"yfcG","hasFamily":true,"annotation":"GSH-dependent disulfide bond oxidoreductase; Exhibits a very robust glutathione (GSH)-dependent disulfide-bond reductase activity toward the model substrate, 2- hydroxyethyl disulfide; the actual physiological substrates are not known. Has also a low GSH-dependent hydroperoxidase activity toward cumene hydroperoxide, but does not reduce H(2)O(2), tert- butyl hydroperoxide, benzyl peroxide, or lauroyl peroxide. Exhibits little or no GSH transferase activity with most typical electrophilic substrates, and has no detectable transferase activity using glutathionylspermidine (GspSH) as the  [...] ","uniprotId":"YFCG_ECOLI"},"6883002":{"id":6883002,"externalId":"511145.b2303","name":"folX","hasFamily":true,"annotation":"D-erythro-7,8-dihydroneopterin triphosphate 2'-epimerase and dihydroneopterin aldolase; Catalyzes the epimerization of carbon 2' of the side chain of dihydroneopterin triphosphate (H2NTP) to form dihydromonapterin triphosphate (H2MTP)","uniprotId":"FOLX_ECOLI"},"6883003":{"id":6883003,"externalId":"511145.b2304","name":"yfcH","hasFamily":true,"annotation":"conserved protein with NAD(P)-binding Rossmann-fold domain","uniprotId":"YFCH_ECOLI"},"6883004":{"id":6883004,"externalId":"511145.b2305","name":"yfcI","hasFamily":true,"annotation":"conserved protein","uniprotId":"YFCI_ECOLI"},"6883005":{"id":6883005,"externalId":"511145.b2306","name":"hisP","hasFamily":true,"annotation":"histidine/lysine/arginine/ornithine transporter subunit; Part of the binding-protein-dependent transport system for histidine. Probably responsible for energy coupling to the transport system","uniprotId":"HISP_ECOLI"},"6883006":{"id":6883006,"externalId":"511145.b2307","name":"hisM","hasFamily":true,"annotation":"histidine/lysine/arginine/ornithine transporter subunit; Part of the binding-protein-dependent transport system for histidine; probably responsible for the translocation of the substrate across the membrane","uniprotId":"HISM_ECOLI"},"6883007":{"id":6883007,"externalId":"511145.b2308","name":"hisQ","hasFamily":true,"annotation":"histidine/lysine/arginine/ornithine transporter permease subunit; Part of the binding-protein-dependent transport system for histidine; probably responsible for the translocation of the substrate across the membrane","uniprotId":"HISQ_ECOLI"},"6883008":{"id":6883008,"externalId":"511145.b2309","name":"hisJ","hasFamily":true,"annotation":"histidine/lysine/arginine/ornithine transporter subunit; Component of the high-affinity histidine permease, a binding-protein-dependent transport system. The other components are proteins HisQ, HisM, and HisP","uniprotId":"HISJ_ECOLI"},"6883009":{"id":6883009,"externalId":"511145.b2310","name":"argT","hasFamily":true,"annotation":"lysine/arginine/ornithine transporter subunit; This periplasmic binding protein is involved in an arginine transport system. ArgT and histidine-binding protein J (HisJ) interact with a common membrane-bound receptor, HisP","uniprotId":"ARGT_ECOLI"},"6883010":{"id":6883010,"externalId":"511145.b2311","name":"ubiX","hasFamily":true,"annotation":"3-octaprenyl-4-hydroxybenzoate carboxy-lyase","uniprotId":"UBIX_ECOLI"},"6883011":{"id":6883011,"externalId":"511145.b2312","name":"purF","hasFamily":true,"annotation":"amidophosphoribosyltransferase","uniprotId":"PUR1_ECOLI"},"6883012":{"id":6883012,"externalId":"511145.b2313","name":"cvpA","hasFamily":true,"annotation":"membrane protein required for colicin V production; Required for colicin V production from plasmid IncFI ColV3-K30","uniprotId":"CVPA_ECOLI"},"6883013":{"id":6883013,"externalId":"511145.b2314","name":"dedD","hasFamily":true,"annotation":"membrane-anchored periplasmic protein involved in septation","uniprotId":"DEDD_ECOLI"},"6883014":{"id":6883014,"externalId":"511145.b2315","name":"folC","hasFamily":true,"annotation":"bifunctional folylpolyglutamate synthase/ dihydrofolate synthase; Conversion of folates to polyglutamate derivatives","uniprotId":"FOLC_ECOLI"},"6883015":{"id":6883015,"externalId":"511145.b2316","name":"accD","hasFamily":true,"annotation":"acetyl-CoA carboxylase, beta (carboxyltransferase) subunit; Component of the acetyl coenzyme A carboxylase (ACC) complex. Biotin carboxylase (BC) catalyzes the carboxylation of biotin on its carrier protein (BCCP) and then the CO(2) group is transferred by the transcarboxylase to acetyl-CoA to form malonyl- CoA (By similarity)","uniprotId":"ACCD_ECOLI"},"6883016":{"id":6883016,"externalId":"511145.b2317","name":"dedA","hasFamily":true,"annotation":"conserved inner membrane protein","uniprotId":"DEDA_ECOLI"},"6883017":{"id":6883017,"externalId":"511145.b2318","name":"truA","hasFamily":true,"annotation":"pseudouridylate synthase I; Formation of pseudouridine at positions 38, 39 and 40 in the anticodon stem and loop of transfer RNAs (By similarity)","uniprotId":"TRUA_ECOLI"},"6883018":{"id":6883018,"externalId":"511145.b2319","name":"usg","hasFamily":true,"annotation":"putative semialdehyde dehydrogenase","uniprotId":"USG_ECOLI"},"6883019":{"id":6883019,"externalId":"511145.b2320","name":"pdxB","hasFamily":true,"annotation":"erythronate-4-phosphate dehydrogenase; Catalyzes the oxidation of erythronate-4-phosphate to 3- hydroxy-2-oxo-4-phosphonooxybutanoate (By similarity)","uniprotId":"PDXB_ECOLI"},"6883020":{"id":6883020,"externalId":"511145.b2321","name":"div","hasFamily":true,"annotation":"putative flagella assembly protein; Acts as a regulator of flagellar gene expression by modulating the protein level of the anti sigma factor FlgM upon sensing ring completion or hook elongation. Flk could inhibit FlgM secretion by acting as a braking system for the flagellar- associated type III secretion system (By similarity)","uniprotId":"FLK_ECOLI"},"6883021":{"id":6883021,"externalId":"511145.b2322","name":"yfcJ","hasFamily":true,"annotation":"predicted transporter","uniprotId":"YFCJ_ECOLI"},"6883022":{"id":6883022,"externalId":"511145.b2323","name":"fabB","hasFamily":true,"annotation":"3-oxoacyl-[acyl-carrier-protein] synthase I; Catalyzes the condensation reaction of fatty acid synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP. Specific for elongation from C-10 to unsaturated C-16 and C-18 fatty acids","uniprotId":"FABB_ECOLI"},"6883023":{"id":6883023,"externalId":"511145.b2324","name":"mnmC","hasFamily":true,"annotation":"fused 5-methylaminomethyl-2-thiouridine-forming enzyme methyltransferase and FAD-dependent demodification enzyme; Catalyzes the last two steps in the biosynthesis of 5- methylaminomethyl-2-thiouridine (mnm(5)s(2)U) at the wobble position (U34) in tRNA. Catalyzes the FAD-dependent demodification of cmnm(5)s(2)U34 to nm(5)s(2)U34, followed by the transfer of a methyl group from S-adenosyl-L-methionine to nm(5)s(2)U34, to form mnm(5)s(2)U34 (By similarity)","uniprotId":"MNMC_ECOLI"},"6883024":{"id":6883024,"externalId":"511145.b2325","name":"yfcL","hasFamily":true,"annotation":"predicted protein","uniprotId":"YFCL_ECOLI"},"6883025":{"id":6883025,"externalId":"511145.b2326","name":"epmC","hasFamily":true,"annotation":"conserved protein; Is involved in the final hydroxylation step of the post- translational modification of translation elongation factor P (EF- P) on 'Lys-34'. Acts after beta-lysylation of 'Lys-34' by EpmA and EpmB. EpmC adds an oxygen atom to the C5 position of 'Lys-34' and does not modify the added beta-lysine","uniprotId":"EPMC_ECOLI"},"6883026":{"id":6883026,"externalId":"511145.b2327","name":"yfcA","hasFamily":true,"annotation":"inner membrane protein, UPF0721 family","uniprotId":"YFCA_ECOLI"},"6883027":{"id":6883027,"externalId":"511145.b2328","name":"mepA","hasFamily":true,"annotation":"murein DD-endopeptidase; Involved in the removal of murein from the sacculus. May also facilitate integration of nascent murein strands into the sacculus by cleaving the peptide bonds between neighboring strands in mature murein (By similarity)","uniprotId":"MEPA_ECOLI"},"6883028":{"id":6883028,"externalId":"511145.b2329","name":"aroC","hasFamily":true,"annotation":"chorismate synthase","uniprotId":"AROC_ECOLI"},"6883029":{"id":6883029,"externalId":"511145.b2330","name":"prmB","hasFamily":true,"annotation":"N5-glutamine methyltransferase; Specifically methylates the 50S ribosomal protein L3 on 'Gln-150'. Does not methylate the translation termination release factors RF1 and RF2","uniprotId":"PRMB_ECOLI"},"6883030":{"id":6883030,"externalId":"511145.b2331","name":"yfcN","hasFamily":true,"annotation":"putative DNA endonuclease","uniprotId":"YFCN_ECOLI"},"6883031":{"id":6883031,"externalId":"511145.b2332","name":"yfcO","hasFamily":true,"annotation":"predicted protein","uniprotId":"YFCO_ECOLI"},"6883032":{"id":6883032,"externalId":"511145.b2333","name":"yfcP","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YFCP_ECOLI"},"6883033":{"id":6883033,"externalId":"511145.b2334","name":"yfcQ","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YFCQ_ECOLI"},"6883034":{"id":6883034,"externalId":"511145.b2335","name":"yfcR","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YFCR_ECOLI"},"6883035":{"id":6883035,"externalId":"511145.b2336","name":"yfcS","hasFamily":true,"annotation":"putative periplasmic pilus chaperone; Could be required for the biogenesis of a putative fimbria (By similarity)","uniprotId":"YFCS_ECOLI"},"6883036":{"id":6883036,"externalId":"511145.b2339","name":"yfcV","hasFamily":true,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YFCV_ECOLI"},"6883037":{"id":6883037,"externalId":"511145.b2340","name":"sixA","hasFamily":true,"annotation":"phosphohistidine phosphatase; Exhibits phosphohistidine phosphatase activity towards the HPt domain of the ArcB sensor involved in the multistep His- Asp phosphorelay","uniprotId":"SIXA_ECOLI"},"6883038":{"id":6883038,"externalId":"511145.b2341","name":"fadJ","hasFamily":true,"annotation":"fused enoyl-CoA hydratase and epimerase and isomerase/3-hydroxyacyl-CoA dehydrogenase; Catalyzes the formation of a hydroxyacyl-CoA by addition of water on enoyl-CoA. Also exhibits 3-hydroxyacyl-CoA epimerase and 3-hydroxyacyl-CoA dehydrogenase activities. Strongly involved in the anaerobic degradation of long and medium-chain fatty acids in the presence of nitrate and weakly involved in the aerobic degradation of long-chain fatty acids","uniprotId":"FADJ_ECOLI"},"6883039":{"id":6883039,"externalId":"511145.b2342","name":"fadI","hasFamily":true,"annotation":"beta-ketoacyl-CoA thiolase, anaerobic, subunit; Catalyzes the final step of fatty acid oxidation in which acetyl-CoA is released and the CoA ester of a fatty acid two carbons shorter is formed. Strongly involved in the anaerobic degradation of long and medium-chain fatty acids in the presence of nitrate and weakly involved in the aerobic degradation of long- chain fatty acids","uniprotId":"FADI_ECOLI"},"6883040":{"id":6883040,"externalId":"511145.b2343","name":"yfcZ","hasFamily":true,"annotation":"conserved protein, UPF0381 family","uniprotId":"YFCZ_ECOLI"},"6883041":{"id":6883041,"externalId":"511145.b2344","name":"fadL","hasFamily":true,"annotation":"long-chain fatty acid outer membrane transporter; Involved in translocation of long-chain fatty acids across the outer membrane. It is a receptor for the bacteriophage T2. FadL may form a specific channel","uniprotId":"FADL_ECOLI"},"6883042":{"id":6883042,"externalId":"511145.b2345","name":"yfdF","hasFamily":false,"annotation":"predicted protein","uniprotId":"YFDF_ECOLI"},"6883043":{"id":6883043,"externalId":"511145.b2346","name":"mlaA","hasFamily":true,"annotation":"ABC transporter maintaining OM lipid asymmetry, OM lipoprotein component; Actively prevents phospholipid accumulation at the cell surface. Probably maintains lipid asymmetry in the outer membrane by retrograde trafficking of phospholipids from the outer membrane to the inner membrane","uniprotId":"MLAA_ECOLI"},"6883044":{"id":6883044,"externalId":"511145.b2347","name":"yfdC","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YFDC_ECOLI"},"6883045":{"id":6883045,"externalId":"511145.b2349","name":"int","hasFamily":false,"annotation":"CPS-53 (KpLE1) prophage; Integrase is necessary for integration of the phage into the host genome by site-specific recombination. In conjunction with excisionase, integrase is also necessary for excision of the prophage from the host genome","uniprotId":"INTS_ECOLI"},"6883046":{"id":6883046,"externalId":"511145.b2350","name":"yfdG","hasFamily":true,"annotation":"CPS-53 (KpLE1) prophage; Involved in O antigen modification. Involved in the translocation of bactoprenol-linked glucose across the cytoplasmic membrane (By similarity)","uniprotId":"GTRA_ECOLI"},"6883047":{"id":6883047,"externalId":"511145.b2351","name":"yfdH","hasFamily":true,"annotation":"CPS-53 (KpLE1) prophage; Involved in O antigen modification. Catalyzes the transfer of the glucose residue from UDP-glucose to a lipid carrier (By similarity)","uniprotId":"GTRB_ECOLI"},"6883048":{"id":6883048,"externalId":"511145.b2352","name":"yfdI","hasFamily":false,"annotation":"serotype-specific glucosyl transferase, CPS-53 (KpLE1) prophage","uniprotId":"YFDI_ECOLI"},"6883049":{"id":6883049,"externalId":"511145.b2354","name":"yfdK","hasFamily":true,"annotation":"CPS-53 (KpLE1) prophage","uniprotId":"YFDK_ECOLI"},"6883050":{"id":6883050,"externalId":"511145.b2357","name":"yfdN","hasFamily":false,"annotation":"CPS-53 (KpLE1) prophage","uniprotId":"YFDN_ECOLI"},"6883051":{"id":6883051,"externalId":"511145.b2359","name":"yfdP","hasFamily":false,"annotation":"CPS-53 (KpLE1) prophage","uniprotId":"YFDP_ECOLI"},"6883052":{"id":6883052,"externalId":"511145.b2360","name":"yfdQ","hasFamily":false,"annotation":"CPS-53 (KpLE1) prophage","uniprotId":"YFDQ_ECOLI"},"6883053":{"id":6883053,"externalId":"511145.b2361","name":"yfdR","hasFamily":false,"annotation":"CPS-53 (KpLE1) prophage; conserved protein","uniprotId":"YFDR_ECOLI"},"6883054":{"id":6883054,"externalId":"511145.b2362","name":"yfdS","hasFamily":false,"annotation":"CPS-53 (KpLE1) prophage","uniprotId":"YFDS_ECOLI"},"6883055":{"id":6883055,"externalId":"511145.b2363","name":"yfdT","hasFamily":false,"annotation":"CPS-53 (KpLE1) prophage","uniprotId":"YFDT_ECOLI"},"6883056":{"id":6883056,"externalId":"511145.b2364","name":"dsdC","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; Regulates the expression of the dsd operon for D-serine deaminase (dsdA)","uniprotId":"DSDC_ECOLI"},"6883057":{"id":6883057,"externalId":"511145.b2365","name":"dsdX","hasFamily":true,"annotation":"D-serine permease; Probable permease for an unknown substrate","uniprotId":"DSDX_ECOLI"},"6883058":{"id":6883058,"externalId":"511145.b2366","name":"dsdA","hasFamily":true,"annotation":"D-serine dehydratase","uniprotId":"SDHD_ECOLI"},"6883059":{"id":6883059,"externalId":"511145.b2367","name":"emrX","hasFamily":true,"annotation":"putative multidrug efflux system","uniprotId":"EMRY_ECOLI"},"6883060":{"id":6883060,"externalId":"511145.b2368","name":"emrK","hasFamily":true,"annotation":"EmrKY-TolC multidrug resistance efflux pump, membrane fusion protein component","uniprotId":"EMRK_ECOLI"},"6883061":{"id":6883061,"externalId":"511145.b2369","name":"evgA","hasFamily":false,"annotation":"DNA-binding response regulator in two-component regulatory system with EvgS; Member of the two-component regulatory system EvgS/EvgA. Regulates the expression of emrKY operon and yfdX. Seems also to control expression of at least one other multidrug efflux operon","uniprotId":"EVGA_ECOLI"},"6883062":{"id":6883062,"externalId":"511145.b2370","name":"evgS","hasFamily":true,"annotation":"hybrid sensory histidine kinase in two-component regulatory system with EvgA; Member of the two-component regulatory system EvgS/EvgA. Phosphorylates EvgA via a four-step phosphorelay in response to environmental signals","uniprotId":"EVGS_ECOLI"},"6883063":{"id":6883063,"externalId":"511145.b2371","name":"yfdE","hasFamily":true,"annotation":"putative CoA-transferase, NAD(P)-binding","uniprotId":"YFDE_ECOLI"},"6883064":{"id":6883064,"externalId":"511145.b2372","name":"yfdV","hasFamily":true,"annotation":"putative transporter","uniprotId":"YFDV_ECOLI"},"6883065":{"id":6883065,"externalId":"511145.b2373","name":"oxc","hasFamily":true,"annotation":"oxalyl CoA decarboxylase, ThDP-dependent","uniprotId":"OXC_ECOLI"},"6883066":{"id":6883066,"externalId":"511145.b2374","name":"frc","hasFamily":true,"annotation":"formyl-CoA transferase, NAD(P)-binding; Catalyzes the transfer of the CoA moiety from formyl-CoA to oxalate (By similarity)","uniprotId":"FCTA_ECOLI"},"6883067":{"id":6883067,"externalId":"511145.b2375","name":"yfdX","hasFamily":true,"annotation":"predicted protein","uniprotId":"YFDX_ECOLI"},"6883068":{"id":6883068,"externalId":"511145.b2376","name":"ypdI","hasFamily":false,"annotation":"putative lipoprotein involved in colanic acid biosynthesis","uniprotId":"YPDI_ECOLI"},"6883069":{"id":6883069,"externalId":"511145.b2377","name":"yfdY","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YFDY_ECOLI"},"6883070":{"id":6883070,"externalId":"511145.b2378","name":"lpxP","hasFamily":true,"annotation":"palmitoleoyl-acyl carrier protein (ACP)-dependent acyltransferase","uniprotId":"DDG_ECOLI"},"6883071":{"id":6883071,"externalId":"511145.b2379","name":"alaC","hasFamily":true,"annotation":"valine-pyruvate aminotransferase 3; Involved in the biosynthesis of alanine","uniprotId":"ALAC_ECOLI"},"6883072":{"id":6883072,"externalId":"511145.b2380","name":"ypdA","hasFamily":true,"annotation":"putative sensory kinase in two-component system with YpdB; Member of the two-component regulatory system YpdA/YpdB involved in carbon control network. Activates YpdB by phosphorylation in response to extracellular pyruvate","uniprotId":"YPDA_ECOLI"},"6883073":{"id":6883073,"externalId":"511145.b2381","name":"ypdB","hasFamily":true,"annotation":"putative response regulator in two-component system with YpdA; Member of the two-component regulatory system YpdA/YpdB involved in carbon control network. Regulates expression of yhjX by binding to its promoter region","uniprotId":"YPDB_ECOLI"},"6883074":{"id":6883074,"externalId":"511145.b2382","name":"ypdC","hasFamily":true,"annotation":"putative DNA-binding protein","uniprotId":"YPDC_ECOLI"},"6883075":{"id":6883075,"externalId":"511145.b2383","name":"fryA","hasFamily":false,"annotation":"fused predicted PTS enzymes: Hpr component/enzyme I component/enzyme IIA component; Multifunctional protein that includes general (non sugar-specific) and sugar-specific components of the phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS). This major carbohydrate active-transport system catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. Enzyme I transfers the phosphoryl group from phosphoenolpyruvate (PEP) to the phosphoryl carrier protein (HPr). HPr transfers the phosphoryl group to the pho [...] ","uniprotId":"PTFX1_ECOLI"},"6883076":{"id":6883076,"externalId":"511145.b2384","name":"ypdE","hasFamily":false,"annotation":"aminopeptidase; Has a broad aminopeptidase activity on non-blocked peptides by progressively cleaving amino acids off the peptide substrate. Aminopeptidase activity stops at the residue before the first proline in the peptide. Can not cleave when proline is the first N-terminal residue","uniprotId":"YPDE_ECOLI"},"6883077":{"id":6883077,"externalId":"511145.b2385","name":"ypdF","hasFamily":true,"annotation":"Xaa-Pro aminopeptidase; Hydrolyzes the N-terminal methionine when the next amino acid is alanine, proline or serine. The substrate preference for methionyl aminopeptidase activity is Pro > Ala > Ser. Also able to hydrolyze the Xaa-Pro peptide bond when the first amino acid is alanine, asparagine or methionine","uniprotId":"YPDF_ECOLI"},"6883078":{"id":6883078,"externalId":"511145.b2386","name":"fryC","hasFamily":false,"annotation":"putative enzyme IIC component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane (By similarity)","uniprotId":"PTFC1_ECOLI"},"6883079":{"id":6883079,"externalId":"511145.b2387","name":"fryB","hasFamily":false,"annotation":"putative enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (By similarity)","uniprotId":"PTFB1_ECOLI"},"6883080":{"id":6883080,"externalId":"511145.b2388","name":"glk","hasFamily":true,"annotation":"glucokinase; Not highly important in E.coli as glucose is transported into the cell by the PTS system already as glucose 6-phosphate","uniprotId":"GLK_ECOLI"},"6883081":{"id":6883081,"externalId":"511145.b2389","name":"yfeO","hasFamily":true,"annotation":"putative ion channel protein","uniprotId":"YFEO_ECOLI"},"6883082":{"id":6883082,"externalId":"511145.b2390","name":"ypeC","hasFamily":true,"annotation":"conserved protein","uniprotId":"YPEC_ECOLI"},"6883083":{"id":6883083,"externalId":"511145.b2392","name":"mntH","hasFamily":true,"annotation":"manganese/divalent cation transporter; H(+)-stimulated, divalent metal cation uptake system. Involved in manganese and iron uptake. Can also transport cadmium, cobalt, zinc and to a lesser extent nickel and copper. Involved in response to reactive oxygen","uniprotId":"MNTH_ECOLI"},"6883084":{"id":6883084,"externalId":"511145.b2393","name":"nupC","hasFamily":true,"annotation":"nucleoside (except guanosine) transporter; Transports nucleosides with a high affinity except guanosine and deoxyguanosine. Driven by a proton motive force","uniprotId":"NUPC_ECOLI"},"6883085":{"id":6883085,"externalId":"511145.b2394","name":"insL1","hasFamily":false,"annotation":"IS186 transposase; Involved in the transposition of the insertion sequence IS186","uniprotId":"INSL1_ECOLI"},"6883086":{"id":6883086,"externalId":"511145.b2395","name":"yfeA","hasFamily":true,"annotation":"putative diguanylate cyclase","uniprotId":"YFEA_ECOLI"},"6883087":{"id":6883087,"externalId":"511145.b2398","name":"yfeC","hasFamily":true,"annotation":"putative DNA-binding protein, DUF1323 family","uniprotId":"YFEC_ECOLI"},"6883088":{"id":6883088,"externalId":"511145.b2399","name":"yfeD","hasFamily":true,"annotation":"conserved protein, DUF1323 family","uniprotId":"YFED_ECOLI"},"6883089":{"id":6883089,"externalId":"511145.b2400","name":"gltX","hasFamily":true,"annotation":"glutamyl-tRNA synthetase; Catalyzes the attachment of glutamate to tRNA(Glu) in a two-step reaction: glutamate is first activated by ATP to form Glu-AMP and then transferred to the acceptor end of tRNA(Glu) (By similarity)","uniprotId":"SYE_ECOLI"},"6883090":{"id":6883090,"externalId":"511145.b2405","name":"xapR","hasFamily":true,"annotation":"DNA-binding transcriptional activator for for xapAB; Positive regulator required for the expression of xapA and xapB. Binds to the inducer xanthosine","uniprotId":"XAPR_ECOLI"},"6883091":{"id":6883091,"externalId":"511145.b2406","name":"xapB","hasFamily":true,"annotation":"xanthosine transporter; Nucleoside transport","uniprotId":"XAPB_ECOLI"},"6883092":{"id":6883092,"externalId":"511145.b2407","name":"xapA","hasFamily":true,"annotation":"purine nucleoside phosphorylase II; The purine nucleoside phosphorylases catalyze the phosphorolytic breakdown of the N-glycosidic bond in the beta- (deoxy)ribonucleoside molecules, with the formation of the corresponding free purine bases and pentose-1-phosphate. This protein can degrade all purine nucleosides including xanthosine, inosine and guanosine, but cannot cleave adenosine, deoxyadenosine or hypoxanthine arabinoside. Has a preference for the neutral over the monoanionic form of xanthosine","uniprotId":"XAPA_ECOLI"},"6883093":{"id":6883093,"externalId":"511145.b2408","name":"yfeN","hasFamily":true,"annotation":"putative outer membrane protein","uniprotId":"YFEN_ECOLI"},"6883094":{"id":6883094,"externalId":"511145.b2409","name":"yfeR","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YFER_ECOLI"},"6883095":{"id":6883095,"externalId":"511145.b2410","name":"yfeH","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YFEH_ECOLI"},"6883096":{"id":6883096,"externalId":"511145.b2411","name":"lig","hasFamily":true,"annotation":"DNA ligase, NAD(+)-dependent; DNA ligase that catalyzes the formation of phosphodiester linkages between 5'-phosphoryl and 3'-hydroxyl groups in double-stranded DNA using NAD as a coenzyme and as the energy source for the reaction. It is essential for DNA replication and repair of damaged DNA (By similarity)","uniprotId":"DNLJ_ECOLI"},"6883097":{"id":6883097,"externalId":"511145.b2412","name":"zipA","hasFamily":true,"annotation":"FtsZ stabilizer; Interacts directly with the cell division protein FtsZ. Probable receptor for the septal ring structure, may anchor it to the inner-membrane (By similarity)","uniprotId":"ZIPA_ECOLI"},"6883098":{"id":6883098,"externalId":"511145.b2413","name":"cysZ","hasFamily":true,"annotation":"putative inner membrane protein; Possibly involved in sulfate transport (By similarity)","uniprotId":"CYSZ_ECOLI"},"6883099":{"id":6883099,"externalId":"511145.b2414","name":"cysK","hasFamily":true,"annotation":"cysteine synthase A, O-acetylserine sulfhydrolase A subunit","uniprotId":"CYSK_ECOLI"},"6883100":{"id":6883100,"externalId":"511145.b2415","name":"ptsH","hasFamily":false,"annotation":"phosphohistidinoprotein-hexose phosphotransferase component of PTS system (Hpr); General (non sugar-specific) component of the phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS). This major carbohydrate active-transport system catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. The phosphoryl group from phosphoenolpyruvate (PEP) is transferred to the phosphoryl carrier protein HPr by enzyme I. Phospho-HPr then transfers it to the permease (enzymes II/III)","uniprotId":"PTHP_ECOLI"},"6883101":{"id":6883101,"externalId":"511145.b2416","name":"ptsI","hasFamily":true,"annotation":"PEP-protein phosphotransferase of PTS system (enzyme I); General (non sugar-specific) component of the phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS). This major carbohydrate active-transport system catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. Enzyme I transfers the phosphoryl group from phosphoenolpyruvate (PEP) to the phosphoryl carrier protein (HPr) (By similarity)","uniprotId":"PT1_ECOLI"},"6883102":{"id":6883102,"externalId":"511145.b2417","name":"crr","hasFamily":true,"annotation":"glucose-specific enzyme IIA component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in glucose transport","uniprotId":"PTGA_ECOLI"},"6883103":{"id":6883103,"externalId":"511145.b2418","name":"pdxK","hasFamily":true,"annotation":"pyridoxal-pyridoxamine kinase/hydroxymethylpyrimidine kinase; Phosphorylates B6 vitamers; functions in a salvage pathway. Uses pyridoxal, pyridoxine, and pyridoxamine as substrates (By similarity)","uniprotId":"PDXK_ECOLI"},"6883104":{"id":6883104,"externalId":"511145.b2419","name":"yfeK","hasFamily":true,"annotation":"predicted protein","uniprotId":"YFEK_ECOLI"},"6883105":{"id":6883105,"externalId":"511145.b2420","name":"yfeS","hasFamily":true,"annotation":"conserved protein","uniprotId":"YFES_ECOLI"},"6883106":{"id":6883106,"externalId":"511145.b2421","name":"cysM","hasFamily":true,"annotation":"cysteine synthase B (O-acetylserine sulfhydrolase B); Two cysteine synthase enzymes are found. Both catalyze the same reaction. Cysteine synthase B can also use thiosulfate in place of sulfide to give cysteine thiosulfonate as a product","uniprotId":"CYSM_ECOLI"},"6883107":{"id":6883107,"externalId":"511145.b2422","name":"cysA","hasFamily":true,"annotation":"sulfate/thiosulfate transporter subunit; Part of the ABC transporter complex CysAWTP involved in sulfate/thiosulfate import. Responsible for energy coupling to the transport system","uniprotId":"CYSA_ECOLI"},"6883108":{"id":6883108,"externalId":"511145.b2423","name":"cysW","hasFamily":true,"annotation":"sulfate/thiosulfate ABC transporter subunit; Part of the ABC transporter complex CysAWTP (TC 3.A.1.6.1) involved in sulfate/thiosulfate import. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"CYSW_ECOLI"},"6883109":{"id":6883109,"externalId":"511145.b2424","name":"cysT","hasFamily":true,"annotation":"sulfate/thiosulfate ABC transporter permease; Part of the ABC transporter complex CysAWTP (TC 3.A.1.6.1) involved in sulfate/thiosulfate import. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"CYST_ECOLI"},"6883110":{"id":6883110,"externalId":"511145.b2425","name":"cysP","hasFamily":true,"annotation":"thiosulfate-binding protein; Part of the ABC transporter complex CysAWTP (TC 3.A.1.6.1) involved in sulfate/thiosulfate import. This protein specifically binds thiosulfate and is involved in its transmembrane transport","uniprotId":"CYSP_ECOLI"},"6883111":{"id":6883111,"externalId":"511145.b2426","name":"ucpA","hasFamily":true,"annotation":"predicted short-chain oxidoreductase","uniprotId":"UCPA_ECOLI"},"6883112":{"id":6883112,"externalId":"511145.b2427","name":"murR","hasFamily":true,"annotation":"Repressor for murPQ, MurNAc 6-P inducible; Represses the expression of the murPQ operon involved in the uptake and degradation of N-acetylmuramic acid (MurNAc). Binds to two adjacent inverted repeats within the operator region. MurNAc 6-phosphate, the substrate of MurQ, is the specific inducer that weakens binding of MurR to the operator. Also represses its own transcription","uniprotId":"MURR_ECOLI"},"6883113":{"id":6883113,"externalId":"511145.b2428","name":"murQ","hasFamily":true,"annotation":"N-acetylmuramic acid 6-phosphate (MurNAc-6-P) etherase; Specifically catalyzes the cleavage of the D-lactyl ether substituent of MurNAc 6-phosphate, producing GlcNAc 6- phosphate and D-lactate. Is required for growth on MurNAc as the sole source of carbon and energy. Together with AnmK, is also required for the utilization of anhydro-N-acetylmuramic acid (anhMurNAc) either imported from the medium or derived from its own cell wall murein, and thus plays a role in cell wall recycling","uniprotId":"MURQ_ECOLI"},"6883114":{"id":6883114,"externalId":"511145.b2429","name":"murP","hasFamily":true,"annotation":"N-acetylmuramic acid permease, EIIBC component, PTS system; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in N-acetylmuramic acid (MurNAc) transport, yielding cytoplasmic MurNAc-6-P. Is responsible for growth on MurNAc as the sole source of carbon and energy. Is also able to take up anhydro-N-acetylmuramic acid (anhMurNAc), but cannot phosphorylate the carbon 6, [...] ","uniprotId":"PTYBC_ECOLI"},"6883115":{"id":6883115,"externalId":"511145.b2430","name":"yfeW","hasFamily":true,"annotation":"penicillin binding protein PBP4B","uniprotId":"YFEW_ECOLI"},"6883116":{"id":6883116,"externalId":"511145.b2431","name":"yfeX","hasFamily":true,"annotation":"deferrrochelatase, cytoplasmic; Involved in the recovery of exogenous heme iron. Extracts iron from heme while preserving the tetrapyrrol ring intact","uniprotId":"YFEX_ECOLI"},"6883117":{"id":6883117,"externalId":"511145.b2432","name":"yfeY","hasFamily":true,"annotation":"RpoE-regulated lipoprotein","uniprotId":"YFEY_ECOLI"},"6883118":{"id":6883118,"externalId":"511145.b2433","name":"yfeZ","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YFEZ_ECOLI"},"6883119":{"id":6883119,"externalId":"511145.b2434","name":"ypeA","hasFamily":true,"annotation":"putative acyltransferase with acyl-CoA N-acyltransferase domain","uniprotId":"YPEA_ECOLI"},"6883120":{"id":6883120,"externalId":"511145.b2435","name":"amiA","hasFamily":true,"annotation":"N-acetylmuramoyl-l-alanine amidase I; Cell-wall hydrolase involved in septum cleavage during cell division. Can also act as powerful autolysin in the presence of murein synthesis inhibitors","uniprotId":"AMIA_ECOLI"},"6883121":{"id":6883121,"externalId":"511145.b2436","name":"hemF","hasFamily":true,"annotation":"coproporphyrinogen III oxidase; Key enzyme in heme biosynthesis. Catalyzes the oxidative decarboxylation of propionic acid side chains of rings A and B of coproporphyrinogen III (By similarity)","uniprotId":"HEM6_ECOLI"},"6883122":{"id":6883122,"externalId":"511145.b2437","name":"eutR","hasFamily":true,"annotation":"eut operon transcriptional activator, AraC family; Activates the transcription of the eut operon. Also positively regulates its own transcription. Probably binds ethanolamine and vitamin B12 as effectors (By similarity)","uniprotId":"EUTR_ECOLI"},"6883123":{"id":6883123,"externalId":"511145.b2438","name":"eutK","hasFamily":true,"annotation":"putative carboxysome structural protein with predicted role in ethanolamine utilization; May be involved in the formation of a specific microcompartment in the cell in which the metabolism of potentially toxic by-products takes place","uniprotId":"EUTK_ECOLI"},"6883124":{"id":6883124,"externalId":"511145.b2439","name":"eutL","hasFamily":true,"annotation":"putative carboxysome structural protein with predicted role in ethanolamine utilization; May be involved in the formation of a specific microcompartment in the cell in which the metabolism of potentially toxic by-products takes place","uniprotId":"EUTL_ECOLI"},"6883125":{"id":6883125,"externalId":"511145.b2440","name":"eutC","hasFamily":true,"annotation":"ethanolamine ammonia-lyase, small subunit (light chain)","uniprotId":"EUTC_ECOLI"},"6883126":{"id":6883126,"externalId":"511145.b2441","name":"eutB","hasFamily":true,"annotation":"ethanolamine ammonia-lyase, large subunit, heavy chain","uniprotId":"EUTB_ECOLI"},"6883127":{"id":6883127,"externalId":"511145.b2442","name":"intZ","hasFamily":true,"annotation":"CPZ-55 prophage; Integrase is necessary for integration of the phage into the host genome by site-specific recombination. In conjunction with excisionase, integrase is also necessary for excision of the prophage from the host genome","uniprotId":"INTZ_ECOLI"},"6883128":{"id":6883128,"externalId":"511145.b2443","name":"yffL","hasFamily":false,"annotation":"CPZ-55 prophage","uniprotId":"YFFL_ECOLI"},"6883129":{"id":6883129,"externalId":"511145.b2444","name":"yffM","hasFamily":false,"annotation":"CPZ-55 prophage","uniprotId":"YFFM_ECOLI"},"6883130":{"id":6883130,"externalId":"511145.b2445","name":"yffN","hasFamily":false,"annotation":"CPZ-55 prophage","uniprotId":"YFFN_ECOLI"},"6883131":{"id":6883131,"externalId":"511145.b2446","name":"yffO","hasFamily":false,"annotation":"CPZ-55 prophage","uniprotId":"YFFO_ECOLI"},"6883132":{"id":6883132,"externalId":"511145.b2447","name":"yffP","hasFamily":false,"annotation":"CPZ-55 prophage","uniprotId":"YFFP_ECOLI"},"6883133":{"id":6883133,"externalId":"511145.b2448","name":"yffQ","hasFamily":false,"annotation":"CPZ-55 prophage","uniprotId":"YFFQ_ECOLI"},"6883134":{"id":6883134,"externalId":"511145.b2449","name":"yffR","hasFamily":false,"annotation":"CPZ-55 prophage","uniprotId":"YFFR_ECOLI"},"6883135":{"id":6883135,"externalId":"511145.b2450","name":"yffS","hasFamily":false,"annotation":"CPZ-55 prophage","uniprotId":"YFFS_ECOLI"},"6883136":{"id":6883136,"externalId":"511145.b2451","name":"eutA","hasFamily":true,"annotation":"reactivating factor for ethanolamine ammonia lyase; May protect the EutBC lyase from inhibition (By similarity)","uniprotId":"EUTA_ECOLI"},"6883137":{"id":6883137,"externalId":"511145.b2452","name":"eutH","hasFamily":true,"annotation":"ethanolamine transporter; Possibly involved in the transport of ethanolamine from the periplasm to the cytoplasm","uniprotId":"EUTH_ECOLI"},"6883138":{"id":6883138,"externalId":"511145.b2453","name":"eutG","hasFamily":true,"annotation":"ethanol dehydrogenase involved in ethanolamine utilization; aldehyde reductase, converts acetaldehyde to ethanol; May act on the acetaldehyde produced from the degradation of ethanolamine","uniprotId":"EUTG_ECOLI"},"6883139":{"id":6883139,"externalId":"511145.b2454","name":"eutJ","hasFamily":true,"annotation":"putative chaperonin, ethanolamine utilization protein","uniprotId":"EUTJ_ECOLI"},"6883140":{"id":6883140,"externalId":"511145.b2455","name":"eutE","hasFamily":true,"annotation":"aldehyde oxidoreductase, ethanolamine utilization protein; May act as an acetaldehyde dehydrogenase that converts acetaldehyde into acetyl-CoA","uniprotId":"EUTE_ECOLI"},"6883141":{"id":6883141,"externalId":"511145.b2456","name":"eutN","hasFamily":true,"annotation":"Ethanolamine catabolic microcompartment shell protein; May be involved in the formation of a specific microcompartment in the cell in which the metabolism of potentially toxic by-products takes place","uniprotId":"EUTN_ECOLI"},"6883142":{"id":6883142,"externalId":"511145.b2457","name":"cchA","hasFamily":true,"annotation":"putative carboxysome structural protein, ethanolamine utilization protein; May be involved in the formation of a specific microcompartment in the cell in which the metabolism of potentially toxic by-products takes place","uniprotId":"EUTM_ECOLI"},"6883143":{"id":6883143,"externalId":"511145.b2458","name":"eutD","hasFamily":true,"annotation":"phosphate acetyltransferase","uniprotId":"EUTD_ECOLI"},"6883144":{"id":6883144,"externalId":"511145.b2459","name":"eutT","hasFamily":true,"annotation":"cobalamin adenosyltransferase involved in ethanolamine utilization; Converts CNB12 to ADOB12 (By similarity)","uniprotId":"EUTT_ECOLI"},"6883145":{"id":6883145,"externalId":"511145.b2460","name":"eutQ","hasFamily":true,"annotation":"conserved protein","uniprotId":"EUTQ_ECOLI"},"6883146":{"id":6883146,"externalId":"511145.b2461","name":"eutP","hasFamily":true,"annotation":"conserved protein with nucleoside triphosphate hydrolase domain","uniprotId":"EUTP_ECOLI"},"6883147":{"id":6883147,"externalId":"511145.b2462","name":"eutS","hasFamily":true,"annotation":"putative carboxysome structural protein with predicted role in ethanol utilization; May be involved in the formation of a specific microcompartment in the cell in which the metabolism of potentially toxic by-products takes place","uniprotId":"EUTS_ECOLI"},"6883148":{"id":6883148,"externalId":"511145.b2463","name":"maeB","hasFamily":true,"annotation":"fused malic enzyme predicted oxidoreductase/predicted phosphotransacetylase","uniprotId":"MAO2_ECOLI"},"6883149":{"id":6883149,"externalId":"511145.b2464","name":"talA","hasFamily":true,"annotation":"transaldolase A; Transaldolase is important for the balance of metabolites in the pentose-phosphate pathway (By similarity)","uniprotId":"TALA_ECOLI"},"6883150":{"id":6883150,"externalId":"511145.b2465","name":"tktB","hasFamily":true,"annotation":"transketolase 2, thiamin-binding; Catalyzes the reversible transfer of a two-carbon ketol group from sedoheptulose-7-phosphate to glyceraldehyde-3- phosphate, producing xylulose-5-phosphate and ribose-5-phosphate. Catalyzes the transfer of a two-carbon ketol group from a ketose donor to an aldose acceptor, via a covalent intermediate with the cofactor thiamine pyrophosphate (By similarity)","uniprotId":"TKT2_ECOLI"},"6883151":{"id":6883151,"externalId":"511145.b2466","name":"ypfG","hasFamily":true,"annotation":"predicted protein","uniprotId":"YPFG_ECOLI"},"6883152":{"id":6883152,"externalId":"511145.b2467","name":"nudK","hasFamily":true,"annotation":"GDP-mannose pyrophosphatase; Catalyzes the hydrolysis of GDP-mannose. Can also use other substrates, catalyzing the hydrolysis of the pyrophosphate bond, releasing a nucleoside monophosphate and a phosphorylated moiety, depending on the substrate","uniprotId":"NUDK_ECOLI"},"6883153":{"id":6883153,"externalId":"511145.b2468","name":"aegA","hasFamily":true,"annotation":"fused predicted oxidoreductase: FeS binding subunit/NAD/FAD-binding subunit","uniprotId":"AEGA_ECOLI"},"6883154":{"id":6883154,"externalId":"511145.b2469","name":"narQ","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with NarP (NarL); Acts as a sensor for nitrate/nitrite and transduces signal of nitrate/nitrite availability to the NarL/NarP proteins. NarQ probably activates NarL and NarP by phosphorylation. NarQ probably negatively regulates the NarL protein by dephosphorylation","uniprotId":"NARQ_ECOLI"},"6883155":{"id":6883155,"externalId":"511145.b2470","name":"acrD","hasFamily":true,"annotation":"aminoglycoside/multidrug efflux system; Participates in the efflux of aminoglycosides. Confers resistance to a variety of these substances","uniprotId":"ACRD_ECOLI"},"6883156":{"id":6883156,"externalId":"511145.b2471","name":"yffB","hasFamily":true,"annotation":"putative reductase, function unknown, ArsC family","uniprotId":"YFFB_ECOLI"},"6883157":{"id":6883157,"externalId":"511145.b2472","name":"dapE","hasFamily":true,"annotation":"N-succinyl-diaminopimelate deacylase; Catalyzes the hydrolysis of N-succinyl-L,L- diaminopimelic acid (SDAP), forming succinate and LL-2,6- diaminoheptanedioate (DAP), an intermediate involved in the bacterial biosynthesis of lysine and meso-diaminopimelic acid, an essential component of bacterial cell walls (By similarity)","uniprotId":"DAPE_ECOLI"},"6883158":{"id":6883158,"externalId":"511145.b2473","name":"ypfH","hasFamily":false,"annotation":"putative hydrolase; Displays esterase activity toward palmitoyl-CoA and pNP- butyrate","uniprotId":"YPFH_ECOLI"},"6883159":{"id":6883159,"externalId":"511145.b2474","name":"tmcA","hasFamily":true,"annotation":"elongator methionine tRNA (ac4C34) acetyltransferase; Catalyzes the formation of N(4)-acetylcytidine (ac(4)C) at the wobble position of tRNA(Met), by using acetyl-CoA as an acetyl donor and ATP (or GTP). It recognizes the wobble base of tRNA(Met), thus distinguishing between tRNA(Met) and the structurally similar tRNA(Ile2)","uniprotId":"TMCA_ECOLI"},"6883160":{"id":6883160,"externalId":"511145.b2475","name":"ypfJ","hasFamily":true,"annotation":"conserved protein","uniprotId":"YPFJ_ECOLI"},"6883161":{"id":6883161,"externalId":"511145.b2476","name":"purC","hasFamily":true,"annotation":"phosphoribosylaminoimidazole-succinocarboxamide synthetase","uniprotId":"PUR7_ECOLI"},"6883162":{"id":6883162,"externalId":"511145.b2477","name":"bamC","hasFamily":true,"annotation":"lipoprotein required for OM biogenesis, in BamABCDE complex; Part of the outer membrane protein assembly complex, which is involved in assembly and insertion of beta-barrel proteins into the outer membrane. Nonessential member of the complex that stabilizes the interaction between the essential proteins BamA and BamD","uniprotId":"BAMC_ECOLI"},"6883163":{"id":6883163,"externalId":"511145.b2478","name":"dapA","hasFamily":true,"annotation":"dihydrodipicolinate synthase; Catalyzes the condensation of (S)-aspartate-beta- semialdehyde [(S)-ASA] and pyruvate to 4-hydroxy- tetrahydrodipicolinate (HTPA) (By similarity)","uniprotId":"DAPA_ECOLI"},"6883164":{"id":6883164,"externalId":"511145.b2479","name":"gcvR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor, regulatory protein accessory to GcvA; Negative transcriptional regulator of the glycine cleavage system operon (GCV). Does not autoregulate its own expression. It is not yet known how GcvR acts as a repressor. It does not seem to bind DNA. It could interact with GcvA and suppress its activatory activity","uniprotId":"GCVR_ECOLI"},"6883165":{"id":6883165,"externalId":"511145.b2480","name":"bcp","hasFamily":true,"annotation":"thiol peroxidase, thioredoxin-dependent","uniprotId":"BCP_ECOLI"},"6883166":{"id":6883166,"externalId":"511145.b2481","name":"hyfA","hasFamily":false,"annotation":"hydrogenase 4, 4Fe-4S subunit; Probable electron transfer protein for hydrogenase 4","uniprotId":"HYFA_ECOLI"},"6883167":{"id":6883167,"externalId":"511145.b2482","name":"hyfB","hasFamily":true,"annotation":"hydrogenase 4, membrane subunit","uniprotId":"HYFB_ECOLI"},"6883168":{"id":6883168,"externalId":"511145.b2483","name":"hyfC","hasFamily":true,"annotation":"hydrogenase 4, membrane subunit","uniprotId":"HYFC_ECOLI"},"6883169":{"id":6883169,"externalId":"511145.b2484","name":"hyfD","hasFamily":true,"annotation":"hydrogenase 4, membrane subunit","uniprotId":"HYFD_ECOLI"},"6883170":{"id":6883170,"externalId":"511145.b2485","name":"hyfE","hasFamily":false,"annotation":"hydrogenase 4, membrane subunit","uniprotId":"HYFE_ECOLI"},"6883171":{"id":6883171,"externalId":"511145.b2486","name":"hyfF","hasFamily":true,"annotation":"hydrogenase 4, membrane subunit","uniprotId":"HYFF_ECOLI"},"6883172":{"id":6883172,"externalId":"511145.b2487","name":"hyfG","hasFamily":false,"annotation":"hydrogenase 4, subunit","uniprotId":"HYFG_ECOLI"},"6883173":{"id":6883173,"externalId":"511145.b2488","name":"hyfH","hasFamily":false,"annotation":"hydrogenase 4, Fe-S subunit; Probable electron transfer protein for hydrogenase 4","uniprotId":"HYFH_ECOLI"},"6883174":{"id":6883174,"externalId":"511145.b2489","name":"hyfI","hasFamily":false,"annotation":"hydrogenase 4, Fe-S subunit","uniprotId":"HYFI_ECOLI"},"6883175":{"id":6883175,"externalId":"511145.b2490","name":"hyfJ","hasFamily":false,"annotation":"putative processing element hydrogenase 4","uniprotId":"HYFJ_ECOLI"},"6883176":{"id":6883176,"externalId":"511145.b2491","name":"hyfR","hasFamily":true,"annotation":"DNA-binding transcriptional activator, formate sensing; Required for induction of expression of the hydrogenase- 4 structural genes","uniprotId":"HYFR_ECOLI"},"6883177":{"id":6883177,"externalId":"511145.b2492","name":"focB","hasFamily":true,"annotation":"putative formate transporter; Involved in the bidirectional transport of formate (By similarity)","uniprotId":"FOCB_ECOLI"},"6883178":{"id":6883178,"externalId":"511145.b2493","name":"perM","hasFamily":true,"annotation":"putative inner membrane permease, UPF0118 family","uniprotId":"PERM_ECOLI"},"6883179":{"id":6883179,"externalId":"511145.b2494","name":"yfgC","hasFamily":true,"annotation":"predicted peptidase, has TPR repeats; May facilitate membrane attachment of LoiP under unfavorable conditions","uniprotId":"YFGC_ECOLI"},"6883180":{"id":6883180,"externalId":"511145.b2495","name":"yfgD","hasFamily":true,"annotation":"putative oxidoreductase","uniprotId":"YFGD_ECOLI"},"6883181":{"id":6883181,"externalId":"511145.b2496","name":"hda","hasFamily":false,"annotation":"ATPase regulatory factor involved in DnaA inactivation; Mediates the interactions of DNA replication inititator protein DnaA with DNA polymerase subunit beta sliding clamp (dnaN). Stimulates hydrolysis of ATP-DnaA to ADP-DnaA, rendering DnaA inactive for reinititation, a process called regulatory inhibition of DnaA or RIDA. ADP-binding activates Hda to hydrolyze DnaA-ATP; Hda monomers bind to ADP with about 200-fold greater affinity than for ATP. RIDA function can be genetically separated from viability, suggesting this protein has another function as well","uniprotId":"HDA_ECOLI"},"6883182":{"id":6883182,"externalId":"511145.b2497","name":"uraA","hasFamily":true,"annotation":"uracil permease; Transport of uracil in the cell","uniprotId":"URAA_ECOLI"},"6883183":{"id":6883183,"externalId":"511145.b2498","name":"upp","hasFamily":true,"annotation":"uracil phosphoribosyltransferase; Catalyzes the conversion of uracil and 5-phospho-alpha- D-ribose 1-diphosphate (PRPP) to UMP and diphosphate (By similarity)","uniprotId":"UPP_ECOLI"},"6883184":{"id":6883184,"externalId":"511145.b2499","name":"purM","hasFamily":true,"annotation":"phosphoribosylaminoimidazole synthetase","uniprotId":"PUR5_ECOLI"},"6883185":{"id":6883185,"externalId":"511145.b2500","name":"purN","hasFamily":true,"annotation":"phosphoribosylglycinamide formyltransferase 1","uniprotId":"PUR3_ECOLI"},"6883186":{"id":6883186,"externalId":"511145.b2501","name":"ppk","hasFamily":true,"annotation":"polyphosphate kinase, component of RNA degradosome; Catalyzes the reversible transfer of the terminal phosphate of ATP to form a long-chain polyphosphate (polyP). Can form linear polymers of orthophosphate with chain lengths up to 1000 or more. Can also act in the reverse direction to form ATP in the presence of excess ADP. Can also use GTP instead of ATP; but the efficiency of GTP is 5% that of ATP","uniprotId":"PPK_ECOLI"},"6883187":{"id":6883187,"externalId":"511145.b2502","name":"ppx","hasFamily":true,"annotation":"exopolyphosphatase; Degradation of inorganic polyphosphates. Orthophosphate is released progressively from the ends of polyphosphate of circa 500 residues long, while chains of circa 15 residues compete poorly with polyphosphate as substrate","uniprotId":"PPX_ECOLI"},"6883188":{"id":6883188,"externalId":"511145.b2503","name":"yfgF","hasFamily":true,"annotation":"cyclic-di-GMP phosphodiesterase, anaerobic; Truncated proteins consisting of the GGDEF/EAL domains (residues 319-747) or of the EAL domain alone (481-747) have c-di- GMP phosphodiesterase activity. They do not have diguanylate cyclase activity. Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"YFGF_ECOLI"},"6883189":{"id":6883189,"externalId":"511145.b2504","name":"yfgG","hasFamily":true,"annotation":"predicted protein","uniprotId":"YFGG_ECOLI"},"6883190":{"id":6883190,"externalId":"511145.b2505","name":"yfgH","hasFamily":false,"annotation":"outer membrane lipoprotein","uniprotId":"YFGH_ECOLI"},"6883191":{"id":6883191,"externalId":"511145.b2506","name":"yfgI","hasFamily":false,"annotation":"conserved protein","uniprotId":"YFGI_ECOLI"},"6883192":{"id":6883192,"externalId":"511145.b2507","name":"guaA","hasFamily":true,"annotation":"GMP synthetase (glutamine aminotransferase); Catalyzes the synthesis of GMP from XMP (By similarity)","uniprotId":"GUAA_ECOLI"},"6883193":{"id":6883193,"externalId":"511145.b2508","name":"guaB","hasFamily":true,"annotation":"IMP dehydrogenase; Catalyzes the conversion of inosine 5'-phosphate (IMP) to xanthosine 5'-phosphate (XMP), the first committed and rate- limiting step in the de novo synthesis of guanine nucleotides, and therefore plays an important role in the regulation of cell growth (By similarity)","uniprotId":"IMDH_ECOLI"},"6883194":{"id":6883194,"externalId":"511145.b2509","name":"xseA","hasFamily":true,"annotation":"exonuclease VII, large subunit; Bidirectionally degrades single-stranded DNA into large acid-insoluble oligonucleotides, which are then degraded further into small acid-soluble oligonucleotides. It can also degrade 3' or 5' ss regions extending from the termini of duplex DNA molecules and displaced ss regions","uniprotId":"EX7L_ECOLI"},"6883195":{"id":6883195,"externalId":"511145.b2510","name":"yfgJ","hasFamily":true,"annotation":"conserved protein, DUF1407 family","uniprotId":"YFGJ_ECOLI"},"6883196":{"id":6883196,"externalId":"511145.b2511","name":"der","hasFamily":true,"annotation":"GTPase; multicopy suppressor of ftsJ; GTPase that plays an essential role in the late steps of ribosome biogenesis. GTPase point mutations (but not a deletion mutant) are suppressed by mild overexpression of RelA, probably due to increased levels of the stringent response mediator (p)ppGpp. 50S subunits assembled in the absence of Der are defective and unable to assemble into 70S ribosomes. GTPase activity is stimulated by YihI. Overexpression rescues an rrmJ deletion, stabilizing the 70S ribosome. Der and RrmJ are likely to share a mechanism to stabilize 50S ribosomal subunits at a ve [...] ","uniprotId":"DER_ECOLI"},"6883197":{"id":6883197,"externalId":"511145.b2512","name":"bamB","hasFamily":true,"annotation":"lipoprotein required for OM biogenesis, in BamABCDE complex; Part of the outer membrane protein assembly complex, which is involved in assembly and insertion of beta-barrel proteins into the outer membrane. Nonessential member of the complex, which may orient the flexible periplasmic domain of BamA for interaction with other Bam components, chaperones and nascent outer membrane proteins","uniprotId":"BAMB_ECOLI"},"6883198":{"id":6883198,"externalId":"511145.b2513","name":"yfgM","hasFamily":true,"annotation":"conserved protein, UPF0070 family","uniprotId":"YFGM_ECOLI"},"6883199":{"id":6883199,"externalId":"511145.b2514","name":"hisS","hasFamily":true,"annotation":"histidyl tRNA synthetase","uniprotId":"SYH_ECOLI"},"6883200":{"id":6883200,"externalId":"511145.b2515","name":"ispG","hasFamily":true,"annotation":"1-hydroxy-2-methyl-2-(E)-butenyl 4-diphosphate synthase; Converts 2C-methyl-D-erythritol 2,4-cyclodiphosphate (ME-2,4cPP) into 1-hydroxy-2-methyl-2-(E)-butenyl 4-diphosphate (By similarity)","uniprotId":"ISPG_ECOLI"},"6883201":{"id":6883201,"externalId":"511145.b2516","name":"rodZ","hasFamily":true,"annotation":"cytoskeletal protein required for MreB assembly; Cytoskeletal protein that is involved in cell-shape control through regulation of the length of the long axis (By similarity)","uniprotId":"RODZ_ECOLI"},"6883202":{"id":6883202,"externalId":"511145.b2517","name":"rlmN","hasFamily":true,"annotation":"23S rRNA m(2)A2503 methyltransferase, SAM-dependen; Specifically methylates position 2 of adenine 2503 in 23S rRNA and position 2 of adenine 37 in tRNAs. m2A2503 modification seems to play a crucial role in the proofreading step occurring at the peptidyl transferase center and thus would serve to optimize ribosomal fidelity. Unmodified tRNA is not a suitable substrate for RlmN, which suggests that RlmN works in a late step during tRNA maturation","uniprotId":"RLMN_ECOLI"},"6883203":{"id":6883203,"externalId":"511145.b2518","name":"ndk","hasFamily":true,"annotation":"multifunctional nucleoside diphosphate kinase and apyrimidinic endonuclease and 3'-phosphodiesterase; Major role in the synthesis of nucleoside triphosphates other than ATP. The ATP gamma phosphate is transferred to the NDP beta phosphate via a ping-pong mechanism, using a phosphorylated active-site intermediate (By similarity)","uniprotId":"NDK_ECOLI"},"6883204":{"id":6883204,"externalId":"511145.b2519","name":"pbpC","hasFamily":true,"annotation":"penicillin-binding protein PBP1C murein transglycosylase; inactive transpeptidase domain; Cell wall formation. The enzyme has a penicillin- insensitive transglycosylase N-terminal domain (formation of linear glycan strands) and a transpeptidase C-terminal domain which may not be functional","uniprotId":"PBPC_ECOLI"},"6883205":{"id":6883205,"externalId":"511145.b2520","name":"yfhM","hasFamily":true,"annotation":"conserved protein","uniprotId":"YFHM_ECOLI"},"6883206":{"id":6883206,"externalId":"511145.b2521","name":"sseA","hasFamily":true,"annotation":"3-mercaptopyruvate sulfurtransferase; Transfers a sulfur ion to cyanide or to other thiol compounds. Also has weak rhodanese activity (130-fold lower). Its participation in detoxification of cyanide may be small. May be involved in the enhancement of serine sensitivity","uniprotId":"THTM_ECOLI"},"6883207":{"id":6883207,"externalId":"511145.b2522","name":"sseB","hasFamily":true,"annotation":"rhodanase-like enzyme, sulfur transfer from thiosulfate; May be involved in the enhancement of serine- sensitivity","uniprotId":"SSEB_ECOLI"},"6883208":{"id":6883208,"externalId":"511145.b2523","name":"pepB","hasFamily":true,"annotation":"aminopeptidase B; Probably plays an important role in intracellular peptide degradation (By similarity)","uniprotId":"PEPB_ECOLI"},"6883209":{"id":6883209,"externalId":"511145.b2524","name":"iscX","hasFamily":true,"annotation":"conserved protein; May function as iron donor in the assembly of iron- sulfur clusters","uniprotId":"ISCX_ECOLI"},"6883210":{"id":6883210,"externalId":"511145.b2525","name":"fdx","hasFamily":true,"annotation":"[2Fe-2S] ferredoxin; Ferredoxin are iron-sulfur proteins that transfer electrons in a wide variety of metabolic reactions. Although the function of this ferredoxin is unknown it is probable that it has a role as a cellular electron transfer protein. Involved in the in vivo assembly of the Fe-S clusters in a wide variety of iron- sulfur proteins","uniprotId":"FER_ECOLI"},"6883211":{"id":6883211,"externalId":"511145.b2526","name":"hscA","hasFamily":true,"annotation":"DnaK-like molecular chaperone specific for IscU; Chaperone involved in the maturation of iron-sulfur cluster-containing proteins. Has a low intrinsic ATPase activity which is markedly stimulated by HscB. Involved in the maturation of IscU (By similarity)","uniprotId":"HSCA_ECOLI"},"6883212":{"id":6883212,"externalId":"511145.b2527","name":"hscB","hasFamily":true,"annotation":"DnaJ-like molecular chaperone specific for IscU; Co-chaperone involved in the maturation of iron-sulfur cluster-containing proteins. Seems to help targeting proteins to be folded toward HscA (By similarity)","uniprotId":"HSCB_ECOLI"},"6883213":{"id":6883213,"externalId":"511145.b2528","name":"iscA","hasFamily":true,"annotation":"FeS cluster assembly protein; Is able to transfer iron-sulfur clusters to apo- ferredoxin. Multiple cycles of [2Fe2S] cluster formation and transfer are observed, suggesting that IscA acts catalytically. Recruits intracellular free iron so as to provide iron for the assembly of transient iron-sulfur cluster in IscU in the presence of IscS, L-cysteine and the thioredoxin reductase system TrxA/TrxB (By similarity)","uniprotId":"ISCA_ECOLI"},"6883214":{"id":6883214,"externalId":"511145.b2529","name":"iscU","hasFamily":true,"annotation":"iron-sulfur cluster assembly scaffold protein; May be involved in the formation or repair of [Fe-S] clusters present in iron-sulfur proteins (Potential)","uniprotId":"NIFU_ECOLI"},"6883215":{"id":6883215,"externalId":"511145.b2530","name":"iscS","hasFamily":true,"annotation":"cysteine desulfurase (tRNA sulfurtransferase), PLP-dependent; Catalyzes the removal of elemental sulfur and selenium atoms from cysteine and selenocysteine to produce alanine. Functions as a sulfur delivery protein for NAD, biotin and Fe-S cluster synthesis. Transfers sulfur on 'Cys-456' of ThiI in a transpersulfidation reaction. Transfers sulfur on 'Cys-19' of TusA in a transpersulfidation reaction. Functions also as a selenium delivery protein in the pathway for the biosynthesis of selenophosphate","uniprotId":"ISCS_ECOLI"},"6883216":{"id":6883216,"externalId":"511145.b2531","name":"iscR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Regulates the transcription of several operons and genes involved in the biogenesis of Fe-S clusters and Fe-S-containing proteins. Transcriptional repressor of the iscRSUA operon, which is involved in the assembly of Fe-S clusters into Fe-S proteins. In its apoform, under conditions of oxidative stress or iron deprivation, it activates the suf operon, which is a second operon involved in the assembly of Fe-S clusters. Represses its own transcription as well as that of toxin rnlA","uniprotId":"ISCR_ECOLI"},"6883217":{"id":6883217,"externalId":"511145.b2532","name":"trmJ","hasFamily":true,"annotation":"tRNA mC32/mU32 methyltransferase, SAM-dependent; Catalyzes the formation of 2'O-methylated cytidine (Cm32) or 2'O-methylated uridine (Um32) at position 32 in tRNA","uniprotId":"TRMJ_ECOLI"},"6883218":{"id":6883218,"externalId":"511145.b2533","name":"suhB","hasFamily":true,"annotation":"inositol monophosphatase","uniprotId":"SUHB_ECOLI"},"6883219":{"id":6883219,"externalId":"511145.b2534","name":"yfhR","hasFamily":true,"annotation":"S9 peptidase family protein, function unknown","uniprotId":"YFHR_ECOLI"},"6883220":{"id":6883220,"externalId":"511145.b2535","name":"csiE","hasFamily":true,"annotation":"stationary phase inducible protein","uniprotId":"CSIE_ECOLI"},"6883221":{"id":6883221,"externalId":"511145.b2536","name":"hcaT","hasFamily":true,"annotation":"putative 3-phenylpropionic transporter; Probable permease involved in the uptake of 3- phenylpropionic acid","uniprotId":"HCAT_ECOLI"},"6883222":{"id":6883222,"externalId":"511145.b2537","name":"hcaR","hasFamily":false,"annotation":"3-hydroxypropionic acid resistance peptide; Transcriptional activator of the hca operon for 3- phenylpropionic acid catabolism","uniprotId":"HCAR_ECOLI"},"6883223":{"id":6883223,"externalId":"511145.b2538","name":"hcaE","hasFamily":true,"annotation":"3-phenylpropionate dioxygenase, large (alpha) subunit; Part of the multicomponent 3-phenylpropionate dioxygenase. Converts 3-phenylpropionic acid (PP) and cinnamic acid (CI) into 3-phenylpropionate-dihydrodiol (PP-dihydrodiol) and cinnamic acid-dihydrodiol (CI-dihydrodiol), respectively (By similarity)","uniprotId":"HCAE_ECOLI"},"6883224":{"id":6883224,"externalId":"511145.b2539","name":"hcaF","hasFamily":false,"annotation":"3-phenylpropionate dioxygenase, small (beta) subunit; Part of the multicomponent 3-phenylpropionate dioxygenase. Converts 3-phenylpropionic acid (PP) and cinnamic acid (CI) into 3-phenylpropionate-dihydrodiol (PP-dihydrodiol) and cinnamic acid-dihydrodiol (CI-dihydrodiol), respectively (By similarity)","uniprotId":"HCAF_ECOLI"},"6883225":{"id":6883225,"externalId":"511145.b2540","name":"hcaC","hasFamily":true,"annotation":"3-phenylpropionate dioxygenase, predicted ferredoxin subunit; Part of the multicomponent 3-phenylpropionate dioxygenase, that converts 3-phenylpropionic acid (PP) and cinnamic acid (CI) into 3-phenylpropionate-dihydrodiol (PP- dihydrodiol) and cinnamic acid-dihydrodiol (CI-dihydrodiol), respectively. This protein seems to be a 2Fe-2S ferredoxin (By similarity)","uniprotId":"HCAC_ECOLI"},"6883226":{"id":6883226,"externalId":"511145.b2541","name":"hcaB","hasFamily":false,"annotation":"2,3-dihydroxy-2,3-dihydrophenylpropionate dehydrogenase; Converts 3-phenylpropionate-dihydrodiol (PP-dihydrodiol) and cinnamic acid-dihydrodiol (CI-dihydrodiol) into 3-(2,3- dihydroxylphenyl)propanoic acid (DHPP) and 2,3-dihydroxicinnamic acid (DHCI), respectively (By similarity)","uniprotId":"HCAB_ECOLI"},"6883227":{"id":6883227,"externalId":"511145.b2542","name":"hcaD","hasFamily":true,"annotation":"phenylpropionate dioxygenase, ferredoxin reductase subunit; Part of the multicomponent 3-phenylpropionate dioxygenase, that converts 3-phenylpropionic acid (PP) and cinnamic acid (CI) into 3-phenylpropionate-dihydrodiol (PP- dihydrodiol) and cinnamic acid-dihydrodiol (CI-dihydrodiol), respectively (By similarity)","uniprotId":"HCAD_ECOLI"},"6883228":{"id":6883228,"externalId":"511145.b2543","name":"yphA","hasFamily":false,"annotation":"putative inner membrane protein","uniprotId":"YPHA_ECOLI"},"6883229":{"id":6883229,"externalId":"511145.b2544","name":"yphB","hasFamily":false,"annotation":"conserved protein","uniprotId":"YPHB_ECOLI"},"6883230":{"id":6883230,"externalId":"511145.b2545","name":"yphC","hasFamily":true,"annotation":"putative oxidoreductase, Zn-dependent and NAD(P)-binding","uniprotId":"YPHC_ECOLI"},"6883231":{"id":6883231,"externalId":"511145.b2546","name":"yphD","hasFamily":false,"annotation":"putative sugar transporter subunit: membrane component of ABC superfamily; Probably part of the binding-protein-dependent transport system YphDEF. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"YPHD_ECOLI"},"6883232":{"id":6883232,"externalId":"511145.b2547","name":"yphE","hasFamily":false,"annotation":"fused predicted sugar transporter subunits of ABC superfamily: ATP-binding components; Probably part of a binding-protein-dependent transport system YphDEF. Probably responsible for energy coupling to the transport system","uniprotId":"YPHE_ECOLI"},"6883233":{"id":6883233,"externalId":"511145.b2548","name":"yphF","hasFamily":false,"annotation":"putative sugar transporter subunit: periplasmic-binding component of ABC superfamily; Probably part of the binding-protein-dependent transport system YphDEF","uniprotId":"YPHF_ECOLI"},"6883234":{"id":6883234,"externalId":"511145.b2549","name":"yphG","hasFamily":true,"annotation":"conserved protein","uniprotId":"YPHG_ECOLI"},"6883235":{"id":6883235,"externalId":"511145.b2550","name":"yphH","hasFamily":false,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YPHH_ECOLI"},"6883236":{"id":6883236,"externalId":"511145.b2551","name":"glyA","hasFamily":true,"annotation":"serine hydroxymethyltransferase; Catalyzes the reversible interconversion of serine and glycine with tetrahydrofolate (THF) serving as the one-carbon carrier. This reaction serves as the major source of one-carbon groups required for the biosynthesis of purines, thymidylate, methionine, and other important biomolecules. Also exhibits THF- independent aldolase activity toward beta-hydroxyamino acids, producing glycine and aldehydes, via a retro-aldol mechanism. Thus, is able to catalyze the cleavage of allothreonine and 3- phenylserine. Also catalyzes the irreversible conversion of 5,10 [...] ","uniprotId":"GLYA_ECOLI"},"6883237":{"id":6883237,"externalId":"511145.b2552","name":"hmp","hasFamily":true,"annotation":"fused nitric oxide dioxygenase/dihydropteridine reductase 2; Is involved in NO detoxification in an aerobic process, termed nitric oxide dioxygenase (NOD) reaction that utilizes O(2) and NAD(P)H to convert NO to nitrate, which protects the bacterium from various noxious nitrogen compounds. Therefore, plays a central role in the inducible response to nitrosative stress (By similarity)","uniprotId":"HMP_ECOLI"},"6883238":{"id":6883238,"externalId":"511145.b2553","name":"glnB","hasFamily":false,"annotation":"regulatory protein P-II for glutamine synthetase; P-II indirectly controls the transcription of the glutamine synthetase gene (glnA). P-II prevents NR-II-catalyzed conversion of NR-I to NR-I-phosphate, the transcriptional activator of GlnA. When P-II is uridylylated to P-II-UMP, these events are reversed. When the ratio of Gln to 2-ketoglutarate decreases, P-II is uridylylated to P-II-UMP, which causes the deadenylation of glutamine synthetase by GlnE, so activating the enzyme","uniprotId":"GLNB_ECOLI"},"6883239":{"id":6883239,"externalId":"511145.b2554","name":"glrR","hasFamily":true,"annotation":"Response regulator regulating glmY sRNA in two-component system with sensor protein GlrK; Member of the two-component regulatory system GlrR/GlrK that up-regulates transcription of the glmY sRNA when cells enter the stationary growth phase. Regulates glmY transcription by binding to three conserved sites in the purL-glmY intergenic region","uniprotId":"GLRR_ECOLI"},"6883240":{"id":6883240,"externalId":"511145.b2555","name":"yfhG","hasFamily":true,"annotation":"conserved protein","uniprotId":"YFHG_ECOLI"},"6883241":{"id":6883241,"externalId":"511145.b2556","name":"glrK","hasFamily":true,"annotation":"Sensor protein kinase regulating glmY sRNA in two-component system with response regulator GlrR; Member of the two-component regulatory system GlrR/GlrK that up-regulates transcription of the glmY sRNA when cells enter the stationary growth phase. Activates GlrR by phosphorylation","uniprotId":"GLRK_ECOLI"},"6883242":{"id":6883242,"externalId":"511145.b2557","name":"purL","hasFamily":true,"annotation":"phosphoribosylformyl-glycineamide synthetase","uniprotId":"PUR4_ECOLI"},"6883243":{"id":6883243,"externalId":"511145.b2558","name":"mltF","hasFamily":true,"annotation":"membrane-bound lytic transglycosylase F, murein hydrolase; Murein-degrading enzyme that degrades murein glycan strands and insoluble, high-molecular weight murein sacculi, with the concomitant formation of a 1,6-anhydromuramoyl product. Lytic transglycosylases (LTs) play an integral role in the metabolism of the peptidoglycan (PG) sacculus. Their lytic action creates space within the PG sacculus to allow for its expansion as well as for the insertion of various structures such as secretion systems and flagella (By similarity)","uniprotId":"MLTF_ECOLI"},"6883244":{"id":6883244,"externalId":"511145.b2559","name":"tadA","hasFamily":true,"annotation":"tRNA-specific adenosine deaminase; Deaminates adenosine-34 to inosine in tRNA-Arg2. Mutation in this protein makes E.coli resistant to the toxic proteins encoded by the gef gene family. Essential for cell viability","uniprotId":"TADA_ECOLI"},"6883245":{"id":6883245,"externalId":"511145.b2560","name":"yfhB","hasFamily":true,"annotation":"conserved protein","uniprotId":"YFHB_ECOLI"},"6883246":{"id":6883246,"externalId":"511145.b2561","name":"yfhH","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YFHH_ECOLI"},"6883247":{"id":6883247,"externalId":"511145.b2562","name":"yfhL","hasFamily":true,"annotation":"putative 4Fe-4S cluster-containing protein","uniprotId":"YFHL_ECOLI"},"6883248":{"id":6883248,"externalId":"511145.b2563","name":"acpS","hasFamily":true,"annotation":"holo-[acyl-carrier-protein] synthase 1; Transfers the 4'-phosphopantetheine moiety from coenzyme A to a Ser of acyl-carrier-protein (By similarity)","uniprotId":"ACPS_ECOLI"},"6883249":{"id":6883249,"externalId":"511145.b2564","name":"pdxJ","hasFamily":true,"annotation":"pyridoxine 5'-phosphate synthase; Catalyzes the complicated ring closure reaction between the two acyclic compounds 1-deoxy-D-xylulose-5-phosphate (DXP) and 3-amino-2-oxopropyl phosphate (1-amino-acetone-3-phosphate or AAP) to form pyridoxine 5'-phosphate (PNP) and inorganic phosphate (By similarity)","uniprotId":"PDXJ_ECOLI"},"6883250":{"id":6883250,"externalId":"511145.b2565","name":"recO","hasFamily":true,"annotation":"gap repair protein; Involved in DNA repair and RecF pathway recombination (By similarity)","uniprotId":"RECO_ECOLI"},"6883251":{"id":6883251,"externalId":"511145.b2566","name":"era","hasFamily":true,"annotation":"membrane-associated, 16S rRNA-binding GTPase; An essential GTPase that binds both GDP and GTP, with nucleotide exchange occurring on the order of seconds whereas hydrolysis occurs on the order of minutes. Plays a role in numerous processes, including cell cycle regulation, energy metabolism, as a chaperone for 16S rRNA processing and 30S ribosomal subunit biogenesis. Its presence in the 30S subunit may prevent translation initiation. Seems to be critical for maintaining cell growth and cell divison rates; a dramatic reduction in Era protein levels temporarily arrests cell growth just b [...] ","uniprotId":"ERA_ECOLI"},"6883252":{"id":6883252,"externalId":"511145.b2567","name":"rnc","hasFamily":true,"annotation":"RNase III; Digests double-stranded RNA formed within single-strand substrates, but not RNA-DNA hybrids. Involved in the processing of rRNA precursors, viral transcripts, some mRNAs and at least 1 tRNA (metY, a minor form of tRNA-init-Met). Cleaves the 30S primary rRNA transcript to yield the immediate precursors to the 16S and 23S rRNAs; cleavage can occur in assembled 30S, 50S and even 70S subunits and is influenced by the presence of ribosomal proteins. The E.coli enzyme does not cleave R.capsulatus rRNA precursor, although R.capsulatus will complement an E.coli disruption, showing s [...] ","uniprotId":"RNC_ECOLI"},"6883253":{"id":6883253,"externalId":"511145.b2568","name":"lepB","hasFamily":true,"annotation":"leader peptidase (signal peptidase I)","uniprotId":"LEP_ECOLI"},"6883254":{"id":6883254,"externalId":"511145.b2569","name":"lepA","hasFamily":true,"annotation":"back-translocating Elongation Factor EF4, GTPase; Required for accurate and efficient protein synthesis under certain stress conditions. May act as a fidelity factor of the translation reaction, by catalyzing a one-codon backward translocation of tRNAs on improperly translocated ribosomes. Back- translocation proceeds from a post-translocation (POST) complex to a pre-translocation (PRE) complex, thus giving elongation factor G a second chance to translocate the tRNAs correctly. Binds to ribosomes in a GTP-dependent manner (By similarity)","uniprotId":"LEPA_ECOLI"},"6883255":{"id":6883255,"externalId":"511145.b2570","name":"rseC","hasFamily":true,"annotation":"RseC protein involved in reduction of the SoxR iron-sulfur cluster; Seems to modulate the activity of RpoE (sigma-E)","uniprotId":"RSEC_ECOLI"},"6883256":{"id":6883256,"externalId":"511145.b2571","name":"rseB","hasFamily":true,"annotation":"anti-sigma E factor, binds RseA; Seems to modulate the activity of RpoE (sigma-E)","uniprotId":"RSEB_ECOLI"},"6883257":{"id":6883257,"externalId":"511145.b2572","name":"rseA","hasFamily":true,"annotation":"anti-sigma factor; Negative regulator of RpoE","uniprotId":"RSEA_ECOLI"},"6883258":{"id":6883258,"externalId":"511145.b2573","name":"rpoE","hasFamily":true,"annotation":"RNA polymerase, sigma 24 (sigma E) factor; Sigma factors are initiation factors that promote the attachment of RNA polymerase to specific initiation sites and are then released. This sigma factor is involved in heat shock and oxidative stress response; it is believed to control protein processing in the extracytoplasmic compartment","uniprotId":"RPOE_ECOLI"},"6883259":{"id":6883259,"externalId":"511145.b2574","name":"nadB","hasFamily":true,"annotation":"quinolinate synthase, L-aspartate oxidase (B protein) subunit; Catalyzes the oxidation of L-aspartate to iminoaspartate","uniprotId":"NADB_ECOLI"},"6883260":{"id":6883260,"externalId":"511145.b2575","name":"yfiC","hasFamily":true,"annotation":"tRNA (adenine-N(6)-)-methyltransferase; Specifically methylates the adenine in position 37 of tRNA(1)(Val) (anticodon cmo5UAC) (By similarity)","uniprotId":"TRMN6_ECOLI"},"6883261":{"id":6883261,"externalId":"511145.b2576","name":"srmB","hasFamily":true,"annotation":"ATP-dependent RNA helicase; Acts as an ATP-dependent RNA helicase, able to unwind ds-RNA","uniprotId":"SRMB_ECOLI"},"6883262":{"id":6883262,"externalId":"511145.b2577","name":"yfiE","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YFIE_ECOLI"},"6883263":{"id":6883263,"externalId":"511145.b2578","name":"eamB","hasFamily":true,"annotation":"cysteine and O-acetylserine exporter; Exporter of O-acetylserine (OAS) and cysteine","uniprotId":"EAMB_ECOLI"},"6883264":{"id":6883264,"externalId":"511145.b2579","name":"grcA","hasFamily":true,"annotation":"autonomous glycyl radical cofactor; Acts as a radical domain for damaged PFL and possibly other radical proteins (By similarity)","uniprotId":"GRCA_ECOLI"},"6883265":{"id":6883265,"externalId":"511145.b2580","name":"ung","hasFamily":true,"annotation":"uracil-DNA-glycosylase; Excises uracil residues from the DNA which can arise as a result of misincorporation of dUMP residues by DNA polymerase or due to deamination of cytosine (By similarity)","uniprotId":"UNG_ECOLI"},"6883266":{"id":6883266,"externalId":"511145.b2581","name":"yfiF","hasFamily":true,"annotation":"putative methyltransferase","uniprotId":"YFIF_ECOLI"},"6883267":{"id":6883267,"externalId":"511145.b2582","name":"trxC","hasFamily":true,"annotation":"thioredoxin 2; Efficient electron donor for the essential enzyme ribonucleotide reductase. Is also able to reduce the interchain disulfide bridges of insulin","uniprotId":"THIO2_ECOLI"},"6883268":{"id":6883268,"externalId":"511145.b2583","name":"yfiP","hasFamily":true,"annotation":"conserved protein, DTW domain","uniprotId":"YFIP_ECOLI"},"6883269":{"id":6883269,"externalId":"511145.b2584","name":"yfiQ","hasFamily":true,"annotation":"protein lysine acetyltransferase","uniprotId":"YFIQ_ECOLI"},"6883270":{"id":6883270,"externalId":"511145.b2585","name":"pssA","hasFamily":true,"annotation":"phosphatidylserine synthase (CDP-diacylglycerol-serine O-phosphatidyltransferase)","uniprotId":"PSS_ECOLI"},"6883271":{"id":6883271,"externalId":"511145.b2586","name":"yfiM","hasFamily":true,"annotation":"required for high salt suppression of motility; probable lipoprotein","uniprotId":"YFIM_ECOLI"},"6883272":{"id":6883272,"externalId":"511145.b2587","name":"kgtP","hasFamily":true,"annotation":"alpha-ketoglutarate transporter; Uptake of alpha-ketoglutarate across the boundary membrane with the concomitant import of a cation (symport system)","uniprotId":"KGTP_ECOLI"},"6883273":{"id":6883273,"externalId":"511145.b2592","name":"clpB","hasFamily":true,"annotation":"protein disaggregation chaperone; Part of a stress-induced multi-chaperone system, it is involved in the recovery of the cell from heat-induced damage, in cooperation with DnaK, DnaJ and GrpE. Acts before DnaK, in the processing of protein aggregates. Protein binding stimulates the ATPase activity; ATP hydrolysis unfolds the denatured protein aggregates, which probably helps expose new hydrophobic binding sites on the surface of ClpB-bound aggregates, contributing to the solubilization and refolding of denatured protein aggregates by DnaK","uniprotId":"CLPB_ECOLI"},"6883274":{"id":6883274,"externalId":"511145.b2593","name":"yfiH","hasFamily":true,"annotation":"conserved protein, UPF0124 family","uniprotId":"YFIH_ECOLI"},"6883275":{"id":6883275,"externalId":"511145.b2594","name":"rluD","hasFamily":true,"annotation":"23S rRNA pseudouridine(1911,1915,1917) synthase (EC:4.2.1.70); Responsible for synthesis of pseudouridine from uracil at positions 1911, 1915 and 1917 in 23S ribosomal RNA. Isomerization occurs as a late step during the assembly of the large ribosomal subunit","uniprotId":"RLUD_ECOLI"},"6883276":{"id":6883276,"externalId":"511145.b2595","name":"bamD","hasFamily":true,"annotation":"lipoprotein required for OM biogenesis, in BamABCDE complex; Part of the outer membrane protein assembly complex, which is involved in assembly and insertion of beta-barrel proteins into the outer membrane. Constitutes, with BamA, the core component of the assembly machinery. Probably involved in transient protein interactions","uniprotId":"BAMD_ECOLI"},"6883277":{"id":6883277,"externalId":"511145.b2597","name":"raiA","hasFamily":true,"annotation":"cold shock protein associated with 30S ribosomal subunit; During stationary phase, prevents 70S dimer formation, probably in order to regulate translation efficiency during transition between the exponential and the stationary phases. In addition, during environmental stress such as cold shock or excessive cell density at stationary phase, stabilizes the 70S ribosome against dissociation, inhibits translation initiation and increase translation accuracy. When normal growth conditions are restored, is quickly released from the ribosome. Inhibits translation initiation by blocking the A- [...] ","uniprotId":"RAIA_ECOLI"},"6883278":{"id":6883278,"externalId":"511145.b2598","name":"pheL","hasFamily":false,"annotation":"pheA gene leader peptide"},"6883279":{"id":6883279,"externalId":"511145.b2599","name":"pheA","hasFamily":true,"annotation":"fused chorismate mutase P/prephenate dehydratase; Catalyzes the Claisen rearrangement of chorismate to prephenate and the decarboxylation/dehydration of prephenate to phenylpyruvate","uniprotId":"PHEA_ECOLI"},"6883280":{"id":6883280,"externalId":"511145.b2600","name":"tyrA","hasFamily":true,"annotation":"fused chorismate mutase T/prephenate dehydrogenase","uniprotId":"TYRA_ECOLI"},"6883281":{"id":6883281,"externalId":"511145.b2601","name":"aroF","hasFamily":true,"annotation":"3-deoxy-D-arabino-heptulosonate-7-phosphate synthase, tyrosine-repressible; Stereospecific condensation of phosphoenolpyruvate (PEP) and D-erythrose-4-phosphate (E4P) giving rise to 3-deoxy-D- arabino-heptulosonate-7-phosphate (DAHP) (By similarity)","uniprotId":"AROF_ECOLI"},"6883282":{"id":6883282,"externalId":"511145.b2602","name":"yfiL","hasFamily":false,"annotation":"lipoprotein","uniprotId":"YFIL_ECOLI"},"6883283":{"id":6883283,"externalId":"511145.b2603","name":"yfiR","hasFamily":true,"annotation":"probable periplasmic inhibitor of YfiN activity","uniprotId":"YFIR_ECOLI"},"6883284":{"id":6883284,"externalId":"511145.b2604","name":"yfiN","hasFamily":true,"annotation":"probable inner membrane diguanylate cyclase; Cyclic-di-GMP is a second messenger which controls cell surface-associated traits in bacteria","uniprotId":"YFIN_ECOLI"},"6883285":{"id":6883285,"externalId":"511145.b2605","name":"yfiB","hasFamily":true,"annotation":"probable positive effector of YfiN activity, OM lipoprotein","uniprotId":"YFIB_ECOLI"},"6883286":{"id":6883286,"externalId":"511145.b2606","name":"rplS","hasFamily":true,"annotation":"50S ribosomal subunit protein L19; This protein is located at the 30S-50S ribosomal subunit interface. In the 70S ribosome (PubMed:12809609) it has been modeled to make two contacts with the 16S rRNA of the 30S subunit forming part of bridges B6 and B8. In the 3.5 A resolved structures (PubMed:16272117) L14 and L19 interact and together make contact with the 16S rRNA. The protein conformation is quite different between the 50S and 70S structures, which may be necessary for translocation","uniprotId":"RL19_ECOLI"},"6883287":{"id":6883287,"externalId":"511145.b2607","name":"trmD","hasFamily":true,"annotation":"tRNA m(1)G37 methyltransferase, SAM-dependent; Specifically methylates guanosine-37 in various tRNAs (By similarity)","uniprotId":"TRMD_ECOLI"},"6883288":{"id":6883288,"externalId":"511145.b2608","name":"rimM","hasFamily":true,"annotation":"16S rRNA processing protein; An accessory protein needed during the final step in assembly of the 30S ribosomal subunit, probably for assembly of the head region (also called the 3' domain). It may act while Era is associated and after RimM in 30S subunit assembly. Interacts with S19. Essential for efficient processing of 16S rRNA; a deletion mutant accumulates 17S rRNA. May be needed both before and after RbfA during the maturation of 16S rRNA. It has affinity for free ribosomal 30S subunits but not for 70S ribosomes","uniprotId":"RIMM_ECOLI"},"6883289":{"id":6883289,"externalId":"511145.b2609","name":"rpsP","hasFamily":true,"annotation":"30S ribosomal subunit protein S16; In addition to being a ribosomal protein, S16 also has a cation-dependent endonuclease activity","uniprotId":"RS16_ECOLI"},"6883290":{"id":6883290,"externalId":"511145.b2610","name":"ffh","hasFamily":true,"annotation":"Signal Recognition Particle (SRP) component with 4.5S RNA (ffs); Involved in targeting and insertion of nascent membrane proteins into the cytoplasmic membrane. Binds to the hydrophobic signal sequence of the ribosome-nascent chain (RNC) as it emerges from the ribosomes. The SRP-RNC complex is then targeted to the cytoplasmic membrane where it interacts with the SRP receptor FtsY. Interaction with FtsY leads to the transfer of the RNC complex to the Sec translocase for insertion into the membrane, the hydrolysis of GTP by both Ffh and FtsY, and the dissociation of the SRP-FtsY complex  [...] ","uniprotId":"SRP54_ECOLI"},"6883291":{"id":6883291,"externalId":"511145.b2611","name":"ypjD","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YPJD_ECOLI"},"6883292":{"id":6883292,"externalId":"511145.b2614","name":"grpE","hasFamily":true,"annotation":"heat shock protein; Participates actively in the response to hyperosmotic and heat shock by preventing the aggregation of stress-denatured proteins, in association with DnaK and GrpE. It is the nucleotide exchange factor for DnaK and may function as a thermosensor. Unfolded proteins bind initially to DnaJ; upon interaction with the DnaJ-bound protein, DnaK hydrolyzes its bound ATP, resulting in the formation of a stable complex. GrpE releases ADP from DnaK; ATP binding to DnaK triggers the release of the substrate protein, thus completing the reaction cycle. Several rounds of ATP- depe [...] ","uniprotId":"GRPE_ECOLI"},"6883293":{"id":6883293,"externalId":"511145.b2615","name":"nadK","hasFamily":true,"annotation":"NAD kinase; Catalyzes the phosphorylation of NAD to NADP. Utilizes ATP and other nucleoside triphosphates as well as inorganic polyphosphate as a source of phosphorus (By similarity)","uniprotId":"PPNK_ECOLI"},"6883294":{"id":6883294,"externalId":"511145.b2616","name":"recN","hasFamily":true,"annotation":"recombination and repair protein; May be involved in recombinational repair of damaged DNA (By similarity)","uniprotId":"RECN_ECOLI"},"6883295":{"id":6883295,"externalId":"511145.b2617","name":"bamE","hasFamily":true,"annotation":"lipoprotein component of BamABCDE OM biogenesis complex; Part of the outer membrane protein assembly complex, which is involved in assembly and insertion of beta-barrel proteins into the outer membrane. Nonessential member of the complex that stabilizes the interaction between the essential proteins BamA and BamD. May modulate the conformation of BamA, likely through interactions with BamD","uniprotId":"BAME_ECOLI"},"6883296":{"id":6883296,"externalId":"511145.b2618","name":"ratB","hasFamily":true,"annotation":"conserved protein, UPF0125 family; Does not function as an antitoxin to the upstream RatA toxin","uniprotId":"RATB_ECOLI"},"6883297":{"id":6883297,"externalId":"511145.b2619","name":"ratA","hasFamily":true,"annotation":"toxic protein, UPF0083 family; Toxic component of a toxin-antitoxin (TA) module. Binds to 50S ribosomal subunits, preventing them from associating with 30S subunits to form 70S ribosomes and reducing polysomes. It does not cause ribosomes to dissociate however. The antibiotic paromomycin blocks the anti-association activity of RatA. Overexpression results in inhibition of growth in liquid cultures, and in a decrease in protein translation. The other gene of this operon, ratB, is not the cognate antitoxin in this strain; in CFT073 however it does fulfill this function","uniprotId":"RATA_ECOLI"},"6883298":{"id":6883298,"externalId":"511145.b2620","name":"smpB","hasFamily":true,"annotation":"trans-translation protein; Binds specifically to the SsrA RNA (tmRNA) and is required for stable association of SsrA with ribosomes (By similarity)","uniprotId":"C3UPD1_ECOLI"},"6883299":{"id":6883299,"externalId":"511145.b2622","name":"intA","hasFamily":true,"annotation":"CP4-57 prophage; Integrase is necessary for integration of the phage into the host genome by site-specific recombination. In conjunction with excisionase, integrase is also necessary for excision of the prophage from the host genome","uniprotId":"INTA_ECOLI"},"6883300":{"id":6883300,"externalId":"511145.b2623","name":"yfjH","hasFamily":false,"annotation":"CP4-57 prophage","uniprotId":"YFJH_ECOLI"},"6883301":{"id":6883301,"externalId":"511145.b2624","name":"alpA","hasFamily":false,"annotation":"CP4-57 prophage; DNA-binding transcriptional activator; Positive regulator of the expression of the slpA gene","uniprotId":"ALPA_ECOLI"},"6883302":{"id":6883302,"externalId":"511145.b2625","name":"yfjI","hasFamily":true,"annotation":"CP4-57 prophage","uniprotId":"YFJI_ECOLI"},"6883303":{"id":6883303,"externalId":"511145.b2626","name":"yfjJ","hasFamily":true,"annotation":"CP4-57 prophage","uniprotId":"YFJJ_ECOLI"},"6883304":{"id":6883304,"externalId":"511145.b2627","name":"yfjK","hasFamily":false,"annotation":"CP4-57 prophage","uniprotId":"YFJK_ECOLI"},"6883305":{"id":6883305,"externalId":"511145.b2628","name":"yfjL","hasFamily":false,"annotation":"CP4-57 prophage","uniprotId":"YFJL_ECOLI"},"6883306":{"id":6883306,"externalId":"511145.b2629","name":"yfjM","hasFamily":false,"annotation":"CP4-57 prophage","uniprotId":"YFJM_ECOLI"},"6883307":{"id":6883307,"externalId":"511145.b2630","name":"rnlA","hasFamily":false,"annotation":"CP4-57 prophage; Toxic component of a toxin-antitoxin (TA) module. A stable (half-life 27.6 minutes) endoribonuclease that in the absence of cognate antitoxin RnlB causes generalized RNA degradation. Degrades late enterobacteria phage T4 mRNAs, protecting the host against T4 reproduction. Activity is inhibited by cognate antitoxin RnlB and by enterobacteria phage T4 protein Dmd. Targets cyaA mRNA","uniprotId":"RNLA_ECOLI"},"6883308":{"id":6883308,"externalId":"511145.b2631","name":"rnlB","hasFamily":false,"annotation":"CP4-57 prophage; Antitoxin component of a toxin-antitoxin (TA) module. A labile antitoxin (half-life of 2.1 minutes) that inhibits the endonuclease activity of cognate toxin RnlA but not that of non- cognate toxin LsoA","uniprotId":"RNLB_ECOLI"},"6883309":{"id":6883309,"externalId":"511145.b2632","name":"yfjP","hasFamily":true,"annotation":"CP4-57 prophage; predicted GTP-binding protein","uniprotId":"YFJP_ECOLI"},"6883310":{"id":6883310,"externalId":"511145.b2633","name":"yfjQ","hasFamily":false,"annotation":"CP4-57 prophage","uniprotId":"YFJQ_ECOLI"},"6883311":{"id":6883311,"externalId":"511145.b2634","name":"yfjR","hasFamily":true,"annotation":"CP4-57 prophage","uniprotId":"YFJR_ECOLI"},"6883312":{"id":6883312,"externalId":"511145.b2635","name":"ypjK","hasFamily":false,"annotation":"CP4-57 prophage","uniprotId":"YPJK_ECOLI"},"6883313":{"id":6883313,"externalId":"511145.b2636","name":"yfjS","hasFamily":false,"annotation":"CP4-57 prophage","uniprotId":"YFJS_ECOLI"},"6883314":{"id":6883314,"externalId":"511145.b2637","name":"yfjT","hasFamily":false,"annotation":"CP4-57 prophage","uniprotId":"YFJT_ECOLI"},"6883315":{"id":6883315,"externalId":"511145.b2642","name":"yfjW","hasFamily":true,"annotation":"CP4-57 prophage","uniprotId":"YFJW_ECOLI"},"6883316":{"id":6883316,"externalId":"511145.b2643","name":"yfjX","hasFamily":false,"annotation":"CP4-57 prophage","uniprotId":"YFJX_ECOLI"},"6883317":{"id":6883317,"externalId":"511145.b2644","name":"yfjY","hasFamily":false,"annotation":"CP4-57 prophage","uniprotId":"YFJY_ECOLI"},"6883318":{"id":6883318,"externalId":"511145.b2645","name":"yfjZ","hasFamily":true,"annotation":"CP4-57 prophage; Might be an antitoxin component of a toxin-antitoxin (TA) module. A putative antitoxin that may inactivate probable toxin YpjF (Potential)","uniprotId":"YFJZ_ECOLI"},"6883319":{"id":6883319,"externalId":"511145.b2646","name":"ypjF","hasFamily":false,"annotation":"CP4-57 prophage; Toxic component of a toxin-antitoxin (TA) module. Overexpression results in inhibition of growth in liquid cultures. Might be inactivated by putative antitoxin YfjZ","uniprotId":"YPJF_ECOLI"},"6883320":{"id":6883320,"externalId":"511145.b2647","name":"ycgI","hasFamily":true,"annotation":"adhesin-like autotransporter","uniprotId":"YCGI_ECOLI"},"6883321":{"id":6883321,"externalId":"511145.b2659","name":"csiD","hasFamily":true,"annotation":"carbon starvation protein; May be involved in the control of utilization of gamma- aminobutyric acid (By similarity)","uniprotId":"CSID_ECOLI"},"6883322":{"id":6883322,"externalId":"511145.b2660","name":"lhgO","hasFamily":true,"annotation":"L-2-hydroxyglutarate oxidase; Oxidizes L-2-hydroxyglutarate, probably to recover alpha-ketoglutarate mistakenly reduced by other enzymes or formed during growth on propionate","uniprotId":"LHGO_ECOLI"},"6883323":{"id":6883323,"externalId":"511145.b2661","name":"gabD","hasFamily":true,"annotation":"succinate-semialdehyde dehydrogenase I, NADP-dependent; Catalyzes the NADP(+)-dependent oxidation of succinate semialdehyde to succinate. It appears to be important for nitrogen metabolism under N limitation conditions","uniprotId":"GABD_ECOLI"},"6883324":{"id":6883324,"externalId":"511145.b2662","name":"gabT","hasFamily":true,"annotation":"4-aminobutyrate aminotransferase, PLP-dependent; Catalyzes the transfer of the amino group from gamma- aminobutyrate (GABA) to alpha-ketoglutarate (KG) to yield succinic semialdehyde (SSA)","uniprotId":"GABT_ECOLI"},"6883325":{"id":6883325,"externalId":"511145.b2663","name":"gabP","hasFamily":true,"annotation":"gamma-aminobutyrate transporter; Transporter for GABA","uniprotId":"GABP_ECOLI"},"6883326":{"id":6883326,"externalId":"511145.b2664","name":"csiR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor of csiD; Modulates expression of csiD and lhgO in a temporal manner during entry into stationary phase or during the first few hours of carbon starvation","uniprotId":"CSIR_ECOLI"},"6883327":{"id":6883327,"externalId":"511145.b2665","name":"ygaU","hasFamily":true,"annotation":"predicted protein","uniprotId":"YGAU_ECOLI"},"6883328":{"id":6883328,"externalId":"511145.b2666","name":"yqaE","hasFamily":true,"annotation":"cyaR sRNA-regulated protein","uniprotId":"YQAE_ECOLI"},"6883329":{"id":6883329,"externalId":"511145.b2667","name":"ygaV","hasFamily":true,"annotation":"tributyltin-inducible repressor of ygaVP; Functions as transcription repressor (Probable)","uniprotId":"YGAV_ECOLI"},"6883330":{"id":6883330,"externalId":"511145.b2668","name":"ygaP","hasFamily":true,"annotation":"putative inner membrane protein with hydrolase activity","uniprotId":"YGAP_ECOLI"},"6883331":{"id":6883331,"externalId":"511145.b2669","name":"stpA","hasFamily":true,"annotation":"DNA binding protein, nucleoid-associated; It may act in a fashion similar to H-NS proteins","uniprotId":"STPA_ECOLI"},"6883332":{"id":6883332,"externalId":"511145.b2670","name":"alaE","hasFamily":true,"annotation":"alanine exporter, alanine-inducible, stress-responsive; Exports L-alanine (By similarity)","uniprotId":"ALAE_ECOLI"},"6883333":{"id":6883333,"externalId":"511145.b2671","name":"ygaC","hasFamily":true,"annotation":"predicted protein","uniprotId":"YGAC_ECOLI"},"6883334":{"id":6883334,"externalId":"511145.b2672","name":"ygaM","hasFamily":true,"annotation":"predicted protein","uniprotId":"YGAM_ECOLI"},"6883335":{"id":6883335,"externalId":"511145.b2673","name":"nrdH","hasFamily":true,"annotation":"Hydrogen donor for NrdEF electron transport system; glutaredoxin-like protein; Electron transport system for the ribonucleotide reductase system NrdEF","uniprotId":"NRDH_ECOLI"},"6883336":{"id":6883336,"externalId":"511145.b2674","name":"nrdI","hasFamily":true,"annotation":"flavodoxin required for NrdEF cluster assembly; Probably involved in ribonucleotide reductase function (By similarity)","uniprotId":"NRDI_ECOLI"},"6883337":{"id":6883337,"externalId":"511145.b2675","name":"nrdE","hasFamily":true,"annotation":"ribonucleoside-diphosphate reductase 2, alpha subunit; Provides the precursors necessary for DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding ribonucleotides. R1E contains the binding sites for both substrates and allosteric effectors and carries out the actual reduction of the ribonucleotide","uniprotId":"RIR3_ECOLI"},"6883338":{"id":6883338,"externalId":"511145.b2676","name":"nrdF","hasFamily":true,"annotation":"ribonucleoside-diphosphate reductase 2, beta subunit, ferritin-like protein; Provides the precursors necessary for DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding ribonucleotides. R2F contains the tyrosyl radical required for catalysis","uniprotId":"RIR4_ECOLI"},"6883339":{"id":6883339,"externalId":"511145.b2677","name":"proV","hasFamily":true,"annotation":"glycine betaine transporter subunit; Involved in a multicomponent binding-protein-dependent transport system for glycine betaine/L-proline","uniprotId":"PROV_ECOLI"},"6883340":{"id":6883340,"externalId":"511145.b2678","name":"proW","hasFamily":true,"annotation":"glycine betaine transporter subunit; Involved in a multicomponent binding-protein-dependent transport system for glycine betaine/L-proline","uniprotId":"PROW_ECOLI"},"6883341":{"id":6883341,"externalId":"511145.b2679","name":"proX","hasFamily":true,"annotation":"glycine betaine transporter subunit; Member of a multicomponent binding-protein-dependent transport system (the ProU transporter) which serves as the glycine betaine/L-proline transporter","uniprotId":"PROX_ECOLI"},"6883342":{"id":6883342,"externalId":"511145.b2682","name":"ygaZ","hasFamily":true,"annotation":"probable L-valine exporter, norvaline resistance","uniprotId":"YGAZ_ECOLI"},"6883343":{"id":6883343,"externalId":"511145.b2683","name":"ygaH","hasFamily":true,"annotation":"probable L-valine exporter, norvaline resistance","uniprotId":"YGAH_ECOLI"},"6883344":{"id":6883344,"externalId":"511145.b2684","name":"emrR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor of microcin B17 synthesis and multidrug efflux; Negative regulator of the multidrug operon emrAB","uniprotId":"MPRA_ECOLI"},"6883345":{"id":6883345,"externalId":"511145.b2685","name":"emrA","hasFamily":true,"annotation":"multidrug efflux system; The emr locus confers resistance to substances of high hydrophobicity. EmrA probably participate in a transport system to extrude toxins and drugs from the cell","uniprotId":"EMRA_ECOLI"},"6883346":{"id":6883346,"externalId":"511145.b2686","name":"emrB","hasFamily":true,"annotation":"multidrug efflux system protein; Translocase that confers resistance to substances of high hydrophobicity","uniprotId":"EMRB_ECOLI"},"6883347":{"id":6883347,"externalId":"511145.b2687","name":"luxS","hasFamily":true,"annotation":"S-ribosylhomocysteine lyase; Involved in the synthesis of autoinducer 2 (AI-2) which is secreted by bacteria and is used to communicate both the cell density and the metabolic potential of the environment. The regulation of gene expression in response to changes in cell density is called quorum sensing. Catalyzes the transformation of S-ribosylhomocysteine (RHC) to homocysteine (HC) and 4,5- dihydroxy-2,3-pentadione (DPD) (By similarity)","uniprotId":"LUXS_ECOLI"},"6883348":{"id":6883348,"externalId":"511145.b2688","name":"gshA","hasFamily":true,"annotation":"glutamate-cysteine ligase","uniprotId":"GSH1_ECOLI"},"6883349":{"id":6883349,"externalId":"511145.b2689","name":"yqaA","hasFamily":true,"annotation":"inner membrane protein, COG1238 family","uniprotId":"YQAA_ECOLI"},"6883350":{"id":6883350,"externalId":"511145.b2690","name":"yqaB","hasFamily":true,"annotation":"fructose-1-P and 6-phosphogluconate phosphatase; Catalyzes strongly the dephosphorylation of fructose-1- phosphate (Fru1P) and slightly the dephosphorylation of 6- phosphogluconate (6P-Glu). It has low beta-phosphoglucomutase activity","uniprotId":"YQAB_ECOLI"},"6883351":{"id":6883351,"externalId":"511145.b2696","name":"csrA","hasFamily":false,"annotation":"pleiotropic regulatory protein for carbon source metabolism; Binds to mRNA to regulate post-transcriptional activity. Affects glycogen biosynthesis, gluconeogenesis, cell size and surface properties. Regulates glycogen synthesis under both aerobic and anaerobic conditions. Seems to accelerate the degradation of glg gene transcripts, potentially through selective RNA binding. Acts to inhibit interaction between the LetD protein and the A subunit of DNA gyrase. Also required for motility and flagellum biosynthesis through the post-transcriptional activation of flhDC expression. This invo [...] ","uniprotId":"CSRA_ECOLI"},"6883352":{"id":6883352,"externalId":"511145.b2697","name":"alaS","hasFamily":true,"annotation":"alanyl-tRNA synthetase; Catalyzes the attachment of alanine to tRNA(Ala) in a two-step reaction: alanine is first activated by ATP to form Ala- AMP and then transferred to the acceptor end of tRNA(Ala). AlaRS also incorrectly activates the sterically smaller than alanine amino acid glycine as well as the sterically larger amino acid serine. These incorrectly charged amino acids occur because the of inherent physicochemical limitations on discrimination between closely related amino acids (glycine and serine) in the charging step","uniprotId":"SYA_ECOLI"},"6883353":{"id":6883353,"externalId":"511145.b2698","name":"oraA","hasFamily":true,"annotation":"regulatory protein for RecA; Modulates RecA activity through direct physical interaction. Can inhibit both RecA recombinase and coprotease activities. May have a regulatory role during the SOS response. Inhibits DNA strand exchange in vitro","uniprotId":"RECX_ECOLI"},"6883354":{"id":6883354,"externalId":"511145.b2699","name":"recA","hasFamily":true,"annotation":"DNA strand exchange and recombination protein with protease and nuclease activity; Can catalyze the hydrolysis of ATP in the presence of single-stranded DNA, the ATP-dependent uptake of single-stranded DNA by duplex DNA, and the ATP-dependent hybridization of homologous single-stranded DNAs. It interacts with LexA causing its activation and leading to its autocatalytic cleavage (By similarity)","uniprotId":"RECA_ECOLI"},"6883355":{"id":6883355,"externalId":"511145.b2700","name":"pncC","hasFamily":true,"annotation":"conserved protein; Has nicotinamidemononucleotide (NMN) aminohydrolase activity, not active on other substrates","uniprotId":"PNCC_ECOLI"},"6883356":{"id":6883356,"externalId":"511145.b2701","name":"mltB","hasFamily":true,"annotation":"membrane-bound lytic murein transglycosylase B; Murein-degrading enzyme. Catalyzes the cleavage of the glycosidic bonds between N-acetylmuramic acid and N- acetylglucosamine residues in peptidoglycan. May play a role in recycling of muropeptides during cell elongation and/or cell division","uniprotId":"MLTB_ECOLI"},"6883357":{"id":6883357,"externalId":"511145.b2702","name":"srlA","hasFamily":true,"annotation":"glucitol/sorbitol-specific enzyme IIC component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in glucitol/sorbitol transport","uniprotId":"PTHC_ECOLI"},"6883358":{"id":6883358,"externalId":"511145.b2703","name":"srlE","hasFamily":true,"annotation":"glucitol/sorbitol-specific enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in glucitol/sorbitol transport","uniprotId":"PTHB_ECOLI"},"6883359":{"id":6883359,"externalId":"511145.b2704","name":"srlB","hasFamily":true,"annotation":"glucitol/sorbitol-specific enzyme IIA component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in glucitol/sorbitol transport","uniprotId":"PTHA_ECOLI"},"6883360":{"id":6883360,"externalId":"511145.b2705","name":"gutD","hasFamily":true,"annotation":"sorbitol-6-phosphate dehydrogenase","uniprotId":"SRLD_ECOLI"},"6883361":{"id":6883361,"externalId":"511145.b2706","name":"gutM","hasFamily":true,"annotation":"DNA-binding transcriptional activator of glucitol operon; Positive regulator for glucitol operon expression","uniprotId":"GUTM_ECOLI"},"6883362":{"id":6883362,"externalId":"511145.b2707","name":"srlR","hasFamily":true,"annotation":"DNA-bindng transcriptional repressor","uniprotId":"SRLR_ECOLI"},"6883363":{"id":6883363,"externalId":"511145.b2708","name":"gutQ","hasFamily":true,"annotation":"D-arabinose 5-phosphate isomerase; Catalyzes the reversible aldol-ketol isomerization between D-ribulose 5-phosphate (Ru5P) and D-arabinose 5-phosphate (A5P). It appears that the physiological function of G-API may be to synthesize the regulatory molecule A5P, which in turn participates in the induction of the gut operon through an unknown mechanism. It is also able of sustaining the biosynthetic pathway of 3-deoxy-D-manno-octulosonate (KDO), a unique 8-carbon sugar component of lipopolysaccharides (LPSs)","uniprotId":"GUTQ_ECOLI"},"6883364":{"id":6883364,"externalId":"511145.b2709","name":"norR","hasFamily":true,"annotation":"Anaerobic nitric oxide reductase DNA-binding transcriptional activator; Required for the expression of anaerobic nitric oxide (NO) reductase, acts as a transcriptional activator for at least the norVW operon. Activation also requires sigma-54. Not required for induction of the aerobic NO-detoxifying enzyme NO dioxygenase. Binds to the promoter region of norVW, to a consensus target sequence, GT-(N7)-AC, which is highly conserved among proteobacteria","uniprotId":"NORR_ECOLI"},"6883365":{"id":6883365,"externalId":"511145.b2710","name":"norV","hasFamily":true,"annotation":"anaerobic nitric oxide reductase flavorubredoxin; Anaerobic nitric oxide reductase; uses NADH to detoxify nitric oxide (NO), protecting several 4Fe-4S NO-sensitive enzymes. Has at least 2 reductase partners, only one of which (NorW, flavorubredoxin reductase) has been identified. NO probably binds to the di-iron center; electrons enter from the NorW at rubredoxin and are transferred sequentially to the FMN center and the di-iron center. Also able to function as an aerobic oxygen reductase (By similarity)","uniprotId":"NORV_ECOLI"},"6883366":{"id":6883366,"externalId":"511145.b2711","name":"norW","hasFamily":true,"annotation":"NADH:flavorubredoxin oxidoreductase; One of at least two accessory proteins for anaerobic nitric oxide (NO) reductase. Reduces the rubredoxin moiety of NO reductase (By similarity)","uniprotId":"NORW_ECOLI"},"6883367":{"id":6883367,"externalId":"511145.b2712","name":"hypF","hasFamily":true,"annotation":"carbamoyl phosphate phosphatase and maturation protein for [NiFe] hydrogenases; Along with HypE, it catalyzes the synthesis of the CN ligands of the active site iron of [NiFe]-hydrogenases using carbamoylphosphate as a substrate. It functions as a carbamoyl transferase using carbamoylphosphate as a substrate and transferring the carboxamido moiety in an ATP-dependent reaction to the thiolate of the C-terminal cysteine of HypE yielding a protein-S-carboxamide","uniprotId":"HYPF_ECOLI"},"6883368":{"id":6883368,"externalId":"511145.b2713","name":"hydN","hasFamily":true,"annotation":"formate dehydrogenase-H, [4Fe-4S] ferredoxin subunit; Electron transport from formate to hydrogen","uniprotId":"HYDN_ECOLI"},"6883369":{"id":6883369,"externalId":"511145.b2714","name":"ascG","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor of the asc operon. The cryptic operon is activated by the insertion of IS186 into the ascG gene","uniprotId":"ASCG_ECOLI"},"6883370":{"id":6883370,"externalId":"511145.b2715","name":"ascF","hasFamily":false,"annotation":"fused cellobiose/arbutin/salicin-specific PTS enzymes: IIB component/IC component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in arbutin, cellobiose, and salicin transport","uniprotId":"PTIBC_ECOLI"},"6883371":{"id":6883371,"externalId":"511145.b2716","name":"ascB","hasFamily":true,"annotation":"cryptic 6-phospho-beta-glucosidase; Can hydrolyze salicin, cellobiose, and probably arbutin","uniprotId":"ASCB_ECOLI"},"6883372":{"id":6883372,"externalId":"511145.b2717","name":"hycI","hasFamily":true,"annotation":"protease involved in processing C-terminal end of HycE; Protease involved in the C-terminal processing of HycE, the large subunit of hydrogenase 3","uniprotId":"HYCI_ECOLI"},"6883373":{"id":6883373,"externalId":"511145.b2718","name":"hycH","hasFamily":true,"annotation":"protein required for maturation of hydrogenase 3; Seems to be required for the conversion of a precursor form of the large subunit of hydrogenlyase (HycE) into a mature form","uniprotId":"HYCH_ECOLI"},"6883374":{"id":6883374,"externalId":"511145.b2719","name":"hycG","hasFamily":true,"annotation":"hydrogenase 3 and formate hydrogenase complex, HycG subunit","uniprotId":"HYCG_ECOLI"},"6883375":{"id":6883375,"externalId":"511145.b2720","name":"hycF","hasFamily":true,"annotation":"formate hydrogenlyase complex iron-sulfur protein; Probable electron transfer protein for hydrogenase 3","uniprotId":"HYCF_ECOLI"},"6883376":{"id":6883376,"externalId":"511145.b2721","name":"hycE","hasFamily":true,"annotation":"hydrogenase 3, large subunit","uniprotId":"HYCE_ECOLI"},"6883377":{"id":6883377,"externalId":"511145.b2722","name":"hycD","hasFamily":true,"annotation":"hydrogenase 3, membrane subunit","uniprotId":"HYCD_ECOLI"},"6883378":{"id":6883378,"externalId":"511145.b2723","name":"hycC","hasFamily":true,"annotation":"hydrogenase 3, membrane subunit","uniprotId":"HYCC_ECOLI"},"6883379":{"id":6883379,"externalId":"511145.b2724","name":"hycB","hasFamily":true,"annotation":"hydrogenase 3, Fe-S subunit; Probable electron transfer protein for hydrogenase 3","uniprotId":"HYCB_ECOLI"},"6883380":{"id":6883380,"externalId":"511145.b2725","name":"hycA","hasFamily":true,"annotation":"regulator of the transcriptional regulator FhlA; Regulatory protein for the formate hydrogenlyase system. Could act by directly interacting with FhlA or by preventing the binding of FhlA to the upstream activatory sequence","uniprotId":"HYCA_ECOLI"},"6883381":{"id":6883381,"externalId":"511145.b2726","name":"b2726","hasFamily":true,"annotation":"protein involved in nickel insertion into hydrogenases 3; Not required for the formation of hydrogenases 1 and 2. Plays probably a regulatory role","uniprotId":"HYPA_ECOLI"},"6883382":{"id":6883382,"externalId":"511145.b2727","name":"hypB","hasFamily":true,"annotation":"GTP hydrolase involved in nickel liganding into hydrogenases; Required for the maturation of the three [NiFe]hydrogenases. Exhibits a low intrinsic GTPase activity. The GTP hydrolysis catalyzed by HypB is an integral process in the incorporation of nickel into hydrogenases","uniprotId":"HYPB_ECOLI"},"6883383":{"id":6883383,"externalId":"511145.b2728","name":"hypC","hasFamily":true,"annotation":"protein required for maturation of hydrogenases 1 and 3; Is required for the formation of all three hydrogenase isoenzymes. May bind to the precursor form of the large subunit of dehydrogenases to keep them in a conformation accessible for metal incorporation","uniprotId":"HYPC_ECOLI"},"6883384":{"id":6883384,"externalId":"511145.b2729","name":"hypD","hasFamily":true,"annotation":"protein required for maturation of hydrogenases; Required for the formation of all three hydrogenase isoenzymes","uniprotId":"HYPD_ECOLI"},"6883385":{"id":6883385,"externalId":"511145.b2730","name":"hypE","hasFamily":true,"annotation":"carbamoyl dehydratase, hydrogenases 1,2,3 maturation protein","uniprotId":"HYPE_ECOLI"},"6883386":{"id":6883386,"externalId":"511145.b2731","name":"fhlA","hasFamily":true,"annotation":"DNA-binding transcriptional activator; Required for induction of expression of the formate dehydrogenase H and hydrogenase-3 structural genes","uniprotId":"FHLA_ECOLI"},"6883387":{"id":6883387,"externalId":"511145.b2732","name":"ygbA","hasFamily":true,"annotation":"predicted protein","uniprotId":"YGBA_ECOLI"},"6883388":{"id":6883388,"externalId":"511145.b2733","name":"mutS","hasFamily":true,"annotation":"methyl-directed mismatch repair protein; This protein is involved in the repair of mismatches in DNA. It is possible that it carries out the mismatch recognition step. This protein has a weak ATPase activity (By similarity)","uniprotId":"MUTS_ECOLI"},"6883389":{"id":6883389,"externalId":"511145.b2734","name":"o218","hasFamily":true,"annotation":"serine/threonine-specific protein phosphatase 2; Has been shown, in vitro, to act on Ser, Thr and Tyr- phosphorylated substrates","uniprotId":"PRP2_ECOLI"},"6883390":{"id":6883390,"externalId":"511145.b2735","name":"ygbI","hasFamily":true,"annotation":"putative transcriptional regulator, DeoR family","uniprotId":"YGBI_ECOLI"},"6883391":{"id":6883391,"externalId":"511145.b2736","name":"ygbJ","hasFamily":true,"annotation":"putative dehydrogenase, with NAD(P)-binding Rossmann-fold domain","uniprotId":"YGBJ_ECOLI"},"6883392":{"id":6883392,"externalId":"511145.b2737","name":"ygbK","hasFamily":true,"annotation":"conserved protein","uniprotId":"YGBK_ECOLI"},"6883393":{"id":6883393,"externalId":"511145.b2738","name":"ygbL","hasFamily":true,"annotation":"putative class II aldolase","uniprotId":"YGBL_ECOLI"},"6883394":{"id":6883394,"externalId":"511145.b2739","name":"ygbM","hasFamily":true,"annotation":"conserved protein; Catalyzes the reversible isomerization between hydroxypyruvate and 2-hydroxy-3-oxopropanoate (also termed tartronate semialdehyde) (By similarity)","uniprotId":"YGBM_ECOLI"},"6883395":{"id":6883395,"externalId":"511145.b2740","name":"ygbN","hasFamily":true,"annotation":"putative transporter","uniprotId":"YGBN_ECOLI"},"6883396":{"id":6883396,"externalId":"511145.b2741","name":"KatF","hasFamily":true,"annotation":"RNA polymerase, sigma S (sigma 38) factor; Sigma transcription factor controlling a regulon of genes required for protection against external stresses. Possibly, it affects the expression of katE (catalase HPII), xthA (endonuclease III), the appABC operon and other genes by modifying the affinity of RNA polymerase for the promoters","uniprotId":"RPOS_ECOLI"},"6883397":{"id":6883397,"externalId":"511145.b2742","name":"nlpD","hasFamily":true,"annotation":"activator of AmiC murein hydrolase activity, lipoprotein; Activator of the cell wall hydrolase AmiC. Required for septal murein cleavage and daughter cell separation during cell division","uniprotId":"NLPD_ECOLI"},"6883398":{"id":6883398,"externalId":"511145.b2743","name":"pcm","hasFamily":true,"annotation":"L-isoaspartate protein carboxylmethyltransferase type II; Catalyzes the methyl esterification of L-isoaspartyl residues in peptides and proteins that result from spontaneous decomposition of normal L-aspartyl and L-asparaginyl residues. It plays a role in the repair and/or degradation of damaged proteins. This enzyme does not act on D-aspartyl residues","uniprotId":"PIMT_ECOLI"},"6883399":{"id":6883399,"externalId":"511145.b2744","name":"surE","hasFamily":true,"annotation":"broad specificity 5'(3')-nucleotidase and polyphosphatase; Nucleotidase with a broad substrate specificity as it can dephosphorylate various ribo- and deoxyribonucleoside 5'- monophosphates and ribonucleoside 3'-monophosphates with highest affinity to 3'-AMP. Also hydrolyzes polyphosphate (exopolyphosphatase activity) with the preference for short-chain- length substrates (P20-25). Might be involved in the regulation of dNTP and NTP pools, and in the turnover of 3'-mononucleotides produced by numerous intracellular RNases (T1, T2, and F) during the degradation of various RNAs. Also pla [...] ","uniprotId":"SURE_ECOLI"},"6883400":{"id":6883400,"externalId":"511145.b2745","name":"truD","hasFamily":true,"annotation":"tRNA(Glu) U13 pseudouridine synthase; Responsible for synthesis of pseudouridine from uracil- 13 in transfer RNAs (By similarity)","uniprotId":"TRUD_ECOLI"},"6883401":{"id":6883401,"externalId":"511145.b2746","name":"ispF","hasFamily":true,"annotation":"2C-methyl-D-erythritol 2,4-cyclodiphosphate synthase; Involved in the biosynthesis of isopentenyl diphosphate (IPP) and dimethylallyl diphosphate (DMAPP), two major building blocks of isoprenoid compounds. Catalyzes the conversion of 4- diphosphocytidyl-2-C-methyl-D-erythritol 2-phosphate (CDP-ME2P) to 2-C-methyl-D-erythritol 2,4-cyclodiphosphate (ME-CPP) with a corresponding release of cytidine 5-monophosphate (CMP). Also converts 4-diphosphocytidyl-2-C-methyl-D-erythritol into 2-C- methyl-D-erythritol 3,4-cyclophosphate and CMP","uniprotId":"ISPF_ECOLI"},"6883402":{"id":6883402,"externalId":"511145.b2747","name":"ispD","hasFamily":true,"annotation":"4-diphosphocytidyl-2C-methyl-D-erythritol synthase; Catalyzes the formation of 4-diphosphocytidyl-2-C- methyl-D-erythritol from CTP and 2-C-methyl-D-erythritol 4- phosphate (MEP) (By similarity)","uniprotId":"ISPD_ECOLI"},"6883403":{"id":6883403,"externalId":"511145.b2748","name":"ftsB","hasFamily":true,"annotation":"cell division protein; Essential cell division protein. May link together the upstream cell division proteins, which are predominantly cytoplasmic, with the downstream cell division proteins, which are predominantly periplasmic (By similarity)","uniprotId":"FTSB_ECOLI"},"6883404":{"id":6883404,"externalId":"511145.b2749","name":"ygbE","hasFamily":true,"annotation":"conserved inner membrane protein","uniprotId":"YGBE_ECOLI"},"6883405":{"id":6883405,"externalId":"511145.b2750","name":"cysC","hasFamily":true,"annotation":"adenosine 5'-phosphosulfate kinase; Catalyzes the synthesis of activated sulfate (By similarity)","uniprotId":"CYSC_ECOLI"},"6883406":{"id":6883406,"externalId":"511145.b2751","name":"cysN","hasFamily":true,"annotation":"sulfate adenylyltransferase, subunit 1; May be the GTPase, regulating ATP sulfurylase activity (By similarity)","uniprotId":"CYSN_ECOLI"},"6883407":{"id":6883407,"externalId":"511145.b2752","name":"cysD","hasFamily":true,"annotation":"sulfate adenylyltransferase, subunit 2","uniprotId":"CYSD_ECOLI"},"6883408":{"id":6883408,"externalId":"511145.b2753","name":"iap","hasFamily":true,"annotation":"aminopeptidase in alkaline phosphatase isozyme conversion; This protein, presumably an aminopeptidase, mediates the conversion of E.coli alkaline phosphatase isozyme 1, to isozymes 2 and 3 by removing, one by one, the two N-terminal arginine residues","uniprotId":"IAP_ECOLI"},"6883409":{"id":6883409,"externalId":"511145.b2754","name":"ygbF","hasFamily":true,"annotation":"probable ssRNA endonuclease, CRISP-associated protein; CRISPR (clustered regularly interspaced short palindromic repeat), is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain sequences complementary to antecedent mobile elements and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA). Functions as a ssRNA-specific endoribonuclease (By similarity)","uniprotId":"CAS2_ECOLI"},"6883410":{"id":6883410,"externalId":"511145.b2755","name":"ygbT","hasFamily":true,"annotation":"CRISP-associated protein Cas1, DNA endonuclease; DNA repair protein; CRISPR (clustered regularly interspaced short palindromic repeat), is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain sequences complementary to antecedent mobile elements and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA). Cas1 is thought to be involved in CRISPR adaptation, the first stage of CRISPR immunity. Might be involved in the addition/remo [...] ","uniprotId":"CAS1_ECOLI"},"6883411":{"id":6883411,"externalId":"511145.b2756","name":"casE","hasFamily":true,"annotation":"CRISPR RNA precursor cleavage enzyme; CRISPR (clustered regularly interspaced short palindromic repeat), is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain sequences complementary to antecedent mobile elements and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA)","uniprotId":"CAS6_ECOLI"},"6883412":{"id":6883412,"externalId":"511145.b2757","name":"casD","hasFamily":true,"annotation":"CRISP RNA (crRNA) containing Cascade antiviral complex protein; CRISPR (clustered regularly interspaced short palindromic repeat), is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain sequences complementary to antecedent mobile elements and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA)","uniprotId":"CAS5_ECOLI"},"6883413":{"id":6883413,"externalId":"511145.b2758","name":"casC","hasFamily":true,"annotation":"CRISP RNA (crRNA) containing Cascade antiviral complex protein; CRISPR (clustered regularly interspaced short palindromic repeat), is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain sequences complementary to antecedent mobile elements and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA)","uniprotId":"CASC_ECOLI"},"6883414":{"id":6883414,"externalId":"511145.b2759","name":"casB","hasFamily":false,"annotation":"CRISP RNA (crRNA) containing Cascade antiviral complex protein; CRISPR (clustered regularly interspaced short palindromic repeat), is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain sequences complementary to antecedent mobile elements and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA)","uniprotId":"CSE2_ECOLI"},"6883415":{"id":6883415,"externalId":"511145.b2760","name":"casA","hasFamily":true,"annotation":"CRISP RNA (crRNA) containing Cascade antiviral complex protein; CRISPR (clustered regularly interspaced short palindromic repeat), is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain sequences complementary to antecedent mobile elements and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA)","uniprotId":"CSE1_ECOLI"},"6883416":{"id":6883416,"externalId":"511145.b2761","name":"ygcB","hasFamily":true,"annotation":"Cas3 predicted helicase needed for Cascade anti-viral activity; CRISPR (clustered regularly interspaced short palindromic repeat), is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain sequences complementary to antecedent mobile elements and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA). Cas3 plus Cascade participate in CRISPR interference, the third stage of CRISPR immunity","uniprotId":"CAS3_ECOLI"},"6883417":{"id":6883417,"externalId":"511145.b2762","name":"cysH","hasFamily":true,"annotation":"3'-phosphoadenosine 5'-phosphosulfate reductase; Reduction of activated sulfate into sulfite (By similarity)","uniprotId":"CYSH_ECOLI"},"6883418":{"id":6883418,"externalId":"511145.b2763","name":"cysI","hasFamily":true,"annotation":"sulfite reductase, beta subunit, NAD(P)-binding, heme-binding; Component of the sulfite reductase complex that catalyzes the 6-electron reduction of sulfite to sulfide. This is one of several activities required for the biosynthesis of L- cysteine from sulfate (By similarity)","uniprotId":"CYSI_ECOLI"},"6883419":{"id":6883419,"externalId":"511145.b2764","name":"cysJ","hasFamily":true,"annotation":"sulfite reductase, alpha subunit, flavoprotein; Component of the sulfite reductase complex that catalyzes the 6-electron reduction of sulfite to sulfide. This is one of several activities required for the biosynthesis of L- cysteine from sulfate. The flavoprotein component catalyzes the electron flow from NADPH -> FAD -> FMN to the hemoprotein component (By similarity)","uniprotId":"CYSJ_ECOLI"},"6883420":{"id":6883420,"externalId":"511145.b2765","name":"queD","hasFamily":true,"annotation":"6-pyruvoyl tetrahydrobiopterin synthase (PTPS); Catalyzes the conversion of 7,8-dihydroneopterin triphosphate (H2NTP) to 6-carboxy-5,6,7,8-tetrahydropterin (CPH4) and acetaldehyde. Can also convert 6-pyruvoyltetrahydropterin (PPH4) and sepiapterin to CPH4; these 2 compounds are probably intermediates in the reaction from H2NTP","uniprotId":"QUED_ECOLI"},"6883421":{"id":6883421,"externalId":"511145.b2766","name":"ygcN","hasFamily":false,"annotation":"putative oxidoreductase with FAD/NAD(P)-binding domain; Probably accepts electrons from YgcQ/YgcR and reduces a quinone","uniprotId":"YGCN_ECOLI"},"6883422":{"id":6883422,"externalId":"511145.b2767","name":"ygcO","hasFamily":false,"annotation":"putative 4Fe-4S cluster-containing protein; Could be a 3Fe-4S cluster-containing protein. Probably participates in a redox process with YgcN, YgcQ and YgcR","uniprotId":"YGCO_ECOLI"},"6883423":{"id":6883423,"externalId":"511145.b2768","name":"ygcP","hasFamily":true,"annotation":"putative anti-terminator regulatory protein","uniprotId":"YGCP_ECOLI"},"6883424":{"id":6883424,"externalId":"511145.b2769","name":"ygcQ","hasFamily":false,"annotation":"putative flavoprotein; May play a role in a redox process","uniprotId":"YGCQ_ECOLI"},"6883425":{"id":6883425,"externalId":"511145.b2770","name":"ygcR","hasFamily":true,"annotation":"putative flavoprotein; May play a role in a redox process","uniprotId":"YGCR_ECOLI"},"6883426":{"id":6883426,"externalId":"511145.b2771","name":"ygcS","hasFamily":true,"annotation":"putative transporter","uniprotId":"YGCS_ECOLI"},"6883427":{"id":6883427,"externalId":"511145.b2774","name":"ygcW","hasFamily":true,"annotation":"putative dehydrogenase","uniprotId":"YGCW_ECOLI"},"6883428":{"id":6883428,"externalId":"511145.b2775","name":"yqcE","hasFamily":false,"annotation":"inner membrane protein, predicted transporter","uniprotId":"YQCE_ECOLI"},"6883429":{"id":6883429,"externalId":"511145.b2776","name":"ygcE","hasFamily":true,"annotation":"putative kinase","uniprotId":"YGCE_ECOLI"},"6883430":{"id":6883430,"externalId":"511145.b2777","name":"queE","hasFamily":true,"annotation":"7-carboxy-7-deazaguanine synthase; queosine biosynthesis; Catalyzes the conversion of 6-carboxy-5,6,7,8- tetrahydropterin (CPH4) to 7-carboxy-7-deazaguanine (CDG) (By similarity)","uniprotId":"QUEE_ECOLI"},"6883431":{"id":6883431,"externalId":"511145.b2778","name":"ygcG","hasFamily":true,"annotation":"predicted protein","uniprotId":"YGCG_ECOLI"},"6883432":{"id":6883432,"externalId":"511145.b2779","name":"eno","hasFamily":true,"annotation":"enolase; Catalyzes the reversible conversion of 2- phosphoglycerate into phosphoenolpyruvate. It is essential for the degradation of carbohydrates via glycolysis. It is also a component of the RNA degradosome, a multi-enzyme complex involved in RNA processing and messenger RNA degradation. Its interaction with RNase E is important for the turnover of mRNA, in particular on transcripts encoding enzymes of energy-generating metabolic routes. Its presence in the degradosome is required for the response to excess phosphosugar. May play a regulatory role in the degradation of specific RNAs, [...] ","uniprotId":"ENO_ECOLI"},"6883433":{"id":6883433,"externalId":"511145.b2780","name":"pyrG","hasFamily":true,"annotation":"CTP synthetase; Catalyzes the ATP-dependent amination of UTP to CTP with either L-glutamine or ammonia as the source of nitrogen (By similarity)","uniprotId":"PYRG_ECOLI"},"6883434":{"id":6883434,"externalId":"511145.b2781","name":"mazG","hasFamily":true,"annotation":"nucleoside triphosphate pyrophosphohydrolase; Involved in the regulation of bacterial cell survival under conditions of nutritional stress. Regulates the MazEF toxin- antitoxin (TA) module that mediates programmed cell death (PCD). This is achieved by lowering the cellular concentration of (p)ppGpp produced by RelA under amino acid starvation, thus protecting the cell from the toxicity of MazF. Reduction of (p)ppGpp can be achieved by direct degradation of (p)ppGpp or by degradation of NTPs, which are substrates for (p)ppGpp synthesis by RelA","uniprotId":"MAZG_ECOLI"},"6883435":{"id":6883435,"externalId":"511145.b2782","name":"chpA","hasFamily":true,"annotation":"mRNA interferase toxin, antitoxin is MazE; Toxic component of a toxin-antitoxin (TA) module. MazF is a sequence-specific mRNA endoribonuclease that inhibits protein synthesis and induces bacterial stasis. It is very stable, single- strand specific and cleavage is independent of the ribosome, although translation enhances cleavage for some mRNAs. Cleavage usually occurs between the first A and C of ACA sequences, yielding a 2',3'-cyclic phosphate and a free 5'-OH. The endoribonuclease activity (a toxin) is inhibited by the labile cognate antitoxin MazE. Toxicity results when the levels  [...] ","uniprotId":"MAZF_ECOLI"},"6883436":{"id":6883436,"externalId":"511145.b2783","name":"chpR","hasFamily":true,"annotation":"antitoxin of the ChpA-ChpR toxin-antitoxin system; Antitoxin component of a toxin-antitoxin (TA) module. Labile antitoxin that binds to the MazF mRNA interferase toxin and neutralizes its endoribonuclease activity. Is considered to be an 'addiction' molecule as the cell will die in its absence. The endoribonuclease activity (MazF, a toxin) is inhibited by the labile cognate antitoxin MazE. Toxicity results when the levels of MazE decrease in the cell, leading to mRNA degradation. This effect can be rescued by expression of MazE, but after 6 hours in rich medium the overexpression of Ma [...] ","uniprotId":"MAZE_ECOLI"},"6883437":{"id":6883437,"externalId":"511145.b2784","name":"relA","hasFamily":true,"annotation":"(p)ppGpp synthetase I/GTP pyrophosphokinase; In eubacteria ppGpp (guanosine 3'-diphosphate 5-' diphosphate) is a mediator of the stringent response that coordinates a variety of cellular activities in response to changes in nutritional abundance. This enzyme catalyzes the formation of pppGpp which is then hydrolyzed to form ppGpp","uniprotId":"RELA_ECOLI"},"6883438":{"id":6883438,"externalId":"511145.b2785","name":"rlmD","hasFamily":true,"annotation":"23S rRNA m(5)U1939 methyltransferase, SAM-dependent; Catalyzes the formation of 5-methyl-uridine at position 1939 (m5U1939) in 23S rRNA (By similarity)","uniprotId":"RLMD_ECOLI"},"6883439":{"id":6883439,"externalId":"511145.b2786","name":"barA","hasFamily":true,"annotation":"hybrid sensory histidine kinase, in two-component regulatory system with UvrY; Member of the two-component regulatory system UvrY/BarA involved in the regulation of carbon metabolism via the CsrA/CsrB regulatory system. Phosphorylates UvrY, probably via a four-step phosphorelay","uniprotId":"BARA_ECOLI"},"6883440":{"id":6883440,"externalId":"511145.b2787","name":"gudD","hasFamily":true,"annotation":"(D)-glucarate dehydratase 1; Catalyzes the dehydration of glucarate to 5-keto-4- deoxy-D-glucarate (5-kdGluc). Also acts on L-idarate","uniprotId":"GUDD_ECOLI"},"6883441":{"id":6883441,"externalId":"511145.b2788","name":"gudX","hasFamily":true,"annotation":"glucarate dehydratase-related protein, substrate unknown; Does not seem to have an in-vivo activity on glucarate or idarate. Its real substrate is unknown","uniprotId":"GUDX_ECOLI"},"6883442":{"id":6883442,"externalId":"511145.b2789","name":"gudP","hasFamily":true,"annotation":"putative D-glucarate transporter; Uptake of D-glucarate","uniprotId":"GUDP_ECOLI"},"6883443":{"id":6883443,"externalId":"511145.b2790","name":"yqcA","hasFamily":true,"annotation":"putative flavoprotein; Probable electron transporter","uniprotId":"YQCA_ECOLI"},"6883444":{"id":6883444,"externalId":"511145.b2791","name":"truC","hasFamily":true,"annotation":"tRNA U65 pseudouridine synthase; Responsible for synthesis of pseudouridine from uracil- 65 in transfer RNAs","uniprotId":"TRUC_ECOLI"},"6883445":{"id":6883445,"externalId":"511145.b2792","name":"yqcC","hasFamily":true,"annotation":"conserved protein","uniprotId":"YQCC_ECOLI"},"6883446":{"id":6883446,"externalId":"511145.b2793","name":"syd","hasFamily":true,"annotation":"secY-interacting protein; Interacts with the SecY protein in vivo. May bind preferentially to an uncomplexed state of SecY, thus functioning either as a chelating agent for excess SecY in the cell or as a regulatory factor that negatively controls the translocase function (By similarity)","uniprotId":"SYDP_ECOLI"},"6883447":{"id":6883447,"externalId":"511145.b2794","name":"queF","hasFamily":true,"annotation":"7-cyano-7-deazaguanine reductase (NADPH-dependent); Catalyzes the NADPH-dependent reduction of 7-cyano-7- deazaguanine (preQ0) to 7-aminomethyl-7-deazaguanine (preQ1) (By similarity)","uniprotId":"QUEF_ECOLI"},"6883448":{"id":6883448,"externalId":"511145.b2795","name":"ygdH","hasFamily":true,"annotation":"conserved protein, UPF0717 family","uniprotId":"YGDH_ECOLI"},"6883449":{"id":6883449,"externalId":"511145.b2796","name":"sdaC","hasFamily":true,"annotation":"putative serine transporter; Involved in the import of serine into the cell. May be required for phage C1 adsorption by interacting with DrcB. May also be involved in ampicillin sensitivity","uniprotId":"SDAC_ECOLI"},"6883450":{"id":6883450,"externalId":"511145.b2797","name":"sdaB","hasFamily":true,"annotation":"L-serine deaminase II; Deaminates also threonine, particularly when it is present in high concentration","uniprotId":"SDHM_ECOLI"},"6883451":{"id":6883451,"externalId":"511145.b2798","name":"xni","hasFamily":true,"annotation":"Ssb-binding protein, misidentified as ExoIX; May have a role in processing of nucleic acids. It does not seem to be an exonuclease, as it can only bind 1 divalent metal cation, not the 2 that are thought to be necessary for exonuclease activity","uniprotId":"XNI_ECOLI"},"6883452":{"id":6883452,"externalId":"511145.b2799","name":"fucO","hasFamily":true,"annotation":"L-1,2-propanediol oxidoreductase","uniprotId":"FUCO_ECOLI"},"6883453":{"id":6883453,"externalId":"511145.b2800","name":"fucA","hasFamily":true,"annotation":"L-fuculose-1-phosphate aldolase","uniprotId":"FUCA_ECOLI"},"6883454":{"id":6883454,"externalId":"511145.b2801","name":"fucP","hasFamily":true,"annotation":"L-fucose transporter; Mediates the uptake of L-fucose across the boundary membrane with the concomitant transport of protons into the cell (symport system). Can also transport L-galactose and D-arabinose, but at reduced rates compared with L-fucose. Is not able to transport L-rhamnose and L-arabinose","uniprotId":"FUCP_ECOLI"},"6883455":{"id":6883455,"externalId":"511145.b2802","name":"fucI","hasFamily":true,"annotation":"L-fucose isomerase; Converts the aldose L-fucose into the corresponding ketose L-fuculose. Is also able to convert D-arabinose into D- ribulose, but this isomerase has a higher affinity for fucose and fuculose than for arabinose and ribulose, respectively","uniprotId":"FUCI_ECOLI"},"6883456":{"id":6883456,"externalId":"511145.b2803","name":"fucK","hasFamily":true,"annotation":"L-fuculokinase","uniprotId":"FUCK_ECOLI"},"6883457":{"id":6883457,"externalId":"511145.b2804","name":"fucU","hasFamily":true,"annotation":"L-fucose mutarotase; Involved in the anomeric conversion of L-fucose. It also exhibits a pyranase activity for D-ribose","uniprotId":"FUCM_ECOLI"},"6883458":{"id":6883458,"externalId":"511145.b2805","name":"fucR","hasFamily":true,"annotation":"DNA-binding transcriptional activator; Transcriptional activator of the fuc operon","uniprotId":"FUCR_ECOLI"},"6883459":{"id":6883459,"externalId":"511145.b2806","name":"rlmM","hasFamily":true,"annotation":"23S rRNA C2498 2'-O-ribose methyltransferase, SAM-dependent; Catalyzes the 2'-O-methylation at nucleotide C2498 in 23S rRNA. Modifies C2498 in naked 23S rRNA, but not in assembled 50S subunits or ribosomes","uniprotId":"RLMM_ECOLI"},"6883460":{"id":6883460,"externalId":"511145.b2807","name":"ygdD","hasFamily":true,"annotation":"inner membrane protein, UPF0382 family","uniprotId":"YGDD_ECOLI"},"6883461":{"id":6883461,"externalId":"511145.b2808","name":"gcvA","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; Regulatory protein for the glycine cleavage system operon (gcv). Mediates activation of gcv by glycine and repression by purines. GcvA is negatively autoregulated. Binds to three sites upstream of the gcv promoter","uniprotId":"GCVA_ECOLI"},"6883462":{"id":6883462,"externalId":"511145.b2809","name":"ygdI","hasFamily":true,"annotation":"novel lipoprotein","uniprotId":"YGDI_ECOLI"},"6883463":{"id":6883463,"externalId":"511145.b2810","name":"csdA","hasFamily":true,"annotation":"cysteine sulfinate desulfinase; Catalyzes the removal of elemental sulfur and selenium atoms from L-cysteine, L-cystine, L-selenocysteine, and L- selenocystine to produce L-alanine. L-cysteine sulfinic acid is the best substrate. Functions as a selenium delivery protein in the pathway for the biosynthesis of selenophosphate","uniprotId":"CSDA_ECOLI"},"6883464":{"id":6883464,"externalId":"511145.b2811","name":"csdE","hasFamily":true,"annotation":"CsdA-binding activator; Fe-S protein","uniprotId":"YGDK_ECOLI"},"6883465":{"id":6883465,"externalId":"511145.b2812","name":"ygdL","hasFamily":true,"annotation":"sulfur acceptor for CsdA","uniprotId":"YGDL_ECOLI"},"6883466":{"id":6883466,"externalId":"511145.b2813","name":"mltA","hasFamily":true,"annotation":"membrane-bound lytic murein transglycosylase A; Murein-degrading enzyme. May play a role in recycling of muropeptides during cell elongation and/or cell division. Degrades murein glycan strands and insoluble, high-molecular weight murein sacculi","uniprotId":"MLTA_ECOLI"},"6883467":{"id":6883467,"externalId":"511145.b2817","name":"amiC","hasFamily":true,"annotation":"N-acetylmuramoyl-L-alanine amidase; Cell-wall hydrolase involved in septum cleavage during cell division. Can also act as powerful autolysin in the presence of murein synthesis inhibitors","uniprotId":"AMIC_ECOLI"},"6883468":{"id":6883468,"externalId":"511145.b2818","name":"argA","hasFamily":true,"annotation":"fused acetylglutamate kinase homolog (inactive)/amino acid N-acetyltransferase","uniprotId":"ARGA_ECOLI"},"6883469":{"id":6883469,"externalId":"511145.b2819","name":"recD","hasFamily":true,"annotation":"exonuclease V (RecBCD complex), alpha chain; Exhibits several catalytic activities, including ATP- dependent exonuclease, ATP-stimulated endonuclease, ATP-dependent unwinding and DNA-dependent ATPase activities. Strand cleavage occurs 5' to 3' during the unwinding of duplex DNA at CHI sequences, which locally stimulate recombination","uniprotId":"EX5A_ECOLI"},"6883470":{"id":6883470,"externalId":"511145.b2820","name":"recB","hasFamily":true,"annotation":"exonuclease V (RecBCD complex), beta subunit; Required for efficient DNA repair; it catalyzes the unwinding of double-stranded DNA and the cleavage of single- stranded DNA and it stimulates local genetic recombination. All of these activities require concomitant hydrolysis of ATP","uniprotId":"EX5B_ECOLI"},"6883471":{"id":6883471,"externalId":"511145.b2821","name":"ptr","hasFamily":true,"annotation":"protease III; Endopeptidase that degrades small peptides of less than 7 kDa, such as glucagon and insulin","uniprotId":"PTRA_ECOLI"},"6883472":{"id":6883472,"externalId":"511145.b2822","name":"recC","hasFamily":true,"annotation":"exonuclease V (RecBCD complex), gamma chain; Exhibits a wide variety of catalytic activities including ATP-dependent exonuclease, ATP-stimulated endonuclease, ATP-dependent helicase and DNA-dependent ATPase activities","uniprotId":"EX5C_ECOLI"},"6883473":{"id":6883473,"externalId":"511145.b2823","name":"ppdC","hasFamily":true,"annotation":"predicted protein; Not yet known","uniprotId":"PPDC_ECOLI"},"6883474":{"id":6883474,"externalId":"511145.b2824","name":"ygdB","hasFamily":true,"annotation":"conserved protein, DUF2509 family","uniprotId":"YGDB_ECOLI"},"6883475":{"id":6883475,"externalId":"511145.b2825","name":"ppdB","hasFamily":true,"annotation":"conserved protein; Not yet known","uniprotId":"PPDB_ECOLI"},"6883476":{"id":6883476,"externalId":"511145.b2826","name":"ppdA","hasFamily":true,"annotation":"conserved protein; Not yet known","uniprotId":"PPDA_ECOLI"},"6883477":{"id":6883477,"externalId":"511145.b2827","name":"thyA","hasFamily":true,"annotation":"thymidylate synthetase; Provides the sole de novo source of dTMP for DNA biosynthesis. This protein also binds to its mRNA thus repressing its own translation","uniprotId":"TYSY_ECOLI"},"6883478":{"id":6883478,"externalId":"511145.b2828","name":"lgt","hasFamily":true,"annotation":"phosphatidylglycerol-prolipoprotein diacylglyceryl transferase; Transfers the N-acyl diglyceride group on what will become the N-terminal cysteine of membrane lipoproteins (By similarity)","uniprotId":"LGT_ECOLI"},"6883479":{"id":6883479,"externalId":"511145.b2829","name":"ptsP","hasFamily":true,"annotation":"fused PTS enzyme: PEP-protein phosphotransferase (enzyme I)/GAF domain containing protein; Component of the phosphoenolpyruvate-dependent nitrogen- metabolic phosphotransferase system (nitrogen-metabolic PTS), that seems to be involved in regulating nitrogen metabolism. Enzyme I- Ntr transfers the phosphoryl group from phosphoenolpyruvate (PEP) to the phosphoryl carrier protein (NPr). Could function in the transcriptional regulation of sigma-54 dependent operons in conjunction with the NPr (PtsO) and EIIA-Ntr (PtsN) proteins","uniprotId":"PT1P_ECOLI"},"6883480":{"id":6883480,"externalId":"511145.b2830","name":"nudH","hasFamily":true,"annotation":"RNA pyrophosphohydrolase; Master regulator of 5'-dependent mRNA decay. Accelerates the degradation of transcripts by removing pyrophosphate from the 5'-end of triphosphorylated RNA, leading to a more labile monophosphorylated state that can stimulate subsequent ribonuclease cleavage. Preferentially hydrolyzes diadenosine penta-phosphate with ATP as one of the reaction products. Also able to hydrolyze diadenosine hexa- and tetra-phosphate. Has no activity on diadenosine tri-phosphate, ADP-ribose, NADH and UDP- glucose. In the meningitis causing strain E.coli K1, has been shown to play a [...] ","uniprotId":"RPPH_ECOLI"},"6883481":{"id":6883481,"externalId":"511145.b2831","name":"mutH","hasFamily":true,"annotation":"methyl-directed mismatch repair protein; Sequence-specific endonuclease that cleaves unmethylated GATC sequences. It is involved in DNA mismatch repair (By similarity)","uniprotId":"MUTH_ECOLI"},"6883482":{"id":6883482,"externalId":"511145.b2832","name":"ygdQ","hasFamily":true,"annotation":"inner membrane protein, UPF0053 family","uniprotId":"YGDQ_ECOLI"},"6883483":{"id":6883483,"externalId":"511145.b2833","name":"ygdR","hasFamily":true,"annotation":"novel lipoprotein","uniprotId":"YGDR_ECOLI"},"6883484":{"id":6883484,"externalId":"511145.b2834","name":"tas","hasFamily":true,"annotation":"putative oxidoreductase, NADP(H)-dependent aldo-keto reductase","uniprotId":"TAS_ECOLI"},"6883485":{"id":6883485,"externalId":"511145.b2835","name":"lplT","hasFamily":true,"annotation":"lysophospholipid transporter; Catalyzes the facilitated diffusion of 2-acyl-glycero-3- phosphoethanolamine (2-acyl-GPE) into the cell (By similarity)","uniprotId":"LPLT_ECOLI"},"6883486":{"id":6883486,"externalId":"511145.b2836","name":"aas","hasFamily":true,"annotation":"fused 2-acylglycerophospho-ethanolamine acyl transferase/acyl-acyl carrier protein synthetase; Plays a role in lysophospholipid acylation. Transfers fatty acids to the 1-position via an enzyme-bound acyl-ACP intermediate in the presence of ATP and magnesium. Its physiological function is to regenerate phosphatidylethanolamine from 2-acyl-glycero-3-phosphoethanolamine (2-acyl-GPE) formed by transacylation reactions or degradation by phospholipase A1 (By similarity)","uniprotId":"AAS_ECOLI"},"6883487":{"id":6883487,"externalId":"511145.b2837","name":"galR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor of the galactose operon. Binds galactose as an inducer","uniprotId":"GALR_ECOLI"},"6883488":{"id":6883488,"externalId":"511145.b2838","name":"lysA","hasFamily":true,"annotation":"diaminopimelate decarboxylase, PLP-binding; Specifically catalyzes the decarboxylation of meso- diaminopimelate (meso-DAP) to L-lysine. Is not active against the DD- or LL-isomers of diaminopimelate","uniprotId":"DCDA_ECOLI"},"6883489":{"id":6883489,"externalId":"511145.b2839","name":"lysR","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; This protein activates the transcription of the lysA gene encoding diaminopimelate decarboxylase. LysR is also a negative regulator of its own expression","uniprotId":"LYSR_ECOLI"},"6883490":{"id":6883490,"externalId":"511145.b2840","name":"ygeA","hasFamily":true,"annotation":"conserved protein, Asp/Glu_racemase family","uniprotId":"YGEA_ECOLI"},"6883491":{"id":6883491,"externalId":"511145.b2841","name":"araE","hasFamily":true,"annotation":"arabinose transporter; Uptake of arabinose across the boundary membrane with the concomitant transport of protons into the cell (symport system)","uniprotId":"ARAE_ECOLI"},"6883492":{"id":6883492,"externalId":"511145.b2842","name":"kduD","hasFamily":true,"annotation":"2-deoxy-D-gluconate 3-dehydrogenase; Catalyzes the reduction of 2,5-diketo-3-deoxygluconate (DKII or 4,6-dihydroxy-2,5-dioxohexanoate) into 2-keto-3- deoxygluconate (KDG or 2-dehydro-3-deoxygluconate) with a concomitant oxidation of NADH (By similarity)","uniprotId":"KDUD_ECOLI"},"6883493":{"id":6883493,"externalId":"511145.b2843","name":"kduI","hasFamily":true,"annotation":"4-deoxy-L-threo-5-hexosulose-uronate ketol-isomerase; Catalyzes the isomerization of 5-dehydro-4-deoxy-D- glucuronate to 3-deoxy-D-glycero-2,5-hexodiulosonate (By similarity)","uniprotId":"KDUI_ECOLI"},"6883494":{"id":6883494,"externalId":"511145.b2844","name":"yqeF","hasFamily":true,"annotation":"putative acyltransferase","uniprotId":"YQEF_ECOLI"},"6883495":{"id":6883495,"externalId":"511145.b2845","name":"yqeG","hasFamily":true,"annotation":"putative transporter","uniprotId":"YQEG_ECOLI"},"6883496":{"id":6883496,"externalId":"511145.b2846","name":"yqeH","hasFamily":true,"annotation":"conserved protein with bipartite regulator domain","uniprotId":"YQEH_ECOLI"},"6883497":{"id":6883497,"externalId":"511145.b2847","name":"yqeI","hasFamily":true,"annotation":"putative transcriptional regulator","uniprotId":"YQEI_ECOLI"},"6883498":{"id":6883498,"externalId":"511145.b2848","name":"yqeJ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YQEJ_ECOLI"},"6883499":{"id":6883499,"externalId":"511145.b2849","name":"yqeK","hasFamily":false,"annotation":"predicted protein","uniprotId":"YQEK_ECOLI"},"6883500":{"id":6883500,"externalId":"511145.b2851","name":"ygeG","hasFamily":false,"annotation":"putative chaperone","uniprotId":"YGEG_ECOLI"},"6883501":{"id":6883501,"externalId":"511145.b2852","name":"ygeH","hasFamily":true,"annotation":"predictedtranscriptional regulator","uniprotId":"YGEH_ECOLI"},"6883502":{"id":6883502,"externalId":"511145.b2853","name":"ygeI","hasFamily":false,"annotation":"predicted protein","uniprotId":"YGEI_ECOLI"},"6883503":{"id":6883503,"externalId":"511145.b2860","name":"b2860","hasFamily":true,"annotation":"IS2 transposase TnpB"},"6883504":{"id":6883504,"externalId":"511145.b2861","name":"b2861","hasFamily":true,"annotation":"IS2 repressor TnpA"},"6883505":{"id":6883505,"externalId":"511145.b2865","name":"ygeR","hasFamily":true,"annotation":"novel lipoprotein, function unknown","uniprotId":"YGER_ECOLI"},"6883506":{"id":6883506,"externalId":"511145.b2866","name":"xdhA","hasFamily":true,"annotation":"xanthine dehydrogenase, molybdenum binding subunit; Presumed to be a dehydrogenase, but possibly an oxidase. Participates in limited purine salvage (requires aspartate) but does not support aerobic growth on purines as the sole carbon source (purine catabolism). Deletion results in increased adenine sensitivity, suggesting that this protein contributes to the conversion of adenine to guanine nucleotides during purine salvage","uniprotId":"XDHA_ECOLI"},"6883507":{"id":6883507,"externalId":"511145.b2867","name":"xdhB","hasFamily":true,"annotation":"xanthine dehydrogenase, FAD-binding subunit; Presumed to be a dehydrogenase, but possibly an oxidase. Participates in limited purine salvage (requires aspartate) but does not support aerobic growth on purines as the sole carbon source (purine catabolism)","uniprotId":"XDHB_ECOLI"},"6883508":{"id":6883508,"externalId":"511145.b2868","name":"xdhC","hasFamily":true,"annotation":"xanthine dehydrogenase, Fe-S binding subunit; Iron-sulfur subunit of the xanthine dehydrogenase complex","uniprotId":"XDHC_ECOLI"},"6883509":{"id":6883509,"externalId":"511145.b2869","name":"ygeV","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YGEV_ECOLI"},"6883510":{"id":6883510,"externalId":"511145.b2870","name":"ygeW","hasFamily":false,"annotation":"putative carbamoyltransferase","uniprotId":"YGEW_ECOLI"},"6883511":{"id":6883511,"externalId":"511145.b2871","name":"ygeX","hasFamily":true,"annotation":"2,3-diaminopropionate ammonia-lyase; Catalyzes the alpha,beta-elimination reaction of both L- and D-alpha,beta-diaminopropionate, the most suitable substrates to form pyruvate and ammonia. The L- and D-isomers of serine are also degraded, though slowly; it is the only serine dehydratase which can eliminate an amino group at the beta-carbon position (By similarity)","uniprotId":"DPAL_ECOLI"},"6883512":{"id":6883512,"externalId":"511145.b2872","name":"ygeY","hasFamily":true,"annotation":"putative peptidase","uniprotId":"YGEY_ECOLI"},"6883513":{"id":6883513,"externalId":"511145.b2873","name":"hyuA","hasFamily":true,"annotation":"D-stereospecific phenylhydantoinase; Catalyzes the stereospecific hydrolysis of the cyclic amide bond of D-hydantoin derivatives with an aromatic side chains at the 5'-position. Has no activity on dihydropyrimidines. The physiological function is unknown (By similarity)","uniprotId":"PHYDA_ECOLI"},"6883514":{"id":6883514,"externalId":"511145.b2874","name":"yqeA","hasFamily":true,"annotation":"putative amino acid kinase","uniprotId":"ARCL_ECOLI"},"6883515":{"id":6883515,"externalId":"511145.b2875","name":"yqeB","hasFamily":true,"annotation":"conserved protein with NAD(P)-binding Rossman fold","uniprotId":"YQEB_ECOLI"},"6883516":{"id":6883516,"externalId":"511145.b2876","name":"yqeC","hasFamily":false,"annotation":"conserved protein","uniprotId":"YQEC_ECOLI"},"6883517":{"id":6883517,"externalId":"511145.b2877","name":"mocA","hasFamily":false,"annotation":"CTP:molybdopterin cytidylyltransferase; Transfers a CMP moiety from CTP to Mo-molybdopterin (Mo- MPT) cofactor (Moco or molybdenum cofactor) to form Mo- molybdopterin cytosine dinucleotide (Mo-MCD) cofactor. Is specific for CTP; other nucleotides such as ATP and GTP can not be utilized. Is also able to convert MPT to MCD in the absence of molybdate, however, with only one catalytic turnover","uniprotId":"MOCA_ECOLI"},"6883518":{"id":6883518,"externalId":"511145.b2878","name":"ygfK","hasFamily":true,"annotation":"putative oxidoreductase, Fe-S subunit","uniprotId":"YGFK_ECOLI"},"6883519":{"id":6883519,"externalId":"511145.b2879","name":"ssnA","hasFamily":true,"annotation":"putative chlorohydrolase/aminohydrolase","uniprotId":"SSNA_ECOLI"},"6883520":{"id":6883520,"externalId":"511145.b2880","name":"ygfM","hasFamily":false,"annotation":"putative oxidoreductase","uniprotId":"YGFM_ECOLI"},"6883521":{"id":6883521,"externalId":"511145.b2881","name":"xdhD","hasFamily":true,"annotation":"probable hypoxanthine oxidase, molybdopterin-binding/Fe-S binding; Probably has no xanthine dehydrogenase activity; however deletion results in increased adenine sensitivity, suggesting that this protein contributes to the conversion of adenine to guanine nucleotides during purine salvage","uniprotId":"XDHD_ECOLI"},"6883522":{"id":6883522,"externalId":"511145.b2882","name":"xanQ","hasFamily":true,"annotation":"xanthine permease; Specific, proton motive force-dependent high-affinity transporter for xanthine","uniprotId":"XANQ_ECOLI"},"6883523":{"id":6883523,"externalId":"511145.b2883","name":"guaD","hasFamily":true,"annotation":"guanine deaminase; Catalyzes the hydrolytic deamination of guanine, producing xanthine and ammonia","uniprotId":"GUAD_ECOLI"},"6883524":{"id":6883524,"externalId":"511145.b2886","name":"ygfS","hasFamily":false,"annotation":"putative oxidoreductase, 4Fe-4S ferredoxin-type subunit","uniprotId":"YGFS_ECOLI"},"6883525":{"id":6883525,"externalId":"511145.b2887","name":"ygfT","hasFamily":true,"annotation":"fused predicted oxidoreductase: Fe-S subunit/nucleotide-binding subunit","uniprotId":"YGFT_ECOLI"},"6883526":{"id":6883526,"externalId":"511145.b2888","name":"uacT","hasFamily":true,"annotation":"uric acid permease; Proton-dependent high-capacity transporter for uric acid. Shows also a low capacity for transport of xanthine at 37 degrees Celsius but not at 25 degrees Celsius","uniprotId":"UACT_ECOLI"},"6883527":{"id":6883527,"externalId":"511145.b2889","name":"idi","hasFamily":true,"annotation":"isopentenyl diphosphate isomerase; Catalyzes the 1,3-allylic rearrangement of the homoallylic substrate isopentenyl (IPP) to its highly electrophilic allylic isomer, dimethylallyl diphosphate (DMAPP) (By similarity)","uniprotId":"IDI_ECOLI"},"6883528":{"id":6883528,"externalId":"511145.b2890","name":"lysS","hasFamily":true,"annotation":"lysine tRNA synthetase, constitutive","uniprotId":"SYK1_ECOLI"},"6883529":{"id":6883529,"externalId":"511145.b2891","name":"prfB","hasFamily":true,"annotation":"peptide chain release factor RF-2; Peptide chain release factor 2 directs the termination of translation in response to the peptide chain termination codons UGA and UAA (By similarity)","uniprotId":"RF2_ECOLI"},"6883530":{"id":6883530,"externalId":"511145.b2892","name":"recJ","hasFamily":true,"annotation":"ssDNA exonuclease, 5' --> 3'-specific; Single-stranded-DNA-specific exonuclease. Required for many types of recombinational events, although the stringency of the requirement for RecJ appears to vary with the type of recombinational event monitored and the other recombination gene products which are available","uniprotId":"RECJ_ECOLI"},"6883531":{"id":6883531,"externalId":"511145.b2893","name":"dsbC","hasFamily":true,"annotation":"protein disulfide isomerase II; Acts as a disulfide isomerase, interacting with incorrectly folded proteins to correct non-native disulfide bonds. DsbG and DsbC are part of a periplasmic reducing system that controls the level of cysteine sulfenylation, and provides reducing equivalents to rescue oxidatively damaged secreted proteins. Acts by transferring its disulfide bond to other proteins and is reduced in the process. DsbC is reoxidized by DsbD","uniprotId":"DSBC_ECOLI"},"6883532":{"id":6883532,"externalId":"511145.b2894","name":"xerD","hasFamily":true,"annotation":"site-specific tyrosine recombinase; Site-specific tyrosine recombinase, which acts by catalyzing the cutting and rejoining of the recombining DNA molecules. Binds cooperatively to specific DNA consensus sequences that are separated from XerC binding sites by a short central region, forming the heterotetrameric XerC-XerD complex that recombines DNA substrates. The complex is essential to convert dimers of the bacterial chromosome into monomers to permit their segregation at cell division. It also contributes to the segregational stability of plasmids at ColE1 xer (or cer) and pSC101 (or [...] ","uniprotId":"XERD_ECOLI"},"6883533":{"id":6883533,"externalId":"511145.b2895","name":"fldB","hasFamily":true,"annotation":"flavodoxin 2; Low-potential electron donor to a number of redox enzymes (By similarity)","uniprotId":"FLAW_ECOLI"},"6883534":{"id":6883534,"externalId":"511145.b2896","name":"cptA","hasFamily":true,"annotation":"predicted protein; Toxic component of a toxin-antitoxin (TA) module. Overexpression leads to growth arrest after 5 hours, with initial elongation of cells, followed by swelling. The toxic effects are abrogated by coexpression with antitoxin CptB. Interacts with cytoskeletal proteins FtsZ and MreB; inhibits FtsZ GTP-dependent polymerization as well as MreB ATP-dependent polymerization","uniprotId":"CPTA_ECOLI"},"6883535":{"id":6883535,"externalId":"511145.b2897","name":"cptB","hasFamily":true,"annotation":"conserved protein; Antitoxin component of a toxin-antitoxin (TA) module. When coexpressed with cognate toxin CptA, the antitoxin neutralizes CptA's toxicity. Has been suggested to be a transcriptional regulator","uniprotId":"CPTB_ECOLI"},"6883536":{"id":6883536,"externalId":"511145.b2898","name":"ygfZ","hasFamily":true,"annotation":"iron-sulfur cluster repair protein, plumbagin resistance; Folate-binding protein involved in regulating the level of ATP-DnaA and in the modification of some tRNAs. It is probably a key factor in regulatory networks that act via tRNA modification, such as initiation of chromosomal replication (By similarity)","uniprotId":"YGFZ_ECOLI"},"6883537":{"id":6883537,"externalId":"511145.b2899","name":"yqfA","hasFamily":true,"annotation":"inner membrane protein, hemolysin III family HylIII","uniprotId":"YQFA_ECOLI"},"6883538":{"id":6883538,"externalId":"511145.b2900","name":"yqfB","hasFamily":true,"annotation":"conserved protein, UPF0267 family","uniprotId":"YQFB_ECOLI"},"6883539":{"id":6883539,"externalId":"511145.b2901","name":"bglA","hasFamily":true,"annotation":"6-phospho-beta-glucosidase A; Catalyzes the hydrolysis of phosphorylated beta- glucosides into glucose-6-phosphate (G-6-P) and aglycone. It has a high affinity for phosphorylated aromatic beta-glucosides (p- nitrophenyl-beta-glucoside, phenyl beta-glucoside, arbutin), with the exception of phosphorylated salicin, and a low affinity for phosphorylated beta-methyl-glucoside. Apparently, it has only a very limited role in the utilization of external beta-glucosides","uniprotId":"BGLA_ECOLI"},"6883540":{"id":6883540,"externalId":"511145.b2902","name":"ygfF","hasFamily":true,"annotation":"putative NAD(P)-binding oxidoreductase with NAD(P)-binding Rossmann-fold domain","uniprotId":"YGFF_ECOLI"},"6883541":{"id":6883541,"externalId":"511145.b2903","name":"gcvP","hasFamily":true,"annotation":"glycine decarboxylase, PLP-dependent, subunit (protein P) of glycine cleavage complex; The glycine cleavage system catalyzes the degradation of glycine. The P protein binds the alpha-amino group of glycine through its pyridoxal phosphate cofactor; CO(2) is released and the remaining methylamine moiety is then transferred to the lipoamide cofactor of the H protein (By similarity)","uniprotId":"GCSP_ECOLI"},"6883542":{"id":6883542,"externalId":"511145.b2904","name":"gcvH","hasFamily":true,"annotation":"glycine cleavage complex lipoylprotein; The glycine cleavage system catalyzes the degradation of glycine. The H protein shuttles the methylamine group of glycine from the P protein to the T protein (By similarity)","uniprotId":"GCSH_ECOLI"},"6883543":{"id":6883543,"externalId":"511145.b2905","name":"gcvT","hasFamily":true,"annotation":"aminomethyltransferase, tetrahydrofolate-dependent, subunit (T protein) of glycine cleavage complex; The glycine cleavage system catalyzes the degradation of glycine (By similarity)","uniprotId":"GCST_ECOLI"},"6883544":{"id":6883544,"externalId":"511145.b2906","name":"visC","hasFamily":true,"annotation":"putative oxidoreductase with FAD/NAD(P)-binding domain; Not yet known","uniprotId":"VISC_ECOLI"},"6883545":{"id":6883545,"externalId":"511145.b2907","name":"ubiH","hasFamily":true,"annotation":"2-octaprenyl-6-methoxyphenol hydroxylase, FAD/NAD(P)-binding; Oxygenase that introduces the hydroxyl group at carbon four of 2-octaprenyl-6-methoxyphenol resulting in the formation of 2-octaprenyl-6-methoxy-1,4-benzoquinone","uniprotId":"UBIH_ECOLI"},"6883546":{"id":6883546,"externalId":"511145.b2908","name":"pepP","hasFamily":true,"annotation":"proline aminopeptidase P II","uniprotId":"AMPP_ECOLI"},"6883547":{"id":6883547,"externalId":"511145.b2909","name":"ygfB","hasFamily":true,"annotation":"conserved protein, UPF0149 family","uniprotId":"YGFB_ECOLI"},"6883548":{"id":6883548,"externalId":"511145.b2910","name":"ygfE","hasFamily":true,"annotation":"FtsZ stabilizer; Activator of cell division through the inhibition of FtsZ GTPase activity, therefore promoting FtsZ assembly into bundles of protofilaments necessary for the formation of the division Z ring. It is recruited early at mid-cell but it is not essential for cell division (By similarity)","uniprotId":"ZAPA_ECOLI"},"6883549":{"id":6883549,"externalId":"511145.b2912","name":"ygfA","hasFamily":true,"annotation":"conserved protein, 5-formyltetrahydrofolate cyclo-ligase family","uniprotId":"YGFA_ECOLI"},"6883550":{"id":6883550,"externalId":"511145.b2913","name":"serA","hasFamily":true,"annotation":"D-3-phosphoglycerate dehydrogenase","uniprotId":"SERA_ECOLI"},"6883551":{"id":6883551,"externalId":"511145.b2914","name":"rpiA","hasFamily":true,"annotation":"ribose 5-phosphate isomerase, constitutive","uniprotId":"RPIA_ECOLI"},"6883552":{"id":6883552,"externalId":"511145.b2916","name":"iciA","hasFamily":true,"annotation":"DNA-binding transcriptional activator, replication initiation inhibitor; Specific inhibitor of chromosomal initiation of replication in vitro. Binds the three 13-mers in the origin (oriC) to block initiation of replication (By similarity)","uniprotId":"ICIA_ECOLI"},"6883553":{"id":6883553,"externalId":"511145.b2917","name":"sbm","hasFamily":true,"annotation":"methylmalonyl-CoA mutase; Catalyzes the interconversion of succinyl-CoA and methylmalonyl-CoA. Could be part of a pathway that converts succinate to propionate","uniprotId":"SCPA_ECOLI"},"6883554":{"id":6883554,"externalId":"511145.b2918","name":"argK","hasFamily":true,"annotation":"membrane ATPase/protein kinase; Binds and cleaves GTP","uniprotId":"ARGK_ECOLI"},"6883555":{"id":6883555,"externalId":"511145.b2919","name":"scpB","hasFamily":true,"annotation":"methylmalonyl-CoA decarboxylase, biotin-independent; Catalyzes the decarboxylation of methylmalonyl-CoA to propionyl-CoA. Could be part of a pathway that converts succinate to propionate","uniprotId":"SCPB_ECOLI"},"6883556":{"id":6883556,"externalId":"511145.b2920","name":"scpC","hasFamily":true,"annotation":"propionyl-CoA:succinate-CoA transferase; Catalyzes the transfer of coenzyme A from propionyl-CoA to succinate. Could be part of a pathway that converts succinate to propionate","uniprotId":"SCPC_ECOLI"},"6883557":{"id":6883557,"externalId":"511145.b2921","name":"ygfI","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YGFI_ECOLI"},"6883558":{"id":6883558,"externalId":"511145.b2922","name":"yggE","hasFamily":true,"annotation":"oxidative stress defense protein","uniprotId":"YGGE_ECOLI"},"6883559":{"id":6883559,"externalId":"511145.b2923","name":"argO","hasFamily":true,"annotation":"arginine transporter; Involved in the export of arginine. Important to control the intracellular level of arginine and the correct balance between arginine and lysine. May also be involved in the export of canavanine (a plant-derived antimetabolite)","uniprotId":"ARGO_ECOLI"},"6883560":{"id":6883560,"externalId":"511145.b2924","name":"mscS","hasFamily":true,"annotation":"mechanosensitive channel protein, small conductance; Mechanosensitive channel that participates in the regulation of osmotic pressure changes within the cell, opening in response to stretch forces in the membrane lipid bilayer, without the need for other proteins. Forms an ion channel of 1.0 nanosiemens conductance with a slight preference for anions. The channel is sensitive to voltage; as the membrane is depolarized, less tension is required to open the channel and vice versa. The channel is characterized by short bursts of activity that last for a few seconds","uniprotId":"MSCS_ECOLI"},"6883561":{"id":6883561,"externalId":"511145.b2925","name":"fba","hasFamily":true,"annotation":"fructose-bisphosphate aldolase, class II; Catalyzes the aldol condensation of dihydroxyacetone phosphate (DHAP or glycerone-phosphate) with glyceraldehyde 3- phosphate (G3P) to form fructose 1,6-bisphosphate (FBP) in gluconeogenesis and the reverse reaction in glycolysis","uniprotId":"ALF_ECOLI"},"6883562":{"id":6883562,"externalId":"511145.b2926","name":"pgk","hasFamily":true,"annotation":"phosphoglycerate kinase","uniprotId":"PGK_ECOLI"},"6883563":{"id":6883563,"externalId":"511145.b2927","name":"epd","hasFamily":true,"annotation":"D-erythrose 4-phosphate dehydrogenase; Catalyzes the NAD-dependent conversion of D-erythrose 4- phosphate to 4-phosphoerythronate (By similarity)","uniprotId":"E4PD_ECOLI"},"6883564":{"id":6883564,"externalId":"511145.b2928","name":"yggC","hasFamily":true,"annotation":"conserved protein with nucleoside triphosphate hydrolase domain","uniprotId":"YGGC_ECOLI"},"6883565":{"id":6883565,"externalId":"511145.b2929","name":"yggD","hasFamily":false,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YGGD_ECOLI"},"6883566":{"id":6883566,"externalId":"511145.b2930","name":"yggF","hasFamily":false,"annotation":"fructose 1,6 bisphosphatase isozyme; Catalyzes the hydrolysis of fructose 1,6-bisphosphate to fructose 6-phosphate. Also displays a low activity toward glucose 1,6-bisphosphate, and no activity against ribulose 1,5- bisphosphate, fructose 2,6-bisphosphate, or fructose 1-phosphate","uniprotId":"GLPX2_ECOLI"},"6883567":{"id":6883567,"externalId":"511145.b2933","name":"cmtA","hasFamily":true,"annotation":"predicted fused mannitol-specific PTS enzymes: IIB component/IIC component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in mannitol transport (By similarity)","uniprotId":"PTMCB_ECOLI"},"6883568":{"id":6883568,"externalId":"511145.b2934","name":"cmtB","hasFamily":true,"annotation":"putative mannitol-specific enzyme IIA component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in mannitol transport (By similarity)","uniprotId":"PTMA_ECOLI"},"6883569":{"id":6883569,"externalId":"511145.b2935","name":"tktA","hasFamily":true,"annotation":"transketolase 1, thiamin-binding; Catalyzes the reversible transfer of a two-carbon ketol group from sedoheptulose-7-phosphate to glyceraldehyde-3- phosphate, producing xylulose-5-phosphate and ribose-5-phosphate. Catalyzes the transfer of a two-carbon ketol group from a ketose donor to an aldose acceptor, via a covalent intermediate with the cofactor thiamine pyrophosphate","uniprotId":"TKT1_ECOLI"},"6883570":{"id":6883570,"externalId":"511145.b2936","name":"loiP","hasFamily":true,"annotation":"heat shock protein binding to Era protein; predicted peptidase; Metalloprotease that cleaves substrates preferentially between Phe-Phe residues. Plays a role in response to some stress conditions. Seems to regulate the expression of speB","uniprotId":"LOIP_ECOLI"},"6883571":{"id":6883571,"externalId":"511145.b2937","name":"speB","hasFamily":true,"annotation":"agmatinase; Catalyzes the formation of putrescine from agmatine (By similarity)","uniprotId":"SPEB_ECOLI"},"6883572":{"id":6883572,"externalId":"511145.b2938","name":"speA","hasFamily":true,"annotation":"biosynthetic arginine decarboxylase, PLP-binding; Catalyzes the biosynthesis of agmatine from arginine (By similarity)","uniprotId":"SPEA_ECOLI"},"6883573":{"id":6883573,"externalId":"511145.b2939","name":"yqgB","hasFamily":true,"annotation":"predicted protein","uniprotId":"YQGB_ECOLI"},"6883574":{"id":6883574,"externalId":"511145.b2940","name":"yqgC","hasFamily":false,"annotation":"predicted protein","uniprotId":"YQGC_ECOLI"},"6883575":{"id":6883575,"externalId":"511145.b2942","name":"metK","hasFamily":true,"annotation":"S-adenosylmethionine synthetase; Catalyzes the formation of S-adenosylmethionine from methionine and ATP. The overall synthetic reaction is composed of two sequential steps, AdoMet formation and the subsequent tripolyphosphate hydrolysis which occurs prior to release of AdoMet from the enzyme. Is essential for growth","uniprotId":"METK_ECOLI"},"6883576":{"id":6883576,"externalId":"511145.b2943","name":"galP","hasFamily":true,"annotation":"D-galactose transporter; Uptake of galactose across the boundary membrane with the concomitant transport of protons into the cell (symport system)","uniprotId":"GALP_ECOLI"},"6883577":{"id":6883577,"externalId":"511145.b2944","name":"sprT","hasFamily":true,"annotation":"conserved protein; Involved in bolA gene expression at the stationary phase","uniprotId":"SPRT_ECOLI"},"6883578":{"id":6883578,"externalId":"511145.b2945","name":"endA","hasFamily":true,"annotation":"DNA-specific endonuclease I; Has double-strand break activity","uniprotId":"END1_ECOLI"},"6883579":{"id":6883579,"externalId":"511145.b2946","name":"rsmE","hasFamily":true,"annotation":"16S rRNA m(3)U1498 methyltransferase, SAM-dependent; Specifically methylates the N3 position of the uracil ring of uridine 1498 (m3U1498) in 16S rRNA. Acts on the fully assembled 30S ribosomal subunit (By similarity)","uniprotId":"RSME_ECOLI"},"6883580":{"id":6883580,"externalId":"511145.b2947","name":"gshB","hasFamily":true,"annotation":"glutathione synthetase","uniprotId":"GSHB_ECOLI"},"6883581":{"id":6883581,"externalId":"511145.b2948","name":"yqgE","hasFamily":true,"annotation":"predicted protein","uniprotId":"YQGE_ECOLI"},"6883582":{"id":6883582,"externalId":"511145.b2949","name":"yqgF","hasFamily":true,"annotation":"putative Holliday junction resolvase; Could be a nuclease that resolves Holliday junction intermediates in genetic recombination (By similarity)","uniprotId":"RUVX_ECOLI"},"6883583":{"id":6883583,"externalId":"511145.b2950","name":"yggR","hasFamily":true,"annotation":"putative pilus retraction ATPase","uniprotId":"YGGR_ECOLI"},"6883584":{"id":6883584,"externalId":"511145.b2951","name":"yggS","hasFamily":true,"annotation":"putative enzyme, binds pyridoxal 5'-phosphate","uniprotId":"YGGS_ECOLI"},"6883585":{"id":6883585,"externalId":"511145.b2952","name":"yggT","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YGGT_ECOLI"},"6883586":{"id":6883586,"externalId":"511145.b2953","name":"yggU","hasFamily":true,"annotation":"conserved protein, UPF0235 family","uniprotId":"YGGU_ECOLI"},"6883587":{"id":6883587,"externalId":"511145.b2954","name":"rdgB","hasFamily":true,"annotation":"dITP/XTP pyrophosphatase; Pyrophosphatase that hydrolyzes the non-canonical purine nucleotides deoxyinosine triphosphate (dITP) and xanthosine triphosphate (XTP) as well as 2'-deoxy-N-6-hydroxylaminopurine triposphate (dHAPTP) to their respective monophosphate derivatives. Probably excludes non-canonical purines from DNA precursor pool, thus preventing their incorporation into DNA and avoiding chromosomal lesions (By similarity)","uniprotId":"RDGB_ECOLI"},"6883588":{"id":6883588,"externalId":"511145.b2955","name":"yggW","hasFamily":true,"annotation":"putative oxidoreductase, HemN family","uniprotId":"YGGW_ECOLI"},"6883589":{"id":6883589,"externalId":"511145.b2956","name":"yggM","hasFamily":true,"annotation":"conserved protein","uniprotId":"YGGM_ECOLI"},"6883590":{"id":6883590,"externalId":"511145.b2957","name":"ansB","hasFamily":true,"annotation":"periplasmic L-asparaginase II","uniprotId":"ASPG2_ECOLI"},"6883591":{"id":6883591,"externalId":"511145.b2958","name":"ecfF","hasFamily":true,"annotation":"predicted protein","uniprotId":"YGGN_ECOLI"},"6883592":{"id":6883592,"externalId":"511145.b2959","name":"yggL","hasFamily":true,"annotation":"conserved protein, DUF469 family","uniprotId":"YGGL_ECOLI"},"6883593":{"id":6883593,"externalId":"511145.b2960","name":"trmB","hasFamily":true,"annotation":"tRNA m(7)G46 methyltransferase, SAM-dependent; Catalyzes the formation of N(7)-methylguanine at position 46 (m7G46) in tRNA (By similarity)","uniprotId":"TRMB_ECOLI"},"6883594":{"id":6883594,"externalId":"511145.b2961","name":"mutY","hasFamily":true,"annotation":"adenine DNA glycosylase; Adenine glycosylase active on G-A mispairs. MutY also corrects error-prone DNA synthesis past GO lesions which are due to the oxidatively damaged form of guanine: 7,8-dihydro-8- oxoguanine (8-oxo-dGTP)","uniprotId":"MUTY_ECOLI"},"6883595":{"id":6883595,"externalId":"511145.b2962","name":"yggX","hasFamily":true,"annotation":"protein that protects iron-sulfur proteins against oxidative damage; Could be a mediator in iron transactions between iron acquisition and iron-requiring processes, such as synthesis and/or repair of Fe-S clusters in biosynthetic enzymes. Necessary to maintain high levels of aconitase under oxidative stress","uniprotId":"FETP_ECOLI"},"6883596":{"id":6883596,"externalId":"511145.b2963","name":"mltC","hasFamily":true,"annotation":"membrane-bound lytic murein transglycosylase C; Murein-degrading enzyme. May play a role in recycling of muropeptides during cell elongation and/or cell division (By similarity)","uniprotId":"MLTC_ECOLI"},"6883597":{"id":6883597,"externalId":"511145.b2964","name":"nupG","hasFamily":true,"annotation":"nucleoside transporter; Transports nucleosides with a high affinity. Driven by a proton motive force","uniprotId":"NUPG_ECOLI"},"6883598":{"id":6883598,"externalId":"511145.b2965","name":"speC","hasFamily":true,"annotation":"ornithine decarboxylase, constitutive","uniprotId":"DCOR_ECOLI"},"6883599":{"id":6883599,"externalId":"511145.b2966","name":"yqgA","hasFamily":true,"annotation":"putative inner membrane protein, DUF554 family","uniprotId":"YQGA_ECOLI"},"6883600":{"id":6883600,"externalId":"511145.b2968","name":"yghD","hasFamily":true,"annotation":"putative secretion pathway M-type protein, membrane anchored; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (By similarity)","uniprotId":"YGHD_ECOLI"},"6883601":{"id":6883601,"externalId":"511145.b2971","name":"yghG","hasFamily":false,"annotation":"predicted protein","uniprotId":"YGHG_ECOLI"},"6883602":{"id":6883602,"externalId":"511145.b2972","name":"pppA","hasFamily":true,"annotation":"bifunctional prepilin leader peptidase/methylase; Cleaves type-4 fimbrial leader sequence and methylates the N-terminal (generally Phe) residue (By similarity)","uniprotId":"PPPA_ECOLI"},"6883603":{"id":6883603,"externalId":"511145.b2975","name":"glcA","hasFamily":true,"annotation":"glycolate transporter; Transports glycolate across the membrane. Can also transport L-lactate and D-lactate. Seems to be driven by a proton motive force","uniprotId":"GLCA_ECOLI"},"6883604":{"id":6883604,"externalId":"511145.b2976","name":"glcB","hasFamily":true,"annotation":"malate synthase G; Accounts for almost the entire malate-synthesizing activity in cells metabolizing glyoxylate","uniprotId":"MASZ_ECOLI"},"6883605":{"id":6883605,"externalId":"511145.b2977","name":"glcG","hasFamily":true,"annotation":"conserved protein","uniprotId":"GLCG_ECOLI"},"6883606":{"id":6883606,"externalId":"511145.b2979","name":"glcD","hasFamily":true,"annotation":"glycolate oxidase subunit, FAD-linked","uniprotId":"GLCD_ECOLI"},"6883607":{"id":6883607,"externalId":"511145.b2980","name":"glcC","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator, glycolate-binding; Activator for the glycolate oxidation locus","uniprotId":"GLCC_ECOLI"},"6883608":{"id":6883608,"externalId":"511145.b2982","name":"b2982","hasFamily":true,"annotation":"IS5 transposase and trans-activator"},"6883609":{"id":6883609,"externalId":"511145.b2983","name":"yghQ","hasFamily":false,"annotation":"putative inner membrane protein","uniprotId":"YGHQ_ECOLI"},"6883610":{"id":6883610,"externalId":"511145.b2984","name":"yghR","hasFamily":false,"annotation":"putative ATP-binding protein","uniprotId":"YGHR_ECOLI"},"6883611":{"id":6883611,"externalId":"511145.b2985","name":"yghS","hasFamily":false,"annotation":"putative ATP-binding protein","uniprotId":"YGHS_ECOLI"},"6883612":{"id":6883612,"externalId":"511145.b2986","name":"yghT","hasFamily":false,"annotation":"putative ATP-binding protein","uniprotId":"YGHT_ECOLI"},"6883613":{"id":6883613,"externalId":"511145.b2987","name":"pitB","hasFamily":true,"annotation":"phosphate transporter; Low-affinity inorganic phosphate transport (By similarity)","uniprotId":"PITB_ECOLI"},"6883614":{"id":6883614,"externalId":"511145.b2988","name":"gsp","hasFamily":true,"annotation":"fused glutathionylspermidine amidase/glutathionylspermidine synthetase; Catalyzes the formation of an amide bond between glutathione and spermidine coupled with hydrolysis of ATP; also catalyzes the hydrolysis of glutathionylspermidine to glutathione and spermidine","uniprotId":"GSP_ECOLI"},"6883615":{"id":6883615,"externalId":"511145.b2989","name":"yghU","hasFamily":true,"annotation":"putative S-transferase; Exhibits a robust glutathione (GSH)-dependent disulfide- bond reductase activity toward the model substrate, 2-hydroxyethyl disulfide; the actual physiological substrates are not known. Also displays a modest GSH-dependent peroxidase activity toward several organic hydroperoxides, such as cumene hydroperoxide and linoleic acid 13(S)-hydroperoxide, but does not reduce H(2)O(2) or tert- butyl hydroperoxide at appreciable rates. Exhibits little or no GSH transferase activity with most typical electrophilic substrates, and has no detectable transferase activity towa [...] ","uniprotId":"YGHU_ECOLI"},"6883616":{"id":6883616,"externalId":"511145.b2990","name":"hybG","hasFamily":true,"annotation":"hydrogenase 2 accessory protein; May have a specific role in the maturation of the large subunits of HYD1 and HYD2","uniprotId":"HYBG_ECOLI"},"6883617":{"id":6883617,"externalId":"511145.b2991","name":"hybF","hasFamily":true,"annotation":"protein involved with the maturation of hydrogenases 1 and 2; Probably plays a role in a hydrogenase nickel cofactor insertion step (By similarity)","uniprotId":"HYBF_ECOLI"},"6883618":{"id":6883618,"externalId":"511145.b2992","name":"hybE","hasFamily":true,"annotation":"hydrogenase 2-specific chaperone","uniprotId":"HYBE_ECOLI"},"6883619":{"id":6883619,"externalId":"511145.b2993","name":"hybD","hasFamily":true,"annotation":"maturation protease for hydrogenase 2; Protease involved in the C-terminal processing of HybC, the large subunit of hydrogenase 2. Specifically cleaves off a 15 amino acid peptide from the C-terminus of the precursor of HybC","uniprotId":"HYBD_ECOLI"},"6883620":{"id":6883620,"externalId":"511145.b2994","name":"hybC","hasFamily":true,"annotation":"hydrogenase 2, large subunit; This is one of three E.coli hydrogenases synthesized in response to different physiological conditions. HYD2 is involved in hydrogen uptake","uniprotId":"MBHM_ECOLI"},"6883621":{"id":6883621,"externalId":"511145.b2995","name":"hybB","hasFamily":true,"annotation":"putative hydrogenase 2 cytochrome b type component; Probable b-type cytochrome","uniprotId":"HYBB_ECOLI"},"6883622":{"id":6883622,"externalId":"511145.b2996","name":"hybA","hasFamily":true,"annotation":"hydrogenase 2 4Fe-4S ferredoxin-type component; Participates in the periplasmic electron-transferring activity of hydrogenase 2 during its catalytic turnover","uniprotId":"HYBA_ECOLI"},"6883623":{"id":6883623,"externalId":"511145.b2997","name":"hybO","hasFamily":true,"annotation":"hydrogenase 2, small subunit; This is one of three E.coli hydrogenases synthesized in response to different physiological conditions. HYD2 is involved in hydrogen uptake","uniprotId":"MBHT_ECOLI"},"6883624":{"id":6883624,"externalId":"511145.b2998","name":"yghW","hasFamily":true,"annotation":"predicted protein","uniprotId":"YGHW_ECOLI"},"6883625":{"id":6883625,"externalId":"511145.b3001","name":"gpr","hasFamily":true,"annotation":"L-glyceraldehyde 3-phosphate reductase; Catalyzes the stereospecific, NADPH-dependent reduction of L-glyceraldehyde 3-phosphate (L-GAP). The physiological role of gpr is the detoxification of L-GAP, which may be formed by non- enzymatic racemization of GAP. Also involved in the stress response as a methylglyoxal reductase which converts the toxic metabolite methylglyoxal to acetol in vitro and in vivo","uniprotId":"GPR_ECOLI"},"6883626":{"id":6883626,"externalId":"511145.b3002","name":"yqhA","hasFamily":true,"annotation":"putative inner membrane protein, UPF0114 family","uniprotId":"YQHA_ECOLI"},"6883627":{"id":6883627,"externalId":"511145.b3003","name":"yghA","hasFamily":true,"annotation":"putative oxidoreductase","uniprotId":"YGHA_ECOLI"},"6883628":{"id":6883628,"externalId":"511145.b3005","name":"exbD","hasFamily":true,"annotation":"membrane spanning protein in TonB-ExbB-ExbD complex; Involved in the TonB-dependent energy-dependent transport of various receptor-bound substrates","uniprotId":"EXBD_ECOLI"},"6883629":{"id":6883629,"externalId":"511145.b3006","name":"exbB","hasFamily":true,"annotation":"membrane spanning protein in TonB-ExbB-ExbD complex; Involved in the TonB-dependent energy-dependent transport of various receptor-bound substrates. Protects ExbD from proteolytic degradation and functionally stabilizes TonB","uniprotId":"EXBB_ECOLI"},"6883630":{"id":6883630,"externalId":"511145.b3008","name":"metC","hasFamily":true,"annotation":"cystathionine beta-lyase, PLP-dependent","uniprotId":"METC_ECOLI"},"6883631":{"id":6883631,"externalId":"511145.b3009","name":"yghB","hasFamily":true,"annotation":"required, with yqjA, for membrane integrity; Required, with YqjA, for the proper export of certain periplasmic amidases and, possibly, other Tat substrates. May play a role in determining membrane lipid composition","uniprotId":"YGHB_ECOLI"},"6883632":{"id":6883632,"externalId":"511145.b3010","name":"yqhC","hasFamily":true,"annotation":"transcriptional activator of yqhD","uniprotId":"YQHC_ECOLI"},"6883633":{"id":6883633,"externalId":"511145.b3011","name":"yqhD","hasFamily":true,"annotation":"aldehyde reductase, NADPH-dependent; NADP-dependent ADH activity","uniprotId":"YQHD_ECOLI"},"6883634":{"id":6883634,"externalId":"511145.b3012","name":"dkgA","hasFamily":true,"annotation":"2,5-diketo-D-gluconate reductase A; Catalyzes the reduction of 2,5-diketo-D-gluconic acid (25DKG) to 2-keto-L-gulonic acid (2KLG). It is also capable of stereoselective -keto ester reductions on ethyl acetoacetate and other 2-substituted derivatives","uniprotId":"DKGA_ECOLI"},"6883635":{"id":6883635,"externalId":"511145.b3013","name":"yqhG","hasFamily":false,"annotation":"conserved protein","uniprotId":"YQHG_ECOLI"},"6883636":{"id":6883636,"externalId":"511145.b3014","name":"yqhH","hasFamily":false,"annotation":"outer membrane lipoprotein, Lpp paralog","uniprotId":"YQHH_ECOLI"},"6883637":{"id":6883637,"externalId":"511145.b3017","name":"ftsP","hasFamily":true,"annotation":"septal ring component that protects the divisome from stress; multicopy suppressor of ftsI(Ts); Cell division protein that is required for growth during stress conditions. May be involved in protecting or stabilizing the divisomal assembly under conditions of stress (By similarity)","uniprotId":"FTSP_ECOLI"},"6883638":{"id":6883638,"externalId":"511145.b3018","name":"plsC","hasFamily":true,"annotation":"1-acyl-sn-glycerol-3-phosphate acyltransferase; Converts lysophosphatidic acid (LPA) into phosphatidic acid by incorporating an acyl moiety at the 2 position. This enzyme can utilize either acyl-CoA or acyl-ACP as the fatty acyl donor","uniprotId":"PLSC_ECOLI"},"6883639":{"id":6883639,"externalId":"511145.b3019","name":"parC","hasFamily":true,"annotation":"DNA topoisomerase IV, subunit A; Topoisomerase IV is essential for chromosome segregation. It relaxes supercoiled DNA. Performs the decatenation events required during the replication of a circular DNA molecule. MukB stimulates the relaxation activity of topoisomerase IV and also has a modest effect on decatenation","uniprotId":"PARC_ECOLI"},"6883640":{"id":6883640,"externalId":"511145.b3020","name":"ygiS","hasFamily":true,"annotation":"putative transporter subunit: periplasmic-binding component of ABC superfamily; Probably part of a binding-protein-dependent transport system","uniprotId":"YGIS_ECOLI"},"6883641":{"id":6883641,"externalId":"511145.b3021","name":"mqsA","hasFamily":true,"annotation":"antitoxin for MqsR toxin; transcriptional repressor; Antitoxin component of a toxin-antitoxin (TA) module. Labile antitoxin that binds to the MqsR mRNA interferase toxin and neutralizes its endoribonuclease activity. Overexpression prevents MqsR-mediated cessation of cell growth and inhibition of cell proliferation. Overexpression also increases biofilm formation, perhaps by repressing cspD","uniprotId":"MQSA_ECOLI"},"6883642":{"id":6883642,"externalId":"511145.b3022","name":"mqsR","hasFamily":true,"annotation":"GCU-specific mRNA interferase toxin of the MqsR-MqsA toxin-antitoxin system; biofilm/motility regulator; Toxic component of a toxin-antitoxin (TA) module. An mRNA interferase which has been reported to be translation- independent (PubMed:19690171, PubMed:19943910) and translation- dependent (PubMed:20041169). Cleavage has been reported to occur on either side of G in the sequence GCU (PubMed:19690171) but also after C in GC(A/U) sequences (PubMed:19943910). There are only 14 genes in E.coli W3110 (and probably also MG1655) that do not have a GCU sequence and thus are resistant to the m [...] ","uniprotId":"MQSR_ECOLI"},"6883643":{"id":6883643,"externalId":"511145.b3023","name":"ygiV","hasFamily":true,"annotation":"transcriptional repressor for mcbR biofilm gene; Represses expression of mcbR","uniprotId":"YGIV_ECOLI"},"6883644":{"id":6883644,"externalId":"511145.b3024","name":"ygiW","hasFamily":true,"annotation":"conserved protein","uniprotId":"YGIW_ECOLI"},"6883645":{"id":6883645,"externalId":"511145.b3025","name":"qseB","hasFamily":true,"annotation":"quorum sensing DNA-binding response regulator in two-component regulatory system with QseC; Member of a two-component regulatory system QseB/QseC. Activates the flagella regulon by activating transcription of FlhDC. Currently it is not known whether this effect is direct or not","uniprotId":"QSEB_ECOLI"},"6883646":{"id":6883646,"externalId":"511145.b3026","name":"qseC","hasFamily":true,"annotation":"quorum sensing sensory histidine kinase in two-component regulatory system with QseB; Member of a two-component regulatory system QseB/QseC. Activates the flagella regulon by activating transcription of FlhDC. May activate QseB by phosphorylation","uniprotId":"QSEC_ECOLI"},"6883647":{"id":6883647,"externalId":"511145.b3027","name":"ygiZ","hasFamily":false,"annotation":"inner membrane protein","uniprotId":"YGIZ_ECOLI"},"6883648":{"id":6883648,"externalId":"511145.b3028","name":"mdaB","hasFamily":true,"annotation":"NADPH quinone reductase","uniprotId":"MDAB_ECOLI"},"6883649":{"id":6883649,"externalId":"511145.b3029","name":"ygiN","hasFamily":true,"annotation":"quinol monooxygenase; Can oxidize menadiol to menadione","uniprotId":"YGIN_ECOLI"},"6883650":{"id":6883650,"externalId":"511145.b3030","name":"parE","hasFamily":true,"annotation":"DNA topoisomerase IV, subunit B; Topoisomerase IV is essential for chromosome segregation. It relaxes supercoiled DNA. Performs the decatenation events required during the replication of a circular DNA molecule. MukB stimulates the relaxation activity of topoisomerase IV and also has a modest effect on decatenation","uniprotId":"PARE_ECOLI"},"6883651":{"id":6883651,"externalId":"511145.b3031","name":"yqiA","hasFamily":true,"annotation":"acyl CoA esterase; Displays esterase activity toward palmitoyl-CoA and pNP- butyrate","uniprotId":"YQIA_ECOLI"},"6883652":{"id":6883652,"externalId":"511145.b3032","name":"icc","hasFamily":true,"annotation":"3',5' cAMP phosphodiesterase; Hydrolyzes cAMP to 5'-AMP. Plays an important regulatory role in modulating the intracellular concentration of cAMP, thereby influencing cAMP-dependent processes. Specific for cAMP","uniprotId":"CPDA_ECOLI"},"6883653":{"id":6883653,"externalId":"511145.b3033","name":"yqiB","hasFamily":true,"annotation":"DUF1249 protein YqiB","uniprotId":"YQIB_ECOLI"},"6883654":{"id":6883654,"externalId":"511145.b3034","name":"nudF","hasFamily":true,"annotation":"ADP-ribose pyrophosphatase; Acts on ADP-mannose and ADP-glucose as well as ADP- ribose. Prevents glycogen biosynthesis. The reaction catalyzed by this enzyme is a limiting step of the gluconeogenic process","uniprotId":"ADPP_ECOLI"},"6883655":{"id":6883655,"externalId":"511145.b3035","name":"tolC","hasFamily":true,"annotation":"transport channel; Required for proper expression of outer membrane protein genes such as ompF, nmpC, protein 2, hemolysin, colicin V, or colicin E1. May be specialized for signal sequence independent, extracellular secretion in Gram-negative bacteria","uniprotId":"TOLC_ECOLI"},"6883656":{"id":6883656,"externalId":"511145.b3037","name":"ygiB","hasFamily":true,"annotation":"conserved protein, UPF0441 family","uniprotId":"YGIB_ECOLI"},"6883657":{"id":6883657,"externalId":"511145.b3038","name":"ygiC","hasFamily":true,"annotation":"Glutathionylspermidine synthase homolog","uniprotId":"YGIC_ECOLI"},"6883658":{"id":6883658,"externalId":"511145.b3039","name":"ygiD","hasFamily":true,"annotation":"putative dioxygenase, LigB family","uniprotId":"YGID_ECOLI"},"6883659":{"id":6883659,"externalId":"511145.b3040","name":"zupT","hasFamily":true,"annotation":"zinc transporter; Mediates zinc uptake. May also transport other divalent cations such as copper and cadmium ions","uniprotId":"ZUPT_ECOLI"},"6883660":{"id":6883660,"externalId":"511145.b3041","name":"ribB","hasFamily":true,"annotation":"3,4-dihydroxy-2-butanone-4-phosphate synthase; Catalyzes the conversion of D-ribulose 5-phosphate to formate and 3,4-dihydroxy-2-butanone 4-phosphate (By similarity)","uniprotId":"RIBB_ECOLI"},"6883661":{"id":6883661,"externalId":"511145.b3042","name":"yqiC","hasFamily":true,"annotation":"conserved protein","uniprotId":"YQIC_ECOLI"},"6883662":{"id":6883662,"externalId":"511145.b3043","name":"ygiL","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YGIL_ECOLI"},"6883663":{"id":6883663,"externalId":"511145.b3044","name":"b3044","hasFamily":true,"annotation":"IS2 repressor TnpA"},"6883664":{"id":6883664,"externalId":"511145.b3045","name":"b3045","hasFamily":true,"annotation":"IS2 transposase TnpB"},"6883665":{"id":6883665,"externalId":"511145.b3047","name":"yqiH","hasFamily":false,"annotation":"putative periplasmic pilin chaperone; Could be required for the biogenesis of a putative fimbria (By similarity)","uniprotId":"YQIH_ECOLI"},"6883666":{"id":6883666,"externalId":"511145.b3048","name":"yqiI","hasFamily":true,"annotation":"conserved protein; May be involved in a fimbrial system chaperoned by YqiH and exported by YqiG","uniprotId":"YQII_ECOLI"},"6883667":{"id":6883667,"externalId":"511145.b3049","name":"glgS","hasFamily":true,"annotation":"putative glycogen synthesis protein; Involved in glycogen synthesis. May be involved in glycogen priming (By similarity)","uniprotId":"GLGS_ECOLI"},"6883668":{"id":6883668,"externalId":"511145.b3050","name":"yqiJ","hasFamily":true,"annotation":"inner membrane protein, DUF1449 family","uniprotId":"YQIJ_ECOLI"},"6883669":{"id":6883669,"externalId":"511145.b3051","name":"yqiK","hasFamily":true,"annotation":"PHB family membrane protein, function unknown","uniprotId":"YQIK_ECOLI"},"6883670":{"id":6883670,"externalId":"511145.b3052","name":"hldE","hasFamily":true,"annotation":"fused heptose 7-phosphate kinase/heptose 1-phosphate adenyltransferase; Catalyzes the phosphorylation of D-glycero-D-manno- heptose 7-phosphate at the C-1 position to form D,D-heptose-1,7- bisphosphate","uniprotId":"HLDE_ECOLI"},"6883671":{"id":6883671,"externalId":"511145.b3053","name":"glnE","hasFamily":true,"annotation":"fused deadenylyltransferase/adenylyltransferase for glutamine synthetase; Adenylation and deadenylation of glutamate--ammonia ligase (By similarity)","uniprotId":"GLNE_ECOLI"},"6883672":{"id":6883672,"externalId":"511145.b3054","name":"ygiF","hasFamily":true,"annotation":"putative adenylate cyclase","uniprotId":"YGIF_ECOLI"},"6883673":{"id":6883673,"externalId":"511145.b3055","name":"ecfG","hasFamily":true,"annotation":"SH3 domain protein","uniprotId":"YGIM_ECOLI"},"6883674":{"id":6883674,"externalId":"511145.b3056","name":"cca","hasFamily":true,"annotation":"fused tRNA nucleotidyl transferase/2'3'-cyclic phosphodiesterase/2'nucleotidase and phosphatase; Catalyzes the addition and repair of the essential 3'- terminal CCA sequence in tRNAs without using a nucleic acid template. Adds these three nucleotides in the order of C, C, and A to the tRNA nucleotide-73, using CTP and ATP as substrates and producing inorganic pyrophosphate. Also shows highest phosphatase activity in the presence of Ni(2+) and hydrolyzes pyrophosphate, canonical 5'-nucleoside tri- and diphosphates, NADP, and 2'-AMP with the production of Pi. Displays a metal-independent [...] ","uniprotId":"CCA_ECOLI"},"6883675":{"id":6883675,"externalId":"511145.b3057","name":"bacA","hasFamily":true,"annotation":"undecaprenyl pyrophosphate phosphatase; Catalyzes the dephosphorylation of undecaprenyl diphosphate (UPP). Confers resistance to bacitracin (By similarity)","uniprotId":"UPPP_ECOLI"},"6883676":{"id":6883676,"externalId":"511145.b3058","name":"folB","hasFamily":true,"annotation":"bifunctional dihydroneopterin aldolase/dihydroneopterin triphosphate 2'-epimerase; Catalyzes the conversion of 7,8-dihydroneopterin to 6- hydroxymethyl-7,8-dihydropterin. Can use L-threo-dihydroneopterin and D-erythro-dihydroneopterin as substrates for the formation of 6-hydroxymethyldihydropterin, but it can also catalyze the epimerization of carbon 2' of dihydroneopterin and dihydromonapterin at appreciable velocity","uniprotId":"FOLB_ECOLI"},"6883677":{"id":6883677,"externalId":"511145.b3059","name":"plsY","hasFamily":true,"annotation":"probable glycerol-3-phosphate acyltransferase; Catalyzes the transfer of an acyl group from acyl-ACP to glycerol-3-phosphate (G3P) to form lysophosphatidic acid (LPA). This enzyme can also utilize acyl-CoA as fatty acyl donor, but not acyl-PO(4) (By similarity)","uniprotId":"PLSY_ECOLI"},"6883678":{"id":6883678,"externalId":"511145.b3060","name":"ttdR","hasFamily":true,"annotation":"transcriptional activator of ttdABT; Positive regulator required for L-tartrate-dependent anaerobic growth on glycerol. Induces expression of the ttdA-ttdB- ygjE operon","uniprotId":"TTDR_ECOLI"},"6883679":{"id":6883679,"externalId":"511145.b3061","name":"ttdA","hasFamily":true,"annotation":"L-tartrate dehydratase, alpha subunit","uniprotId":"TTDA_ECOLI"},"6883680":{"id":6883680,"externalId":"511145.b3062","name":"ttdB","hasFamily":true,"annotation":"L-tartrate dehydratase, beta subunit","uniprotId":"TTDB_ECOLI"},"6883681":{"id":6883681,"externalId":"511145.b3063","name":"ttdT","hasFamily":false,"annotation":"L-tartrate/succinate antiporte; Catalyzes the uptake of tartrate in exchange for intracellular succinate. Essential for anaerobic L-tartrate fermentation","uniprotId":"TTDT_ECOLI"},"6883682":{"id":6883682,"externalId":"511145.b3064","name":"gcp","hasFamily":true,"annotation":"t(6)A tRNA modification protein; glycation-binding protein; genome maintenance protein; Required for the formation of a threonylcarbamoyl group on adenosine at position 37 (t(6)A37) in tRNAs that read codons beginning with adenine. May also be involved in the metabolism of glycated proteins","uniprotId":"GCP_ECOLI"},"6883683":{"id":6883683,"externalId":"511145.b3065","name":"rpsU","hasFamily":false,"annotation":"30S ribosomal subunit protein S21","uniprotId":"RS21_ECOLI"},"6883684":{"id":6883684,"externalId":"511145.b3066","name":"dnaG","hasFamily":true,"annotation":"DNA primase; DNA primase is the polymerase that synthesizes small RNA primers for the Okazaki fragments on both template strands at replication forks during chromosomal DNA synthesis (By similarity)","uniprotId":"PRIM_ECOLI"},"6883685":{"id":6883685,"externalId":"511145.b3067","name":"rpoD","hasFamily":true,"annotation":"RNA polymerase, sigma 70 (sigma D) factor; Sigma factors are initiation factors that promote the attachment of RNA polymerase to specific initiation sites and are then released. This is the primary sigma factor of this bacterium","uniprotId":"RPOD_ECOLI"},"6883686":{"id":6883686,"externalId":"511145.b3068","name":"mug","hasFamily":true,"annotation":"G/U mismatch-specific DNA glycosylase; xanthine DNA glycosylase; Excises ethenocytosine and uracil, which can arise by alkylation or deamination of cytosine, respectively, from the corresponding mispairs with guanine in ds-DNA. It is capable of hydrolyzing the carbon-nitrogen bond between the sugar-phosphate backbone of the DNA and the mispaired base. The complementary strand guanine functions in substrate recognition. Required for DNA damage lesion repair in stationary-phase cells (By similarity)","uniprotId":"MUG_ECOLI"},"6883687":{"id":6883687,"externalId":"511145.b3070","name":"yqjH","hasFamily":true,"annotation":"putative siderophore interacting protein; Plays a role in iron homeostasis under excess nickel conditions","uniprotId":"YQJH_ECOLI"},"6883688":{"id":6883688,"externalId":"511145.b3071","name":"yqjI","hasFamily":true,"annotation":"putative transcriptional regulator, PadR family; Represses the expression of YqjH which is involved in iron homeostasis under excess nickel conditions. Also represses its own expression","uniprotId":"YQJI_ECOLI"},"6883689":{"id":6883689,"externalId":"511145.b3072","name":"aer","hasFamily":true,"annotation":"fused signal transducer for aerotaxis sensory component/methyl accepting chemotaxis component; Signal transducer for aerotaxis. The aerotactic response is the accumulation of cells around air bubbles. The nature of the sensory stimulus detected by this protein is the proton motive force or cellular redox state. It uses a FAD prosthetic group as a redox sensor to monitor oxygen levels","uniprotId":"AER_ECOLI"},"6883690":{"id":6883690,"externalId":"511145.b3073","name":"patA","hasFamily":true,"annotation":"putrescine:2-oxoglutaric acid aminotransferase, PLP-dependent; Catalyzes the aminotransferase reaction from putrescine to 2-oxoglutarate, leading to glutamate and 4-aminobutanal, which spontaneously cyclizes to form 1-pyrroline. Is also able to transaminate cadaverine and, in lower extent, spermidine, but not ornithine. Alpha-ketobutyrate and pyruvate can also act as amino acceptors, although much less efficiently","uniprotId":"PAT_ECOLI"},"6883691":{"id":6883691,"externalId":"511145.b3074","name":"ygjH","hasFamily":true,"annotation":"conserved tRNA binding protein","uniprotId":"YGJH_ECOLI"},"6883692":{"id":6883692,"externalId":"511145.b3075","name":"ECs3957","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor for beta galactosidase alpha and beta subunits (ebgA and ebgC). Binds lactose as an inducer","uniprotId":"EBGR_ECOLI"},"6883693":{"id":6883693,"externalId":"511145.b3076","name":"ebgA","hasFamily":true,"annotation":"cryptic beta-D-galactosidase, alpha subunit; The wild-type enzyme is an ineffective lactase. Two classes of point mutations dramatically improve activity of the enzyme","uniprotId":"BGA2_ECOLI"},"6883694":{"id":6883694,"externalId":"511145.b3077","name":"ebgC","hasFamily":false,"annotation":"cryptic beta-D-galactosidase, beta subunit; Required for full activity of the EbgA enzyme. Exact function not known","uniprotId":"EBGC_ECOLI"},"6883695":{"id":6883695,"externalId":"511145.b3078","name":"ygjI","hasFamily":true,"annotation":"putative transporter","uniprotId":"YGJI_ECOLI"},"6883696":{"id":6883696,"externalId":"511145.b3079","name":"ygjJ","hasFamily":false,"annotation":"conserved protein","uniprotId":"YGJJ_ECOLI"},"6883697":{"id":6883697,"externalId":"511145.b3080","name":"ygjK","hasFamily":false,"annotation":"alpha-glucosidase; Glucoside hydrolase that cleaves the alpha-1,3- glucosidic linkage in nigerose. Has very low activity towards maltooligosaccharides, soluble starch, nigerotriose, kojibiose and trehalose","uniprotId":"YGJK_ECOLI"},"6883698":{"id":6883698,"externalId":"511145.b3081","name":"fadH","hasFamily":true,"annotation":"2,4-dienoyl-CoA reductase, NADH and FMN-linked; Catalyzes the NADP-dependent reduction of 2,4-dienoyl- CoA to yield trans-2- enoyl-CoA","uniprotId":"FADH_ECOLI"},"6883699":{"id":6883699,"externalId":"511145.b3082","name":"higA","hasFamily":true,"annotation":"antitoxinof the HigB-HigA toxin-antitoxin system; Antitoxin component of a toxin-antitoxin (TA) module. Functions as an mRNA interferase antitoxin; overexpression prevents HigB-mediated cessation of cell growth and inhibition of cell proliferation","uniprotId":"HIGA_ECOLI"},"6883700":{"id":6883700,"externalId":"511145.b3083","name":"higB","hasFamily":true,"annotation":"mRNA interferase toxin of the HigB-HigA toxin-antitoxin system; Toxic component of a toxin-antitoxin (TA) module. A probable translation-dependent mRNA interferase. Overexpression causes cessation of cell growth and inhibits cell proliferation via inhibition of translation; this blockage is overcome by subsequent expression of antitoxin HigA. Overexpression causes cleavage of a number of mRNAs in a translation-dependent fashion, suggesting this is an mRNA interferase","uniprotId":"HIGB_ECOLI"},"6883701":{"id":6883701,"externalId":"511145.b3084","name":"rlmG","hasFamily":true,"annotation":"23S rRNA mG1835 methyltransferase, SAM-dependent; Specifically methylates the guanine in position 1835 (m2G1835) of 23S rRNA (By similarity)","uniprotId":"RLMG_ECOLI"},"6883702":{"id":6883702,"externalId":"511145.b3085","name":"ygjP","hasFamily":true,"annotation":"putative metal dependent hydrolase","uniprotId":"YGJP_ECOLI"},"6883703":{"id":6883703,"externalId":"511145.b3086","name":"ygjQ","hasFamily":true,"annotation":"conserved protein, SanA family, DUF218 superfamily","uniprotId":"YGJQ_ECOLI"},"6883704":{"id":6883704,"externalId":"511145.b3087","name":"ygjR","hasFamily":true,"annotation":"putative NAD(P)-binding dehydrogenase","uniprotId":"YGJR_ECOLI"},"6883705":{"id":6883705,"externalId":"511145.b3088","name":"alx","hasFamily":true,"annotation":"inner membrane protein, part of terminus","uniprotId":"ALX_ECOLI"},"6883706":{"id":6883706,"externalId":"511145.b3089","name":"sstT","hasFamily":true,"annotation":"sodium:serine/threonine symporter; Involved in the import of serine and threonine into the cell, with the concomitant import of sodium (symport system) (By similarity)","uniprotId":"SSTT_ECOLI"},"6883707":{"id":6883707,"externalId":"511145.b3090","name":"ygjV","hasFamily":true,"annotation":"inner membrane protein, Imp-YgjV family","uniprotId":"YGJV_ECOLI"},"6883708":{"id":6883708,"externalId":"511145.b3091","name":"uxaA","hasFamily":true,"annotation":"altronate hydrolase; Catalyzes the dehydration of D-altronate","uniprotId":"UXAA_ECOLI"},"6883709":{"id":6883709,"externalId":"511145.b3092","name":"uxaC","hasFamily":true,"annotation":"uronate isomerase","uniprotId":"UXAC_ECOLI"},"6883710":{"id":6883710,"externalId":"511145.b3093","name":"exuT","hasFamily":true,"annotation":"hexuronate transporter; Aldohexuronate transport system","uniprotId":"EXUT_ECOLI"},"6883711":{"id":6883711,"externalId":"511145.b3094","name":"exuR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor for the exu regulon that encode genes involved in hexuronate utilization. It regulates the ExuT, UxaCA and UxuRAB operons. Binds D-tagaturonate and D-fructuronate as inducers","uniprotId":"EXUR_ECOLI"},"6883712":{"id":6883712,"externalId":"511145.b3095","name":"ecfL","hasFamily":true,"annotation":"required, with yghB, for membrane integrity; Required, with YghB, for the proper export of certain periplasmic amidases and, possibly, other Tat substrates. May play a role in determining membrane lipid composition","uniprotId":"YQJA_ECOLI"},"6883713":{"id":6883713,"externalId":"511145.b3096","name":"ecfM","hasFamily":true,"annotation":"modulator of EnvZ/OmpR regulon; Modulates the activity of the EnvZ/OmpR two-component regulatory system, probably by directly modulating EnvZ enzymatic activity and increasing stability of phosphorylated OmpR. Links the two-component systems CpxA/CpxR and EnvZ/OmpR","uniprotId":"MZRA_ECOLI"},"6883714":{"id":6883714,"externalId":"511145.b3097","name":"yqjC","hasFamily":true,"annotation":"conserved protein","uniprotId":"YQJC_ECOLI"},"6883715":{"id":6883715,"externalId":"511145.b3098","name":"yqjD","hasFamily":true,"annotation":"conserved protein; Upon overexpression inhibits growth","uniprotId":"YQJD_ECOLI"},"6883716":{"id":6883716,"externalId":"511145.b3099","name":"yqjE","hasFamily":true,"annotation":"inner membrane protein, DUF1469 family","uniprotId":"YQJE_ECOLI"},"6883717":{"id":6883717,"externalId":"511145.b3100","name":"yqjK","hasFamily":true,"annotation":"conserved protein","uniprotId":"YQJK_ECOLI"},"6883718":{"id":6883718,"externalId":"511145.b3101","name":"yqjF","hasFamily":true,"annotation":"putative quinol oxidase subunit","uniprotId":"YQJF_ECOLI"},"6883719":{"id":6883719,"externalId":"511145.b3102","name":"yqjG","hasFamily":true,"annotation":"putative S-transferase; Catalyzes glutathione (GSH)-dependent reduction of glutathionyl-hydroquinones (GS-HQs) to the corresponding hydroquinones. Can use a variety of GS-HQs as substrates, such as GS-p-hydroquinone (GS-HQ), GS-hydroxy-p-hydroquinone (GS-HHQ), GS- methyl-p-hydroquinone (GS-MHQ), GS-menadiol, and GS-trichloro-p- hydroquinone (GS-TriCH). Also displays GSH-dependent disulfide- bond reduction activity toward HED (2-hydroxyethyl disulfide), and is able to catalyze DMA (dimethylarsinate) reduction. Exhibits no GSH transferase activity with 1-chloro-2,4-dinitrobenzene (CDNB)","uniprotId":"YQJG_ECOLI"},"6883720":{"id":6883720,"externalId":"511145.b3103","name":"yhaH","hasFamily":true,"annotation":"inner membrane protein, DUF805 family","uniprotId":"YHAH_ECOLI"},"6883721":{"id":6883721,"externalId":"511145.b3104","name":"yhaI","hasFamily":true,"annotation":"inner membrane protein, DUF805 family","uniprotId":"YHAI_ECOLI"},"6883722":{"id":6883722,"externalId":"511145.b3105","name":"yhaJ","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YHAJ_ECOLI"},"6883723":{"id":6883723,"externalId":"511145.b3106","name":"yhaK","hasFamily":true,"annotation":"redox-sensitive bicupin; Does not have quercetin 2,3-dioxygenase activity","uniprotId":"YHAK_ECOLI"},"6883724":{"id":6883724,"externalId":"511145.b3107","name":"yhaL","hasFamily":true,"annotation":"predicted protein","uniprotId":"YHAL_ECOLI"},"6883725":{"id":6883725,"externalId":"511145.b3110","name":"yhaO","hasFamily":true,"annotation":"putative transporter","uniprotId":"YHAO_ECOLI"},"6883726":{"id":6883726,"externalId":"511145.b3113","name":"tdcF","hasFamily":true,"annotation":"predicted L-PSP (mRNA) endoribonuclease; May be a post-translational regulator that controls the metabolic fate of L-threonine or the potentially toxic intermediate 2-ketobutyrate","uniprotId":"TDCF_ECOLI"},"6883727":{"id":6883727,"externalId":"511145.b3114","name":"tdcE","hasFamily":true,"annotation":"pyruvate formate-lyase 4/2-ketobutyrate formate-lyase; Catalyzes the cleavage of 2-ketobutyrate to propionyl- CoA and formate. It can also use pyruvate as substrate","uniprotId":"TDCE_ECOLI"},"6883728":{"id":6883728,"externalId":"511145.b3115","name":"tdcD","hasFamily":true,"annotation":"propionate kinase/acetate kinase C, anaerobic; Catalyzes the conversion of propionyl phosphate and ADP to propionate and ATP. It can also use acetyl phosphate as phosphate group acceptor","uniprotId":"TDCD_ECOLI"},"6883729":{"id":6883729,"externalId":"511145.b3116","name":"tdcC","hasFamily":true,"annotation":"L-threonine/L-serine transporter; Involved in the import of threonine and serine into the cell, with the concomitant import of a proton (symport system) (By similarity)","uniprotId":"TDCC_ECOLI"},"6883730":{"id":6883730,"externalId":"511145.b3117","name":"tdcB","hasFamily":true,"annotation":"catabolic threonine dehydratase, PLP-dependent; Catalyzes the anaerobic formation of alpha-ketobutyrate and ammonia from threonine in a two-step reaction. The first step involved a dehydration of threonine and a production of enamine intermediates (aminocrotonate), which tautomerizes to its imine form (iminobutyrate). Both intermediates are unstable and short- lived. The second step is the nonenzymatic hydrolysis of the enamine/imine intermediates to form 2-ketobutyrate and free ammonia. In the low water environment of the cell, the second step is accelerated by RidA. TdcB also dehydra [...] ","uniprotId":"TDCB_ECOLI"},"6883731":{"id":6883731,"externalId":"511145.b3118","name":"tdcA","hasFamily":true,"annotation":"DNA-binding transcriptional activator; Transcriptional activator for the tdcABCDE operon","uniprotId":"TDCA_ECOLI"},"6883732":{"id":6883732,"externalId":"511145.b3119","name":"tdcR","hasFamily":false,"annotation":"DNA-binding transcriptional activator; Probable trans-acting positive activator for the tdc operon","uniprotId":"TDCR_ECOLI"},"6883733":{"id":6883733,"externalId":"511145.b3120","name":"yhaB","hasFamily":false,"annotation":"predicted protein","uniprotId":"YHAB_ECOLI"},"6883734":{"id":6883734,"externalId":"511145.b3121","name":"yhaC","hasFamily":false,"annotation":"predicted protein","uniprotId":"YHAC_ECOLI"},"6883735":{"id":6883735,"externalId":"511145.b3124","name":"garK","hasFamily":true,"annotation":"glycerate kinase I","uniprotId":"GLXK1_ECOLI"},"6883736":{"id":6883736,"externalId":"511145.b3125","name":"garR","hasFamily":true,"annotation":"tartronate semialdehyde reductase","uniprotId":"GARR_ECOLI"},"6883737":{"id":6883737,"externalId":"511145.b3126","name":"garL","hasFamily":true,"annotation":"alpha-dehydro-beta-deoxy-D-glucarate aldolase; Catalyzes the reversible retro-aldol cleavage of both 5- keto-4-deoxy-D-glucarate and 2-keto-3-deoxy-D-glucarate to pyruvate and tartronic semialdehyde (By similarity)","uniprotId":"GARL_ECOLI"},"6883738":{"id":6883738,"externalId":"511145.b3127","name":"garP","hasFamily":true,"annotation":"putative (D)-galactarate transporter; Uptake of D-galactarate","uniprotId":"GARP_ECOLI"},"6883739":{"id":6883739,"externalId":"511145.b3128","name":"garD","hasFamily":true,"annotation":"(D)-galactarate dehydrogenase","uniprotId":"GARD_ECOLI"},"6883740":{"id":6883740,"externalId":"511145.b3129","name":"prlF","hasFamily":true,"annotation":"antitoxin of the SohA(PrlF)-YhaV toxin-antitoxin system; Antitoxin component of a toxin-antitoxin (TA) module. Labile antitoxin that binds to the YhaV toxin and neutralizes its ribonuclease activity. Also acts as a transcription factor. The YhaV/PrlF complex binds the prlF-yhaV operon, probably negatively regulating its expression","uniprotId":"PRLF_ECOLI"},"6883741":{"id":6883741,"externalId":"511145.b3130","name":"yhaV","hasFamily":true,"annotation":"toxin of the SohB(PrlF)-YhaV toxin-antitoxin system; Toxic component of a toxin-antitoxin (TA) module. Has RNase activity in vitro. Overexpression leads to growth arrest after 30 minutes; these effects are overcome by concomitant expression of antitoxin SohA (PrlF). Massive overexpression is toxic. Unlike most other characterized TA systems degrades rRNA, and co-folding of the proteins is necessary in vitro for inhibition of the RNase activity. It is not known if it has any sequence-specificity. Acts as a transcription factor. The YhaV/PrlF complex binds the prlF-yhaV operon, probably  [...] ","uniprotId":"YHAV_ECOLI"},"6883742":{"id":6883742,"externalId":"511145.b3131","name":"agaR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor of the aga regulon; Probable repressor for the aga operon for N-acetyl galactosamine transport and metabolism","uniprotId":"AGAR_ECOLI"},"6883743":{"id":6883743,"externalId":"511145.b3132","name":"agaZ","hasFamily":true,"annotation":"tagatose 6-phosphate aldolase 1, kbaZ subunit; Component of the tagatose-1,6-bisphosphate aldolase KbaYZ that is required for full activity and stability of the Y subunit. Could have a chaperone-like function for the proper and stable folding of KbaY. When expressed alone, KbaZ does not show any aldolase activity. Is involved in the catabolism of N- acetylgalactosamine and D-galactosamine","uniprotId":"KBAZ_ECOLI"},"6883744":{"id":6883744,"externalId":"511145.b3133","name":"agaV","hasFamily":true,"annotation":"N-acetylgalactosamine-specific enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in N-acetylgalactosamine transport","uniprotId":"PTPB2_ECOLI"},"6883745":{"id":6883745,"externalId":"511145.b3136","name":"agaS","hasFamily":true,"annotation":"tagatose-6-phosphate ketose/aldose isomerase; May be a tagatose-6-phosphate ketose/aldose isomerase (By similarity)","uniprotId":"AGAS_ECOLI"},"6883746":{"id":6883746,"externalId":"511145.b3137","name":"agaY","hasFamily":true,"annotation":"tagatose 6-phosphate aldolase 1, kbaY subunit; Catalytic subunit of the tagatose-1,6-bisphosphate aldolase KbaYZ, which catalyzes the reversible aldol condensation of dihydroxyacetone phosphate (DHAP or glycerone-phosphate) with glyceraldehyde 3-phosphate (G3P) to produce tagatose 1,6- bisphosphate (TBP). Requires KbaZ subunit for full activity and stability. Is involved in the catabolism of N-acetylgalactosamine and D-galactosamine","uniprotId":"KBAY_ECOLI"},"6883747":{"id":6883747,"externalId":"511145.b3138","name":"agaB","hasFamily":false,"annotation":"N-acetylgalactosamine-specific enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in N-acetylgalactosamine transport","uniprotId":"PTPB1_ECOLI"},"6883748":{"id":6883748,"externalId":"511145.b3139","name":"agaC","hasFamily":true,"annotation":"N-acetylgalactosamine-specific enzyme IIC component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in N-acetylgalactosamine transport","uniprotId":"PTPC1_ECOLI"},"6883749":{"id":6883749,"externalId":"511145.b3140","name":"agaD","hasFamily":true,"annotation":"N-acetylgalactosamine-specific enzyme IID component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in N-acetylgalactosamine transport","uniprotId":"PTPD_ECOLI"},"6883750":{"id":6883750,"externalId":"511145.b3141","name":"agaI","hasFamily":true,"annotation":"galactosamine-6-phosphate isomerase","uniprotId":"AGAI_ECOLI"},"6883751":{"id":6883751,"externalId":"511145.b3142","name":"yraH","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YRAH_ECOLI"},"6883752":{"id":6883752,"externalId":"511145.b3143","name":"yraI","hasFamily":true,"annotation":"putative periplasmic pilin chaperone; Could be required for the biogenesis of the putative YraH fimbria","uniprotId":"YRAI_ECOLI"},"6883753":{"id":6883753,"externalId":"511145.b3144","name":"yraJ","hasFamily":true,"annotation":"putative outer membrane protein; Could be involved in the export and assembly of the putative YraH fimbrial subunit across the outer membrane","uniprotId":"YRAJ_ECOLI"},"6883754":{"id":6883754,"externalId":"511145.b3145","name":"yraK","hasFamily":false,"annotation":"putative fimbrial-like adhesin protein","uniprotId":"YRAK_ECOLI"},"6883755":{"id":6883755,"externalId":"511145.b3146","name":"rsmI","hasFamily":true,"annotation":"16S rRNA C1402 2'-O-ribose methyltransferase, SAM-dependent; Catalyzes the 2'-O-methylation of the ribose of cytidine 1402 (C1402) in 16S rRNA. In vitro, active on the assembled 30S subunit, but not naked 16S rRNA or 70S ribosomes","uniprotId":"RSMI_ECOLI"},"6883756":{"id":6883756,"externalId":"511145.b3147","name":"lpoA","hasFamily":true,"annotation":"conserved protein; Regulator of peptidoglycan synthesis that is essential for the function of penicillin-binding protein 1A (PBP1a). Stimulates transpeptidase activity of PBP1a in vitro","uniprotId":"LPOA_ECOLI"},"6883757":{"id":6883757,"externalId":"511145.b3148","name":"yraN","hasFamily":true,"annotation":"conserved protein, UPF0102 family","uniprotId":"YRAN_ECOLI"},"6883758":{"id":6883758,"externalId":"511145.b3149","name":"diaA","hasFamily":true,"annotation":"DnaA initiator-associating factor for replication initiation; Required for the timely initiation of chromosomal replication via direct interactions with the DnaA initiator protein (By similarity)","uniprotId":"DIAA_ECOLI"},"6883759":{"id":6883759,"externalId":"511145.b3150","name":"ecfH","hasFamily":true,"annotation":"OM lipoprotein","uniprotId":"YRAP_ECOLI"},"6883760":{"id":6883760,"externalId":"511145.b3151","name":"yraQ","hasFamily":false,"annotation":"putative permease","uniprotId":"YRAQ_ECOLI"},"6883761":{"id":6883761,"externalId":"511145.b3152","name":"yraR","hasFamily":true,"annotation":"putative nucleoside-diphosphate-sugar epimerase","uniprotId":"YRAR_ECOLI"},"6883762":{"id":6883762,"externalId":"511145.b3153","name":"yhbO","hasFamily":true,"annotation":"stress-resistance protein","uniprotId":"YHBO_ECOLI"},"6883763":{"id":6883763,"externalId":"511145.b3154","name":"yhbP","hasFamily":true,"annotation":"conserved protein, UPF0306 family","uniprotId":"YHBP_ECOLI"},"6883764":{"id":6883764,"externalId":"511145.b3155","name":"yhbQ","hasFamily":true,"annotation":"conserved protein, GIY-YIG nuclease superfamily","uniprotId":"YHBQ_ECOLI"},"6883765":{"id":6883765,"externalId":"511145.b3156","name":"yhbS","hasFamily":true,"annotation":"putative acyltransferase with acyl-CoA N-acyltransferase domain","uniprotId":"YHBS_ECOLI"},"6883766":{"id":6883766,"externalId":"511145.b3157","name":"yhbT","hasFamily":true,"annotation":"putative lipid carrier protein, COG3154 family","uniprotId":"YHBT_ECOLI"},"6883767":{"id":6883767,"externalId":"511145.b3158","name":"yhbU","hasFamily":true,"annotation":"putative peptidase (collagenase-like)","uniprotId":"YHBU_ECOLI"},"6883768":{"id":6883768,"externalId":"511145.b3159","name":"yhbV","hasFamily":true,"annotation":"putative protease","uniprotId":"YHBV_ECOLI"},"6883769":{"id":6883769,"externalId":"511145.b3160","name":"yhbW","hasFamily":true,"annotation":"putative enzyme","uniprotId":"YHBW_ECOLI"},"6883770":{"id":6883770,"externalId":"511145.b3161","name":"mtr","hasFamily":true,"annotation":"tryptophan transporter of high affinity; Involved in transporting tryptophan across the cytoplasmic membrane","uniprotId":"MTR_ECOLI"},"6883771":{"id":6883771,"externalId":"511145.b3162","name":"deaD","hasFamily":true,"annotation":"ATP-dependent RNA helicase; Acts as an ATP-dependent RNA helicase, able to unwind ds-RNA. May play a role in the RNA degradosome at low temperatures","uniprotId":"DEAD_ECOLI"},"6883772":{"id":6883772,"externalId":"511145.b3163","name":"nlpI","hasFamily":true,"annotation":"lipoprotein involved in osmotic sensitivity and filamentation; May be involved in cell division. May play a role in bacterial septation or regulation of cell wall degradation during cell division. Negatively controls the production of extracellular DNA (eDNA)","uniprotId":"NLPI_ECOLI"},"6883773":{"id":6883773,"externalId":"511145.b3164","name":"pnp","hasFamily":true,"annotation":"polynucleotide phosphorylase/polyadenylase; Involved in mRNA degradation. Hydrolyzes single-stranded polyribonucleotides processively in the 3'- to 5'-direction (By similarity)","uniprotId":"PNP_ECOLI"},"6883774":{"id":6883774,"externalId":"511145.b3165","name":"rpsO","hasFamily":true,"annotation":"30S ribosomal subunit protein S15; One of the primary rRNA binding proteins, it binds directly to 16S rRNA where it helps nucleate assembly of the platform of the 30S subunit by binding and bridging several RNA helices of the 16S rRNA","uniprotId":"RS15_ECOLI"},"6883775":{"id":6883775,"externalId":"511145.b3166","name":"truB","hasFamily":true,"annotation":"tRNA U55 pseudouridine synthase; Responsible for synthesis of pseudouridine from uracil- 55 in the psi GC loop of transfer RNAs (By similarity)","uniprotId":"TRUB_ECOLI"},"6883776":{"id":6883776,"externalId":"511145.b3167","name":"rbfA","hasFamily":true,"annotation":"30s ribosome binding factor; Essential for efficient processing of 16S rRNA. Probably part of the 30S subunit prior to or during the final step in the processing of 16S free 30S ribosomal subunits. Could be some accessory protein needed for efficient assembly of the 30S subunit. May interact with the 5'-terminal helix region of 16S rRNA. Has affinity for free ribosomal 30S subunits but not for 70S ribosomes. Overexpression suppresses a cold-sensitive C23U 16S rRNA mutation and rimM deletion mutants. Its function probably overlaps Era","uniprotId":"RBFA_ECOLI"},"6883777":{"id":6883777,"externalId":"511145.b3168","name":"infB","hasFamily":true,"annotation":"fused protein chain initiation factor 2, IF2: membrane protein/conserved protein; One of the essential components for the initiation of protein synthesis. Protects formylmethionyl-tRNA from spontaneous hydrolysis and promotes its binding to the 30S ribosomal subunits. Also involved in the hydrolysis of GTP during the formation of the 70S ribosomal complex (By similarity)"},"6883778":{"id":6883778,"externalId":"511145.b3169","name":"nusA","hasFamily":true,"annotation":"transcription termination/antitermination L factor; Participates in both transcription termination and antitermination. Involved in a variety of cellular and viral termination and antitermination processes, such as Rho-dependent transcriptional termination, intrinsic termination, and phage lambda N-mediated transcriptional antitermination. Also important for coordinating the cellular responses to DNA damage by coupling the processes of nucleotide excision repair and translesion synthesis to transcription","uniprotId":"NUSA_ECOLI"},"6883779":{"id":6883779,"externalId":"511145.b3170","name":"rimP","hasFamily":true,"annotation":"ribosome maturation factor for 30S subunits; Required for maturation of 30S ribosomal subunits, probably at a late stage of ribosomal protein binding, while Era is associated and after RimM","uniprotId":"RIMP_ECOLI"},"6883780":{"id":6883780,"externalId":"511145.b3172","name":"argG","hasFamily":true,"annotation":"argininosuccinate synthetase","uniprotId":"ASSY_ECOLI"},"6883781":{"id":6883781,"externalId":"511145.b3173","name":"yhbX","hasFamily":true,"annotation":"putative hydrolase, inner membrane","uniprotId":"YHBX_ECOLI"},"6883782":{"id":6883782,"externalId":"511145.b3175","name":"secG","hasFamily":true,"annotation":"preprotein translocase membrane subunit; Subunit of the protein translocation channel SecYEG. Overexpression of some hybrid proteins has been thought to jam the protein secretion apparatus resulting in cell death; while this may be true it also results in FtsH-mediated degradation of SecY. Treatment with antibiotics that block translation elongation such as chloramphenicol also leads to degradation of SecY and SecE but not SecG","uniprotId":"SECG_ECOLI"},"6883783":{"id":6883783,"externalId":"511145.b3176","name":"glmM","hasFamily":true,"annotation":"phosphoglucosamine mutase; Catalyzes the conversion of glucosamine-6-phosphate to glucosamine-1-phosphate. Can also catalyze the formation of glucose-6-P from glucose-1-P, although at a 1400-fold lower rate","uniprotId":"GLMM_ECOLI"},"6883784":{"id":6883784,"externalId":"511145.b3177","name":"dhps","hasFamily":true,"annotation":"7,8-dihydropteroate synthase; DHPS catalyzes the formation of the immediate precursor of folic acid. It is implicated in resistance to sulfonamide","uniprotId":"DHPS_ECOLI"},"6883785":{"id":6883785,"externalId":"511145.b3178","name":"ftsH","hasFamily":true,"annotation":"protease, ATP-dependent zinc-metallo; Acts as a processive, ATP-dependent zinc metallopeptidase for both cytoplasmic and membrane proteins. Plays a role in the quality control of integral membrane proteins. Degrades a few membrane proteins that have not been assembled into complexes such as SecY, F(0) ATPase subunit a and YccA, and also cytoplasmic proteins sigma-32, LpxC, KdtA and phage lambda cII protein among others. Degrades membrane proteins in a processive manner starting at either the N- or C-terminus; recognition requires a cytoplasmic tail of about 20 residues with no apparent [...] ","uniprotId":"FTSH_ECOLI"},"6883786":{"id":6883786,"externalId":"511145.b3179","name":"rlmE","hasFamily":true,"annotation":"23S rRNA U2552 ribose 2'-O-methyltransferase, SAM-dependent; Specifically methylates the uridine in position 2552 of 23S rRNA at the 2'-O position of the ribose in the fully assembled 50S ribosomal subunit (By similarity)","uniprotId":"RLME_ECOLI"},"6883787":{"id":6883787,"externalId":"511145.b3180","name":"yhbY","hasFamily":true,"annotation":"RNA binding protein associated with pre-50S ribosomal subunits","uniprotId":"YHBY_ECOLI"},"6883788":{"id":6883788,"externalId":"511145.b3181","name":"greA","hasFamily":true,"annotation":"transcript cleavage factor; Necessary for efficient RNA polymerase transcription elongation past template-encoded arresting sites. The arresting sites in DNA have the property of trapping a certain fraction of elongating RNA polymerases that pass through, resulting in locked ternary complexes. Cleavage of the nascent transcript by cleavage factors such as GreA or GreB allows the resumption of elongation from the new 3'terminus. GreA releases sequences of 2 to 3 nucleotides (By similarity)","uniprotId":"GREA_ECOLI"},"6883789":{"id":6883789,"externalId":"511145.b3182","name":"dacB","hasFamily":true,"annotation":"D-alanyl-D-alanine carboxypeptidase; Not involved in transpeptidation but exclusively catalyzes a DD-carboxypeptidase and DD-endopeptidase reaction","uniprotId":"DACB_ECOLI"},"6883790":{"id":6883790,"externalId":"511145.b3183","name":"obgE","hasFamily":true,"annotation":"GTPase involved in cell partioning and DNA repair; An abundant, essential GTPase which binds GTP, GDP and ppGpp with moderate affinity, shows high guanine nucleotide exchange rate constants for GTP and GDP, and has a relatively low GTP hydrolysis rate. It is estimated there are 34000 molecules in log-phase cells and 5600 molecules in stationary-phase cells as the monomer. Required for chromosome segregation. It also plays a role in the stringent response, and a non-essential role in the late steps of ribosome biogenesis, but how it performs these functions remains unelucidated","uniprotId":"OBG_ECOLI"},"6883791":{"id":6883791,"externalId":"511145.b3184","name":"yhbE","hasFamily":true,"annotation":"putative inner membrane permease","uniprotId":"YHBE_ECOLI"},"6883792":{"id":6883792,"externalId":"511145.b3185","name":"rpmA","hasFamily":true,"annotation":"50S ribosomal subunit protein L27","uniprotId":"RL27_ECOLI"},"6883793":{"id":6883793,"externalId":"511145.b3186","name":"rplU","hasFamily":true,"annotation":"50S ribosomal subunit protein L21; This protein binds to 23S rRNA in the presence of protein L20 (By similarity)","uniprotId":"RL21_ECOLI"},"6883794":{"id":6883794,"externalId":"511145.b3187","name":"ispB","hasFamily":true,"annotation":"octaprenyl diphosphate synthase; Supplies octaprenyl diphosphate, the precursor for the side chain of the isoprenoid quinones ubiquinone and menaquinone","uniprotId":"ISPB_ECOLI"},"6883795":{"id":6883795,"externalId":"511145.b3188","name":"nlp","hasFamily":true,"annotation":"DNA-binding transcriptional activator of maltose metabolism; This protein is involved in positive regulation of the metabolism of sugars (By similarity)","uniprotId":"SFSB_ECOLI"},"6883796":{"id":6883796,"externalId":"511145.b3189","name":"murA","hasFamily":true,"annotation":"UDP-N-acetylglucosamine 1-carboxyvinyltransferase; Cell wall formation. Adds enolpyruvyl to UDP-N- acetylglucosamine. Target for the antibiotic phosphomycin","uniprotId":"MURA_ECOLI"},"6883797":{"id":6883797,"externalId":"511145.b3190","name":"yrbA","hasFamily":true,"annotation":"acid stress protein","uniprotId":"YRBA_ECOLI"},"6883798":{"id":6883798,"externalId":"511145.b3191","name":"mlaB","hasFamily":true,"annotation":"ABC transporter maintaining OM lipid asymmetry, cytoplasmic STAS component; Part of the ABC transporter complex MlaFEDB that actively prevents phospholipid accumulation at the cell surface. Probably maintains lipid asymmetry in the outer membrane by retrograde trafficking of phospholipids from the outer membrane to the inner membrane","uniprotId":"MLAB_ECOLI"},"6883799":{"id":6883799,"externalId":"511145.b3192","name":"mlaC","hasFamily":true,"annotation":"ABC transporter maintaining OM lipid asymmetry, periplasmic binding protein; Actively prevents phospholipid accumulation at the cell surface. Probably maintains lipid asymmetry in the outer membrane by retrograde trafficking of phospholipids from the outer membrane to the inner membrane. May transfer phospholipid across the periplasmic space and deliver it to the MlaFEDB complex at the inner membrane","uniprotId":"MLAC_ECOLI"},"6883800":{"id":6883800,"externalId":"511145.b3193","name":"mlaD","hasFamily":true,"annotation":"ABC transporter maintaining OM lipid asymmetry, anchored periplasmic binding protein; Part of the ABC transporter complex MlaFEDB that actively prevents phospholipid accumulation at the cell surface. Probably maintains lipid asymmetry in the outer membrane by retrograde trafficking of phospholipids from the outer membrane to the inner membrane","uniprotId":"MLAD_ECOLI"},"6883801":{"id":6883801,"externalId":"511145.b3194","name":"mlaE","hasFamily":true,"annotation":"ABC transporter maintaining OM lipid asymmetry, inner membrane permease protein; Part of the ABC transporter complex MlaFEDB that actively prevents phospholipid accumulation at the cell surface. Probably maintains lipid asymmetry in the outer membrane by retrograde trafficking of phospholipids from the outer membrane to the inner membrane","uniprotId":"MLAE_ECOLI"},"6883802":{"id":6883802,"externalId":"511145.b3195","name":"mlaF","hasFamily":true,"annotation":"ABC transporter maintaining OM lipid asymmetry, ATP-binding protein; Part of the ABC transporter complex MlaFEDB that actively prevents phospholipid accumulation at the cell surface. Probably maintains lipid asymmetry in the outer membrane by retrograde trafficking of phospholipids from the outer membrane to the inner membrane. Probably responsible for energy coupling to the transport system","uniprotId":"MLAF_ECOLI"},"6883803":{"id":6883803,"externalId":"511145.b3196","name":"yrbG","hasFamily":true,"annotation":"putative calcium/sodium:proton antiporter","uniprotId":"YRBG_ECOLI"},"6883804":{"id":6883804,"externalId":"511145.b3197","name":"kdsD","hasFamily":true,"annotation":"D-arabinose 5-phosphate isomerase; Involved in the biosynthesis of 3-deoxy-D-manno- octulosonate (KDO), a unique 8-carbon sugar component of lipopolysaccharides (LPSs). KdsD is not essential in the KDO biosynthesis and can be substituted by GutQ. Catalyzes the reversible aldol-ketol isomerization between D-ribulose 5- phosphate (Ru5P) and D-arabinose 5-phosphate (A5P)","uniprotId":"KDSD_ECOLI"},"6883805":{"id":6883805,"externalId":"511145.b3198","name":"kdsC","hasFamily":true,"annotation":"3-deoxy-D-manno-octulosonate 8-phosphate phosphatase; Involved in the biosynthesis of lipopolysaccharides (LPSs), but is not essential. Catalyzes the hydrolysis of 3-deoxy- D-manno-octulosonate 8-phosphate (KDO 8-P) to 3-deoxy-D-manno- octulosonate (KDO) and inorganic phosphate","uniprotId":"KDSC_ECOLI"},"6883806":{"id":6883806,"externalId":"511145.b3199","name":"lptC","hasFamily":true,"annotation":"lipopolysaccharide export, IM-tethered periplasmic protein of the LptBFGC export complex; Required for the translocation of lipopolysaccharide (LPS) from the inner membrane to the outer membrane. It facilitates the transfer of LPS from the inner membrane to a periplasmic chaperone, LptA","uniprotId":"LPTC_ECOLI"},"6883807":{"id":6883807,"externalId":"511145.b3200","name":"lptA","hasFamily":true,"annotation":"periplasmic LPS-binding protein; Required for the translocation of lipopolysaccharide (LPS) from the inner membrane to the outer membrane. May act as a chaperone that facilitates LPS transfer across the aquaeous environment of the periplasm. Interacts specifically with the lipid A domain of LPS","uniprotId":"LPTA_ECOLI"},"6883808":{"id":6883808,"externalId":"511145.b3201","name":"lptB","hasFamily":true,"annotation":"lipopolysaccharide export ABC transporter ATP-binding protein of the LptBFGC export complex; Part of the ABC transporter complex LptBFG involved in the translocation of lipopolysaccharide (LPS) from the inner membrane to the outer membrane. Probably responsible for energy coupling to the transport system","uniprotId":"LPTB_ECOLI"},"6883809":{"id":6883809,"externalId":"511145.b3202","name":"rpoN","hasFamily":true,"annotation":"RNA polymerase, sigma 54 (sigma N) factor; Sigma factors are initiation factors that promote the attachment of RNA polymerase to specific initiation sites and are then released. This sigma factor is responsible for the expression of enzymes involved in arginine catabolism. The open complex (sigma-54 and core RNA polymerase) serves as the receptor for the receipt of the melting signal from the remotely bound activator protein GlnG(NtrC)","uniprotId":"RP54_ECOLI"},"6883810":{"id":6883810,"externalId":"511145.b3203","name":"hpf","hasFamily":true,"annotation":"ribosome hibernation promoting factor HPF; stabilizes 70S dimers (100S); During stationary phase, promotes and stabilizes dimerization of 70S ribosomes by the ribosome modulation factor (RMF), leading to the formation of inactive 100S ribosomes. May convert immature 90S particles formed by RMF into 100S ribosomes. May also function as a potential translational inhibitor","uniprotId":"HPF_ECOLI"},"6883811":{"id":6883811,"externalId":"511145.b3204","name":"ptsN","hasFamily":true,"annotation":"sugar-specific enzyme IIA component of PTS; Seems to have a role in regulating nitrogen assimilation (By similarity)","uniprotId":"PTSN_ECOLI"},"6883812":{"id":6883812,"externalId":"511145.b3205","name":"yhbJ","hasFamily":true,"annotation":"glmZ(sRNA)-inactivating NTPase, glucosamine-6-phosphate regulated; Displays ATPase and GTPase activities. Can also hydrolyze pNPP. May regulate expression of glmS, by affecting the processing and stability of a small RNA, glmZ","uniprotId":"YHBJ_ECOLI"},"6883813":{"id":6883813,"externalId":"511145.b3206","name":"npr","hasFamily":true,"annotation":"phosphohistidinoprotein-hexose phosphotransferase component of N-regulated PTS system (Npr); Component of the phosphoenolpyruvate-dependent nitrogen- metabolic phosphotransferase system (nitrogen-metabolic PTS), that seems to be involved in regulating nitrogen metabolism. The phosphoryl group from phosphoenolpyruvate (PEP) is transferred to the phosphoryl carrier protein NPr by enzyme I-Ntr. Phospho-NPr then transfers it to EIIA-Ntr. Could function in the transcriptional regulation of sigma-54 dependent operons in conjunction with the NPr (PtsO) and EIIA-Ntr (PtsN) proteins","uniprotId":"PTSO_ECOLI"},"6883814":{"id":6883814,"externalId":"511145.b3207","name":"yrbL","hasFamily":true,"annotation":"predicted protein","uniprotId":"YRBL_ECOLI"},"6883815":{"id":6883815,"externalId":"511145.b3208","name":"mtgA","hasFamily":true,"annotation":"biosynthetic peptidoglycan transglycosylase; Cell wall formation (By similarity)","uniprotId":"MTGA_ECOLI"},"6883816":{"id":6883816,"externalId":"511145.b3209","name":"elbB","hasFamily":true,"annotation":"isoprenoid biosynthesis protein with amidotransferase-like domain; May be involved in the early steps of isoprenoid biosynthesis","uniprotId":"ELBB_ECOLI"},"6883817":{"id":6883817,"externalId":"511145.b3210","name":"arcB","hasFamily":true,"annotation":"aerobic respiration control sensor histidine protein kinase, cognate to two-component response regulators ArcA and RssB; Member of the two-component regulatory system ArcB/ArcA. Sensor-regulator protein for anaerobic repression of the arc modulon. Activates ArcA via a four-step phosphorelay. ArcB can also dephosphorylate ArcA by a reverse phosphorelay involving His- 717 and Asp-576","uniprotId":"ARCB_ECOLI"},"6883818":{"id":6883818,"externalId":"511145.b3211","name":"yhcC","hasFamily":true,"annotation":"putative Fe-S oxidoreductase","uniprotId":"YHCC_ECOLI"},"6883819":{"id":6883819,"externalId":"511145.b3212","name":"gltB","hasFamily":true,"annotation":"glutamate synthase, large subunit","uniprotId":"GLTB_ECOLI"},"6883820":{"id":6883820,"externalId":"511145.b3213","name":"gltD","hasFamily":true,"annotation":"glutamate synthase, 4Fe-4S protein, small subunit","uniprotId":"GLTD_ECOLI"},"6883821":{"id":6883821,"externalId":"511145.b3214","name":"gltF","hasFamily":false,"annotation":"periplasmic protein; Involved in induction of the so-called NTR enzymes in response to nitrogen deprivation, as well as in glutamate biosynthesis. May mediate the glutamate-dependent repression of the GLT operon","uniprotId":"GLTF_ECOLI"},"6883822":{"id":6883822,"externalId":"511145.b3215","name":"yhcA","hasFamily":true,"annotation":"putative periplasmic chaperone protein; Could be required for the biogenesis of a putative fimbria (By similarity)","uniprotId":"YHCA_ECOLI"},"6883823":{"id":6883823,"externalId":"511145.b3216","name":"yhcD","hasFamily":true,"annotation":"putative outer membrane mbrial subunitusher protein; Involved in the export and assembly of a fimbrial subunit across the outer membrane (By similarity)","uniprotId":"YHCD_ECOLI"},"6883824":{"id":6883824,"externalId":"511145.b3218","name":"b3218","hasFamily":true,"annotation":"IS5 transposase and trans-activator"},"6883825":{"id":6883825,"externalId":"511145.b3219","name":"yhcF","hasFamily":true,"annotation":"putative transcriptional regulator","uniprotId":"YHCF_ECOLI"},"6883826":{"id":6883826,"externalId":"511145.b3220","name":"yhcG","hasFamily":true,"annotation":"conserved protein","uniprotId":"YHCG_ECOLI"},"6883827":{"id":6883827,"externalId":"511145.b3221","name":"yhcH","hasFamily":true,"annotation":"conserved protein","uniprotId":"YHCH_ECOLI"},"6883828":{"id":6883828,"externalId":"511145.b3222","name":"nanK","hasFamily":true,"annotation":"N-acetylmannosamine kinase; Catalyzes the phosphorylation of N-acetylmannosamine (ManNAc) to ManNAc-6-P. Has also low level glucokinase activity in vitro","uniprotId":"NANK_ECOLI"},"6883829":{"id":6883829,"externalId":"511145.b3223","name":"nanE","hasFamily":true,"annotation":"putative N-acetylmannosamine-6-P epimerase; Converts N-acetylmannosamine-6-phosphate (ManNAc-6-P) to N-acetylglucosamine-6-phosphate (GlcNAc-6-P) (By similarity)","uniprotId":"NANE_ECOLI"},"6883830":{"id":6883830,"externalId":"511145.b3224","name":"nanT","hasFamily":true,"annotation":"sialic acid transporter; May be a sugar-cation symporter involved in sialic acid transport as it is required for sialic acid uptake","uniprotId":"NANT_ECOLI"},"6883831":{"id":6883831,"externalId":"511145.b3225","name":"nanA","hasFamily":true,"annotation":"N-acetylneuraminate lyase; Catalyzes the cleavage of N-acetylneuraminic acid (sialic acid) to form pyruvate and N-acetylmannosamine via a Schiff base intermediate (By similarity)","uniprotId":"NANA_ECOLI"},"6883832":{"id":6883832,"externalId":"511145.b3226","name":"nanR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor of the nan operon, induced by sialic acid; Transcriptional repressor of the nan operon that encodes proteins involved in sialic acid utilization (By similarity)","uniprotId":"NANR_ECOLI"},"6883833":{"id":6883833,"externalId":"511145.b3227","name":"dcuD","hasFamily":false,"annotation":"putative transporter; May have a role in the transport of dicarboxylates from the periplasm across the inner membrane. Seems not to be essential for growth. It is likely to be a cryptic gene, as no or low expression has been found under a number of growth conditions and when tested with a large number of potential substrates","uniprotId":"DCUD_ECOLI"},"6883834":{"id":6883834,"externalId":"511145.b3228","name":"sspB","hasFamily":true,"annotation":"ClpXP protease specificity enhancing factor; Seems to act in concert with SspA in the regulation of several proteins during exponential and stationary-phase growth. The exact function of SspB is not yet known","uniprotId":"SSPB_ECOLI"},"6883835":{"id":6883835,"externalId":"511145.b3229","name":"sspA","hasFamily":true,"annotation":"stringent starvation protein A; Forms an equimolar complex with the RNA polymerase holoenzyme (RNAP) but not with the core enzyme. It is synthesized predominantly when cells are exposed to amino acid starvation, at which time it accounts for over 50% of the total protein synthesized. It is involved in the transition from P1 early to P1 late gene expression. Rnk and SspA can functionally replace P.aeruginosa alginate regulatory gene algR2","uniprotId":"SSPA_ECOLI"},"6883836":{"id":6883836,"externalId":"511145.b3230","name":"rpsI","hasFamily":true,"annotation":"30S ribosomal subunit protein S9; The C-terminal tail plays a role in the affinity of the 30S P site for different tRNAs. Mutations that decrease this affinity are suppressed in the 70S ribosome","uniprotId":"RS9_ECOLI"},"6883837":{"id":6883837,"externalId":"511145.b3231","name":"rplM","hasFamily":false,"annotation":"50S ribosomal subunit protein L13; This protein is one of the early assembly proteins of the 50S ribosomal subunit, although it is not seen to bind rRNA by itself. It is important during the early stages of 50S assembly (By similarity)","uniprotId":"RL13_ECOLI"},"6883838":{"id":6883838,"externalId":"511145.b3232","name":"yhcM","hasFamily":true,"annotation":"conserved protein with nucleoside triphosphate hydrolase domain","uniprotId":"YHCM_ECOLI"},"6883839":{"id":6883839,"externalId":"511145.b3233","name":"yhcB","hasFamily":true,"annotation":"conserved protein","uniprotId":"YHCB_ECOLI"},"6883840":{"id":6883840,"externalId":"511145.b3234","name":"degQ","hasFamily":true,"annotation":"serine endoprotease, periplasmic; DegQ could degrade transiently denatured and unfolded proteins which accumulate in the periplasm following stress conditions. DegQ is efficient with Val-Xaa and Ile-Xaa peptide bonds, suggesting a preference for a beta-branched side chain amino acids. Only unfolded proteins devoid of disulfide bonds appear capable to be cleaved, thereby preventing non-specific proteolysis of folded proteins. DegQ can substitute for the periplasmic protease DegP","uniprotId":"DEGQ_ECOLI"},"6883841":{"id":6883841,"externalId":"511145.b3235","name":"degS","hasFamily":true,"annotation":"serine endoprotease, periplasmic; When heat shock or other environmental stresses disrupt protein folding in the periplasm, DegS senses the accumulation of unassembled outer membrane porins (OMPs) and then initiates RseA (anti sigma-E factor) degradation by cleaving it in its periplasmic domain, making it an attractive substrate for subsequent cleavage by RseP. This cascade that ultimately leads to the sigma-E-driven expression of a variety of factors dealing with folding stress in the periplasm and OMP assembly","uniprotId":"DEGS_ECOLI"},"6883842":{"id":6883842,"externalId":"511145.b3236","name":"mdh","hasFamily":true,"annotation":"malate dehydrogenase, NAD(P)-binding; Catalyzes the reversible oxidation of malate to oxaloacetate (By similarity)","uniprotId":"Q1PI59_ECOLI"},"6883843":{"id":6883843,"externalId":"511145.b3237","name":"argR","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator, L-arginine-binding; Negatively controls the expression of the four operons of arginine biosynthesis in addition to the carAB operon. Predominantly interacts with A/T residues in ARG boxes. It also binds to a specific site in cer locus. Thus it is essential for cer-mediated site-specific recombination in ColE1. It is necessary for monomerization of the plasmid ColE1","uniprotId":"ARGR_ECOLI"},"6883844":{"id":6883844,"externalId":"511145.b3238","name":"yhcN","hasFamily":true,"annotation":"conserved protein","uniprotId":"YHCN_ECOLI"},"6883845":{"id":6883845,"externalId":"511145.b3239","name":"yhcO","hasFamily":true,"annotation":"putative barnase inhibitor","uniprotId":"YHCO_ECOLI"},"6883846":{"id":6883846,"externalId":"511145.b3240","name":"aaeB","hasFamily":true,"annotation":"p-hydroxybenzoic acid efflux system component; Efflux pump whose substrates are p-hydroxybenzoic acid (pHBA), 6-hydroxy-2-naphthoic and 2-hydroxycinnamate. Could function as a metabolic relief valve, allowing to eliminate certain compounds when they accumulate to high levels in the cell (By similarity)","uniprotId":"AAEB_ECOLI"},"6883847":{"id":6883847,"externalId":"511145.b3241","name":"aaeA","hasFamily":true,"annotation":"p-hydroxybenzoic acid efflux system component","uniprotId":"AAEA_ECOLI"},"6883848":{"id":6883848,"externalId":"511145.b3242","name":"aaeX","hasFamily":true,"annotation":"membrane protein of efflux system","uniprotId":"AAEX_ECOLI"},"6883849":{"id":6883849,"externalId":"511145.b3243","name":"aaeR","hasFamily":true,"annotation":"transcriptional regulator for aaeXAB operon; Activates transcription of the aaeXAB operon","uniprotId":"AAER_ECOLI"},"6883850":{"id":6883850,"externalId":"511145.b3244","name":"tldD","hasFamily":true,"annotation":"putative peptidase; Suppresses the inhibitory activity of the carbon storage regulator (CsrA)","uniprotId":"TLDD_ECOLI"},"6883851":{"id":6883851,"externalId":"511145.b3247","name":"rng","hasFamily":true,"annotation":"ribonuclease G; Involved in the processing of the 5'-end of 16S rRNA. Could be involved in chromosome segregation and cell division. It may be one of the components of the cytoplasmic axial filaments bundles or merely regulate the formation of this structure","uniprotId":"RNG_ECOLI"},"6883852":{"id":6883852,"externalId":"511145.b3248","name":"yhdE","hasFamily":true,"annotation":"Maf-like protein","uniprotId":"YHDE_ECOLI"},"6883853":{"id":6883853,"externalId":"511145.b3249","name":"mreD","hasFamily":true,"annotation":"cell wall structural complex MreBCD transmembrane component MreD; Involved in formation of the rod shape of the cell. May also contribute to regulation of formation of penicillin-binding proteins","uniprotId":"MRED_ECOLI"},"6883854":{"id":6883854,"externalId":"511145.b3250","name":"mreC","hasFamily":true,"annotation":"cell wall structural complex MreBCD transmembrane component MreC; Involved in formation and maintenance of cell shape. Responsible for formation of rod shape. May also contribute to regulation of formation of penicillin-binding proteins","uniprotId":"MREC_ECOLI"},"6883855":{"id":6883855,"externalId":"511145.b3251","name":"mreB","hasFamily":true,"annotation":"cell wall structural complex MreBCD, actin-like component MreB; Involved in formation of the rod shape of the cell. May act as a negative regulator of FtsI. Polymerization and bundle formation is enhanced by CbeA","uniprotId":"MREB_ECOLI"},"6883856":{"id":6883856,"externalId":"511145.b3252","name":"csrD","hasFamily":true,"annotation":"targeting factor for csrBC sRNA degradation; Serves as a specificity factor required for RNase E- mediated decay of the small global regulatory RNAs CsrB and CsrC, it is probably not a nuclease. Nor does its activity involve c-di- GMP, despite its domain composition. Positively modulates motility gene expression, is also required for curli expression","uniprotId":"CSRD_ECOLI"},"6883857":{"id":6883857,"externalId":"511145.b3253","name":"acuI","hasFamily":true,"annotation":"putative acryloyl-CoA reductase; Probably catalyzes the NADPH-dependent reduction of acrylyl-CoA to propanoyl-CoA","uniprotId":"ACUI_ECOLI"},"6883858":{"id":6883858,"externalId":"511145.b3255","name":"accB","hasFamily":true,"annotation":"acetyl CoA carboxylase, BCCP subunit; This protein is a component of the acetyl coenzyme A carboxylase complex; first, biotin carboxylase catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA","uniprotId":"BCCP_ECOLI"},"6883859":{"id":6883859,"externalId":"511145.b3256","name":"accC","hasFamily":true,"annotation":"acetyl-CoA carboxylase, biotin carboxylase subunit; This protein is a component of the acetyl coenzyme A carboxylase complex; first, biotin carboxylase catalyzes the carboxylation of the carrier protein and then the transcarboxylase transfers the carboxyl group to form malonyl-CoA","uniprotId":"ACCC_ECOLI"},"6883860":{"id":6883860,"externalId":"511145.b3257","name":"yhdT","hasFamily":true,"annotation":"conserved inner membrane protein","uniprotId":"YHDT_ECOLI"},"6883861":{"id":6883861,"externalId":"511145.b3258","name":"panF","hasFamily":true,"annotation":"pantothenate:sodium symporter; Catalyzes the sodium-dependent uptake of extracellular pantothenate","uniprotId":"PANF_ECOLI"},"6883862":{"id":6883862,"externalId":"511145.b3259","name":"prmA","hasFamily":true,"annotation":"methyltransferase for 50S ribosomal subunit protein L11; Methylates ribosomal protein L11 (By similarity)","uniprotId":"PRMA_ECOLI"},"6883863":{"id":6883863,"externalId":"511145.b3260","name":"dusB","hasFamily":true,"annotation":"tRNA-dihydrouridine synthase B; Catalyzes the synthesis of dihydrouridine, a modified base found in the D-loop of most tRNAs (By similarity)","uniprotId":"DUSB_ECOLI"},"6883864":{"id":6883864,"externalId":"511145.b3261","name":"fis","hasFamily":false,"annotation":"global DNA-binding transcriptional dual regulator; Activates ribosomal RNA transcription, as well other genes. Plays a direct role in upstream activation of rRNA promoters. Binds to a recombinational enhancer sequence that is required to stimulate hin-mediated DNA inversion. Prevents initiation of DNA replication from oriC","uniprotId":"FIS_ECOLI"},"6883865":{"id":6883865,"externalId":"511145.b3262","name":"yhdJ","hasFamily":true,"annotation":"DNA adenine methyltransferase, SAM-dependent; Methylates the second adenine of the NsiI recognition sequence (5'-ATGCAT-3')","uniprotId":"YHDJ_ECOLI"},"6883866":{"id":6883866,"externalId":"511145.b3263","name":"yhdU","hasFamily":true,"annotation":"putative membrane protein","uniprotId":"YHDU_ECOLI"},"6883867":{"id":6883867,"externalId":"511145.b3264","name":"envR","hasFamily":true,"annotation":"DNA-binding transcriptional regulator; Potential regulator protein for the acrEF/envCD genes","uniprotId":"ENVR_ECOLI"},"6883868":{"id":6883868,"externalId":"511145.b3265","name":"acrE","hasFamily":true,"annotation":"cytoplasmic membrane lipoprotein; May affect specific membrane functions, such as septum formation during cell division, and cell membrane permeability","uniprotId":"ACRE_ECOLI"},"6883869":{"id":6883869,"externalId":"511145.b3266","name":"acrF","hasFamily":true,"annotation":"multidrug efflux system protein; Involved in cell envelope formation. Is produced in extremely low amounts","uniprotId":"ACRF_ECOLI"},"6883870":{"id":6883870,"externalId":"511145.b3267","name":"yhdV","hasFamily":true,"annotation":"putative outer membrane protein","uniprotId":"YHDV_ECOLI"},"6883871":{"id":6883871,"externalId":"511145.b3269","name":"yhdX","hasFamily":true,"annotation":"putative amino-acid transporter subunit; Probably part of the binding-protein-dependent transport system YdhWXYZ for an amino acid; probably responsible for the translocation of the substrate across the membrane","uniprotId":"YHDX_ECOLI"},"6883872":{"id":6883872,"externalId":"511145.b3270","name":"yhdY","hasFamily":true,"annotation":"putative amino-acid transporter subunit; Probably part of the binding-protein-dependent transport system YdhWXYZ for an amino acid; probably responsible for the translocation of the substrate across the membrane","uniprotId":"YHDY_ECOLI"},"6883873":{"id":6883873,"externalId":"511145.b3271","name":"yhdZ","hasFamily":true,"annotation":"putative amino-acid transporter subunit; Probably part of a binding-protein-dependent transport system YdhWXYZ for an amino acid. Probably responsible for energy coupling to the transport system","uniprotId":"YHDZ_ECOLI"},"6883874":{"id":6883874,"externalId":"511145.b3279","name":"yrdA","hasFamily":true,"annotation":"conserved protein","uniprotId":"YRDA_ECOLI"},"6883875":{"id":6883875,"externalId":"511145.b3280","name":"yrdB","hasFamily":true,"annotation":"conserved protein","uniprotId":"YRDB_ECOLI"},"6883876":{"id":6883876,"externalId":"511145.b3281","name":"aroE","hasFamily":true,"annotation":"dehydroshikimate reductase, NAD(P)-binding","uniprotId":"AROE_ECOLI"},"6883877":{"id":6883877,"externalId":"511145.b3282","name":"rimN","hasFamily":true,"annotation":"tRNA(ANN) t(6)A37 threonylcarbamoyladenosine modification protein; Required for the formation of a threonylcarbamoyl group on adenosine at position 37 (t(6)A37) in tRNAs that read codons beginning with adenine. Binds ATP, and to fully modified tRNA(Thr) and partially modified tRNA(Thr). Could also be required for the maturation of 16S rRNA. Binds to double-stranded RNA but does not interact tightly with either of the ribosomal subunits, or the 70S particles","uniprotId":"RIMN_ECOLI"},"6883878":{"id":6883878,"externalId":"511145.b3283","name":"yrdD","hasFamily":true,"annotation":"putative DNA topoisomerase","uniprotId":"YRDD_ECOLI"},"6883879":{"id":6883879,"externalId":"511145.b3284","name":"smg","hasFamily":true,"annotation":"conserved protein","uniprotId":"SMG_ECOLI"},"6883880":{"id":6883880,"externalId":"511145.b3287","name":"def","hasFamily":true,"annotation":"peptide deformylase; Removes the formyl group from the N-terminal Met of newly synthesized proteins. Requires at least a dipeptide for an efficient rate of reaction. N-terminal L-methionine is a prerequisite for activity but the enzyme has broad specificity at other positions (By similarity)","uniprotId":"DEF_ECOLI"},"6883881":{"id":6883881,"externalId":"511145.b3288","name":"fmt","hasFamily":true,"annotation":"10-formyltetrahydrofolate:L-methionyl-tRNA(fMet) N-formyltransferase; Modifies the free amino group of the aminoacyl moiety of methionyl-tRNA(fMet). The formyl group appears to play a dual role in the initiator identity of N-formylmethionyl-tRNA by: (I) promoting its recognition by IF2 and (II) impairing its binding to EFTu-GTP (By similarity)","uniprotId":"FMT_ECOLI"},"6883882":{"id":6883882,"externalId":"511145.b3289","name":"sun","hasFamily":true,"annotation":"16S rRNA m(5)C967 methyltransferase, SAM-dependent (EC:2.1.1.61); Specifically methylates the cytosine at position 967 (m5C967) of 16S rRNA (By similarity)","uniprotId":"RSMB_ECOLI"},"6883883":{"id":6883883,"externalId":"511145.b3290","name":"trkA","hasFamily":true,"annotation":"NAD-binding component of TrK potassium transporter; Part of the constitutive potassium transport systems TrkG and TrkH. May regulate the transport activity of TrkG and TrkH systems. Binds to NAD(+) and NADH","uniprotId":"TRKA_ECOLI"},"6883884":{"id":6883884,"externalId":"511145.b3291","name":"mscL","hasFamily":true,"annotation":"mechanosensitive channel protein, high conductance; Mechanosensitive channel that opens in response to stretch forces in the membrane lipid bilayer. Participates in the regulation of osmotic pressure changes within the cell. Forms a nonselective ion channel of 2.5 nanosiemens conductance. Opens at a pressure just below that which would cause cell disruption and death","uniprotId":"MSCL_ECOLI"},"6883885":{"id":6883885,"externalId":"511145.b3292","name":"yhdM","hasFamily":true,"annotation":"DNA-binding transcriptional activator in response to Zn(II); Zinc-responsive transcriptional regulator of zntA","uniprotId":"ZNTR_ECOLI"},"6883886":{"id":6883886,"externalId":"511145.b3293","name":"yhdN","hasFamily":true,"annotation":"conserved protein, DUF1992 family","uniprotId":"YHDN_ECOLI"},"6883887":{"id":6883887,"externalId":"511145.b3294","name":"rplQ","hasFamily":true,"annotation":"50S ribosomal subunit protein L17; Requires L15 for assembly into the 50S subunit","uniprotId":"RL17_ECOLI"},"6883888":{"id":6883888,"externalId":"511145.b3295","name":"rpoA","hasFamily":false,"annotation":"RNA polymerase, alpha subunit; DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates. This subunit plays an important role in subunit assembly since its dimerization is the first step in the sequential assembly of subunits to form the holoenzyme","uniprotId":"RPOA_ECOLI"},"6883889":{"id":6883889,"externalId":"511145.b3296","name":"rpsD","hasFamily":true,"annotation":"30S ribosomal subunit protein S4; Also functions as a rho-dependent antiterminator of rRNA transcription, increasing the synthesis of rRNA under conditions of excess protein, allowing a more rapid return to homeostasis. Binds directly to RNA polymerase","uniprotId":"RS4_ECOLI"},"6883890":{"id":6883890,"externalId":"511145.b3297","name":"rpsK","hasFamily":true,"annotation":"30S ribosomal subunit protein S11; Located on the platform of the 30S subunit, it bridges several disparate RNA helices of the 16S rRNA. Forms part of the Shine-Dalgarno cleft in the 70S ribosome (By similarity)","uniprotId":"RS11_ECOLI"},"6883891":{"id":6883891,"externalId":"511145.b3298","name":"rpsM","hasFamily":true,"annotation":"30S ribosomal subunit protein S13; Located at the top of the head of the 30S subunit, it contacts several helices of the 16S rRNA. In the 70S ribosome it contacts the 23S rRNA (bridge B1a) and protein L5 of the 50S subunit (bridge B1b), connecting the 2 subunits; these bridges are implicated in subunit movement. Contacts the tRNAs in the A and P- sites (By similarity)","uniprotId":"RS13_ECOLI"},"6883892":{"id":6883892,"externalId":"511145.b3299","name":"rpmJ","hasFamily":false,"annotation":"50S ribosomal subunit protein L36","uniprotId":"RL36_ECOLI"},"6883893":{"id":6883893,"externalId":"511145.b3300","name":"prlA","hasFamily":true,"annotation":"preprotein translocase membrane subunit; The central subunit of the protein translocation channel SecYEG. Consists of two halves formed by TMs 1-5 and 6-10. These two domains form a lateral gate at the front which open onto the bilayer between TMs 2 and 7, and are clamped together by SecE at the back. The channel is closed by both a pore ring composed of hydrophobic SecY resides and a short helix (helix 2A) on the extracellular side of the membrane which forms a plug. The plug probably moves laterally to allow the channel to open. The ring and the pore may move independently. SecY is r [...] ","uniprotId":"SECY_ECOLI"},"6883894":{"id":6883894,"externalId":"511145.b3301","name":"rplO","hasFamily":true,"annotation":"50S ribosomal subunit protein L15; This protein binds the 5S rRNA. It is required for the late stages of subunit assembly, and is essential for 5S rRNA assembly onto the ribosome","uniprotId":"RL15_ECOLI"},"6883895":{"id":6883895,"externalId":"511145.b3302","name":"rpmD","hasFamily":true,"annotation":"50S ribosomal subunit protein L30","uniprotId":"RL30_ECOLI"},"6883896":{"id":6883896,"externalId":"511145.b3303","name":"rpsE","hasFamily":false,"annotation":"30S ribosomal subunit protein S5; With S4 and S12 plays an important role in translational accuracy. Many suppressors of streptomycin-dependent mutants of protein S12 are found in this protein, some but not all of which decrease translational accuracy (ram, ribosomal ambiguity mutations)","uniprotId":"RS5_ECOLI"},"6883897":{"id":6883897,"externalId":"511145.b3304","name":"rplR","hasFamily":true,"annotation":"50S ribosomal subunit protein L18; This is one of the proteins that mediates the attachment of the 5S rRNA subcomplex onto the large ribosomal subunit where it forms part of the central protuberance. Binds stably to 5S rRNA; increases binding abilities of L5 in a cooperative fashion; both proteins together confer 23S rRNA binding. The 5S rRNA and some of its associated proteins might help stabilize positioning of ribosome-bound tRNAs","uniprotId":"RL18_ECOLI"},"6883898":{"id":6883898,"externalId":"511145.b3305","name":"rplF","hasFamily":true,"annotation":"50S ribosomal subunit protein L6; This protein binds directly to at least 2 domains of the 23S ribosomal RNA, thus is important in its secondary structure. It is located near the subunit interface in the base of the L7/L12 stalk, and near the tRNA binding site of the peptidyltransferase center","uniprotId":"RL6_ECOLI"},"6883899":{"id":6883899,"externalId":"511145.b3306","name":"rpsH","hasFamily":false,"annotation":"30S ribosomal subunit protein S8; One of the primary rRNA binding proteins, it binds directly to 16S rRNA central domain where it helps coordinate assembly of the platform of the 30S subunit (By similarity)","uniprotId":"RS8_ECOLI"},"6883900":{"id":6883900,"externalId":"511145.b3307","name":"rpsN","hasFamily":true,"annotation":"30S ribosomal subunit protein S14; Binds 16S rRNA, required for the assembly of 30S particles and may also be responsible for determining the conformation of the 16S rRNA at the A site (By similarity)","uniprotId":"RS14_ECOLI"},"6883901":{"id":6883901,"externalId":"511145.b3308","name":"rplE","hasFamily":true,"annotation":"50S ribosomal subunit protein L5; This is 1 of the proteins that binds and probably mediates the attachment of the 5S RNA into the large ribosomal subunit, where it forms part of the central protuberance. In the 70S ribosome it contacts protein S13 of the 30S subunit (bridge B1b), connecting the 2 subunits; this bridge is implicated in subunit movement. Contacts the P site tRNA; the 5S rRNA and some of its associated proteins might help stabilize positioning of ribosome-bound tRNAs (By similarity)","uniprotId":"RL5_ECOLI"},"6883902":{"id":6883902,"externalId":"511145.b3309","name":"rplX","hasFamily":true,"annotation":"50S ribosomal subunit protein L24; One of two assembly initiator proteins, it binds directly to the 5'-end of the 23S rRNA, where it nucleates assembly of the 50S subunit. It is not thought to be involved in the functions of the mature 50S subunit in vitro","uniprotId":"RL24_ECOLI"},"6883903":{"id":6883903,"externalId":"511145.b3310","name":"rplN","hasFamily":true,"annotation":"50S ribosomal subunit protein L14; This protein binds directly to 23S ribosomal RNA. In the E.coli 70S ribosome (PubMed:12809609) it has been modeled to make two contacts with the 16S rRNA of the 30S subunit, forming part of bridges B5 and B8, connecting the 2 subunits. Although the protein undergoes significant rotation during the transition from an initiation to and EF-G bound state, the bridges remain stable. In the 3.5 A resolved structures (PubMed:16272117) L14 and L19 interact and together make contact with the 16S rRNA in bridges B5 and B8","uniprotId":"RL14_ECOLI"},"6883904":{"id":6883904,"externalId":"511145.b3311","name":"rpsQ","hasFamily":true,"annotation":"30S ribosomal subunit protein S17; One of the primary rRNA binding proteins, it binds specifically to the 5'-end of 16S ribosomal RNA. Also plays a role in translational accuracy; neamine-resistant ribosomes show reduced neamine-induced misreading in vitro","uniprotId":"RS17_ECOLI"},"6883905":{"id":6883905,"externalId":"511145.b3312","name":"rpmC","hasFamily":true,"annotation":"50S ribosomal subunit protein L29; Binds 23S rRNA. It is not essential for growth","uniprotId":"RL29_ECOLI"},"6883906":{"id":6883906,"externalId":"511145.b3313","name":"rplP","hasFamily":true,"annotation":"50S ribosomal subunit protein L16; This protein binds directly to 23S ribosomal RNA and is located at the A site of the peptidyltransferase center. It contacts the A and P site tRNAs. It has an essential role in subunit assembly, which is not well understood","uniprotId":"RL16_ECOLI"},"6883907":{"id":6883907,"externalId":"511145.b3314","name":"rpsC","hasFamily":true,"annotation":"30S ribosomal subunit protein S3; Binds the lower part of the 30S subunit head. Binds mRNA in the 70S ribosome, positioning it for translation (By similarity)","uniprotId":"RS3_ECOLI"},"6883908":{"id":6883908,"externalId":"511145.b3315","name":"rplV","hasFamily":true,"annotation":"50S ribosomal subunit protein L22; This protein binds specifically to 23S rRNA; its binding is stimulated by other ribosomal proteins, e.g. L4, L17, and L20. It is important during the early stages of 50S assembly. It makes multiple contacts with different domains of the 23S rRNA in the assembled 50S subunit and ribosome","uniprotId":"RL22_ECOLI"},"6883909":{"id":6883909,"externalId":"511145.b3316","name":"rpsS","hasFamily":true,"annotation":"30S ribosomal subunit protein S19; In the E.coli 70S ribosome in the initiation state (PubMed:12809609) it has been modeled to contact the 23S rRNA of the 50S subunit forming part of bridge B1a; this bridge is broken in the model with bound EF-G. The 23S rRNA contact site in bridge B1a is modeled to differ in different ribosomal states (PubMed:12859903), contacting alternately S13 or S19. In the 3.5 angstroms resolved ribosome structures (PubMed:16272117) the contacts between L5, S13 and S19 bridge B1b are different, confirming the dynamic nature of this interaction. Bridge B1a is not  [...] ","uniprotId":"RS19_ECOLI"},"6883910":{"id":6883910,"externalId":"511145.b3317","name":"rplB","hasFamily":true,"annotation":"50S ribosomal subunit protein L2; One of the primary rRNA binding proteins. Required for association of the 30S and 50S subunits to form the 70S ribosome, for tRNA binding and peptide bond formation. It has been suggested to have peptidyltransferase activity; this is somewhat controversial. Makes several contacts with the 16S rRNA in the 70S ribosome (By similarity)","uniprotId":"RL2_ECOLI"},"6883911":{"id":6883911,"externalId":"511145.b3318","name":"rplW","hasFamily":true,"annotation":"50S ribosomal subunit protein L23; One of the early assembly proteins, it binds 23S rRNA; is essential for growth. One of the proteins that surround the polypeptide exit tunnel on the outside of the subunit. Acts as the docking site for trigger factor (PubMed:12226666) for Ffh binding to the ribosome (SRP54, PubMed:12756233 and PubMed:12702815) and to nascent polypeptide chains (PubMed:12756233)","uniprotId":"RL23_ECOLI"},"6883912":{"id":6883912,"externalId":"511145.b3319","name":"rplD","hasFamily":true,"annotation":"50S ribosomal subunit protein L4; One of the primary rRNA binding proteins, this protein initially binds near the 5'-end of the 23S rRNA. It is important during the early stages of 50S assembly. It makes multiple contacts with different domains of the 23S rRNA in the assembled 50S subunit and ribosome","uniprotId":"RL4_ECOLI"},"6883913":{"id":6883913,"externalId":"511145.b3320","name":"rplC","hasFamily":true,"annotation":"50S ribosomal subunit protein L3; One of the primary rRNA binding proteins, it binds directly near the 3'-end of the 23S rRNA, where it nucleates assembly of the 50S subunit (By similarity)","uniprotId":"RL3_ECOLI"},"6883914":{"id":6883914,"externalId":"511145.b3321","name":"rpsJ","hasFamily":true,"annotation":"30S ribosomal subunit protein S10; Involved in the binding of tRNA to the ribosomes (By similarity)","uniprotId":"RS10_ECOLI"},"6883915":{"id":6883915,"externalId":"511145.b3322","name":"gspB","hasFamily":false,"annotation":"part of gsp divergon involved in type II protein secretion; Calcium-binding protein that may be required for the initiation of chromosome replication. May play a regulatory role under conditions of derepressed gsp gene expression","uniprotId":"GSPB_ECOLI"},"6883916":{"id":6883916,"externalId":"511145.b3323","name":"gspA","hasFamily":true,"annotation":"general secretory pathway component, cryptic; May play a regulatory role under conditions of derepressed gsp gene expression","uniprotId":"GSPA_ECOLI"},"6883917":{"id":6883917,"externalId":"511145.b3324","name":"gspC","hasFamily":true,"annotation":"general secretory pathway component, cryptic; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (Potential)","uniprotId":"GSPC_ECOLI"},"6883918":{"id":6883918,"externalId":"511145.b3325","name":"gspD","hasFamily":true,"annotation":"general secretory pathway component, cryptic; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (Potential)","uniprotId":"GSPD_ECOLI"},"6883919":{"id":6883919,"externalId":"511145.b3326","name":"gspE","hasFamily":true,"annotation":"general secretory pathway component, cryptic; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (Potential)","uniprotId":"GSPE_ECOLI"},"6883920":{"id":6883920,"externalId":"511145.b3327","name":"gspF","hasFamily":true,"annotation":"general secretory pathway component, cryptic; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (Potential)","uniprotId":"GSPF_ECOLI"},"6883921":{"id":6883921,"externalId":"511145.b3328","name":"gspG","hasFamily":true,"annotation":"pseudopilin, cryptic, general secretion pathway; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (Potential)","uniprotId":"GSPG_ECOLI"},"6883922":{"id":6883922,"externalId":"511145.b3329","name":"gspH","hasFamily":true,"annotation":"putative general secretory pathway component, cryptic; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (Potential)","uniprotId":"GSPH_ECOLI"},"6883923":{"id":6883923,"externalId":"511145.b3330","name":"gspI","hasFamily":true,"annotation":"general secretory pathway component, cryptic; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (Potential)","uniprotId":"GSPI_ECOLI"},"6883924":{"id":6883924,"externalId":"511145.b3331","name":"gspJ","hasFamily":true,"annotation":"putative general secretory pathway component, cryptic; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (Potential)","uniprotId":"GSPJ_ECOLI"},"6883925":{"id":6883925,"externalId":"511145.b3332","name":"gspK","hasFamily":true,"annotation":"general secretory pathway component, cryptic; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (Potential)","uniprotId":"GSPK_ECOLI"},"6883926":{"id":6883926,"externalId":"511145.b3333","name":"gspL","hasFamily":true,"annotation":"general secretory pathway component, cryptic; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (By similarity)","uniprotId":"GSPL_ECOLI"},"6883927":{"id":6883927,"externalId":"511145.b3334","name":"pshM","hasFamily":false,"annotation":"general secretory pathway component, cryptic; Involved in a type II secretion system (T2SS, formerly general secretion pathway, GSP) for the export of proteins (Potential)","uniprotId":"GSPM_ECOLI"},"6883928":{"id":6883928,"externalId":"511145.b3335","name":"gspO","hasFamily":false,"annotation":"bifunctional prepilin leader peptidase/ methylase; Cleaves type-4 fimbrial leader sequence and methylates the N-terminal (generally Phe) residue (By similarity)","uniprotId":"LEP4_ECOLI"},"6883929":{"id":6883929,"externalId":"511145.b3336","name":"bfr","hasFamily":true,"annotation":"bacterioferritin, iron storage and detoxification protein; Iron-storage protein, whose ferroxidase center binds Fe(2+) ions, oxidizes them by dioxygen to Fe(3+), and participates in the subsequent Fe(3+) oxide mineral core formation within the central cavity of the protein complex. The mineralized iron core can contain as many as 2700 iron atoms/24-meric molecule","uniprotId":"BFR_ECOLI"},"6883930":{"id":6883930,"externalId":"511145.b3337","name":"bfd","hasFamily":false,"annotation":"bacterioferritin-associated ferredoxin; Seems to associate with BFR; could be a general redox and/or regulatory component participating in the iron storage mobilization functions of BFR. Could participate in the release or the delivery of iron from/to bacterioferritin (or other iron complexes)","uniprotId":"BFD_ECOLI"},"6883931":{"id":6883931,"externalId":"511145.b3338","name":"chiA","hasFamily":true,"annotation":"periplasmic endochitinase; Bifunctional enzyme with lysozyme/chitinase activity","uniprotId":"CHIA_ECOLI"},"6883932":{"id":6883932,"externalId":"511145.b3339","name":"tufA","hasFamily":true,"annotation":"protein chain elongation factor EF-Tu (duplicate of tufB); This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis (By similarity)","uniprotId":"EFTU1_ECOLI"},"6883933":{"id":6883933,"externalId":"511145.b3340","name":"fusA","hasFamily":true,"annotation":"protein chain elongation factor EF-G, GTP-binding; Catalyzes the GTP-dependent ribosomal translocation step during translation elongation. During this step, the ribosome changes from the pre-translocational (PRE) to the post- translocational (POST) state as the newly formed A-site-bound peptidyl-tRNA and P-site-bound deacylated tRNA move to the P and E sites, respectively. Catalyzes the coordinated movement of the two tRNA molecules, the mRNA and conformational changes in the ribosome (By similarity)","uniprotId":"EFG_ECOLI"},"6883934":{"id":6883934,"externalId":"511145.b3341","name":"rpsG","hasFamily":true,"annotation":"30S ribosomal subunit protein S7; One of the primary rRNA binding proteins, it binds directly to 16S rRNA where it nucleates assembly of the head domain of the 30S subunit. Is located at the subunit interface close to the decoding center, where it has been shown to contact mRNA. Has been shown to contact tRNA in both the P and E sites; it probably blocks exit of the E site tRNA","uniprotId":"RS7_ECOLI"},"6883935":{"id":6883935,"externalId":"511145.b3342","name":"rpsL","hasFamily":false,"annotation":"30S ribosomal subunit protein S12; Interacts with and stabilizes bases of the 16S rRNA that are involved in tRNA selection in the A site and with the mRNA backbone. Located at the interface of the 30S and 50S subunits, it traverses the body of the 30S subunit contacting proteins on the other side and probably holding the rRNA structure together. The combined cluster of proteins S8, S12 and S17 appears to hold together the shoulder and platform of the 30S subunit (By similarity)","uniprotId":"RS12_ECOLI"},"6883936":{"id":6883936,"externalId":"511145.b3343","name":"tusB","hasFamily":true,"annotation":"protein required for 2-thiolation step of mnm(5)-s(2)U34-tRNA synthesis; Part of a sulfur-relay system required for 2-thiolation of 5-methylaminomethyl-2-thiouridine (mnm(5)s(2)U) at tRNA wobble positions (By similarity)","uniprotId":"TUSB_ECOLI"},"6883937":{"id":6883937,"externalId":"511145.b3344","name":"tusC","hasFamily":true,"annotation":"protein required for 2-thiolation step of mnm(5)-s(2)U34-tRNA synthesis; Part of a sulfur-relay system required for 2-thiolation of 5-methylaminomethyl-2-thiouridine (mnm(5)s(2)U) at tRNA wobble positions (By similarity)","uniprotId":"TUSC_ECOLI"},"6883938":{"id":6883938,"externalId":"511145.b3345","name":"tusD","hasFamily":true,"annotation":"sulfurtransferase for 2-thiolation step of mnm(5)-s(2)U34-tRNA synthesis; Part of a sulfur-relay system required for 2-thiolation of 5-methylaminomethyl-2-thiouridine (mnm(5)s(2)U) at tRNA wobble positions. Accepts sulfur from TusA and transfers it in turn to TusE (By similarity)","uniprotId":"TUSD_ECOLI"},"6883939":{"id":6883939,"externalId":"511145.b3346","name":"yheO","hasFamily":true,"annotation":"conserved protein","uniprotId":"YHEO_ECOLI"},"6883940":{"id":6883940,"externalId":"511145.b3347","name":"fkpA","hasFamily":true,"annotation":"FKBP-type peptidyl-prolyl cis-trans isomerase (rotamase); PPIases accelerate the folding of proteins. It catalyzes the cis-trans isomerization of proline imidic peptide bonds in oligopeptides","uniprotId":"FKBA_ECOLI"},"6883941":{"id":6883941,"externalId":"511145.b3348","name":"slyX","hasFamily":true,"annotation":"protein required for phi X174 lysis","uniprotId":"SLYX_ECOLI"},"6883942":{"id":6883942,"externalId":"511145.b3349","name":"slyD","hasFamily":true,"annotation":"FKBP-type peptidyl prolyl cis-trans isomerase (rotamase); Folding helper with both chaperone and peptidyl-prolyl cis-trans isomerase (PPIase) activities. Chaperone activity prevents aggregation of unfolded or partially folded proteins and promotes their correct folding. PPIases catalyze the cis-trans isomerization of Xaa-Pro bonds of peptides, which accelerates slow steps of protein folding and thus shortens the lifetime of intermediates. Both strategies lower the concentration of intermediates and increase the productivity and yield of the folding reaction. SlyD could be involved in T [...] ","uniprotId":"SLYD_ECOLI"},"6883943":{"id":6883943,"externalId":"511145.b3350","name":"kefB","hasFamily":true,"annotation":"potassium:proton antiporter; Transport system that facilitates potassium-efflux, possibly by potassium-proton antiport (By similarity)","uniprotId":"KEFB_ECOLI"},"6883944":{"id":6883944,"externalId":"511145.b3351","name":"kefG","hasFamily":true,"annotation":"potassium-efflux system ancillary protein for KefB, glutathione-regulated; Required for full activity of KefB (By similarity)","uniprotId":"KEFG_ECOLI"},"6883945":{"id":6883945,"externalId":"511145.b3352","name":"yheS","hasFamily":true,"annotation":"fused predicted transporter subunits of ABC superfamily: ATP-binding components","uniprotId":"YHES_ECOLI"},"6883946":{"id":6883946,"externalId":"511145.b3353","name":"yheT","hasFamily":true,"annotation":"putative hydrolase","uniprotId":"YHET_ECOLI"},"6883947":{"id":6883947,"externalId":"511145.b3354","name":"yheU","hasFamily":true,"annotation":"conserved protein","uniprotId":"YHEU_ECOLI"},"6883948":{"id":6883948,"externalId":"511145.b3355","name":"prkB","hasFamily":true,"annotation":"putative phosphoribulokinase","uniprotId":"KPPR_ECOLI"},"6883949":{"id":6883949,"externalId":"511145.b3356","name":"yhfA","hasFamily":true,"annotation":"conserved protein, OsmC family","uniprotId":"YHFA_ECOLI"},"6883950":{"id":6883950,"externalId":"511145.b3357","name":"crp","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; This protein complexes with cyclic AMP and binds to specific DNA sites near the promoter to regulate the transcription of several catabolite-sensitive operons. The protein induces a severe bend in the DNA. Acts as a negative regulator of its own synthesis as well as for adenylate cyclase (cyaA), which generates cAMP","uniprotId":"CRP_ECOLI"},"6883951":{"id":6883951,"externalId":"511145.b3358","name":"yhfK","hasFamily":true,"annotation":"conserved inner membrane protein","uniprotId":"YHFK_ECOLI"},"6883952":{"id":6883952,"externalId":"511145.b3359","name":"argD","hasFamily":true,"annotation":"bifunctional acetylornithine aminotransferase/ succinyldiaminopimelate aminotransferase; Involved in both the arginine and lysine biosynthetic pathways (By similarity)","uniprotId":"ARGD_ECOLI"},"6883953":{"id":6883953,"externalId":"511145.b3360","name":"pabA","hasFamily":true,"annotation":"aminodeoxychorismate synthase, subunit II; Catalyzes the biosynthesis of 4-amino-4-deoxychorismate (ADC) from chorismate and glutamine","uniprotId":"PABA_ECOLI"},"6883954":{"id":6883954,"externalId":"511145.b3361","name":"fic","hasFamily":true,"annotation":"stationary-phase protein, cell division; Probable adenylyltransferase that mediates the addition of adenosine 5'-monophosphate (AMP) to specific residues of target proteins (By similarity). Involved in cell filamentation induced by cyclic AMP. May have some role in cell division","uniprotId":"FIC_ECOLI"},"6883955":{"id":6883955,"externalId":"511145.b3362","name":"yhfG","hasFamily":true,"annotation":"predicted protein","uniprotId":"YHFG_ECOLI"},"6883956":{"id":6883956,"externalId":"511145.b3363","name":"ppiA","hasFamily":true,"annotation":"peptidyl-prolyl cis-trans isomerase A (rotamase A); PPIases accelerate the folding of proteins. It catalyzes the cis-trans isomerization of proline imidic peptide bonds in oligopeptides","uniprotId":"PPIA_ECOLI"},"6883957":{"id":6883957,"externalId":"511145.b3364","name":"tsgA","hasFamily":true,"annotation":"putative transporter","uniprotId":"TSGA_ECOLI"},"6883958":{"id":6883958,"externalId":"511145.b3365","name":"nirB","hasFamily":true,"annotation":"nitrite reductase, large subunit, NAD(P)H-binding","uniprotId":"NIRB_ECOLI"},"6883959":{"id":6883959,"externalId":"511145.b3366","name":"nirD","hasFamily":true,"annotation":"nitrite reductase, NAD(P)H-binding, small subunit; Required for activity of the reductase","uniprotId":"NIRD_ECOLI"},"6883960":{"id":6883960,"externalId":"511145.b3367","name":"nirC","hasFamily":true,"annotation":"nitrite transporter; May act as a nitrite transporter","uniprotId":"NIRC_ECOLI"},"6883961":{"id":6883961,"externalId":"511145.b3368","name":"cysG","hasFamily":true,"annotation":"fused siroheme synthase 1,3-dimethyluroporphyriongen III dehydrogenase and siroheme ferrochelatase/uroporphyrinogen methyltransferase; Multifunctional enzyme that catalyzes the SAM-dependent methylation of uroporphyrinogen III at position C-2 and C-7 to form precorrin-2 and then position C-12 or C-18 to form trimethylpyrrocorphin 2. It also catalyzes the conversion of precorrin-2 into siroheme. This reaction consists of the NAD- dependent oxidation of precorrin-2 into sirohydrochlorin and its subsequent ferrochelation into siroheme (By similarity)","uniprotId":"CYSG_ECOLI"},"6883962":{"id":6883962,"externalId":"511145.b3369","name":"yhfL","hasFamily":true,"annotation":"small lipoprotein","uniprotId":"YHFL_ECOLI"},"6883963":{"id":6883963,"externalId":"511145.b3370","name":"frlA","hasFamily":true,"annotation":"putative fructoselysine transporter; May be involved in fructoselysine transport","uniprotId":"FRLA_ECOLI"},"6883964":{"id":6883964,"externalId":"511145.b3371","name":"frlB","hasFamily":true,"annotation":"fructoselysine-6-P-deglycase; Catalyzes the conversion of fructoselysine 6-phosphate to glucose 6-phosphate and lysine","uniprotId":"FRLB_ECOLI"},"6883965":{"id":6883965,"externalId":"511145.b3374","name":"frlD","hasFamily":true,"annotation":"fructoselysine 6-kinase; Phosphorylates fructoselysine to yield fructoselysine 6- phosphate","uniprotId":"FRLD_ECOLI"},"6883966":{"id":6883966,"externalId":"511145.b3375","name":"frlR","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator; May regulate the transcription of the frlABCD operon","uniprotId":"FRLR_ECOLI"},"6883967":{"id":6883967,"externalId":"511145.b3376","name":"yhfS","hasFamily":false,"annotation":"conserved protein","uniprotId":"YHFS_ECOLI"},"6883968":{"id":6883968,"externalId":"511145.b3377","name":"yhfT","hasFamily":false,"annotation":"putative inner membrane protein","uniprotId":"YHFT_ECOLI"},"6883969":{"id":6883969,"externalId":"511145.b3378","name":"yhfU","hasFamily":false,"annotation":"predicted protein","uniprotId":"YHFU_ECOLI"},"6883970":{"id":6883970,"externalId":"511145.b3379","name":"php","hasFamily":true,"annotation":"putative hydrolase; Its real enzymatic activity is not yet known. It was tested for general esterase, aminopeptidase, sulfatase, phosphatase, carbonic anhydrase, phosphodiesterase, and phosphotriesterase activities with the following substrates: P- nitrophenyl acetate, L-alanine nitroanilide, P-nitrophenyl sulfate, bis(P-nitrophenyl) phosphate, paraoxon, and P-nitrophenyl phosphate. No enzymatic activity was detected with any of these nonspecific substrates","uniprotId":"PHP_ECOLI"},"6883971":{"id":6883971,"externalId":"511145.b3380","name":"yhfW","hasFamily":false,"annotation":"putative mutase","uniprotId":"YHFW_ECOLI"},"6883972":{"id":6883972,"externalId":"511145.b3381","name":"yhfX","hasFamily":false,"annotation":"putative amino acid racemase","uniprotId":"YHFX_ECOLI"},"6883973":{"id":6883973,"externalId":"511145.b3382","name":"yhfY","hasFamily":false,"annotation":"conserved protein","uniprotId":"YHFY_ECOLI"},"6883974":{"id":6883974,"externalId":"511145.b3383","name":"yhfZ","hasFamily":false,"annotation":"conserved protein","uniprotId":"YHFZ_ECOLI"},"6883975":{"id":6883975,"externalId":"511145.b3384","name":"trpS","hasFamily":true,"annotation":"tryptophanyl-tRNA synthetase","uniprotId":"SYW_ECOLI"},"6883976":{"id":6883976,"externalId":"511145.b3385","name":"gph","hasFamily":true,"annotation":"phosphoglycolate phosphatase; Specifically catalyzes the dephosphorylation of 2- phosphoglycolate. Is involved in the dissimilation of the intracellular 2-phosphoglycolate formed during the DNA repair of 3'-phosphoglycolate ends, a major class of DNA lesions induced by oxidative stress (By similarity)","uniprotId":"GPH_ECOLI"},"6883977":{"id":6883977,"externalId":"511145.b3386","name":"rpe","hasFamily":true,"annotation":"D-ribulose-5-phosphate 3-epimerase; Catalyzes the reversible epimerization of D-ribulose 5- phosphate to D-xylulose 5-phosphate (By similarity)","uniprotId":"RPE_ECOLI"},"6883978":{"id":6883978,"externalId":"511145.b3387","name":"dam","hasFamily":true,"annotation":"DNA adenine methyltransferase; Methylates DNA within the sequence GATC and protects the DNA from cleavage by the restriction endonuclease MboI. Although it shares sequence specificity with a number of type II restriction endonucleases and methylases, it is thought to act in postreplication mismatch repair rather than as a part of a restriction modification system. May also play a role in DNA replication","uniprotId":"DMA_ECOLI"},"6883979":{"id":6883979,"externalId":"511145.b3388","name":"damX","hasFamily":true,"annotation":"cell division protein that binds to the septal ring; Directly or indirectly interferes with cell division","uniprotId":"DAMX_ECOLI"},"6883980":{"id":6883980,"externalId":"511145.b3389","name":"aroB","hasFamily":true,"annotation":"3-dehydroquinate synthase","uniprotId":"AROB_ECOLI"},"6883981":{"id":6883981,"externalId":"511145.b3390","name":"aroK","hasFamily":true,"annotation":"shikimate kinase I; Catalyzes the specific phosphorylation of the 3-hydroxyl group of shikimic acid using ATP as a cosubstrate (By similarity)","uniprotId":"AROK_ECOLI"},"6883982":{"id":6883982,"externalId":"511145.b3391","name":"hofQ","hasFamily":true,"annotation":"protein required for the utilization of DNA as a carbon source; fimbrial transporter homolog; Required for the use of extracellular DNA as a nutrient. Could be the porin responsible for transport of DNA across the outer membrane","uniprotId":"HOFQ_ECOLI"},"6883983":{"id":6883983,"externalId":"511145.b3392","name":"hofP","hasFamily":true,"annotation":"protein required for the utilization of DNA as a carbon source; Required for the use of extracellular DNA as a nutrient","uniprotId":"HOFP_ECOLI"},"6883984":{"id":6883984,"externalId":"511145.b3393","name":"hofO","hasFamily":true,"annotation":"protein required for the utilization of DNA as a carbon source; Required for the use of extracellular DNA as a nutrient","uniprotId":"HOFO_ECOLI"},"6883985":{"id":6883985,"externalId":"511145.b3394","name":"hofN","hasFamily":true,"annotation":"protein required for the utilization of DNA as a carbon source; fimbrial assembly protein homolog; Required for the use of extracellular DNA as a nutrient","uniprotId":"HOFN_ECOLI"},"6883986":{"id":6883986,"externalId":"511145.b3395","name":"hofM","hasFamily":true,"annotation":"protein required for the utilization of DNA as a carbon source, pilus assembly protein homolog; Required for the use of extracellular DNA as a nutrient","uniprotId":"HOFM_ECOLI"},"6883987":{"id":6883987,"externalId":"511145.b3396","name":"mrcA","hasFamily":true,"annotation":"fused penicillin-binding protein 1a: murein transglycosylase/murein transpeptidase; Cell wall formation. Synthesis of cross-linked peptidoglycan from the lipid intermediates. The enzyme has a penicillin-insensitive transglycosylase N-terminal domain (formation of linear glycan strands) and a penicillin-sensitive transpeptidase C-terminal domain (cross-linking of the peptide subunits)","uniprotId":"PBPA_ECOLI"},"6883988":{"id":6883988,"externalId":"511145.b3397","name":"nudE","hasFamily":true,"annotation":"adenosine nucleotide hydrolase; substrates include Ap3A, Ap2A, ADP-ribose, NADH; Active on adenosine(5')triphospho(5')adenosine (Ap3A), ADP-ribose, NADH, adenosine(5')diphospho(5')adenosine (Ap2A)","uniprotId":"NUDE_ECOLI"},"6883989":{"id":6883989,"externalId":"511145.b3398","name":"yrfF","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"IGAA_ECOLI"},"6883990":{"id":6883990,"externalId":"511145.b3399","name":"yrfG","hasFamily":true,"annotation":"GMP/IMP nucleotidase; Catalyzes the dephosphorylation of different purine nucleotides (GMP and IMP). Also hydrolyzes flavin mononucleotide (FMN)","uniprotId":"YRFG_ECOLI"},"6883991":{"id":6883991,"externalId":"511145.b3400","name":"hslR","hasFamily":true,"annotation":"ribosome-associated heat shock protein Hsp15; Involved in the recycling of free 50S ribosomal subunits that still carry a nascent chain. Binds RNA more specifically than DNA. Binds with very high affinity to the free 50S ribosomal subunit. Does not bind it when it is part of the 70S ribosome","uniprotId":"HSLR_ECOLI"},"6883992":{"id":6883992,"externalId":"511145.b3401","name":"hslO","hasFamily":true,"annotation":"heat shock protein Hsp33; Redox regulated molecular chaperone. Protects both thermally unfolding and oxidatively damaged proteins from irreversible aggregation. Plays an important role in the bacterial defense system toward oxidative stress (By similarity)","uniprotId":"HSLO_ECOLI"},"6883993":{"id":6883993,"externalId":"511145.b3402","name":"yhgE","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YHGE_ECOLI"},"6883994":{"id":6883994,"externalId":"511145.b3403","name":"pck","hasFamily":true,"annotation":"phosphoenolpyruvate carboxykinase","uniprotId":"PCKA_ECOLI"},"6883995":{"id":6883995,"externalId":"511145.b3404","name":"envZ","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with OmpR; Member of the two-component regulatory system EnvZ/OmpR involved in the regulation of osmoregulation (genes ompF and ompC). EnvZ functions as a membrane-associated protein kinase that phosphorylates OmpR in response to environmental signals","uniprotId":"ENVZ_ECOLI"},"6883996":{"id":6883996,"externalId":"511145.b3405","name":"ompR","hasFamily":false,"annotation":"DNA-binding response regulator in two-component regulatory system with EnvZ; The N-terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC; its C-terminal moiety mediates the multimerization of the OmpR protein. As a multimer, it turns on the expression of the ompC gene; as a monomer, it turns on the expression of the ompF gene","uniprotId":"OMPR_ECOLI"},"6883997":{"id":6883997,"externalId":"511145.b3406","name":"greB","hasFamily":true,"annotation":"transcript cleavage factor; Necessary for efficient RNA polymerase transcription elongation past template-encoded arresting sites. The arresting sites in DNA have the property of trapping a certain fraction of elongating RNA polymerases that pass through, resulting in locked ternary complexes. Cleavage of the nascent transcript by cleavage factors such as GreA or GreB allows the resumption of elongation from the new 3'terminus. GreB releases sequences of up to 9 nucleotides in length","uniprotId":"GREB_ECOLI"},"6883998":{"id":6883998,"externalId":"511145.b3407","name":"yhgF","hasFamily":true,"annotation":"putative transcriptional accessory protein","uniprotId":"YHGF_ECOLI"},"6883999":{"id":6883999,"externalId":"511145.b3408","name":"feoA","hasFamily":true,"annotation":"ferrous iron transporter, protein A; Not known. Involved in Fe(2+) ion uptake","uniprotId":"FEOA_ECOLI"},"6884000":{"id":6884000,"externalId":"511145.b3409","name":"feoB","hasFamily":true,"annotation":"fused ferrous iron transporter, protein B: GTP-binding protein/membrane protein; GTP-driven Fe(2+) uptake system","uniprotId":"FEOB_ECOLI"},"6884001":{"id":6884001,"externalId":"511145.b3410","name":"feoC","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator; May function as a transcriptional regulator that controls feoABC expression (By similarity)","uniprotId":"FEOC_ECOLI"},"6884002":{"id":6884002,"externalId":"511145.b3411","name":"yhgA","hasFamily":false,"annotation":"putative transposase","uniprotId":"YHGA_ECOLI"},"6884003":{"id":6884003,"externalId":"511145.b3412","name":"bioH","hasFamily":true,"annotation":"pimeloyl-ACP carboxylesterase; The physiological role of BioH is to remove the methyl group introduced by BioC when the pimeloyl moiety is complete. It allows to synthesize pimeloyl-ACP via the fatty acid synthetic pathway through the hydrolysis of the ester bonds of pimeloyl-ACP esters. E.coli employs a methylation and demethylation strategy to allow elongation of a temporarily disguised malonate moiety to a pimelate moiety by the fatty acid synthetic enzymes. BioH shows a preference for short chain fatty acid esters (acyl chain length of up to 6 carbons) and short chain p-nitrophenyl [...] ","uniprotId":"BIOH_ECOLI"},"6884004":{"id":6884004,"externalId":"511145.b3413","name":"gntX","hasFamily":true,"annotation":"protein required for the utilization of DNA as a carbon source; Could be involved in gluconate metabolism","uniprotId":"GNTX_ECOLI"},"6884005":{"id":6884005,"externalId":"511145.b3414","name":"gntY","hasFamily":true,"annotation":"Fe/S biogenesis protein; possible scaffold/chaperone for damaged Fe/S proteins; Involved in iron-sulfur cluster biogenesis under severe conditions such as iron starvation or oxidative stress. Binds a 4Fe-4S cluster, can transfer this cluster to apoproteins, and thereby intervenes in the maturation of Fe/S proteins. Could also act as a scaffold/chaperone for damaged Fe/S proteins. Required for E.coli to sustain oxidative stress and iron starvation. Also necessary for the use of extracellular DNA as the sole source of carbon and energy","uniprotId":"NFUA_ECOLI"},"6884006":{"id":6884006,"externalId":"511145.b3415","name":"gntT","hasFamily":true,"annotation":"gluconate transporter, high-affinity GNT I system; Part of the gluconate utilization system Gnt-I; high- affinity intake of gluconate","uniprotId":"GNTT_ECOLI"},"6884007":{"id":6884007,"externalId":"511145.b3416","name":"malQ","hasFamily":true,"annotation":"4-alpha-glucanotransferase (amylomaltase)","uniprotId":"MALQ_ECOLI"},"6884008":{"id":6884008,"externalId":"511145.b3417","name":"malP","hasFamily":true,"annotation":"maltodextrin phosphorylase; Phosphorylase is an important allosteric enzyme in carbohydrate metabolism. Enzymes from different sources differ in their regulatory mechanisms and in their natural substrates. However, all known phosphorylases share catalytic and structural properties (By similarity)","uniprotId":"PHSM_ECOLI"},"6884009":{"id":6884009,"externalId":"511145.b3418","name":"malT","hasFamily":true,"annotation":"DNA-binding transcriptional activator for the mal regulon and maltotriose-ATP-binding protein; Positively regulates the transcription of the maltose regulon whose gene products are responsible for uptake and catabolism of malto-oligosaccharides. Binds and recognizes a DNA motif (called the malT box): 5'-GGA[TG]GA-3' (By similarity)","uniprotId":"MALT_ECOLI"},"6884010":{"id":6884010,"externalId":"511145.b3421","name":"rtcB","hasFamily":true,"annotation":"conserved protein; RNA ligase that mediates the joining of broken tRNA-like stem-loop structures in case of tRNA damage. Probably participates to tRNA restriction-repair by ligating broken tRNA-like stem-loop structures with 2',3'-cyclic phosphate and 5'-OH ends to form a splice junction with a 2'-OH, 3',5'-phosphodiester","uniprotId":"RTCB_ECOLI"},"6884011":{"id":6884011,"externalId":"511145.b3422","name":"rtcR","hasFamily":true,"annotation":"sigma 54-dependent transcriptional regulator of rtcBA expression; Transcriptional repressor of the rtcAB genes. Interacts with sigma-54","uniprotId":"RTCR_ECOLI"},"6884012":{"id":6884012,"externalId":"511145.b3423","name":"glpR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor of the glycerol-3-phosphate regulon","uniprotId":"GLPR_ECOLI"},"6884013":{"id":6884013,"externalId":"511145.b3424","name":"glpG","hasFamily":true,"annotation":"rhomboid intramembrane serine protease; Rhomboid-type serine protease that catalyzes intramembrane proteolysis (By similarity)","uniprotId":"GLPG_ECOLI"},"6884014":{"id":6884014,"externalId":"511145.b3425","name":"glpE","hasFamily":true,"annotation":"thiosulfate:cyanide sulfurtransferase (rhodanese); Catalyzes, although with low efficiency, the sulfur transfer reaction from thiosulfate to cyanide. The relatively low affinity of GlpE for both thiosulfate and cyanide suggests that these compounds are not the physiological substrates. Thioredoxin 1 or related dithiol proteins could instead be the physiological sulfur-acceptor substrate. Possible association with the metabolism of glycerol-phosphate remains to be elucidated","uniprotId":"GLPE_ECOLI"},"6884015":{"id":6884015,"externalId":"511145.b3426","name":"glpD","hasFamily":true,"annotation":"sn-glycerol-3-phosphate dehydrogenase, aerobic, FAD/NAD(P)-binding; Conversion of glycerol 3-phosphate to dihydroxyacetone. Uses molecular oxygen or nitrate as electron acceptor","uniprotId":"GLPD_ECOLI"},"6884016":{"id":6884016,"externalId":"511145.b3428","name":"glgP","hasFamily":true,"annotation":"glycogen phosphorylase; Phosphorylase is an important allosteric enzyme in carbohydrate metabolism. Enzymes from different sources differ in their regulatory mechanisms and in their natural substrates. However, all known phosphorylases share catalytic and structural properties (By similarity)","uniprotId":"PHSG_ECOLI"},"6884017":{"id":6884017,"externalId":"511145.b3429","name":"glgA","hasFamily":true,"annotation":"glycogen synthase; Synthesizes alpha-1,4-glucan chains using ADP-glucose (By similarity)","uniprotId":"GLGA_ECOLI"},"6884018":{"id":6884018,"externalId":"511145.b3430","name":"glgC","hasFamily":true,"annotation":"glucose-1-phosphate adenylyltransferase; Catalyzes the synthesis of ADP-glucose, a sugar donor used in elongation reactions on alpha-glucans (By similarity)","uniprotId":"GLGC_ECOLI"},"6884019":{"id":6884019,"externalId":"511145.b3431","name":"glgX","hasFamily":true,"annotation":"glycogen debranching enzyme; Hydrolyzes the alpha-1,6-glucosidic linkages in glycogen which has first been partially depolymerized by phosphorylase. Shows only very little activity with native glycogen (By similarity)","uniprotId":"GLGX_ECOLI"},"6884020":{"id":6884020,"externalId":"511145.b3432","name":"glgB","hasFamily":true,"annotation":"1,4-alpha-glucan branching enzyme; Catalyzes the formation of the alpha-1,6-glucosidic linkages in glycogen by scission of a 1,4-alpha-linked oligosaccharide from growing alpha-1,4-glucan chains and the subsequent attachment of the oligosaccharide to the alpha-1,6 position (By similarity)","uniprotId":"GLGB_ECOLI"},"6884021":{"id":6884021,"externalId":"511145.b3433","name":"asd","hasFamily":true,"annotation":"aspartate-semialdehyde dehydrogenase, NAD(P)-binding; Catalyzes the NADPH-dependent formation of L-aspartate- semialdehyde (L-ASA) by the reductive dephosphorylation of L- aspartyl-4-phosphate (By similarity)","uniprotId":"DHAS_ECOLI"},"6884022":{"id":6884022,"externalId":"511145.b3434","name":"yhgN","hasFamily":true,"annotation":"putative antibiotic transporter","uniprotId":"YHGN_ECOLI"},"6884023":{"id":6884023,"externalId":"511145.b3437","name":"gntK","hasFamily":true,"annotation":"gluconate kinase 2","uniprotId":"GNTK_ECOLI"},"6884024":{"id":6884024,"externalId":"511145.b3438","name":"gntR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Negative regulator for the gluconate utilization system GNT-I, the gntUKR operon","uniprotId":"GNTR_ECOLI"},"6884025":{"id":6884025,"externalId":"511145.b3439","name":"yhhW","hasFamily":true,"annotation":"quercetinase activity in vitro; Has quercetin 2,3-dioxygenase activity in vitro. Its physiological role is unknown; however, may provide a mechanism that would avoid inhibition of key cellular proteins, such as DNA gyrase, by quercetin","uniprotId":"YHHW_ECOLI"},"6884026":{"id":6884026,"externalId":"511145.b3440","name":"yhhX","hasFamily":true,"annotation":"putative oxidoreductase with NAD(P)-binding Rossmann-fold domain","uniprotId":"YHHX_ECOLI"},"6884027":{"id":6884027,"externalId":"511145.b3441","name":"yhhY","hasFamily":true,"annotation":"putative acetyltransferase","uniprotId":"YHHY_ECOLI"},"6884028":{"id":6884028,"externalId":"511145.b3442","name":"yhhZ","hasFamily":true,"annotation":"conserved protein","uniprotId":"YHHZ_ECOLI"},"6884029":{"id":6884029,"externalId":"511145.b3444","name":"insA1","hasFamily":false,"annotation":"IS1 repressor TnpA; Absolutely required for transposition of IS1","uniprotId":"INSA1_ECOLI"},"6884030":{"id":6884030,"externalId":"511145.b3445","name":"is1B","hasFamily":true,"annotation":"IS1 transposase B; Absolutely required for transposition of IS1","uniprotId":"INSB1_ECOLI"},"6884031":{"id":6884031,"externalId":"511145.b3446","name":"yrhB","hasFamily":false,"annotation":"predicted protein","uniprotId":"YRHB_ECOLI"},"6884032":{"id":6884032,"externalId":"511145.b3447","name":"ggt","hasFamily":true,"annotation":"gamma-glutamyltranspeptidase","uniprotId":"GGT_ECOLI"},"6884033":{"id":6884033,"externalId":"511145.b3448","name":"yhhA","hasFamily":true,"annotation":"conserved protein, DUF2756 family","uniprotId":"YHHA_ECOLI"},"6884034":{"id":6884034,"externalId":"511145.b3449","name":"ugpQ","hasFamily":true,"annotation":"glycerophosphodiester phosphodiesterase, cytosolic; Glycerophosphoryl diester phosphodiesterase hydrolyzes deacylated phospholipids to G3P and the corresponding alcohols","uniprotId":"UGPQ_ECOLI"},"6884035":{"id":6884035,"externalId":"511145.b3450","name":"ugpC","hasFamily":true,"annotation":"glycerol-3-phosphate transporter subunit; Part of the ABC transporter complex UgpABCE involved in sn-glycerol-3-phosphate import. Responsible for energy coupling to the transport system (Probable). Can also transport glycerophosphoryl diesters","uniprotId":"UGPC_ECOLI"},"6884036":{"id":6884036,"externalId":"511145.b3451","name":"ugpE","hasFamily":true,"annotation":"glycerol-3-phosphate transporter subunit; Part of the binding-protein-dependent transport system for sn-glycerol-3-phosphate; probably responsible for the translocation of the substrate across the membrane","uniprotId":"UGPE_ECOLI"},"6884037":{"id":6884037,"externalId":"511145.b3452","name":"ugpA","hasFamily":true,"annotation":"glycerol-3-phosphate transporter subunit; Part of the binding-protein-dependent transport system for sn-glycerol-3-phosphate; probably responsible for the translocation of the substrate across the membrane","uniprotId":"UGPA_ECOLI"},"6884038":{"id":6884038,"externalId":"511145.b3453","name":"ugpB","hasFamily":true,"annotation":"glycerol-3-phosphate transporter subunit; sn-glycerol-3-phosphate and glycerophosphoryl diester- binding protein interacts with the binding protein-dependent transport system UgpACE","uniprotId":"UGPB_ECOLI"},"6884039":{"id":6884039,"externalId":"511145.b3454","name":"livF","hasFamily":true,"annotation":"leucine/isoleucine/valine transporter subunit; Component of the leucine-specific transport system","uniprotId":"LIVF_ECOLI"},"6884040":{"id":6884040,"externalId":"511145.b3455","name":"livG","hasFamily":true,"annotation":"leucine/isoleucine/valine transporter subunit; Component of the leucine-specific transport system","uniprotId":"LIVG_ECOLI"},"6884041":{"id":6884041,"externalId":"511145.b3456","name":"livM","hasFamily":true,"annotation":"leucine/isoleucine/valine transporter subunit; Part of the binding-protein-dependent transport system for branched-chain amino acids. Probably responsible for the translocation of the substrates across the membrane","uniprotId":"LIVM_ECOLI"},"6884042":{"id":6884042,"externalId":"511145.b3457","name":"livH","hasFamily":true,"annotation":"leucine/isoleucine/valine transporter subunit; Part of the binding-protein-dependent transport system for branched-chain amino acids. Probably responsible for the translocation of the substrates across the membrane","uniprotId":"LIVH_ECOLI"},"6884043":{"id":6884043,"externalId":"511145.b3458","name":"livK","hasFamily":true,"annotation":"leucine transporter subunit; This protein is a component of the leucine-specific transport system, which is one of the two periplasmic binding protein-dependent transport systems of the high-affinity transport of the branched-chain amino acids in E.coli","uniprotId":"LIVK_ECOLI"},"6884044":{"id":6884044,"externalId":"511145.b3459","name":"yhhK","hasFamily":true,"annotation":"PanD autocleavage accelerator, panothenate synthesis","uniprotId":"YHHK_ECOLI"},"6884045":{"id":6884045,"externalId":"511145.b3460","name":"livJ","hasFamily":true,"annotation":"leucine/isoleucine/valine transporter subunit; This protein is a component of the leucine, isoleucine, valine, (threonine) transport system, which is one of the two periplasmic binding protein-dependent transport systems of the high-affinity transport of the branched-chain amino acids","uniprotId":"LIVJ_ECOLI"},"6884046":{"id":6884046,"externalId":"511145.b3461","name":"rpoH","hasFamily":true,"annotation":"RNA polymerase, sigma 32 (sigma H) factor; Sigma factors are initiation factors that promote the attachment of RNA polymerase to specific initiation sites and are then released. This sigma factor is responsible for the expression of heat shock promoters","uniprotId":"RP32_ECOLI"},"6884047":{"id":6884047,"externalId":"511145.b3462","name":"ftsX","hasFamily":true,"annotation":"putative transporter subunit: membrane component of ABC superfamily; Part of the ABC transporter FtsEX involved in cellular division. Important for assembly or stability of the septal ring. Encoded in an operon consisting of genes ftsY, ftsE and ftsX","uniprotId":"FTSX_ECOLI"},"6884048":{"id":6884048,"externalId":"511145.b3463","name":"ftsE","hasFamily":true,"annotation":"putative transporter subunit: ATP-binding component of ABC superfamily; Not known. Is coded in an operon essential for cell division","uniprotId":"FTSE_ECOLI"},"6884049":{"id":6884049,"externalId":"511145.b3464","name":"ftsY","hasFamily":true,"annotation":"Signal Recognition Particle (SRP) receptor; Involved in targeting and insertion of nascent membrane proteins into the cytoplasmic membrane. Acts as a receptor for the complex formed by the signal recognition particle (SRP) and the ribosome-nascent chain (RNC). Interaction with SRP-RNC leads to the transfer of the RNC complex to the Sec translocase for insertion into the membrane, the hydrolysis of GTP by both Ffh and FtsY, and the dissociation of the SRP-FtsY complex into the individual components (By similarity)","uniprotId":"FTSY_ECOLI"},"6884050":{"id":6884050,"externalId":"511145.b3465","name":"rsmD","hasFamily":true,"annotation":"16S rRNA m(2)G966 methyltransferase, SAM-dependent; Specifically methylates the guanine in position 966 of 16S rRNA in the assembled 30S particle","uniprotId":"RSMD_ECOLI"},"6884051":{"id":6884051,"externalId":"511145.b3466","name":"yhhL","hasFamily":true,"annotation":"conserved protein, DUF1145 family","uniprotId":"YHHL_ECOLI"},"6884052":{"id":6884052,"externalId":"511145.b3467","name":"yhhM","hasFamily":true,"annotation":"conserved protein, DUF2500 family","uniprotId":"YHHM_ECOLI"},"6884053":{"id":6884053,"externalId":"511145.b3468","name":"yhhN","hasFamily":true,"annotation":"putative inner membrane protein, TMEM86 family","uniprotId":"YHHN_ECOLI"},"6884054":{"id":6884054,"externalId":"511145.b3469","name":"zntA","hasFamily":true,"annotation":"zinc, cobalt and lead efflux system; Involved in export of lead, cadmium, zinc and mercury","uniprotId":"ATZN_ECOLI"},"6884055":{"id":6884055,"externalId":"511145.b3470","name":"sirA","hasFamily":true,"annotation":"sulfurtransferase required 2-thiolation step of mnm(5)-s(2)U34-tRNA synthesis; Part of a sulfur-relay system required for 2-thiolation of 5-methylaminomethyl-2-thiouridine (mnm(5)s(2)U) at tRNA wobble positions. Interacts with IscS and stimulates its activity. Then, accepts a sulfur from IscS and transfers it in turn to TusD. Seems to affect the stability of sigma-S, particularly during the logarithmic growth phase","uniprotId":"TUSA_ECOLI"},"6884056":{"id":6884056,"externalId":"511145.b3471","name":"yhhQ","hasFamily":true,"annotation":"inner membrane protein, DUF165 family","uniprotId":"YHHQ_ECOLI"},"6884057":{"id":6884057,"externalId":"511145.b3472","name":"dcrB","hasFamily":true,"annotation":"periplasmic protein, predicted lipoprotein; Required for phage C1 adsorption","uniprotId":"DCRB_ECOLI"},"6884058":{"id":6884058,"externalId":"511145.b3473","name":"yhhS","hasFamily":true,"annotation":"putative arabinose efflux transporter","uniprotId":"YHHS_ECOLI"},"6884059":{"id":6884059,"externalId":"511145.b3474","name":"yhhT","hasFamily":true,"annotation":"inner membrane protein, predicted membrane permease, UPF0118 family","uniprotId":"YHHT_ECOLI"},"6884060":{"id":6884060,"externalId":"511145.b3475","name":"acpT","hasFamily":true,"annotation":"holo-(acyl carrier protein) synthase 2; May be involved in an alternative pathway for phosphopantetheinyl transfer and holo-ACP synthesis in E.coli. The native apo-protein substrate is unknown. Is able to functionally replace AcpS in vivo but only when expressed at high levels","uniprotId":"ACPT_ECOLI"},"6884061":{"id":6884061,"externalId":"511145.b3476","name":"nikA","hasFamily":true,"annotation":"Nickel-binding, heme-binding periplasmic protein; Involved in a nickel transport system, probably represents the nickel binder","uniprotId":"NIKA_ECOLI"},"6884062":{"id":6884062,"externalId":"511145.b3477","name":"nikB","hasFamily":true,"annotation":"nickel transporter subunit; Involved in a nickel transport system, probably translocates nickel through the bacterial inner membrane","uniprotId":"NIKB_ECOLI"},"6884063":{"id":6884063,"externalId":"511145.b3478","name":"nikC","hasFamily":false,"annotation":"nickel transporter subunit; Involved in a nickel transport system, probably translocates nickel through the bacterial inner membrane","uniprotId":"NIKC_ECOLI"},"6884064":{"id":6884064,"externalId":"511145.b3479","name":"nikD","hasFamily":false,"annotation":"nickel transporter subunit; Part of the ABC transporter complex NikABCDE involved in nickel import. Responsible for energy coupling to the transport system (By similarity)","uniprotId":"NIKD_ECOLI"},"6884065":{"id":6884065,"externalId":"511145.b3480","name":"nikE","hasFamily":false,"annotation":"nickel transporter subunit; Part of the ABC transporter complex NikABCDE involved in nickel import. Responsible for energy coupling to the transport system (By similarity)","uniprotId":"NIKE_ECOLI"},"6884066":{"id":6884066,"externalId":"511145.b3481","name":"nikR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor, Ni-binding; Transcriptional repressor of the nikABCDE operon. Is active in the presence of excessive concentrations of intracellular nickel (By similarity)","uniprotId":"NIKR_ECOLI"},"6884067":{"id":6884067,"externalId":"511145.b3482","name":"rhsB","hasFamily":true,"annotation":"rhsB element core protein RshB; Rhs elements have a nonessential function. They may play an important role in the natural ecology of the cell","uniprotId":"RHSB_ECOLI"},"6884068":{"id":6884068,"externalId":"511145.b3483","name":"yhhH","hasFamily":false,"annotation":"predicted protein","uniprotId":"YHHH_ECOLI"},"6884069":{"id":6884069,"externalId":"511145.b3484","name":"yafF","hasFamily":true,"annotation":"putative transposase","uniprotId":"YAFF_ECOLI"},"6884070":{"id":6884070,"externalId":"511145.b3485","name":"yhhJ","hasFamily":true,"annotation":"putative permease subunit, ABC transporter superfamily, inner membrane protein","uniprotId":"YHHJ_ECOLI"},"6884071":{"id":6884071,"externalId":"511145.b3486","name":"rbbA","hasFamily":true,"annotation":"fused ribosome-associated ATPase: ATP-binding protein/ATP-binding protein/predicted membrane protein","uniprotId":"YHIH_ECOLI"},"6884072":{"id":6884072,"externalId":"511145.b3487","name":"yhiI","hasFamily":true,"annotation":"putative membrane fusion protein (MFP) of efflux pump","uniprotId":"YHII_ECOLI"},"6884073":{"id":6884073,"externalId":"511145.b3488","name":"yhiJ","hasFamily":false,"annotation":"predicted protein","uniprotId":"YHIJ_ECOLI"},"6884074":{"id":6884074,"externalId":"511145.b3491","name":"yhiM","hasFamily":true,"annotation":"inner membrane protein, DUF1323 family","uniprotId":"YHIM_ECOLI"},"6884075":{"id":6884075,"externalId":"511145.b3492","name":"yhiN","hasFamily":true,"annotation":"putative oxidoreductase with FAD/NAD(P)-binding domain","uniprotId":"YHIN_ECOLI"},"6884076":{"id":6884076,"externalId":"511145.b3493","name":"pitA","hasFamily":true,"annotation":"phosphate transporter, low-affinity; Low-affinity inorganic phosphate transport. Can also transport arsenate","uniprotId":"PITA_ECOLI"},"6884077":{"id":6884077,"externalId":"511145.b3494","name":"uspB","hasFamily":true,"annotation":"universal stress (ethanol tolerance) protein B","uniprotId":"USPB_ECOLI"},"6884078":{"id":6884078,"externalId":"511145.b3495","name":"uspA","hasFamily":false,"annotation":"universal stress global response regulator; Required for resistance to DNA-damaging agents","uniprotId":"USPA_ECOLI"},"6884079":{"id":6884079,"externalId":"511145.b3496","name":"dtpB","hasFamily":true,"annotation":"dipeptide and tripeptide permease B; Proton-dependent permease that transports di- and tripeptides. Has a clear preference for dipeptides and tripeptides composed of L-amino acids, and discriminates dipeptides on the basis of the position of charges within the substrate","uniprotId":"DTPB_ECOLI"},"6884080":{"id":6884080,"externalId":"511145.b3497","name":"rsmJ","hasFamily":true,"annotation":"predicted SAM-dependent methyltransferase; Specifically methylates the guanosine in position 1516 of 16S rRNA (By similarity)","uniprotId":"RSMJ_ECOLI"},"6884081":{"id":6884081,"externalId":"511145.b3498","name":"prlC","hasFamily":true,"annotation":"oligopeptidase A; May play a specific role in the degradation of signal peptides after they are released from precursor forms of secreted proteins. Can cleave N-acetyl-L-Ala(4)","uniprotId":"OPDA_ECOLI"},"6884082":{"id":6884082,"externalId":"511145.b3499","name":"rlmJ","hasFamily":true,"annotation":"23S rRNA m(6)A2030 methyltransferase, SAM-dependent; Specifically methylates the adenine in position 2030 of 23S rRNA. Nascent 23S rRNA seems to be the natural substrate. Appears to be not necessary for ribosome assembly. Seems to be required for the utilization of extracellular DNA as the sole source of carbon and energy","uniprotId":"RLMJ_ECOLI"},"6884083":{"id":6884083,"externalId":"511145.b3500","name":"gor","hasFamily":true,"annotation":"glutathione oxidoreductase; Maintains high levels of reduced glutathione in the cytosol","uniprotId":"GSHR_ECOLI"},"6884084":{"id":6884084,"externalId":"511145.b3501","name":"arsR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Transcriptional repressor for the arsEFG operon. ArsE is a trans-acting regulatory protein which controls its own expression. The repressive effect of ArsE is alleviated by oxyions of +III oxidation state of arsenic, antimony, and bismuth, as well as arsenate (As(V)) (By similarity)","uniprotId":"ARSR_ECOLI"},"6884085":{"id":6884085,"externalId":"511145.b3502","name":"arsB","hasFamily":true,"annotation":"arsenite/antimonite transporter; Involved in arsenical resistance. Thought to form the channel of an arsenite pump (By similarity)","uniprotId":"ARSB_ECOLI"},"6884086":{"id":6884086,"externalId":"511145.b3503","name":"arsC","hasFamily":true,"annotation":"arsenate reductase; Reduction of arsenate [As(V)] to arsenite [As(III)]. This protein expands the substrate specificity of ArsAB pump which can extrude arsenite and antimonite to allow for arsenate pumping and resistance (By similarity)","uniprotId":"ARSC_ECOLI"},"6884087":{"id":6884087,"externalId":"511145.b3505","name":"is5B","hasFamily":true,"annotation":"IS5 transposase and trans-activator; Involved in the transposition of the insertion sequence IS5","uniprotId":"INH10_ECOLI"},"6884088":{"id":6884088,"externalId":"511145.b3506","name":"slp","hasFamily":true,"annotation":"outer membrane lipoprotein; The induction of slp may help to stabilize the outer membrane during carbon starvation and stationary phase","uniprotId":"SLP_ECOLI"},"6884089":{"id":6884089,"externalId":"511145.b3507","name":"dctR","hasFamily":false,"annotation":"predicted DNA-binding ranscriptional regulator; May act as a transcriptional regulator of dctA","uniprotId":"DCTR_ECOLI"},"6884090":{"id":6884090,"externalId":"511145.b3508","name":"yhiD","hasFamily":true,"annotation":"putative Mg(2+) transport ATPase, inner membrane protein","uniprotId":"YHID_ECOLI"},"6884091":{"id":6884091,"externalId":"511145.b3509","name":"hdeB","hasFamily":true,"annotation":"acid-resistance protein; Required for optimal acid stress protection, which is important for survival of enteric bacteria in the acidic environment of the host stomach. Exhibits a chaperone-like activity at acidic pH by preventing the aggregation of many different periplasmic proteins","uniprotId":"HDEB_ECOLI"},"6884092":{"id":6884092,"externalId":"511145.b3510","name":"hdeA","hasFamily":false,"annotation":"stress response protein acid-resistance protein; Required for optimal acid stress protection. Exhibits a chaperone-like activity only at pH below 3 by suppressing non- specifically the aggregation of denaturated periplasmic proteins. Important for survival of enteric bacteria in the acidic environment of the host stomach. Also promotes the solubilization at neutral pH of proteins that had aggregated in their presence at acidic pHs. May cooperate with other periplasmic chaperones such as DegP and SurA","uniprotId":"HDEA_ECOLI"},"6884093":{"id":6884093,"externalId":"511145.b3511","name":"hdeD","hasFamily":true,"annotation":"acid-resistance membrane protein","uniprotId":"HDED_ECOLI"},"6884094":{"id":6884094,"externalId":"511145.b3512","name":"gadE","hasFamily":false,"annotation":"DNA-binding transcriptional activator; Regulates the expression of several genes involved in acid resistance. Required for the expression of gadA and gadBC, among others, regardless of media or growth conditions. Binds directly to the 20 bp GAD box found in the control regions of both loci","uniprotId":"GADE_ECOLI"},"6884095":{"id":6884095,"externalId":"511145.b3513","name":"mdtE","hasFamily":true,"annotation":"anaerobic multidrug efflux transporter, ArcA-regulated; Part of a multidrug resistance efflux system that confers resistance to compounds such as rhodamine 6G, erythromycin, doxorubicin, ethidium bromide, TPP, SDS, deoxycholate, crystal violet and benzalkonium","uniprotId":"MDTE_ECOLI"},"6884096":{"id":6884096,"externalId":"511145.b3514","name":"mdtF","hasFamily":false,"annotation":"anaerobic multidrug efflux transporter, ArcA-regulated; Part of a multidrug resistance efflux system that confers resistance to compounds such as rhodamine 6G, erythromycin, doxorubicin, ethidium bromide, TPP, SDS, deoxycholate, crystal violet and benzalkonium","uniprotId":"MDTF_ECOLI"},"6884097":{"id":6884097,"externalId":"511145.b3515","name":"gadW","hasFamily":false,"annotation":"transcriptional activator of gadA and gadBC; Depending on the conditions (growth phase and medium), acts as a positive or negative regulator of gadA and gadBC. Repression occurs directly or via the repression of the expression of gadX. Activation occurs directly by the binding of GadW to the gadA and gadBC promoters","uniprotId":"GADW_ECOLI"},"6884098":{"id":6884098,"externalId":"511145.b3516","name":"gadX","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; Positively regulates the expression of about fifteen genes involved in acid resistance such as gadA, gadB and gadC. Depending on the conditions (growth phase and medium), can repress gadW","uniprotId":"GADX_ECOLI"},"6884099":{"id":6884099,"externalId":"511145.b3517","name":"gadA","hasFamily":true,"annotation":"glutamate decarboxylase A, PLP-dependent; Converts glutamate to gamma-aminobutyrate (GABA), consuming one intracellular proton in the reaction. The gad system helps to maintain a near-neutral intracellular pH when cells are exposed to extremely acidic conditions. The ability to survive transit through the acidic conditions of the stomach is essential for successful colonization of the mammalian host by commensal and pathogenic bacteria","uniprotId":"DCEA_ECOLI"},"6884100":{"id":6884100,"externalId":"511145.b3518","name":"yhjA","hasFamily":true,"annotation":"putative cytochrome C peroxidase","uniprotId":"YHJA_ECOLI"},"6884101":{"id":6884101,"externalId":"511145.b3519","name":"treF","hasFamily":true,"annotation":"cytoplasmic trehalase; Hydrolyzes trehalose to glucose. Could be involved, in cells returning to low osmolarity conditions, in the utilization of the accumulated cytoplasmic trehalose, which was synthesized in response to high osmolarity (By similarity)","uniprotId":"TREF_ECOLI"},"6884102":{"id":6884102,"externalId":"511145.b3520","name":"yhjB","hasFamily":true,"annotation":"putative two-component system response regulator, LuxR-like HTH domain, function unknown","uniprotId":"YHJB_ECOLI"},"6884103":{"id":6884103,"externalId":"511145.b3521","name":"yhjC","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YHJC_ECOLI"},"6884104":{"id":6884104,"externalId":"511145.b3522","name":"yhjD","hasFamily":true,"annotation":"putative alternate lipid exporter, suppressor of msbA and KDO essentiality, inner membrane protein","uniprotId":"YHJD_ECOLI"},"6884105":{"id":6884105,"externalId":"511145.b3523","name":"yhjE","hasFamily":true,"annotation":"inner membrane protein, predicted transporter","uniprotId":"YHJE_ECOLI"},"6884106":{"id":6884106,"externalId":"511145.b3524","name":"yhjG","hasFamily":true,"annotation":"Inner membrane protein, AsmA family","uniprotId":"YHJG_ECOLI"},"6884107":{"id":6884107,"externalId":"511145.b3525","name":"yhjH","hasFamily":true,"annotation":"cyclic-di-GMP phosphodiesterase, FlhDC-regulated; A bis-(3'-5') cyclic diguanylic acid (c-di-GMP) phosphodiesterase involved in regulating the levels of c-di-GMP that control cell motility (the flagella) and adhesion (adhesive curli fimbriae). The YhjH effect on flagella is controlled via the c-di-GMP-binding flagellar brake protein YcgR, however curli expression is not regulated via YcgR. Forms a network with different diguanylate cyclases (DosC, YeaJ, YedQ, YegE have been identified) to regulate c-di-GMP levels. Flagellar activity is high at low c-di-GMP levels whereas curli fimbria  [...] ","uniprotId":"YHJH_ECOLI"},"6884108":{"id":6884108,"externalId":"511145.b3526","name":"kdgK","hasFamily":true,"annotation":"ketodeoxygluconokinase","uniprotId":"KDGK_ECOLI"},"6884109":{"id":6884109,"externalId":"511145.b3527","name":"yhjJ","hasFamily":true,"annotation":"putative zinc-dependent peptidase","uniprotId":"YHJJ_ECOLI"},"6884110":{"id":6884110,"externalId":"511145.b3528","name":"dctA","hasFamily":true,"annotation":"C4-dicarboxylic acid, orotate and citrate transporter; Responsible for the aerobic transport of the dicarboxylates fumarate, L- and D-malate and to a lesser extent succinate, from the periplasm across the inner membrane","uniprotId":"DCTA_ECOLI"},"6884111":{"id":6884111,"externalId":"511145.b3529","name":"yhjK","hasFamily":true,"annotation":"predicted diguanylate cyclase","uniprotId":"YHJK_ECOLI"},"6884112":{"id":6884112,"externalId":"511145.b3530","name":"bcsC","hasFamily":true,"annotation":"cellulose synthase subunit; Required for maximal bacterial cellulose synthesis (By similarity)","uniprotId":"BCSC_ECOLI"},"6884113":{"id":6884113,"externalId":"511145.b3531","name":"bcsZ","hasFamily":true,"annotation":"endo-1,4-D-glucanase; Hydrolyzes carboxymethylcellulose","uniprotId":"GUN_ECOLI"},"6884114":{"id":6884114,"externalId":"511145.b3532","name":"bcsB","hasFamily":true,"annotation":"regulator of cellulose synthase, cyclic di-GMP binding; Binds the cellulose synthase activator, bis-(3'-5') cyclic diguanylic acid (c-di-GMP) (By similarity)","uniprotId":"BCSB_ECOLI"},"6884115":{"id":6884115,"externalId":"511145.b3533","name":"bcsA","hasFamily":true,"annotation":"cellulose synthase, catalytic subunit; Catalytic subunit of cellulose synthase. It polymerizes uridine 5'-diphosphate glucose to cellulose, which is produced as an extracellular component for mechanical and chemical protection at the onset of the stationary phase, when the cells exhibit multicellular behavior (rdar morphotype). Coexpression of cellulose and thin aggregative fimbriae leads to a hydrophobic network with tightly packed cells embedded in a highly inert matrix","uniprotId":"BCSA_ECOLI"},"6884116":{"id":6884116,"externalId":"511145.b3535","name":"yhjR","hasFamily":true,"annotation":"putative protein","uniprotId":"YHJR_ECOLI"},"6884117":{"id":6884117,"externalId":"511145.b3536","name":"bcsE","hasFamily":true,"annotation":"cellulose production protein","uniprotId":"BCSE_ECOLI"},"6884118":{"id":6884118,"externalId":"511145.b3537","name":"bcsF","hasFamily":true,"annotation":"predicted protein","uniprotId":"BCSF_ECOLI"},"6884119":{"id":6884119,"externalId":"511145.b3538","name":"bcsG","hasFamily":true,"annotation":"inner membrane protein, predicted endoglucanase, DUF3260 family","uniprotId":"BCSG_ECOLI"},"6884120":{"id":6884120,"externalId":"511145.b3539","name":"yhjV","hasFamily":true,"annotation":"putative transporter","uniprotId":"YHJV_ECOLI"},"6884121":{"id":6884121,"externalId":"511145.b3540","name":"dppF","hasFamily":true,"annotation":"dipeptide transporter; Part of the binding-protein-dependent transport system for dipeptides. Probably responsible for energy coupling to the transport system","uniprotId":"DPPF_ECOLI"},"6884122":{"id":6884122,"externalId":"511145.b3541","name":"dppD","hasFamily":true,"annotation":"dipeptide/heme transporter; Part of the binding-protein-dependent transport system for dipeptides. Probably responsible for energy coupling to the transport system","uniprotId":"DPPD_ECOLI"},"6884123":{"id":6884123,"externalId":"511145.b3542","name":"dppC","hasFamily":true,"annotation":"dipeptide/heme transporter; Part of the binding-protein-dependent transport system for dipeptides; probably responsible for the translocation of the substrate across the membrane","uniprotId":"DPPC_ECOLI"},"6884124":{"id":6884124,"externalId":"511145.b3543","name":"dppB","hasFamily":true,"annotation":"dipeptide/heme transporter; Part of the binding-protein-dependent transport system for dipeptides; probably responsible for the translocation of the substrate across the membrane","uniprotId":"DPPB_ECOLI"},"6884125":{"id":6884125,"externalId":"511145.b3544","name":"dppA","hasFamily":true,"annotation":"dipeptide transporter; Dipeptide-binding protein of a transport system that can be subject to osmotic shock. DppA is also required for peptide chemotaxis","uniprotId":"DPPA_ECOLI"},"6884126":{"id":6884126,"externalId":"511145.b3546","name":"eptB","hasFamily":true,"annotation":"KDO phosphoethanolamine transferase, Ca(2+)-inducible; Catalyzes the addition of a phosphoethanolamine (pEtN) moiety to the outer 3-deoxy-D-manno-octulosonic acid (KDO) residue of a KDO(2)-lipid A. Phosphatidylethanolamines with one unsaturated acyl group functions as pEtN donors and the reaction releases diacylglycerol","uniprotId":"EPTB_ECOLI"},"6884127":{"id":6884127,"externalId":"511145.b3547","name":"yhjX","hasFamily":true,"annotation":"Inner membrane protein, predicted oxalate-formate antiporter","uniprotId":"YHJX_ECOLI"},"6884128":{"id":6884128,"externalId":"511145.b3548","name":"yhjY","hasFamily":true,"annotation":"conserved protein","uniprotId":"YHJY_ECOLI"},"6884129":{"id":6884129,"externalId":"511145.b3549","name":"tag","hasFamily":true,"annotation":"3-methyl-adenine DNA glycosylase I, constitutive; Hydrolysis of the deoxyribose N-glycosidic bond to excise 3-methyladenine from the damaged DNA polymer formed by alkylation lesions","uniprotId":"3MG1_ECOLI"},"6884130":{"id":6884130,"externalId":"511145.b3550","name":"yiaC","hasFamily":true,"annotation":"putative acyltransferase with acyl-CoA N-acyltransferase domain","uniprotId":"YIAC_ECOLI"},"6884131":{"id":6884131,"externalId":"511145.b3551","name":"bisC","hasFamily":true,"annotation":"biotin sulfoxide reductase; This enzyme may serve as a scavenger, allowing the cell to utilize biotin sulfoxide as a biotin source. It reduces a spontaneous oxidation product of biotin, D-biotin D-sulfoxide (BSO or BDS), back to biotin","uniprotId":"BISC_ECOLI"},"6884132":{"id":6884132,"externalId":"511145.b3552","name":"yiaD","hasFamily":true,"annotation":"multicopy suppressor of bamB","uniprotId":"YIAD_ECOLI"},"6884133":{"id":6884133,"externalId":"511145.b3553","name":"ghrB","hasFamily":true,"annotation":"glyoxylate/hydroxypyruvate reductase B; Catalyzes the NADPH-dependent reduction of glyoxylate and hydroxypyruvate into glycolate and glycerate, respectively. Can also reduce 2,5-diketo-D-gluconate (25DKG) to 5-keto-D- gluconate (5KDG), 2-keto-D-gluconate (2KDG) to D-gluconate, and 2- keto-L-gulonate (2KLG) to L-idonate (IA), but it is not its physiological function. Inactive towards 2-oxoglutarate, oxaloacetate, pyruvate, 5-keto-D-gluconate, D-fructose and L- sorbose. Activity with NAD is very low","uniprotId":"GHRB_ECOLI"},"6884134":{"id":6884134,"externalId":"511145.b3554","name":"yiaF","hasFamily":true,"annotation":"conserved protein","uniprotId":"YIAF_ECOLI"},"6884135":{"id":6884135,"externalId":"511145.b3555","name":"yiaG","hasFamily":true,"annotation":"putative transcriptional regulator, HTH_CROC1 family","uniprotId":"YIAG_ECOLI"},"6884136":{"id":6884136,"externalId":"511145.b3556","name":"cspA","hasFamily":false,"annotation":"RNA chaperone and anti-terminator, cold-inducible; Binds to and stimulates the transcription of the CCAAT- containing, cold-shock-inducible promoters of the H-NS and GyrA proteins. Binds also to the inverted repeat 5'-ATTGG-3'","uniprotId":"CSPA_ECOLI"},"6884137":{"id":6884137,"externalId":"511145.b3557","name":"insJ","hasFamily":true,"annotation":"IS150 transposase A","uniprotId":"INSJ_ECOLI"},"6884138":{"id":6884138,"externalId":"511145.b3558","name":"insK","hasFamily":true,"annotation":"IS150 transposase B; Involved in the transposition of the insertion sequence IS150","uniprotId":"INSK_ECOLI"},"6884139":{"id":6884139,"externalId":"511145.b3559","name":"glyS","hasFamily":true,"annotation":"glycine tRNA synthetase, beta subunit","uniprotId":"SYGB_ECOLI"},"6884140":{"id":6884140,"externalId":"511145.b3560","name":"glyQ","hasFamily":false,"annotation":"glycine tRNA synthetase, alpha subunit","uniprotId":"SYGA_ECOLI"},"6884141":{"id":6884141,"externalId":"511145.b3561","name":"wech","hasFamily":true,"annotation":"O-acetyltransferase for enterobacterial common antigen (ECA)","uniprotId":"YIAH_ECOLI"},"6884142":{"id":6884142,"externalId":"511145.b3562","name":"yiaA","hasFamily":true,"annotation":"inner membrane protein, YiaAB family","uniprotId":"YIAA_ECOLI"},"6884143":{"id":6884143,"externalId":"511145.b3563","name":"yiaB","hasFamily":false,"annotation":"inner membrane protein, YiaAB family","uniprotId":"YIAB_ECOLI"},"6884144":{"id":6884144,"externalId":"511145.b3564","name":"xylB","hasFamily":true,"annotation":"xylulokinase","uniprotId":"XYLB_ECOLI"},"6884145":{"id":6884145,"externalId":"511145.b3565","name":"xylA","hasFamily":true,"annotation":"D-xylose isomerase","uniprotId":"XYLA_ECOLI"},"6884146":{"id":6884146,"externalId":"511145.b3566","name":"xylF","hasFamily":true,"annotation":"D-xylose transporter subunit; Involved in the high-affinity D-xylose membrane transport system. Binds with high affinity to xylose","uniprotId":"XYLF_ECOLI"},"6884147":{"id":6884147,"externalId":"511145.b3567","name":"xylG","hasFamily":true,"annotation":"fused D-xylose transporter subunits of ABC superfamily: ATP-binding components; Part of the ABC transporter complex XylFGH involved in xylose import. Responsible for energy coupling to the transport system (Probable). The XylFGH system can also transport ribose in absence of xylose","uniprotId":"XYLG_ECOLI"},"6884148":{"id":6884148,"externalId":"511145.b3568","name":"xylH","hasFamily":true,"annotation":"D-xylose ABC transporter permease subunit; Part of the binding-protein-dependent transport system for D-xylose. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"XYLH_ECOLI"},"6884149":{"id":6884149,"externalId":"511145.b3569","name":"xylR","hasFamily":true,"annotation":"DNA-binding transcriptional activator, xylose-binding; Regulatory protein for the xylBAFGHR operon","uniprotId":"XYLR_ECOLI"},"6884150":{"id":6884150,"externalId":"511145.b3570","name":"bax","hasFamily":true,"annotation":"conserved protein","uniprotId":"BAX_ECOLI"},"6884151":{"id":6884151,"externalId":"511145.b3571","name":"malS","hasFamily":true,"annotation":"alpha-amylase; Since only maltooligosaccharides up to a chain length of 6 glucose units are actively transported through the cytoplasmic membrane via the membrane-bound complex of three proteins, MalF, MalG, and MalK, longer maltooligosaccharides must first be degraded by the periplasmic alpha-amylase, the MalS protein","uniprotId":"AMY1_ECOLI"},"6884152":{"id":6884152,"externalId":"511145.b3572","name":"avtA","hasFamily":true,"annotation":"valine-pyruvate aminotransferase 1; Involved in the biosynthesis of alanine","uniprotId":"AVTA_ECOLI"},"6884153":{"id":6884153,"externalId":"511145.b3573","name":"yiaI","hasFamily":true,"annotation":"putative hydrogenase, 4Fe-4S ferredoxin-type component","uniprotId":"YSAA_ECOLI"},"6884154":{"id":6884154,"externalId":"511145.b3574","name":"yiaJ","hasFamily":true,"annotation":"DNA-binding transcriptional repressor of yiaK-S operon; Negatively controls the transcription of the yiaKLMNOPQRS operon, which may be involved in the utilization of 2,3-diketo-L-gulonate","uniprotId":"YIAJ_ECOLI"},"6884155":{"id":6884155,"externalId":"511145.b3575","name":"dlgD","hasFamily":true,"annotation":"2,3-diketo-L-gulonate reductase, NADH-dependent; Catalyzes the reduction of 2,3-diketo-L-gulonate in the presence of NADH, to form 3-keto-L-gulonate (By similarity)","uniprotId":"DLGD_ECOLI"},"6884156":{"id":6884156,"externalId":"511145.b3576","name":"yiaL","hasFamily":true,"annotation":"conserved protein","uniprotId":"YIAL_ECOLI"},"6884157":{"id":6884157,"externalId":"511145.b3577","name":"yiaM","hasFamily":true,"annotation":"L-dehydroascorbate transporter, TRAP permease small subunit for TRAP (TRipartite ATP-independent Periplasmic) family transporter YiaMNO; Part of the tripartite ATP-independent periplasmic (TRAP) transport system YiaMNO involved in the uptake of 2,3- diketo-L-gulonate","uniprotId":"YIAM_ECOLI"},"6884158":{"id":6884158,"externalId":"511145.b3578","name":"yiaN","hasFamily":true,"annotation":"L-dehydroascorbate transporter, TRAP permease large subunit for TRAP (TRipartite ATP-independent Periplasmic) family transporter YiaMNO; Part of the tripartite ATP-independent periplasmic (TRAP) transport system YiaMNO involved in the uptake of 2,3- diketo-L-gulonate","uniprotId":"YIAN_ECOLI"},"6884159":{"id":6884159,"externalId":"511145.b3579","name":"yiaO","hasFamily":true,"annotation":"L-dehydroascorbate transporter, periplasmic binding protein for TRAP (TRipartite ATP-independent Periplasmic) family transporter YiaMNO; Part of the tripartite ATP-independent periplasmic (TRAP) transport system YiaMNO involved in the uptake of 2,3- diketo-L-gulonate. This protein specifically binds 2,3-diketo-L- gulonate. Is not able to bind either L-ascorbate or dehydroascorbate","uniprotId":"YIAO_ECOLI"},"6884160":{"id":6884160,"externalId":"511145.b3580","name":"lyx","hasFamily":true,"annotation":"L-xylulose kinase; Catalyzes the phosphorylation of L-xylulose and 3-keto- L-gulonate. Is involved in L-lyxose utilization via xylulose, and may also be involved in the utilization of 2,3-diketo-L-gulonate","uniprotId":"LYXK_ECOLI"},"6884161":{"id":6884161,"externalId":"511145.b3581","name":"sgbH","hasFamily":true,"annotation":"3-keto-L-gulonate 6-phosphate decarboxylase; Catalyzes the decarboxylation of 3-keto-L-gulonate-6-P into L-xylulose-5-P. May be involved in the utilization of 2,3- diketo-L-gulonate","uniprotId":"SGBH_ECOLI"},"6884162":{"id":6884162,"externalId":"511145.b3582","name":"sgbU","hasFamily":true,"annotation":"putative L-xylulose 5-phosphate 3-epimerase; Catalyzes the isomerization of L-xylulose-5-phosphate to L-ribulose-5-phosphate (Potential). May be involved in the utilization of 2,3-diketo-L-gulonate","uniprotId":"SGBU_ECOLI"},"6884163":{"id":6884163,"externalId":"511145.b3583","name":"sgbE","hasFamily":true,"annotation":"L-ribulose-5-phosphate 4-epimerase; Catalyzes the isomerization of L-ribulose 5-phosphate to D-xylulose 5-phosphate. May be involved in the utilization of 2,3- diketo-L-gulonate","uniprotId":"SGBE_ECOLI"},"6884164":{"id":6884164,"externalId":"511145.b3584","name":"yiaT","hasFamily":false,"annotation":"putative outer membrane protein","uniprotId":"YIAT_ECOLI"},"6884165":{"id":6884165,"externalId":"511145.b3585","name":"yiaU","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YIAU_ECOLI"},"6884166":{"id":6884166,"externalId":"511145.b3586","name":"yiaV","hasFamily":true,"annotation":"membrane fusion protein (MFP) component of efflux pump, signal anchor","uniprotId":"YIAV_ECOLI"},"6884167":{"id":6884167,"externalId":"511145.b3587","name":"yiaW","hasFamily":false,"annotation":"inner membrane protein, DUF3302 family","uniprotId":"YIAW_ECOLI"},"6884168":{"id":6884168,"externalId":"511145.b3588","name":"aldB","hasFamily":true,"annotation":"aldehyde dehydrogenase B; Catalyzes the NADP-dependent oxidation of diverse aldehydes such as chloroacetaldehyde, acetaldehyde, propionaldehyde, benzaldehyde, mafosfamide, 4- hydroperoxycyclophosphamide. Its preferred substrates are acetaldehyde and chloroacetaldehyde","uniprotId":"ALDB_ECOLI"},"6884169":{"id":6884169,"externalId":"511145.b3589","name":"yiaY","hasFamily":true,"annotation":"putative Fe-containing alcohol dehydrogenase, Pfam00465 family","uniprotId":"ADH2_ECOLI"},"6884170":{"id":6884170,"externalId":"511145.b3590","name":"selB","hasFamily":true,"annotation":"selenocysteinyl-tRNA-specific translation factor; Translation factor necessary for the incorporation of selenocysteine into proteins. It probably replaces EF-Tu for the insertion of selenocysteine directed by the UGA codon. SelB binds GTP and GDP","uniprotId":"SELB_ECOLI"},"6884171":{"id":6884171,"externalId":"511145.b3591","name":"selA","hasFamily":true,"annotation":"selenocysteine synthase; Converts seryl-tRNA(Sec) to selenocysteinyl-tRNA(Sec) required for selenoprotein biosynthesis. Requires selenophosphate as the selenium-donor molecule","uniprotId":"SELA_ECOLI"},"6884172":{"id":6884172,"externalId":"511145.b3592","name":"yibF","hasFamily":true,"annotation":"glutathione S-transferase homolog; Glutathione (GSH) transferase homolog, that might be involved in selenium metabolism","uniprotId":"YIBF_ECOLI"},"6884173":{"id":6884173,"externalId":"511145.b3593","name":"ybfO","hasFamily":true,"annotation":"rhsA element core protein RshA; Rhs elements have a nonessential function. They may play an important role in the natural ecology of the cell","uniprotId":"RHSA_ECOLI"},"6884174":{"id":6884174,"externalId":"511145.b3594","name":"yibA","hasFamily":true,"annotation":"putative lyase containing HEAT-repeat","uniprotId":"YIBA_ECOLI"},"6884175":{"id":6884175,"externalId":"511145.b3596","name":"yibG","hasFamily":false,"annotation":"conserved protein","uniprotId":"YIBG_ECOLI"},"6884176":{"id":6884176,"externalId":"511145.b3597","name":"yibH","hasFamily":true,"annotation":"predicted protein","uniprotId":"YIBH_ECOLI"},"6884177":{"id":6884177,"externalId":"511145.b3598","name":"yibI","hasFamily":false,"annotation":"inner membrane protein, DUF3302 family","uniprotId":"YIBI_ECOLI"},"6884178":{"id":6884178,"externalId":"511145.b3599","name":"mtlA","hasFamily":true,"annotation":"fused mannitol-specific PTS enzymes: IIA components/IIB components/IIC components; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in mannitol transport","uniprotId":"PTM3C_ECOLI"},"6884179":{"id":6884179,"externalId":"511145.b3600","name":"mtlD","hasFamily":true,"annotation":"mannitol-1-phosphate dehydrogenase, NAD-dependent","uniprotId":"MTLD_ECOLI"},"6884180":{"id":6884180,"externalId":"511145.b3601","name":"mtlR","hasFamily":true,"annotation":"mannitol operon repressor; Acts as a repressor of the mtlAD operon","uniprotId":"MTLR_ECOLI"},"6884181":{"id":6884181,"externalId":"511145.b3602","name":"yibL","hasFamily":true,"annotation":"conserved protein, ribosome-associated","uniprotId":"YIBL_ECOLI"},"6884182":{"id":6884182,"externalId":"511145.b3603","name":"lldP","hasFamily":true,"annotation":"L-lactate permease; Transports L-lactate across the membrane. Can also transport D-lactate and glycolate. Seems to be driven by a proton motive force","uniprotId":"LLDP_ECOLI"},"6884183":{"id":6884183,"externalId":"511145.b3604","name":"lldR","hasFamily":true,"annotation":"dual role activator/repressor for lldPRD operon; May be a regulatory protein for the LCT genes","uniprotId":"LLDR_ECOLI"},"6884184":{"id":6884184,"externalId":"511145.b3605","name":"lldD","hasFamily":true,"annotation":"L-lactate dehydrogenase, FMN-linked","uniprotId":"LLDD_ECOLI"},"6884185":{"id":6884185,"externalId":"511145.b3606","name":"trmL","hasFamily":true,"annotation":"tRNA Leu mC34,mU34 2'-O-methyltransferase, SAM-dependent; Methylates the ribose at the nucleotide 34 wobble position in the two leucyl isoacceptors tRNA(Leu)(CmAA) and tRNA(Leu)(cmnm5UmAA). Catalyzes the methyl transfer from S- adenosyl-L-methionine to the 2'-OH of the wobble nucleotide. Recognition of the target requires a pyridine at position 34 and N(6)-(isopentenyl)-2-methylthioadenosine at position 37","uniprotId":"TRML_ECOLI"},"6884186":{"id":6884186,"externalId":"511145.b3607","name":"cysE","hasFamily":true,"annotation":"serine acetyltransferase","uniprotId":"CYSE_ECOLI"},"6884187":{"id":6884187,"externalId":"511145.b3608","name":"gpsA","hasFamily":true,"annotation":"glycerol-3-phosphate dehydrogenase (NAD+)","uniprotId":"GPDA_ECOLI"},"6884188":{"id":6884188,"externalId":"511145.b3609","name":"secB","hasFamily":true,"annotation":"protein export chaperone; One of the proteins required for the normal export of some preproteins out of the cell cytoplasm. It is a molecular chaperone that binds to a subset of precursor proteins, maintaining them in a translocation-competent state. It also specifically binds to its receptor SecA. Its substrates include AmpC, DegP, FhuA, FkpA, GBP, LamB, MalE (MBP), OmpA, OmpF, OmpT, OmpX, OppA, PhoE, TolB, TolC, YbgF, YcgK, YgiW and YncE","uniprotId":"SECB_ECOLI"},"6884189":{"id":6884189,"externalId":"511145.b3610","name":"grxC","hasFamily":true,"annotation":"glutaredoxin 3; The disulfide bond functions as an electron carrier in the glutathione-dependent synthesis of deoxyribonucleotides by the enzyme ribonucleotide reductase. In addition, it is also involved in reducing some disulfide bonds in a coupled system with glutathione reductase","uniprotId":"GLRX3_ECOLI"},"6884190":{"id":6884190,"externalId":"511145.b3611","name":"yibN","hasFamily":true,"annotation":"putative rhodanese-related sulfurtransferase","uniprotId":"YIBN_ECOLI"},"6884191":{"id":6884191,"externalId":"511145.b3612","name":"gpmI","hasFamily":true,"annotation":"phosphoglycero mutase III, cofactor-independent; Catalyzes the interconversion of 2-phosphoglycerate and 3-phosphoglycerate (By similarity)","uniprotId":"GPMI_ECOLI"},"6884192":{"id":6884192,"externalId":"511145.b3613","name":"envC","hasFamily":true,"annotation":"activator of AmiB,C murein hydrolases, septal ring factor; Activator of the cell wall hydrolases AmiA and AmiB. Required for septal murein cleavage and daughter cell separation during cell division. In vitro, exhibits weak endoproteolytic activity on beta-casein","uniprotId":"ENVC_ECOLI"},"6884193":{"id":6884193,"externalId":"511145.b3614","name":"yibQ","hasFamily":true,"annotation":"putative polysaccharide deacetylase","uniprotId":"YIBQ_ECOLI"},"6884194":{"id":6884194,"externalId":"511145.b3615","name":"yibD","hasFamily":true,"annotation":"putative glycosyl transferase","uniprotId":"YIBD_ECOLI"},"6884195":{"id":6884195,"externalId":"511145.b3616","name":"tdh","hasFamily":true,"annotation":"threonine 3-dehydrogenase, NAD(P)-binding","uniprotId":"TDH_ECOLI"},"6884196":{"id":6884196,"externalId":"511145.b3617","name":"kbl","hasFamily":true,"annotation":"glycine C-acetyltransferase; Catalyzes the cleavage of 2-amino-3-ketobutyrate to glycine and acetyl-CoA","uniprotId":"KBL_ECOLI"},"6884197":{"id":6884197,"externalId":"511145.b3618","name":"htrL","hasFamily":true,"annotation":"conserved protein, pfam09612 family; Involved in lipopolysaccharide biosynthesis","uniprotId":"HTRL_ECOLI"},"6884198":{"id":6884198,"externalId":"511145.b3619","name":"hldD","hasFamily":true,"annotation":"ADP-L-glycero-D-mannoheptose-6-epimerase, NAD(P)-binding; Catalyzes the interconversion between ADP-D-glycero- beta-D-manno-heptose and ADP-L-glycero-beta-D-manno-heptose via an epimerization at carbon 6 of the heptose (By similarity)","uniprotId":"HLDD_ECOLI"},"6884199":{"id":6884199,"externalId":"511145.b3620","name":"rfaF","hasFamily":true,"annotation":"ADP-heptose:LPS heptosyltransferase II","uniprotId":"RFAF_ECOLI"},"6884200":{"id":6884200,"externalId":"511145.b3621","name":"rfaC","hasFamily":true,"annotation":"ADP-heptose:LPS heptosyl transferase I; Heptose transfer to the lipopolysaccharide core. It transfers the innermost heptose to [4'-P](3-deoxy-D-manno- octulosonic acid)2-IVA","uniprotId":"RFAC_ECOLI"},"6884201":{"id":6884201,"externalId":"511145.b3622","name":"rfaL","hasFamily":false,"annotation":"O-antigen ligase; Adds the O-antigen on the glucose group of LPS","uniprotId":"RFAL_ECOLI"},"6884202":{"id":6884202,"externalId":"511145.b3623","name":"waaU","hasFamily":false,"annotation":"lipopolysaccharide core biosynthesis; Adds the terminal N-acetyl-D-glucosamine group on the glucose(II) group of LPS","uniprotId":"WAAU_ECOLI"},"6884203":{"id":6884203,"externalId":"511145.b3624","name":"rfaZ","hasFamily":true,"annotation":"lipopolysaccharide core biosynthesis protein","uniprotId":"RFAZ_ECOLI"},"6884204":{"id":6884204,"externalId":"511145.b3625","name":"rfaY","hasFamily":true,"annotation":"lipopolysaccharide core biosynthesis protein; Catalyzes the phosphorylation of heptose(II) of the outer membrane lipopolysaccharide core (By similarity)","uniprotId":"RFAY_ECOLI"},"6884205":{"id":6884205,"externalId":"511145.b3626","name":"rfaJ","hasFamily":true,"annotation":"UDP-D-glucose:(galactosyl)lipopolysaccharide glucosyltransferase","uniprotId":"RFAJ_ECOLI"},"6884206":{"id":6884206,"externalId":"511145.b3627","name":"rfaI","hasFamily":true,"annotation":"UDP-D-galactose:(glucosyl)lipopolysaccharide- alpha-1,3-D-galactosyltransferase","uniprotId":"RFAI_ECOLI"},"6884207":{"id":6884207,"externalId":"511145.b3628","name":"rfaB","hasFamily":true,"annotation":"UDP-D-galactose:(glucosyl)lipopolysaccharide-1, 6-D-galactosyltransferase; Adds a galactose goup to a glucose group of LPS","uniprotId":"RFAB_ECOLI"},"6884208":{"id":6884208,"externalId":"511145.b3629","name":"rfaS","hasFamily":false,"annotation":"lipopolysaccharide core biosynthesis protein","uniprotId":"RFAS_ECOLI"},"6884209":{"id":6884209,"externalId":"511145.b3630","name":"rfaP","hasFamily":true,"annotation":"kinase that phosphorylates core heptose of lipopolysaccharide; Catalyzes the phosphorylation of heptose(I) of the outer membrane lipopolysaccharide core (By similarity)","uniprotId":"RFAP_ECOLI"},"6884210":{"id":6884210,"externalId":"511145.b3631","name":"rfaG","hasFamily":true,"annotation":"glucosyltransferase I; Involved in the addition of the first glucose residue to the lipopolysaccharide core","uniprotId":"RFAG_ECOLI"},"6884211":{"id":6884211,"externalId":"511145.b3632","name":"rfaQ","hasFamily":true,"annotation":"lipopolysaccharide core biosynthesis protein; Catalyzes heptose transfer to the lipopolysaccharide core. It transfers a heptose, called heptose(III), to the heptose(II) of the inner core (By similarity)","uniprotId":"RFAQ_ECOLI"},"6884212":{"id":6884212,"externalId":"511145.b3633","name":"waaA","hasFamily":true,"annotation":"3-deoxy-D-manno-octulosonic-acid transferase (KDO transferase); Involved in lipopolysaccharide (LPS) biosynthesis. Catalyzes the transfer of two 3-deoxy-D-manno-octulosonate (Kdo) residues from CMP-Kdo to lipid IV(A), the tetraacyldisaccharide- 1,4'-bisphosphate precursor of lipid A","uniprotId":"KDTA_ECOLI"},"6884213":{"id":6884213,"externalId":"511145.b3634","name":"coaD","hasFamily":true,"annotation":"pantetheine-phosphate adenylyltransferase; Reversibly transfers an adenylyl group from ATP to 4'- phosphopantetheine, yielding dephospho-CoA (dPCoA) and pyrophosphate (By similarity)","uniprotId":"COAD_ECOLI"},"6884214":{"id":6884214,"externalId":"511145.b3635","name":"mutM","hasFamily":true,"annotation":"formamidopyrimidine/5-formyluracil/ 5-hydroxymethyluracil DNA glycosylase; Involved in base excision repair of DNA damaged by oxidation or by mutagenic agents. Acts as DNA glycosylase that recognizes and removes damaged bases. Has a preference for oxidized purines, such as 7,8-dihydro-8-oxoguanine (8-oxoG) and its derivatives such as guanidinohydantoin:C and spiroiminodihydantoin:C, however it also acts on thymine glycol:G, 5,6-dihydrouracil:G and 5-hydroxyuracil:G. Has AP (apurinic/apyrimidinic) lyase activity and introduces nicks in the DNA strand. Cleaves the DNA backbone by beta-de [...] ","uniprotId":"FPG_ECOLI"},"6884215":{"id":6884215,"externalId":"511145.b3636","name":"rpmG","hasFamily":false,"annotation":"50S ribosomal subunit protein L33","uniprotId":"RL33_ECOLI"},"6884216":{"id":6884216,"externalId":"511145.b3637","name":"rpmB","hasFamily":true,"annotation":"50S ribosomal subunit protein L28","uniprotId":"RL28_ECOLI"},"6884217":{"id":6884217,"externalId":"511145.b3638","name":"radC","hasFamily":true,"annotation":"conserved protein, UPF0758 family","uniprotId":"YICR_ECOLI"},"6884218":{"id":6884218,"externalId":"511145.b3639","name":"dfp","hasFamily":true,"annotation":"fused 4'-phosphopantothenoylcysteine decarboxylase/phosphopantothenoylcysteine synthetase, FMN-binding; Catalyzes two steps in the biosynthesis of coenzyme A. In the first step cysteine is conjugated to 4'-phosphopantothenate to form 4-phosphopantothenoylcysteine, in the latter compound is decarboxylated to form 4'-phosphopantotheine","uniprotId":"COABC_ECOLI"},"6884219":{"id":6884219,"externalId":"511145.b3640","name":"dut","hasFamily":true,"annotation":"deoxyuridinetriphosphatase; This enzyme is involved in nucleotide metabolism: it produces dUMP, the immediate precursor of thymidine nucleotides and it decreases the intracellular concentration of dUTP so that uracil cannot be incorporated into DNA (By similarity)","uniprotId":"DUT_ECOLI"},"6884220":{"id":6884220,"externalId":"511145.b3641","name":"ttk","hasFamily":true,"annotation":"DNA-binding anti-FtsZ division inhibitor; Required for nucleoid occlusion (NO) phenomenon, which prevents Z-ring formation and cell division over the nucleoid. Acts as a DNA-associated cell division inhibitor that binds simultaneously chromosomal DNA and FtsZ, and disrupts the assembly of FtsZ polymers. SlmA-DNA-binding sequences (SBS) are dispersed on non-Ter regions of the chromosome, preventing FtsZ polymerization at these regions (By similarity)","uniprotId":"SLMA_ECOLI"},"6884221":{"id":6884221,"externalId":"511145.b3642","name":"pyrE","hasFamily":true,"annotation":"orotate phosphoribosyltransferase; Catalyzes the transfer of a ribosyl phosphate group from 5-phosphoribose 1-diphosphate to orotate, leading to the formation of orotidine monophosphate (OMP) (By similarity)","uniprotId":"PYRE_ECOLI"},"6884222":{"id":6884222,"externalId":"511145.b3643","name":"rph","hasFamily":true,"annotation":"defective ribonuclease PH; Phosphorolytic exoribonuclease that removes nucleotide residues following the -CCA terminus of tRNA and adds nucleotides to the ends of RNA molecules by using nucleoside diphosphates as substrates. Also acts to regulate the attenuation of PyrE","uniprotId":"RNPH_ECOLI"},"6884223":{"id":6884223,"externalId":"511145.b3644","name":"yicC","hasFamily":true,"annotation":"conserved protein, UPF0701 family","uniprotId":"YICC_ECOLI"},"6884224":{"id":6884224,"externalId":"511145.b3645","name":"dinD","hasFamily":false,"annotation":"DNA-damage-inducible protein","uniprotId":"DIND_ECOLI"},"6884225":{"id":6884225,"externalId":"511145.b3646","name":"yicG","hasFamily":true,"annotation":"inner membrane protein, UPF0126 family","uniprotId":"YICG_ECOLI"},"6884226":{"id":6884226,"externalId":"511145.b3647","name":"ligB","hasFamily":true,"annotation":"DNA ligase, NAD(+)-dependent; Catalyzes the formation of phosphodiester linkages between 5'-phosphoryl and 3'-hydroxyl groups in double-stranded DNA using NAD as a coenzyme and as the energy source for the reaction (By similarity)","uniprotId":"LIGB_ECOLI"},"6884227":{"id":6884227,"externalId":"511145.b3648","name":"gmk","hasFamily":true,"annotation":"guanylate kinase; Essential for recycling GMP and indirectly, cGMP (By similarity)","uniprotId":"KGUA_ECOLI"},"6884228":{"id":6884228,"externalId":"511145.b3649","name":"rpoZ","hasFamily":true,"annotation":"RNA polymerase, omega subunit; Promotes RNA polymerase assembly. Latches the N- and C- terminal regions of the beta' subunit thereby facilitating its interaction with the beta and alpha subunits (By similarity)","uniprotId":"RPOZ_ECOLI"},"6884229":{"id":6884229,"externalId":"511145.b3650","name":"spoT","hasFamily":true,"annotation":"bifunctional (p)ppGpp synthetase II/ guanosine-3',5'-bis pyrophosphate 3'-pyrophosphohydrolase; In eubacteria ppGpp (guanosine 3'-diphosphate 5-' diphosphate) is a mediator of the stringent response that coordinates a variety of cellular activities in response to changes in nutritional abundance. This enzyme catalyzes both the synthesis and degradation of ppGpp. The second messengers ppGpp and c-di-GMP together control biofilm formation in response to translational stress; ppGpp represses biofilm formation while c- di-GMP induces it","uniprotId":"SPOT_ECOLI"},"6884230":{"id":6884230,"externalId":"511145.b3651","name":"spoU","hasFamily":true,"annotation":"tRNA mG18-2'-O-methyltransferase, SAM-dependen; Specifically methylates guanosine-18 in various tRNAs","uniprotId":"TRMH_ECOLI"},"6884231":{"id":6884231,"externalId":"511145.b3652","name":"recG","hasFamily":true,"annotation":"ATP-dependent DNA helicase; Critical role in recombination and DNA repair. Help process Holliday junction intermediates to mature products by catalyzing branch migration. Has a DNA unwinding activity characteristic of a DNA helicase with a 3' to 5' polarity. RecG unwind branched duplex DNA (Y-DNA). Has a role in constitutive stable DNA replication (CsdR) and R-loop formation","uniprotId":"RECG_ECOLI"},"6884232":{"id":6884232,"externalId":"511145.b3653","name":"gltS","hasFamily":true,"annotation":"glutamate transporter; Catalyzes the sodium-dependent uptake of extracellular glutamate","uniprotId":"GLTS_ECOLI"},"6884233":{"id":6884233,"externalId":"511145.b3654","name":"xanP","hasFamily":true,"annotation":"xanthine permease; Specific, proton motive force-dependent high-affinity transporter for xanthine","uniprotId":"XANP_ECOLI"},"6884234":{"id":6884234,"externalId":"511145.b3655","name":"yicH","hasFamily":true,"annotation":"conserved protein","uniprotId":"YICH_ECOLI"},"6884235":{"id":6884235,"externalId":"511145.b3656","name":"yicI","hasFamily":true,"annotation":"putative alpha-glucosidase; Can catalyze the transfer of alpha-xylosyl residue from alpha-xyloside to xylose, glucose, mannose, fructose, maltose, isomaltose, nigerose, kojibiose, sucrose and trehalose","uniprotId":"XYLS_ECOLI"},"6884236":{"id":6884236,"externalId":"511145.b3657","name":"yicJ","hasFamily":true,"annotation":"putative transporter","uniprotId":"YICJ_ECOLI"},"6884237":{"id":6884237,"externalId":"511145.b3659","name":"setC","hasFamily":true,"annotation":"predicted sugar efflux system; Involved in the efflux of sugars. The physiological role may be the detoxification of non-metabolizable sugar analogs","uniprotId":"SETC_ECOLI"},"6884238":{"id":6884238,"externalId":"511145.b3660","name":"yicL","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YICL_ECOLI"},"6884239":{"id":6884239,"externalId":"511145.b3661","name":"nlpA","hasFamily":true,"annotation":"cytoplasmic membrane lipoprotein-28","uniprotId":"NLPA_ECOLI"},"6884240":{"id":6884240,"externalId":"511145.b3662","name":"nepI","hasFamily":true,"annotation":"putative transporter; Involved in the efflux of purine ribonucleosides, such as guanosine, adenosine and especially inosine. Involved in the resistance to 6-mercaptopurine (By similarity)","uniprotId":"NEPI_ECOLI"},"6884241":{"id":6884241,"externalId":"511145.b3663","name":"yicN","hasFamily":true,"annotation":"conserved protein","uniprotId":"YICN_ECOLI"},"6884242":{"id":6884242,"externalId":"511145.b3664","name":"yicO","hasFamily":true,"annotation":"Probable adenine permease","uniprotId":"YICO_ECOLI"},"6884243":{"id":6884243,"externalId":"511145.b3665","name":"ade","hasFamily":true,"annotation":"cryptic adenine deaminase","uniprotId":"ADEC_ECOLI"},"6884244":{"id":6884244,"externalId":"511145.b3666","name":"uhpT","hasFamily":true,"annotation":"hexose phosphate transporter; Transport protein for sugar phosphate uptake","uniprotId":"UHPT_ECOLI"},"6884245":{"id":6884245,"externalId":"511145.b3667","name":"uhpC","hasFamily":true,"annotation":"membrane protein regulates uhpT expression; UhpC is required for uhpT expression, it may act jointly with the UhpB signaling protein","uniprotId":"UHPC_ECOLI"},"6884246":{"id":6884246,"externalId":"511145.b3668","name":"uhpB","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory sytem with UhpA; Member of the two-component regulatory system UhpB/UhpA involved in the regulation of the uptake of hexose phosphates. UhpB functions as a membrane-associated protein kinase that phosphorylates UhpA in response to environmental signals","uniprotId":"UHPB_ECOLI"},"6884247":{"id":6884247,"externalId":"511145.b3669","name":"uhpA","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system wtih UhpB; Appears to be the positive activator of UhpT transcription","uniprotId":"UHPA_ECOLI"},"6884248":{"id":6884248,"externalId":"511145.b3670","name":"ilvN","hasFamily":true,"annotation":"acetolactate synthase I, small subunit","uniprotId":"ILVN_ECOLI"},"6884249":{"id":6884249,"externalId":"511145.b3671","name":"ilvB","hasFamily":true,"annotation":"acetolactate synthase I, large subunit","uniprotId":"ILVB_ECOLI"},"6884250":{"id":6884250,"externalId":"511145.b3672","name":"ivbL","hasFamily":false,"annotation":"ilvB operon leader peptide; This protein is involved in control of the biosynthesis of isoleucine, leucine, and valine","uniprotId":"LPIV_ECOLI"},"6884251":{"id":6884251,"externalId":"511145.b3673","name":"emrD","hasFamily":true,"annotation":"multidrug efflux system protein; Multidrug resistance pump that participates in a low energy shock adaptative response","uniprotId":"EMRD_ECOLI"},"6884252":{"id":6884252,"externalId":"511145.b3674","name":"yidF","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YIDF_ECOLI"},"6884253":{"id":6884253,"externalId":"511145.b3675","name":"yidG","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YIDG_ECOLI"},"6884254":{"id":6884254,"externalId":"511145.b3676","name":"yidH","hasFamily":true,"annotation":"inner membrane protein, DUF202 family","uniprotId":"YIDH_ECOLI"},"6884255":{"id":6884255,"externalId":"511145.b3677","name":"yidI","hasFamily":false,"annotation":"inner membrane protein","uniprotId":"YIDI_ECOLI"},"6884256":{"id":6884256,"externalId":"511145.b3678","name":"yidJ","hasFamily":true,"annotation":"putative sulfatase/phosphatase","uniprotId":"YIDJ_ECOLI"},"6884257":{"id":6884257,"externalId":"511145.b3679","name":"yidK","hasFamily":true,"annotation":"putative transporter","uniprotId":"YIDK_ECOLI"},"6884258":{"id":6884258,"externalId":"511145.b3680","name":"yidL","hasFamily":false,"annotation":"putative transcriptional regulator, AraC family","uniprotId":"YIDL_ECOLI"},"6884259":{"id":6884259,"externalId":"511145.b3684","name":"yidP","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YIDP_ECOLI"},"6884260":{"id":6884260,"externalId":"511145.b3685","name":"yidE","hasFamily":true,"annotation":"putative transporter","uniprotId":"YIDE_ECOLI"},"6884261":{"id":6884261,"externalId":"511145.b3686","name":"ibpB","hasFamily":true,"annotation":"heat shock chaperone; Associates with aggregated proteins, together with IbpA, to stabilize and protect them from irreversible denaturation and extensive proteolysis during heat shock and oxidative stress. Aggregated proteins bound to the IbpAB complex are more efficiently refolded and reactivated by the ATP-dependent chaperone systems ClpB and DnaK/DnaJ/GrpE. Its activity is ATP- independent (By similarity)","uniprotId":"IBPB_ECOLI"},"6884262":{"id":6884262,"externalId":"511145.b3687","name":"ibpA","hasFamily":true,"annotation":"heat shock chaperone; Associates with aggregated proteins, together with IbpB, to stabilize and protect them from irreversible denaturation and extensive proteolysis during heat shock and oxidative stress. Aggregated proteins bound to the IbpAB complex are more efficiently refolded and reactivated by the ATP-dependent chaperone systems ClpB and DnaK/DnaJ/GrpE. Its activity is ATP- independent (By similarity)","uniprotId":"IBPA_ECOLI"},"6884263":{"id":6884263,"externalId":"511145.b3688","name":"ecfI","hasFamily":true,"annotation":"conserved outer membrane protein","uniprotId":"YIDQ_ECOLI"},"6884264":{"id":6884264,"externalId":"511145.b3689","name":"yidR","hasFamily":true,"annotation":"conserved protein","uniprotId":"YIDR_ECOLI"},"6884265":{"id":6884265,"externalId":"511145.b3690","name":"cbrA","hasFamily":false,"annotation":"colicin M resistance protein","uniprotId":"CBRA_ECOLI"},"6884266":{"id":6884266,"externalId":"511145.b3691","name":"dgoT","hasFamily":true,"annotation":"D-galactonate transporter; Intake of galactonate into the cell","uniprotId":"DGOT_ECOLI"},"6884267":{"id":6884267,"externalId":"511145.b3693","name":"dgoK","hasFamily":true,"annotation":"2-oxo-3-deoxygalactonate kinase","uniprotId":"DGOK_ECOLI"},"6884268":{"id":6884268,"externalId":"511145.b3696","name":"yidX","hasFamily":true,"annotation":"putative lipoproteinC","uniprotId":"YIDX_ECOLI"},"6884269":{"id":6884269,"externalId":"511145.b3697","name":"yidA","hasFamily":true,"annotation":"sugar phosphate phosphatase; substrates include erythrose 4-P and mannose 1-P phosphatase; Catalyzes the dephosphorylation of different sugar phosphates including erythrose-4-phosphate (Ery4P), ribose-5- phosphate (Ribu5P), fructose-1-phosphate (Fru1P), fructose-6- phosphate (Fru6P), glucose-6-P (Glu6P), and also imidodiphosphate (Imido-di-P) and acetyl phosphate (Acetyl-P). Selectively hydrolyzes alpha-D-glucose-1-phosphate (Glu1P) and has no activity with the beta form","uniprotId":"YIDA_ECOLI"},"6884270":{"id":6884270,"externalId":"511145.b3698","name":"yidB","hasFamily":false,"annotation":"conserved protein, DUF937 family","uniprotId":"YIDB_ECOLI"},"6884271":{"id":6884271,"externalId":"511145.b3699","name":"gyrB","hasFamily":true,"annotation":"DNA gyrase, subunit B; DNA gyrase negatively supercoils closed circular double- stranded DNA in an ATP-dependent manner and also catalyzes the interconversion of other topological isomers of double-stranded DNA rings, including catenanes and knotted rings (By similarity)","uniprotId":"GYRB_ECOLI"},"6884272":{"id":6884272,"externalId":"511145.b3700","name":"recF","hasFamily":true,"annotation":"gap repair protein; The RecF protein is involved in DNA metabolism; it is required for DNA replication and normal SOS inducibility. RecF binds preferentially to single-stranded, linear DNA. It also seems to bind ATP (By similarity)","uniprotId":"RECF_ECOLI"},"6884273":{"id":6884273,"externalId":"511145.b3701","name":"dnaN","hasFamily":true,"annotation":"DNA polymerase III, beta subunit; DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. This DNA polymerase also exhibits 3' to 5' exonuclease activity. The beta chain is required for initiation of replication; once it is clamped onto DNA it slides freely (bidirectional and ATP- independent) along duplex DNA. Coordinates protein traffic at the replication fork, where it interacts with multiple DNA polymerases","uniprotId":"DPO3B_ECOLI"},"6884274":{"id":6884274,"externalId":"511145.b3702","name":"dnaA","hasFamily":true,"annotation":"chromosomal replication initiator protein DnaA, DNA-binding transcriptional dual regulator; Plays a key role in the initiation and regulation of chromosomal replication. Binds in an ATP-dependent fashion to the origin of replication (oriC) to inititate formation of the DNA replication inititation complex exactly once per cell cycle. Binds the DnaA box (consensus sequence 5'-TTATC[CA]A[CA]A-3'); subsequent binding of DNA polymerase III subunits leads to replisome formation. The DnaA-ATP form converts to DnaA-ADP; once converted to ADP the protein cannot inititate replication, ensuring o [...] ","uniprotId":"DNAA_ECOLI"},"6884275":{"id":6884275,"externalId":"511145.b3703","name":"rpmH","hasFamily":false,"annotation":"50S ribosomal subunit protein L34","uniprotId":"RL34_ECOLI"},"6884276":{"id":6884276,"externalId":"511145.b3704","name":"rnpA","hasFamily":true,"annotation":"protein C5 component of RNase P; RNaseP catalyzes the removal of the 5'-leader sequence from pre-tRNA to produce the mature 5'-terminus. It can also cleave other RNA substrates such as 4.5S RNA. The protein component plays an auxiliary but essential role in vivo by binding to the 5'-leader sequence and broadening the substrate specificity of the ribozyme (By similarity)","uniprotId":"RNPA_ECOLI"},"6884277":{"id":6884277,"externalId":"511145.b3705","name":"yidC","hasFamily":true,"annotation":"membrane protein insertase; Inner membrane protein required for the insertion and/or proper folding and/or complex formation of integral inner membrane proteins. Involved in integration of membrane proteins that insert dependently and independently of the Sec translocase complex, as well as at least 2 lipoproteins. Its own insertion requires SRP and is Sec translocase-dependent. Essential for the integration of Sec-dependent subunit a of the F(0)ATP synthase, FtsQ and SecE proteins and for Sec-independent subunit c of the F(0)ATP synthase, M13 phage procoat and the N-terminus of leader [...] ","uniprotId":"YIDC_ECOLI"},"6884278":{"id":6884278,"externalId":"511145.b3706","name":"mnmE","hasFamily":true,"annotation":"GTPase; Exhibits a very high intrinsic GTPase hydrolysis rate. Involved in the addition of a carboxymethylaminomethyl (cmnm) group at the wobble position (U34) of certain tRNAs, forming tRNA- cmnm(5)s(2)U34 (By similarity)","uniprotId":"MNME_ECOLI"},"6884279":{"id":6884279,"externalId":"511145.b3707","name":"tnaL","hasFamily":false,"annotation":"tryptophanase leader peptide","uniprotId":"LPTN_ECOLI"},"6884280":{"id":6884280,"externalId":"511145.b3708","name":"tnaA","hasFamily":true,"annotation":"tryptophanase/L-cysteine desulfhydrase, PLP-dependent","uniprotId":"TNAA_ECOLI"},"6884281":{"id":6884281,"externalId":"511145.b3709","name":"tnaB","hasFamily":false,"annotation":"tryptophan transporter of low affinity; Involved in tryptophan transport across the cytoplasmic membrane. Plays a role in transporting tryptophan which is to be used catabolically","uniprotId":"TNAB_ECOLI"},"6884282":{"id":6884282,"externalId":"511145.b3710","name":"mdtL","hasFamily":true,"annotation":"multidrug efflux system protein; Confers resistance to chloramphenicol (By similarity)","uniprotId":"MDTL_ECOLI"},"6884283":{"id":6884283,"externalId":"511145.b3711","name":"yidZ","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator; Involved in anaerobic NO protection (By similarity)","uniprotId":"YIDZ_ECOLI"},"6884284":{"id":6884284,"externalId":"511145.b3712","name":"yieE","hasFamily":true,"annotation":"putative phosphopantetheinyl transferase, COG2091 family","uniprotId":"YIEE_ECOLI"},"6884285":{"id":6884285,"externalId":"511145.b3713","name":"chrR","hasFamily":true,"annotation":"chromate reductase, Class I, flavoprotein","uniprotId":"YIEF_ECOLI"},"6884286":{"id":6884286,"externalId":"511145.b3714","name":"purP","hasFamily":true,"annotation":"adenine permease, high affinity; High-affinity uptake of adenine","uniprotId":"PURP_ECOLI"},"6884287":{"id":6884287,"externalId":"511145.b3715","name":"yieH","hasFamily":true,"annotation":"phosphoenolpyruvate and 6-phosphogluconate phosphatase; Catalyzes strongly the dephosphorylation of 6- phosphogluconate (6P-Glu) and slightly the dephosphorylation of dihydroxyacetone phosphate (DHAP) and phosphoenolpyruvate (PEP). Also hydrolyzes both purines (GMP and IMP) and pyrimidines as secondary substrates","uniprotId":"YIEH_ECOLI"},"6884288":{"id":6884288,"externalId":"511145.b3716","name":"cbrB","hasFamily":false,"annotation":"inner membrane protein, creBC regulon","uniprotId":"CBRB_ECOLI"},"6884289":{"id":6884289,"externalId":"511145.b3717","name":"cbrC","hasFamily":true,"annotation":"conserved protein, UPF0167 family","uniprotId":"CBRC_ECOLI"},"6884290":{"id":6884290,"externalId":"511145.b3718","name":"yieK","hasFamily":true,"annotation":"putative 6-phosphogluconolactonase","uniprotId":"YIEK_ECOLI"},"6884291":{"id":6884291,"externalId":"511145.b3719","name":"yieL","hasFamily":false,"annotation":"putative xylanase","uniprotId":"YIEL_ECOLI"},"6884292":{"id":6884292,"externalId":"511145.b3720","name":"bglH","hasFamily":true,"annotation":"carbohydrate-specific outer membrane porin, cryptic; Part of a cryptic operon that is poorly expressed in vivo. May be an ancestral sugar porin with a broad carbohydrate specificity; it binds aromatic beta-D-glucosides such as arbutin and salicin, but with low affinity compared to the binding of maltooligosaccharides to the LamB porin","uniprotId":"BGLH_ECOLI"},"6884293":{"id":6884293,"externalId":"511145.b3721","name":"bglB","hasFamily":true,"annotation":"cryptic phospho-beta-glucosidase B; Catalyzes the hydrolysis of phosphorylated beta- glucosides into glucose-6-phosphate (G-6-P) and aglycone. It has a high affinity for phosphorylated aromatic beta-glucosides (p- nitrophenyl-beta-glucoside, phenyl beta-glucoside, arbutin and phosphorylated salicin), and a low affinity for phosphorylated beta-methyl-glucoside","uniprotId":"BGLB_ECOLI"},"6884294":{"id":6884294,"externalId":"511145.b3722","name":"bglF","hasFamily":true,"annotation":"fused beta-glucoside-specific PTS enzymes: IIA component/IIB component/IIC component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in beta-glucoside transport","uniprotId":"PTV3B_ECOLI"},"6884295":{"id":6884295,"externalId":"511145.b3723","name":"bglG","hasFamily":true,"annotation":"transcriptional antiterminator of the bgl operon; Mediates the positive regulation of the beta-glucoside (bgl) operon by functioning as a transcriptional antiterminator. This is a RNA-binding protein that recognizes a specific sequence located just upstream of two termination sites within the operon","uniprotId":"BGLG_ECOLI"},"6884296":{"id":6884296,"externalId":"511145.b3724","name":"phoU","hasFamily":true,"annotation":"negative regulator of PhoR/PhoB two-component regulator; Part of the phosphate (Pho) regulon, which plays a key role in phosphate homeostasis. Encoded together with proteins of the phosphate-specific transport (Pst) system in the polycistronic pstSCAB-phoU operon. PhoU is essential for the repression of the Pho regulon at high phosphate conditions. In this role, it may bind, possibly as a chaperone, to PhoR, PhoB or a PhoR-PhoB complex to promote dephosphorylation of phospho-PhoB, or inhibit formation of the PhoR-PhoB transitory complex. Is also part of complex networks important for b [...] ","uniprotId":"PHOU_ECOLI"},"6884297":{"id":6884297,"externalId":"511145.b3725","name":"pstB","hasFamily":true,"annotation":"phosphate transporter subunit; Part of the ABC transporter complex PstSACB involved in phosphate import. Responsible for energy coupling to the transport system (By similarity)","uniprotId":"PSTB_ECOLI"},"6884298":{"id":6884298,"externalId":"511145.b3726","name":"pstA","hasFamily":true,"annotation":"phosphate transporter subunit; Part of the binding-protein-dependent transport system for phosphate; probably responsible for the translocation of the substrate across the membrane","uniprotId":"PSTA_ECOLI"},"6884299":{"id":6884299,"externalId":"511145.b3727","name":"pstC","hasFamily":true,"annotation":"phosphate transporter subunit; Part of the binding-protein-dependent transport system for phosphate; probably responsible for the translocation of the substrate across the membrane","uniprotId":"PSTC_ECOLI"},"6884300":{"id":6884300,"externalId":"511145.b3728","name":"phoS","hasFamily":true,"annotation":"periplasmic phosphate binding protein, high-affinity; Part of the ABC transporter complex PstSACB involved in phosphate import (By similarity)","uniprotId":"PSTS_ECOLI"},"6884301":{"id":6884301,"externalId":"511145.b3729","name":"glmS","hasFamily":true,"annotation":"L-glutamine:D-fructose-6-phosphate aminotransferase; Catalyzes the first step in hexosamine metabolism, converting fructose-6P into glucosamine-6P using glutamine as a nitrogen source (By similarity)","uniprotId":"GLMS_ECOLI"},"6884302":{"id":6884302,"externalId":"511145.b3730","name":"glmU","hasFamily":true,"annotation":"fused N-acetyl glucosamine-1-phosphate uridyltransferase/glucosamine-1-phosphate acetyl transferase; Catalyzes the last two sequential reactions in the de novo biosynthetic pathway for UDP-N-acetylglucosamine (UDP- GlcNAc). The C-terminal domain catalyzes the transfer of acetyl group from acetyl coenzyme A to glucosamine-1-phosphate (GlcN-1-P) to produce N-acetylglucosamine-1-phosphate (GlcNAc-1-P), which is converted into UDP-GlcNAc by the transfer of uridine 5- monophosphate (from uridine 5-triphosphate), a reaction catalyzed by the N-terminal domain (By similarity)","uniprotId":"GLMU_ECOLI"},"6884303":{"id":6884303,"externalId":"511145.b3731","name":"atpC","hasFamily":true,"annotation":"F1 sector of membrane-bound ATP synthase, epsilon subunit; Produces ATP from ADP in the presence of a proton gradient across the membrane (By similarity)","uniprotId":"ATPE_ECOLI"},"6884304":{"id":6884304,"externalId":"511145.b3732","name":"atpD","hasFamily":true,"annotation":"F1 sector of membrane-bound ATP synthase, beta subunit; Produces ATP from ADP in the presence of a proton gradient across the membrane. The catalytic sites are hosted primarily by the beta subunits (By similarity)","uniprotId":"ATPB_ECOLI"},"6884305":{"id":6884305,"externalId":"511145.b3733","name":"atpG","hasFamily":true,"annotation":"F1 sector of membrane-bound ATP synthase, gamma subunit; Produces ATP from ADP in the presence of a proton gradient across the membrane. The gamma chain is believed to be important in regulating ATPase activity and the flow of protons through the CF(0) complex (By similarity)","uniprotId":"ATPG_ECOLI"},"6884306":{"id":6884306,"externalId":"511145.b3734","name":"atpA","hasFamily":true,"annotation":"F1 sector of membrane-bound ATP synthase, alpha subunit; Produces ATP from ADP in the presence of a proton gradient across the membrane. The alpha chain is a regulatory subunit (By similarity)","uniprotId":"ATPA_ECOLI"},"6884307":{"id":6884307,"externalId":"511145.b3735","name":"atpH","hasFamily":true,"annotation":"F1 sector of membrane-bound ATP synthase, delta subunit; F(1)F(0) ATP synthase produces ATP from ADP in the presence of a proton or sodium gradient. F-type ATPases consist of two structural domains, F(1) containing the extramembraneous catalytic core and F(0) containing the membrane proton channel, linked together by a central stalk and a peripheral stalk. During catalysis, ATP synthesis in the catalytic domain of F(1) is coupled via a rotary mechanism of the central stalk subunits to proton translocation (By similarity)","uniprotId":"ATPD_ECOLI"},"6884308":{"id":6884308,"externalId":"511145.b3736","name":"atpF","hasFamily":true,"annotation":"F0 sector of membrane-bound ATP synthase, subunit b; F(1)F(0) ATP synthase produces ATP from ADP in the presence of a proton or sodium gradient. F-type ATPases consist of two structural domains, F(1) containing the extramembraneous catalytic core and F(0) containing the membrane proton channel, linked together by a central stalk and a peripheral stalk. During catalysis, ATP synthesis in the catalytic domain of F(1) is coupled via a rotary mechanism of the central stalk subunits to proton translocation (By similarity)","uniprotId":"ATPF_ECOLI"},"6884309":{"id":6884309,"externalId":"511145.b3737","name":"atpE","hasFamily":false,"annotation":"F0 sector of membrane-bound ATP synthase, subunit c; F(1)F(0) ATP synthase produces ATP from ADP in the presence of a proton or sodium gradient. F-type ATPases consist of two structural domains, F(1) containing the extramembraneous catalytic core and F(0) containing the membrane proton channel, linked together by a central stalk and a peripheral stalk. During catalysis, ATP synthesis in the catalytic domain of F(1) is coupled via a rotary mechanism of the central stalk subunits to proton translocation","uniprotId":"ATPL_ECOLI"},"6884310":{"id":6884310,"externalId":"511145.b3738","name":"atpB","hasFamily":true,"annotation":"F0 sector of membrane-bound ATP synthase, subunit a; Key component of the proton channel; it plays a direct role in the translocation of protons across the membrane (By similarity)","uniprotId":"ATP6_ECOLI"},"6884311":{"id":6884311,"externalId":"511145.b3739","name":"atpI","hasFamily":true,"annotation":"ATP synthase, membrane-bound accessory factor; A possible function for this protein is to guide the assembly of the membrane sector of the ATPase enzyme complex","uniprotId":"ATPZ_ECOLI"},"6884312":{"id":6884312,"externalId":"511145.b3740","name":"gidB","hasFamily":true,"annotation":"16S rRNA m(7)G527 methyltransferase, SAM-dependent; glucose-inhibited cell-division protein; Specifically methylates the N7 position of guanine in position 527 of 16S rRNA. Requires the intact 30S subunit for methylation","uniprotId":"RSMG_ECOLI"},"6884313":{"id":6884313,"externalId":"511145.b3741","name":"gidA","hasFamily":true,"annotation":"5-methylaminomethyl-2-thiouridine modification at tRNA U34; NAD-binding protein involved in the addition of a carboxymethylaminomethyl (cmnm) group at the wobble position (U34) of certain tRNAs, forming tRNA-cmnm(5)s(2)U34 (By similarity)","uniprotId":"MNMG_ECOLI"},"6884314":{"id":6884314,"externalId":"511145.b3742","name":"mioC","hasFamily":true,"annotation":"FMN-binding protein MioC; Probable electron transporter required for biotin synthase activity","uniprotId":"MIOC_ECOLI"},"6884315":{"id":6884315,"externalId":"511145.b3743","name":"asnC","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; Activator of asnA transcription; autogenous regulator of its own transcription; and repressor of the expression of gidA at a post-transcriptional level","uniprotId":"ASNC_ECOLI"},"6884316":{"id":6884316,"externalId":"511145.b3744","name":"asnA","hasFamily":true,"annotation":"asparagine synthetase A; May amidate Asp of the extracellular death factor precursor Asn-Asn-Trp-Asp-Asn to generate Asn-Asn-Trp-Asn-Asn (Probable)","uniprotId":"ASNA_ECOLI"},"6884317":{"id":6884317,"externalId":"511145.b3745","name":"viaA","hasFamily":true,"annotation":"putative von Willibrand factor containing protein","uniprotId":"VIAA_ECOLI"},"6884318":{"id":6884318,"externalId":"511145.b3746","name":"ravA","hasFamily":true,"annotation":"fused predicted transcriptional regulator: sigma54 activator protein/conserved protein; Functions as an ATPase. May play a role in metal insertion (metal-chelatase) or as a chaperone (By similarity)","uniprotId":"RAVA_ECOLI"},"6884319":{"id":6884319,"externalId":"511145.b3747","name":"kup","hasFamily":true,"annotation":"potassium transporter; Responsible for the low-affinity transport of potassium into the cell, with the probable concomitant uptake of protons (symport system). Can also transport cesium","uniprotId":"KUP_ECOLI"},"6884320":{"id":6884320,"externalId":"511145.b3748","name":"rbsD","hasFamily":true,"annotation":"putative cytoplasmic sugar-binding protein; Catalyzes the interconversion of beta-pyran and beta- furan forms of D-ribose. It also catalyzes the conversion between beta-allofuranose and beta-allopyranose","uniprotId":"RBSD_ECOLI"},"6884321":{"id":6884321,"externalId":"511145.b3749","name":"rbsA","hasFamily":true,"annotation":"fused D-ribose transporter subunits of ABC superfamily: ATP-binding components; Part of the ABC transporter complex RbsABCD involved in ribose import. Responsible for energy coupling to the transport system (By similarity)","uniprotId":"RBSA_ECOLI"},"6884322":{"id":6884322,"externalId":"511145.b3750","name":"rbsC","hasFamily":true,"annotation":"D-ribose transporter subunit; Part of the binding-protein-dependent transport system for ribose. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"RBSC_ECOLI"},"6884323":{"id":6884323,"externalId":"511145.b3751","name":"rbsB","hasFamily":true,"annotation":"D-ribose transporter subunit; Involved in the high-affinity D-ribose membrane transport system and also serves as the primary chemoreceptor for chemotaxis","uniprotId":"RBSB_ECOLI"},"6884324":{"id":6884324,"externalId":"511145.b3752","name":"rbsK","hasFamily":true,"annotation":"ribokinase","uniprotId":"RBSK_ECOLI"},"6884325":{"id":6884325,"externalId":"511145.b3753","name":"rbsR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor of ribose metabolism; Transcriptional repressor for the ribose rbsDACBK operon. RbsR binds to a region of perfect dyad symmetry spanning the rbs operon transcriptional start site. The affinity for the rbs operator is reduced by addition of ribose, consistent with ribose being the inducer of the operon","uniprotId":"RBSR_ECOLI"},"6884326":{"id":6884326,"externalId":"511145.b3754","name":"hsrA","hasFamily":true,"annotation":"putative multidrug or homocysteine efflux system","uniprotId":"HSRA_ECOLI"},"6884327":{"id":6884327,"externalId":"511145.b3755","name":"yieP","hasFamily":true,"annotation":"putative transcriptional regulator","uniprotId":"YIEP_ECOLI"},"6884328":{"id":6884328,"externalId":"511145.b3764","name":"yifE","hasFamily":true,"annotation":"conserved protein, UPF0438 family","uniprotId":"YIFE_ECOLI"},"6884329":{"id":6884329,"externalId":"511145.b3765","name":"yifB","hasFamily":true,"annotation":"putative bifunctional enzyme and transcriptional regulator","uniprotId":"YIFB_ECOLI"},"6884330":{"id":6884330,"externalId":"511145.b3766","name":"ilvL","hasFamily":false,"annotation":"ilvG operon leader peptide","uniprotId":"LPID_ECOLI"},"6884331":{"id":6884331,"externalId":"511145.b3769","name":"ilvM","hasFamily":true,"annotation":"acetolactate synthase II, small subunit","uniprotId":"ILVM_ECOLI"},"6884332":{"id":6884332,"externalId":"511145.b3770","name":"ilvE","hasFamily":true,"annotation":"branched-chain amino-acid aminotransferase; Acts on leucine, isoleucine and valine","uniprotId":"ILVE_ECOLI"},"6884333":{"id":6884333,"externalId":"511145.b3771","name":"ilvD","hasFamily":true,"annotation":"dihydroxyacid dehydratase","uniprotId":"ILVD_ECOLI"},"6884334":{"id":6884334,"externalId":"511145.b3772","name":"ilvA","hasFamily":true,"annotation":"threonine deaminase; Catalyzes the anaerobic formation of alpha-ketobutyrate and ammonia from threonine in a two-step reaction. The first step involved a dehydration of threonine and a production of enamine intermediates (aminocrotonate), which tautomerizes to its imine form (iminobutyrate). Both intermediates are unstable and short- lived. The second step is the nonenzymatic hydrolysis of the enamine/imine intermediates to form 2-ketobutyrate and free ammonia. In the low water environment of the cell, the second step is accelerated by RidA","uniprotId":"ILVA_ECOLI"},"6884335":{"id":6884335,"externalId":"511145.b3773","name":"ilvY","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; This protein activates the transcription of the ilvC gene in the presence of acetolactate or acetohydroxybutyrate. IlvY is also a negative regulator of its own expression","uniprotId":"ILVY_ECOLI"},"6884336":{"id":6884336,"externalId":"511145.b3774","name":"ilvC","hasFamily":true,"annotation":"ketol-acid reductoisomerase, NAD(P)-binding","uniprotId":"ILVC_ECOLI"},"6884337":{"id":6884337,"externalId":"511145.b3775","name":"ppiC","hasFamily":true,"annotation":"peptidyl-prolyl cis-trans isomerase C (rotamase C); PPIases accelerate the folding of proteins. It prefers amino acid residues with hydrophobic side chains like leucine and phenylalanine in the P1 position of the peptides substrates","uniprotId":"PPIC_ECOLI"},"6884338":{"id":6884338,"externalId":"511145.b3778","name":"rep","hasFamily":true,"annotation":"DNA helicase and single-stranded DNA-dependent ATPase; Rep helicase is a single-stranded DNA-dependent ATPase involved in DNA replication; it can initiate unwinding at a nick in the DNA. It binds to the single-stranded DNA and acts in a progressive fashion along the DNA in the 3' to 5' direction","uniprotId":"REP_ECOLI"},"6884339":{"id":6884339,"externalId":"511145.b3779","name":"gpp","hasFamily":true,"annotation":"guanosine pentaphosphatase/exopolyphosphatase; Conversion of pppGpp to ppGpp. Guanosine pentaphosphate (pppGpp) is a cytoplasmic signaling molecule which together with ppGpp controls the \"stringent response\", an adaptive process that allows bacteria to respond to amino acid starvation, resulting in the coordinated regulation of numerous cellular activities (By similarity)","uniprotId":"GPPA_ECOLI"},"6884340":{"id":6884340,"externalId":"511145.b3780","name":"rhlB","hasFamily":true,"annotation":"ATP-dependent RNA helicase; Can carry out ATP-dependent unwinding of double stranded RNA. Has a role in RNA decay. Involved in the RNA degradosome, a multi-enzyme complex important in RNA processing and messenger RNA degradation (By similarity)","uniprotId":"RHLB_ECOLI"},"6884341":{"id":6884341,"externalId":"511145.b3781","name":"trxA","hasFamily":true,"annotation":"thioredoxin 1; Participates in various redox reactions through the reversible oxidation of its active center dithiol to a disulfide and catalyzes dithiol-disulfide exchange reactions","uniprotId":"THIO_ECOLI"},"6884342":{"id":6884342,"externalId":"511145.b3783","name":"rho","hasFamily":true,"annotation":"transcription termination factor; Facilitates transcription termination by a mechanism that involves Rho binding to the nascent RNA, activation of Rho's RNA-dependent ATPase activity, and release of the mRNA from the DNA template. RNA-dependent NTPase which utilizes all four ribonucleoside triphosphates as substrates","uniprotId":"RHO_ECOLI"},"6884343":{"id":6884343,"externalId":"511145.b3784","name":"rfe","hasFamily":true,"annotation":"UDP-GlcNAc:undecaprenylphosphate GlcNAc-1-phosphate transferase; Catalyzes the transfer of the GlcNAc-1-phosphate moiety from UDP-GlcNAc onto the carrier lipid undecaprenyl phosphate (C55-P), yielding GlcNAc-pyrophosphoryl-undecaprenyl (GlcNAc-PP- C55). It is the first lipid-linked intermediate involved in enterobacterial common antigen (ECA) synthesis and an acceptor for the addition of subsequent sugars to complete the biosynthesis of O-antigen. The apparent affinity of WecA for the polyisoprenyl phosphate substrates increases with the polyisoprenyl chain length. WecA is unable to ut [...] ","uniprotId":"WECA_ECOLI"},"6884344":{"id":6884344,"externalId":"511145.b3785","name":"wzzE","hasFamily":true,"annotation":"Entobacterial Common Antigen (ECA) polysaccharide chain length modulation protein","uniprotId":"WZZE_ECOLI"},"6884345":{"id":6884345,"externalId":"511145.b3786","name":"rffE","hasFamily":true,"annotation":"UDP-N-acetyl glucosamine-2-epimerase; Catalyzes the reversible epimerization at C-2 of UDP-N- acetylglucosamine (UDP-GlcNAc) and thereby provides bacteria with UDP-N-acetylmannosamine (UDP-ManNAc), the activated donor of ManNAc residues. Also involved in bacteriophage N4 adsorption","uniprotId":"WECB_ECOLI"},"6884346":{"id":6884346,"externalId":"511145.b3787","name":"rffD","hasFamily":true,"annotation":"UDP-N-acetyl-D-mannosaminuronic acid dehydrogenase; Catalyzes the four-electron oxidation of UDP-N-acetyl-D- mannosamine (UDP-ManNAc), reducing NAD(+) and releasing UDP-N- acetylmannosaminuronic acid (UDP-ManNAcA)","uniprotId":"WECC_ECOLI"},"6884347":{"id":6884347,"externalId":"511145.b3788","name":"rffG","hasFamily":true,"annotation":"dTDP-glucose 4,6-dehydratase; Catalyzes the dehydration of dTDP-D-glucose to form dTDP-6-deoxy-D-xylo-4-hexulose via a three-step process involving oxidation, dehydration and reduction","uniprotId":"RMLB2_ECOLI"},"6884348":{"id":6884348,"externalId":"511145.b3789","name":"rffH","hasFamily":true,"annotation":"glucose-1-phosphate thymidylyltransferase; Catalyzes the formation of dTDP-glucose, from dTTP and glucose 1-phosphate, as well as its pyrophosphorolysis (By similarity)","uniprotId":"RMLA2_ECOLI"},"6884349":{"id":6884349,"externalId":"511145.b3790","name":"rffC","hasFamily":true,"annotation":"TDP-fucosamine acetyltransferase; Catalyzes the acetylation of dTDP-fucosamine (dTDP-4- amino-4,6-dideoxy-D-galactose) to dTDP-Fuc4NAc, which is utilized in the biosynthesis of the enterobacterial common antigen (ECA) (By similarity)","uniprotId":"WECD_ECOLI"},"6884350":{"id":6884350,"externalId":"511145.b3791","name":"rffA","hasFamily":true,"annotation":"TDP-4-oxo-6-deoxy-D-glucose transaminase; Involved in ECA elongation","uniprotId":"RFFA_ECOLI"},"6884351":{"id":6884351,"externalId":"511145.b3792","name":"wzxE","hasFamily":true,"annotation":"O-antigen translocase","uniprotId":"WZXE_ECOLI"},"6884352":{"id":6884352,"externalId":"511145.b3793","name":"wzyE","hasFamily":true,"annotation":"putative Wzy protein involved in ECA polysaccharide chain elongation; Could be involved in ECA polysaccharide chain elongation (By similarity)","uniprotId":"WZYE_ECOLI"},"6884353":{"id":6884353,"externalId":"511145.b3794","name":"wecG","hasFamily":true,"annotation":"UDP-N-acetyl-D-mannosaminuronic acid transferase; Catalyzes the synthesis of Und-PP-GlcNAc-ManNAcA (Lipid II), the second lipid-linked intermediate involved in ECA synthesis (By similarity)","uniprotId":"WECG_ECOLI"},"6884354":{"id":6884354,"externalId":"511145.b3795","name":"yifK","hasFamily":true,"annotation":"putative transporter; Probable amino-acid or metabolite transport protein","uniprotId":"YIFK_ECOLI"},"6884355":{"id":6884355,"externalId":"511145.b3800","name":"aslB","hasFamily":true,"annotation":"putative regulator of arylsulfatase activity","uniprotId":"ASLB_ECOLI"},"6884356":{"id":6884356,"externalId":"511145.b3801","name":"aslA","hasFamily":true,"annotation":"acrylsulfatase-like enzyme","uniprotId":"ASLA_ECOLI"},"6884357":{"id":6884357,"externalId":"511145.b3802","name":"hemY","hasFamily":true,"annotation":"putative protoheme IX synthesis protein; Involved in a late step of protoheme IX synthesis","uniprotId":"HEMY_ECOLI"},"6884358":{"id":6884358,"externalId":"511145.b3803","name":"hemX","hasFamily":true,"annotation":"predicted uroporphyrinogen III methyltransferase","uniprotId":"HEMX_ECOLI"},"6884359":{"id":6884359,"externalId":"511145.b3804","name":"hemD","hasFamily":true,"annotation":"uroporphyrinogen III synthase; Catalyzes cyclization of the linear tetrapyrrole, hydroxymethylbilane, to the macrocyclic uroporphyrinogen III (By similarity)","uniprotId":"HEM4_ECOLI"},"6884360":{"id":6884360,"externalId":"511145.b3805","name":"hemC","hasFamily":true,"annotation":"hydroxymethylbilane synthase; Tetrapolymerization of the monopyrrole PBG into the hydroxymethylbilane pre-uroporphyrinogen in several discrete steps (By similarity)","uniprotId":"HEM3_ECOLI"},"6884361":{"id":6884361,"externalId":"511145.b3806","name":"cyaA","hasFamily":true,"annotation":"adenylate cyclase; Catalyzes the formation of the second messenger cAMP from ATP. Its transcript is probably degraded by endoribonuclease LS (rnlA), decreasing cAMP levels and the negative regulator Crp- cAMP, which then induces its own transcription again","uniprotId":"CYAA_ECOLI"},"6884362":{"id":6884362,"externalId":"511145.b3807","name":"cyaY","hasFamily":true,"annotation":"iron-dependent inhibitor of iron-sulfur cluster formation; frataxin","uniprotId":"CYAY_ECOLI"},"6884363":{"id":6884363,"externalId":"511145.b3809","name":"dapF","hasFamily":true,"annotation":"diaminopimelate epimerase; Catalyzes the stereoinversion of LL-2,6- diaminoheptanedioate (L,L-DAP) to meso-diaminoheptanedioate (meso- DAP), a precursor of L-lysine and an essential component of the bacterial peptidoglycan. Only accepts DAP isomers with the L configuration","uniprotId":"DAPF_ECOLI"},"6884364":{"id":6884364,"externalId":"511145.b3810","name":"yigA","hasFamily":true,"annotation":"conserved protein, DUF484 family","uniprotId":"YIGA_ECOLI"},"6884365":{"id":6884365,"externalId":"511145.b3811","name":"xerC","hasFamily":true,"annotation":"site-specific tyrosine recombinase; Site-specific tyrosine recombinase, which acts by catalyzing the cutting and rejoining of the recombining DNA molecules. Binds cooperatively to specific DNA consensus sequences that are separated from XerD binding sites by a short central region, forming the heterotetrameric XerC-XerD complex that recombines DNA substrates. The complex is essential to convert dimers of the bacterial chromosome into monomers to permit their segregation at cell division. It also contributes to the segregational stability of plasmids at ColE1 xer (or cer) and pSC101 (or [...] ","uniprotId":"XERC_ECOLI"},"6884366":{"id":6884366,"externalId":"511145.b3812","name":"yigB","hasFamily":true,"annotation":"FMN phosphatase; Catalyzes the dephosphorylation of flavin mononucleotide (FMN). YigB is important for formation of dormant persister cells","uniprotId":"YIGB_ECOLI"},"6884367":{"id":6884367,"externalId":"511145.b3813","name":"mutU","hasFamily":true,"annotation":"DNA-dependent ATPase I and helicase II; Has both ATPase and helicase activities. Unwinds DNA duplexes with 3' to 5' polarity with respect to the bound strand and initiates unwinding most effectively when a single-stranded region is present. Involved in the post-incision events of nucleotide excision repair and methyl-directed mismatch repair","uniprotId":"UVRD_ECOLI"},"6884368":{"id":6884368,"externalId":"511145.b3816","name":"corA","hasFamily":true,"annotation":"magnesium/nickel/cobalt transporter; Mediates influx of magnesium ions. Can also mediate cobalt and manganese uptake","uniprotId":"CORA_ECOLI"},"6884369":{"id":6884369,"externalId":"511145.b3817","name":"yigF","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YIGF_ECOLI"},"6884370":{"id":6884370,"externalId":"511145.b3818","name":"yigG","hasFamily":true,"annotation":"conserved inner membrane protein","uniprotId":"YIGG_ECOLI"},"6884371":{"id":6884371,"externalId":"511145.b3819","name":"rarD","hasFamily":true,"annotation":"putative chloramphenical resistance permease","uniprotId":"RARD_ECOLI"},"6884372":{"id":6884372,"externalId":"511145.b3820","name":"yigI","hasFamily":true,"annotation":"conserved protein, 4HBT family of thioesterases","uniprotId":"YIGI_ECOLI"},"6884373":{"id":6884373,"externalId":"511145.b3821","name":"pldA","hasFamily":true,"annotation":"outer membrane phospholipase A; Has broad substrate specificity including hydrolysis of phosphatidylcholine with phospholipase A2 (EC 3.1.1.4) and phospholipase A1 (EC 3.1.1.32) activities. Strong expression leads to outer membrane breakdown and cell death; is dormant in normal growing cells. Required for efficient secretion of bacteriocins","uniprotId":"PA1_ECOLI"},"6884374":{"id":6884374,"externalId":"511145.b3822","name":"recQ","hasFamily":true,"annotation":"ATP-dependent DNA helicase; Involved in the RecF recombination pathway; its gene expression is under the regulation of the SOS system. It is a DNA helicase","uniprotId":"RECQ_ECOLI"},"6884375":{"id":6884375,"externalId":"511145.b3823","name":"rhtC","hasFamily":true,"annotation":"threonine efflux pump; Conducts the efflux of threonine","uniprotId":"RHTC_ECOLI"},"6884376":{"id":6884376,"externalId":"511145.b3824","name":"rhtB","hasFamily":true,"annotation":"homoserine, homoserine lactone and S-methyl-methionine efflux pump; Conducts the efflux of homoserine and homoserine lactone","uniprotId":"RHTB_ECOLI"},"6884377":{"id":6884377,"externalId":"511145.b3825","name":"pldB","hasFamily":true,"annotation":"lysophospholipase L2","uniprotId":"PLDB_ECOLI"},"6884378":{"id":6884378,"externalId":"511145.b3826","name":"yigL","hasFamily":true,"annotation":"putative hydrolase; Catalyzes Strongly the dephosphorylation of pyridoxal- phosphate (PLP) and moderately the dephosphorylation of 2- deoxyglucose 6-phosphate (2bGLU6P) and beta-glucose 6-phosphate (bGlu6P). Also hydrolyzes both purines (GMP and IMP) and pyrimidines as secondary substrates","uniprotId":"YIGL_ECOLI"},"6884379":{"id":6884379,"externalId":"511145.b3827","name":"yigM","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YIGM_ECOLI"},"6884380":{"id":6884380,"externalId":"511145.b3828","name":"metR","hasFamily":true,"annotation":"DNA-binding transcriptional activator, homocysteine-binding; Control of the last step in methionine biosynthesis; MetR is a positive activator of the metA, metE and metH genes. MetR is also a negative regulator of its own expression. Binds homocysteine as an inducer","uniprotId":"METR_ECOLI"},"6884381":{"id":6884381,"externalId":"511145.b3829","name":"metE","hasFamily":true,"annotation":"5-methyltetrahydropteroyltriglutamate- homocysteine S-methyltransferase; Catalyzes the transfer of a methyl group from 5- methyltetrahydrofolate to homocysteine resulting in methionine formation (By similarity)","uniprotId":"METE_ECOLI"},"6884382":{"id":6884382,"externalId":"511145.b3830","name":"ysgA","hasFamily":true,"annotation":"putative hydrolase","uniprotId":"DLHH_ECOLI"},"6884383":{"id":6884383,"externalId":"511145.b3831","name":"udp","hasFamily":true,"annotation":"uridine phosphorylase; Catalyzes the reversible phosphorylytic cleavage of uridine and deoxyuridine to uracil and ribose- or deoxyribose-1- phosphate. The produced molecules are then utilized as carbon and energy sources or in the rescue of pyrimidine bases for nucleotide synthesis","uniprotId":"UDP_ECOLI"},"6884384":{"id":6884384,"externalId":"511145.b3832","name":"rmuC","hasFamily":true,"annotation":"putative recombination limiting protein; Involved in DNA recombination","uniprotId":"RMUC_ECOLI"},"6884385":{"id":6884385,"externalId":"511145.b3833","name":"ubiE","hasFamily":true,"annotation":"bifunctional 2-octaprenyl-6-methoxy-1,4-benzoquinone methylase/ S-adenosylmethionine:2-DMK methyltransferase; Methyltransferase required for the conversion of demethylmenaquinone (DMKH2) to menaquinone (MKH2) and the conversion of 2-polyprenyl-6-methoxy-1,4-benzoquinol (DDMQH2) to 2-polyprenyl-3-methyl-6-methoxy-1,4-benzoquinol (DMQH2) (By similarity)","uniprotId":"UBIE_ECOLI"},"6884386":{"id":6884386,"externalId":"511145.b3834","name":"yigP","hasFamily":true,"annotation":"conserved protein, SCP2 family","uniprotId":"YIGP_ECOLI"},"6884387":{"id":6884387,"externalId":"511145.b3835","name":"yigR","hasFamily":true,"annotation":"2-octaprenylphenol hydroxylase; Required, probably indirectly, for the hydroxylation of 2-octaprenylphenol to 2-octaprenyl-6-hydroxy-phenol, the fourth step in ubiquinone biosynthesis. Specific for aerobically grown log-phase cells","uniprotId":"UBIB_ECOLI"},"6884388":{"id":6884388,"externalId":"511145.b3836","name":"tatA","hasFamily":true,"annotation":"TatABCE protein translocation system subunit; Part of the twin-arginine translocation (Tat) system that transports large folded proteins containing a characteristic twin-arginine motif in their signal peptide across membranes. TatA could form the protein-conducting channel of the Tat system (By similarity)","uniprotId":"TATA_ECOLI"},"6884389":{"id":6884389,"externalId":"511145.b3838","name":"tatB","hasFamily":true,"annotation":"TatABCE protein translocation system subunit; Part of the twin-arginine translocation (Tat) system that transports large folded proteins containing a characteristic twin-arginine motif in their signal peptide across membranes. Together with TatC, TatB is part of a receptor directly interacting with Tat signal peptides. TatB may form an oligomeric binding site that transiently accommodates folded Tat precursor proteins before their translocation (By similarity)","uniprotId":"TATB_ECOLI"},"6884390":{"id":6884390,"externalId":"511145.b3839","name":"tatC","hasFamily":true,"annotation":"TatABCE protein translocation system subunit; Part of the twin-arginine translocation (Tat) system that transports large folded proteins containing a characteristic twin-arginine motif in their signal peptide across membranes. Together with TatB, TatC is part of a receptor directly interacting with Tat signal peptides (By similarity)","uniprotId":"TATC_ECOLI"},"6884391":{"id":6884391,"externalId":"511145.b3842","name":"rfaH","hasFamily":true,"annotation":"DNA-binding transcriptional antiterminator; Enhances distal genes transcription elongation in a specialized subset of operons that encode extracytoplasmic components. RfaH is recruited into a multi-component RNA polymerase complex by the ops element, which is a short conserved DNA sequence located downstream of the main promoter of these operons. Once bound, RfaH suppresses pausing and inhibits Rho- dependent and intrinsic termination at a subset of sites. Termination signals are bypassed, which allows complete synthesis of long RNA chains. Enhances expression of several operons involv [...] ","uniprotId":"RFAH_ECOLI"},"6884392":{"id":6884392,"externalId":"511145.b3843","name":"ubiD","hasFamily":true,"annotation":"3-octaprenyl-4-hydroxybenzoate decarboxylase; Catalyzes the decarboxylation of 3-octaprenyl-4-hydroxy benzoate to 2-octaprenylphenol (By similarity)","uniprotId":"UBID_ECOLI"},"6884393":{"id":6884393,"externalId":"511145.b3844","name":"fre","hasFamily":true,"annotation":"NAD(P)H-flavin reductase; Catalyzes the reduction of soluble flavins by reduced pyridine nucleotides. Seems to reduce the complexed Fe(3+) iron of siderophores to Fe(2+), thus releasing it from the chelator","uniprotId":"FRE_ECOLI"},"6884394":{"id":6884394,"externalId":"511145.b3845","name":"fadA","hasFamily":true,"annotation":"3-ketoacyl-CoA thiolase (thiolase I); Catalyzes the final step of fatty acid oxidation in which acetyl-CoA is released and the CoA ester of a fatty acid two carbons shorter is formed. Involved in the aerobic and anaerobic degradation of long-chain fatty acids","uniprotId":"FADA_ECOLI"},"6884395":{"id":6884395,"externalId":"511145.b3846","name":"fadB","hasFamily":true,"annotation":"fused 3-hydroxybutyryl-CoA epimerase/delta(3)-cis-delta(2)-trans-enoyl-CoA isomerase/enoyl-CoA hydratase/3-hydroxyacyl-CoA dehydrogenase; Catalyzes the formation of a hydroxyacyl-CoA by addition of water on enoyl-CoA. Also exhibits 3-hydroxyacyl-CoA epimerase and 3-hydroxyacyl-CoA dehydrogenase activities. Involved in the aerobic and anaerobic degradation of long-chain fatty acids","uniprotId":"FADB_ECOLI"},"6884396":{"id":6884396,"externalId":"511145.b3847","name":"pepQ","hasFamily":true,"annotation":"proline dipeptidase; Splits dipeptides with a prolyl residue in the C- terminal position and a polar or nonpolar amino acid at the N- terminal position. With much lower efficiency, also catalyzes the stereoselective hydrolysis of a wide variety of organophosphate triesters and organophosphonate diesters. Is able to hydrolyze the organophosphorus insecticide paraoxon and the p-nitrophenyl analogs of the nerve agents GB (sarin), GD (soman), GF, Vx and rVX","uniprotId":"PEPQ_ECOLI"},"6884397":{"id":6884397,"externalId":"511145.b3848","name":"yigZ","hasFamily":true,"annotation":"conserved protein, UPF0029 family","uniprotId":"YIGZ_ECOLI"},"6884398":{"id":6884398,"externalId":"511145.b3849","name":"trkH","hasFamily":true,"annotation":"potassium transporter; Low-affinity potassium transport system. Interacts with Trk system potassium uptake protein TrkA and requires TrkE for transport activity","uniprotId":"TRKH_ECOLI"},"6884399":{"id":6884399,"externalId":"511145.b3850","name":"hemG","hasFamily":true,"annotation":"protoporphyrin oxidase, flavoprotein; Catalyzes the 6-electron oxidation of protoporphyrinogen-IX to form protoporphyrin-IX using menaquinone as electron acceptor","uniprotId":"HEMG_ECOLI"},"6884400":{"id":6884400,"externalId":"511145.b3856","name":"mobB","hasFamily":true,"annotation":"molybdopterin-guanine dinucleotide biosynthesis protein B; GTP-binding protein that is not required for the biosynthesis of Mo-molybdopterin guanine dinucleotide (Mo-MGD) cofactor, and not necessary for the formation of active molybdoenzymes using this form of molybdenum cofactor. May act as an adapter protein to achieve the efficient biosynthesis and utilization of MGD. Displays a weak intrinsic GTPase activity. Is also able to bind the nucleotides ATP, TTP and GDP, but with lower affinity than GTP","uniprotId":"MOBB_ECOLI"},"6884401":{"id":6884401,"externalId":"511145.b3857","name":"mobA","hasFamily":true,"annotation":"molybdopterin-guanine dinucleotide synthase; Transfers a GMP moiety from GTP to Mo-molybdopterin (Mo- MPT) cofactor (Moco or molybdenum cofactor) to form Mo- molybdopterin guanine dinucleotide (Mo-MGD) cofactor. Is also involved in the biosynthesis of the bis-MGD form of the Moco cofactor (Mo-bisMGD) in which the metal is symmetrically ligated by the dithiolene groups of two MGD molecules. Is necessary and sufficient for the in vitro activation of the DMSOR molybdoenzyme that uses the Mo-bisMGD form of molybdenum cofactor, which implies formation and efficient insertion of the cofactor [...] ","uniprotId":"MOBA_ECOLI"},"6884402":{"id":6884402,"externalId":"511145.b3858","name":"yihD","hasFamily":true,"annotation":"DUF1040 protein YihD","uniprotId":"YIHD_ECOLI"},"6884403":{"id":6884403,"externalId":"511145.b3859","name":"rdoA","hasFamily":true,"annotation":"Thr/Ser kinase implicated in Cpx stress response; Probably involved in the extracytoplasmic stress response (By similarity)","uniprotId":"RDOA_ECOLI"},"6884404":{"id":6884404,"externalId":"511145.b3860","name":"dsbA","hasFamily":true,"annotation":"periplasmic protein disulfide isomerase I; Required for disulfide bond formation in some periplasmic proteins such as PhoA or OmpA. Acts by transferring its disulfide bond to other proteins and is reduced in the process. DsbA is reoxidized by DsbB. It is required for pilus biogenesis","uniprotId":"DSBA_ECOLI"},"6884405":{"id":6884405,"externalId":"511145.b3861","name":"yihF","hasFamily":false,"annotation":"conserved protein, DUF945 family","uniprotId":"YIHF_ECOLI"},"6884406":{"id":6884406,"externalId":"511145.b3862","name":"yihG","hasFamily":true,"annotation":"inner membrane protein, Predicted acyltransferas","uniprotId":"YIHG_ECOLI"},"6884407":{"id":6884407,"externalId":"511145.b3863","name":"polA","hasFamily":true,"annotation":"fused DNA polymerase I 5'->3' polymerase/3'->5' exonuclease/5'->3' exonuclease; In addition to polymerase activity, this DNA polymerase exhibits 3' to 5' and 5' to 3' exonuclease activity. It is able to utilize nicked circular duplex DNA as a template and can unwind the parental DNA strand from its template","uniprotId":"DPO1_ECOLI"},"6884408":{"id":6884408,"externalId":"511145.b3865","name":"engB","hasFamily":true,"annotation":"GTP-binding protein required for normal cell division; Necessary for normal cell division and for the maintenance of normal septation. Depletion of this protein leads to a severe reduction in growth rate and to extensive filamentation, with a block beyond the stage of segregation. Essential for bacteria survival","uniprotId":"ENGB_ECOLI"},"6884409":{"id":6884409,"externalId":"511145.b3866","name":"yihI","hasFamily":true,"annotation":"activator of Der GTPase; A GTPase-activating protein (GAP) that modifies Der/EngA GTPase function, negatively regulating cell growth, probably via ribosome assembly. Stimulates the GTPase activity of Der; a construct missing the first 45 residues is even more stimulatory. Does not stimulate 2 other GTPases (ObgE and Era). Overexpression inhibits cell growth; precursor 16S rRNA accumulates, the 23S rRNA is 6-7 bases longer than usual, and 50S ribosomal subunits are improperly assembled, leading to 45S subunits lacking proteins L9, L18 and L25. Overexpression of Der in the same cells sup [...] ","uniprotId":"YIHI_ECOLI"},"6884410":{"id":6884410,"externalId":"511145.b3867","name":"hemN","hasFamily":true,"annotation":"coproporphyrinogen III oxidase, SAM and NAD(P)H dependent, oxygen-independent; Anaerobic transformation of coproporphyrinogen-III into protoporphyrinogen-IX","uniprotId":"HEMN_ECOLI"},"6884411":{"id":6884411,"externalId":"511145.b3868","name":"glnG","hasFamily":true,"annotation":"fused DNA-binding response regulator in two-component regulatory system with GlnL: response regulator/sigma54 interaction protein; Member of the two-component regulatory system NtrB/NtrC involved in the activation of nitrogen assimilatory genes such as GlnA. NtrC is phosphorylated by NtrB and interacts with sigma-54","uniprotId":"NTRC_ECOLI"},"6884412":{"id":6884412,"externalId":"511145.b3869","name":"glnL","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with GlnG; NtrB acts as a signal transducer which responds to the nitrogen level of cell and modulates the activity of NtrC. In nitrogen limitation NtrB activates NtrC by phosphorylating it, while in nitrogen excess NtrC is dephosphorylated and consequently inactivated by NtrB","uniprotId":"NTRB_ECOLI"},"6884413":{"id":6884413,"externalId":"511145.b3870","name":"glnA","hasFamily":true,"annotation":"glutamine synthetase","uniprotId":"GLNA_ECOLI"},"6884414":{"id":6884414,"externalId":"511145.b3871","name":"typA","hasFamily":true,"annotation":"GTP-binding protein; Probably interacts with the ribosomes in a GTP dependent manner. GTPase that mediates interactions between enteropathogenic E.coli (EPEC) and epithelial cells","uniprotId":"TYPA_ECOLI"},"6884415":{"id":6884415,"externalId":"511145.b3872","name":"yihL","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YIHL_ECOLI"},"6884416":{"id":6884416,"externalId":"511145.b3873","name":"yihM","hasFamily":false,"annotation":"putative sugar phosphate isomerase","uniprotId":"YIHM_ECOLI"},"6884417":{"id":6884417,"externalId":"511145.b3874","name":"yihN","hasFamily":true,"annotation":"inner membrane protein, predicted transporter","uniprotId":"YIHN_ECOLI"},"6884418":{"id":6884418,"externalId":"511145.b3875","name":"ompL","hasFamily":true,"annotation":"outer membrane porin L; Outer membrane channel protein that allows an efficient diffusion of low-molecular-weight solutes such as small sugars and tetraglycine. However, the specific substrate recognized by the OmpL channel is unknown","uniprotId":"OMPL_ECOLI"},"6884419":{"id":6884419,"externalId":"511145.b3876","name":"yihO","hasFamily":true,"annotation":"putative transporter","uniprotId":"YIHO_ECOLI"},"6884420":{"id":6884420,"externalId":"511145.b3877","name":"yihP","hasFamily":true,"annotation":"putative transporter","uniprotId":"YIHP_ECOLI"},"6884421":{"id":6884421,"externalId":"511145.b3878","name":"yihQ","hasFamily":true,"annotation":"alpha-glucosidase; Exhibits hydrolysis activity against alpha-glucosyl fluoride, although natural substrates, such as alpha-glucobioses are scarcely hydrolyzed","uniprotId":"YIHQ_ECOLI"},"6884422":{"id":6884422,"externalId":"511145.b3879","name":"yihR","hasFamily":true,"annotation":"putative aldose-1-epimerase","uniprotId":"YIHR_ECOLI"},"6884423":{"id":6884423,"externalId":"511145.b3880","name":"yihS","hasFamily":true,"annotation":"aldose-ketose isomerase","uniprotId":"YIHS_ECOLI"},"6884424":{"id":6884424,"externalId":"511145.b3881","name":"yihT","hasFamily":true,"annotation":"putative aldolase","uniprotId":"YIHT_ECOLI"},"6884425":{"id":6884425,"externalId":"511145.b3882","name":"yihU","hasFamily":true,"annotation":"gamma-hydroxybutyrate dehydrogenase, NADH-dependent","uniprotId":"YIHU_ECOLI"},"6884426":{"id":6884426,"externalId":"511145.b3883","name":"yihV","hasFamily":true,"annotation":"putative sugar kinase","uniprotId":"YIHV_ECOLI"},"6884427":{"id":6884427,"externalId":"511145.b3884","name":"yihW","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YIHW_ECOLI"},"6884428":{"id":6884428,"externalId":"511145.b3885","name":"yihX","hasFamily":true,"annotation":"alpha-D-Glucose-1-P phosphatase, anomer-specific; Catalyzes the dephosphorylation of alpha-D-glucose-1- phosphate (Glu1P) and has no activity with the beta form. In addition, YihX has phosphatase activity against sugar-phosphate and pyridoxal phosphate (PLP) and low beta-phosphoglucomutase activity","uniprotId":"YIHX_ECOLI"},"6884429":{"id":6884429,"externalId":"511145.b3886","name":"yihY","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YIHY_ECOLI"},"6884430":{"id":6884430,"externalId":"511145.b3887","name":"dtd","hasFamily":true,"annotation":"D-tyr-tRNA(Tyr) deacylase; Hydrolyzes D-tyrosyl-tRNA(Tyr) into D-tyrosine and free tRNA(Tyr). Could be a defense mechanism against a harmful effect of D-tyrosine. Can also cleave D-aspartyl-tRNA(Asp) and D- tryptophanyl-tRNA(Trp)","uniprotId":"DTD_ECOLI"},"6884431":{"id":6884431,"externalId":"511145.b3888","name":"yiiD","hasFamily":true,"annotation":"putative acetyltransferase","uniprotId":"YIID_ECOLI"},"6884432":{"id":6884432,"externalId":"511145.b3889","name":"yiiE","hasFamily":false,"annotation":"putative transcriptional regulator","uniprotId":"YIIE_ECOLI"},"6884433":{"id":6884433,"externalId":"511145.b3890","name":"yiiF","hasFamily":false,"annotation":"conserved protein","uniprotId":"YIIF_ECOLI"},"6884434":{"id":6884434,"externalId":"511145.b3891","name":"fdhE","hasFamily":true,"annotation":"formate dehydrogenase formation protein; Necessary for formate dehydrogenase activity (By similarity)","uniprotId":"FDHE_ECOLI"},"6884435":{"id":6884435,"externalId":"511145.b3892","name":"fdoI","hasFamily":true,"annotation":"formate dehydrogenase-O, cytochrome b556 subunit; Allows to use formate as major electron donor during aerobic respiration. Subunit gamma is probably the cytochrome b556(FDO) component of the formate dehydrogenase","uniprotId":"FDOI_ECOLI"},"6884436":{"id":6884436,"externalId":"511145.b3893","name":"fdoH","hasFamily":true,"annotation":"formate dehydrogenase-O, Fe-S subunit; Allows to use formate as major electron donor during aerobic respiration. The beta chain is an electron transfer unit containing 4 cysteine clusters involved in the formation of iron- sulfur centers. Electrons are transferred from the gamma chain to the molybdenum cofactor of the alpha subunit (By similarity)","uniprotId":"FDOH_ECOLI"},"6884437":{"id":6884437,"externalId":"511145.b3894","name":"fdoG","hasFamily":true,"annotation":"formate dehydrogenase-O, large subunit; Allows to use formate as major electron donor during aerobic respiration. Subunit alpha possibly forms the active site","uniprotId":"FDOG_ECOLI"},"6884438":{"id":6884438,"externalId":"511145.b3895","name":"fdhD","hasFamily":true,"annotation":"formate dehydrogenase formation protein; Necessary for nitrate inducible formate dehydrogenase activity (By similarity)","uniprotId":"FDHD_ECOLI"},"6884439":{"id":6884439,"externalId":"511145.b3896","name":"yiiG","hasFamily":true,"annotation":"conserved lipoprotein","uniprotId":"YIIG_ECOLI"},"6884440":{"id":6884440,"externalId":"511145.b3897","name":"frvR","hasFamily":false,"annotation":"putative frv operon regulator; Could be involved in the regulation of the transcription of the FRV operon","uniprotId":"FRVR_ECOLI"},"6884441":{"id":6884441,"externalId":"511145.b3898","name":"frvX","hasFamily":true,"annotation":"putative peptidase","uniprotId":"FRVX_ECOLI"},"6884442":{"id":6884442,"externalId":"511145.b3899","name":"frvB","hasFamily":false,"annotation":"fused predicted PTS enzymes: IIB component/IIC component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (By similarity)","uniprotId":"PTFLB_ECOLI"},"6884443":{"id":6884443,"externalId":"511145.b3900","name":"frvA","hasFamily":false,"annotation":"predicted enzyme IIA component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (By similarity)","uniprotId":"PTFA_ECOLI"},"6884444":{"id":6884444,"externalId":"511145.b3901","name":"rhaM","hasFamily":true,"annotation":"L-rhamnose mutarotase; Involved in the anomeric conversion of L-rhamnose (By similarity)","uniprotId":"RHAM_ECOLI"},"6884445":{"id":6884445,"externalId":"511145.b3902","name":"rhaD","hasFamily":true,"annotation":"rhamnulose-1-phosphate aldolase; Catalyzes the reversible cleavage of L-rhamnulose-1- phosphate to dihydroxyacetone phosphate (DHAP) and L-lactaldehyde (By similarity)","uniprotId":"RHAD_ECOLI"},"6884446":{"id":6884446,"externalId":"511145.b3903","name":"rhaA","hasFamily":true,"annotation":"L-rhamnose isomerase","uniprotId":"RHAA_ECOLI"},"6884447":{"id":6884447,"externalId":"511145.b3904","name":"rhaB","hasFamily":true,"annotation":"rhamnulokinase","uniprotId":"RHAB_ECOLI"},"6884448":{"id":6884448,"externalId":"511145.b3905","name":"rhaS","hasFamily":true,"annotation":"DNA-binding transcriptional activator for rhaBAD and rhaT, L-rhamnose-binding; Activates expression of the rhaBAD and rhaT operons (By similarity)","uniprotId":"RHAS_ECOLI"},"6884449":{"id":6884449,"externalId":"511145.b3906","name":"rhaR","hasFamily":true,"annotation":"DNA-binding transcriptional activator for rhaSR, L-rhamnose-binding; Activates expression of the rhaSR operon in response to L-rhamnose (By similarity)","uniprotId":"RHAR_ECOLI"},"6884450":{"id":6884450,"externalId":"511145.b3907","name":"rhaT","hasFamily":true,"annotation":"L-rhamnose:proton symporter; Uptake of L-rhamnose across the boundary membrane with the concomitant transport of protons into the cell (symport system). Can also transport L-mannose and L-xylose, but at reduced rates","uniprotId":"RHAT_ECOLI"},"6884451":{"id":6884451,"externalId":"511145.b3908","name":"sodA","hasFamily":true,"annotation":"superoxide dismutase, Mn; Destroys superoxide anion radicals which are normally produced within the cells and which are toxic to biological systems","uniprotId":"SODM_ECOLI"},"6884452":{"id":6884452,"externalId":"511145.b3909","name":"kdgT","hasFamily":true,"annotation":"2-keto-3-deoxy-D-gluconate transporter; The 2-keto-3-deoxygluconate permease transports the degraded pectin products into the bacterial cell, where they serve as carbon and energy sources. This is a hydrogen coupled transport system (By similarity)","uniprotId":"KDGT_ECOLI"},"6884453":{"id":6884453,"externalId":"511145.b3910","name":"yiiM","hasFamily":true,"annotation":"6-N-hydroxylaminopurine resistance protein","uniprotId":"YIIM_ECOLI"},"6884454":{"id":6884454,"externalId":"511145.b3911","name":"cpxA","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with CpxR; This protein is involved in several diverse cellular processes, such as the functioning of acetohydroxyacid synthetase I, in the biosynthesis of isoleucine and valine, the TraJ protein activation activity for tra gene expression in F plasmid, and the synthesis, translocation, or stability of cell envelope proteins. Activates CpxR by phosphorylation","uniprotId":"CPXA_ECOLI"},"6884455":{"id":6884455,"externalId":"511145.b3912","name":"cpxR","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with CpxA; Member of the two-component regulatory system CpxA/CpxR. This system combats a variety of extracytoplasmic protein-mediated toxicities. It performs this function by increasing the synthesis of the periplasmic protease, DegP as well as that of CpxP protein","uniprotId":"CPXR_ECOLI"},"6884456":{"id":6884456,"externalId":"511145.b3915","name":"fieF","hasFamily":true,"annotation":"ferrous iron and zinc transporter; Iron-efflux transporter responsible for iron detoxification. Also able to transport Zn(2+) in a proton- dependent manner (By similarity)","uniprotId":"FIEF_ECOLI"},"6884457":{"id":6884457,"externalId":"511145.b3916","name":"pfkA","hasFamily":true,"annotation":"6-phosphofructokinase I","uniprotId":"K6PF1_ECOLI"},"6884458":{"id":6884458,"externalId":"511145.b3917","name":"sbp","hasFamily":true,"annotation":"sulfate transporter subunit; This protein specifically binds sulfate and is involved in its transmembrane transport","uniprotId":"SUBI_ECOLI"},"6884459":{"id":6884459,"externalId":"511145.b3918","name":"cdh","hasFamily":true,"annotation":"CDP-diacylglycerol phosphotidylhydrolase","uniprotId":"CDH_ECOLI"},"6884460":{"id":6884460,"externalId":"511145.b3919","name":"tpiA","hasFamily":true,"annotation":"triosephosphate isomerase","uniprotId":"TPIS_ECOLI"},"6884461":{"id":6884461,"externalId":"511145.b3920","name":"yiiQ","hasFamily":true,"annotation":"conserved protein","uniprotId":"YIIQ_ECOLI"},"6884462":{"id":6884462,"externalId":"511145.b3921","name":"yiiR","hasFamily":true,"annotation":"putative inner membrane protein, DUF805 family","uniprotId":"YIIR_ECOLI"},"6884463":{"id":6884463,"externalId":"511145.b3922","name":"yiiS","hasFamily":false,"annotation":"conserved protein, UPF0381 family","uniprotId":"YIIS_ECOLI"},"6884464":{"id":6884464,"externalId":"511145.b3923","name":"uspD","hasFamily":false,"annotation":"stress-induced protein; Required for resistance to DNA-damaging agents","uniprotId":"USPD_ECOLI"},"6884465":{"id":6884465,"externalId":"511145.b3924","name":"fpr","hasFamily":true,"annotation":"ferredoxin-NADP reductase; Transports electrons between flavodoxin or ferredoxin and NADPH. Involved in the reductive activation of cobalamin- independent methionine synthase, pyruvate formate lyase and anaerobic ribonucleotide reductase. Also protects against superoxide radicals due to methyl viologen in the presence of oxygen","uniprotId":"FENR_ECOLI"},"6884466":{"id":6884466,"externalId":"511145.b3925","name":"glpX","hasFamily":true,"annotation":"fructose 1,6-bisphosphatase II; Catalyzes the hydrolysis of fructose 1,6-bisphosphate to fructose 6-phosphate. Is likely to be involved in gluconeogenesis during growth on glycerol. Also displays a low activity toward glucose 1,6-bisphosphate, and no activity against ribulose 1,5- bisphosphate, fructose 2,6-bisphosphate, or fructose 1-phosphate","uniprotId":"GLPX_ECOLI"},"6884467":{"id":6884467,"externalId":"511145.b3926","name":"glpK","hasFamily":true,"annotation":"glycerol kinase; Key enzyme in the regulation of glycerol uptake and metabolism (By similarity)","uniprotId":"GLPK_ECOLI"},"6884468":{"id":6884468,"externalId":"511145.b3927","name":"glpF","hasFamily":true,"annotation":"glycerol facilitator; Transporter of glycerol across the cytoplasmic membrane, with limited permeability to water and small uncharged compounds such as polyols","uniprotId":"GLPF_ECOLI"},"6884469":{"id":6884469,"externalId":"511145.b3928","name":"yiiU","hasFamily":true,"annotation":"septal ring assembly factor, stimulates cell division; Non-essential, abundant cell division factor that is required for proper Z-ring formation. It is recruited early to the divisome by direct interaction with FtsZ, stimulating Z-ring assembly and thereby promoting cell division earlier in the cell cycle. Its recruitment to the Z-ring requires functional FtsA or ZipA (By similarity)","uniprotId":"ZAPB_ECOLI"},"6884470":{"id":6884470,"externalId":"511145.b3929","name":"rraA","hasFamily":true,"annotation":"ribonuclease E (RNase E) inhibitor protein; Globally modulates RNA abundance by binding to RNase E (Rne) and regulating its endonucleolytic activity. Can modulate Rne action in a substrate-dependent manner by altering the composition of the degradosome. Modulates RNA-binding and helicase activities of the degradosome (By similarity)","uniprotId":"RRAA_ECOLI"},"6884471":{"id":6884471,"externalId":"511145.b3930","name":"menA","hasFamily":true,"annotation":"1,4-dihydroxy-2-naphthoate octaprenyltransferase; Conversion of 1,4-dihydroxy-2-naphthoate (DHNA) to dimethylmenaquinone (DMK). Attaches octaprenylpyrophosphate, a membrane-bound 40-carbon side chain to DHNA. The conversion of DHNA to DMK proceeds in three stages: the removal of the carboxyl group of DHNA as CO(2), the attachment of the isoprenoid side chain, and a quinol-to-quinone oxidation, which is thought to be spontaneous","uniprotId":"MENA_ECOLI"},"6884472":{"id":6884472,"externalId":"511145.b3931","name":"hslU","hasFamily":true,"annotation":"molecular chaperone and ATPase component of HslUV protease; ATPase subunit of a proteasome-like degradation complex; this subunit has chaperone activity. The binding of ATP and its subsequent hydrolysis by HslU are essential for unfolding of protein substrates subsequently hydrolyzed by HslV. HslU recognizes the N-terminal part of its protein substrates and unfolds these before they are guided to HslV for hydrolysis (By similarity)","uniprotId":"HSLU_ECOLI"},"6884473":{"id":6884473,"externalId":"511145.b3932","name":"hslV","hasFamily":true,"annotation":"peptidase component of the HslUV protease; Protease subunit of a proteasome-like degradation complex believed to be a general protein degrading machinery. The complex has been shown to be involved in the specific degradation of heat shock induced transcription factors such as RpoH and SulA. In addition, small hydrophobic peptides are also hydrolyzed by HslV. HslV has weak protease activity even in the absence of HslU, but this activity is induced more than 100-fold in the presence of HslU. HslU recognizes protein substrates and unfolds these before guiding them to HslV for hydrolysis.  [...] ","uniprotId":"HSLV_ECOLI"},"6884474":{"id":6884474,"externalId":"511145.b3933","name":"ftsN","hasFamily":true,"annotation":"essential cell division protein; Essential cell division protein. May play a role in the assembly or stability of the septation machinery. May interact with FtsA, PBP3 and FtsQ","uniprotId":"FTSN_ECOLI"},"6884475":{"id":6884475,"externalId":"511145.b3934","name":"cytR","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; This protein negatively controls the transcription initiation of genes such as deoCABD, udp, and cdd encoding catabolizing enzymes and nupC, nupG, and tsx encoding transporting and pore-forming proteins. Binds cytidine and adenosine as effectors","uniprotId":"CYTR_ECOLI"},"6884476":{"id":6884476,"externalId":"511145.b3935","name":"priA","hasFamily":true,"annotation":"Primosome factor n' (replication factor Y); Recognizes a specific hairpin sequence on phiX ssDNA. This structure is then recognized and bound by proteins PriB and PriC. Formation of the primosome proceeds with the subsequent actions of DnaB, DnaC, DnaT and primase. PriA then functions as a helicase within the primosome","uniprotId":"PRIA_ECOLI"},"6884477":{"id":6884477,"externalId":"511145.b3936","name":"rpmE","hasFamily":true,"annotation":"50S ribosomal subunit protein L31; Binds the 23S rRNA (By similarity)","uniprotId":"RL31_ECOLI"},"6884478":{"id":6884478,"externalId":"511145.b3937","name":"yiiX","hasFamily":false,"annotation":"conserved protein, DUF830 family","uniprotId":"YIIX_ECOLI"},"6884479":{"id":6884479,"externalId":"511145.b3938","name":"metJ","hasFamily":true,"annotation":"DNA-binding transcriptional repressor, S-adenosylmethionine-binding; This regulatory protein, when combined with SAM (S- adenosylmethionine) represses the expression of the methionine regulon and of enzymes involved in SAM synthesis. It is also autoregulated","uniprotId":"METJ_ECOLI"},"6884480":{"id":6884480,"externalId":"511145.b3939","name":"metB","hasFamily":true,"annotation":"cystathionine gamma-synthase, PLP-dependent; Catalyzes the formation of L-cystathionine from O- succinyl-L-homoserine (OSHS) and L-cysteine, via a gamma- replacement reaction. In the absence of thiol, catalyzes gamma- elimination to form 2-oxobutanoate, succinate and ammonia","uniprotId":"METB_ECOLI"},"6884481":{"id":6884481,"externalId":"511145.b3940","name":"metL","hasFamily":true,"annotation":"fused aspartokinase II/homoserine dehydrogenase II","uniprotId":"AK2H_ECOLI"},"6884482":{"id":6884482,"externalId":"511145.b3941","name":"metF","hasFamily":true,"annotation":"5,10-methylenetetrahydrofolate reductase","uniprotId":"METF_ECOLI"},"6884483":{"id":6884483,"externalId":"511145.b3942","name":"katG","hasFamily":true,"annotation":"catalase-peroxidase HPI, heme b-containing; Bifunctional enzyme with both catalase and broad- spectrum peroxidase activity. Displays also NADH oxidase, INH lyase and isonicotinoyl-NAD synthase activity","uniprotId":"KATG_ECOLI"},"6884484":{"id":6884484,"externalId":"511145.b3943","name":"yijE","hasFamily":false,"annotation":"inner membrane protein, predicted permease","uniprotId":"YIJE_ECOLI"},"6884485":{"id":6884485,"externalId":"511145.b3944","name":"yijF","hasFamily":true,"annotation":"conserved protein","uniprotId":"YIJF_ECOLI"},"6884486":{"id":6884486,"externalId":"511145.b3945","name":"gldA","hasFamily":true,"annotation":"glycerol dehydrogenase, NAD; Catalyzes the NAD-dependent oxidation of glycerol to dihydroxyacetone (glycerone). Allows microorganisms to utilize glycerol as a source of carbon under anaerobic conditions. In E.coli, an important role of GldA is also likely to regulate the intracellular level of dihydroxyacetone by catalyzing the reverse reaction, i.e. the conversion of dihydroxyacetone into glycerol. Possesses a broad substrate specificity, since it is also able to oxidize 1,2-propanediol and to reduce glycolaldehyde, methylglyoxal and hydroxyacetone into ethylene glycol, lactaldehyde a [...] ","uniprotId":"GLDA_ECOLI"},"6884487":{"id":6884487,"externalId":"511145.b3946","name":"fsaB","hasFamily":true,"annotation":"fructose-6-phosphate aldolase 2","uniprotId":"FSAB_ECOLI"},"6884488":{"id":6884488,"externalId":"511145.b3947","name":"ptsA","hasFamily":true,"annotation":"fused predicted PTS enzymes: Hpr component/enzyme I component/enzyme IIA component; Multifunctional protein that includes general (non sugar-specific) and sugar-specific components of the phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS). This major carbohydrate active-transport system catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. Enzyme I transfers the phosphoryl group from phosphoenolpyruvate (PEP) to the phosphoryl carrier protein (HPr). HPr transfers the phosphoryl group to the pho [...] ","uniprotId":"PTFX2_ECOLI"},"6884489":{"id":6884489,"externalId":"511145.b3949","name":"frwC","hasFamily":true,"annotation":"putative enzyme IIC component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane (By similarity)","uniprotId":"PTFC2_ECOLI"},"6884490":{"id":6884490,"externalId":"511145.b3950","name":"frwB","hasFamily":true,"annotation":"predicted enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (By similarity)","uniprotId":"PTFB2_ECOLI"},"6884491":{"id":6884491,"externalId":"511145.b3951","name":"pflD","hasFamily":true,"annotation":"putative formate acetyltransferase 2 (pyruvate formate lyase II)","uniprotId":"PFLD_ECOLI"},"6884492":{"id":6884492,"externalId":"511145.b3952","name":"pflC","hasFamily":true,"annotation":"pyruvate formate lyase II activase; Activation of pyruvate formate-lyase 2 under anaerobic conditions by generation of an organic free radical, using S- adenosylmethionine and reduced flavodoxin as cosubstrates to produce 5'-deoxy-adenosine (By similarity)","uniprotId":"PFLC_ECOLI"},"6884493":{"id":6884493,"externalId":"511145.b3953","name":"frwD","hasFamily":true,"annotation":"predicted enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (By similarity)","uniprotId":"PTFB3_ECOLI"},"6884494":{"id":6884494,"externalId":"511145.b3954","name":"yijO","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YIJO_ECOLI"},"6884495":{"id":6884495,"externalId":"511145.b3955","name":"eptC","hasFamily":true,"annotation":"conserved inner membrane protein; Catalyzes the addition of a phosphoethanolamine moiety to the outer membrane lipopolysaccharide core (By similarity). Plays a role in the pathogenesis of E.coli meningitis. Required for invasion of E.coli K1 into brain microvascular endothelial cells (BMEC). Contributes to E.coli traversal across the blood- brain barrier","uniprotId":"EPTC_ECOLI"},"6884496":{"id":6884496,"externalId":"511145.b3956","name":"ppc","hasFamily":true,"annotation":"phosphoenolpyruvate carboxylase; Forms oxaloacetate, a four-carbon dicarboxylic acid source for the tricarboxylic acid cycle (By similarity)","uniprotId":"CAPP_ECOLI"},"6884497":{"id":6884497,"externalId":"511145.b3957","name":"argE","hasFamily":true,"annotation":"acetylornithine deacetylase; Displays a broad specificity and can also deacylate substrates such as acetylarginine, acetylhistidine or acetylglutamate semialdehyde","uniprotId":"ARGE_ECOLI"},"6884498":{"id":6884498,"externalId":"511145.b3958","name":"argC","hasFamily":true,"annotation":"N-acetyl-gamma-glutamylphosphate reductase, NAD(P)-binding","uniprotId":"ARGC_ECOLI"},"6884499":{"id":6884499,"externalId":"511145.b3959","name":"argB","hasFamily":true,"annotation":"acetylglutamate kinase","uniprotId":"ARGB_ECOLI"},"6884500":{"id":6884500,"externalId":"511145.b3960","name":"argH","hasFamily":true,"annotation":"argininosuccinate lyase","uniprotId":"ARLY_ECOLI"},"6884501":{"id":6884501,"externalId":"511145.b3961","name":"oxyR","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; Hydrogen peroxide sensor. Activates the expression of a regulon of hydrogen peroxide-inducible genes such as katG, gor, ahpC, ahpF, oxyS (a regulatory RNA), dps, fur and grxA. OxyR expression is negatively autoregulated by binding to a 43 bp region upstream of its own coding sequence. OxyR is inactivated by reduction of its essential disulfide bond by the product of GrxA, itself positively regulated by OxyR. Has also a positive regulatory effect on the production of surface proteins that control the colony morphology and auto-aggregation ability","uniprotId":"OXYR_ECOLI"},"6884502":{"id":6884502,"externalId":"511145.b3962","name":"sthA","hasFamily":true,"annotation":"pyridine nucleotide transhydrogenase, soluble; Conversion of NADPH, generated by peripheral catabolic pathways, to NADH, which can enter the respiratory chain for energy generation (By similarity)","uniprotId":"STHA_ECOLI"},"6884503":{"id":6884503,"externalId":"511145.b3963","name":"fabR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Represses the transcription of fabB, involved in unsaturated fatty acid (UFA) biosynthesis. By controlling UFA production, FabR directly influences the physical properties of the membrane bilayer (By similarity)","uniprotId":"FABR_ECOLI"},"6884504":{"id":6884504,"externalId":"511145.b3964","name":"yijD","hasFamily":true,"annotation":"inner membrane protein, DUF1422 family","uniprotId":"YIJD_ECOLI"},"6884505":{"id":6884505,"externalId":"511145.b3965","name":"trmA","hasFamily":true,"annotation":"tRNA m(5)U54 methyltransferase, SAM-dependen; Catalyzes the formation of 5-methyl-uridine at position 54 (m5U54) in all tRNAs (By similarity)","uniprotId":"TRMA_ECOLI"},"6884506":{"id":6884506,"externalId":"511145.b3966","name":"btuB","hasFamily":true,"annotation":"vitamin B12/cobalamin outer membrane transporter; Involved in the active translocation of vitamin B12 (cyanocobalamin) across the outer membrane to the periplasmic space. It derives its energy for transport by interacting with the trans-periplasmic membrane protein TonB. Is also a receptor for bacteriophages BF23 and C1, and for A and E colicins","uniprotId":"BTUB_ECOLI"},"6884507":{"id":6884507,"externalId":"511145.b3967","name":"murI","hasFamily":true,"annotation":"glutamate racemase; Provides the (R)-glutamate required for cell wall biosynthesis (By similarity)","uniprotId":"MURI_ECOLI"},"6884508":{"id":6884508,"externalId":"511145.b3972","name":"murB","hasFamily":true,"annotation":"UDP-N-acetylenolpyruvoylglucosamine reductase, FAD-binding; Cell wall formation (By similarity)","uniprotId":"MURB_ECOLI"},"6884509":{"id":6884509,"externalId":"511145.b3973","name":"birA","hasFamily":true,"annotation":"bifunctional biotin-[acetylCoA carboxylase] holoenzyme synthetase/ DNA-binding transcriptional repressor, bio-5'-AMP-binding; BirA acts both as a biotin-operon repressor and as the enzyme that synthesizes the corepressor, acetyl-CoA:carbon-dioxide ligase. This protein also activates biotin to form biotinyl-5'- adenylate and transfers the biotin moiety to biotin-accepting proteins","uniprotId":"BIRA_ECOLI"},"6884510":{"id":6884510,"externalId":"511145.b3974","name":"coaA","hasFamily":true,"annotation":"pantothenate kinase","uniprotId":"COAA_ECOLI"},"6884511":{"id":6884511,"externalId":"511145.b3980","name":"tuf","hasFamily":true,"annotation":"protein chain elongation factor EF-Tu (duplicate of tufA); This protein promotes the GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis (By similarity)","uniprotId":"EFTU2_ECOLI"},"6884512":{"id":6884512,"externalId":"511145.b3981","name":"secE","hasFamily":true,"annotation":"preprotein translocase membrane subunit; Essential subunit of the protein translocation channel SecYEG. Clamps together the 2 halves of SecY. May contact the channel plug during translocation. Overexpression of some hybrid proteins has been thought to jam the protein secretion apparatus resulting in cell death; while this may be true it also results in FtsH-mediated degradation of SecY","uniprotId":"SECE_ECOLI"},"6884513":{"id":6884513,"externalId":"511145.b3982","name":"nusG","hasFamily":true,"annotation":"transcription termination factor; Participates in transcription elongation, termination and antitermination. In the absence of Rho, increases the rate of transcription elongation by the RNA polymerase (RNAP), probably by partially suppressing pausing. In the presence of Rho, modulates most Rho-dependent termination events by interacting with the RNAP to render the complex more susceptible to the termination activity of Rho. May be required to overcome a kinetic limitation of Rho to function at certain terminators. Also involved in ribosomal RNA and phage lambda N-mediated transcription [...] ","uniprotId":"NUSG_ECOLI"},"6884514":{"id":6884514,"externalId":"511145.b3983","name":"rplK","hasFamily":true,"annotation":"50S ribosomal subunit protein L11; This protein binds directly to 23S ribosomal RNA. Forms the L11 stalk, which is mobile in the ribosome, indicating its contribution to the activity of initiation, elongation and release factors","uniprotId":"RL11_ECOLI"},"6884515":{"id":6884515,"externalId":"511145.b3984","name":"rplA","hasFamily":true,"annotation":"50S ribosomal subunit protein L1; One of the primary rRNA binding proteins, it binds very close to the 3'-end of the 23S rRNA. Forms part of the L1 stalk. It is often not seen in high-resolution crystal structures, but can be seen in cryo_EM and 3D reconstruction models. These indicate that the distal end of the stalk moves by approximately 20 angstroms (PubMed:12859903). This stalk movement is thought to be coupled to movement of deacylated tRNA into and out of the E site, and thus to participate in tRNA translocation (PubMed:12859903). Contacts the P and E site tRNAs","uniprotId":"RL1_ECOLI"},"6884516":{"id":6884516,"externalId":"511145.b3985","name":"rplJ","hasFamily":true,"annotation":"50S ribosomal subunit protein L10; Protein L10 is also a translational repressor protein. It controls the translation of the rplJL-rpoBC operon by binding to its mRNA","uniprotId":"RL10_ECOLI"},"6884517":{"id":6884517,"externalId":"511145.b3986","name":"rplL","hasFamily":true,"annotation":"50S ribosomal subunit protein L7/L12; Seems to be the binding site for several of the factors involved in protein synthesis and appears to be essential for accurate translation (By similarity)","uniprotId":"RL7_ECOLI"},"6884518":{"id":6884518,"externalId":"511145.b3987","name":"rpoB","hasFamily":true,"annotation":"RNA polymerase, beta subunit; DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates (By similarity)","uniprotId":"RPOB_ECOLI"},"6884519":{"id":6884519,"externalId":"511145.b3988","name":"rpoC","hasFamily":true,"annotation":"RNA polymerase, beta prime subunit; DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates (By similarity)","uniprotId":"RPOC_ECOLI"},"6884520":{"id":6884520,"externalId":"511145.b3989","name":"htrC","hasFamily":false,"annotation":"stationary phase growth adaptation protein","uniprotId":"HTRC_ECOLI"},"6884521":{"id":6884521,"externalId":"511145.b3990","name":"thiH","hasFamily":true,"annotation":"tyrosine lyase, involved in thiamin-thiazole moiety synthesis; Catalyzes the radical-mediated cleavage of tyrosine to 2-iminoacetate and 4-cresol","uniprotId":"THIH_ECOLI"},"6884522":{"id":6884522,"externalId":"511145.b3991","name":"thiG","hasFamily":true,"annotation":"thiamin biosynthesis ThiGH complex subunit; Catalyzes the rearrangement of 1-deoxy-D-xylulose 5- phosphate (DXP) to produce the thiazole phosphate moiety of thiamine. Sulfur is provided by the thiocarboxylate moiety of the carrier protein ThiS. In vitro, sulfur can be provided by H(2)S (By similarity)","uniprotId":"THIG_ECOLI"},"6884523":{"id":6884523,"externalId":"511145.b3992","name":"thiF","hasFamily":true,"annotation":"adenylyltransferase, modifies ThiS C-terminus; Catalyzes the adenylation by ATP of the carboxyl group of the C-terminal glycine of sulfur carrier protein ThiS","uniprotId":"THIF_ECOLI"},"6884524":{"id":6884524,"externalId":"511145.b3993","name":"thiE","hasFamily":true,"annotation":"thiamin phosphate synthase (thiamin phosphate pyrophosphorylase); Condenses 4-methyl-5-(beta-hydroxyethyl)thiazole monophosphate (THZ-P) and 2-methyl-4-amino-5-hydroxymethyl pyrimidine pyrophosphate (HMP-PP) to form thiamine monophosphate (TMP) (By similarity)","uniprotId":"THIE_ECOLI"},"6884525":{"id":6884525,"externalId":"511145.b3994","name":"thiC","hasFamily":true,"annotation":"thiamin (pyrimidine moiety) biosynthesis protein; Catalyzes the synthesis of the hydroxymethylpyrimidine phosphate (HMP-P) moiety of thiamine from aminoimidazole ribotide (AIR) in a radical S-adenosyl-L-methionine (SAM)-dependent reaction (By similarity)","uniprotId":"THIC_ECOLI"},"6884526":{"id":6884526,"externalId":"511145.b3995","name":"rsd","hasFamily":true,"annotation":"stationary phase protein, binds sigma 70 RNA polymerase subunit; Has binding activity to the major sigma subunit of RNA polymerase. May be involved in replacement of the RNA polymerase sigma subunit from sigma-70 to sigma-S during the transition from exponential growth to the stationary phase (By similarity)","uniprotId":"RSD_ECOLI"},"6884527":{"id":6884527,"externalId":"511145.b3996","name":"nudC","hasFamily":true,"annotation":"NADH pyrophosphatase","uniprotId":"NUDC_ECOLI"},"6884528":{"id":6884528,"externalId":"511145.b3997","name":"hemE","hasFamily":true,"annotation":"uroporphyrinogen decarboxylase; Catalyzes the decarboxylation of four acetate groups of uroporphyrinogen-III to yield coproporphyrinogen-III (By similarity)","uniprotId":"DCUP_ECOLI"},"6884529":{"id":6884529,"externalId":"511145.b3998","name":"nfi","hasFamily":true,"annotation":"endonuclease V; Selectively cleaves double-stranded DNA at the second phosphodiester bond 3' to a deoxyinosine leaving behind the intact lesion on the nicked DNA. Has a wide substrate spectrum. In addition to deoxyinosine-containing DNA, the enzyme cleaves DNA containing urea residues, AP sites, base mismatches, insertion/deletion mismatches, flaps, and pseudo-Y structures. Participates in the excision repair of hypoxanthine and xanthine (deaminated adenine and guanine) in DNA. It thereby reduces the mutagenic effects of nitrous acid by attacking lesions caused by nitrosative deaminati [...] ","uniprotId":"NFI_ECOLI"},"6884530":{"id":6884530,"externalId":"511145.b3999","name":"yjaG","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJAG_ECOLI"},"6884531":{"id":6884531,"externalId":"511145.b4000","name":"hupA","hasFamily":true,"annotation":"HU, DNA-binding transcriptional regulator, alpha subunit; Histone-like DNA-binding protein which is capable of wrapping DNA to stabilize it, and thus to prevent its denaturation under extreme environmental conditions","uniprotId":"DBHA_ECOLI"},"6884532":{"id":6884532,"externalId":"511145.b4001","name":"yjaH","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJAH_ECOLI"},"6884533":{"id":6884533,"externalId":"511145.b4002","name":"yjaI","hasFamily":true,"annotation":"Zn-binding periplasmic protein; Binds zinc. Could be an important component of the zinc- balancing mechanism","uniprotId":"ZRAP_ECOLI"},"6884534":{"id":6884534,"externalId":"511145.b4003","name":"zraS","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with ZraR; Member of the two-component regulatory system ZraS/ZraR. May function as a membrane-associated protein kinase that phosphorylates ZraR in response to high concentrations of zinc or lead in the medium","uniprotId":"ZRAS_ECOLI"},"6884535":{"id":6884535,"externalId":"511145.b4004","name":"hydG","hasFamily":true,"annotation":"fused DNA-binding response regulator in two-component regulatory system with ZraS: response regulator/sigma54 interaction protein; Member of the two-component regulatory system ZraS/ZraR. When activated by ZraS it acts in conjunction with sigma-54 to regulate the expression of zraP. Positively autoregulates the expression of the zraSR operon","uniprotId":"ZRAR_ECOLI"},"6884536":{"id":6884536,"externalId":"511145.b4005","name":"purD","hasFamily":true,"annotation":"phosphoribosylglycinamide synthetase phosphoribosylamine-glycine ligase","uniprotId":"PUR2_ECOLI"},"6884537":{"id":6884537,"externalId":"511145.b4006","name":"purH","hasFamily":true,"annotation":"fused IMP cyclohydrolase/phosphoribosylaminoimidazolecarboxamide formyltransferase","uniprotId":"PUR9_ECOLI"},"6884538":{"id":6884538,"externalId":"511145.b4011","name":"yjaA","hasFamily":true,"annotation":"conserved stress-induced protein","uniprotId":"YJAA_ECOLI"},"6884539":{"id":6884539,"externalId":"511145.b4012","name":"yjaB","hasFamily":true,"annotation":"putative acetyltransferase","uniprotId":"YJAB_ECOLI"},"6884540":{"id":6884540,"externalId":"511145.b4013","name":"metA","hasFamily":true,"annotation":"homoserine O-transsuccinylase","uniprotId":"META_ECOLI"},"6884541":{"id":6884541,"externalId":"511145.b4014","name":"aceB","hasFamily":true,"annotation":"malate synthase A","uniprotId":"MASY_ECOLI"},"6884542":{"id":6884542,"externalId":"511145.b4015","name":"aceA","hasFamily":true,"annotation":"isocitrate lyase; Catalyzes the formation of succinate and glyoxylate from isocitrate, a key step of the glyoxylate cycle. May be involved in the assimilation of one-carbon compounds via the isocitrate lyase- positive serine pathway (By similarity)","uniprotId":"ACEA_ECOLI"},"6884543":{"id":6884543,"externalId":"511145.b4016","name":"aceK","hasFamily":true,"annotation":"isocitrate dehydrogenase kinase/phosphatase; Bifunctional enzyme which can phosphorylate or dephosphorylate isocitrate dehydrogenase (IDH) on a specific serine residue. This is a regulatory mechanism which enables bacteria to bypass the Krebs cycle via the glyoxylate shunt in response to the source of carbon. When bacteria are grown on glucose, IDH is fully active and unphosphorylated, but when grown on acetate or ethanol, the activity of IDH declines drastically concomitant with its phosphorylation (By similarity)","uniprotId":"ACEK_ECOLI"},"6884544":{"id":6884544,"externalId":"511145.b4017","name":"arpA","hasFamily":false,"annotation":"ankyrin repeat protein","uniprotId":"ARPA_ECOLI"},"6884545":{"id":6884545,"externalId":"511145.b4018","name":"iclR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Regulation of the glyoxylate bypass operon (aceBAK), which encodes isocitrate lyase, malate synthase as well as isocitrate dehydrogenase kinase/phosphorylase. Glyoxylate disrupts the interaction with the promoter by favoring the inactive dimeric form. Pyruvate enhances promoter binding by stabilizing the tetrameric form","uniprotId":"ICLR_ECOLI"},"6884546":{"id":6884546,"externalId":"511145.b4019","name":"metH","hasFamily":true,"annotation":"homocysteine-N5-methyltetrahydrofolate transmethylase, B12-dependent; Catalyzes the transfer of a methyl group from methyl- cobalamin to homocysteine, yielding enzyme-bound cob(I)alamin and methionine. Subsequently, remethylates the cofactor using methyltetrahydrofolate","uniprotId":"METH_ECOLI"},"6884547":{"id":6884547,"externalId":"511145.b4020","name":"yjbB","hasFamily":true,"annotation":"putative Na+/Pi-cotransporter","uniprotId":"YJBB_ECOLI"},"6884548":{"id":6884548,"externalId":"511145.b4021","name":"pepE","hasFamily":true,"annotation":"(alpha)-aspartyl dipeptidase; Hydrolyzes dipeptides containing N-terminal aspartate residues. May play a role in allowing the cell to use peptide aspartate to spare carbon otherwise required for the synthesis of the aspartate family of amino acids (By similarity)","uniprotId":"PEPE_ECOLI"},"6884549":{"id":6884549,"externalId":"511145.b4022","name":"rluF","hasFamily":true,"annotation":"23S rRNA U2604 pseudouridine synthase; Responsible for synthesis of pseudouridine from uracil- 2604 in 23S ribosomal RNA. Can, to a small extent, also react with uracil-2605","uniprotId":"RLUF_ECOLI"},"6884550":{"id":6884550,"externalId":"511145.b4023","name":"yjbD","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJBD_ECOLI"},"6884551":{"id":6884551,"externalId":"511145.b4024","name":"lysC","hasFamily":true,"annotation":"aspartokinase III","uniprotId":"AK3_ECOLI"},"6884552":{"id":6884552,"externalId":"511145.b4025","name":"pgi","hasFamily":true,"annotation":"glucosephosphate isomerase","uniprotId":"G6PI_ECOLI"},"6884553":{"id":6884553,"externalId":"511145.b4026","name":"yjbE","hasFamily":false,"annotation":"predicted protein","uniprotId":"YJBE_ECOLI"},"6884554":{"id":6884554,"externalId":"511145.b4027","name":"yjbF","hasFamily":true,"annotation":"putative lipoprotein","uniprotId":"YJBF_ECOLI"},"6884555":{"id":6884555,"externalId":"511145.b4028","name":"yjbG","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJBG_ECOLI"},"6884556":{"id":6884556,"externalId":"511145.b4029","name":"yjbH","hasFamily":true,"annotation":"putative porin","uniprotId":"YJBH_ECOLI"},"6884557":{"id":6884557,"externalId":"511145.b4030","name":"psiE","hasFamily":true,"annotation":"phosphate starvation inducible protein","uniprotId":"PSIE_ECOLI"},"6884558":{"id":6884558,"externalId":"511145.b4031","name":"xylE","hasFamily":true,"annotation":"D-xylose transporter; Uptake of D-xylose across the boundary membrane with the concomitant transport of protons into the cell (symport system)","uniprotId":"XYLE_ECOLI"},"6884559":{"id":6884559,"externalId":"511145.b4032","name":"malG","hasFamily":true,"annotation":"maltose transporter subunit; Part of the binding-protein-dependent transport system for maltose; probably responsible for the translocation of the substrate across the membrane","uniprotId":"MALG_ECOLI"},"6884560":{"id":6884560,"externalId":"511145.b4033","name":"malF","hasFamily":true,"annotation":"maltose transporter subunit; Part of the binding-protein-dependent transport system for maltose; probably responsible for the translocation of the substrate across the membrane","uniprotId":"MALF_ECOLI"},"6884561":{"id":6884561,"externalId":"511145.b4034","name":"malE","hasFamily":true,"annotation":"maltose transporter subunit; Involved in the high-affinity maltose membrane transport system MalEFGK. Initial receptor for the active transport of and chemotaxis toward maltooligosaccharides","uniprotId":"MALE_ECOLI"},"6884562":{"id":6884562,"externalId":"511145.b4035","name":"malK","hasFamily":true,"annotation":"fused maltose transport subunit, ATP-binding component of ABC superfamily/regulatory protein; Part of the ABC transporter complex MalEFGK involved in maltose/maltodextrin import. Responsible for energy coupling to the transport system","uniprotId":"MALK_ECOLI"},"6884563":{"id":6884563,"externalId":"511145.b4036","name":"lamB","hasFamily":true,"annotation":"maltose outer membrane porin (maltoporin); Involved in the transport of maltose and maltodextrins, indispensable for translocation of dextrins containing more than three glucosyl moieties. A hydrophobic path (\"greasy slide\") of aromatic residues serves to guide and select the sugars for transport through the channel. Also acts as a receptor for several bacteriophages including lambda","uniprotId":"LAMB_ECOLI"},"6884564":{"id":6884564,"externalId":"511145.b4037","name":"malM","hasFamily":true,"annotation":"maltose regulon periplasmic protein; Not yet known. Might function in the uptake of a still unidentified substrate","uniprotId":"MALM_ECOLI"},"6884565":{"id":6884565,"externalId":"511145.b4039","name":"ubiC","hasFamily":true,"annotation":"chorismate--pyruvate lyase; Removes the pyruvyl group from chorismate, with concomitant aromatization of the ring, to provide 4- hydroxybenzoate (4HB) for the ubiquinone pathway (By similarity)","uniprotId":"UBIC_ECOLI"},"6884566":{"id":6884566,"externalId":"511145.b4040","name":"ubiA","hasFamily":true,"annotation":"p-hydroxybenzoate octaprenyltransferase; Synthesis of 3-octaprenyl-4-hydroxybenzoate (By similarity)","uniprotId":"UBIA_ECOLI"},"6884567":{"id":6884567,"externalId":"511145.b4041","name":"plsB","hasFamily":true,"annotation":"glycerol-3-phosphate O-acyltransferase; Catalyzes the transfer of an acyl group from acyl-ACP to glycerol-3-phosphate (G3P) to form lysophosphatidic acid (LPA). This enzyme can utilize either acyl-CoA or acyl-ACP as the fatty acyl donor","uniprotId":"PLSB_ECOLI"},"6884568":{"id":6884568,"externalId":"511145.b4042","name":"dgkA","hasFamily":true,"annotation":"diacylglycerol kinase; Recycling of diacylglycerol produced during the turnover of membrane phospholipid","uniprotId":"KDGL_ECOLI"},"6884569":{"id":6884569,"externalId":"511145.b4043","name":"lexA","hasFamily":true,"annotation":"DNA-binding transcriptional repressor of SOS regulon; Represses a number of genes involved in the response to DNA damage (SOS response), including recA and lexA. Binds to the 16 bp palindromic sequence 5'-CTGTATATATATACAG-3'. In the presence of single-stranded DNA, RecA interacts with LexA causing an autocatalytic cleavage which disrupts the DNA-binding part of LexA, leading to derepression of the SOS regulon and eventually DNA repair (By similarity)","uniprotId":"LEXA_ECOLI"},"6884570":{"id":6884570,"externalId":"511145.b4044","name":"dinF","hasFamily":true,"annotation":"DNA-damage-inducible SOS response protein","uniprotId":"DINF_ECOLI"},"6884571":{"id":6884571,"externalId":"511145.b4045","name":"yjbJ","hasFamily":true,"annotation":"stress-induced protein, UPF0337 family","uniprotId":"YJBJ_ECOLI"},"6884572":{"id":6884572,"externalId":"511145.b4046","name":"zur","hasFamily":true,"annotation":"DNA-binding transcriptional repressor, Zn(II)-binding; Acts as a negative controlling element, employing Zn(2+) as a cofactor to bind the operator of the repressed genes (znuACB)","uniprotId":"ZUR_ECOLI"},"6884573":{"id":6884573,"externalId":"511145.b4047","name":"yjbL","hasFamily":false,"annotation":"predicted protein","uniprotId":"YJBL_ECOLI"},"6884574":{"id":6884574,"externalId":"511145.b4048","name":"yjbM","hasFamily":false,"annotation":"predicted protein","uniprotId":"YJBM_ECOLI"},"6884575":{"id":6884575,"externalId":"511145.b4049","name":"dusA","hasFamily":true,"annotation":"tRNA-dihydrouridine synthase A; Catalyzes the synthesis of dihydrouridine, a modified base found in the D-loop of most tRNAs (By similarity)","uniprotId":"DUSA_ECOLI"},"6884576":{"id":6884576,"externalId":"511145.b4050","name":"pspG","hasFamily":true,"annotation":"phage shock protein G; Effector of the phage shock response","uniprotId":"PSPG_ECOLI"},"6884577":{"id":6884577,"externalId":"511145.b4051","name":"qor","hasFamily":true,"annotation":"quinone oxidoreductase, NADPH-dependent","uniprotId":"QOR1_ECOLI"},"6884578":{"id":6884578,"externalId":"511145.b4052","name":"dnaB","hasFamily":true,"annotation":"replicative DNA helicase; Participates in initiation and elongation during chromosome replication; it exhibits DNA-dependent ATPase activity and contains distinct active sites for ATP binding, DNA binding, and interaction with DnaC protein, primase, and other prepriming proteins","uniprotId":"DNAB_ECOLI"},"6884579":{"id":6884579,"externalId":"511145.b4053","name":"alr","hasFamily":true,"annotation":"alanine racemase 1, PLP-binding, biosynthetic; Catalyzes the interconversion of L-alanine and D- alanine. Provides the D-alanine required for cell wall biosynthesis","uniprotId":"ALR1_ECOLI"},"6884580":{"id":6884580,"externalId":"511145.b4054","name":"tyrB","hasFamily":true,"annotation":"tyrosine aminotransferase, tyrosine-repressible, PLP-dependent","uniprotId":"TYRB_ECOLI"},"6884581":{"id":6884581,"externalId":"511145.b4055","name":"aphA","hasFamily":true,"annotation":"acid phosphatase/phosphotransferase, class B, non-specific; Dephosphorylates several organic phosphate monoesters including 3'- and 5'-nucleotides, 2'-deoxy-5'-nucleotides, pNPP, phenyl phosphate, glycerol 2-phosphate, ribose 5-phosphate, O- phospho-L-amino acids and phytic acid, showing the highest activity with aryl phosphoesters (pNPP, phenyl phosphate and O- phospho-L-tyrosine), and to a lesser extent with 3'- and 5'- nucleotides. No activity toward ATP, phosphodiesters, glycerol-1- phosphate, glucose 1-phosphate, glucose 6-phosphate, NADP, GTP or 3',5'-cAMP, ADP or ATP. Also has a [...] ","uniprotId":"APHA_ECOLI"},"6884582":{"id":6884582,"externalId":"511145.b4056","name":"yjbQ","hasFamily":true,"annotation":"thiamin phosphate synthase","uniprotId":"YJBQ_ECOLI"},"6884583":{"id":6884583,"externalId":"511145.b4057","name":"yjbR","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJBR_ECOLI"},"6884584":{"id":6884584,"externalId":"511145.b4058","name":"uvrA","hasFamily":true,"annotation":"ATPase and DNA damage recognition protein of nucleotide excision repair excinuclease UvrABC; The UvrABC repair system catalyzes the recognition and processing of DNA lesions. UvrA is an ATPase and a DNA-binding protein. A damage recognition complex composed of 2 UvrA and 2 UvrB subunits scans DNA for abnormalities. When the presence of a lesion has been verified by UvrB, the UvrA molecules dissociate (By similarity)","uniprotId":"UVRA_ECOLI"},"6884585":{"id":6884585,"externalId":"511145.b4059","name":"ssb","hasFamily":true,"annotation":"single-stranded DNA-binding protein; This protein is essential for replication of the chromosomes and its single-stranded DNA phages. It is also involved in DNA recombination and repair","uniprotId":"SSB_ECOLI"},"6884586":{"id":6884586,"externalId":"511145.b4060","name":"yjcB","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJCB_ECOLI"},"6884587":{"id":6884587,"externalId":"511145.b4061","name":"yjcC","hasFamily":true,"annotation":"predicted cyclic-di-GMP phosphodiesterase; Overexpression reduces biofilm formation","uniprotId":"YJCC_ECOLI"},"6884588":{"id":6884588,"externalId":"511145.b4062","name":"soxS","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; Transcriptional activator of the superoxide response regulon of E.coli that includes at least 10 genes such as sodA, nfo, zwf and micF. Binds the DNA sequence 5'-GCACN(7)CAA-3'. It also facilitates the subsequent binding of RNA polymerase to the micF and the nfo promoters","uniprotId":"SOXS_ECOLI"},"6884589":{"id":6884589,"externalId":"511145.b4063","name":"soxR","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator, Fe-S center for redox-sensing; Activates the transcription of the soxS gene which itself controls the superoxide response regulon. SoxR contains a 2Fe-2S iron-sulfur cluster that may act as a redox sensor system that recognizes superoxide. The variable redox state of the Fe-S cluster is employed in vivo to modulate the transcriptional activity of SoxR in response to specific types of oxidative stress. Upon reduction of 2Fe-2S cluster, SoxR reversibly loses its transcriptional activity, but retains its DNA binding affinity","uniprotId":"SOXR_ECOLI"},"6884590":{"id":6884590,"externalId":"511145.b4064","name":"yjcD","hasFamily":true,"annotation":"putative permease","uniprotId":"YJCD_ECOLI"},"6884591":{"id":6884591,"externalId":"511145.b4065","name":"yjcE","hasFamily":true,"annotation":"putative cation/proton antiporter","uniprotId":"YJCE_ECOLI"},"6884592":{"id":6884592,"externalId":"511145.b4066","name":"yjcF","hasFamily":false,"annotation":"pentapeptide repeat protein","uniprotId":"YJCF_ECOLI"},"6884593":{"id":6884593,"externalId":"511145.b4067","name":"actP","hasFamily":true,"annotation":"acetate transporter; Transports acetate. Also able to transport glycolate","uniprotId":"ACTP_ECOLI"},"6884594":{"id":6884594,"externalId":"511145.b4068","name":"yjcH","hasFamily":true,"annotation":"inner membrane protein, DUF485 family","uniprotId":"YJCH_ECOLI"},"6884595":{"id":6884595,"externalId":"511145.b4069","name":"acs","hasFamily":true,"annotation":"acetyl-CoA synthetase; Catalyzes the conversion of acetate into acetyl-CoA (AcCoA), an essential intermediate at the junction of anabolic and catabolic pathways. Acs undergoes a two-step reaction. In the first half reaction, Acs combines acetate with ATP to form acetyl- adenylate (AcAMP) intermediate. In the second half reaction, it can then transfer the acetyl group from AcAMP to the sulfhydryl group of CoA, forming the product AcCoA (By similarity)","uniprotId":"ACSA_ECOLI"},"6884596":{"id":6884596,"externalId":"511145.b4070","name":"nrfA","hasFamily":true,"annotation":"nitrite reductase, formate-dependent, cytochrome; Plays a role in nitrite reduction (By similarity)","uniprotId":"NRFA_ECOLI"},"6884597":{"id":6884597,"externalId":"511145.b4071","name":"nrfB","hasFamily":true,"annotation":"nitrite reductase, formate-dependent, penta-heme cytochrome c; Plays a role in nitrite reduction","uniprotId":"NRFB_ECOLI"},"6884598":{"id":6884598,"externalId":"511145.b4072","name":"nrfC","hasFamily":true,"annotation":"formate-dependent nitrite reductase, 4Fe4S subunit; Probably involved in the transfer of electrons from the quinone pool to the type-c cytochromes","uniprotId":"NRFC_ECOLI"},"6884599":{"id":6884599,"externalId":"511145.b4073","name":"nrfD","hasFamily":true,"annotation":"formate-dependent nitrite reductase, membrane subunit; Probably involved in the transfer of electrons from the quinone pool to the type-c cytochromes","uniprotId":"NRFD_ECOLI"},"6884600":{"id":6884600,"externalId":"511145.b4074","name":"nrfE","hasFamily":true,"annotation":"heme lyase (NrfEFG) for insertion of heme into c552, subunit NrfE; May be required for the biogenesis of c-type cytochromes. Possible subunit of a heme lyase","uniprotId":"NRFE_ECOLI"},"6884601":{"id":6884601,"externalId":"511145.b4075","name":"nrfF","hasFamily":true,"annotation":"heme lyase (NrfEFG) for insertion of heme into c552, subunit NrfF; Not required for the biosynthesis of any of the c-type cytochromes. Possible subunit of a heme lyase","uniprotId":"NRFF_ECOLI"},"6884602":{"id":6884602,"externalId":"511145.b4076","name":"nrfG","hasFamily":true,"annotation":"heme lyase (NrfEFG) for insertion of heme into c552, subunit NrfG; Required for formate-dependent nitrite reduction. Not required for the biosynthesis of any of the c-type cytochromes nor for the secretion of the periplasmic cytochromes","uniprotId":"NRFG_ECOLI"},"6884603":{"id":6884603,"externalId":"511145.b4077","name":"gltP","hasFamily":true,"annotation":"glutamate/aspartate:proton symporter; This carrier protein is part of the Na(+)-independent, binding-protein-independent glutamate-aspartate transport system","uniprotId":"GLTP_ECOLI"},"6884604":{"id":6884604,"externalId":"511145.b4078","name":"yjcO","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJCO_ECOLI"},"6884605":{"id":6884605,"externalId":"511145.b4079","name":"fdhF","hasFamily":true,"annotation":"formate dehydrogenase-H, selenopolypeptide subunit; Decomposes formic acid to hydrogen and carbon dioxide under anaerobic conditions in the absence of exogenous electron acceptors","uniprotId":"FDHF_ECOLI"},"6884606":{"id":6884606,"externalId":"511145.b4080","name":"mdtP","hasFamily":true,"annotation":"outer membrane factor of efflux pump; Could be involved in resistance to puromycin, acriflavine and tetraphenylarsonium chloride","uniprotId":"MDTP_ECOLI"},"6884607":{"id":6884607,"externalId":"511145.b4081","name":"mdtO","hasFamily":true,"annotation":"membrane translocase (MDR) of MdtNOP efflux pump, PET family; Could be involved in resistance to puromycin, acriflavine and tetraphenylarsonium chloride","uniprotId":"MDTO_ECOLI"},"6884608":{"id":6884608,"externalId":"511145.b4082","name":"mdtN","hasFamily":true,"annotation":"membrane fusion protein of efflux pump; Could be involved in resistance to puromycin, acriflavine and tetraphenylarsonium chloride","uniprotId":"MDTN_ECOLI"},"6884609":{"id":6884609,"externalId":"511145.b4083","name":"yjcS","hasFamily":true,"annotation":"conserved protein, metallo-beta-lactamase superfamily","uniprotId":"YJCS_ECOLI"},"6884610":{"id":6884610,"externalId":"511145.b4084","name":"alsK","hasFamily":false,"annotation":"D-allose kinase; Catalyzes the phosphorylation of D-allose to D-allose 6- P. Has also low level glucokinase activity in vitro","uniprotId":"ALSK_ECOLI"},"6884611":{"id":6884611,"externalId":"511145.b4085","name":"alsE","hasFamily":true,"annotation":"allulose-6-phosphate 3-epimerase; Catalyzes the reversible epimerization of D-allulose 6- phosphate to D-fructose 6-phosphate. Can also catalyze with lower efficiency the reversible epimerization of D-ribulose 5-phosphate to D-xylulose 5-phosphate","uniprotId":"ALSE_ECOLI"},"6884612":{"id":6884612,"externalId":"511145.b4086","name":"alsC","hasFamily":false,"annotation":"D-allose transporter subunit; Part of the binding-protein-dependent transport system AlsBAC for D-allose; probably responsible for the translocation of the substrate across the membrane","uniprotId":"ALSC_ECOLI"},"6884613":{"id":6884613,"externalId":"511145.b4087","name":"alsA","hasFamily":true,"annotation":"fused D-allose transporter subunits of ABC superfamily: ATP-binding components; Part of the ABC transporter complex AlsBAC involved in D-allose import. Probably responsible for energy coupling to the transport system","uniprotId":"ALSA_ECOLI"},"6884614":{"id":6884614,"externalId":"511145.b4088","name":"alsB","hasFamily":false,"annotation":"D-allose transporter subunit; Part of the binding-protein-dependent transport system AlsBAC for D-allose","uniprotId":"ALSB_ECOLI"},"6884615":{"id":6884615,"externalId":"511145.b4089","name":"rpiR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Regulatory protein involved in rpiB gene repression. Also involved in als operon repression","uniprotId":"RPIR_ECOLI"},"6884616":{"id":6884616,"externalId":"511145.b4090","name":"rpiB","hasFamily":true,"annotation":"ribose 5-phosphate isomerase B/allose 6-phosphate isomerase; In addition to its activity on D-ribose 5-phosphate it probably also has activity on D-allose 6-phosphate","uniprotId":"RPIB_ECOLI"},"6884617":{"id":6884617,"externalId":"511145.b4092","name":"phnP","hasFamily":true,"annotation":"5-phospho-alpha-D-ribosyl 1,2-cyclic phosphate phosphodiesterase; Catalyzes the hydrolysis of the cyclic ribose-phosphate to form alpha-D-ribose 1,5-bisphosphate","uniprotId":"PHNP_ECOLI"},"6884618":{"id":6884618,"externalId":"511145.b4093","name":"phnO","hasFamily":true,"annotation":"putative aminoalkylphosphonic acid N-acetyltransferase; Belongs to an operon involved in alkylphosphonate uptake and C-P lyase. Exact function not known","uniprotId":"PHNO_ECOLI"},"6884619":{"id":6884619,"externalId":"511145.b4094","name":"phnN","hasFamily":true,"annotation":"ribose 1,5-bisphosphokinase; Catalyzes the phosphorylation of ribose 1,5-bisphosphate to 5-phospho-D-ribosyl alpha-1-diphosphate (PRPP). Accepts ATP but not GTP as a phosphoryl donor, and uses ribose 1,5-bisphosphate but not ribose, ribose 1-phosphate, or ribose 5-phosphate as a phosphoryl acceptor","uniprotId":"PHNN_ECOLI"},"6884620":{"id":6884620,"externalId":"511145.b4095","name":"phnM","hasFamily":true,"annotation":"ribophosphonate triphosphate hydrolase; Catalyzes the hydrolysis of alpha-D-ribose 1- methylphosphonate triphosphate (RPnTP) to form alpha-D-ribose 1- methylphosphonate phosphate (PRPn) and diphosphate","uniprotId":"PHNM_ECOLI"},"6884621":{"id":6884621,"externalId":"511145.b4096","name":"phnL","hasFamily":true,"annotation":"carbon-phosphorus lyase complex subunit; Together with PhnG, PhnH and PhnI is required for the transfer of the ribose triphosphate moiety from ATP to methyl phosphonate","uniprotId":"PHNL_ECOLI"},"6884622":{"id":6884622,"externalId":"511145.b4097","name":"phnK","hasFamily":true,"annotation":"carbon-phosphorus lyase complex subunit; Belongs to an operon involved in alkylphosphonate uptake and C-P lyase. Exact function not known. PhnK is not required for the ribophosphonate triphosphate (RPnTP) synthase reaction","uniprotId":"PHNK_ECOLI"},"6884623":{"id":6884623,"externalId":"511145.b4098","name":"phnJ","hasFamily":true,"annotation":"carbon-phosphorus lyase, SAM-dependent; Catalyzes the breakage of the C-P bond in alpha-D-ribose 1-methylphosphonate 5-phosphate (PRPn) forming alpha-D-ribose 1,2- cyclic phosphate 5-phosphate (PRcP)","uniprotId":"PHNJ_ECOLI"},"6884624":{"id":6884624,"externalId":"511145.b4099","name":"phnI","hasFamily":true,"annotation":"carbon-phosphorus lyase complex subunit; Together with PhnG, PhnH and PhnL is required for the transfer of the ribose triphosphate moiety from ATP to methyl phosphonate. PhnI alone has nucleosidase activity, catalyzing the hydrolysis of ATP or GTP forming alpha-D-ribose 5-triphosphate and adenine or guanine, respectively","uniprotId":"PHNI_ECOLI"},"6884625":{"id":6884625,"externalId":"511145.b4100","name":"phnH","hasFamily":true,"annotation":"carbon-phosphorus lyase complex subunit; Together with PhnG, PhnI and PhnL is required for the transfer of the ribose triphosphate moiety from ATP to methyl phosphonate","uniprotId":"PHNH_ECOLI"},"6884626":{"id":6884626,"externalId":"511145.b4101","name":"phnG","hasFamily":true,"annotation":"carbon-phosphorus lyase complex subunit; Together with PhnH, PhnI and PhnL is required for the transfer of the ribose triphosphate moiety from ATP to methyl phosphonate","uniprotId":"PHNG_ECOLI"},"6884627":{"id":6884627,"externalId":"511145.b4102","name":"phnF","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator of phosphonate uptake and biodegradation; Belongs to an operon involved in alkylphosphonate uptake and C-P lyase. Exact function not known. By similarity could be a transcriptional regulator","uniprotId":"PHNF_ECOLI"},"6884628":{"id":6884628,"externalId":"511145.b4105","name":"phnD","hasFamily":true,"annotation":"phosphonate/organophosphate ester transporter subunit; Phosphonate binding protein that is part of the phosphonate uptake system. Exhibits high affinity for 2- aminoethylphosphonate, and somewhat less affinity to ethylphosphonate, methylphosphonate, phosphonoacetate and phenylphosphonate","uniprotId":"PHND_ECOLI"},"6884629":{"id":6884629,"externalId":"511145.b4106","name":"phnC","hasFamily":true,"annotation":"phosphonate ABC transporter ATP-binding subunit; Part of the ABC transporter complex PhnCDE involved in phosphonates, phosphate esters, phosphite and phosphate import. Responsible for energy coupling to the transport system","uniprotId":"PHNC_ECOLI"},"6884630":{"id":6884630,"externalId":"511145.b4107","name":"phnB","hasFamily":true,"annotation":"conserved protein","uniprotId":"PHNB_ECOLI"},"6884631":{"id":6884631,"externalId":"511145.b4108","name":"phnA","hasFamily":true,"annotation":"conserved protein","uniprotId":"PHNA_ECOLI"},"6884632":{"id":6884632,"externalId":"511145.b4109","name":"yjdA","hasFamily":true,"annotation":"mutational suppressor of yhjH motility mutation, function unknown; Belongs to an operon involved in alkylphosphonate uptake and C-P lyase. Exact function not known","uniprotId":"YJDA_ECOLI"},"6884633":{"id":6884633,"externalId":"511145.b4110","name":"yjcZ","hasFamily":false,"annotation":"mutational suppressor of yhjH motility mutation, function unknown","uniprotId":"YJCZ_ECOLI"},"6884634":{"id":6884634,"externalId":"511145.b4111","name":"proP","hasFamily":true,"annotation":"proline/glycine betaine transporter; Proton symporter that senses osmotic shifts and responds by importing osmolytes such as proline, glycine betaine, stachydrine, pipecolic acid, ectoine and taurine. It is both an osmosensor and an osmoregulator which is available to participate early in the bacterial osmoregulatory response","uniprotId":"PROP_ECOLI"},"6884635":{"id":6884635,"externalId":"511145.b4112","name":"basS","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with BasR; Member of the two-component regulatory system BasS/BasR Autophosphorylates and activates BasR by phosphorylation","uniprotId":"BASS_ECOLI"},"6884636":{"id":6884636,"externalId":"511145.b4113","name":"basR","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with BasS; Member of the two-component regulatory system BasS/BasR. BasR induces the transcription of the ugd, ais, arnBCADTEF and eptA-basRS loci, all involved in resistance to polymyxin (By similarity)","uniprotId":"BASR_ECOLI"},"6884637":{"id":6884637,"externalId":"511145.b4114","name":"eptA","hasFamily":true,"annotation":"predicted metal dependent hydrolase; Catalyzes the addition of a phosphoethanolamine moiety to the lipid A. The phosphoethanolamine modification is required for resistance to polymyxin (By similarity)","uniprotId":"EPTA_ECOLI"},"6884638":{"id":6884638,"externalId":"511145.b4115","name":"adiC","hasFamily":true,"annotation":"arginine:agmatin; Major component of the acid-resistance (AR) system allowing enteric pathogens to survive the acidic environment in the stomach (By similarity). Exchanges extracellular arginine for its intracellular decarboxylation product agmatine (Agm) thereby expelling intracellular protons","uniprotId":"ADIC_ECOLI"},"6884639":{"id":6884639,"externalId":"511145.b4116","name":"adiY","hasFamily":true,"annotation":"DNA-binding transcriptional activator","uniprotId":"ADIY_ECOLI"},"6884640":{"id":6884640,"externalId":"511145.b4117","name":"adiA","hasFamily":true,"annotation":"arginine decarboxylase; ADC can be found in two forms: biodegradative and biosynthetic. The biodegradative form may play a role in regulating pH by consuming proteins","uniprotId":"ADIA_ECOLI"},"6884641":{"id":6884641,"externalId":"511145.b4118","name":"melR","hasFamily":true,"annotation":"DNA-binding transcriptional dual regulator; Transcription activator for the expression of the melAB operon. MelR binds at two sites located upstream of the melAB transcription site","uniprotId":"MELR_ECOLI"},"6884642":{"id":6884642,"externalId":"511145.b4119","name":"melA","hasFamily":true,"annotation":"alpha-galactosidase, NAD(P)-binding","uniprotId":"AGAL_ECOLI"},"6884643":{"id":6884643,"externalId":"511145.b4120","name":"melB","hasFamily":true,"annotation":"melibiose:sodium symporter; Responsible for melibiose and other galactoside transport. It is capable of using hydrogen, sodium, and lithium cations as coupling cations for cotransport, depending on the particular sugar transported (symport system)","uniprotId":"MELB_ECOLI"},"6884644":{"id":6884644,"externalId":"511145.b4121","name":"yjdF","hasFamily":true,"annotation":"conserved inner membrane protein","uniprotId":"YJDF_ECOLI"},"6884645":{"id":6884645,"externalId":"511145.b4122","name":"fumB","hasFamily":true,"annotation":"anaerobic class I fumarate hydratase (fumarase B); It functions in the generation of fumarate for use as an anaerobic electron acceptor","uniprotId":"FUMB_ECOLI"},"6884646":{"id":6884646,"externalId":"511145.b4123","name":"dcuB","hasFamily":true,"annotation":"C4-dicarboxylate transporter, anaerobic; DcuS co-sensor; Responsible for the transport of C4-dicarboxylates from the periplasm across the inner membrane","uniprotId":"DCUB_ECOLI"},"6884647":{"id":6884647,"externalId":"511145.b4124","name":"dcuR","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with DcuS; Member of the two-component regulatory system DcuR/DcuS. Involved in the C4-dicarboxylate-stimulated regulation of the genes encoding the anaerobic fumarate respiratory system (frdABCD; nuoAN; dcuB; dcuC; sdhCDAB; etc.). Weakly regulates the aerobic C4-dicarboxylate transporter dctA","uniprotId":"DCUR_ECOLI"},"6884648":{"id":6884648,"externalId":"511145.b4125","name":"dcuS","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with DcuR, regulator of anaerobic fumarate respiration; Member of the two-component regulatory system DcuR/DcuS. Involved in the C4-dicarboxylate-stimulated regulation of the genes encoding the anaerobic fumarate respiratory system (frdABCD; nuoAN; dcuB; dcuC; sdhCDAB; etc.). Weakly regulates the aerobic C4-dicarboxylate transporter dctA. Activates DcuR by phosphorylation","uniprotId":"DCUS_ECOLI"},"6884649":{"id":6884649,"externalId":"511145.b4126","name":"yjdI","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJDI_ECOLI"},"6884650":{"id":6884650,"externalId":"511145.b4127","name":"yjdJ","hasFamily":true,"annotation":"putative acyltransferase with acyl-CoA N-acyltransferase domain","uniprotId":"YJDJ_ECOLI"},"6884651":{"id":6884651,"externalId":"511145.b4128","name":"yjdK","hasFamily":false,"annotation":"predicted protein","uniprotId":"YJDK_ECOLI"},"6884652":{"id":6884652,"externalId":"511145.b4129","name":"lysU","hasFamily":true,"annotation":"lysine tRNA synthetase, inducible; Also can synthesize a number of adenyl dinucleotides (in particular AppppA). These dinucleotides have been proposed to act as modulators of the heat-shock response and stress response","uniprotId":"SYK2_ECOLI"},"6884653":{"id":6884653,"externalId":"511145.b4130","name":"yjdL","hasFamily":true,"annotation":"dipeptide and tripeptide permease; Probable proton-dependent permease that transports di- and tripeptides. Shows significantly higher specificity towards dipeptides","uniprotId":"YJDL_ECOLI"},"6884654":{"id":6884654,"externalId":"511145.b4131","name":"cadA","hasFamily":true,"annotation":"lysine decarboxylase, acid-inducible; Plays a role in pH homeostasis by consuming protons and neutralizing the acidic by-products of carbohydrate fermentation","uniprotId":"LDCI_ECOLI"},"6884655":{"id":6884655,"externalId":"511145.b4132","name":"cadB","hasFamily":true,"annotation":"putative lysine/cadaverine transporter; Probable cadaverine/lysine antiporter or part of it","uniprotId":"CADB_ECOLI"},"6884656":{"id":6884656,"externalId":"511145.b4133","name":"cadC","hasFamily":true,"annotation":"DNA-binding transcriptional activator; Required for Pcad induction, a promoter upstream of cadBA that is responsible for the pH-regulated expression of CadA and CadB. Probably acts as an activating transcription factor","uniprotId":"CADC_ECOLI"},"6884657":{"id":6884657,"externalId":"511145.b4135","name":"yjdC","hasFamily":true,"annotation":"putative transcriptional regulator","uniprotId":"YJDC_ECOLI"},"6884658":{"id":6884658,"externalId":"511145.b4136","name":"dipZ","hasFamily":true,"annotation":"fused thiol:disulfide interchange protein: activator of DsbC/conserved protein; Required to facilitate the formation of correct disulfide bonds in some periplasmic proteins and for the assembly of the periplasmic c-type cytochromes. Acts by transferring electrons from cytoplasmic thioredoxin to the periplasm, thereby maintaining the active site of DsbC, DsbE and DsbG in a reduced state. This transfer involves a cascade of disulfide bond formation and reduction steps","uniprotId":"DSBD_ECOLI"},"6884659":{"id":6884659,"externalId":"511145.b4137","name":"cutA","hasFamily":true,"annotation":"divalent-cation tolerance protein, copper sensitivity; Involved in resistance toward heavy metals (By similarity)","uniprotId":"CUTA_ECOLI"},"6884660":{"id":6884660,"externalId":"511145.b4138","name":"dcuA","hasFamily":true,"annotation":"C4-dicarboxylate antiporter; Responsible for the transport of C4-dicarboxylates from the periplasm across the inner membrane","uniprotId":"DCUA_ECOLI"},"6884661":{"id":6884661,"externalId":"511145.b4139","name":"aspA","hasFamily":true,"annotation":"aspartate ammonia-lyase","uniprotId":"ASPA_ECOLI"},"6884662":{"id":6884662,"externalId":"511145.b4140","name":"fxsA","hasFamily":true,"annotation":"suppressor of F exclusion of phage T7; Overexpression alleviates the exclusion of phage T7 in cells harboring the F plasmid","uniprotId":"FXSA_ECOLI"},"6884663":{"id":6884663,"externalId":"511145.b4141","name":"yjeH","hasFamily":true,"annotation":"putative transporter","uniprotId":"YJEH_ECOLI"},"6884664":{"id":6884664,"externalId":"511145.b4142","name":"groS","hasFamily":true,"annotation":"Cpn10 chaperonin GroES, small subunit of GroESL; Binds to Cpn60 in the presence of Mg-ATP and suppresses the ATPase activity of the latter (By similarity)","uniprotId":"CH10_ECOLI"},"6884665":{"id":6884665,"externalId":"511145.b4143","name":"groL","hasFamily":true,"annotation":"Cpn60 chaperonin GroEL, large subunit of GroESL; Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions (By similarity)","uniprotId":"CH60_ECOLI"},"6884666":{"id":6884666,"externalId":"511145.b4144","name":"yjeI","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJEI_ECOLI"},"6884667":{"id":6884667,"externalId":"511145.b4145","name":"yjeJ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YJEJ_ECOLI"},"6884668":{"id":6884668,"externalId":"511145.b4146","name":"epmB","hasFamily":true,"annotation":"EF-P-Lys34 lysylation protein; With EpmA is involved in the beta-lysylation step of the post-translational modification of translation elongation factor P (EF-P) on 'Lys-34'. EpmB appears to act before EpmA. Displays lysine 2,3-aminomutase activity, producing (R)-beta-lysine from (S)-alpha-lysine (L-lysine). Cannot use (S)-ornithine or (R)- alpha-lysine as a substrate","uniprotId":"EPMB_ECOLI"},"6884669":{"id":6884669,"externalId":"511145.b4147","name":"efp","hasFamily":true,"annotation":"elongation factor EF-P; Involved in peptide bond synthesis. Alleviates ribosome stalling that occurs when 3 or more consecutive Pro residues or the sequence PPG is present in a protein, possibly by augmenting the peptidyl transferase activity of the ribosome. Beta-lysylation at Lys-34 is required for alleviation. The Pro codons and their context do not affect activity; only consecutive Pro residues (not another amino acid) are affected by EF-P. Has stimulatory effects on peptide bond formation between ribosome-bound initiator tRNA(fMet) and puromycin, and N-acetyl-Phe tRNA(Phe)-primed  [...] ","uniprotId":"EFP_ECOLI"},"6884670":{"id":6884670,"externalId":"511145.b4148","name":"sugE","hasFamily":true,"annotation":"multidrug efflux system protein; Quaternary ammonium compound efflux pump. Confers resistance to cetylpyridinium, cetyldimethylethyl ammonium and cetrimide cations","uniprotId":"SUGE_ECOLI"},"6884671":{"id":6884671,"externalId":"511145.b4149","name":"blc","hasFamily":true,"annotation":"outer membrane lipoprotein (lipocalin), cell division and growth function; Involved in the storage or transport of lipids necessary for membrane maintenance under stressful conditions. Displays a binding preference for lysophospholipids","uniprotId":"BLC_ECOLI"},"6884672":{"id":6884672,"externalId":"511145.b4150","name":"ampC","hasFamily":true,"annotation":"beta-lactamase/D-alanine carboxypeptidase; This protein is a serine beta-lactamase with a substrate specificity for cephalosporins","uniprotId":"AMPC_ECOLI"},"6884673":{"id":6884673,"externalId":"511145.b4151","name":"frdD","hasFamily":true,"annotation":"fumarate reductase (anaerobic), membrane anchor subunit; Seems to be involved in the anchoring of the catalytic components of the fumarate reductase complex to the cytoplasmic membrane (By similarity)","uniprotId":"FRDD_ECOLI"},"6884674":{"id":6884674,"externalId":"511145.b4152","name":"frdC","hasFamily":true,"annotation":"fumarate reductase (anaerobic), membrane anchor subunit; Seems to be involved in the anchoring of the catalytic components of the fumarate reductase complex to the cytoplasmic membrane (By similarity)","uniprotId":"FRDC_ECOLI"},"6884675":{"id":6884675,"externalId":"511145.b4153","name":"frdB","hasFamily":true,"annotation":"fumarate reductase (anaerobic), Fe-S subunit; Two distinct, membrane-bound, FAD-containing enzymes are responsible for the catalysis of fumarate and succinate interconversion; the fumarate reductase is used in anaerobic growth, and the succinate dehydrogenase is used in aerobic growth","uniprotId":"FRDB_ECOLI"},"6884676":{"id":6884676,"externalId":"511145.b4154","name":"frdA","hasFamily":true,"annotation":"fumarate reductase (anaerobic) catalytic and NAD/flavoprotein subunit; Two distinct, membrane-bound, FAD-containing enzymes are responsible for the catalysis of fumarate and succinate interconversion; the fumarate reductase is used in anaerobic growth, and the succinate dehydrogenase is used in aerobic growth","uniprotId":"FRDA_ECOLI"},"6884677":{"id":6884677,"externalId":"511145.b4155","name":"epmA","hasFamily":true,"annotation":"EF-P-lysine34-lysine ligase; With EpmB is involved in the beta-lysylation step of the post-translational modification of translation elongation factor P (EF-P) on 'Lys-34'. Catalyzes the ATP-dependent activation of (R)- beta-lysine produced by EpmB, forming a lysyl-adenylate, from which the beta-lysyl moiety is then transferred to the epsilon- amino group of EF-P 'Lys-34'. (R)-beta-lysine is 100-fold more efficient as a substrate than either (S)-beta-lysine or L-alpha- lysine. Cannot ligate lysine to any tRNA","uniprotId":"EPMA_ECOLI"},"6884678":{"id":6884678,"externalId":"511145.b4156","name":"yjeM","hasFamily":true,"annotation":"putative transporter","uniprotId":"YJEM_ECOLI"},"6884679":{"id":6884679,"externalId":"511145.b4157","name":"yjeN","hasFamily":false,"annotation":"predicted protein","uniprotId":"YJEN_ECOLI"},"6884680":{"id":6884680,"externalId":"511145.b4158","name":"yjeO","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"YJEO_ECOLI"},"6884681":{"id":6884681,"externalId":"511145.b4159","name":"yjeP","hasFamily":true,"annotation":"predicted mechanosensitive channel","uniprotId":"YJEP_ECOLI"},"6884682":{"id":6884682,"externalId":"511145.b4160","name":"psd","hasFamily":true,"annotation":"phosphatidylserine decarboxylase","uniprotId":"PSD_ECOLI"},"6884683":{"id":6884683,"externalId":"511145.b4161","name":"rsgA","hasFamily":true,"annotation":"ribosome small subunit-dependent GTPase A; May play a role in 30S ribosomal subunit biogenesis. Unusual circulary permuted GTPase that catalyzes rapid hydrolysis of GTP with a slow catalytic turnover. Dispensible for viability, but important for overall fitness. The intrinsic GTPase activity is stimulated by the presence of 30S (160-fold increase in kcat) or 70S (96 fold increase in kcat) ribosomes (PubMed:14973029). The GTPase is inhibited by aminoglycoside antibiotics such as neomycin and paromycin (PubMed:15466596) streptomycin and spectinomycin (PubMed:15828870). This inhibition is [...] ","uniprotId":"RSGA_ECOLI"},"6884684":{"id":6884684,"externalId":"511145.b4162","name":"orn","hasFamily":true,"annotation":"oligoribonuclease; 3'-to-5' exoribonuclease specific for small oligoribonucleotides 2 to 5 nucleotides in length, as well as small (2 to 5 nucleotides) ssDNA oligomers. Probably responsible for the final step in mRNA degradation","uniprotId":"ORN_ECOLI"},"6884685":{"id":6884685,"externalId":"511145.b4166","name":"queG","hasFamily":true,"annotation":"epoxyqueuosine reductase, cobalamine-stimulated; Catalyzes the conversion of epoxyqueuosine (oQ) to queuosine (Q), which is a hypermodified base found in the wobble positions of tRNA(Asp), tRNA(Asn), tRNA(His) and tRNA(Tyr) (By similarity)","uniprotId":"QUEG_ECOLI"},"6884686":{"id":6884686,"externalId":"511145.b4167","name":"nnr","hasFamily":true,"annotation":"predicted carbohydrate kinase; Bifunctional enzyme that catalyzes the epimerization of the S- and R-forms of NAD(P)HX and the dehydration of the S-form of NAD(P)HX at the expense of ADP, which is converted to AMP. This allows the repair of both epimers of NAD(P)HX, a damaged form of NAD(P)H that is a result of enzymatic or heat-dependent hydration","uniprotId":"NNR_ECOLI"},"6884687":{"id":6884687,"externalId":"511145.b4168","name":"yjeE","hasFamily":true,"annotation":"ADP-binding protein essential for nucleoid integrity","uniprotId":"YJEE_ECOLI"},"6884688":{"id":6884688,"externalId":"511145.b4169","name":"amiB","hasFamily":true,"annotation":"N-acetylmuramoyl-l-alanine amidase II; Cell-wall hydrolase involved in septum cleavage during cell division. Can also act as powerful autolysin in the presence of murein synthesis inhibitors","uniprotId":"AMIB_ECOLI"},"6884689":{"id":6884689,"externalId":"511145.b4170","name":"mutL","hasFamily":true,"annotation":"methyl-directed mismatch repair protein; This protein is involved in the repair of mismatches in DNA. It is required for dam-dependent methyl-directed DNA mismatch repair. May act as a \"molecular matchmaker\", a protein that promotes the formation of a stable complex between two or more DNA-binding proteins in an ATP-dependent manner without itself being part of the final effector complex. The ATPase activity of MutL is stimulated by DNA","uniprotId":"MUTL_ECOLI"},"6884690":{"id":6884690,"externalId":"511145.b4171","name":"miaA","hasFamily":true,"annotation":"delta(2)-isopentenylpyrophosphate tRNA-adenosine transferase; Catalyzes the transfer of a dimethylallyl group onto the adenine at position 37 in tRNAs that read codons beginning with uridine, leading to the formation of N6-(dimethylallyl)adenosine (i(6)A) (By similarity)","uniprotId":"MIAA_ECOLI"},"6884691":{"id":6884691,"externalId":"511145.b4172","name":"hfq","hasFamily":true,"annotation":"HF-I, host factor for RNA phage Q beta replication; RNA chaperone that binds small regulatory RNA (sRNAs) and mRNAs to facilitate mRNA translational regulation in response to envelope stress, environmental stress and changes in metabolite concentrations. Involved in the regulation of stress responses mediated by the sigma factors RpoS, sigma-E and sigma-32. Binds with high specificity to tRNAs. In vitro, stimulates synthesis of long tails by poly(A) polymerase I. Required for RNA phage Qbeta replication","uniprotId":"HFQ_ECOLI"},"6884692":{"id":6884692,"externalId":"511145.b4173","name":"hflX","hasFamily":true,"annotation":"GTPase, stimulated by 50S subunit binding; GTPase that associates with the 50S ribosomal subunit and may have a role during protein synthesis or ribosome biogenesis. In vitro, also exhibits ATPase activity","uniprotId":"HFLX_ECOLI"},"6884693":{"id":6884693,"externalId":"511145.b4174","name":"hflK","hasFamily":true,"annotation":"modulator for HflB protease specific for phage lambda cII repressor; HflC and HflK help govern the stability of phage lambda cII protein, and thereby control the lysogenization frequency of phage lambda. HflKC inhibits the SecY-degrading activity of FtsH, possibly helping quality control of integral membrane proteins","uniprotId":"HFLK_ECOLI"},"6884694":{"id":6884694,"externalId":"511145.b4175","name":"hflC","hasFamily":true,"annotation":"modulator for HflB protease specific for phage lambda cII repressor; HflC and HflK help govern the stability of phage lambda cII protein, and thereby control the lysogenization frequency of phage lambda. HflKC inhibits the SecY-degrading activity of FtsH, possibly helping quality control of integral membrane proteins","uniprotId":"HFLC_ECOLI"},"6884695":{"id":6884695,"externalId":"511145.b4176","name":"yjeT","hasFamily":true,"annotation":"conserved protein, DUF2065 family","uniprotId":"YJET_ECOLI"},"6884696":{"id":6884696,"externalId":"511145.b4177","name":"purA","hasFamily":true,"annotation":"adenylosuccinate synthetase; Plays an important role in the de novo pathway of purine nucleotide biosynthesis. Catalyzes the first committed step in the biosynthesis of AMP from IMP (By similarity)","uniprotId":"PURA_ECOLI"},"6884697":{"id":6884697,"externalId":"511145.b4178","name":"nsrR","hasFamily":true,"annotation":"nitric oxide-sensitive repressor for NO regulon; Nitric oxide-sensitive repressor of genes involved in protecting the cell against nitrosative stress, such as ytfE, hmpA and ygbA. May require iron for activity. Does not regulates its own transcription","uniprotId":"NSRR_ECOLI"},"6884698":{"id":6884698,"externalId":"511145.b4179","name":"rnr","hasFamily":true,"annotation":"exoribonuclease R, RNase R; 3'-5' exoribonuclease that releases 5'-nucleoside monophosphates and is involved in maturation of structured RNAs (rRNAs, tRNAs and SsrA/tmRNA). In stationary phase, involved in the post-transcriptional regulation of ompA mRNA stability. Shortens RNA processively to di- and trinucleotides. In vitro, exhibits helicase activity, which is independent of its RNase activity. Required for the expression of virulence genes in enteroinvasive strains of E.coli","uniprotId":"RNR_ECOLI"},"6884699":{"id":6884699,"externalId":"511145.b4180","name":"rlmB","hasFamily":true,"annotation":"23S rRNA (Gm2251)-methyltransferase; Specifically methylates the ribose of guanosine 2251 in 23S rRNA (By similarity)","uniprotId":"RLMB_ECOLI"},"6884700":{"id":6884700,"externalId":"511145.b4181","name":"yjfI","hasFamily":true,"annotation":"conserved protein, DUF2170 family","uniprotId":"YJFI_ECOLI"},"6884701":{"id":6884701,"externalId":"511145.b4182","name":"yjfJ","hasFamily":true,"annotation":"conserved protein, PspA/IM30 family","uniprotId":"YJFJ_ECOLI"},"6884702":{"id":6884702,"externalId":"511145.b4183","name":"yjfK","hasFamily":true,"annotation":"conserved protein, DUF2491 family","uniprotId":"YJFK_ECOLI"},"6884703":{"id":6884703,"externalId":"511145.b4184","name":"yjfL","hasFamily":true,"annotation":"inner membrane protein, UPF0719 family","uniprotId":"YJFL_ECOLI"},"6884704":{"id":6884704,"externalId":"511145.b4185","name":"yjfM","hasFamily":true,"annotation":"conserved protein, DUF1190 family","uniprotId":"YJFM_ECOLI"},"6884705":{"id":6884705,"externalId":"511145.b4186","name":"yjfC","hasFamily":true,"annotation":"glutathionylspermidine synthase homolog","uniprotId":"YJFC_ECOLI"},"6884706":{"id":6884706,"externalId":"511145.b4187","name":"aidB","hasFamily":true,"annotation":"isovaleryl CoA dehydrogenase; Part of the adaptive DNA-repair response to alkylating agents. Could prevent alkylation damage by protecting DNA and destroying alkylating agents that have yet to reach their DNA target. Binds to double-stranded DNA with a preference for a DNA region that includes its own promoter. Shows weak isovaleryl-CoA dehydrogenase activity in vitro","uniprotId":"AIDB_ECOLI"},"6884707":{"id":6884707,"externalId":"511145.b4188","name":"yjfN","hasFamily":true,"annotation":"predicted protein","uniprotId":"YJFN_ECOLI"},"6884708":{"id":6884708,"externalId":"511145.b4189","name":"bsmA","hasFamily":true,"annotation":"bioflm peroxide resistance protein; Involved in protection of biofilms against oxidative stress","uniprotId":"BSMA_ECOLI"},"6884709":{"id":6884709,"externalId":"511145.b4190","name":"yjfP","hasFamily":true,"annotation":"acyl CoA esterase; Displays esterase activity toward palmitoyl-CoA and pNP- butyrate","uniprotId":"YJFP_ECOLI"},"6884710":{"id":6884710,"externalId":"511145.b4191","name":"ulaR","hasFamily":true,"annotation":"transcriptional tepressor for the L-ascorbate utilization (ula) divergon; Represses ulaG and the ulaABCDEF operon. Two ulaR binding sites have been identified in each promoter. Full activity requires simultaneous interaction of UlaR with both divergent promoters and seems to be dependent on repressor-mediated DNA loop formation, which is helped by the action of integration host factor","uniprotId":"ULAR_ECOLI"},"6884711":{"id":6884711,"externalId":"511145.b4192","name":"ulaG","hasFamily":true,"annotation":"L-ascorbate 6-phosphate lactonase; Probably catalyzes the hydrolysis of L-ascorbate-6-P into 3-keto-L-gulonate-6-P. Is essential for L-ascorbate utilization under anaerobic conditions. Also shows phosphodiesterase activity, hydrolyzing phosphodiester bond in the artificial chromogenic substrate bis-p-nitrophenyl phosphate (bis- pNPP)","uniprotId":"ULAG_ECOLI"},"6884712":{"id":6884712,"externalId":"511145.b4193","name":"sgaT","hasFamily":true,"annotation":"L-ascorbate-specific enzyme IIC component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active- transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane. This system is involved in ascorbate transport","uniprotId":"ULAA_ECOLI"},"6884713":{"id":6884713,"externalId":"511145.b4194","name":"ulaB","hasFamily":true,"annotation":"L-ascorbate-specific enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in ascorbate transport","uniprotId":"ULAB_ECOLI"},"6884714":{"id":6884714,"externalId":"511145.b4195","name":"ulaC","hasFamily":true,"annotation":"L-ascorbate-specific enzyme IIA component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in ascorbate transport","uniprotId":"ULAC_ECOLI"},"6884715":{"id":6884715,"externalId":"511145.b4196","name":"sgaH","hasFamily":false,"annotation":"3-keto-L-gulonate 6-phosphate decarboxylase; Catalyzes the decarboxylation of 3-keto-L-gulonate-6-P into L-xylulose-5-P. Is involved in the anaerobic L-ascorbate utilization (By similarity)","uniprotId":"ULAD_ECOLI"},"6884716":{"id":6884716,"externalId":"511145.b4197","name":"sgaU","hasFamily":true,"annotation":"L-xylulose 5-phosphate 3-epimerase; Catalyzes the isomerization of L-xylulose-5-phosphate to L-ribulose-5-phosphate. Is involved in the anaerobic L-ascorbate utilization (By similarity)","uniprotId":"ULAE_ECOLI"},"6884717":{"id":6884717,"externalId":"511145.b4198","name":"sgaE","hasFamily":true,"annotation":"L-ribulose 5-phosphate 4-epimerase; Catalyzes the isomerization of L-ribulose 5-phosphate to D-xylulose 5-phosphate. Is involved in the anaerobic L-ascorbate utilization (By similarity)","uniprotId":"ULAF_ECOLI"},"6884718":{"id":6884718,"externalId":"511145.b4199","name":"yjfY","hasFamily":true,"annotation":"predicted protein","uniprotId":"YJFY_ECOLI"},"6884719":{"id":6884719,"externalId":"511145.b4200","name":"rpsF","hasFamily":true,"annotation":"30S ribosomal subunit protein S6; Binds together with S18 to 16S ribosomal RNA (By similarity)","uniprotId":"RS6_ECOLI"},"6884720":{"id":6884720,"externalId":"511145.b4201","name":"priB","hasFamily":true,"annotation":"primosomal protein N; Binds single-stranded DNA at the primosome assembly site (PAS). During primosome assembly it facilitates the complex formation between PriA and DnaT (By similarity)","uniprotId":"PRIB_ECOLI"},"6884721":{"id":6884721,"externalId":"511145.b4202","name":"rpsR","hasFamily":true,"annotation":"30S ribosomal subunit protein S18; Binds as a heterodimer with protein S6 to the central domain of the 16S rRNA, where it helps stabilize the platform of the 30S subunit (By similarity)","uniprotId":"RS18_ECOLI"},"6884722":{"id":6884722,"externalId":"511145.b4203","name":"rplI","hasFamily":true,"annotation":"50S ribosomal subunit protein L9; One of the primary rRNA binding proteins, it binds very close to the 3' end of the 23S rRNA","uniprotId":"RL9_ECOLI"},"6884723":{"id":6884723,"externalId":"511145.b4204","name":"yjfZ","hasFamily":false,"annotation":"predicted protein","uniprotId":"YJFZ_ECOLI"},"6884724":{"id":6884724,"externalId":"511145.b4206","name":"ytfB","hasFamily":true,"annotation":"putative cell envelope opacity-associated protein","uniprotId":"YTFB_ECOLI"},"6884725":{"id":6884725,"externalId":"511145.b4207","name":"fklB","hasFamily":true,"annotation":"FKBP-type peptidyl-prolyl cis-trans isomerase (rotamase); PPIases accelerate the folding of proteins. It catalyzes the cis-trans isomerization of proline imidic peptide bonds in oligopeptides","uniprotId":"FKBB_ECOLI"},"6884726":{"id":6884726,"externalId":"511145.b4208","name":"cycA","hasFamily":true,"annotation":"D-alanine/D-serine/glycine transporter; Permease that is involved in the transport across the cytoplasmic membrane of D-alanine, D-serine and glycine","uniprotId":"CYCA_ECOLI"},"6884727":{"id":6884727,"externalId":"511145.b4209","name":"ytfE","hasFamily":true,"annotation":"iron-sulfur cluster repair protein RIC; Di-iron-containing protein involved in the repair of iron-sulfur clusters damaged by oxidative and nitrosative stress conditions (By similarity)","uniprotId":"YTFE_ECOLI"},"6884728":{"id":6884728,"externalId":"511145.b4210","name":"ytfF","hasFamily":true,"annotation":"inner membrane protein, DMT transporter family","uniprotId":"YTFF_ECOLI"},"6884729":{"id":6884729,"externalId":"511145.b4211","name":"qorB","hasFamily":true,"annotation":"NAD(P)H:quinone oxidoreductase; Quinone oxidoreductase that may play some additional role beyond quinone reduction. Potential redox sensor protein. Overexpression induces retardation of growth","uniprotId":"QOR2_ECOLI"},"6884730":{"id":6884730,"externalId":"511145.b4212","name":"ytfH","hasFamily":true,"annotation":"putative transcriptional regulator, HxlR-type, DUF24 family","uniprotId":"YTFH_ECOLI"},"6884731":{"id":6884731,"externalId":"511145.b4213","name":"cpdB","hasFamily":true,"annotation":"2':3'-cyclic-nucleotide 2'-phosphodiesterase; This bifunctional enzyme catalyzes two consecutive reactions during ribonucleic acid degradation. Converts a 2',3'- cyclic nucleotide to a 3'-nucleotide and then the 3'-nucleotide to the corresponding nucleoside and phosphate","uniprotId":"CPDB_ECOLI"},"6884732":{"id":6884732,"externalId":"511145.b4214","name":"cysQ","hasFamily":true,"annotation":"PAPS (adenosine 3'-phosphate 5'-phosphosulfate) 3'(2'),5'-bisphosphate nucleotidase; Converts 3'(2')-phosphoadenosine 5'-phosphate (PAP) to AMP. May also convert adenosine 3'-phosphate 5'-phosphosulfate (PAPS) to adenosine 5'-phosphosulfate (APS). Has 10000-fold lower activity towards inositol 1,4-bisphosphate (Ins(1,4)P2)","uniprotId":"CYSQ_ECOLI"},"6884733":{"id":6884733,"externalId":"511145.b4215","name":"ytfI","hasFamily":false,"annotation":"predicted protein","uniprotId":"YTFI_ECOLI"},"6884734":{"id":6884734,"externalId":"511145.b4216","name":"ecfJ","hasFamily":true,"annotation":"putative transcriptional regulator","uniprotId":"YTFJ_ECOLI"},"6884735":{"id":6884735,"externalId":"511145.b4217","name":"ytfK","hasFamily":true,"annotation":"conserved protein, DUF1107 family","uniprotId":"YTFK_ECOLI"},"6884736":{"id":6884736,"externalId":"511145.b4218","name":"ytfL","hasFamily":true,"annotation":"inner membrane protein, UPF0053 family","uniprotId":"YTFL_ECOLI"},"6884737":{"id":6884737,"externalId":"511145.b4219","name":"msrA","hasFamily":true,"annotation":"methionine sulfoxide reductase A; Has an important function as a repair enzyme for proteins that have been inactivated by oxidation. Catalyzes the reversible oxidation-reduction of methionine sulfoxide in proteins to methionine (By similarity)","uniprotId":"MSRA_ECOLI"},"6884738":{"id":6884738,"externalId":"511145.b4220","name":"tamA","hasFamily":true,"annotation":"predicted outer membrane protein and surface antigen; Part of the translocation and assembly module (TAM) autotransporter assembly complex, which functions in translocation of autotransporters across the outer membrane. Has anion selective channel-forming ability, but the physiological relevance of this activity is unclear","uniprotId":"TAMA_ECOLI"},"6884739":{"id":6884739,"externalId":"511145.b4221","name":"tamB","hasFamily":true,"annotation":"large conserved protein, DUF490 family; Part of the translocation and assembly module (TAM) autotransporter assembly complex, which functions in translocation of autotransporters across the outer membrane","uniprotId":"TAMB_ECOLI"},"6884740":{"id":6884740,"externalId":"511145.b4222","name":"ytfP","hasFamily":true,"annotation":"GGCT-like protein; May play a role in antibiotic biosynthesis (Potential)","uniprotId":"YTFP_ECOLI"},"6884741":{"id":6884741,"externalId":"511145.b4224","name":"chpS","hasFamily":false,"annotation":"antitoxin of the ChpBS toxin-antitoxin system; Antitoxin component of a toxin-antitoxin (TA) module. May be involved in the regulation of cell growth. It acts as a suppressor of the inhibitory function of ChpB protein. Both ChpS and ChpB probably bind to the promoter region of the chpSB operon to autoregulate their synthesis","uniprotId":"CHPS_ECOLI"},"6884742":{"id":6884742,"externalId":"511145.b4225","name":"chpB","hasFamily":true,"annotation":"toxin of the ChpB-ChpS toxin-antitoxin system; Toxic component of a toxin-antitoxin (TA) module. ChpB is a sequence-specific mRNA and (weak) tmRNA endoribonuclease that inhibits protein synthesis and induces bacterial stasis. Cleavage is independent of the ribosome. Cleavage occurs at ACY sequences where Y is not C. The endoribonuclease activity is not as strong as that of MazF. The endoribonuclease activity (a toxin) is inhibited by the labile cognate antitoxin ChpS. Toxicity results when the levels of ChpS decrease in the cell, leading to mRNA degradation. Both ChpS and ChpB probably [...] ","uniprotId":"CHPB_ECOLI"},"6884743":{"id":6884743,"externalId":"511145.b4226","name":"ppa","hasFamily":true,"annotation":"inorganic pyrophosphatase","uniprotId":"IPYR_ECOLI"},"6884744":{"id":6884744,"externalId":"511145.b4227","name":"ytfQ","hasFamily":true,"annotation":"galactofuranose binding proteint: periplasmic-binding component of ABC superfamily","uniprotId":"YTFQ_ECOLI"},"6884745":{"id":6884745,"externalId":"511145.b4230","name":"ytfT","hasFamily":true,"annotation":"putative sugar transporter subunit: membrane component of ABC superfamily; Probably part of a binding-protein-dependent transport system. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"YTFT_ECOLI"},"6884746":{"id":6884746,"externalId":"511145.b4231","name":"yjfF","hasFamily":true,"annotation":"putative sugar transporter subunit: membrane component of ABC superfamily; Probably part of a binding-protein-dependent transport system. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"YJFF_ECOLI"},"6884747":{"id":6884747,"externalId":"511145.b4232","name":"fbp","hasFamily":true,"annotation":"fructose-1,6-bisphosphatase I","uniprotId":"F16PA_ECOLI"},"6884748":{"id":6884748,"externalId":"511145.b4233","name":"mpl","hasFamily":true,"annotation":"UDP-N-acetylmuramate:L-alanyl-gamma-D-glutamyl- meso-diaminopimelate ligase; Reutilizes the intact tripeptide L-alanyl-gamma-D- glutamyl-meso-diaminopimelate by linking it to UDP-N-acetylmuramic acid","uniprotId":"MPL_ECOLI"},"6884749":{"id":6884749,"externalId":"511145.b4234","name":"yjgA","hasFamily":true,"annotation":"conserved protein, ribosome-associated","uniprotId":"YJGA_ECOLI"},"6884750":{"id":6884750,"externalId":"511145.b4235","name":"pmbA","hasFamily":true,"annotation":"putative peptidase required for the maturation and secretion of the antibiotic peptide MccB17; May facilitate the secretion of the peptide antibiotic microcin B17 (MccB17) by completing its maturation. Suppresses the inhibitory activity of the carbon storage regulator (CsrA)","uniprotId":"PMBA_ECOLI"},"6884751":{"id":6884751,"externalId":"511145.b4237","name":"nrdG","hasFamily":true,"annotation":"anaerobic ribonucleotide reductase activating protein; Activation of anaerobic ribonucleoside-triphosphate reductase under anaerobic conditions by generation of an organic free radical, using S-adenosylmethionine and reduced flavodoxin as cosubstrates to produce 5'-deoxy-adenosine (By similarity)","uniprotId":"NRDG_ECOLI"},"6884752":{"id":6884752,"externalId":"511145.b4238","name":"nrdD","hasFamily":true,"annotation":"anaerobic ribonucleoside-triphosphate reductase","uniprotId":"NRDD_ECOLI"},"6884753":{"id":6884753,"externalId":"511145.b4239","name":"treC","hasFamily":true,"annotation":"trehalose-6-P hydrolase","uniprotId":"TREC_ECOLI"},"6884754":{"id":6884754,"externalId":"511145.b4240","name":"treB","hasFamily":true,"annotation":"fused trehalose(maltose)-specific PTS enzyme: IIB component/IIC component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane. This system is involved in trehalose transport","uniprotId":"PTTBC_ECOLI"},"6884755":{"id":6884755,"externalId":"511145.b4241","name":"treR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor of the treBC operon. Binds trehalose 6- phosphate as an inducer","uniprotId":"TRER_ECOLI"},"6884756":{"id":6884756,"externalId":"511145.b4242","name":"mgtA","hasFamily":true,"annotation":"magnesium transporter; Mediates magnesium influx to the cytosol","uniprotId":"ATMA_ECOLI"},"6884757":{"id":6884757,"externalId":"511145.b4243","name":"ridA","hasFamily":true,"annotation":"conserved protein, UPF0131 family; Accelerates the release of ammonia from reactive enamine/imine intermediates of the PLP-dependent threonine dehydratase (IlvA) in the low water environment of the cell. It catalyzes the deamination of enamine/imine intermediates to yield 2-ketobutyrate and ammonia. It is required for the detoxification of reactive intermediates of IlvA due to their highly nucleophilic abilities. Involved in the isoleucine biosynthesis (By similarity)","uniprotId":"RIDA_ECOLI"},"6884758":{"id":6884758,"externalId":"511145.b4244","name":"pyrI","hasFamily":true,"annotation":"aspartate carbamoyltransferase, regulatory subunit; Involved in allosteric regulation of aspartate carbamoyltransferase (By similarity)","uniprotId":"PYRI_ECOLI"},"6884759":{"id":6884759,"externalId":"511145.b4245","name":"pyrB","hasFamily":true,"annotation":"aspartate carbamoyltransferase, catalytic subunit","uniprotId":"PYRB_ECOLI"},"6884760":{"id":6884760,"externalId":"511145.b4246","name":"pyrL","hasFamily":true,"annotation":"pyrBI operon leader peptide","uniprotId":"LPPY_ECOLI"},"6884761":{"id":6884761,"externalId":"511145.b4248","name":"yjgH","hasFamily":true,"annotation":"conserved protein, UPF0131 family","uniprotId":"YJGH_ECOLI"},"6884762":{"id":6884762,"externalId":"511145.b4249","name":"bdcA","hasFamily":true,"annotation":"c-di-GMP-binding biofilm dispersal mediator protein; Increases biofilm dispersal. Acts by binding directly to the signaling molecule cyclic-di-GMP, which decreases the intracellular concentration of cyclic-di-GMP and leads to biofilm dispersal. Also controls other biofilm-related phenotypes such as cell motility, cell size, cell aggregation and production of extracellular DNA and extracellular polysaccharides (EPS). Does not act as a phosphodiesterase","uniprotId":"BDCA_ECOLI"},"6884763":{"id":6884763,"externalId":"511145.b4251","name":"bdcR","hasFamily":true,"annotation":"predicted transcriptional regulator; Negatively regulates expression of bdcA","uniprotId":"BDCR_ECOLI"},"6884764":{"id":6884764,"externalId":"511145.b4252","name":"tabA","hasFamily":true,"annotation":"biofilm modulator regulated by toxins; Influences biofilm formation. Represses fimbria genes in 8 hours biofilms. May act in response to the combined activity of several toxin-antitoxin (TA) systems","uniprotId":"TABA_ECOLI"},"6884765":{"id":6884765,"externalId":"511145.b4253","name":"yjgL","hasFamily":false,"annotation":"predicted protein","uniprotId":"YJGL_ECOLI"},"6884766":{"id":6884766,"externalId":"511145.b4254","name":"argI","hasFamily":true,"annotation":"ornithine carbamoyltransferase 1; Reversibly catalyzes the transfer of the carbamoyl group from carbamoyl phosphate (CP) to the N(epsilon) atom of ornithine (ORN) to produce L-citrulline, which is a substrate for argininosuccinate synthetase, the enzyme involved in the final step in arginine biosynthesis","uniprotId":"OTC1_ECOLI"},"6884767":{"id":6884767,"externalId":"511145.b4255","name":"rraB","hasFamily":true,"annotation":"protein inhibitor of RNase E; Globally modulates RNA abundance by binding to RNase E (Rne) and regulating its endonucleolytic activity. Can modulate Rne action in a substrate-dependent manner by altering the composition of the degradosome (By similarity)","uniprotId":"RRAB_ECOLI"},"6884768":{"id":6884768,"externalId":"511145.b4256","name":"yjgM","hasFamily":true,"annotation":"putative acetyltransferase","uniprotId":"YJGM_ECOLI"},"6884769":{"id":6884769,"externalId":"511145.b4257","name":"yjgN","hasFamily":true,"annotation":"inner membrane protein, DUF898 family","uniprotId":"YJGN_ECOLI"},"6884770":{"id":6884770,"externalId":"511145.b4258","name":"valS","hasFamily":true,"annotation":"valyl-tRNA synthetase; Catalyzes the attachment of valine to tRNA(Val). As ValRS can inadvertently accommodate and process structurally similar amino acids such as threonine, to avoid such errors, it has a \"posttransfer\" editing activity that hydrolyzes mischarged Thr-tRNA(Val) in a tRNA-dependent manner (By similarity)","uniprotId":"SYV_ECOLI"},"6884771":{"id":6884771,"externalId":"511145.b4259","name":"holC","hasFamily":true,"annotation":"DNA polymerase III, chi subunit; DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. This DNA polymerase also exhibits 3' to 5' exonuclease activity","uniprotId":"HOLC_ECOLI"},"6884772":{"id":6884772,"externalId":"511145.b4260","name":"pepA","hasFamily":true,"annotation":"multifunctional aminopeptidase A: a cyteinylglycinase, transcription regulator and site-specific recombination factor; Presumably involved in the processing and regular turnover of intracellular proteins. Catalyzes the removal of unsubstituted N-terminal amino acids from various peptides. Required for plasmid ColE1 site-specific recombination but not in its aminopeptidase activity. Could act as a structural component of the putative nucleoprotein complex in which the Xer recombination reaction takes place","uniprotId":"AMPA_ECOLI"},"6884773":{"id":6884773,"externalId":"511145.b4261","name":"lptF","hasFamily":true,"annotation":"lipopolysaccharide export ABC permease of the LptBFGC export complex; Part of the ABC transporter complex LptBFG involved in the translocation of lipopolysaccharide (LPS) from the inner membrane to the outer membrane","uniprotId":"LPTF_ECOLI"},"6884774":{"id":6884774,"externalId":"511145.b4262","name":"lptG","hasFamily":true,"annotation":"lipopolysaccharide export ABC permease of the LptBFGC export complex; Part of the ABC transporter complex LptBFG involved in the translocation of lipopolysaccharide (LPS) from the inner membrane to the outer membrane","uniprotId":"LPTG_ECOLI"},"6884775":{"id":6884775,"externalId":"511145.b4263","name":"yjgR","hasFamily":true,"annotation":"conserved protein, DUF853 family with NTPase fold","uniprotId":"YJGR_ECOLI"},"6884776":{"id":6884776,"externalId":"511145.b4264","name":"idnR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor, 5-gluconate-binding; Idn operon regulator. May repress gntKU and gntT genes when growing on L-idonate","uniprotId":"IDNR_ECOLI"},"6884777":{"id":6884777,"externalId":"511145.b4265","name":"idnT","hasFamily":true,"annotation":"L-idonate and D-gluconate transporter; Transports L-idonate, D-gluconate and 5-keto-D- gluconate, from the periplasm across the inner membrane","uniprotId":"IDNT_ECOLI"},"6884778":{"id":6884778,"externalId":"511145.b4266","name":"idnO","hasFamily":true,"annotation":"5-keto-D-gluconate-5-reductase; Catalyzes a reversible reduction of 5-ketoglutanate to form D-gluconate. Dependent on NADP, almost inactive with NAD","uniprotId":"IDNO_ECOLI"},"6884779":{"id":6884779,"externalId":"511145.b4267","name":"idnD","hasFamily":true,"annotation":"L-idonate 5-dehydrogenase, NAD-binding; Catalyzes the NADH/NADPH-dependent oxidation of L- idonate to 5-ketogluconate (5KG)","uniprotId":"IDND_ECOLI"},"6884780":{"id":6884780,"externalId":"511145.b4268","name":"idnK","hasFamily":true,"annotation":"D-gluconate kinase, thermosensitive","uniprotId":"IDNK_ECOLI"},"6884781":{"id":6884781,"externalId":"511145.b4269","name":"yjgB","hasFamily":true,"annotation":"putative alcohol dehydrogenase, Zn-dependent and NAD(P)-binding","uniprotId":"YJGB_ECOLI"},"6884782":{"id":6884782,"externalId":"511145.b4272","name":"insC","hasFamily":true,"annotation":"IS2 repressor TnpA"},"6884783":{"id":6884783,"externalId":"511145.b4273","name":"insD","hasFamily":true,"annotation":"IS2 transposase TnpB; Involved in the transposition of the insertion sequence IS2 (By similarity)","uniprotId":"IND10_ECOLI"},"6884784":{"id":6884784,"externalId":"511145.b4277","name":"yjgZ","hasFamily":false,"annotation":"KpLE2 phage-like element","uniprotId":"YJGZ_ECOLI"},"6884785":{"id":6884785,"externalId":"511145.b4278","name":"insG","hasFamily":true,"annotation":"IS4 transposase; Involved in the transposition of the insertion sequence IS4","uniprotId":"INSG_ECOLI"},"6884786":{"id":6884786,"externalId":"511145.b4279","name":"yjhB","hasFamily":true,"annotation":"putative transporter","uniprotId":"YJHB_ECOLI"},"6884787":{"id":6884787,"externalId":"511145.b4280","name":"yjhC","hasFamily":true,"annotation":"putative oxidoreductase","uniprotId":"YJHC_ECOLI"},"6884788":{"id":6884788,"externalId":"511145.b4284","name":"insI","hasFamily":true,"annotation":"IS30 transposase; Required for the transposition of the insertion element (Potential)","uniprotId":"INSI3_ECOLI"},"6884789":{"id":6884789,"externalId":"511145.b4287","name":"fecE","hasFamily":true,"annotation":"KpLE2 phage-like element; iron-dicitrate transporter subunit; Part of the binding-protein-dependent transport system for citrate-dependent Fe(3+). Probably responsible for energy coupling to the transport system","uniprotId":"FECE_ECOLI"},"6884790":{"id":6884790,"externalId":"511145.b4288","name":"fecD","hasFamily":true,"annotation":"KpLE2 phage-like element; iron-dicitrate transporter subunit; Part of the binding-protein-dependent transport system for citrate-dependent Fe(3+). Probably responsible for the translocation of the substrate across the membrane","uniprotId":"FECD_ECOLI"},"6884791":{"id":6884791,"externalId":"511145.b4289","name":"fecC","hasFamily":true,"annotation":"KpLE2 phage-like element; iron-dicitrate transporter subunit; Part of the binding-protein-dependent transport system for citrate-dependent Fe(3+). Probably responsible for the translocation of the substrate across the membrane","uniprotId":"FECC_ECOLI"},"6884792":{"id":6884792,"externalId":"511145.b4290","name":"fecB","hasFamily":true,"annotation":"KpLE2 phage-like element; iron-dicitrate transporter subunit; Binds citrate-dependent Fe(3+); part of the binding- protein-dependent transport system for uptake of citrate-dependent Fe(3+)","uniprotId":"FECB_ECOLI"},"6884793":{"id":6884793,"externalId":"511145.b4291","name":"fecA","hasFamily":true,"annotation":"KpLE2 phage-like element; ferric citrate outer membrane transporter; FecA is the outer membrane receptor protein in the Fe(3+) dicitrate transport system","uniprotId":"FECA_ECOLI"},"6884794":{"id":6884794,"externalId":"511145.b4292","name":"fecR","hasFamily":true,"annotation":"KpLE2 phage-like element; transmembrane signal transducer for ferric citrate transport; Regulation of iron dicitrate transport. In the absence of citrate FecR inactivates fecI. FecR is probably a sensor that recognizes iron dicitrate in the periplasm","uniprotId":"FECR_ECOLI"},"6884795":{"id":6884795,"externalId":"511145.b4293","name":"fecI","hasFamily":true,"annotation":"KpLE2 phage-like element; RNA polymerase, sigma 19 factor; Sigma factors are initiation factors that promote the attachment of RNA polymerase to specific initiation sites and are then released. This sigma factor regulates the fec genes for iron dicitrate transport (Probable)","uniprotId":"FECI_ECOLI"},"6884796":{"id":6884796,"externalId":"511145.b4294","name":"aec39","hasFamily":false,"annotation":"IS1 repressor TnpA; Absolutely required for transposition of IS1","uniprotId":"INSA7_ECOLI"},"6884797":{"id":6884797,"externalId":"511145.b4295","name":"yjhU","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator","uniprotId":"YJHU_ECOLI"},"6884798":{"id":6884798,"externalId":"511145.b4296","name":"yjhF","hasFamily":true,"annotation":"KpLE2 phage-like element; predicted transporter","uniprotId":"YJHF_ECOLI"},"6884799":{"id":6884799,"externalId":"511145.b4297","name":"yjhG","hasFamily":true,"annotation":"KpLE2 phage-like element","uniprotId":"YJHG_ECOLI"},"6884800":{"id":6884800,"externalId":"511145.b4298","name":"yjhH","hasFamily":true,"annotation":"KpLE2 phage-like element; predicted lyase/synthase","uniprotId":"YJHH_ECOLI"},"6884801":{"id":6884801,"externalId":"511145.b4299","name":"yjhI","hasFamily":false,"annotation":"KpLE2 phage-like element","uniprotId":"YJHI_ECOLI"},"6884802":{"id":6884802,"externalId":"511145.b4300","name":"sgcR","hasFamily":true,"annotation":"KpLE2 phage-like element; Putative transcriptional regulator for the sgcREAQCX region","uniprotId":"SGCR_ECOLI"},"6884803":{"id":6884803,"externalId":"511145.b4301","name":"sgcE","hasFamily":true,"annotation":"KpLE2 phage-like element; predicted epimerase; Probable pentose-5-phosphate 3-epimerase","uniprotId":"SGCE_ECOLI"},"6884804":{"id":6884804,"externalId":"511145.b4302","name":"sgcA","hasFamily":true,"annotation":"KpLE2 phage-like element; predicted phosphotransferase enzyme IIA component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (By similarity)","uniprotId":"SGCA_ECOLI"},"6884805":{"id":6884805,"externalId":"511145.b4303","name":"sgcQ","hasFamily":true,"annotation":"KpLE2 phage-like element; predicted nucleoside triphosphatase","uniprotId":"SGCQ_ECOLI"},"6884806":{"id":6884806,"externalId":"511145.b4304","name":"sgcC","hasFamily":true,"annotation":"KpLE2 phage-like element; predicted phosphotransferase enzyme IIC component; The phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane (By similarity)","uniprotId":"SGCC_ECOLI"},"6884807":{"id":6884807,"externalId":"511145.b4305","name":"sgcX","hasFamily":true,"annotation":"KpLE2 phage-like element","uniprotId":"SGCX_ECOLI"},"6884808":{"id":6884808,"externalId":"511145.b4306","name":"yjhP","hasFamily":true,"annotation":"KpLE2 phage-like element","uniprotId":"YJHP_ECOLI"},"6884809":{"id":6884809,"externalId":"511145.b4307","name":"yjhQ","hasFamily":true,"annotation":"KpLE2 phage-like element","uniprotId":"YJHQ_ECOLI"},"6884810":{"id":6884810,"externalId":"511145.b4309","name":"nanS","hasFamily":false,"annotation":"9-O-acetyl N-acetylneuraminic acid esterase; Probably catalyzes the hydrolysis of the 9-O-acetyl group of 9-O-acetyl-N-acetylneuraminate (Neu5,9Ac2). Is required for growth of E.coli on Neu5,9Ac2, an alternative sialic acid commonly found in mammalian host mucosal sites, in particular in the human intestine","uniprotId":"NANS_ECOLI"},"6884811":{"id":6884811,"externalId":"511145.b4310","name":"nanM","hasFamily":true,"annotation":"N-acetylneuraminic acid mutarotase; Converts alpha-N-acetylneuranimic acid (Neu5Ac) to the beta-anomer, accelerating the equilibrium between the alpha- and beta-anomers. Probably facilitates sialidase-negative bacteria to compete sucessfully for limited amounts of extracellular Neu5Ac, which is likely taken up in the beta-anomer. In addition, the rapid removal of sialic acid from solution might be advantageous to the bacterium to damp down host responses (By similarity)","uniprotId":"NANM_ECOLI"},"6884812":{"id":6884812,"externalId":"511145.b4311","name":"nanC","hasFamily":false,"annotation":"N-acetylnuraminic acid outer membrane channel protein; Outer membrane channel protein allowing the entry of N- acetylneuraminic acid (Neu5Ac, the most abundant sialic acid on host cell surfaces) into the bacteria (Probable). NanC proteins form high-conductance channels which are open at low membrane potentials and which have a weak anion selectivity","uniprotId":"NANC_ECOLI"},"6884813":{"id":6884813,"externalId":"511145.b4312","name":"fimB","hasFamily":true,"annotation":"tyrosine recombinase/inversion of on/off regulator of fimA; FimB is one of the 2 regulatory proteins which control the phase variation of type 1 fimbriae in E.coli. These proteins mediate the periodic inversion of a 300bp DNA segment that harbors the promoter for the fimbrial structural gene, FimA. FimB switches FimA on","uniprotId":"FIMB_ECOLI"},"6884814":{"id":6884814,"externalId":"511145.b4313","name":"fimE","hasFamily":false,"annotation":"tyrosine recombinase/inversion of on/off regulator of fimA; FimE is one of the 2 regulatory proteins which control the phase variation of type 1 fimbriae in E.coli. These proteins mediate the periodic inversion of a 300bp DNA segment that harbors the promoter for the fimbrial structural gene, fimA. FimE switches fimA off","uniprotId":"FIME_ECOLI"},"6884815":{"id":6884815,"externalId":"511145.b4314","name":"fimA","hasFamily":false,"annotation":"major type 1 subunit fimbrin (pilin); Fimbriae (also called pili), polar filaments radiating from the surface of the bacterium to a length of 0.5-1.5 micrometers and numbering 100-300 per cell, enable bacteria to colonize the epithelium of specific host organs","uniprotId":"FIMA1_ECOLI"},"6884816":{"id":6884816,"externalId":"511145.b4315","name":"fimI","hasFamily":false,"annotation":"fimbrial protein involved in type 1 pilus biosynthesis","uniprotId":"FIMI_ECOLI"},"6884817":{"id":6884817,"externalId":"511145.b4316","name":"fimC","hasFamily":false,"annotation":"periplasmic chaperone; Required for the biogenesis of type 1 fimbriae. Binds and interact with FimH","uniprotId":"FIMC_ECOLI"},"6884818":{"id":6884818,"externalId":"511145.b4317","name":"fimD","hasFamily":true,"annotation":"outer membrane usher protein, type 1 fimbrial synthesis; Involved in the export and assembly of FimA fimbrial subunits across the outer membrane","uniprotId":"FIMD_ECOLI"},"6884819":{"id":6884819,"externalId":"511145.b4318","name":"fimF","hasFamily":false,"annotation":"minor component of type 1 fimbriae; Involved in regulation of length and mediation of adhesion of type 1 fimbriae (but not necessary for the production of fimbriae). Involved in the integration of FimH in the fimbriae","uniprotId":"FIMF_ECOLI"},"6884820":{"id":6884820,"externalId":"511145.b4319","name":"fimG","hasFamily":false,"annotation":"minor component of type 1 fimbriae; Involved in regulation of length and mediation of adhesion of type 1 fimbriae (but not necessary for the production of fimbriae). Involved in the integration of FimH in the fimbriae","uniprotId":"FIMG_ECOLI"},"6884821":{"id":6884821,"externalId":"511145.b4320","name":"fimH","hasFamily":false,"annotation":"minor component of type 1 fimbriae; Involved in regulation of length and mediation of adhesion of type 1 fimbriae (but not necessary for the production of fimbriae). Adhesin responsible for the binding to D-mannose. It is laterally positioned at intervals in the structure of the type 1 fimbriae. In order to integrate FimH in the fimbriae FimF and FimG are needed","uniprotId":"FIMH_ECOLI"},"6884822":{"id":6884822,"externalId":"511145.b4321","name":"gntP","hasFamily":true,"annotation":"fructuronate transporter; High-affinity gluconate transporter with fairly broad specificity, including low affinity for glucuronate, several disaccharides, and some hexoses, but not glucose","uniprotId":"GNTP_ECOLI"},"6884823":{"id":6884823,"externalId":"511145.b4322","name":"uxuA","hasFamily":true,"annotation":"mannonate hydrolase; Catalyzes the dehydration of D-mannonate (By similarity)","uniprotId":"UXUA_ECOLI"},"6884824":{"id":6884824,"externalId":"511145.b4323","name":"uxuB","hasFamily":true,"annotation":"D-mannonate oxidoreductase, NAD-dependent","uniprotId":"UXUB_ECOLI"},"6884825":{"id":6884825,"externalId":"511145.b4324","name":"uxuR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor; Repressor for the uxuRBA operon","uniprotId":"UXUR_ECOLI"},"6884826":{"id":6884826,"externalId":"511145.b4325","name":"yjiC","hasFamily":false,"annotation":"predicted protein","uniprotId":"YJIC_ECOLI"},"6884827":{"id":6884827,"externalId":"511145.b4326","name":"iraD","hasFamily":true,"annotation":"RpoS stabilzer after DNA damage, anti-RssB factor; Inhibits RpoS proteolysis by regulating RssB activity, thereby increasing the stability of the sigma stress factor RpoS during oxidative stress. Its effect on RpoS stability is due to its interaction with RssB, which probably blocks the interaction of RssB with RpoS, and the consequent delivery of the RssB-RpoS complex to the ClpXP protein degradation pathway","uniprotId":"IRAD_ECOLI"},"6884828":{"id":6884828,"externalId":"511145.b4327","name":"yjiE","hasFamily":true,"annotation":"Cell density-dependent motility repressor; Protects cells from HOCl (hypochlorite) stress but not peroxide or diamide stress. Decreases the intracellular load of reactive oxygen species by up-regulating genes involved in methionine and cysteine biosynthesis and down-regulating Fur- regulated genes involved in iron acquisition. Has also been suggested to down-regulate expression of the flagellar regulon, decreasing motility, but this activity was not confirmed in a second study (PubMed:22223481)","uniprotId":"YJIE_ECOLI"},"6884829":{"id":6884829,"externalId":"511145.b4328","name":"iadA","hasFamily":true,"annotation":"isoaspartyl dipeptidase; Catalyzes the hydrolytic cleavage of a subset of L- isoaspartyl (L-beta-aspartyl) dipeptides. Used to degrade proteins damaged by L-isoaspartyl residues formation. The best substrate for the enzyme reported thus far is iso-Asp-Leu","uniprotId":"IADA_ECOLI"},"6884830":{"id":6884830,"externalId":"511145.b4329","name":"yjiG","hasFamily":true,"annotation":"inner membrane protein, SpmB family","uniprotId":"YJIG_ECOLI"},"6884831":{"id":6884831,"externalId":"511145.b4330","name":"yjiH","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YJIH_ECOLI"},"6884832":{"id":6884832,"externalId":"511145.b4331","name":"kptA","hasFamily":true,"annotation":"RNA 2'-phosphotransferase; Removes the 2'-phosphate from RNA via an intermediate in which the phosphate is ADP-ribosylated by NAD followed by a presumed transesterification to release the RNA and generate ADP- ribose 1''-2''-cyclic phosphate (APPR>P). May function as an ADP- ribosylase (By similarity)","uniprotId":"KPTA_ECOLI"},"6884833":{"id":6884833,"externalId":"511145.b4332","name":"yjiJ","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YJIJ_ECOLI"},"6884834":{"id":6884834,"externalId":"511145.b4333","name":"yjiK","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJIK_ECOLI"},"6884835":{"id":6884835,"externalId":"511145.b4334","name":"yjiL","hasFamily":true,"annotation":"putative ATPase, activator of (R)-hydroxyglutaryl-CoA dehydratase","uniprotId":"YJIL_ECOLI"},"6884836":{"id":6884836,"externalId":"511145.b4335","name":"yjiM","hasFamily":true,"annotation":"putative 2-hydroxyglutaryl-CoA dehydratase","uniprotId":"YJIM_ECOLI"},"6884837":{"id":6884837,"externalId":"511145.b4336","name":"yjiN","hasFamily":true,"annotation":"zinc-type alcohol dehydrogenase-like protein","uniprotId":"YJIN_ECOLI"},"6884838":{"id":6884838,"externalId":"511145.b4337","name":"mdtM","hasFamily":true,"annotation":"multidrug efflux system protein; Confers resistance to acriflavine, chloramphenicol, norfloxacin, ethidium bromide and TPP","uniprotId":"MDTM_ECOLI"},"6884839":{"id":6884839,"externalId":"511145.b4340","name":"yjiR","hasFamily":true,"annotation":"fused predicted DNA-binding transcriptional regulator/predicted aminotransferase","uniprotId":"YJIR_ECOLI"},"6884840":{"id":6884840,"externalId":"511145.b4341","name":"yjiS","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJIS_ECOLI"},"6884841":{"id":6884841,"externalId":"511145.b4345","name":"mcrC","hasFamily":false,"annotation":"5-methylcytosine-specific restriction enzyme McrBC, subunit McrC; Modifies the specificity of McrB restriction by expanding the range of modified sequences restricted. Does not bind to DNA","uniprotId":"MCRC_ECOLI"},"6884842":{"id":6884842,"externalId":"511145.b4346","name":"mcrB","hasFamily":true,"annotation":"5-methylcytosine-specific restriction enzyme McrBC, subunit McrB; Recognizes N4- and C5-methylcytosine (and 5-hydroxy- methylcytosines) produced by a broad range of DNA methylases and appears to act against 5-methylcytosine preceded by a purine residue. Binds to DNA containing methylated cytosines; also binds to GTP. Isoform 33 kDa is less active than isoform 51 kDa and may play a role in regulating the activity of isoform 51 kDa by competing with it in DNA and protein binding abilities","uniprotId":"MCRB_ECOLI"},"6884843":{"id":6884843,"externalId":"511145.b4347","name":"symE","hasFamily":true,"annotation":"toxic peptide regulated by antisense sRNA symR; Involved in the degradation and recycling of damaged RNA. It is itself a target for degradation by the ATP-dependent protease La (By similarity)","uniprotId":"SYME_ECOLI"},"6884844":{"id":6884844,"externalId":"511145.b4348","name":"hsdS","hasFamily":true,"annotation":"specificity determinant for hsdM and hsdR; The M and S subunits together form a methyltransferase (MTase) that methylates two adenine residues in complementary strands of a bipartite DNA recognition sequence. In the presence of the R subunit the complex can also act as an endonuclease, binding to the same target sequence but cutting the DNA some distance from this site. Whether the DNA is cut or modified depends on the methylation state of the target sequence. When the target site is unmodified, the DNA is cut. When the target site is hemimethylated, the complex acts as a maintenance M [...] ","uniprotId":"T1SK_ECOLI"},"6884845":{"id":6884845,"externalId":"511145.b4349","name":"hsdM","hasFamily":true,"annotation":"DNA methyltransferase M; The M and S subunits together form a methyltransferase (MTase) that methylates two adenine residues in complementary strands of a bipartite DNA recognition sequence. In the presence of the R subunit the complex can also act as an endonuclease, binding to the same target sequence but cutting the DNA some distance from this site. Whether the DNA is cut or modified depends on the methylation state of the target sequence. When the target site is unmodified, the DNA is cut. When the target site is hemimethylated, the complex acts as a maintenance MTase modifying the [...] ","uniprotId":"T1MK_ECOLI"},"6884846":{"id":6884846,"externalId":"511145.b4350","name":"hsdR","hasFamily":true,"annotation":"endonuclease R Type I restriction enzyme; The EcoKI enzyme recognizes 5'-AACN(6)GTGC-3'. Subunit R is required for both nuclease and ATPase activities, but not for modification","uniprotId":"T1RK_ECOLI"},"6884847":{"id":6884847,"externalId":"511145.b4351","name":"mrr","hasFamily":true,"annotation":"methylated adenine and cytosine restriction protein; Involved in the acceptance of foreign DNA which is modified. Restricts both adenine- and cytosine-methylated DNA","uniprotId":"MRR_ECOLI"},"6884848":{"id":6884848,"externalId":"511145.b4352","name":"yjiA","hasFamily":true,"annotation":"putative GTPase; Binds GTP. May function as GTP-dependent regulator","uniprotId":"YJIA_ECOLI"},"6884849":{"id":6884849,"externalId":"511145.b4353","name":"yjiX","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJIX_ECOLI"},"6884850":{"id":6884850,"externalId":"511145.b4354","name":"cts1","hasFamily":true,"annotation":"putative inner membrane protein","uniprotId":"YJIY_ECOLI"},"6884851":{"id":6884851,"externalId":"511145.b4355","name":"tsr","hasFamily":true,"annotation":"methyl-accepting chemotaxis protein I, serine sensor receptor; Receptor for the attractant L-serine and related amino acids. Is also responsible for chemotaxis away from a wide range of repellents, including leucine, indole, and weak acids","uniprotId":"MCP1_ECOLI"},"6884852":{"id":6884852,"externalId":"511145.b4356","name":"yjiZ","hasFamily":true,"annotation":"L-galactonate transporter; Responsible for the transport of L-galactonate from the periplasm across the inner membrane (Probable). Is essential for growth on L-galactonate as the sole carbon source","uniprotId":"YJJL_ECOLI"},"6884853":{"id":6884853,"externalId":"511145.b4357","name":"yjjM","hasFamily":false,"annotation":"DNA-binding transcriptional activator for yjjMN; May be a positive transcriptional regulator for yjjM and/or yjjN. Is essential for growth on L-galactonate as the sole carbon source","uniprotId":"YJJM_ECOLI"},"6884854":{"id":6884854,"externalId":"511145.b4358","name":"yjjN","hasFamily":true,"annotation":"L-galactonate oxidoreductase; Putative L-galactonate oxidoreductase that is required for growth on L-galactonate as the sole carbon source","uniprotId":"YJJN_ECOLI"},"6884855":{"id":6884855,"externalId":"511145.b4359","name":"mdoB","hasFamily":true,"annotation":"phosphoglycerol transferases I and II; Transfers a phosphoglycerol residue from phosphatidylglycerol to the membrane-bound nascent glucan backbones (By similarity)","uniprotId":"OPGB_ECOLI"},"6884856":{"id":6884856,"externalId":"511145.b4360","name":"yjjA","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJJA_ECOLI"},"6884857":{"id":6884857,"externalId":"511145.b4361","name":"dnaC","hasFamily":true,"annotation":"DNA biosynthesis protein; This protein is required for chromosomal replication. It forms, in concert with DnaB protein and other prepriming proteins DnaT, N, N', N'' a prepriming protein complex on the specific site of the template DNA recognized by protein N'","uniprotId":"DNAC_ECOLI"},"6884858":{"id":6884858,"externalId":"511145.b4362","name":"dnaT","hasFamily":true,"annotation":"DNA biosynthesis protein (primosomal protein I); This protein is required for primosome-dependent normal DNA replication; it is also involved in inducing stable DNA replication during SOS response. It forms, in concert with DnaB protein and other prepriming proteins DnaC, N, N', N'' a prepriming protein complex on the specific site of the template DNA recognized by protein N' (By similarity)","uniprotId":"DNAT_ECOLI"},"6884859":{"id":6884859,"externalId":"511145.b4363","name":"yjjB","hasFamily":true,"annotation":"conserved inner membrane protein","uniprotId":"YJJB_ECOLI"},"6884860":{"id":6884860,"externalId":"511145.b4364","name":"yjjP","hasFamily":true,"annotation":"inner membrane protein, H-NS-repressed, DUF1212 family","uniprotId":"YJJP_ECOLI"},"6884861":{"id":6884861,"externalId":"511145.b4365","name":"yjjQ","hasFamily":true,"annotation":"DNA-binding transcriptional regulator","uniprotId":"YJJQ_ECOLI"},"6884862":{"id":6884862,"externalId":"511145.b4366","name":"bglJ","hasFamily":true,"annotation":"bgl operon transcriptional activator; A crytic transcriptional activator. When its expression is induced it relieves H-NS repression of the bgl operon. Acts independently of transcription factor LeuO","uniprotId":"BGLJ_ECOLI"},"6884863":{"id":6884863,"externalId":"511145.b4367","name":"fhuF","hasFamily":true,"annotation":"ferric iron reductase involved in ferric hydroximate transport; Involved in the reduction of ferric iron in cytoplasmic ferrioxamine B","uniprotId":"FHUF_ECOLI"},"6884864":{"id":6884864,"externalId":"511145.b4371","name":"rsmC","hasFamily":true,"annotation":"16S rRNA m(2)G1207 methyltransferase; Specifically methylates the guanine in position 1207 of 16S rRNA in the 30S particle (By similarity)","uniprotId":"RSMC_ECOLI"},"6884865":{"id":6884865,"externalId":"511145.b4372","name":"holD","hasFamily":true,"annotation":"DNA polymerase III, psi subunit; DNA polymerase III is a complex, multichain enzyme responsible for most of the replicative synthesis in bacteria. This DNA polymerase also exhibits 3' to 5' exonuclease activity. The exact function of the psi subunit is unknown","uniprotId":"HOLD_ECOLI"},"6884866":{"id":6884866,"externalId":"511145.b4373","name":"rimI","hasFamily":true,"annotation":"ribosomal-protein-S18-alanine N-acetyltransferase; This enzyme acetylates the N-terminal alanine of ribosomal protein S18","uniprotId":"RIMI_ECOLI"},"6884867":{"id":6884867,"externalId":"511145.b4374","name":"yjjG","hasFamily":true,"annotation":"dUMP phosphatase; Nucleotidase that shows high phosphatase activity toward non-canonical pyrimidine nucleotides and three canonical nucleoside 5'-monophosphates (UMP, dUMP, and dTMP), and very low activity against TDP, IMP, UDP, GMP, dGMP, AMP, dAMP, and 6- phosphogluconate. Appears to function as a house-cleaning nucleotidase in vivo, since the general nucleotidase activity of YjjG allows it to protect cells against non-canonical pyrimidine derivatives such as 5-fluoro-2'-deoxyuridine, 5-fluorouridine, 5- fluoroorotate, 5-fluorouracil, and 5-aza-2'-deoxycytidine, and prevents the inco [...] ","uniprotId":"YJJG_ECOLI"},"6884868":{"id":6884868,"externalId":"511145.b4375","name":"prfC","hasFamily":true,"annotation":"peptide chain release factor RF-3; Increases the formation of ribosomal termination complexes and stimulates activities of RF-1 and RF-2. It binds guanine nucleotides and has strong preference for UGA stop codons. It may interact directly with the ribosome. The stimulation of RF- 1 and RF-2 is significantly reduced by GTP and GDP, but not by GMP (By similarity)","uniprotId":"RF3_ECOLI"},"6884869":{"id":6884869,"externalId":"511145.b4376","name":"osmY","hasFamily":true,"annotation":"periplasmic protein","uniprotId":"OSMY_ECOLI"},"6884870":{"id":6884870,"externalId":"511145.b4377","name":"yjjU","hasFamily":true,"annotation":"putative phospholipase, patatin-like family; Probable lipid hydrolase (By similarity)","uniprotId":"YJJU_ECOLI"},"6884871":{"id":6884871,"externalId":"511145.b4378","name":"yjjV","hasFamily":true,"annotation":"putative DNase; Putative deoxyribonuclease (By similarity)","uniprotId":"YJJV_ECOLI"},"6884872":{"id":6884872,"externalId":"511145.b4379","name":"yjjW","hasFamily":true,"annotation":"putative pyruvate formate lyase activating enzyme","uniprotId":"YJJW_ECOLI"},"6884873":{"id":6884873,"externalId":"511145.b4380","name":"yjjI","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJJI_ECOLI"},"6884874":{"id":6884874,"externalId":"511145.b4381","name":"deoC","hasFamily":true,"annotation":"2-deoxyribose-5-phosphate aldolase, NAD(P)-linked; Catalyzes a reversible aldol reaction between acetaldehyde and D-glyceraldehyde 3-phosphate to generate 2-deoxy- D-ribose 5-phosphate (By similarity)","uniprotId":"DEOC_ECOLI"},"6884875":{"id":6884875,"externalId":"511145.b4382","name":"deoA","hasFamily":true,"annotation":"thymidine phosphorylase; The enzymes which catalyze the reversible phosphorolysis of pyrimidine nucleosides are involved in the degradation of these compounds and in their utilization as carbon and energy sources, or in the rescue of pyrimidine bases for nucleotide synthesis (By similarity)","uniprotId":"TYPH_ECOLI"},"6884876":{"id":6884876,"externalId":"511145.b4383","name":"deoB","hasFamily":true,"annotation":"phosphopentomutase; Phosphotransfer between the C1 and C5 carbon atoms of pentose (By similarity)","uniprotId":"DEOB_ECOLI"},"6884877":{"id":6884877,"externalId":"511145.b4384","name":"deoD","hasFamily":true,"annotation":"purine-nucleoside phosphorylase; Cleavage of guanosine or inosine to respective bases and sugar-1-phosphate molecules","uniprotId":"DEOD_ECOLI"},"6884878":{"id":6884878,"externalId":"511145.b4385","name":"yjjJ","hasFamily":false,"annotation":"HipA family protein","uniprotId":"YJJJ_ECOLI"},"6884879":{"id":6884879,"externalId":"511145.b4386","name":"lplA","hasFamily":true,"annotation":"lipoate-protein ligase A; Catalyzes both the ATP-dependent activation of exogenously supplied lipoate to lipoyl-AMP and the transfer of the activated lipoyl onto the lipoyl domains of lipoate-dependent enzymes. Is also able to catalyze very poorly the transfer of lipoyl and octanoyl moiety from their acyl carrier protein","uniprotId":"LPLA_ECOLI"},"6884880":{"id":6884880,"externalId":"511145.b4387","name":"smp","hasFamily":true,"annotation":"conserved membrane protein","uniprotId":"SMP_ECOLI"},"6884881":{"id":6884881,"externalId":"511145.b4388","name":"serB","hasFamily":true,"annotation":"3-phosphoserine phosphatase; Catalyzes the dephosphorylation of phosphoserine (P- Ser). Also catalyzes the hydrolysis of phosphothreonine (P-Thr)","uniprotId":"SERB_ECOLI"},"6884882":{"id":6884882,"externalId":"511145.b4389","name":"sms","hasFamily":true,"annotation":"DNA repair protein; May play a role in the repair of endogenous alkylation damage (By similarity)","uniprotId":"RADA_ECOLI"},"6884883":{"id":6884883,"externalId":"511145.b4390","name":"nadR","hasFamily":true,"annotation":"bifunctional DNA-binding transcriptional repressor/ NMN adenylyltransferase; This enzyme has three activities: DNA binding, nicotinamide mononucleotide (NMN) adenylyltransferase and ribosylnicotinamide (RN) kinase. The DNA-binding domain binds to the nadB operator sequence in an NAD- and ATP-dependent manner. As NAD levels increase within the cell, the affinity of NadR for the nadB operator regions of nadA, nadB, and pncB increases, repressing the transcription of these genes. The RN kinase activity catalyzes the phosphorylation of RN to form nicotinamide ribonucleotide. The NMN adenyl [...] ","uniprotId":"NADR_ECOLI"},"6884884":{"id":6884884,"externalId":"511145.b4391","name":"yjjK","hasFamily":true,"annotation":"fused predicted transporter subunits of ABC superfamily: ATP-binding components","uniprotId":"YJJK_ECOLI"},"6884885":{"id":6884885,"externalId":"511145.b4392","name":"slt","hasFamily":true,"annotation":"lytic murein transglycosylase, soluble; Murein-degrading enzyme. Catalyzes the cleavage of the glycosidic bonds between N-acetylmuramic acid and N- acetylglucosamine residues in peptidoglycan. May play a role in recycling of muropeptides during cell elongation and/or cell division","uniprotId":"SLT_ECOLI"},"6884886":{"id":6884886,"externalId":"511145.b4393","name":"trpR","hasFamily":true,"annotation":"DNA-binding transcriptional repressor, tryptophan-binding; This protein is an aporepressor. When complexed with L- tryptophan it binds the operator region of the trp operon (5'- ACTAGT-'3') and prevents the initiation of transcription. The complex also regulates trp repressor biosynthesis by binding to its regulatory region (By similarity)","uniprotId":"TRPR_ECOLI"},"6884887":{"id":6884887,"externalId":"511145.b4394","name":"yjjX","hasFamily":true,"annotation":"inosine/xanthosine triphosphatase; Phosphatase that hydrolyzes non-canonical purine nucleotides such as XTP and ITP to their respective diphosphate derivatives. Probably excludes non-canonical purines from DNA precursor pool, thus preventing their incorporation into DNA and avoiding chromosomal lesions. ITP is the best substrate, followed by XTP, GDP or dITP. Also implicated in the resistance against the thiamine metabolism inhibitors bacimethrin and CF3-HMP","uniprotId":"NCPP_ECOLI"},"6884888":{"id":6884888,"externalId":"511145.b4395","name":"gpmB","hasFamily":true,"annotation":"phosphatase","uniprotId":"GPMB_ECOLI"},"6884889":{"id":6884889,"externalId":"511145.b4396","name":"rob","hasFamily":true,"annotation":"right oriC-binding transcriptional activator, AraC family; Binds to the right arm of the replication origin oriC of the chromosome. Rob binding may influence the formation of the nucleoprotein structure, required for oriC function in the initiation of replication","uniprotId":"ROB_ECOLI"},"6884890":{"id":6884890,"externalId":"511145.b4397","name":"creA","hasFamily":true,"annotation":"conserved protein","uniprotId":"CREA_ECOLI"},"6884891":{"id":6884891,"externalId":"511145.b4398","name":"creB","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with CreC; Member of the two-component regulatory system CreC/CreB involved in catabolic regulation","uniprotId":"CREB_ECOLI"},"6884892":{"id":6884892,"externalId":"511145.b4399","name":"creC","hasFamily":true,"annotation":"sensory histidine kinase in two-component regulatory system with CreB or PhoB, regulator of the CreBC regulon; Member of the two-component regulatory system CreC/CreB involved in catabolic regulation. CreC may function as a membrane- associated protein kinase that phosphorylates CreB in response to environmental signals. CreC can also phosphorylate PhoB","uniprotId":"CREC_ECOLI"},"6884893":{"id":6884893,"externalId":"511145.b4400","name":"creD","hasFamily":true,"annotation":"inner membrane protein","uniprotId":"CRED_ECOLI"},"6884894":{"id":6884894,"externalId":"511145.b4401","name":"arcA2","hasFamily":true,"annotation":"DNA-binding response regulator in two-component regulatory system with ArcB or CpxA; Member of the two-component regulatory system ArcB/ArcA. Represses a wide variety of aerobic enzymes under anaerobic conditions. Controls the resistance of E.coli to dyes; required for expression of the alkaline phosphatase and sex factor F genes; It also may be involved in the osmoregulation of envelope proteins. When activated by ArcB, it negatively regulates the expression of genes of aerobic function. Activates the transcription of the plfB operon by binding to its promoter","uniprotId":"A5PFJ8_ECOLI"},"6884895":{"id":6884895,"externalId":"511145.b4402","name":"yjjY","hasFamily":true,"annotation":"predicted protein","uniprotId":"YJJY_ECOLI"},"6884896":{"id":6884896,"externalId":"511145.b4403","name":"lasT","hasFamily":true,"annotation":"putative methyltransferase","uniprotId":"LAST_ECOLI"},"6884897":{"id":6884897,"externalId":"511145.b4406","name":"yaeP","hasFamily":true,"annotation":"conserved protein","uniprotId":"YAEP_ECOLI"},"6884898":{"id":6884898,"externalId":"511145.b4407","name":"thiS","hasFamily":true,"annotation":"immediate sulfur donor in thiazole formation; Is the sulfur donor in the synthesis of the thiazole phosphate moiety of thiamine phosphate","uniprotId":"THIS_ECOLI"},"6884899":{"id":6884899,"externalId":"511145.b4409","name":"blr","hasFamily":false,"annotation":"beta-lactam resistance membrane protein; Component of the cell division machinery, which is probably involved in the stabilization of the divisome under certain stress conditions","uniprotId":"BLR_ECOLI"},"6884900":{"id":6884900,"externalId":"511145.b4410","name":"ecnA","hasFamily":true,"annotation":"entericidin A membrane lipoprotein, antidote entericidin B; Acts as antidote to the effect of entericidin B","uniprotId":"ECNA_ECOLI"},"6884901":{"id":6884901,"externalId":"511145.b4411","name":"ecnB","hasFamily":true,"annotation":"entericidin B membrane lipoprotein; Plays a role in the bacteriolysis. Is activated under conditions of high osmolarity by the factor sigma S. Entericidin A functions as an antidote","uniprotId":"ECNB_ECOLI"},"6884902":{"id":6884902,"externalId":"511145.b4412","name":"b4412","hasFamily":false,"annotation":"toxic membrane protein, small"},"6884903":{"id":6884903,"externalId":"511145.b4415","name":"hokE","hasFamily":false,"annotation":"toxic polypeptide, small","uniprotId":"HOKE_ECOLI"},"6884904":{"id":6884904,"externalId":"511145.b4419","name":"b4419","hasFamily":false,"annotation":"toxic polypeptide, small"},"6884905":{"id":6884905,"externalId":"511145.b4421","name":"ldrB","hasFamily":false,"annotation":"toxic polypeptide, small; Toxic peptide whose overexpression causes rapid cell killing and nucleoid condensation of the host cell (By similarity)","uniprotId":"LDRB_ECOLI"},"6884906":{"id":6884906,"externalId":"511145.b4423","name":"ldrC","hasFamily":false,"annotation":"toxic polypeptide, small; Toxic peptide whose overexpression causes rapid cell killing and nucleoid condensation of the host cell (By similarity)","uniprotId":"LDRA_ECOLI"},"6884907":{"id":6884907,"externalId":"511145.b4428","name":"hokB","hasFamily":false,"annotation":"toxic polypeptide, small","uniprotId":"HOKB_ECOLI"},"6884908":{"id":6884908,"externalId":"511145.b4453","name":"ldrD","hasFamily":false,"annotation":"toxic polypeptide, small; Toxic peptide whose overexpression causes rapid cell killing and nucleoid condensation of the host cell. Overexpression induces stress-response and a number of membrane protein genes","uniprotId":"LDRD_ECOLI"},"6884909":{"id":6884909,"externalId":"511145.b4455","name":"hokA","hasFamily":false,"annotation":"toxic polypeptide, small; Overproduction is toxic","uniprotId":"HOKA_ECOLI"},"6884910":{"id":6884910,"externalId":"511145.b4460","name":"araH","hasFamily":true,"annotation":"fused L-arabinose transporter subunits of ABC superfamily: membrane components; Part of the binding-protein-dependent transport system for L-arabinose. Probably responsible for the translocation of the substrate across the membrane","uniprotId":"ARAH_ECOLI"},"6884911":{"id":6884911,"externalId":"511145.b4461","name":"yfjD","hasFamily":true,"annotation":"inner membrane protein, UPF0053 family","uniprotId":"YFJD_ECOLI"},"6884912":{"id":6884912,"externalId":"511145.b4463","name":"ygcU","hasFamily":true,"annotation":"putative FAD containing dehydrogenase","uniprotId":"YGCU_ECOLI"},"6884913":{"id":6884913,"externalId":"511145.b4464","name":"ygfQ","hasFamily":false,"annotation":"putative purine permease","uniprotId":"YGFQ_ECOLI"},"6884914":{"id":6884914,"externalId":"511145.b4465","name":"yggP","hasFamily":false,"annotation":"putative dehydrogenase","uniprotId":"YGGP_ECOLI"},"6884915":{"id":6884915,"externalId":"511145.b4466","name":"yghJ","hasFamily":true,"annotation":"putative inner membrane lipoprotein","uniprotId":"ACFD_ECOLI"},"6884916":{"id":6884916,"externalId":"511145.b4467","name":"b4467","hasFamily":true,"annotation":"glycolate oxidase 4Fe-4S iron-sulfur cluster subunit","uniprotId":"GLCF_ECOLI"},"6884917":{"id":6884917,"externalId":"511145.b4468","name":"glcE","hasFamily":true,"annotation":"glycolate oxidase FAD binding subunit","uniprotId":"GLCE_ECOLI"},"6884918":{"id":6884918,"externalId":"511145.b4469","name":"ygiQ","hasFamily":true,"annotation":"conserved protein","uniprotId":"YGIQ_ECOLI"},"6884919":{"id":6884919,"externalId":"511145.b4470","name":"yhaM","hasFamily":true,"annotation":"conserved protein","uniprotId":"YHAM_ECOLI"},"6884920":{"id":6884920,"externalId":"511145.b4471","name":"tdcG","hasFamily":true,"annotation":"L-serine dehydratase 3","uniprotId":"TDCG_ECOLI"},"6884921":{"id":6884921,"externalId":"511145.b4472","name":"yhdP","hasFamily":true,"annotation":"conserved membrane protein, predicted transporter","uniprotId":"YHDP_ECOLI"},"6884922":{"id":6884922,"externalId":"511145.b4473","name":"smf","hasFamily":true,"annotation":"conserved protein","uniprotId":"SMF_ECOLI"},"6884923":{"id":6884923,"externalId":"511145.b4474","name":"frlC","hasFamily":true,"annotation":"fructoselysine 3-epimerase; Not clear, may be involved in an isomerization step","uniprotId":"FRLC_ECOLI"},"6884924":{"id":6884924,"externalId":"511145.b4475","name":"rtcA","hasFamily":true,"annotation":"RNA 3'-terminal phosphate cyclase; Catalyzes the conversion of 3'-phosphate to a 2',3'- cyclic phosphodiester at the end of RNA. The mechanism of action of the enzyme occurs in 3 steps: (A) adenylation of the enzyme by ATP; (B) transfer of adenylate to an RNA-N3'P to produce RNA- N3'PP5'A; (C) and attack of the adjacent 2'-hydroxyl on the 3'- phosphorus in the diester linkage to produce the cyclic end product. The biological role of this enzyme is unknown but it is likely to function in some aspects of cellular RNA processing (By similarity)","uniprotId":"RTCA_ECOLI"},"6884925":{"id":6884925,"externalId":"511145.b4476","name":"gntU","hasFamily":true,"annotation":"gluconate transporter, low affinity GNT 1 system; Part of the gluconate utilization system Gnt-I; low- affinity intake of gluconate","uniprotId":"GNTU_ECOLI"},"6884926":{"id":6884926,"externalId":"511145.b4477","name":"b4477","hasFamily":true,"annotation":"2-oxo-3-deoxygalactonate 6-phosphate aldolase","uniprotId":"DGOA_ECOLI"},"6884927":{"id":6884927,"externalId":"511145.b4478","name":"dgoD","hasFamily":true,"annotation":"galactonate dehydratase; Catalyzes the dehydration of D-galactonate to 2-keto-3- deoxy-D-galactonate (By similarity)","uniprotId":"DGOD_ECOLI"},"6884928":{"id":6884928,"externalId":"511145.b4479","name":"dgoR","hasFamily":true,"annotation":"putative DNA-binding transcriptional regulator; Repressor for the dgoRKAT operon. Binds D-galactonate as an inducer","uniprotId":"DGOR_ECOLI"},"6884929":{"id":6884929,"externalId":"511145.b4480","name":"hdfR","hasFamily":true,"annotation":"DNA-binding transcriptional regulator; Negatively regulates the transcription of the flagellar master operon flhDC by binding to the upstream region of the operon (By similarity)","uniprotId":"HDFR_ECOLI"},"6884930":{"id":6884930,"externalId":"511145.b4481","name":"rffT","hasFamily":true,"annotation":"TDP-Fuc4NAc:lipidIIFuc4NAc transferase; Catalyzes the synthesis of Und-PP-GlcNAc-ManNAcA-Fuc4NAc (Lipid III), the third lipid-linked intermediate involved in ECA synthesis (By similarity)","uniprotId":"WECF_ECOLI"},"6884931":{"id":6884931,"externalId":"511145.b4482","name":"yigE","hasFamily":true,"annotation":"putative protein, DUF2233 family","uniprotId":"YIGE_ECOLI"},"6884932":{"id":6884932,"externalId":"511145.b4483","name":"mttC","hasFamily":true,"annotation":"quality control of Tat-exported FeS proteins; Mg-dependent cytoplasmic DNase; Part of the twin-arginine translocation (Tat) system that transports large folded proteins containing a characteristic twin-arginine motif in their signal peptide across membranes. Although TatD is not required for export activity, it is a central component of a quality control system that is linked to the Tat translocation system. May act by degrading wild-type pre-protein molecules that are misfolded. Shows magnesium-dependent DNase activity (By similarity)","uniprotId":"TATD_ECOLI"},"6884933":{"id":6884933,"externalId":"511145.b4484","name":"cpxP","hasFamily":true,"annotation":"inhibitor of the cpx response; periplasmic adaptor protein; Aids in combating extracytoplasmic protein-mediated toxicity","uniprotId":"CPXP_ECOLI"},"6884934":{"id":6884934,"externalId":"511145.b4485","name":"ytfR","hasFamily":true,"annotation":"putative sugar transporter subunit: ATP-binding component of ABC superfamily","uniprotId":"YTFR_ECOLI"},"6884935":{"id":6884935,"externalId":"511145.b4487","name":"yjdP","hasFamily":false,"annotation":"conserved protein","uniprotId":"YJDP_ECOLI"},"6884936":{"id":6884936,"externalId":"511145.b4501","name":"torI","hasFamily":false,"annotation":"response regulator inhibitor for tor operon; Transcription inhibitory protein for the torCAD operon. Also acts as an excisionase and plays an essential role in the defective prophage CPS53 excision","uniprotId":"TORI_ECOLI"},"6884937":{"id":6884937,"externalId":"511145.b4502","name":"yeiW","hasFamily":true,"annotation":"conserved protein, UPF0153 family","uniprotId":"YEIW_ECOLI"},"6884938":{"id":6884938,"externalId":"511145.b4504","name":"ykfH","hasFamily":false,"annotation":"predicted protein","uniprotId":"YKFH_ECOLI"},"6884939":{"id":6884939,"externalId":"511145.b4506","name":"ykgO","hasFamily":true,"annotation":"RpmJ-like protein","uniprotId":"RL362_ECOLI"},"6884940":{"id":6884940,"externalId":"511145.b4509","name":"ylcG","hasFamily":false,"annotation":"expressed protein, DLP12 prophage","uniprotId":"YLCG_ECOLI"},"6884941":{"id":6884941,"externalId":"511145.b4510","name":"rzoD","hasFamily":false,"annotation":"DLP12 prophage; Involvement in host cell lysis. Promotes membrane fusion. The local destruction of the membrane is accompanied by leakage of the cell contents (By similarity)","uniprotId":"RZOD_ECOLI"},"6884942":{"id":6884942,"externalId":"511145.b4511","name":"ybdZ","hasFamily":true,"annotation":"conserved protein; Could be involved in enterobactin synthesis","uniprotId":"YBDZ_ECOLI"},"6884943":{"id":6884943,"externalId":"511145.b4512","name":"ybdD","hasFamily":true,"annotation":"conserved protein, DUF466 family","uniprotId":"YBDD_ECOLI"},"6884944":{"id":6884944,"externalId":"511145.b4513","name":"kdpF","hasFamily":false,"annotation":"potassium ion accessory transporter subunit","uniprotId":"KDPF_ECOLI"},"6884945":{"id":6884945,"externalId":"511145.b4515","name":"ybgT","hasFamily":true,"annotation":"conserved protein","uniprotId":"YBGT_ECOLI"},"6884946":{"id":6884946,"externalId":"511145.b4516","name":"insA3","hasFamily":false,"annotation":"IS1 repressor TnpA; Absolutely required for transposition of IS1","uniprotId":"INSA2_ECOLI"},"6884947":{"id":6884947,"externalId":"511145.b4517","name":"gnsA","hasFamily":true,"annotation":"multicopy suppressor of secG(Cs) and fabA6(Ts); Overexpression increases levels of unsaturated fatty acids and suppresses both the temperature-sensitive fabA6 mutation and cold-sensitive secG null mutation","uniprotId":"GNSA_ECOLI"},"6884948":{"id":6884948,"externalId":"511145.b4518","name":"ymdF","hasFamily":true,"annotation":"conserved protein","uniprotId":"YMDF_ECOLI"},"6884949":{"id":6884949,"externalId":"511145.b4520","name":"ymgF","hasFamily":false,"annotation":"inner membrane division septum protein; Could be involved in cell division. May participate in the stabilization of the cell divisome under specific conditions","uniprotId":"YMGF_ECOLI"},"6884950":{"id":6884950,"externalId":"511145.b4522","name":"ymiA","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YMIA_ECOLI"},"6884951":{"id":6884951,"externalId":"511145.b4523","name":"yciX","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YCIX_ECOLI"},"6884952":{"id":6884952,"externalId":"511145.b4526","name":"ydaE","hasFamily":false,"annotation":"Rac prophage","uniprotId":"YDAE_ECOLI"},"6884953":{"id":6884953,"externalId":"511145.b4527","name":"ydaF","hasFamily":false,"annotation":"Rac prophage","uniprotId":"YDAF_ECOLI"},"6884954":{"id":6884954,"externalId":"511145.b4528","name":"rzoR","hasFamily":false,"annotation":"Rac prophage; Involvement in host cell lysis. Promotes membrane fusion. The local destruction of the membrane is accompanied by leakage of the cell contents (By similarity)","uniprotId":"RZOR_ECOLI"},"6884955":{"id":6884955,"externalId":"511145.b4529","name":"ydbJ","hasFamily":true,"annotation":"putative lipoprotein, DUF333 family","uniprotId":"YDBJ_ECOLI"},"6884956":{"id":6884956,"externalId":"511145.b4532","name":"hicA","hasFamily":false,"annotation":"mRNA interferase toxin of the HicAB toxin-antitoxin system; Toxic component of a toxin-antitoxin (TA) module. A probable translation-independent mRNA interferase. Overexpression causes cessation of cell growth and inhibits cell proliferation via inhibition of translation; this blockage is overcome (after 90 minutes) by subsequent expression of antitoxin HicB. Overexpression causes cleavage of a number of mRNAs and tmRNA, in a translation-independent fashion, suggesting this is an mRNA interferase","uniprotId":"HICA_ECOLI"},"6884957":{"id":6884957,"externalId":"511145.b4533","name":"ynfO","hasFamily":false,"annotation":"hypothetical protein, Qin prophage","uniprotId":"YNFO_ECOLI"},"6884958":{"id":6884958,"externalId":"511145.b4535","name":"yniD","hasFamily":false,"annotation":"predicted protein","uniprotId":"YNID_ECOLI"},"6884959":{"id":6884959,"externalId":"511145.b4536","name":"yobH","hasFamily":true,"annotation":"predicted protein","uniprotId":"YOBH_ECOLI"},"6884960":{"id":6884960,"externalId":"511145.b4537","name":"yecJ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YECJ_ECOLI"},"6884961":{"id":6884961,"externalId":"511145.b4539","name":"yoeB","hasFamily":true,"annotation":"toxin of the YoeB-YefM toxin-antitoxin system; Toxic component of a toxin-antitoxin (TA) module. Its mode of function is controversial; it has been proposed to be an mRNA interferase but also an inhibitor of translation initiation. When overproduced in wild-type cells, inhibits bacterial growth and translation by cleavage of mRNA molecules while it has a weak effect on colony forming ability. Overproduction of Lon protease specifically activates YoeB-dependent mRNA cleavage, leading to lethality. YefM binds to the promoter region of the yefM-yeoB operon to repress transcription, YeoB a [...] ","uniprotId":"YOEB_ECOLI"},"6884962":{"id":6884962,"externalId":"511145.b4541","name":"yehK","hasFamily":false,"annotation":"predicted protein","uniprotId":"YEHK_ECOLI"},"6884963":{"id":6884963,"externalId":"511145.b4542","name":"yohO","hasFamily":false,"annotation":"predicted protein","uniprotId":"YOHO_ECOLI"},"6884964":{"id":6884964,"externalId":"511145.b4543","name":"ypaA","hasFamily":false,"annotation":"predicted protein"},"6884965":{"id":6884965,"externalId":"511145.b4544","name":"arnE","hasFamily":true,"annotation":"undecaprenyl phosphate-alpha-L-ara4N exporter; flippase ArnEF subunit; Translocates 4-amino-4-deoxy-L-arabinose- phosphoundecaprenol (alpha-L-Ara4N-phosphoundecaprenol) from the cytoplasmic to the periplasmic side of the inner membrane (By similarity)","uniprotId":"ARNE_ECOLI"},"6884966":{"id":6884966,"externalId":"511145.b4546","name":"ypeB","hasFamily":true,"annotation":"predicted protein","uniprotId":"YPEB_ECOLI"},"6884967":{"id":6884967,"externalId":"511145.b4547","name":"ypfN","hasFamily":true,"annotation":"putative membrane protein, UPF0370 family","uniprotId":"YPFN_ECOLI"},"6884968":{"id":6884968,"externalId":"511145.b4548","name":"ypjJ","hasFamily":false,"annotation":"predicted protein","uniprotId":"YPJJ_ECOLI"},"6884969":{"id":6884969,"externalId":"511145.b4550","name":"arfA","hasFamily":true,"annotation":"conserved protein, DUF331 family; Rescues the ribosomes stalled at the 3' end of non-stop mRNAs. May induce hydrolysis of the ribosome-bound peptidyl-tRNA. This activity is crucial when the stalled ribosome can not be rescued by the SsrA(tmRNA)-SmpB quality control system","uniprotId":"ARFA_ECOLI"},"6884970":{"id":6884970,"externalId":"511145.b4551","name":"yheV","hasFamily":true,"annotation":"predicted protein","uniprotId":"YHEV_ECOLI"},"6884971":{"id":6884971,"externalId":"511145.b4553","name":"ysaB","hasFamily":true,"annotation":"predicted protein","uniprotId":"YSAB_ECOLI"},"6884972":{"id":6884972,"externalId":"511145.b4554","name":"yibT","hasFamily":true,"annotation":"predicted protein","uniprotId":"YIBT_ECOLI"},"6884973":{"id":6884973,"externalId":"511145.b4555","name":"yicS","hasFamily":true,"annotation":"putative periplasmic protein","uniprotId":"YICS_ECOLI"},"6884974":{"id":6884974,"externalId":"511145.b4557","name":"yidD","hasFamily":true,"annotation":"conserved protein, UPF0161 family; Could be involved in insertion of integral membrane proteins into the membrane (By similarity)","uniprotId":"YIDD_ECOLI"},"6884975":{"id":6884975,"externalId":"511145.b4558","name":"yifL","hasFamily":true,"annotation":"putative lipoprotein","uniprotId":"YIFL_ECOLI"},"6884976":{"id":6884976,"externalId":"511145.b4559","name":"yjdO","hasFamily":false,"annotation":"predicted protein","uniprotId":"YJDO_ECOLI"},"6884977":{"id":6884977,"externalId":"511145.b4565","name":"sgcB","hasFamily":true,"annotation":"putative enzyme IIB component of PTS; The phosphoenolpyruvate-dependent sugar phosphotransferase system (sugar PTS), a major carbohydrate active -transport system, catalyzes the phosphorylation of incoming sugar substrates concomitantly with their translocation across the cell membrane (By similarity)","uniprotId":"SGCB_ECOLI"},"6884978":{"id":6884978,"externalId":"511145.b4566","name":"yjhX","hasFamily":true,"annotation":"conserved protein","uniprotId":"YJHX_ECOLI"},"6884979":{"id":6884979,"externalId":"511145.b4567","name":"yjjZ","hasFamily":true,"annotation":"predicted protein","uniprotId":"YJJZ_ECOLI"},"6884980":{"id":6884980,"externalId":"511145.b4568","name":"ytjA","hasFamily":false,"annotation":"predicted protein","uniprotId":"YTJA_ECOLI"},"6884981":{"id":6884981,"externalId":"511145.b4586","name":"ykfM","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YKFM_ECOLI"},"6884982":{"id":6884982,"externalId":"511145.b4588","name":"ylcH","hasFamily":false,"annotation":"hypothetical protein, DLP12 prophage","uniprotId":"YLCH_ECOLI"},"6884983":{"id":6884983,"externalId":"511145.b4589","name":"ylcI","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YLCI_ECOLI"},"6884984":{"id":6884984,"externalId":"511145.b4590","name":"ybfK","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YBFK_ECOLI"},"6884985":{"id":6884985,"externalId":"511145.b4592","name":"yccB","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YCCB_ECOLI"},"6884986":{"id":6884986,"externalId":"511145.b4593","name":"ymgI","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YMGI_ECOLI"},"6884987":{"id":6884987,"externalId":"511145.b4594","name":"ymgJ","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YMGJ_ECOLI"},"6884988":{"id":6884988,"externalId":"511145.b4595","name":"yciY","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YCIY_ECOLI"},"6884989":{"id":6884989,"externalId":"511145.b4596","name":"yciZ","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YCIZ_ECOLI"},"6884990":{"id":6884990,"externalId":"511145.b4598","name":"yncL","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YNCL_ECOLI"},"6884991":{"id":6884991,"externalId":"511145.b4599","name":"yneM","hasFamily":false,"annotation":"expressed protein, membrane-associated","uniprotId":"YNEM_ECOLI"},"6884992":{"id":6884992,"externalId":"511145.b4601","name":"ydgU","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YDGU_ECOLI"},"6884993":{"id":6884993,"externalId":"511145.b4602","name":"ynhF","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YNHF_ECOLI"},"6884994":{"id":6884994,"externalId":"511145.b4604","name":"yojO","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YOJO_ECOLI"},"6884995":{"id":6884995,"externalId":"511145.b4606","name":"ypfM","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YPFM_ECOLI"},"6884996":{"id":6884996,"externalId":"511145.b4612","name":"yrhD","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YRHD_ECOLI"},"6884997":{"id":6884997,"externalId":"511145.b4613","name":"dinQ","hasFamily":false,"annotation":"damage inducible protein","uniprotId":"DINQ_ECOLI"},"6884998":{"id":6884998,"externalId":"511145.b4618","name":"tisB","hasFamily":false,"annotation":"toxic membrane persister formation peptide, LexA-regulated; A toxic protein; overexpression causes cessation of growth. Overexpression induces stress-response, a number of membrane protein genes, and leads to cell death. Part of the programmed response to DNA damage; damage leads to increased accumulation of the protein which slows or stops bacterial growth, probably allowing DNA repair before cells continue to grow","uniprotId":"TISB_ECOLI"},"6884999":{"id":6884999,"externalId":"511145.b4620","name":"yjbT","hasFamily":false,"annotation":"hypothetical protein, no homologs","uniprotId":"YJBT_ECOLI"},"6885000":{"id":6885000,"externalId":"511145.b4621","name":"yjbS","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YJBS_ECOLI"},"6885001":{"id":6885001,"externalId":"511145.b4622","name":"ytcA","hasFamily":false,"annotation":"hypothetical protein","uniprotId":"YTCA_ECOLI"},"6885002":{"id":6885002,"externalId":"511145.b4637","name":"uof","hasFamily":false,"annotation":"ryhB-regulated fur leader peptide; Cotranscribed with fur, it is essential for fur translation. The fur ribosomal binding site (RBS) is occluded by the 5'-mRNA secondary structure, which is opened by uof translation","uniprotId":"LPFUR_ECOLI"},"6885003":{"id":6885003,"externalId":"511145.b4655","name":"ythA","hasFamily":false,"annotation":"expressed protein","uniprotId":"YTHA_ECOLI"},"6885004":{"id":6885004,"externalId":"511145.b4662","name":"sgrT","hasFamily":false,"annotation":"Inhibitor of glucose uptake; Acts to promote recovery from glucose-phosphate stress due to intracellular accumulation of glucose-6-phosphate caused by disruption of glycolytic flux or in the presence of (toxic) non- metabolizable glucose phosphate analogs. It may do so by inhibiting the transporter activity for glucose uptake (PtsG) as cells that overexpress this protein do not seem to import glucose although they have nearly wild-type levels of PtsG","uniprotId":"SGRT_ECOLI"},"6885005":{"id":6885005,"externalId":"511145.b4663","name":"azuC","hasFamily":false,"annotation":"expressed protein","uniprotId":"AZUC_ECOLI"},"6885006":{"id":6885006,"externalId":"511145.b4664","name":"ibsD","hasFamily":false,"annotation":"toxic membrane protein"},"6885007":{"id":6885007,"externalId":"511145.b4665","name":"ibsC","hasFamily":false,"annotation":"toxic membrane protein"},"6885008":{"id":6885008,"externalId":"511145.b4666","name":"ibsE","hasFamily":false,"annotation":"toxic membrane protein"},"6885009":{"id":6885009,"externalId":"511145.b4667","name":"ibsA","hasFamily":false,"annotation":"toxic membrane protein"},"6885010":{"id":6885010,"externalId":"511145.b4668","name":"ibsB","hasFamily":false,"annotation":"toxic membrane protein"},"6885011":{"id":6885011,"externalId":"511145.b4669","name":"ilvX","hasFamily":false,"annotation":"expressed protein"},"6885012":{"id":6885012,"externalId":"511145.b4670","name":"yjeV","hasFamily":false,"annotation":"expressed protein","uniprotId":"YJEV_ECOLI"},"6885013":{"id":6885013,"externalId":"511145.b4671","name":"ykgR","hasFamily":false,"annotation":"expressed protein","uniprotId":"YKGR_ECOLI"},"6885014":{"id":6885014,"externalId":"511145.b4672","name":"ymiB","hasFamily":false,"annotation":"expressed protein","uniprotId":"YMIB_ECOLI"},"6885015":{"id":6885015,"externalId":"511145.b4674","name":"ynbG","hasFamily":false,"annotation":"expressed protein","uniprotId":"YNBG_ECOLI"},"6885016":{"id":6885016,"externalId":"511145.b4675","name":"yoaJ","hasFamily":false,"annotation":"expressed protein, membrane-associated","uniprotId":"YOAJ_ECOLI"},"6885017":{"id":6885017,"externalId":"511145.b4676","name":"yoaK","hasFamily":false,"annotation":"expressed protein, membrane-associated","uniprotId":"YOAK_ECOLI"},"6885018":{"id":6885018,"externalId":"511145.b4677","name":"yobI","hasFamily":false,"annotation":"expressed protein","uniprotId":"YOBI_ECOLI"},"6885019":{"id":6885019,"externalId":"511145.b4678","name":"yoeI","hasFamily":false,"annotation":"expressed protein","uniprotId":"YOEI_ECOLI"},"6885020":{"id":6885020,"externalId":"511145.b4679","name":"yohP","hasFamily":false,"annotation":"expressed protein","uniprotId":"YOHP_ECOLI"},"6885021":{"id":6885021,"externalId":"511145.b4680","name":"ypdK","hasFamily":false,"annotation":"expressed protein, membrane-associated","uniprotId":"YPDK_ECOLI"},"6885022":{"id":6885022,"externalId":"511145.b4682","name":"yqcG","hasFamily":false,"annotation":"expressed protein","uniprotId":"YQCG_ECOLI"},"6885023":{"id":6885023,"externalId":"511145.b4683","name":"yqeL","hasFamily":false,"annotation":"expressed protein","uniprotId":"YQEL_ECOLI"},"6885024":{"id":6885024,"externalId":"511145.b4684","name":"yqfG","hasFamily":false,"annotation":"expressed protein","uniprotId":"YQFG_ECOLI"},"6885025":{"id":6885025,"externalId":"511145.b4685","name":"yrbN","hasFamily":false,"annotation":"expressed protein","uniprotId":"YRBN_ECOLI"},"6885026":{"id":6885026,"externalId":"511145.b4686","name":"yshB","hasFamily":false,"annotation":"expressed protein","uniprotId":"YSHB_ECOLI"},"6885027":{"id":6885027,"externalId":"511145.b4687","name":"shoB","hasFamily":false,"annotation":"toxic membrane protein; May be a toxic protein; overexpression causes cessation of growth and rapid membrane depolarization. Overexpression induces stress-response and a number of membrane protein genes","uniprotId":"SHOB_ECOLI"},"6885028":{"id":6885028,"externalId":"511145.b4689","name":"rzoQ","hasFamily":false,"annotation":"probable Rz1-like lipoprotein, Qin prophage","uniprotId":"RZOQ_ECOLI"},"6885029":{"id":6885029,"externalId":"511145.b4702","name":"mgtL","hasFamily":false,"annotation":"regulatory leader peptide for mgtA"},"6885030":{"id":6885030,"externalId":"511145.b4703","name":"pmrR","hasFamily":false,"annotation":"putative regulator of BasS activity, membrane protein; May bind to BasS and modulate its sensor kinase activity (Potential)","uniprotId":"PMRR_ECOLI"}};
module.exports = proteins;